<!DOCTYPE html>
<html lang="en">

  <head>
    <title>CancerFeed</title>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta name="robots" content="noindex, nofollow" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
    <link rel="alternate" type="application/rss+xml" title="CancerFeed" href="feed.atom" />
    <link href="index.css?v1.14.4" rel="stylesheet" />
    <style>
  :root {
    /**
     * Solarized Light
     * Ethan Schoonover (modified by aramisgithub)
     */
    --base00: #fdf6e3;
    --base01: #eee8d5;
    --base02: #93a1a1;
    --base03: #839496;
    --base04: #657b83;
    --base05: #586e75;
    --base06: #073642;
    --base07: #002b36;
    --base08: #dc322f;
    --base09: #cb4b16;
    --base0A: #b58900;
    --base0B: #859900;
    --base0C: #2aa198;
    --base0D: #268bd2;
    --base0E: #6c71c4;
    --base0F: #d33682;

    /**
    * Components
    */
    --source-name-color: var(--base0C);
    /**
   * Typograph
   */
    --font-family-default: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell,
      "Helvetica Neue", sans-serif;
    --font-size-L: 3.6rem;
    --font-size-m: 2.2rem;
    --font-size-s: 1.8rem;
    --font-size-scaler: 52.5%; /* 1rem = 10px */
    --line-height-body: 1.4;
    --line-height-heading: 1.2;

  }
</style>

  </head>

  <body>
    <!-- %after-body-begin.html% -->
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-11-28 datetime="2022-11-28T11:00:00.000Z">2022-11-28</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-11-28 datetime="2022-11-28T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.8&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20221128200439&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36441964"
                        open
                      >
                        <summary class="article-expander__title">Associations of clonal haematopoiesis with recurrent vascular events and death in patients with incident ischemic stroke</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36441964/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221128200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is common among older people and associated with an increased risk of atherosclerosis, inflammation, and shorter overall survival. Age and inflammation are major risk factors for ischemic stroke, yet the association of CH with risk of secondary vascular events and death is unknown. We investigated CH in peripheral blood DNA from 581 patients with first-ever ischemic stroke from the Prospective Cohort with Incident Stroke-Berlin study (PROSCIS-B) using error-corrected...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36439451"
                        open
                      >
                        <summary class="article-expander__title">Towards clinical implementation of circulating tumor DNA in metastatic prostate cancer: Opportunities for integration and pitfalls to interpretation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36439451/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221128200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Plasma circulating tumor DNA (ctDNA) represents short fragments of tumor-derived DNA released into the bloodstream primarily from cancer cells undergoing apoptosis. In metastatic castration-resistant prostate cancer (mCRPC), characterizing genomic alterations in ctDNA identifies mutations, copy number alterations, and structural rearrangements with predictive and prognostic biomarker utility. These associations with clinical outcomes have resulted in ctDNA increasingly incorporated into routine...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221128200439&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.8">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36439408"
                        open
                      >
                        <summary class="article-expander__title">Mutational patterns along different evolution paths of follicular lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36439408/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221128200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Follicular lymphoma (FL) is an indolent disease, characterized by a median life expectancy of 18-20 years and by intermittent periods of relapse and remission. FL frequently transforms into the more aggressive diffuse large B cell lymphoma (t-FL). In previous studies, the analysis of immunoglobulin heavy chain variable region (IgHV) genes in sequential biopsies from the same patient revealed two different patterns of tumor clonal evolution: direct evolution, through acquisition of additional...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-11-27 datetime="2022-11-27T11:00:00.000Z">2022-11-27</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-11-27 datetime="2022-11-27T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.8&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20221128200439&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36437309"
                        open
                      >
                        <summary class="article-expander__title">Transcriptomic profiling of calcified aortic valves in clonal hematopoiesis of indeterminate potential carriers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36437309/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221128200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is characterized by the presence of clones of mutated blood cells without overt blood diseases. In the last few years, it has emerged that CHIP is associated with atherosclerosis and coronary calcification and that it is an independent determinant of cardiovascular mortality. Recently, CHIP has been found to occur frequently in patients with calcific aortic valve disease (CAVD) and it is associated with a poor prognosis after valve...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-11-26 datetime="2022-11-26T11:00:00.000Z">2022-11-26</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-11-26 datetime="2022-11-26T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.8&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20221128200439&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36428749"
                        open
                      >
                        <summary class="article-expander__title">Bone Marrow Immune Microenvironment in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36428749/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221128200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The BM, the major hematopoietic organ in humans, consists of a pleiomorphic environment of cellular, extracellular, and bioactive compounds with continuous and complex interactions between them, leading to the formation of mature blood cells found in the peripheral circulation. Systemic and local inflammation in the BM elicit stress hematopoiesis and drive hematopoietic stem cells (HSCs) out of their quiescent state, as part of a protective pathophysiologic process. However, sustained chronic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36428627"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndromes with Isolated del(5q): Value of Molecular Alterations for Diagnostic and Prognostic Assessment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36428627/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221128200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a group of clonal hematological neoplasms characterized by ineffective hematopoiesis in one or more bone marrow cell lineages. Consequently, patients present with variable degrees of cytopenia and dysplasia. These characteristics constitute the basis for the World Health Organization (WHO) classification criteria of MDS, among other parameters, for the current prognostic scoring system. Although nearly half of newly diagnosed patients present a cytogenetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36428749"
                        open
                      >
                        <summary class="article-expander__title">Bone Marrow Immune Microenvironment in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36428749/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221127200227&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The BM, the major hematopoietic organ in humans, consists of a pleiomorphic environment of cellular, extracellular, and bioactive compounds with continuous and complex interactions between them, leading to the formation of mature blood cells found in the peripheral circulation. Systemic and local inflammation in the BM elicit stress hematopoiesis and drive hematopoietic stem cells (HSCs) out of their quiescent state, as part of a protective pathophysiologic process. However, sustained chronic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36428627"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndromes with Isolated del(5q): Value of Molecular Alterations for Diagnostic and Prognostic Assessment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36428627/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221127200227&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a group of clonal hematological neoplasms characterized by ineffective hematopoiesis in one or more bone marrow cell lineages. Consequently, patients present with variable degrees of cytopenia and dysplasia. These characteristics constitute the basis for the World Health Organization (WHO) classification criteria of MDS, among other parameters, for the current prognostic scoring system. Although nearly half of newly diagnosed patients present a cytogenetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36428749"
                        open
                      >
                        <summary class="article-expander__title">Bone Marrow Immune Microenvironment in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36428749/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221126200943&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The BM, the major hematopoietic organ in humans, consists of a pleiomorphic environment of cellular, extracellular, and bioactive compounds with continuous and complex interactions between them, leading to the formation of mature blood cells found in the peripheral circulation. Systemic and local inflammation in the BM elicit stress hematopoiesis and drive hematopoietic stem cells (HSCs) out of their quiescent state, as part of a protective pathophysiologic process. However, sustained chronic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36428627"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndromes with Isolated del(5q): Value of Molecular Alterations for Diagnostic and Prognostic Assessment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36428627/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221126200943&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a group of clonal hematological neoplasms characterized by ineffective hematopoiesis in one or more bone marrow cell lineages. Consequently, patients present with variable degrees of cytopenia and dysplasia. These characteristics constitute the basis for the World Health Organization (WHO) classification criteria of MDS, among other parameters, for the current prognostic scoring system. Although nearly half of newly diagnosed patients present a cytogenetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221128200439&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.8">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36433937"
                        open
                      >
                        <summary class="article-expander__title">Evaluating cancer etiology and risk with a mathematical model of tumor evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36433937/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221128200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent evidence arising from DNA sequencing of healthy human tissues has clearly indicated that our organs accumulate a relevant number of somatic mutations due to normal endogenous mutational processes, in addition to those caused by environmental factors. A deeper understanding of the evolution of this endogenous mutational load is critical for understanding what causes cancer. Here we present a mathematical model of tumor evolution that is able to predict the expected number of endogenous...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36428782"
                        open
                      >
                        <summary class="article-expander__title">The Impact of Clonal Hierarchy and Heterogeneity on Phenotypic Manifestations of Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36428782/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221128200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Until recently, conventional prognostication of myelodysplastic neoplasms (MDS) was performed using the revised International Prognostic Scoring System (IPSS-R), with additional adverse prognoses conferred by select mutations. Nonetheless, the clonal diversity and dynamics of coexisting mutations have been shown to alter the prognosis and treatment response in patients with MDS. Often in the process of clonal evolution, various initial hits are preferentially followed by a specific spectrum of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36433937"
                        open
                      >
                        <summary class="article-expander__title">Evaluating cancer etiology and risk with a mathematical model of tumor evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36433937/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221127200227&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent evidence arising from DNA sequencing of healthy human tissues has clearly indicated that our organs accumulate a relevant number of somatic mutations due to normal endogenous mutational processes, in addition to those caused by environmental factors. A deeper understanding of the evolution of this endogenous mutational load is critical for understanding what causes cancer. Here we present a mathematical model of tumor evolution that is able to predict the expected number of endogenous...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36428782"
                        open
                      >
                        <summary class="article-expander__title">The Impact of Clonal Hierarchy and Heterogeneity on Phenotypic Manifestations of Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36428782/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221127200227&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Until recently, conventional prognostication of myelodysplastic neoplasms (MDS) was performed using the revised International Prognostic Scoring System (IPSS-R), with additional adverse prognoses conferred by select mutations. Nonetheless, the clonal diversity and dynamics of coexisting mutations have been shown to alter the prognosis and treatment response in patients with MDS. Often in the process of clonal evolution, various initial hits are preferentially followed by a specific spectrum of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36433937"
                        open
                      >
                        <summary class="article-expander__title">Evaluating cancer etiology and risk with a mathematical model of tumor evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36433937/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221126200943&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent evidence arising from DNA sequencing of healthy human tissues has clearly indicated that our organs accumulate a relevant number of somatic mutations due to normal endogenous mutational processes, in addition to those caused by environmental factors. A deeper understanding of the evolution of this endogenous mutational load is critical for understanding what causes cancer. Here we present a mathematical model of tumor evolution that is able to predict the expected number of endogenous...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36428782"
                        open
                      >
                        <summary class="article-expander__title">The Impact of Clonal Hierarchy and Heterogeneity on Phenotypic Manifestations of Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36428782/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221126200943&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Until recently, conventional prognostication of myelodysplastic neoplasms (MDS) was performed using the revised International Prognostic Scoring System (IPSS-R), with additional adverse prognoses conferred by select mutations. Nonetheless, the clonal diversity and dynamics of coexisting mutations have been shown to alter the prognosis and treatment response in patients with MDS. Often in the process of clonal evolution, various initial hits are preferentially followed by a specific spectrum of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-11-25 datetime="2022-11-25T11:00:00.000Z">2022-11-25</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-11-25 datetime="2022-11-25T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.8&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20221128200439&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36426217"
                        open
                      >
                        <summary class="article-expander__title">Radiation therapy induces immunosenescence mediated by p90RSK</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36426217/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221128200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Radiation therapy (RT) to the chest increases the patients&#x27; risk of cardiovascular disease (CVD). A complete understanding of the mechanisms by which RT induces CVD could lead to specific preventive, therapeutic approaches. It is becoming evident that both genotoxic chemotherapy agents and radiation induce mitochondrial dysfunction and cellular senescence. Notably, one of the common phenotypes observed in cancer survivors is accelerated senescence, and immunosenescence is closely related to both...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36426217"
                        open
                      >
                        <summary class="article-expander__title">Radiation therapy induces immunosenescence mediated by p90RSK</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36426217/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221127200227&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Radiation therapy (RT) to the chest increases the patients&#x27; risk of cardiovascular disease (CVD). A complete understanding of the mechanisms by which RT induces CVD could lead to specific preventive, therapeutic approaches. It is becoming evident that both genotoxic chemotherapy agents and radiation induce mitochondrial dysfunction and cellular senescence. Notably, one of the common phenotypes observed in cancer survivors is accelerated senescence, and immunosenescence is closely related to both...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36426217"
                        open
                      >
                        <summary class="article-expander__title">Radiation therapy induces immunosenescence mediated by p90RSK</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36426217/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221126200943&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Radiation therapy (RT) to the chest increases the patients&#x27; risk of cardiovascular disease (CVD). A complete understanding of the mechanisms by which RT induces CVD could lead to specific preventive, therapeutic approaches. It is becoming evident that both genotoxic chemotherapy agents and radiation induce mitochondrial dysfunction and cellular senescence. Notably, one of the common phenotypes observed in cancer survivors is accelerated senescence, and immunosenescence is closely related to both...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36426217"
                        open
                      >
                        <summary class="article-expander__title">Radiation therapy induces immunosenescence mediated by p90RSK</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36426217/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221125200054&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Radiation therapy (RT) to the chest increases the patients&#x27; risk of cardiovascular disease (CVD). A complete understanding of the mechanisms by which RT induces CVD could lead to specific preventive, therapeutic approaches. It is becoming evident that both genotoxic chemotherapy agents and radiation induce mitochondrial dysfunction and cellular senescence. Notably, one of the common phenotypes observed in cancer survivors is accelerated senescence, and immunosenescence is closely related to both...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-11-24 datetime="2022-11-24T11:00:00.000Z">2022-11-24</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-11-24 datetime="2022-11-24T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221128200439&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.8">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36424557"
                        open
                      >
                        <summary class="article-expander__title">Temporal and spatial characteristics of tumor evolution in a mouse model of oral squamous cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36424557/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221128200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Tumor heterogeneity is significant in primary tumor cells and disseminated tumor cells. OSCC cells probably disseminate to lymph nodes in the early stage of oral carcinogenesis. OSCC is characterized by polyclonal dissemination, and the evolutionary trajectory of DTCs is potentially dominated by the tumor microenvironment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36424487"
                        open
                      >
                        <summary class="article-expander__title">Functional analysis of structural variants in single cells using Strand-seq</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36424487/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221128200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic structural variants (SVs) are widespread in cancer, but their impact on disease evolution is understudied due to a lack of methods to directly characterize their functional consequences. We present a computational method, scNOVA, which uses Strand-seq to perform haplotype-aware integration of SV discovery and molecular phenotyping in single cells by using nucleosome occupancy to infer gene expression as a readout. Application to leukemias and cell lines identifies local effects of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36423362"
                        open
                      >
                        <summary class="article-expander__title">COGNITION: a prospective precision oncology trial for patients with early breast cancer at high risk following neoadjuvant chemotherapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36423362/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221128200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: While most precision oncology initiatives focus on metastatic disease, the presented concept offers the opportunity to empower novel therapy options for patients with high-risk early breast cancer in the post-neoadjuvant setting within a biomarker-driven trial and provides the basis to test the value of precision oncology in a curative setting with the overarching goal to increase cure rates.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36424557"
                        open
                      >
                        <summary class="article-expander__title">Temporal and spatial characteristics of tumor evolution in a mouse model of oral squamous cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36424557/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221127200227&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Tumor heterogeneity is significant in primary tumor cells and disseminated tumor cells. OSCC cells probably disseminate to lymph nodes in the early stage of oral carcinogenesis. OSCC is characterized by polyclonal dissemination, and the evolutionary trajectory of DTCs is potentially dominated by the tumor microenvironment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36424487"
                        open
                      >
                        <summary class="article-expander__title">Functional analysis of structural variants in single cells using Strand-seq</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36424487/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221127200227&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic structural variants (SVs) are widespread in cancer, but their impact on disease evolution is understudied due to a lack of methods to directly characterize their functional consequences. We present a computational method, scNOVA, which uses Strand-seq to perform haplotype-aware integration of SV discovery and molecular phenotyping in single cells by using nucleosome occupancy to infer gene expression as a readout. Application to leukemias and cell lines identifies local effects of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36423362"
                        open
                      >
                        <summary class="article-expander__title">COGNITION: a prospective precision oncology trial for patients with early breast cancer at high risk following neoadjuvant chemotherapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36423362/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221127200227&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: While most precision oncology initiatives focus on metastatic disease, the presented concept offers the opportunity to empower novel therapy options for patients with high-risk early breast cancer in the post-neoadjuvant setting within a biomarker-driven trial and provides the basis to test the value of precision oncology in a curative setting with the overarching goal to increase cure rates.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36424557"
                        open
                      >
                        <summary class="article-expander__title">Temporal and spatial characteristics of tumor evolution in a mouse model of oral squamous cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36424557/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221126200943&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Tumor heterogeneity is significant in primary tumor cells and disseminated tumor cells. OSCC cells probably disseminate to lymph nodes in the early stage of oral carcinogenesis. OSCC is characterized by polyclonal dissemination, and the evolutionary trajectory of DTCs is potentially dominated by the tumor microenvironment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36424487"
                        open
                      >
                        <summary class="article-expander__title">Functional analysis of structural variants in single cells using Strand-seq</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36424487/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221126200943&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic structural variants (SVs) are widespread in cancer, but their impact on disease evolution is understudied due to a lack of methods to directly characterize their functional consequences. We present a computational method, scNOVA, which uses Strand-seq to perform haplotype-aware integration of SV discovery and molecular phenotyping in single cells by using nucleosome occupancy to infer gene expression as a readout. Application to leukemias and cell lines identifies local effects of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36423362"
                        open
                      >
                        <summary class="article-expander__title">COGNITION: a prospective precision oncology trial for patients with early breast cancer at high risk following neoadjuvant chemotherapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36423362/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221126200943&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: While most precision oncology initiatives focus on metastatic disease, the presented concept offers the opportunity to empower novel therapy options for patients with high-risk early breast cancer in the post-neoadjuvant setting within a biomarker-driven trial and provides the basis to test the value of precision oncology in a curative setting with the overarching goal to increase cure rates.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36424557"
                        open
                      >
                        <summary class="article-expander__title">Temporal and spatial characteristics of tumor evolution in a mouse model of oral squamous cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36424557/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221125200054&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Tumor heterogeneity is significant in primary tumor cells and disseminated tumor cells. OSCC cells probably disseminate to lymph nodes in the early stage of oral carcinogenesis. OSCC is characterized by polyclonal dissemination, and the evolutionary trajectory of DTCs is potentially dominated by the tumor microenvironment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36424487"
                        open
                      >
                        <summary class="article-expander__title">Functional analysis of structural variants in single cells using Strand-seq</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36424487/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221125200054&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic structural variants (SVs) are widespread in cancer, but their impact on disease evolution is understudied due to a lack of methods to directly characterize their functional consequences. We present a computational method, scNOVA, which uses Strand-seq to perform haplotype-aware integration of SV discovery and molecular phenotyping in single cells by using nucleosome occupancy to infer gene expression as a readout. Application to leukemias and cell lines identifies local effects of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36423362"
                        open
                      >
                        <summary class="article-expander__title">COGNITION: a prospective precision oncology trial for patients with early breast cancer at high risk following neoadjuvant chemotherapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36423362/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221125200054&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: While most precision oncology initiatives focus on metastatic disease, the presented concept offers the opportunity to empower novel therapy options for patients with high-risk early breast cancer in the post-neoadjuvant setting within a biomarker-driven trial and provides the basis to test the value of precision oncology in a curative setting with the overarching goal to increase cure rates.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-11-22 datetime="2022-11-22T11:00:00.000Z">2022-11-22</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-11-22 datetime="2022-11-22T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.8&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20221128200439&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36414227"
                        open
                      >
                        <summary class="article-expander__title">Risk Factors for Clonal Hematopoiesis of Indeterminate Potential and Mosaic Chromosomal Alterations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36414227/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221128200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) and mosaic chromosomal alterations (mCAs) of the autosomes, X, and Y chromosomes are aging-related somatic mutations detectable in peripheral blood. The presence of these acquired mutations predisposes otherwise healthy adults to increased risk of several chronic aging-related conditions including hematologic cancers, atherosclerotic cardiovascular diseases, other inflammatory conditions, and mortality. While the public health impact and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36414227"
                        open
                      >
                        <summary class="article-expander__title">Risk Factors for Clonal Hematopoiesis of Indeterminate Potential and Mosaic Chromosomal Alterations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36414227/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221127200227&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) and mosaic chromosomal alterations (mCAs) of the autosomes, X, and Y chromosomes are aging-related somatic mutations detectable in peripheral blood. The presence of these acquired mutations predisposes otherwise healthy adults to increased risk of several chronic aging-related conditions including hematologic cancers, atherosclerotic cardiovascular diseases, other inflammatory conditions, and mortality. While the public health impact and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36414227"
                        open
                      >
                        <summary class="article-expander__title">Risk Factors for Clonal Hematopoiesis of Indeterminate Potential and Mosaic Chromosomal Alterations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36414227/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221126200943&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) and mosaic chromosomal alterations (mCAs) of the autosomes, X, and Y chromosomes are aging-related somatic mutations detectable in peripheral blood. The presence of these acquired mutations predisposes otherwise healthy adults to increased risk of several chronic aging-related conditions including hematologic cancers, atherosclerotic cardiovascular diseases, other inflammatory conditions, and mortality. While the public health impact and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36414227"
                        open
                      >
                        <summary class="article-expander__title">Risk Factors for Clonal Hematopoiesis of Indeterminate Potential and Mosaic Chromosomal Alterations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36414227/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221125200054&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) and mosaic chromosomal alterations (mCAs) of the autosomes, X, and Y chromosomes are aging-related somatic mutations detectable in peripheral blood. The presence of these acquired mutations predisposes otherwise healthy adults to increased risk of several chronic aging-related conditions including hematologic cancers, atherosclerotic cardiovascular diseases, other inflammatory conditions, and mortality. While the public health impact and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36414227"
                        open
                      >
                        <summary class="article-expander__title">Risk Factors for Clonal Hematopoiesis of Indeterminate Potential and Mosaic Chromosomal Alterations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36414227/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221124200507&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) and mosaic chromosomal alterations (mCAs) of the autosomes, X, and Y chromosomes are aging-related somatic mutations detectable in peripheral blood. The presence of these acquired mutations predisposes otherwise healthy adults to increased risk of several chronic aging-related conditions including hematologic cancers, atherosclerotic cardiovascular diseases, other inflammatory conditions, and mortality. While the public health impact and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36414227"
                        open
                      >
                        <summary class="article-expander__title">Risk Factors for Clonal Hematopoiesis of Indeterminate Potential and Mosaic Chromosomal Alterations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36414227/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221123201010&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) and mosaic chromosomal alterations (mCAs) of the autosomes, X, and Y chromosomes are aging-related somatic mutations detectable in peripheral blood. The presence of these acquired mutations predisposes otherwise healthy adults to increased risk of several chronic aging-related conditions including hematologic cancers, atherosclerotic cardiovascular diseases, other inflammatory conditions, and mortality. While the public health impact and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221128200439&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.8">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36414803"
                        open
                      >
                        <summary class="article-expander__title">Plasma cell neoplasms and related entities-evolution in diagnosis and classification</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36414803/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221128200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Plasma cell neoplasms including multiple myeloma (MM) and related terminally differentiated B-cell neoplasms are characterized by secretion of monoclonal immunoglobulin and stepwise development from a preneoplastic clonal B and/or plasma cell proliferation called monoclonal gammopathy of undetermined significance (MGUS). Diagnosis of these disorders requires integration of clinical, laboratory, and morphological features. While their classification mostly remains unchanged compared to the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36412315"
                        open
                      >
                        <summary class="article-expander__title">Derivative 6 as an additional chromosomal abnormality along with t(15;17): A case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36412315/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221128200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute promyelocytic leukemia (APL) is a type of acute myeloid leukemia (AML) characterized by the presence of t(15;17)(q22;q21) translocation leading to fusion between PML and RARa gene. Treatment combining all-trans retinoic acid (ATRA) and arsenic trioxide (ATO) has dramatically improved the prognosis of APL. We report a rare finding of primary clone of t(15;17) followed by a sequential clonal evolution of additional derivative chromosome 6 formation by a two hit mechanism. Our case showed a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36414803"
                        open
                      >
                        <summary class="article-expander__title">Plasma cell neoplasms and related entities-evolution in diagnosis and classification</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36414803/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221127200227&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Plasma cell neoplasms including multiple myeloma (MM) and related terminally differentiated B-cell neoplasms are characterized by secretion of monoclonal immunoglobulin and stepwise development from a preneoplastic clonal B and/or plasma cell proliferation called monoclonal gammopathy of undetermined significance (MGUS). Diagnosis of these disorders requires integration of clinical, laboratory, and morphological features. While their classification mostly remains unchanged compared to the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36412315"
                        open
                      >
                        <summary class="article-expander__title">Derivative 6 as an additional chromosomal abnormality along with t(15;17): A case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36412315/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221127200227&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute promyelocytic leukemia (APL) is a type of acute myeloid leukemia (AML) characterized by the presence of t(15;17)(q22;q21) translocation leading to fusion between PML and RARa gene. Treatment combining all-trans retinoic acid (ATRA) and arsenic trioxide (ATO) has dramatically improved the prognosis of APL. We report a rare finding of primary clone of t(15;17) followed by a sequential clonal evolution of additional derivative chromosome 6 formation by a two hit mechanism. Our case showed a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36414803"
                        open
                      >
                        <summary class="article-expander__title">Plasma cell neoplasms and related entities-evolution in diagnosis and classification</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36414803/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221126200943&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Plasma cell neoplasms including multiple myeloma (MM) and related terminally differentiated B-cell neoplasms are characterized by secretion of monoclonal immunoglobulin and stepwise development from a preneoplastic clonal B and/or plasma cell proliferation called monoclonal gammopathy of undetermined significance (MGUS). Diagnosis of these disorders requires integration of clinical, laboratory, and morphological features. While their classification mostly remains unchanged compared to the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36412315"
                        open
                      >
                        <summary class="article-expander__title">Derivative 6 as an additional chromosomal abnormality along with t(15;17): A case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36412315/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221126200943&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute promyelocytic leukemia (APL) is a type of acute myeloid leukemia (AML) characterized by the presence of t(15;17)(q22;q21) translocation leading to fusion between PML and RARa gene. Treatment combining all-trans retinoic acid (ATRA) and arsenic trioxide (ATO) has dramatically improved the prognosis of APL. We report a rare finding of primary clone of t(15;17) followed by a sequential clonal evolution of additional derivative chromosome 6 formation by a two hit mechanism. Our case showed a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36414803"
                        open
                      >
                        <summary class="article-expander__title">Plasma cell neoplasms and related entities-evolution in diagnosis and classification</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36414803/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221125200054&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Plasma cell neoplasms including multiple myeloma (MM) and related terminally differentiated B-cell neoplasms are characterized by secretion of monoclonal immunoglobulin and stepwise development from a preneoplastic clonal B and/or plasma cell proliferation called monoclonal gammopathy of undetermined significance (MGUS). Diagnosis of these disorders requires integration of clinical, laboratory, and morphological features. While their classification mostly remains unchanged compared to the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36412315"
                        open
                      >
                        <summary class="article-expander__title">Derivative 6 as an additional chromosomal abnormality along with t(15;17): A case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36412315/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221125200054&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute promyelocytic leukemia (APL) is a type of acute myeloid leukemia (AML) characterized by the presence of t(15;17)(q22;q21) translocation leading to fusion between PML and RARa gene. Treatment combining all-trans retinoic acid (ATRA) and arsenic trioxide (ATO) has dramatically improved the prognosis of APL. We report a rare finding of primary clone of t(15;17) followed by a sequential clonal evolution of additional derivative chromosome 6 formation by a two hit mechanism. Our case showed a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36414803"
                        open
                      >
                        <summary class="article-expander__title">Plasma cell neoplasms and related entities-evolution in diagnosis and classification</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36414803/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221124200507&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Plasma cell neoplasms including multiple myeloma (MM) and related terminally differentiated B-cell neoplasms are characterized by secretion of monoclonal immunoglobulin and stepwise development from a preneoplastic clonal B and/or plasma cell proliferation called monoclonal gammopathy of undetermined significance (MGUS). Diagnosis of these disorders requires integration of clinical, laboratory, and morphological features. While their classification mostly remains unchanged compared to the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36412315"
                        open
                      >
                        <summary class="article-expander__title">Derivative 6 as an additional chromosomal abnormality along with t(15;17): A case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36412315/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221124200507&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute promyelocytic leukemia (APL) is a type of acute myeloid leukemia (AML) characterized by the presence of t(15;17)(q22;q21) translocation leading to fusion between PML and RARa gene. Treatment combining all-trans retinoic acid (ATRA) and arsenic trioxide (ATO) has dramatically improved the prognosis of APL. We report a rare finding of primary clone of t(15;17) followed by a sequential clonal evolution of additional derivative chromosome 6 formation by a two hit mechanism. Our case showed a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36414803"
                        open
                      >
                        <summary class="article-expander__title">Plasma cell neoplasms and related entities-evolution in diagnosis and classification</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36414803/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221123201010&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Plasma cell neoplasms including multiple myeloma (MM) and related terminally differentiated B-cell neoplasms are characterized by secretion of monoclonal immunoglobulin and stepwise development from a preneoplastic clonal B and/or plasma cell proliferation called monoclonal gammopathy of undetermined significance (MGUS). Diagnosis of these disorders requires integration of clinical, laboratory, and morphological features. While their classification mostly remains unchanged compared to the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36412315"
                        open
                      >
                        <summary class="article-expander__title">Derivative 6 as an additional chromosomal abnormality along with t(15;17): A case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36412315/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221123201010&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute promyelocytic leukemia (APL) is a type of acute myeloid leukemia (AML) characterized by the presence of t(15;17)(q22;q21) translocation leading to fusion between PML and RARa gene. Treatment combining all-trans retinoic acid (ATRA) and arsenic trioxide (ATO) has dramatically improved the prognosis of APL. We report a rare finding of primary clone of t(15;17) followed by a sequential clonal evolution of additional derivative chromosome 6 formation by a two hit mechanism. Our case showed a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-11-21 datetime="2022-11-21T11:00:00.000Z">2022-11-21</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-11-21 datetime="2022-11-21T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221128200439&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.8">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36405946"
                        open
                      >
                        <summary class="article-expander__title">Precision medicine in the real-world setting. Clinical activity of talazoparib in a woman with hormone receptor-positive HER2-negative metastatic breast cancer with pathogenic mutation in somatic BRCA2</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36405946/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221128200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Undoubtedly, more information regarding the cost effectiveness of NGS is needed to develop adequate reimbursement policies for this technology. It should be highlighted that the generalisation of MTBs and the implementation of molecular screening programmes are greatly needed in our region to gain more knowledge of somatic mutations implicated in the Hispanic and Latin-American population with BC diagnosis. Recently presented results of randomised studies may support the evaluation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36405946"
                        open
                      >
                        <summary class="article-expander__title">Precision medicine in the real-world setting. Clinical activity of talazoparib in a woman with hormone receptor-positive HER2-negative metastatic breast cancer with pathogenic mutation in somatic BRCA2</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36405946/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221127200227&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Undoubtedly, more information regarding the cost effectiveness of NGS is needed to develop adequate reimbursement policies for this technology. It should be highlighted that the generalisation of MTBs and the implementation of molecular screening programmes are greatly needed in our region to gain more knowledge of somatic mutations implicated in the Hispanic and Latin-American population with BC diagnosis. Recently presented results of randomised studies may support the evaluation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36405946"
                        open
                      >
                        <summary class="article-expander__title">Precision medicine in the real-world setting. Clinical activity of talazoparib in a woman with hormone receptor-positive HER2-negative metastatic breast cancer with pathogenic mutation in somatic BRCA2</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36405946/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221126200943&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Undoubtedly, more information regarding the cost effectiveness of NGS is needed to develop adequate reimbursement policies for this technology. It should be highlighted that the generalisation of MTBs and the implementation of molecular screening programmes are greatly needed in our region to gain more knowledge of somatic mutations implicated in the Hispanic and Latin-American population with BC diagnosis. Recently presented results of randomised studies may support the evaluation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36405946"
                        open
                      >
                        <summary class="article-expander__title">Precision medicine in the real-world setting. Clinical activity of talazoparib in a woman with hormone receptor-positive HER2-negative metastatic breast cancer with pathogenic mutation in somatic BRCA2</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36405946/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221125200054&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Undoubtedly, more information regarding the cost effectiveness of NGS is needed to develop adequate reimbursement policies for this technology. It should be highlighted that the generalisation of MTBs and the implementation of molecular screening programmes are greatly needed in our region to gain more knowledge of somatic mutations implicated in the Hispanic and Latin-American population with BC diagnosis. Recently presented results of randomised studies may support the evaluation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36405946"
                        open
                      >
                        <summary class="article-expander__title">Precision medicine in the real-world setting. Clinical activity of talazoparib in a woman with hormone receptor-positive HER2-negative metastatic breast cancer with pathogenic mutation in somatic BRCA2</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36405946/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221124200507&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Undoubtedly, more information regarding the cost effectiveness of NGS is needed to develop adequate reimbursement policies for this technology. It should be highlighted that the generalisation of MTBs and the implementation of molecular screening programmes are greatly needed in our region to gain more knowledge of somatic mutations implicated in the Hispanic and Latin-American population with BC diagnosis. Recently presented results of randomised studies may support the evaluation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36405946"
                        open
                      >
                        <summary class="article-expander__title">Precision medicine in the real-world setting. Clinical activity of talazoparib in a woman with hormone receptor-positive HER2-negative metastatic breast cancer with pathogenic mutation in somatic BRCA2</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36405946/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221123201010&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Undoubtedly, more information regarding the cost effectiveness of NGS is needed to develop adequate reimbursement policies for this technology. It should be highlighted that the generalisation of MTBs and the implementation of molecular screening programmes are greatly needed in our region to gain more knowledge of somatic mutations implicated in the Hispanic and Latin-American population with BC diagnosis. Recently presented results of randomised studies may support the evaluation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36405946"
                        open
                      >
                        <summary class="article-expander__title">Precision medicine in the real-world setting. Clinical activity of talazoparib in a woman with hormone receptor-positive HER2-negative metastatic breast cancer with pathogenic mutation in somatic BRCA2</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36405946/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221122200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Undoubtedly, more information regarding the cost effectiveness of NGS is needed to develop adequate reimbursement policies for this technology. It should be highlighted that the generalisation of MTBs and the implementation of molecular screening programmes are greatly needed in our region to gain more knowledge of somatic mutations implicated in the Hispanic and Latin-American population with BC diagnosis. Recently presented results of randomised studies may support the evaluation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36405946"
                        open
                      >
                        <summary class="article-expander__title">Precision medicine in the real-world setting. Clinical activity of talazoparib in a woman with hormone receptor-positive HER2-negative metastatic breast cancer with pathogenic mutation in somatic BRCA2</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36405946/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221121201257&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Undoubtedly, more information regarding the cost effectiveness of NGS is needed to develop adequate reimbursement policies for this technology. It should be highlighted that the generalisation of MTBs and the implementation of molecular screening programmes are greatly needed in our region to gain more knowledge of somatic mutations implicated in the Hispanic and Latin-American population with BC diagnosis. Recently presented results of randomised studies may support the evaluation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-11-18 datetime="2022-11-18T11:00:00.000Z">2022-11-18</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-11-18 datetime="2022-11-18T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.8&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20221128200439&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36400843"
                        open
                      >
                        <summary class="article-expander__title">Stem cell homeostasis regulated by hierarchy and neutral competition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36400843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221128200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue stem cells maintain themselves through self-renewal while constantly supplying differentiating cells. Two distinct models have been proposed as mechanisms of stem cell homeostasis. According to the classical model, there is hierarchy among stem cells, and master stem cells produce stem cells by asymmetric division; whereas, according to the recent model, stem cells are equipotent and neutrally compete. However, the mechanism remains controversial in several tissues and species. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36400843"
                        open
                      >
                        <summary class="article-expander__title">Stem cell homeostasis regulated by hierarchy and neutral competition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36400843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221127200227&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue stem cells maintain themselves through self-renewal while constantly supplying differentiating cells. Two distinct models have been proposed as mechanisms of stem cell homeostasis. According to the classical model, there is hierarchy among stem cells, and master stem cells produce stem cells by asymmetric division; whereas, according to the recent model, stem cells are equipotent and neutrally compete. However, the mechanism remains controversial in several tissues and species. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36400843"
                        open
                      >
                        <summary class="article-expander__title">Stem cell homeostasis regulated by hierarchy and neutral competition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36400843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221126200943&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue stem cells maintain themselves through self-renewal while constantly supplying differentiating cells. Two distinct models have been proposed as mechanisms of stem cell homeostasis. According to the classical model, there is hierarchy among stem cells, and master stem cells produce stem cells by asymmetric division; whereas, according to the recent model, stem cells are equipotent and neutrally compete. However, the mechanism remains controversial in several tissues and species. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36400843"
                        open
                      >
                        <summary class="article-expander__title">Stem cell homeostasis regulated by hierarchy and neutral competition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36400843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221125200054&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue stem cells maintain themselves through self-renewal while constantly supplying differentiating cells. Two distinct models have been proposed as mechanisms of stem cell homeostasis. According to the classical model, there is hierarchy among stem cells, and master stem cells produce stem cells by asymmetric division; whereas, according to the recent model, stem cells are equipotent and neutrally compete. However, the mechanism remains controversial in several tissues and species. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36400843"
                        open
                      >
                        <summary class="article-expander__title">Stem cell homeostasis regulated by hierarchy and neutral competition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36400843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221124200507&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue stem cells maintain themselves through self-renewal while constantly supplying differentiating cells. Two distinct models have been proposed as mechanisms of stem cell homeostasis. According to the classical model, there is hierarchy among stem cells, and master stem cells produce stem cells by asymmetric division; whereas, according to the recent model, stem cells are equipotent and neutrally compete. However, the mechanism remains controversial in several tissues and species. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36400843"
                        open
                      >
                        <summary class="article-expander__title">Stem cell homeostasis regulated by hierarchy and neutral competition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36400843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221123201010&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue stem cells maintain themselves through self-renewal while constantly supplying differentiating cells. Two distinct models have been proposed as mechanisms of stem cell homeostasis. According to the classical model, there is hierarchy among stem cells, and master stem cells produce stem cells by asymmetric division; whereas, according to the recent model, stem cells are equipotent and neutrally compete. However, the mechanism remains controversial in several tissues and species. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36400843"
                        open
                      >
                        <summary class="article-expander__title">Stem cell homeostasis regulated by hierarchy and neutral competition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36400843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221122200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue stem cells maintain themselves through self-renewal while constantly supplying differentiating cells. Two distinct models have been proposed as mechanisms of stem cell homeostasis. According to the classical model, there is hierarchy among stem cells, and master stem cells produce stem cells by asymmetric division; whereas, according to the recent model, stem cells are equipotent and neutrally compete. However, the mechanism remains controversial in several tissues and species. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36400843"
                        open
                      >
                        <summary class="article-expander__title">Stem cell homeostasis regulated by hierarchy and neutral competition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36400843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221121201257&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue stem cells maintain themselves through self-renewal while constantly supplying differentiating cells. Two distinct models have been proposed as mechanisms of stem cell homeostasis. According to the classical model, there is hierarchy among stem cells, and master stem cells produce stem cells by asymmetric division; whereas, according to the recent model, stem cells are equipotent and neutrally compete. However, the mechanism remains controversial in several tissues and species. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36400843"
                        open
                      >
                        <summary class="article-expander__title">Stem cell homeostasis regulated by hierarchy and neutral competition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36400843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221120201223&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue stem cells maintain themselves through self-renewal while constantly supplying differentiating cells. Two distinct models have been proposed as mechanisms of stem cell homeostasis. According to the classical model, there is hierarchy among stem cells, and master stem cells produce stem cells by asymmetric division; whereas, according to the recent model, stem cells are equipotent and neutrally compete. However, the mechanism remains controversial in several tissues and species. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36400843"
                        open
                      >
                        <summary class="article-expander__title">Stem cell homeostasis regulated by hierarchy and neutral competition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36400843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221119201545&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue stem cells maintain themselves through self-renewal while constantly supplying differentiating cells. Two distinct models have been proposed as mechanisms of stem cell homeostasis. According to the classical model, there is hierarchy among stem cells, and master stem cells produce stem cells by asymmetric division; whereas, according to the recent model, stem cells are equipotent and neutrally compete. However, the mechanism remains controversial in several tissues and species. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-11-17 datetime="2022-11-17T11:00:00.000Z">2022-11-17</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-11-17 datetime="2022-11-17T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.8&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20221128200439&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36387170"
                        open
                      >
                        <summary class="article-expander__title">Navigating the contested borders between myelodysplastic syndrome and acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36387170/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221128200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome and acute myeloid leukemia are heterogeneous myeloid neoplasms which arise from the accumulation of mutations in a myeloid stem cell or progenitor that confer survival or growth advantages. These disease processes are formally differentiated by clinical, laboratory, and morphological presentations, especially with regard to the preponderance of blasts in the peripheral blood or bone marrow (AML); however, they are closely associated through their shared lineage as well...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36384248"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in diffuse large B-cell lymphoma: clinical impact and genetic relatedness to lymphoma and therapy-related myeloid neoplasm</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36384248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221128200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Not available.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36387170"
                        open
                      >
                        <summary class="article-expander__title">Navigating the contested borders between myelodysplastic syndrome and acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36387170/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221127200227&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome and acute myeloid leukemia are heterogeneous myeloid neoplasms which arise from the accumulation of mutations in a myeloid stem cell or progenitor that confer survival or growth advantages. These disease processes are formally differentiated by clinical, laboratory, and morphological presentations, especially with regard to the preponderance of blasts in the peripheral blood or bone marrow (AML); however, they are closely associated through their shared lineage as well...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36384248"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in diffuse large B-cell lymphoma: clinical impact and genetic relatedness to lymphoma and therapy-related myeloid neoplasm</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36384248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221127200227&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Not available.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36387170"
                        open
                      >
                        <summary class="article-expander__title">Navigating the contested borders between myelodysplastic syndrome and acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36387170/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221126200943&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome and acute myeloid leukemia are heterogeneous myeloid neoplasms which arise from the accumulation of mutations in a myeloid stem cell or progenitor that confer survival or growth advantages. These disease processes are formally differentiated by clinical, laboratory, and morphological presentations, especially with regard to the preponderance of blasts in the peripheral blood or bone marrow (AML); however, they are closely associated through their shared lineage as well...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36384248"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in diffuse large B-cell lymphoma: clinical impact and genetic relatedness to lymphoma and therapy-related myeloid neoplasm</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36384248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221126200943&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Not available.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36387170"
                        open
                      >
                        <summary class="article-expander__title">Navigating the contested borders between myelodysplastic syndrome and acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36387170/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221125200054&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome and acute myeloid leukemia are heterogeneous myeloid neoplasms which arise from the accumulation of mutations in a myeloid stem cell or progenitor that confer survival or growth advantages. These disease processes are formally differentiated by clinical, laboratory, and morphological presentations, especially with regard to the preponderance of blasts in the peripheral blood or bone marrow (AML); however, they are closely associated through their shared lineage as well...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36384248"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in diffuse large B-cell lymphoma: clinical impact and genetic relatedness to lymphoma and therapy-related myeloid neoplasm</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36384248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221125200054&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Not available.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36387170"
                        open
                      >
                        <summary class="article-expander__title">Navigating the contested borders between myelodysplastic syndrome and acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36387170/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221124200507&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome and acute myeloid leukemia are heterogeneous myeloid neoplasms which arise from the accumulation of mutations in a myeloid stem cell or progenitor that confer survival or growth advantages. These disease processes are formally differentiated by clinical, laboratory, and morphological presentations, especially with regard to the preponderance of blasts in the peripheral blood or bone marrow (AML); however, they are closely associated through their shared lineage as well...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36384248"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in diffuse large B-cell lymphoma: clinical impact and genetic relatedness to lymphoma and therapy-related myeloid neoplasm</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36384248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221124200507&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Not available.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36387170"
                        open
                      >
                        <summary class="article-expander__title">Navigating the contested borders between myelodysplastic syndrome and acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36387170/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221123201010&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome and acute myeloid leukemia are heterogeneous myeloid neoplasms which arise from the accumulation of mutations in a myeloid stem cell or progenitor that confer survival or growth advantages. These disease processes are formally differentiated by clinical, laboratory, and morphological presentations, especially with regard to the preponderance of blasts in the peripheral blood or bone marrow (AML); however, they are closely associated through their shared lineage as well...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36384248"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in diffuse large B-cell lymphoma: clinical impact and genetic relatedness to lymphoma and therapy-related myeloid neoplasm</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36384248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221123201010&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Not available.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36387170"
                        open
                      >
                        <summary class="article-expander__title">Navigating the contested borders between myelodysplastic syndrome and acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36387170/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221122200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome and acute myeloid leukemia are heterogeneous myeloid neoplasms which arise from the accumulation of mutations in a myeloid stem cell or progenitor that confer survival or growth advantages. These disease processes are formally differentiated by clinical, laboratory, and morphological presentations, especially with regard to the preponderance of blasts in the peripheral blood or bone marrow (AML); however, they are closely associated through their shared lineage as well...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36384248"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in diffuse large B-cell lymphoma: clinical impact and genetic relatedness to lymphoma and therapy-related myeloid neoplasm</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36384248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221122200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Not available.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36387170"
                        open
                      >
                        <summary class="article-expander__title">Navigating the contested borders between myelodysplastic syndrome and acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36387170/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221121201257&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome and acute myeloid leukemia are heterogeneous myeloid neoplasms which arise from the accumulation of mutations in a myeloid stem cell or progenitor that confer survival or growth advantages. These disease processes are formally differentiated by clinical, laboratory, and morphological presentations, especially with regard to the preponderance of blasts in the peripheral blood or bone marrow (AML); however, they are closely associated through their shared lineage as well...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36384248"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in diffuse large B-cell lymphoma: clinical impact and genetic relatedness to lymphoma and therapy-related myeloid neoplasm</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36384248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221121201257&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Not available.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36387170"
                        open
                      >
                        <summary class="article-expander__title">Navigating the contested borders between myelodysplastic syndrome and acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36387170/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221120201223&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome and acute myeloid leukemia are heterogeneous myeloid neoplasms which arise from the accumulation of mutations in a myeloid stem cell or progenitor that confer survival or growth advantages. These disease processes are formally differentiated by clinical, laboratory, and morphological presentations, especially with regard to the preponderance of blasts in the peripheral blood or bone marrow (AML); however, they are closely associated through their shared lineage as well...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36384248"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in diffuse large B-cell lymphoma: clinical impact and genetic relatedness to lymphoma and therapy-related myeloid neoplasm</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36384248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221120201223&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Not available.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36387170"
                        open
                      >
                        <summary class="article-expander__title">Navigating the contested borders between myelodysplastic syndrome and acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36387170/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221119201545&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome and acute myeloid leukemia are heterogeneous myeloid neoplasms which arise from the accumulation of mutations in a myeloid stem cell or progenitor that confer survival or growth advantages. These disease processes are formally differentiated by clinical, laboratory, and morphological presentations, especially with regard to the preponderance of blasts in the peripheral blood or bone marrow (AML); however, they are closely associated through their shared lineage as well...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36384248"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in diffuse large B-cell lymphoma: clinical impact and genetic relatedness to lymphoma and therapy-related myeloid neoplasm</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36384248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221119201545&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Not available.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36387170"
                        open
                      >
                        <summary class="article-expander__title">Navigating the contested borders between myelodysplastic syndrome and acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36387170/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221118201229&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome and acute myeloid leukemia are heterogeneous myeloid neoplasms which arise from the accumulation of mutations in a myeloid stem cell or progenitor that confer survival or growth advantages. These disease processes are formally differentiated by clinical, laboratory, and morphological presentations, especially with regard to the preponderance of blasts in the peripheral blood or bone marrow (AML); however, they are closely associated through their shared lineage as well...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36384248"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in diffuse large B-cell lymphoma: clinical impact and genetic relatedness to lymphoma and therapy-related myeloid neoplasm</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36384248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221118201229&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Not available.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36387170"
                        open
                      >
                        <summary class="article-expander__title">Navigating the contested borders between myelodysplastic syndrome and acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36387170/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221117201524&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome and acute myeloid leukemia are heterogeneous myeloid neoplasms which arise from the accumulation of mutations in a myeloid stem cell or progenitor that confer survival or growth advantages. These disease processes are formally differentiated by clinical, laboratory, and morphological presentations, especially with regard to the preponderance of blasts in the peripheral blood or bone marrow (AML); however, they are closely associated through their shared lineage as well...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36384248"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in diffuse large B-cell lymphoma: clinical impact and genetic relatedness to lymphoma and therapy-related myeloid neoplasm</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36384248/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221117201524&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Not available.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-11-15 datetime="2022-11-15T11:00:00.000Z">2022-11-15</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-11-15 datetime="2022-11-15T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.8&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20221128200439&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36379023"
                        open
                      >
                        <summary class="article-expander__title">Aging drives Tet2+/- clonal hematopoiesis via IL-1 signaling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36379023/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221128200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP), also referred to as aging-related clonal hematopoiesis (ARCH), is defined as an asymptomatic clonal expansion of mutant mature hematopoietic cells over 4% of blood leukocytes. CHIP associates with advanced age and increased risk for hematological malignancy, cardiovascular disease and all-cause mortality. Loss-of-function somatic mutations in TET2 are frequent drivers of CHIP. However, the contribution of aging-associated cooperating...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36379023"
                        open
                      >
                        <summary class="article-expander__title">Aging drives Tet2+/- clonal hematopoiesis via IL-1 signaling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36379023/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221127200227&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP), also referred to as aging-related clonal hematopoiesis (ARCH), is defined as an asymptomatic clonal expansion of mutant mature hematopoietic cells over 4% of blood leukocytes. CHIP associates with advanced age and increased risk for hematological malignancy, cardiovascular disease and all-cause mortality. Loss-of-function somatic mutations in TET2 are frequent drivers of CHIP. However, the contribution of aging-associated cooperating...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36379023"
                        open
                      >
                        <summary class="article-expander__title">Aging drives Tet2+/- clonal hematopoiesis via IL-1 signaling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36379023/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221126200943&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP), also referred to as aging-related clonal hematopoiesis (ARCH), is defined as an asymptomatic clonal expansion of mutant mature hematopoietic cells over 4% of blood leukocytes. CHIP associates with advanced age and increased risk for hematological malignancy, cardiovascular disease and all-cause mortality. Loss-of-function somatic mutations in TET2 are frequent drivers of CHIP. However, the contribution of aging-associated cooperating...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36379023"
                        open
                      >
                        <summary class="article-expander__title">Aging drives Tet2+/- clonal hematopoiesis via IL-1 signaling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36379023/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221125200054&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP), also referred to as aging-related clonal hematopoiesis (ARCH), is defined as an asymptomatic clonal expansion of mutant mature hematopoietic cells over 4% of blood leukocytes. CHIP associates with advanced age and increased risk for hematological malignancy, cardiovascular disease and all-cause mortality. Loss-of-function somatic mutations in TET2 are frequent drivers of CHIP. However, the contribution of aging-associated cooperating...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36379023"
                        open
                      >
                        <summary class="article-expander__title">Aging drives Tet2+/- clonal hematopoiesis via IL-1 signaling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36379023/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221124200507&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP), also referred to as aging-related clonal hematopoiesis (ARCH), is defined as an asymptomatic clonal expansion of mutant mature hematopoietic cells over 4% of blood leukocytes. CHIP associates with advanced age and increased risk for hematological malignancy, cardiovascular disease and all-cause mortality. Loss-of-function somatic mutations in TET2 are frequent drivers of CHIP. However, the contribution of aging-associated cooperating...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36379023"
                        open
                      >
                        <summary class="article-expander__title">Aging drives Tet2+/- clonal hematopoiesis via IL-1 signaling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36379023/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221123201010&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP), also referred to as aging-related clonal hematopoiesis (ARCH), is defined as an asymptomatic clonal expansion of mutant mature hematopoietic cells over 4% of blood leukocytes. CHIP associates with advanced age and increased risk for hematological malignancy, cardiovascular disease and all-cause mortality. Loss-of-function somatic mutations in TET2 are frequent drivers of CHIP. However, the contribution of aging-associated cooperating...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36379023"
                        open
                      >
                        <summary class="article-expander__title">Aging drives Tet2+/- clonal hematopoiesis via IL-1 signaling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36379023/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221122200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP), also referred to as aging-related clonal hematopoiesis (ARCH), is defined as an asymptomatic clonal expansion of mutant mature hematopoietic cells over 4% of blood leukocytes. CHIP associates with advanced age and increased risk for hematological malignancy, cardiovascular disease and all-cause mortality. Loss-of-function somatic mutations in TET2 are frequent drivers of CHIP. However, the contribution of aging-associated cooperating...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36379023"
                        open
                      >
                        <summary class="article-expander__title">Aging drives Tet2+/- clonal hematopoiesis via IL-1 signaling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36379023/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221121201257&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP), also referred to as aging-related clonal hematopoiesis (ARCH), is defined as an asymptomatic clonal expansion of mutant mature hematopoietic cells over 4% of blood leukocytes. CHIP associates with advanced age and increased risk for hematological malignancy, cardiovascular disease and all-cause mortality. Loss-of-function somatic mutations in TET2 are frequent drivers of CHIP. However, the contribution of aging-associated cooperating...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36379023"
                        open
                      >
                        <summary class="article-expander__title">Aging drives Tet2+/- clonal hematopoiesis via IL-1 signaling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36379023/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221120201223&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP), also referred to as aging-related clonal hematopoiesis (ARCH), is defined as an asymptomatic clonal expansion of mutant mature hematopoietic cells over 4% of blood leukocytes. CHIP associates with advanced age and increased risk for hematological malignancy, cardiovascular disease and all-cause mortality. Loss-of-function somatic mutations in TET2 are frequent drivers of CHIP. However, the contribution of aging-associated cooperating...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36379023"
                        open
                      >
                        <summary class="article-expander__title">Aging drives Tet2+/- clonal hematopoiesis via IL-1 signaling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36379023/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221119201545&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP), also referred to as aging-related clonal hematopoiesis (ARCH), is defined as an asymptomatic clonal expansion of mutant mature hematopoietic cells over 4% of blood leukocytes. CHIP associates with advanced age and increased risk for hematological malignancy, cardiovascular disease and all-cause mortality. Loss-of-function somatic mutations in TET2 are frequent drivers of CHIP. However, the contribution of aging-associated cooperating...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36379023"
                        open
                      >
                        <summary class="article-expander__title">Aging drives Tet2+/- clonal hematopoiesis via IL-1 signaling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36379023/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221118201229&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP), also referred to as aging-related clonal hematopoiesis (ARCH), is defined as an asymptomatic clonal expansion of mutant mature hematopoietic cells over 4% of blood leukocytes. CHIP associates with advanced age and increased risk for hematological malignancy, cardiovascular disease and all-cause mortality. Loss-of-function somatic mutations in TET2 are frequent drivers of CHIP. However, the contribution of aging-associated cooperating...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36379023"
                        open
                      >
                        <summary class="article-expander__title">Aging drives Tet2+/- clonal hematopoiesis via IL-1 signaling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36379023/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221117201524&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP), also referred to as aging-related clonal hematopoiesis (ARCH), is defined as an asymptomatic clonal expansion of mutant mature hematopoietic cells over 4% of blood leukocytes. CHIP associates with advanced age and increased risk for hematological malignancy, cardiovascular disease and all-cause mortality. Loss-of-function somatic mutations in TET2 are frequent drivers of CHIP. However, the contribution of aging-associated cooperating...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36379023"
                        open
                      >
                        <summary class="article-expander__title">Aging drives Tet2+/- &gt;clonal hematopoiesis via IL-1 signaling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36379023/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221116201100&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP), also referred to as aging-related clonal hematopoiesis (ARCH), is defined as an asymptomatic clonal expansion of mutant mature hematopoietic cells over 4% of blood leukocytes. CHIP associates with advanced age and increased risk for hematological malignancy, cardiovascular disease and all-cause mortality. Loss-of-function somatic mutations in TET2 are frequent drivers of CHIP. However, the contribution of aging-associated cooperating...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36379023"
                        open
                      >
                        <summary class="article-expander__title">Aging drives Tet2+/- &gt;clonal hematopoiesis via IL-1 signaling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36379023/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221115201214&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP), also referred to as aging-related clonal hematopoiesis (ARCH), is defined as an asymptomatic clonal expansion of mutant mature hematopoietic cells over 4% of blood leukocytes. CHIP associates with advanced age and increased risk for hematological malignancy, cardiovascular disease and all-cause mortality. Loss-of-function somatic mutations in TET2 are frequent drivers of CHIP. However, the contribution of aging-associated cooperating...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221128200439&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.8">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36379206"
                        open
                      >
                        <summary class="article-expander__title">Loss of p53 and genetic evolution in pancreatic cancer: Ordered chaos after the guardian is gone</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36379206/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221128200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A recent Nature study delineates a stepwise genomic evolution during pancreatic cancer development, employing an engineered mutant Kras and heterozygous Trp53 mouse model that identifies cells undergoing Trp53 loss of heterozygosity (LOH). Genetic progression post-Trp53 LOH involves clonal deletions, then genome doubling and subsequent accumulation of subclonal gains and amplifications.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36376909"
                        open
                      >
                        <summary class="article-expander__title">MEDICC2: whole-genome doubling aware copy-number phylogenies for cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36376909/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221128200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aneuploidy, chromosomal instability, somatic copy-number alterations, and whole-genome doubling (WGD) play key roles in cancer evolution and provide information for the complex task of phylogenetic inference. We present MEDICC2, a method for inferring evolutionary trees and WGD using haplotype-specific somatic copy-number alterations from single-cell or bulk data. MEDICC2 eschews simplifications such as the infinite sites assumption, allowing multiple mutations and parallel evolution, and does...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36379206"
                        open
                      >
                        <summary class="article-expander__title">Loss of p53 and genetic evolution in pancreatic cancer: Ordered chaos after the guardian is gone</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36379206/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221127200227&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A recent Nature study delineates a stepwise genomic evolution during pancreatic cancer development, employing an engineered mutant Kras and heterozygous Trp53 mouse model that identifies cells undergoing Trp53 loss of heterozygosity (LOH). Genetic progression post-Trp53 LOH involves clonal deletions, then genome doubling and subsequent accumulation of subclonal gains and amplifications.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36376909"
                        open
                      >
                        <summary class="article-expander__title">MEDICC2: whole-genome doubling aware copy-number phylogenies for cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36376909/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221127200227&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aneuploidy, chromosomal instability, somatic copy-number alterations, and whole-genome doubling (WGD) play key roles in cancer evolution and provide information for the complex task of phylogenetic inference. We present MEDICC2, a method for inferring evolutionary trees and WGD using haplotype-specific somatic copy-number alterations from single-cell or bulk data. MEDICC2 eschews simplifications such as the infinite sites assumption, allowing multiple mutations and parallel evolution, and does...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36379206"
                        open
                      >
                        <summary class="article-expander__title">Loss of p53 and genetic evolution in pancreatic cancer: Ordered chaos after the guardian is gone</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36379206/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221126200943&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A recent Nature study delineates a stepwise genomic evolution during pancreatic cancer development, employing an engineered mutant Kras and heterozygous Trp53 mouse model that identifies cells undergoing Trp53 loss of heterozygosity (LOH). Genetic progression post-Trp53 LOH involves clonal deletions, then genome doubling and subsequent accumulation of subclonal gains and amplifications.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36376909"
                        open
                      >
                        <summary class="article-expander__title">MEDICC2: whole-genome doubling aware copy-number phylogenies for cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36376909/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221126200943&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aneuploidy, chromosomal instability, somatic copy-number alterations, and whole-genome doubling (WGD) play key roles in cancer evolution and provide information for the complex task of phylogenetic inference. We present MEDICC2, a method for inferring evolutionary trees and WGD using haplotype-specific somatic copy-number alterations from single-cell or bulk data. MEDICC2 eschews simplifications such as the infinite sites assumption, allowing multiple mutations and parallel evolution, and does...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36379206"
                        open
                      >
                        <summary class="article-expander__title">Loss of p53 and genetic evolution in pancreatic cancer: Ordered chaos after the guardian is gone</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36379206/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221125200054&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A recent Nature study delineates a stepwise genomic evolution during pancreatic cancer development, employing an engineered mutant Kras and heterozygous Trp53 mouse model that identifies cells undergoing Trp53 loss of heterozygosity (LOH). Genetic progression post-Trp53 LOH involves clonal deletions, then genome doubling and subsequent accumulation of subclonal gains and amplifications.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36376909"
                        open
                      >
                        <summary class="article-expander__title">MEDICC2: whole-genome doubling aware copy-number phylogenies for cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36376909/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221125200054&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aneuploidy, chromosomal instability, somatic copy-number alterations, and whole-genome doubling (WGD) play key roles in cancer evolution and provide information for the complex task of phylogenetic inference. We present MEDICC2, a method for inferring evolutionary trees and WGD using haplotype-specific somatic copy-number alterations from single-cell or bulk data. MEDICC2 eschews simplifications such as the infinite sites assumption, allowing multiple mutations and parallel evolution, and does...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36379206"
                        open
                      >
                        <summary class="article-expander__title">Loss of p53 and genetic evolution in pancreatic cancer: Ordered chaos after the guardian is gone</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36379206/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221124200507&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A recent Nature study delineates a stepwise genomic evolution during pancreatic cancer development, employing an engineered mutant Kras and heterozygous Trp53 mouse model that identifies cells undergoing Trp53 loss of heterozygosity (LOH). Genetic progression post-Trp53 LOH involves clonal deletions, then genome doubling and subsequent accumulation of subclonal gains and amplifications.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36376909"
                        open
                      >
                        <summary class="article-expander__title">MEDICC2: whole-genome doubling aware copy-number phylogenies for cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36376909/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221124200507&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aneuploidy, chromosomal instability, somatic copy-number alterations, and whole-genome doubling (WGD) play key roles in cancer evolution and provide information for the complex task of phylogenetic inference. We present MEDICC2, a method for inferring evolutionary trees and WGD using haplotype-specific somatic copy-number alterations from single-cell or bulk data. MEDICC2 eschews simplifications such as the infinite sites assumption, allowing multiple mutations and parallel evolution, and does...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36379206"
                        open
                      >
                        <summary class="article-expander__title">Loss of p53 and genetic evolution in pancreatic cancer: Ordered chaos after the guardian is gone</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36379206/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221123201010&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A recent Nature study delineates a stepwise genomic evolution during pancreatic cancer development, employing an engineered mutant Kras and heterozygous Trp53 mouse model that identifies cells undergoing Trp53 loss of heterozygosity (LOH). Genetic progression post-Trp53 LOH involves clonal deletions, then genome doubling and subsequent accumulation of subclonal gains and amplifications.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36376909"
                        open
                      >
                        <summary class="article-expander__title">MEDICC2: whole-genome doubling aware copy-number phylogenies for cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36376909/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221123201010&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aneuploidy, chromosomal instability, somatic copy-number alterations, and whole-genome doubling (WGD) play key roles in cancer evolution and provide information for the complex task of phylogenetic inference. We present MEDICC2, a method for inferring evolutionary trees and WGD using haplotype-specific somatic copy-number alterations from single-cell or bulk data. MEDICC2 eschews simplifications such as the infinite sites assumption, allowing multiple mutations and parallel evolution, and does...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36379206"
                        open
                      >
                        <summary class="article-expander__title">Loss of p53 and genetic evolution in pancreatic cancer: Ordered chaos after the guardian is gone</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36379206/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221122200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A recent Nature study delineates a stepwise genomic evolution during pancreatic cancer development, employing an engineered mutant Kras and heterozygous Trp53 mouse model that identifies cells undergoing Trp53 loss of heterozygosity (LOH). Genetic progression post-Trp53 LOH involves clonal deletions, then genome doubling and subsequent accumulation of subclonal gains and amplifications.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36376909"
                        open
                      >
                        <summary class="article-expander__title">MEDICC2: whole-genome doubling aware copy-number phylogenies for cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36376909/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221122200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aneuploidy, chromosomal instability, somatic copy-number alterations, and whole-genome doubling (WGD) play key roles in cancer evolution and provide information for the complex task of phylogenetic inference. We present MEDICC2, a method for inferring evolutionary trees and WGD using haplotype-specific somatic copy-number alterations from single-cell or bulk data. MEDICC2 eschews simplifications such as the infinite sites assumption, allowing multiple mutations and parallel evolution, and does...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36379206"
                        open
                      >
                        <summary class="article-expander__title">Loss of p53 and genetic evolution in pancreatic cancer: Ordered chaos after the guardian is gone</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36379206/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221121201257&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A recent Nature study delineates a stepwise genomic evolution during pancreatic cancer development, employing an engineered mutant Kras and heterozygous Trp53 mouse model that identifies cells undergoing Trp53 loss of heterozygosity (LOH). Genetic progression post-Trp53 LOH involves clonal deletions, then genome doubling and subsequent accumulation of subclonal gains and amplifications.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36376909"
                        open
                      >
                        <summary class="article-expander__title">MEDICC2: whole-genome doubling aware copy-number phylogenies for cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36376909/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221121201257&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aneuploidy, chromosomal instability, somatic copy-number alterations, and whole-genome doubling (WGD) play key roles in cancer evolution and provide information for the complex task of phylogenetic inference. We present MEDICC2, a method for inferring evolutionary trees and WGD using haplotype-specific somatic copy-number alterations from single-cell or bulk data. MEDICC2 eschews simplifications such as the infinite sites assumption, allowing multiple mutations and parallel evolution, and does...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36379206"
                        open
                      >
                        <summary class="article-expander__title">Loss of p53 and genetic evolution in pancreatic cancer: Ordered chaos after the guardian is gone</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36379206/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221120201223&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A recent Nature study delineates a stepwise genomic evolution during pancreatic cancer development, employing an engineered mutant Kras and heterozygous Trp53 mouse model that identifies cells undergoing Trp53 loss of heterozygosity (LOH). Genetic progression post-Trp53 LOH involves clonal deletions, then genome doubling and subsequent accumulation of subclonal gains and amplifications.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36376909"
                        open
                      >
                        <summary class="article-expander__title">MEDICC2: whole-genome doubling aware copy-number phylogenies for cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36376909/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221120201223&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aneuploidy, chromosomal instability, somatic copy-number alterations, and whole-genome doubling (WGD) play key roles in cancer evolution and provide information for the complex task of phylogenetic inference. We present MEDICC2, a method for inferring evolutionary trees and WGD using haplotype-specific somatic copy-number alterations from single-cell or bulk data. MEDICC2 eschews simplifications such as the infinite sites assumption, allowing multiple mutations and parallel evolution, and does...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36379206"
                        open
                      >
                        <summary class="article-expander__title">Loss of p53 and genetic evolution in pancreatic cancer: Ordered chaos after the guardian is gone</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36379206/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221119201544&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A recent Nature study delineates a stepwise genomic evolution during pancreatic cancer development, employing an engineered mutant Kras and heterozygous Trp53 mouse model that identifies cells undergoing Trp53 loss of heterozygosity (LOH). Genetic progression post-Trp53 LOH involves clonal deletions, then genome doubling and subsequent accumulation of subclonal gains and amplifications.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36376909"
                        open
                      >
                        <summary class="article-expander__title">MEDICC2: whole-genome doubling aware copy-number phylogenies for cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36376909/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221119201544&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aneuploidy, chromosomal instability, somatic copy-number alterations, and whole-genome doubling (WGD) play key roles in cancer evolution and provide information for the complex task of phylogenetic inference. We present MEDICC2, a method for inferring evolutionary trees and WGD using haplotype-specific somatic copy-number alterations from single-cell or bulk data. MEDICC2 eschews simplifications such as the infinite sites assumption, allowing multiple mutations and parallel evolution, and does...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36379206"
                        open
                      >
                        <summary class="article-expander__title">Loss of p53 and genetic evolution in pancreatic cancer: Ordered chaos after the guardian is gone</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36379206/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221118201229&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A recent Nature study delineates a stepwise genomic evolution during pancreatic cancer development, employing an engineered mutant Kras and heterozygous Trp53 mouse model that identifies cells undergoing Trp53 loss of heterozygosity (LOH). Genetic progression post-Trp53 LOH involves clonal deletions, then genome doubling and subsequent accumulation of subclonal gains and amplifications.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36376909"
                        open
                      >
                        <summary class="article-expander__title">MEDICC2: whole-genome doubling aware copy-number phylogenies for cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36376909/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221118201229&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aneuploidy, chromosomal instability, somatic copy-number alterations, and whole-genome doubling (WGD) play key roles in cancer evolution and provide information for the complex task of phylogenetic inference. We present MEDICC2, a method for inferring evolutionary trees and WGD using haplotype-specific somatic copy-number alterations from single-cell or bulk data. MEDICC2 eschews simplifications such as the infinite sites assumption, allowing multiple mutations and parallel evolution, and does...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36376909"
                        open
                      >
                        <summary class="article-expander__title">MEDICC2: whole-genome doubling aware copy-number phylogenies for cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36376909/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221117201522&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aneuploidy, chromosomal instability, somatic copy-number alterations, and whole-genome doubling (WGD) play key roles in cancer evolution and provide information for the complex task of phylogenetic inference. We present MEDICC2, a method for inferring evolutionary trees and WGD using haplotype-specific somatic copy-number alterations from single-cell or bulk data. MEDICC2 eschews simplifications such as the infinite sites assumption, allowing multiple mutations and parallel evolution, and does...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36376909"
                        open
                      >
                        <summary class="article-expander__title">MEDICC2: whole-genome doubling aware copy-number phylogenies for cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36376909/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221116201100&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aneuploidy, chromosomal instability, somatic copy-number alterations, and whole-genome doubling (WGD) play key roles in cancer evolution and provide information for the complex task of phylogenetic inference. We present MEDICC2, a method for inferring evolutionary trees and WGD using haplotype-specific somatic copy-number alterations from single-cell or bulk data. MEDICC2 eschews simplifications such as the infinite sites assumption, allowing multiple mutations and parallel evolution, and does...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36376909"
                        open
                      >
                        <summary class="article-expander__title">MEDICC2: whole-genome doubling aware copy-number phylogenies for cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36376909/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221115201214&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aneuploidy, chromosomal instability, somatic copy-number alterations, and whole-genome doubling (WGD) play key roles in cancer evolution and provide information for the complex task of phylogenetic inference. We present MEDICC2, a method for inferring evolutionary trees and WGD using haplotype-specific somatic copy-number alterations from single-cell or bulk data. MEDICC2 eschews simplifications such as the infinite sites assumption, allowing multiple mutations and parallel evolution, and does...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-11-14 datetime="2022-11-14T11:00:00.000Z">2022-11-14</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-11-14 datetime="2022-11-14T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.8&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20221128200439&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36374003"
                        open
                      >
                        <summary class="article-expander__title">Probing Clonal Hematopoiesis in Liquid Biopsy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36374003/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221128200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Increased use of liquid biopsy to pinpoint actionable genomic alterations in cancer has been accompanied by a rise in the incidental detection of clonal hematopoiesis. Its clinical significance remains unclear, however, as well as how best to risk-stratify patients for further evaluation and, if necessary, treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36374003"
                        open
                      >
                        <summary class="article-expander__title">Probing Clonal Hematopoiesis in Liquid Biopsy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36374003/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221127200227&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Increased use of liquid biopsy to pinpoint actionable genomic alterations in cancer has been accompanied by a rise in the incidental detection of clonal hematopoiesis. Its clinical significance remains unclear, however, as well as how best to risk-stratify patients for further evaluation and, if necessary, treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36374003"
                        open
                      >
                        <summary class="article-expander__title">Probing Clonal Hematopoiesis in Liquid Biopsy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36374003/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221126200943&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Increased use of liquid biopsy to pinpoint actionable genomic alterations in cancer has been accompanied by a rise in the incidental detection of clonal hematopoiesis. Its clinical significance remains unclear, however, as well as how best to risk-stratify patients for further evaluation and, if necessary, treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36374003"
                        open
                      >
                        <summary class="article-expander__title">Probing Clonal Hematopoiesis in Liquid Biopsy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36374003/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221125200054&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Increased use of liquid biopsy to pinpoint actionable genomic alterations in cancer has been accompanied by a rise in the incidental detection of clonal hematopoiesis. Its clinical significance remains unclear, however, as well as how best to risk-stratify patients for further evaluation and, if necessary, treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36374003"
                        open
                      >
                        <summary class="article-expander__title">Probing Clonal Hematopoiesis in Liquid Biopsy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36374003/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221124200507&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Increased use of liquid biopsy to pinpoint actionable genomic alterations in cancer has been accompanied by a rise in the incidental detection of clonal hematopoiesis. Its clinical significance remains unclear, however, as well as how best to risk-stratify patients for further evaluation and, if necessary, treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36374003"
                        open
                      >
                        <summary class="article-expander__title">Probing Clonal Hematopoiesis in Liquid Biopsy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36374003/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221123201010&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Increased use of liquid biopsy to pinpoint actionable genomic alterations in cancer has been accompanied by a rise in the incidental detection of clonal hematopoiesis. Its clinical significance remains unclear, however, as well as how best to risk-stratify patients for further evaluation and, if necessary, treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36374003"
                        open
                      >
                        <summary class="article-expander__title">Probing Clonal Hematopoiesis in Liquid Biopsy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36374003/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221122200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Increased use of liquid biopsy to pinpoint actionable genomic alterations in cancer has been accompanied by a rise in the incidental detection of clonal hematopoiesis. Its clinical significance remains unclear, however, as well as how best to risk-stratify patients for further evaluation and, if necessary, treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36374003"
                        open
                      >
                        <summary class="article-expander__title">Probing Clonal Hematopoiesis in Liquid Biopsy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36374003/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221121201257&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Increased use of liquid biopsy to pinpoint actionable genomic alterations in cancer has been accompanied by a rise in the incidental detection of clonal hematopoiesis. Its clinical significance remains unclear, however, as well as how best to risk-stratify patients for further evaluation and, if necessary, treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36374003"
                        open
                      >
                        <summary class="article-expander__title">Probing Clonal Hematopoiesis in Liquid Biopsy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36374003/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221120201223&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Increased use of liquid biopsy to pinpoint actionable genomic alterations in cancer has been accompanied by a rise in the incidental detection of clonal hematopoiesis. Its clinical significance remains unclear, however, as well as how best to risk-stratify patients for further evaluation and, if necessary, treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36374003"
                        open
                      >
                        <summary class="article-expander__title">Probing Clonal Hematopoiesis in Liquid Biopsy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36374003/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221119201545&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Increased use of liquid biopsy to pinpoint actionable genomic alterations in cancer has been accompanied by a rise in the incidental detection of clonal hematopoiesis. Its clinical significance remains unclear, however, as well as how best to risk-stratify patients for further evaluation and, if necessary, treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36374003"
                        open
                      >
                        <summary class="article-expander__title">Probing Clonal Hematopoiesis in Liquid Biopsy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36374003/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221118201229&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Increased use of liquid biopsy to pinpoint actionable genomic alterations in cancer has been accompanied by a rise in the incidental detection of clonal hematopoiesis. Its clinical significance remains unclear, however, as well as how best to risk-stratify patients for further evaluation and, if necessary, treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36374003"
                        open
                      >
                        <summary class="article-expander__title">Probing Clonal Hematopoiesis in Liquid Biopsy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36374003/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221117201524&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Increased use of liquid biopsy to pinpoint actionable genomic alterations in cancer has been accompanied by a rise in the incidental detection of clonal hematopoiesis. Its clinical significance remains unclear, however, as well as how best to risk-stratify patients for further evaluation and, if necessary, treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36374003"
                        open
                      >
                        <summary class="article-expander__title">Probing Clonal Hematopoiesis in Liquid Biopsy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36374003/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221116201100&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Increased use of liquid biopsy to pinpoint actionable genomic alterations in cancer has been accompanied by a rise in the incidental detection of clonal hematopoiesis. Its clinical significance remains unclear, however, as well as how best to risk-stratify patients for further evaluation and, if necessary, treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36374003"
                        open
                      >
                        <summary class="article-expander__title">Probing Clonal Hematopoiesis in Liquid Biopsy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36374003/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221115201214&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Increased use of liquid biopsy to pinpoint actionable genomic alterations in cancer has been accompanied by a rise in the incidental detection of clonal hematopoiesis. Its clinical significance remains unclear, however, as well as how best to risk-stratify patients for further evaluation and, if necessary, treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36374003"
                        open
                      >
                        <summary class="article-expander__title">Probing Clonal Hematopoiesis in Liquid Biopsy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36374003/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221114201139&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Increased use of liquid biopsy to pinpoint actionable genomic alterations in cancer has been accompanied by a rise in the incidental detection of clonal hematopoiesis. Its clinical significance remains unclear, however, as well as how best to risk-stratify patients for further evaluation and, if necessary, treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221128200439&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.8">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36373660"
                        open
                      >
                        <summary class="article-expander__title">CIViCdb 2022: evolution of an open-access cancer variant interpretation knowledgebase</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36373660/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221128200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CIViC (Clinical Interpretation of Variants in Cancer; civicdb.org) is a crowd-sourced, public domain knowledgebase composed of literature-derived evidence characterizing the clinical utility of cancer variants. As clinical sequencing becomes more prevalent in cancer management, the need for cancer variant interpretation has grown beyond the capability of any single institution. CIViC contains peer-reviewed, published literature curated and expertly-moderated into structured data units (Evidence...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36372873"
                        open
                      >
                        <summary class="article-expander__title">Dynamic monitoring of cerebrospinal fluid circulating tumor DNA to identify unique genetic profiles of brain metastatic tumors and better predict intracranial tumor responses in non-small cell lung cancer patients with brain metastases: a prospective cohort study (GASTO 1028)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36372873/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221128200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CSF ctDNA exhibited unique genetic profiles of brain metastases, and dynamic changes in CSF ctDNA could better predict intracranial tumor responses and track clonal evolution during treatment in NSCLC patients with brain metastases.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36373660"
                        open
                      >
                        <summary class="article-expander__title">CIViCdb 2022: evolution of an open-access cancer variant interpretation knowledgebase</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36373660/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221127200227&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CIViC (Clinical Interpretation of Variants in Cancer; civicdb.org) is a crowd-sourced, public domain knowledgebase composed of literature-derived evidence characterizing the clinical utility of cancer variants. As clinical sequencing becomes more prevalent in cancer management, the need for cancer variant interpretation has grown beyond the capability of any single institution. CIViC contains peer-reviewed, published literature curated and expertly-moderated into structured data units (Evidence...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36372873"
                        open
                      >
                        <summary class="article-expander__title">Dynamic monitoring of cerebrospinal fluid circulating tumor DNA to identify unique genetic profiles of brain metastatic tumors and better predict intracranial tumor responses in non-small cell lung cancer patients with brain metastases: a prospective cohort study (GASTO 1028)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36372873/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221127200227&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CSF ctDNA exhibited unique genetic profiles of brain metastases, and dynamic changes in CSF ctDNA could better predict intracranial tumor responses and track clonal evolution during treatment in NSCLC patients with brain metastases.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36373660"
                        open
                      >
                        <summary class="article-expander__title">CIViCdb 2022: evolution of an open-access cancer variant interpretation knowledgebase</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36373660/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221126200943&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CIViC (Clinical Interpretation of Variants in Cancer; civicdb.org) is a crowd-sourced, public domain knowledgebase composed of literature-derived evidence characterizing the clinical utility of cancer variants. As clinical sequencing becomes more prevalent in cancer management, the need for cancer variant interpretation has grown beyond the capability of any single institution. CIViC contains peer-reviewed, published literature curated and expertly-moderated into structured data units (Evidence...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36372873"
                        open
                      >
                        <summary class="article-expander__title">Dynamic monitoring of cerebrospinal fluid circulating tumor DNA to identify unique genetic profiles of brain metastatic tumors and better predict intracranial tumor responses in non-small cell lung cancer patients with brain metastases: a prospective cohort study (GASTO 1028)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36372873/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221126200943&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CSF ctDNA exhibited unique genetic profiles of brain metastases, and dynamic changes in CSF ctDNA could better predict intracranial tumor responses and track clonal evolution during treatment in NSCLC patients with brain metastases.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36373660"
                        open
                      >
                        <summary class="article-expander__title">CIViCdb 2022: evolution of an open-access cancer variant interpretation knowledgebase</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36373660/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221125200054&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CIViC (Clinical Interpretation of Variants in Cancer; civicdb.org) is a crowd-sourced, public domain knowledgebase composed of literature-derived evidence characterizing the clinical utility of cancer variants. As clinical sequencing becomes more prevalent in cancer management, the need for cancer variant interpretation has grown beyond the capability of any single institution. CIViC contains peer-reviewed, published literature curated and expertly-moderated into structured data units (Evidence...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36372873"
                        open
                      >
                        <summary class="article-expander__title">Dynamic monitoring of cerebrospinal fluid circulating tumor DNA to identify unique genetic profiles of brain metastatic tumors and better predict intracranial tumor responses in non-small cell lung cancer patients with brain metastases: a prospective cohort study (GASTO 1028)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36372873/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221125200054&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CSF ctDNA exhibited unique genetic profiles of brain metastases, and dynamic changes in CSF ctDNA could better predict intracranial tumor responses and track clonal evolution during treatment in NSCLC patients with brain metastases.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36373660"
                        open
                      >
                        <summary class="article-expander__title">CIViCdb 2022: evolution of an open-access cancer variant interpretation knowledgebase</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36373660/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221124200507&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CIViC (Clinical Interpretation of Variants in Cancer; civicdb.org) is a crowd-sourced, public domain knowledgebase composed of literature-derived evidence characterizing the clinical utility of cancer variants. As clinical sequencing becomes more prevalent in cancer management, the need for cancer variant interpretation has grown beyond the capability of any single institution. CIViC contains peer-reviewed, published literature curated and expertly-moderated into structured data units (Evidence...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36372873"
                        open
                      >
                        <summary class="article-expander__title">Dynamic monitoring of cerebrospinal fluid circulating tumor DNA to identify unique genetic profiles of brain metastatic tumors and better predict intracranial tumor responses in non-small cell lung cancer patients with brain metastases: a prospective cohort study (GASTO 1028)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36372873/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221124200507&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CSF ctDNA exhibited unique genetic profiles of brain metastases, and dynamic changes in CSF ctDNA could better predict intracranial tumor responses and track clonal evolution during treatment in NSCLC patients with brain metastases.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36373660"
                        open
                      >
                        <summary class="article-expander__title">CIViCdb 2022: evolution of an open-access cancer variant interpretation knowledgebase</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36373660/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221123201010&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CIViC (Clinical Interpretation of Variants in Cancer; civicdb.org) is a crowd-sourced, public domain knowledgebase composed of literature-derived evidence characterizing the clinical utility of cancer variants. As clinical sequencing becomes more prevalent in cancer management, the need for cancer variant interpretation has grown beyond the capability of any single institution. CIViC contains peer-reviewed, published literature curated and expertly-moderated into structured data units (Evidence...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36372873"
                        open
                      >
                        <summary class="article-expander__title">Dynamic monitoring of cerebrospinal fluid circulating tumor DNA to identify unique genetic profiles of brain metastatic tumors and better predict intracranial tumor responses in non-small cell lung cancer patients with brain metastases: a prospective cohort study (GASTO 1028)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36372873/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221123201010&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CSF ctDNA exhibited unique genetic profiles of brain metastases, and dynamic changes in CSF ctDNA could better predict intracranial tumor responses and track clonal evolution during treatment in NSCLC patients with brain metastases.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36373660"
                        open
                      >
                        <summary class="article-expander__title">CIViCdb 2022: evolution of an open-access cancer variant interpretation knowledgebase</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36373660/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221122200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CIViC (Clinical Interpretation of Variants in Cancer; civicdb.org) is a crowd-sourced, public domain knowledgebase composed of literature-derived evidence characterizing the clinical utility of cancer variants. As clinical sequencing becomes more prevalent in cancer management, the need for cancer variant interpretation has grown beyond the capability of any single institution. CIViC contains peer-reviewed, published literature curated and expertly-moderated into structured data units (Evidence...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36372873"
                        open
                      >
                        <summary class="article-expander__title">Dynamic monitoring of cerebrospinal fluid circulating tumor DNA to identify unique genetic profiles of brain metastatic tumors and better predict intracranial tumor responses in non-small cell lung cancer patients with brain metastases: a prospective cohort study (GASTO 1028)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36372873/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221122200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CSF ctDNA exhibited unique genetic profiles of brain metastases, and dynamic changes in CSF ctDNA could better predict intracranial tumor responses and track clonal evolution during treatment in NSCLC patients with brain metastases.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36373660"
                        open
                      >
                        <summary class="article-expander__title">CIViCdb 2022: evolution of an open-access cancer variant interpretation knowledgebase</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36373660/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221121201257&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CIViC (Clinical Interpretation of Variants in Cancer; civicdb.org) is a crowd-sourced, public domain knowledgebase composed of literature-derived evidence characterizing the clinical utility of cancer variants. As clinical sequencing becomes more prevalent in cancer management, the need for cancer variant interpretation has grown beyond the capability of any single institution. CIViC contains peer-reviewed, published literature curated and expertly-moderated into structured data units (Evidence...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36372873"
                        open
                      >
                        <summary class="article-expander__title">Dynamic monitoring of cerebrospinal fluid circulating tumor DNA to identify unique genetic profiles of brain metastatic tumors and better predict intracranial tumor responses in non-small cell lung cancer patients with brain metastases: a prospective cohort study (GASTO 1028)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36372873/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221121201257&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CSF ctDNA exhibited unique genetic profiles of brain metastases, and dynamic changes in CSF ctDNA could better predict intracranial tumor responses and track clonal evolution during treatment in NSCLC patients with brain metastases.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36373660"
                        open
                      >
                        <summary class="article-expander__title">CIViCdb 2022: evolution of an open-access cancer variant interpretation knowledgebase</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36373660/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221120201223&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CIViC (Clinical Interpretation of Variants in Cancer; civicdb.org) is a crowd-sourced, public domain knowledgebase composed of literature-derived evidence characterizing the clinical utility of cancer variants. As clinical sequencing becomes more prevalent in cancer management, the need for cancer variant interpretation has grown beyond the capability of any single institution. CIViC contains peer-reviewed, published literature curated and expertly-moderated into structured data units (Evidence...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36372873"
                        open
                      >
                        <summary class="article-expander__title">Dynamic monitoring of cerebrospinal fluid circulating tumor DNA to identify unique genetic profiles of brain metastatic tumors and better predict intracranial tumor responses in non-small cell lung cancer patients with brain metastases: a prospective cohort study (GASTO 1028)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36372873/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221120201223&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CSF ctDNA exhibited unique genetic profiles of brain metastases, and dynamic changes in CSF ctDNA could better predict intracranial tumor responses and track clonal evolution during treatment in NSCLC patients with brain metastases.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36373660"
                        open
                      >
                        <summary class="article-expander__title">CIViCdb 2022: evolution of an open-access cancer variant interpretation knowledgebase</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36373660/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221119201544&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CIViC (Clinical Interpretation of Variants in Cancer; civicdb.org) is a crowd-sourced, public domain knowledgebase composed of literature-derived evidence characterizing the clinical utility of cancer variants. As clinical sequencing becomes more prevalent in cancer management, the need for cancer variant interpretation has grown beyond the capability of any single institution. CIViC contains peer-reviewed, published literature curated and expertly-moderated into structured data units (Evidence...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36372873"
                        open
                      >
                        <summary class="article-expander__title">Dynamic monitoring of cerebrospinal fluid circulating tumor DNA to identify unique genetic profiles of brain metastatic tumors and better predict intracranial tumor responses in non-small cell lung cancer patients with brain metastases: a prospective cohort study (GASTO 1028)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36372873/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221119201544&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CSF ctDNA exhibited unique genetic profiles of brain metastases, and dynamic changes in CSF ctDNA could better predict intracranial tumor responses and track clonal evolution during treatment in NSCLC patients with brain metastases.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36373660"
                        open
                      >
                        <summary class="article-expander__title">CIViCdb 2022: evolution of an open-access cancer variant interpretation knowledgebase</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36373660/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221118201229&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CIViC (Clinical Interpretation of Variants in Cancer; civicdb.org) is a crowd-sourced, public domain knowledgebase composed of literature-derived evidence characterizing the clinical utility of cancer variants. As clinical sequencing becomes more prevalent in cancer management, the need for cancer variant interpretation has grown beyond the capability of any single institution. CIViC contains peer-reviewed, published literature curated and expertly-moderated into structured data units (Evidence...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36372873"
                        open
                      >
                        <summary class="article-expander__title">Dynamic monitoring of cerebrospinal fluid circulating tumor DNA to identify unique genetic profiles of brain metastatic tumors and better predict intracranial tumor responses in non-small cell lung cancer patients with brain metastases: a prospective cohort study (GASTO 1028)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36372873/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221118201229&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CSF ctDNA exhibited unique genetic profiles of brain metastases, and dynamic changes in CSF ctDNA could better predict intracranial tumor responses and track clonal evolution during treatment in NSCLC patients with brain metastases.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36373660"
                        open
                      >
                        <summary class="article-expander__title">CIViCdb 2022: evolution of an open-access cancer variant interpretation knowledgebase</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36373660/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221117201522&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CIViC (Clinical Interpretation of Variants in Cancer; civicdb.org) is a crowd-sourced, public domain knowledgebase composed of literature-derived evidence characterizing the clinical utility of cancer variants. As clinical sequencing becomes more prevalent in cancer management, the need for cancer variant interpretation has grown beyond the capability of any single institution. CIViC contains peer-reviewed, published literature curated and expertly-moderated into structured data units (Evidence...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36372873"
                        open
                      >
                        <summary class="article-expander__title">Dynamic monitoring of cerebrospinal fluid circulating tumor DNA to identify unique genetic profiles of brain metastatic tumors and better predict intracranial tumor responses in non-small cell lung cancer patients with brain metastases: a prospective cohort study (GASTO 1028)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36372873/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221117201522&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CSF ctDNA exhibited unique genetic profiles of brain metastases, and dynamic changes in CSF ctDNA could better predict intracranial tumor responses and track clonal evolution during treatment in NSCLC patients with brain metastases.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36373660"
                        open
                      >
                        <summary class="article-expander__title">CIViCdb 2022: evolution of an open-access cancer variant interpretation knowledgebase</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36373660/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221116201100&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CIViC (Clinical Interpretation of Variants in Cancer; civicdb.org) is a crowd-sourced, public domain knowledgebase composed of literature-derived evidence characterizing the clinical utility of cancer variants. As clinical sequencing becomes more prevalent in cancer management, the need for cancer variant interpretation has grown beyond the capability of any single institution. CIViC contains peer-reviewed, published literature curated and expertly-moderated into structured data units (Evidence...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36372873"
                        open
                      >
                        <summary class="article-expander__title">Dynamic monitoring of cerebrospinal fluid circulating tumor DNA to identify unique genetic profiles of brain metastatic tumors and better predict intracranial tumor responses in non-small cell lung cancer patients with brain metastases: a prospective cohort study (GASTO 1028)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36372873/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221116201100&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CSF ctDNA exhibited unique genetic profiles of brain metastases, and dynamic changes in CSF ctDNA could better predict intracranial tumor responses and track clonal evolution during treatment in NSCLC patients with brain metastases.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36373660"
                        open
                      >
                        <summary class="article-expander__title">CIViCdb 2022: evolution of an open-access cancer variant interpretation knowledgebase</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36373660/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221115201214&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CIViC (Clinical Interpretation of Variants in Cancer; civicdb.org) is a crowd-sourced, public domain knowledgebase composed of literature-derived evidence characterizing the clinical utility of cancer variants. As clinical sequencing becomes more prevalent in cancer management, the need for cancer variant interpretation has grown beyond the capability of any single institution. CIViC contains peer-reviewed, published literature curated and expertly-moderated into structured data units (Evidence...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36372873"
                        open
                      >
                        <summary class="article-expander__title">Dynamic monitoring of cerebrospinal fluid circulating tumor DNA to identify unique genetic profiles of brain metastatic tumors and better predict intracranial tumor responses in non-small cell lung cancer patients with brain metastases: a prospective cohort study (GASTO 1028)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36372873/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221115201214&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CSF ctDNA exhibited unique genetic profiles of brain metastases, and dynamic changes in CSF ctDNA could better predict intracranial tumor responses and track clonal evolution during treatment in NSCLC patients with brain metastases.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36373660"
                        open
                      >
                        <summary class="article-expander__title">CIViCdb 2022: evolution of an open-access cancer variant interpretation knowledgebase</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36373660/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221114201139&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CIViC (Clinical Interpretation of Variants in Cancer; civicdb.org) is a crowd-sourced, public domain knowledgebase composed of literature-derived evidence characterizing the clinical utility of cancer variants. As clinical sequencing becomes more prevalent in cancer management, the need for cancer variant interpretation has grown beyond the capability of any single institution. CIViC contains peer-reviewed, published literature curated and expertly-moderated into structured data units (Evidence...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36372873"
                        open
                      >
                        <summary class="article-expander__title">Dynamic monitoring of cerebrospinal fluid circulating tumor DNA to identify unique genetic profiles of brain metastatic tumors and better predict intracranial tumor responses in non-small cell lung cancer patients with brain metastases: a prospective cohort study (GASTO 1028)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36372873/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221114201139&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CSF ctDNA exhibited unique genetic profiles of brain metastases, and dynamic changes in CSF ctDNA could better predict intracranial tumor responses and track clonal evolution during treatment in NSCLC patients with brain metastases.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-11-12 datetime="2022-11-12T11:00:00.000Z">2022-11-12</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-11-12 datetime="2022-11-12T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221128200439&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.8">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36371022"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution and expansion associated with therapy resistance and relapse of colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36371022/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221128200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal cancer (CRC) arises by a continuous process of genetic diversification and clonal evolution. Multiple genes and pathways have a role in tumor initiation and progression. The gradual accumulation of genetic and epigenetic processes leads to the establishment of adenoma and cancer. The important &#x27;driver&#x27; mutations in tumor suppressor genes (such as TP53, APC, and SMAD4) and oncogenes (such as KRAS, NRAS, MET, and PIK3CA) confer selective growth advantages and cause CRC advancement....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36371022"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution and expansion associated with therapy resistance and relapse of colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36371022/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221127200227&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal cancer (CRC) arises by a continuous process of genetic diversification and clonal evolution. Multiple genes and pathways have a role in tumor initiation and progression. The gradual accumulation of genetic and epigenetic processes leads to the establishment of adenoma and cancer. The important &#x27;driver&#x27; mutations in tumor suppressor genes (such as TP53, APC, and SMAD4) and oncogenes (such as KRAS, NRAS, MET, and PIK3CA) confer selective growth advantages and cause CRC advancement....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36371022"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution and expansion associated with therapy resistance and relapse of colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36371022/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221126200943&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal cancer (CRC) arises by a continuous process of genetic diversification and clonal evolution. Multiple genes and pathways have a role in tumor initiation and progression. The gradual accumulation of genetic and epigenetic processes leads to the establishment of adenoma and cancer. The important &#x27;driver&#x27; mutations in tumor suppressor genes (such as TP53, APC, and SMAD4) and oncogenes (such as KRAS, NRAS, MET, and PIK3CA) confer selective growth advantages and cause CRC advancement....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36371022"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution and expansion associated with therapy resistance and relapse of colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36371022/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221125200054&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal cancer (CRC) arises by a continuous process of genetic diversification and clonal evolution. Multiple genes and pathways have a role in tumor initiation and progression. The gradual accumulation of genetic and epigenetic processes leads to the establishment of adenoma and cancer. The important &#x27;driver&#x27; mutations in tumor suppressor genes (such as TP53, APC, and SMAD4) and oncogenes (such as KRAS, NRAS, MET, and PIK3CA) confer selective growth advantages and cause CRC advancement....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36371022"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution and expansion associated with therapy resistance and relapse of Colorectal Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36371022/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221124200507&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal cancer (CRC) arises by a continuous process of genetic diversification and clonal evolution. Multiple genes and pathways have a role in tumor initiation and progression. The gradual accumulation of genetic and epigenetic processes leads to the establishment of adenoma and cancer. The important &#x27;driver&#x27; mutations in tumor suppressor genes (such as TP53, APC, and SMAD4) and oncogenes (such as KRAS, NRAS, MET, and PIK3CA) confer selective growth advantages and cause CRC advancement....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36371022"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution and expansion associated with therapy resistance and relapse of Colorectal Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36371022/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221123201010&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal cancer (CRC) arises by a continuous process of genetic diversification and clonal evolution. Multiple genes and pathways have a role in tumor initiation and progression. The gradual accumulation of genetic and epigenetic processes leads to the establishment of adenoma and cancer. The important &#x27;driver&#x27; mutations in tumor suppressor genes (such as TP53, APC, and SMAD4) and oncogenes (such as KRAS, NRAS, MET, and PIK3CA) confer selective growth advantages and cause CRC advancement....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36371022"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution and expansion associated with therapy resistance and relapse of Colorectal Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36371022/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221122200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal cancer (CRC) arises by a continuous process of genetic diversification and clonal evolution. Multiple genes and pathways have a role in tumor initiation and progression. The gradual accumulation of genetic and epigenetic processes leads to the establishment of adenoma and cancer. The important &#x27;driver&#x27; mutations in tumor suppressor genes (such as TP53, APC, and SMAD4) and oncogenes (such as KRAS, NRAS, MET, and PIK3CA) confer selective growth advantages and cause CRC advancement....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36371022"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution and expansion associated with therapy resistance and relapse of Colorectal Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36371022/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221121201257&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal cancer (CRC) arises by a continuous process of genetic diversification and clonal evolution. Multiple genes and pathways have a role in tumor initiation and progression. The gradual accumulation of genetic and epigenetic processes leads to the establishment of adenoma and cancer. The important &#x27;driver&#x27; mutations in tumor suppressor genes (such as TP53, APC, and SMAD4) and oncogenes (such as KRAS, NRAS, MET, and PIK3CA) confer selective growth advantages and cause CRC advancement....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36371022"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution and expansion associated with therapy resistance and relapse of Colorectal Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36371022/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221120201223&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal cancer (CRC) arises by a continuous process of genetic diversification and clonal evolution. Multiple genes and pathways have a role in tumor initiation and progression. The gradual accumulation of genetic and epigenetic processes leads to the establishment of adenoma and cancer. The important &#x27;driver&#x27; mutations in tumor suppressor genes (such as TP53, APC, and SMAD4) and oncogenes (such as KRAS, NRAS, MET, and PIK3CA) confer selective growth advantages and cause CRC advancement....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36371022"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution and expansion associated with therapy resistance and relapse of Colorectal Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36371022/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221119201544&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal cancer (CRC) arises by a continuous process of genetic diversification and clonal evolution. Multiple genes and pathways have a role in tumor initiation and progression. The gradual accumulation of genetic and epigenetic processes leads to the establishment of adenoma and cancer. The important &#x27;driver&#x27; mutations in tumor suppressor genes (such as TP53, APC, and SMAD4) and oncogenes (such as KRAS, NRAS, MET, and PIK3CA) confer selective growth advantages and cause CRC advancement....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36371022"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution and expansion associated with therapy resistance and relapse of Colorectal Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36371022/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221118201229&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal cancer (CRC) arises by a continuous process of genetic diversification and clonal evolution. Multiple genes and pathways have a role in tumor initiation and progression. The gradual accumulation of genetic and epigenetic processes leads to the establishment of adenoma and cancer. The important &#x27;driver&#x27; mutations in tumor suppressor genes (such as TP53, APC, and SMAD4) and oncogenes (such as KRAS, NRAS, MET, and PIK3CA) confer selective growth advantages and cause CRC advancement....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36371022"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution and expansion associated with therapy resistance and relapse of Colorectal Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36371022/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221117201522&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal cancer (CRC) arises by a continuous process of genetic diversification and clonal evolution. Multiple genes and pathways have a role in tumor initiation and progression. The gradual accumulation of genetic and epigenetic processes leads to the establishment of adenoma and cancer. The important &#x27;driver&#x27; mutations in tumor suppressor genes (such as TP53, APC, and SMAD4) and oncogenes (such as KRAS, NRAS, MET, and PIK3CA) confer selective growth advantages and cause CRC advancement....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36371022"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution and expansion associated with therapy resistance and relapse of Colorectal Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36371022/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221116201100&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal cancer (CRC) arises by a continuous process of genetic diversification and clonal evolution. Multiple genes and pathways have a role in tumor initiation and progression. The gradual accumulation of genetic and epigenetic processes leads to the establishment of adenoma and cancer. The important &#x27;driver&#x27; mutations in tumor suppressor genes (such as TP53, APC, and SMAD4) and oncogenes (such as KRAS, NRAS, MET, and PIK3CA) confer selective growth advantages and cause CRC advancement....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36371022"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution and expansion associated with therapy resistance and relapse of Colorectal Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36371022/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221115201214&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal cancer (CRC) arises by a continuous process of genetic diversification and clonal evolution. Multiple genes and pathways have a role in tumor initiation and progression. The gradual accumulation of genetic and epigenetic processes leads to the establishment of adenoma and cancer. The important &#x27;driver&#x27; mutations in tumor suppressor genes (such as TP53, APC, and SMAD4) and oncogenes (such as KRAS, NRAS, MET, and PIK3CA) confer selective growth advantages and cause CRC advancement....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36371022"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution and expansion associated with therapy resistance and relapse of Colorectal Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36371022/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221114201139&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal cancer (CRC) arises by a continuous process of genetic diversification and clonal evolution. Multiple genes and pathways have a role in tumor initiation and progression. The gradual accumulation of genetic and epigenetic processes leads to the establishment of adenoma and cancer. The important &#x27;driver&#x27; mutations in tumor suppressor genes (such as TP53, APC, and SMAD4) and oncogenes (such as KRAS, NRAS, MET, and PIK3CA) confer selective growth advantages and cause CRC advancement....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36371022"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution and expansion associated with therapy resistance and relapse of Colorectal Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36371022/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221113201202&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal cancer (CRC) arises by a continuous process of genetic diversification and clonal evolution. Multiple genes and pathways have a role in tumor initiation and progression. The gradual accumulation of genetic and epigenetic processes leads to the establishment of adenoma and cancer. The important &#x27;driver&#x27; mutations in tumor suppressor genes (such as TP53, APC, and SMAD4) and oncogenes (such as KRAS, NRAS, MET, and PIK3CA) confer selective growth advantages and cause CRC advancement....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-11-11 datetime="2022-11-11T11:00:00.000Z">2022-11-11</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-11-11 datetime="2022-11-11T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.8&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20221128200439&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36358655"
                        open
                      >
                        <summary class="article-expander__title">LGL Clonal Expansion and Unexplained Cytopenia: Two Clues Don&#x27;t Make an Evidence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36358655/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221128200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal expansions of large granular lymphocytes (LGL) have been reported in a wide spectrum of conditions, with LGL leukemia (LGLL) being the most extreme. However, the boundaries between LGLL and LGL clones are often subtle, and both conditions can be detected in several clinical scenarios, particularly in patients with cytopenias. The intricate overlap of LGL clonal expansion with other disease entities characterized by unexplained cytopenias makes their classification challenging. Indeed,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36358655"
                        open
                      >
                        <summary class="article-expander__title">LGL Clonal Expansion and Unexplained Cytopenia: Two Clues Don&#x27;t Make an Evidence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36358655/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221127200227&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal expansions of large granular lymphocytes (LGL) have been reported in a wide spectrum of conditions, with LGL leukemia (LGLL) being the most extreme. However, the boundaries between LGLL and LGL clones are often subtle, and both conditions can be detected in several clinical scenarios, particularly in patients with cytopenias. The intricate overlap of LGL clonal expansion with other disease entities characterized by unexplained cytopenias makes their classification challenging. Indeed,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36358655"
                        open
                      >
                        <summary class="article-expander__title">LGL Clonal Expansion and Unexplained Cytopenia: Two Clues Don&#x27;t Make an Evidence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36358655/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221126200943&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal expansions of large granular lymphocytes (LGL) have been reported in a wide spectrum of conditions, with LGL leukemia (LGLL) being the most extreme. However, the boundaries between LGLL and LGL clones are often subtle, and both conditions can be detected in several clinical scenarios, particularly in patients with cytopenias. The intricate overlap of LGL clonal expansion with other disease entities characterized by unexplained cytopenias makes their classification challenging. Indeed,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36358655"
                        open
                      >
                        <summary class="article-expander__title">LGL Clonal Expansion and Unexplained Cytopenia: Two Clues Don&#x27;t Make an Evidence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36358655/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221125200054&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal expansions of large granular lymphocytes (LGL) have been reported in a wide spectrum of conditions, with LGL leukemia (LGLL) being the most extreme. However, the boundaries between LGLL and LGL clones are often subtle, and both conditions can be detected in several clinical scenarios, particularly in patients with cytopenias. The intricate overlap of LGL clonal expansion with other disease entities characterized by unexplained cytopenias makes their classification challenging. Indeed,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36358655"
                        open
                      >
                        <summary class="article-expander__title">LGL Clonal Expansion and Unexplained Cytopenia: Two Clues Don&#x27;t Make an Evidence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36358655/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221124200507&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal expansions of large granular lymphocytes (LGL) have been reported in a wide spectrum of conditions, with LGL leukemia (LGLL) being the most extreme. However, the boundaries between LGLL and LGL clones are often subtle, and both conditions can be detected in several clinical scenarios, particularly in patients with cytopenias. The intricate overlap of LGL clonal expansion with other disease entities characterized by unexplained cytopenias makes their classification challenging. Indeed,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36358655"
                        open
                      >
                        <summary class="article-expander__title">LGL Clonal Expansion and Unexplained Cytopenia: Two Clues Don&#x27;t Make an Evidence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36358655/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221123201010&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal expansions of large granular lymphocytes (LGL) have been reported in a wide spectrum of conditions, with LGL leukemia (LGLL) being the most extreme. However, the boundaries between LGLL and LGL clones are often subtle, and both conditions can be detected in several clinical scenarios, particularly in patients with cytopenias. The intricate overlap of LGL clonal expansion with other disease entities characterized by unexplained cytopenias makes their classification challenging. Indeed,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36358655"
                        open
                      >
                        <summary class="article-expander__title">LGL Clonal Expansion and Unexplained Cytopenia: Two Clues Don&#x27;t Make an Evidence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36358655/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221122200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal expansions of large granular lymphocytes (LGL) have been reported in a wide spectrum of conditions, with LGL leukemia (LGLL) being the most extreme. However, the boundaries between LGLL and LGL clones are often subtle, and both conditions can be detected in several clinical scenarios, particularly in patients with cytopenias. The intricate overlap of LGL clonal expansion with other disease entities characterized by unexplained cytopenias makes their classification challenging. Indeed,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36358655"
                        open
                      >
                        <summary class="article-expander__title">LGL Clonal Expansion and Unexplained Cytopenia: Two Clues Don&#x27;t Make an Evidence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36358655/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221121201257&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal expansions of large granular lymphocytes (LGL) have been reported in a wide spectrum of conditions, with LGL leukemia (LGLL) being the most extreme. However, the boundaries between LGLL and LGL clones are often subtle, and both conditions can be detected in several clinical scenarios, particularly in patients with cytopenias. The intricate overlap of LGL clonal expansion with other disease entities characterized by unexplained cytopenias makes their classification challenging. Indeed,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36358655"
                        open
                      >
                        <summary class="article-expander__title">LGL Clonal Expansion and Unexplained Cytopenia: Two Clues Don&#x27;t Make an Evidence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36358655/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221120201223&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal expansions of large granular lymphocytes (LGL) have been reported in a wide spectrum of conditions, with LGL leukemia (LGLL) being the most extreme. However, the boundaries between LGLL and LGL clones are often subtle, and both conditions can be detected in several clinical scenarios, particularly in patients with cytopenias. The intricate overlap of LGL clonal expansion with other disease entities characterized by unexplained cytopenias makes their classification challenging. Indeed,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36358655"
                        open
                      >
                        <summary class="article-expander__title">LGL Clonal Expansion and Unexplained Cytopenia: Two Clues Don&#x27;t Make an Evidence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36358655/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221119201545&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal expansions of large granular lymphocytes (LGL) have been reported in a wide spectrum of conditions, with LGL leukemia (LGLL) being the most extreme. However, the boundaries between LGLL and LGL clones are often subtle, and both conditions can be detected in several clinical scenarios, particularly in patients with cytopenias. The intricate overlap of LGL clonal expansion with other disease entities characterized by unexplained cytopenias makes their classification challenging. Indeed,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36358655"
                        open
                      >
                        <summary class="article-expander__title">LGL Clonal Expansion and Unexplained Cytopenia: Two Clues Don&#x27;t Make an Evidence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36358655/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221118201229&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal expansions of large granular lymphocytes (LGL) have been reported in a wide spectrum of conditions, with LGL leukemia (LGLL) being the most extreme. However, the boundaries between LGLL and LGL clones are often subtle, and both conditions can be detected in several clinical scenarios, particularly in patients with cytopenias. The intricate overlap of LGL clonal expansion with other disease entities characterized by unexplained cytopenias makes their classification challenging. Indeed,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36358655"
                        open
                      >
                        <summary class="article-expander__title">LGL Clonal Expansion and Unexplained Cytopenia: Two Clues Don&#x27;t Make an Evidence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36358655/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221117201524&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal expansions of large granular lymphocytes (LGL) have been reported in a wide spectrum of conditions, with LGL leukemia (LGLL) being the most extreme. However, the boundaries between LGLL and LGL clones are often subtle, and both conditions can be detected in several clinical scenarios, particularly in patients with cytopenias. The intricate overlap of LGL clonal expansion with other disease entities characterized by unexplained cytopenias makes their classification challenging. Indeed,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36358655"
                        open
                      >
                        <summary class="article-expander__title">LGL Clonal Expansion and Unexplained Cytopenia: Two Clues Don&#x27;t Make an Evidence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36358655/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221116201100&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal expansions of large granular lymphocytes (LGL) have been reported in a wide spectrum of conditions, with LGL leukemia (LGLL) being the most extreme. However, the boundaries between LGLL and LGL clones are often subtle, and both conditions can be detected in several clinical scenarios, particularly in patients with cytopenias. The intricate overlap of LGL clonal expansion with other disease entities characterized by unexplained cytopenias makes their classification challenging. Indeed,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36358655"
                        open
                      >
                        <summary class="article-expander__title">LGL Clonal Expansion and Unexplained Cytopenia: Two Clues Don&#x27;t Make an Evidence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36358655/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221115201214&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal expansions of large granular lymphocytes (LGL) have been reported in a wide spectrum of conditions, with LGL leukemia (LGLL) being the most extreme. However, the boundaries between LGLL and LGL clones are often subtle, and both conditions can be detected in several clinical scenarios, particularly in patients with cytopenias. The intricate overlap of LGL clonal expansion with other disease entities characterized by unexplained cytopenias makes their classification challenging. Indeed,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36358655"
                        open
                      >
                        <summary class="article-expander__title">LGL Clonal Expansion and Unexplained Cytopenia: Two Clues Don&#x27;t Make an Evidence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36358655/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221114201139&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal expansions of large granular lymphocytes (LGL) have been reported in a wide spectrum of conditions, with LGL leukemia (LGLL) being the most extreme. However, the boundaries between LGLL and LGL clones are often subtle, and both conditions can be detected in several clinical scenarios, particularly in patients with cytopenias. The intricate overlap of LGL clonal expansion with other disease entities characterized by unexplained cytopenias makes their classification challenging. Indeed,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36358655"
                        open
                      >
                        <summary class="article-expander__title">LGL Clonal Expansion and Unexplained Cytopenia: Two Clues Don&#x27;t Make an Evidence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36358655/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221113201202&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal expansions of large granular lymphocytes (LGL) have been reported in a wide spectrum of conditions, with LGL leukemia (LGLL) being the most extreme. However, the boundaries between LGLL and LGL clones are often subtle, and both conditions can be detected in several clinical scenarios, particularly in patients with cytopenias. The intricate overlap of LGL clonal expansion with other disease entities characterized by unexplained cytopenias makes their classification challenging. Indeed,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36358655"
                        open
                      >
                        <summary class="article-expander__title">LGL Clonal Expansion and Unexplained Cytopenia: Two Clues Don&#x27;t Make an Evidence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36358655/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221112201337&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal expansions of large granular lymphocytes (LGL) have been reported in a wide spectrum of conditions, with LGL leukemia (LGLL) being the most extreme. However, the boundaries between LGLL and LGL clones are often subtle, and both conditions can be detected in several clinical scenarios, particularly in patients with cytopenias. The intricate overlap of LGL clonal expansion with other disease entities characterized by unexplained cytopenias makes their classification challenging. Indeed,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36358655"
                        open
                      >
                        <summary class="article-expander__title">LGL Clonal Expansion and Unexplained Cytopenia: Two Clues Don&#x27;t Make an Evidence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36358655/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221111201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal expansions of large granular lymphocytes (LGL) have been reported in a wide spectrum of conditions, with LGL leukemia (LGLL) being the most extreme. However, the boundaries between LGLL and LGL clones are often subtle, and both conditions can be detected in several clinical scenarios, particularly in patients with cytopenias. The intricate overlap of LGL clonal expansion with other disease entities characterized by unexplained cytopenias makes their classification challenging. Indeed,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221128200439&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.8">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36358353"
                        open
                      >
                        <summary class="article-expander__title">Multiparametric Longitudinal Profiling of RCAS-tva-Induced PDGFB-Driven Experimental Glioma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36358353/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221128200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Glioblastomas are incurable primary brain tumors harboring a heterogeneous landscape of genetic and metabolic alterations. Longitudinal imaging by MRI and [^(18)F]FET-PET measurements enable us to visualize the features of evolving tumors in a dynamic manner. Yet, close-meshed longitudinal imaging time points for characterizing temporal and spatial metabolic alterations during tumor evolution in patients is not feasible because patients usually present with already established tumors. The...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36358353"
                        open
                      >
                        <summary class="article-expander__title">Multiparametric Longitudinal Profiling of RCAS-tva-Induced PDGFB-Driven Experimental Glioma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36358353/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221127200227&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Glioblastomas are incurable primary brain tumors harboring a heterogeneous landscape of genetic and metabolic alterations. Longitudinal imaging by MRI and [^(18)F]FET-PET measurements enable us to visualize the features of evolving tumors in a dynamic manner. Yet, close-meshed longitudinal imaging time points for characterizing temporal and spatial metabolic alterations during tumor evolution in patients is not feasible because patients usually present with already established tumors. The...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36358353"
                        open
                      >
                        <summary class="article-expander__title">Multiparametric Longitudinal Profiling of RCAS-tva-Induced PDGFB-Driven Experimental Glioma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36358353/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221126200943&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Glioblastomas are incurable primary brain tumors harboring a heterogeneous landscape of genetic and metabolic alterations. Longitudinal imaging by MRI and [^(18)F]FET-PET measurements enable us to visualize the features of evolving tumors in a dynamic manner. Yet, close-meshed longitudinal imaging time points for characterizing temporal and spatial metabolic alterations during tumor evolution in patients is not feasible because patients usually present with already established tumors. The...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36358353"
                        open
                      >
                        <summary class="article-expander__title">Multiparametric Longitudinal Profiling of RCAS-tva-Induced PDGFB-Driven Experimental Glioma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36358353/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221125200054&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Glioblastomas are incurable primary brain tumors harboring a heterogeneous landscape of genetic and metabolic alterations. Longitudinal imaging by MRI and [^(18)F]FET-PET measurements enable us to visualize the features of evolving tumors in a dynamic manner. Yet, close-meshed longitudinal imaging time points for characterizing temporal and spatial metabolic alterations during tumor evolution in patients is not feasible because patients usually present with already established tumors. The...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36358353"
                        open
                      >
                        <summary class="article-expander__title">Multiparametric Longitudinal Profiling of RCAS-tva-Induced PDGFB-Driven Experimental Glioma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36358353/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221124200507&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Glioblastomas are incurable primary brain tumors harboring a heterogeneous landscape of genetic and metabolic alterations. Longitudinal imaging by MRI and [^(18)F]FET-PET measurements enable us to visualize the features of evolving tumors in a dynamic manner. Yet, close-meshed longitudinal imaging time points for characterizing temporal and spatial metabolic alterations during tumor evolution in patients is not feasible because patients usually present with already established tumors. The...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36358353"
                        open
                      >
                        <summary class="article-expander__title">Multiparametric Longitudinal Profiling of RCAS-tva-Induced PDGFB-Driven Experimental Glioma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36358353/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221123201010&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Glioblastomas are incurable primary brain tumors harboring a heterogeneous landscape of genetic and metabolic alterations. Longitudinal imaging by MRI and [^(18)F]FET-PET measurements enable us to visualize the features of evolving tumors in a dynamic manner. Yet, close-meshed longitudinal imaging time points for characterizing temporal and spatial metabolic alterations during tumor evolution in patients is not feasible because patients usually present with already established tumors. The...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36358353"
                        open
                      >
                        <summary class="article-expander__title">Multiparametric Longitudinal Profiling of RCAS-tva-Induced PDGFB-Driven Experimental Glioma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36358353/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221122200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Glioblastomas are incurable primary brain tumors harboring a heterogeneous landscape of genetic and metabolic alterations. Longitudinal imaging by MRI and [^(18)F]FET-PET measurements enable us to visualize the features of evolving tumors in a dynamic manner. Yet, close-meshed longitudinal imaging time points for characterizing temporal and spatial metabolic alterations during tumor evolution in patients is not feasible because patients usually present with already established tumors. The...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36358353"
                        open
                      >
                        <summary class="article-expander__title">Multiparametric Longitudinal Profiling of RCAS-tva-Induced PDGFB-Driven Experimental Glioma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36358353/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221121201257&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Glioblastomas are incurable primary brain tumors harboring a heterogeneous landscape of genetic and metabolic alterations. Longitudinal imaging by MRI and [^(18)F]FET-PET measurements enable us to visualize the features of evolving tumors in a dynamic manner. Yet, close-meshed longitudinal imaging time points for characterizing temporal and spatial metabolic alterations during tumor evolution in patients is not feasible because patients usually present with already established tumors. The...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36358353"
                        open
                      >
                        <summary class="article-expander__title">Multiparametric Longitudinal Profiling of RCAS-tva-Induced PDGFB-Driven Experimental Glioma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36358353/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221120201223&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Glioblastomas are incurable primary brain tumors harboring a heterogeneous landscape of genetic and metabolic alterations. Longitudinal imaging by MRI and [^(18)F]FET-PET measurements enable us to visualize the features of evolving tumors in a dynamic manner. Yet, close-meshed longitudinal imaging time points for characterizing temporal and spatial metabolic alterations during tumor evolution in patients is not feasible because patients usually present with already established tumors. The...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36358353"
                        open
                      >
                        <summary class="article-expander__title">Multiparametric Longitudinal Profiling of RCAS-tva-Induced PDGFB-Driven Experimental Glioma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36358353/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221119201544&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Glioblastomas are incurable primary brain tumors harboring a heterogeneous landscape of genetic and metabolic alterations. Longitudinal imaging by MRI and [^(18)F]FET-PET measurements enable us to visualize the features of evolving tumors in a dynamic manner. Yet, close-meshed longitudinal imaging time points for characterizing temporal and spatial metabolic alterations during tumor evolution in patients is not feasible because patients usually present with already established tumors. The...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36358353"
                        open
                      >
                        <summary class="article-expander__title">Multiparametric Longitudinal Profiling of RCAS-tva-Induced PDGFB-Driven Experimental Glioma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36358353/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221118201229&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Glioblastomas are incurable primary brain tumors harboring a heterogeneous landscape of genetic and metabolic alterations. Longitudinal imaging by MRI and [^(18)F]FET-PET measurements enable us to visualize the features of evolving tumors in a dynamic manner. Yet, close-meshed longitudinal imaging time points for characterizing temporal and spatial metabolic alterations during tumor evolution in patients is not feasible because patients usually present with already established tumors. The...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36358353"
                        open
                      >
                        <summary class="article-expander__title">Multiparametric Longitudinal Profiling of RCAS-tva-Induced PDGFB-Driven Experimental Glioma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36358353/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221117201522&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Glioblastomas are incurable primary brain tumors harboring a heterogeneous landscape of genetic and metabolic alterations. Longitudinal imaging by MRI and [^(18)F]FET-PET measurements enable us to visualize the features of evolving tumors in a dynamic manner. Yet, close-meshed longitudinal imaging time points for characterizing temporal and spatial metabolic alterations during tumor evolution in patients is not feasible because patients usually present with already established tumors. The...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36358353"
                        open
                      >
                        <summary class="article-expander__title">Multiparametric Longitudinal Profiling of RCAS-tva-Induced PDGFB-Driven Experimental Glioma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36358353/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221116201100&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Glioblastomas are incurable primary brain tumors harboring a heterogeneous landscape of genetic and metabolic alterations. Longitudinal imaging by MRI and [^(18)F]FET-PET measurements enable us to visualize the features of evolving tumors in a dynamic manner. Yet, close-meshed longitudinal imaging time points for characterizing temporal and spatial metabolic alterations during tumor evolution in patients is not feasible because patients usually present with already established tumors. The...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36358353"
                        open
                      >
                        <summary class="article-expander__title">Multiparametric Longitudinal Profiling of RCAS-tva-Induced PDGFB-Driven Experimental Glioma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36358353/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221115201214&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Glioblastomas are incurable primary brain tumors harboring a heterogeneous landscape of genetic and metabolic alterations. Longitudinal imaging by MRI and [^(18)F]FET-PET measurements enable us to visualize the features of evolving tumors in a dynamic manner. Yet, close-meshed longitudinal imaging time points for characterizing temporal and spatial metabolic alterations during tumor evolution in patients is not feasible because patients usually present with already established tumors. The...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36358353"
                        open
                      >
                        <summary class="article-expander__title">Multiparametric Longitudinal Profiling of RCAS-tva-Induced PDGFB-Driven Experimental Glioma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36358353/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221114201139&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Glioblastomas are incurable primary brain tumors harboring a heterogeneous landscape of genetic and metabolic alterations. Longitudinal imaging by MRI and [^(18)F]FET-PET measurements enable us to visualize the features of evolving tumors in a dynamic manner. Yet, close-meshed longitudinal imaging time points for characterizing temporal and spatial metabolic alterations during tumor evolution in patients is not feasible because patients usually present with already established tumors. The...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36358353"
                        open
                      >
                        <summary class="article-expander__title">Multiparametric Longitudinal Profiling of RCAS-tva-Induced PDGFB-Driven Experimental Glioma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36358353/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221113201202&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Glioblastomas are incurable primary brain tumors harboring a heterogeneous landscape of genetic and metabolic alterations. Longitudinal imaging by MRI and [^(18)F]FET-PET measurements enable us to visualize the features of evolving tumors in a dynamic manner. Yet, close-meshed longitudinal imaging time points for characterizing temporal and spatial metabolic alterations during tumor evolution in patients is not feasible because patients usually present with already established tumors. The...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36358353"
                        open
                      >
                        <summary class="article-expander__title">Multiparametric Longitudinal Profiling of RCAS-tva-Induced PDGFB-Driven Experimental Glioma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36358353/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221112201337&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Glioblastomas are incurable primary brain tumors harboring a heterogeneous landscape of genetic and metabolic alterations. Longitudinal imaging by MRI and [^(18)F]FET-PET measurements enable us to visualize the features of evolving tumors in a dynamic manner. Yet, close-meshed longitudinal imaging time points for characterizing temporal and spatial metabolic alterations during tumor evolution in patients is not feasible because patients usually present with already established tumors. The...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36358353"
                        open
                      >
                        <summary class="article-expander__title">Multiparametric Longitudinal Profiling of RCAS-tva-Induced PDGFB-Driven Experimental Glioma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36358353/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221111201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Glioblastomas are incurable primary brain tumors harboring a heterogeneous landscape of genetic and metabolic alterations. Longitudinal imaging by MRI and [^(18)F]FET-PET measurements enable us to visualize the features of evolving tumors in a dynamic manner. Yet, close-meshed longitudinal imaging time points for characterizing temporal and spatial metabolic alterations during tumor evolution in patients is not feasible because patients usually present with already established tumors. The...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-11-10 datetime="2022-11-10T11:00:00.000Z">2022-11-10</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-11-10 datetime="2022-11-10T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.8&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20221128200439&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36355225"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and cardiovascular disease: deciphering interconnections</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36355225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221128200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiovascular and oncological diseases represent the global major causes of death. For both, a novel and far-reaching risk factor has been identified: clonal hematopoiesis (CH). CH is defined as clonal expansion of peripheral blood cells on the basis of somatic mutations, without overt hematological malignancy. The most commonly affected genes are TET2, DNMT3A, ASXL1 and JAK2. By the age of 70, at least 20-50% of all individuals carry a CH clone, conveying a striking clinical impact by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36353993"
                        open
                      >
                        <summary class="article-expander__title">2022 Jeffrey M. Hoeg Award Lecture: Genomic Aging, Clonal Hematopoiesis, and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36353993/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221128200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronologic age is the dominant risk factor for coronary artery disease but the features of aging promoting coronary artery disease are poorly understood. Advances in human genetics and population-based genetic profiling of blood cells have uncovered the surprising role of age-related subclinical leukemogenic mutations in blood cells, termed &quot;clonal hematopoiesis of indeterminate potential,&quot; in coronary artery disease. Such mutations typically occur in DNMT3A, TET2, ASXL1, and JAK2. Murine and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36355225"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and cardiovascular disease: deciphering interconnections</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36355225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221127200227&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiovascular and oncological diseases represent the global major causes of death. For both, a novel and far-reaching risk factor has been identified: clonal hematopoiesis (CH). CH is defined as clonal expansion of peripheral blood cells on the basis of somatic mutations, without overt hematological malignancy. The most commonly affected genes are TET2, DNMT3A, ASXL1 and JAK2. By the age of 70, at least 20-50% of all individuals carry a CH clone, conveying a striking clinical impact by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36353993"
                        open
                      >
                        <summary class="article-expander__title">2022 Jeffrey M. Hoeg Award Lecture: Genomic Aging, Clonal Hematopoiesis, and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36353993/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221127200227&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronologic age is the dominant risk factor for coronary artery disease but the features of aging promoting coronary artery disease are poorly understood. Advances in human genetics and population-based genetic profiling of blood cells have uncovered the surprising role of age-related subclinical leukemogenic mutations in blood cells, termed &quot;clonal hematopoiesis of indeterminate potential,&quot; in coronary artery disease. Such mutations typically occur in DNMT3A, TET2, ASXL1, and JAK2. Murine and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36353970"
                        open
                      >
                        <summary class="article-expander__title">Recurrent missense variants in clonal hematopoiesis-related genes present in the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36353970/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221127200227&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) consists in an abnormal expansion of a hematopoietic stem cell bearing an advantageous somatic variant. A survey of known recurrent somatic missense variants in DNMT3A, SF3B1, SRSF2 and TP53, some of the most prominent genes underlying CH of indeterminate potential (CHIP), in gnomAD non-cancer database shows the presence of 73 variants. Many of them reach frequencies higher than 0.01% in various populations and, in many cases, are enriched in specific populations....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36355225"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and cardiovascular disease: deciphering interconnections</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36355225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221126200943&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiovascular and oncological diseases represent the global major causes of death. For both, a novel and far-reaching risk factor has been identified: clonal hematopoiesis (CH). CH is defined as clonal expansion of peripheral blood cells on the basis of somatic mutations, without overt hematological malignancy. The most commonly affected genes are TET2, DNMT3A, ASXL1 and JAK2. By the age of 70, at least 20-50% of all individuals carry a CH clone, conveying a striking clinical impact by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36353993"
                        open
                      >
                        <summary class="article-expander__title">2022 Jeffrey M. Hoeg Award Lecture: Genomic Aging, Clonal Hematopoiesis, and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36353993/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221126200943&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronologic age is the dominant risk factor for coronary artery disease but the features of aging promoting coronary artery disease are poorly understood. Advances in human genetics and population-based genetic profiling of blood cells have uncovered the surprising role of age-related subclinical leukemogenic mutations in blood cells, termed &quot;clonal hematopoiesis of indeterminate potential,&quot; in coronary artery disease. Such mutations typically occur in DNMT3A, TET2, ASXL1, and JAK2. Murine and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36353970"
                        open
                      >
                        <summary class="article-expander__title">Recurrent missense variants in clonal hematopoiesis-related genes present in the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36353970/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221126200943&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) consists in an abnormal expansion of a hematopoietic stem cell bearing an advantageous somatic variant. A survey of known recurrent somatic missense variants in DNMT3A, SF3B1, SRSF2 and TP53, some of the most prominent genes underlying CH of indeterminate potential (CHIP), in gnomAD non-cancer database shows the presence of 73 variants. Many of them reach frequencies higher than 0.01% in various populations and, in many cases, are enriched in specific populations....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36355225"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and cardiovascular disease: deciphering interconnections</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36355225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221125200054&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiovascular and oncological diseases represent the global major causes of death. For both, a novel and far-reaching risk factor has been identified: clonal hematopoiesis (CH). CH is defined as clonal expansion of peripheral blood cells on the basis of somatic mutations, without overt hematological malignancy. The most commonly affected genes are TET2, DNMT3A, ASXL1 and JAK2. By the age of 70, at least 20-50% of all individuals carry a CH clone, conveying a striking clinical impact by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36353993"
                        open
                      >
                        <summary class="article-expander__title">2022 Jeffrey M. Hoeg Award Lecture: Genomic Aging, Clonal Hematopoiesis, and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36353993/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221125200054&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronologic age is the dominant risk factor for coronary artery disease but the features of aging promoting coronary artery disease are poorly understood. Advances in human genetics and population-based genetic profiling of blood cells have uncovered the surprising role of age-related subclinical leukemogenic mutations in blood cells, termed &quot;clonal hematopoiesis of indeterminate potential,&quot; in coronary artery disease. Such mutations typically occur in DNMT3A, TET2, ASXL1, and JAK2. Murine and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36353970"
                        open
                      >
                        <summary class="article-expander__title">Recurrent missense variants in clonal hematopoiesis-related genes present in the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36353970/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221125200054&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) consists in an abnormal expansion of a hematopoietic stem cell bearing an advantageous somatic variant. A survey of known recurrent somatic missense variants in DNMT3A, SF3B1, SRSF2 and TP53, some of the most prominent genes underlying CH of indeterminate potential (CHIP), in gnomAD non-cancer database shows the presence of 73 variants. Many of them reach frequencies higher than 0.01% in various populations and, in many cases, are enriched in specific populations....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36355225"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and cardiovascular disease: deciphering interconnections</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36355225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221124200507&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiovascular and oncological diseases represent the global major causes of death. For both, a novel and far-reaching risk factor has been identified: clonal hematopoiesis (CH). CH is defined as clonal expansion of peripheral blood cells on the basis of somatic mutations, without overt hematological malignancy. The most commonly affected genes are TET2, DNMT3A, ASXL1 and JAK2. By the age of 70, at least 20-50% of all individuals carry a CH clone, conveying a striking clinical impact by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36353993"
                        open
                      >
                        <summary class="article-expander__title">2022 Jeffrey M. Hoeg Award Lecture: Genomic Aging, Clonal Hematopoiesis, and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36353993/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221124200507&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronologic age is the dominant risk factor for coronary artery disease but the features of aging promoting coronary artery disease are poorly understood. Advances in human genetics and population-based genetic profiling of blood cells have uncovered the surprising role of age-related subclinical leukemogenic mutations in blood cells, termed &quot;clonal hematopoiesis of indeterminate potential,&quot; in coronary artery disease. Such mutations typically occur in DNMT3A, TET2, ASXL1, and JAK2. Murine and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36353970"
                        open
                      >
                        <summary class="article-expander__title">Recurrent missense variants in clonal hematopoiesis-related genes present in the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36353970/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221124200507&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) consists in an abnormal expansion of a hematopoietic stem cell bearing an advantageous somatic variant. A survey of known recurrent somatic missense variants in DNMT3A, SF3B1, SRSF2 and TP53, some of the most prominent genes underlying CH of indeterminate potential (CHIP), in gnomAD non-cancer database shows the presence of 73 variants. Many of them reach frequencies higher than 0.01% in various populations and, in many cases, are enriched in specific populations....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36355225"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and cardiovascular disease: deciphering interconnections</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36355225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221123201010&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiovascular and oncological diseases represent the global major causes of death. For both, a novel and far-reaching risk factor has been identified: clonal hematopoiesis (CH). CH is defined as clonal expansion of peripheral blood cells on the basis of somatic mutations, without overt hematological malignancy. The most commonly affected genes are TET2, DNMT3A, ASXL1 and JAK2. By the age of 70, at least 20-50% of all individuals carry a CH clone, conveying a striking clinical impact by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36353993"
                        open
                      >
                        <summary class="article-expander__title">2022 Jeffrey M. Hoeg Award Lecture: Genomic Aging, Clonal Hematopoiesis, and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36353993/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221123201010&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronologic age is the dominant risk factor for coronary artery disease but the features of aging promoting coronary artery disease are poorly understood. Advances in human genetics and population-based genetic profiling of blood cells have uncovered the surprising role of age-related subclinical leukemogenic mutations in blood cells, termed &quot;clonal hematopoiesis of indeterminate potential,&quot; in coronary artery disease. Such mutations typically occur in DNMT3A, TET2, ASXL1, and JAK2. Murine and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36353970"
                        open
                      >
                        <summary class="article-expander__title">Recurrent missense variants in clonal hematopoiesis-related genes present in the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36353970/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221123201010&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) consists in an abnormal expansion of a hematopoietic stem cell bearing an advantageous somatic variant. A survey of known recurrent somatic missense variants in DNMT3A, SF3B1, SRSF2 and TP53, some of the most prominent genes underlying CH of indeterminate potential (CHIP), in gnomAD non-cancer database shows the presence of 73 variants. Many of them reach frequencies higher than 0.01% in various populations and, in many cases, are enriched in specific populations....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36355225"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and cardiovascular disease: deciphering interconnections</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36355225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221122200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiovascular and oncological diseases represent the global major causes of death. For both, a novel and far-reaching risk factor has been identified: clonal hematopoiesis (CH). CH is defined as clonal expansion of peripheral blood cells on the basis of somatic mutations, without overt hematological malignancy. The most commonly affected genes are TET2, DNMT3A, ASXL1 and JAK2. By the age of 70, at least 20-50% of all individuals carry a CH clone, conveying a striking clinical impact by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36353993"
                        open
                      >
                        <summary class="article-expander__title">2022 Jeffrey M. Hoeg Award Lecture: Genomic Aging, Clonal Hematopoiesis, and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36353993/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221122200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronologic age is the dominant risk factor for coronary artery disease but the features of aging promoting coronary artery disease are poorly understood. Advances in human genetics and population-based genetic profiling of blood cells have uncovered the surprising role of age-related subclinical leukemogenic mutations in blood cells, termed &quot;clonal hematopoiesis of indeterminate potential,&quot; in coronary artery disease. Such mutations typically occur in DNMT3A, TET2, ASXL1, and JAK2. Murine and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36353970"
                        open
                      >
                        <summary class="article-expander__title">Recurrent missense variants in clonal hematopoiesis-related genes present in the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36353970/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221122200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) consists in an abnormal expansion of a hematopoietic stem cell bearing an advantageous somatic variant. A survey of known recurrent somatic missense variants in DNMT3A, SF3B1, SRSF2 and TP53, some of the most prominent genes underlying CH of indeterminate potential (CHIP), in gnomAD non-cancer database shows the presence of 73 variants. Many of them reach frequencies higher than 0.01% in various populations and, in many cases, are enriched in specific populations....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36355225"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and cardiovascular disease: deciphering interconnections</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36355225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221121201257&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiovascular and oncological diseases represent the global major causes of death. For both, a novel and far-reaching risk factor has been identified: clonal hematopoiesis (CH). CH is defined as clonal expansion of peripheral blood cells on the basis of somatic mutations, without overt hematological malignancy. The most commonly affected genes are TET2, DNMT3A, ASXL1 and JAK2. By the age of 70, at least 20-50% of all individuals carry a CH clone, conveying a striking clinical impact by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36353993"
                        open
                      >
                        <summary class="article-expander__title">2022 Jeffrey M. Hoeg Award Lecture: Genomic Aging, Clonal Hematopoiesis, and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36353993/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221121201257&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronologic age is the dominant risk factor for coronary artery disease but the features of aging promoting coronary artery disease are poorly understood. Advances in human genetics and population-based genetic profiling of blood cells have uncovered the surprising role of age-related subclinical leukemogenic mutations in blood cells, termed &quot;clonal hematopoiesis of indeterminate potential,&quot; in coronary artery disease. Such mutations typically occur in DNMT3A, TET2, ASXL1, and JAK2. Murine and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36353970"
                        open
                      >
                        <summary class="article-expander__title">Recurrent missense variants in clonal hematopoiesis-related genes present in the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36353970/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221121201257&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) consists in an abnormal expansion of a hematopoietic stem cell bearing an advantageous somatic variant. A survey of known recurrent somatic missense variants in DNMT3A, SF3B1, SRSF2 and TP53, some of the most prominent genes underlying CH of indeterminate potential (CHIP), in gnomAD non-cancer database shows the presence of 73 variants. Many of them reach frequencies higher than 0.01% in various populations and, in many cases, are enriched in specific populations....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36355225"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and cardiovascular disease: deciphering interconnections</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36355225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221120201223&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiovascular and oncological diseases represent the global major causes of death. For both, a novel and far-reaching risk factor has been identified: clonal hematopoiesis (CH). CH is defined as clonal expansion of peripheral blood cells on the basis of somatic mutations, without overt hematological malignancy. The most commonly affected genes are TET2, DNMT3A, ASXL1 and JAK2. By the age of 70, at least 20-50% of all individuals carry a CH clone, conveying a striking clinical impact by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36353993"
                        open
                      >
                        <summary class="article-expander__title">2022 Jeffrey M. Hoeg Award Lecture: Genomic Aging, Clonal Hematopoiesis, and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36353993/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221120201223&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronologic age is the dominant risk factor for coronary artery disease but the features of aging promoting coronary artery disease are poorly understood. Advances in human genetics and population-based genetic profiling of blood cells have uncovered the surprising role of age-related subclinical leukemogenic mutations in blood cells, termed &quot;clonal hematopoiesis of indeterminate potential,&quot; in coronary artery disease. Such mutations typically occur in DNMT3A, TET2, ASXL1, and JAK2. Murine and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36353970"
                        open
                      >
                        <summary class="article-expander__title">Recurrent missense variants in clonal hematopoiesis-related genes present in the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36353970/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221120201223&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) consists in an abnormal expansion of a hematopoietic stem cell bearing an advantageous somatic variant. A survey of known recurrent somatic missense variants in DNMT3A, SF3B1, SRSF2 and TP53, some of the most prominent genes underlying CH of indeterminate potential (CHIP), in gnomAD non-cancer database shows the presence of 73 variants. Many of them reach frequencies higher than 0.01% in various populations and, in many cases, are enriched in specific populations....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36355225"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and cardiovascular disease: deciphering interconnections</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36355225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221119201545&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiovascular and oncological diseases represent the global major causes of death. For both, a novel and far-reaching risk factor has been identified: clonal hematopoiesis (CH). CH is defined as clonal expansion of peripheral blood cells on the basis of somatic mutations, without overt hematological malignancy. The most commonly affected genes are TET2, DNMT3A, ASXL1 and JAK2. By the age of 70, at least 20-50% of all individuals carry a CH clone, conveying a striking clinical impact by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36353993"
                        open
                      >
                        <summary class="article-expander__title">2022 Jeffrey M. Hoeg Award Lecture: Genomic Aging, Clonal Hematopoiesis, and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36353993/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221119201545&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronologic age is the dominant risk factor for coronary artery disease but the features of aging promoting coronary artery disease are poorly understood. Advances in human genetics and population-based genetic profiling of blood cells have uncovered the surprising role of age-related subclinical leukemogenic mutations in blood cells, termed &quot;clonal hematopoiesis of indeterminate potential,&quot; in coronary artery disease. Such mutations typically occur in DNMT3A, TET2, ASXL1, and JAK2. Murine and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36353970"
                        open
                      >
                        <summary class="article-expander__title">Recurrent missense variants in clonal hematopoiesis-related genes present in the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36353970/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221119201545&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) consists in an abnormal expansion of a hematopoietic stem cell bearing an advantageous somatic variant. A survey of known recurrent somatic missense variants in DNMT3A, SF3B1, SRSF2 and TP53, some of the most prominent genes underlying CH of indeterminate potential (CHIP), in gnomAD non-cancer database shows the presence of 73 variants. Many of them reach frequencies higher than 0.01% in various populations and, in many cases, are enriched in specific populations....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36355225"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and cardiovascular disease: deciphering interconnections</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36355225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221118201229&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiovascular and oncological diseases represent the global major causes of death. For both, a novel and far-reaching risk factor has been identified: clonal hematopoiesis (CH). CH is defined as clonal expansion of peripheral blood cells on the basis of somatic mutations, without overt hematological malignancy. The most commonly affected genes are TET2, DNMT3A, ASXL1 and JAK2. By the age of 70, at least 20-50% of all individuals carry a CH clone, conveying a striking clinical impact by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36353993"
                        open
                      >
                        <summary class="article-expander__title">2022 Jeffrey M. Hoeg Award Lecture: Genomic Aging, Clonal Hematopoiesis, and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36353993/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221118201229&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronologic age is the dominant risk factor for coronary artery disease but the features of aging promoting coronary artery disease are poorly understood. Advances in human genetics and population-based genetic profiling of blood cells have uncovered the surprising role of age-related subclinical leukemogenic mutations in blood cells, termed &quot;clonal hematopoiesis of indeterminate potential,&quot; in coronary artery disease. Such mutations typically occur in DNMT3A, TET2, ASXL1, and JAK2. Murine and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36353970"
                        open
                      >
                        <summary class="article-expander__title">Recurrent missense variants in clonal hematopoiesis-related genes present in the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36353970/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221118201229&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) consists in an abnormal expansion of a hematopoietic stem cell bearing an advantageous somatic variant. A survey of known recurrent somatic missense variants in DNMT3A, SF3B1, SRSF2 and TP53, some of the most prominent genes underlying CH of indeterminate potential (CHIP), in gnomAD non-cancer database shows the presence of 73 variants. Many of them reach frequencies higher than 0.01% in various populations and, in many cases, are enriched in specific populations....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36355225"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and cardiovascular disease: deciphering interconnections</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36355225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221117201524&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiovascular and oncological diseases represent the global major causes of death. For both, a novel and far-reaching risk factor has been identified: clonal hematopoiesis (CH). CH is defined as clonal expansion of peripheral blood cells on the basis of somatic mutations, without overt hematological malignancy. The most commonly affected genes are TET2, DNMT3A, ASXL1 and JAK2. By the age of 70, at least 20-50% of all individuals carry a CH clone, conveying a striking clinical impact by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36353993"
                        open
                      >
                        <summary class="article-expander__title">2022 Jeffrey M. Hoeg Award Lecture: Genomic Aging, Clonal Hematopoiesis, and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36353993/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221117201524&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronologic age is the dominant risk factor for coronary artery disease but the features of aging promoting coronary artery disease are poorly understood. Advances in human genetics and population-based genetic profiling of blood cells have uncovered the surprising role of age-related subclinical leukemogenic mutations in blood cells, termed &quot;clonal hematopoiesis of indeterminate potential,&quot; in coronary artery disease. Such mutations typically occur in DNMT3A, TET2, ASXL1, and JAK2. Murine and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36353970"
                        open
                      >
                        <summary class="article-expander__title">Recurrent missense variants in clonal hematopoiesis-related genes present in the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36353970/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221117201524&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) consists in an abnormal expansion of a hematopoietic stem cell bearing an advantageous somatic variant. A survey of known recurrent somatic missense variants in DNMT3A, SF3B1, SRSF2 and TP53, some of the most prominent genes underlying CH of indeterminate potential (CHIP), in gnomAD non-cancer database shows the presence of 73 variants. Many of them reach frequencies higher than 0.01% in various populations and, in many cases, are enriched in specific populations....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36355225"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and cardiovascular disease: deciphering interconnections</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36355225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221116201100&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiovascular and oncological diseases represent the global major causes of death. For both, a novel and far-reaching risk factor has been identified: clonal hematopoiesis (CH). CH is defined as clonal expansion of peripheral blood cells on the basis of somatic mutations, without overt hematological malignancy. The most commonly affected genes are TET2, DNMT3A, ASXL1 and JAK2. By the age of 70, at least 20-50% of all individuals carry a CH clone, conveying a striking clinical impact by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36353993"
                        open
                      >
                        <summary class="article-expander__title">2022 Jeffrey M. Hoeg Award Lecture: Genomic Aging, Clonal Hematopoiesis, and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36353993/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221116201100&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronologic age is the dominant risk factor for coronary artery disease but the features of aging promoting coronary artery disease are poorly understood. Advances in human genetics and population-based genetic profiling of blood cells have uncovered the surprising role of age-related subclinical leukemogenic mutations in blood cells, termed &quot;clonal hematopoiesis of indeterminate potential,&quot; in coronary artery disease. Such mutations typically occur in DNMT3A, TET2, ASXL1, and JAK2. Murine and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36353970"
                        open
                      >
                        <summary class="article-expander__title">Recurrent missense variants in clonal hematopoiesis-related genes present in the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36353970/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221116201100&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) consists in an abnormal expansion of a hematopoietic stem cell bearing an advantageous somatic variant. A survey of known recurrent somatic missense variants in DNMT3A, SF3B1, SRSF2 and TP53, some of the most prominent genes underlying CH of indeterminate potential (CHIP), in gnomAD non-cancer database shows the presence of 73 variants. Many of them reach frequencies higher than 0.01% in various populations and, in many cases, are enriched in specific populations....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36355225"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and cardiovascular disease: deciphering interconnections</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36355225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221115201214&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiovascular and oncological diseases represent the global major causes of death. For both, a novel and far-reaching risk factor has been identified: clonal hematopoiesis (CH). CH is defined as clonal expansion of peripheral blood cells on the basis of somatic mutations, without overt hematological malignancy. The most commonly affected genes are TET2, DNMT3A, ASXL1 and JAK2. By the age of 70, at least 20-50% of all individuals carry a CH clone, conveying a striking clinical impact by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36353993"
                        open
                      >
                        <summary class="article-expander__title">2022 Jeffrey M. Hoeg Award Lecture: Genomic Aging, Clonal Hematopoiesis, and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36353993/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221115201214&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronologic age is the dominant risk factor for coronary artery disease but the features of aging promoting coronary artery disease are poorly understood. Advances in human genetics and population-based genetic profiling of blood cells have uncovered the surprising role of age-related subclinical leukemogenic mutations in blood cells, termed &quot;clonal hematopoiesis of indeterminate potential,&quot; in coronary artery disease. Such mutations typically occur in DNMT3A, TET2, ASXL1, and JAK2. Murine and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36353970"
                        open
                      >
                        <summary class="article-expander__title">Recurrent missense variants in clonal hematopoiesis-related genes present in the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36353970/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221115201214&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) consists in an abnormal expansion of a hematopoietic stem cell bearing an advantageous somatic variant. A survey of known recurrent somatic missense variants in DNMT3A, SF3B1, SRSF2 and TP53, some of the most prominent genes underlying CH of indeterminate potential (CHIP), in gnomAD non-cancer database shows the presence of 73 variants. Many of them reach frequencies higher than 0.01% in various populations and, in many cases, are enriched in specific populations....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36355225"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and cardiovascular disease: deciphering interconnections</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36355225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221114201139&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiovascular and oncological diseases represent the global major causes of death. For both, a novel and far-reaching risk factor has been identified: clonal hematopoiesis (CH). CH is defined as clonal expansion of peripheral blood cells on the basis of somatic mutations, without overt hematological malignancy. The most commonly affected genes are TET2, DNMT3A, ASXL1 and JAK2. By the age of 70, at least 20-50% of all individuals carry a CH clone, conveying a striking clinical impact by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36353993"
                        open
                      >
                        <summary class="article-expander__title">2022 Jeffrey M. Hoeg Award Lecture: Genomic Aging, Clonal Hematopoiesis, and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36353993/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221114201139&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronologic age is the dominant risk factor for coronary artery disease but the features of aging promoting coronary artery disease are poorly understood. Advances in human genetics and population-based genetic profiling of blood cells have uncovered the surprising role of age-related subclinical leukemogenic mutations in blood cells, termed &quot;clonal hematopoiesis of indeterminate potential,&quot; in coronary artery disease. Such mutations typically occur in DNMT3A, TET2, ASXL1, and JAK2. Murine and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36353970"
                        open
                      >
                        <summary class="article-expander__title">Recurrent missense variants in clonal hematopoiesis-related genes present in the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36353970/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221114201139&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) consists in an abnormal expansion of a hematopoietic stem cell bearing an advantageous somatic variant. A survey of known recurrent somatic missense variants in DNMT3A, SF3B1, SRSF2 and TP53, some of the most prominent genes underlying CH of indeterminate potential (CHIP), in gnomAD non-cancer database shows the presence of 73 variants. Many of them reach frequencies higher than 0.01% in various populations and, in many cases, are enriched in specific populations....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36355225"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and cardiovascular disease: deciphering interconnections</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36355225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221113201202&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiovascular and oncological diseases represent the global major causes of death. For both, a novel and far-reaching risk factor has been identified: clonal hematopoiesis (CH). CH is defined as clonal expansion of peripheral blood cells on the basis of somatic mutations, without overt hematological malignancy. The most commonly affected genes are TET2, DNMT3A, ASXL1 and JAK2. By the age of 70, at least 20-50% of all individuals carry a CH clone, conveying a striking clinical impact by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36353993"
                        open
                      >
                        <summary class="article-expander__title">2022 Jeffrey M. Hoeg Award Lecture: Genomic Aging, Clonal Hematopoiesis, and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36353993/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221113201202&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronologic age is the dominant risk factor for coronary artery disease but the features of aging promoting coronary artery disease are poorly understood. Advances in human genetics and population-based genetic profiling of blood cells have uncovered the surprising role of age-related subclinical leukemogenic mutations in blood cells, termed &quot;clonal hematopoiesis of indeterminate potential,&quot; in coronary artery disease. Such mutations typically occur in DNMT3A, TET2, ASXL1, and JAK2. Murine and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36353970"
                        open
                      >
                        <summary class="article-expander__title">Recurrent missense variants in clonal hematopoiesis-related genes present in the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36353970/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221113201202&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) consists in an abnormal expansion of a hematopoietic stem cell bearing an advantageous somatic variant. A survey of known recurrent somatic missense variants in DNMT3A, SF3B1, SRSF2 and TP53, some of the most prominent genes underlying CH of indeterminate potential (CHIP), in gnomAD non-cancer database shows the presence of 73 variants. Many of them reach frequencies higher than 0.01% in various populations and, in many cases, are enriched in specific populations....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36355225"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and cardiovascular disease: deciphering interconnections</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36355225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221112201337&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiovascular and oncological diseases represent the global major causes of death. For both, a novel and far-reaching risk factor has been identified: clonal hematopoiesis (CH). CH is defined as clonal expansion of peripheral blood cells on the basis of somatic mutations, without overt hematological malignancy. The most commonly affected genes are TET2, DNMT3A, ASXL1 and JAK2. By the age of 70, at least 20-50% of all individuals carry a CH clone, conveying a striking clinical impact by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36353993"
                        open
                      >
                        <summary class="article-expander__title">2022 Jeffrey M. Hoeg Award Lecture: Genomic Aging, Clonal Hematopoiesis, and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36353993/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221112201337&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronologic age is the dominant risk factor for coronary artery disease but the features of aging promoting coronary artery disease are poorly understood. Advances in human genetics and population-based genetic profiling of blood cells have uncovered the surprising role of age-related subclinical leukemogenic mutations in blood cells, termed &quot;clonal hematopoiesis of indeterminate potential,&quot; in coronary artery disease. Such mutations typically occur in DNMT3A, TET2, ASXL1, and JAK2. Murine and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36353970"
                        open
                      >
                        <summary class="article-expander__title">Recurrent missense variants in clonal hematopoiesis-related genes present in the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36353970/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221112201337&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) consists in an abnormal expansion of a hematopoietic stem cell bearing an advantageous somatic variant. A survey of known recurrent somatic missense variants in DNMT3A, SF3B1, SRSF2 and TP53, some of the most prominent genes underlying CH of indeterminate potential (CHIP), in gnomAD non-cancer database shows the presence of 73 variants. Many of them reach frequencies higher than 0.01% in various populations and, in many cases, are enriched in specific populations....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36355225"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and cardiovascular disease: deciphering interconnections</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36355225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221111201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiovascular and oncological diseases represent the global major causes of death. For both, a novel and far-reaching risk factor has been identified: clonal hematopoiesis (CH). CH is defined as clonal expansion of peripheral blood cells on the basis of somatic mutations, without overt hematological malignancy. The most commonly affected genes are TET2, DNMT3A, ASXL1 and JAK2. By the age of 70, at least 20-50% of all individuals carry a CH clone, conveying a striking clinical impact by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36353993"
                        open
                      >
                        <summary class="article-expander__title">2022 Jeffrey M. Hoeg Award Lecture: Genomic Aging, Clonal Hematopoiesis, and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36353993/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221111201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronologic age is the dominant risk factor for coronary artery disease but the features of aging promoting coronary artery disease are poorly understood. Advances in human genetics and population-based genetic profiling of blood cells have uncovered the surprising role of age-related subclinical leukemogenic mutations in blood cells, termed &quot;clonal hematopoiesis of indeterminate potential,&quot; in coronary artery disease. Such mutations typically occur in DNMT3A, TET2, ASXL1, and JAK2. Murine and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36353970"
                        open
                      >
                        <summary class="article-expander__title">Recurrent missense variants in clonal hematopoiesis-related genes present in the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36353970/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221111201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) consists in an abnormal expansion of a hematopoietic stem cell bearing an advantageous somatic variant. A survey of known recurrent somatic missense variants in DNMT3A, SF3B1, SRSF2 and TP53, some of the most prominent genes underlying CH of indeterminate potential (CHIP), in gnomAD non-cancer database shows the presence of 73 variants. Many of them reach frequencies higher than 0.01% in various populations and, in many cases, are enriched in specific populations....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36355225"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and cardiovascular disease: deciphering interconnections</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36355225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221110201819&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiovascular and oncological diseases represent the global major causes of death. For both, a novel and far-reaching risk factor has been identified: clonal hematopoiesis (CH). CH is defined as clonal expansion of peripheral blood cells on the basis of somatic mutations, without overt hematological malignancy. The most commonly affected genes are TET2, DNMT3A, ASXL1 and JAK2. By the age of 70, at least 20-50% of all individuals carry a CH clone, conveying a striking clinical impact by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36353993"
                        open
                      >
                        <summary class="article-expander__title">2022 Jeffrey M. Hoeg Award Lecture: Genomic Aging, Clonal Hematopoiesis, and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36353993/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221110201819&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronologic age is the dominant risk factor for coronary artery disease but the features of aging promoting coronary artery disease are poorly understood. Advances in human genetics and population-based genetic profiling of blood cells have uncovered the surprising role of age-related subclinical leukemogenic mutations in blood cells, termed &quot;clonal hematopoiesis of indeterminate potential,&quot; in coronary artery disease. Such mutations typically occur in DNMT3A, TET2, ASXL1, and JAK2. Murine and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36353970"
                        open
                      >
                        <summary class="article-expander__title">Recurrent missense variants in clonal hematopoiesis-related genes present in the general population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36353970/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221110201819&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) consists in an abnormal expansion of a hematopoietic stem cell bearing an advantageous somatic variant. A survey of known recurrent somatic missense variants in DNMT3A, SF3B1, SRSF2 and TP53, some of the most prominent genes underlying CH of indeterminate potential (CHIP), in gnomAD non-cancer database shows the presence of 73 variants. Many of them reach frequencies higher than 0.01% in various populations and, in many cases, are enriched in specific populations....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221128200439&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.8">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36354097"
                        open
                      >
                        <summary class="article-expander__title">Reciprocal interplays between MicroRNAs and pluripotency transcription factors in dictating stemness features in human cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36354097/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221127200227&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The interplay between microRNAs (miRNAs) and pluripotency transcription factors (TFs) orchestrates the acquisition of cancer stem cell (CSC) features during the course of malignant transformation, rendering them essential cancer cell dependencies and therapeutic vulnerabilities. In this review, we discuss emerging themes in tumor heterogeneity, including the clonal evolution and the CSC models and their implications in resistance to cancer therapies, and then provide thorough coverage on the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36354097"
                        open
                      >
                        <summary class="article-expander__title">Reciprocal interplays between MicroRNAs and pluripotency transcription factors in dictating stemness features in human cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36354097/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221126200943&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The interplay between microRNAs (miRNAs) and pluripotency transcription factors (TFs) orchestrates the acquisition of cancer stem cell (CSC) features during the course of malignant transformation, rendering them essential cancer cell dependencies and therapeutic vulnerabilities. In this review, we discuss emerging themes in tumor heterogeneity, including the clonal evolution and the CSC models and their implications in resistance to cancer therapies, and then provide thorough coverage on the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36354097"
                        open
                      >
                        <summary class="article-expander__title">Reciprocal interplays between MicroRNAs and pluripotency transcription factors in dictating stemness features in human cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36354097/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221125200054&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The interplay between microRNAs (miRNAs) and pluripotency transcription factors (TFs) orchestrates the acquisition of cancer stem cell (CSC) features during the course of malignant transformation, rendering them essential cancer cell dependencies and therapeutic vulnerabilities. In this review, we discuss emerging themes in tumor heterogeneity, including the clonal evolution and the CSC models and their implications in resistance to cancer therapies, and then provide thorough coverage on the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36354097"
                        open
                      >
                        <summary class="article-expander__title">Reciprocal interplays between MicroRNAs and pluripotency transcription factors in dictating stemness features in human cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36354097/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221124200507&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The interplay between microRNAs (miRNAs) and pluripotency transcription factors (TFs) orchestrates the acquisition of cancer stem cell (CSC) features during the course of malignant transformation, rendering them essential cancer cell dependencies and therapeutic vulnerabilities. In this review, we discuss emerging themes in tumor heterogeneity, including the clonal evolution and the CSC models and their implications in resistance to cancer therapies, and then provide thorough coverage on the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36354097"
                        open
                      >
                        <summary class="article-expander__title">Reciprocal interplays between MicroRNAs and pluripotency transcription factors in dictating stemness features in human cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36354097/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221123201010&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The interplay between microRNAs (miRNAs) and pluripotency transcription factors (TFs) orchestrates the acquisition of cancer stem cell (CSC) features during the course of malignant transformation, rendering them essential cancer cell dependencies and therapeutic vulnerabilities. In this review, we discuss emerging themes in tumor heterogeneity, including the clonal evolution and the CSC models and their implications in resistance to cancer therapies, and then provide thorough coverage on the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36354097"
                        open
                      >
                        <summary class="article-expander__title">Reciprocal interplays between MicroRNAs and pluripotency transcription factors in dictating stemness features in human cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36354097/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221122200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The interplay between microRNAs (miRNAs) and pluripotency transcription factors (TFs) orchestrates the acquisition of cancer stem cell (CSC) features during the course of malignant transformation, rendering them essential cancer cell dependencies and therapeutic vulnerabilities. In this review, we discuss emerging themes in tumor heterogeneity, including the clonal evolution and the CSC models and their implications in resistance to cancer therapies, and then provide thorough coverage on the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36354097"
                        open
                      >
                        <summary class="article-expander__title">Reciprocal interplays between MicroRNAs and pluripotency transcription factors in dictating stemness features in human cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36354097/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221121201257&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The interplay between microRNAs (miRNAs) and pluripotency transcription factors (TFs) orchestrates the acquisition of cancer stem cell (CSC) features during the course of malignant transformation, rendering them essential cancer cell dependencies and therapeutic vulnerabilities. In this review, we discuss emerging themes in tumor heterogeneity, including the clonal evolution and the CSC models and their implications in resistance to cancer therapies, and then provide thorough coverage on the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36354097"
                        open
                      >
                        <summary class="article-expander__title">Reciprocal interplays between MicroRNAs and pluripotency transcription factors in dictating stemness features in human cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36354097/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221120201223&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The interplay between microRNAs (miRNAs) and pluripotency transcription factors (TFs) orchestrates the acquisition of cancer stem cell (CSC) features during the course of malignant transformation, rendering them essential cancer cell dependencies and therapeutic vulnerabilities. In this review, we discuss emerging themes in tumor heterogeneity, including the clonal evolution and the CSC models and their implications in resistance to cancer therapies, and then provide thorough coverage on the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36354097"
                        open
                      >
                        <summary class="article-expander__title">Reciprocal interplays between MicroRNAs and pluripotency transcription factors in dictating stemness features in human cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36354097/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221119201544&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The interplay between microRNAs (miRNAs) and pluripotency transcription factors (TFs) orchestrates the acquisition of cancer stem cell (CSC) features during the course of malignant transformation, rendering them essential cancer cell dependencies and therapeutic vulnerabilities. In this review, we discuss emerging themes in tumor heterogeneity, including the clonal evolution and the CSC models and their implications in resistance to cancer therapies, and then provide thorough coverage on the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36354097"
                        open
                      >
                        <summary class="article-expander__title">Reciprocal interplays between MicroRNAs and pluripotency transcription factors in dictating stemness features in human cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36354097/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221118201229&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The interplay between microRNAs (miRNAs) and pluripotency transcription factors (TFs) orchestrates the acquisition of cancer stem cell (CSC) features during the course of malignant transformation, rendering them essential cancer cell dependencies and therapeutic vulnerabilities. In this review, we discuss emerging themes in tumor heterogeneity, including the clonal evolution and the CSC models and their implications in resistance to cancer therapies, and then provide thorough coverage on the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36354097"
                        open
                      >
                        <summary class="article-expander__title">Reciprocal interplays between MicroRNAs and pluripotency transcription factors in dictating stemness features in human cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36354097/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221117201522&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The interplay between microRNAs (miRNAs) and pluripotency transcription factors (TFs) orchestrates the acquisition of cancer stem cell (CSC) features during the course of malignant transformation, rendering them essential cancer cell dependencies and therapeutic vulnerabilities. In this review, we discuss emerging themes in tumor heterogeneity, including the clonal evolution and the CSC models and their implications in resistance to cancer therapies, and then provide thorough coverage on the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36354097"
                        open
                      >
                        <summary class="article-expander__title">Reciprocal interplays between MicroRNAs and pluripotency transcription factors in dictating stemness features in human cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36354097/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221116201100&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The interplay between microRNAs (miRNAs) and pluripotency transcription factors (TFs) orchestrates the acquisition of cancer stem cell (CSC) features during the course of malignant transformation, rendering them essential cancer cell dependencies and therapeutic vulnerabilities. In this review, we discuss emerging themes in tumor heterogeneity, including the clonal evolution and the CSC models and their implications in resistance to cancer therapies, and then provide thorough coverage on the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36354097"
                        open
                      >
                        <summary class="article-expander__title">Reciprocal interplays between MicroRNAs and pluripotency transcription factors in dictating stemness features in human cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36354097/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221115201214&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The interplay between microRNAs (miRNAs) and pluripotency transcription factors (TFs) orchestrates the acquisition of cancer stem cell (CSC) features during the course of malignant transformation, rendering them essential cancer cell dependencies and therapeutic vulnerabilities. In this review, we discuss emerging themes in tumor heterogeneity, including the clonal evolution and the CSC models and their implications in resistance to cancer therapies, and then provide thorough coverage on the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36354097"
                        open
                      >
                        <summary class="article-expander__title">Reciprocal interplays between MicroRNAs and pluripotency transcription factors in dictating stemness features in human cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36354097/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221114201139&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The interplay between microRNAs (miRNAs) and pluripotency transcription factors (TFs) orchestrates the acquisition of cancer stem cell (CSC) features during the course of malignant transformation, rendering them essential cancer cell dependencies and therapeutic vulnerabilities. In this review, we discuss emerging themes in tumor heterogeneity, including the clonal evolution and the CSC models and their implications in resistance to cancer therapies, and then provide thorough coverage on the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36354097"
                        open
                      >
                        <summary class="article-expander__title">Reciprocal interplays between MicroRNAs and pluripotency transcription factors in dictating stemness features in human cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36354097/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221113201202&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The interplay between microRNAs (miRNAs) and pluripotency transcription factors (TFs) orchestrates the acquisition of cancer stem cell (CSC) features during the course of malignant transformation, rendering them essential cancer cell dependencies and therapeutic vulnerabilities. In this review, we discuss emerging themes in tumor heterogeneity, including the clonal evolution and the CSC models and their implications in resistance to cancer therapies, and then provide thorough coverage on the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36354097"
                        open
                      >
                        <summary class="article-expander__title">Reciprocal interplays between MicroRNAs and pluripotency transcription factors in dictating stemness features in human cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36354097/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221112201337&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The interplay between microRNAs (miRNAs) and pluripotency transcription factors (TFs) orchestrates the acquisition of cancer stem cell (CSC) features during the course of malignant transformation, rendering them essential cancer cell dependencies and therapeutic vulnerabilities. In this review, we discuss emerging themes in tumor heterogeneity, including the clonal evolution and the CSC models and their implications in resistance to cancer therapies, and then provide thorough coverage on the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36354097"
                        open
                      >
                        <summary class="article-expander__title">Reciprocal interplays between MicroRNAs and pluripotency transcription factors in dictating stemness features in human cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36354097/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221111201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The interplay between microRNAs (miRNAs) and pluripotency transcription factors (TFs) orchestrates the acquisition of cancer stem cell (CSC) features during the course of malignant transformation, rendering them essential cancer cell dependencies and therapeutic vulnerabilities. In this review, we discuss emerging themes in tumor heterogeneity, including the clonal evolution and the CSC models and their implications in resistance to cancer therapies, and then provide thorough coverage on the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36354097"
                        open
                      >
                        <summary class="article-expander__title">Reciprocal interplays between MicroRNAs and pluripotency transcription factors in dictating stemness features in human cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36354097/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221110201819&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The interplay between microRNAs (miRNAs) and pluripotency transcription factors (TFs) orchestrates the acquisition of cancer stem cell (CSC) features during the course of malignant transformation, rendering them essential cancer cell dependencies and therapeutic vulnerabilities. In this review, we discuss emerging themes in tumor heterogeneity, including the clonal evolution and the CSC models and their implications in resistance to cancer therapies, and then provide thorough coverage on the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-11-09 datetime="2022-11-09T11:00:00.000Z">2022-11-09</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-11-09 datetime="2022-11-09T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221128200439&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.8">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36352222"
                        open
                      >
                        <summary class="article-expander__title">Spatial genomics maps the structure, nature and evolution of cancer clones</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36352222/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221125200054&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genome sequencing of cancers often reveals mosaics of different subclones present in the same tumour1-3. Although these are believed to arise according to the principles of somatic evolution, the exact spatial growth patterns and underlying mechanisms remain elusive4,5. Here, to address this need, we developed a workflow that generates detailed quantitative maps of genetic subclone composition across whole-tumour sections. These provide the basis for studying clonal growth...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;19
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36351210"
                        open
                      >
                        <summary class="article-expander__title">Resistance Mechanisms to Anti-EGFR Therapy in RAS/RAF Wildtype Colorectal Cancer Vary by Regimen and Line of Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36351210/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221125200054&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: These findings support a model of resistance whereby transcriptomic mechanisms of resistance predominate in the presence of active cytotoxic chemotherapy combined with EGFRi, with a greater predominance of acquired MAPK mutations after single-agent EGFRi. The proposed model has implications for prospective studies evaluating EGFRi rechallenge strategies guided by acquired MAPK mutations, and highlights the need to address transcriptional mechanisms of resistance.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36352222"
                        open
                      >
                        <summary class="article-expander__title">Spatial genomics maps the structure, nature and evolution of cancer clones</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36352222/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221124200507&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genome sequencing of cancers often reveals mosaics of different subclones present in the same tumour1-3. Although these are believed to arise according to the principles of somatic evolution, the exact spatial growth patterns and underlying mechanisms remain elusive4,5. Here, to address this need, we developed a workflow that generates detailed quantitative maps of genetic subclone composition across whole-tumour sections. These provide the basis for studying clonal growth...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;19
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36351210"
                        open
                      >
                        <summary class="article-expander__title">Resistance Mechanisms to Anti-EGFR Therapy in RAS/RAF Wildtype Colorectal Cancer Vary by Regimen and Line of Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36351210/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221124200507&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: These findings support a model of resistance whereby transcriptomic mechanisms of resistance predominate in the presence of active cytotoxic chemotherapy combined with EGFRi, with a greater predominance of acquired MAPK mutations after single-agent EGFRi. The proposed model has implications for prospective studies evaluating EGFRi rechallenge strategies guided by acquired MAPK mutations, and highlights the need to address transcriptional mechanisms of resistance.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36352222"
                        open
                      >
                        <summary class="article-expander__title">Spatial genomics maps the structure, nature and evolution of cancer clones</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36352222/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221123201010&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genome sequencing of cancers often reveals mosaics of different subclones present in the same tumour1-3. Although these are believed to arise according to the principles of somatic evolution, the exact spatial growth patterns and underlying mechanisms remain elusive4,5. Here, to address this need, we developed a workflow that generates detailed quantitative maps of genetic subclone composition across whole-tumour sections. These provide the basis for studying clonal growth...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;19
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36351210"
                        open
                      >
                        <summary class="article-expander__title">Resistance Mechanisms to Anti-EGFR Therapy in RAS/RAF Wildtype Colorectal Cancer Vary by Regimen and Line of Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36351210/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221123201010&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: These findings support a model of resistance whereby transcriptomic mechanisms of resistance predominate in the presence of active cytotoxic chemotherapy combined with EGFRi, with a greater predominance of acquired MAPK mutations after single-agent EGFRi. The proposed model has implications for prospective studies evaluating EGFRi rechallenge strategies guided by acquired MAPK mutations, and highlights the need to address transcriptional mechanisms of resistance.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36352222"
                        open
                      >
                        <summary class="article-expander__title">Spatial genomics maps the structure, nature and evolution of cancer clones</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36352222/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221122200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genome sequencing of cancers often reveals mosaics of different subclones present in the same tumour1-3. Although these are believed to arise according to the principles of somatic evolution, the exact spatial growth patterns and underlying mechanisms remain elusive4,5. Here, to address this need, we developed a workflow that generates detailed quantitative maps of genetic subclone composition across whole-tumour sections. These provide the basis for studying clonal growth...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;19
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36351210"
                        open
                      >
                        <summary class="article-expander__title">Resistance Mechanisms to Anti-EGFR Therapy in RAS/RAF Wildtype Colorectal Cancer Vary by Regimen and Line of Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36351210/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221122200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: These findings support a model of resistance whereby transcriptomic mechanisms of resistance predominate in the presence of active cytotoxic chemotherapy combined with EGFRi, with a greater predominance of acquired MAPK mutations after single-agent EGFRi. The proposed model has implications for prospective studies evaluating EGFRi rechallenge strategies guided by acquired MAPK mutations, and highlights the need to address transcriptional mechanisms of resistance.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36352222"
                        open
                      >
                        <summary class="article-expander__title">Spatial genomics maps the structure, nature and evolution of cancer clones</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36352222/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221121201257&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genome sequencing of cancers often reveals mosaics of different subclones present in the same tumour1-3. Although these are believed to arise according to the principles of somatic evolution, the exact spatial growth patterns and underlying mechanisms remain elusive4,5. Here, to address this need, we developed a workflow that generates detailed quantitative maps of genetic subclone composition across whole-tumour sections. These provide the basis for studying clonal growth...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;19
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36351210"
                        open
                      >
                        <summary class="article-expander__title">Resistance Mechanisms to Anti-EGFR Therapy in RAS/RAF Wildtype Colorectal Cancer Vary by Regimen and Line of Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36351210/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221121201257&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: These findings support a model of resistance whereby transcriptomic mechanisms of resistance predominate in the presence of active cytotoxic chemotherapy combined with EGFRi, with a greater predominance of acquired MAPK mutations after single-agent EGFRi. The proposed model has implications for prospective studies evaluating EGFRi rechallenge strategies guided by acquired MAPK mutations, and highlights the need to address transcriptional mechanisms of resistance.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36352222"
                        open
                      >
                        <summary class="article-expander__title">Spatial genomics maps the structure, nature and evolution of cancer clones</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36352222/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221120201223&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genome sequencing of cancers often reveals mosaics of different subclones present in the same tumour^(1-3). Although these are believed to arise according to the principles of somatic evolution, the exact spatial growth patterns and underlying mechanisms remain elusive^(4,5). Here, to address this need, we developed a workflow that generates detailed quantitative maps of genetic subclone composition across whole-tumour sections. These provide the basis for studying clonal growth patterns, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36351210"
                        open
                      >
                        <summary class="article-expander__title">Resistance Mechanisms to Anti-EGFR Therapy in RAS/RAF Wildtype Colorectal Cancer Vary by Regimen and Line of Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36351210/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221120201223&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: These findings support a model of resistance whereby transcriptomic mechanisms of resistance predominate in the presence of active cytotoxic chemotherapy combined with EGFRi, with a greater predominance of acquired MAPK mutations after single-agent EGFRi. The proposed model has implications for prospective studies evaluating EGFRi rechallenge strategies guided by acquired MAPK mutations, and highlights the need to address transcriptional mechanisms of resistance.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36352222"
                        open
                      >
                        <summary class="article-expander__title">Spatial genomics maps the structure, nature and evolution of cancer clones</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36352222/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221119201544&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genome sequencing of cancers often reveals mosaics of different subclones present in the same tumour^(1-3). Although these are believed to arise according to the principles of somatic evolution, the exact spatial growth patterns and underlying mechanisms remain elusive^(4,5). Here, to address this need, we developed a workflow that generates detailed quantitative maps of genetic subclone composition across whole-tumour sections. These provide the basis for studying clonal growth patterns, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36351210"
                        open
                      >
                        <summary class="article-expander__title">Resistance Mechanisms to Anti-EGFR Therapy in RAS/RAF Wildtype Colorectal Cancer Vary by Regimen and Line of Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36351210/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221119201544&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: These findings support a model of resistance whereby transcriptomic mechanisms of resistance predominate in the presence of active cytotoxic chemotherapy combined with EGFRi, with a greater predominance of acquired MAPK mutations after single-agent EGFRi. The proposed model has implications for prospective studies evaluating EGFRi rechallenge strategies guided by acquired MAPK mutations, and highlights the need to address transcriptional mechanisms of resistance.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36352222"
                        open
                      >
                        <summary class="article-expander__title">Spatial genomics maps the structure, nature and evolution of cancer clones</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36352222/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221118201229&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genome sequencing of cancers often reveals mosaics of different subclones present in the same tumour^(1-3). Although these are believed to arise according to the principles of somatic evolution, the exact spatial growth patterns and underlying mechanisms remain elusive^(4,5). Here, to address this need, we developed a workflow that generates detailed quantitative maps of genetic subclone composition across whole-tumour sections. These provide the basis for studying clonal growth patterns, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36351210"
                        open
                      >
                        <summary class="article-expander__title">Resistance Mechanisms to Anti-EGFR Therapy in RAS/RAF Wildtype Colorectal Cancer Vary by Regimen and Line of Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36351210/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221118201229&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: These findings support a model of resistance whereby transcriptomic mechanisms of resistance predominate in the presence of active cytotoxic chemotherapy combined with EGFRi, with a greater predominance of acquired MAPK mutations after single-agent EGFRi. The proposed model has implications for prospective studies evaluating EGFRi rechallenge strategies guided by acquired MAPK mutations, and highlights the need to address transcriptional mechanisms of resistance.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36352222"
                        open
                      >
                        <summary class="article-expander__title">Spatial genomics maps the structure, nature and evolution of cancer clones</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36352222/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221117201522&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genome sequencing of cancers often reveals mosaics of different subclones present in the same tumour^(1-3). Although these are believed to arise according to the principles of somatic evolution, the exact spatial growth patterns and underlying mechanisms remain elusive^(4,5). Here, to address this need, we developed a workflow that generates detailed quantitative maps of genetic subclone composition across whole-tumour sections. These provide the basis for studying clonal growth patterns, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36351210"
                        open
                      >
                        <summary class="article-expander__title">Resistance Mechanisms to Anti-EGFR Therapy in RAS/RAF Wildtype Colorectal Cancer Vary by Regimen and Line of Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36351210/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221117201522&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: These findings support a model of resistance whereby transcriptomic mechanisms of resistance predominate in the presence of active cytotoxic chemotherapy combined with EGFRi, with a greater predominance of acquired MAPK mutations after single-agent EGFRi. The proposed model has implications for prospective studies evaluating EGFRi rechallenge strategies guided by acquired MAPK mutations, and highlights the need to address transcriptional mechanisms of resistance.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36352222"
                        open
                      >
                        <summary class="article-expander__title">Spatial genomics maps the structure, nature and evolution of cancer clones</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36352222/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221116201100&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genome sequencing of cancers often reveals mosaics of different subclones present in the same tumour^(1-3). Although these are believed to arise according to the principles of somatic evolution, the exact spatial growth patterns and underlying mechanisms remain elusive^(4,5). Here, to address this need, we developed a workflow that generates detailed quantitative maps of genetic subclone composition across whole-tumour sections. These provide the basis for studying clonal growth patterns, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36351210"
                        open
                      >
                        <summary class="article-expander__title">Resistance Mechanisms to Anti-EGFR Therapy in RAS/RAF Wildtype Colorectal Cancer Vary by Regimen and Line of Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36351210/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221116201100&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: These findings support a model of resistance whereby transcriptomic mechanisms of resistance predominate in the presence of active cytotoxic chemotherapy combined with EGFRi, with a greater predominance of acquired MAPK mutations after single-agent EGFRi. The proposed model has implications for prospective studies evaluating EGFRi rechallenge strategies guided by acquired MAPK mutations, and highlights the need to address transcriptional mechanisms of resistance.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36352222"
                        open
                      >
                        <summary class="article-expander__title">Spatial genomics maps the structure, nature and evolution of cancer clones</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36352222/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221115201214&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genome sequencing of cancers often reveals mosaics of different subclones present in the same tumour^(1-3). Although these are believed to arise according to the principles of somatic evolution, the exact spatial growth patterns and underlying mechanisms remain elusive^(4,5). Here, to address this need, we developed a workflow that generates detailed quantitative maps of genetic subclone composition across whole-tumour sections. These provide the basis for studying clonal growth patterns, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36351210"
                        open
                      >
                        <summary class="article-expander__title">Resistance Mechanisms to Anti-EGFR Therapy in RAS/RAF Wildtype Colorectal Cancer Vary by Regimen and Line of Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36351210/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221115201214&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: These findings support a model of resistance whereby transcriptomic mechanisms of resistance predominate in the presence of active cytotoxic chemotherapy combined with EGFRi, with a greater predominance of acquired MAPK mutations after single-agent EGFRi. The proposed model has implications for prospective studies evaluating EGFRi rechallenge strategies guided by acquired MAPK mutations, and highlights the need to address transcriptional mechanisms of resistance.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36352222"
                        open
                      >
                        <summary class="article-expander__title">Spatial genomics maps the structure, nature and evolution of cancer clones</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36352222/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221114201139&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genome sequencing of cancers often reveals mosaics of different subclones present in the same tumour^(1-3). Although these are believed to arise according to the principles of somatic evolution, the exact spatial growth patterns and underlying mechanisms remain elusive^(4,5). Here, to address this need, we developed a workflow that generates detailed quantitative maps of genetic subclone composition across whole-tumour sections. These provide the basis for studying clonal growth patterns, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36351210"
                        open
                      >
                        <summary class="article-expander__title">Resistance Mechanisms to Anti-EGFR Therapy in RAS/RAF Wildtype Colorectal Cancer Vary by Regimen and Line of Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36351210/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221114201139&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: These findings support a model of resistance whereby transcriptomic mechanisms of resistance predominate in the presence of active cytotoxic chemotherapy combined with EGFRi, with a greater predominance of acquired MAPK mutations after single-agent EGFRi. The proposed model has implications for prospective studies evaluating EGFRi rechallenge strategies guided by acquired MAPK mutations, and highlights the need to address transcriptional mechanisms of resistance.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36352222"
                        open
                      >
                        <summary class="article-expander__title">Spatial genomics maps the structure, nature and evolution of cancer clones</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36352222/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221113201202&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genome sequencing of cancers often reveals mosaics of different subclones present in the same tumour^(1-3). Although these are believed to arise according to the principles of somatic evolution, the exact spatial growth patterns and underlying mechanisms remain elusive^(4,5). Here, to address this need, we developed a workflow that generates detailed quantitative maps of genetic subclone composition across whole-tumour sections. These provide the basis for studying clonal growth patterns, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36351210"
                        open
                      >
                        <summary class="article-expander__title">Resistance Mechanisms to Anti-EGFR Therapy in RAS/RAF Wildtype Colorectal Cancer Vary by Regimen and Line of Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36351210/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221113201202&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: These findings support a model of resistance whereby transcriptomic mechanisms of resistance predominate in the presence of active cytotoxic chemotherapy combined with EGFRi, with a greater predominance of acquired MAPK mutations after single-agent EGFRi. The proposed model has implications for prospective studies evaluating EGFRi rechallenge strategies guided by acquired MAPK mutations, and highlights the need to address transcriptional mechanisms of resistance.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36352222"
                        open
                      >
                        <summary class="article-expander__title">Spatial genomics maps the structure, nature and evolution of cancer clones</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36352222/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221112201337&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genome sequencing of cancers often reveals mosaics of different subclones present in the same tumour^(1-3). Although these are believed to arise according to the principles of somatic evolution, the exact spatial growth patterns and underlying mechanisms remain elusive^(4,5). Here, to address this need, we developed a workflow that generates detailed quantitative maps of genetic subclone composition across whole-tumour sections. These provide the basis for studying clonal growth patterns, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36351210"
                        open
                      >
                        <summary class="article-expander__title">Resistance Mechanisms to Anti-EGFR Therapy in RAS/RAF Wildtype Colorectal Cancer Vary by Regimen and Line of Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36351210/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221112201337&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: These findings support a model of resistance whereby transcriptomic mechanisms of resistance predominate in the presence of active cytotoxic chemotherapy combined with EGFRi, with a greater predominance of acquired MAPK mutations after single-agent EGFRi. The proposed model has implications for prospective studies evaluating EGFRi rechallenge strategies guided by acquired MAPK mutations, and highlights the need to address transcriptional mechanisms of resistance.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36352222"
                        open
                      >
                        <summary class="article-expander__title">Spatial genomics maps the structure, nature and evolution of cancer clones</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36352222/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221111201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genome sequencing of cancers often reveals mosaics of different subclones present in the same tumour^(1-3). Although these are believed to arise according to the principles of somatic evolution, the exact spatial growth patterns and underlying mechanisms remain elusive^(4,5). Here, to address this need, we developed a workflow that generates detailed quantitative maps of genetic subclone composition across whole-tumour sections. These provide the basis for studying clonal growth patterns, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36351210"
                        open
                      >
                        <summary class="article-expander__title">Resistance Mechanisms to Anti-EGFR Therapy in RAS/RAF Wildtype Colorectal Cancer Vary by Regimen and Line of Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36351210/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221111201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: These findings support a model of resistance whereby transcriptomic mechanisms of resistance predominate in the presence of active cytotoxic chemotherapy combined with EGFRi, with a greater predominance of acquired MAPK mutations after single-agent EGFRi. The proposed model has implications for prospective studies evaluating EGFRi rechallenge strategies guided by acquired MAPK mutations, and highlights the need to address transcriptional mechanisms of resistance.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36352222"
                        open
                      >
                        <summary class="article-expander__title">Spatial genomics maps the structure, nature and evolution of cancer clones</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36352222/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221110201819&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genome sequencing of cancers often reveals mosaics of different subclones present in the same tumour^(1-3). Although these are believed to arise according to the principles of somatic evolution, the exact spatial growth patterns and underlying mechanisms remain elusive^(4,5). Here, to address this need, we developed a workflow that generates detailed quantitative maps of genetic subclone composition across whole-tumour sections. These provide the basis for studying clonal growth patterns, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36351210"
                        open
                      >
                        <summary class="article-expander__title">Resistance Mechanisms to Anti-EGFR Therapy in RAS/RAF Wildtype Colorectal Cancer Vary by Regimen and Line of Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36351210/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221110201819&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: These findings support a model of resistance whereby transcriptomic mechanisms of resistance predominate in the presence of active cytotoxic chemotherapy combined with EGFRi, with a greater predominance of acquired MAPK mutations after single-agent EGFRi. The proposed model has implications for prospective studies evaluating EGFRi rechallenge strategies guided by acquired MAPK mutations, and highlights the need to address transcriptional mechanisms of resistance.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36351210"
                        open
                      >
                        <summary class="article-expander__title">Resistance Mechanisms to Anti-EGFR Therapy in RAS/RAF Wildtype Colorectal Cancer Vary by Regimen and Line of Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36351210/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221109201530&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: These findings support a model of resistance whereby transcriptomic mechanisms of resistance predominate in the presence of active cytotoxic chemotherapy combined with EGFRi, with a greater predominance of acquired MAPK mutations after single-agent EGFRi. The proposed model has implications for prospective studies evaluating EGFRi rechallenge strategies guided by acquired MAPK mutations, and highlights the need to address transcriptional mechanisms of resistance.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-11-08 datetime="2022-11-08T11:00:00.000Z">2022-11-08</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-11-08 datetime="2022-11-08T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.8&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20221128200439&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36347490"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and bone marrow inflammation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36347490/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221127200227&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) occurs in hematopoietic stem cells with increased risks of progressing to hematologic malignancies. CH mutations are predominantly found in aged populations and correlate with an increased incidence of cardiovascular and other diseases. Increased lines of evidence demonstrate that CH mutations are closely related to the inflammatory bone marrow microenvironment. In this review, we summarize the recent advances in this topic starting from the discovery of CH and its...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36347013"
                        open
                      >
                        <summary class="article-expander__title">Genetic basis and molecular profiling in myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36347013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221127200227&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BCR::ABL1-negative myeloproliferative neoplasms (MPN) are clonal diseases originating from a single hematopoietic stem cell (HSC) that cause excessive production of mature blood cells. The three subtypes polycythemia vera (PV), essential thrombocythemia (ET) and primary myelofibrosis (PMF) are diagnosed according to the World Health Organization (WHO) and International Consensus Classification (ICC) criteria. Acquired gain-of-function mutations in one of three disease driver genes, JAK2, CALR...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36347490"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and bone marrow inflammation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36347490/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221126200943&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) occurs in hematopoietic stem cells with increased risks of progressing to hematologic malignancies. CH mutations are predominantly found in aged populations and correlate with an increased incidence of cardiovascular and other diseases. Increased lines of evidence demonstrate that CH mutations are closely related to the inflammatory bone marrow microenvironment. In this review, we summarize the recent advances in this topic starting from the discovery of CH and its...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36347013"
                        open
                      >
                        <summary class="article-expander__title">Genetic basis and molecular profiling in myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36347013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221126200943&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BCR::ABL1-negative myeloproliferative neoplasms (MPN) are clonal diseases originating from a single hematopoietic stem cell (HSC) that cause excessive production of mature blood cells. The three subtypes polycythemia vera (PV), essential thrombocythemia (ET) and primary myelofibrosis (PMF) are diagnosed according to the World Health Organization (WHO) and International Consensus Classification (ICC) criteria. Acquired gain-of-function mutations in one of three disease driver genes, JAK2, CALR...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36347490"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and bone marrow inflammation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36347490/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221125200054&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) occurs in hematopoietic stem cells with increased risks of progressing to hematologic malignancies. CH mutations are predominantly found in aged populations and correlate with an increased incidence of cardiovascular and other diseases. Increased lines of evidence demonstrate that CH mutations are closely related to the inflammatory bone marrow microenvironment. In this review, we summarize the recent advances in this topic starting from the discovery of CH and its...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36347013"
                        open
                      >
                        <summary class="article-expander__title">Genetic basis and molecular profiling in myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36347013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221125200054&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BCR::ABL1-negative myeloproliferative neoplasms (MPN) are clonal diseases originating from a single hematopoietic stem cell (HSC) that cause excessive production of mature blood cells. The three subtypes polycythemia vera (PV), essential thrombocythemia (ET) and primary myelofibrosis (PMF) are diagnosed according to the World Health Organization (WHO) and International Consensus Classification (ICC) criteria. Acquired gain-of-function mutations in one of three disease driver genes, JAK2, CALR...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36346484"
                        open
                      >
                        <summary class="article-expander__title">Factors associated with clonal hematopoiesis and interaction with marrow environment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36346484/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221125200054&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an expansion of clones in individuals without any hematologic abnormalities, often carrying the driver mutations implicated in myeloid tumors, such as DNMT3A, TET2, and ASXL1. Most notably, CH is an age-related event, accounting for ~ 10% of cases in people over 60 years old. CH may also be correlated with a previous history of cancer treatment with chemotherapeutic drugs/radiation and infection episodes. The link between aging and CH acquisition is best explained by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36347490"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and bone marrow inflammation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36347490/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221124200507&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) occurs in hematopoietic stem cells with increased risks of progressing to hematologic malignancies. CH mutations are predominantly found in aged populations and correlate with an increased incidence of cardiovascular and other diseases. Increased lines of evidence demonstrate that CH mutations are closely related to the inflammatory bone marrow microenvironment. In this review, we summarize the recent advances in this topic starting from the discovery of CH and its...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36347013"
                        open
                      >
                        <summary class="article-expander__title">Genetic basis and molecular profiling in myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36347013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221124200507&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BCR::ABL1-negative myeloproliferative neoplasms (MPN) are clonal diseases originating from a single hematopoietic stem cell (HSC) that cause excessive production of mature blood cells. The three subtypes polycythemia vera (PV), essential thrombocythemia (ET) and primary myelofibrosis (PMF) are diagnosed according to the World Health Organization (WHO) and International Consensus Classification (ICC) criteria. Acquired gain-of-function mutations in one of three disease driver genes, JAK2, CALR...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36346484"
                        open
                      >
                        <summary class="article-expander__title">Factors associated with clonal hematopoiesis and interaction with marrow environment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36346484/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221124200507&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an expansion of clones in individuals without any hematologic abnormalities, often carrying the driver mutations implicated in myeloid tumors, such as DNMT3A, TET2, and ASXL1. Most notably, CH is an age-related event, accounting for ~ 10% of cases in people over 60 years old. CH may also be correlated with a previous history of cancer treatment with chemotherapeutic drugs/radiation and infection episodes. The link between aging and CH acquisition is best explained by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36347490"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and bone marrow inflammation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36347490/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221123201010&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) occurs in hematopoietic stem cells with increased risks of progressing to hematologic malignancies. CH mutations are predominantly found in aged populations and correlate with an increased incidence of cardiovascular and other diseases. Increased lines of evidence demonstrate that CH mutations are closely related to the inflammatory bone marrow microenvironment. In this review, we summarize the recent advances in this topic starting from the discovery of CH and its...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36347013"
                        open
                      >
                        <summary class="article-expander__title">Genetic basis and molecular profiling in myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36347013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221123201010&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BCR::ABL1-negative myeloproliferative neoplasms (MPN) are clonal diseases originating from a single hematopoietic stem cell (HSC) that cause excessive production of mature blood cells. The three subtypes polycythemia vera (PV), essential thrombocythemia (ET) and primary myelofibrosis (PMF) are diagnosed according to the World Health Organization (WHO) and International Consensus Classification (ICC) criteria. Acquired gain-of-function mutations in one of three disease driver genes, JAK2, CALR...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36346484"
                        open
                      >
                        <summary class="article-expander__title">Factors associated with clonal hematopoiesis and interaction with marrow environment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36346484/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221123201010&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an expansion of clones in individuals without any hematologic abnormalities, often carrying the driver mutations implicated in myeloid tumors, such as DNMT3A, TET2, and ASXL1. Most notably, CH is an age-related event, accounting for ~ 10% of cases in people over 60 years old. CH may also be correlated with a previous history of cancer treatment with chemotherapeutic drugs/radiation and infection episodes. The link between aging and CH acquisition is best explained by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36347490"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and bone marrow inflammation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36347490/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221122200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) occurs in hematopoietic stem cells with increased risks of progressing to hematologic malignancies. CH mutations are predominantly found in aged populations and correlate with an increased incidence of cardiovascular and other diseases. Increased lines of evidence demonstrate that CH mutations are closely related to the inflammatory bone marrow microenvironment. In this review, we summarize the recent advances in this topic starting from the discovery of CH and its...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36347013"
                        open
                      >
                        <summary class="article-expander__title">Genetic basis and molecular profiling in myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36347013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221122200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BCR::ABL1-negative myeloproliferative neoplasms (MPN) are clonal diseases originating from a single hematopoietic stem cell (HSC) that cause excessive production of mature blood cells. The three subtypes polycythemia vera (PV), essential thrombocythemia (ET) and primary myelofibrosis (PMF) are diagnosed according to the World Health Organization (WHO) and International Consensus Classification (ICC) criteria. Acquired gain-of-function mutations in one of three disease driver genes, JAK2, CALR...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36346484"
                        open
                      >
                        <summary class="article-expander__title">Factors associated with clonal hematopoiesis and interaction with marrow environment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36346484/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221122200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an expansion of clones in individuals without any hematologic abnormalities, often carrying the driver mutations implicated in myeloid tumors, such as DNMT3A, TET2, and ASXL1. Most notably, CH is an age-related event, accounting for ~ 10% of cases in people over 60 years old. CH may also be correlated with a previous history of cancer treatment with chemotherapeutic drugs/radiation and infection episodes. The link between aging and CH acquisition is best explained by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36347490"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and bone marrow inflammation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36347490/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221121201257&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) occurs in hematopoietic stem cells with increased risks of progressing to hematologic malignancies. CH mutations are predominantly found in aged populations and correlate with an increased incidence of cardiovascular and other diseases. Increased lines of evidence demonstrate that CH mutations are closely related to the inflammatory bone marrow microenvironment. In this review, we summarize the recent advances in this topic starting from the discovery of CH and its...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36347013"
                        open
                      >
                        <summary class="article-expander__title">Genetic basis and molecular profiling in myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36347013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221121201257&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BCR::ABL1-negative myeloproliferative neoplasms (MPN) are clonal diseases originating from a single hematopoietic stem cell (HSC) that cause excessive production of mature blood cells. The three subtypes polycythemia vera (PV), essential thrombocythemia (ET) and primary myelofibrosis (PMF) are diagnosed according to the World Health Organization (WHO) and International Consensus Classification (ICC) criteria. Acquired gain-of-function mutations in one of three disease driver genes, JAK2, CALR...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36346484"
                        open
                      >
                        <summary class="article-expander__title">Factors associated with clonal hematopoiesis and interaction with marrow environment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36346484/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221121201257&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an expansion of clones in individuals without any hematologic abnormalities, often carrying the driver mutations implicated in myeloid tumors, such as DNMT3A, TET2, and ASXL1. Most notably, CH is an age-related event, accounting for ~ 10% of cases in people over 60 years old. CH may also be correlated with a previous history of cancer treatment with chemotherapeutic drugs/radiation and infection episodes. The link between aging and CH acquisition is best explained by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36347490"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and bone marrow inflammation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36347490/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221120201223&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) occurs in hematopoietic stem cells with increased risks of progressing to hematologic malignancies. CH mutations are predominantly found in aged populations and correlate with an increased incidence of cardiovascular and other diseases. Increased lines of evidence demonstrate that CH mutations are closely related to the inflammatory bone marrow microenvironment. In this review, we summarize the recent advances in this topic starting from the discovery of CH and its...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36347013"
                        open
                      >
                        <summary class="article-expander__title">Genetic basis and molecular profiling in myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36347013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221120201223&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BCR::ABL1-negative myeloproliferative neoplasms (MPN) are clonal diseases originating from a single hematopoietic stem cell (HSC) that cause excessive production of mature blood cells. The three subtypes polycythemia vera (PV), essential thrombocythemia (ET) and primary myelofibrosis (PMF) are diagnosed according to the World Health Organization (WHO) and International Consensus Classification (ICC) criteria. Acquired gain-of-function mutations in one of three disease driver genes, JAK2, CALR...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36346484"
                        open
                      >
                        <summary class="article-expander__title">Factors associated with clonal hematopoiesis and interaction with marrow environment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36346484/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221120201223&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an expansion of clones in individuals without any hematologic abnormalities, often carrying the driver mutations implicated in myeloid tumors, such as DNMT3A, TET2, and ASXL1. Most notably, CH is an age-related event, accounting for ~ 10% of cases in people over 60 years old. CH may also be correlated with a previous history of cancer treatment with chemotherapeutic drugs/radiation and infection episodes. The link between aging and CH acquisition is best explained by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36347490"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and bone marrow inflammation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36347490/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221119201545&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) occurs in hematopoietic stem cells with increased risks of progressing to hematologic malignancies. CH mutations are predominantly found in aged populations and correlate with an increased incidence of cardiovascular and other diseases. Increased lines of evidence demonstrate that CH mutations are closely related to the inflammatory bone marrow microenvironment. In this review, we summarize the recent advances in this topic starting from the discovery of CH and its...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36347013"
                        open
                      >
                        <summary class="article-expander__title">Genetic basis and molecular profiling in myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36347013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221119201545&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BCR::ABL1-negative myeloproliferative neoplasms (MPN) are clonal diseases originating from a single hematopoietic stem cell (HSC) that cause excessive production of mature blood cells. The three subtypes polycythemia vera (PV), essential thrombocythemia (ET) and primary myelofibrosis (PMF) are diagnosed according to the World Health Organization (WHO) and International Consensus Classification (ICC) criteria. Acquired gain-of-function mutations in one of three disease driver genes, JAK2, CALR...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36346484"
                        open
                      >
                        <summary class="article-expander__title">Factors associated with clonal hematopoiesis and interaction with marrow environment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36346484/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221119201545&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an expansion of clones in individuals without any hematologic abnormalities, often carrying the driver mutations implicated in myeloid tumors, such as DNMT3A, TET2, and ASXL1. Most notably, CH is an age-related event, accounting for ~ 10% of cases in people over 60 years old. CH may also be correlated with a previous history of cancer treatment with chemotherapeutic drugs/radiation and infection episodes. The link between aging and CH acquisition is best explained by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36347490"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and bone marrow inflammation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36347490/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221118201229&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) occurs in hematopoietic stem cells with increased risks of progressing to hematologic malignancies. CH mutations are predominantly found in aged populations and correlate with an increased incidence of cardiovascular and other diseases. Increased lines of evidence demonstrate that CH mutations are closely related to the inflammatory bone marrow microenvironment. In this review, we summarize the recent advances in this topic starting from the discovery of CH and its...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36347013"
                        open
                      >
                        <summary class="article-expander__title">Genetic basis and molecular profiling in myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36347013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221118201229&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BCR::ABL1-negative myeloproliferative neoplasms (MPN) are clonal diseases originating from a single hematopoietic stem cell (HSC) that cause excessive production of mature blood cells. The three subtypes polycythemia vera (PV), essential thrombocythemia (ET) and primary myelofibrosis (PMF) are diagnosed according to the World Health Organization (WHO) and International Consensus Classification (ICC) criteria. Acquired gain-of-function mutations in one of three disease driver genes, JAK2, CALR...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36346484"
                        open
                      >
                        <summary class="article-expander__title">Factors associated with clonal hematopoiesis and interaction with marrow environment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36346484/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221118201229&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an expansion of clones in individuals without any hematologic abnormalities, often carrying the driver mutations implicated in myeloid tumors, such as DNMT3A, TET2, and ASXL1. Most notably, CH is an age-related event, accounting for ~ 10% of cases in people over 60 years old. CH may also be correlated with a previous history of cancer treatment with chemotherapeutic drugs/radiation and infection episodes. The link between aging and CH acquisition is best explained by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36347490"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and bone marrow inflammation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36347490/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221117201524&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) occurs in hematopoietic stem cells with increased risks of progressing to hematologic malignancies. CH mutations are predominantly found in aged populations and correlate with an increased incidence of cardiovascular and other diseases. Increased lines of evidence demonstrate that CH mutations are closely related to the inflammatory bone marrow microenvironment. In this review, we summarize the recent advances in this topic starting from the discovery of CH and its...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36347013"
                        open
                      >
                        <summary class="article-expander__title">Genetic basis and molecular profiling in myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36347013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221117201524&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BCR::ABL1-negative myeloproliferative neoplasms (MPN) are clonal diseases originating from a single hematopoietic stem cell (HSC) that cause excessive production of mature blood cells. The three subtypes polycythemia vera (PV), essential thrombocythemia (ET) and primary myelofibrosis (PMF) are diagnosed according to the World Health Organization (WHO) and International Consensus Classification (ICC) criteria. Acquired gain-of-function mutations in one of three disease driver genes, JAK2, CALR...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36346484"
                        open
                      >
                        <summary class="article-expander__title">Factors associated with clonal hematopoiesis and interaction with marrow environment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36346484/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221117201524&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an expansion of clones in individuals without any hematologic abnormalities, often carrying the driver mutations implicated in myeloid tumors, such as DNMT3A, TET2, and ASXL1. Most notably, CH is an age-related event, accounting for ~ 10% of cases in people over 60 years old. CH may also be correlated with a previous history of cancer treatment with chemotherapeutic drugs/radiation and infection episodes. The link between aging and CH acquisition is best explained by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36347490"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and bone marrow inflammation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36347490/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221116201100&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) occurs in hematopoietic stem cells with increased risks of progressing to hematologic malignancies. CH mutations are predominantly found in aged populations and correlate with an increased incidence of cardiovascular and other diseases. Increased lines of evidence demonstrate that CH mutations are closely related to the inflammatory bone marrow microenvironment. In this review, we summarize the recent advances in this topic starting from the discovery of CH and its...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36347013"
                        open
                      >
                        <summary class="article-expander__title">Genetic basis and molecular profiling in myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36347013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221116201100&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BCR::ABL1-negative myeloproliferative neoplasms (MPN) are clonal diseases originating from a single hematopoietic stem cell (HSC) that cause excessive production of mature blood cells. The three subtypes polycythemia vera (PV), essential thrombocythemia (ET) and primary myelofibrosis (PMF) are diagnosed according to the World Health Organization (WHO) and International Consensus Classification (ICC) criteria. Acquired gain-of-function mutations in one of three disease driver genes, JAK2, CALR...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36346484"
                        open
                      >
                        <summary class="article-expander__title">Factors associated with clonal hematopoiesis and interaction with marrow environment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36346484/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221116201100&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an expansion of clones in individuals without any hematologic abnormalities, often carrying the driver mutations implicated in myeloid tumors, such as DNMT3A, TET2, and ASXL1. Most notably, CH is an age-related event, accounting for ~ 10% of cases in people over 60 years old. CH may also be correlated with a previous history of cancer treatment with chemotherapeutic drugs/radiation and infection episodes. The link between aging and CH acquisition is best explained by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36347490"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and bone marrow inflammation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36347490/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221115201214&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) occurs in hematopoietic stem cells with increased risks of progressing to hematologic malignancies. CH mutations are predominantly found in aged populations and correlate with an increased incidence of cardiovascular and other diseases. Increased lines of evidence demonstrate that CH mutations are closely related to the inflammatory bone marrow microenvironment. In this review, we summarize the recent advances in this topic starting from the discovery of CH and its...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36347013"
                        open
                      >
                        <summary class="article-expander__title">Genetic basis and molecular profiling in myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36347013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221115201214&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BCR::ABL1-negative myeloproliferative neoplasms (MPN) are clonal diseases originating from a single hematopoietic stem cell (HSC) that cause excessive production of mature blood cells. The three subtypes polycythemia vera (PV), essential thrombocythemia (ET) and primary myelofibrosis (PMF) are diagnosed according to the World Health Organization (WHO) and International Consensus Classification (ICC) criteria. Acquired gain-of-function mutations in one of three disease driver genes, JAK2, CALR...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36346484"
                        open
                      >
                        <summary class="article-expander__title">Factors associated with clonal hematopoiesis and interaction with marrow environment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36346484/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221115201214&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an expansion of clones in individuals without any hematologic abnormalities, often carrying the driver mutations implicated in myeloid tumors, such as DNMT3A, TET2, and ASXL1. Most notably, CH is an age-related event, accounting for ~ 10% of cases in people over 60 years old. CH may also be correlated with a previous history of cancer treatment with chemotherapeutic drugs/radiation and infection episodes. The link between aging and CH acquisition is best explained by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36347490"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and bone marrow inflammation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36347490/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221114201139&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) occurs in hematopoietic stem cells with increased risks of progressing to hematologic malignancies. CH mutations are predominantly found in aged populations and correlate with an increased incidence of cardiovascular and other diseases. Increased lines of evidence demonstrate that CH mutations are closely related to the inflammatory bone marrow microenvironment. In this review, we summarize the recent advances in this topic starting from the discovery of CH and its...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36347013"
                        open
                      >
                        <summary class="article-expander__title">Genetic basis and molecular profiling in myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36347013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221114201139&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BCR::ABL1-negative myeloproliferative neoplasms (MPN) are clonal diseases originating from a single hematopoietic stem cell (HSC) that cause excessive production of mature blood cells. The three subtypes polycythemia vera (PV), essential thrombocythemia (ET) and primary myelofibrosis (PMF) are diagnosed according to the World Health Organization (WHO) and International Consensus Classification (ICC) criteria. Acquired gain-of-function mutations in one of three disease driver genes, JAK2, CALR...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36346484"
                        open
                      >
                        <summary class="article-expander__title">Factors associated with clonal hematopoiesis and interaction with marrow environment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36346484/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221114201139&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an expansion of clones in individuals without any hematologic abnormalities, often carrying the driver mutations implicated in myeloid tumors, such as DNMT3A, TET2, and ASXL1. Most notably, CH is an age-related event, accounting for ~ 10% of cases in people over 60 years old. CH may also be correlated with a previous history of cancer treatment with chemotherapeutic drugs/radiation and infection episodes. The link between aging and CH acquisition is best explained by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36347490"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and bone marrow inflammation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36347490/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221113201202&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) occurs in hematopoietic stem cells with increased risks of progressing to hematologic malignancies. CH mutations are predominantly found in aged populations and correlate with an increased incidence of cardiovascular and other diseases. Increased lines of evidence demonstrate that CH mutations are closely related to the inflammatory bone marrow microenvironment. In this review, we summarize the recent advances in this topic starting from the discovery of CH and its...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36347013"
                        open
                      >
                        <summary class="article-expander__title">Genetic basis and molecular profiling in myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36347013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221113201202&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BCR::ABL1-negative myeloproliferative neoplasms (MPN) are clonal diseases originating from a single hematopoietic stem cell (HSC) that cause excessive production of mature blood cells. The three subtypes polycythemia vera (PV), essential thrombocythemia (ET) and primary myelofibrosis (PMF) are diagnosed according to the World Health Organization (WHO) and International Consensus Classification (ICC) criteria. Acquired gain-of-function mutations in one of three disease driver genes, JAK2, CALR...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36346484"
                        open
                      >
                        <summary class="article-expander__title">Factors associated with clonal hematopoiesis and interaction with marrow environment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36346484/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221113201202&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an expansion of clones in individuals without any hematologic abnormalities, often carrying the driver mutations implicated in myeloid tumors, such as DNMT3A, TET2, and ASXL1. Most notably, CH is an age-related event, accounting for ~ 10% of cases in people over 60 years old. CH may also be correlated with a previous history of cancer treatment with chemotherapeutic drugs/radiation and infection episodes. The link between aging and CH acquisition is best explained by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36347490"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and bone marrow inflammation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36347490/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221112201337&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) occurs in hematopoietic stem cells with increased risks of progressing to hematologic malignancies. CH mutations are predominantly found in aged populations and correlate with an increased incidence of cardiovascular and other diseases. Increased lines of evidence demonstrate that CH mutations are closely related to the inflammatory bone marrow microenvironment. In this review, we summarize the recent advances in this topic starting from the discovery of CH and its...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36347013"
                        open
                      >
                        <summary class="article-expander__title">Genetic basis and molecular profiling in myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36347013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221112201337&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BCR::ABL1-negative myeloproliferative neoplasms (MPN) are clonal diseases originating from a single hematopoietic stem cell (HSC) that cause excessive production of mature blood cells. The three subtypes polycythemia vera (PV), essential thrombocythemia (ET) and primary myelofibrosis (PMF) are diagnosed according to the World Health Organization (WHO) and International Consensus Classification (ICC) criteria. Acquired gain-of-function mutations in one of three disease driver genes, JAK2, CALR...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36346484"
                        open
                      >
                        <summary class="article-expander__title">Factors associated with clonal hematopoiesis and interaction with marrow environment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36346484/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221112201337&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an expansion of clones in individuals without any hematologic abnormalities, often carrying the driver mutations implicated in myeloid tumors, such as DNMT3A, TET2, and ASXL1. Most notably, CH is an age-related event, accounting for ~ 10% of cases in people over 60 years old. CH may also be correlated with a previous history of cancer treatment with chemotherapeutic drugs/radiation and infection episodes. The link between aging and CH acquisition is best explained by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36347490"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and bone marrow inflammation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36347490/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221111201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) occurs in hematopoietic stem cells with increased risks of progressing to hematologic malignancies. CH mutations are predominantly found in aged populations and correlate with an increased incidence of cardiovascular and other diseases. Increased lines of evidence demonstrate that CH mutations are closely related to the inflammatory bone marrow microenvironment. In this review, we summarize the recent advances in this topic starting from the discovery of CH and its...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36347013"
                        open
                      >
                        <summary class="article-expander__title">Genetic basis and molecular profiling in myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36347013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221111201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BCR::ABL1-negative myeloproliferative neoplasms (MPN) are clonal diseases originating from a single hematopoietic stem cell (HSC) that cause excessive production of mature blood cells. The three subtypes polycythemia vera (PV), essential thrombocythemia (ET) and primary myelofibrosis (PMF) are diagnosed according to the World Health Organization (WHO) and International Consensus Classification (ICC) criteria. Acquired gain-of-function mutations in one of three disease driver genes, JAK2, CALR...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36346484"
                        open
                      >
                        <summary class="article-expander__title">Factors associated with clonal hematopoiesis and interaction with marrow environment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36346484/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221111201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an expansion of clones in individuals without any hematologic abnormalities, often carrying the driver mutations implicated in myeloid tumors, such as DNMT3A, TET2, and ASXL1. Most notably, CH is an age-related event, accounting for ~ 10% of cases in people over 60 years old. CH may also be correlated with a previous history of cancer treatment with chemotherapeutic drugs/radiation and infection episodes. The link between aging and CH acquisition is best explained by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36347490"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and bone marrow inflammation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36347490/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221110201819&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) occurs in hematopoietic stem cells with increased risks of progressing to hematologic malignancies. CH mutations are predominantly found in aged populations and correlate with an increased incidence of cardiovascular and other diseases. Increased lines of evidence demonstrate that CH mutations are closely related to the inflammatory bone marrow microenvironment. In this review, we summarize the recent advances in this topic starting from the discovery of CH and its...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36347013"
                        open
                      >
                        <summary class="article-expander__title">Genetic basis and molecular profiling in myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36347013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221110201819&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BCR::ABL1-negative myeloproliferative neoplasms (MPN) are clonal diseases originating from a single hematopoietic stem cell (HSC) that cause excessive production of mature blood cells. The three subtypes polycythemia vera (PV), essential thrombocythemia (ET) and primary myelofibrosis (PMF) are diagnosed according to the World Health Organization (WHO) and International Consensus Classification (ICC) criteria. Acquired gain-of-function mutations in one of three disease driver genes, JAK2, CALR...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36346484"
                        open
                      >
                        <summary class="article-expander__title">Factors associated with clonal hematopoiesis and interaction with marrow environment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36346484/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221110201819&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an expansion of clones in individuals without any hematologic abnormalities, often carrying the driver mutations implicated in myeloid tumors, such as DNMT3A, TET2, and ASXL1. Most notably, CH is an age-related event, accounting for ~ 10% of cases in people over 60 years old. CH may also be correlated with a previous history of cancer treatment with chemotherapeutic drugs/radiation and infection episodes. The link between aging and CH acquisition is best explained by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36347490"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and bone marrow inflammation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36347490/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221109201530&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) occurs in hematopoietic stem cells with increased risks of progressing to hematologic malignancies. CH mutations are predominantly found in aged populations and correlate with an increased incidence of cardiovascular and other diseases. Increased lines of evidence demonstrate that CH mutations are closely related to the inflammatory bone marrow microenvironment. In this review, we summarize the recent advances in this topic starting from the discovery of CH and its...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36347013"
                        open
                      >
                        <summary class="article-expander__title">Genetic basis and molecular profiling in myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36347013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221109201530&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BCR::ABL1-negative myeloproliferative neoplasms (MPN) are clonal diseases originating from a single hematopoietic stem cell (HSC) that cause excessive production of mature blood cells. The three subtypes polycythemia vera (PV), essential thrombocythemia (ET) and primary myelofibrosis (PMF) are diagnosed according to the World Health Organization (WHO) and International Consensus Classification (ICC) criteria. Acquired gain-of-function mutations in one of three disease driver genes, JAK2, CALR...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36346484"
                        open
                      >
                        <summary class="article-expander__title">Factors associated with clonal hematopoiesis and interaction with marrow environment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36346484/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221109201530&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an expansion of clones in individuals without any hematologic abnormalities, often carrying the driver mutations implicated in myeloid tumors, such as DNMT3A, TET2, and ASXL1. Most notably, CH is an age-related event, accounting for ~ 10% of cases in people over 60 years old. CH may also be correlated with a previous history of cancer treatment with chemotherapeutic drugs/radiation and infection episodes. The link between aging and CH acquisition is best explained by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36347013"
                        open
                      >
                        <summary class="article-expander__title">Genetic basis and molecular profiling in myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36347013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221108201514&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BCR::ABL1-negative myeloproliferative neoplasms (MPN) are clonal diseases originating from a single hematopoietic stem cell (HSC) that cause excessive production of mature blood cells. The three subtypes polycythemia vera (PV), essential thrombocythemia (ET) and primary myelofibrosis (PMF) are diagnosed according to the World Health Organization (WHO) and International Consensus Classification (ICC) criteria. Acquired gain-of-function mutations in one of three disease driver genes, JAK2, CALR...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36346484"
                        open
                      >
                        <summary class="article-expander__title">Factors associated with clonal hematopoiesis and interaction with marrow environment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36346484/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221108201514&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an expansion of clones in individuals without any hematologic abnormalities, often carrying the driver mutations implicated in myeloid tumors, such as DNMT3A, TET2, and ASXL1. Most notably, CH is an age-related event, accounting for ~ 10% of cases in people over 60 years old. CH may also be correlated with a previous history of cancer treatment with chemotherapeutic drugs/radiation and infection episodes. The link between aging and CH acquisition is best explained by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-11-07 datetime="2022-11-07T11:00:00.000Z">2022-11-07</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-11-07 datetime="2022-11-07T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.8&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20221128200439&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36339018"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis, multi-omics and coronary artery disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36339018/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221125200054&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Studies of the genetic architecture of cardiovascular disease once focused on heritable germline factors. Newer work has shed light on the role of somatic mutations in blood cells. These mechanistic and multi-omics studies, along with phenotypic analyses, offer the prospect of new precision cardiovascular medicine paradigms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36339018"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis, multi-omics and coronary artery disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36339018/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221124200507&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Studies of the genetic architecture of cardiovascular disease once focused on heritable germline factors. Newer work has shed light on the role of somatic mutations in blood cells. These mechanistic and multi-omics studies, along with phenotypic analyses, offer the prospect of new precision cardiovascular medicine paradigms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36337911"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in cardiovascular disease and therapeutic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36337911/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221124200507&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis arises from somatic mutations that provide a fitness advantage to hematopoietic stem cells and the outgrowth of clones of blood cells. Clonal hematopoiesis commonly involves mutations in genes that are involved in epigenetic modifications, signaling and DNA damage repair. Clonal hematopoiesis has emerged as a major independent risk factor in atherosclerotic cardiovascular disease, thrombosis and heart failure. Studies in mouse models of clonal hematopoiesis have shown an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36339018"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis, multi-omics and coronary artery disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36339018/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221123201010&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Studies of the genetic architecture of cardiovascular disease once focused on heritable germline factors. Newer work has shed light on the role of somatic mutations in blood cells. These mechanistic and multi-omics studies, along with phenotypic analyses, offer the prospect of new precision cardiovascular medicine paradigms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36337911"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in cardiovascular disease and therapeutic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36337911/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221123201010&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis arises from somatic mutations that provide a fitness advantage to hematopoietic stem cells and the outgrowth of clones of blood cells. Clonal hematopoiesis commonly involves mutations in genes that are involved in epigenetic modifications, signaling and DNA damage repair. Clonal hematopoiesis has emerged as a major independent risk factor in atherosclerotic cardiovascular disease, thrombosis and heart failure. Studies in mouse models of clonal hematopoiesis have shown an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36339018"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis, multi-omics and coronary artery disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36339018/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221122200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Studies of the genetic architecture of cardiovascular disease once focused on heritable germline factors. Newer work has shed light on the role of somatic mutations in blood cells. These mechanistic and multi-omics studies, along with phenotypic analyses, offer the prospect of new precision cardiovascular medicine paradigms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36337911"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in cardiovascular disease and therapeutic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36337911/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221122200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis arises from somatic mutations that provide a fitness advantage to hematopoietic stem cells and the outgrowth of clones of blood cells. Clonal hematopoiesis commonly involves mutations in genes that are involved in epigenetic modifications, signaling and DNA damage repair. Clonal hematopoiesis has emerged as a major independent risk factor in atherosclerotic cardiovascular disease, thrombosis and heart failure. Studies in mouse models of clonal hematopoiesis have shown an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36339018"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis, multi-omics and coronary artery disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36339018/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221121201257&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Studies of the genetic architecture of cardiovascular disease once focused on heritable germline factors. Newer work has shed light on the role of somatic mutations in blood cells. These mechanistic and multi-omics studies, along with phenotypic analyses, offer the prospect of new precision cardiovascular medicine paradigms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36337911"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in cardiovascular disease and therapeutic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36337911/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221121201257&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis arises from somatic mutations that provide a fitness advantage to hematopoietic stem cells and the outgrowth of clones of blood cells. Clonal hematopoiesis commonly involves mutations in genes that are involved in epigenetic modifications, signaling and DNA damage repair. Clonal hematopoiesis has emerged as a major independent risk factor in atherosclerotic cardiovascular disease, thrombosis and heart failure. Studies in mouse models of clonal hematopoiesis have shown an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36339018"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis, multi-omics and coronary artery disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36339018/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221120201223&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Studies of the genetic architecture of cardiovascular disease once focused on heritable germline factors. Newer work has shed light on the role of somatic mutations in blood cells. These mechanistic and multi-omics studies, along with phenotypic analyses, offer the prospect of new precision cardiovascular medicine paradigms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36337911"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in cardiovascular disease and therapeutic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36337911/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221120201223&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis arises from somatic mutations that provide a fitness advantage to hematopoietic stem cells and the outgrowth of clones of blood cells. Clonal hematopoiesis commonly involves mutations in genes that are involved in epigenetic modifications, signaling and DNA damage repair. Clonal hematopoiesis has emerged as a major independent risk factor in atherosclerotic cardiovascular disease, thrombosis and heart failure. Studies in mouse models of clonal hematopoiesis have shown an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36339018"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis, multi-omics and coronary artery disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36339018/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221119201545&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Studies of the genetic architecture of cardiovascular disease once focused on heritable germline factors. Newer work has shed light on the role of somatic mutations in blood cells. These mechanistic and multi-omics studies, along with phenotypic analyses, offer the prospect of new precision cardiovascular medicine paradigms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36337911"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in cardiovascular disease and therapeutic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36337911/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221119201545&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis arises from somatic mutations that provide a fitness advantage to hematopoietic stem cells and the outgrowth of clones of blood cells. Clonal hematopoiesis commonly involves mutations in genes that are involved in epigenetic modifications, signaling and DNA damage repair. Clonal hematopoiesis has emerged as a major independent risk factor in atherosclerotic cardiovascular disease, thrombosis and heart failure. Studies in mouse models of clonal hematopoiesis have shown an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36339018"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis, multi-omics and coronary artery disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36339018/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221118201229&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Studies of the genetic architecture of cardiovascular disease once focused on heritable germline factors. Newer work has shed light on the role of somatic mutations in blood cells. These mechanistic and multi-omics studies, along with phenotypic analyses, offer the prospect of new precision cardiovascular medicine paradigms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36337911"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in cardiovascular disease and therapeutic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36337911/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221118201229&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis arises from somatic mutations that provide a fitness advantage to hematopoietic stem cells and the outgrowth of clones of blood cells. Clonal hematopoiesis commonly involves mutations in genes that are involved in epigenetic modifications, signaling and DNA damage repair. Clonal hematopoiesis has emerged as a major independent risk factor in atherosclerotic cardiovascular disease, thrombosis and heart failure. Studies in mouse models of clonal hematopoiesis have shown an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36339018"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis, multi-omics and coronary artery disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36339018/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221117201524&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Studies of the genetic architecture of cardiovascular disease once focused on heritable germline factors. Newer work has shed light on the role of somatic mutations in blood cells. These mechanistic and multi-omics studies, along with phenotypic analyses, offer the prospect of new precision cardiovascular medicine paradigms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36337911"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in cardiovascular disease and therapeutic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36337911/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221117201524&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis arises from somatic mutations that provide a fitness advantage to hematopoietic stem cells and the outgrowth of clones of blood cells. Clonal hematopoiesis commonly involves mutations in genes that are involved in epigenetic modifications, signaling and DNA damage repair. Clonal hematopoiesis has emerged as a major independent risk factor in atherosclerotic cardiovascular disease, thrombosis and heart failure. Studies in mouse models of clonal hematopoiesis have shown an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36339018"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis, multi-omics and coronary artery disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36339018/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221116201100&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Studies of the genetic architecture of cardiovascular disease once focused on heritable germline factors. Newer work has shed light on the role of somatic mutations in blood cells. These mechanistic and multi-omics studies, along with phenotypic analyses, offer the prospect of new precision cardiovascular medicine paradigms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36337911"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in cardiovascular disease and therapeutic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36337911/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221116201100&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis arises from somatic mutations that provide a fitness advantage to hematopoietic stem cells and the outgrowth of clones of blood cells. Clonal hematopoiesis commonly involves mutations in genes that are involved in epigenetic modifications, signaling and DNA damage repair. Clonal hematopoiesis has emerged as a major independent risk factor in atherosclerotic cardiovascular disease, thrombosis and heart failure. Studies in mouse models of clonal hematopoiesis have shown an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36339018"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis, multi-omics and coronary artery disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36339018/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221115201214&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Studies of the genetic architecture of cardiovascular disease once focused on heritable germline factors. Newer work has shed light on the role of somatic mutations in blood cells. These mechanistic and multi-omics studies, along with phenotypic analyses, offer the prospect of new precision cardiovascular medicine paradigms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36337911"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in cardiovascular disease and therapeutic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36337911/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221115201214&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis arises from somatic mutations that provide a fitness advantage to hematopoietic stem cells and the outgrowth of clones of blood cells. Clonal hematopoiesis commonly involves mutations in genes that are involved in epigenetic modifications, signaling and DNA damage repair. Clonal hematopoiesis has emerged as a major independent risk factor in atherosclerotic cardiovascular disease, thrombosis and heart failure. Studies in mouse models of clonal hematopoiesis have shown an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36339018"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis, multi-omics and coronary artery disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36339018/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221114201139&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Studies of the genetic architecture of cardiovascular disease once focused on heritable germline factors. Newer work has shed light on the role of somatic mutations in blood cells. These mechanistic and multi-omics studies, along with phenotypic analyses, offer the prospect of new precision cardiovascular medicine paradigms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36337911"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in cardiovascular disease and therapeutic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36337911/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221114201139&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis arises from somatic mutations that provide a fitness advantage to hematopoietic stem cells and the outgrowth of clones of blood cells. Clonal hematopoiesis commonly involves mutations in genes that are involved in epigenetic modifications, signaling and DNA damage repair. Clonal hematopoiesis has emerged as a major independent risk factor in atherosclerotic cardiovascular disease, thrombosis and heart failure. Studies in mouse models of clonal hematopoiesis have shown an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36339018"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis, multi-omics and coronary artery disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36339018/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221113201202&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Studies of the genetic architecture of cardiovascular disease once focused on heritable germline factors. Newer work has shed light on the role of somatic mutations in blood cells. These mechanistic and multi-omics studies, along with phenotypic analyses, offer the prospect of new precision cardiovascular medicine paradigms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36337911"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in cardiovascular disease and therapeutic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36337911/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221113201202&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis arises from somatic mutations that provide a fitness advantage to hematopoietic stem cells and the outgrowth of clones of blood cells. Clonal hematopoiesis commonly involves mutations in genes that are involved in epigenetic modifications, signaling and DNA damage repair. Clonal hematopoiesis has emerged as a major independent risk factor in atherosclerotic cardiovascular disease, thrombosis and heart failure. Studies in mouse models of clonal hematopoiesis have shown an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36339018"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis, multi-omics and coronary artery disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36339018/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221112201337&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Studies of the genetic architecture of cardiovascular disease once focused on heritable germline factors. Newer work has shed light on the role of somatic mutations in blood cells. These mechanistic and multi-omics studies, along with phenotypic analyses, offer the prospect of new precision cardiovascular medicine paradigms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36337911"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in cardiovascular disease and therapeutic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36337911/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221112201337&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis arises from somatic mutations that provide a fitness advantage to hematopoietic stem cells and the outgrowth of clones of blood cells. Clonal hematopoiesis commonly involves mutations in genes that are involved in epigenetic modifications, signaling and DNA damage repair. Clonal hematopoiesis has emerged as a major independent risk factor in atherosclerotic cardiovascular disease, thrombosis and heart failure. Studies in mouse models of clonal hematopoiesis have shown an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36339018"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis, multi-omics and coronary artery disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36339018/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221111201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Studies of the genetic architecture of cardiovascular disease once focused on heritable germline factors. Newer work has shed light on the role of somatic mutations in blood cells. These mechanistic and multi-omics studies, along with phenotypic analyses, offer the prospect of new precision cardiovascular medicine paradigms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36337911"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in cardiovascular disease and therapeutic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36337911/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221111201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis arises from somatic mutations that provide a fitness advantage to hematopoietic stem cells and the outgrowth of clones of blood cells. Clonal hematopoiesis commonly involves mutations in genes that are involved in epigenetic modifications, signaling and DNA damage repair. Clonal hematopoiesis has emerged as a major independent risk factor in atherosclerotic cardiovascular disease, thrombosis and heart failure. Studies in mouse models of clonal hematopoiesis have shown an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36339018"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis, multi-omics and coronary artery disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36339018/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221110201819&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Studies of the genetic architecture of cardiovascular disease once focused on heritable germline factors. Newer work has shed light on the role of somatic mutations in blood cells. These mechanistic and multi-omics studies, along with phenotypic analyses, offer the prospect of new precision cardiovascular medicine paradigms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36337911"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in cardiovascular disease and therapeutic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36337911/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221110201819&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis arises from somatic mutations that provide a fitness advantage to hematopoietic stem cells and the outgrowth of clones of blood cells. Clonal hematopoiesis commonly involves mutations in genes that are involved in epigenetic modifications, signaling and DNA damage repair. Clonal hematopoiesis has emerged as a major independent risk factor in atherosclerotic cardiovascular disease, thrombosis and heart failure. Studies in mouse models of clonal hematopoiesis have shown an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36339018"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis, multi-omics and coronary artery disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36339018/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221109201530&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Studies of the genetic architecture of cardiovascular disease once focused on heritable germline factors. Newer work has shed light on the role of somatic mutations in blood cells. These mechanistic and multi-omics studies, along with phenotypic analyses, offer the prospect of new precision cardiovascular medicine paradigms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36337911"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in cardiovascular disease and therapeutic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36337911/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221109201530&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis arises from somatic mutations that provide a fitness advantage to hematopoietic stem cells and the outgrowth of clones of blood cells. Clonal hematopoiesis commonly involves mutations in genes that are involved in epigenetic modifications, signaling and DNA damage repair. Clonal hematopoiesis has emerged as a major independent risk factor in atherosclerotic cardiovascular disease, thrombosis and heart failure. Studies in mouse models of clonal hematopoiesis have shown an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36339018"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis, multi-omics and coronary artery disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36339018/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221108201514&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Studies of the genetic architecture of cardiovascular disease once focused on heritable germline factors. Newer work has shed light on the role of somatic mutations in blood cells. These mechanistic and multi-omics studies, along with phenotypic analyses, offer the prospect of new precision cardiovascular medicine paradigms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36337911"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in cardiovascular disease and therapeutic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36337911/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221108201514&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis arises from somatic mutations that provide a fitness advantage to hematopoietic stem cells and the outgrowth of clones of blood cells. Clonal hematopoiesis commonly involves mutations in genes that are involved in epigenetic modifications, signaling and DNA damage repair. Clonal hematopoiesis has emerged as a major independent risk factor in atherosclerotic cardiovascular disease, thrombosis and heart failure. Studies in mouse models of clonal hematopoiesis have shown an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36339018"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis, multi-omics and coronary artery disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36339018/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221107201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Studies of the genetic architecture of cardiovascular disease once focused on heritable germline factors. Newer work has shed light on the role of somatic mutations in blood cells. These mechanistic and multi-omics studies, along with phenotypic analyses, offer the prospect of new precision cardiovascular medicine paradigms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36337911"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in cardiovascular disease and therapeutic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36337911/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221107201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis arises from somatic mutations that provide a fitness advantage to hematopoietic stem cells and the outgrowth of clones of blood cells. Clonal hematopoiesis commonly involves mutations in genes that are involved in epigenetic modifications, signaling and DNA damage repair. Clonal hematopoiesis has emerged as a major independent risk factor in atherosclerotic cardiovascular disease, thrombosis and heart failure. Studies in mouse models of clonal hematopoiesis have shown an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221128200439&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.8">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36338524"
                        open
                      >
                        <summary class="article-expander__title">Clinical utility of liquid biopsy-based companion diagnostics in the non-small-cell lung cancer treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36338524/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221124200507&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recently, technological advances in the detection and biological characterization of circulating tumor DNA (ctDNA) have enabled the implementation of liquid biopsy testing into clinical practice. Methods for analysis of liquid biopsies have rapidly evolved over the past few years and have continued to advance, thus providing details about tumor biological characteristics such as tumor progression, metastasis, tumor heterogeneity, genomic mutation profile, clonal evolution, etc. In tandem with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36338524"
                        open
                      >
                        <summary class="article-expander__title">Clinical utility of liquid biopsy-based companion diagnostics in the non-small-cell lung cancer treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36338524/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221123201010&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recently, technological advances in the detection and biological characterization of circulating tumor DNA (ctDNA) have enabled the implementation of liquid biopsy testing into clinical practice. Methods for analysis of liquid biopsies have rapidly evolved over the past few years and have continued to advance, thus providing details about tumor biological characteristics such as tumor progression, metastasis, tumor heterogeneity, genomic mutation profile, clonal evolution, etc. In tandem with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36338524"
                        open
                      >
                        <summary class="article-expander__title">Clinical utility of liquid biopsy-based companion diagnostics in the non-small-cell lung cancer treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36338524/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221122200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recently, technological advances in the detection and biological characterization of circulating tumor DNA (ctDNA) have enabled the implementation of liquid biopsy testing into clinical practice. Methods for analysis of liquid biopsies have rapidly evolved over the past few years and have continued to advance, thus providing details about tumor biological characteristics such as tumor progression, metastasis, tumor heterogeneity, genomic mutation profile, clonal evolution, etc. In tandem with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36338524"
                        open
                      >
                        <summary class="article-expander__title">Clinical utility of liquid biopsy-based companion diagnostics in the non-small-cell lung cancer treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36338524/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221121201257&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recently, technological advances in the detection and biological characterization of circulating tumor DNA (ctDNA) have enabled the implementation of liquid biopsy testing into clinical practice. Methods for analysis of liquid biopsies have rapidly evolved over the past few years and have continued to advance, thus providing details about tumor biological characteristics such as tumor progression, metastasis, tumor heterogeneity, genomic mutation profile, clonal evolution, etc. In tandem with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36338524"
                        open
                      >
                        <summary class="article-expander__title">Clinical utility of liquid biopsy-based companion diagnostics in the non-small-cell lung cancer treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36338524/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221120201223&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recently, technological advances in the detection and biological characterization of circulating tumor DNA (ctDNA) have enabled the implementation of liquid biopsy testing into clinical practice. Methods for analysis of liquid biopsies have rapidly evolved over the past few years and have continued to advance, thus providing details about tumor biological characteristics such as tumor progression, metastasis, tumor heterogeneity, genomic mutation profile, clonal evolution, etc. In tandem with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36338524"
                        open
                      >
                        <summary class="article-expander__title">Clinical utility of liquid biopsy-based companion diagnostics in the non-small-cell lung cancer treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36338524/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221119201544&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recently, technological advances in the detection and biological characterization of circulating tumor DNA (ctDNA) have enabled the implementation of liquid biopsy testing into clinical practice. Methods for analysis of liquid biopsies have rapidly evolved over the past few years and have continued to advance, thus providing details about tumor biological characteristics such as tumor progression, metastasis, tumor heterogeneity, genomic mutation profile, clonal evolution, etc. In tandem with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36338524"
                        open
                      >
                        <summary class="article-expander__title">Clinical utility of liquid biopsy-based companion diagnostics in the non-small-cell lung cancer treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36338524/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221118201229&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recently, technological advances in the detection and biological characterization of circulating tumor DNA (ctDNA) have enabled the implementation of liquid biopsy testing into clinical practice. Methods for analysis of liquid biopsies have rapidly evolved over the past few years and have continued to advance, thus providing details about tumor biological characteristics such as tumor progression, metastasis, tumor heterogeneity, genomic mutation profile, clonal evolution, etc. In tandem with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36338524"
                        open
                      >
                        <summary class="article-expander__title">Clinical utility of liquid biopsy-based companion diagnostics in the non-small-cell lung cancer treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36338524/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221117201522&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recently, technological advances in the detection and biological characterization of circulating tumor DNA (ctDNA) have enabled the implementation of liquid biopsy testing into clinical practice. Methods for analysis of liquid biopsies have rapidly evolved over the past few years and have continued to advance, thus providing details about tumor biological characteristics such as tumor progression, metastasis, tumor heterogeneity, genomic mutation profile, clonal evolution, etc. In tandem with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36338524"
                        open
                      >
                        <summary class="article-expander__title">Clinical utility of liquid biopsy-based companion diagnostics in the non-small-cell lung cancer treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36338524/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221116201100&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recently, technological advances in the detection and biological characterization of circulating tumor DNA (ctDNA) have enabled the implementation of liquid biopsy testing into clinical practice. Methods for analysis of liquid biopsies have rapidly evolved over the past few years and have continued to advance, thus providing details about tumor biological characteristics such as tumor progression, metastasis, tumor heterogeneity, genomic mutation profile, clonal evolution, etc. In tandem with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36338524"
                        open
                      >
                        <summary class="article-expander__title">Clinical utility of liquid biopsy-based companion diagnostics in the non-small-cell lung cancer treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36338524/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221115201214&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recently, technological advances in the detection and biological characterization of circulating tumor DNA (ctDNA) have enabled the implementation of liquid biopsy testing into clinical practice. Methods for analysis of liquid biopsies have rapidly evolved over the past few years and have continued to advance, thus providing details about tumor biological characteristics such as tumor progression, metastasis, tumor heterogeneity, genomic mutation profile, clonal evolution, etc. In tandem with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36338524"
                        open
                      >
                        <summary class="article-expander__title">Clinical utility of liquid biopsy-based companion diagnostics in the non-small-cell lung cancer treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36338524/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221114201139&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recently, technological advances in the detection and biological characterization of circulating tumor DNA (ctDNA) have enabled the implementation of liquid biopsy testing into clinical practice. Methods for analysis of liquid biopsies have rapidly evolved over the past few years and have continued to advance, thus providing details about tumor biological characteristics such as tumor progression, metastasis, tumor heterogeneity, genomic mutation profile, clonal evolution, etc. In tandem with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36338524"
                        open
                      >
                        <summary class="article-expander__title">Clinical utility of liquid biopsy-based companion diagnostics in the non-small-cell lung cancer treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36338524/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221113201202&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recently, technological advances in the detection and biological characterization of circulating tumor DNA (ctDNA) have enabled the implementation of liquid biopsy testing into clinical practice. Methods for analysis of liquid biopsies have rapidly evolved over the past few years and have continued to advance, thus providing details about tumor biological characteristics such as tumor progression, metastasis, tumor heterogeneity, genomic mutation profile, clonal evolution, etc. In tandem with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36338524"
                        open
                      >
                        <summary class="article-expander__title">Clinical utility of liquid biopsy-based companion diagnostics in the non-small-cell lung cancer treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36338524/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221112201337&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recently, technological advances in the detection and biological characterization of circulating tumor DNA (ctDNA) have enabled the implementation of liquid biopsy testing into clinical practice. Methods for analysis of liquid biopsies have rapidly evolved over the past few years and have continued to advance, thus providing details about tumor biological characteristics such as tumor progression, metastasis, tumor heterogeneity, genomic mutation profile, clonal evolution, etc. In tandem with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36338524"
                        open
                      >
                        <summary class="article-expander__title">Clinical utility of liquid biopsy-based companion diagnostics in the non-small-cell lung cancer treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36338524/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221111201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recently, technological advances in the detection and biological characterization of circulating tumor DNA (ctDNA) have enabled the implementation of liquid biopsy testing into clinical practice. Methods for analysis of liquid biopsies have rapidly evolved over the past few years and have continued to advance, thus providing details about tumor biological characteristics such as tumor progression, metastasis, tumor heterogeneity, genomic mutation profile, clonal evolution, etc. In tandem with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36338524"
                        open
                      >
                        <summary class="article-expander__title">Clinical utility of liquid biopsy-based companion diagnostics in the non-small-cell lung cancer treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36338524/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221110201819&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recently, technological advances in the detection and biological characterization of circulating tumor DNA (ctDNA) have enabled the implementation of liquid biopsy testing into clinical practice. Methods for analysis of liquid biopsies have rapidly evolved over the past few years and have continued to advance, thus providing details about tumor biological characteristics such as tumor progression, metastasis, tumor heterogeneity, genomic mutation profile, clonal evolution, etc. In tandem with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36338524"
                        open
                      >
                        <summary class="article-expander__title">Clinical utility of liquid biopsy-based companion diagnostics in the non-small-cell lung cancer treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36338524/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221109201530&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recently, technological advances in the detection and biological characterization of circulating tumor DNA (ctDNA) have enabled the implementation of liquid biopsy testing into clinical practice. Methods for analysis of liquid biopsies have rapidly evolved over the past few years and have continued to advance, thus providing details about tumor biological characteristics such as tumor progression, metastasis, tumor heterogeneity, genomic mutation profile, clonal evolution, etc. In tandem with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36338524"
                        open
                      >
                        <summary class="article-expander__title">Clinical utility of liquid biopsy-based companion diagnostics in the non-small-cell lung cancer treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36338524/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221108201514&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recently, technological advances in the detection and biological characterization of circulating tumor DNA (ctDNA) have enabled the implementation of liquid biopsy testing into clinical practice. Methods for analysis of liquid biopsies have rapidly evolved over the past few years and have continued to advance, thus providing details about tumor biological characteristics such as tumor progression, metastasis, tumor heterogeneity, genomic mutation profile, clonal evolution, etc. In tandem with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36338524"
                        open
                      >
                        <summary class="article-expander__title">Clinical utility of liquid biopsy-based companion diagnostics in the non-small-cell lung cancer treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36338524/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221107201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recently, technological advances in the detection and biological characterization of circulating tumor DNA (ctDNA) have enabled the implementation of liquid biopsy testing into clinical practice. Methods for analysis of liquid biopsies have rapidly evolved over the past few years and have continued to advance, thus providing details about tumor biological characteristics such as tumor progression, metastasis, tumor heterogeneity, genomic mutation profile, clonal evolution, etc. In tandem with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-11-06 datetime="2022-11-06T11:00:00.000Z">2022-11-06</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-11-06 datetime="2022-11-06T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.8&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20221128200439&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36336766"
                        open
                      >
                        <summary class="article-expander__title">Molecular Pathogenesis of Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36336766/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221122200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Myeloproliferative neoplasms (MPNs) are chronic hematological malignancies characterized by increased proliferation of MPN stem and myeloid progenitor cells with or without bone marrow fibrosis that typically lead to increased peripheral blood cell counts. The genetic and cytogenetic alterations that initiate and drive the development of MPNs have largely been defined, and we summarize these here.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36336766"
                        open
                      >
                        <summary class="article-expander__title">Molecular Pathogenesis of Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36336766/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221121201257&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Myeloproliferative neoplasms (MPNs) are chronic hematological malignancies characterized by increased proliferation of MPN stem and myeloid progenitor cells with or without bone marrow fibrosis that typically lead to increased peripheral blood cell counts. The genetic and cytogenetic alterations that initiate and drive the development of MPNs have largely been defined, and we summarize these here.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36336766"
                        open
                      >
                        <summary class="article-expander__title">Molecular Pathogenesis of Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36336766/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221120201223&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Myeloproliferative neoplasms (MPNs) are chronic hematological malignancies characterized by increased proliferation of MPN stem and myeloid progenitor cells with or without bone marrow fibrosis that typically lead to increased peripheral blood cell counts. The genetic and cytogenetic alterations that initiate and drive the development of MPNs have largely been defined, and we summarize these here.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36336766"
                        open
                      >
                        <summary class="article-expander__title">Molecular Pathogenesis of Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36336766/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221119201545&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Myeloproliferative neoplasms (MPNs) are chronic hematological malignancies characterized by increased proliferation of MPN stem and myeloid progenitor cells with or without bone marrow fibrosis that typically lead to increased peripheral blood cell counts. The genetic and cytogenetic alterations that initiate and drive the development of MPNs have largely been defined, and we summarize these here.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36336766"
                        open
                      >
                        <summary class="article-expander__title">Molecular Pathogenesis of Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36336766/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221118201229&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Myeloproliferative neoplasms (MPNs) are chronic hematological malignancies characterized by increased proliferation of MPN stem and myeloid progenitor cells with or without bone marrow fibrosis that typically lead to increased peripheral blood cell counts. The genetic and cytogenetic alterations that initiate and drive the development of MPNs have largely been defined, and we summarize these here.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36336766"
                        open
                      >
                        <summary class="article-expander__title">Molecular Pathogenesis of Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36336766/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221117201524&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Myeloproliferative neoplasms (MPNs) are chronic hematological malignancies characterized by increased proliferation of MPN stem and myeloid progenitor cells with or without bone marrow fibrosis that typically lead to increased peripheral blood cell counts. The genetic and cytogenetic alterations that initiate and drive the development of MPNs have largely been defined, and we summarize these here.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36336766"
                        open
                      >
                        <summary class="article-expander__title">Molecular Pathogenesis of Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36336766/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221116201100&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Myeloproliferative neoplasms (MPNs) are chronic hematological malignancies characterized by increased proliferation of MPN stem and myeloid progenitor cells with or without bone marrow fibrosis that typically lead to increased peripheral blood cell counts. The genetic and cytogenetic alterations that initiate and drive the development of MPNs have largely been defined, and we summarize these here.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36336766"
                        open
                      >
                        <summary class="article-expander__title">Molecular Pathogenesis of Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36336766/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221115201214&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Myeloproliferative neoplasms (MPNs) are chronic hematological malignancies characterized by increased proliferation of MPN stem and myeloid progenitor cells with or without bone marrow fibrosis that typically lead to increased peripheral blood cell counts. The genetic and cytogenetic alterations that initiate and drive the development of MPNs have largely been defined, and we summarize these here.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36336766"
                        open
                      >
                        <summary class="article-expander__title">Molecular Pathogenesis of Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36336766/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221114201139&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Myeloproliferative neoplasms (MPNs) are chronic hematological malignancies characterized by increased proliferation of MPN stem and myeloid progenitor cells with or without bone marrow fibrosis that typically lead to increased peripheral blood cell counts. The genetic and cytogenetic alterations that initiate and drive the development of MPNs have largely been defined, and we summarize these here.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36336766"
                        open
                      >
                        <summary class="article-expander__title">Molecular Pathogenesis of Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36336766/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221113201202&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Myeloproliferative neoplasms (MPNs) are chronic hematological malignancies characterized by increased proliferation of MPN stem and myeloid progenitor cells with or without bone marrow fibrosis that typically lead to increased peripheral blood cell counts. The genetic and cytogenetic alterations that initiate and drive the development of MPNs have largely been defined, and we summarize these here.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36336766"
                        open
                      >
                        <summary class="article-expander__title">Molecular Pathogenesis of Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36336766/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221112201337&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Myeloproliferative neoplasms (MPNs) are chronic hematological malignancies characterized by increased proliferation of MPN stem and myeloid progenitor cells with or without bone marrow fibrosis that typically lead to increased peripheral blood cell counts. The genetic and cytogenetic alterations that initiate and drive the development of MPNs have largely been defined, and we summarize these here.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36336766"
                        open
                      >
                        <summary class="article-expander__title">Molecular Pathogenesis of Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36336766/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221111201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Myeloproliferative neoplasms (MPNs) are chronic hematological malignancies characterized by increased proliferation of MPN stem and myeloid progenitor cells with or without bone marrow fibrosis that typically lead to increased peripheral blood cell counts. The genetic and cytogenetic alterations that initiate and drive the development of MPNs have largely been defined, and we summarize these here.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36336766"
                        open
                      >
                        <summary class="article-expander__title">Molecular Pathogenesis of Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36336766/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221110201819&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Myeloproliferative neoplasms (MPNs) are chronic hematological malignancies characterized by increased proliferation of MPN stem and myeloid progenitor cells with or without bone marrow fibrosis that typically lead to increased peripheral blood cell counts. The genetic and cytogenetic alterations that initiate and drive the development of MPNs have largely been defined, and we summarize these here.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36336766"
                        open
                      >
                        <summary class="article-expander__title">Molecular Pathogenesis of Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36336766/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221109201530&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Myeloproliferative neoplasms (MPNs) are chronic hematological malignancies characterized by increased proliferation of MPN stem and myeloid progenitor cells with or without bone marrow fibrosis that typically lead to increased peripheral blood cell counts. The genetic and cytogenetic alterations that initiate and drive the development of MPNs have largely been defined, and we summarize these here.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36336766"
                        open
                      >
                        <summary class="article-expander__title">Molecular Pathogenesis of Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36336766/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221108201514&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Myeloproliferative neoplasms (MPNs) are chronic hematological malignancies characterized by increased proliferation of MPN stem and myeloid progenitor cells with or without bone marrow fibrosis that typically lead to increased peripheral blood cell counts. The genetic and cytogenetic alterations that initiate and drive the development of MPNs have largely been defined, and we summarize these here.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36336766"
                        open
                      >
                        <summary class="article-expander__title">Molecular Pathogenesis of Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36336766/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221107201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Myeloproliferative neoplasms (MPNs) are chronic hematological malignancies characterized by increased proliferation of MPN stem and myeloid progenitor cells with or without bone marrow fibrosis that typically lead to increased peripheral blood cell counts. The genetic and cytogenetic alterations that initiate and drive the development of MPNs have largely been defined, and we summarize these here.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-11-05 datetime="2022-11-05T10:00:00.000Z">2022-11-05</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-11-05 datetime="2022-11-05T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221128200439&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.8">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36333584"
                        open
                      >
                        <summary class="article-expander__title">In vivo PDX CRISPR/Cas9 screens reveal mutual therapeutic targets to overcome heterogeneous acquired chemo-resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36333584/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221124200507&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Resistance towards cancer treatment represents a major clinical obstacle, preventing cure of cancer patients. To gain mechanistic insights, we developed a model for acquired resistance to chemotherapy by treating mice carrying patient derived xenografts (PDX) of acute lymphoblastic leukemia with widely-used cytotoxic drugs for 18 consecutive weeks. In two distinct PDX samples, tumors initially responded to treatment, until stable disease and eventually tumor re-growth evolved under therapy, at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36333584"
                        open
                      >
                        <summary class="article-expander__title">In vivo PDX CRISPR/Cas9 screens reveal mutual therapeutic targets to overcome heterogeneous acquired chemo-resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36333584/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221123201010&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Resistance towards cancer treatment represents a major clinical obstacle, preventing cure of cancer patients. To gain mechanistic insights, we developed a model for acquired resistance to chemotherapy by treating mice carrying patient derived xenografts (PDX) of acute lymphoblastic leukemia with widely-used cytotoxic drugs for 18 consecutive weeks. In two distinct PDX samples, tumors initially responded to treatment, until stable disease and eventually tumor re-growth evolved under therapy, at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36333584"
                        open
                      >
                        <summary class="article-expander__title">In vivo PDX CRISPR/Cas9 screens reveal mutual therapeutic targets to overcome heterogeneous acquired chemo-resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36333584/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221122200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Resistance towards cancer treatment represents a major clinical obstacle, preventing cure of cancer patients. To gain mechanistic insights, we developed a model for acquired resistance to chemotherapy by treating mice carrying patient derived xenografts (PDX) of acute lymphoblastic leukemia with widely-used cytotoxic drugs for 18 consecutive weeks. In two distinct PDX samples, tumors initially responded to treatment, until stable disease and eventually tumor re-growth evolved under therapy, at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36333584"
                        open
                      >
                        <summary class="article-expander__title">In vivo PDX CRISPR/Cas9 screens reveal mutual therapeutic targets to overcome heterogeneous acquired chemo-resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36333584/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221121201257&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Resistance towards cancer treatment represents a major clinical obstacle, preventing cure of cancer patients. To gain mechanistic insights, we developed a model for acquired resistance to chemotherapy by treating mice carrying patient derived xenografts (PDX) of acute lymphoblastic leukemia with widely-used cytotoxic drugs for 18 consecutive weeks. In two distinct PDX samples, tumors initially responded to treatment, until stable disease and eventually tumor re-growth evolved under therapy, at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36333584"
                        open
                      >
                        <summary class="article-expander__title">In vivo PDX CRISPR/Cas9 screens reveal mutual therapeutic targets to overcome heterogeneous acquired chemo-resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36333584/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221120201223&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Resistance towards cancer treatment represents a major clinical obstacle, preventing cure of cancer patients. To gain mechanistic insights, we developed a model for acquired resistance to chemotherapy by treating mice carrying patient derived xenografts (PDX) of acute lymphoblastic leukemia with widely-used cytotoxic drugs for 18 consecutive weeks. In two distinct PDX samples, tumors initially responded to treatment, until stable disease and eventually tumor re-growth evolved under therapy, at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36333584"
                        open
                      >
                        <summary class="article-expander__title">In vivo PDX CRISPR/Cas9 screens reveal mutual therapeutic targets to overcome heterogeneous acquired chemo-resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36333584/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221119201544&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Resistance towards cancer treatment represents a major clinical obstacle, preventing cure of cancer patients. To gain mechanistic insights, we developed a model for acquired resistance to chemotherapy by treating mice carrying patient derived xenografts (PDX) of acute lymphoblastic leukemia with widely-used cytotoxic drugs for 18 consecutive weeks. In two distinct PDX samples, tumors initially responded to treatment, until stable disease and eventually tumor re-growth evolved under therapy, at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36333584"
                        open
                      >
                        <summary class="article-expander__title">In vivo PDX CRISPR/Cas9 screens reveal mutual therapeutic targets to overcome heterogeneous acquired chemo-resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36333584/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221118201229&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Resistance towards cancer treatment represents a major clinical obstacle, preventing cure of cancer patients. To gain mechanistic insights, we developed a model for acquired resistance to chemotherapy by treating mice carrying patient derived xenografts (PDX) of acute lymphoblastic leukemia with widely-used cytotoxic drugs for 18 consecutive weeks. In two distinct PDX samples, tumors initially responded to treatment, until stable disease and eventually tumor re-growth evolved under therapy, at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36333584"
                        open
                      >
                        <summary class="article-expander__title">In vivo PDX CRISPR/Cas9 screens reveal mutual therapeutic targets to overcome heterogeneous acquired chemo-resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36333584/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221117201522&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Resistance towards cancer treatment represents a major clinical obstacle, preventing cure of cancer patients. To gain mechanistic insights, we developed a model for acquired resistance to chemotherapy by treating mice carrying patient derived xenografts (PDX) of acute lymphoblastic leukemia with widely-used cytotoxic drugs for 18 consecutive weeks. In two distinct PDX samples, tumors initially responded to treatment, until stable disease and eventually tumor re-growth evolved under therapy, at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36333584"
                        open
                      >
                        <summary class="article-expander__title">In vivo PDX CRISPR/Cas9 screens reveal mutual therapeutic targets to overcome heterogeneous acquired chemo-resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36333584/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221116201100&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Resistance towards cancer treatment represents a major clinical obstacle, preventing cure of cancer patients. To gain mechanistic insights, we developed a model for acquired resistance to chemotherapy by treating mice carrying patient derived xenografts (PDX) of acute lymphoblastic leukemia with widely-used cytotoxic drugs for 18 consecutive weeks. In two distinct PDX samples, tumors initially responded to treatment, until stable disease and eventually tumor re-growth evolved under therapy, at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36333584"
                        open
                      >
                        <summary class="article-expander__title">In vivo PDX CRISPR/Cas9 screens reveal mutual therapeutic targets to overcome heterogeneous acquired chemo-resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36333584/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221115201214&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Resistance towards cancer treatment represents a major clinical obstacle, preventing cure of cancer patients. To gain mechanistic insights, we developed a model for acquired resistance to chemotherapy by treating mice carrying patient derived xenografts (PDX) of acute lymphoblastic leukemia with widely-used cytotoxic drugs for 18 consecutive weeks. In two distinct PDX samples, tumors initially responded to treatment, until stable disease and eventually tumor re-growth evolved under therapy, at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36333584"
                        open
                      >
                        <summary class="article-expander__title">In vivo PDX CRISPR/Cas9 screens reveal mutual therapeutic targets to overcome heterogeneous acquired chemo-resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36333584/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221114201139&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Resistance towards cancer treatment represents a major clinical obstacle, preventing cure of cancer patients. To gain mechanistic insights, we developed a model for acquired resistance to chemotherapy by treating mice carrying patient derived xenografts (PDX) of acute lymphoblastic leukemia with widely-used cytotoxic drugs for 18 consecutive weeks. In two distinct PDX samples, tumors initially responded to treatment, until stable disease and eventually tumor re-growth evolved under therapy, at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36333584"
                        open
                      >
                        <summary class="article-expander__title">In vivo PDX CRISPR/Cas9 screens reveal mutual therapeutic targets to overcome heterogeneous acquired chemo-resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36333584/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221113201202&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Resistance towards cancer treatment represents a major clinical obstacle, preventing cure of cancer patients. To gain mechanistic insights, we developed a model for acquired resistance to chemotherapy by treating mice carrying patient derived xenografts (PDX) of acute lymphoblastic leukemia with widely-used cytotoxic drugs for 18 consecutive weeks. In two distinct PDX samples, tumors initially responded to treatment, until stable disease and eventually tumor re-growth evolved under therapy, at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36333584"
                        open
                      >
                        <summary class="article-expander__title">In vivo PDX CRISPR/Cas9 screens reveal mutual therapeutic targets to overcome heterogeneous acquired chemo-resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36333584/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221112201337&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Resistance towards cancer treatment represents a major clinical obstacle, preventing cure of cancer patients. To gain mechanistic insights, we developed a model for acquired resistance to chemotherapy by treating mice carrying patient derived xenografts (PDX) of acute lymphoblastic leukemia with widely-used cytotoxic drugs for 18 consecutive weeks. In two distinct PDX samples, tumors initially responded to treatment, until stable disease and eventually tumor re-growth evolved under therapy, at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36333584"
                        open
                      >
                        <summary class="article-expander__title">In vivo PDX CRISPR/Cas9 screens reveal mutual therapeutic targets to overcome heterogeneous acquired chemo-resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36333584/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221111201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Resistance towards cancer treatment represents a major clinical obstacle, preventing cure of cancer patients. To gain mechanistic insights, we developed a model for acquired resistance to chemotherapy by treating mice carrying patient derived xenografts (PDX) of acute lymphoblastic leukemia with widely-used cytotoxic drugs for 18 consecutive weeks. In two distinct PDX samples, tumors initially responded to treatment, until stable disease and eventually tumor re-growth evolved under therapy, at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36333584"
                        open
                      >
                        <summary class="article-expander__title">In vivo PDX CRISPR/Cas9 screens reveal mutual therapeutic targets to overcome heterogeneous acquired chemo-resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36333584/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221110201819&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Resistance towards cancer treatment represents a major clinical obstacle, preventing cure of cancer patients. To gain mechanistic insights, we developed a model for acquired resistance to chemotherapy by treating mice carrying patient derived xenografts (PDX) of acute lymphoblastic leukemia with widely-used cytotoxic drugs for 18 consecutive weeks. In two distinct PDX samples, tumors initially responded to treatment, until stable disease and eventually tumor re-growth evolved under therapy, at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36333584"
                        open
                      >
                        <summary class="article-expander__title">In vivo PDX CRISPR/Cas9 screens reveal mutual therapeutic targets to overcome heterogeneous acquired chemo-resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36333584/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221109201530&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Resistance towards cancer treatment represents a major clinical obstacle, preventing cure of cancer patients. To gain mechanistic insights, we developed a model for acquired resistance to chemotherapy by treating mice carrying patient derived xenografts (PDX) of acute lymphoblastic leukemia with widely-used cytotoxic drugs for 18 consecutive weeks. In two distinct PDX samples, tumors initially responded to treatment, until stable disease and eventually tumor re-growth evolved under therapy, at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36333584"
                        open
                      >
                        <summary class="article-expander__title">In vivo PDX CRISPR/Cas9 screens reveal mutual therapeutic targets to overcome heterogeneous acquired chemo-resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36333584/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221108201514&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Resistance towards cancer treatment represents a major clinical obstacle, preventing cure of cancer patients. To gain mechanistic insights, we developed a model for acquired resistance to chemotherapy by treating mice carrying patient derived xenografts (PDX) of acute lymphoblastic leukemia with widely-used cytotoxic drugs for 18 consecutive weeks. In two distinct PDX samples, tumors initially responded to treatment, until stable disease and eventually tumor re-growth evolved under therapy, at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36333584"
                        open
                      >
                        <summary class="article-expander__title">In vivo PDX CRISPR/Cas9 screens reveal mutual therapeutic targets to overcome heterogeneous acquired chemo-resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36333584/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221107201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Resistance towards cancer treatment represents a major clinical obstacle, preventing cure of cancer patients. To gain mechanistic insights, we developed a model for acquired resistance to chemotherapy by treating mice carrying patient derived xenografts (PDX) of acute lymphoblastic leukemia with widely-used cytotoxic drugs for 18 consecutive weeks. In two distinct PDX samples, tumors initially responded to treatment, until stable disease and eventually tumor re-growth evolved under therapy, at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36333584"
                        open
                      >
                        <summary class="article-expander__title">In vivo PDX CRISPR/Cas9 screens reveal mutual therapeutic targets to overcome heterogeneous acquired chemo-resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36333584/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221106201343&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Resistance towards cancer treatment represents a major clinical obstacle, preventing cure of cancer patients. To gain mechanistic insights, we developed a model for acquired resistance to chemotherapy by treating mice carrying patient derived xenografts (PDX) of acute lymphoblastic leukemia with widely-used cytotoxic drugs for 18 consecutive weeks. In two distinct PDX samples, tumors initially responded to treatment, until stable disease and eventually tumor re-growth evolved under therapy, at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36333584"
                        open
                      >
                        <summary class="article-expander__title">In vivo PDX CRISPR/Cas9 screens reveal mutual therapeutic targets to overcome heterogeneous acquired chemo-resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36333584/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221105211555&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Resistance towards cancer treatment represents a major clinical obstacle, preventing cure of cancer patients. To gain mechanistic insights, we developed a model for acquired resistance to chemotherapy by treating mice carrying patient derived xenografts (PDX) of acute lymphoblastic leukemia with widely-used cytotoxic drugs for 18 consecutive weeks. In two distinct PDX samples, tumors initially responded to treatment, until stable disease and eventually tumor re-growth evolved under therapy, at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-11-04 datetime="2022-11-04T10:00:00.000Z">2022-11-04</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-11-04 datetime="2022-11-04T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221128200439&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.8">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36331987"
                        open
                      >
                        <summary class="article-expander__title">Inferring parameters of cancer evolution in chronic lymphocytic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36331987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221124200507&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>As a cancer develops, its cells accrue new mutations, resulting in a heterogeneous, complex genomic profile. We make use of this heterogeneity to derive simple, analytic estimates of parameters driving carcinogenesis and reconstruct the timeline of selective events following initiation of an individual cancer, where two longitudinal samples are available for sequencing. Using stochastic computer simulations of cancer growth, we show that we can accurately estimate mutation rate, time before and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36331987"
                        open
                      >
                        <summary class="article-expander__title">Inferring parameters of cancer evolution in chronic lymphocytic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36331987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221123201010&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>As a cancer develops, its cells accrue new mutations, resulting in a heterogeneous, complex genomic profile. We make use of this heterogeneity to derive simple, analytic estimates of parameters driving carcinogenesis and reconstruct the timeline of selective events following initiation of an individual cancer, where two longitudinal samples are available for sequencing. Using stochastic computer simulations of cancer growth, we show that we can accurately estimate mutation rate, time before and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36331987"
                        open
                      >
                        <summary class="article-expander__title">Inferring parameters of cancer evolution in chronic lymphocytic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36331987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221122200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>As a cancer develops, its cells accrue new mutations, resulting in a heterogeneous, complex genomic profile. We make use of this heterogeneity to derive simple, analytic estimates of parameters driving carcinogenesis and reconstruct the timeline of selective events following initiation of an individual cancer, where two longitudinal samples are available for sequencing. Using stochastic computer simulations of cancer growth, we show that we can accurately estimate mutation rate, time before and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36331987"
                        open
                      >
                        <summary class="article-expander__title">Inferring parameters of cancer evolution in chronic lymphocytic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36331987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221121201257&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>As a cancer develops, its cells accrue new mutations, resulting in a heterogeneous, complex genomic profile. We make use of this heterogeneity to derive simple, analytic estimates of parameters driving carcinogenesis and reconstruct the timeline of selective events following initiation of an individual cancer, where two longitudinal samples are available for sequencing. Using stochastic computer simulations of cancer growth, we show that we can accurately estimate mutation rate, time before and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36331987"
                        open
                      >
                        <summary class="article-expander__title">Inferring parameters of cancer evolution in chronic lymphocytic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36331987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221120201223&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>As a cancer develops, its cells accrue new mutations, resulting in a heterogeneous, complex genomic profile. We make use of this heterogeneity to derive simple, analytic estimates of parameters driving carcinogenesis and reconstruct the timeline of selective events following initiation of an individual cancer, where two longitudinal samples are available for sequencing. Using stochastic computer simulations of cancer growth, we show that we can accurately estimate mutation rate, time before and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36331987"
                        open
                      >
                        <summary class="article-expander__title">Inferring parameters of cancer evolution in chronic lymphocytic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36331987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221119201544&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>As a cancer develops, its cells accrue new mutations, resulting in a heterogeneous, complex genomic profile. We make use of this heterogeneity to derive simple, analytic estimates of parameters driving carcinogenesis and reconstruct the timeline of selective events following initiation of an individual cancer, where two longitudinal samples are available for sequencing. Using stochastic computer simulations of cancer growth, we show that we can accurately estimate mutation rate, time before and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36331987"
                        open
                      >
                        <summary class="article-expander__title">Inferring parameters of cancer evolution in chronic lymphocytic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36331987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221118201229&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>As a cancer develops, its cells accrue new mutations, resulting in a heterogeneous, complex genomic profile. We make use of this heterogeneity to derive simple, analytic estimates of parameters driving carcinogenesis and reconstruct the timeline of selective events following initiation of an individual cancer, where two longitudinal samples are available for sequencing. Using stochastic computer simulations of cancer growth, we show that we can accurately estimate mutation rate, time before and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36331987"
                        open
                      >
                        <summary class="article-expander__title">Inferring parameters of cancer evolution in chronic lymphocytic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36331987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221117201522&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>As a cancer develops, its cells accrue new mutations, resulting in a heterogeneous, complex genomic profile. We make use of this heterogeneity to derive simple, analytic estimates of parameters driving carcinogenesis and reconstruct the timeline of selective events following initiation of an individual cancer, where two longitudinal samples are available for sequencing. Using stochastic computer simulations of cancer growth, we show that we can accurately estimate mutation rate, time before and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36331987"
                        open
                      >
                        <summary class="article-expander__title">Inferring parameters of cancer evolution in chronic lymphocytic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36331987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221116201100&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>As a cancer develops, its cells accrue new mutations, resulting in a heterogeneous, complex genomic profile. We make use of this heterogeneity to derive simple, analytic estimates of parameters driving carcinogenesis and reconstruct the timeline of selective events following initiation of an individual cancer, where two longitudinal samples are available for sequencing. Using stochastic computer simulations of cancer growth, we show that we can accurately estimate mutation rate, time before and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36331987"
                        open
                      >
                        <summary class="article-expander__title">Inferring parameters of cancer evolution in chronic lymphocytic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36331987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221115201214&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>As a cancer develops, its cells accrue new mutations, resulting in a heterogeneous, complex genomic profile. We make use of this heterogeneity to derive simple, analytic estimates of parameters driving carcinogenesis and reconstruct the timeline of selective events following initiation of an individual cancer, where two longitudinal samples are available for sequencing. Using stochastic computer simulations of cancer growth, we show that we can accurately estimate mutation rate, time before and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36331987"
                        open
                      >
                        <summary class="article-expander__title">Inferring parameters of cancer evolution in chronic lymphocytic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36331987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221114201139&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>As a cancer develops, its cells accrue new mutations, resulting in a heterogeneous, complex genomic profile. We make use of this heterogeneity to derive simple, analytic estimates of parameters driving carcinogenesis and reconstruct the timeline of selective events following initiation of an individual cancer, where two longitudinal samples are available for sequencing. Using stochastic computer simulations of cancer growth, we show that we can accurately estimate mutation rate, time before and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36331987"
                        open
                      >
                        <summary class="article-expander__title">Inferring parameters of cancer evolution in chronic lymphocytic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36331987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221113201202&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>As a cancer develops, its cells accrue new mutations, resulting in a heterogeneous, complex genomic profile. We make use of this heterogeneity to derive simple, analytic estimates of parameters driving carcinogenesis and reconstruct the timeline of selective events following initiation of an individual cancer, where two longitudinal samples are available for sequencing. Using stochastic computer simulations of cancer growth, we show that we can accurately estimate mutation rate, time before and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36331987"
                        open
                      >
                        <summary class="article-expander__title">Inferring parameters of cancer evolution in chronic lymphocytic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36331987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221112201337&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>As a cancer develops, its cells accrue new mutations, resulting in a heterogeneous, complex genomic profile. We make use of this heterogeneity to derive simple, analytic estimates of parameters driving carcinogenesis and reconstruct the timeline of selective events following initiation of an individual cancer, where two longitudinal samples are available for sequencing. Using stochastic computer simulations of cancer growth, we show that we can accurately estimate mutation rate, time before and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36331987"
                        open
                      >
                        <summary class="article-expander__title">Inferring parameters of cancer evolution in chronic lymphocytic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36331987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221111201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>As a cancer develops, its cells accrue new mutations, resulting in a heterogeneous, complex genomic profile. We make use of this heterogeneity to derive simple, analytic estimates of parameters driving carcinogenesis and reconstruct the timeline of selective events following initiation of an individual cancer, where two longitudinal samples are available for sequencing. Using stochastic computer simulations of cancer growth, we show that we can accurately estimate mutation rate, time before and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36331987"
                        open
                      >
                        <summary class="article-expander__title">Inferring parameters of cancer evolution in chronic lymphocytic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36331987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221110201819&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>As a cancer develops, its cells accrue new mutations, resulting in a heterogeneous, complex genomic profile. We make use of this heterogeneity to derive simple, analytic estimates of parameters driving carcinogenesis and reconstruct the timeline of selective events following initiation of an individual cancer, where two longitudinal samples are available for sequencing. Using stochastic computer simulations of cancer growth, we show that we can accurately estimate mutation rate, time before and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36331987"
                        open
                      >
                        <summary class="article-expander__title">Inferring parameters of cancer evolution in chronic lymphocytic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36331987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221109201530&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>As a cancer develops, its cells accrue new mutations, resulting in a heterogeneous, complex genomic profile. We make use of this heterogeneity to derive simple, analytic estimates of parameters driving carcinogenesis and reconstruct the timeline of selective events following initiation of an individual cancer, where two longitudinal samples are available for sequencing. Using stochastic computer simulations of cancer growth, we show that we can accurately estimate mutation rate, time before and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36331987"
                        open
                      >
                        <summary class="article-expander__title">Inferring parameters of cancer evolution in chronic lymphocytic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36331987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221108201514&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>As a cancer develops, its cells accrue new mutations, resulting in a heterogeneous, complex genomic profile. We make use of this heterogeneity to derive simple, analytic estimates of parameters driving carcinogenesis and reconstruct the timeline of selective events following initiation of an individual cancer, where two longitudinal samples are available for sequencing. Using stochastic computer simulations of cancer growth, we show that we can accurately estimate mutation rate, time before and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36331987"
                        open
                      >
                        <summary class="article-expander__title">Inferring parameters of cancer evolution in chronic lymphocytic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36331987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221107201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>As a cancer develops, its cells accrue new mutations, resulting in a heterogeneous, complex genomic profile. We make use of this heterogeneity to derive simple, analytic estimates of parameters driving carcinogenesis and reconstruct the timeline of selective events following initiation of an individual cancer, where two longitudinal samples are available for sequencing. Using stochastic computer simulations of cancer growth, we show that we can accurately estimate mutation rate, time before and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36331987"
                        open
                      >
                        <summary class="article-expander__title">Inferring parameters of cancer evolution in chronic lymphocytic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36331987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221106201343&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>As a cancer develops, its cells accrue new mutations, resulting in a heterogeneous, complex genomic profile. We make use of this heterogeneity to derive simple, analytic estimates of parameters driving carcinogenesis and reconstruct the timeline of selective events following initiation of an individual cancer, where two longitudinal samples are available for sequencing. Using stochastic computer simulations of cancer growth, we show that we can accurately estimate mutation rate, time before and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36331987"
                        open
                      >
                        <summary class="article-expander__title">Inferring parameters of cancer evolution in chronic lymphocytic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36331987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221105211555&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>As a cancer develops, its cells accrue new mutations, resulting in a heterogeneous, complex genomic profile. We make use of this heterogeneity to derive simple, analytic estimates of parameters driving carcinogenesis and reconstruct the timeline of selective events following initiation of an individual cancer, where two longitudinal samples are available for sequencing. Using stochastic computer simulations of cancer growth, we show that we can accurately estimate mutation rate, time before and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36331987"
                        open
                      >
                        <summary class="article-expander__title">Inferring parameters of cancer evolution in chronic lymphocytic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36331987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221104211729&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>As a cancer develops, its cells accrue new mutations, resulting in a heterogeneous, complex genomic profile. We make use of this heterogeneity to derive simple, analytic estimates of parameters driving carcinogenesis and reconstruct the timeline of selective events following initiation of an individual cancer, where two longitudinal samples are available for sequencing. Using stochastic computer simulations of cancer growth, we show that we can accurately estimate mutation rate, time before and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-11-03 datetime="2022-11-03T10:00:00.000Z">2022-11-03</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-11-03 datetime="2022-11-03T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.8&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20221128200439&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36325893"
                        open
                      >
                        <summary class="article-expander__title">Genomics improves risk stratification of adults with T-cell acute lymphoblastic leukemia patients enrolled in measurable residual disease-oriented trials</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36325893/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221118201229&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genetic information has been crucial to understand the pathogenesis of T-cell acute lymphoblastic leukemia (T-ALL) at diagnosis and at relapse, but still nowadays has a limited value in a clinical context. Few genetic markers are associated with the outcome of T-ALL patients, independently of measurable residual disease (MRD) status after therapy. In addition, the prognostic relevance of genetic features may be modulated by the specific treatment used. We analyzed the genetic profile of 145...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36325893"
                        open
                      >
                        <summary class="article-expander__title">Genomics improves risk stratification of adults with T-cell acute lymphoblastic leukemia patients enrolled in measurable residual disease-oriented trials</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36325893/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221117201524&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genetic information has been crucial to understand the pathogenesis of T-cell acute lymphoblastic leukemia (T-ALL) at diagnosis and at relapse, but still nowadays has a limited value in a clinical context. Few genetic markers are associated with the outcome of T-ALL patients, independently of measurable residual disease (MRD) status after therapy. In addition, the prognostic relevance of genetic features may be modulated by the specific treatment used. We analyzed the genetic profile of 145...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36325893"
                        open
                      >
                        <summary class="article-expander__title">Genomics improves risk stratification of adults with T-cell acute lymphoblastic leukemia patients enrolled in measurable residual disease-oriented trials</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36325893/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221116201100&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genetic information has been crucial to understand the pathogenesis of T-cell acute lymphoblastic leukemia (T-ALL) at diagnosis and at relapse, but still nowadays has a limited value in a clinical context. Few genetic markers are associated with the outcome of T-ALL patients, independently of measurable residual disease (MRD) status after therapy. In addition, the prognostic relevance of genetic features may be modulated by the specific treatment used. We analyzed the genetic profile of 145...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36325893"
                        open
                      >
                        <summary class="article-expander__title">Genomics improves risk stratification of adults with T-cell acute lymphoblastic leukemia patients enrolled in measurable residual disease-oriented trials</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36325893/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221115201214&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genetic information has been crucial to understand the pathogenesis of T-cell acute lymphoblastic leukemia (T-ALL) at diagnosis and at relapse, but still nowadays has a limited value in a clinical context. Few genetic markers are associated with the outcome of T-ALL patients, independently of measurable residual disease (MRD) status after therapy. In addition, the prognostic relevance of genetic features may be modulated by the specific treatment used. We analyzed the genetic profile of 145...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36325893"
                        open
                      >
                        <summary class="article-expander__title">Genomics improves risk stratification of adults with T-cell acute lymphoblastic leukemia patients enrolled in measurable residual disease-oriented trials</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36325893/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221114201139&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genetic information has been crucial to understand the pathogenesis of T-cell acute lymphoblastic leukemia (T-ALL) at diagnosis and at relapse, but still nowadays has a limited value in a clinical context. Few genetic markers are associated with the outcome of T-ALL patients, independently of measurable residual disease (MRD) status after therapy. In addition, the prognostic relevance of genetic features may be modulated by the specific treatment used. We analyzed the genetic profile of 145...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36325893"
                        open
                      >
                        <summary class="article-expander__title">Genomics improves risk stratification of adults with T-cell acute lymphoblastic leukemia patients enrolled in measurable residual disease-oriented trials</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36325893/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221113201202&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genetic information has been crucial to understand the pathogenesis of T-cell acute lymphoblastic leukemia (T-ALL) at diagnosis and at relapse, but still nowadays has a limited value in a clinical context. Few genetic markers are associated with the outcome of T-ALL patients, independently of measurable residual disease (MRD) status after therapy. In addition, the prognostic relevance of genetic features may be modulated by the specific treatment used. We analyzed the genetic profile of 145...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36325893"
                        open
                      >
                        <summary class="article-expander__title">Genomics improves risk stratification of adults with T-cell acute lymphoblastic leukemia patients enrolled in measurable residual disease-oriented trials</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36325893/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221112201337&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genetic information has been crucial to understand the pathogenesis of T-cell acute lymphoblastic leukemia (T-ALL) at diagnosis and at relapse, but still nowadays has a limited value in a clinical context. Few genetic markers are associated with the outcome of T-ALL patients, independently of measurable residual disease (MRD) status after therapy. In addition, the prognostic relevance of genetic features may be modulated by the specific treatment used. We analyzed the genetic profile of 145...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36325893"
                        open
                      >
                        <summary class="article-expander__title">Genomics improves risk stratification of adults with T-cell acute lymphoblastic leukemia patients enrolled in measurable residual disease-oriented trials</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36325893/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221111201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genetic information has been crucial to understand the pathogenesis of T-cell acute lymphoblastic leukemia (T-ALL) at diagnosis and at relapse, but still nowadays has a limited value in a clinical context. Few genetic markers are associated with the outcome of T-ALL patients, independently of measurable residual disease (MRD) status after therapy. In addition, the prognostic relevance of genetic features may be modulated by the specific treatment used. We analyzed the genetic profile of 145...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36325893"
                        open
                      >
                        <summary class="article-expander__title">Genomics improves risk stratification of adults with T-cell acute lymphoblastic leukemia patients enrolled in measurable residual disease-oriented trials</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36325893/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221110201819&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genetic information has been crucial to understand the pathogenesis of T-cell acute lymphoblastic leukemia (T-ALL) at diagnosis and at relapse, but still nowadays has a limited value in a clinical context. Few genetic markers are associated with the outcome of T-ALL patients, independently of measurable residual disease (MRD) status after therapy. In addition, the prognostic relevance of genetic features may be modulated by the specific treatment used. We analyzed the genetic profile of 145...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36325893"
                        open
                      >
                        <summary class="article-expander__title">Genomics improves risk stratification of adults with T-cell acute lymphoblastic leukemia patients enrolled in measurable residual disease-oriented trials</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36325893/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221109201530&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genetic information has been crucial to understand the pathogenesis of T-cell acute lymphoblastic leukemia (T-ALL) at diagnosis and at relapse, but still nowadays has a limited value in a clinical context. Few genetic markers are associated with the outcome of T-ALL patients, independently of measurable residual disease (MRD) status after therapy. In addition, the prognostic relevance of genetic features may be modulated by the specific treatment used. We analyzed the genetic profile of 145...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36325893"
                        open
                      >
                        <summary class="article-expander__title">Genomics improves risk stratification of adults with T-cell acute lymphoblastic leukemia patients enrolled in measurable residual disease-oriented trials</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36325893/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221108201514&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genetic information has been crucial to understand the pathogenesis of T-cell acute lymphoblastic leukemia (T-ALL) at diagnosis and at relapse, but still nowadays has a limited value in a clinical context. Few genetic markers are associated with the outcome of T-ALL patients, independently of measurable residual disease (MRD) status after therapy. In addition, the prognostic relevance of genetic features may be modulated by the specific treatment used. We analyzed the genetic profile of 145...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36325893"
                        open
                      >
                        <summary class="article-expander__title">Genomics improves risk stratification of adults with T-cell acute lymphoblastic leukemia patients enrolled in measurable residual disease-oriented trials</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36325893/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221107201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genetic information has been crucial to understand the pathogenesis of T-cell acute lymphoblastic leukemia (T-ALL) at diagnosis and at relapse, but still nowadays has a limited value in a clinical context. Few genetic markers are associated with the outcome of T-ALL patients, independently of measurable residual disease (MRD) status after therapy. In addition, the prognostic relevance of genetic features may be modulated by the specific treatment used. We analyzed the genetic profile of 145...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36325893"
                        open
                      >
                        <summary class="article-expander__title">Genomics improves risk stratification of adults with T-cell acute lymphoblastic leukemia patients enrolled in measurable residual disease-oriented trials</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36325893/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221106201343&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genetic information has been crucial to understand the pathogenesis of T-cell acute lymphoblastic leukemia (T-ALL) at diagnosis and at relapse, but still nowadays has a limited value in a clinical context. Few genetic markers are associated with the outcome of T-ALL patients, independently of measurable residual disease (MRD) status after therapy. In addition, the prognostic relevance of genetic features may be modulated by the specific treatment used. We analyzed the genetic profile of 145...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36325893"
                        open
                      >
                        <summary class="article-expander__title">Genomics improves risk stratification of adults with T-cell acute lymphoblastic leukemia patients enrolled in measurable residual disease-oriented trials</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36325893/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221105211555&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genetic information has been crucial to understand the pathogenesis of T-cell acute lymphoblastic leukemia (T-ALL) at diagnosis and at relapse, but still nowadays has a limited value in a clinical context. Few genetic markers are associated with the outcome of T-ALL patients, independently of measurable residual disease (MRD) status after therapy. In addition, the prognostic relevance of genetic features may be modulated by the specific treatment used. We analyzed the genetic profile of 145...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36325893"
                        open
                      >
                        <summary class="article-expander__title">Genomics improves risk stratification of adults with T-cell acute lymphoblastic leukemia patients enrolled in measurable residual disease-oriented trials</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36325893/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221104211730&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genetic information has been crucial to understand the pathogenesis of T-cell acute lymphoblastic leukemia (T-ALL) at diagnosis and at relapse, but still nowadays has a limited value in a clinical context. Few genetic markers are associated with the outcome of T-ALL patients, independently of measurable residual disease (MRD) status after therapy. In addition, the prognostic relevance of genetic features may be modulated by the specific treatment used. We analyzed the genetic profile of 145...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36325893"
                        open
                      >
                        <summary class="article-expander__title">Genomics improves risk stratification of adults with T-cell acute lymphoblastic leukemia patients enrolled in measurable residual disease-oriented trials</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36325893/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221103212023&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genetic information has been crucial to understand the pathogenesis of T-cell acute lymphoblastic leukemia (T-ALL) at diagnosis and at relapse, but still nowadays has a limited value in a clinical context. Few genetic markers are associated with the outcome of T-ALL patients, independently of measurable residual disease (MRD) status after therapy. In addition, the prognostic relevance of genetic features may be modulated by the specific treatment used. We analyzed the genetic profile of 145...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-11-02 datetime="2022-11-02T10:00:00.000Z">2022-11-02</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-11-02 datetime="2022-11-02T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.8&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20221128200439&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36322239"
                        open
                      >
                        <summary class="article-expander__title">Over expression of mTOR gene predicts overall survival in myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36322239/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221116201100&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our study showed an elevation of TNF and AMPK signalling pathways in MDS. TNF signalling might be mediating the proliferative advantage to myeloid clonal cells (mutation carrying cells) over normal cells, whereas, AMPK signalling could be acting as protector against it (favouring normal cells). Hence it would be interesting to explore the functions and pathways associated with mTOR, AMPK, MAPK8 and JUNB in myelopoiesis related diseases like MDS.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36322239"
                        open
                      >
                        <summary class="article-expander__title">Over expression of mTOR gene predicts overall survival in myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36322239/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221115201214&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our study showed an elevation of TNF and AMPK signalling pathways in MDS. TNF signalling might be mediating the proliferative advantage to myeloid clonal cells (mutation carrying cells) over normal cells, whereas, AMPK signalling could be acting as protector against it (favouring normal cells). Hence it would be interesting to explore the functions and pathways associated with mTOR, AMPK, MAPK8 and JUNB in myelopoiesis related diseases like MDS.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36322239"
                        open
                      >
                        <summary class="article-expander__title">Over expression of mTOR gene predicts overall survival in myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36322239/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221114201139&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our study showed an elevation of TNF and AMPK signalling pathways in MDS. TNF signalling might be mediating the proliferative advantage to myeloid clonal cells (mutation carrying cells) over normal cells, whereas, AMPK signalling could be acting as protector against it (favouring normal cells). Hence it would be interesting to explore the functions and pathways associated with mTOR, AMPK, MAPK8 and JUNB in myelopoiesis related diseases like MDS.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36322239"
                        open
                      >
                        <summary class="article-expander__title">Over expression of mTOR gene predicts overall survival in myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36322239/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221113201202&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our study showed an elevation of TNF and AMPK signalling pathways in MDS. TNF signalling might be mediating the proliferative advantage to myeloid clonal cells (mutation carrying cells) over normal cells, whereas, AMPK signalling could be acting as protector against it (favouring normal cells). Hence it would be interesting to explore the functions and pathways associated with mTOR, AMPK, MAPK8 and JUNB in myelopoiesis related diseases like MDS.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36322239"
                        open
                      >
                        <summary class="article-expander__title">Over expression of mTOR gene predicts overall survival in myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36322239/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221112201337&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our study showed an elevation of TNF and AMPK signalling pathways in MDS. TNF signalling might be mediating the proliferative advantage to myeloid clonal cells (mutation carrying cells) over normal cells, whereas, AMPK signalling could be acting as protector against it (favouring normal cells). Hence it would be interesting to explore the functions and pathways associated with mTOR, AMPK, MAPK8 and JUNB in myelopoiesis related diseases like MDS.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36322239"
                        open
                      >
                        <summary class="article-expander__title">Over expression of mTOR gene predicts overall survival in myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36322239/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221111201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our study showed an elevation of TNF and AMPK signalling pathways in MDS. TNF signalling might be mediating the proliferative advantage to myeloid clonal cells (mutation carrying cells) over normal cells, whereas, AMPK signalling could be acting as protector against it (favouring normal cells). Hence it would be interesting to explore the functions and pathways associated with mTOR, AMPK, MAPK8 and JUNB in myelopoiesis related diseases like MDS.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36322239"
                        open
                      >
                        <summary class="article-expander__title">Over expression of mTOR gene predicts overall survival in myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36322239/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221110201819&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our study showed an elevation of TNF and AMPK signalling pathways in MDS. TNF signalling might be mediating the proliferative advantage to myeloid clonal cells (mutation carrying cells) over normal cells, whereas, AMPK signalling could be acting as protector against it (favouring normal cells). Hence it would be interesting to explore the functions and pathways associated with mTOR, AMPK, MAPK8 and JUNB in myelopoiesis related diseases like MDS.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36322239"
                        open
                      >
                        <summary class="article-expander__title">Over expression of mTOR gene predicts overall survival in myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36322239/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221109201530&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our study showed an elevation of TNF and AMPK signalling pathways in MDS. TNF signalling might be mediating the proliferative advantage to myeloid clonal cells (mutation carrying cells) over normal cells, whereas, AMPK signalling could be acting as protector against it (favouring normal cells). Hence it would be interesting to explore the functions and pathways associated with mTOR, AMPK, MAPK8 and JUNB in myelopoiesis related diseases like MDS.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36322239"
                        open
                      >
                        <summary class="article-expander__title">Over expression of mTOR gene predicts overall survival in myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36322239/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221108201514&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our study showed an elevation of TNF and AMPK signalling pathways in MDS. TNF signalling might be mediating the proliferative advantage to myeloid clonal cells (mutation carrying cells) over normal cells, whereas, AMPK signalling could be acting as protector against it (favouring normal cells). Hence it would be interesting to explore the functions and pathways associated with mTOR, AMPK, MAPK8 and JUNB in myelopoiesis related diseases like MDS.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36322239"
                        open
                      >
                        <summary class="article-expander__title">Over expression of mTOR gene predicts overall survival in myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36322239/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221107201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our study showed an elevation of TNF and AMPK signalling pathways in MDS. TNF signalling might be mediating the proliferative advantage to myeloid clonal cells (mutation carrying cells) over normal cells, whereas, AMPK signalling could be acting as protector against it (favouring normal cells). Hence it would be interesting to explore the functions and pathways associated with mTOR, AMPK, MAPK8 and JUNB in myelopoiesis related diseases like MDS.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36322239"
                        open
                      >
                        <summary class="article-expander__title">Over expression of mTOR gene predicts overall survival in myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36322239/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221106201343&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our study showed an elevation of TNF and AMPK signalling pathways in MDS. TNF signalling might be mediating the proliferative advantage to myeloid clonal cells (mutation carrying cells) over normal cells, whereas, AMPK signalling could be acting as protector against it (favouring normal cells). Hence it would be interesting to explore the functions and pathways associated with mTOR, AMPK, MAPK8 and JUNB in myelopoiesis related diseases like MDS.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36322239"
                        open
                      >
                        <summary class="article-expander__title">Over expression of mTOR gene predicts overall survival in myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36322239/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221105211555&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our study showed an elevation of TNF and AMPK signalling pathways in MDS. TNF signalling might be mediating the proliferative advantage to myeloid clonal cells (mutation carrying cells) over normal cells, whereas, AMPK signalling could be acting as protector against it (favouring normal cells). Hence it would be interesting to explore the functions and pathways associated with mTOR, AMPK, MAPK8 and JUNB in myelopoiesis related diseases like MDS.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36322239"
                        open
                      >
                        <summary class="article-expander__title">Over expression of mTOR gene predicts overall survival in myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36322239/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221104211730&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our study showed an elevation of TNF and AMPK signalling pathways in MDS. TNF signalling might be mediating the proliferative advantage to myeloid clonal cells (mutation carrying cells) over normal cells, whereas, AMPK signalling could be acting as protector against it (favouring normal cells). Hence it would be interesting to explore the functions and pathways associated with mTOR, AMPK, MAPK8 and JUNB in myelopoiesis related diseases like MDS.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36322239"
                        open
                      >
                        <summary class="article-expander__title">Over expression of mTOR gene predicts overall survival in myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36322239/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221103212023&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our study showed an elevation of TNF and AMPK signalling pathways in MDS. TNF signalling might be mediating the proliferative advantage to myeloid clonal cells (mutation carrying cells) over normal cells, whereas, AMPK signalling could be acting as protector against it (favouring normal cells). Hence it would be interesting to explore the functions and pathways associated with mTOR, AMPK, MAPK8 and JUNB in myelopoiesis related diseases like MDS.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36322239"
                        open
                      >
                        <summary class="article-expander__title">Over expression of mTOR gene predicts overall survival in myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36322239/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221102211743&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our study showed an elevation of TNF and AMPK signalling pathways in MDS. TNF signalling might be mediating the proliferative advantage to myeloid clonal cells (mutation carrying cells) over normal cells, whereas, AMPK signalling could be acting as protector against it (favouring normal cells). Hence it would be interesting to explore the functions and pathways associated with mTOR, AMPK, MAPK8 and JUNB in myelopoiesis related diseases like MDS.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221128200439&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.8">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36323507"
                        open
                      >
                        <summary class="article-expander__title">KIT genetic alterations in breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36323507/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221122200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: KIT genetic alterations are vanishingly rare in BC. KIT-altered BCs are of high grade but lack distinctive histological features. Genetic alterations in KIT might be late events in the evolution and/or progression of BC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36322938"
                        open
                      >
                        <summary class="article-expander__title">Single-cell methods in myeloproliferative neoplasms - old questions, new technologies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36322938/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221122200439&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are a group of clonal, stem cell derived hematopoietic malignancies driven by aberrant JAK/STAT signalling. Although they are genetically simple diseases, MPNs are phenotypically heterogeneous, reflecting underlying intra-tumoral heterogeneity driven by the interplay of genetic and non-genetic factors. As their evolution is determined by factors that enable certain cellular subsets to outcompete others, techniques that resolve cellular heterogeneity at the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36323507"
                        open
                      >
                        <summary class="article-expander__title">KIT genetic alterations in breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36323507/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221121201257&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: KIT genetic alterations are vanishingly rare in BC. KIT-altered BCs are of high grade but lack distinctive histological features. Genetic alterations in KIT might be late events in the evolution and/or progression of BC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36322938"
                        open
                      >
                        <summary class="article-expander__title">Single-cell methods in myeloproliferative neoplasms - old questions, new technologies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36322938/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221121201257&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are a group of clonal, stem cell derived hematopoietic malignancies driven by aberrant JAK/STAT signalling. Although they are genetically simple diseases, MPNs are phenotypically heterogeneous, reflecting underlying intra-tumoral heterogeneity driven by the interplay of genetic and non-genetic factors. As their evolution is determined by factors that enable certain cellular subsets to outcompete others, techniques that resolve cellular heterogeneity at the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36323507"
                        open
                      >
                        <summary class="article-expander__title">KIT genetic alterations in breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36323507/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221120201223&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: KIT genetic alterations are vanishingly rare in BC. KIT-altered BCs are of high grade but lack distinctive histological features. Genetic alterations in KIT might be late events in the evolution and/or progression of BC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36322938"
                        open
                      >
                        <summary class="article-expander__title">Single-cell methods in myeloproliferative neoplasms - old questions, new technologies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36322938/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221120201223&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are a group of clonal, stem cell derived hematopoietic malignancies driven by aberrant JAK/STAT signalling. Although they are genetically simple diseases, MPNs are phenotypically heterogeneous, reflecting underlying intra-tumoral heterogeneity driven by the interplay of genetic and non-genetic factors. As their evolution is determined by factors that enable certain cellular subsets to outcompete others, techniques that resolve cellular heterogeneity at the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36323507"
                        open
                      >
                        <summary class="article-expander__title">KIT genetic alterations in breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36323507/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221119201544&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: KIT genetic alterations are vanishingly rare in BC. KIT-altered BCs are of high grade but lack distinctive histological features. Genetic alterations in KIT might be late events in the evolution and/or progression of BC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36322938"
                        open
                      >
                        <summary class="article-expander__title">Single-cell methods in myeloproliferative neoplasms - old questions, new technologies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36322938/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221119201544&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are a group of clonal, stem cell derived hematopoietic malignancies driven by aberrant JAK/STAT signalling. Although they are genetically simple diseases, MPNs are phenotypically heterogeneous, reflecting underlying intra-tumoral heterogeneity driven by the interplay of genetic and non-genetic factors. As their evolution is determined by factors that enable certain cellular subsets to outcompete others, techniques that resolve cellular heterogeneity at the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36323507"
                        open
                      >
                        <summary class="article-expander__title">KIT genetic alterations in breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36323507/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221118201229&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: KIT genetic alterations are vanishingly rare in BC. KIT-altered BCs are of high grade but lack distinctive histological features. Genetic alterations in KIT might be late events in the evolution and/or progression of BC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36322938"
                        open
                      >
                        <summary class="article-expander__title">Single-cell methods in myeloproliferative neoplasms - old questions, new technologies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36322938/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221118201229&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are a group of clonal, stem cell derived hematopoietic malignancies driven by aberrant JAK/STAT signalling. Although they are genetically simple diseases, MPNs are phenotypically heterogeneous, reflecting underlying intra-tumoral heterogeneity driven by the interplay of genetic and non-genetic factors. As their evolution is determined by factors that enable certain cellular subsets to outcompete others, techniques that resolve cellular heterogeneity at the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36323507"
                        open
                      >
                        <summary class="article-expander__title">KIT genetic alterations in breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36323507/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221117201522&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: KIT genetic alterations are vanishingly rare in BC. KIT-altered BCs are of high grade but lack distinctive histological features. Genetic alterations in KIT might be late events in the evolution and/or progression of BC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36322938"
                        open
                      >
                        <summary class="article-expander__title">Single-cell methods in myeloproliferative neoplasms - old questions, new technologies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36322938/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221117201522&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are a group of clonal, stem cell derived hematopoietic malignancies driven by aberrant JAK/STAT signalling. Although they are genetically simple diseases, MPNs are phenotypically heterogeneous, reflecting underlying intra-tumoral heterogeneity driven by the interplay of genetic and non-genetic factors. As their evolution is determined by factors that enable certain cellular subsets to outcompete others, techniques that resolve cellular heterogeneity at the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36323507"
                        open
                      >
                        <summary class="article-expander__title">KIT genetic alterations in breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36323507/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221116201100&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: KIT genetic alterations are vanishingly rare in BC. KIT-altered BCs are of high grade but lack distinctive histological features. Genetic alterations in KIT might be late events in the evolution and/or progression of BC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36322938"
                        open
                      >
                        <summary class="article-expander__title">Single-cell methods in myeloproliferative neoplasms - old questions, new technologies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36322938/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221116201100&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are a group of clonal, stem cell derived hematopoietic malignancies driven by aberrant JAK/STAT signalling. Although they are genetically simple diseases, MPNs are phenotypically heterogeneous, reflecting underlying intra-tumoral heterogeneity driven by the interplay of genetic and non-genetic factors. As their evolution is determined by factors that enable certain cellular subsets to outcompete others, techniques that resolve cellular heterogeneity at the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36323507"
                        open
                      >
                        <summary class="article-expander__title">KIT genetic alterations in breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36323507/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221115201214&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: KIT genetic alterations are vanishingly rare in BC. KIT-altered BCs are of high grade but lack distinctive histological features. Genetic alterations in KIT might be late events in the evolution and/or progression of BC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36322938"
                        open
                      >
                        <summary class="article-expander__title">Single-cell methods in myeloproliferative neoplasms - old questions, new technologies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36322938/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221115201214&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are a group of clonal, stem cell derived hematopoietic malignancies driven by aberrant JAK/STAT signalling. Although they are genetically simple diseases, MPNs are phenotypically heterogeneous, reflecting underlying intra-tumoral heterogeneity driven by the interplay of genetic and non-genetic factors. As their evolution is determined by factors that enable certain cellular subsets to outcompete others, techniques that resolve cellular heterogeneity at the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36323507"
                        open
                      >
                        <summary class="article-expander__title">KIT genetic alterations in breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36323507/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221114201139&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: KIT genetic alterations are vanishingly rare in BC. KIT-altered BCs are of high grade but lack distinctive histological features. Genetic alterations in KIT might be late events in the evolution and/or progression of BC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36322938"
                        open
                      >
                        <summary class="article-expander__title">Single-cell methods in myeloproliferative neoplasms - old questions, new technologies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36322938/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221114201139&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are a group of clonal, stem cell derived hematopoietic malignancies driven by aberrant JAK/STAT signalling. Although they are genetically simple diseases, MPNs are phenotypically heterogeneous, reflecting underlying intra-tumoral heterogeneity driven by the interplay of genetic and non-genetic factors. As their evolution is determined by factors that enable certain cellular subsets to outcompete others, techniques that resolve cellular heterogeneity at the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36323507"
                        open
                      >
                        <summary class="article-expander__title">KIT genetic alterations in breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36323507/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221113201202&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: KIT genetic alterations are vanishingly rare in BC. KIT-altered BCs are of high grade but lack distinctive histological features. Genetic alterations in KIT might be late events in the evolution and/or progression of BC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36322938"
                        open
                      >
                        <summary class="article-expander__title">Single-cell methods in myeloproliferative neoplasms - old questions, new technologies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36322938/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221113201202&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are a group of clonal, stem cell derived hematopoietic malignancies driven by aberrant JAK/STAT signalling. Although they are genetically simple diseases, MPNs are phenotypically heterogeneous, reflecting underlying intra-tumoral heterogeneity driven by the interplay of genetic and non-genetic factors. As their evolution is determined by factors that enable certain cellular subsets to outcompete others, techniques that resolve cellular heterogeneity at the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36323507"
                        open
                      >
                        <summary class="article-expander__title">KIT genetic alterations in breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36323507/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221112201337&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: KIT genetic alterations are vanishingly rare in BC. KIT-altered BCs are of high grade but lack distinctive histological features. Genetic alterations in KIT might be late events in the evolution and/or progression of BC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36322938"
                        open
                      >
                        <summary class="article-expander__title">Single-cell methods in myeloproliferative neoplasms - old questions, new technologies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36322938/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221112201337&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are a group of clonal, stem cell derived hematopoietic malignancies driven by aberrant JAK/STAT signalling. Although they are genetically simple diseases, MPNs are phenotypically heterogeneous, reflecting underlying intra-tumoral heterogeneity driven by the interplay of genetic and non-genetic factors. As their evolution is determined by factors that enable certain cellular subsets to outcompete others, techniques that resolve cellular heterogeneity at the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36323507"
                        open
                      >
                        <summary class="article-expander__title">KIT genetic alterations in breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36323507/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221111201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: KIT genetic alterations are vanishingly rare in BC. KIT-altered BCs are of high grade but lack distinctive histological features. Genetic alterations in KIT might be late events in the evolution and/or progression of BC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36322938"
                        open
                      >
                        <summary class="article-expander__title">Single-cell methods in myeloproliferative neoplasms - old questions, new technologies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36322938/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221111201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are a group of clonal, stem cell derived hematopoietic malignancies driven by aberrant JAK/STAT signalling. Although they are genetically simple diseases, MPNs are phenotypically heterogeneous, reflecting underlying intra-tumoral heterogeneity driven by the interplay of genetic and non-genetic factors. As their evolution is determined by factors that enable certain cellular subsets to outcompete others, techniques that resolve cellular heterogeneity at the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36323507"
                        open
                      >
                        <summary class="article-expander__title">KIT genetic alterations in breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36323507/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221110201819&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: KIT genetic alterations are vanishingly rare in BC. KIT-altered BCs are of high grade but lack distinctive histological features. Genetic alterations in KIT might be late events in the evolution and/or progression of BC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36322938"
                        open
                      >
                        <summary class="article-expander__title">Single-cell methods in myeloproliferative neoplasms - old questions, new technologies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36322938/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221110201819&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are a group of clonal, stem cell derived hematopoietic malignancies driven by aberrant JAK/STAT signalling. Although they are genetically simple diseases, MPNs are phenotypically heterogeneous, reflecting underlying intra-tumoral heterogeneity driven by the interplay of genetic and non-genetic factors. As their evolution is determined by factors that enable certain cellular subsets to outcompete others, techniques that resolve cellular heterogeneity at the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36323507"
                        open
                      >
                        <summary class="article-expander__title">KIT genetic alterations in breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36323507/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221109201530&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: KIT genetic alterations are vanishingly rare in BC. KIT-altered BCs are of high grade but lack distinctive histological features. Genetic alterations in KIT might be late events in the evolution and/or progression of BC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36322938"
                        open
                      >
                        <summary class="article-expander__title">Single-cell methods in myeloproliferative neoplasms - old questions, new technologies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36322938/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221109201530&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are a group of clonal, stem cell derived hematopoietic malignancies driven by aberrant JAK/STAT signalling. Although they are genetically simple diseases, MPNs are phenotypically heterogeneous, reflecting underlying intra-tumoral heterogeneity driven by the interplay of genetic and non-genetic factors. As their evolution is determined by factors that enable certain cellular subsets to outcompete others, techniques that resolve cellular heterogeneity at the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36323507"
                        open
                      >
                        <summary class="article-expander__title">KIT genetic alterations in breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36323507/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221108201514&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: KIT genetic alterations are vanishingly rare in BC. KIT-altered BCs are of high grade but lack distinctive histological features. Genetic alterations in KIT might be late events in the evolution and/or progression of BC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36322938"
                        open
                      >
                        <summary class="article-expander__title">Single-cell methods in myeloproliferative neoplasms - old questions, new technologies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36322938/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221108201514&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are a group of clonal, stem cell derived hematopoietic malignancies driven by aberrant JAK/STAT signalling. Although they are genetically simple diseases, MPNs are phenotypically heterogeneous, reflecting underlying intra-tumoral heterogeneity driven by the interplay of genetic and non-genetic factors. As their evolution is determined by factors that enable certain cellular subsets to outcompete others, techniques that resolve cellular heterogeneity at the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36323507"
                        open
                      >
                        <summary class="article-expander__title">KIT genetic alterations in breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36323507/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221107201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: KIT genetic alterations are vanishingly rare in BC. KIT-altered BCs are of high grade but lack distinctive histological features. Genetic alterations in KIT might be late events in the evolution and/or progression of BC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36322938"
                        open
                      >
                        <summary class="article-expander__title">Single-cell methods in myeloproliferative neoplasms - old questions, new technologies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36322938/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221107201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are a group of clonal, stem cell derived hematopoietic malignancies driven by aberrant JAK/STAT signalling. Although they are genetically simple diseases, MPNs are phenotypically heterogeneous, reflecting underlying intra-tumoral heterogeneity driven by the interplay of genetic and non-genetic factors. As their evolution is determined by factors that enable certain cellular subsets to outcompete others, techniques that resolve cellular heterogeneity at the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36323507"
                        open
                      >
                        <summary class="article-expander__title">KIT genetic alterations in breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36323507/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221106201343&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: KIT genetic alterations are vanishingly rare in BC. KIT-altered BCs are of high grade but lack distinctive histological features. Genetic alterations in KIT might be late events in the evolution and/or progression of BC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36322938"
                        open
                      >
                        <summary class="article-expander__title">Single-cell methods in myeloproliferative neoplasms - old questions, new technologies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36322938/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221106201343&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are a group of clonal, stem cell derived hematopoietic malignancies driven by aberrant JAK/STAT signalling. Although they are genetically simple diseases, MPNs are phenotypically heterogeneous, reflecting underlying intra-tumoral heterogeneity driven by the interplay of genetic and non-genetic factors. As their evolution is determined by factors that enable certain cellular subsets to outcompete others, techniques that resolve cellular heterogeneity at the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36323507"
                        open
                      >
                        <summary class="article-expander__title">KIT genetic alterations in breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36323507/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221105211555&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: KIT genetic alterations are vanishingly rare in BC. KIT-altered BCs are of high grade but lack distinctive histological features. Genetic alterations in KIT might be late events in the evolution and/or progression of BC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36322938"
                        open
                      >
                        <summary class="article-expander__title">Single-cell methods in myeloproliferative neoplasms - old questions, new technologies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36322938/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221105211555&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are a group of clonal, stem cell derived hematopoietic malignancies driven by aberrant JAK/STAT signalling. Although they are genetically simple diseases, MPNs are phenotypically heterogeneous, reflecting underlying intra-tumoral heterogeneity driven by the interplay of genetic and non-genetic factors. As their evolution is determined by factors that enable certain cellular subsets to outcompete others, techniques that resolve cellular heterogeneity at the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36323507"
                        open
                      >
                        <summary class="article-expander__title">KIT genetic alterations in breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36323507/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221104211729&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: KIT genetic alterations are vanishingly rare in BC. KIT-altered BCs are of high grade but lack distinctive histological features. Genetic alterations in KIT might be late events in the evolution and/or progression of BC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36322938"
                        open
                      >
                        <summary class="article-expander__title">Single-cell methods in myeloproliferative neoplasms - old questions, new technologies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36322938/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221104211729&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are a group of clonal, stem cell derived hematopoietic malignancies driven by aberrant JAK/STAT signalling. Although they are genetically simple diseases, MPNs are phenotypically heterogeneous, reflecting underlying intra-tumoral heterogeneity driven by the interplay of genetic and non-genetic factors. As their evolution is determined by factors that enable certain cellular subsets to outcompete others, techniques that resolve cellular heterogeneity at the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36323507"
                        open
                      >
                        <summary class="article-expander__title">KIT genetic alterations in breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36323507/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221103212023&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: KIT genetic alterations are vanishingly rare in BC. KIT-altered BCs are of high grade but lack distinctive histological features. Genetic alterations in KIT might be late events in the evolution and/or progression of BC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36322938"
                        open
                      >
                        <summary class="article-expander__title">Single-cell methods in myeloproliferative neoplasms - old questions, new technologies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36322938/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221103212023&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are a group of clonal, stem cell derived hematopoietic malignancies driven by aberrant JAK/STAT signalling. Although they are genetically simple diseases, MPNs are phenotypically heterogeneous, reflecting underlying intra-tumoral heterogeneity driven by the interplay of genetic and non-genetic factors. As their evolution is determined by factors that enable certain cellular subsets to outcompete others, techniques that resolve cellular heterogeneity at the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36322938"
                        open
                      >
                        <summary class="article-expander__title">Single-cell methods in myeloproliferative neoplasms - old questions, new technologies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36322938/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221102211743&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are a group of clonal, stem cell derived hematopoietic malignancies driven by aberrant JAK/STAT signalling. Although they are genetically simple diseases, MPNs are phenotypically heterogeneous, reflecting underlying intra-tumoral heterogeneity driven by the interplay of genetic and non-genetic factors. As their evolution is determined by factors that enable certain cellular subsets to outcompete others, techniques that resolve cellular heterogeneity at the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-31 datetime="2022-10-31T10:00:00.000Z">2022-10-31</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-31 datetime="2022-10-31T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.8&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20221128200439&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36311800"
                        open
                      >
                        <summary class="article-expander__title">Increased prevalence of clonal hematopoiesis of indeterminate potential in hospitalized patients with COVID-19</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36311800/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221116201100&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) leads to higher mortality, carries a cardiovascular risk and alters inflammation. All three aspects harbor overlaps with the clinical manifestation of COVID-19. This study aimed to identify the impact of CHIP on COVID-19 pathophysiology. 90 hospitalized patients were analyzed for CHIP. In addition, their disease course and outcome were evaluated. With a prevalence of 37.8%, the frequency of a CHIP-driver mutation was significantly higher...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36311800"
                        open
                      >
                        <summary class="article-expander__title">Increased prevalence of clonal hematopoiesis of indeterminate potential in hospitalized patients with COVID-19</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36311800/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221115201214&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) leads to higher mortality, carries a cardiovascular risk and alters inflammation. All three aspects harbor overlaps with the clinical manifestation of COVID-19. This study aimed to identify the impact of CHIP on COVID-19 pathophysiology. 90 hospitalized patients were analyzed for CHIP. In addition, their disease course and outcome were evaluated. With a prevalence of 37.8%, the frequency of a CHIP-driver mutation was significantly higher...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36311800"
                        open
                      >
                        <summary class="article-expander__title">Increased prevalence of clonal hematopoiesis of indeterminate potential in hospitalized patients with COVID-19</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36311800/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221114201139&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) leads to higher mortality, carries a cardiovascular risk and alters inflammation. All three aspects harbor overlaps with the clinical manifestation of COVID-19. This study aimed to identify the impact of CHIP on COVID-19 pathophysiology. 90 hospitalized patients were analyzed for CHIP. In addition, their disease course and outcome were evaluated. With a prevalence of 37.8%, the frequency of a CHIP-driver mutation was significantly higher...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36311538"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in understanding telomere diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36311538/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221114201139&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline genetic defects impairing telomere length maintenance may result in severe medical conditions in humans, from aplastic anemia and myeloid neoplasms to interstitial lung disease and liver cirrhosis, from childhood (dyskeratosis congenita) to old age (pulmonary fibrosis). The molecular mechanisms underlying these clinically distinct disorders are pathologically excessive telomere erosion, limiting cell proliferation and differentiation, tissue regeneration, and increasing genomic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36311800"
                        open
                      >
                        <summary class="article-expander__title">Increased prevalence of clonal hematopoiesis of indeterminate potential in hospitalized patients with COVID-19</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36311800/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221113201202&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) leads to higher mortality, carries a cardiovascular risk and alters inflammation. All three aspects harbor overlaps with the clinical manifestation of COVID-19. This study aimed to identify the impact of CHIP on COVID-19 pathophysiology. 90 hospitalized patients were analyzed for CHIP. In addition, their disease course and outcome were evaluated. With a prevalence of 37.8%, the frequency of a CHIP-driver mutation was significantly higher...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36311538"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in understanding telomere diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36311538/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221113201202&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline genetic defects impairing telomere length maintenance may result in severe medical conditions in humans, from aplastic anemia and myeloid neoplasms to interstitial lung disease and liver cirrhosis, from childhood (dyskeratosis congenita) to old age (pulmonary fibrosis). The molecular mechanisms underlying these clinically distinct disorders are pathologically excessive telomere erosion, limiting cell proliferation and differentiation, tissue regeneration, and increasing genomic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36311800"
                        open
                      >
                        <summary class="article-expander__title">Increased prevalence of clonal hematopoiesis of indeterminate potential in hospitalized patients with COVID-19</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36311800/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221112201337&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) leads to higher mortality, carries a cardiovascular risk and alters inflammation. All three aspects harbor overlaps with the clinical manifestation of COVID-19. This study aimed to identify the impact of CHIP on COVID-19 pathophysiology. 90 hospitalized patients were analyzed for CHIP. In addition, their disease course and outcome were evaluated. With a prevalence of 37.8%, the frequency of a CHIP-driver mutation was significantly higher...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36311538"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in understanding telomere diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36311538/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221112201337&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline genetic defects impairing telomere length maintenance may result in severe medical conditions in humans, from aplastic anemia and myeloid neoplasms to interstitial lung disease and liver cirrhosis, from childhood (dyskeratosis congenita) to old age (pulmonary fibrosis). The molecular mechanisms underlying these clinically distinct disorders are pathologically excessive telomere erosion, limiting cell proliferation and differentiation, tissue regeneration, and increasing genomic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36311800"
                        open
                      >
                        <summary class="article-expander__title">Increased prevalence of clonal hematopoiesis of indeterminate potential in hospitalized patients with COVID-19</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36311800/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221111201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) leads to higher mortality, carries a cardiovascular risk and alters inflammation. All three aspects harbor overlaps with the clinical manifestation of COVID-19. This study aimed to identify the impact of CHIP on COVID-19 pathophysiology. 90 hospitalized patients were analyzed for CHIP. In addition, their disease course and outcome were evaluated. With a prevalence of 37.8%, the frequency of a CHIP-driver mutation was significantly higher...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36311538"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in understanding telomere diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36311538/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221111201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline genetic defects impairing telomere length maintenance may result in severe medical conditions in humans, from aplastic anemia and myeloid neoplasms to interstitial lung disease and liver cirrhosis, from childhood (dyskeratosis congenita) to old age (pulmonary fibrosis). The molecular mechanisms underlying these clinically distinct disorders are pathologically excessive telomere erosion, limiting cell proliferation and differentiation, tissue regeneration, and increasing genomic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36311800"
                        open
                      >
                        <summary class="article-expander__title">Increased prevalence of clonal hematopoiesis of indeterminate potential in hospitalized patients with COVID-19</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36311800/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221110201819&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) leads to higher mortality, carries a cardiovascular risk and alters inflammation. All three aspects harbor overlaps with the clinical manifestation of COVID-19. This study aimed to identify the impact of CHIP on COVID-19 pathophysiology. 90 hospitalized patients were analyzed for CHIP. In addition, their disease course and outcome were evaluated. With a prevalence of 37.8%, the frequency of a CHIP-driver mutation was significantly higher...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36311538"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in understanding telomere diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36311538/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221110201819&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline genetic defects impairing telomere length maintenance may result in severe medical conditions in humans, from aplastic anemia and myeloid neoplasms to interstitial lung disease and liver cirrhosis, from childhood (dyskeratosis congenita) to old age (pulmonary fibrosis). The molecular mechanisms underlying these clinically distinct disorders are pathologically excessive telomere erosion, limiting cell proliferation and differentiation, tissue regeneration, and increasing genomic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36311800"
                        open
                      >
                        <summary class="article-expander__title">Increased prevalence of clonal hematopoiesis of indeterminate potential in hospitalized patients with COVID-19</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36311800/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221109201530&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) leads to higher mortality, carries a cardiovascular risk and alters inflammation. All three aspects harbor overlaps with the clinical manifestation of COVID-19. This study aimed to identify the impact of CHIP on COVID-19 pathophysiology. 90 hospitalized patients were analyzed for CHIP. In addition, their disease course and outcome were evaluated. With a prevalence of 37.8%, the frequency of a CHIP-driver mutation was significantly higher...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36311538"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in understanding telomere diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36311538/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221109201530&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline genetic defects impairing telomere length maintenance may result in severe medical conditions in humans, from aplastic anemia and myeloid neoplasms to interstitial lung disease and liver cirrhosis, from childhood (dyskeratosis congenita) to old age (pulmonary fibrosis). The molecular mechanisms underlying these clinically distinct disorders are pathologically excessive telomere erosion, limiting cell proliferation and differentiation, tissue regeneration, and increasing genomic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36311800"
                        open
                      >
                        <summary class="article-expander__title">Increased prevalence of clonal hematopoiesis of indeterminate potential in hospitalized patients with COVID-19</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36311800/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221108201514&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) leads to higher mortality, carries a cardiovascular risk and alters inflammation. All three aspects harbor overlaps with the clinical manifestation of COVID-19. This study aimed to identify the impact of CHIP on COVID-19 pathophysiology. 90 hospitalized patients were analyzed for CHIP. In addition, their disease course and outcome were evaluated. With a prevalence of 37.8%, the frequency of a CHIP-driver mutation was significantly higher...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36311538"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in understanding telomere diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36311538/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221108201514&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline genetic defects impairing telomere length maintenance may result in severe medical conditions in humans, from aplastic anemia and myeloid neoplasms to interstitial lung disease and liver cirrhosis, from childhood (dyskeratosis congenita) to old age (pulmonary fibrosis). The molecular mechanisms underlying these clinically distinct disorders are pathologically excessive telomere erosion, limiting cell proliferation and differentiation, tissue regeneration, and increasing genomic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36311800"
                        open
                      >
                        <summary class="article-expander__title">Increased prevalence of clonal hematopoiesis of indeterminate potential in hospitalized patients with COVID-19</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36311800/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221107201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) leads to higher mortality, carries a cardiovascular risk and alters inflammation. All three aspects harbor overlaps with the clinical manifestation of COVID-19. This study aimed to identify the impact of CHIP on COVID-19 pathophysiology. 90 hospitalized patients were analyzed for CHIP. In addition, their disease course and outcome were evaluated. With a prevalence of 37.8%, the frequency of a CHIP-driver mutation was significantly higher...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36311538"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in understanding telomere diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36311538/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221107201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline genetic defects impairing telomere length maintenance may result in severe medical conditions in humans, from aplastic anemia and myeloid neoplasms to interstitial lung disease and liver cirrhosis, from childhood (dyskeratosis congenita) to old age (pulmonary fibrosis). The molecular mechanisms underlying these clinically distinct disorders are pathologically excessive telomere erosion, limiting cell proliferation and differentiation, tissue regeneration, and increasing genomic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36311800"
                        open
                      >
                        <summary class="article-expander__title">Increased prevalence of clonal hematopoiesis of indeterminate potential in hospitalized patients with COVID-19</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36311800/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221106201343&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) leads to higher mortality, carries a cardiovascular risk and alters inflammation. All three aspects harbor overlaps with the clinical manifestation of COVID-19. This study aimed to identify the impact of CHIP on COVID-19 pathophysiology. 90 hospitalized patients were analyzed for CHIP. In addition, their disease course and outcome were evaluated. With a prevalence of 37.8%, the frequency of a CHIP-driver mutation was significantly higher...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36311538"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in understanding telomere diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36311538/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221106201343&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline genetic defects impairing telomere length maintenance may result in severe medical conditions in humans, from aplastic anemia and myeloid neoplasms to interstitial lung disease and liver cirrhosis, from childhood (dyskeratosis congenita) to old age (pulmonary fibrosis). The molecular mechanisms underlying these clinically distinct disorders are pathologically excessive telomere erosion, limiting cell proliferation and differentiation, tissue regeneration, and increasing genomic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36311800"
                        open
                      >
                        <summary class="article-expander__title">Increased prevalence of clonal hematopoiesis of indeterminate potential in hospitalized patients with COVID-19</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36311800/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221105211555&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) leads to higher mortality, carries a cardiovascular risk and alters inflammation. All three aspects harbor overlaps with the clinical manifestation of COVID-19. This study aimed to identify the impact of CHIP on COVID-19 pathophysiology. 90 hospitalized patients were analyzed for CHIP. In addition, their disease course and outcome were evaluated. With a prevalence of 37.8%, the frequency of a CHIP-driver mutation was significantly higher...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36311538"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in understanding telomere diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36311538/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221105211555&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline genetic defects impairing telomere length maintenance may result in severe medical conditions in humans, from aplastic anemia and myeloid neoplasms to interstitial lung disease and liver cirrhosis, from childhood (dyskeratosis congenita) to old age (pulmonary fibrosis). The molecular mechanisms underlying these clinically distinct disorders are pathologically excessive telomere erosion, limiting cell proliferation and differentiation, tissue regeneration, and increasing genomic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36311800"
                        open
                      >
                        <summary class="article-expander__title">Increased prevalence of clonal hematopoiesis of indeterminate potential in hospitalized patients with COVID-19</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36311800/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221104211730&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) leads to higher mortality, carries a cardiovascular risk and alters inflammation. All three aspects harbor overlaps with the clinical manifestation of COVID-19. This study aimed to identify the impact of CHIP on COVID-19 pathophysiology. 90 hospitalized patients were analyzed for CHIP. In addition, their disease course and outcome were evaluated. With a prevalence of 37.8%, the frequency of a CHIP-driver mutation was significantly higher...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36311538"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in understanding telomere diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36311538/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221104211730&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline genetic defects impairing telomere length maintenance may result in severe medical conditions in humans, from aplastic anemia and myeloid neoplasms to interstitial lung disease and liver cirrhosis, from childhood (dyskeratosis congenita) to old age (pulmonary fibrosis). The molecular mechanisms underlying these clinically distinct disorders are pathologically excessive telomere erosion, limiting cell proliferation and differentiation, tissue regeneration, and increasing genomic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36311800"
                        open
                      >
                        <summary class="article-expander__title">Increased prevalence of clonal hematopoiesis of indeterminate potential in hospitalized patients with COVID-19</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36311800/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221103212023&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) leads to higher mortality, carries a cardiovascular risk and alters inflammation. All three aspects harbor overlaps with the clinical manifestation of COVID-19. This study aimed to identify the impact of CHIP on COVID-19 pathophysiology. 90 hospitalized patients were analyzed for CHIP. In addition, their disease course and outcome were evaluated. With a prevalence of 37.8%, the frequency of a CHIP-driver mutation was significantly higher...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36311538"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in understanding telomere diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36311538/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221103212023&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline genetic defects impairing telomere length maintenance may result in severe medical conditions in humans, from aplastic anemia and myeloid neoplasms to interstitial lung disease and liver cirrhosis, from childhood (dyskeratosis congenita) to old age (pulmonary fibrosis). The molecular mechanisms underlying these clinically distinct disorders are pathologically excessive telomere erosion, limiting cell proliferation and differentiation, tissue regeneration, and increasing genomic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36311800"
                        open
                      >
                        <summary class="article-expander__title">Increased prevalence of clonal hematopoiesis of indeterminate potential in hospitalized patients with COVID-19</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36311800/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221102211743&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) leads to higher mortality, carries a cardiovascular risk and alters inflammation. All three aspects harbor overlaps with the clinical manifestation of COVID-19. This study aimed to identify the impact of CHIP on COVID-19 pathophysiology. 90 hospitalized patients were analyzed for CHIP. In addition, their disease course and outcome were evaluated. With a prevalence of 37.8%, the frequency of a CHIP-driver mutation was significantly higher...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36311538"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in understanding telomere diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36311538/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221102211743&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline genetic defects impairing telomere length maintenance may result in severe medical conditions in humans, from aplastic anemia and myeloid neoplasms to interstitial lung disease and liver cirrhosis, from childhood (dyskeratosis congenita) to old age (pulmonary fibrosis). The molecular mechanisms underlying these clinically distinct disorders are pathologically excessive telomere erosion, limiting cell proliferation and differentiation, tissue regeneration, and increasing genomic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36311800"
                        open
                      >
                        <summary class="article-expander__title">Increased prevalence of clonal hematopoiesis of indeterminate potential in hospitalized patients with COVID-19</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36311800/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221101212552&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) leads to higher mortality, carries a cardiovascular risk and alters inflammation. All three aspects harbor overlaps with the clinical manifestation of COVID-19. This study aimed to identify the impact of CHIP on COVID-19 pathophysiology. 90 hospitalized patients were analyzed for CHIP. In addition, their disease course and outcome were evaluated. With a prevalence of 37.8%, the frequency of a CHIP-driver mutation was significantly higher...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36311538"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in understanding telomere diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36311538/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221101212552&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline genetic defects impairing telomere length maintenance may result in severe medical conditions in humans, from aplastic anemia and myeloid neoplasms to interstitial lung disease and liver cirrhosis, from childhood (dyskeratosis congenita) to old age (pulmonary fibrosis). The molecular mechanisms underlying these clinically distinct disorders are pathologically excessive telomere erosion, limiting cell proliferation and differentiation, tissue regeneration, and increasing genomic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36311800"
                        open
                      >
                        <summary class="article-expander__title">Increased prevalence of clonal hematopoiesis of indeterminate potential in hospitalized patients with COVID-19</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36311800/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221031214247&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) leads to higher mortality, carries a cardiovascular risk and alters inflammation. All three aspects harbor overlaps with the clinical manifestation of COVID-19. This study aimed to identify the impact of CHIP on COVID-19 pathophysiology. 90 hospitalized patients were analyzed for CHIP. In addition, their disease course and outcome were evaluated. With a prevalence of 37.8%, the frequency of a CHIP-driver mutation was significantly higher...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36311538"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in understanding telomere diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36311538/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221031214247&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline genetic defects impairing telomere length maintenance may result in severe medical conditions in humans, from aplastic anemia and myeloid neoplasms to interstitial lung disease and liver cirrhosis, from childhood (dyskeratosis congenita) to old age (pulmonary fibrosis). The molecular mechanisms underlying these clinically distinct disorders are pathologically excessive telomere erosion, limiting cell proliferation and differentiation, tissue regeneration, and increasing genomic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221128200439&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.8">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36315915"
                        open
                      >
                        <summary class="article-expander__title">APOBEC Mutational Signatures in Hormone Receptor-Positive Human Epidermal Growth Factor Receptor 2-Negative Breast Cancers Are Associated With Poor Outcomes on CDK4/6 Inhibitors and Endocrine Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36315915/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221120201223&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: APOBEC+ HR+ HER2- patients had shorter TTD on first-line ET plus CDK4/6i relative to APOBEC- patients. Further research is needed to optimize the treatment of APOBEC+ HR+ HER2- BC and to investigate the efficacy of immunotherapeutic strategies in this population.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36315915"
                        open
                      >
                        <summary class="article-expander__title">APOBEC Mutational Signatures in Hormone Receptor-Positive Human Epidermal Growth Factor Receptor 2-Negative Breast Cancers Are Associated With Poor Outcomes on CDK4/6 Inhibitors and Endocrine Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36315915/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221119201544&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: APOBEC+ HR+ HER2- patients had shorter TTD on first-line ET plus CDK4/6i relative to APOBEC- patients. Further research is needed to optimize the treatment of APOBEC+ HR+ HER2- BC and to investigate the efficacy of immunotherapeutic strategies in this population.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36315915"
                        open
                      >
                        <summary class="article-expander__title">APOBEC Mutational Signatures in Hormone Receptor-Positive Human Epidermal Growth Factor Receptor 2-Negative Breast Cancers Are Associated With Poor Outcomes on CDK4/6 Inhibitors and Endocrine Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36315915/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221118201229&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: APOBEC+ HR+ HER2- patients had shorter TTD on first-line ET plus CDK4/6i relative to APOBEC- patients. Further research is needed to optimize the treatment of APOBEC+ HR+ HER2- BC and to investigate the efficacy of immunotherapeutic strategies in this population.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36315915"
                        open
                      >
                        <summary class="article-expander__title">APOBEC Mutational Signatures in Hormone Receptor-Positive Human Epidermal Growth Factor Receptor 2-Negative Breast Cancers Are Associated With Poor Outcomes on CDK4/6 Inhibitors and Endocrine Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36315915/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221117201522&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: APOBEC+ HR+ HER2- patients had shorter TTD on first-line ET plus CDK4/6i relative to APOBEC- patients. Further research is needed to optimize the treatment of APOBEC+ HR+ HER2- BC and to investigate the efficacy of immunotherapeutic strategies in this population.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36311816"
                        open
                      >
                        <summary class="article-expander__title">Uterine lavage identifies cancer mutations and increased TP53 somatic mutation burden in individuals with ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36311816/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221117201522&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Current screening methods for ovarian cancer (OC) have failed to demonstrate a significant reduction in mortality. Uterine lavage combined with TP53 ultra-deep sequencing for the detection of disseminated OC cells has emerged as a promising tool, but this approach has not been tested for early-stage disease or non-serous histologies. In addition, lavages carry multiple background mutations, the significance of which is poorly understood. Uterine lavage was collected preoperatively in 34 patients...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36315915"
                        open
                      >
                        <summary class="article-expander__title">APOBEC Mutational Signatures in Hormone Receptor-Positive Human Epidermal Growth Factor Receptor 2-Negative Breast Cancers Are Associated With Poor Outcomes on CDK4/6 Inhibitors and Endocrine Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36315915/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221116201100&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: APOBEC+ HR+ HER2- patients had shorter TTD on first-line ET plus CDK4/6i relative to APOBEC- patients. Further research is needed to optimize the treatment of APOBEC+ HR+ HER2- BC and to investigate the efficacy of immunotherapeutic strategies in this population.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36311816"
                        open
                      >
                        <summary class="article-expander__title">Uterine lavage identifies cancer mutations and increased TP53 somatic mutation burden in individuals with ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36311816/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221116201100&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Current screening methods for ovarian cancer (OC) have failed to demonstrate a significant reduction in mortality. Uterine lavage combined with TP53 ultra-deep sequencing for the detection of disseminated OC cells has emerged as a promising tool, but this approach has not been tested for early-stage disease or non-serous histologies. In addition, lavages carry multiple background mutations, the significance of which is poorly understood. Uterine lavage was collected preoperatively in 34 patients...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36315915"
                        open
                      >
                        <summary class="article-expander__title">APOBEC Mutational Signatures in Hormone Receptor-Positive Human Epidermal Growth Factor Receptor 2-Negative Breast Cancers Are Associated With Poor Outcomes on CDK4/6 Inhibitors and Endocrine Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36315915/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221115201214&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: APOBEC+ HR+ HER2- patients had shorter TTD on first-line ET plus CDK4/6i relative to APOBEC- patients. Further research is needed to optimize the treatment of APOBEC+ HR+ HER2- BC and to investigate the efficacy of immunotherapeutic strategies in this population.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36311816"
                        open
                      >
                        <summary class="article-expander__title">Uterine lavage identifies cancer mutations and increased TP53 somatic mutation burden in individuals with ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36311816/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221115201214&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Current screening methods for ovarian cancer (OC) have failed to demonstrate a significant reduction in mortality. Uterine lavage combined with TP53 ultra-deep sequencing for the detection of disseminated OC cells has emerged as a promising tool, but this approach has not been tested for early-stage disease or non-serous histologies. In addition, lavages carry multiple background mutations, the significance of which is poorly understood. Uterine lavage was collected preoperatively in 34 patients...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36315915"
                        open
                      >
                        <summary class="article-expander__title">APOBEC Mutational Signatures in Hormone Receptor-Positive Human Epidermal Growth Factor Receptor 2-Negative Breast Cancers Are Associated With Poor Outcomes on CDK4/6 Inhibitors and Endocrine Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36315915/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221114201139&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: APOBEC+ HR+ HER2- patients had shorter TTD on first-line ET plus CDK4/6i relative to APOBEC- patients. Further research is needed to optimize the treatment of APOBEC+ HR+ HER2- BC and to investigate the efficacy of immunotherapeutic strategies in this population.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36311816"
                        open
                      >
                        <summary class="article-expander__title">Uterine lavage identifies cancer mutations and increased TP53 somatic mutation burden in individuals with ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36311816/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221114201139&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Current screening methods for ovarian cancer (OC) have failed to demonstrate a significant reduction in mortality. Uterine lavage combined with TP53 ultra-deep sequencing for the detection of disseminated OC cells has emerged as a promising tool, but this approach has not been tested for early-stage disease or non-serous histologies. In addition, lavages carry multiple background mutations, the significance of which is poorly understood. Uterine lavage was collected preoperatively in 34 patients...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36315915"
                        open
                      >
                        <summary class="article-expander__title">APOBEC Mutational Signatures in Hormone Receptor-Positive Human Epidermal Growth Factor Receptor 2-Negative Breast Cancers Are Associated With Poor Outcomes on CDK4/6 Inhibitors and Endocrine Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36315915/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221113201202&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: APOBEC+ HR+ HER2- patients had shorter TTD on first-line ET plus CDK4/6i relative to APOBEC- patients. Further research is needed to optimize the treatment of APOBEC+ HR+ HER2- BC and to investigate the efficacy of immunotherapeutic strategies in this population.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36311816"
                        open
                      >
                        <summary class="article-expander__title">Uterine lavage identifies cancer mutations and increased TP53 somatic mutation burden in individuals with ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36311816/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221113201202&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Current screening methods for ovarian cancer (OC) have failed to demonstrate a significant reduction in mortality. Uterine lavage combined with TP53 ultra-deep sequencing for the detection of disseminated OC cells has emerged as a promising tool, but this approach has not been tested for early-stage disease or non-serous histologies. In addition, lavages carry multiple background mutations, the significance of which is poorly understood. Uterine lavage was collected preoperatively in 34 patients...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36315915"
                        open
                      >
                        <summary class="article-expander__title">APOBEC Mutational Signatures in Hormone Receptor-Positive Human Epidermal Growth Factor Receptor 2-Negative Breast Cancers Are Associated With Poor Outcomes on CDK4/6 Inhibitors and Endocrine Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36315915/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221112201337&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: APOBEC+ HR+ HER2- patients had shorter TTD on first-line ET plus CDK4/6i relative to APOBEC- patients. Further research is needed to optimize the treatment of APOBEC+ HR+ HER2- BC and to investigate the efficacy of immunotherapeutic strategies in this population.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36311816"
                        open
                      >
                        <summary class="article-expander__title">Uterine lavage identifies cancer mutations and increased TP53 somatic mutation burden in individuals with ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36311816/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221112201337&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Current screening methods for ovarian cancer (OC) have failed to demonstrate a significant reduction in mortality. Uterine lavage combined with TP53 ultra-deep sequencing for the detection of disseminated OC cells has emerged as a promising tool, but this approach has not been tested for early-stage disease or non-serous histologies. In addition, lavages carry multiple background mutations, the significance of which is poorly understood. Uterine lavage was collected preoperatively in 34 patients...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36315915"
                        open
                      >
                        <summary class="article-expander__title">APOBEC Mutational Signatures in Hormone Receptor-Positive Human Epidermal Growth Factor Receptor 2-Negative Breast Cancers Are Associated With Poor Outcomes on CDK4/6 Inhibitors and Endocrine Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36315915/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221111201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: APOBEC+ HR+ HER2- patients had shorter TTD on first-line ET plus CDK4/6i relative to APOBEC- patients. Further research is needed to optimize the treatment of APOBEC+ HR+ HER2- BC and to investigate the efficacy of immunotherapeutic strategies in this population.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36311816"
                        open
                      >
                        <summary class="article-expander__title">Uterine lavage identifies cancer mutations and increased TP53 somatic mutation burden in individuals with ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36311816/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221111201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Current screening methods for ovarian cancer (OC) have failed to demonstrate a significant reduction in mortality. Uterine lavage combined with TP53 ultra-deep sequencing for the detection of disseminated OC cells has emerged as a promising tool, but this approach has not been tested for early-stage disease or non-serous histologies. In addition, lavages carry multiple background mutations, the significance of which is poorly understood. Uterine lavage was collected preoperatively in 34 patients...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36315915"
                        open
                      >
                        <summary class="article-expander__title">APOBEC Mutational Signatures in Hormone Receptor-Positive Human Epidermal Growth Factor Receptor 2-Negative Breast Cancers Are Associated With Poor Outcomes on CDK4/6 Inhibitors and Endocrine Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36315915/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221110201819&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: APOBEC+ HR+ HER2- patients had shorter TTD on first-line ET plus CDK4/6i relative to APOBEC- patients. Further research is needed to optimize the treatment of APOBEC+ HR+ HER2- BC and to investigate the efficacy of immunotherapeutic strategies in this population.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36311816"
                        open
                      >
                        <summary class="article-expander__title">Uterine lavage identifies cancer mutations and increased TP53 somatic mutation burden in individuals with ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36311816/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221110201819&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Current screening methods for ovarian cancer (OC) have failed to demonstrate a significant reduction in mortality. Uterine lavage combined with TP53 ultra-deep sequencing for the detection of disseminated OC cells has emerged as a promising tool, but this approach has not been tested for early-stage disease or non-serous histologies. In addition, lavages carry multiple background mutations, the significance of which is poorly understood. Uterine lavage was collected preoperatively in 34 patients...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36315915"
                        open
                      >
                        <summary class="article-expander__title">APOBEC Mutational Signatures in Hormone Receptor-Positive Human Epidermal Growth Factor Receptor 2-Negative Breast Cancers Are Associated With Poor Outcomes on CDK4/6 Inhibitors and Endocrine Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36315915/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221109201530&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: APOBEC+ HR+ HER2- patients had shorter TTD on first-line ET plus CDK4/6i relative to APOBEC- patients. Further research is needed to optimize the treatment of APOBEC+ HR+ HER2- BC and to investigate the efficacy of immunotherapeutic strategies in this population.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36311816"
                        open
                      >
                        <summary class="article-expander__title">Uterine lavage identifies cancer mutations and increased TP53 somatic mutation burden in individuals with ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36311816/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221109201530&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Current screening methods for ovarian cancer (OC) have failed to demonstrate a significant reduction in mortality. Uterine lavage combined with TP53 ultra-deep sequencing for the detection of disseminated OC cells has emerged as a promising tool, but this approach has not been tested for early-stage disease or non-serous histologies. In addition, lavages carry multiple background mutations, the significance of which is poorly understood. Uterine lavage was collected preoperatively in 34 patients...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36315915"
                        open
                      >
                        <summary class="article-expander__title">APOBEC Mutational Signatures in Hormone Receptor-Positive Human Epidermal Growth Factor Receptor 2-Negative Breast Cancers Are Associated With Poor Outcomes on CDK4/6 Inhibitors and Endocrine Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36315915/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221108201514&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: APOBEC+ HR+ HER2- patients had shorter TTD on first-line ET plus CDK4/6i relative to APOBEC- patients. Further research is needed to optimize the treatment of APOBEC+ HR+ HER2- BC and to investigate the efficacy of immunotherapeutic strategies in this population.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36311816"
                        open
                      >
                        <summary class="article-expander__title">Uterine lavage identifies cancer mutations and increased TP53 somatic mutation burden in individuals with ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36311816/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221108201514&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Current screening methods for ovarian cancer (OC) have failed to demonstrate a significant reduction in mortality. Uterine lavage combined with TP53 ultra-deep sequencing for the detection of disseminated OC cells has emerged as a promising tool, but this approach has not been tested for early-stage disease or non-serous histologies. In addition, lavages carry multiple background mutations, the significance of which is poorly understood. Uterine lavage was collected preoperatively in 34 patients...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36315915"
                        open
                      >
                        <summary class="article-expander__title">APOBEC Mutational Signatures in Hormone Receptor-Positive Human Epidermal Growth Factor Receptor 2-Negative Breast Cancers Are Associated With Poor Outcomes on CDK4/6 Inhibitors and Endocrine Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36315915/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221107201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: APOBEC+ HR+ HER2- patients had shorter TTD on first-line ET plus CDK4/6i relative to APOBEC- patients. Further research is needed to optimize the treatment of APOBEC+ HR+ HER2- BC and to investigate the efficacy of immunotherapeutic strategies in this population.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36311816"
                        open
                      >
                        <summary class="article-expander__title">Uterine lavage identifies cancer mutations and increased TP53 somatic mutation burden in individuals with ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36311816/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221107201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Current screening methods for ovarian cancer (OC) have failed to demonstrate a significant reduction in mortality. Uterine lavage combined with TP53 ultra-deep sequencing for the detection of disseminated OC cells has emerged as a promising tool, but this approach has not been tested for early-stage disease or non-serous histologies. In addition, lavages carry multiple background mutations, the significance of which is poorly understood. Uterine lavage was collected preoperatively in 34 patients...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36315915"
                        open
                      >
                        <summary class="article-expander__title">APOBEC Mutational Signatures in Hormone Receptor-Positive Human Epidermal Growth Factor Receptor 2-Negative Breast Cancers Are Associated With Poor Outcomes on CDK4/6 Inhibitors and Endocrine Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36315915/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221106201343&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: APOBEC+ HR+ HER2- patients had shorter TTD on first-line ET plus CDK4/6i relative to APOBEC- patients. Further research is needed to optimize the treatment of APOBEC+ HR+ HER2- BC and to investigate the efficacy of immunotherapeutic strategies in this population.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36311816"
                        open
                      >
                        <summary class="article-expander__title">Uterine lavage identifies cancer mutations and increased TP53 somatic mutation burden in individuals with ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36311816/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221106201343&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Current screening methods for ovarian cancer (OC) have failed to demonstrate a significant reduction in mortality. Uterine lavage combined with TP53 ultra-deep sequencing for the detection of disseminated OC cells has emerged as a promising tool, but this approach has not been tested for early-stage disease or non-serous histologies. In addition, lavages carry multiple background mutations, the significance of which is poorly understood. Uterine lavage was collected preoperatively in 34 patients...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36315915"
                        open
                      >
                        <summary class="article-expander__title">APOBEC Mutational Signatures in Hormone Receptor-Positive Human Epidermal Growth Factor Receptor 2-Negative Breast Cancers Are Associated With Poor Outcomes on CDK4/6 Inhibitors and Endocrine Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36315915/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221105211555&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: APOBEC+ HR+ HER2- patients had shorter TTD on first-line ET plus CDK4/6i relative to APOBEC- patients. Further research is needed to optimize the treatment of APOBEC+ HR+ HER2- BC and to investigate the efficacy of immunotherapeutic strategies in this population.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36311816"
                        open
                      >
                        <summary class="article-expander__title">Uterine lavage identifies cancer mutations and increased TP53 somatic mutation burden in individuals with ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36311816/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221105211555&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Current screening methods for ovarian cancer (OC) have failed to demonstrate a significant reduction in mortality. Uterine lavage combined with TP53 ultra-deep sequencing for the detection of disseminated OC cells has emerged as a promising tool, but this approach has not been tested for early-stage disease or non-serous histologies. In addition, lavages carry multiple background mutations, the significance of which is poorly understood. Uterine lavage was collected preoperatively in 34 patients...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36315915"
                        open
                      >
                        <summary class="article-expander__title">APOBEC Mutational Signatures in Hormone Receptor-Positive Human Epidermal Growth Factor Receptor 2-Negative Breast Cancers Are Associated With Poor Outcomes on CDK4/6 Inhibitors and Endocrine Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36315915/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221104211729&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: APOBEC+ HR+ HER2- patients had shorter TTD on first-line ET plus CDK4/6i relative to APOBEC- patients. Further research is needed to optimize the treatment of APOBEC+ HR+ HER2- BC and to investigate the efficacy of immunotherapeutic strategies in this population.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36311816"
                        open
                      >
                        <summary class="article-expander__title">Uterine lavage identifies cancer mutations and increased TP53 somatic mutation burden in individuals with ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36311816/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221104211729&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Current screening methods for ovarian cancer (OC) have failed to demonstrate a significant reduction in mortality. Uterine lavage combined with TP53 ultra-deep sequencing for the detection of disseminated OC cells has emerged as a promising tool, but this approach has not been tested for early-stage disease or non-serous histologies. In addition, lavages carry multiple background mutations, the significance of which is poorly understood. Uterine lavage was collected preoperatively in 34 patients...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36315915"
                        open
                      >
                        <summary class="article-expander__title">APOBEC Mutational Signatures in Hormone Receptor-Positive Human Epidermal Growth Factor Receptor 2-Negative Breast Cancers Are Associated With Poor Outcomes on CDK4/6 Inhibitors and Endocrine Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36315915/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221103212023&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: APOBEC+ HR+ HER2- patients had shorter TTD on first-line ET plus CDK4/6i relative to APOBEC- patients. Further research is needed to optimize the treatment of APOBEC+ HR+ HER2- BC and to investigate the efficacy of immunotherapeutic strategies in this population.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36311816"
                        open
                      >
                        <summary class="article-expander__title">Uterine lavage identifies cancer mutations and increased TP53 somatic mutation burden in individuals with ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36311816/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221103212023&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Current screening methods for ovarian cancer (OC) have failed to demonstrate a significant reduction in mortality. Uterine lavage combined with TP53 ultra-deep sequencing for the detection of disseminated OC cells has emerged as a promising tool, but this approach has not been tested for early-stage disease or non-serous histologies. In addition, lavages carry multiple background mutations, the significance of which is poorly understood. Uterine lavage was collected preoperatively in 34 patients...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36315915"
                        open
                      >
                        <summary class="article-expander__title">APOBEC Mutational Signatures in Hormone Receptor-Positive Human Epidermal Growth Factor Receptor 2-Negative Breast Cancers Are Associated With Poor Outcomes on CDK4/6 Inhibitors and Endocrine Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36315915/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221102211743&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: APOBEC+ HR+ HER2- patients had shorter TTD on first-line ET plus CDK4/6i relative to APOBEC- patients. Further research is needed to optimize the treatment of APOBEC+ HR+ HER2- BC and to investigate the efficacy of immunotherapeutic strategies in this population.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36311816"
                        open
                      >
                        <summary class="article-expander__title">Uterine lavage identifies cancer mutations and increased TP53 somatic mutation burden in individuals with ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36311816/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221102211743&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Current screening methods for ovarian cancer (OC) have failed to demonstrate a significant reduction in mortality. Uterine lavage combined with TP53 ultra-deep sequencing for the detection of disseminated OC cells has emerged as a promising tool, but this approach has not been tested for early-stage disease or non-serous histologies. In addition, lavages carry multiple background mutations, the significance of which is poorly understood. Uterine lavage was collected preoperatively in 34 patients...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36315915"
                        open
                      >
                        <summary class="article-expander__title">APOBEC Mutational Signatures in Hormone Receptor-Positive Human Epidermal Growth Factor Receptor 2-Negative Breast Cancers Are Associated With Poor Outcomes on CDK4/6 Inhibitors and Endocrine Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36315915/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221101212552&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: APOBEC+ HR+ HER2- patients had shorter TTD on first-line ET plus CDK4/6i relative to APOBEC- patients. Further research is needed to optimize the treatment of APOBEC+ HR+ HER2- BC and to investigate the efficacy of immunotherapeutic strategies in this population.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36311816"
                        open
                      >
                        <summary class="article-expander__title">Uterine lavage identifies cancer mutations and increased TP53 somatic mutation burden in individuals with ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36311816/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221101212552&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Current screening methods for ovarian cancer (OC) have failed to demonstrate a significant reduction in mortality. Uterine lavage combined with TP53 ultra-deep sequencing for the detection of disseminated OC cells has emerged as a promising tool, but this approach has not been tested for early-stage disease or non-serous histologies. In addition, lavages carry multiple background mutations, the significance of which is poorly understood. Uterine lavage was collected preoperatively in 34 patients...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36315915"
                        open
                      >
                        <summary class="article-expander__title">APOBEC Mutational Signatures in Hormone Receptor-Positive Human Epidermal Growth Factor Receptor 2-Negative Breast Cancers Are Associated With Poor Outcomes on CDK4/6 Inhibitors and Endocrine Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36315915/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221031214247&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: APOBEC+ HR+ HER2- patients had shorter TTD on first-line ET plus CDK4/6i relative to APOBEC- patients. Further research is needed to optimize the treatment of APOBEC+ HR+ HER2- BC and to investigate the efficacy of immunotherapeutic strategies in this population.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36311816"
                        open
                      >
                        <summary class="article-expander__title">Uterine lavage identifies cancer mutations and increased TP53 somatic mutation burden in individuals with ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36311816/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221031214247&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Current screening methods for ovarian cancer (OC) have failed to demonstrate a significant reduction in mortality. Uterine lavage combined with TP53 ultra-deep sequencing for the detection of disseminated OC cells has emerged as a promising tool, but this approach has not been tested for early-stage disease or non-serous histologies. In addition, lavages carry multiple background mutations, the significance of which is poorly understood. Uterine lavage was collected preoperatively in 34 patients...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-29 datetime="2022-10-29T10:00:00.000Z">2022-10-29</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-29 datetime="2022-10-29T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221128200439&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.8">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36309296"
                        open
                      >
                        <summary class="article-expander__title">Clonal Relationships of Adjacent Gleason Pattern 3 and Gleason Pattern 5 Lesions in Gleason Scores 3+5&#x3D;8 and 5+3&#x3D;8</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36309296/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221114201139&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genomic studies have demonstrated a high level of intra-tumoral heterogeneity in prostate cancer. There is strong evidence suggesting that individual tumor foci can arise as genetically distinct, clonally independent lesions. However, recent studies have also demonstrated that adjacent Gleason pattern (GP) 3 and GP4 lesions can originate from the same clone but follow divergent genetic and morphologic evolution. The clonal relationship of adjacent GP3 and GP5 lesions has thus far not been...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36309296"
                        open
                      >
                        <summary class="article-expander__title">Clonal Relationships of Adjacent Gleason Pattern 3 and Gleason Pattern 5 Lesions in Gleason Scores 3+5&#x3D;8 and 5+3&#x3D;8</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36309296/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221113201202&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genomic studies have demonstrated a high level of intra-tumoral heterogeneity in prostate cancer. There is strong evidence suggesting that individual tumor foci can arise as genetically distinct, clonally independent lesions. However, recent studies have also demonstrated that adjacent Gleason pattern (GP) 3 and GP4 lesions can originate from the same clone but follow divergent genetic and morphologic evolution. The clonal relationship of adjacent GP3 and GP5 lesions has thus far not been...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36309296"
                        open
                      >
                        <summary class="article-expander__title">Clonal Relationships of Adjacent Gleason Pattern 3 and Gleason Pattern 5 Lesions in Gleason Scores 3+5&#x3D;8 and 5+3&#x3D;8</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36309296/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221112201337&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genomic studies have demonstrated a high level of intra-tumoral heterogeneity in prostate cancer. There is strong evidence suggesting that individual tumor foci can arise as genetically distinct, clonally independent lesions. However, recent studies have also demonstrated that adjacent Gleason pattern (GP) 3 and GP4 lesions can originate from the same clone but follow divergent genetic and morphologic evolution. The clonal relationship of adjacent GP3 and GP5 lesions has thus far not been...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36309296"
                        open
                      >
                        <summary class="article-expander__title">Clonal Relationships of Adjacent Gleason Pattern 3 and Gleason Pattern 5 Lesions in Gleason Scores 3+5&#x3D;8 and 5+3&#x3D;8</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36309296/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221111201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genomic studies have demonstrated a high level of intra-tumoral heterogeneity in prostate cancer. There is strong evidence suggesting that individual tumor foci can arise as genetically distinct, clonally independent lesions. However, recent studies have also demonstrated that adjacent Gleason pattern (GP) 3 and GP4 lesions can originate from the same clone but follow divergent genetic and morphologic evolution. The clonal relationship of adjacent GP3 and GP5 lesions has thus far not been...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36309296"
                        open
                      >
                        <summary class="article-expander__title">Clonal Relationships of Adjacent Gleason Pattern 3 and Gleason Pattern 5 Lesions in Gleason Scores 3+5&#x3D;8 and 5+3&#x3D;8</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36309296/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221110201819&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genomic studies have demonstrated a high level of intra-tumoral heterogeneity in prostate cancer. There is strong evidence suggesting that individual tumor foci can arise as genetically distinct, clonally independent lesions. However, recent studies have also demonstrated that adjacent Gleason pattern (GP) 3 and GP4 lesions can originate from the same clone but follow divergent genetic and morphologic evolution. The clonal relationship of adjacent GP3 and GP5 lesions has thus far not been...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36309296"
                        open
                      >
                        <summary class="article-expander__title">Clonal Relationships of Adjacent Gleason Pattern 3 and Gleason Pattern 5 Lesions in Gleason Scores 3+5&#x3D;8 and 5+3&#x3D;8</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36309296/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221109201530&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genomic studies have demonstrated a high level of intra-tumoral heterogeneity in prostate cancer. There is strong evidence suggesting that individual tumor foci can arise as genetically distinct, clonally independent lesions. However, recent studies have also demonstrated that adjacent Gleason pattern (GP) 3 and GP4 lesions can originate from the same clone but follow divergent genetic and morphologic evolution. The clonal relationship of adjacent GP3 and GP5 lesions has thus far not been...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36309296"
                        open
                      >
                        <summary class="article-expander__title">Clonal Relationships of Adjacent Gleason Pattern 3 and Gleason Pattern 5 Lesions in Gleason Scores 3+5&#x3D;8 and 5+3&#x3D;8</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36309296/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221108201514&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genomic studies have demonstrated a high level of intra-tumoral heterogeneity in prostate cancer. There is strong evidence suggesting that individual tumor foci can arise as genetically distinct, clonally independent lesions. However, recent studies have also demonstrated that adjacent Gleason pattern (GP) 3 and GP4 lesions can originate from the same clone but follow divergent genetic and morphologic evolution. The clonal relationship of adjacent GP3 and GP5 lesions has thus far not been...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36309296"
                        open
                      >
                        <summary class="article-expander__title">Clonal Relationships of Adjacent Gleason Pattern 3 and Gleason Pattern 5 Lesions in Gleason Scores 3+5&#x3D;8 and 5+3&#x3D;8</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36309296/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221107201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genomic studies have demonstrated a high level of intra-tumoral heterogeneity in prostate cancer. There is strong evidence suggesting that individual tumor foci can arise as genetically distinct, clonally independent lesions. However, recent studies have also demonstrated that adjacent Gleason pattern (GP) 3 and GP4 lesions can originate from the same clone but follow divergent genetic and morphologic evolution. The clonal relationship of adjacent GP3 and GP5 lesions has thus far not been...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36309296"
                        open
                      >
                        <summary class="article-expander__title">Clonal Relationships of Adjacent Gleason Pattern 3 and Gleason Pattern 5 Lesions in Gleason Scores 3+5&#x3D;8 and 5+3&#x3D;8</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36309296/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221106201343&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genomic studies have demonstrated a high level of intra-tumoral heterogeneity in prostate cancer. There is strong evidence suggesting that individual tumor foci can arise as genetically distinct, clonally independent lesions. However, recent studies have also demonstrated that adjacent Gleason pattern (GP) 3 and GP4 lesions can originate from the same clone but follow divergent genetic and morphologic evolution. The clonal relationship of adjacent GP3 and GP5 lesions has thus far not been...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36309296"
                        open
                      >
                        <summary class="article-expander__title">Clonal Relationships of Adjacent Gleason Pattern 3 and Gleason Pattern 5 Lesions in Gleason Scores 3+5&#x3D;8 and 5+3&#x3D;8</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36309296/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221105211555&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genomic studies have demonstrated a high level of intra-tumoral heterogeneity in prostate cancer. There is strong evidence suggesting that individual tumor foci can arise as genetically distinct, clonally independent lesions. However, recent studies have also demonstrated that adjacent Gleason pattern (GP) 3 and GP4 lesions can originate from the same clone but follow divergent genetic and morphologic evolution. The clonal relationship of adjacent GP3 and GP5 lesions has thus far not been...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36309296"
                        open
                      >
                        <summary class="article-expander__title">Clonal Relationships of Adjacent Gleason Pattern 3 and Gleason Pattern 5 Lesions in Gleason Scores 3+5&#x3D;8 and 5+3&#x3D;8</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36309296/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221104211729&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genomic studies have demonstrated a high level of intra-tumoral heterogeneity in prostate cancer. There is strong evidence suggesting that individual tumor foci can arise as genetically distinct, clonally independent lesions. However, recent studies have also demonstrated that adjacent Gleason pattern (GP) 3 and GP4 lesions can originate from the same clone but follow divergent genetic and morphologic evolution. The clonal relationship of adjacent GP3 and GP5 lesions has thus far not been...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36309296"
                        open
                      >
                        <summary class="article-expander__title">Clonal Relationships of Adjacent Gleason Pattern 3 and Gleason Pattern 5 Lesions in Gleason Scores 3+5&#x3D;8 and 5+3&#x3D;8</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36309296/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221103212023&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genomic studies have demonstrated a high level of intra-tumoral heterogeneity in prostate cancer. There is strong evidence suggesting that individual tumor foci can arise as genetically distinct, clonally independent lesions. However, recent studies have also demonstrated that adjacent Gleason pattern (GP) 3 and GP4 lesions can originate from the same clone but follow divergent genetic and morphologic evolution. The clonal relationship of adjacent GP3 and GP5 lesions has thus far not been...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36309296"
                        open
                      >
                        <summary class="article-expander__title">Clonal Relationships of Adjacent Gleason Pattern 3 and Gleason Pattern 5 Lesions in Gleason Scores 3+5&#x3D;8 and 5+3&#x3D;8</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36309296/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221102211743&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genomic studies have demonstrated a high level of intra-tumoral heterogeneity in prostate cancer. There is strong evidence suggesting that individual tumor foci can arise as genetically distinct, clonally independent lesions. However, recent studies have also demonstrated that adjacent Gleason pattern (GP) 3 and GP4 lesions can originate from the same clone but follow divergent genetic and morphologic evolution. The clonal relationship of adjacent GP3 and GP5 lesions has thus far not been...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36309296"
                        open
                      >
                        <summary class="article-expander__title">Clonal Relationships of Adjacent Gleason Pattern 3 and Gleason Pattern 5 Lesions in Gleason Scores 3+5&#x3D;8 and 5+3&#x3D;8</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36309296/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221101212552&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genomic studies have demonstrated a high level of intra-tumoral heterogeneity in prostate cancer. There is strong evidence suggesting that individual tumor foci can arise as genetically distinct, clonally independent lesions. However, recent studies have also demonstrated that adjacent Gleason pattern (GP) 3 and GP4 lesions can originate from the same clone but follow divergent genetic and morphologic evolution. The clonal relationship of adjacent GP3 and GP5 lesions has thus far not been...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36309296"
                        open
                      >
                        <summary class="article-expander__title">Clonal Relationships of Adjacent Gleason Pattern 3 and Gleason Pattern 5 Lesions in Gleason Scores 3+5&#x3D;8 and 5+3&#x3D;8</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36309296/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221031214247&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genomic studies have demonstrated a high level of intra-tumoral heterogeneity in prostate cancer. There is strong evidence suggesting that individual tumor foci can arise as genetically distinct, clonally independent lesions. However, recent studies have also demonstrated that adjacent Gleason pattern (GP) 3 and GP4 lesions can originate from the same clone but follow divergent genetic and morphologic evolution. The clonal relationship of adjacent GP3 and GP5 lesions has thus far not been...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36309296"
                        open
                      >
                        <summary class="article-expander__title">Clonal Relationships of Adjacent Gleason Pattern 3 and Gleason Pattern 5 Lesions in Gleason Scores 3+5&#x3D;8 and 5+3&#x3D;8</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36309296/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221030212817&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genomic studies have demonstrated a high level of intra-tumoral heterogeneity in prostate cancer. There is strong evidence suggesting that individual tumor foci can arise as genetically distinct, clonally independent lesions. However, recent studies have also demonstrated that adjacent Gleason pattern (GP) 3 and GP4 lesions can originate from the same clone but follow divergent genetic and morphologic evolution. The clonal relationship of adjacent GP3 and GP5 lesions has thus far not been...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-28 datetime="2022-10-28T10:00:00.000Z">2022-10-28</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-28 datetime="2022-10-28T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.8&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20221128200439&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36306593"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic neoplasms: An overview on diagnosis, risk-stratification, molecular pathogenesis, and treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36306593/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221113201202&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic neoplasms are clonal hematological malignancies arising from hematopoietic stem cells that accumulate various mutations. MDS is heterogeneous in nature but uniformly characterized by ineffective hematopoiesis, dysplasia of one or more cell lineages, and an increased risk of transformation to acute myeloid leukemia. Disease-related risk is commonly assessed using the Revised International Prognostic Scoring System based on five cytogenetic risk groups, together with refined...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36306593"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic neoplasms: An overview on diagnosis, risk-stratification, molecular pathogenesis, and treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36306593/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221112201337&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic neoplasms are clonal hematological malignancies arising from hematopoietic stem cells that accumulate various mutations. MDS is heterogeneous in nature but uniformly characterized by ineffective hematopoiesis, dysplasia of one or more cell lineages, and an increased risk of transformation to acute myeloid leukemia. Disease-related risk is commonly assessed using the Revised International Prognostic Scoring System based on five cytogenetic risk groups, together with refined...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36306593"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic neoplasms: An overview on diagnosis, risk-stratification, molecular pathogenesis, and treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36306593/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221111201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic neoplasms are clonal hematological malignancies arising from hematopoietic stem cells that accumulate various mutations. MDS is heterogeneous in nature but uniformly characterized by ineffective hematopoiesis, dysplasia of one or more cell lineages, and an increased risk of transformation to acute myeloid leukemia. Disease-related risk is commonly assessed using the Revised International Prognostic Scoring System based on five cytogenetic risk groups, together with refined...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36306593"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic neoplasms: An overview on diagnosis, risk-stratification, molecular pathogenesis, and treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36306593/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221110201819&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic neoplasms are clonal hematological malignancies arising from hematopoietic stem cells that accumulate various mutations. MDS is heterogeneous in nature but uniformly characterized by ineffective hematopoiesis, dysplasia of one or more cell lineages, and an increased risk of transformation to acute myeloid leukemia. Disease-related risk is commonly assessed using the Revised International Prognostic Scoring System based on five cytogenetic risk groups, together with refined...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36303184"
                        open
                      >
                        <summary class="article-expander__title">Bone marrow mesenchymal stem cells regulate the dysfunction of NK cells via the T cell immunoglobulin and ITIM domain in patients with myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36303184/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221110201819&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The co-culture results of BMSCs and NK cells demonstrated that BMSCs regulate NK cells through the TIGIT/CD155 interaction, indicating that NK cells play a vital role in MDS progression. BMSCs regulate the function of NK cells via TIGIT/CD155. Video Abstract.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36306593"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic neoplasms: An overview on diagnosis, risk-stratification, molecular pathogenesis, and treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36306593/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221109201530&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic neoplasms are clonal hematological malignancies arising from hematopoietic stem cells that accumulate various mutations. MDS is heterogeneous in nature but uniformly characterized by ineffective hematopoiesis, dysplasia of one or more cell lineages, and an increased risk of transformation to acute myeloid leukemia. Disease-related risk is commonly assessed using the Revised International Prognostic Scoring System based on five cytogenetic risk groups, together with refined...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36303184"
                        open
                      >
                        <summary class="article-expander__title">Bone marrow mesenchymal stem cells regulate the dysfunction of NK cells via the T cell immunoglobulin and ITIM domain in patients with myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36303184/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221109201530&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The co-culture results of BMSCs and NK cells demonstrated that BMSCs regulate NK cells through the TIGIT/CD155 interaction, indicating that NK cells play a vital role in MDS progression. BMSCs regulate the function of NK cells via TIGIT/CD155. Video Abstract.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36306593"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic neoplasms: An overview on diagnosis, risk-stratification, molecular pathogenesis, and treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36306593/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221108201514&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic neoplasms are clonal hematological malignancies arising from hematopoietic stem cells that accumulate various mutations. MDS is heterogeneous in nature but uniformly characterized by ineffective hematopoiesis, dysplasia of one or more cell lineages, and an increased risk of transformation to acute myeloid leukemia. Disease-related risk is commonly assessed using the Revised International Prognostic Scoring System based on five cytogenetic risk groups, together with refined...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36303184"
                        open
                      >
                        <summary class="article-expander__title">Bone marrow mesenchymal stem cells regulate the dysfunction of NK cells via the T cell immunoglobulin and ITIM domain in patients with myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36303184/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221108201514&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The co-culture results of BMSCs and NK cells demonstrated that BMSCs regulate NK cells through the TIGIT/CD155 interaction, indicating that NK cells play a vital role in MDS progression. BMSCs regulate the function of NK cells via TIGIT/CD155. Video Abstract.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36306593"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic neoplasms: An overview on diagnosis, risk-stratification, molecular pathogenesis, and treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36306593/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221107201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic neoplasms are clonal hematological malignancies arising from hematopoietic stem cells that accumulate various mutations. MDS is heterogeneous in nature but uniformly characterized by ineffective hematopoiesis, dysplasia of one or more cell lineages, and an increased risk of transformation to acute myeloid leukemia. Disease-related risk is commonly assessed using the Revised International Prognostic Scoring System based on five cytogenetic risk groups, together with refined...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36303184"
                        open
                      >
                        <summary class="article-expander__title">Bone marrow mesenchymal stem cells regulate the dysfunction of NK cells via the T cell immunoglobulin and ITIM domain in patients with myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36303184/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221107201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The co-culture results of BMSCs and NK cells demonstrated that BMSCs regulate NK cells through the TIGIT/CD155 interaction, indicating that NK cells play a vital role in MDS progression. BMSCs regulate the function of NK cells via TIGIT/CD155. Video Abstract.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36306593"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic neoplasms: An overview on diagnosis, risk-stratification, molecular pathogenesis, and treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36306593/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221106201343&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic neoplasms are clonal hematological malignancies arising from hematopoietic stem cells that accumulate various mutations. MDS is heterogeneous in nature but uniformly characterized by ineffective hematopoiesis, dysplasia of one or more cell lineages, and an increased risk of transformation to acute myeloid leukemia. Disease-related risk is commonly assessed using the Revised International Prognostic Scoring System based on five cytogenetic risk groups, together with refined...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36303184"
                        open
                      >
                        <summary class="article-expander__title">Bone marrow mesenchymal stem cells regulate the dysfunction of NK cells via the T cell immunoglobulin and ITIM domain in patients with myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36303184/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221106201343&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The co-culture results of BMSCs and NK cells demonstrated that BMSCs regulate NK cells through the TIGIT/CD155 interaction, indicating that NK cells play a vital role in MDS progression. BMSCs regulate the function of NK cells via TIGIT/CD155. Video Abstract.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36306593"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic neoplasms: An overview on diagnosis, risk-stratification, molecular pathogenesis, and treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36306593/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221105211555&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic neoplasms are clonal hematological malignancies arising from hematopoietic stem cells that accumulate various mutations. MDS is heterogeneous in nature but uniformly characterized by ineffective hematopoiesis, dysplasia of one or more cell lineages, and an increased risk of transformation to acute myeloid leukemia. Disease-related risk is commonly assessed using the Revised International Prognostic Scoring System based on five cytogenetic risk groups, together with refined...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36303184"
                        open
                      >
                        <summary class="article-expander__title">Bone marrow mesenchymal stem cells regulate the dysfunction of NK cells via the T cell immunoglobulin and ITIM domain in patients with myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36303184/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221105211555&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The co-culture results of BMSCs and NK cells demonstrated that BMSCs regulate NK cells through the TIGIT/CD155 interaction, indicating that NK cells play a vital role in MDS progression. BMSCs regulate the function of NK cells via TIGIT/CD155. Video Abstract.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36306593"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic neoplasms: An overview on diagnosis, risk-stratification, molecular pathogenesis, and treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36306593/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221104211730&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic neoplasms are clonal hematological malignancies arising from hematopoietic stem cells that accumulate various mutations. MDS is heterogeneous in nature but uniformly characterized by ineffective hematopoiesis, dysplasia of one or more cell lineages, and an increased risk of transformation to acute myeloid leukemia. Disease-related risk is commonly assessed using the Revised International Prognostic Scoring System based on five cytogenetic risk groups, together with refined...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36303184"
                        open
                      >
                        <summary class="article-expander__title">Bone marrow mesenchymal stem cells regulate the dysfunction of NK cells via the T cell immunoglobulin and ITIM domain in patients with myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36303184/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221104211730&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The co-culture results of BMSCs and NK cells demonstrated that BMSCs regulate NK cells through the TIGIT/CD155 interaction, indicating that NK cells play a vital role in MDS progression. BMSCs regulate the function of NK cells via TIGIT/CD155. Video Abstract.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36306593"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic neoplasms: An overview on diagnosis, risk-stratification, molecular pathogenesis, and treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36306593/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221103212023&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic neoplasms are clonal hematological malignancies arising from hematopoietic stem cells that accumulate various mutations. MDS is heterogeneous in nature but uniformly characterized by ineffective hematopoiesis, dysplasia of one or more cell lineages, and an increased risk of transformation to acute myeloid leukemia. Disease-related risk is commonly assessed using the Revised International Prognostic Scoring System based on five cytogenetic risk groups, together with refined...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36303184"
                        open
                      >
                        <summary class="article-expander__title">Bone marrow mesenchymal stem cells regulate the dysfunction of NK cells via the T cell immunoglobulin and ITIM domain in patients with myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36303184/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221103212023&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The co-culture results of BMSCs and NK cells demonstrated that BMSCs regulate NK cells through the TIGIT/CD155 interaction, indicating that NK cells play a vital role in MDS progression. BMSCs regulate the function of NK cells via TIGIT/CD155. Video Abstract.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36306593"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic neoplasms: An overview on diagnosis, risk-stratification, molecular pathogenesis, and treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36306593/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221102211743&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic neoplasms are clonal hematological malignancies arising from hematopoietic stem cells that accumulate various mutations. MDS is heterogeneous in nature but uniformly characterized by ineffective hematopoiesis, dysplasia of one or more cell lineages, and an increased risk of transformation to acute myeloid leukemia. Disease-related risk is commonly assessed using the Revised International Prognostic Scoring System based on five cytogenetic risk groups, together with refined...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36303184"
                        open
                      >
                        <summary class="article-expander__title">Bone marrow mesenchymal stem cells regulate the dysfunction of NK cells via the T cell immunoglobulin and ITIM domain in patients with myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36303184/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221102211743&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The co-culture results of BMSCs and NK cells demonstrated that BMSCs regulate NK cells through the TIGIT/CD155 interaction, indicating that NK cells play a vital role in MDS progression. BMSCs regulate the function of NK cells via TIGIT/CD155. Video Abstract.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36306593"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic neoplasms: An overview on diagnosis, risk-stratification, molecular pathogenesis, and treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36306593/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221101212552&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic neoplasms are clonal hematological malignancies arising from hematopoietic stem cells that accumulate various mutations. MDS is heterogeneous in nature but uniformly characterized by ineffective hematopoiesis, dysplasia of one or more cell lineages, and an increased risk of transformation to acute myeloid leukemia. Disease-related risk is commonly assessed using the Revised International Prognostic Scoring System based on five cytogenetic risk groups, together with refined...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36303184"
                        open
                      >
                        <summary class="article-expander__title">Bone marrow mesenchymal stem cells regulate the dysfunction of NK cells via the T cell immunoglobulin and ITIM domain in patients with myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36303184/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221101212552&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The co-culture results of BMSCs and NK cells demonstrated that BMSCs regulate NK cells through the TIGIT/CD155 interaction, indicating that NK cells play a vital role in MDS progression. BMSCs regulate the function of NK cells via TIGIT/CD155. Video Abstract.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36306593"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic neoplasms: An overview on diagnosis, risk-stratification, molecular pathogenesis, and treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36306593/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221031214247&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic neoplasms are clonal hematological malignancies arising from hematopoietic stem cells that accumulate various mutations. MDS is heterogeneous in nature but uniformly characterized by ineffective hematopoiesis, dysplasia of one or more cell lineages, and an increased risk of transformation to acute myeloid leukemia. Disease-related risk is commonly assessed using the Revised International Prognostic Scoring System based on five cytogenetic risk groups, together with refined...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36303184"
                        open
                      >
                        <summary class="article-expander__title">Bone marrow mesenchymal stem cells regulate the dysfunction of NK cells via the T cell immunoglobulin and ITIM domain in patients with myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36303184/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221031214247&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The co-culture results of BMSCs and NK cells demonstrated that BMSCs regulate NK cells through the TIGIT/CD155 interaction, indicating that NK cells play a vital role in MDS progression. BMSCs regulate the function of NK cells via TIGIT/CD155. Video Abstract.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36306593"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic neoplasms: An overview on diagnosis, risk-stratification, molecular pathogenesis, and treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36306593/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221030212817&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic neoplasms are clonal hematological malignancies arising from hematopoietic stem cells that accumulate various mutations. MDS is heterogeneous in nature but uniformly characterized by ineffective hematopoiesis, dysplasia of one or more cell lineages, and an increased risk of transformation to acute myeloid leukemia. Disease-related risk is commonly assessed using the Revised International Prognostic Scoring System based on five cytogenetic risk groups, together with refined...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36303184"
                        open
                      >
                        <summary class="article-expander__title">Bone marrow mesenchymal stem cells regulate the dysfunction of NK cells via the T cell immunoglobulin and ITIM domain in patients with myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36303184/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221030212817&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The co-culture results of BMSCs and NK cells demonstrated that BMSCs regulate NK cells through the TIGIT/CD155 interaction, indicating that NK cells play a vital role in MDS progression. BMSCs regulate the function of NK cells via TIGIT/CD155. Video Abstract.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36306593"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic neoplasms: An overview on diagnosis, risk-stratification, molecular pathogenesis, and treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36306593/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221029212729&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic neoplasms are clonal hematological malignancies arising from hematopoietic stem cells that accumulate various mutations. MDS is heterogeneous in nature but uniformly characterized by ineffective hematopoiesis, dysplasia of one or more cell lineages, and an increased risk of transformation to acute myeloid leukemia. Disease-related risk is commonly assessed using the Revised International Prognostic Scoring System based on five cytogenetic risk groups, together with refined...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36303184"
                        open
                      >
                        <summary class="article-expander__title">Bone marrow mesenchymal stem cells regulate the dysfunction of NK cells via the T cell immunoglobulin and ITIM domain in patients with myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36303184/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221029212729&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The co-culture results of BMSCs and NK cells demonstrated that BMSCs regulate NK cells through the TIGIT/CD155 interaction, indicating that NK cells play a vital role in MDS progression. BMSCs regulate the function of NK cells via TIGIT/CD155. Video Abstract.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36303184"
                        open
                      >
                        <summary class="article-expander__title">Bone marrow mesenchymal stem cells regulate the dysfunction of NK cells via the T cell immunoglobulin and ITIM domain in patients with myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36303184/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221028211238&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The co-culture results of BMSCs and NK cells demonstrated that BMSCs regulate NK cells through the TIGIT/CD155 interaction, indicating that NK cells play a vital role in MDS progression. BMSCs regulate the function of NK cells via TIGIT/CD155. Video Abstract.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221128200439&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.8">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36303541"
                        open
                      >
                        <summary class="article-expander__title">Advances in single-cell sequencing technology in the field of hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36303541/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221113201202&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tumors are a class of diseases characterized by altered genetic information and uncontrolled growth. Sequencing technology provide researchers with a better way to explore specific tumor pathogenesis. In recent years, single-cell sequencing technology has shone in tumor research, especially in the study of liver cancer, revealing phenomena that were unexplored by previous studies. Single-cell sequencing (SCS) is a technique for sequencing the cellular genome, transcriptome, epigenome,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36303541"
                        open
                      >
                        <summary class="article-expander__title">Advances in single-cell sequencing technology in the field of hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36303541/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221112201337&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tumors are a class of diseases characterized by altered genetic information and uncontrolled growth. Sequencing technology provide researchers with a better way to explore specific tumor pathogenesis. In recent years, single-cell sequencing technology has shone in tumor research, especially in the study of liver cancer, revealing phenomena that were unexplored by previous studies. Single-cell sequencing (SCS) is a technique for sequencing the cellular genome, transcriptome, epigenome,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36303541"
                        open
                      >
                        <summary class="article-expander__title">Advances in single-cell sequencing technology in the field of hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36303541/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221111201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tumors are a class of diseases characterized by altered genetic information and uncontrolled growth. Sequencing technology provide researchers with a better way to explore specific tumor pathogenesis. In recent years, single-cell sequencing technology has shone in tumor research, especially in the study of liver cancer, revealing phenomena that were unexplored by previous studies. Single-cell sequencing (SCS) is a technique for sequencing the cellular genome, transcriptome, epigenome,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36303541"
                        open
                      >
                        <summary class="article-expander__title">Advances in single-cell sequencing technology in the field of hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36303541/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221110201819&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tumors are a class of diseases characterized by altered genetic information and uncontrolled growth. Sequencing technology provide researchers with a better way to explore specific tumor pathogenesis. In recent years, single-cell sequencing technology has shone in tumor research, especially in the study of liver cancer, revealing phenomena that were unexplored by previous studies. Single-cell sequencing (SCS) is a technique for sequencing the cellular genome, transcriptome, epigenome,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36303541"
                        open
                      >
                        <summary class="article-expander__title">Advances in single-cell sequencing technology in the field of hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36303541/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221109201530&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tumors are a class of diseases characterized by altered genetic information and uncontrolled growth. Sequencing technology provide researchers with a better way to explore specific tumor pathogenesis. In recent years, single-cell sequencing technology has shone in tumor research, especially in the study of liver cancer, revealing phenomena that were unexplored by previous studies. Single-cell sequencing (SCS) is a technique for sequencing the cellular genome, transcriptome, epigenome,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36303541"
                        open
                      >
                        <summary class="article-expander__title">Advances in single-cell sequencing technology in the field of hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36303541/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221108201514&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tumors are a class of diseases characterized by altered genetic information and uncontrolled growth. Sequencing technology provide researchers with a better way to explore specific tumor pathogenesis. In recent years, single-cell sequencing technology has shone in tumor research, especially in the study of liver cancer, revealing phenomena that were unexplored by previous studies. Single-cell sequencing (SCS) is a technique for sequencing the cellular genome, transcriptome, epigenome,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36303541"
                        open
                      >
                        <summary class="article-expander__title">Advances in single-cell sequencing technology in the field of hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36303541/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221107201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tumors are a class of diseases characterized by altered genetic information and uncontrolled growth. Sequencing technology provide researchers with a better way to explore specific tumor pathogenesis. In recent years, single-cell sequencing technology has shone in tumor research, especially in the study of liver cancer, revealing phenomena that were unexplored by previous studies. Single-cell sequencing (SCS) is a technique for sequencing the cellular genome, transcriptome, epigenome,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36303541"
                        open
                      >
                        <summary class="article-expander__title">Advances in single-cell sequencing technology in the field of hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36303541/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221106201343&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tumors are a class of diseases characterized by altered genetic information and uncontrolled growth. Sequencing technology provide researchers with a better way to explore specific tumor pathogenesis. In recent years, single-cell sequencing technology has shone in tumor research, especially in the study of liver cancer, revealing phenomena that were unexplored by previous studies. Single-cell sequencing (SCS) is a technique for sequencing the cellular genome, transcriptome, epigenome,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36303541"
                        open
                      >
                        <summary class="article-expander__title">Advances in single-cell sequencing technology in the field of hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36303541/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221105211555&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tumors are a class of diseases characterized by altered genetic information and uncontrolled growth. Sequencing technology provide researchers with a better way to explore specific tumor pathogenesis. In recent years, single-cell sequencing technology has shone in tumor research, especially in the study of liver cancer, revealing phenomena that were unexplored by previous studies. Single-cell sequencing (SCS) is a technique for sequencing the cellular genome, transcriptome, epigenome,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36303541"
                        open
                      >
                        <summary class="article-expander__title">Advances in single-cell sequencing technology in the field of hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36303541/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221104211729&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tumors are a class of diseases characterized by altered genetic information and uncontrolled growth. Sequencing technology provide researchers with a better way to explore specific tumor pathogenesis. In recent years, single-cell sequencing technology has shone in tumor research, especially in the study of liver cancer, revealing phenomena that were unexplored by previous studies. Single-cell sequencing (SCS) is a technique for sequencing the cellular genome, transcriptome, epigenome,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36303541"
                        open
                      >
                        <summary class="article-expander__title">Advances in single-cell sequencing technology in the field of hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36303541/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221103212023&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tumors are a class of diseases characterized by altered genetic information and uncontrolled growth. Sequencing technology provide researchers with a better way to explore specific tumor pathogenesis. In recent years, single-cell sequencing technology has shone in tumor research, especially in the study of liver cancer, revealing phenomena that were unexplored by previous studies. Single-cell sequencing (SCS) is a technique for sequencing the cellular genome, transcriptome, epigenome,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36303541"
                        open
                      >
                        <summary class="article-expander__title">Advances in single-cell sequencing technology in the field of hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36303541/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221102211743&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tumors are a class of diseases characterized by altered genetic information and uncontrolled growth. Sequencing technology provide researchers with a better way to explore specific tumor pathogenesis. In recent years, single-cell sequencing technology has shone in tumor research, especially in the study of liver cancer, revealing phenomena that were unexplored by previous studies. Single-cell sequencing (SCS) is a technique for sequencing the cellular genome, transcriptome, epigenome,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36303541"
                        open
                      >
                        <summary class="article-expander__title">Advances in single-cell sequencing technology in the field of hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36303541/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221101212552&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tumors are a class of diseases characterized by altered genetic information and uncontrolled growth. Sequencing technology provide researchers with a better way to explore specific tumor pathogenesis. In recent years, single-cell sequencing technology has shone in tumor research, especially in the study of liver cancer, revealing phenomena that were unexplored by previous studies. Single-cell sequencing (SCS) is a technique for sequencing the cellular genome, transcriptome, epigenome,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36303541"
                        open
                      >
                        <summary class="article-expander__title">Advances in single-cell sequencing technology in the field of hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36303541/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221031214247&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tumors are a class of diseases characterized by altered genetic information and uncontrolled growth. Sequencing technology provide researchers with a better way to explore specific tumor pathogenesis. In recent years, single-cell sequencing technology has shone in tumor research, especially in the study of liver cancer, revealing phenomena that were unexplored by previous studies. Single-cell sequencing (SCS) is a technique for sequencing the cellular genome, transcriptome, epigenome,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36303541"
                        open
                      >
                        <summary class="article-expander__title">Advances in single-cell sequencing technology in the field of hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36303541/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221030212817&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tumors are a class of diseases characterized by altered genetic information and uncontrolled growth. Sequencing technology provide researchers with a better way to explore specific tumor pathogenesis. In recent years, single-cell sequencing technology has shone in tumor research, especially in the study of liver cancer, revealing phenomena that were unexplored by previous studies. Single-cell sequencing (SCS) is a technique for sequencing the cellular genome, transcriptome, epigenome,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36303541"
                        open
                      >
                        <summary class="article-expander__title">Advances in single-cell sequencing technology in the field of hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36303541/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221029212729&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tumors are a class of diseases characterized by altered genetic information and uncontrolled growth. Sequencing technology provide researchers with a better way to explore specific tumor pathogenesis. In recent years, single-cell sequencing technology has shone in tumor research, especially in the study of liver cancer, revealing phenomena that were unexplored by previous studies. Single-cell sequencing (SCS) is a technique for sequencing the cellular genome, transcriptome, epigenome,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36303541"
                        open
                      >
                        <summary class="article-expander__title">Advances in single-cell sequencing technology in the field of hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36303541/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221028211238&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tumors are a class of diseases characterized by altered genetic information and uncontrolled growth. Sequencing technology provide researchers with a better way to explore specific tumor pathogenesis. In recent years, single-cell sequencing technology has shone in tumor research, especially in the study of liver cancer, revealing phenomena that were unexplored by previous studies. Single-cell sequencing (SCS) is a technique for sequencing the cellular genome, transcriptome, epigenome,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-27 datetime="2022-10-27T10:00:00.000Z">2022-10-27</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-27 datetime="2022-10-27T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.8&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20221128200439&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36290845"
                        open
                      >
                        <summary class="article-expander__title">Mutational Landscape of Patients Referred for Elevated Hemoglobin Level</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36290845/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221109201530&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Background: Since the identification of JAK2 V617F and exon 12 mutations as driver mutations in polycythemia vera (PV) in 2005, molecular testing of these mutations for patients with erythrocytosis has become a routine clinical practice. However, the incidence of myeloid mutations other than the common JAK2 V617F mutation in unselected patients referred for elevated hemoglobin is not well studied. This study aimed to characterize the mutational landscape in a real-world population of patients...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36290314"
                        open
                      >
                        <summary class="article-expander__title">Lysosomes and Their Role in Regulating the Metabolism of Hematopoietic Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36290314/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221109201530&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) have the capacity to renew blood cells at all stages of life and are largely quiescent at a steady state. It is essential to understand the processes that govern quiescence in HSCs to enhance bone marrow transplantation. It is hypothesized that in their quiescent state, HSCs primarily use glycolysis for energy production rather than mitochondrial oxidative phosphorylation (OXPHOS). In addition, the HSC switch from quiescence to activation occurs along a continuous...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36290845"
                        open
                      >
                        <summary class="article-expander__title">Mutational Landscape of Patients Referred for Elevated Hemoglobin Level</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36290845/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221108201514&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Background: Since the identification of JAK2 V617F and exon 12 mutations as driver mutations in polycythemia vera (PV) in 2005, molecular testing of these mutations for patients with erythrocytosis has become a routine clinical practice. However, the incidence of myeloid mutations other than the common JAK2 V617F mutation in unselected patients referred for elevated hemoglobin is not well studied. This study aimed to characterize the mutational landscape in a real-world population of patients...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36290314"
                        open
                      >
                        <summary class="article-expander__title">Lysosomes and Their Role in Regulating the Metabolism of Hematopoietic Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36290314/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221108201514&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) have the capacity to renew blood cells at all stages of life and are largely quiescent at a steady state. It is essential to understand the processes that govern quiescence in HSCs to enhance bone marrow transplantation. It is hypothesized that in their quiescent state, HSCs primarily use glycolysis for energy production rather than mitochondrial oxidative phosphorylation (OXPHOS). In addition, the HSC switch from quiescence to activation occurs along a continuous...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36290845"
                        open
                      >
                        <summary class="article-expander__title">Mutational Landscape of Patients Referred for Elevated Hemoglobin Level</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36290845/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221107201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Background: Since the identification of JAK2 V617F and exon 12 mutations as driver mutations in polycythemia vera (PV) in 2005, molecular testing of these mutations for patients with erythrocytosis has become a routine clinical practice. However, the incidence of myeloid mutations other than the common JAK2 V617F mutation in unselected patients referred for elevated hemoglobin is not well studied. This study aimed to characterize the mutational landscape in a real-world population of patients...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36290314"
                        open
                      >
                        <summary class="article-expander__title">Lysosomes and Their Role in Regulating the Metabolism of Hematopoietic Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36290314/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221107201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) have the capacity to renew blood cells at all stages of life and are largely quiescent at a steady state. It is essential to understand the processes that govern quiescence in HSCs to enhance bone marrow transplantation. It is hypothesized that in their quiescent state, HSCs primarily use glycolysis for energy production rather than mitochondrial oxidative phosphorylation (OXPHOS). In addition, the HSC switch from quiescence to activation occurs along a continuous...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36290845"
                        open
                      >
                        <summary class="article-expander__title">Mutational Landscape of Patients Referred for Elevated Hemoglobin Level</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36290845/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221106201343&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Background: Since the identification of JAK2 V617F and exon 12 mutations as driver mutations in polycythemia vera (PV) in 2005, molecular testing of these mutations for patients with erythrocytosis has become a routine clinical practice. However, the incidence of myeloid mutations other than the common JAK2 V617F mutation in unselected patients referred for elevated hemoglobin is not well studied. This study aimed to characterize the mutational landscape in a real-world population of patients...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36290314"
                        open
                      >
                        <summary class="article-expander__title">Lysosomes and Their Role in Regulating the Metabolism of Hematopoietic Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36290314/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221106201343&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) have the capacity to renew blood cells at all stages of life and are largely quiescent at a steady state. It is essential to understand the processes that govern quiescence in HSCs to enhance bone marrow transplantation. It is hypothesized that in their quiescent state, HSCs primarily use glycolysis for energy production rather than mitochondrial oxidative phosphorylation (OXPHOS). In addition, the HSC switch from quiescence to activation occurs along a continuous...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36290845"
                        open
                      >
                        <summary class="article-expander__title">Mutational Landscape of Patients Referred for Elevated Hemoglobin Level</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36290845/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221105211555&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Background: Since the identification of JAK2 V617F and exon 12 mutations as driver mutations in polycythemia vera (PV) in 2005, molecular testing of these mutations for patients with erythrocytosis has become a routine clinical practice. However, the incidence of myeloid mutations other than the common JAK2 V617F mutation in unselected patients referred for elevated hemoglobin is not well studied. This study aimed to characterize the mutational landscape in a real-world population of patients...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36290314"
                        open
                      >
                        <summary class="article-expander__title">Lysosomes and Their Role in Regulating the Metabolism of Hematopoietic Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36290314/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221105211555&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) have the capacity to renew blood cells at all stages of life and are largely quiescent at a steady state. It is essential to understand the processes that govern quiescence in HSCs to enhance bone marrow transplantation. It is hypothesized that in their quiescent state, HSCs primarily use glycolysis for energy production rather than mitochondrial oxidative phosphorylation (OXPHOS). In addition, the HSC switch from quiescence to activation occurs along a continuous...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36290845"
                        open
                      >
                        <summary class="article-expander__title">Mutational Landscape of Patients Referred for Elevated Hemoglobin Level</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36290845/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221104211730&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Background: Since the identification of JAK2 V617F and exon 12 mutations as driver mutations in polycythemia vera (PV) in 2005, molecular testing of these mutations for patients with erythrocytosis has become a routine clinical practice. However, the incidence of myeloid mutations other than the common JAK2 V617F mutation in unselected patients referred for elevated hemoglobin is not well studied. This study aimed to characterize the mutational landscape in a real-world population of patients...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36290314"
                        open
                      >
                        <summary class="article-expander__title">Lysosomes and Their Role in Regulating the Metabolism of Hematopoietic Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36290314/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221104211730&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) have the capacity to renew blood cells at all stages of life and are largely quiescent at a steady state. It is essential to understand the processes that govern quiescence in HSCs to enhance bone marrow transplantation. It is hypothesized that in their quiescent state, HSCs primarily use glycolysis for energy production rather than mitochondrial oxidative phosphorylation (OXPHOS). In addition, the HSC switch from quiescence to activation occurs along a continuous...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36290845"
                        open
                      >
                        <summary class="article-expander__title">Mutational Landscape of Patients Referred for Elevated Hemoglobin Level</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36290845/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221103212023&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Background: Since the identification of JAK2 V617F and exon 12 mutations as driver mutations in polycythemia vera (PV) in 2005, molecular testing of these mutations for patients with erythrocytosis has become a routine clinical practice. However, the incidence of myeloid mutations other than the common JAK2 V617F mutation in unselected patients referred for elevated hemoglobin is not well studied. This study aimed to characterize the mutational landscape in a real-world population of patients...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36290314"
                        open
                      >
                        <summary class="article-expander__title">Lysosomes and Their Role in Regulating the Metabolism of Hematopoietic Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36290314/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221103212023&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) have the capacity to renew blood cells at all stages of life and are largely quiescent at a steady state. It is essential to understand the processes that govern quiescence in HSCs to enhance bone marrow transplantation. It is hypothesized that in their quiescent state, HSCs primarily use glycolysis for energy production rather than mitochondrial oxidative phosphorylation (OXPHOS). In addition, the HSC switch from quiescence to activation occurs along a continuous...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36290845"
                        open
                      >
                        <summary class="article-expander__title">Mutational Landscape of Patients Referred for Elevated Hemoglobin Level</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36290845/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221102211743&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Background: Since the identification of JAK2 V617F and exon 12 mutations as driver mutations in polycythemia vera (PV) in 2005, molecular testing of these mutations for patients with erythrocytosis has become a routine clinical practice. However, the incidence of myeloid mutations other than the common JAK2 V617F mutation in unselected patients referred for elevated hemoglobin is not well studied. This study aimed to characterize the mutational landscape in a real-world population of patients...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36290314"
                        open
                      >
                        <summary class="article-expander__title">Lysosomes and Their Role in Regulating the Metabolism of Hematopoietic Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36290314/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221102211743&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) have the capacity to renew blood cells at all stages of life and are largely quiescent at a steady state. It is essential to understand the processes that govern quiescence in HSCs to enhance bone marrow transplantation. It is hypothesized that in their quiescent state, HSCs primarily use glycolysis for energy production rather than mitochondrial oxidative phosphorylation (OXPHOS). In addition, the HSC switch from quiescence to activation occurs along a continuous...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36290845"
                        open
                      >
                        <summary class="article-expander__title">Mutational Landscape of Patients Referred for Elevated Hemoglobin Level</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36290845/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221101212552&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Background: Since the identification of JAK2 V617F and exon 12 mutations as driver mutations in polycythemia vera (PV) in 2005, molecular testing of these mutations for patients with erythrocytosis has become a routine clinical practice. However, the incidence of myeloid mutations other than the common JAK2 V617F mutation in unselected patients referred for elevated hemoglobin is not well studied. This study aimed to characterize the mutational landscape in a real-world population of patients...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36290314"
                        open
                      >
                        <summary class="article-expander__title">Lysosomes and Their Role in Regulating the Metabolism of Hematopoietic Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36290314/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221101212552&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) have the capacity to renew blood cells at all stages of life and are largely quiescent at a steady state. It is essential to understand the processes that govern quiescence in HSCs to enhance bone marrow transplantation. It is hypothesized that in their quiescent state, HSCs primarily use glycolysis for energy production rather than mitochondrial oxidative phosphorylation (OXPHOS). In addition, the HSC switch from quiescence to activation occurs along a continuous...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36290845"
                        open
                      >
                        <summary class="article-expander__title">Mutational Landscape of Patients Referred for Elevated Hemoglobin Level</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36290845/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221031214247&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Background: Since the identification of JAK2 V617F and exon 12 mutations as driver mutations in polycythemia vera (PV) in 2005, molecular testing of these mutations for patients with erythrocytosis has become a routine clinical practice. However, the incidence of myeloid mutations other than the common JAK2 V617F mutation in unselected patients referred for elevated hemoglobin is not well studied. This study aimed to characterize the mutational landscape in a real-world population of patients...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36290314"
                        open
                      >
                        <summary class="article-expander__title">Lysosomes and Their Role in Regulating the Metabolism of Hematopoietic Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36290314/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221031214247&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) have the capacity to renew blood cells at all stages of life and are largely quiescent at a steady state. It is essential to understand the processes that govern quiescence in HSCs to enhance bone marrow transplantation. It is hypothesized that in their quiescent state, HSCs primarily use glycolysis for energy production rather than mitochondrial oxidative phosphorylation (OXPHOS). In addition, the HSC switch from quiescence to activation occurs along a continuous...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36290845"
                        open
                      >
                        <summary class="article-expander__title">Mutational Landscape of Patients Referred for Elevated Hemoglobin Level</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36290845/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221030212817&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Background: Since the identification of JAK2 V617F and exon 12 mutations as driver mutations in polycythemia vera (PV) in 2005, molecular testing of these mutations for patients with erythrocytosis has become a routine clinical practice. However, the incidence of myeloid mutations other than the common JAK2 V617F mutation in unselected patients referred for elevated hemoglobin is not well studied. This study aimed to characterize the mutational landscape in a real-world population of patients...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36290314"
                        open
                      >
                        <summary class="article-expander__title">Lysosomes and Their Role in Regulating the Metabolism of Hematopoietic Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36290314/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221030212817&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) have the capacity to renew blood cells at all stages of life and are largely quiescent at a steady state. It is essential to understand the processes that govern quiescence in HSCs to enhance bone marrow transplantation. It is hypothesized that in their quiescent state, HSCs primarily use glycolysis for energy production rather than mitochondrial oxidative phosphorylation (OXPHOS). In addition, the HSC switch from quiescence to activation occurs along a continuous...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36290845"
                        open
                      >
                        <summary class="article-expander__title">Mutational Landscape of Patients Referred for Elevated Hemoglobin Level</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36290845/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221029212729&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Background: Since the identification of JAK2 V617F and exon 12 mutations as driver mutations in polycythemia vera (PV) in 2005, molecular testing of these mutations for patients with erythrocytosis has become a routine clinical practice. However, the incidence of myeloid mutations other than the common JAK2 V617F mutation in unselected patients referred for elevated hemoglobin is not well studied. This study aimed to characterize the mutational landscape in a real-world population of patients...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36290314"
                        open
                      >
                        <summary class="article-expander__title">Lysosomes and Their Role in Regulating the Metabolism of Hematopoietic Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36290314/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221029212729&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) have the capacity to renew blood cells at all stages of life and are largely quiescent at a steady state. It is essential to understand the processes that govern quiescence in HSCs to enhance bone marrow transplantation. It is hypothesized that in their quiescent state, HSCs primarily use glycolysis for energy production rather than mitochondrial oxidative phosphorylation (OXPHOS). In addition, the HSC switch from quiescence to activation occurs along a continuous...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36290845"
                        open
                      >
                        <summary class="article-expander__title">Mutational Landscape of Patients Referred for Elevated Hemoglobin Level</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36290845/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221028211238&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Background: Since the identification of JAK2 V617F and exon 12 mutations as driver mutations in polycythemia vera (PV) in 2005, molecular testing of these mutations for patients with erythrocytosis has become a routine clinical practice. However, the incidence of myeloid mutations other than the common JAK2 V617F mutation in unselected patients referred for elevated hemoglobin is not well studied. This study aimed to characterize the mutational landscape in a real-world population of patients...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36290314"
                        open
                      >
                        <summary class="article-expander__title">Lysosomes and Their Role in Regulating the Metabolism of Hematopoietic Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36290314/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221028211238&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) have the capacity to renew blood cells at all stages of life and are largely quiescent at a steady state. It is essential to understand the processes that govern quiescence in HSCs to enhance bone marrow transplantation. It is hypothesized that in their quiescent state, HSCs primarily use glycolysis for energy production rather than mitochondrial oxidative phosphorylation (OXPHOS). In addition, the HSC switch from quiescence to activation occurs along a continuous...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36290845"
                        open
                      >
                        <summary class="article-expander__title">Mutational Landscape of Patients Referred for Elevated Hemoglobin Level</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36290845/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221027212537&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Background: Since the identification of JAK2 V617F and exon 12 mutations as driver mutations in polycythemia vera (PV) in 2005, molecular testing of these mutations for patients with erythrocytosis has become a routine clinical practice. However, the incidence of myeloid mutations other than the common JAK2 V617F mutation in unselected patients referred for elevated hemoglobin is not well studied. This study aimed to characterize the mutational landscape in a real-world population of patients...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36290314"
                        open
                      >
                        <summary class="article-expander__title">Lysosomes and Their Role in Regulating the Metabolism of Hematopoietic Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36290314/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221027212537&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) have the capacity to renew blood cells at all stages of life and are largely quiescent at a steady state. It is essential to understand the processes that govern quiescence in HSCs to enhance bone marrow transplantation. It is hypothesized that in their quiescent state, HSCs primarily use glycolysis for energy production rather than mitochondrial oxidative phosphorylation (OXPHOS). In addition, the HSC switch from quiescence to activation occurs along a continuous...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221128200439&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.8">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36291798"
                        open
                      >
                        <summary class="article-expander__title">Application of CRISPR for In Vivo Mouse Cancer Studies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36291798/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221113201202&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clustered regularly interspaced short palindromic repeats (CRISPR) are widely used in cancer research to edit specific genes and study their functions. This applies both to in vitro and in vivo studies where CRISPR technology has accelerated the generation of specific loss- or gain-of-function mutations. This review focuses on CRISPR for generating in vivo models of cancer by editing somatic cells in specific organs. The delivery of CRISPR/Cas to designated tissues and specific cell compartments...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36291798"
                        open
                      >
                        <summary class="article-expander__title">Application of CRISPR for In Vivo Mouse Cancer Studies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36291798/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221112201337&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clustered regularly interspaced short palindromic repeats (CRISPR) are widely used in cancer research to edit specific genes and study their functions. This applies both to in vitro and in vivo studies where CRISPR technology has accelerated the generation of specific loss- or gain-of-function mutations. This review focuses on CRISPR for generating in vivo models of cancer by editing somatic cells in specific organs. The delivery of CRISPR/Cas to designated tissues and specific cell compartments...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36291798"
                        open
                      >
                        <summary class="article-expander__title">Application of CRISPR for In Vivo Mouse Cancer Studies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36291798/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221111201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clustered regularly interspaced short palindromic repeats (CRISPR) are widely used in cancer research to edit specific genes and study their functions. This applies both to in vitro and in vivo studies where CRISPR technology has accelerated the generation of specific loss- or gain-of-function mutations. This review focuses on CRISPR for generating in vivo models of cancer by editing somatic cells in specific organs. The delivery of CRISPR/Cas to designated tissues and specific cell compartments...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36291798"
                        open
                      >
                        <summary class="article-expander__title">Application of CRISPR for In Vivo Mouse Cancer Studies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36291798/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221110201819&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clustered regularly interspaced short palindromic repeats (CRISPR) are widely used in cancer research to edit specific genes and study their functions. This applies both to in vitro and in vivo studies where CRISPR technology has accelerated the generation of specific loss- or gain-of-function mutations. This review focuses on CRISPR for generating in vivo models of cancer by editing somatic cells in specific organs. The delivery of CRISPR/Cas to designated tissues and specific cell compartments...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36291798"
                        open
                      >
                        <summary class="article-expander__title">Application of CRISPR for In Vivo Mouse Cancer Studies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36291798/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221109201530&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clustered regularly interspaced short palindromic repeats (CRISPR) are widely used in cancer research to edit specific genes and study their functions. This applies both to in vitro and in vivo studies where CRISPR technology has accelerated the generation of specific loss- or gain-of-function mutations. This review focuses on CRISPR for generating in vivo models of cancer by editing somatic cells in specific organs. The delivery of CRISPR/Cas to designated tissues and specific cell compartments...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36291798"
                        open
                      >
                        <summary class="article-expander__title">Application of CRISPR for In Vivo Mouse Cancer Studies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36291798/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221108201514&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clustered regularly interspaced short palindromic repeats (CRISPR) are widely used in cancer research to edit specific genes and study their functions. This applies both to in vitro and in vivo studies where CRISPR technology has accelerated the generation of specific loss- or gain-of-function mutations. This review focuses on CRISPR for generating in vivo models of cancer by editing somatic cells in specific organs. The delivery of CRISPR/Cas to designated tissues and specific cell compartments...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36291798"
                        open
                      >
                        <summary class="article-expander__title">Application of CRISPR for In Vivo Mouse Cancer Studies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36291798/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221107201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clustered regularly interspaced short palindromic repeats (CRISPR) are widely used in cancer research to edit specific genes and study their functions. This applies both to in vitro and in vivo studies where CRISPR technology has accelerated the generation of specific loss- or gain-of-function mutations. This review focuses on CRISPR for generating in vivo models of cancer by editing somatic cells in specific organs. The delivery of CRISPR/Cas to designated tissues and specific cell compartments...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36291798"
                        open
                      >
                        <summary class="article-expander__title">Application of CRISPR for In Vivo Mouse Cancer Studies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36291798/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221106201343&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clustered regularly interspaced short palindromic repeats (CRISPR) are widely used in cancer research to edit specific genes and study their functions. This applies both to in vitro and in vivo studies where CRISPR technology has accelerated the generation of specific loss- or gain-of-function mutations. This review focuses on CRISPR for generating in vivo models of cancer by editing somatic cells in specific organs. The delivery of CRISPR/Cas to designated tissues and specific cell compartments...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36291798"
                        open
                      >
                        <summary class="article-expander__title">Application of CRISPR for In Vivo Mouse Cancer Studies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36291798/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221105211555&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clustered regularly interspaced short palindromic repeats (CRISPR) are widely used in cancer research to edit specific genes and study their functions. This applies both to in vitro and in vivo studies where CRISPR technology has accelerated the generation of specific loss- or gain-of-function mutations. This review focuses on CRISPR for generating in vivo models of cancer by editing somatic cells in specific organs. The delivery of CRISPR/Cas to designated tissues and specific cell compartments...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36291798"
                        open
                      >
                        <summary class="article-expander__title">Application of CRISPR for In Vivo Mouse Cancer Studies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36291798/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221104211729&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clustered regularly interspaced short palindromic repeats (CRISPR) are widely used in cancer research to edit specific genes and study their functions. This applies both to in vitro and in vivo studies where CRISPR technology has accelerated the generation of specific loss- or gain-of-function mutations. This review focuses on CRISPR for generating in vivo models of cancer by editing somatic cells in specific organs. The delivery of CRISPR/Cas to designated tissues and specific cell compartments...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36291798"
                        open
                      >
                        <summary class="article-expander__title">Application of CRISPR for In Vivo Mouse Cancer Studies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36291798/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221103212023&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clustered regularly interspaced short palindromic repeats (CRISPR) are widely used in cancer research to edit specific genes and study their functions. This applies both to in vitro and in vivo studies where CRISPR technology has accelerated the generation of specific loss- or gain-of-function mutations. This review focuses on CRISPR for generating in vivo models of cancer by editing somatic cells in specific organs. The delivery of CRISPR/Cas to designated tissues and specific cell compartments...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36291798"
                        open
                      >
                        <summary class="article-expander__title">Application of CRISPR for In Vivo Mouse Cancer Studies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36291798/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221102211743&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clustered regularly interspaced short palindromic repeats (CRISPR) are widely used in cancer research to edit specific genes and study their functions. This applies both to in vitro and in vivo studies where CRISPR technology has accelerated the generation of specific loss- or gain-of-function mutations. This review focuses on CRISPR for generating in vivo models of cancer by editing somatic cells in specific organs. The delivery of CRISPR/Cas to designated tissues and specific cell compartments...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36291798"
                        open
                      >
                        <summary class="article-expander__title">Application of CRISPR for In Vivo Mouse Cancer Studies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36291798/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221101212552&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clustered regularly interspaced short palindromic repeats (CRISPR) are widely used in cancer research to edit specific genes and study their functions. This applies both to in vitro and in vivo studies where CRISPR technology has accelerated the generation of specific loss- or gain-of-function mutations. This review focuses on CRISPR for generating in vivo models of cancer by editing somatic cells in specific organs. The delivery of CRISPR/Cas to designated tissues and specific cell compartments...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36291798"
                        open
                      >
                        <summary class="article-expander__title">Application of CRISPR for In Vivo Mouse Cancer Studies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36291798/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221031214247&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clustered regularly interspaced short palindromic repeats (CRISPR) are widely used in cancer research to edit specific genes and study their functions. This applies both to in vitro and in vivo studies where CRISPR technology has accelerated the generation of specific loss- or gain-of-function mutations. This review focuses on CRISPR for generating in vivo models of cancer by editing somatic cells in specific organs. The delivery of CRISPR/Cas to designated tissues and specific cell compartments...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36291798"
                        open
                      >
                        <summary class="article-expander__title">Application of CRISPR for In Vivo Mouse Cancer Studies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36291798/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221030212817&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clustered regularly interspaced short palindromic repeats (CRISPR) are widely used in cancer research to edit specific genes and study their functions. This applies both to in vitro and in vivo studies where CRISPR technology has accelerated the generation of specific loss- or gain-of-function mutations. This review focuses on CRISPR for generating in vivo models of cancer by editing somatic cells in specific organs. The delivery of CRISPR/Cas to designated tissues and specific cell compartments...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36291798"
                        open
                      >
                        <summary class="article-expander__title">Application of CRISPR for In Vivo Mouse Cancer Studies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36291798/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221029212729&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clustered regularly interspaced short palindromic repeats (CRISPR) are widely used in cancer research to edit specific genes and study their functions. This applies both to in vitro and in vivo studies where CRISPR technology has accelerated the generation of specific loss- or gain-of-function mutations. This review focuses on CRISPR for generating in vivo models of cancer by editing somatic cells in specific organs. The delivery of CRISPR/Cas to designated tissues and specific cell compartments...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36291798"
                        open
                      >
                        <summary class="article-expander__title">Application of CRISPR for In Vivo Mouse Cancer Studies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36291798/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221028211238&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clustered regularly interspaced short palindromic repeats (CRISPR) are widely used in cancer research to edit specific genes and study their functions. This applies both to in vitro and in vivo studies where CRISPR technology has accelerated the generation of specific loss- or gain-of-function mutations. This review focuses on CRISPR for generating in vivo models of cancer by editing somatic cells in specific organs. The delivery of CRISPR/Cas to designated tissues and specific cell compartments...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36291798"
                        open
                      >
                        <summary class="article-expander__title">Application of CRISPR for In Vivo Mouse Cancer Studies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36291798/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221027212535&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clustered regularly interspaced short palindromic repeats (CRISPR) are widely used in cancer research to edit specific genes and study their functions. This applies both to in vitro and in vivo studies where CRISPR technology has accelerated the generation of specific loss- or gain-of-function mutations. This review focuses on CRISPR for generating in vivo models of cancer by editing somatic cells in specific organs. The delivery of CRISPR/Cas to designated tissues and specific cell compartments...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-26 datetime="2022-10-26T10:00:00.000Z">2022-10-26</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-26 datetime="2022-10-26T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221128200439&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.8">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36289336"
                        open
                      >
                        <summary class="article-expander__title">Phenotypic plasticity and genetic control in colorectal cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36289336/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221112201337&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genetic and epigenetic variation, together with transcriptional plasticity, contribute to intratumour heterogeneity. The interplay of these biological processes and their respective contributions to tumour evolution remain unknown. Here we show that intratumour genetic ancestry only infrequently affects gene expression traits and subclonal evolution in colorectal cancer (CRC). Using spatially resolved paired whole-genome and transcriptome sequencing, we find that the majority of intratumour...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36289336"
                        open
                      >
                        <summary class="article-expander__title">Phenotypic plasticity and genetic control in colorectal cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36289336/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221111201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genetic and epigenetic variation, together with transcriptional plasticity, contribute to intratumour heterogeneity. The interplay of these biological processes and their respective contributions to tumour evolution remain unknown. Here we show that intratumour genetic ancestry only infrequently affects gene expression traits and subclonal evolution in colorectal cancer (CRC). Using spatially resolved paired whole-genome and transcriptome sequencing, we find that the majority of intratumour...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36289336"
                        open
                      >
                        <summary class="article-expander__title">Phenotypic plasticity and genetic control in colorectal cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36289336/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221110201819&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genetic and epigenetic variation, together with transcriptional plasticity, contribute to intratumour heterogeneity. The interplay of these biological processes and their respective contributions to tumour evolution remain unknown. Here we show that intratumour genetic ancestry only infrequently affects gene expression traits and subclonal evolution in colorectal cancer (CRC). Using spatially resolved paired whole-genome and transcriptome sequencing, we find that the majority of intratumour...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36289335"
                        open
                      >
                        <summary class="article-expander__title">The co-evolution of the genome and epigenome in colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36289335/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221110201819&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal malignancies are a leading cause of cancer-related death^(1 )and have undergone extensive genomic study^(2,3). However, DNA mutations alone do not fully explain malignant transformation^(4-7). Here we investigate the co-evolution of the genome and epigenome of colorectal tumours at single-clone resolution using spatial multi-omic profiling of individual glands. We collected 1,370 samples from 30 primary cancers and 8 concomitant adenomas and generated 1,207 chromatin accessibility...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36289336"
                        open
                      >
                        <summary class="article-expander__title">Phenotypic plasticity and genetic control in colorectal cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36289336/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221109201530&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genetic and epigenetic variation, together with transcriptional plasticity, contribute to intratumour heterogeneity. The interplay of these biological processes and their respective contributions to tumour evolution remain unknown. Here we show that intratumour genetic ancestry only infrequently affects gene expression traits and subclonal evolution in colorectal cancer (CRC). Using spatially resolved paired whole-genome and transcriptome sequencing, we find that the majority of intratumour...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36289335"
                        open
                      >
                        <summary class="article-expander__title">The co-evolution of the genome and epigenome in colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36289335/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221109201530&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal malignancies are a leading cause of cancer-related death^(1 )and have undergone extensive genomic study^(2,3). However, DNA mutations alone do not fully explain malignant transformation^(4-7). Here we investigate the co-evolution of the genome and epigenome of colorectal tumours at single-clone resolution using spatial multi-omic profiling of individual glands. We collected 1,370 samples from 30 primary cancers and 8 concomitant adenomas and generated 1,207 chromatin accessibility...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36289203"
                        open
                      >
                        <summary class="article-expander__title">Clonal somatic copy number altered driver events inform drug sensitivity in high-grade serous ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36289203/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221109201530&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosomal instability is a major challenge to patient stratification and targeted drug development for high-grade serous ovarian carcinoma (HGSOC). Here we show that somatic copy number alterations (SCNAs) in frequently amplified HGSOC cancer genes significantly correlate with gene expression and methylation status. We identify five prevalent clonal driver SCNAs (chromosomal amplifications encompassing MYC, PIK3CA, CCNE1, KRAS and TERT) from multi-regional HGSOC data and reason that their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36289336"
                        open
                      >
                        <summary class="article-expander__title">Phenotypic plasticity and genetic control in colorectal cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36289336/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221108201514&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genetic and epigenetic variation, together with transcriptional plasticity, contribute to intratumour heterogeneity. The interplay of these biological processes and their respective contributions to tumour evolution remain unknown. Here we show that intratumour genetic ancestry only infrequently affects gene expression traits and subclonal evolution in colorectal cancer (CRC). Using spatially resolved paired whole-genome and transcriptome sequencing, we find that the majority of intratumour...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36289335"
                        open
                      >
                        <summary class="article-expander__title">The co-evolution of the genome and epigenome in colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36289335/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221108201514&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal malignancies are a leading cause of cancer-related death^(1 )and have undergone extensive genomic study^(2,3). However, DNA mutations alone do not fully explain malignant transformation^(4-7). Here we investigate the co-evolution of the genome and epigenome of colorectal tumours at single-clone resolution using spatial multi-omic profiling of individual glands. We collected 1,370 samples from 30 primary cancers and 8 concomitant adenomas and generated 1,207 chromatin accessibility...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36289203"
                        open
                      >
                        <summary class="article-expander__title">Clonal somatic copy number altered driver events inform drug sensitivity in high-grade serous ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36289203/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221108201514&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosomal instability is a major challenge to patient stratification and targeted drug development for high-grade serous ovarian carcinoma (HGSOC). Here we show that somatic copy number alterations (SCNAs) in frequently amplified HGSOC cancer genes significantly correlate with gene expression and methylation status. We identify five prevalent clonal driver SCNAs (chromosomal amplifications encompassing MYC, PIK3CA, CCNE1, KRAS and TERT) from multi-regional HGSOC data and reason that their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36289336"
                        open
                      >
                        <summary class="article-expander__title">Phenotypic plasticity and genetic control in colorectal cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36289336/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221107201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genetic and epigenetic variation, together with transcriptional plasticity, contribute to intratumour heterogeneity. The interplay of these biological processes and their respective contributions to tumour evolution remain unknown. Here we show that intratumour genetic ancestry only infrequently affects gene expression traits and subclonal evolution in colorectal cancer (CRC). Using spatially resolved paired whole-genome and transcriptome sequencing, we find that the majority of intratumour...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36289335"
                        open
                      >
                        <summary class="article-expander__title">The co-evolution of the genome and epigenome in colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36289335/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221107201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal malignancies are a leading cause of cancer-related death^(1 )and have undergone extensive genomic study^(2,3). However, DNA mutations alone do not fully explain malignant transformation^(4-7). Here we investigate the co-evolution of the genome and epigenome of colorectal tumours at single-clone resolution using spatial multi-omic profiling of individual glands. We collected 1,370 samples from 30 primary cancers and 8 concomitant adenomas and generated 1,207 chromatin accessibility...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36289203"
                        open
                      >
                        <summary class="article-expander__title">Clonal somatic copy number altered driver events inform drug sensitivity in high-grade serous ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36289203/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221107201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosomal instability is a major challenge to patient stratification and targeted drug development for high-grade serous ovarian carcinoma (HGSOC). Here we show that somatic copy number alterations (SCNAs) in frequently amplified HGSOC cancer genes significantly correlate with gene expression and methylation status. We identify five prevalent clonal driver SCNAs (chromosomal amplifications encompassing MYC, PIK3CA, CCNE1, KRAS and TERT) from multi-regional HGSOC data and reason that their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36289336"
                        open
                      >
                        <summary class="article-expander__title">Phenotypic plasticity and genetic control in colorectal cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36289336/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221106201343&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genetic and epigenetic variation, together with transcriptional plasticity, contribute to intratumour heterogeneity. The interplay of these biological processes and their respective contributions to tumour evolution remain unknown. Here we show that intratumour genetic ancestry only infrequently affects gene expression traits and subclonal evolution in colorectal cancer (CRC). Using spatially resolved paired whole-genome and transcriptome sequencing, we find that the majority of intratumour...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36289335"
                        open
                      >
                        <summary class="article-expander__title">The co-evolution of the genome and epigenome in colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36289335/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221106201343&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal malignancies are a leading cause of cancer-related death^(1 )and have undergone extensive genomic study^(2,3). However, DNA mutations alone do not fully explain malignant transformation^(4-7). Here we investigate the co-evolution of the genome and epigenome of colorectal tumours at single-clone resolution using spatial multi-omic profiling of individual glands. We collected 1,370 samples from 30 primary cancers and 8 concomitant adenomas and generated 1,207 chromatin accessibility...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36289203"
                        open
                      >
                        <summary class="article-expander__title">Clonal somatic copy number altered driver events inform drug sensitivity in high-grade serous ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36289203/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221106201343&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosomal instability is a major challenge to patient stratification and targeted drug development for high-grade serous ovarian carcinoma (HGSOC). Here we show that somatic copy number alterations (SCNAs) in frequently amplified HGSOC cancer genes significantly correlate with gene expression and methylation status. We identify five prevalent clonal driver SCNAs (chromosomal amplifications encompassing MYC, PIK3CA, CCNE1, KRAS and TERT) from multi-regional HGSOC data and reason that their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36289336"
                        open
                      >
                        <summary class="article-expander__title">Phenotypic plasticity and genetic control in colorectal cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36289336/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221105211555&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genetic and epigenetic variation, together with transcriptional plasticity, contribute to intratumour heterogeneity. The interplay of these biological processes and their respective contributions to tumour evolution remain unknown. Here we show that intratumour genetic ancestry only infrequently affects gene expression traits and subclonal evolution in colorectal cancer (CRC). Using spatially resolved paired whole-genome and transcriptome sequencing, we find that the majority of intratumour...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36289335"
                        open
                      >
                        <summary class="article-expander__title">The co-evolution of the genome and epigenome in colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36289335/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221105211555&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal malignancies are a leading cause of cancer-related death^(1 )and have undergone extensive genomic study^(2,3). However, DNA mutations alone do not fully explain malignant transformation^(4-7). Here we investigate the co-evolution of the genome and epigenome of colorectal tumours at single-clone resolution using spatial multi-omic profiling of individual glands. We collected 1,370 samples from 30 primary cancers and 8 concomitant adenomas and generated 1,207 chromatin accessibility...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36289203"
                        open
                      >
                        <summary class="article-expander__title">Clonal somatic copy number altered driver events inform drug sensitivity in high-grade serous ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36289203/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221105211555&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosomal instability is a major challenge to patient stratification and targeted drug development for high-grade serous ovarian carcinoma (HGSOC). Here we show that somatic copy number alterations (SCNAs) in frequently amplified HGSOC cancer genes significantly correlate with gene expression and methylation status. We identify five prevalent clonal driver SCNAs (chromosomal amplifications encompassing MYC, PIK3CA, CCNE1, KRAS and TERT) from multi-regional HGSOC data and reason that their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36289336"
                        open
                      >
                        <summary class="article-expander__title">Phenotypic plasticity and genetic control in colorectal cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36289336/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221104211729&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genetic and epigenetic variation, together with transcriptional plasticity, contribute to intratumour heterogeneity. The interplay of these biological processes and their respective contributions to tumour evolution remain unknown. Here we show that intratumour genetic ancestry only infrequently affects gene expression traits and subclonal evolution in colorectal cancer (CRC). Using spatially resolved paired whole-genome and transcriptome sequencing, we find that the majority of intratumour...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36289335"
                        open
                      >
                        <summary class="article-expander__title">The co-evolution of the genome and epigenome in colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36289335/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221104211729&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal malignancies are a leading cause of cancer-related death^(1 )and have undergone extensive genomic study^(2,3). However, DNA mutations alone do not fully explain malignant transformation^(4-7). Here we investigate the co-evolution of the genome and epigenome of colorectal tumours at single-clone resolution using spatial multi-omic profiling of individual glands. We collected 1,370 samples from 30 primary cancers and 8 concomitant adenomas and generated 1,207 chromatin accessibility...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36289203"
                        open
                      >
                        <summary class="article-expander__title">Clonal somatic copy number altered driver events inform drug sensitivity in high-grade serous ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36289203/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221104211729&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosomal instability is a major challenge to patient stratification and targeted drug development for high-grade serous ovarian carcinoma (HGSOC). Here we show that somatic copy number alterations (SCNAs) in frequently amplified HGSOC cancer genes significantly correlate with gene expression and methylation status. We identify five prevalent clonal driver SCNAs (chromosomal amplifications encompassing MYC, PIK3CA, CCNE1, KRAS and TERT) from multi-regional HGSOC data and reason that their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36289336"
                        open
                      >
                        <summary class="article-expander__title">Phenotypic plasticity and genetic control in colorectal cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36289336/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221103212023&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genetic and epigenetic variation, together with transcriptional plasticity, contribute to intratumour heterogeneity. The interplay of these biological processes and their respective contributions to tumour evolution remain unknown. Here we show that intratumour genetic ancestry only infrequently affects gene expression traits and subclonal evolution in colorectal cancer (CRC). Using spatially resolved paired whole-genome and transcriptome sequencing, we find that the majority of intratumour...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36289335"
                        open
                      >
                        <summary class="article-expander__title">The co-evolution of the genome and epigenome in colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36289335/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221103212023&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal malignancies are a leading cause of cancer-related death^(1 )and have undergone extensive genomic study^(2,3). However, DNA mutations alone do not fully explain malignant transformation^(4-7). Here we investigate the co-evolution of the genome and epigenome of colorectal tumours at single-clone resolution using spatial multi-omic profiling of individual glands. We collected 1,370 samples from 30 primary cancers and 8 concomitant adenomas and generated 1,207 chromatin accessibility...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36289203"
                        open
                      >
                        <summary class="article-expander__title">Clonal somatic copy number altered driver events inform drug sensitivity in high-grade serous ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36289203/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221103212023&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosomal instability is a major challenge to patient stratification and targeted drug development for high-grade serous ovarian carcinoma (HGSOC). Here we show that somatic copy number alterations (SCNAs) in frequently amplified HGSOC cancer genes significantly correlate with gene expression and methylation status. We identify five prevalent clonal driver SCNAs (chromosomal amplifications encompassing MYC, PIK3CA, CCNE1, KRAS and TERT) from multi-regional HGSOC data and reason that their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36289336"
                        open
                      >
                        <summary class="article-expander__title">Phenotypic plasticity and genetic control in colorectal cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36289336/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221102211743&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genetic and epigenetic variation, together with transcriptional plasticity, contribute to intratumour heterogeneity. The interplay of these biological processes and their respective contributions to tumour evolution remain unknown. Here we show that intratumour genetic ancestry only infrequently affects gene expression traits and subclonal evolution in colorectal cancer (CRC). Using spatially resolved paired whole-genome and transcriptome sequencing, we find that the majority of intratumour...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36289335"
                        open
                      >
                        <summary class="article-expander__title">The co-evolution of the genome and epigenome in colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36289335/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221102211743&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal malignancies are a leading cause of cancer-related death^(1 )and have undergone extensive genomic study^(2,3). However, DNA mutations alone do not fully explain malignant transformation^(4-7). Here we investigate the co-evolution of the genome and epigenome of colorectal tumours at single-clone resolution using spatial multi-omic profiling of individual glands. We collected 1,370 samples from 30 primary cancers and 8 concomitant adenomas and generated 1,207 chromatin accessibility...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36289203"
                        open
                      >
                        <summary class="article-expander__title">Clonal somatic copy number altered driver events inform drug sensitivity in high-grade serous ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36289203/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221102211743&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosomal instability is a major challenge to patient stratification and targeted drug development for high-grade serous ovarian carcinoma (HGSOC). Here we show that somatic copy number alterations (SCNAs) in frequently amplified HGSOC cancer genes significantly correlate with gene expression and methylation status. We identify five prevalent clonal driver SCNAs (chromosomal amplifications encompassing MYC, PIK3CA, CCNE1, KRAS and TERT) from multi-regional HGSOC data and reason that their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36289336"
                        open
                      >
                        <summary class="article-expander__title">Phenotypic plasticity and genetic control in colorectal cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36289336/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221101212552&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genetic and epigenetic variation, together with transcriptional plasticity, contribute to intratumour heterogeneity. The interplay of these biological processes and their respective contributions to tumour evolution remain unknown. Here we show that intratumour genetic ancestry only infrequently affects gene expression traits and subclonal evolution in colorectal cancer (CRC). Using spatially resolved paired whole-genome and transcriptome sequencing, we find that the majority of intratumour...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36289335"
                        open
                      >
                        <summary class="article-expander__title">The co-evolution of the genome and epigenome in colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36289335/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221101212552&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal malignancies are a leading cause of cancer-related death^(1 )and have undergone extensive genomic study^(2,3). However, DNA mutations alone do not fully explain malignant transformation^(4-7). Here we investigate the co-evolution of the genome and epigenome of colorectal tumours at single-clone resolution using spatial multi-omic profiling of individual glands. We collected 1,370 samples from 30 primary cancers and 8 concomitant adenomas and generated 1,207 chromatin accessibility...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36289203"
                        open
                      >
                        <summary class="article-expander__title">Clonal somatic copy number altered driver events inform drug sensitivity in high-grade serous ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36289203/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221101212552&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosomal instability is a major challenge to patient stratification and targeted drug development for high-grade serous ovarian carcinoma (HGSOC). Here we show that somatic copy number alterations (SCNAs) in frequently amplified HGSOC cancer genes significantly correlate with gene expression and methylation status. We identify five prevalent clonal driver SCNAs (chromosomal amplifications encompassing MYC, PIK3CA, CCNE1, KRAS and TERT) from multi-regional HGSOC data and reason that their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36289336"
                        open
                      >
                        <summary class="article-expander__title">Phenotypic plasticity and genetic control in colorectal cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36289336/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221031214247&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genetic and epigenetic variation, together with transcriptional plasticity, contribute to intratumour heterogeneity. The interplay of these biological processes and their respective contributions to tumour evolution remain unknown. Here we show that intratumour genetic ancestry only infrequently affects gene expression traits and subclonal evolution in colorectal cancer (CRC). Using spatially resolved paired whole-genome and transcriptome sequencing, we find that the majority of intratumour...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36289335"
                        open
                      >
                        <summary class="article-expander__title">The co-evolution of the genome and epigenome in colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36289335/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221031214247&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal malignancies are a leading cause of cancer-related death^(1 )and have undergone extensive genomic study^(2,3). However, DNA mutations alone do not fully explain malignant transformation^(4-7). Here we investigate the co-evolution of the genome and epigenome of colorectal tumours at single-clone resolution using spatial multi-omic profiling of individual glands. We collected 1,370 samples from 30 primary cancers and 8 concomitant adenomas and generated 1,207 chromatin accessibility...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36289203"
                        open
                      >
                        <summary class="article-expander__title">Clonal somatic copy number altered driver events inform drug sensitivity in high-grade serous ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36289203/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221031214247&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosomal instability is a major challenge to patient stratification and targeted drug development for high-grade serous ovarian carcinoma (HGSOC). Here we show that somatic copy number alterations (SCNAs) in frequently amplified HGSOC cancer genes significantly correlate with gene expression and methylation status. We identify five prevalent clonal driver SCNAs (chromosomal amplifications encompassing MYC, PIK3CA, CCNE1, KRAS and TERT) from multi-regional HGSOC data and reason that their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36289336"
                        open
                      >
                        <summary class="article-expander__title">Phenotypic plasticity and genetic control in colorectal cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36289336/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221030212817&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genetic and epigenetic variation, together with transcriptional plasticity, contribute to intratumour heterogeneity. The interplay of these biological processes and their respective contributions to tumour evolution remain unknown. Here we show that intratumour genetic ancestry only infrequently affects gene expression traits and subclonal evolution in colorectal cancer (CRC). Using spatially resolved paired whole-genome and transcriptome sequencing, we find that the majority of intratumour...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36289335"
                        open
                      >
                        <summary class="article-expander__title">The co-evolution of the genome and epigenome in colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36289335/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221030212817&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal malignancies are a leading cause of cancer-related death^(1 )and have undergone extensive genomic study^(2,3). However, DNA mutations alone do not fully explain malignant transformation^(4-7). Here we investigate the co-evolution of the genome and epigenome of colorectal tumours at single-clone resolution using spatial multi-omic profiling of individual glands. We collected 1,370 samples from 30 primary cancers and 8 concomitant adenomas and generated 1,207 chromatin accessibility...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36289203"
                        open
                      >
                        <summary class="article-expander__title">Clonal somatic copy number altered driver events inform drug sensitivity in high-grade serous ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36289203/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221030212817&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosomal instability is a major challenge to patient stratification and targeted drug development for high-grade serous ovarian carcinoma (HGSOC). Here we show that somatic copy number alterations (SCNAs) in frequently amplified HGSOC cancer genes significantly correlate with gene expression and methylation status. We identify five prevalent clonal driver SCNAs (chromosomal amplifications encompassing MYC, PIK3CA, CCNE1, KRAS and TERT) from multi-regional HGSOC data and reason that their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36289336"
                        open
                      >
                        <summary class="article-expander__title">Phenotypic plasticity and genetic control in colorectal cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36289336/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221029212729&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genetic and epigenetic variation, together with transcriptional plasticity, contribute to intratumour heterogeneity. The interplay of these biological processes and their respective contributions to tumour evolution remain unknown. Here we show that intratumour genetic ancestry only infrequently affects gene expression traits and subclonal evolution in colorectal cancer (CRC). Using spatially resolved paired whole-genome and transcriptome sequencing, we find that the majority of intratumour...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36289335"
                        open
                      >
                        <summary class="article-expander__title">The co-evolution of the genome and epigenome in colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36289335/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221029212729&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal malignancies are a leading cause of cancer-related death^(1 )and have undergone extensive genomic study^(2,3). However, DNA mutations alone do not fully explain malignant transformation^(4-7). Here we investigate the co-evolution of the genome and epigenome of colorectal tumours at single-clone resolution using spatial multi-omic profiling of individual glands. We collected 1,370 samples from 30 primary cancers and 8 concomitant adenomas and generated 1,207 chromatin accessibility...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36289203"
                        open
                      >
                        <summary class="article-expander__title">Clonal somatic copy number altered driver events inform drug sensitivity in high-grade serous ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36289203/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221029212729&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosomal instability is a major challenge to patient stratification and targeted drug development for high-grade serous ovarian carcinoma (HGSOC). Here we show that somatic copy number alterations (SCNAs) in frequently amplified HGSOC cancer genes significantly correlate with gene expression and methylation status. We identify five prevalent clonal driver SCNAs (chromosomal amplifications encompassing MYC, PIK3CA, CCNE1, KRAS and TERT) from multi-regional HGSOC data and reason that their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36289336"
                        open
                      >
                        <summary class="article-expander__title">Phenotypic plasticity and genetic control in colorectal cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36289336/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221028211238&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genetic and epigenetic variation, together with transcriptional plasticity, contribute to intratumour heterogeneity. The interplay of these biological processes and their respective contributions to tumour evolution remain unknown. Here we show that intratumour genetic ancestry only infrequently affects gene expression traits and subclonal evolution in colorectal cancer (CRC). Using spatially resolved paired whole-genome and transcriptome sequencing, we find that the majority of intratumour...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36289335"
                        open
                      >
                        <summary class="article-expander__title">The co-evolution of the genome and epigenome in colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36289335/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221028211238&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal malignancies are a leading cause of cancer-related death^(1 )and have undergone extensive genomic study^(2,3). However, DNA mutations alone do not fully explain malignant transformation^(4-7). Here we investigate the co-evolution of the genome and epigenome of colorectal tumours at single-clone resolution using spatial multi-omic profiling of individual glands. We collected 1,370 samples from 30 primary cancers and 8 concomitant adenomas and generated 1,207 chromatin accessibility...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36289203"
                        open
                      >
                        <summary class="article-expander__title">Clonal somatic copy number altered driver events inform drug sensitivity in high-grade serous ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36289203/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221028211238&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosomal instability is a major challenge to patient stratification and targeted drug development for high-grade serous ovarian carcinoma (HGSOC). Here we show that somatic copy number alterations (SCNAs) in frequently amplified HGSOC cancer genes significantly correlate with gene expression and methylation status. We identify five prevalent clonal driver SCNAs (chromosomal amplifications encompassing MYC, PIK3CA, CCNE1, KRAS and TERT) from multi-regional HGSOC data and reason that their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36289336"
                        open
                      >
                        <summary class="article-expander__title">Phenotypic plasticity and genetic control in colorectal cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36289336/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221027212535&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Genetic and epigenetic variation, together with transcriptional plasticity, contribute to intratumour heterogeneity. The interplay of these biological processes and their respective contributions to tumour evolution remain unknown. Here we show that intratumour genetic ancestry only infrequently affects gene expression traits and subclonal evolution in colorectal cancer (CRC). Using spatially resolved paired whole-genome and transcriptome sequencing, we find that the majority of intratumour...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36289335"
                        open
                      >
                        <summary class="article-expander__title">The co-evolution of the genome and epigenome in colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36289335/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221027212535&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal malignancies are a leading cause of cancer-related death^(1 )and have undergone extensive genomic study^(2,3). However, DNA mutations alone do not fully explain malignant transformation^(4-7). Here we investigate the co-evolution of the genome and epigenome of colorectal tumours at single-clone resolution using spatial multi-omic profiling of individual glands. We collected 1,370 samples from 30 primary cancers and 8 concomitant adenomas and generated 1,207 chromatin accessibility...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-25 datetime="2022-10-25T10:00:00.000Z">2022-10-25</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-25 datetime="2022-10-25T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.8&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20221128200439&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36281508"
                        open
                      >
                        <summary class="article-expander__title">Clinical, Mutational, and Transcriptomic Characteristics in Elderly Korean Individuals With Clonal Hematopoiesis Driver Mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36281508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221109201530&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Somatic CHIP driver mutations are common among the elderly in Korea and are detected in various genes, including DNMT3A and TET2. Our study highlights that chronic dysregulation of innate immune signaling is associated with the pathogenesis of various diseases, including hematologic malignancies.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36281508"
                        open
                      >
                        <summary class="article-expander__title">Clinical, Mutational, and Transcriptomic Characteristics in Elderly Korean Individuals With Clonal Hematopoiesis Driver Mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36281508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221108201514&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Somatic CHIP driver mutations are common among the elderly in Korea and are detected in various genes, including DNMT3A and TET2. Our study highlights that chronic dysregulation of innate immune signaling is associated with the pathogenesis of various diseases, including hematologic malignancies.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36281508"
                        open
                      >
                        <summary class="article-expander__title">Clinical, Mutational, and Transcriptomic Characteristics in Elderly Korean Individuals With Clonal Hematopoiesis Driver Mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36281508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221107201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Somatic CHIP driver mutations are common among the elderly in Korea and are detected in various genes, including DNMT3A and TET2. Our study highlights that chronic dysregulation of innate immune signaling is associated with the pathogenesis of various diseases, including hematologic malignancies.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36281508"
                        open
                      >
                        <summary class="article-expander__title">Clinical, Mutational, and Transcriptomic Characteristics in Elderly Korean Individuals With Clonal Hematopoiesis Driver Mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36281508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221106201343&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Somatic CHIP driver mutations are common among the elderly in Korea and are detected in various genes, including DNMT3A and TET2. Our study highlights that chronic dysregulation of innate immune signaling is associated with the pathogenesis of various diseases, including hematologic malignancies.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36281508"
                        open
                      >
                        <summary class="article-expander__title">Clinical, Mutational, and Transcriptomic Characteristics in Elderly Korean Individuals With Clonal Hematopoiesis Driver Mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36281508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221105211555&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Somatic CHIP driver mutations are common among the elderly in Korea and are detected in various genes, including DNMT3A and TET2. Our study highlights that chronic dysregulation of innate immune signaling is associated with the pathogenesis of various diseases, including hematologic malignancies.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36281508"
                        open
                      >
                        <summary class="article-expander__title">Clinical, Mutational, and Transcriptomic Characteristics in Elderly Korean Individuals With Clonal Hematopoiesis Driver Mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36281508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221104211730&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Somatic CHIP driver mutations are common among the elderly in Korea and are detected in various genes, including DNMT3A and TET2. Our study highlights that chronic dysregulation of innate immune signaling is associated with the pathogenesis of various diseases, including hematologic malignancies.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36281508"
                        open
                      >
                        <summary class="article-expander__title">Clinical, Mutational, and Transcriptomic Characteristics in Elderly Korean Individuals With Clonal Hematopoiesis Driver Mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36281508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221103212023&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Somatic CHIP driver mutations are common among the elderly in Korea and are detected in various genes, including DNMT3A and TET2. Our study highlights that chronic dysregulation of innate immune signaling is associated with the pathogenesis of various diseases, including hematologic malignancies.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36281508"
                        open
                      >
                        <summary class="article-expander__title">Clinical, Mutational, and Transcriptomic Characteristics in Elderly Korean Individuals With Clonal Hematopoiesis Driver Mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36281508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221102211743&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Somatic CHIP driver mutations are common among the elderly in Korea and are detected in various genes, including DNMT3A and TET2. Our study highlights that chronic dysregulation of innate immune signaling is associated with the pathogenesis of various diseases, including hematologic malignancies.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36281508"
                        open
                      >
                        <summary class="article-expander__title">Clinical, Mutational, and Transcriptomic Characteristics in Elderly Korean Individuals With Clonal Hematopoiesis Driver Mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36281508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221101212552&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Somatic CHIP driver mutations are common among the elderly in Korea and are detected in various genes, including DNMT3A and TET2. Our study highlights that chronic dysregulation of innate immune signaling is associated with the pathogenesis of various diseases, including hematologic malignancies.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36281508"
                        open
                      >
                        <summary class="article-expander__title">Clinical, Mutational, and Transcriptomic Characteristics in Elderly Korean Individuals With Clonal Hematopoiesis Driver Mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36281508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221031214247&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Somatic CHIP driver mutations are common among the elderly in Korea and are detected in various genes, including DNMT3A and TET2. Our study highlights that chronic dysregulation of innate immune signaling is associated with the pathogenesis of various diseases, including hematologic malignancies.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36281508"
                        open
                      >
                        <summary class="article-expander__title">Clinical, Mutational, and Transcriptomic Characteristics in Elderly Korean Individuals With Clonal Hematopoiesis Driver Mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36281508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221030212817&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Somatic CHIP driver mutations are common among the elderly in Korea and are detected in various genes, including DNMT3A and TET2. Our study highlights that chronic dysregulation of innate immune signaling is associated with the pathogenesis of various diseases, including hematologic malignancies.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36281508"
                        open
                      >
                        <summary class="article-expander__title">Clinical, Mutational, and Transcriptomic Characteristics in Elderly Korean Individuals With Clonal Hematopoiesis Driver Mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36281508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221029212729&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Somatic CHIP driver mutations are common among the elderly in Korea and are detected in various genes, including DNMT3A and TET2. Our study highlights that chronic dysregulation of innate immune signaling is associated with the pathogenesis of various diseases, including hematologic malignancies.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36281508"
                        open
                      >
                        <summary class="article-expander__title">Clinical, Mutational, and Transcriptomic Characteristics in Elderly Korean Individuals With Clonal Hematopoiesis Driver Mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36281508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221028211238&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Somatic CHIP driver mutations are common among the elderly in Korea and are detected in various genes, including DNMT3A and TET2. Our study highlights that chronic dysregulation of innate immune signaling is associated with the pathogenesis of various diseases, including hematologic malignancies.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36281508"
                        open
                      >
                        <summary class="article-expander__title">Clinical, Mutational, and Transcriptomic Characteristics in Elderly Korean Individuals With Clonal Hematopoiesis Driver Mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36281508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221027212537&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Somatic CHIP driver mutations are common among the elderly in Korea and are detected in various genes, including DNMT3A and TET2. Our study highlights that chronic dysregulation of innate immune signaling is associated with the pathogenesis of various diseases, including hematologic malignancies.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36281508"
                        open
                      >
                        <summary class="article-expander__title">Clinical, Mutational, and Transcriptomic Characteristics in Elderly Korean Individuals With Clonal Hematopoiesis Driver Mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36281508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221026211957&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Somatic CHIP driver mutations are common among the elderly in Korea and are detected in various genes, including DNMT3A and TET2. Our study highlights that chronic dysregulation of innate immune signaling is associated with the pathogenesis of various diseases, including hematologic malignancies.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36281508"
                        open
                      >
                        <summary class="article-expander__title">Clinical, Mutational, and Transcriptomic Characteristics in Elderly Korean Individuals With Clonal Hematopoiesis Driver Mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36281508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221025212118&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Somatic CHIP driver mutations are common among the elderly in Korea and are detected in various genes, including DNMT3A and TET2. Our study highlights that chronic dysregulation of innate immune signaling is associated with the pathogenesis of various diseases, including hematologic malignancies.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221128200439&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.8">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36281508"
                        open
                      >
                        <summary class="article-expander__title">Clinical, Mutational, and Transcriptomic Characteristics in Elderly Korean Individuals With Clonal Hematopoiesis Driver Mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36281508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221109201530&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Somatic CHIP driver mutations are common among the elderly in Korea and are detected in various genes, including DNMT3A and TET2. Our study highlights that chronic dysregulation of innate immune signaling is associated with the pathogenesis of various diseases, including hematologic malignancies.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36281508"
                        open
                      >
                        <summary class="article-expander__title">Clinical, Mutational, and Transcriptomic Characteristics in Elderly Korean Individuals With Clonal Hematopoiesis Driver Mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36281508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221108201514&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Somatic CHIP driver mutations are common among the elderly in Korea and are detected in various genes, including DNMT3A and TET2. Our study highlights that chronic dysregulation of innate immune signaling is associated with the pathogenesis of various diseases, including hematologic malignancies.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36281508"
                        open
                      >
                        <summary class="article-expander__title">Clinical, Mutational, and Transcriptomic Characteristics in Elderly Korean Individuals With Clonal Hematopoiesis Driver Mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36281508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221107201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Somatic CHIP driver mutations are common among the elderly in Korea and are detected in various genes, including DNMT3A and TET2. Our study highlights that chronic dysregulation of innate immune signaling is associated with the pathogenesis of various diseases, including hematologic malignancies.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36281508"
                        open
                      >
                        <summary class="article-expander__title">Clinical, Mutational, and Transcriptomic Characteristics in Elderly Korean Individuals With Clonal Hematopoiesis Driver Mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36281508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221106201343&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Somatic CHIP driver mutations are common among the elderly in Korea and are detected in various genes, including DNMT3A and TET2. Our study highlights that chronic dysregulation of innate immune signaling is associated with the pathogenesis of various diseases, including hematologic malignancies.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36281508"
                        open
                      >
                        <summary class="article-expander__title">Clinical, Mutational, and Transcriptomic Characteristics in Elderly Korean Individuals With Clonal Hematopoiesis Driver Mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36281508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221105211555&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Somatic CHIP driver mutations are common among the elderly in Korea and are detected in various genes, including DNMT3A and TET2. Our study highlights that chronic dysregulation of innate immune signaling is associated with the pathogenesis of various diseases, including hematologic malignancies.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36281508"
                        open
                      >
                        <summary class="article-expander__title">Clinical, Mutational, and Transcriptomic Characteristics in Elderly Korean Individuals With Clonal Hematopoiesis Driver Mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36281508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221104211729&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Somatic CHIP driver mutations are common among the elderly in Korea and are detected in various genes, including DNMT3A and TET2. Our study highlights that chronic dysregulation of innate immune signaling is associated with the pathogenesis of various diseases, including hematologic malignancies.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36281508"
                        open
                      >
                        <summary class="article-expander__title">Clinical, Mutational, and Transcriptomic Characteristics in Elderly Korean Individuals With Clonal Hematopoiesis Driver Mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36281508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221103212023&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Somatic CHIP driver mutations are common among the elderly in Korea and are detected in various genes, including DNMT3A and TET2. Our study highlights that chronic dysregulation of innate immune signaling is associated with the pathogenesis of various diseases, including hematologic malignancies.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36281508"
                        open
                      >
                        <summary class="article-expander__title">Clinical, Mutational, and Transcriptomic Characteristics in Elderly Korean Individuals With Clonal Hematopoiesis Driver Mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36281508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221102211743&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Somatic CHIP driver mutations are common among the elderly in Korea and are detected in various genes, including DNMT3A and TET2. Our study highlights that chronic dysregulation of innate immune signaling is associated with the pathogenesis of various diseases, including hematologic malignancies.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36281508"
                        open
                      >
                        <summary class="article-expander__title">Clinical, Mutational, and Transcriptomic Characteristics in Elderly Korean Individuals With Clonal Hematopoiesis Driver Mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36281508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221101212552&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Somatic CHIP driver mutations are common among the elderly in Korea and are detected in various genes, including DNMT3A and TET2. Our study highlights that chronic dysregulation of innate immune signaling is associated with the pathogenesis of various diseases, including hematologic malignancies.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36281508"
                        open
                      >
                        <summary class="article-expander__title">Clinical, Mutational, and Transcriptomic Characteristics in Elderly Korean Individuals With Clonal Hematopoiesis Driver Mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36281508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221031214247&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Somatic CHIP driver mutations are common among the elderly in Korea and are detected in various genes, including DNMT3A and TET2. Our study highlights that chronic dysregulation of innate immune signaling is associated with the pathogenesis of various diseases, including hematologic malignancies.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36281508"
                        open
                      >
                        <summary class="article-expander__title">Clinical, Mutational, and Transcriptomic Characteristics in Elderly Korean Individuals With Clonal Hematopoiesis Driver Mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36281508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221030212817&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Somatic CHIP driver mutations are common among the elderly in Korea and are detected in various genes, including DNMT3A and TET2. Our study highlights that chronic dysregulation of innate immune signaling is associated with the pathogenesis of various diseases, including hematologic malignancies.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36281508"
                        open
                      >
                        <summary class="article-expander__title">Clinical, Mutational, and Transcriptomic Characteristics in Elderly Korean Individuals With Clonal Hematopoiesis Driver Mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36281508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221029212729&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Somatic CHIP driver mutations are common among the elderly in Korea and are detected in various genes, including DNMT3A and TET2. Our study highlights that chronic dysregulation of innate immune signaling is associated with the pathogenesis of various diseases, including hematologic malignancies.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36281508"
                        open
                      >
                        <summary class="article-expander__title">Clinical, Mutational, and Transcriptomic Characteristics in Elderly Korean Individuals With Clonal Hematopoiesis Driver Mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36281508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221028211238&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Somatic CHIP driver mutations are common among the elderly in Korea and are detected in various genes, including DNMT3A and TET2. Our study highlights that chronic dysregulation of innate immune signaling is associated with the pathogenesis of various diseases, including hematologic malignancies.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36281508"
                        open
                      >
                        <summary class="article-expander__title">Clinical, Mutational, and Transcriptomic Characteristics in Elderly Korean Individuals With Clonal Hematopoiesis Driver Mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36281508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221027212535&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Somatic CHIP driver mutations are common among the elderly in Korea and are detected in various genes, including DNMT3A and TET2. Our study highlights that chronic dysregulation of innate immune signaling is associated with the pathogenesis of various diseases, including hematologic malignancies.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36281508"
                        open
                      >
                        <summary class="article-expander__title">Clinical, Mutational, and Transcriptomic Characteristics in Elderly Korean Individuals With Clonal Hematopoiesis Driver Mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36281508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221026211957&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Somatic CHIP driver mutations are common among the elderly in Korea and are detected in various genes, including DNMT3A and TET2. Our study highlights that chronic dysregulation of innate immune signaling is associated with the pathogenesis of various diseases, including hematologic malignancies.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-24 datetime="2022-10-24T10:00:00.000Z">2022-10-24</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-24 datetime="2022-10-24T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.8&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20221128200439&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36278584"
                        open
                      >
                        <summary class="article-expander__title">Luspatercept: A New Tool for the Treatment of Anemia Related to -Thalassemia, Myelodysplastic Syndromes and Primary Myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36278584/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221108201514&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Anemia is a common feature of both benign and malignant hematologic diseases. Beta-thalassemia (-thalassemia) syndromes are a group of hereditary disorders characterized by ineffective erythropoiesis, due to a genetic deficiency in the synthesis of the beta chains of hemoglobin, often accompanied by severe anemia and the need for red blood cell (RBC) transfusions. Myelodysplastic syndromes (MDS) are characterized by cytopenia(s) and ineffective hematopoiesis, despite a hypercellular bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36278584"
                        open
                      >
                        <summary class="article-expander__title">Luspatercept: A New Tool for the Treatment of Anemia Related to -Thalassemia, Myelodysplastic Syndromes and Primary Myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36278584/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221107201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Anemia is a common feature of both benign and malignant hematologic diseases. Beta-thalassemia (-thalassemia) syndromes are a group of hereditary disorders characterized by ineffective erythropoiesis, due to a genetic deficiency in the synthesis of the beta chains of hemoglobin, often accompanied by severe anemia and the need for red blood cell (RBC) transfusions. Myelodysplastic syndromes (MDS) are characterized by cytopenia(s) and ineffective hematopoiesis, despite a hypercellular bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36278517"
                        open
                      >
                        <summary class="article-expander__title">A Case Report on Hepatic Extramedullary Hematopoiesis as the Manifestation of Progression to Secondary Myelofibrosis in a Patient with Essential Thrombocytopenia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36278517/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221107201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN), which include primary myelofibrosis (PMF) and essential thrombocytopenia (ET), are characterized by the clonal proliferation of mature blood cells as a result of the overactivation of the JAK/STAT pathway. Extramedullary hematopoiesis (EMH), a common complication of PMF, occurs due to the dysregulation of the bone marrow microenvironment. We report an interesting case of a 73-year-old female with a working diagnosis of ET who was found to have EMH in the liver...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36278584"
                        open
                      >
                        <summary class="article-expander__title">Luspatercept: A New Tool for the Treatment of Anemia Related to -Thalassemia, Myelodysplastic Syndromes and Primary Myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36278584/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221106201343&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Anemia is a common feature of both benign and malignant hematologic diseases. Beta-thalassemia (-thalassemia) syndromes are a group of hereditary disorders characterized by ineffective erythropoiesis, due to a genetic deficiency in the synthesis of the beta chains of hemoglobin, often accompanied by severe anemia and the need for red blood cell (RBC) transfusions. Myelodysplastic syndromes (MDS) are characterized by cytopenia(s) and ineffective hematopoiesis, despite a hypercellular bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36278517"
                        open
                      >
                        <summary class="article-expander__title">A Case Report on Hepatic Extramedullary Hematopoiesis as the Manifestation of Progression to Secondary Myelofibrosis in a Patient with Essential Thrombocytopenia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36278517/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221106201343&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN), which include primary myelofibrosis (PMF) and essential thrombocytopenia (ET), are characterized by the clonal proliferation of mature blood cells as a result of the overactivation of the JAK/STAT pathway. Extramedullary hematopoiesis (EMH), a common complication of PMF, occurs due to the dysregulation of the bone marrow microenvironment. We report an interesting case of a 73-year-old female with a working diagnosis of ET who was found to have EMH in the liver...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36278584"
                        open
                      >
                        <summary class="article-expander__title">Luspatercept: A New Tool for the Treatment of Anemia Related to -Thalassemia, Myelodysplastic Syndromes and Primary Myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36278584/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221105211555&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Anemia is a common feature of both benign and malignant hematologic diseases. Beta-thalassemia (-thalassemia) syndromes are a group of hereditary disorders characterized by ineffective erythropoiesis, due to a genetic deficiency in the synthesis of the beta chains of hemoglobin, often accompanied by severe anemia and the need for red blood cell (RBC) transfusions. Myelodysplastic syndromes (MDS) are characterized by cytopenia(s) and ineffective hematopoiesis, despite a hypercellular bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36278517"
                        open
                      >
                        <summary class="article-expander__title">A Case Report on Hepatic Extramedullary Hematopoiesis as the Manifestation of Progression to Secondary Myelofibrosis in a Patient with Essential Thrombocytopenia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36278517/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221105211555&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN), which include primary myelofibrosis (PMF) and essential thrombocytopenia (ET), are characterized by the clonal proliferation of mature blood cells as a result of the overactivation of the JAK/STAT pathway. Extramedullary hematopoiesis (EMH), a common complication of PMF, occurs due to the dysregulation of the bone marrow microenvironment. We report an interesting case of a 73-year-old female with a working diagnosis of ET who was found to have EMH in the liver...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36278584"
                        open
                      >
                        <summary class="article-expander__title">Luspatercept: A New Tool for the Treatment of Anemia Related to -Thalassemia, Myelodysplastic Syndromes and Primary Myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36278584/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221104211730&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Anemia is a common feature of both benign and malignant hematologic diseases. Beta-thalassemia (-thalassemia) syndromes are a group of hereditary disorders characterized by ineffective erythropoiesis, due to a genetic deficiency in the synthesis of the beta chains of hemoglobin, often accompanied by severe anemia and the need for red blood cell (RBC) transfusions. Myelodysplastic syndromes (MDS) are characterized by cytopenia(s) and ineffective hematopoiesis, despite a hypercellular bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36278517"
                        open
                      >
                        <summary class="article-expander__title">A Case Report on Hepatic Extramedullary Hematopoiesis as the Manifestation of Progression to Secondary Myelofibrosis in a Patient with Essential Thrombocytopenia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36278517/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221104211730&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN), which include primary myelofibrosis (PMF) and essential thrombocytopenia (ET), are characterized by the clonal proliferation of mature blood cells as a result of the overactivation of the JAK/STAT pathway. Extramedullary hematopoiesis (EMH), a common complication of PMF, occurs due to the dysregulation of the bone marrow microenvironment. We report an interesting case of a 73-year-old female with a working diagnosis of ET who was found to have EMH in the liver...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36278584"
                        open
                      >
                        <summary class="article-expander__title">Luspatercept: A New Tool for the Treatment of Anemia Related to -Thalassemia, Myelodysplastic Syndromes and Primary Myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36278584/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221103212023&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Anemia is a common feature of both benign and malignant hematologic diseases. Beta-thalassemia (-thalassemia) syndromes are a group of hereditary disorders characterized by ineffective erythropoiesis, due to a genetic deficiency in the synthesis of the beta chains of hemoglobin, often accompanied by severe anemia and the need for red blood cell (RBC) transfusions. Myelodysplastic syndromes (MDS) are characterized by cytopenia(s) and ineffective hematopoiesis, despite a hypercellular bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36278517"
                        open
                      >
                        <summary class="article-expander__title">A Case Report on Hepatic Extramedullary Hematopoiesis as the Manifestation of Progression to Secondary Myelofibrosis in a Patient with Essential Thrombocytopenia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36278517/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221103212023&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN), which include primary myelofibrosis (PMF) and essential thrombocytopenia (ET), are characterized by the clonal proliferation of mature blood cells as a result of the overactivation of the JAK/STAT pathway. Extramedullary hematopoiesis (EMH), a common complication of PMF, occurs due to the dysregulation of the bone marrow microenvironment. We report an interesting case of a 73-year-old female with a working diagnosis of ET who was found to have EMH in the liver...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36278584"
                        open
                      >
                        <summary class="article-expander__title">Luspatercept: A New Tool for the Treatment of Anemia Related to -Thalassemia, Myelodysplastic Syndromes and Primary Myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36278584/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221102211743&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Anemia is a common feature of both benign and malignant hematologic diseases. Beta-thalassemia (-thalassemia) syndromes are a group of hereditary disorders characterized by ineffective erythropoiesis, due to a genetic deficiency in the synthesis of the beta chains of hemoglobin, often accompanied by severe anemia and the need for red blood cell (RBC) transfusions. Myelodysplastic syndromes (MDS) are characterized by cytopenia(s) and ineffective hematopoiesis, despite a hypercellular bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36278517"
                        open
                      >
                        <summary class="article-expander__title">A Case Report on Hepatic Extramedullary Hematopoiesis as the Manifestation of Progression to Secondary Myelofibrosis in a Patient with Essential Thrombocytopenia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36278517/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221102211743&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN), which include primary myelofibrosis (PMF) and essential thrombocytopenia (ET), are characterized by the clonal proliferation of mature blood cells as a result of the overactivation of the JAK/STAT pathway. Extramedullary hematopoiesis (EMH), a common complication of PMF, occurs due to the dysregulation of the bone marrow microenvironment. We report an interesting case of a 73-year-old female with a working diagnosis of ET who was found to have EMH in the liver...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36278584"
                        open
                      >
                        <summary class="article-expander__title">Luspatercept: A New Tool for the Treatment of Anemia Related to -Thalassemia, Myelodysplastic Syndromes and Primary Myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36278584/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221101212552&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Anemia is a common feature of both benign and malignant hematologic diseases. Beta-thalassemia (-thalassemia) syndromes are a group of hereditary disorders characterized by ineffective erythropoiesis, due to a genetic deficiency in the synthesis of the beta chains of hemoglobin, often accompanied by severe anemia and the need for red blood cell (RBC) transfusions. Myelodysplastic syndromes (MDS) are characterized by cytopenia(s) and ineffective hematopoiesis, despite a hypercellular bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36278517"
                        open
                      >
                        <summary class="article-expander__title">A Case Report on Hepatic Extramedullary Hematopoiesis as the Manifestation of Progression to Secondary Myelofibrosis in a Patient with Essential Thrombocytopenia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36278517/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221101212552&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN), which include primary myelofibrosis (PMF) and essential thrombocytopenia (ET), are characterized by the clonal proliferation of mature blood cells as a result of the overactivation of the JAK/STAT pathway. Extramedullary hematopoiesis (EMH), a common complication of PMF, occurs due to the dysregulation of the bone marrow microenvironment. We report an interesting case of a 73-year-old female with a working diagnosis of ET who was found to have EMH in the liver...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36278584"
                        open
                      >
                        <summary class="article-expander__title">Luspatercept: A New Tool for the Treatment of Anemia Related to -Thalassemia, Myelodysplastic Syndromes and Primary Myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36278584/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221031214247&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Anemia is a common feature of both benign and malignant hematologic diseases. Beta-thalassemia (-thalassemia) syndromes are a group of hereditary disorders characterized by ineffective erythropoiesis, due to a genetic deficiency in the synthesis of the beta chains of hemoglobin, often accompanied by severe anemia and the need for red blood cell (RBC) transfusions. Myelodysplastic syndromes (MDS) are characterized by cytopenia(s) and ineffective hematopoiesis, despite a hypercellular bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36278517"
                        open
                      >
                        <summary class="article-expander__title">A Case Report on Hepatic Extramedullary Hematopoiesis as the Manifestation of Progression to Secondary Myelofibrosis in a Patient with Essential Thrombocytopenia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36278517/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221031214247&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN), which include primary myelofibrosis (PMF) and essential thrombocytopenia (ET), are characterized by the clonal proliferation of mature blood cells as a result of the overactivation of the JAK/STAT pathway. Extramedullary hematopoiesis (EMH), a common complication of PMF, occurs due to the dysregulation of the bone marrow microenvironment. We report an interesting case of a 73-year-old female with a working diagnosis of ET who was found to have EMH in the liver...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36278584"
                        open
                      >
                        <summary class="article-expander__title">Luspatercept: A New Tool for the Treatment of Anemia Related to -Thalassemia, Myelodysplastic Syndromes and Primary Myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36278584/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221030212817&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Anemia is a common feature of both benign and malignant hematologic diseases. Beta-thalassemia (-thalassemia) syndromes are a group of hereditary disorders characterized by ineffective erythropoiesis, due to a genetic deficiency in the synthesis of the beta chains of hemoglobin, often accompanied by severe anemia and the need for red blood cell (RBC) transfusions. Myelodysplastic syndromes (MDS) are characterized by cytopenia(s) and ineffective hematopoiesis, despite a hypercellular bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36278517"
                        open
                      >
                        <summary class="article-expander__title">A Case Report on Hepatic Extramedullary Hematopoiesis as the Manifestation of Progression to Secondary Myelofibrosis in a Patient with Essential Thrombocytopenia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36278517/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221030212817&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN), which include primary myelofibrosis (PMF) and essential thrombocytopenia (ET), are characterized by the clonal proliferation of mature blood cells as a result of the overactivation of the JAK/STAT pathway. Extramedullary hematopoiesis (EMH), a common complication of PMF, occurs due to the dysregulation of the bone marrow microenvironment. We report an interesting case of a 73-year-old female with a working diagnosis of ET who was found to have EMH in the liver...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36278584"
                        open
                      >
                        <summary class="article-expander__title">Luspatercept: A New Tool for the Treatment of Anemia Related to -Thalassemia, Myelodysplastic Syndromes and Primary Myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36278584/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221029212729&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Anemia is a common feature of both benign and malignant hematologic diseases. Beta-thalassemia (-thalassemia) syndromes are a group of hereditary disorders characterized by ineffective erythropoiesis, due to a genetic deficiency in the synthesis of the beta chains of hemoglobin, often accompanied by severe anemia and the need for red blood cell (RBC) transfusions. Myelodysplastic syndromes (MDS) are characterized by cytopenia(s) and ineffective hematopoiesis, despite a hypercellular bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36278517"
                        open
                      >
                        <summary class="article-expander__title">A Case Report on Hepatic Extramedullary Hematopoiesis as the Manifestation of Progression to Secondary Myelofibrosis in a Patient with Essential Thrombocytopenia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36278517/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221029212729&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN), which include primary myelofibrosis (PMF) and essential thrombocytopenia (ET), are characterized by the clonal proliferation of mature blood cells as a result of the overactivation of the JAK/STAT pathway. Extramedullary hematopoiesis (EMH), a common complication of PMF, occurs due to the dysregulation of the bone marrow microenvironment. We report an interesting case of a 73-year-old female with a working diagnosis of ET who was found to have EMH in the liver...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36278584"
                        open
                      >
                        <summary class="article-expander__title">Luspatercept: A New Tool for the Treatment of Anemia Related to -Thalassemia, Myelodysplastic Syndromes and Primary Myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36278584/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221028211238&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Anemia is a common feature of both benign and malignant hematologic diseases. Beta-thalassemia (-thalassemia) syndromes are a group of hereditary disorders characterized by ineffective erythropoiesis, due to a genetic deficiency in the synthesis of the beta chains of hemoglobin, often accompanied by severe anemia and the need for red blood cell (RBC) transfusions. Myelodysplastic syndromes (MDS) are characterized by cytopenia(s) and ineffective hematopoiesis, despite a hypercellular bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36278517"
                        open
                      >
                        <summary class="article-expander__title">A Case Report on Hepatic Extramedullary Hematopoiesis as the Manifestation of Progression to Secondary Myelofibrosis in a Patient with Essential Thrombocytopenia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36278517/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221028211238&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN), which include primary myelofibrosis (PMF) and essential thrombocytopenia (ET), are characterized by the clonal proliferation of mature blood cells as a result of the overactivation of the JAK/STAT pathway. Extramedullary hematopoiesis (EMH), a common complication of PMF, occurs due to the dysregulation of the bone marrow microenvironment. We report an interesting case of a 73-year-old female with a working diagnosis of ET who was found to have EMH in the liver...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36278584"
                        open
                      >
                        <summary class="article-expander__title">Luspatercept: A New Tool for the Treatment of Anemia Related to -Thalassemia, Myelodysplastic Syndromes and Primary Myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36278584/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221027212537&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Anemia is a common feature of both benign and malignant hematologic diseases. Beta-thalassemia (-thalassemia) syndromes are a group of hereditary disorders characterized by ineffective erythropoiesis, due to a genetic deficiency in the synthesis of the beta chains of hemoglobin, often accompanied by severe anemia and the need for red blood cell (RBC) transfusions. Myelodysplastic syndromes (MDS) are characterized by cytopenia(s) and ineffective hematopoiesis, despite a hypercellular bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36278517"
                        open
                      >
                        <summary class="article-expander__title">A Case Report on Hepatic Extramedullary Hematopoiesis as the Manifestation of Progression to Secondary Myelofibrosis in a Patient with Essential Thrombocytopenia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36278517/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221027212537&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN), which include primary myelofibrosis (PMF) and essential thrombocytopenia (ET), are characterized by the clonal proliferation of mature blood cells as a result of the overactivation of the JAK/STAT pathway. Extramedullary hematopoiesis (EMH), a common complication of PMF, occurs due to the dysregulation of the bone marrow microenvironment. We report an interesting case of a 73-year-old female with a working diagnosis of ET who was found to have EMH in the liver...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36278584"
                        open
                      >
                        <summary class="article-expander__title">Luspatercept: A New Tool for the Treatment of Anemia Related to -Thalassemia, Myelodysplastic Syndromes and Primary Myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36278584/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221026211957&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Anemia is a common feature of both benign and malignant hematologic diseases. Beta-thalassemia (-thalassemia) syndromes are a group of hereditary disorders characterized by ineffective erythropoiesis, due to a genetic deficiency in the synthesis of the beta chains of hemoglobin, often accompanied by severe anemia and the need for red blood cell (RBC) transfusions. Myelodysplastic syndromes (MDS) are characterized by cytopenia(s) and ineffective hematopoiesis, despite a hypercellular bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36278517"
                        open
                      >
                        <summary class="article-expander__title">A Case Report on Hepatic Extramedullary Hematopoiesis as the Manifestation of Progression to Secondary Myelofibrosis in a Patient with Essential Thrombocytopenia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36278517/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221026211957&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN), which include primary myelofibrosis (PMF) and essential thrombocytopenia (ET), are characterized by the clonal proliferation of mature blood cells as a result of the overactivation of the JAK/STAT pathway. Extramedullary hematopoiesis (EMH), a common complication of PMF, occurs due to the dysregulation of the bone marrow microenvironment. We report an interesting case of a 73-year-old female with a working diagnosis of ET who was found to have EMH in the liver...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36278584"
                        open
                      >
                        <summary class="article-expander__title">Luspatercept: A New Tool for the Treatment of Anemia Related to -Thalassemia, Myelodysplastic Syndromes and Primary Myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36278584/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221025212118&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Anemia is a common feature of both benign and malignant hematologic diseases. Beta-thalassemia (-thalassemia) syndromes are a group of hereditary disorders characterized by ineffective erythropoiesis, due to a genetic deficiency in the synthesis of the beta chains of hemoglobin, often accompanied by severe anemia and the need for red blood cell (RBC) transfusions. Myelodysplastic syndromes (MDS) are characterized by cytopenia(s) and ineffective hematopoiesis, despite a hypercellular bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36278517"
                        open
                      >
                        <summary class="article-expander__title">A Case Report on Hepatic Extramedullary Hematopoiesis as the Manifestation of Progression to Secondary Myelofibrosis in a Patient with Essential Thrombocytopenia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36278517/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221025212118&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN), which include primary myelofibrosis (PMF) and essential thrombocytopenia (ET), are characterized by the clonal proliferation of mature blood cells as a result of the overactivation of the JAK/STAT pathway. Extramedullary hematopoiesis (EMH), a common complication of PMF, occurs due to the dysregulation of the bone marrow microenvironment. We report an interesting case of a 73-year-old female with a working diagnosis of ET who was found to have EMH in the liver...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36278584"
                        open
                      >
                        <summary class="article-expander__title">Luspatercept: A New Tool for the Treatment of Anemia Related to -Thalassemia, Myelodysplastic Syndromes and Primary Myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36278584/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221024214453&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Anemia is a common feature of both benign and malignant hematologic diseases. Beta-thalassemia (-thalassemia) syndromes are a group of hereditary disorders characterized by ineffective erythropoiesis, due to a genetic deficiency in the synthesis of the beta chains of hemoglobin, often accompanied by severe anemia and the need for red blood cell (RBC) transfusions. Myelodysplastic syndromes (MDS) are characterized by cytopenia(s) and ineffective hematopoiesis, despite a hypercellular bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36278517"
                        open
                      >
                        <summary class="article-expander__title">A Case Report on Hepatic Extramedullary Hematopoiesis as the Manifestation of Progression to Secondary Myelofibrosis in a Patient with Essential Thrombocytopenia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36278517/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221024214453&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN), which include primary myelofibrosis (PMF) and essential thrombocytopenia (ET), are characterized by the clonal proliferation of mature blood cells as a result of the overactivation of the JAK/STAT pathway. Extramedullary hematopoiesis (EMH), a common complication of PMF, occurs due to the dysregulation of the bone marrow microenvironment. We report an interesting case of a 73-year-old female with a working diagnosis of ET who was found to have EMH in the liver...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-21 datetime="2022-10-21T10:00:00.000Z">2022-10-21</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-21 datetime="2022-10-21T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.8&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20221128200439&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36271153"
                        open
                      >
                        <summary class="article-expander__title">Srsf2P95H/+ co-operates with loss of TET2 to promote myeloid bias and initiate a chronic myelomonocytic leukemia-like disease in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36271153/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221107201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recurrent mutations in RNA splicing proteins and epigenetic regulators contribute to the development of myelodysplastic syndrome (MDS) and related myeloid neoplasms. In chronic myelomonocytic leukemia (CMML), SRSF2 mutations occur in ~50% of patients and TET2 mutations in ~60%. Clonal analysis indicates that either mutation can arise as the founder lesion. Based on human cancer genetics we crossed an inducible Srsf2^(P95H/+) mutant model with Tet2^(fl/fl) mice to mutate both concomitantly in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36271153"
                        open
                      >
                        <summary class="article-expander__title">Srsf2P95H/+ co-operates with loss of TET2 to promote myeloid bias and initiate a chronic myelomonocytic leukemia-like disease in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36271153/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221106201343&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recurrent mutations in RNA splicing proteins and epigenetic regulators contribute to the development of myelodysplastic syndrome (MDS) and related myeloid neoplasms. In chronic myelomonocytic leukemia (CMML), SRSF2 mutations occur in ~50% of patients and TET2 mutations in ~60%. Clonal analysis indicates that either mutation can arise as the founder lesion. Based on human cancer genetics we crossed an inducible Srsf2^(P95H/+) mutant model with Tet2^(fl/fl) mice to mutate both concomitantly in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36271153"
                        open
                      >
                        <summary class="article-expander__title">Srsf2P95H/+ co-operates with loss of TET2 to promote myeloid bias and initiate a chronic myelomonocytic leukemia-like disease in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36271153/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221105211555&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recurrent mutations in RNA splicing proteins and epigenetic regulators contribute to the development of myelodysplastic syndrome (MDS) and related myeloid neoplasms. In chronic myelomonocytic leukemia (CMML), SRSF2 mutations occur in ~50% of patients and TET2 mutations in ~60%. Clonal analysis indicates that either mutation can arise as the founder lesion. Based on human cancer genetics we crossed an inducible Srsf2^(P95H/+) mutant model with Tet2^(fl/fl) mice to mutate both concomitantly in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36271153"
                        open
                      >
                        <summary class="article-expander__title">Srsf2P95H/+ co-operates with loss of TET2 to promote myeloid bias and initiate a chronic myelomonocytic leukemia-like disease in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36271153/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221104211730&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recurrent mutations in RNA splicing proteins and epigenetic regulators contribute to the development of myelodysplastic syndrome (MDS) and related myeloid neoplasms. In chronic myelomonocytic leukemia (CMML), SRSF2 mutations occur in ~50% of patients and TET2 mutations in ~60%. Clonal analysis indicates that either mutation can arise as the founder lesion. Based on human cancer genetics we crossed an inducible Srsf2^(P95H/+) mutant model with Tet2^(fl/fl) mice to mutate both concomitantly in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36271153"
                        open
                      >
                        <summary class="article-expander__title">Srsf2P95H/+ co-operates with loss of TET2 to promote myeloid bias and initiate a chronic myelomonocytic leukemia-like disease in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36271153/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221103212023&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recurrent mutations in RNA splicing proteins and epigenetic regulators contribute to the development of myelodysplastic syndrome (MDS) and related myeloid neoplasms. In chronic myelomonocytic leukemia (CMML), SRSF2 mutations occur in ~50% of patients and TET2 mutations in ~60%. Clonal analysis indicates that either mutation can arise as the founder lesion. Based on human cancer genetics we crossed an inducible Srsf2^(P95H/+) mutant model with Tet2^(fl/fl) mice to mutate both concomitantly in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36271153"
                        open
                      >
                        <summary class="article-expander__title">Srsf2P95H/+ co-operates with loss of TET2 to promote myeloid bias and initiate a chronic myelomonocytic leukemia-like disease in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36271153/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221102211743&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recurrent mutations in RNA splicing proteins and epigenetic regulators contribute to the development of myelodysplastic syndrome (MDS) and related myeloid neoplasms. In chronic myelomonocytic leukemia (CMML), SRSF2 mutations occur in ~50% of patients and TET2 mutations in ~60%. Clonal analysis indicates that either mutation can arise as the founder lesion. Based on human cancer genetics we crossed an inducible Srsf2^(P95H/+) mutant model with Tet2^(fl/fl) mice to mutate both concomitantly in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36271153"
                        open
                      >
                        <summary class="article-expander__title">Srsf2P95H/+ co-operates with loss of TET2 to promote myeloid bias and initiate a chronic myelomonocytic leukemia-like disease in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36271153/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221101212552&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recurrent mutations in RNA splicing proteins and epigenetic regulators contribute to the development of myelodysplastic syndrome (MDS) and related myeloid neoplasms. In chronic myelomonocytic leukemia (CMML), SRSF2 mutations occur in ~50% of patients and TET2 mutations in ~60%. Clonal analysis indicates that either mutation can arise as the founder lesion. Based on human cancer genetics we crossed an inducible Srsf2^(P95H/+) mutant model with Tet2^(fl/fl) mice to mutate both concomitantly in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36271153"
                        open
                      >
                        <summary class="article-expander__title">Srsf2P95H/+ co-operates with loss of TET2 to promote myeloid bias and initiate a chronic myelomonocytic leukemia-like disease in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36271153/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221031214247&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recurrent mutations in RNA splicing proteins and epigenetic regulators contribute to the development of myelodysplastic syndrome (MDS) and related myeloid neoplasms. In chronic myelomonocytic leukemia (CMML), SRSF2 mutations occur in ~50% of patients and TET2 mutations in ~60%. Clonal analysis indicates that either mutation can arise as the founder lesion. Based on human cancer genetics we crossed an inducible Srsf2^(P95H/+) mutant model with Tet2^(fl/fl) mice to mutate both concomitantly in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36271153"
                        open
                      >
                        <summary class="article-expander__title">Srsf2P95H/+ co-operates with loss of TET2 to promote myeloid bias and initiate a chronic myelomonocytic leukemia-like disease in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36271153/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221030212817&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recurrent mutations in RNA splicing proteins and epigenetic regulators contribute to the development of myelodysplastic syndrome (MDS) and related myeloid neoplasms. In chronic myelomonocytic leukemia (CMML), SRSF2 mutations occur in ~50% of patients and TET2 mutations in ~60%. Clonal analysis indicates that either mutation can arise as the founder lesion. Based on human cancer genetics we crossed an inducible Srsf2^(P95H/+) mutant model with Tet2^(fl/fl) mice to mutate both concomitantly in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36271153"
                        open
                      >
                        <summary class="article-expander__title">Srsf2P95H/+ co-operates with loss of TET2 to promote myeloid bias and initiate a chronic myelomonocytic leukemia-like disease in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36271153/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221029212729&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recurrent mutations in RNA splicing proteins and epigenetic regulators contribute to the development of myelodysplastic syndrome (MDS) and related myeloid neoplasms. In chronic myelomonocytic leukemia (CMML), SRSF2 mutations occur in ~50% of patients and TET2 mutations in ~60%. Clonal analysis indicates that either mutation can arise as the founder lesion. Based on human cancer genetics we crossed an inducible Srsf2^(P95H/+) mutant model with Tet2^(fl/fl) mice to mutate both concomitantly in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36271153"
                        open
                      >
                        <summary class="article-expander__title">Srsf2P95H/+ co-operates with loss of TET2 to promote myeloid bias and initiate a chronic myelomonocytic leukemia-like disease in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36271153/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221028211238&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recurrent mutations in RNA splicing proteins and epigenetic regulators contribute to the development of myelodysplastic syndrome (MDS) and related myeloid neoplasms. In chronic myelomonocytic leukemia (CMML), SRSF2 mutations occur in ~50% of patients and TET2 mutations in ~60%. Clonal analysis indicates that either mutation can arise as the founder lesion. Based on human cancer genetics we crossed an inducible Srsf2^(P95H/+) mutant model with Tet2^(fl/fl) mice to mutate both concomitantly in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36271153"
                        open
                      >
                        <summary class="article-expander__title">Srsf2P95H/+ co-operates with loss of TET2 to promote myeloid bias and initiate a chronic myelomonocytic leukemia-like disease in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36271153/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221027212537&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recurrent mutations in RNA splicing proteins and epigenetic regulators contribute to the development of myelodysplastic syndrome (MDS) and related myeloid neoplasms. In chronic myelomonocytic leukemia (CMML), SRSF2 mutations occur in ~50% of patients and TET2 mutations in ~60%. Clonal analysis indicates that either mutation can arise as the founder lesion. Based on human cancer genetics we crossed an inducible Srsf2^(P95H/+) mutant model with Tet2^(fl/fl) mice to mutate both concomitantly in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36271153"
                        open
                      >
                        <summary class="article-expander__title">Srsf2P95H/+ co-operates with loss of TET2 to promote myeloid bias and initiate a chronic myelomonocytic leukemia-like disease in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36271153/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221026211957&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recurrent mutations in RNA splicing proteins and epigenetic regulators contribute to the development of myelodysplastic syndrome (MDS) and related myeloid neoplasms. In chronic myelomonocytic leukemia (CMML), SRSF2 mutations occur in ~50% of patients and TET2 mutations in ~60%. Clonal analysis indicates that either mutation can arise as the founder lesion. Based on human cancer genetics we crossed an inducible Srsf2^(P95H/+) mutant model with Tet2^(fl/fl) mice to mutate both concomitantly in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36271153"
                        open
                      >
                        <summary class="article-expander__title">Srsf2P95H/+ co-operates with loss of TET2 to promote myeloid bias and initiate a chronic myelomonocytic leukemia-like disease in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36271153/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221025212118&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recurrent mutations in RNA splicing proteins and epigenetic regulators contribute to the development of myelodysplastic syndrome (MDS) and related myeloid neoplasms. In chronic myelomonocytic leukemia (CMML), SRSF2 mutations occur in ~50% of patients and TET2 mutations in ~60%. Clonal analysis indicates that either mutation can arise as the founder lesion. Based on human cancer genetics we crossed an inducible Srsf2^(P95H/+) mutant model with Tet2^(fl/fl) mice to mutate both concomitantly in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36271153"
                        open
                      >
                        <summary class="article-expander__title">Srsf2P95H/+ co-operates with loss of TET2 to promote myeloid bias and initiate a chronic myelomonocytic leukemia-like disease in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36271153/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221024214453&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recurrent mutations in RNA splicing proteins and epigenetic regulators contribute to the development of myelodysplastic syndrome (MDS) and related myeloid neoplasms. In chronic myelomonocytic leukemia (CMML), SRSF2 mutations occur in ~50% of patients and TET2 mutations in ~60%. Clonal analysis indicates that either mutation can arise as the founder lesion. Based on human cancer genetics we crossed an inducible Srsf2^(P95H/+) mutant model with Tet2^(fl/fl) mice to mutate both concomitantly in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36271153"
                        open
                      >
                        <summary class="article-expander__title">Srsf2P95H/+ co-operates with loss of TET2 to promote myeloid bias and initiate a chronic myelomonocytic leukemia-like disease in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36271153/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221023215029&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recurrent mutations in RNA splicing proteins and epigenetic regulators contribute to the development of myelodysplastic syndrome (MDS) and related myeloid neoplasms. In chronic myelomonocytic leukemia (CMML), SRSF2 mutations occur in ~50% of patients and TET2 mutations in ~60%. Clonal analysis indicates that either mutation can arise as the founder lesion. Based on human cancer genetics we crossed an inducible Srsf2^(P95H/+) mutant model with Tet2^(fl/fl) mice to mutate both concomitantly in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36271153"
                        open
                      >
                        <summary class="article-expander__title">Srsf2P95H/+ co-operates with loss of TET2 to promote myeloid bias and initiate a chronic myelomonocytic leukemia-like disease in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36271153/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221022213551&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recurrent mutations in RNA splicing proteins and epigenetic regulators contribute to the development of myelodysplastic syndrome (MDS) and related myeloid neoplasms. In chronic myelomonocytic leukemia (CMML), SRSF2 mutations occur in ~50% of patients and TET2 mutations in ~60%. Clonal analysis indicates that either mutation can arise as the founder lesion. Based on human cancer genetics we crossed an inducible Srsf2^(P95H/+) mutant model with Tet2^(fl/fl) mice to mutate both concomitantly in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221128200439&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.8">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36270293"
                        open
                      >
                        <summary class="article-expander__title">New Insights Into the Pathogenesis of Mastocytosis: Emerging Concepts in Diagnosis and Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36270293/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221108201514&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mastocytosis is a heterogeneous group of neoplasms defined by a numerical increase and accumulation of clonal mast cells (MCs) in various organ systems. The disease may present as cutaneous mastocytosis or systemic mastocytosis (SM). On the basis of histopathological and molecular features, clinical variables, and organ involvement, SM is divided into indolent SM, smoldering SM, SM with an associated hematologic neoplasm, aggressive SM, and MC leukemia. Each variant is defined by unique...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36270293"
                        open
                      >
                        <summary class="article-expander__title">New Insights Into the Pathogenesis of Mastocytosis: Emerging Concepts in Diagnosis and Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36270293/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221107201407&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mastocytosis is a heterogeneous group of neoplasms defined by a numerical increase and accumulation of clonal mast cells (MCs) in various organ systems. The disease may present as cutaneous mastocytosis or systemic mastocytosis (SM). On the basis of histopathological and molecular features, clinical variables, and organ involvement, SM is divided into indolent SM, smoldering SM, SM with an associated hematologic neoplasm, aggressive SM, and MC leukemia. Each variant is defined by unique...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36270293"
                        open
                      >
                        <summary class="article-expander__title">New Insights Into the Pathogenesis of Mastocytosis: Emerging Concepts in Diagnosis and Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36270293/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221106201343&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mastocytosis is a heterogeneous group of neoplasms defined by a numerical increase and accumulation of clonal mast cells (MCs) in various organ systems. The disease may present as cutaneous mastocytosis or systemic mastocytosis (SM). On the basis of histopathological and molecular features, clinical variables, and organ involvement, SM is divided into indolent SM, smoldering SM, SM with an associated hematologic neoplasm, aggressive SM, and MC leukemia. Each variant is defined by unique...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36270293"
                        open
                      >
                        <summary class="article-expander__title">New Insights Into the Pathogenesis of Mastocytosis: Emerging Concepts in Diagnosis and Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36270293/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221105211555&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mastocytosis is a heterogeneous group of neoplasms defined by a numerical increase and accumulation of clonal mast cells (MCs) in various organ systems. The disease may present as cutaneous mastocytosis or systemic mastocytosis (SM). On the basis of histopathological and molecular features, clinical variables, and organ involvement, SM is divided into indolent SM, smoldering SM, SM with an associated hematologic neoplasm, aggressive SM, and MC leukemia. Each variant is defined by unique...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36270293"
                        open
                      >
                        <summary class="article-expander__title">New Insights Into the Pathogenesis of Mastocytosis: Emerging Concepts in Diagnosis and Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36270293/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221104211729&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mastocytosis is a heterogeneous group of neoplasms defined by a numerical increase and accumulation of clonal mast cells (MCs) in various organ systems. The disease may present as cutaneous mastocytosis or systemic mastocytosis (SM). On the basis of histopathological and molecular features, clinical variables, and organ involvement, SM is divided into indolent SM, smoldering SM, SM with an associated hematologic neoplasm, aggressive SM, and MC leukemia. Each variant is defined by unique...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36270293"
                        open
                      >
                        <summary class="article-expander__title">New Insights Into the Pathogenesis of Mastocytosis: Emerging Concepts in Diagnosis and Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36270293/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221103212023&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mastocytosis is a heterogeneous group of neoplasms defined by a numerical increase and accumulation of clonal mast cells (MCs) in various organ systems. The disease may present as cutaneous mastocytosis or systemic mastocytosis (SM). On the basis of histopathological and molecular features, clinical variables, and organ involvement, SM is divided into indolent SM, smoldering SM, SM with an associated hematologic neoplasm, aggressive SM, and MC leukemia. Each variant is defined by unique...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36270293"
                        open
                      >
                        <summary class="article-expander__title">New Insights Into the Pathogenesis of Mastocytosis: Emerging Concepts in Diagnosis and Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36270293/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221102211743&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mastocytosis is a heterogeneous group of neoplasms defined by a numerical increase and accumulation of clonal mast cells (MCs) in various organ systems. The disease may present as cutaneous mastocytosis or systemic mastocytosis (SM). On the basis of histopathological and molecular features, clinical variables, and organ involvement, SM is divided into indolent SM, smoldering SM, SM with an associated hematologic neoplasm, aggressive SM, and MC leukemia. Each variant is defined by unique...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36270293"
                        open
                      >
                        <summary class="article-expander__title">New Insights Into the Pathogenesis of Mastocytosis: Emerging Concepts in Diagnosis and Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36270293/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221101212552&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mastocytosis is a heterogeneous group of neoplasms defined by a numerical increase and accumulation of clonal mast cells (MCs) in various organ systems. The disease may present as cutaneous mastocytosis or systemic mastocytosis (SM). On the basis of histopathological and molecular features, clinical variables, and organ involvement, SM is divided into indolent SM, smoldering SM, SM with an associated hematologic neoplasm, aggressive SM, and MC leukemia. Each variant is defined by unique...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36270293"
                        open
                      >
                        <summary class="article-expander__title">New Insights Into the Pathogenesis of Mastocytosis: Emerging Concepts in Diagnosis and Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36270293/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221031214247&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mastocytosis is a heterogeneous group of neoplasms defined by a numerical increase and accumulation of clonal mast cells (MCs) in various organ systems. The disease may present as cutaneous mastocytosis or systemic mastocytosis (SM). On the basis of histopathological and molecular features, clinical variables, and organ involvement, SM is divided into indolent SM, smoldering SM, SM with an associated hematologic neoplasm, aggressive SM, and MC leukemia. Each variant is defined by unique...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36270293"
                        open
                      >
                        <summary class="article-expander__title">New Insights Into the Pathogenesis of Mastocytosis: Emerging Concepts in Diagnosis and Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36270293/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221030212817&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mastocytosis is a heterogeneous group of neoplasms defined by a numerical increase and accumulation of clonal mast cells (MCs) in various organ systems. The disease may present as cutaneous mastocytosis or systemic mastocytosis (SM). On the basis of histopathological and molecular features, clinical variables, and organ involvement, SM is divided into indolent SM, smoldering SM, SM with an associated hematologic neoplasm, aggressive SM, and MC leukemia. Each variant is defined by unique...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36270293"
                        open
                      >
                        <summary class="article-expander__title">New Insights Into the Pathogenesis of Mastocytosis: Emerging Concepts in Diagnosis and Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36270293/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221029212729&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mastocytosis is a heterogeneous group of neoplasms defined by a numerical increase and accumulation of clonal mast cells (MCs) in various organ systems. The disease may present as cutaneous mastocytosis or systemic mastocytosis (SM). On the basis of histopathological and molecular features, clinical variables, and organ involvement, SM is divided into indolent SM, smoldering SM, SM with an associated hematologic neoplasm, aggressive SM, and MC leukemia. Each variant is defined by unique...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36270293"
                        open
                      >
                        <summary class="article-expander__title">New Insights Into the Pathogenesis of Mastocytosis: Emerging Concepts in Diagnosis and Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36270293/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221028211238&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mastocytosis is a heterogeneous group of neoplasms defined by a numerical increase and accumulation of clonal mast cells (MCs) in various organ systems. The disease may present as cutaneous mastocytosis or systemic mastocytosis (SM). On the basis of histopathological and molecular features, clinical variables, and organ involvement, SM is divided into indolent SM, smoldering SM, SM with an associated hematologic neoplasm, aggressive SM, and MC leukemia. Each variant is defined by unique...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36270293"
                        open
                      >
                        <summary class="article-expander__title">New Insights Into the Pathogenesis of Mastocytosis: Emerging Concepts in Diagnosis and Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36270293/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221027212535&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mastocytosis is a heterogeneous group of neoplasms defined by a numerical increase and accumulation of clonal mast cells (MCs) in various organ systems. The disease may present as cutaneous mastocytosis or systemic mastocytosis (SM). On the basis of histopathological and molecular features, clinical variables, and organ involvement, SM is divided into indolent SM, smoldering SM, SM with an associated hematologic neoplasm, aggressive SM, and MC leukemia. Each variant is defined by unique...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36270293"
                        open
                      >
                        <summary class="article-expander__title">New Insights Into the Pathogenesis of Mastocytosis: Emerging Concepts in Diagnosis and Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36270293/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221026211957&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mastocytosis is a heterogeneous group of neoplasms defined by a numerical increase and accumulation of clonal mast cells (MCs) in various organ systems. The disease may present as cutaneous mastocytosis or systemic mastocytosis (SM). On the basis of histopathological and molecular features, clinical variables, and organ involvement, SM is divided into indolent SM, smoldering SM, SM with an associated hematologic neoplasm, aggressive SM, and MC leukemia. Each variant is defined by unique...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36270293"
                        open
                      >
                        <summary class="article-expander__title">New Insights Into the Pathogenesis of Mastocytosis: Emerging Concepts in Diagnosis and Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36270293/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221025212118&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mastocytosis is a heterogeneous group of neoplasms defined by a numerical increase and accumulation of clonal mast cells (MCs) in various organ systems. The disease may present as cutaneous mastocytosis or systemic mastocytosis (SM). On the basis of histopathological and molecular features, clinical variables, and organ involvement, SM is divided into indolent SM, smoldering SM, SM with an associated hematologic neoplasm, aggressive SM, and MC leukemia. Each variant is defined by unique...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36270293"
                        open
                      >
                        <summary class="article-expander__title">New Insights Into the Pathogenesis of Mastocytosis: Emerging Concepts in Diagnosis and Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36270293/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221024214453&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mastocytosis is a heterogeneous group of neoplasms defined by a numerical increase and accumulation of clonal mast cells (MCs) in various organ systems. The disease may present as cutaneous mastocytosis or systemic mastocytosis (SM). On the basis of histopathological and molecular features, clinical variables, and organ involvement, SM is divided into indolent SM, smoldering SM, SM with an associated hematologic neoplasm, aggressive SM, and MC leukemia. Each variant is defined by unique...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36270293"
                        open
                      >
                        <summary class="article-expander__title">New Insights Into the Pathogenesis of Mastocytosis: Emerging Concepts in Diagnosis and Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36270293/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221023215029&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mastocytosis is a heterogeneous group of neoplasms defined by a numerical increase and accumulation of clonal mast cells (MCs) in various organ systems. The disease may present as cutaneous mastocytosis or systemic mastocytosis (SM). On the basis of histopathological and molecular features, clinical variables, and organ involvement, SM is divided into indolent SM, smoldering SM, SM with an associated hematologic neoplasm, aggressive SM, and MC leukemia. Each variant is defined by unique...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36270293"
                        open
                      >
                        <summary class="article-expander__title">New Insights Into the Pathogenesis of Mastocytosis: Emerging Concepts in Diagnosis and Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36270293/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221022213551&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mastocytosis is a heterogeneous group of neoplasms defined by a numerical increase and accumulation of clonal mast cells (MCs) in various organ systems. The disease may present as cutaneous mastocytosis or systemic mastocytosis (SM). On the basis of histopathological and molecular features, clinical variables, and organ involvement, SM is divided into indolent SM, smoldering SM, SM with an associated hematologic neoplasm, aggressive SM, and MC leukemia. Each variant is defined by unique...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-20 datetime="2022-10-20T10:00:00.000Z">2022-10-20</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-20 datetime="2022-10-20T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.8&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20221128200439&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36264554"
                        open
                      >
                        <summary class="article-expander__title">Practical Considerations for the Use of Circulating Tumor DNA in the Treatment of Patients With Cancer: A Narrative Review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36264554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221106201343&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS AND RELEVANCE: This review found that liquid biopsy is increasingly being used clinically in advanced lung cancer, and ongoing research is identifying applications of circulating tumor DNA-based testing that complement tissue analysis across a broad range of clinical settings. Circulating tumor DNA technologies are advancing quickly and are demonstrating potential benefits for patients, health care practitioners, health care systems, and researchers, at many stages of the patient...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36264554"
                        open
                      >
                        <summary class="article-expander__title">Practical Considerations for the Use of Circulating Tumor DNA in the Treatment of Patients With Cancer: A Narrative Review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36264554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221105211555&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS AND RELEVANCE: This review found that liquid biopsy is increasingly being used clinically in advanced lung cancer, and ongoing research is identifying applications of circulating tumor DNA-based testing that complement tissue analysis across a broad range of clinical settings. Circulating tumor DNA technologies are advancing quickly and are demonstrating potential benefits for patients, health care practitioners, health care systems, and researchers, at many stages of the patient...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36264554"
                        open
                      >
                        <summary class="article-expander__title">Practical Considerations for the Use of Circulating Tumor DNA in the Treatment of Patients With Cancer: A Narrative Review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36264554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221104211730&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS AND RELEVANCE: This review found that liquid biopsy is increasingly being used clinically in advanced lung cancer, and ongoing research is identifying applications of circulating tumor DNA-based testing that complement tissue analysis across a broad range of clinical settings. Circulating tumor DNA technologies are advancing quickly and are demonstrating potential benefits for patients, health care practitioners, health care systems, and researchers, at many stages of the patient...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36264554"
                        open
                      >
                        <summary class="article-expander__title">Practical Considerations for the Use of Circulating Tumor DNA in the Treatment of Patients With Cancer: A Narrative Review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36264554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221103212023&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS AND RELEVANCE: This review found that liquid biopsy is increasingly being used clinically in advanced lung cancer, and ongoing research is identifying applications of circulating tumor DNA-based testing that complement tissue analysis across a broad range of clinical settings. Circulating tumor DNA technologies are advancing quickly and are demonstrating potential benefits for patients, health care practitioners, health care systems, and researchers, at many stages of the patient...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36264554"
                        open
                      >
                        <summary class="article-expander__title">Practical Considerations for the Use of Circulating Tumor DNA in the Treatment of Patients With Cancer: A Narrative Review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36264554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221102211743&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS AND RELEVANCE: This review found that liquid biopsy is increasingly being used clinically in advanced lung cancer, and ongoing research is identifying applications of circulating tumor DNA-based testing that complement tissue analysis across a broad range of clinical settings. Circulating tumor DNA technologies are advancing quickly and are demonstrating potential benefits for patients, health care practitioners, health care systems, and researchers, at many stages of the patient...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36264554"
                        open
                      >
                        <summary class="article-expander__title">Practical Considerations for the Use of Circulating Tumor DNA in the Treatment of Patients With Cancer: A Narrative Review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36264554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221101212552&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS AND RELEVANCE: This review found that liquid biopsy is increasingly being used clinically in advanced lung cancer, and ongoing research is identifying applications of circulating tumor DNA-based testing that complement tissue analysis across a broad range of clinical settings. Circulating tumor DNA technologies are advancing quickly and are demonstrating potential benefits for patients, health care practitioners, health care systems, and researchers, at many stages of the patient...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36264554"
                        open
                      >
                        <summary class="article-expander__title">Practical Considerations for the Use of Circulating Tumor DNA in the Treatment of Patients With Cancer: A Narrative Review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36264554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221031214247&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS AND RELEVANCE: This review found that liquid biopsy is increasingly being used clinically in advanced lung cancer, and ongoing research is identifying applications of circulating tumor DNA-based testing that complement tissue analysis across a broad range of clinical settings. Circulating tumor DNA technologies are advancing quickly and are demonstrating potential benefits for patients, health care practitioners, health care systems, and researchers, at many stages of the patient...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36264554"
                        open
                      >
                        <summary class="article-expander__title">Practical Considerations for the Use of Circulating Tumor DNA in the Treatment of Patients With Cancer: A Narrative Review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36264554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221030212817&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS AND RELEVANCE: This review found that liquid biopsy is increasingly being used clinically in advanced lung cancer, and ongoing research is identifying applications of circulating tumor DNA-based testing that complement tissue analysis across a broad range of clinical settings. Circulating tumor DNA technologies are advancing quickly and are demonstrating potential benefits for patients, health care practitioners, health care systems, and researchers, at many stages of the patient...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36264554"
                        open
                      >
                        <summary class="article-expander__title">Practical Considerations for the Use of Circulating Tumor DNA in the Treatment of Patients With Cancer: A Narrative Review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36264554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221029212729&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS AND RELEVANCE: This review found that liquid biopsy is increasingly being used clinically in advanced lung cancer, and ongoing research is identifying applications of circulating tumor DNA-based testing that complement tissue analysis across a broad range of clinical settings. Circulating tumor DNA technologies are advancing quickly and are demonstrating potential benefits for patients, health care practitioners, health care systems, and researchers, at many stages of the patient...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36264554"
                        open
                      >
                        <summary class="article-expander__title">Practical Considerations for the Use of Circulating Tumor DNA in the Treatment of Patients With Cancer: A Narrative Review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36264554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221028211238&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS AND RELEVANCE: This review found that liquid biopsy is increasingly being used clinically in advanced lung cancer, and ongoing research is identifying applications of circulating tumor DNA-based testing that complement tissue analysis across a broad range of clinical settings. Circulating tumor DNA technologies are advancing quickly and are demonstrating potential benefits for patients, health care practitioners, health care systems, and researchers, at many stages of the patient...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36264554"
                        open
                      >
                        <summary class="article-expander__title">Practical Considerations for the Use of Circulating Tumor DNA in the Treatment of Patients With Cancer: A Narrative Review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36264554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221027212537&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS AND RELEVANCE: This review found that liquid biopsy is increasingly being used clinically in advanced lung cancer, and ongoing research is identifying applications of circulating tumor DNA-based testing that complement tissue analysis across a broad range of clinical settings. Circulating tumor DNA technologies are advancing quickly and are demonstrating potential benefits for patients, health care practitioners, health care systems, and researchers, at many stages of the patient...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36264554"
                        open
                      >
                        <summary class="article-expander__title">Practical Considerations for the Use of Circulating Tumor DNA in the Treatment of Patients With Cancer: A Narrative Review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36264554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221026211957&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS AND RELEVANCE: This review found that liquid biopsy is increasingly being used clinically in advanced lung cancer, and ongoing research is identifying applications of circulating tumor DNA-based testing that complement tissue analysis across a broad range of clinical settings. Circulating tumor DNA technologies are advancing quickly and are demonstrating potential benefits for patients, health care practitioners, health care systems, and researchers, at many stages of the patient...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36264554"
                        open
                      >
                        <summary class="article-expander__title">Practical Considerations for the Use of Circulating Tumor DNA in the Treatment of Patients With Cancer: A Narrative Review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36264554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221025212118&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS AND RELEVANCE: This review found that liquid biopsy is increasingly being used clinically in advanced lung cancer, and ongoing research is identifying applications of circulating tumor DNA-based testing that complement tissue analysis across a broad range of clinical settings. Circulating tumor DNA technologies are advancing quickly and are demonstrating potential benefits for patients, health care practitioners, health care systems, and researchers, at many stages of the patient...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36264554"
                        open
                      >
                        <summary class="article-expander__title">Practical Considerations for the Use of Circulating Tumor DNA in the Treatment of Patients With Cancer: A Narrative Review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36264554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221024214453&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS AND RELEVANCE: This review found that liquid biopsy is increasingly being used clinically in advanced lung cancer, and ongoing research is identifying applications of circulating tumor DNA-based testing that complement tissue analysis across a broad range of clinical settings. Circulating tumor DNA technologies are advancing quickly and are demonstrating potential benefits for patients, health care practitioners, health care systems, and researchers, at many stages of the patient...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36264554"
                        open
                      >
                        <summary class="article-expander__title">Practical Considerations for the Use of Circulating Tumor DNA in the Treatment of Patients With Cancer: A Narrative Review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36264554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221023215029&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS AND RELEVANCE: This review found that liquid biopsy is increasingly being used clinically in advanced lung cancer, and ongoing research is identifying applications of circulating tumor DNA-based testing that complement tissue analysis across a broad range of clinical settings. Circulating tumor DNA technologies are advancing quickly and are demonstrating potential benefits for patients, health care practitioners, health care systems, and researchers, at many stages of the patient...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36264554"
                        open
                      >
                        <summary class="article-expander__title">Practical Considerations for the Use of Circulating Tumor DNA in the Treatment of Patients With Cancer: A Narrative Review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36264554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221022213551&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS AND RELEVANCE: This review found that liquid biopsy is increasingly being used clinically in advanced lung cancer, and ongoing research is identifying applications of circulating tumor DNA-based testing that complement tissue analysis across a broad range of clinical settings. Circulating tumor DNA technologies are advancing quickly and are demonstrating potential benefits for patients, health care practitioners, health care systems, and researchers, at many stages of the patient...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36264554"
                        open
                      >
                        <summary class="article-expander__title">Practical Considerations for the Use of Circulating Tumor DNA in the Treatment of Patients With Cancer: A Narrative Review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36264554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221021213516&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS AND RELEVANCE: This review found that liquid biopsy is increasingly being used clinically in advanced lung cancer, and ongoing research is identifying applications of circulating tumor DNA-based testing that complement tissue analysis across a broad range of clinical settings. Circulating tumor DNA technologies are advancing quickly and are demonstrating potential benefits for patients, health care practitioners, health care systems, and researchers, at many stages of the patient...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36264554"
                        open
                      >
                        <summary class="article-expander__title">Practical Considerations for the Use of Circulating Tumor DNA in the Treatment of Patients With Cancer: A Narrative Review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36264554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221020212123&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS AND RELEVANCE: This review found that liquid biopsy is increasingly being used clinically in advanced lung cancer, and ongoing research is identifying applications of circulating tumor DNA-based testing that complement tissue analysis across a broad range of clinical settings. Circulating tumor DNA technologies are advancing quickly and are demonstrating potential benefits for patients, health care practitioners, health care systems, and researchers, at many stages of the patient...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221128200439&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.8">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36265329"
                        open
                      >
                        <summary class="article-expander__title">The thin red line between the immune system and cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36265329/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221106201343&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The cancer immunoediting theory describes the dual ability of endogenous antitumor immunity to inhibit or promote progressing cancers. Tumor-specific neoantigens arising from somatic mutations serve as targets for the endogenous T-cell-mediated antitumor immunity and therefore possess a crucial role for tumor development. Additionally, targeting these molecules is conceptually appealing because neoantigens are not expressed in healthy tissue and therefore confer less toxicity and greater...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36265329"
                        open
                      >
                        <summary class="article-expander__title">The thin red line between the immune system and cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36265329/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221105211555&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The cancer immunoediting theory describes the dual ability of endogenous antitumor immunity to inhibit or promote progressing cancers. Tumor-specific neoantigens arising from somatic mutations serve as targets for the endogenous T-cell-mediated antitumor immunity and therefore possess a crucial role for tumor development. Additionally, targeting these molecules is conceptually appealing because neoantigens are not expressed in healthy tissue and therefore confer less toxicity and greater...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36265329"
                        open
                      >
                        <summary class="article-expander__title">The thin red line between the immune system and cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36265329/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221104211729&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The cancer immunoediting theory describes the dual ability of endogenous antitumor immunity to inhibit or promote progressing cancers. Tumor-specific neoantigens arising from somatic mutations serve as targets for the endogenous T-cell-mediated antitumor immunity and therefore possess a crucial role for tumor development. Additionally, targeting these molecules is conceptually appealing because neoantigens are not expressed in healthy tissue and therefore confer less toxicity and greater...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36265329"
                        open
                      >
                        <summary class="article-expander__title">The thin red line between the immune system and cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36265329/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221103212023&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The cancer immunoediting theory describes the dual ability of endogenous antitumor immunity to inhibit or promote progressing cancers. Tumor-specific neoantigens arising from somatic mutations serve as targets for the endogenous T-cell-mediated antitumor immunity and therefore possess a crucial role for tumor development. Additionally, targeting these molecules is conceptually appealing because neoantigens are not expressed in healthy tissue and therefore confer less toxicity and greater...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36265329"
                        open
                      >
                        <summary class="article-expander__title">The thin red line between the immune system and cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36265329/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221102211743&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The cancer immunoediting theory describes the dual ability of endogenous antitumor immunity to inhibit or promote progressing cancers. Tumor-specific neoantigens arising from somatic mutations serve as targets for the endogenous T-cell-mediated antitumor immunity and therefore possess a crucial role for tumor development. Additionally, targeting these molecules is conceptually appealing because neoantigens are not expressed in healthy tissue and therefore confer less toxicity and greater...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36265329"
                        open
                      >
                        <summary class="article-expander__title">The thin red line between the immune system and cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36265329/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221101212552&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The cancer immunoediting theory describes the dual ability of endogenous antitumor immunity to inhibit or promote progressing cancers. Tumor-specific neoantigens arising from somatic mutations serve as targets for the endogenous T-cell-mediated antitumor immunity and therefore possess a crucial role for tumor development. Additionally, targeting these molecules is conceptually appealing because neoantigens are not expressed in healthy tissue and therefore confer less toxicity and greater...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36265329"
                        open
                      >
                        <summary class="article-expander__title">The thin red line between the immune system and cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36265329/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221031214247&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The cancer immunoediting theory describes the dual ability of endogenous antitumor immunity to inhibit or promote progressing cancers. Tumor-specific neoantigens arising from somatic mutations serve as targets for the endogenous T-cell-mediated antitumor immunity and therefore possess a crucial role for tumor development. Additionally, targeting these molecules is conceptually appealing because neoantigens are not expressed in healthy tissue and therefore confer less toxicity and greater...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36265329"
                        open
                      >
                        <summary class="article-expander__title">The thin red line between the immune system and cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36265329/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221030212817&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The cancer immunoediting theory describes the dual ability of endogenous antitumor immunity to inhibit or promote progressing cancers. Tumor-specific neoantigens arising from somatic mutations serve as targets for the endogenous T-cell-mediated antitumor immunity and therefore possess a crucial role for tumor development. Additionally, targeting these molecules is conceptually appealing because neoantigens are not expressed in healthy tissue and therefore confer less toxicity and greater...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36265329"
                        open
                      >
                        <summary class="article-expander__title">The thin red line between the immune system and cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36265329/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221029212729&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The cancer immunoediting theory describes the dual ability of endogenous antitumor immunity to inhibit or promote progressing cancers. Tumor-specific neoantigens arising from somatic mutations serve as targets for the endogenous T-cell-mediated antitumor immunity and therefore possess a crucial role for tumor development. Additionally, targeting these molecules is conceptually appealing because neoantigens are not expressed in healthy tissue and therefore confer less toxicity and greater...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36265329"
                        open
                      >
                        <summary class="article-expander__title">The thin red line between the immune system and cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36265329/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221028211238&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The cancer immunoediting theory describes the dual ability of endogenous antitumor immunity to inhibit or promote progressing cancers. Tumor-specific neoantigens arising from somatic mutations serve as targets for the endogenous T-cell-mediated antitumor immunity and therefore possess a crucial role for tumor development. Additionally, targeting these molecules is conceptually appealing because neoantigens are not expressed in healthy tissue and therefore confer less toxicity and greater...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36265329"
                        open
                      >
                        <summary class="article-expander__title">The thin red line between the immune system and cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36265329/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221027212535&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The cancer immunoediting theory describes the dual ability of endogenous antitumor immunity to inhibit or promote progressing cancers. Tumor-specific neoantigens arising from somatic mutations serve as targets for the endogenous T-cell-mediated antitumor immunity and therefore possess a crucial role for tumor development. Additionally, targeting these molecules is conceptually appealing because neoantigens are not expressed in healthy tissue and therefore confer less toxicity and greater...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36265329"
                        open
                      >
                        <summary class="article-expander__title">The thin red line between the immune system and cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36265329/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221026211957&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The cancer immunoediting theory describes the dual ability of endogenous antitumor immunity to inhibit or promote progressing cancers. Tumor-specific neoantigens arising from somatic mutations serve as targets for the endogenous T-cell-mediated antitumor immunity and therefore possess a crucial role for tumor development. Additionally, targeting these molecules is conceptually appealing because neoantigens are not expressed in healthy tissue and therefore confer less toxicity and greater...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36265329"
                        open
                      >
                        <summary class="article-expander__title">The thin red line between the immune system and cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36265329/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221025212118&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The cancer immunoediting theory describes the dual ability of endogenous antitumor immunity to inhibit or promote progressing cancers. Tumor-specific neoantigens arising from somatic mutations serve as targets for the endogenous T-cell-mediated antitumor immunity and therefore possess a crucial role for tumor development. Additionally, targeting these molecules is conceptually appealing because neoantigens are not expressed in healthy tissue and therefore confer less toxicity and greater...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36265329"
                        open
                      >
                        <summary class="article-expander__title">The thin red line between the immune system and cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36265329/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221024214453&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The cancer immunoediting theory describes the dual ability of endogenous antitumor immunity to inhibit or promote progressing cancers. Tumor-specific neoantigens arising from somatic mutations serve as targets for the endogenous T-cell-mediated antitumor immunity and therefore possess a crucial role for tumor development. Additionally, targeting these molecules is conceptually appealing because neoantigens are not expressed in healthy tissue and therefore confer less toxicity and greater...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36265329"
                        open
                      >
                        <summary class="article-expander__title">The thin red line between the immune system and cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36265329/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221023215029&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The cancer immunoediting theory describes the dual ability of endogenous antitumor immunity to inhibit or promote progressing cancers. Tumor-specific neoantigens arising from somatic mutations serve as targets for the endogenous T-cell-mediated antitumor immunity and therefore possess a crucial role for tumor development. Additionally, targeting these molecules is conceptually appealing because neoantigens are not expressed in healthy tissue and therefore confer less toxicity and greater...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36265329"
                        open
                      >
                        <summary class="article-expander__title">The thin red line between the immune system and cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36265329/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221022213551&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The cancer immunoediting theory describes the dual ability of endogenous antitumor immunity to inhibit or promote progressing cancers. Tumor-specific neoantigens arising from somatic mutations serve as targets for the endogenous T-cell-mediated antitumor immunity and therefore possess a crucial role for tumor development. Additionally, targeting these molecules is conceptually appealing because neoantigens are not expressed in healthy tissue and therefore confer less toxicity and greater...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36265329"
                        open
                      >
                        <summary class="article-expander__title">The thin red line between the immune system and cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36265329/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221021213516&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The cancer immunoediting theory describes the dual ability of endogenous antitumor immunity to inhibit or promote progressing cancers. Tumor-specific neoantigens arising from somatic mutations serve as targets for the endogenous T-cell-mediated antitumor immunity and therefore possess a crucial role for tumor development. Additionally, targeting these molecules is conceptually appealing because neoantigens are not expressed in healthy tissue and therefore confer less toxicity and greater...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-19 datetime="2022-10-19T10:00:00.000Z">2022-10-19</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-19 datetime="2022-10-19T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221128200439&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.8">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36261585"
                        open
                      >
                        <summary class="article-expander__title">Clonal relationship and alcohol consumption-associated mutational signature in synchronous hypopharyngeal tumours and oesophageal squamous cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36261585/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221104211729&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our hypothesis-generating M-WES ITH assessment data have implications for prognostication. Collectively, our findings support multicentric independent clonal evolution, the field cancerisation theory, and suggest novel insights implicating an aetiologic role of alcohol metabolism in dual ESCC/HPC carcinogenesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36261585"
                        open
                      >
                        <summary class="article-expander__title">Clonal relationship and alcohol consumption-associated mutational signature in synchronous hypopharyngeal tumours and oesophageal squamous cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36261585/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221103212023&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our hypothesis-generating M-WES ITH assessment data have implications for prognostication. Collectively, our findings support multicentric independent clonal evolution, the field cancerisation theory, and suggest novel insights implicating an aetiologic role of alcohol metabolism in dual ESCC/HPC carcinogenesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36261585"
                        open
                      >
                        <summary class="article-expander__title">Clonal relationship and alcohol consumption-associated mutational signature in synchronous hypopharyngeal tumours and oesophageal squamous cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36261585/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221102211743&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our hypothesis-generating M-WES ITH assessment data have implications for prognostication. Collectively, our findings support multicentric independent clonal evolution, the field cancerisation theory, and suggest novel insights implicating an aetiologic role of alcohol metabolism in dual ESCC/HPC carcinogenesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36261585"
                        open
                      >
                        <summary class="article-expander__title">Clonal relationship and alcohol consumption-associated mutational signature in synchronous hypopharyngeal tumours and oesophageal squamous cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36261585/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221101212552&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our hypothesis-generating M-WES ITH assessment data have implications for prognostication. Collectively, our findings support multicentric independent clonal evolution, the field cancerisation theory, and suggest novel insights implicating an aetiologic role of alcohol metabolism in dual ESCC/HPC carcinogenesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36261585"
                        open
                      >
                        <summary class="article-expander__title">Clonal relationship and alcohol consumption-associated mutational signature in synchronous hypopharyngeal tumours and oesophageal squamous cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36261585/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221031214247&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our hypothesis-generating M-WES ITH assessment data have implications for prognostication. Collectively, our findings support multicentric independent clonal evolution, the field cancerisation theory, and suggest novel insights implicating an aetiologic role of alcohol metabolism in dual ESCC/HPC carcinogenesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36261585"
                        open
                      >
                        <summary class="article-expander__title">Clonal relationship and alcohol consumption-associated mutational signature in synchronous hypopharyngeal tumours and oesophageal squamous cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36261585/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221030212817&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our hypothesis-generating M-WES ITH assessment data have implications for prognostication. Collectively, our findings support multicentric independent clonal evolution, the field cancerisation theory, and suggest novel insights implicating an aetiologic role of alcohol metabolism in dual ESCC/HPC carcinogenesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36261585"
                        open
                      >
                        <summary class="article-expander__title">Clonal relationship and alcohol consumption-associated mutational signature in synchronous hypopharyngeal tumours and oesophageal squamous cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36261585/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221029212729&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our hypothesis-generating M-WES ITH assessment data have implications for prognostication. Collectively, our findings support multicentric independent clonal evolution, the field cancerisation theory, and suggest novel insights implicating an aetiologic role of alcohol metabolism in dual ESCC/HPC carcinogenesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36261585"
                        open
                      >
                        <summary class="article-expander__title">Clonal relationship and alcohol consumption-associated mutational signature in synchronous hypopharyngeal tumours and oesophageal squamous cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36261585/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221028211238&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our hypothesis-generating M-WES ITH assessment data have implications for prognostication. Collectively, our findings support multicentric independent clonal evolution, the field cancerisation theory, and suggest novel insights implicating an aetiologic role of alcohol metabolism in dual ESCC/HPC carcinogenesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36261585"
                        open
                      >
                        <summary class="article-expander__title">Clonal relationship and alcohol consumption-associated mutational signature in synchronous hypopharyngeal tumours and oesophageal squamous cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36261585/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221027212535&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our hypothesis-generating M-WES ITH assessment data have implications for prognostication. Collectively, our findings support multicentric independent clonal evolution, the field cancerisation theory, and suggest novel insights implicating an aetiologic role of alcohol metabolism in dual ESCC/HPC carcinogenesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36261585"
                        open
                      >
                        <summary class="article-expander__title">Clonal relationship and alcohol consumption-associated mutational signature in synchronous hypopharyngeal tumours and oesophageal squamous cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36261585/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221026211957&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our hypothesis-generating M-WES ITH assessment data have implications for prognostication. Collectively, our findings support multicentric independent clonal evolution, the field cancerisation theory, and suggest novel insights implicating an aetiologic role of alcohol metabolism in dual ESCC/HPC carcinogenesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36261585"
                        open
                      >
                        <summary class="article-expander__title">Clonal relationship and alcohol consumption-associated mutational signature in synchronous hypopharyngeal tumours and oesophageal squamous cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36261585/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221025212118&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our hypothesis-generating M-WES ITH assessment data have implications for prognostication. Collectively, our findings support multicentric independent clonal evolution, the field cancerisation theory, and suggest novel insights implicating an aetiologic role of alcohol metabolism in dual ESCC/HPC carcinogenesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36261585"
                        open
                      >
                        <summary class="article-expander__title">Clonal relationship and alcohol consumption-associated mutational signature in synchronous hypopharyngeal tumours and oesophageal squamous cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36261585/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221024214453&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our hypothesis-generating M-WES ITH assessment data have implications for prognostication. Collectively, our findings support multicentric independent clonal evolution, the field cancerisation theory, and suggest novel insights implicating an aetiologic role of alcohol metabolism in dual ESCC/HPC carcinogenesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36261585"
                        open
                      >
                        <summary class="article-expander__title">Clonal relationship and alcohol consumption-associated mutational signature in synchronous hypopharyngeal tumours and oesophageal squamous cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36261585/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221023215029&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our hypothesis-generating M-WES ITH assessment data have implications for prognostication. Collectively, our findings support multicentric independent clonal evolution, the field cancerisation theory, and suggest novel insights implicating an aetiologic role of alcohol metabolism in dual ESCC/HPC carcinogenesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36261585"
                        open
                      >
                        <summary class="article-expander__title">Clonal relationship and alcohol consumption-associated mutational signature in synchronous hypopharyngeal tumours and oesophageal squamous cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36261585/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221022213551&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our hypothesis-generating M-WES ITH assessment data have implications for prognostication. Collectively, our findings support multicentric independent clonal evolution, the field cancerisation theory, and suggest novel insights implicating an aetiologic role of alcohol metabolism in dual ESCC/HPC carcinogenesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36261585"
                        open
                      >
                        <summary class="article-expander__title">Clonal relationship and alcohol consumption-associated mutational signature in synchronous hypopharyngeal tumours and oesophageal squamous cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36261585/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221021213516&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our hypothesis-generating M-WES ITH assessment data have implications for prognostication. Collectively, our findings support multicentric independent clonal evolution, the field cancerisation theory, and suggest novel insights implicating an aetiologic role of alcohol metabolism in dual ESCC/HPC carcinogenesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36261585"
                        open
                      >
                        <summary class="article-expander__title">Clonal relationship and alcohol consumption-associated mutational signature in synchronous hypopharyngeal tumours and oesophageal squamous cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36261585/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221020212123&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our hypothesis-generating M-WES ITH assessment data have implications for prognostication. Collectively, our findings support multicentric independent clonal evolution, the field cancerisation theory, and suggest novel insights implicating an aetiologic role of alcohol metabolism in dual ESCC/HPC carcinogenesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-17 datetime="2022-10-17T10:00:00.000Z">2022-10-17</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-17 datetime="2022-10-17T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.8&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20221128200439&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36253429"
                        open
                      >
                        <summary class="article-expander__title">Molecular landscape of immune pressure and escape in aplastic anemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36253429/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221106201343&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Idiopathic aplastic anemia (IAA) pathophysiology is dominated by autoreactivity of human leukocyte antigen (HLA)-restricted T-cells against antigens presented by hematopoietic stem and progenitor cells (HSPCs). Expansion of PIGA and HLA class I mutant HSPCs have been linked to immune evasion from T-cell mediated pressures. We hypothesized that in analogy with antitumor immunity, the pathophysiological cascade of immune escape in IAA is initiated by immunoediting pressures and culminates with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36252553"
                        open
                      >
                        <summary class="article-expander__title">JAK: Not Just Another Kinase</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36252553/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221106201343&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The Janus kinase/signal transducer and activator of transcription (JAK/STAT) axis is implicated in cancer, inflammation, and immunity.Numerous cytokines/growth factors affect JAK/STAT signaling. JAKs noncovalently associate with cytokine receptors, mediate receptor tyrosine phosphorylation, and recruit STAT proteins.Tyrosine-phosphorylated STATs dimerize and are transported into the nucleus to function as transcription factors. Signaling is attenuated by specific suppressor of cytokine signaling...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36253429"
                        open
                      >
                        <summary class="article-expander__title">Molecular landscape of immune pressure and escape in aplastic anemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36253429/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221105211555&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Idiopathic aplastic anemia (IAA) pathophysiology is dominated by autoreactivity of human leukocyte antigen (HLA)-restricted T-cells against antigens presented by hematopoietic stem and progenitor cells (HSPCs). Expansion of PIGA and HLA class I mutant HSPCs have been linked to immune evasion from T-cell mediated pressures. We hypothesized that in analogy with antitumor immunity, the pathophysiological cascade of immune escape in IAA is initiated by immunoediting pressures and culminates with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36252553"
                        open
                      >
                        <summary class="article-expander__title">JAK: Not Just Another Kinase</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36252553/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221105211555&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The Janus kinase/signal transducer and activator of transcription (JAK/STAT) axis is implicated in cancer, inflammation, and immunity.Numerous cytokines/growth factors affect JAK/STAT signaling. JAKs noncovalently associate with cytokine receptors, mediate receptor tyrosine phosphorylation, and recruit STAT proteins.Tyrosine-phosphorylated STATs dimerize and are transported into the nucleus to function as transcription factors. Signaling is attenuated by specific suppressor of cytokine signaling...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36253429"
                        open
                      >
                        <summary class="article-expander__title">Molecular landscape of immune pressure and escape in aplastic anemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36253429/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221104211730&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Idiopathic aplastic anemia (IAA) pathophysiology is dominated by autoreactivity of human leukocyte antigen (HLA)-restricted T-cells against antigens presented by hematopoietic stem and progenitor cells (HSPCs). Expansion of PIGA and HLA class I mutant HSPCs have been linked to immune evasion from T-cell mediated pressures. We hypothesized that in analogy with antitumor immunity, the pathophysiological cascade of immune escape in IAA is initiated by immunoediting pressures and culminates with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36252553"
                        open
                      >
                        <summary class="article-expander__title">JAK: Not Just Another Kinase</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36252553/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221104211730&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The Janus kinase/signal transducer and activator of transcription (JAK/STAT) axis is implicated in cancer, inflammation, and immunity.Numerous cytokines/growth factors affect JAK/STAT signaling. JAKs noncovalently associate with cytokine receptors, mediate receptor tyrosine phosphorylation, and recruit STAT proteins.Tyrosine-phosphorylated STATs dimerize and are transported into the nucleus to function as transcription factors. Signaling is attenuated by specific suppressor of cytokine signaling...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36253429"
                        open
                      >
                        <summary class="article-expander__title">Molecular landscape of immune pressure and escape in aplastic anemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36253429/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221103212023&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Idiopathic aplastic anemia (IAA) pathophysiology is dominated by autoreactivity of human leukocyte antigen (HLA)-restricted T-cells against antigens presented by hematopoietic stem and progenitor cells (HSPCs). Expansion of PIGA and HLA class I mutant HSPCs have been linked to immune evasion from T-cell mediated pressures. We hypothesized that in analogy with antitumor immunity, the pathophysiological cascade of immune escape in IAA is initiated by immunoediting pressures and culminates with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36252553"
                        open
                      >
                        <summary class="article-expander__title">JAK: Not Just Another Kinase</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36252553/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221103212023&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The Janus kinase/signal transducer and activator of transcription (JAK/STAT) axis is implicated in cancer, inflammation, and immunity.Numerous cytokines/growth factors affect JAK/STAT signaling. JAKs noncovalently associate with cytokine receptors, mediate receptor tyrosine phosphorylation, and recruit STAT proteins.Tyrosine-phosphorylated STATs dimerize and are transported into the nucleus to function as transcription factors. Signaling is attenuated by specific suppressor of cytokine signaling...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36253429"
                        open
                      >
                        <summary class="article-expander__title">Molecular landscape of immune pressure and escape in aplastic anemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36253429/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221102211743&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Idiopathic aplastic anemia (IAA) pathophysiology is dominated by autoreactivity of human leukocyte antigen (HLA)-restricted T-cells against antigens presented by hematopoietic stem and progenitor cells (HSPCs). Expansion of PIGA and HLA class I mutant HSPCs have been linked to immune evasion from T-cell mediated pressures. We hypothesized that in analogy with antitumor immunity, the pathophysiological cascade of immune escape in IAA is initiated by immunoediting pressures and culminates with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36252553"
                        open
                      >
                        <summary class="article-expander__title">JAK: Not Just Another Kinase</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36252553/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221102211743&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The Janus kinase/signal transducer and activator of transcription (JAK/STAT) axis is implicated in cancer, inflammation, and immunity.Numerous cytokines/growth factors affect JAK/STAT signaling. JAKs noncovalently associate with cytokine receptors, mediate receptor tyrosine phosphorylation, and recruit STAT proteins.Tyrosine-phosphorylated STATs dimerize and are transported into the nucleus to function as transcription factors. Signaling is attenuated by specific suppressor of cytokine signaling...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36251745"
                        open
                      >
                        <summary class="article-expander__title">Distinct clonal identities of B-ALLs arising after Lenolidomide therapy for multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36251745/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221102211743&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Patients with multiple myeloma (MM) who are treated with lenalidomide rarely develop a secondary B-cell acute lymphoblastic leukemia (B-ALL). The clonal and biological relationship between these sequential malignancies is not yet clear. We identified 17 patients with MM treated with lenalidomide, who subsequently developed B-ALL. Samples were evaluated with sequencing, cytogenetics/FISH, immunohistochemical staining (IHC), and IgH clonality assessment. Samples were assessed for shared mutations...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36253429"
                        open
                      >
                        <summary class="article-expander__title">Molecular landscape of immune pressure and escape in aplastic anemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36253429/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221101212552&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Idiopathic aplastic anemia (IAA) pathophysiology is dominated by autoreactivity of human leukocyte antigen (HLA)-restricted T-cells against antigens presented by hematopoietic stem and progenitor cells (HSPCs). Expansion of PIGA and HLA class I mutant HSPCs have been linked to immune evasion from T-cell mediated pressures. We hypothesized that in analogy with antitumor immunity, the pathophysiological cascade of immune escape in IAA is initiated by immunoediting pressures and culminates with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36252553"
                        open
                      >
                        <summary class="article-expander__title">JAK: Not Just Another Kinase</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36252553/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221101212552&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The Janus kinase/signal transducer and activator of transcription (JAK/STAT) axis is implicated in cancer, inflammation, and immunity.Numerous cytokines/growth factors affect JAK/STAT signaling. JAKs noncovalently associate with cytokine receptors, mediate receptor tyrosine phosphorylation, and recruit STAT proteins.Tyrosine-phosphorylated STATs dimerize and are transported into the nucleus to function as transcription factors. Signaling is attenuated by specific suppressor of cytokine signaling...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36251745"
                        open
                      >
                        <summary class="article-expander__title">Distinct clonal identities of B-ALLs arising after Lenolidomide therapy for multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36251745/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221101212552&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Patients with multiple myeloma (MM) who are treated with lenalidomide rarely develop a secondary B-cell acute lymphoblastic leukemia (B-ALL). The clonal and biological relationship between these sequential malignancies is not yet clear. We identified 17 patients with MM treated with lenalidomide, who subsequently developed B-ALL. Samples were evaluated with sequencing, cytogenetics/FISH, immunohistochemical staining (IHC), and IgH clonality assessment. Samples were assessed for shared mutations...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36253429"
                        open
                      >
                        <summary class="article-expander__title">Molecular landscape of immune pressure and escape in aplastic anemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36253429/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221031214247&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Idiopathic aplastic anemia (IAA) pathophysiology is dominated by autoreactivity of human leukocyte antigen (HLA)-restricted T-cells against antigens presented by hematopoietic stem and progenitor cells (HSPCs). Expansion of PIGA and HLA class I mutant HSPCs have been linked to immune evasion from T-cell mediated pressures. We hypothesized that in analogy with antitumor immunity, the pathophysiological cascade of immune escape in IAA is initiated by immunoediting pressures and culminates with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36252553"
                        open
                      >
                        <summary class="article-expander__title">JAK: Not Just Another Kinase</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36252553/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221031214247&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The Janus kinase/signal transducer and activator of transcription (JAK/STAT) axis is implicated in cancer, inflammation, and immunity.Numerous cytokines/growth factors affect JAK/STAT signaling. JAKs noncovalently associate with cytokine receptors, mediate receptor tyrosine phosphorylation, and recruit STAT proteins.Tyrosine-phosphorylated STATs dimerize and are transported into the nucleus to function as transcription factors. Signaling is attenuated by specific suppressor of cytokine signaling...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36251745"
                        open
                      >
                        <summary class="article-expander__title">Distinct clonal identities of B-ALLs arising after Lenolidomide therapy for multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36251745/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221031214247&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Patients with multiple myeloma (MM) who are treated with lenalidomide rarely develop a secondary B-cell acute lymphoblastic leukemia (B-ALL). The clonal and biological relationship between these sequential malignancies is not yet clear. We identified 17 patients with MM treated with lenalidomide, who subsequently developed B-ALL. Samples were evaluated with sequencing, cytogenetics/FISH, immunohistochemical staining (IHC), and IgH clonality assessment. Samples were assessed for shared mutations...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36253429"
                        open
                      >
                        <summary class="article-expander__title">Molecular landscape of immune pressure and escape in aplastic anemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36253429/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221030212817&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Idiopathic aplastic anemia (IAA) pathophysiology is dominated by autoreactivity of human leukocyte antigen (HLA)-restricted T-cells against antigens presented by hematopoietic stem and progenitor cells (HSPCs). Expansion of PIGA and HLA class I mutant HSPCs have been linked to immune evasion from T-cell mediated pressures. We hypothesized that in analogy with antitumor immunity, the pathophysiological cascade of immune escape in IAA is initiated by immunoediting pressures and culminates with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36252553"
                        open
                      >
                        <summary class="article-expander__title">JAK: Not Just Another Kinase</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36252553/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221030212817&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The Janus kinase/signal transducer and activator of transcription (JAK/STAT) axis is implicated in cancer, inflammation, and immunity.Numerous cytokines/growth factors affect JAK/STAT signaling. JAKs noncovalently associate with cytokine receptors, mediate receptor tyrosine phosphorylation, and recruit STAT proteins.Tyrosine-phosphorylated STATs dimerize and are transported into the nucleus to function as transcription factors. Signaling is attenuated by specific suppressor of cytokine signaling...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36251745"
                        open
                      >
                        <summary class="article-expander__title">Distinct clonal identities of B-ALLs arising after Lenolidomide therapy for multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36251745/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221030212817&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Patients with multiple myeloma (MM) who are treated with lenalidomide rarely develop a secondary B-cell acute lymphoblastic leukemia (B-ALL). The clonal and biological relationship between these sequential malignancies is not yet clear. We identified 17 patients with MM treated with lenalidomide, who subsequently developed B-ALL. Samples were evaluated with sequencing, cytogenetics/FISH, immunohistochemical staining (IHC), and IgH clonality assessment. Samples were assessed for shared mutations...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36253429"
                        open
                      >
                        <summary class="article-expander__title">Molecular landscape of immune pressure and escape in aplastic anemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36253429/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221029212729&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Idiopathic aplastic anemia (IAA) pathophysiology is dominated by autoreactivity of human leukocyte antigen (HLA)-restricted T-cells against antigens presented by hematopoietic stem and progenitor cells (HSPCs). Expansion of PIGA and HLA class I mutant HSPCs have been linked to immune evasion from T-cell mediated pressures. We hypothesized that in analogy with antitumor immunity, the pathophysiological cascade of immune escape in IAA is initiated by immunoediting pressures and culminates with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36252553"
                        open
                      >
                        <summary class="article-expander__title">JAK: Not Just Another Kinase</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36252553/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221029212729&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The Janus kinase/signal transducer and activator of transcription (JAK/STAT) axis is implicated in cancer, inflammation, and immunity.Numerous cytokines/growth factors affect JAK/STAT signaling. JAKs noncovalently associate with cytokine receptors, mediate receptor tyrosine phosphorylation, and recruit STAT proteins.Tyrosine-phosphorylated STATs dimerize and are transported into the nucleus to function as transcription factors. Signaling is attenuated by specific suppressor of cytokine signaling...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36251745"
                        open
                      >
                        <summary class="article-expander__title">Distinct clonal identities of B-ALLs arising after Lenolidomide therapy for multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36251745/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221029212729&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Patients with multiple myeloma (MM) who are treated with lenalidomide rarely develop a secondary B-cell acute lymphoblastic leukemia (B-ALL). The clonal and biological relationship between these sequential malignancies is not yet clear. We identified 17 patients with MM treated with lenalidomide, who subsequently developed B-ALL. Samples were evaluated with sequencing, cytogenetics/FISH, immunohistochemical staining (IHC), and IgH clonality assessment. Samples were assessed for shared mutations...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36253429"
                        open
                      >
                        <summary class="article-expander__title">Molecular landscape of immune pressure and escape in aplastic anemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36253429/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221028211238&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Idiopathic aplastic anemia (IAA) pathophysiology is dominated by autoreactivity of human leukocyte antigen (HLA)-restricted T-cells against antigens presented by hematopoietic stem and progenitor cells (HSPCs). Expansion of PIGA and HLA class I mutant HSPCs have been linked to immune evasion from T-cell mediated pressures. We hypothesized that in analogy with antitumor immunity, the pathophysiological cascade of immune escape in IAA is initiated by immunoediting pressures and culminates with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36252553"
                        open
                      >
                        <summary class="article-expander__title">JAK: Not Just Another Kinase</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36252553/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221028211238&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The Janus kinase/signal transducer and activator of transcription (JAK/STAT) axis is implicated in cancer, inflammation, and immunity.Numerous cytokines/growth factors affect JAK/STAT signaling. JAKs noncovalently associate with cytokine receptors, mediate receptor tyrosine phosphorylation, and recruit STAT proteins.Tyrosine-phosphorylated STATs dimerize and are transported into the nucleus to function as transcription factors. Signaling is attenuated by specific suppressor of cytokine signaling...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36251745"
                        open
                      >
                        <summary class="article-expander__title">Distinct clonal identities of B-ALLs arising after Lenolidomide therapy for multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36251745/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221028211238&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Patients with multiple myeloma (MM) who are treated with lenalidomide rarely develop a secondary B-cell acute lymphoblastic leukemia (B-ALL). The clonal and biological relationship between these sequential malignancies is not yet clear. We identified 17 patients with MM treated with lenalidomide, who subsequently developed B-ALL. Samples were evaluated with sequencing, cytogenetics/FISH, immunohistochemical staining (IHC), and IgH clonality assessment. Samples were assessed for shared mutations...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36253429"
                        open
                      >
                        <summary class="article-expander__title">Molecular landscape of immune pressure and escape in aplastic anemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36253429/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221027212537&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Idiopathic aplastic anemia (IAA) pathophysiology is dominated by autoreactivity of human leukocyte antigen (HLA)-restricted T-cells against antigens presented by hematopoietic stem and progenitor cells (HSPCs). Expansion of PIGA and HLA class I mutant HSPCs have been linked to immune evasion from T-cell mediated pressures. We hypothesized that in analogy with antitumor immunity, the pathophysiological cascade of immune escape in IAA is initiated by immunoediting pressures and culminates with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36252553"
                        open
                      >
                        <summary class="article-expander__title">JAK: Not Just Another Kinase</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36252553/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221027212537&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The Janus kinase/signal transducer and activator of transcription (JAK/STAT) axis is implicated in cancer, inflammation, and immunity.Numerous cytokines/growth factors affect JAK/STAT signaling. JAKs noncovalently associate with cytokine receptors, mediate receptor tyrosine phosphorylation, and recruit STAT proteins.Tyrosine-phosphorylated STATs dimerize and are transported into the nucleus to function as transcription factors. Signaling is attenuated by specific suppressor of cytokine signaling...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36251745"
                        open
                      >
                        <summary class="article-expander__title">Distinct clonal identities of B-ALLs arising after Lenolidomide therapy for multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36251745/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221027212537&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Patients with multiple myeloma (MM) who are treated with lenalidomide rarely develop a secondary B-cell acute lymphoblastic leukemia (B-ALL). The clonal and biological relationship between these sequential malignancies is not yet clear. We identified 17 patients with MM treated with lenalidomide, who subsequently developed B-ALL. Samples were evaluated with sequencing, cytogenetics/FISH, immunohistochemical staining (IHC), and IgH clonality assessment. Samples were assessed for shared mutations...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36253429"
                        open
                      >
                        <summary class="article-expander__title">Molecular landscape of immune pressure and escape in aplastic anemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36253429/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221026211957&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Idiopathic aplastic anemia (IAA) pathophysiology is dominated by autoreactivity of human leukocyte antigen (HLA)-restricted T-cells against antigens presented by hematopoietic stem and progenitor cells (HSPCs). Expansion of PIGA and HLA class I mutant HSPCs have been linked to immune evasion from T-cell mediated pressures. We hypothesized that in analogy with antitumor immunity, the pathophysiological cascade of immune escape in IAA is initiated by immunoediting pressures and culminates with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36252553"
                        open
                      >
                        <summary class="article-expander__title">JAK: Not Just Another Kinase</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36252553/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221026211957&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The Janus kinase/signal transducer and activator of transcription (JAK/STAT) axis is implicated in cancer, inflammation, and immunity.Numerous cytokines/growth factors affect JAK/STAT signaling. JAKs noncovalently associate with cytokine receptors, mediate receptor tyrosine phosphorylation, and recruit STAT proteins.Tyrosine-phosphorylated STATs dimerize and are transported into the nucleus to function as transcription factors. Signaling is attenuated by specific suppressor of cytokine signaling...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36251745"
                        open
                      >
                        <summary class="article-expander__title">Distinct clonal identities of B-ALLs arising after Lenolidomide therapy for multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36251745/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221026211957&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Patients with multiple myeloma (MM) who are treated with lenalidomide rarely develop a secondary B-cell acute lymphoblastic leukemia (B-ALL). The clonal and biological relationship between these sequential malignancies is not yet clear. We identified 17 patients with MM treated with lenalidomide, who subsequently developed B-ALL. Samples were evaluated with sequencing, cytogenetics/FISH, immunohistochemical staining (IHC), and IgH clonality assessment. Samples were assessed for shared mutations...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36253429"
                        open
                      >
                        <summary class="article-expander__title">Molecular landscape of immune pressure and escape in aplastic anemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36253429/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221025212118&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Idiopathic aplastic anemia (IAA) pathophysiology is dominated by autoreactivity of human leukocyte antigen (HLA)-restricted T-cells against antigens presented by hematopoietic stem and progenitor cells (HSPCs). Expansion of PIGA and HLA class I mutant HSPCs have been linked to immune evasion from T-cell mediated pressures. We hypothesized that in analogy with antitumor immunity, the pathophysiological cascade of immune escape in IAA is initiated by immunoediting pressures and culminates with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36252553"
                        open
                      >
                        <summary class="article-expander__title">JAK: Not Just Another Kinase</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36252553/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221025212118&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The Janus kinase/signal transducer and activator of transcription (JAK/STAT) axis is implicated in cancer, inflammation, and immunity.Numerous cytokines/growth factors affect JAK/STAT signaling. JAKs noncovalently associate with cytokine receptors, mediate receptor tyrosine phosphorylation, and recruit STAT proteins.Tyrosine-phosphorylated STATs dimerize and are transported into the nucleus to function as transcription factors. Signaling is attenuated by specific suppressor of cytokine signaling...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36251745"
                        open
                      >
                        <summary class="article-expander__title">Distinct clonal identities of B-ALLs arising after Lenolidomide therapy for multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36251745/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221025212118&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Patients with multiple myeloma (MM) who are treated with lenalidomide rarely develop a secondary B-cell acute lymphoblastic leukemia (B-ALL). The clonal and biological relationship between these sequential malignancies is not yet clear. We identified 17 patients with MM treated with lenalidomide, who subsequently developed B-ALL. Samples were evaluated with sequencing, cytogenetics/FISH, immunohistochemical staining (IHC), and IgH clonality assessment. Samples were assessed for shared mutations...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36253429"
                        open
                      >
                        <summary class="article-expander__title">Molecular landscape of immune pressure and escape in aplastic anemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36253429/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221024214453&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Idiopathic aplastic anemia (IAA) pathophysiology is dominated by autoreactivity of human leukocyte antigen (HLA)-restricted T-cells against antigens presented by hematopoietic stem and progenitor cells (HSPCs). Expansion of PIGA and HLA class I mutant HSPCs have been linked to immune evasion from T-cell mediated pressures. We hypothesized that in analogy with antitumor immunity, the pathophysiological cascade of immune escape in IAA is initiated by immunoediting pressures and culminates with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36252553"
                        open
                      >
                        <summary class="article-expander__title">JAK: Not Just Another Kinase</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36252553/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221024214453&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The Janus kinase/signal transducer and activator of transcription (JAK/STAT) axis is implicated in cancer, inflammation, and immunity.Numerous cytokines/growth factors affect JAK/STAT signaling. JAKs noncovalently associate with cytokine receptors, mediate receptor tyrosine phosphorylation, and recruit STAT proteins.Tyrosine-phosphorylated STATs dimerize and are transported into the nucleus to function as transcription factors. Signaling is attenuated by specific suppressor of cytokine signaling...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36251745"
                        open
                      >
                        <summary class="article-expander__title">Distinct clonal identities of B-ALLs arising after Lenolidomide therapy for multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36251745/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221024214453&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Patients with multiple myeloma (MM) who are treated with lenalidomide rarely develop a secondary B-cell acute lymphoblastic leukemia (B-ALL). The clonal and biological relationship between these sequential malignancies is not yet clear. We identified 17 patients with MM treated with lenalidomide, who subsequently developed B-ALL. Samples were evaluated with sequencing, cytogenetics/FISH, immunohistochemical staining (IHC), and IgH clonality assessment. Samples were assessed for shared mutations...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36253429"
                        open
                      >
                        <summary class="article-expander__title">Molecular landscape of immune pressure and escape in aplastic anemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36253429/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221023215029&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Idiopathic aplastic anemia (IAA) pathophysiology is dominated by autoreactivity of human leukocyte antigen (HLA)-restricted T-cells against antigens presented by hematopoietic stem and progenitor cells (HSPCs). Expansion of PIGA and HLA class I mutant HSPCs have been linked to immune evasion from T-cell mediated pressures. We hypothesized that in analogy with antitumor immunity, the pathophysiological cascade of immune escape in IAA is initiated by immunoediting pressures and culminates with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36252553"
                        open
                      >
                        <summary class="article-expander__title">JAK: Not Just Another Kinase</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36252553/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221023215029&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The Janus kinase/signal transducer and activator of transcription (JAK/STAT) axis is implicated in cancer, inflammation, and immunity.Numerous cytokines/growth factors affect JAK/STAT signaling. JAKs noncovalently associate with cytokine receptors, mediate receptor tyrosine phosphorylation, and recruit STAT proteins.Tyrosine-phosphorylated STATs dimerize and are transported into the nucleus to function as transcription factors. Signaling is attenuated by specific suppressor of cytokine signaling...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36251745"
                        open
                      >
                        <summary class="article-expander__title">Distinct clonal identities of B-ALLs arising after Lenolidomide therapy for multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36251745/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221023215029&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Patients with multiple myeloma (MM) who are treated with lenalidomide rarely develop a secondary B-cell acute lymphoblastic leukemia (B-ALL). The clonal and biological relationship between these sequential malignancies is not yet clear. We identified 17 patients with MM treated with lenalidomide, who subsequently developed B-ALL. Samples were evaluated with sequencing, cytogenetics/FISH, immunohistochemical staining (IHC), and IgH clonality assessment. Samples were assessed for shared mutations...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36253429"
                        open
                      >
                        <summary class="article-expander__title">Molecular landscape of immune pressure and escape in aplastic anemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36253429/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221022213551&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Idiopathic aplastic anemia (IAA) pathophysiology is dominated by autoreactivity of human leukocyte antigen (HLA)-restricted T-cells against antigens presented by hematopoietic stem and progenitor cells (HSPCs). Expansion of PIGA and HLA class I mutant HSPCs have been linked to immune evasion from T-cell mediated pressures. We hypothesized that in analogy with antitumor immunity, the pathophysiological cascade of immune escape in IAA is initiated by immunoediting pressures and culminates with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36252553"
                        open
                      >
                        <summary class="article-expander__title">JAK: Not Just Another Kinase</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36252553/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221022213551&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The Janus kinase/signal transducer and activator of transcription (JAK/STAT) axis is implicated in cancer, inflammation, and immunity.Numerous cytokines/growth factors affect JAK/STAT signaling. JAKs noncovalently associate with cytokine receptors, mediate receptor tyrosine phosphorylation, and recruit STAT proteins.Tyrosine-phosphorylated STATs dimerize and are transported into the nucleus to function as transcription factors. Signaling is attenuated by specific suppressor of cytokine signaling...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36251745"
                        open
                      >
                        <summary class="article-expander__title">Distinct clonal identities of B-ALLs arising after Lenolidomide therapy for multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36251745/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221022213551&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Patients with multiple myeloma (MM) who are treated with lenalidomide rarely develop a secondary B-cell acute lymphoblastic leukemia (B-ALL). The clonal and biological relationship between these sequential malignancies is not yet clear. We identified 17 patients with MM treated with lenalidomide, who subsequently developed B-ALL. Samples were evaluated with sequencing, cytogenetics/FISH, immunohistochemical staining (IHC), and IgH clonality assessment. Samples were assessed for shared mutations...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36253429"
                        open
                      >
                        <summary class="article-expander__title">Molecular landscape of immune pressure and escape in aplastic anemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36253429/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221021213516&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Idiopathic aplastic anemia (IAA) pathophysiology is dominated by autoreactivity of human leukocyte antigen (HLA)-restricted T-cells against antigens presented by hematopoietic stem and progenitor cells (HSPCs). Expansion of PIGA and HLA class I mutant HSPCs have been linked to immune evasion from T-cell mediated pressures. We hypothesized that in analogy with antitumor immunity, the pathophysiological cascade of immune escape in IAA is initiated by immunoediting pressures and culminates with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36252553"
                        open
                      >
                        <summary class="article-expander__title">JAK: Not Just Another Kinase</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36252553/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221021213516&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The Janus kinase/signal transducer and activator of transcription (JAK/STAT) axis is implicated in cancer, inflammation, and immunity.Numerous cytokines/growth factors affect JAK/STAT signaling. JAKs noncovalently associate with cytokine receptors, mediate receptor tyrosine phosphorylation, and recruit STAT proteins.Tyrosine-phosphorylated STATs dimerize and are transported into the nucleus to function as transcription factors. Signaling is attenuated by specific suppressor of cytokine signaling...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36251745"
                        open
                      >
                        <summary class="article-expander__title">Distinct clonal identities of B-ALLs arising after Lenolidomide therapy for multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36251745/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221021213516&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Patients with multiple myeloma (MM) who are treated with lenalidomide rarely develop a secondary B-cell acute lymphoblastic leukemia (B-ALL). The clonal and biological relationship between these sequential malignancies is not yet clear. We identified 17 patients with MM treated with lenalidomide, who subsequently developed B-ALL. Samples were evaluated with sequencing, cytogenetics/FISH, immunohistochemical staining (IHC), and IgH clonality assessment. Samples were assessed for shared mutations...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36253429"
                        open
                      >
                        <summary class="article-expander__title">Molecular landscape of immune pressure and escape in aplastic anemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36253429/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221020212123&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Idiopathic aplastic anemia (IAA) pathophysiology is dominated by autoreactivity of human leukocyte antigen (HLA)-restricted T-cells against antigens presented by hematopoietic stem and progenitor cells (HSPCs). Expansion of PIGA and HLA class I mutant HSPCs have been linked to immune evasion from T-cell mediated pressures. We hypothesized that in analogy with antitumor immunity, the pathophysiological cascade of immune escape in IAA is initiated by immunoediting pressures and culminates with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36252553"
                        open
                      >
                        <summary class="article-expander__title">JAK: Not Just Another Kinase</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36252553/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221020212123&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The Janus kinase/signal transducer and activator of transcription (JAK/STAT) axis is implicated in cancer, inflammation, and immunity.Numerous cytokines/growth factors affect JAK/STAT signaling. JAKs noncovalently associate with cytokine receptors, mediate receptor tyrosine phosphorylation, and recruit STAT proteins.Tyrosine-phosphorylated STATs dimerize and are transported into the nucleus to function as transcription factors. Signaling is attenuated by specific suppressor of cytokine signaling...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36251745"
                        open
                      >
                        <summary class="article-expander__title">Distinct clonal identities of B-ALLs arising after Lenolidomide therapy for multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36251745/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221020212123&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Patients with multiple myeloma (MM) who are treated with lenalidomide rarely develop a secondary B-cell acute lymphoblastic leukemia (B-ALL). The clonal and biological relationship between these sequential malignancies is not yet clear. We identified 17 patients with MM treated with lenalidomide, who subsequently developed B-ALL. Samples were evaluated with sequencing, cytogenetics/FISH, immunohistochemical staining (IHC), and IgH clonality assessment. Samples were assessed for shared mutations...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36253429"
                        open
                      >
                        <summary class="article-expander__title">Molecular landscape of immune pressure and escape in aplastic anemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36253429/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221019212642&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Idiopathic aplastic anemia (IAA) pathophysiology is dominated by autoreactivity of human leukocyte antigen (HLA)-restricted T-cells against antigens presented by hematopoietic stem and progenitor cells (HSPCs). Expansion of PIGA and HLA class I mutant HSPCs have been linked to immune evasion from T-cell mediated pressures. We hypothesized that in analogy with antitumor immunity, the pathophysiological cascade of immune escape in IAA is initiated by immunoediting pressures and culminates with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36252553"
                        open
                      >
                        <summary class="article-expander__title">JAK: Not Just Another Kinase</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36252553/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221019212642&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The Janus kinase/signal transducer and activator of transcription (JAK/STAT) axis is implicated in cancer, inflammation, and immunity.Numerous cytokines/growth factors affect JAK/STAT signaling. JAKs noncovalently associate with cytokine receptors, mediate receptor tyrosine phosphorylation, and recruit STAT proteins.Tyrosine-phosphorylated STATs dimerize and are transported into the nucleus to function as transcription factors. Signaling is attenuated by specific suppressor of cytokine signaling...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36251745"
                        open
                      >
                        <summary class="article-expander__title">Distinct clonal identities of B-ALLs arising after Lenolidomide therapy for multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36251745/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221019212642&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Patients with multiple myeloma (MM) who are treated with lenalidomide rarely develop a secondary B-cell acute lymphoblastic leukemia (B-ALL). The clonal and biological relationship between these sequential malignancies is not yet clear. We identified 17 patients with MM treated with lenalidomide, who subsequently developed B-ALL. Samples were evaluated with sequencing, cytogenetics/FISH, immunohistochemical staining (IHC), and IgH clonality assessment. Samples were assessed for shared mutations...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36253429"
                        open
                      >
                        <summary class="article-expander__title">Molecular landscape of immune pressure and escape in aplastic anemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36253429/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221018213749&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Idiopathic aplastic anemia (IAA) pathophysiology is dominated by autoreactivity of human leukocyte antigen (HLA)-restricted T-cells against antigens presented by hematopoietic stem and progenitor cells (HSPCs). Expansion of PIGA and HLA class I mutant HSPCs have been linked to immune evasion from T-cell mediated pressures. We hypothesized that in analogy with antitumor immunity, the pathophysiological cascade of immune escape in IAA is initiated by immunoediting pressures and culminates with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36252553"
                        open
                      >
                        <summary class="article-expander__title">JAK: Not Just Another Kinase</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36252553/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221018213749&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The Janus kinase/signal transducer and activator of transcription (JAK/STAT) axis is implicated in cancer, inflammation, and immunity.Numerous cytokines/growth factors affect JAK/STAT signaling. JAKs noncovalently associate with cytokine receptors, mediate receptor tyrosine phosphorylation, and recruit STAT proteins.Tyrosine-phosphorylated STATs dimerize and are transported into the nucleus to function as transcription factors. Signaling is attenuated by specific suppressor of cytokine signaling...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36251745"
                        open
                      >
                        <summary class="article-expander__title">Distinct clonal identities of B-ALLs arising after Lenolidomide therapy for multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36251745/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221018213749&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Patients with multiple myeloma (MM) who are treated with lenalidomide rarely develop a secondary B-cell acute lymphoblastic leukemia (B-ALL). The clonal and biological relationship between these sequential malignancies is not yet clear. We identified 17 patients with MM treated with lenalidomide, who subsequently developed B-ALL. Samples were evaluated with sequencing, cytogenetics/FISH, immunohistochemical staining (IHC), and IgH clonality assessment. Samples were assessed for shared mutations...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36251745"
                        open
                      >
                        <summary class="article-expander__title">Distinct clonal identities of B-ALLs arising after Lenolidomide therapy for multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36251745/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221017214255&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Patients with multiple myeloma (MM) who are treated with lenalidomide rarely develop a secondary B-cell acute lymphoblastic leukemia (B-ALL). The clonal and biological relationship between these sequential malignancies is not yet clear. We identified 17 patients with MM treated with lenalidomide, who subsequently developed B-ALL. Samples were evaluated with sequencing, cytogenetics/FISH, immunohistochemical staining (IHC), and IgH clonality assessment. Samples were assessed for shared mutations...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221128200439&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.8">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36252999"
                        open
                      >
                        <summary class="article-expander__title">IgE type multiple myeloma exhibits hypermutated phenotype and tumor reactive T cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36252999/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221103212023&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a hematological malignancy originating from malignant and clonally expanding plasma cells. MM can be molecularly stratified, and its clonal evolution deciphered based on the Ig heavy and light chains of the respective malignant plasma cell clone. Of all MM subtypes, IgE type MM accounts for only &lt;0.1% of cases and is associated with an aggressive clinical course and consequentially dismal prognosis. In such malignancies, adoptive transfer of autologous lymphocytes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36252999"
                        open
                      >
                        <summary class="article-expander__title">IgE type multiple myeloma exhibits hypermutated phenotype and tumor reactive T cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36252999/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221102211743&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a hematological malignancy originating from malignant and clonally expanding plasma cells. MM can be molecularly stratified, and its clonal evolution deciphered based on the Ig heavy and light chains of the respective malignant plasma cell clone. Of all MM subtypes, IgE type MM accounts for only &lt;0.1% of cases and is associated with an aggressive clinical course and consequentially dismal prognosis. In such malignancies, adoptive transfer of autologous lymphocytes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36252999"
                        open
                      >
                        <summary class="article-expander__title">IgE type multiple myeloma exhibits hypermutated phenotype and tumor reactive T cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36252999/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221101212552&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a hematological malignancy originating from malignant and clonally expanding plasma cells. MM can be molecularly stratified, and its clonal evolution deciphered based on the Ig heavy and light chains of the respective malignant plasma cell clone. Of all MM subtypes, IgE type MM accounts for only &lt;0.1% of cases and is associated with an aggressive clinical course and consequentially dismal prognosis. In such malignancies, adoptive transfer of autologous lymphocytes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36252999"
                        open
                      >
                        <summary class="article-expander__title">IgE type multiple myeloma exhibits hypermutated phenotype and tumor reactive T cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36252999/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221031214247&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a hematological malignancy originating from malignant and clonally expanding plasma cells. MM can be molecularly stratified, and its clonal evolution deciphered based on the Ig heavy and light chains of the respective malignant plasma cell clone. Of all MM subtypes, IgE type MM accounts for only &lt;0.1% of cases and is associated with an aggressive clinical course and consequentially dismal prognosis. In such malignancies, adoptive transfer of autologous lymphocytes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36252999"
                        open
                      >
                        <summary class="article-expander__title">IgE type multiple myeloma exhibits hypermutated phenotype and tumor reactive T cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36252999/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221030212817&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a hematological malignancy originating from malignant and clonally expanding plasma cells. MM can be molecularly stratified, and its clonal evolution deciphered based on the Ig heavy and light chains of the respective malignant plasma cell clone. Of all MM subtypes, IgE type MM accounts for only &lt;0.1% of cases and is associated with an aggressive clinical course and consequentially dismal prognosis. In such malignancies, adoptive transfer of autologous lymphocytes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36252999"
                        open
                      >
                        <summary class="article-expander__title">IgE type multiple myeloma exhibits hypermutated phenotype and tumor reactive T cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36252999/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221029212729&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a hematological malignancy originating from malignant and clonally expanding plasma cells. MM can be molecularly stratified, and its clonal evolution deciphered based on the Ig heavy and light chains of the respective malignant plasma cell clone. Of all MM subtypes, IgE type MM accounts for only &lt;0.1% of cases and is associated with an aggressive clinical course and consequentially dismal prognosis. In such malignancies, adoptive transfer of autologous lymphocytes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36252999"
                        open
                      >
                        <summary class="article-expander__title">IgE type multiple myeloma exhibits hypermutated phenotype and tumor reactive T cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36252999/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221028211238&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a hematological malignancy originating from malignant and clonally expanding plasma cells. MM can be molecularly stratified, and its clonal evolution deciphered based on the Ig heavy and light chains of the respective malignant plasma cell clone. Of all MM subtypes, IgE type MM accounts for only &lt;0.1% of cases and is associated with an aggressive clinical course and consequentially dismal prognosis. In such malignancies, adoptive transfer of autologous lymphocytes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36252999"
                        open
                      >
                        <summary class="article-expander__title">IgE type multiple myeloma exhibits hypermutated phenotype and tumor reactive T cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36252999/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221027212535&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a hematological malignancy originating from malignant and clonally expanding plasma cells. MM can be molecularly stratified, and its clonal evolution deciphered based on the Ig heavy and light chains of the respective malignant plasma cell clone. Of all MM subtypes, IgE type MM accounts for only &lt;0.1% of cases and is associated with an aggressive clinical course and consequentially dismal prognosis. In such malignancies, adoptive transfer of autologous lymphocytes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36252999"
                        open
                      >
                        <summary class="article-expander__title">IgE type multiple myeloma exhibits hypermutated phenotype and tumor reactive T cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36252999/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221026211957&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a hematological malignancy originating from malignant and clonally expanding plasma cells. MM can be molecularly stratified, and its clonal evolution deciphered based on the Ig heavy and light chains of the respective malignant plasma cell clone. Of all MM subtypes, IgE type MM accounts for only &lt;0.1% of cases and is associated with an aggressive clinical course and consequentially dismal prognosis. In such malignancies, adoptive transfer of autologous lymphocytes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36252999"
                        open
                      >
                        <summary class="article-expander__title">IgE type multiple myeloma exhibits hypermutated phenotype and tumor reactive T cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36252999/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221025212118&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a hematological malignancy originating from malignant and clonally expanding plasma cells. MM can be molecularly stratified, and its clonal evolution deciphered based on the Ig heavy and light chains of the respective malignant plasma cell clone. Of all MM subtypes, IgE type MM accounts for only &lt;0.1% of cases and is associated with an aggressive clinical course and consequentially dismal prognosis. In such malignancies, adoptive transfer of autologous lymphocytes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36252999"
                        open
                      >
                        <summary class="article-expander__title">IgE type multiple myeloma exhibits hypermutated phenotype and tumor reactive T cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36252999/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221024214453&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a hematological malignancy originating from malignant and clonally expanding plasma cells. MM can be molecularly stratified, and its clonal evolution deciphered based on the Ig heavy and light chains of the respective malignant plasma cell clone. Of all MM subtypes, IgE type MM accounts for only &lt;0.1% of cases and is associated with an aggressive clinical course and consequentially dismal prognosis. In such malignancies, adoptive transfer of autologous lymphocytes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36252999"
                        open
                      >
                        <summary class="article-expander__title">IgE type multiple myeloma exhibits hypermutated phenotype and tumor reactive T cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36252999/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221023215029&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a hematological malignancy originating from malignant and clonally expanding plasma cells. MM can be molecularly stratified, and its clonal evolution deciphered based on the Ig heavy and light chains of the respective malignant plasma cell clone. Of all MM subtypes, IgE type MM accounts for only &lt;0.1% of cases and is associated with an aggressive clinical course and consequentially dismal prognosis. In such malignancies, adoptive transfer of autologous lymphocytes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36252999"
                        open
                      >
                        <summary class="article-expander__title">IgE type multiple myeloma exhibits hypermutated phenotype and tumor reactive T cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36252999/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221022213551&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a hematological malignancy originating from malignant and clonally expanding plasma cells. MM can be molecularly stratified, and its clonal evolution deciphered based on the Ig heavy and light chains of the respective malignant plasma cell clone. Of all MM subtypes, IgE type MM accounts for only &lt;0.1% of cases and is associated with an aggressive clinical course and consequentially dismal prognosis. In such malignancies, adoptive transfer of autologous lymphocytes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36252999"
                        open
                      >
                        <summary class="article-expander__title">IgE type multiple myeloma exhibits hypermutated phenotype and tumor reactive T cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36252999/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221021213516&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a hematological malignancy originating from malignant and clonally expanding plasma cells. MM can be molecularly stratified, and its clonal evolution deciphered based on the Ig heavy and light chains of the respective malignant plasma cell clone. Of all MM subtypes, IgE type MM accounts for only &lt;0.1% of cases and is associated with an aggressive clinical course and consequentially dismal prognosis. In such malignancies, adoptive transfer of autologous lymphocytes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36252999"
                        open
                      >
                        <summary class="article-expander__title">IgE type multiple myeloma exhibits hypermutated phenotype and tumor reactive T cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36252999/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221020212123&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a hematological malignancy originating from malignant and clonally expanding plasma cells. MM can be molecularly stratified, and its clonal evolution deciphered based on the Ig heavy and light chains of the respective malignant plasma cell clone. Of all MM subtypes, IgE type MM accounts for only &lt;0.1% of cases and is associated with an aggressive clinical course and consequentially dismal prognosis. In such malignancies, adoptive transfer of autologous lymphocytes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36252999"
                        open
                      >
                        <summary class="article-expander__title">IgE type multiple myeloma exhibits hypermutated phenotype and tumor reactive T cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36252999/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221019212642&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a hematological malignancy originating from malignant and clonally expanding plasma cells. MM can be molecularly stratified, and its clonal evolution deciphered based on the Ig heavy and light chains of the respective malignant plasma cell clone. Of all MM subtypes, IgE type MM accounts for only &lt;0.1% of cases and is associated with an aggressive clinical course and consequentially dismal prognosis. In such malignancies, adoptive transfer of autologous lymphocytes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36252999"
                        open
                      >
                        <summary class="article-expander__title">IgE type multiple myeloma exhibits hypermutated phenotype and tumor reactive T cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36252999/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221018213749&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a hematological malignancy originating from malignant and clonally expanding plasma cells. MM can be molecularly stratified, and its clonal evolution deciphered based on the Ig heavy and light chains of the respective malignant plasma cell clone. Of all MM subtypes, IgE type MM accounts for only &lt;0.1% of cases and is associated with an aggressive clinical course and consequentially dismal prognosis. In such malignancies, adoptive transfer of autologous lymphocytes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-14 datetime="2022-10-14T10:00:00.000Z">2022-10-14</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-14 datetime="2022-10-14T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.8&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20221128200439&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36241679"
                        open
                      >
                        <summary class="article-expander__title">Massive expansion of multiple clones in the mouse hematopoietic system long after whole-body X-irradiation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36241679/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221101212552&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is prevalent in the elderly and associates with hematologic malignancy and cardiovascular disease. Although the risk of developing these diseases increases with radiation doses in atomic-bomb survivors, the causal relationship between radiation exposure and CH is unclear. This study investigated whether radiation exposure induces CH in mice 12-18 months after 3-Gy whole-body irradiation. We found radiation-associated increases in peripheral blood myeloid cells and red...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36241679"
                        open
                      >
                        <summary class="article-expander__title">Massive expansion of multiple clones in the mouse hematopoietic system long after whole-body X-irradiation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36241679/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221031214247&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is prevalent in the elderly and associates with hematologic malignancy and cardiovascular disease. Although the risk of developing these diseases increases with radiation doses in atomic-bomb survivors, the causal relationship between radiation exposure and CH is unclear. This study investigated whether radiation exposure induces CH in mice 12-18 months after 3-Gy whole-body irradiation. We found radiation-associated increases in peripheral blood myeloid cells and red...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36241679"
                        open
                      >
                        <summary class="article-expander__title">Massive expansion of multiple clones in the mouse hematopoietic system long after whole-body X-irradiation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36241679/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221030212817&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is prevalent in the elderly and associates with hematologic malignancy and cardiovascular disease. Although the risk of developing these diseases increases with radiation doses in atomic-bomb survivors, the causal relationship between radiation exposure and CH is unclear. This study investigated whether radiation exposure induces CH in mice 12-18 months after 3-Gy whole-body irradiation. We found radiation-associated increases in peripheral blood myeloid cells and red...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36241009"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential and Kidney Function Decline in the General Population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36241009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221030212817&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We report an association between CHIP and kidney function decline in three general population cohorts without known kidney disease. Further studies are needed to investigate this novel condition and its potential impact among individuals with overt kidney disease.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36241679"
                        open
                      >
                        <summary class="article-expander__title">Massive expansion of multiple clones in the mouse hematopoietic system long after whole-body X-irradiation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36241679/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221029212729&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is prevalent in the elderly and associates with hematologic malignancy and cardiovascular disease. Although the risk of developing these diseases increases with radiation doses in atomic-bomb survivors, the causal relationship between radiation exposure and CH is unclear. This study investigated whether radiation exposure induces CH in mice 12-18 months after 3-Gy whole-body irradiation. We found radiation-associated increases in peripheral blood myeloid cells and red...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36241009"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential and Kidney Function Decline in the General Population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36241009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221029212729&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We report an association between CHIP and kidney function decline in three general population cohorts without known kidney disease. Further studies are needed to investigate this novel condition and its potential impact among individuals with overt kidney disease.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36241679"
                        open
                      >
                        <summary class="article-expander__title">Massive expansion of multiple clones in the mouse hematopoietic system long after whole-body X-irradiation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36241679/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221028211238&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is prevalent in the elderly and associates with hematologic malignancy and cardiovascular disease. Although the risk of developing these diseases increases with radiation doses in atomic-bomb survivors, the causal relationship between radiation exposure and CH is unclear. This study investigated whether radiation exposure induces CH in mice 12-18 months after 3-Gy whole-body irradiation. We found radiation-associated increases in peripheral blood myeloid cells and red...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36241009"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential and Kidney Function Decline in the General Population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36241009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221028211238&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We report an association between CHIP and kidney function decline in three general population cohorts without known kidney disease. Further studies are needed to investigate this novel condition and its potential impact among individuals with overt kidney disease.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36241679"
                        open
                      >
                        <summary class="article-expander__title">Massive expansion of multiple clones in the mouse hematopoietic system long after whole-body X-irradiation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36241679/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221027212537&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is prevalent in the elderly and associates with hematologic malignancy and cardiovascular disease. Although the risk of developing these diseases increases with radiation doses in atomic-bomb survivors, the causal relationship between radiation exposure and CH is unclear. This study investigated whether radiation exposure induces CH in mice 12-18 months after 3-Gy whole-body irradiation. We found radiation-associated increases in peripheral blood myeloid cells and red...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36241009"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential and Kidney Function Decline in the General Population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36241009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221027212537&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We report an association between CHIP and kidney function decline in three general population cohorts without known kidney disease. Further studies are needed to investigate this novel condition and its potential impact among individuals with overt kidney disease.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36241679"
                        open
                      >
                        <summary class="article-expander__title">Massive expansion of multiple clones in the mouse hematopoietic system long after whole-body X-irradiation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36241679/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221026211957&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is prevalent in the elderly and associates with hematologic malignancy and cardiovascular disease. Although the risk of developing these diseases increases with radiation doses in atomic-bomb survivors, the causal relationship between radiation exposure and CH is unclear. This study investigated whether radiation exposure induces CH in mice 12-18 months after 3-Gy whole-body irradiation. We found radiation-associated increases in peripheral blood myeloid cells and red...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36241009"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential and Kidney Function Decline in the General Population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36241009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221026211957&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We report an association between CHIP and kidney function decline in three general population cohorts without known kidney disease. Further studies are needed to investigate this novel condition and its potential impact among individuals with overt kidney disease.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36241679"
                        open
                      >
                        <summary class="article-expander__title">Massive expansion of multiple clones in the mouse hematopoietic system long after whole-body X-irradiation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36241679/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221025212118&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is prevalent in the elderly and associates with hematologic malignancy and cardiovascular disease. Although the risk of developing these diseases increases with radiation doses in atomic-bomb survivors, the causal relationship between radiation exposure and CH is unclear. This study investigated whether radiation exposure induces CH in mice 12-18 months after 3-Gy whole-body irradiation. We found radiation-associated increases in peripheral blood myeloid cells and red...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36241009"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential and Kidney Function Decline in the General Population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36241009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221025212118&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We report an association between CHIP and kidney function decline in three general population cohorts without known kidney disease. Further studies are needed to investigate this novel condition and its potential impact among individuals with overt kidney disease.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36241679"
                        open
                      >
                        <summary class="article-expander__title">Massive expansion of multiple clones in the mouse hematopoietic system long after whole-body X-irradiation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36241679/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221024214453&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is prevalent in the elderly and associates with hematologic malignancy and cardiovascular disease. Although the risk of developing these diseases increases with radiation doses in atomic-bomb survivors, the causal relationship between radiation exposure and CH is unclear. This study investigated whether radiation exposure induces CH in mice 12-18 months after 3-Gy whole-body irradiation. We found radiation-associated increases in peripheral blood myeloid cells and red...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36241009"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential and Kidney Function Decline in the General Population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36241009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221024214453&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We report an association between CHIP and kidney function decline in three general population cohorts without known kidney disease. Further studies are needed to investigate this novel condition and its potential impact among individuals with overt kidney disease.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36241679"
                        open
                      >
                        <summary class="article-expander__title">Massive expansion of multiple clones in the mouse hematopoietic system long after whole-body X-irradiation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36241679/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221023215029&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is prevalent in the elderly and associates with hematologic malignancy and cardiovascular disease. Although the risk of developing these diseases increases with radiation doses in atomic-bomb survivors, the causal relationship between radiation exposure and CH is unclear. This study investigated whether radiation exposure induces CH in mice 12-18 months after 3-Gy whole-body irradiation. We found radiation-associated increases in peripheral blood myeloid cells and red...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36241009"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential and Kidney Function Decline in the General Population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36241009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221023215029&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We report an association between CHIP and kidney function decline in three general population cohorts without known kidney disease. Further studies are needed to investigate this novel condition and its potential impact among individuals with overt kidney disease.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36241679"
                        open
                      >
                        <summary class="article-expander__title">Massive expansion of multiple clones in the mouse hematopoietic system long after whole-body X-irradiation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36241679/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221022213551&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is prevalent in the elderly and associates with hematologic malignancy and cardiovascular disease. Although the risk of developing these diseases increases with radiation doses in atomic-bomb survivors, the causal relationship between radiation exposure and CH is unclear. This study investigated whether radiation exposure induces CH in mice 12-18 months after 3-Gy whole-body irradiation. We found radiation-associated increases in peripheral blood myeloid cells and red...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36241009"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential and Kidney Function Decline in the General Population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36241009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221022213551&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We report an association between CHIP and kidney function decline in three general population cohorts without known kidney disease. Further studies are needed to investigate this novel condition and its potential impact among individuals with overt kidney disease.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36241679"
                        open
                      >
                        <summary class="article-expander__title">Massive expansion of multiple clones in the mouse hematopoietic system long after whole-body X-irradiation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36241679/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221021213516&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is prevalent in the elderly and associates with hematologic malignancy and cardiovascular disease. Although the risk of developing these diseases increases with radiation doses in atomic-bomb survivors, the causal relationship between radiation exposure and CH is unclear. This study investigated whether radiation exposure induces CH in mice 12-18 months after 3-Gy whole-body irradiation. We found radiation-associated increases in peripheral blood myeloid cells and red...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36241009"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential and Kidney Function Decline in the General Population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36241009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221021213516&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We report an association between CHIP and kidney function decline in three general population cohorts without known kidney disease. Further studies are needed to investigate this novel condition and its potential impact among individuals with overt kidney disease.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36241679"
                        open
                      >
                        <summary class="article-expander__title">Massive expansion of multiple clones in the mouse hematopoietic system long after whole-body X-irradiation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36241679/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221020212123&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is prevalent in the elderly and associates with hematologic malignancy and cardiovascular disease. Although the risk of developing these diseases increases with radiation doses in atomic-bomb survivors, the causal relationship between radiation exposure and CH is unclear. This study investigated whether radiation exposure induces CH in mice 12-18 months after 3-Gy whole-body irradiation. We found radiation-associated increases in peripheral blood myeloid cells and red...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36241009"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential and Kidney Function Decline in the General Population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36241009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221020212123&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We report an association between CHIP and kidney function decline in three general population cohorts without known kidney disease. Further studies are needed to investigate this novel condition and its potential impact among individuals with overt kidney disease.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36241679"
                        open
                      >
                        <summary class="article-expander__title">Massive expansion of multiple clones in the mouse hematopoietic system long after whole-body X-irradiation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36241679/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221019212642&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is prevalent in the elderly and associates with hematologic malignancy and cardiovascular disease. Although the risk of developing these diseases increases with radiation doses in atomic-bomb survivors, the causal relationship between radiation exposure and CH is unclear. This study investigated whether radiation exposure induces CH in mice 12-18 months after 3-Gy whole-body irradiation. We found radiation-associated increases in peripheral blood myeloid cells and red...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36241009"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential and Kidney Function Decline in the General Population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36241009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221019212642&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We report an association between CHIP and kidney function decline in three general population cohorts without known kidney disease. Further studies are needed to investigate this novel condition and its potential impact among individuals with overt kidney disease.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36241679"
                        open
                      >
                        <summary class="article-expander__title">Massive expansion of multiple clones in the mouse hematopoietic system long after whole-body X-irradiation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36241679/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221018213749&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is prevalent in the elderly and associates with hematologic malignancy and cardiovascular disease. Although the risk of developing these diseases increases with radiation doses in atomic-bomb survivors, the causal relationship between radiation exposure and CH is unclear. This study investigated whether radiation exposure induces CH in mice 12-18 months after 3-Gy whole-body irradiation. We found radiation-associated increases in peripheral blood myeloid cells and red...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36241009"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential and Kidney Function Decline in the General Population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36241009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221018213749&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We report an association between CHIP and kidney function decline in three general population cohorts without known kidney disease. Further studies are needed to investigate this novel condition and its potential impact among individuals with overt kidney disease.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36241679"
                        open
                      >
                        <summary class="article-expander__title">Massive expansion of multiple clones in the mouse hematopoietic system long after whole-body X-irradiation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36241679/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221017214255&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is prevalent in the elderly and associates with hematologic malignancy and cardiovascular disease. Although the risk of developing these diseases increases with radiation doses in atomic-bomb survivors, the causal relationship between radiation exposure and CH is unclear. This study investigated whether radiation exposure induces CH in mice 12-18 months after 3-Gy whole-body irradiation. We found radiation-associated increases in peripheral blood myeloid cells and red...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36241009"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential and Kidney Function Decline in the General Population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36241009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221017214255&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We report an association between CHIP and kidney function decline in three general population cohorts without known kidney disease. Further studies are needed to investigate this novel condition and its potential impact among individuals with overt kidney disease.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36241679"
                        open
                      >
                        <summary class="article-expander__title">Massive expansion of multiple clones in the mouse hematopoietic system long after whole-body X-irradiation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36241679/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221016214939&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is prevalent in the elderly and associates with hematologic malignancy and cardiovascular disease. Although the risk of developing these diseases increases with radiation doses in atomic-bomb survivors, the causal relationship between radiation exposure and CH is unclear. This study investigated whether radiation exposure induces CH in mice 12-18 months after 3-Gy whole-body irradiation. We found radiation-associated increases in peripheral blood myeloid cells and red...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36241009"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential and Kidney Function Decline in the General Population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36241009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221016214939&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We report an association between CHIP and kidney function decline in three general population cohorts without known kidney disease. Further studies are needed to investigate this novel condition and its potential impact among individuals with overt kidney disease.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36241679"
                        open
                      >
                        <summary class="article-expander__title">Massive expansion of multiple clones in the mouse hematopoietic system long after whole-body X-irradiation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36241679/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221015214355&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is prevalent in the elderly and associates with hematologic malignancy and cardiovascular disease. Although the risk of developing these diseases increases with radiation doses in atomic-bomb survivors, the causal relationship between radiation exposure and CH is unclear. This study investigated whether radiation exposure induces CH in mice 12-18 months after 3-Gy whole-body irradiation. We found radiation-associated increases in peripheral blood myeloid cells and red...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36241009"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential and Kidney Function Decline in the General Population</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36241009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221015214355&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We report an association between CHIP and kidney function decline in three general population cohorts without known kidney disease. Further studies are needed to investigate this novel condition and its potential impact among individuals with overt kidney disease.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221128200439&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.8">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36240778"
                        open
                      >
                        <summary class="article-expander__title">Spatial epitope barcoding reveals clonal tumor patch behaviors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36240778/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221102211743&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are still largely unsuitable to accurately track phenotypes and clonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitopes for imaging using combinatorial tagging (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36240778"
                        open
                      >
                        <summary class="article-expander__title">Spatial epitope barcoding reveals clonal tumor patch behaviors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36240778/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221101212552&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are still largely unsuitable to accurately track phenotypes and clonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitopes for imaging using combinatorial tagging (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36230551"
                        open
                      >
                        <summary class="article-expander__title">The Clonal Diversity of Peripheral B Cell Receptor Immune Repertoire Impaired by Residual Malignant B Cells Predicts Treatment Efficacy in B Cell Lymphoma Patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36230551/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221101212552&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germinal center (GC) is the vital locus for the evolution of nave B cells into memory B and plasma cells, but also a hotbed for the proliferation of malignant B cells. We hypothesized that malignant B cells may locally or globally impact GCs to produce peripheral B cell receptor immune repertoire (BCR IR) with reduced clonal diversity. In this study, we first validated our hypothesis in a novel human in-vitro GC (hiGC) model. The addition of the diffuse large B cell lymphoma (DLBCL) cells to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36240778"
                        open
                      >
                        <summary class="article-expander__title">Spatial epitope barcoding reveals clonal tumor patch behaviors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36240778/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221031214247&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are still largely unsuitable to accurately track phenotypes and clonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitopes for imaging using combinatorial tagging (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36230551"
                        open
                      >
                        <summary class="article-expander__title">The Clonal Diversity of Peripheral B Cell Receptor Immune Repertoire Impaired by Residual Malignant B Cells Predicts Treatment Efficacy in B Cell Lymphoma Patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36230551/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221031214247&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germinal center (GC) is the vital locus for the evolution of nave B cells into memory B and plasma cells, but also a hotbed for the proliferation of malignant B cells. We hypothesized that malignant B cells may locally or globally impact GCs to produce peripheral B cell receptor immune repertoire (BCR IR) with reduced clonal diversity. In this study, we first validated our hypothesis in a novel human in-vitro GC (hiGC) model. The addition of the diffuse large B cell lymphoma (DLBCL) cells to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36240778"
                        open
                      >
                        <summary class="article-expander__title">Spatial epitope barcoding reveals clonal tumor patch behaviors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36240778/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221030212817&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are still largely unsuitable to accurately track phenotypes and clonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitopes for imaging using combinatorial tagging (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36230551"
                        open
                      >
                        <summary class="article-expander__title">The Clonal Diversity of Peripheral B Cell Receptor Immune Repertoire Impaired by Residual Malignant B Cells Predicts Treatment Efficacy in B Cell Lymphoma Patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36230551/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221030212817&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germinal center (GC) is the vital locus for the evolution of nave B cells into memory B and plasma cells, but also a hotbed for the proliferation of malignant B cells. We hypothesized that malignant B cells may locally or globally impact GCs to produce peripheral B cell receptor immune repertoire (BCR IR) with reduced clonal diversity. In this study, we first validated our hypothesis in a novel human in-vitro GC (hiGC) model. The addition of the diffuse large B cell lymphoma (DLBCL) cells to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36240778"
                        open
                      >
                        <summary class="article-expander__title">Spatial epitope barcoding reveals clonal tumor patch behaviors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36240778/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221029212729&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are still largely unsuitable to accurately track phenotypes and clonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitopes for imaging using combinatorial tagging (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36230551"
                        open
                      >
                        <summary class="article-expander__title">The Clonal Diversity of Peripheral B Cell Receptor Immune Repertoire Impaired by Residual Malignant B Cells Predicts Treatment Efficacy in B Cell Lymphoma Patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36230551/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221029212729&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germinal center (GC) is the vital locus for the evolution of nave B cells into memory B and plasma cells, but also a hotbed for the proliferation of malignant B cells. We hypothesized that malignant B cells may locally or globally impact GCs to produce peripheral B cell receptor immune repertoire (BCR IR) with reduced clonal diversity. In this study, we first validated our hypothesis in a novel human in-vitro GC (hiGC) model. The addition of the diffuse large B cell lymphoma (DLBCL) cells to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36240778"
                        open
                      >
                        <summary class="article-expander__title">Spatial epitope barcoding reveals clonal tumor patch behaviors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36240778/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221028211238&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are still largely unsuitable to accurately track phenotypes and clonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitopes for imaging using combinatorial tagging (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36230551"
                        open
                      >
                        <summary class="article-expander__title">The Clonal Diversity of Peripheral B Cell Receptor Immune Repertoire Impaired by Residual Malignant B Cells Predicts Treatment Efficacy in B Cell Lymphoma Patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36230551/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221028211238&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germinal center (GC) is the vital locus for the evolution of nave B cells into memory B and plasma cells, but also a hotbed for the proliferation of malignant B cells. We hypothesized that malignant B cells may locally or globally impact GCs to produce peripheral B cell receptor immune repertoire (BCR IR) with reduced clonal diversity. In this study, we first validated our hypothesis in a novel human in-vitro GC (hiGC) model. The addition of the diffuse large B cell lymphoma (DLBCL) cells to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36240778"
                        open
                      >
                        <summary class="article-expander__title">Spatial epitope barcoding reveals clonal tumor patch behaviors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36240778/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221027212535&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are still largely unsuitable to accurately track phenotypes and clonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitopes for imaging using combinatorial tagging (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36230551"
                        open
                      >
                        <summary class="article-expander__title">The Clonal Diversity of Peripheral B Cell Receptor Immune Repertoire Impaired by Residual Malignant B Cells Predicts Treatment Efficacy in B Cell Lymphoma Patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36230551/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221027212535&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germinal center (GC) is the vital locus for the evolution of nave B cells into memory B and plasma cells, but also a hotbed for the proliferation of malignant B cells. We hypothesized that malignant B cells may locally or globally impact GCs to produce peripheral B cell receptor immune repertoire (BCR IR) with reduced clonal diversity. In this study, we first validated our hypothesis in a novel human in-vitro GC (hiGC) model. The addition of the diffuse large B cell lymphoma (DLBCL) cells to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36240778"
                        open
                      >
                        <summary class="article-expander__title">Spatial epitope barcoding reveals clonal tumor patch behaviors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36240778/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221026211957&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are still largely unsuitable to accurately track phenotypes and clonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitopes for imaging using combinatorial tagging (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36230551"
                        open
                      >
                        <summary class="article-expander__title">The Clonal Diversity of Peripheral B Cell Receptor Immune Repertoire Impaired by Residual Malignant B Cells Predicts Treatment Efficacy in B Cell Lymphoma Patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36230551/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221026211957&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germinal center (GC) is the vital locus for the evolution of nave B cells into memory B and plasma cells, but also a hotbed for the proliferation of malignant B cells. We hypothesized that malignant B cells may locally or globally impact GCs to produce peripheral B cell receptor immune repertoire (BCR IR) with reduced clonal diversity. In this study, we first validated our hypothesis in a novel human in-vitro GC (hiGC) model. The addition of the diffuse large B cell lymphoma (DLBCL) cells to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36240778"
                        open
                      >
                        <summary class="article-expander__title">Spatial epitope barcoding reveals clonal tumor patch behaviors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36240778/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221025212118&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are still largely unsuitable to accurately track phenotypes and clonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitopes for imaging using combinatorial tagging (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36230551"
                        open
                      >
                        <summary class="article-expander__title">The Clonal Diversity of Peripheral B Cell Receptor Immune Repertoire Impaired by Residual Malignant B Cells Predicts Treatment Efficacy in B Cell Lymphoma Patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36230551/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221025212118&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germinal center (GC) is the vital locus for the evolution of nave B cells into memory B and plasma cells, but also a hotbed for the proliferation of malignant B cells. We hypothesized that malignant B cells may locally or globally impact GCs to produce peripheral B cell receptor immune repertoire (BCR IR) with reduced clonal diversity. In this study, we first validated our hypothesis in a novel human in-vitro GC (hiGC) model. The addition of the diffuse large B cell lymphoma (DLBCL) cells to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36240778"
                        open
                      >
                        <summary class="article-expander__title">Spatial epitope barcoding reveals clonal tumor patch behaviors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36240778/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221024214453&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are still largely unsuitable to accurately track phenotypes and clonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitopes for imaging using combinatorial tagging (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36230551"
                        open
                      >
                        <summary class="article-expander__title">The Clonal Diversity of Peripheral B Cell Receptor Immune Repertoire Impaired by Residual Malignant B Cells Predicts Treatment Efficacy in B Cell Lymphoma Patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36230551/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221024214453&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germinal center (GC) is the vital locus for the evolution of nave B cells into memory B and plasma cells, but also a hotbed for the proliferation of malignant B cells. We hypothesized that malignant B cells may locally or globally impact GCs to produce peripheral B cell receptor immune repertoire (BCR IR) with reduced clonal diversity. In this study, we first validated our hypothesis in a novel human in-vitro GC (hiGC) model. The addition of the diffuse large B cell lymphoma (DLBCL) cells to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36240778"
                        open
                      >
                        <summary class="article-expander__title">Spatial epitope barcoding reveals clonal tumor patch behaviors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36240778/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221023215029&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are still largely unsuitable to accurately track phenotypes and clonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitopes for imaging using combinatorial tagging (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36230551"
                        open
                      >
                        <summary class="article-expander__title">The Clonal Diversity of Peripheral B Cell Receptor Immune Repertoire Impaired by Residual Malignant B Cells Predicts Treatment Efficacy in B Cell Lymphoma Patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36230551/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221023215029&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germinal center (GC) is the vital locus for the evolution of nave B cells into memory B and plasma cells, but also a hotbed for the proliferation of malignant B cells. We hypothesized that malignant B cells may locally or globally impact GCs to produce peripheral B cell receptor immune repertoire (BCR IR) with reduced clonal diversity. In this study, we first validated our hypothesis in a novel human in-vitro GC (hiGC) model. The addition of the diffuse large B cell lymphoma (DLBCL) cells to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36240778"
                        open
                      >
                        <summary class="article-expander__title">Spatial epitope barcoding reveals clonal tumor patch behaviors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36240778/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221022213551&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are still largely unsuitable to accurately track phenotypes and clonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitopes for imaging using combinatorial tagging (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36230551"
                        open
                      >
                        <summary class="article-expander__title">The Clonal Diversity of Peripheral B Cell Receptor Immune Repertoire Impaired by Residual Malignant B Cells Predicts Treatment Efficacy in B Cell Lymphoma Patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36230551/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221022213551&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germinal center (GC) is the vital locus for the evolution of nave B cells into memory B and plasma cells, but also a hotbed for the proliferation of malignant B cells. We hypothesized that malignant B cells may locally or globally impact GCs to produce peripheral B cell receptor immune repertoire (BCR IR) with reduced clonal diversity. In this study, we first validated our hypothesis in a novel human in-vitro GC (hiGC) model. The addition of the diffuse large B cell lymphoma (DLBCL) cells to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36240778"
                        open
                      >
                        <summary class="article-expander__title">Spatial epitope barcoding reveals clonal tumor patch behaviors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36240778/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221021213516&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are still largely unsuitable to accurately track phenotypes and clonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitopes for imaging using combinatorial tagging (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36230551"
                        open
                      >
                        <summary class="article-expander__title">The Clonal Diversity of Peripheral B Cell Receptor Immune Repertoire Impaired by Residual Malignant B Cells Predicts Treatment Efficacy in B Cell Lymphoma Patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36230551/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221021213516&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germinal center (GC) is the vital locus for the evolution of nave B cells into memory B and plasma cells, but also a hotbed for the proliferation of malignant B cells. We hypothesized that malignant B cells may locally or globally impact GCs to produce peripheral B cell receptor immune repertoire (BCR IR) with reduced clonal diversity. In this study, we first validated our hypothesis in a novel human in-vitro GC (hiGC) model. The addition of the diffuse large B cell lymphoma (DLBCL) cells to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36240778"
                        open
                      >
                        <summary class="article-expander__title">Spatial epitope barcoding reveals clonal tumor patch behaviors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36240778/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221020212123&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are still largely unsuitable to accurately track phenotypes and clonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitopes for imaging using combinatorial tagging (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36230551"
                        open
                      >
                        <summary class="article-expander__title">The Clonal Diversity of Peripheral B Cell Receptor Immune Repertoire Impaired by Residual Malignant B Cells Predicts Treatment Efficacy in B Cell Lymphoma Patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36230551/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221020212123&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germinal center (GC) is the vital locus for the evolution of nave B cells into memory B and plasma cells, but also a hotbed for the proliferation of malignant B cells. We hypothesized that malignant B cells may locally or globally impact GCs to produce peripheral B cell receptor immune repertoire (BCR IR) with reduced clonal diversity. In this study, we first validated our hypothesis in a novel human in-vitro GC (hiGC) model. The addition of the diffuse large B cell lymphoma (DLBCL) cells to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36240778"
                        open
                      >
                        <summary class="article-expander__title">Spatial epitope barcoding reveals clonal tumor patch behaviors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36240778/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221019212642&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are still largely unsuitable to accurately track phenotypes and clonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitopes for imaging using combinatorial tagging (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36230551"
                        open
                      >
                        <summary class="article-expander__title">The Clonal Diversity of Peripheral B Cell Receptor Immune Repertoire Impaired by Residual Malignant B Cells Predicts Treatment Efficacy in B Cell Lymphoma Patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36230551/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221019212642&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germinal center (GC) is the vital locus for the evolution of nave B cells into memory B and plasma cells, but also a hotbed for the proliferation of malignant B cells. We hypothesized that malignant B cells may locally or globally impact GCs to produce peripheral B cell receptor immune repertoire (BCR IR) with reduced clonal diversity. In this study, we first validated our hypothesis in a novel human in-vitro GC (hiGC) model. The addition of the diffuse large B cell lymphoma (DLBCL) cells to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36240778"
                        open
                      >
                        <summary class="article-expander__title">Spatial epitope barcoding reveals clonal tumor patch behaviors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36240778/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221018213749&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are still largely unsuitable to accurately track phenotypes and clonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitopes for imaging using combinatorial tagging (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36230551"
                        open
                      >
                        <summary class="article-expander__title">The Clonal Diversity of Peripheral B Cell Receptor Immune Repertoire Impaired by Residual Malignant B Cells Predicts Treatment Efficacy in B Cell Lymphoma Patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36230551/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221018213749&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germinal center (GC) is the vital locus for the evolution of nave B cells into memory B and plasma cells, but also a hotbed for the proliferation of malignant B cells. We hypothesized that malignant B cells may locally or globally impact GCs to produce peripheral B cell receptor immune repertoire (BCR IR) with reduced clonal diversity. In this study, we first validated our hypothesis in a novel human in-vitro GC (hiGC) model. The addition of the diffuse large B cell lymphoma (DLBCL) cells to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36240778"
                        open
                      >
                        <summary class="article-expander__title">Spatial epitope barcoding reveals clonal tumor patch behaviors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36240778/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221017214255&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are still largely unsuitable to accurately track phenotypes and clonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitopes for imaging using combinatorial tagging (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36230551"
                        open
                      >
                        <summary class="article-expander__title">The Clonal Diversity of Peripheral B Cell Receptor Immune Repertoire Impaired by Residual Malignant B Cells Predicts Treatment Efficacy in B Cell Lymphoma Patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36230551/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221017214255&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germinal center (GC) is the vital locus for the evolution of nave B cells into memory B and plasma cells, but also a hotbed for the proliferation of malignant B cells. We hypothesized that malignant B cells may locally or globally impact GCs to produce peripheral B cell receptor immune repertoire (BCR IR) with reduced clonal diversity. In this study, we first validated our hypothesis in a novel human in-vitro GC (hiGC) model. The addition of the diffuse large B cell lymphoma (DLBCL) cells to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36240778"
                        open
                      >
                        <summary class="article-expander__title">Spatial epitope barcoding reveals clonal tumor patch behaviors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36240778/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221016214941&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are still largely unsuitable to accurately track phenotypes and clonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitopes for imaging using combinatorial tagging (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36230551"
                        open
                      >
                        <summary class="article-expander__title">The Clonal Diversity of Peripheral B Cell Receptor Immune Repertoire Impaired by Residual Malignant B Cells Predicts Treatment Efficacy in B Cell Lymphoma Patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36230551/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221016214941&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germinal center (GC) is the vital locus for the evolution of nave B cells into memory B and plasma cells, but also a hotbed for the proliferation of malignant B cells. We hypothesized that malignant B cells may locally or globally impact GCs to produce peripheral B cell receptor immune repertoire (BCR IR) with reduced clonal diversity. In this study, we first validated our hypothesis in a novel human in-vitro GC (hiGC) model. The addition of the diffuse large B cell lymphoma (DLBCL) cells to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36240778"
                        open
                      >
                        <summary class="article-expander__title">Spatial epitope barcoding reveals clonal tumor patch behaviors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36240778/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221015214357&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are still largely unsuitable to accurately track phenotypes and clonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitopes for imaging using combinatorial tagging (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36230551"
                        open
                      >
                        <summary class="article-expander__title">The Clonal Diversity of Peripheral B Cell Receptor Immune Repertoire Impaired by Residual Malignant B Cells Predicts Treatment Efficacy in B Cell Lymphoma Patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36230551/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221015214357&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germinal center (GC) is the vital locus for the evolution of nave B cells into memory B and plasma cells, but also a hotbed for the proliferation of malignant B cells. We hypothesized that malignant B cells may locally or globally impact GCs to produce peripheral B cell receptor immune repertoire (BCR IR) with reduced clonal diversity. In this study, we first validated our hypothesis in a novel human in-vitro GC (hiGC) model. The addition of the diffuse large B cell lymphoma (DLBCL) cells to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36230551"
                        open
                      >
                        <summary class="article-expander__title">The Clonal Diversity of Peripheral B Cell Receptor Immune Repertoire Impaired by Residual Malignant B Cells Predicts Treatment Efficacy in B Cell Lymphoma Patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36230551/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221014214034&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germinal center (GC) is the vital locus for the evolution of nave B cells into memory B and plasma cells, but also a hotbed for the proliferation of malignant B cells. We hypothesized that malignant B cells may locally or globally impact GCs to produce peripheral B cell receptor immune repertoire (BCR IR) with reduced clonal diversity. In this study, we first validated our hypothesis in a novel human in-vitro GC (hiGC) model. The addition of the diffuse large B cell lymphoma (DLBCL) cells to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-13 datetime="2022-10-13T10:00:00.000Z">2022-10-13</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-13 datetime="2022-10-13T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.8&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20221128200439&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36229591"
                        open
                      >
                        <summary class="article-expander__title">Single-cell transcriptome analyses reveal distinct gene expression signatures of severe COVID-19 in the presence of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36229591/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221030212817&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP), a common aging-related process that predisposes individuals to various inflammatory responses, has been reported to be associated with COVID-19 severity. However, the immunological signature and the exact gene expression program by which the presence of CHIP exerts its clinical impact on COVID-19 remain to be elucidated. In this study, we generated a single-cell transcriptome landscape of severe COVID-19 according to the presence of CHIP...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36229591"
                        open
                      >
                        <summary class="article-expander__title">Single-cell transcriptome analyses reveal distinct gene expression signatures of severe COVID-19 in the presence of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36229591/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221029212729&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP), a common aging-related process that predisposes individuals to various inflammatory responses, has been reported to be associated with COVID-19 severity. However, the immunological signature and the exact gene expression program by which the presence of CHIP exerts its clinical impact on COVID-19 remain to be elucidated. In this study, we generated a single-cell transcriptome landscape of severe COVID-19 according to the presence of CHIP...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36229591"
                        open
                      >
                        <summary class="article-expander__title">Single-cell transcriptome analyses reveal distinct gene expression signatures of severe COVID-19 in the presence of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36229591/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221028211238&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP), a common aging-related process that predisposes individuals to various inflammatory responses, has been reported to be associated with COVID-19 severity. However, the immunological signature and the exact gene expression program by which the presence of CHIP exerts its clinical impact on COVID-19 remain to be elucidated. In this study, we generated a single-cell transcriptome landscape of severe COVID-19 according to the presence of CHIP...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36229091"
                        open
                      >
                        <summary class="article-expander__title">Impact of Clonal Hematopoiesis in Patients With Cardiogenic Shock Complicating Acute Myocardial Infarction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36229091/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221028211238&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP is frequent among AMI and CS patients and is associated with impaired clinical outcome. CHIP assessment may facilitate risk stratification in patients with CS and imply novel treatment targets. (Culprit Lesion Only vs Multivessel Percutaneous Coronary Intervention in Cardiogenic Shock [CULPRIT-SHOCK]; NCT01927549).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36229591"
                        open
                      >
                        <summary class="article-expander__title">Single-cell transcriptome analyses reveal distinct gene expression signatures of severe COVID-19 in the presence of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36229591/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221027212537&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP), a common aging-related process that predisposes individuals to various inflammatory responses, has been reported to be associated with COVID-19 severity. However, the immunological signature and the exact gene expression program by which the presence of CHIP exerts its clinical impact on COVID-19 remain to be elucidated. In this study, we generated a single-cell transcriptome landscape of severe COVID-19 according to the presence of CHIP...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36229091"
                        open
                      >
                        <summary class="article-expander__title">Impact of Clonal Hematopoiesis in Patients With Cardiogenic Shock Complicating Acute Myocardial Infarction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36229091/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221027212537&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP is frequent among AMI and CS patients and is associated with impaired clinical outcome. CHIP assessment may facilitate risk stratification in patients with CS and imply novel treatment targets. (Culprit Lesion Only vs Multivessel Percutaneous Coronary Intervention in Cardiogenic Shock [CULPRIT-SHOCK]; NCT01927549).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36226499"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis by DNMT3A mutations as a common finding in idiopathic splanchnic vein thrombosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36226499/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221027212537&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Not available.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36229591"
                        open
                      >
                        <summary class="article-expander__title">Single-cell transcriptome analyses reveal distinct gene expression signatures of severe COVID-19 in the presence of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36229591/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221026211957&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP), a common aging-related process that predisposes individuals to various inflammatory responses, has been reported to be associated with COVID-19 severity. However, the immunological signature and the exact gene expression program by which the presence of CHIP exerts its clinical impact on COVID-19 remain to be elucidated. In this study, we generated a single-cell transcriptome landscape of severe COVID-19 according to the presence of CHIP...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36229091"
                        open
                      >
                        <summary class="article-expander__title">Impact of Clonal Hematopoiesis in Patients With Cardiogenic Shock Complicating Acute Myocardial Infarction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36229091/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221026211957&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP is frequent among AMI and CS patients and is associated with impaired clinical outcome. CHIP assessment may facilitate risk stratification in patients with CS and imply novel treatment targets. (Culprit Lesion Only vs Multivessel Percutaneous Coronary Intervention in Cardiogenic Shock [CULPRIT-SHOCK]; NCT01927549).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36226499"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis by DNMT3A mutations as a common finding in idiopathic splanchnic vein thrombosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36226499/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221026211957&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Not available.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36229591"
                        open
                      >
                        <summary class="article-expander__title">Single-cell transcriptome analyses reveal distinct gene expression signatures of severe COVID-19 in the presence of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36229591/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221025212118&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP), a common aging-related process that predisposes individuals to various inflammatory responses, has been reported to be associated with COVID-19 severity. However, the immunological signature and the exact gene expression program by which the presence of CHIP exerts its clinical impact on COVID-19 remain to be elucidated. In this study, we generated a single-cell transcriptome landscape of severe COVID-19 according to the presence of CHIP...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36229091"
                        open
                      >
                        <summary class="article-expander__title">Impact of Clonal Hematopoiesis in Patients With Cardiogenic Shock Complicating Acute Myocardial Infarction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36229091/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221025212118&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP is frequent among AMI and CS patients and is associated with impaired clinical outcome. CHIP assessment may facilitate risk stratification in patients with CS and imply novel treatment targets. (Culprit Lesion Only vs Multivessel Percutaneous Coronary Intervention in Cardiogenic Shock [CULPRIT-SHOCK]; NCT01927549).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36226499"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis by DNMT3A mutations as a common finding in idiopathic splanchnic vein thrombosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36226499/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221025212118&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Not available.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36229591"
                        open
                      >
                        <summary class="article-expander__title">Single-cell transcriptome analyses reveal distinct gene expression signatures of severe COVID-19 in the presence of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36229591/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221024214453&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP), a common aging-related process that predisposes individuals to various inflammatory responses, has been reported to be associated with COVID-19 severity. However, the immunological signature and the exact gene expression program by which the presence of CHIP exerts its clinical impact on COVID-19 remain to be elucidated. In this study, we generated a single-cell transcriptome landscape of severe COVID-19 according to the presence of CHIP...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36229091"
                        open
                      >
                        <summary class="article-expander__title">Impact of Clonal Hematopoiesis in Patients With Cardiogenic Shock Complicating Acute Myocardial Infarction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36229091/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221024214453&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP is frequent among AMI and CS patients and is associated with impaired clinical outcome. CHIP assessment may facilitate risk stratification in patients with CS and imply novel treatment targets. (Culprit Lesion Only vs Multivessel Percutaneous Coronary Intervention in Cardiogenic Shock [CULPRIT-SHOCK]; NCT01927549).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36226499"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis by DNMT3A mutations as a common finding in idiopathic splanchnic vein thrombosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36226499/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221024214453&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Not available.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36229591"
                        open
                      >
                        <summary class="article-expander__title">Single-cell transcriptome analyses reveal distinct gene expression signatures of severe COVID-19 in the presence of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36229591/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221023215029&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP), a common aging-related process that predisposes individuals to various inflammatory responses, has been reported to be associated with COVID-19 severity. However, the immunological signature and the exact gene expression program by which the presence of CHIP exerts its clinical impact on COVID-19 remain to be elucidated. In this study, we generated a single-cell transcriptome landscape of severe COVID-19 according to the presence of CHIP...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36229091"
                        open
                      >
                        <summary class="article-expander__title">Impact of Clonal Hematopoiesis in Patients With Cardiogenic Shock Complicating Acute Myocardial Infarction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36229091/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221023215029&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP is frequent among AMI and CS patients and is associated with impaired clinical outcome. CHIP assessment may facilitate risk stratification in patients with CS and imply novel treatment targets. (Culprit Lesion Only vs Multivessel Percutaneous Coronary Intervention in Cardiogenic Shock [CULPRIT-SHOCK]; NCT01927549).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36226499"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis by DNMT3A mutations as a common finding in idiopathic splanchnic vein thrombosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36226499/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221023215029&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Not available.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36229591"
                        open
                      >
                        <summary class="article-expander__title">Single-cell transcriptome analyses reveal distinct gene expression signatures of severe COVID-19 in the presence of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36229591/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221022213551&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP), a common aging-related process that predisposes individuals to various inflammatory responses, has been reported to be associated with COVID-19 severity. However, the immunological signature and the exact gene expression program by which the presence of CHIP exerts its clinical impact on COVID-19 remain to be elucidated. In this study, we generated a single-cell transcriptome landscape of severe COVID-19 according to the presence of CHIP...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36229091"
                        open
                      >
                        <summary class="article-expander__title">Impact of Clonal Hematopoiesis in Patients With Cardiogenic Shock Complicating Acute Myocardial Infarction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36229091/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221022213551&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP is frequent among AMI and CS patients and is associated with impaired clinical outcome. CHIP assessment may facilitate risk stratification in patients with CS and imply novel treatment targets. (Culprit Lesion Only vs Multivessel Percutaneous Coronary Intervention in Cardiogenic Shock [CULPRIT-SHOCK]; NCT01927549).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36226499"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis by DNMT3A mutations as a common finding in idiopathic splanchnic vein thrombosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36226499/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221022213551&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Not available.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36229591"
                        open
                      >
                        <summary class="article-expander__title">Single-cell transcriptome analyses reveal distinct gene expression signatures of severe COVID-19 in the presence of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36229591/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221021213516&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP), a common aging-related process that predisposes individuals to various inflammatory responses, has been reported to be associated with COVID-19 severity. However, the immunological signature and the exact gene expression program by which the presence of CHIP exerts its clinical impact on COVID-19 remain to be elucidated. In this study, we generated a single-cell transcriptome landscape of severe COVID-19 according to the presence of CHIP...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36229091"
                        open
                      >
                        <summary class="article-expander__title">Impact of Clonal Hematopoiesis in Patients With Cardiogenic Shock Complicating Acute Myocardial Infarction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36229091/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221021213516&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP is frequent among AMI and CS patients and is associated with impaired clinical outcome. CHIP assessment may facilitate risk stratification in patients with CS and imply novel treatment targets. (Culprit Lesion Only vs Multivessel Percutaneous Coronary Intervention in Cardiogenic Shock [CULPRIT-SHOCK]; NCT01927549).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36226499"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis by DNMT3A mutations as a common finding in idiopathic splanchnic vein thrombosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36226499/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221021213516&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Not available.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36229591"
                        open
                      >
                        <summary class="article-expander__title">Single-cell transcriptome analyses reveal distinct gene expression signatures of severe COVID-19 in the presence of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36229591/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221020212123&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP), a common aging-related process that predisposes individuals to various inflammatory responses, has been reported to be associated with COVID-19 severity. However, the immunological signature and the exact gene expression program by which the presence of CHIP exerts its clinical impact on COVID-19 remain to be elucidated. In this study, we generated a single-cell transcriptome landscape of severe COVID-19 according to the presence of CHIP...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36229091"
                        open
                      >
                        <summary class="article-expander__title">Impact of Clonal Hematopoiesis in Patients With Cardiogenic Shock Complicating Acute Myocardial Infarction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36229091/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221020212123&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP is frequent among AMI and CS patients and is associated with impaired clinical outcome. CHIP assessment may facilitate risk stratification in patients with CS and imply novel treatment targets. (Culprit Lesion Only vs Multivessel Percutaneous Coronary Intervention in Cardiogenic Shock [CULPRIT-SHOCK]; NCT01927549).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36226499"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis by DNMT3A mutations as a common finding in idiopathic splanchnic vein thrombosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36226499/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221020212123&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Not available.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36229591"
                        open
                      >
                        <summary class="article-expander__title">Single-cell transcriptome analyses reveal distinct gene expression signatures of severe COVID-19 in the presence of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36229591/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221019212642&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP), a common aging-related process that predisposes individuals to various inflammatory responses, has been reported to be associated with COVID-19 severity. However, the immunological signature and the exact gene expression program by which the presence of CHIP exerts its clinical impact on COVID-19 remain to be elucidated. In this study, we generated a single-cell transcriptome landscape of severe COVID-19 according to the presence of CHIP...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36229091"
                        open
                      >
                        <summary class="article-expander__title">Impact of Clonal Hematopoiesis in Patients With Cardiogenic Shock Complicating Acute Myocardial Infarction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36229091/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221019212642&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP is frequent among AMI and CS patients and is associated with impaired clinical outcome. CHIP assessment may facilitate risk stratification in patients with CS and imply novel treatment targets. (Culprit Lesion Only vs Multivessel Percutaneous Coronary Intervention in Cardiogenic Shock [CULPRIT-SHOCK]; NCT01927549).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36226499"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis by DNMT3A mutations as a common finding in idiopathic splanchnic vein thrombosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36226499/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221019212642&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Not available.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36229591"
                        open
                      >
                        <summary class="article-expander__title">Single-cell transcriptome analyses reveal distinct gene expression signatures of severe COVID-19 in the presence of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36229591/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221018213749&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP), a common aging-related process that predisposes individuals to various inflammatory responses, has been reported to be associated with COVID-19 severity. However, the immunological signature and the exact gene expression program by which the presence of CHIP exerts its clinical impact on COVID-19 remain to be elucidated. In this study, we generated a single-cell transcriptome landscape of severe COVID-19 according to the presence of CHIP...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36229091"
                        open
                      >
                        <summary class="article-expander__title">Impact of Clonal Hematopoiesis in Patients With Cardiogenic Shock Complicating Acute Myocardial Infarction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36229091/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221018213749&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP is frequent among AMI and CS patients and is associated with impaired clinical outcome. CHIP assessment may facilitate risk stratification in patients with CS and imply novel treatment targets. (Culprit Lesion Only vs Multivessel Percutaneous Coronary Intervention in Cardiogenic Shock [CULPRIT-SHOCK]; NCT01927549).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36226499"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis by DNMT3A mutations as a common finding in idiopathic splanchnic vein thrombosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36226499/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221018213749&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Not available.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36229591"
                        open
                      >
                        <summary class="article-expander__title">Single-cell transcriptome analyses reveal distinct gene expression signatures of severe COVID-19 in the presence of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36229591/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221017214255&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP), a common aging-related process that predisposes individuals to various inflammatory responses, has been reported to be associated with COVID-19 severity. However, the immunological signature and the exact gene expression program by which the presence of CHIP exerts its clinical impact on COVID-19 remain to be elucidated. In this study, we generated a single-cell transcriptome landscape of severe COVID-19 according to the presence of CHIP...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36229091"
                        open
                      >
                        <summary class="article-expander__title">Impact of Clonal Hematopoiesis in Patients With Cardiogenic Shock Complicating Acute Myocardial Infarction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36229091/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221017214255&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP is frequent among AMI and CS patients and is associated with impaired clinical outcome. CHIP assessment may facilitate risk stratification in patients with CS and imply novel treatment targets. (Culprit Lesion Only vs Multivessel Percutaneous Coronary Intervention in Cardiogenic Shock [CULPRIT-SHOCK]; NCT01927549).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36226499"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis by DNMT3A mutations as a common finding in idiopathic splanchnic vein thrombosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36226499/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221017214255&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Not available.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36229591"
                        open
                      >
                        <summary class="article-expander__title">Single-cell transcriptome analyses reveal distinct gene expression signatures of severe COVID-19 in the presence of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36229591/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221016214939&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP), a common aging-related process that predisposes individuals to various inflammatory responses, has been reported to be associated with COVID-19 severity. However, the immunological signature and the exact gene expression program by which the presence of CHIP exerts its clinical impact on COVID-19 remain to be elucidated. In this study, we generated a single-cell transcriptome landscape of severe COVID-19 according to the presence of CHIP...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36229091"
                        open
                      >
                        <summary class="article-expander__title">Impact of Clonal Hematopoiesis in Patients With Cardiogenic Shock Complicating Acute Myocardial Infarction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36229091/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221016214939&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP is frequent among AMI and CS patients and is associated with impaired clinical outcome. CHIP assessment may facilitate risk stratification in patients with CS and imply novel treatment targets. (Culprit Lesion Only vs Multivessel Percutaneous Coronary Intervention in Cardiogenic Shock [CULPRIT-SHOCK]; NCT01927549).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36226499"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis by DNMT3A mutations as a common finding in idiopathic splanchnic vein thrombosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36226499/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221016214939&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Not available.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36229591"
                        open
                      >
                        <summary class="article-expander__title">Single-cell transcriptome analyses reveal distinct gene expression signatures of severe COVID-19 in the presence of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36229591/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221015214355&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP), a common aging-related process that predisposes individuals to various inflammatory responses, has been reported to be associated with COVID-19 severity. However, the immunological signature and the exact gene expression program by which the presence of CHIP exerts its clinical impact on COVID-19 remain to be elucidated. In this study, we generated a single-cell transcriptome landscape of severe COVID-19 according to the presence of CHIP...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36229091"
                        open
                      >
                        <summary class="article-expander__title">Impact of Clonal Hematopoiesis in Patients With Cardiogenic Shock Complicating Acute Myocardial Infarction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36229091/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221015214355&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP is frequent among AMI and CS patients and is associated with impaired clinical outcome. CHIP assessment may facilitate risk stratification in patients with CS and imply novel treatment targets. (Culprit Lesion Only vs Multivessel Percutaneous Coronary Intervention in Cardiogenic Shock [CULPRIT-SHOCK]; NCT01927549).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36226499"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis by DNMT3A mutations as a common finding in idiopathic splanchnic vein thrombosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36226499/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221015214355&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Not available.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36229591"
                        open
                      >
                        <summary class="article-expander__title">Single-cell transcriptome analyses reveal distinct gene expression signatures of severe COVID-19 in the presence of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36229591/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221014214034&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP), a common aging-related process that predisposes individuals to various inflammatory responses, has been reported to be associated with COVID-19 severity. However, the immunological signature and the exact gene expression program by which the presence of CHIP exerts its clinical impact on COVID-19 remain to be elucidated. In this study, we generated a single-cell transcriptome landscape of severe COVID-19 according to the presence of CHIP...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36229091"
                        open
                      >
                        <summary class="article-expander__title">Impact of Clonal Hematopoiesis in Patients With Cardiogenic Shock Complicating Acute Myocardial Infarction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36229091/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221014214034&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP is frequent among AMI and CS patients and is associated with impaired clinical outcome. CHIP assessment may facilitate risk stratification in patients with CS and imply novel treatment targets. (Culprit Lesion Only vs Multivessel Percutaneous Coronary Intervention in Cardiogenic Shock [CULPRIT-SHOCK]; NCT01927549).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36226499"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis by DNMT3A mutations as a common finding in idiopathic splanchnic vein thrombosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36226499/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221014214034&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Not available.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36226499"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis by DNMT3A mutations as a common finding in idiopathic splanchnic vein thrombosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36226499/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221013214955&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Not available.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-12 datetime="2022-10-12T10:00:00.000Z">2022-10-12</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-12 datetime="2022-10-12T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.8&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20221128200439&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36221810"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential: associations with heart failure incidence, clinical parameters and biomarkers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36221810/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221024214453&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: CHIP correlates with HF risk factors and biomarkers, and is associated with incident HF in subjects &lt;65 years of age. This article is protected by copyright. All rights reserved.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36221810"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential: associations with heart failure incidence, clinical parameters and biomarkers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36221810/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221023215029&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: CHIP correlates with HF risk factors and biomarkers, and is associated with incident HF in subjects &lt;65 years of age. This article is protected by copyright. All rights reserved.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36221810"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential: associations with heart failure incidence, clinical parameters and biomarkers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36221810/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221022213551&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: CHIP correlates with HF risk factors and biomarkers, and is associated with incident HF in subjects &lt;65 years of age. This article is protected by copyright. All rights reserved.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36221810"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential: associations with heart failure incidence, clinical parameters and biomarkers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36221810/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221021213516&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: CHIP correlates with HF risk factors and biomarkers, and is associated with incident HF in subjects &lt;65 years of age. This article is protected by copyright. All rights reserved.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36221810"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential: associations with heart failure incidence, clinical parameters and biomarkers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36221810/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221020212123&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: CHIP correlates with HF risk factors and biomarkers, and is associated with incident HF in subjects &lt;65 years of age. This article is protected by copyright. All rights reserved.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36221810"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential: associations with heart failure incidence, clinical parameters and biomarkers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36221810/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221019212642&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: CHIP correlates with HF risk factors and biomarkers, and is associated with incident HF in subjects &lt;65 years of age. This article is protected by copyright. All rights reserved.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36221810"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential: associations with heart failure incidence, clinical parameters and biomarkers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36221810/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221018213749&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: CHIP correlates with HF risk factors and biomarkers, and is associated with incident HF in subjects &lt;65 years of age. This article is protected by copyright. All rights reserved.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36221810"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential: associations with heart failure incidence, clinical parameters and biomarkers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36221810/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221017214255&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: CHIP correlates with HF risk factors and biomarkers, and is associated with incident HF in subjects &lt;65 years of age. This article is protected by copyright. All rights reserved.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36221810"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential: associations with heart failure incidence, clinical parameters and biomarkers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36221810/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221016214939&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: CHIP correlates with HF risk factors and biomarkers, and is associated with incident HF in subjects &lt;65 years of age. This article is protected by copyright. All rights reserved.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36221810"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential: associations with heart failure incidence, clinical parameters and biomarkers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36221810/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221015214355&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: CHIP correlates with HF risk factors and biomarkers, and is associated with incident HF in subjects &lt;65 years of age. This article is protected by copyright. All rights reserved.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36221810"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential: associations with heart failure incidence, clinical parameters and biomarkers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36221810/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221014214034&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: CHIP correlates with HF risk factors and biomarkers, and is associated with incident HF in subjects &lt;65 years of age. This article is protected by copyright. All rights reserved.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36221810"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential: associations with heart failure incidence, clinical parameters and biomarkers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36221810/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221013214955&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: CHIP correlates with HF risk factors and biomarkers, and is associated with incident HF in subjects &lt;65 years of age. This article is protected by copyright. All rights reserved.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36221810"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential: associations with heart failure incidence, clinical parameters and biomarkers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36221810/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221012213728&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: CHIP correlates with HF risk factors and biomarkers, and is associated with incident HF in subjects &lt;65 years of age. This article is protected by copyright. All rights reserved.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221128200439&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.8">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36223594"
                        open
                      >
                        <summary class="article-expander__title">Whole-genome analysis identifies novel drivers and high-risk double-hit events in relapsed/refractory myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36223594/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221030212817&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Large-scale analyses of genomic data from newly-diagnosed multiple myeloma patients (ndMM) have been undertaken, however, large-scale analysis of relapsed/refractory multiple myeloma (rrMM) has not been performed. We hypothesize that somatic variants chronicle the therapeutic exposures and clonal structure of myeloma from ndMM to rrMM stages. We generated whole genome sequencing (WGS) data from 418 tumors (386 patients) derived from six rrMM clinical trials and compared them with WGS from 198...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36223594"
                        open
                      >
                        <summary class="article-expander__title">Whole-genome analysis identifies novel drivers and high-risk double-hit events in relapsed/refractory myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36223594/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221029212729&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Large-scale analyses of genomic data from newly-diagnosed multiple myeloma patients (ndMM) have been undertaken, however, large-scale analysis of relapsed/refractory multiple myeloma (rrMM) has not been performed. We hypothesize that somatic variants chronicle the therapeutic exposures and clonal structure of myeloma from ndMM to rrMM stages. We generated whole genome sequencing (WGS) data from 418 tumors (386 patients) derived from six rrMM clinical trials and compared them with WGS from 198...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36223594"
                        open
                      >
                        <summary class="article-expander__title">Whole-genome analysis identifies novel drivers and high-risk double-hit events in relapsed/refractory myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36223594/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221028211238&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Large-scale analyses of genomic data from newly-diagnosed multiple myeloma patients (ndMM) have been undertaken, however, large-scale analysis of relapsed/refractory multiple myeloma (rrMM) has not been performed. We hypothesize that somatic variants chronicle the therapeutic exposures and clonal structure of myeloma from ndMM to rrMM stages. We generated whole genome sequencing (WGS) data from 418 tumors (386 patients) derived from six rrMM clinical trials and compared them with WGS from 198...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36223594"
                        open
                      >
                        <summary class="article-expander__title">Whole-genome analysis identifies novel drivers and high-risk double-hit events in relapsed/refractory myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36223594/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221027212535&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Large-scale analyses of genomic data from newly-diagnosed multiple myeloma patients (ndMM) have been undertaken, however, large-scale analysis of relapsed/refractory multiple myeloma (rrMM) has not been performed. We hypothesize that somatic variants chronicle the therapeutic exposures and clonal structure of myeloma from ndMM to rrMM stages. We generated whole genome sequencing (WGS) data from 418 tumors (386 patients) derived from six rrMM clinical trials and compared them with WGS from 198...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36223594"
                        open
                      >
                        <summary class="article-expander__title">Whole-genome analysis identifies novel drivers and high-risk double-hit events in relapsed/refractory myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36223594/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221026211957&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Large-scale analyses of genomic data from newly-diagnosed multiple myeloma patients (ndMM) have been undertaken, however, large-scale analysis of relapsed/refractory multiple myeloma (rrMM) has not been performed. We hypothesize that somatic variants chronicle the therapeutic exposures and clonal structure of myeloma from ndMM to rrMM stages. We generated whole genome sequencing (WGS) data from 418 tumors (386 patients) derived from six rrMM clinical trials and compared them with WGS from 198...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36223594"
                        open
                      >
                        <summary class="article-expander__title">Whole-genome analysis identifies novel drivers and high-risk double-hit events in relapsed/refractory myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36223594/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221025212118&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Large-scale analyses of genomic data from newly-diagnosed multiple myeloma patients (ndMM) have been undertaken, however, large-scale analysis of relapsed/refractory multiple myeloma (rrMM) has not been performed. We hypothesize that somatic variants chronicle the therapeutic exposures and clonal structure of myeloma from ndMM to rrMM stages. We generated whole genome sequencing (WGS) data from 418 tumors (386 patients) derived from six rrMM clinical trials and compared them with WGS from 198...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36223594"
                        open
                      >
                        <summary class="article-expander__title">Whole-genome analysis identifies novel drivers and high-risk double-hit events in relapsed/refractory myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36223594/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221024214453&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Large-scale analyses of genomic data from newly-diagnosed multiple myeloma patients (ndMM) have been undertaken, however, large-scale analysis of relapsed/refractory multiple myeloma (rrMM) has not been performed. We hypothesize that somatic variants chronicle the therapeutic exposures and clonal structure of myeloma from ndMM to rrMM stages. We generated whole genome sequencing (WGS) data from 418 tumors (386 patients) derived from six rrMM clinical trials and compared them with WGS from 198...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36223594"
                        open
                      >
                        <summary class="article-expander__title">Whole-genome analysis identifies novel drivers and high-risk double-hit events in relapsed/refractory myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36223594/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221023215029&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Large-scale analyses of genomic data from newly-diagnosed multiple myeloma patients (ndMM) have been undertaken, however, large-scale analysis of relapsed/refractory multiple myeloma (rrMM) has not been performed. We hypothesize that somatic variants chronicle the therapeutic exposures and clonal structure of myeloma from ndMM to rrMM stages. We generated whole genome sequencing (WGS) data from 418 tumors (386 patients) derived from six rrMM clinical trials and compared them with WGS from 198...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36223594"
                        open
                      >
                        <summary class="article-expander__title">Whole-genome analysis identifies novel drivers and high-risk double-hit events in relapsed/refractory myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36223594/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221022213551&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Large-scale analyses of genomic data from newly-diagnosed multiple myeloma patients (ndMM) have been undertaken, however, large-scale analysis of relapsed/refractory multiple myeloma (rrMM) has not been performed. We hypothesize that somatic variants chronicle the therapeutic exposures and clonal structure of myeloma from ndMM to rrMM stages. We generated whole genome sequencing (WGS) data from 418 tumors (386 patients) derived from six rrMM clinical trials and compared them with WGS from 198...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36223594"
                        open
                      >
                        <summary class="article-expander__title">Whole-genome analysis identifies novel drivers and high-risk double-hit events in relapsed/refractory myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36223594/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221021213516&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Large-scale analyses of genomic data from newly-diagnosed multiple myeloma patients (ndMM) have been undertaken, however, large-scale analysis of relapsed/refractory multiple myeloma (rrMM) has not been performed. We hypothesize that somatic variants chronicle the therapeutic exposures and clonal structure of myeloma from ndMM to rrMM stages. We generated whole genome sequencing (WGS) data from 418 tumors (386 patients) derived from six rrMM clinical trials and compared them with WGS from 198...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36223594"
                        open
                      >
                        <summary class="article-expander__title">Whole-genome analysis identifies novel drivers and high-risk double-hit events in relapsed/refractory myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36223594/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221020212123&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Large-scale analyses of genomic data from newly-diagnosed multiple myeloma patients (ndMM) have been undertaken, however, large-scale analysis of relapsed/refractory multiple myeloma (rrMM) has not been performed. We hypothesize that somatic variants chronicle the therapeutic exposures and clonal structure of myeloma from ndMM to rrMM stages. We generated whole genome sequencing (WGS) data from 418 tumors (386 patients) derived from six rrMM clinical trials and compared them with WGS from 198...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36223594"
                        open
                      >
                        <summary class="article-expander__title">Whole-genome analysis identifies novel drivers and high-risk double-hit events in relapsed/refractory myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36223594/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221019212642&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Large-scale analyses of genomic data from newly-diagnosed multiple myeloma patients (ndMM) have been undertaken, however, large-scale analysis of relapsed/refractory multiple myeloma (rrMM) has not been performed. We hypothesize that somatic variants chronicle the therapeutic exposures and clonal structure of myeloma from ndMM to rrMM stages. We generated whole genome sequencing (WGS) data from 418 tumors (386 patients) derived from six rrMM clinical trials and compared them with WGS from 198...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36223594"
                        open
                      >
                        <summary class="article-expander__title">Whole-genome analysis identifies novel drivers and high-risk double-hit events in relapsed/refractory myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36223594/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221018213749&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Large-scale analyses of genomic data from newly-diagnosed multiple myeloma patients (ndMM) have been undertaken, however, large-scale analysis of relapsed/refractory multiple myeloma (rrMM) has not been performed. We hypothesize that somatic variants chronicle the therapeutic exposures and clonal structure of myeloma from ndMM to rrMM stages. We generated whole genome sequencing (WGS) data from 418 tumors (386 patients) derived from six rrMM clinical trials and compared them with WGS from 198...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36223594"
                        open
                      >
                        <summary class="article-expander__title">Whole-genome analysis identifies novel drivers and high-risk double-hit events in relapsed/refractory myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36223594/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221017214255&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Large-scale analyses of genomic data from newly-diagnosed multiple myeloma patients (ndMM) have been undertaken, however, large-scale analysis of relapsed/refractory multiple myeloma (rrMM) has not been performed. We hypothesize that somatic variants chronicle the therapeutic exposures and clonal structure of myeloma from ndMM to rrMM stages. We generated whole genome sequencing (WGS) data from 418 tumors (386 patients) derived from six rrMM clinical trials and compared them with WGS from 198...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36223594"
                        open
                      >
                        <summary class="article-expander__title">Whole-genome analysis identifies novel drivers and high-risk double-hit events in relapsed/refractory myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36223594/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221016214941&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Large-scale analyses of genomic data from newly-diagnosed multiple myeloma patients (ndMM) have been undertaken, however, large-scale analysis of relapsed/refractory multiple myeloma (rrMM) has not been performed. We hypothesize that somatic variants chronicle the therapeutic exposures and clonal structure of myeloma from ndMM to rrMM stages. We generated whole genome sequencing (WGS) data from 418 tumors (386 patients) derived from six rrMM clinical trials and compared them with WGS from 198...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36223594"
                        open
                      >
                        <summary class="article-expander__title">Whole-genome analysis identifies novel drivers and high-risk double-hit events in relapsed/refractory myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36223594/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221015214357&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Large-scale analyses of genomic data from newly-diagnosed multiple myeloma patients (ndMM) have been undertaken, however, large-scale analysis of relapsed/refractory multiple myeloma (rrMM) has not been performed. We hypothesize that somatic variants chronicle the therapeutic exposures and clonal structure of myeloma from ndMM to rrMM stages. We generated whole genome sequencing (WGS) data from 418 tumors (386 patients) derived from six rrMM clinical trials and compared them with WGS from 198...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36223594"
                        open
                      >
                        <summary class="article-expander__title">Whole-genome analysis identifies novel drivers and high-risk double-hit events in relapsed/refractory myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36223594/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221014214034&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Large-scale analyses of genomic data from newly-diagnosed multiple myeloma patients (ndMM) have been undertaken, however, large-scale analysis of relapsed/refractory multiple myeloma (rrMM) has not been performed. We hypothesize that somatic variants chronicle the therapeutic exposures and clonal structure of myeloma from ndMM to rrMM stages. We generated whole genome sequencing (WGS) data from 418 tumors (386 patients) derived from six rrMM clinical trials and compared them with WGS from 198...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36223594"
                        open
                      >
                        <summary class="article-expander__title">Whole-genome analysis identifies novel drivers and high-risk double-hit events in relapsed/refractory myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36223594/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221013214955&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Large-scale analyses of genomic data from newly-diagnosed multiple myeloma patients (ndMM) have been undertaken, however, large-scale analysis of relapsed/refractory multiple myeloma (rrMM) has not been performed. We hypothesize that somatic variants chronicle the therapeutic exposures and clonal structure of myeloma from ndMM to rrMM stages. We generated whole genome sequencing (WGS) data from 418 tumors (386 patients) derived from six rrMM clinical trials and compared them with WGS from 198...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36223594"
                        open
                      >
                        <summary class="article-expander__title">Whole-genome analysis identifies novel drivers and high-risk double-hit events in relapsed/refractory myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36223594/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221012213728&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Large-scale analyses of genomic data from newly-diagnosed multiple myeloma patients (ndMM) have been undertaken, however, large-scale analysis of relapsed/refractory multiple myeloma (rrMM) has not been performed. We hypothesize that somatic variants chronicle the therapeutic exposures and clonal structure of myeloma from ndMM to rrMM stages. We generated whole genome sequencing (WGS) data from 418 tumors (386 patients) derived from six rrMM clinical trials and compared them with WGS from 198...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-11 datetime="2022-10-11T10:00:00.000Z">2022-10-11</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-11 datetime="2022-10-11T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.8&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20221128200439&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36219593"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in patients with stem cell mobilization failure: a nested case-control study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36219593/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221026211957&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inadequate mobilization of peripheral blood progenitor cells (PBPC) is a limiting factor to proceed with autologous hematopoietic cell transplantation (auto-HCT). To assess the impact of clonal hematopoiesis (CH) on mobilization failure of PBPC for auto-HCT, we investigated the characteristics of poor mobilizers (with total PBPC collection &lt;2x106 CD34+ cells/kg) in a consecutive single center cohort of 776 patients. Targeted error corrected next-generation sequencing of 28 genes was performed in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36219593"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in patients with stem cell mobilization failure: a nested case-control study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36219593/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221025212118&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inadequate mobilization of peripheral blood progenitor cells (PBPC) is a limiting factor to proceed with autologous hematopoietic cell transplantation (auto-HCT). To assess the impact of clonal hematopoiesis (CH) on mobilization failure of PBPC for auto-HCT, we investigated the characteristics of poor mobilizers (with total PBPC collection &lt;2x106 CD34+ cells/kg) in a consecutive single center cohort of 776 patients. Targeted error corrected next-generation sequencing of 28 genes was performed in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36219593"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in patients with stem cell mobilization failure: a nested case-control study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36219593/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221024214453&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inadequate mobilization of peripheral blood progenitor cells (PBPC) is a limiting factor to proceed with autologous hematopoietic cell transplantation (auto-HCT). To assess the impact of clonal hematopoiesis (CH) on mobilization failure of PBPC for auto-HCT, we investigated the characteristics of poor mobilizers (with total PBPC collection &lt;2x106 CD34+ cells/kg) in a consecutive single center cohort of 776 patients. Targeted error corrected next-generation sequencing of 28 genes was performed in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36219593"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in patients with stem cell mobilization failure: a nested case-control study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36219593/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221023215029&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inadequate mobilization of peripheral blood progenitor cells (PBPC) is a limiting factor to proceed with autologous hematopoietic cell transplantation (auto-HCT). To assess the impact of clonal hematopoiesis (CH) on mobilization failure of PBPC for auto-HCT, we investigated the characteristics of poor mobilizers (with total PBPC collection &lt;2x106 CD34+ cells/kg) in a consecutive single center cohort of 776 patients. Targeted error corrected next-generation sequencing of 28 genes was performed in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36219593"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in patients with stem cell mobilization failure: a nested case-control study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36219593/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221022213551&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inadequate mobilization of peripheral blood progenitor cells (PBPC) is a limiting factor to proceed with autologous hematopoietic cell transplantation (auto-HCT). To assess the impact of clonal hematopoiesis (CH) on mobilization failure of PBPC for auto-HCT, we investigated the characteristics of poor mobilizers (with total PBPC collection &lt;2x106 CD34+ cells/kg) in a consecutive single center cohort of 776 patients. Targeted error corrected next-generation sequencing of 28 genes was performed in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36219593"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in patients with stem cell mobilization failure: a nested case-control study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36219593/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221021213516&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inadequate mobilization of peripheral blood progenitor cells (PBPC) is a limiting factor to proceed with autologous hematopoietic cell transplantation (auto-HCT). To assess the impact of clonal hematopoiesis (CH) on mobilization failure of PBPC for auto-HCT, we investigated the characteristics of poor mobilizers (with total PBPC collection &lt;2x106 CD34+ cells/kg) in a consecutive single center cohort of 776 patients. Targeted error corrected next-generation sequencing of 28 genes was performed in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36219593"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in patients with stem cell mobilization failure: a nested case-control study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36219593/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221020212123&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inadequate mobilization of peripheral blood progenitor cells (PBPC) is a limiting factor to proceed with autologous hematopoietic cell transplantation (auto-HCT). To assess the impact of clonal hematopoiesis (CH) on mobilization failure of PBPC for auto-HCT, we investigated the characteristics of poor mobilizers (with total PBPC collection &lt;2x106 CD34+ cells/kg) in a consecutive single center cohort of 776 patients. Targeted error corrected next-generation sequencing of 28 genes was performed in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36219593"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in patients with stem cell mobilization failure: a nested case-control study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36219593/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221019212642&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inadequate mobilization of peripheral blood progenitor cells (PBPC) is a limiting factor to proceed with autologous hematopoietic cell transplantation (auto-HCT). To assess the impact of clonal hematopoiesis (CH) on mobilization failure of PBPC for auto-HCT, we investigated the characteristics of poor mobilizers (with total PBPC collection &lt;2x106 CD34+ cells/kg) in a consecutive single center cohort of 776 patients. Targeted error corrected next-generation sequencing of 28 genes was performed in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36219593"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in patients with stem cell mobilization failure: a nested case-control study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36219593/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221018213749&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inadequate mobilization of peripheral blood progenitor cells (PBPC) is a limiting factor to proceed with autologous hematopoietic cell transplantation (auto-HCT). To assess the impact of clonal hematopoiesis (CH) on mobilization failure of PBPC for auto-HCT, we investigated the characteristics of poor mobilizers (with total PBPC collection &lt;2x106 CD34+ cells/kg) in a consecutive single center cohort of 776 patients. Targeted error corrected next-generation sequencing of 28 genes was performed in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36219593"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in patients with stem cell mobilization failure: a nested case-control study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36219593/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221017214255&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inadequate mobilization of peripheral blood progenitor cells (PBPC) is a limiting factor to proceed with autologous hematopoietic cell transplantation (auto-HCT). To assess the impact of clonal hematopoiesis (CH) on mobilization failure of PBPC for auto-HCT, we investigated the characteristics of poor mobilizers (with total PBPC collection &lt;2x106 CD34+ cells/kg) in a consecutive single center cohort of 776 patients. Targeted error corrected next-generation sequencing of 28 genes was performed in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36219593"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in patients with stem cell mobilization failure: a nested case-control study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36219593/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221016214939&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inadequate mobilization of peripheral blood progenitor cells (PBPC) is a limiting factor to proceed with autologous hematopoietic cell transplantation (auto-HCT). To assess the impact of clonal hematopoiesis (CH) on mobilization failure of PBPC for auto-HCT, we investigated the characteristics of poor mobilizers (with total PBPC collection &lt;2x106 CD34+ cells/kg) in a consecutive single center cohort of 776 patients. Targeted error corrected next-generation sequencing of 28 genes was performed in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36219593"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in patients with stem cell mobilization failure: a nested case-control study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36219593/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221015214355&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inadequate mobilization of peripheral blood progenitor cells (PBPC) is a limiting factor to proceed with autologous hematopoietic cell transplantation (auto-HCT). To assess the impact of clonal hematopoiesis (CH) on mobilization failure of PBPC for auto-HCT, we investigated the characteristics of poor mobilizers (with total PBPC collection &lt;2x106 CD34+ cells/kg) in a consecutive single center cohort of 776 patients. Targeted error corrected next-generation sequencing of 28 genes was performed in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36219593"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in patients with stem cell mobilization failure: a nested case-control study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36219593/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221014214034&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inadequate mobilization of peripheral blood progenitor cells (PBPC) is a limiting factor to proceed with autologous hematopoietic cell transplantation (auto-HCT). To assess the impact of clonal hematopoiesis (CH) on mobilization failure of PBPC for auto-HCT, we investigated the characteristics of poor mobilizers (with total PBPC collection &lt;2x106 CD34+ cells/kg) in a consecutive single center cohort of 776 patients. Targeted error corrected next-generation sequencing of 28 genes was performed in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36219593"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in patients with stem cell mobilization failure: a nested case-control study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36219593/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221013214955&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inadequate mobilization of peripheral blood progenitor cells (PBPC) is a limiting factor to proceed with autologous hematopoietic cell transplantation (auto-HCT). To assess the impact of clonal hematopoiesis (CH) on mobilization failure of PBPC for auto-HCT, we investigated the characteristics of poor mobilizers (with total PBPC collection &lt;2x106 CD34+ cells/kg) in a consecutive single center cohort of 776 patients. Targeted error corrected next-generation sequencing of 28 genes was performed in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36219593"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in patients with stem cell mobilization failure: a nested case-control study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36219593/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221012213728&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inadequate mobilization of peripheral blood progenitor cells (PBPC) is a limiting factor to proceed with autologous hematopoietic cell transplantation (auto-HCT). To assess the impact of clonal hematopoiesis (CH) on mobilization failure of PBPC for auto-HCT, we investigated the characteristics of poor mobilizers (with total PBPC collection &lt;2x106 CD34+ cells/kg) in a consecutive single center cohort of 776 patients. Targeted error corrected next-generation sequencing of 28 genes was performed in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36219593"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in patients with stem cell mobilization failure: a nested case-control study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36219593/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221011214956&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inadequate mobilization of peripheral blood progenitor cells (PBPC) is a limiting factor to proceed with autologous hematopoietic cell transplantation (auto-HCT). To assess the impact of clonal hematopoiesis (CH) on mobilization failure of PBPC for auto-HCT, we investigated the characteristics of poor mobilizers (with total PBPC collection &lt;2x106 CD34+ cells/kg) in a consecutive single center cohort of 776 patients. Targeted error corrected next-generation sequencing of 28 genes was performed in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-10 datetime="2022-10-10T10:00:00.000Z">2022-10-10</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-10 datetime="2022-10-10T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.8&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20221128200439&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36214849"
                        open
                      >
                        <summary class="article-expander__title">Influence of clonal hematopoiesis on non-hematological diseases and aging processes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36214849/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221026211957&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The occurrence of clonal hematopoiesis, caused by acquired somatic mutations of leukemia-associated genes in blood stem cells is very common in the population and increases with age. Besides an increased risk of developing myeloid neoplasms, an unexpected causal relationship between clonal hematopoiesis and cardiovascular diseases was recently discovered. Clonal hematopoiesis presents as a new independent and strong risk factor for cardiovascular diseases, such as atherosclerosis, coronary heart...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36214849"
                        open
                      >
                        <summary class="article-expander__title">Influence of clonal hematopoiesis on non-hematological diseases and aging processes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36214849/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221025212118&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The occurrence of clonal hematopoiesis, caused by acquired somatic mutations of leukemia-associated genes in blood stem cells is very common in the population and increases with age. Besides an increased risk of developing myeloid neoplasms, an unexpected causal relationship between clonal hematopoiesis and cardiovascular diseases was recently discovered. Clonal hematopoiesis presents as a new independent and strong risk factor for cardiovascular diseases, such as atherosclerosis, coronary heart...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36214849"
                        open
                      >
                        <summary class="article-expander__title">Influence of clonal hematopoiesis on non-hematological diseases and aging processes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36214849/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221024214453&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The occurrence of clonal hematopoiesis, caused by acquired somatic mutations of leukemia-associated genes in blood stem cells is very common in the population and increases with age. Besides an increased risk of developing myeloid neoplasms, an unexpected causal relationship between clonal hematopoiesis and cardiovascular diseases was recently discovered. Clonal hematopoiesis presents as a new independent and strong risk factor for cardiovascular diseases, such as atherosclerosis, coronary heart...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36214849"
                        open
                      >
                        <summary class="article-expander__title">Influence of clonal hematopoiesis on non-hematological diseases and aging processes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36214849/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221023215029&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The occurrence of clonal hematopoiesis, caused by acquired somatic mutations of leukemia-associated genes in blood stem cells is very common in the population and increases with age. Besides an increased risk of developing myeloid neoplasms, an unexpected causal relationship between clonal hematopoiesis and cardiovascular diseases was recently discovered. Clonal hematopoiesis presents as a new independent and strong risk factor for cardiovascular diseases, such as atherosclerosis, coronary heart...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36210552"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and cardiovascular disease in cancer patients and survivors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36210552/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221023215029&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes have long been known to carry a high number of somatic mutations distributed across many genes. However, recent sequencing studies have unveiled that non-cancerous cells also carry a considerable number of somatic mutations, which are acquired continuously through the lifespan. Accordingly, the pathophysiological relevance of somatic mutagenesis beyond cancer has become a topic of intensive research. Human genetic studies and experiments in mice have shown that some somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36214849"
                        open
                      >
                        <summary class="article-expander__title">Influence of clonal hematopoiesis on non-hematological diseases and aging processes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36214849/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221022213551&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The occurrence of clonal hematopoiesis, caused by acquired somatic mutations of leukemia-associated genes in blood stem cells is very common in the population and increases with age. Besides an increased risk of developing myeloid neoplasms, an unexpected causal relationship between clonal hematopoiesis and cardiovascular diseases was recently discovered. Clonal hematopoiesis presents as a new independent and strong risk factor for cardiovascular diseases, such as atherosclerosis, coronary heart...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36210552"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and cardiovascular disease in cancer patients and survivors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36210552/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221022213551&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes have long been known to carry a high number of somatic mutations distributed across many genes. However, recent sequencing studies have unveiled that non-cancerous cells also carry a considerable number of somatic mutations, which are acquired continuously through the lifespan. Accordingly, the pathophysiological relevance of somatic mutagenesis beyond cancer has become a topic of intensive research. Human genetic studies and experiments in mice have shown that some somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36214849"
                        open
                      >
                        <summary class="article-expander__title">Influence of clonal hematopoiesis on non-hematological diseases and aging processes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36214849/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221021213516&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The occurrence of clonal hematopoiesis, caused by acquired somatic mutations of leukemia-associated genes in blood stem cells is very common in the population and increases with age. Besides an increased risk of developing myeloid neoplasms, an unexpected causal relationship between clonal hematopoiesis and cardiovascular diseases was recently discovered. Clonal hematopoiesis presents as a new independent and strong risk factor for cardiovascular diseases, such as atherosclerosis, coronary heart...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36210552"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and cardiovascular disease in cancer patients and survivors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36210552/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221021213516&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes have long been known to carry a high number of somatic mutations distributed across many genes. However, recent sequencing studies have unveiled that non-cancerous cells also carry a considerable number of somatic mutations, which are acquired continuously through the lifespan. Accordingly, the pathophysiological relevance of somatic mutagenesis beyond cancer has become a topic of intensive research. Human genetic studies and experiments in mice have shown that some somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36214849"
                        open
                      >
                        <summary class="article-expander__title">Influence of clonal hematopoiesis on non-hematological diseases and aging processes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36214849/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221020212123&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The occurrence of clonal hematopoiesis, caused by acquired somatic mutations of leukemia-associated genes in blood stem cells is very common in the population and increases with age. Besides an increased risk of developing myeloid neoplasms, an unexpected causal relationship between clonal hematopoiesis and cardiovascular diseases was recently discovered. Clonal hematopoiesis presents as a new independent and strong risk factor for cardiovascular diseases, such as atherosclerosis, coronary heart...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36210552"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and cardiovascular disease in cancer patients and survivors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36210552/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221020212123&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes have long been known to carry a high number of somatic mutations distributed across many genes. However, recent sequencing studies have unveiled that non-cancerous cells also carry a considerable number of somatic mutations, which are acquired continuously through the lifespan. Accordingly, the pathophysiological relevance of somatic mutagenesis beyond cancer has become a topic of intensive research. Human genetic studies and experiments in mice have shown that some somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36214849"
                        open
                      >
                        <summary class="article-expander__title">Influence of clonal hematopoiesis on non-hematological diseases and aging processes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36214849/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221019212642&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The occurrence of clonal hematopoiesis, caused by acquired somatic mutations of leukemia-associated genes in blood stem cells is very common in the population and increases with age. Besides an increased risk of developing myeloid neoplasms, an unexpected causal relationship between clonal hematopoiesis and cardiovascular diseases was recently discovered. Clonal hematopoiesis presents as a new independent and strong risk factor for cardiovascular diseases, such as atherosclerosis, coronary heart...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36210552"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and cardiovascular disease in cancer patients and survivors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36210552/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221019212642&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes have long been known to carry a high number of somatic mutations distributed across many genes. However, recent sequencing studies have unveiled that non-cancerous cells also carry a considerable number of somatic mutations, which are acquired continuously through the lifespan. Accordingly, the pathophysiological relevance of somatic mutagenesis beyond cancer has become a topic of intensive research. Human genetic studies and experiments in mice have shown that some somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36214849"
                        open
                      >
                        <summary class="article-expander__title">Influence of clonal hematopoiesis on non-hematological diseases and aging processes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36214849/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221018213749&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The occurrence of clonal hematopoiesis, caused by acquired somatic mutations of leukemia-associated genes in blood stem cells is very common in the population and increases with age. Besides an increased risk of developing myeloid neoplasms, an unexpected causal relationship between clonal hematopoiesis and cardiovascular diseases was recently discovered. Clonal hematopoiesis presents as a new independent and strong risk factor for cardiovascular diseases, such as atherosclerosis, coronary heart...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36210552"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and cardiovascular disease in cancer patients and survivors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36210552/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221018213749&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes have long been known to carry a high number of somatic mutations distributed across many genes. However, recent sequencing studies have unveiled that non-cancerous cells also carry a considerable number of somatic mutations, which are acquired continuously through the lifespan. Accordingly, the pathophysiological relevance of somatic mutagenesis beyond cancer has become a topic of intensive research. Human genetic studies and experiments in mice have shown that some somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36214849"
                        open
                      >
                        <summary class="article-expander__title">Influence of clonal hematopoiesis on non-hematological diseases and aging processes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36214849/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221017214255&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The occurrence of clonal hematopoiesis, caused by acquired somatic mutations of leukemia-associated genes in blood stem cells is very common in the population and increases with age. Besides an increased risk of developing myeloid neoplasms, an unexpected causal relationship between clonal hematopoiesis and cardiovascular diseases was recently discovered. Clonal hematopoiesis presents as a new independent and strong risk factor for cardiovascular diseases, such as atherosclerosis, coronary heart...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36210552"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and cardiovascular disease in cancer patients and survivors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36210552/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221017214255&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes have long been known to carry a high number of somatic mutations distributed across many genes. However, recent sequencing studies have unveiled that non-cancerous cells also carry a considerable number of somatic mutations, which are acquired continuously through the lifespan. Accordingly, the pathophysiological relevance of somatic mutagenesis beyond cancer has become a topic of intensive research. Human genetic studies and experiments in mice have shown that some somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36214849"
                        open
                      >
                        <summary class="article-expander__title">Influence of clonal hematopoiesis on non-hematological diseases and aging processes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36214849/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221016214939&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The occurrence of clonal hematopoiesis, caused by acquired somatic mutations of leukemia-associated genes in blood stem cells is very common in the population and increases with age. Besides an increased risk of developing myeloid neoplasms, an unexpected causal relationship between clonal hematopoiesis and cardiovascular diseases was recently discovered. Clonal hematopoiesis presents as a new independent and strong risk factor for cardiovascular diseases, such as atherosclerosis, coronary heart...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36210552"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and cardiovascular disease in cancer patients and survivors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36210552/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221016214939&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes have long been known to carry a high number of somatic mutations distributed across many genes. However, recent sequencing studies have unveiled that non-cancerous cells also carry a considerable number of somatic mutations, which are acquired continuously through the lifespan. Accordingly, the pathophysiological relevance of somatic mutagenesis beyond cancer has become a topic of intensive research. Human genetic studies and experiments in mice have shown that some somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36214849"
                        open
                      >
                        <summary class="article-expander__title">Influence of clonal hematopoiesis on non-hematological diseases and aging processes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36214849/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221015214355&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The occurrence of clonal hematopoiesis, caused by acquired somatic mutations of leukemia-associated genes in blood stem cells is very common in the population and increases with age. Besides an increased risk of developing myeloid neoplasms, an unexpected causal relationship between clonal hematopoiesis and cardiovascular diseases was recently discovered. Clonal hematopoiesis presents as a new independent and strong risk factor for cardiovascular diseases, such as atherosclerosis, coronary heart...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36210552"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and cardiovascular disease in cancer patients and survivors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36210552/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221015214355&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes have long been known to carry a high number of somatic mutations distributed across many genes. However, recent sequencing studies have unveiled that non-cancerous cells also carry a considerable number of somatic mutations, which are acquired continuously through the lifespan. Accordingly, the pathophysiological relevance of somatic mutagenesis beyond cancer has become a topic of intensive research. Human genetic studies and experiments in mice have shown that some somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36214849"
                        open
                      >
                        <summary class="article-expander__title">Influence of clonal hematopoiesis on non-hematological diseases and aging processes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36214849/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221014214034&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The occurrence of clonal hematopoiesis, caused by acquired somatic mutations of leukemia-associated genes in blood stem cells is very common in the population and increases with age. Besides an increased risk of developing myeloid neoplasms, an unexpected causal relationship between clonal hematopoiesis and cardiovascular diseases was recently discovered. Clonal hematopoiesis presents as a new independent and strong risk factor for cardiovascular diseases, such as atherosclerosis, coronary heart...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36210552"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and cardiovascular disease in cancer patients and survivors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36210552/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221014214034&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes have long been known to carry a high number of somatic mutations distributed across many genes. However, recent sequencing studies have unveiled that non-cancerous cells also carry a considerable number of somatic mutations, which are acquired continuously through the lifespan. Accordingly, the pathophysiological relevance of somatic mutagenesis beyond cancer has become a topic of intensive research. Human genetic studies and experiments in mice have shown that some somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36214849"
                        open
                      >
                        <summary class="article-expander__title">Influence of clonal hematopoiesis on non-hematological diseases and aging processes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36214849/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221013214955&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The occurrence of clonal hematopoiesis, caused by acquired somatic mutations of leukemia-associated genes in blood stem cells is very common in the population and increases with age. Besides an increased risk of developing myeloid neoplasms, an unexpected causal relationship between clonal hematopoiesis and cardiovascular diseases was recently discovered. Clonal hematopoiesis presents as a new independent and strong risk factor for cardiovascular diseases, such as atherosclerosis, coronary heart...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36210552"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and cardiovascular disease in cancer patients and survivors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36210552/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221013214955&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes have long been known to carry a high number of somatic mutations distributed across many genes. However, recent sequencing studies have unveiled that non-cancerous cells also carry a considerable number of somatic mutations, which are acquired continuously through the lifespan. Accordingly, the pathophysiological relevance of somatic mutagenesis beyond cancer has become a topic of intensive research. Human genetic studies and experiments in mice have shown that some somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36214849"
                        open
                      >
                        <summary class="article-expander__title">Influence of clonal hematopoiesis on non-hematological diseases and aging processes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36214849/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221012213728&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The occurrence of clonal hematopoiesis, caused by acquired somatic mutations of leukemia-associated genes in blood stem cells is very common in the population and increases with age. Besides an increased risk of developing myeloid neoplasms, an unexpected causal relationship between clonal hematopoiesis and cardiovascular diseases was recently discovered. Clonal hematopoiesis presents as a new independent and strong risk factor for cardiovascular diseases, such as atherosclerosis, coronary heart...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36210552"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and cardiovascular disease in cancer patients and survivors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36210552/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221012213728&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes have long been known to carry a high number of somatic mutations distributed across many genes. However, recent sequencing studies have unveiled that non-cancerous cells also carry a considerable number of somatic mutations, which are acquired continuously through the lifespan. Accordingly, the pathophysiological relevance of somatic mutagenesis beyond cancer has become a topic of intensive research. Human genetic studies and experiments in mice have shown that some somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36214849"
                        open
                      >
                        <summary class="article-expander__title">Influence of clonal hematopoiesis on non-hematological diseases and aging processes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36214849/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221011214956&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The occurrence of clonal hematopoiesis, caused by acquired somatic mutations of leukemia-associated genes in blood stem cells is very common in the population and increases with age. Besides an increased risk of developing myeloid neoplasms, an unexpected causal relationship between clonal hematopoiesis and cardiovascular diseases was recently discovered. Clonal hematopoiesis presents as a new independent and strong risk factor for cardiovascular diseases, such as atherosclerosis, coronary heart...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36210552"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and cardiovascular disease in cancer patients and survivors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36210552/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221011214956&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes have long been known to carry a high number of somatic mutations distributed across many genes. However, recent sequencing studies have unveiled that non-cancerous cells also carry a considerable number of somatic mutations, which are acquired continuously through the lifespan. Accordingly, the pathophysiological relevance of somatic mutagenesis beyond cancer has become a topic of intensive research. Human genetic studies and experiments in mice have shown that some somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36214849"
                        open
                      >
                        <summary class="article-expander__title">Influence of clonal hematopoiesis on non-hematological diseases and aging processes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36214849/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221010212330&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The occurrence of clonal hematopoiesis, caused by acquired somatic mutations of leukemia-associated genes in blood stem cells is very common in the population and increases with age. Besides an increased risk of developing myeloid neoplasms, an unexpected causal relationship between clonal hematopoiesis and cardiovascular diseases was recently discovered. Clonal hematopoiesis presents as a new independent and strong risk factor for cardiovascular diseases, such as atherosclerosis, coronary heart...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36210552"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and cardiovascular disease in cancer patients and survivors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36210552/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221010212330&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes have long been known to carry a high number of somatic mutations distributed across many genes. However, recent sequencing studies have unveiled that non-cancerous cells also carry a considerable number of somatic mutations, which are acquired continuously through the lifespan. Accordingly, the pathophysiological relevance of somatic mutagenesis beyond cancer has become a topic of intensive research. Human genetic studies and experiments in mice have shown that some somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221128200439&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.8">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36215125"
                        open
                      >
                        <summary class="article-expander__title">Clinical Efficacy and Whole Exome Sequencing of Liquid Biopsies in a Phase IB/II study of Bazedoxifene and Palbociclib in Hormone Receptor-Positive Breast Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36215125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221030212817&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The combination of palbocilib and bazedoxifene has clinical efficacy and an acceptable safety profile in a heavily pre-treated patient population with advanced HR+/HER2- breast cancer. These results merit continued investigation of bazedoxifene in breast cancer.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36215125"
                        open
                      >
                        <summary class="article-expander__title">Clinical Efficacy and Whole Exome Sequencing of Liquid Biopsies in a Phase IB/II study of Bazedoxifene and Palbociclib in Hormone Receptor-Positive Breast Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36215125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221029212729&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The combination of palbocilib and bazedoxifene has clinical efficacy and an acceptable safety profile in a heavily pre-treated patient population with advanced HR+/HER2- breast cancer. These results merit continued investigation of bazedoxifene in breast cancer.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36214849"
                        open
                      >
                        <summary class="article-expander__title">Influence of clonal hematopoiesis on non-hematological diseases and aging processes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36214849/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221029212729&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The occurrence of clonal hematopoiesis, caused by acquired somatic mutations of leukemia-associated genes in blood stem cells is very common in the population and increases with age. Besides an increased risk of developing myeloid neoplasms, an unexpected causal relationship between clonal hematopoiesis and cardiovascular diseases was recently discovered. Clonal hematopoiesis presents as a new independent and strong risk factor for cardiovascular diseases, such as atherosclerosis, coronary heart...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36215125"
                        open
                      >
                        <summary class="article-expander__title">Clinical Efficacy and Whole Exome Sequencing of Liquid Biopsies in a Phase IB/II study of Bazedoxifene and Palbociclib in Hormone Receptor-Positive Breast Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36215125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221028211238&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The combination of palbocilib and bazedoxifene has clinical efficacy and an acceptable safety profile in a heavily pre-treated patient population with advanced HR+/HER2- breast cancer. These results merit continued investigation of bazedoxifene in breast cancer.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36214849"
                        open
                      >
                        <summary class="article-expander__title">Influence of clonal hematopoiesis on non-hematological diseases and aging processes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36214849/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221028211238&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The occurrence of clonal hematopoiesis, caused by acquired somatic mutations of leukemia-associated genes in blood stem cells is very common in the population and increases with age. Besides an increased risk of developing myeloid neoplasms, an unexpected causal relationship between clonal hematopoiesis and cardiovascular diseases was recently discovered. Clonal hematopoiesis presents as a new independent and strong risk factor for cardiovascular diseases, such as atherosclerosis, coronary heart...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36215125"
                        open
                      >
                        <summary class="article-expander__title">Clinical Efficacy and Whole Exome Sequencing of Liquid Biopsies in a Phase IB/II study of Bazedoxifene and Palbociclib in Hormone Receptor-Positive Breast Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36215125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221027212535&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The combination of palbocilib and bazedoxifene has clinical efficacy and an acceptable safety profile in a heavily pre-treated patient population with advanced HR+/HER2- breast cancer. These results merit continued investigation of bazedoxifene in breast cancer.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36214625"
                        open
                      >
                        <summary class="article-expander__title">Cell competition in carcinogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36214625/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221027212535&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The majority of human cancers evolve over time through the stepwise accumulation of somatic mutations followed by clonal selection akin to Darwinian evolution. However, the in-depth mechanisms that govern clonal dynamics and selection remain elusive, particularly during the earliest stages of tissue transformation. Cell competition, often referred to as &#x27;survival of the fittest&#x27; at the cellular level, results in the elimination of less fit cells by their more fit neighbors supporting optimal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36210552"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and cardiovascular disease in cancer patients and survivors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36210552/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221027212535&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes have long been known to carry a high number of somatic mutations distributed across many genes. However, recent sequencing studies have unveiled that non-cancerous cells also carry a considerable number of somatic mutations, which are acquired continuously through the lifespan. Accordingly, the pathophysiological relevance of somatic mutagenesis beyond cancer has become a topic of intensive research. Human genetic studies and experiments in mice have shown that some somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36215125"
                        open
                      >
                        <summary class="article-expander__title">Clinical Efficacy and Whole Exome Sequencing of Liquid Biopsies in a Phase IB/II study of Bazedoxifene and Palbociclib in Hormone Receptor-Positive Breast Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36215125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221026211957&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The combination of palbocilib and bazedoxifene has clinical efficacy and an acceptable safety profile in a heavily pre-treated patient population with advanced HR+/HER2- breast cancer. These results merit continued investigation of bazedoxifene in breast cancer.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36214625"
                        open
                      >
                        <summary class="article-expander__title">Cell competition in carcinogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36214625/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221026211957&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The majority of human cancers evolve over time through the stepwise accumulation of somatic mutations followed by clonal selection akin to Darwinian evolution. However, the in-depth mechanisms that govern clonal dynamics and selection remain elusive, particularly during the earliest stages of tissue transformation. Cell competition, often referred to as &#x27;survival of the fittest&#x27; at the cellular level, results in the elimination of less fit cells by their more fit neighbors supporting optimal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36210552"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and cardiovascular disease in cancer patients and survivors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36210552/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221026211957&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes have long been known to carry a high number of somatic mutations distributed across many genes. However, recent sequencing studies have unveiled that non-cancerous cells also carry a considerable number of somatic mutations, which are acquired continuously through the lifespan. Accordingly, the pathophysiological relevance of somatic mutagenesis beyond cancer has become a topic of intensive research. Human genetic studies and experiments in mice have shown that some somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36215125"
                        open
                      >
                        <summary class="article-expander__title">Clinical Efficacy and Whole Exome Sequencing of Liquid Biopsies in a Phase IB/II study of Bazedoxifene and Palbociclib in Hormone Receptor-Positive Breast Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36215125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221025212118&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The combination of palbocilib and bazedoxifene has clinical efficacy and an acceptable safety profile in a heavily pre-treated patient population with advanced HR+/HER2- breast cancer. These results merit continued investigation of bazedoxifene in breast cancer.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36214625"
                        open
                      >
                        <summary class="article-expander__title">Cell competition in carcinogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36214625/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221025212118&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The majority of human cancers evolve over time through the stepwise accumulation of somatic mutations followed by clonal selection akin to Darwinian evolution. However, the in-depth mechanisms that govern clonal dynamics and selection remain elusive, particularly during the earliest stages of tissue transformation. Cell competition, often referred to as &#x27;survival of the fittest&#x27; at the cellular level, results in the elimination of less fit cells by their more fit neighbors supporting optimal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36210552"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and cardiovascular disease in cancer patients and survivors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36210552/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221025212118&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes have long been known to carry a high number of somatic mutations distributed across many genes. However, recent sequencing studies have unveiled that non-cancerous cells also carry a considerable number of somatic mutations, which are acquired continuously through the lifespan. Accordingly, the pathophysiological relevance of somatic mutagenesis beyond cancer has become a topic of intensive research. Human genetic studies and experiments in mice have shown that some somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36215125"
                        open
                      >
                        <summary class="article-expander__title">Clinical Efficacy and Whole Exome Sequencing of Liquid Biopsies in a Phase IB/II study of Bazedoxifene and Palbociclib in Hormone Receptor-Positive Breast Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36215125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221024214453&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The combination of palbocilib and bazedoxifene has clinical efficacy and an acceptable safety profile in a heavily pre-treated patient population with advanced HR+/HER2- breast cancer. These results merit continued investigation of bazedoxifene in breast cancer.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36214625"
                        open
                      >
                        <summary class="article-expander__title">Cell competition in carcinogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36214625/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221024214453&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The majority of human cancers evolve over time through the stepwise accumulation of somatic mutations followed by clonal selection akin to Darwinian evolution. However, the in-depth mechanisms that govern clonal dynamics and selection remain elusive, particularly during the earliest stages of tissue transformation. Cell competition, often referred to as &#x27;survival of the fittest&#x27; at the cellular level, results in the elimination of less fit cells by their more fit neighbors supporting optimal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36210552"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and cardiovascular disease in cancer patients and survivors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36210552/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221024214453&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes have long been known to carry a high number of somatic mutations distributed across many genes. However, recent sequencing studies have unveiled that non-cancerous cells also carry a considerable number of somatic mutations, which are acquired continuously through the lifespan. Accordingly, the pathophysiological relevance of somatic mutagenesis beyond cancer has become a topic of intensive research. Human genetic studies and experiments in mice have shown that some somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36215125"
                        open
                      >
                        <summary class="article-expander__title">Clinical Efficacy and Whole Exome Sequencing of Liquid Biopsies in a Phase IB/II study of Bazedoxifene and Palbociclib in Hormone Receptor-Positive Breast Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36215125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221023215029&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The combination of palbocilib and bazedoxifene has clinical efficacy and an acceptable safety profile in a heavily pre-treated patient population with advanced HR+/HER2- breast cancer. These results merit continued investigation of bazedoxifene in breast cancer.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36214625"
                        open
                      >
                        <summary class="article-expander__title">Cell competition in carcinogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36214625/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221023215029&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The majority of human cancers evolve over time through the stepwise accumulation of somatic mutations followed by clonal selection akin to Darwinian evolution. However, the in-depth mechanisms that govern clonal dynamics and selection remain elusive, particularly during the earliest stages of tissue transformation. Cell competition, often referred to as &#x27;survival of the fittest&#x27; at the cellular level, results in the elimination of less fit cells by their more fit neighbors supporting optimal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36210552"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and cardiovascular disease in cancer patients and survivors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36210552/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221023215029&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes have long been known to carry a high number of somatic mutations distributed across many genes. However, recent sequencing studies have unveiled that non-cancerous cells also carry a considerable number of somatic mutations, which are acquired continuously through the lifespan. Accordingly, the pathophysiological relevance of somatic mutagenesis beyond cancer has become a topic of intensive research. Human genetic studies and experiments in mice have shown that some somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36215125"
                        open
                      >
                        <summary class="article-expander__title">Clinical Efficacy and Whole Exome Sequencing of Liquid Biopsies in a Phase IB/II study of Bazedoxifene and Palbociclib in Hormone Receptor-Positive Breast Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36215125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221022213551&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The combination of palbocilib and bazedoxifene has clinical efficacy and an acceptable safety profile in a heavily pre-treated patient population with advanced HR+/HER2- breast cancer. These results merit continued investigation of bazedoxifene in breast cancer.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36214625"
                        open
                      >
                        <summary class="article-expander__title">Cell competition in carcinogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36214625/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221022213551&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The majority of human cancers evolve over time through the stepwise accumulation of somatic mutations followed by clonal selection akin to Darwinian evolution. However, the in-depth mechanisms that govern clonal dynamics and selection remain elusive, particularly during the earliest stages of tissue transformation. Cell competition, often referred to as &#x27;survival of the fittest&#x27; at the cellular level, results in the elimination of less fit cells by their more fit neighbors supporting optimal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36210552"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and cardiovascular disease in cancer patients and survivors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36210552/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221022213551&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes have long been known to carry a high number of somatic mutations distributed across many genes. However, recent sequencing studies have unveiled that non-cancerous cells also carry a considerable number of somatic mutations, which are acquired continuously through the lifespan. Accordingly, the pathophysiological relevance of somatic mutagenesis beyond cancer has become a topic of intensive research. Human genetic studies and experiments in mice have shown that some somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36215125"
                        open
                      >
                        <summary class="article-expander__title">Clinical Efficacy and Whole Exome Sequencing of Liquid Biopsies in a Phase IB/II study of Bazedoxifene and Palbociclib in Hormone Receptor-Positive Breast Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36215125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221021213516&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The combination of palbocilib and bazedoxifene has clinical efficacy and an acceptable safety profile in a heavily pre-treated patient population with advanced HR+/HER2- breast cancer. These results merit continued investigation of bazedoxifene in breast cancer.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36214625"
                        open
                      >
                        <summary class="article-expander__title">Cell competition in carcinogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36214625/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221021213516&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The majority of human cancers evolve over time through the stepwise accumulation of somatic mutations followed by clonal selection akin to Darwinian evolution. However, the in-depth mechanisms that govern clonal dynamics and selection remain elusive, particularly during the earliest stages of tissue transformation. Cell competition, often referred to as &#x27;survival of the fittest&#x27; at the cellular level, results in the elimination of less fit cells by their more fit neighbors supporting optimal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36210552"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and cardiovascular disease in cancer patients and survivors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36210552/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221021213516&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes have long been known to carry a high number of somatic mutations distributed across many genes. However, recent sequencing studies have unveiled that non-cancerous cells also carry a considerable number of somatic mutations, which are acquired continuously through the lifespan. Accordingly, the pathophysiological relevance of somatic mutagenesis beyond cancer has become a topic of intensive research. Human genetic studies and experiments in mice have shown that some somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36215125"
                        open
                      >
                        <summary class="article-expander__title">Clinical Efficacy and Whole Exome Sequencing of Liquid Biopsies in a Phase IB/II study of Bazedoxifene and Palbociclib in Hormone Receptor-Positive Breast Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36215125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221020212123&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The combination of palbocilib and bazedoxifene has clinical efficacy and an acceptable safety profile in a heavily pre-treated patient population with advanced HR+/HER2- breast cancer. These results merit continued investigation of bazedoxifene in breast cancer.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36214625"
                        open
                      >
                        <summary class="article-expander__title">Cell competition in carcinogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36214625/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221020212123&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The majority of human cancers evolve over time through the stepwise accumulation of somatic mutations followed by clonal selection akin to Darwinian evolution. However, the in-depth mechanisms that govern clonal dynamics and selection remain elusive, particularly during the earliest stages of tissue transformation. Cell competition, often referred to as &#x27;survival of the fittest&#x27; at the cellular level, results in the elimination of less fit cells by their more fit neighbors supporting optimal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36210552"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and cardiovascular disease in cancer patients and survivors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36210552/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221020212123&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes have long been known to carry a high number of somatic mutations distributed across many genes. However, recent sequencing studies have unveiled that non-cancerous cells also carry a considerable number of somatic mutations, which are acquired continuously through the lifespan. Accordingly, the pathophysiological relevance of somatic mutagenesis beyond cancer has become a topic of intensive research. Human genetic studies and experiments in mice have shown that some somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36215125"
                        open
                      >
                        <summary class="article-expander__title">Clinical Efficacy and Whole Exome Sequencing of Liquid Biopsies in a Phase IB/II study of Bazedoxifene and Palbociclib in Hormone Receptor-Positive Breast Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36215125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221019212642&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The combination of palbocilib and bazedoxifene has clinical efficacy and an acceptable safety profile in a heavily pre-treated patient population with advanced HR+/HER2- breast cancer. These results merit continued investigation of bazedoxifene in breast cancer.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36214625"
                        open
                      >
                        <summary class="article-expander__title">Cell competition in carcinogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36214625/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221019212642&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The majority of human cancers evolve over time through the stepwise accumulation of somatic mutations followed by clonal selection akin to Darwinian evolution. However, the in-depth mechanisms that govern clonal dynamics and selection remain elusive, particularly during the earliest stages of tissue transformation. Cell competition, often referred to as &#x27;survival of the fittest&#x27; at the cellular level, results in the elimination of less fit cells by their more fit neighbors supporting optimal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36210552"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and cardiovascular disease in cancer patients and survivors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36210552/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221019212642&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes have long been known to carry a high number of somatic mutations distributed across many genes. However, recent sequencing studies have unveiled that non-cancerous cells also carry a considerable number of somatic mutations, which are acquired continuously through the lifespan. Accordingly, the pathophysiological relevance of somatic mutagenesis beyond cancer has become a topic of intensive research. Human genetic studies and experiments in mice have shown that some somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36215125"
                        open
                      >
                        <summary class="article-expander__title">Clinical Efficacy and Whole Exome Sequencing of Liquid Biopsies in a Phase IB/II study of Bazedoxifene and Palbociclib in Hormone Receptor-Positive Breast Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36215125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221018213749&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The combination of palbocilib and bazedoxifene has clinical efficacy and an acceptable safety profile in a heavily pre-treated patient population with advanced HR+/HER2- breast cancer. These results merit continued investigation of bazedoxifene in breast cancer.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36214625"
                        open
                      >
                        <summary class="article-expander__title">Cell competition in carcinogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36214625/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221018213749&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The majority of human cancers evolve over time through the stepwise accumulation of somatic mutations followed by clonal selection akin to Darwinian evolution. However, the in-depth mechanisms that govern clonal dynamics and selection remain elusive, particularly during the earliest stages of tissue transformation. Cell competition, often referred to as &#x27;survival of the fittest&#x27; at the cellular level, results in the elimination of less fit cells by their more fit neighbors supporting optimal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36210552"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and cardiovascular disease in cancer patients and survivors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36210552/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221018213749&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes have long been known to carry a high number of somatic mutations distributed across many genes. However, recent sequencing studies have unveiled that non-cancerous cells also carry a considerable number of somatic mutations, which are acquired continuously through the lifespan. Accordingly, the pathophysiological relevance of somatic mutagenesis beyond cancer has become a topic of intensive research. Human genetic studies and experiments in mice have shown that some somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36215125"
                        open
                      >
                        <summary class="article-expander__title">Clinical Efficacy and Whole Exome Sequencing of Liquid Biopsies in a Phase IB/II study of Bazedoxifene and Palbociclib in Hormone Receptor-Positive Breast Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36215125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221017214255&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The combination of palbocilib and bazedoxifene has clinical efficacy and an acceptable safety profile in a heavily pre-treated patient population with advanced HR+/HER2- breast cancer. These results merit continued investigation of bazedoxifene in breast cancer.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36214625"
                        open
                      >
                        <summary class="article-expander__title">Cell competition in carcinogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36214625/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221017214255&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The majority of human cancers evolve over time through the stepwise accumulation of somatic mutations followed by clonal selection akin to Darwinian evolution. However, the in-depth mechanisms that govern clonal dynamics and selection remain elusive, particularly during the earliest stages of tissue transformation. Cell competition, often referred to as &#x27;survival of the fittest&#x27; at the cellular level, results in the elimination of less fit cells by their more fit neighbors supporting optimal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36210552"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and cardiovascular disease in cancer patients and survivors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36210552/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221017214255&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes have long been known to carry a high number of somatic mutations distributed across many genes. However, recent sequencing studies have unveiled that non-cancerous cells also carry a considerable number of somatic mutations, which are acquired continuously through the lifespan. Accordingly, the pathophysiological relevance of somatic mutagenesis beyond cancer has become a topic of intensive research. Human genetic studies and experiments in mice have shown that some somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36215125"
                        open
                      >
                        <summary class="article-expander__title">Clinical Efficacy and Whole Exome Sequencing of Liquid Biopsies in a Phase IB/II study of Bazedoxifene and Palbociclib in Hormone Receptor-Positive Breast Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36215125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221016214941&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The combination of palbocilib and bazedoxifene has clinical efficacy and an acceptable safety profile in a heavily pre-treated patient population with advanced HR+/HER2- breast cancer. These results merit continued investigation of bazedoxifene in breast cancer.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36214625"
                        open
                      >
                        <summary class="article-expander__title">Cell competition in carcinogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36214625/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221016214941&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The majority of human cancers evolve over time through the stepwise accumulation of somatic mutations followed by clonal selection akin to Darwinian evolution. However, the in-depth mechanisms that govern clonal dynamics and selection remain elusive, particularly during the earliest stages of tissue transformation. Cell competition, often referred to as &#x27;survival of the fittest&#x27; at the cellular level, results in the elimination of less fit cells by their more fit neighbors supporting optimal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36210552"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and cardiovascular disease in cancer patients and survivors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36210552/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221016214941&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes have long been known to carry a high number of somatic mutations distributed across many genes. However, recent sequencing studies have unveiled that non-cancerous cells also carry a considerable number of somatic mutations, which are acquired continuously through the lifespan. Accordingly, the pathophysiological relevance of somatic mutagenesis beyond cancer has become a topic of intensive research. Human genetic studies and experiments in mice have shown that some somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36215125"
                        open
                      >
                        <summary class="article-expander__title">Clinical Efficacy and Whole Exome Sequencing of Liquid Biopsies in a Phase IB/II study of Bazedoxifene and Palbociclib in Hormone Receptor-Positive Breast Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36215125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221015214357&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The combination of palbocilib and bazedoxifene has clinical efficacy and an acceptable safety profile in a heavily pre-treated patient population with advanced HR+/HER2- breast cancer. These results merit continued investigation of bazedoxifene in breast cancer.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36214625"
                        open
                      >
                        <summary class="article-expander__title">Cell competition in carcinogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36214625/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221015214357&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The majority of human cancers evolve over time through the stepwise accumulation of somatic mutations followed by clonal selection akin to Darwinian evolution. However, the in-depth mechanisms that govern clonal dynamics and selection remain elusive, particularly during the earliest stages of tissue transformation. Cell competition, often referred to as &#x27;survival of the fittest&#x27; at the cellular level, results in the elimination of less fit cells by their more fit neighbors supporting optimal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36210552"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and cardiovascular disease in cancer patients and survivors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36210552/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221015214357&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes have long been known to carry a high number of somatic mutations distributed across many genes. However, recent sequencing studies have unveiled that non-cancerous cells also carry a considerable number of somatic mutations, which are acquired continuously through the lifespan. Accordingly, the pathophysiological relevance of somatic mutagenesis beyond cancer has become a topic of intensive research. Human genetic studies and experiments in mice have shown that some somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36215125"
                        open
                      >
                        <summary class="article-expander__title">Clinical Efficacy and Whole Exome Sequencing of Liquid Biopsies in a Phase IB/II study of Bazedoxifene and Palbociclib in Hormone Receptor-Positive Breast Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36215125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221014214034&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The combination of palbocilib and bazedoxifene has clinical efficacy and an acceptable safety profile in a heavily pre-treated patient population with advanced HR+/HER2- breast cancer. These results merit continued investigation of bazedoxifene in breast cancer.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36214625"
                        open
                      >
                        <summary class="article-expander__title">Cell competition in carcinogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36214625/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221014214034&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The majority of human cancers evolve over time through the stepwise accumulation of somatic mutations followed by clonal selection akin to Darwinian evolution. However, the in-depth mechanisms that govern clonal dynamics and selection remain elusive, particularly during the earliest stages of tissue transformation. Cell competition, often referred to as &#x27;survival of the fittest&#x27; at the cellular level, results in the elimination of less fit cells by their more fit neighbors supporting optimal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36210552"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and cardiovascular disease in cancer patients and survivors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36210552/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221014214034&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes have long been known to carry a high number of somatic mutations distributed across many genes. However, recent sequencing studies have unveiled that non-cancerous cells also carry a considerable number of somatic mutations, which are acquired continuously through the lifespan. Accordingly, the pathophysiological relevance of somatic mutagenesis beyond cancer has become a topic of intensive research. Human genetic studies and experiments in mice have shown that some somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36215125"
                        open
                      >
                        <summary class="article-expander__title">Clinical Efficacy and Whole Exome Sequencing of Liquid Biopsies in a Phase IB/II study of Bazedoxifene and Palbociclib in Hormone Receptor-Positive Breast Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36215125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221013214955&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The combination of palbocilib and bazedoxifene has clinical efficacy and an acceptable safety profile in a heavily pre-treated patient population with advanced HR+/HER2- breast cancer. These results merit continued investigation of bazedoxifene in breast cancer.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36214625"
                        open
                      >
                        <summary class="article-expander__title">Cell competition in carcinogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36214625/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221013214955&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The majority of human cancers evolve over time through the stepwise accumulation of somatic mutations followed by clonal selection akin to Darwinian evolution. However, the in-depth mechanisms that govern clonal dynamics and selection remain elusive, particularly during the earliest stages of tissue transformation. Cell competition, often referred to as &#x27;survival of the fittest&#x27; at the cellular level, results in the elimination of less fit cells by their more fit neighbors supporting optimal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36210552"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and cardiovascular disease in cancer patients and survivors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36210552/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221013214955&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes have long been known to carry a high number of somatic mutations distributed across many genes. However, recent sequencing studies have unveiled that non-cancerous cells also carry a considerable number of somatic mutations, which are acquired continuously through the lifespan. Accordingly, the pathophysiological relevance of somatic mutagenesis beyond cancer has become a topic of intensive research. Human genetic studies and experiments in mice have shown that some somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36215125"
                        open
                      >
                        <summary class="article-expander__title">Clinical Efficacy and Whole Exome Sequencing of Liquid Biopsies in a Phase IB/II study of Bazedoxifene and Palbociclib in Hormone Receptor-Positive Breast Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36215125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221012213728&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The combination of palbocilib and bazedoxifene has clinical efficacy and an acceptable safety profile in a heavily pre-treated patient population with advanced HR+/HER2- breast cancer. These results merit continued investigation of bazedoxifene in breast cancer.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36214625"
                        open
                      >
                        <summary class="article-expander__title">Cell competition in carcinogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36214625/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221012213728&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The majority of human cancers evolve over time through the stepwise accumulation of somatic mutations followed by clonal selection akin to Darwinian evolution. However, the in-depth mechanisms that govern clonal dynamics and selection remain elusive, particularly during the earliest stages of tissue transformation. Cell competition, often referred to as &#x27;survival of the fittest&#x27; at the cellular level, results in the elimination of less fit cells by their more fit neighbors supporting optimal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36210552"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and cardiovascular disease in cancer patients and survivors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36210552/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221012213728&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes have long been known to carry a high number of somatic mutations distributed across many genes. However, recent sequencing studies have unveiled that non-cancerous cells also carry a considerable number of somatic mutations, which are acquired continuously through the lifespan. Accordingly, the pathophysiological relevance of somatic mutagenesis beyond cancer has become a topic of intensive research. Human genetic studies and experiments in mice have shown that some somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36215125"
                        open
                      >
                        <summary class="article-expander__title">Clinical Efficacy and Whole Exome Sequencing of Liquid Biopsies in a Phase IB/II study of Bazedoxifene and Palbociclib in Hormone Receptor-Positive Breast Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36215125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221011214956&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The combination of palbocilib and bazedoxifene has clinical efficacy and an acceptable safety profile in a heavily pre-treated patient population with advanced HR+/HER2- breast cancer. These results merit continued investigation of bazedoxifene in breast cancer.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36214625"
                        open
                      >
                        <summary class="article-expander__title">Cell competition in carcinogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36214625/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221011214956&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The majority of human cancers evolve over time through the stepwise accumulation of somatic mutations followed by clonal selection akin to Darwinian evolution. However, the in-depth mechanisms that govern clonal dynamics and selection remain elusive, particularly during the earliest stages of tissue transformation. Cell competition, often referred to as &#x27;survival of the fittest&#x27; at the cellular level, results in the elimination of less fit cells by their more fit neighbors supporting optimal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36210552"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and cardiovascular disease in cancer patients and survivors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36210552/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221011214956&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes have long been known to carry a high number of somatic mutations distributed across many genes. However, recent sequencing studies have unveiled that non-cancerous cells also carry a considerable number of somatic mutations, which are acquired continuously through the lifespan. Accordingly, the pathophysiological relevance of somatic mutagenesis beyond cancer has become a topic of intensive research. Human genetic studies and experiments in mice have shown that some somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36215125"
                        open
                      >
                        <summary class="article-expander__title">Clinical Efficacy and Whole Exome Sequencing of Liquid Biopsies in a Phase IB/II study of Bazedoxifene and Palbociclib in Hormone Receptor-Positive Breast Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36215125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221010212330&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The combination of palbocilib and bazedoxifene has clinical efficacy and an acceptable safety profile in a heavily pre-treated patient population with advanced HR+/HER2- breast cancer. These results merit continued investigation of bazedoxifene in breast cancer.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36214625"
                        open
                      >
                        <summary class="article-expander__title">Cell competition in carcinogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36214625/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221010212330&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The majority of human cancers evolve over time through the stepwise accumulation of somatic mutations followed by clonal selection akin to Darwinian evolution. However, the in-depth mechanisms that govern clonal dynamics and selection remain elusive, particularly during the earliest stages of tissue transformation. Cell competition, often referred to as &#x27;survival of the fittest&#x27; at the cellular level, results in the elimination of less fit cells by their more fit neighbors supporting optimal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Tools of the Trade
  | Nature Reviews Cancer</button>
                  <a class="source-heading__link" href="https://www.nature.com/nrc/articles?type&#x3D;tools-of-the-trade">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="a7dd854ecb00659bf268ce51147a1486"
                        open
                      >
                        <summary class="article-expander__title">Tools of the Trade</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://rss.app/articles/cb4e791f6f6d729c04434f0c71d7a19e191b5e221032fdefd0e1c81b97926ec4ee1ab6032d919e3dfee63522835712cc36c82cb9970f3f56da6f9070d29d1478c6e821">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Browse the archive of articles on Nature Reviews Cancer</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="ff61af20bfc0e108a272ed7c86a30985"
                        open
                      >
                        <summary class="article-expander__title">Shining a light on tumour behaviour</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://rss.app/articles/cb4e791f6f6d729c04434f0c71d7a19e191b5e221032fde0d0f68e1989837488f14bf4096fd0c368f1a0777ddf110d957dd3">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Nature Reviews Cancer - In this Tools of the Trade article, Venkataramani describes the development of in vivo imaging workflows that allow the acquisition of imaging data with improved...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-06 datetime="2022-10-06T10:00:00.000Z">2022-10-06</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-06 datetime="2022-10-06T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.8&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20221128200439&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36203205"
                        open
                      >
                        <summary class="article-expander__title">Role of TET dioxygenases in the regulation of both normal and pathological hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36203205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221023215029&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The family of ten-eleven translocation dioxygenases (TETs) consists of TET1, TET2, and TET3. Although all TETs are expressed in hematopoietic tissues, only TET2 is commonly found to be mutated in age-related clonal hematopoiesis and hematopoietic malignancies. TET2 mutation causes abnormal epigenetic landscape changes and results in multiple stages of lineage commitment/differentiation defects as well as genetic instability in hematopoietic stem/progenitor cells (HSPCs). TET2 mutations are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36203205"
                        open
                      >
                        <summary class="article-expander__title">Role of TET dioxygenases in the regulation of both normal and pathological hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36203205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221022213551&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The family of ten-eleven translocation dioxygenases (TETs) consists of TET1, TET2, and TET3. Although all TETs are expressed in hematopoietic tissues, only TET2 is commonly found to be mutated in age-related clonal hematopoiesis and hematopoietic malignancies. TET2 mutation causes abnormal epigenetic landscape changes and results in multiple stages of lineage commitment/differentiation defects as well as genetic instability in hematopoietic stem/progenitor cells (HSPCs). TET2 mutations are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36203205"
                        open
                      >
                        <summary class="article-expander__title">Role of TET dioxygenases in the regulation of both normal and pathological hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36203205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221021213516&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The family of ten-eleven translocation dioxygenases (TETs) consists of TET1, TET2, and TET3. Although all TETs are expressed in hematopoietic tissues, only TET2 is commonly found to be mutated in age-related clonal hematopoiesis and hematopoietic malignancies. TET2 mutation causes abnormal epigenetic landscape changes and results in multiple stages of lineage commitment/differentiation defects as well as genetic instability in hematopoietic stem/progenitor cells (HSPCs). TET2 mutations are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36203205"
                        open
                      >
                        <summary class="article-expander__title">Role of TET dioxygenases in the regulation of both normal and pathological hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36203205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221020212123&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The family of ten-eleven translocation dioxygenases (TETs) consists of TET1, TET2, and TET3. Although all TETs are expressed in hematopoietic tissues, only TET2 is commonly found to be mutated in age-related clonal hematopoiesis and hematopoietic malignancies. TET2 mutation causes abnormal epigenetic landscape changes and results in multiple stages of lineage commitment/differentiation defects as well as genetic instability in hematopoietic stem/progenitor cells (HSPCs). TET2 mutations are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36203205"
                        open
                      >
                        <summary class="article-expander__title">Role of TET dioxygenases in the regulation of both normal and pathological hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36203205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221019212642&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The family of ten-eleven translocation dioxygenases (TETs) consists of TET1, TET2, and TET3. Although all TETs are expressed in hematopoietic tissues, only TET2 is commonly found to be mutated in age-related clonal hematopoiesis and hematopoietic malignancies. TET2 mutation causes abnormal epigenetic landscape changes and results in multiple stages of lineage commitment/differentiation defects as well as genetic instability in hematopoietic stem/progenitor cells (HSPCs). TET2 mutations are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36203205"
                        open
                      >
                        <summary class="article-expander__title">Role of TET dioxygenases in the regulation of both normal and pathological hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36203205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221018213749&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The family of ten-eleven translocation dioxygenases (TETs) consists of TET1, TET2, and TET3. Although all TETs are expressed in hematopoietic tissues, only TET2 is commonly found to be mutated in age-related clonal hematopoiesis and hematopoietic malignancies. TET2 mutation causes abnormal epigenetic landscape changes and results in multiple stages of lineage commitment/differentiation defects as well as genetic instability in hematopoietic stem/progenitor cells (HSPCs). TET2 mutations are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36203205"
                        open
                      >
                        <summary class="article-expander__title">Role of TET dioxygenases in the regulation of both normal and pathological hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36203205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221017214255&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The family of ten-eleven translocation dioxygenases (TETs) consists of TET1, TET2, and TET3. Although all TETs are expressed in hematopoietic tissues, only TET2 is commonly found to be mutated in age-related clonal hematopoiesis and hematopoietic malignancies. TET2 mutation causes abnormal epigenetic landscape changes and results in multiple stages of lineage commitment/differentiation defects as well as genetic instability in hematopoietic stem/progenitor cells (HSPCs). TET2 mutations are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36203205"
                        open
                      >
                        <summary class="article-expander__title">Role of TET dioxygenases in the regulation of both normal and pathological hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36203205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221016214939&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The family of ten-eleven translocation dioxygenases (TETs) consists of TET1, TET2, and TET3. Although all TETs are expressed in hematopoietic tissues, only TET2 is commonly found to be mutated in age-related clonal hematopoiesis and hematopoietic malignancies. TET2 mutation causes abnormal epigenetic landscape changes and results in multiple stages of lineage commitment/differentiation defects as well as genetic instability in hematopoietic stem/progenitor cells (HSPCs). TET2 mutations are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36203205"
                        open
                      >
                        <summary class="article-expander__title">Role of TET dioxygenases in the regulation of both normal and pathological hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36203205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221015214355&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The family of ten-eleven translocation dioxygenases (TETs) consists of TET1, TET2, and TET3. Although all TETs are expressed in hematopoietic tissues, only TET2 is commonly found to be mutated in age-related clonal hematopoiesis and hematopoietic malignancies. TET2 mutation causes abnormal epigenetic landscape changes and results in multiple stages of lineage commitment/differentiation defects as well as genetic instability in hematopoietic stem/progenitor cells (HSPCs). TET2 mutations are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36203205"
                        open
                      >
                        <summary class="article-expander__title">Role of TET dioxygenases in the regulation of both normal and pathological hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36203205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221014214034&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The family of ten-eleven translocation dioxygenases (TETs) consists of TET1, TET2, and TET3. Although all TETs are expressed in hematopoietic tissues, only TET2 is commonly found to be mutated in age-related clonal hematopoiesis and hematopoietic malignancies. TET2 mutation causes abnormal epigenetic landscape changes and results in multiple stages of lineage commitment/differentiation defects as well as genetic instability in hematopoietic stem/progenitor cells (HSPCs). TET2 mutations are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36203205"
                        open
                      >
                        <summary class="article-expander__title">Role of TET dioxygenases in the regulation of both normal and pathological hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36203205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221013214955&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The family of ten-eleven translocation dioxygenases (TETs) consists of TET1, TET2, and TET3. Although all TETs are expressed in hematopoietic tissues, only TET2 is commonly found to be mutated in age-related clonal hematopoiesis and hematopoietic malignancies. TET2 mutation causes abnormal epigenetic landscape changes and results in multiple stages of lineage commitment/differentiation defects as well as genetic instability in hematopoietic stem/progenitor cells (HSPCs). TET2 mutations are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36203205"
                        open
                      >
                        <summary class="article-expander__title">Role of TET dioxygenases in the regulation of both normal and pathological hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36203205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221012213728&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The family of ten-eleven translocation dioxygenases (TETs) consists of TET1, TET2, and TET3. Although all TETs are expressed in hematopoietic tissues, only TET2 is commonly found to be mutated in age-related clonal hematopoiesis and hematopoietic malignancies. TET2 mutation causes abnormal epigenetic landscape changes and results in multiple stages of lineage commitment/differentiation defects as well as genetic instability in hematopoietic stem/progenitor cells (HSPCs). TET2 mutations are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36203205"
                        open
                      >
                        <summary class="article-expander__title">Role of TET dioxygenases in the regulation of both normal and pathological hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36203205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221011214956&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The family of ten-eleven translocation dioxygenases (TETs) consists of TET1, TET2, and TET3. Although all TETs are expressed in hematopoietic tissues, only TET2 is commonly found to be mutated in age-related clonal hematopoiesis and hematopoietic malignancies. TET2 mutation causes abnormal epigenetic landscape changes and results in multiple stages of lineage commitment/differentiation defects as well as genetic instability in hematopoietic stem/progenitor cells (HSPCs). TET2 mutations are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36203205"
                        open
                      >
                        <summary class="article-expander__title">Role of TET dioxygenases in the regulation of both normal and pathological hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36203205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221010212330&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The family of ten-eleven translocation dioxygenases (TETs) consists of TET1, TET2, and TET3. Although all TETs are expressed in hematopoietic tissues, only TET2 is commonly found to be mutated in age-related clonal hematopoiesis and hematopoietic malignancies. TET2 mutation causes abnormal epigenetic landscape changes and results in multiple stages of lineage commitment/differentiation defects as well as genetic instability in hematopoietic stem/progenitor cells (HSPCs). TET2 mutations are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36203205"
                        open
                      >
                        <summary class="article-expander__title">Role of TET dioxygenases in the regulation of both normal and pathological hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36203205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221009214149&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The family of ten-eleven translocation dioxygenases (TETs) consists of TET1, TET2, and TET3. Although all TETs are expressed in hematopoietic tissues, only TET2 is commonly found to be mutated in age-related clonal hematopoiesis and hematopoietic malignancies. TET2 mutation causes abnormal epigenetic landscape changes and results in multiple stages of lineage commitment/differentiation defects as well as genetic instability in hematopoietic stem/progenitor cells (HSPCs). TET2 mutations are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36203205"
                        open
                      >
                        <summary class="article-expander__title">Role of TET dioxygenases in the regulation of both normal and pathological hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36203205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221008212536&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The family of ten-eleven translocation dioxygenases (TETs) consists of TET1, TET2, and TET3. Although all TETs are expressed in hematopoietic tissues, only TET2 is commonly found to be mutated in age-related clonal hematopoiesis and hematopoietic malignancies. TET2 mutation causes abnormal epigenetic landscape changes and results in multiple stages of lineage commitment/differentiation defects as well as genetic instability in hematopoietic stem/progenitor cells (HSPCs). TET2 mutations are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36203205"
                        open
                      >
                        <summary class="article-expander__title">Role of TET dioxygenases in the regulation of both normal and pathological hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36203205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221007211807&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The family of ten-eleven translocation dioxygenases (TETs) consists of TET1, TET2, and TET3. Although all TETs are expressed in hematopoietic tissues, only TET2 is commonly found to be mutated in age-related clonal hematopoiesis and hematopoietic malignancies. TET2 mutation causes abnormal epigenetic landscape changes and results in multiple stages of lineage commitment/differentiation defects as well as genetic instability in hematopoietic stem/progenitor cells (HSPCs). TET2 mutations are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221128200439&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.8">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36202798"
                        open
                      >
                        <summary class="article-expander__title">Lineage-coupled clonal capture identifies clonal evolution mechanisms and vulnerabilities of BRAFV600E inhibition resistance in melanoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36202798/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221027212535&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Targeted cancer therapies have revolutionized treatment but their efficacies are limited by the development of resistance driven by clonal evolution within tumors. We developed &quot;CAPTURE&quot;, a single-cell barcoding approach to comprehensively trace clonal dynamics and capture live lineage-coupled resistant cells for in-depth multi-omics analysis and functional exploration. We demonstrate that heterogeneous clones, either preexisting or emerging from drug-tolerant persister cells, dominated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36202798"
                        open
                      >
                        <summary class="article-expander__title">Lineage-coupled clonal capture identifies clonal evolution mechanisms and vulnerabilities of BRAFV600E inhibition resistance in melanoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36202798/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221026211957&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Targeted cancer therapies have revolutionized treatment but their efficacies are limited by the development of resistance driven by clonal evolution within tumors. We developed &quot;CAPTURE&quot;, a single-cell barcoding approach to comprehensively trace clonal dynamics and capture live lineage-coupled resistant cells for in-depth multi-omics analysis and functional exploration. We demonstrate that heterogeneous clones, either preexisting or emerging from drug-tolerant persister cells, dominated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36202798"
                        open
                      >
                        <summary class="article-expander__title">Lineage-coupled clonal capture identifies clonal evolution mechanisms and vulnerabilities of BRAFV600E inhibition resistance in melanoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36202798/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221025212118&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Targeted cancer therapies have revolutionized treatment but their efficacies are limited by the development of resistance driven by clonal evolution within tumors. We developed &quot;CAPTURE&quot;, a single-cell barcoding approach to comprehensively trace clonal dynamics and capture live lineage-coupled resistant cells for in-depth multi-omics analysis and functional exploration. We demonstrate that heterogeneous clones, either preexisting or emerging from drug-tolerant persister cells, dominated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36202798"
                        open
                      >
                        <summary class="article-expander__title">Lineage-coupled clonal capture identifies clonal evolution mechanisms and vulnerabilities of BRAFV600E inhibition resistance in melanoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36202798/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221024214453&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Targeted cancer therapies have revolutionized treatment but their efficacies are limited by the development of resistance driven by clonal evolution within tumors. We developed &quot;CAPTURE&quot;, a single-cell barcoding approach to comprehensively trace clonal dynamics and capture live lineage-coupled resistant cells for in-depth multi-omics analysis and functional exploration. We demonstrate that heterogeneous clones, either preexisting or emerging from drug-tolerant persister cells, dominated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36202798"
                        open
                      >
                        <summary class="article-expander__title">Lineage-coupled clonal capture identifies clonal evolution mechanisms and vulnerabilities of BRAFV600E inhibition resistance in melanoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36202798/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221023215029&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Targeted cancer therapies have revolutionized treatment but their efficacies are limited by the development of resistance driven by clonal evolution within tumors. We developed &quot;CAPTURE&quot;, a single-cell barcoding approach to comprehensively trace clonal dynamics and capture live lineage-coupled resistant cells for in-depth multi-omics analysis and functional exploration. We demonstrate that heterogeneous clones, either preexisting or emerging from drug-tolerant persister cells, dominated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36202798"
                        open
                      >
                        <summary class="article-expander__title">Lineage-coupled clonal capture identifies clonal evolution mechanisms and vulnerabilities of BRAFV600E inhibition resistance in melanoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36202798/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221022213551&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Targeted cancer therapies have revolutionized treatment but their efficacies are limited by the development of resistance driven by clonal evolution within tumors. We developed &quot;CAPTURE&quot;, a single-cell barcoding approach to comprehensively trace clonal dynamics and capture live lineage-coupled resistant cells for in-depth multi-omics analysis and functional exploration. We demonstrate that heterogeneous clones, either preexisting or emerging from drug-tolerant persister cells, dominated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36202798"
                        open
                      >
                        <summary class="article-expander__title">Lineage-coupled clonal capture identifies clonal evolution mechanisms and vulnerabilities of BRAFV600E inhibition resistance in melanoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36202798/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221021213516&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Targeted cancer therapies have revolutionized treatment but their efficacies are limited by the development of resistance driven by clonal evolution within tumors. We developed &quot;CAPTURE&quot;, a single-cell barcoding approach to comprehensively trace clonal dynamics and capture live lineage-coupled resistant cells for in-depth multi-omics analysis and functional exploration. We demonstrate that heterogeneous clones, either preexisting or emerging from drug-tolerant persister cells, dominated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36202798"
                        open
                      >
                        <summary class="article-expander__title">Lineage-coupled clonal capture identifies clonal evolution mechanisms and vulnerabilities of BRAFV600E inhibition resistance in melanoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36202798/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221020212123&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Targeted cancer therapies have revolutionized treatment but their efficacies are limited by the development of resistance driven by clonal evolution within tumors. We developed &quot;CAPTURE&quot;, a single-cell barcoding approach to comprehensively trace clonal dynamics and capture live lineage-coupled resistant cells for in-depth multi-omics analysis and functional exploration. We demonstrate that heterogeneous clones, either preexisting or emerging from drug-tolerant persister cells, dominated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36202798"
                        open
                      >
                        <summary class="article-expander__title">Lineage-coupled clonal capture identifies clonal evolution mechanisms and vulnerabilities of BRAFV600E inhibition resistance in melanoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36202798/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221019212642&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Targeted cancer therapies have revolutionized treatment but their efficacies are limited by the development of resistance driven by clonal evolution within tumors. We developed &quot;CAPTURE&quot;, a single-cell barcoding approach to comprehensively trace clonal dynamics and capture live lineage-coupled resistant cells for in-depth multi-omics analysis and functional exploration. We demonstrate that heterogeneous clones, either preexisting or emerging from drug-tolerant persister cells, dominated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36202798"
                        open
                      >
                        <summary class="article-expander__title">Lineage-coupled clonal capture identifies clonal evolution mechanisms and vulnerabilities of BRAFV600E inhibition resistance in melanoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36202798/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221018213749&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Targeted cancer therapies have revolutionized treatment but their efficacies are limited by the development of resistance driven by clonal evolution within tumors. We developed &quot;CAPTURE&quot;, a single-cell barcoding approach to comprehensively trace clonal dynamics and capture live lineage-coupled resistant cells for in-depth multi-omics analysis and functional exploration. We demonstrate that heterogeneous clones, either preexisting or emerging from drug-tolerant persister cells, dominated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36202798"
                        open
                      >
                        <summary class="article-expander__title">Lineage-coupled clonal capture identifies clonal evolution mechanisms and vulnerabilities of BRAFV600E inhibition resistance in melanoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36202798/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221017214255&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Targeted cancer therapies have revolutionized treatment but their efficacies are limited by the development of resistance driven by clonal evolution within tumors. We developed &quot;CAPTURE&quot;, a single-cell barcoding approach to comprehensively trace clonal dynamics and capture live lineage-coupled resistant cells for in-depth multi-omics analysis and functional exploration. We demonstrate that heterogeneous clones, either preexisting or emerging from drug-tolerant persister cells, dominated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36202798"
                        open
                      >
                        <summary class="article-expander__title">Lineage-coupled clonal capture identifies clonal evolution mechanisms and vulnerabilities of BRAFV600E inhibition resistance in melanoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36202798/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221016214941&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Targeted cancer therapies have revolutionized treatment but their efficacies are limited by the development of resistance driven by clonal evolution within tumors. We developed &quot;CAPTURE&quot;, a single-cell barcoding approach to comprehensively trace clonal dynamics and capture live lineage-coupled resistant cells for in-depth multi-omics analysis and functional exploration. We demonstrate that heterogeneous clones, either preexisting or emerging from drug-tolerant persister cells, dominated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36202798"
                        open
                      >
                        <summary class="article-expander__title">Lineage-coupled clonal capture identifies clonal evolution mechanisms and vulnerabilities of BRAFV600E inhibition resistance in melanoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36202798/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221015214357&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Targeted cancer therapies have revolutionized treatment but their efficacies are limited by the development of resistance driven by clonal evolution within tumors. We developed &quot;CAPTURE&quot;, a single-cell barcoding approach to comprehensively trace clonal dynamics and capture live lineage-coupled resistant cells for in-depth multi-omics analysis and functional exploration. We demonstrate that heterogeneous clones, either preexisting or emerging from drug-tolerant persister cells, dominated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36202798"
                        open
                      >
                        <summary class="article-expander__title">Lineage-coupled clonal capture identifies clonal evolution mechanisms and vulnerabilities of BRAFV600E inhibition resistance in melanoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36202798/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221014214034&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Targeted cancer therapies have revolutionized treatment but their efficacies are limited by the development of resistance driven by clonal evolution within tumors. We developed &quot;CAPTURE&quot;, a single-cell barcoding approach to comprehensively trace clonal dynamics and capture live lineage-coupled resistant cells for in-depth multi-omics analysis and functional exploration. We demonstrate that heterogeneous clones, either preexisting or emerging from drug-tolerant persister cells, dominated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36202798"
                        open
                      >
                        <summary class="article-expander__title">Lineage-coupled clonal capture identifies clonal evolution mechanisms and vulnerabilities of BRAFV600E inhibition resistance in melanoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36202798/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221013214955&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Targeted cancer therapies have revolutionized treatment but their efficacies are limited by the development of resistance driven by clonal evolution within tumors. We developed &quot;CAPTURE&quot;, a single-cell barcoding approach to comprehensively trace clonal dynamics and capture live lineage-coupled resistant cells for in-depth multi-omics analysis and functional exploration. We demonstrate that heterogeneous clones, either preexisting or emerging from drug-tolerant persister cells, dominated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36202798"
                        open
                      >
                        <summary class="article-expander__title">Lineage-coupled clonal capture identifies clonal evolution mechanisms and vulnerabilities of BRAFV600E inhibition resistance in melanoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36202798/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221012213728&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Targeted cancer therapies have revolutionized treatment but their efficacies are limited by the development of resistance driven by clonal evolution within tumors. We developed &quot;CAPTURE&quot;, a single-cell barcoding approach to comprehensively trace clonal dynamics and capture live lineage-coupled resistant cells for in-depth multi-omics analysis and functional exploration. We demonstrate that heterogeneous clones, either preexisting or emerging from drug-tolerant persister cells, dominated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36202798"
                        open
                      >
                        <summary class="article-expander__title">Lineage-coupled clonal capture identifies clonal evolution mechanisms and vulnerabilities of BRAFV600E inhibition resistance in melanoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36202798/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221011214956&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Targeted cancer therapies have revolutionized treatment but their efficacies are limited by the development of resistance driven by clonal evolution within tumors. We developed &quot;CAPTURE&quot;, a single-cell barcoding approach to comprehensively trace clonal dynamics and capture live lineage-coupled resistant cells for in-depth multi-omics analysis and functional exploration. We demonstrate that heterogeneous clones, either preexisting or emerging from drug-tolerant persister cells, dominated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36202798"
                        open
                      >
                        <summary class="article-expander__title">Lineage-coupled clonal capture identifies clonal evolution mechanisms and vulnerabilities of BRAFV600E inhibition resistance in melanoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36202798/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221010212330&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Targeted cancer therapies have revolutionized treatment but their efficacies are limited by the development of resistance driven by clonal evolution within tumors. We developed &quot;CAPTURE&quot;, a single-cell barcoding approach to comprehensively trace clonal dynamics and capture live lineage-coupled resistant cells for in-depth multi-omics analysis and functional exploration. We demonstrate that heterogeneous clones, either preexisting or emerging from drug-tolerant persister cells, dominated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36202798"
                        open
                      >
                        <summary class="article-expander__title">Lineage-coupled clonal capture identifies clonal evolution mechanisms and vulnerabilities of BRAFV600E inhibition resistance in melanoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36202798/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221009214149&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Targeted cancer therapies have revolutionized treatment but their efficacies are limited by the development of resistance driven by clonal evolution within tumors. We developed &quot;CAPTURE&quot;, a single-cell barcoding approach to comprehensively trace clonal dynamics and capture live lineage-coupled resistant cells for in-depth multi-omics analysis and functional exploration. We demonstrate that heterogeneous clones, either preexisting or emerging from drug-tolerant persister cells, dominated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36202798"
                        open
                      >
                        <summary class="article-expander__title">Lineage-coupled clonal capture identifies clonal evolution mechanisms and vulnerabilities of BRAFV600E inhibition resistance in melanoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36202798/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221008212536&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Targeted cancer therapies have revolutionized treatment but their efficacies are limited by the development of resistance driven by clonal evolution within tumors. We developed &quot;CAPTURE&quot;, a single-cell barcoding approach to comprehensively trace clonal dynamics and capture live lineage-coupled resistant cells for in-depth multi-omics analysis and functional exploration. We demonstrate that heterogeneous clones, either preexisting or emerging from drug-tolerant persister cells, dominated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36202798"
                        open
                      >
                        <summary class="article-expander__title">Lineage-coupled clonal capture identifies clonal evolution mechanisms and vulnerabilities of BRAFV600E inhibition resistance in melanoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36202798/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221007211807&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Targeted cancer therapies have revolutionized treatment but their efficacies are limited by the development of resistance driven by clonal evolution within tumors. We developed &quot;CAPTURE&quot;, a single-cell barcoding approach to comprehensively trace clonal dynamics and capture live lineage-coupled resistant cells for in-depth multi-omics analysis and functional exploration. We demonstrate that heterogeneous clones, either preexisting or emerging from drug-tolerant persister cells, dominated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-05 datetime="2022-10-05T10:00:00.000Z">2022-10-05</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-05 datetime="2022-10-05T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.8&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20221128200439&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198536"
                        open
                      >
                        <summary class="article-expander__title">Myeloproliferative neoplasms and vascular diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198536/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221021213516&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Vascular diseases are a feature of myeloproliferative neoplasms (MPNs) with a driver mutation, such as JAK2V617F. There is growing evidence that clonal hematopoiesis with JAK2V617F (JAK2V617F-CH) is often complicated with vascular diseases, even in the absence of MPN onset. Such vascular diseases associated with the MPN driver mutation include arterial/venous thrombosis, atherosclerotic coronary artery disease and aortic aneurysm, and pulmonary hypertension. Murine studies have recently revealed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198536"
                        open
                      >
                        <summary class="article-expander__title">Myeloproliferative neoplasms and vascular diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198536/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221020212123&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Vascular diseases are a feature of myeloproliferative neoplasms (MPNs) with a driver mutation, such as JAK2V617F. There is growing evidence that clonal hematopoiesis with JAK2V617F (JAK2V617F-CH) is often complicated with vascular diseases, even in the absence of MPN onset. Such vascular diseases associated with the MPN driver mutation include arterial/venous thrombosis, atherosclerotic coronary artery disease and aortic aneurysm, and pulmonary hypertension. Murine studies have recently revealed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198536"
                        open
                      >
                        <summary class="article-expander__title">Myeloproliferative neoplasms and vascular diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198536/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221019212642&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Vascular diseases are a feature of myeloproliferative neoplasms (MPNs) with a driver mutation, such as JAK2V617F. There is growing evidence that clonal hematopoiesis with JAK2V617F (JAK2V617F-CH) is often complicated with vascular diseases, even in the absence of MPN onset. Such vascular diseases associated with the MPN driver mutation include arterial/venous thrombosis, atherosclerotic coronary artery disease and aortic aneurysm, and pulmonary hypertension. Murine studies have recently revealed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198527"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in aplastic anemia and paroxysmal nocturnal hemoglobinuria</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198527/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221019212642&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Similar with myelodysplastic syndromes (MDS), aplastic anemia (AA) and paroxysmal nocturnal hemoglobinuria (PNH) are the major bone marrow failure syndromes. Approximately 10-20% of patients with AA/PNH present with transformation into MDS. Clonal hematopoiesis in AA/PNH affected by karyotypic abnormalities and genetic mutations should be discriminated from MDS clone, which is sometimes difficult due to shared genetic events among these diseases. In patients with AA/PNH, clones with UPD6p and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198536"
                        open
                      >
                        <summary class="article-expander__title">Myeloproliferative neoplasms and vascular diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198536/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221018213749&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Vascular diseases are a feature of myeloproliferative neoplasms (MPNs) with a driver mutation, such as JAK2V617F. There is growing evidence that clonal hematopoiesis with JAK2V617F (JAK2V617F-CH) is often complicated with vascular diseases, even in the absence of MPN onset. Such vascular diseases associated with the MPN driver mutation include arterial/venous thrombosis, atherosclerotic coronary artery disease and aortic aneurysm, and pulmonary hypertension. Murine studies have recently revealed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198527"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in aplastic anemia and paroxysmal nocturnal hemoglobinuria</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198527/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221018213749&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Similar with myelodysplastic syndromes (MDS), aplastic anemia (AA) and paroxysmal nocturnal hemoglobinuria (PNH) are the major bone marrow failure syndromes. Approximately 10-20% of patients with AA/PNH present with transformation into MDS. Clonal hematopoiesis in AA/PNH affected by karyotypic abnormalities and genetic mutations should be discriminated from MDS clone, which is sometimes difficult due to shared genetic events among these diseases. In patients with AA/PNH, clones with UPD6p and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198536"
                        open
                      >
                        <summary class="article-expander__title">Myeloproliferative neoplasms and vascular diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198536/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221017214255&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Vascular diseases are a feature of myeloproliferative neoplasms (MPNs) with a driver mutation, such as JAK2V617F. There is growing evidence that clonal hematopoiesis with JAK2V617F (JAK2V617F-CH) is often complicated with vascular diseases, even in the absence of MPN onset. Such vascular diseases associated with the MPN driver mutation include arterial/venous thrombosis, atherosclerotic coronary artery disease and aortic aneurysm, and pulmonary hypertension. Murine studies have recently revealed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198527"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in aplastic anemia and paroxysmal nocturnal hemoglobinuria</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198527/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221017214255&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Similar with myelodysplastic syndromes (MDS), aplastic anemia (AA) and paroxysmal nocturnal hemoglobinuria (PNH) are the major bone marrow failure syndromes. Approximately 10-20% of patients with AA/PNH present with transformation into MDS. Clonal hematopoiesis in AA/PNH affected by karyotypic abnormalities and genetic mutations should be discriminated from MDS clone, which is sometimes difficult due to shared genetic events among these diseases. In patients with AA/PNH, clones with UPD6p and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198536"
                        open
                      >
                        <summary class="article-expander__title">Myeloproliferative neoplasms and vascular diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198536/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221016214939&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Vascular diseases are a feature of myeloproliferative neoplasms (MPNs) with a driver mutation, such as JAK2V617F. There is growing evidence that clonal hematopoiesis with JAK2V617F (JAK2V617F-CH) is often complicated with vascular diseases, even in the absence of MPN onset. Such vascular diseases associated with the MPN driver mutation include arterial/venous thrombosis, atherosclerotic coronary artery disease and aortic aneurysm, and pulmonary hypertension. Murine studies have recently revealed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198527"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in aplastic anemia and paroxysmal nocturnal hemoglobinuria</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198527/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221016214939&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Similar with myelodysplastic syndromes (MDS), aplastic anemia (AA) and paroxysmal nocturnal hemoglobinuria (PNH) are the major bone marrow failure syndromes. Approximately 10-20% of patients with AA/PNH present with transformation into MDS. Clonal hematopoiesis in AA/PNH affected by karyotypic abnormalities and genetic mutations should be discriminated from MDS clone, which is sometimes difficult due to shared genetic events among these diseases. In patients with AA/PNH, clones with UPD6p and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198536"
                        open
                      >
                        <summary class="article-expander__title">Myeloproliferative neoplasms and vascular diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198536/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221015214355&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Vascular diseases are a feature of myeloproliferative neoplasms (MPNs) with a driver mutation, such as JAK2V617F. There is growing evidence that clonal hematopoiesis with JAK2V617F (JAK2V617F-CH) is often complicated with vascular diseases, even in the absence of MPN onset. Such vascular diseases associated with the MPN driver mutation include arterial/venous thrombosis, atherosclerotic coronary artery disease and aortic aneurysm, and pulmonary hypertension. Murine studies have recently revealed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198527"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in aplastic anemia and paroxysmal nocturnal hemoglobinuria</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198527/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221015214355&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Similar with myelodysplastic syndromes (MDS), aplastic anemia (AA) and paroxysmal nocturnal hemoglobinuria (PNH) are the major bone marrow failure syndromes. Approximately 10-20% of patients with AA/PNH present with transformation into MDS. Clonal hematopoiesis in AA/PNH affected by karyotypic abnormalities and genetic mutations should be discriminated from MDS clone, which is sometimes difficult due to shared genetic events among these diseases. In patients with AA/PNH, clones with UPD6p and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198536"
                        open
                      >
                        <summary class="article-expander__title">Myeloproliferative neoplasms and vascular diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198536/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221014214034&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Vascular diseases are a feature of myeloproliferative neoplasms (MPNs) with a driver mutation, such as JAK2V617F. There is growing evidence that clonal hematopoiesis with JAK2V617F (JAK2V617F-CH) is often complicated with vascular diseases, even in the absence of MPN onset. Such vascular diseases associated with the MPN driver mutation include arterial/venous thrombosis, atherosclerotic coronary artery disease and aortic aneurysm, and pulmonary hypertension. Murine studies have recently revealed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198527"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in aplastic anemia and paroxysmal nocturnal hemoglobinuria</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198527/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221014214034&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Similar with myelodysplastic syndromes (MDS), aplastic anemia (AA) and paroxysmal nocturnal hemoglobinuria (PNH) are the major bone marrow failure syndromes. Approximately 10-20% of patients with AA/PNH present with transformation into MDS. Clonal hematopoiesis in AA/PNH affected by karyotypic abnormalities and genetic mutations should be discriminated from MDS clone, which is sometimes difficult due to shared genetic events among these diseases. In patients with AA/PNH, clones with UPD6p and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198536"
                        open
                      >
                        <summary class="article-expander__title">Myeloproliferative neoplasms and vascular diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198536/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221013214955&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Vascular diseases are a feature of myeloproliferative neoplasms (MPNs) with a driver mutation, such as JAK2V617F. There is growing evidence that clonal hematopoiesis with JAK2V617F (JAK2V617F-CH) is often complicated with vascular diseases, even in the absence of MPN onset. Such vascular diseases associated with the MPN driver mutation include arterial/venous thrombosis, atherosclerotic coronary artery disease and aortic aneurysm, and pulmonary hypertension. Murine studies have recently revealed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198527"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in aplastic anemia and paroxysmal nocturnal hemoglobinuria</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198527/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221013214955&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Similar with myelodysplastic syndromes (MDS), aplastic anemia (AA) and paroxysmal nocturnal hemoglobinuria (PNH) are the major bone marrow failure syndromes. Approximately 10-20% of patients with AA/PNH present with transformation into MDS. Clonal hematopoiesis in AA/PNH affected by karyotypic abnormalities and genetic mutations should be discriminated from MDS clone, which is sometimes difficult due to shared genetic events among these diseases. In patients with AA/PNH, clones with UPD6p and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198536"
                        open
                      >
                        <summary class="article-expander__title">Myeloproliferative neoplasms and vascular diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198536/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221012213728&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Vascular diseases are a feature of myeloproliferative neoplasms (MPNs) with a driver mutation, such as JAK2V617F. There is growing evidence that clonal hematopoiesis with JAK2V617F (JAK2V617F-CH) is often complicated with vascular diseases, even in the absence of MPN onset. Such vascular diseases associated with the MPN driver mutation include arterial/venous thrombosis, atherosclerotic coronary artery disease and aortic aneurysm, and pulmonary hypertension. Murine studies have recently revealed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198527"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in aplastic anemia and paroxysmal nocturnal hemoglobinuria</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198527/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221012213728&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Similar with myelodysplastic syndromes (MDS), aplastic anemia (AA) and paroxysmal nocturnal hemoglobinuria (PNH) are the major bone marrow failure syndromes. Approximately 10-20% of patients with AA/PNH present with transformation into MDS. Clonal hematopoiesis in AA/PNH affected by karyotypic abnormalities and genetic mutations should be discriminated from MDS clone, which is sometimes difficult due to shared genetic events among these diseases. In patients with AA/PNH, clones with UPD6p and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198536"
                        open
                      >
                        <summary class="article-expander__title">Myeloproliferative neoplasms and vascular diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198536/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221011214956&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Vascular diseases are a feature of myeloproliferative neoplasms (MPNs) with a driver mutation, such as JAK2V617F. There is growing evidence that clonal hematopoiesis with JAK2V617F (JAK2V617F-CH) is often complicated with vascular diseases, even in the absence of MPN onset. Such vascular diseases associated with the MPN driver mutation include arterial/venous thrombosis, atherosclerotic coronary artery disease and aortic aneurysm, and pulmonary hypertension. Murine studies have recently revealed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198527"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in aplastic anemia and paroxysmal nocturnal hemoglobinuria</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198527/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221011214956&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Similar with myelodysplastic syndromes (MDS), aplastic anemia (AA) and paroxysmal nocturnal hemoglobinuria (PNH) are the major bone marrow failure syndromes. Approximately 10-20% of patients with AA/PNH present with transformation into MDS. Clonal hematopoiesis in AA/PNH affected by karyotypic abnormalities and genetic mutations should be discriminated from MDS clone, which is sometimes difficult due to shared genetic events among these diseases. In patients with AA/PNH, clones with UPD6p and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198536"
                        open
                      >
                        <summary class="article-expander__title">Myeloproliferative neoplasms and vascular diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198536/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221010212330&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Vascular diseases are a feature of myeloproliferative neoplasms (MPNs) with a driver mutation, such as JAK2V617F. There is growing evidence that clonal hematopoiesis with JAK2V617F (JAK2V617F-CH) is often complicated with vascular diseases, even in the absence of MPN onset. Such vascular diseases associated with the MPN driver mutation include arterial/venous thrombosis, atherosclerotic coronary artery disease and aortic aneurysm, and pulmonary hypertension. Murine studies have recently revealed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198527"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in aplastic anemia and paroxysmal nocturnal hemoglobinuria</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198527/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221010212330&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Similar with myelodysplastic syndromes (MDS), aplastic anemia (AA) and paroxysmal nocturnal hemoglobinuria (PNH) are the major bone marrow failure syndromes. Approximately 10-20% of patients with AA/PNH present with transformation into MDS. Clonal hematopoiesis in AA/PNH affected by karyotypic abnormalities and genetic mutations should be discriminated from MDS clone, which is sometimes difficult due to shared genetic events among these diseases. In patients with AA/PNH, clones with UPD6p and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198536"
                        open
                      >
                        <summary class="article-expander__title">Myeloproliferative neoplasms and vascular diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198536/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221009214149&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Vascular diseases are a feature of myeloproliferative neoplasms (MPNs) with a driver mutation, such as JAK2V617F. There is growing evidence that clonal hematopoiesis with JAK2V617F (JAK2V617F-CH) is often complicated with vascular diseases, even in the absence of MPN onset. Such vascular diseases associated with the MPN driver mutation include arterial/venous thrombosis, atherosclerotic coronary artery disease and aortic aneurysm, and pulmonary hypertension. Murine studies have recently revealed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198527"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in aplastic anemia and paroxysmal nocturnal hemoglobinuria</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198527/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221009214149&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Similar with myelodysplastic syndromes (MDS), aplastic anemia (AA) and paroxysmal nocturnal hemoglobinuria (PNH) are the major bone marrow failure syndromes. Approximately 10-20% of patients with AA/PNH present with transformation into MDS. Clonal hematopoiesis in AA/PNH affected by karyotypic abnormalities and genetic mutations should be discriminated from MDS clone, which is sometimes difficult due to shared genetic events among these diseases. In patients with AA/PNH, clones with UPD6p and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198536"
                        open
                      >
                        <summary class="article-expander__title">Myeloproliferative neoplasms and vascular diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198536/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221008212536&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Vascular diseases are a feature of myeloproliferative neoplasms (MPNs) with a driver mutation, such as JAK2V617F. There is growing evidence that clonal hematopoiesis with JAK2V617F (JAK2V617F-CH) is often complicated with vascular diseases, even in the absence of MPN onset. Such vascular diseases associated with the MPN driver mutation include arterial/venous thrombosis, atherosclerotic coronary artery disease and aortic aneurysm, and pulmonary hypertension. Murine studies have recently revealed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198527"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in aplastic anemia and paroxysmal nocturnal hemoglobinuria</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198527/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221008212536&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Similar with myelodysplastic syndromes (MDS), aplastic anemia (AA) and paroxysmal nocturnal hemoglobinuria (PNH) are the major bone marrow failure syndromes. Approximately 10-20% of patients with AA/PNH present with transformation into MDS. Clonal hematopoiesis in AA/PNH affected by karyotypic abnormalities and genetic mutations should be discriminated from MDS clone, which is sometimes difficult due to shared genetic events among these diseases. In patients with AA/PNH, clones with UPD6p and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198536"
                        open
                      >
                        <summary class="article-expander__title">Myeloproliferative neoplasms and vascular diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198536/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221007211807&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Vascular diseases are a feature of myeloproliferative neoplasms (MPNs) with a driver mutation, such as JAK2V617F. There is growing evidence that clonal hematopoiesis with JAK2V617F (JAK2V617F-CH) is often complicated with vascular diseases, even in the absence of MPN onset. Such vascular diseases associated with the MPN driver mutation include arterial/venous thrombosis, atherosclerotic coronary artery disease and aortic aneurysm, and pulmonary hypertension. Murine studies have recently revealed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198527"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in aplastic anemia and paroxysmal nocturnal hemoglobinuria</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198527/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221007211807&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Similar with myelodysplastic syndromes (MDS), aplastic anemia (AA) and paroxysmal nocturnal hemoglobinuria (PNH) are the major bone marrow failure syndromes. Approximately 10-20% of patients with AA/PNH present with transformation into MDS. Clonal hematopoiesis in AA/PNH affected by karyotypic abnormalities and genetic mutations should be discriminated from MDS clone, which is sometimes difficult due to shared genetic events among these diseases. In patients with AA/PNH, clones with UPD6p and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198536"
                        open
                      >
                        <summary class="article-expander__title">Myeloproliferative neoplasms and vascular diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198536/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221006214146&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Vascular diseases are a feature of myeloproliferative neoplasms (MPNs) with a driver mutation, such as JAK2V617F. There is growing evidence that clonal hematopoiesis with JAK2V617F (JAK2V617F-CH) is often complicated with vascular diseases, even in the absence of MPN onset. Such vascular diseases associated with the MPN driver mutation include arterial/venous thrombosis, atherosclerotic coronary artery disease and aortic aneurysm, and pulmonary hypertension. Murine studies have recently revealed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198527"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in aplastic anemia and paroxysmal nocturnal hemoglobinuria</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198527/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221006214146&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Similar with myelodysplastic syndromes (MDS), aplastic anemia (AA) and paroxysmal nocturnal hemoglobinuria (PNH) are the major bone marrow failure syndromes. Approximately 10-20% of patients with AA/PNH present with transformation into MDS. Clonal hematopoiesis in AA/PNH affected by karyotypic abnormalities and genetic mutations should be discriminated from MDS clone, which is sometimes difficult due to shared genetic events among these diseases. In patients with AA/PNH, clones with UPD6p and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221128200439&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.8">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36199157"
                        open
                      >
                        <summary class="article-expander__title">Determination of ERG(+), EZH2, NKX3.1, and SPINK-1 subtypes to evaluate their association with clonal origin and disease progression in multifocal prostate cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36199157/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221026211957&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The predominance of molecular discordance in prostatic foci per patient, which reflects the multifocal origin of PCa foci, highlights the importance of analyzing multiple samples to establish the prognostic and therapeutic relevance of molecular subtypes in a patient. All the subtypes analyzed here are of early onset, starting from preneoplastic lesions. NKX3.1 gene expression is the only molecular characteristic that shows a progression pattern by sample.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198773"
                        open
                      >
                        <summary class="article-expander__title">Targeted-sequence of normal urothelium and tumor of patients with non-muscle invasive bladder cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198773/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221026211957&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>During tumorigenesis, certain tissues are colonized by mutant clones with oncogenic driver mutations as precancer lesions. These mutations can facilitate clonal expansion and may contribute to malignant transformation. The molecular features of low-grade non-muscle invasive bladder cancer (NMIBC) and high-grade bladder cancer are so distinct that they are thought to follow different evolutionary tumorigenesis pathways. Although NMIBC accounts for most bladder tumors, the somatic mutation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198542"
                        open
                      >
                        <summary class="article-expander__title">Treatment strategies for multiple myeloma based on molecular pathogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198542/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221026211957&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It is well documented that multiple myeloma (MM) originates in a single plasma cell transformed by chromosome 14q translocations or chromosomal hyperdiploidy and evolves with the accumulation of point mutations of driver genes and/or cytogenetic abnormalities. Furthermore, disease progression is accomplished by branching patterns of subclonal evolution from reservoir clones with a propagating potential and/or the emergence of minor clones, which already exist at premalignant stages and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36199157"
                        open
                      >
                        <summary class="article-expander__title">Determination of ERG(+), EZH2, NKX3.1, and SPINK-1 subtypes to evaluate their association with clonal origin and disease progression in multifocal prostate cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36199157/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221025212118&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The predominance of molecular discordance in prostatic foci per patient, which reflects the multifocal origin of PCa foci, highlights the importance of analyzing multiple samples to establish the prognostic and therapeutic relevance of molecular subtypes in a patient. All the subtypes analyzed here are of early onset, starting from preneoplastic lesions. NKX3.1 gene expression is the only molecular characteristic that shows a progression pattern by sample.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198773"
                        open
                      >
                        <summary class="article-expander__title">Targeted-sequence of normal urothelium and tumor of patients with non-muscle invasive bladder cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198773/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221025212118&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>During tumorigenesis, certain tissues are colonized by mutant clones with oncogenic driver mutations as precancer lesions. These mutations can facilitate clonal expansion and may contribute to malignant transformation. The molecular features of low-grade non-muscle invasive bladder cancer (NMIBC) and high-grade bladder cancer are so distinct that they are thought to follow different evolutionary tumorigenesis pathways. Although NMIBC accounts for most bladder tumors, the somatic mutation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198542"
                        open
                      >
                        <summary class="article-expander__title">Treatment strategies for multiple myeloma based on molecular pathogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198542/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221025212118&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It is well documented that multiple myeloma (MM) originates in a single plasma cell transformed by chromosome 14q translocations or chromosomal hyperdiploidy and evolves with the accumulation of point mutations of driver genes and/or cytogenetic abnormalities. Furthermore, disease progression is accomplished by branching patterns of subclonal evolution from reservoir clones with a propagating potential and/or the emergence of minor clones, which already exist at premalignant stages and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36199157"
                        open
                      >
                        <summary class="article-expander__title">Determination of ERG(+), EZH2, NKX3.1, and SPINK-1 subtypes to evaluate their association with clonal origin and disease progression in multifocal prostate cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36199157/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221024214453&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The predominance of molecular discordance in prostatic foci per patient, which reflects the multifocal origin of PCa foci, highlights the importance of analyzing multiple samples to establish the prognostic and therapeutic relevance of molecular subtypes in a patient. All the subtypes analyzed here are of early onset, starting from preneoplastic lesions. NKX3.1 gene expression is the only molecular characteristic that shows a progression pattern by sample.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198773"
                        open
                      >
                        <summary class="article-expander__title">Targeted-sequence of normal urothelium and tumor of patients with non-muscle invasive bladder cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198773/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221024214453&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>During tumorigenesis, certain tissues are colonized by mutant clones with oncogenic driver mutations as precancer lesions. These mutations can facilitate clonal expansion and may contribute to malignant transformation. The molecular features of low-grade non-muscle invasive bladder cancer (NMIBC) and high-grade bladder cancer are so distinct that they are thought to follow different evolutionary tumorigenesis pathways. Although NMIBC accounts for most bladder tumors, the somatic mutation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198542"
                        open
                      >
                        <summary class="article-expander__title">Treatment strategies for multiple myeloma based on molecular pathogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198542/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221024214453&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It is well documented that multiple myeloma (MM) originates in a single plasma cell transformed by chromosome 14q translocations or chromosomal hyperdiploidy and evolves with the accumulation of point mutations of driver genes and/or cytogenetic abnormalities. Furthermore, disease progression is accomplished by branching patterns of subclonal evolution from reservoir clones with a propagating potential and/or the emergence of minor clones, which already exist at premalignant stages and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36199157"
                        open
                      >
                        <summary class="article-expander__title">Determination of ERG(+), EZH2, NKX3.1, and SPINK-1 subtypes to evaluate their association with clonal origin and disease progression in multifocal prostate cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36199157/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221023215029&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The predominance of molecular discordance in prostatic foci per patient, which reflects the multifocal origin of PCa foci, highlights the importance of analyzing multiple samples to establish the prognostic and therapeutic relevance of molecular subtypes in a patient. All the subtypes analyzed here are of early onset, starting from preneoplastic lesions. NKX3.1 gene expression is the only molecular characteristic that shows a progression pattern by sample.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198773"
                        open
                      >
                        <summary class="article-expander__title">Targeted-sequence of normal urothelium and tumor of patients with non-muscle invasive bladder cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198773/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221023215029&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>During tumorigenesis, certain tissues are colonized by mutant clones with oncogenic driver mutations as precancer lesions. These mutations can facilitate clonal expansion and may contribute to malignant transformation. The molecular features of low-grade non-muscle invasive bladder cancer (NMIBC) and high-grade bladder cancer are so distinct that they are thought to follow different evolutionary tumorigenesis pathways. Although NMIBC accounts for most bladder tumors, the somatic mutation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198542"
                        open
                      >
                        <summary class="article-expander__title">Treatment strategies for multiple myeloma based on molecular pathogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198542/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221023215029&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It is well documented that multiple myeloma (MM) originates in a single plasma cell transformed by chromosome 14q translocations or chromosomal hyperdiploidy and evolves with the accumulation of point mutations of driver genes and/or cytogenetic abnormalities. Furthermore, disease progression is accomplished by branching patterns of subclonal evolution from reservoir clones with a propagating potential and/or the emergence of minor clones, which already exist at premalignant stages and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36199157"
                        open
                      >
                        <summary class="article-expander__title">Determination of ERG(+), EZH2, NKX3.1, and SPINK-1 subtypes to evaluate their association with clonal origin and disease progression in multifocal prostate cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36199157/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221022213551&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The predominance of molecular discordance in prostatic foci per patient, which reflects the multifocal origin of PCa foci, highlights the importance of analyzing multiple samples to establish the prognostic and therapeutic relevance of molecular subtypes in a patient. All the subtypes analyzed here are of early onset, starting from preneoplastic lesions. NKX3.1 gene expression is the only molecular characteristic that shows a progression pattern by sample.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198773"
                        open
                      >
                        <summary class="article-expander__title">Targeted-sequence of normal urothelium and tumor of patients with non-muscle invasive bladder cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198773/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221022213551&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>During tumorigenesis, certain tissues are colonized by mutant clones with oncogenic driver mutations as precancer lesions. These mutations can facilitate clonal expansion and may contribute to malignant transformation. The molecular features of low-grade non-muscle invasive bladder cancer (NMIBC) and high-grade bladder cancer are so distinct that they are thought to follow different evolutionary tumorigenesis pathways. Although NMIBC accounts for most bladder tumors, the somatic mutation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198542"
                        open
                      >
                        <summary class="article-expander__title">Treatment strategies for multiple myeloma based on molecular pathogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198542/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221022213551&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It is well documented that multiple myeloma (MM) originates in a single plasma cell transformed by chromosome 14q translocations or chromosomal hyperdiploidy and evolves with the accumulation of point mutations of driver genes and/or cytogenetic abnormalities. Furthermore, disease progression is accomplished by branching patterns of subclonal evolution from reservoir clones with a propagating potential and/or the emergence of minor clones, which already exist at premalignant stages and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36199157"
                        open
                      >
                        <summary class="article-expander__title">Determination of ERG(+), EZH2, NKX3.1, and SPINK-1 subtypes to evaluate their association with clonal origin and disease progression in multifocal prostate cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36199157/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221021213516&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The predominance of molecular discordance in prostatic foci per patient, which reflects the multifocal origin of PCa foci, highlights the importance of analyzing multiple samples to establish the prognostic and therapeutic relevance of molecular subtypes in a patient. All the subtypes analyzed here are of early onset, starting from preneoplastic lesions. NKX3.1 gene expression is the only molecular characteristic that shows a progression pattern by sample.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198773"
                        open
                      >
                        <summary class="article-expander__title">Targeted-sequence of normal urothelium and tumor of patients with non-muscle invasive bladder cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198773/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221021213516&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>During tumorigenesis, certain tissues are colonized by mutant clones with oncogenic driver mutations as precancer lesions. These mutations can facilitate clonal expansion and may contribute to malignant transformation. The molecular features of low-grade non-muscle invasive bladder cancer (NMIBC) and high-grade bladder cancer are so distinct that they are thought to follow different evolutionary tumorigenesis pathways. Although NMIBC accounts for most bladder tumors, the somatic mutation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198542"
                        open
                      >
                        <summary class="article-expander__title">Treatment strategies for multiple myeloma based on molecular pathogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198542/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221021213516&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It is well documented that multiple myeloma (MM) originates in a single plasma cell transformed by chromosome 14q translocations or chromosomal hyperdiploidy and evolves with the accumulation of point mutations of driver genes and/or cytogenetic abnormalities. Furthermore, disease progression is accomplished by branching patterns of subclonal evolution from reservoir clones with a propagating potential and/or the emergence of minor clones, which already exist at premalignant stages and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36199157"
                        open
                      >
                        <summary class="article-expander__title">Determination of ERG(+), EZH2, NKX3.1, and SPINK-1 subtypes to evaluate their association with clonal origin and disease progression in multifocal prostate cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36199157/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221020212123&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The predominance of molecular discordance in prostatic foci per patient, which reflects the multifocal origin of PCa foci, highlights the importance of analyzing multiple samples to establish the prognostic and therapeutic relevance of molecular subtypes in a patient. All the subtypes analyzed here are of early onset, starting from preneoplastic lesions. NKX3.1 gene expression is the only molecular characteristic that shows a progression pattern by sample.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198773"
                        open
                      >
                        <summary class="article-expander__title">Targeted-sequence of normal urothelium and tumor of patients with non-muscle invasive bladder cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198773/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221020212123&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>During tumorigenesis, certain tissues are colonized by mutant clones with oncogenic driver mutations as precancer lesions. These mutations can facilitate clonal expansion and may contribute to malignant transformation. The molecular features of low-grade non-muscle invasive bladder cancer (NMIBC) and high-grade bladder cancer are so distinct that they are thought to follow different evolutionary tumorigenesis pathways. Although NMIBC accounts for most bladder tumors, the somatic mutation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198542"
                        open
                      >
                        <summary class="article-expander__title">Treatment strategies for multiple myeloma based on molecular pathogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198542/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221020212123&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It is well documented that multiple myeloma (MM) originates in a single plasma cell transformed by chromosome 14q translocations or chromosomal hyperdiploidy and evolves with the accumulation of point mutations of driver genes and/or cytogenetic abnormalities. Furthermore, disease progression is accomplished by branching patterns of subclonal evolution from reservoir clones with a propagating potential and/or the emergence of minor clones, which already exist at premalignant stages and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36199157"
                        open
                      >
                        <summary class="article-expander__title">Determination of ERG(+), EZH2, NKX3.1, and SPINK-1 subtypes to evaluate their association with clonal origin and disease progression in multifocal prostate cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36199157/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221019212642&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The predominance of molecular discordance in prostatic foci per patient, which reflects the multifocal origin of PCa foci, highlights the importance of analyzing multiple samples to establish the prognostic and therapeutic relevance of molecular subtypes in a patient. All the subtypes analyzed here are of early onset, starting from preneoplastic lesions. NKX3.1 gene expression is the only molecular characteristic that shows a progression pattern by sample.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198773"
                        open
                      >
                        <summary class="article-expander__title">Targeted-sequence of normal urothelium and tumor of patients with non-muscle invasive bladder cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198773/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221019212642&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>During tumorigenesis, certain tissues are colonized by mutant clones with oncogenic driver mutations as precancer lesions. These mutations can facilitate clonal expansion and may contribute to malignant transformation. The molecular features of low-grade non-muscle invasive bladder cancer (NMIBC) and high-grade bladder cancer are so distinct that they are thought to follow different evolutionary tumorigenesis pathways. Although NMIBC accounts for most bladder tumors, the somatic mutation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198542"
                        open
                      >
                        <summary class="article-expander__title">Treatment strategies for multiple myeloma based on molecular pathogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198542/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221019212642&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It is well documented that multiple myeloma (MM) originates in a single plasma cell transformed by chromosome 14q translocations or chromosomal hyperdiploidy and evolves with the accumulation of point mutations of driver genes and/or cytogenetic abnormalities. Furthermore, disease progression is accomplished by branching patterns of subclonal evolution from reservoir clones with a propagating potential and/or the emergence of minor clones, which already exist at premalignant stages and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36199157"
                        open
                      >
                        <summary class="article-expander__title">Determination of ERG(+), EZH2, NKX3.1, and SPINK-1 subtypes to evaluate their association with clonal origin and disease progression in multifocal prostate cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36199157/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221018213749&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The predominance of molecular discordance in prostatic foci per patient, which reflects the multifocal origin of PCa foci, highlights the importance of analyzing multiple samples to establish the prognostic and therapeutic relevance of molecular subtypes in a patient. All the subtypes analyzed here are of early onset, starting from preneoplastic lesions. NKX3.1 gene expression is the only molecular characteristic that shows a progression pattern by sample.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198773"
                        open
                      >
                        <summary class="article-expander__title">Targeted-sequence of normal urothelium and tumor of patients with non-muscle invasive bladder cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198773/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221018213749&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>During tumorigenesis, certain tissues are colonized by mutant clones with oncogenic driver mutations as precancer lesions. These mutations can facilitate clonal expansion and may contribute to malignant transformation. The molecular features of low-grade non-muscle invasive bladder cancer (NMIBC) and high-grade bladder cancer are so distinct that they are thought to follow different evolutionary tumorigenesis pathways. Although NMIBC accounts for most bladder tumors, the somatic mutation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198542"
                        open
                      >
                        <summary class="article-expander__title">Treatment strategies for multiple myeloma based on molecular pathogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198542/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221018213749&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It is well documented that multiple myeloma (MM) originates in a single plasma cell transformed by chromosome 14q translocations or chromosomal hyperdiploidy and evolves with the accumulation of point mutations of driver genes and/or cytogenetic abnormalities. Furthermore, disease progression is accomplished by branching patterns of subclonal evolution from reservoir clones with a propagating potential and/or the emergence of minor clones, which already exist at premalignant stages and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36199157"
                        open
                      >
                        <summary class="article-expander__title">Determination of ERG(+), EZH2, NKX3.1, and SPINK-1 subtypes to evaluate their association with clonal origin and disease progression in multifocal prostate cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36199157/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221017214255&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The predominance of molecular discordance in prostatic foci per patient, which reflects the multifocal origin of PCa foci, highlights the importance of analyzing multiple samples to establish the prognostic and therapeutic relevance of molecular subtypes in a patient. All the subtypes analyzed here are of early onset, starting from preneoplastic lesions. NKX3.1 gene expression is the only molecular characteristic that shows a progression pattern by sample.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198773"
                        open
                      >
                        <summary class="article-expander__title">Targeted-sequence of normal urothelium and tumor of patients with non-muscle invasive bladder cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198773/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221017214255&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>During tumorigenesis, certain tissues are colonized by mutant clones with oncogenic driver mutations as precancer lesions. These mutations can facilitate clonal expansion and may contribute to malignant transformation. The molecular features of low-grade non-muscle invasive bladder cancer (NMIBC) and high-grade bladder cancer are so distinct that they are thought to follow different evolutionary tumorigenesis pathways. Although NMIBC accounts for most bladder tumors, the somatic mutation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198542"
                        open
                      >
                        <summary class="article-expander__title">Treatment strategies for multiple myeloma based on molecular pathogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198542/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221017214255&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It is well documented that multiple myeloma (MM) originates in a single plasma cell transformed by chromosome 14q translocations or chromosomal hyperdiploidy and evolves with the accumulation of point mutations of driver genes and/or cytogenetic abnormalities. Furthermore, disease progression is accomplished by branching patterns of subclonal evolution from reservoir clones with a propagating potential and/or the emergence of minor clones, which already exist at premalignant stages and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36199157"
                        open
                      >
                        <summary class="article-expander__title">Determination of ERG(+), EZH2, NKX3.1, and SPINK-1 subtypes to evaluate their association with clonal origin and disease progression in multifocal prostate cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36199157/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221016214941&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The predominance of molecular discordance in prostatic foci per patient, which reflects the multifocal origin of PCa foci, highlights the importance of analyzing multiple samples to establish the prognostic and therapeutic relevance of molecular subtypes in a patient. All the subtypes analyzed here are of early onset, starting from preneoplastic lesions. NKX3.1 gene expression is the only molecular characteristic that shows a progression pattern by sample.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198773"
                        open
                      >
                        <summary class="article-expander__title">Targeted-sequence of normal urothelium and tumor of patients with non-muscle invasive bladder cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198773/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221016214941&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>During tumorigenesis, certain tissues are colonized by mutant clones with oncogenic driver mutations as precancer lesions. These mutations can facilitate clonal expansion and may contribute to malignant transformation. The molecular features of low-grade non-muscle invasive bladder cancer (NMIBC) and high-grade bladder cancer are so distinct that they are thought to follow different evolutionary tumorigenesis pathways. Although NMIBC accounts for most bladder tumors, the somatic mutation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198542"
                        open
                      >
                        <summary class="article-expander__title">Treatment strategies for multiple myeloma based on molecular pathogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198542/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221016214941&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It is well documented that multiple myeloma (MM) originates in a single plasma cell transformed by chromosome 14q translocations or chromosomal hyperdiploidy and evolves with the accumulation of point mutations of driver genes and/or cytogenetic abnormalities. Furthermore, disease progression is accomplished by branching patterns of subclonal evolution from reservoir clones with a propagating potential and/or the emergence of minor clones, which already exist at premalignant stages and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36199157"
                        open
                      >
                        <summary class="article-expander__title">Determination of ERG(+), EZH2, NKX3.1, and SPINK-1 subtypes to evaluate their association with clonal origin and disease progression in multifocal prostate cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36199157/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221015214357&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The predominance of molecular discordance in prostatic foci per patient, which reflects the multifocal origin of PCa foci, highlights the importance of analyzing multiple samples to establish the prognostic and therapeutic relevance of molecular subtypes in a patient. All the subtypes analyzed here are of early onset, starting from preneoplastic lesions. NKX3.1 gene expression is the only molecular characteristic that shows a progression pattern by sample.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198773"
                        open
                      >
                        <summary class="article-expander__title">Targeted-sequence of normal urothelium and tumor of patients with non-muscle invasive bladder cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198773/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221015214357&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>During tumorigenesis, certain tissues are colonized by mutant clones with oncogenic driver mutations as precancer lesions. These mutations can facilitate clonal expansion and may contribute to malignant transformation. The molecular features of low-grade non-muscle invasive bladder cancer (NMIBC) and high-grade bladder cancer are so distinct that they are thought to follow different evolutionary tumorigenesis pathways. Although NMIBC accounts for most bladder tumors, the somatic mutation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198542"
                        open
                      >
                        <summary class="article-expander__title">Treatment strategies for multiple myeloma based on molecular pathogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198542/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221015214357&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It is well documented that multiple myeloma (MM) originates in a single plasma cell transformed by chromosome 14q translocations or chromosomal hyperdiploidy and evolves with the accumulation of point mutations of driver genes and/or cytogenetic abnormalities. Furthermore, disease progression is accomplished by branching patterns of subclonal evolution from reservoir clones with a propagating potential and/or the emergence of minor clones, which already exist at premalignant stages and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36199157"
                        open
                      >
                        <summary class="article-expander__title">Determination of ERG(+), EZH2, NKX3.1, and SPINK-1 subtypes to evaluate their association with clonal origin and disease progression in multifocal prostate cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36199157/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221014214034&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The predominance of molecular discordance in prostatic foci per patient, which reflects the multifocal origin of PCa foci, highlights the importance of analyzing multiple samples to establish the prognostic and therapeutic relevance of molecular subtypes in a patient. All the subtypes analyzed here are of early onset, starting from preneoplastic lesions. NKX3.1 gene expression is the only molecular characteristic that shows a progression pattern by sample.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198773"
                        open
                      >
                        <summary class="article-expander__title">Targeted-sequence of normal urothelium and tumor of patients with non-muscle invasive bladder cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198773/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221014214034&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>During tumorigenesis, certain tissues are colonized by mutant clones with oncogenic driver mutations as precancer lesions. These mutations can facilitate clonal expansion and may contribute to malignant transformation. The molecular features of low-grade non-muscle invasive bladder cancer (NMIBC) and high-grade bladder cancer are so distinct that they are thought to follow different evolutionary tumorigenesis pathways. Although NMIBC accounts for most bladder tumors, the somatic mutation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198542"
                        open
                      >
                        <summary class="article-expander__title">Treatment strategies for multiple myeloma based on molecular pathogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198542/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221014214034&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It is well documented that multiple myeloma (MM) originates in a single plasma cell transformed by chromosome 14q translocations or chromosomal hyperdiploidy and evolves with the accumulation of point mutations of driver genes and/or cytogenetic abnormalities. Furthermore, disease progression is accomplished by branching patterns of subclonal evolution from reservoir clones with a propagating potential and/or the emergence of minor clones, which already exist at premalignant stages and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36199157"
                        open
                      >
                        <summary class="article-expander__title">Determination of ERG(+), EZH2, NKX3.1, and SPINK-1 subtypes to evaluate their association with clonal origin and disease progression in multifocal prostate cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36199157/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221013214955&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The predominance of molecular discordance in prostatic foci per patient, which reflects the multifocal origin of PCa foci, highlights the importance of analyzing multiple samples to establish the prognostic and therapeutic relevance of molecular subtypes in a patient. All the subtypes analyzed here are of early onset, starting from preneoplastic lesions. NKX3.1 gene expression is the only molecular characteristic that shows a progression pattern by sample.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198773"
                        open
                      >
                        <summary class="article-expander__title">Targeted-sequence of normal urothelium and tumor of patients with non-muscle invasive bladder cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198773/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221013214955&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>During tumorigenesis, certain tissues are colonized by mutant clones with oncogenic driver mutations as precancer lesions. These mutations can facilitate clonal expansion and may contribute to malignant transformation. The molecular features of low-grade non-muscle invasive bladder cancer (NMIBC) and high-grade bladder cancer are so distinct that they are thought to follow different evolutionary tumorigenesis pathways. Although NMIBC accounts for most bladder tumors, the somatic mutation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198542"
                        open
                      >
                        <summary class="article-expander__title">Treatment strategies for multiple myeloma based on molecular pathogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198542/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221013214955&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It is well documented that multiple myeloma (MM) originates in a single plasma cell transformed by chromosome 14q translocations or chromosomal hyperdiploidy and evolves with the accumulation of point mutations of driver genes and/or cytogenetic abnormalities. Furthermore, disease progression is accomplished by branching patterns of subclonal evolution from reservoir clones with a propagating potential and/or the emergence of minor clones, which already exist at premalignant stages and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36199157"
                        open
                      >
                        <summary class="article-expander__title">Determination of ERG(+), EZH2, NKX3.1, and SPINK-1 subtypes to evaluate their association with clonal origin and disease progression in multifocal prostate cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36199157/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221012213728&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The predominance of molecular discordance in prostatic foci per patient, which reflects the multifocal origin of PCa foci, highlights the importance of analyzing multiple samples to establish the prognostic and therapeutic relevance of molecular subtypes in a patient. All the subtypes analyzed here are of early onset, starting from preneoplastic lesions. NKX3.1 gene expression is the only molecular characteristic that shows a progression pattern by sample.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198773"
                        open
                      >
                        <summary class="article-expander__title">Targeted-sequence of normal urothelium and tumor of patients with non-muscle invasive bladder cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198773/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221012213728&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>During tumorigenesis, certain tissues are colonized by mutant clones with oncogenic driver mutations as precancer lesions. These mutations can facilitate clonal expansion and may contribute to malignant transformation. The molecular features of low-grade non-muscle invasive bladder cancer (NMIBC) and high-grade bladder cancer are so distinct that they are thought to follow different evolutionary tumorigenesis pathways. Although NMIBC accounts for most bladder tumors, the somatic mutation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198542"
                        open
                      >
                        <summary class="article-expander__title">Treatment strategies for multiple myeloma based on molecular pathogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198542/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221012213728&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It is well documented that multiple myeloma (MM) originates in a single plasma cell transformed by chromosome 14q translocations or chromosomal hyperdiploidy and evolves with the accumulation of point mutations of driver genes and/or cytogenetic abnormalities. Furthermore, disease progression is accomplished by branching patterns of subclonal evolution from reservoir clones with a propagating potential and/or the emergence of minor clones, which already exist at premalignant stages and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36199157"
                        open
                      >
                        <summary class="article-expander__title">Determination of ERG(+), EZH2, NKX3.1, and SPINK-1 subtypes to evaluate their association with clonal origin and disease progression in multifocal prostate cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36199157/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221011214956&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The predominance of molecular discordance in prostatic foci per patient, which reflects the multifocal origin of PCa foci, highlights the importance of analyzing multiple samples to establish the prognostic and therapeutic relevance of molecular subtypes in a patient. All the subtypes analyzed here are of early onset, starting from preneoplastic lesions. NKX3.1 gene expression is the only molecular characteristic that shows a progression pattern by sample.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198773"
                        open
                      >
                        <summary class="article-expander__title">Targeted-sequence of normal urothelium and tumor of patients with non-muscle invasive bladder cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198773/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221011214956&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>During tumorigenesis, certain tissues are colonized by mutant clones with oncogenic driver mutations as precancer lesions. These mutations can facilitate clonal expansion and may contribute to malignant transformation. The molecular features of low-grade non-muscle invasive bladder cancer (NMIBC) and high-grade bladder cancer are so distinct that they are thought to follow different evolutionary tumorigenesis pathways. Although NMIBC accounts for most bladder tumors, the somatic mutation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198542"
                        open
                      >
                        <summary class="article-expander__title">Treatment strategies for multiple myeloma based on molecular pathogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198542/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221011214956&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It is well documented that multiple myeloma (MM) originates in a single plasma cell transformed by chromosome 14q translocations or chromosomal hyperdiploidy and evolves with the accumulation of point mutations of driver genes and/or cytogenetic abnormalities. Furthermore, disease progression is accomplished by branching patterns of subclonal evolution from reservoir clones with a propagating potential and/or the emergence of minor clones, which already exist at premalignant stages and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36199157"
                        open
                      >
                        <summary class="article-expander__title">Determination of ERG(+), EZH2, NKX3.1, and SPINK-1 subtypes to evaluate their association with clonal origin and disease progression in multifocal prostate cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36199157/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221010212330&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The predominance of molecular discordance in prostatic foci per patient, which reflects the multifocal origin of PCa foci, highlights the importance of analyzing multiple samples to establish the prognostic and therapeutic relevance of molecular subtypes in a patient. All the subtypes analyzed here are of early onset, starting from preneoplastic lesions. NKX3.1 gene expression is the only molecular characteristic that shows a progression pattern by sample.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198773"
                        open
                      >
                        <summary class="article-expander__title">Targeted-sequence of normal urothelium and tumor of patients with non-muscle invasive bladder cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198773/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221010212330&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>During tumorigenesis, certain tissues are colonized by mutant clones with oncogenic driver mutations as precancer lesions. These mutations can facilitate clonal expansion and may contribute to malignant transformation. The molecular features of low-grade non-muscle invasive bladder cancer (NMIBC) and high-grade bladder cancer are so distinct that they are thought to follow different evolutionary tumorigenesis pathways. Although NMIBC accounts for most bladder tumors, the somatic mutation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36199157"
                        open
                      >
                        <summary class="article-expander__title">Determination of ERG(+), EZH2, NKX3.1, and SPINK-1 subtypes to evaluate their association with clonal origin and disease progression in multifocal prostate cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36199157/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221009214149&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The predominance of molecular discordance in prostatic foci per patient, which reflects the multifocal origin of PCa foci, highlights the importance of analyzing multiple samples to establish the prognostic and therapeutic relevance of molecular subtypes in a patient. All the subtypes analyzed here are of early onset, starting from preneoplastic lesions. NKX3.1 gene expression is the only molecular characteristic that shows a progression pattern by sample.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198773"
                        open
                      >
                        <summary class="article-expander__title">Targeted-sequence of normal urothelium and tumor of patients with non-muscle invasive bladder cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198773/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221009214149&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>During tumorigenesis, certain tissues are colonized by mutant clones with oncogenic driver mutations as precancer lesions. These mutations can facilitate clonal expansion and may contribute to malignant transformation. The molecular features of low-grade non-muscle invasive bladder cancer (NMIBC) and high-grade bladder cancer are so distinct that they are thought to follow different evolutionary tumorigenesis pathways. Although NMIBC accounts for most bladder tumors, the somatic mutation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36199157"
                        open
                      >
                        <summary class="article-expander__title">Determination of ERG(+), EZH2, NKX3.1, and SPINK-1 subtypes to evaluate their association with clonal origin and disease progression in multifocal prostate cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36199157/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221008212536&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The predominance of molecular discordance in prostatic foci per patient, which reflects the multifocal origin of PCa foci, highlights the importance of analyzing multiple samples to establish the prognostic and therapeutic relevance of molecular subtypes in a patient. All the subtypes analyzed here are of early onset, starting from preneoplastic lesions. NKX3.1 gene expression is the only molecular characteristic that shows a progression pattern by sample.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198773"
                        open
                      >
                        <summary class="article-expander__title">Targeted-sequence of normal urothelium and tumor of patients with non-muscle invasive bladder cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198773/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221008212536&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>During tumorigenesis, certain tissues are colonized by mutant clones with oncogenic driver mutations as precancer lesions. These mutations can facilitate clonal expansion and may contribute to malignant transformation. The molecular features of low-grade non-muscle invasive bladder cancer (NMIBC) and high-grade bladder cancer are so distinct that they are thought to follow different evolutionary tumorigenesis pathways. Although NMIBC accounts for most bladder tumors, the somatic mutation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36199157"
                        open
                      >
                        <summary class="article-expander__title">Determination of ERG(+), EZH2, NKX3.1, and SPINK-1 subtypes to evaluate their association with clonal origin and disease progression in multifocal prostate cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36199157/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221007211807&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The predominance of molecular discordance in prostatic foci per patient, which reflects the multifocal origin of PCa foci, highlights the importance of analyzing multiple samples to establish the prognostic and therapeutic relevance of molecular subtypes in a patient. All the subtypes analyzed here are of early onset, starting from preneoplastic lesions. NKX3.1 gene expression is the only molecular characteristic that shows a progression pattern by sample.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198773"
                        open
                      >
                        <summary class="article-expander__title">Targeted-sequence of normal urothelium and tumor of patients with non-muscle invasive bladder cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198773/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221007211807&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>During tumorigenesis, certain tissues are colonized by mutant clones with oncogenic driver mutations as precancer lesions. These mutations can facilitate clonal expansion and may contribute to malignant transformation. The molecular features of low-grade non-muscle invasive bladder cancer (NMIBC) and high-grade bladder cancer are so distinct that they are thought to follow different evolutionary tumorigenesis pathways. Although NMIBC accounts for most bladder tumors, the somatic mutation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36199157"
                        open
                      >
                        <summary class="article-expander__title">Determination of ERG(+), EZH2, NKX3.1, and SPINK-1 subtypes to evaluate their association with clonal origin and disease progression in multifocal prostate cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36199157/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221006214146&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The predominance of molecular discordance in prostatic foci per patient, which reflects the multifocal origin of PCa foci, highlights the importance of analyzing multiple samples to establish the prognostic and therapeutic relevance of molecular subtypes in a patient. All the subtypes analyzed here are of early onset, starting from preneoplastic lesions. NKX3.1 gene expression is the only molecular characteristic that shows a progression pattern by sample.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36198773"
                        open
                      >
                        <summary class="article-expander__title">Targeted-sequence of normal urothelium and tumor of patients with non-muscle invasive bladder cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36198773/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221006214146&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>During tumorigenesis, certain tissues are colonized by mutant clones with oncogenic driver mutations as precancer lesions. These mutations can facilitate clonal expansion and may contribute to malignant transformation. The molecular features of low-grade non-muscle invasive bladder cancer (NMIBC) and high-grade bladder cancer are so distinct that they are thought to follow different evolutionary tumorigenesis pathways. Although NMIBC accounts for most bladder tumors, the somatic mutation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-03 datetime="2022-10-03T10:00:00.000Z">2022-10-03</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-03 datetime="2022-10-03T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.8&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20221128200439&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36190670"
                        open
                      >
                        <summary class="article-expander__title">Pathogenic Mechanisms in Acute Myeloid Leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36190670/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221017214255&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is the most common form of leukemia in adults, leading to the highest number of annual leukemia-associated deaths in the USA. Although most AML patients initially enter remission following induction therapy, most eventually relapse, underscoring the unmet need for more effective therapies. In recent years, novel high-throughput sequencing techniques, and mouse and human models of disease have increased our understanding of the molecular mechanisms that lead to AML....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36188489"
                        open
                      >
                        <summary class="article-expander__title">Cancer and aging: A call to action</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36188489/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221017214255&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Research is needed to accelerate knowledge regarding the dynamic interplay of cancer and aging and optimize care in diverse older adults to achieve equity in cancer outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36190670"
                        open
                      >
                        <summary class="article-expander__title">Pathogenic Mechanisms in Acute Myeloid Leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36190670/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221016214939&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is the most common form of leukemia in adults, leading to the highest number of annual leukemia-associated deaths in the USA. Although most AML patients initially enter remission following induction therapy, most eventually relapse, underscoring the unmet need for more effective therapies. In recent years, novel high-throughput sequencing techniques, and mouse and human models of disease have increased our understanding of the molecular mechanisms that lead to AML....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36188489"
                        open
                      >
                        <summary class="article-expander__title">Cancer and aging: A call to action</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36188489/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221016214939&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Research is needed to accelerate knowledge regarding the dynamic interplay of cancer and aging and optimize care in diverse older adults to achieve equity in cancer outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36190670"
                        open
                      >
                        <summary class="article-expander__title">Pathogenic Mechanisms in Acute Myeloid Leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36190670/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221015214355&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is the most common form of leukemia in adults, leading to the highest number of annual leukemia-associated deaths in the USA. Although most AML patients initially enter remission following induction therapy, most eventually relapse, underscoring the unmet need for more effective therapies. In recent years, novel high-throughput sequencing techniques, and mouse and human models of disease have increased our understanding of the molecular mechanisms that lead to AML....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36188489"
                        open
                      >
                        <summary class="article-expander__title">Cancer and aging: A call to action</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36188489/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221015214355&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Research is needed to accelerate knowledge regarding the dynamic interplay of cancer and aging and optimize care in diverse older adults to achieve equity in cancer outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36190670"
                        open
                      >
                        <summary class="article-expander__title">Pathogenic Mechanisms in Acute Myeloid Leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36190670/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221014214034&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is the most common form of leukemia in adults, leading to the highest number of annual leukemia-associated deaths in the USA. Although most AML patients initially enter remission following induction therapy, most eventually relapse, underscoring the unmet need for more effective therapies. In recent years, novel high-throughput sequencing techniques, and mouse and human models of disease have increased our understanding of the molecular mechanisms that lead to AML....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36188489"
                        open
                      >
                        <summary class="article-expander__title">Cancer and aging: A call to action</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36188489/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221014214034&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Research is needed to accelerate knowledge regarding the dynamic interplay of cancer and aging and optimize care in diverse older adults to achieve equity in cancer outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36190670"
                        open
                      >
                        <summary class="article-expander__title">Pathogenic Mechanisms in Acute Myeloid Leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36190670/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221013214955&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is the most common form of leukemia in adults, leading to the highest number of annual leukemia-associated deaths in the USA. Although most AML patients initially enter remission following induction therapy, most eventually relapse, underscoring the unmet need for more effective therapies. In recent years, novel high-throughput sequencing techniques, and mouse and human models of disease have increased our understanding of the molecular mechanisms that lead to AML....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36188489"
                        open
                      >
                        <summary class="article-expander__title">Cancer and aging: A call to action</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36188489/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221013214955&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Research is needed to accelerate knowledge regarding the dynamic interplay of cancer and aging and optimize care in diverse older adults to achieve equity in cancer outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36190670"
                        open
                      >
                        <summary class="article-expander__title">Pathogenic Mechanisms in Acute Myeloid Leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36190670/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221012213728&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is the most common form of leukemia in adults, leading to the highest number of annual leukemia-associated deaths in the USA. Although most AML patients initially enter remission following induction therapy, most eventually relapse, underscoring the unmet need for more effective therapies. In recent years, novel high-throughput sequencing techniques, and mouse and human models of disease have increased our understanding of the molecular mechanisms that lead to AML....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36188489"
                        open
                      >
                        <summary class="article-expander__title">Cancer and aging: A call to action</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36188489/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221012213728&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Research is needed to accelerate knowledge regarding the dynamic interplay of cancer and aging and optimize care in diverse older adults to achieve equity in cancer outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36190670"
                        open
                      >
                        <summary class="article-expander__title">Pathogenic Mechanisms in Acute Myeloid Leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36190670/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221011214956&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is the most common form of leukemia in adults, leading to the highest number of annual leukemia-associated deaths in the USA. Although most AML patients initially enter remission following induction therapy, most eventually relapse, underscoring the unmet need for more effective therapies. In recent years, novel high-throughput sequencing techniques, and mouse and human models of disease have increased our understanding of the molecular mechanisms that lead to AML....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36188489"
                        open
                      >
                        <summary class="article-expander__title">Cancer and aging: A call to action</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36188489/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221011214956&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Research is needed to accelerate knowledge regarding the dynamic interplay of cancer and aging and optimize care in diverse older adults to achieve equity in cancer outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36190670"
                        open
                      >
                        <summary class="article-expander__title">Pathogenic Mechanisms in Acute Myeloid Leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36190670/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221010212330&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is the most common form of leukemia in adults, leading to the highest number of annual leukemia-associated deaths in the USA. Although most AML patients initially enter remission following induction therapy, most eventually relapse, underscoring the unmet need for more effective therapies. In recent years, novel high-throughput sequencing techniques, and mouse and human models of disease have increased our understanding of the molecular mechanisms that lead to AML....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36188489"
                        open
                      >
                        <summary class="article-expander__title">Cancer and aging: A call to action</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36188489/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221010212330&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Research is needed to accelerate knowledge regarding the dynamic interplay of cancer and aging and optimize care in diverse older adults to achieve equity in cancer outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36190670"
                        open
                      >
                        <summary class="article-expander__title">Pathogenic Mechanisms in Acute Myeloid Leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36190670/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221009214149&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is the most common form of leukemia in adults, leading to the highest number of annual leukemia-associated deaths in the USA. Although most AML patients initially enter remission following induction therapy, most eventually relapse, underscoring the unmet need for more effective therapies. In recent years, novel high-throughput sequencing techniques, and mouse and human models of disease have increased our understanding of the molecular mechanisms that lead to AML....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36188489"
                        open
                      >
                        <summary class="article-expander__title">Cancer and aging: A call to action</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36188489/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221009214149&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Research is needed to accelerate knowledge regarding the dynamic interplay of cancer and aging and optimize care in diverse older adults to achieve equity in cancer outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36190670"
                        open
                      >
                        <summary class="article-expander__title">Pathogenic Mechanisms in Acute Myeloid Leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36190670/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221008212536&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is the most common form of leukemia in adults, leading to the highest number of annual leukemia-associated deaths in the USA. Although most AML patients initially enter remission following induction therapy, most eventually relapse, underscoring the unmet need for more effective therapies. In recent years, novel high-throughput sequencing techniques, and mouse and human models of disease have increased our understanding of the molecular mechanisms that lead to AML....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36188489"
                        open
                      >
                        <summary class="article-expander__title">Cancer and aging: A call to action</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36188489/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221008212536&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Research is needed to accelerate knowledge regarding the dynamic interplay of cancer and aging and optimize care in diverse older adults to achieve equity in cancer outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36190670"
                        open
                      >
                        <summary class="article-expander__title">Pathogenic Mechanisms in Acute Myeloid Leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36190670/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221007211807&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is the most common form of leukemia in adults, leading to the highest number of annual leukemia-associated deaths in the USA. Although most AML patients initially enter remission following induction therapy, most eventually relapse, underscoring the unmet need for more effective therapies. In recent years, novel high-throughput sequencing techniques, and mouse and human models of disease have increased our understanding of the molecular mechanisms that lead to AML....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36188489"
                        open
                      >
                        <summary class="article-expander__title">Cancer and aging: A call to action</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36188489/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221007211807&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Research is needed to accelerate knowledge regarding the dynamic interplay of cancer and aging and optimize care in diverse older adults to achieve equity in cancer outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36190670"
                        open
                      >
                        <summary class="article-expander__title">Pathogenic Mechanisms in Acute Myeloid Leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36190670/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221006214146&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is the most common form of leukemia in adults, leading to the highest number of annual leukemia-associated deaths in the USA. Although most AML patients initially enter remission following induction therapy, most eventually relapse, underscoring the unmet need for more effective therapies. In recent years, novel high-throughput sequencing techniques, and mouse and human models of disease have increased our understanding of the molecular mechanisms that lead to AML....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36188489"
                        open
                      >
                        <summary class="article-expander__title">Cancer and aging: A call to action</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36188489/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221006214146&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Research is needed to accelerate knowledge regarding the dynamic interplay of cancer and aging and optimize care in diverse older adults to achieve equity in cancer outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36190670"
                        open
                      >
                        <summary class="article-expander__title">Pathogenic Mechanisms in Acute Myeloid Leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36190670/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221005212152&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is the most common form of leukemia in adults, leading to the highest number of annual leukemia-associated deaths in the USA. Although most AML patients initially enter remission following induction therapy, most eventually relapse, underscoring the unmet need for more effective therapies. In recent years, novel high-throughput sequencing techniques, and mouse and human models of disease have increased our understanding of the molecular mechanisms that lead to AML....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36188489"
                        open
                      >
                        <summary class="article-expander__title">Cancer and aging: A call to action</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36188489/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221005212152&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Research is needed to accelerate knowledge regarding the dynamic interplay of cancer and aging and optimize care in diverse older adults to achieve equity in cancer outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36190670"
                        open
                      >
                        <summary class="article-expander__title">Pathogenic Mechanisms in Acute Myeloid Leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36190670/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221004212702&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is the most common form of leukemia in adults, leading to the highest number of annual leukemia-associated deaths in the USA. Although most AML patients initially enter remission following induction therapy, most eventually relapse, underscoring the unmet need for more effective therapies. In recent years, novel high-throughput sequencing techniques, and mouse and human models of disease have increased our understanding of the molecular mechanisms that lead to AML....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36188489"
                        open
                      >
                        <summary class="article-expander__title">Cancer and aging: A call to action</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36188489/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221004212702&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Research is needed to accelerate knowledge regarding the dynamic interplay of cancer and aging and optimize care in diverse older adults to achieve equity in cancer outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36190670"
                        open
                      >
                        <summary class="article-expander__title">Pathogenic Mechanisms in Acute Myeloid Leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36190670/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221003213836&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is the most common form of leukemia in adults, leading to the highest number of annual leukemia-associated deaths in the USA. Although most AML patients initially enter remission following induction therapy, most eventually relapse, underscoring the unmet need for more effective therapies. In recent years, novel high-throughput sequencing techniques, and mouse and human models of disease have increased our understanding of the molecular mechanisms that lead to AML....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36188489"
                        open
                      >
                        <summary class="article-expander__title">Cancer and aging: A call to action</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36188489/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221003213836&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Research is needed to accelerate knowledge regarding the dynamic interplay of cancer and aging and optimize care in diverse older adults to achieve equity in cancer outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221128200439&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.8">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36186629"
                        open
                      >
                        <summary class="article-expander__title">Potential role of MAP2K1 mutation in the trans-differentiation of interdigitating dendritic cell sarcoma: Case report and literature review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36186629/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221025212118&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A 5-year-old male child was diagnosed with interdigitating dendritic cell sarcoma (IDCS) during his maintenance therapy for B-cell precursor acute lymphoblastic leukemia (B-ALL). Multiplex lymph node involvements of the neck were found by positron emission tomography CT (PET-CT). Treatments, including surgical and chemotherapy, resulted in complete remission. Four years later, systemic bone infiltration was discovered. Surgical resection of the IV rib and intensive chemotherapy led to a complete...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36186629"
                        open
                      >
                        <summary class="article-expander__title">Potential role of MAP2K1 mutation in the trans-differentiation of interdigitating dendritic cell sarcoma: Case report and literature review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36186629/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221024214453&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A 5-year-old male child was diagnosed with interdigitating dendritic cell sarcoma (IDCS) during his maintenance therapy for B-cell precursor acute lymphoblastic leukemia (B-ALL). Multiplex lymph node involvements of the neck were found by positron emission tomography CT (PET-CT). Treatments, including surgical and chemotherapy, resulted in complete remission. Four years later, systemic bone infiltration was discovered. Surgical resection of the IV rib and intensive chemotherapy led to a complete...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36186629"
                        open
                      >
                        <summary class="article-expander__title">Potential role of MAP2K1 mutation in the trans-differentiation of interdigitating dendritic cell sarcoma: Case report and literature review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36186629/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221023215029&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A 5-year-old male child was diagnosed with interdigitating dendritic cell sarcoma (IDCS) during his maintenance therapy for B-cell precursor acute lymphoblastic leukemia (B-ALL). Multiplex lymph node involvements of the neck were found by positron emission tomography CT (PET-CT). Treatments, including surgical and chemotherapy, resulted in complete remission. Four years later, systemic bone infiltration was discovered. Surgical resection of the IV rib and intensive chemotherapy led to a complete...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36186629"
                        open
                      >
                        <summary class="article-expander__title">Potential role of MAP2K1 mutation in the trans-differentiation of interdigitating dendritic cell sarcoma: Case report and literature review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36186629/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221022213551&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A 5-year-old male child was diagnosed with interdigitating dendritic cell sarcoma (IDCS) during his maintenance therapy for B-cell precursor acute lymphoblastic leukemia (B-ALL). Multiplex lymph node involvements of the neck were found by positron emission tomography CT (PET-CT). Treatments, including surgical and chemotherapy, resulted in complete remission. Four years later, systemic bone infiltration was discovered. Surgical resection of the IV rib and intensive chemotherapy led to a complete...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36186629"
                        open
                      >
                        <summary class="article-expander__title">Potential role of MAP2K1 mutation in the trans-differentiation of interdigitating dendritic cell sarcoma: Case report and literature review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36186629/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221021213516&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A 5-year-old male child was diagnosed with interdigitating dendritic cell sarcoma (IDCS) during his maintenance therapy for B-cell precursor acute lymphoblastic leukemia (B-ALL). Multiplex lymph node involvements of the neck were found by positron emission tomography CT (PET-CT). Treatments, including surgical and chemotherapy, resulted in complete remission. Four years later, systemic bone infiltration was discovered. Surgical resection of the IV rib and intensive chemotherapy led to a complete...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36185218"
                        open
                      >
                        <summary class="article-expander__title">Mutational signature and clonal relatedness of recurrent urothelial carcinomas with aristolochic acid</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36185218/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221021213516&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Urothelial carcinomas (UCs) are malignant tumors that arise from the lower and upper urinary tract and are characterized by multiple recurrences. Aristolochic acid (AA) is a potent nephrotoxin and human carcinogen associated with UC. East Asian populations with a high UC prevalence have an unusual genome-wide AA-induced mutational pattern. To address the genomic differences and clonal relatedness between primary and recurrent tumors in the UCs with AA pattern, we investigated the genomic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36186629"
                        open
                      >
                        <summary class="article-expander__title">Potential role of MAP2K1 mutation in the trans-differentiation of interdigitating dendritic cell sarcoma: Case report and literature review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36186629/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221020212123&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A 5-year-old male child was diagnosed with interdigitating dendritic cell sarcoma (IDCS) during his maintenance therapy for B-cell precursor acute lymphoblastic leukemia (B-ALL). Multiplex lymph node involvements of the neck were found by positron emission tomography CT (PET-CT). Treatments, including surgical and chemotherapy, resulted in complete remission. Four years later, systemic bone infiltration was discovered. Surgical resection of the IV rib and intensive chemotherapy led to a complete...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36185218"
                        open
                      >
                        <summary class="article-expander__title">Mutational signature and clonal relatedness of recurrent urothelial carcinomas with aristolochic acid</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36185218/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221020212123&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Urothelial carcinomas (UCs) are malignant tumors that arise from the lower and upper urinary tract and are characterized by multiple recurrences. Aristolochic acid (AA) is a potent nephrotoxin and human carcinogen associated with UC. East Asian populations with a high UC prevalence have an unusual genome-wide AA-induced mutational pattern. To address the genomic differences and clonal relatedness between primary and recurrent tumors in the UCs with AA pattern, we investigated the genomic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36186629"
                        open
                      >
                        <summary class="article-expander__title">Potential role of MAP2K1 mutation in the trans-differentiation of interdigitating dendritic cell sarcoma: Case report and literature review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36186629/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221019212642&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A 5-year-old male child was diagnosed with interdigitating dendritic cell sarcoma (IDCS) during his maintenance therapy for B-cell precursor acute lymphoblastic leukemia (B-ALL). Multiplex lymph node involvements of the neck were found by positron emission tomography CT (PET-CT). Treatments, including surgical and chemotherapy, resulted in complete remission. Four years later, systemic bone infiltration was discovered. Surgical resection of the IV rib and intensive chemotherapy led to a complete...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36185218"
                        open
                      >
                        <summary class="article-expander__title">Mutational signature and clonal relatedness of recurrent urothelial carcinomas with aristolochic acid</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36185218/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221019212642&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Urothelial carcinomas (UCs) are malignant tumors that arise from the lower and upper urinary tract and are characterized by multiple recurrences. Aristolochic acid (AA) is a potent nephrotoxin and human carcinogen associated with UC. East Asian populations with a high UC prevalence have an unusual genome-wide AA-induced mutational pattern. To address the genomic differences and clonal relatedness between primary and recurrent tumors in the UCs with AA pattern, we investigated the genomic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36186629"
                        open
                      >
                        <summary class="article-expander__title">Potential role of MAP2K1 mutation in the trans-differentiation of interdigitating dendritic cell sarcoma: Case report and literature review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36186629/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221018213749&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A 5-year-old male child was diagnosed with interdigitating dendritic cell sarcoma (IDCS) during his maintenance therapy for B-cell precursor acute lymphoblastic leukemia (B-ALL). Multiplex lymph node involvements of the neck were found by positron emission tomography CT (PET-CT). Treatments, including surgical and chemotherapy, resulted in complete remission. Four years later, systemic bone infiltration was discovered. Surgical resection of the IV rib and intensive chemotherapy led to a complete...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36185218"
                        open
                      >
                        <summary class="article-expander__title">Mutational signature and clonal relatedness of recurrent urothelial carcinomas with aristolochic acid</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36185218/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221018213749&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Urothelial carcinomas (UCs) are malignant tumors that arise from the lower and upper urinary tract and are characterized by multiple recurrences. Aristolochic acid (AA) is a potent nephrotoxin and human carcinogen associated with UC. East Asian populations with a high UC prevalence have an unusual genome-wide AA-induced mutational pattern. To address the genomic differences and clonal relatedness between primary and recurrent tumors in the UCs with AA pattern, we investigated the genomic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36186629"
                        open
                      >
                        <summary class="article-expander__title">Potential role of MAP2K1 mutation in the trans-differentiation of interdigitating dendritic cell sarcoma: Case report and literature review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36186629/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221017214255&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A 5-year-old male child was diagnosed with interdigitating dendritic cell sarcoma (IDCS) during his maintenance therapy for B-cell precursor acute lymphoblastic leukemia (B-ALL). Multiplex lymph node involvements of the neck were found by positron emission tomography CT (PET-CT). Treatments, including surgical and chemotherapy, resulted in complete remission. Four years later, systemic bone infiltration was discovered. Surgical resection of the IV rib and intensive chemotherapy led to a complete...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36185218"
                        open
                      >
                        <summary class="article-expander__title">Mutational signature and clonal relatedness of recurrent urothelial carcinomas with aristolochic acid</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36185218/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221017214255&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Urothelial carcinomas (UCs) are malignant tumors that arise from the lower and upper urinary tract and are characterized by multiple recurrences. Aristolochic acid (AA) is a potent nephrotoxin and human carcinogen associated with UC. East Asian populations with a high UC prevalence have an unusual genome-wide AA-induced mutational pattern. To address the genomic differences and clonal relatedness between primary and recurrent tumors in the UCs with AA pattern, we investigated the genomic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36186629"
                        open
                      >
                        <summary class="article-expander__title">Potential role of MAP2K1 mutation in the trans-differentiation of interdigitating dendritic cell sarcoma: Case report and literature review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36186629/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221016214941&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A 5-year-old male child was diagnosed with interdigitating dendritic cell sarcoma (IDCS) during his maintenance therapy for B-cell precursor acute lymphoblastic leukemia (B-ALL). Multiplex lymph node involvements of the neck were found by positron emission tomography CT (PET-CT). Treatments, including surgical and chemotherapy, resulted in complete remission. Four years later, systemic bone infiltration was discovered. Surgical resection of the IV rib and intensive chemotherapy led to a complete...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36185218"
                        open
                      >
                        <summary class="article-expander__title">Mutational signature and clonal relatedness of recurrent urothelial carcinomas with aristolochic acid</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36185218/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221016214941&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Urothelial carcinomas (UCs) are malignant tumors that arise from the lower and upper urinary tract and are characterized by multiple recurrences. Aristolochic acid (AA) is a potent nephrotoxin and human carcinogen associated with UC. East Asian populations with a high UC prevalence have an unusual genome-wide AA-induced mutational pattern. To address the genomic differences and clonal relatedness between primary and recurrent tumors in the UCs with AA pattern, we investigated the genomic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36186629"
                        open
                      >
                        <summary class="article-expander__title">Potential role of MAP2K1 mutation in the trans-differentiation of interdigitating dendritic cell sarcoma: Case report and literature review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36186629/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221015214357&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A 5-year-old male child was diagnosed with interdigitating dendritic cell sarcoma (IDCS) during his maintenance therapy for B-cell precursor acute lymphoblastic leukemia (B-ALL). Multiplex lymph node involvements of the neck were found by positron emission tomography CT (PET-CT). Treatments, including surgical and chemotherapy, resulted in complete remission. Four years later, systemic bone infiltration was discovered. Surgical resection of the IV rib and intensive chemotherapy led to a complete...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36185218"
                        open
                      >
                        <summary class="article-expander__title">Mutational signature and clonal relatedness of recurrent urothelial carcinomas with aristolochic acid</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36185218/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221015214357&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Urothelial carcinomas (UCs) are malignant tumors that arise from the lower and upper urinary tract and are characterized by multiple recurrences. Aristolochic acid (AA) is a potent nephrotoxin and human carcinogen associated with UC. East Asian populations with a high UC prevalence have an unusual genome-wide AA-induced mutational pattern. To address the genomic differences and clonal relatedness between primary and recurrent tumors in the UCs with AA pattern, we investigated the genomic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36186629"
                        open
                      >
                        <summary class="article-expander__title">Potential role of MAP2K1 mutation in the trans-differentiation of interdigitating dendritic cell sarcoma: Case report and literature review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36186629/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221014214034&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A 5-year-old male child was diagnosed with interdigitating dendritic cell sarcoma (IDCS) during his maintenance therapy for B-cell precursor acute lymphoblastic leukemia (B-ALL). Multiplex lymph node involvements of the neck were found by positron emission tomography CT (PET-CT). Treatments, including surgical and chemotherapy, resulted in complete remission. Four years later, systemic bone infiltration was discovered. Surgical resection of the IV rib and intensive chemotherapy led to a complete...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36185218"
                        open
                      >
                        <summary class="article-expander__title">Mutational signature and clonal relatedness of recurrent urothelial carcinomas with aristolochic acid</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36185218/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221014214034&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Urothelial carcinomas (UCs) are malignant tumors that arise from the lower and upper urinary tract and are characterized by multiple recurrences. Aristolochic acid (AA) is a potent nephrotoxin and human carcinogen associated with UC. East Asian populations with a high UC prevalence have an unusual genome-wide AA-induced mutational pattern. To address the genomic differences and clonal relatedness between primary and recurrent tumors in the UCs with AA pattern, we investigated the genomic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36186629"
                        open
                      >
                        <summary class="article-expander__title">Potential role of MAP2K1 mutation in the trans-differentiation of interdigitating dendritic cell sarcoma: Case report and literature review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36186629/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221013214955&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A 5-year-old male child was diagnosed with interdigitating dendritic cell sarcoma (IDCS) during his maintenance therapy for B-cell precursor acute lymphoblastic leukemia (B-ALL). Multiplex lymph node involvements of the neck were found by positron emission tomography CT (PET-CT). Treatments, including surgical and chemotherapy, resulted in complete remission. Four years later, systemic bone infiltration was discovered. Surgical resection of the IV rib and intensive chemotherapy led to a complete...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36185218"
                        open
                      >
                        <summary class="article-expander__title">Mutational signature and clonal relatedness of recurrent urothelial carcinomas with aristolochic acid</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36185218/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221013214955&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Urothelial carcinomas (UCs) are malignant tumors that arise from the lower and upper urinary tract and are characterized by multiple recurrences. Aristolochic acid (AA) is a potent nephrotoxin and human carcinogen associated with UC. East Asian populations with a high UC prevalence have an unusual genome-wide AA-induced mutational pattern. To address the genomic differences and clonal relatedness between primary and recurrent tumors in the UCs with AA pattern, we investigated the genomic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36186629"
                        open
                      >
                        <summary class="article-expander__title">Potential role of MAP2K1 mutation in the trans-differentiation of interdigitating dendritic cell sarcoma: Case report and literature review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36186629/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221012213728&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A 5-year-old male child was diagnosed with interdigitating dendritic cell sarcoma (IDCS) during his maintenance therapy for B-cell precursor acute lymphoblastic leukemia (B-ALL). Multiplex lymph node involvements of the neck were found by positron emission tomography CT (PET-CT). Treatments, including surgical and chemotherapy, resulted in complete remission. Four years later, systemic bone infiltration was discovered. Surgical resection of the IV rib and intensive chemotherapy led to a complete...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36185218"
                        open
                      >
                        <summary class="article-expander__title">Mutational signature and clonal relatedness of recurrent urothelial carcinomas with aristolochic acid</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36185218/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221012213728&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Urothelial carcinomas (UCs) are malignant tumors that arise from the lower and upper urinary tract and are characterized by multiple recurrences. Aristolochic acid (AA) is a potent nephrotoxin and human carcinogen associated with UC. East Asian populations with a high UC prevalence have an unusual genome-wide AA-induced mutational pattern. To address the genomic differences and clonal relatedness between primary and recurrent tumors in the UCs with AA pattern, we investigated the genomic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36186629"
                        open
                      >
                        <summary class="article-expander__title">Potential role of MAP2K1 mutation in the trans-differentiation of interdigitating dendritic cell sarcoma: Case report and literature review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36186629/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221011214956&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A 5-year-old male child was diagnosed with interdigitating dendritic cell sarcoma (IDCS) during his maintenance therapy for B-cell precursor acute lymphoblastic leukemia (B-ALL). Multiplex lymph node involvements of the neck were found by positron emission tomography CT (PET-CT). Treatments, including surgical and chemotherapy, resulted in complete remission. Four years later, systemic bone infiltration was discovered. Surgical resection of the IV rib and intensive chemotherapy led to a complete...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36185218"
                        open
                      >
                        <summary class="article-expander__title">Mutational signature and clonal relatedness of recurrent urothelial carcinomas with aristolochic acid</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36185218/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221011214956&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Urothelial carcinomas (UCs) are malignant tumors that arise from the lower and upper urinary tract and are characterized by multiple recurrences. Aristolochic acid (AA) is a potent nephrotoxin and human carcinogen associated with UC. East Asian populations with a high UC prevalence have an unusual genome-wide AA-induced mutational pattern. To address the genomic differences and clonal relatedness between primary and recurrent tumors in the UCs with AA pattern, we investigated the genomic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36186629"
                        open
                      >
                        <summary class="article-expander__title">Potential role of MAP2K1 mutation in the trans-differentiation of interdigitating dendritic cell sarcoma: Case report and literature review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36186629/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221010212330&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A 5-year-old male child was diagnosed with interdigitating dendritic cell sarcoma (IDCS) during his maintenance therapy for B-cell precursor acute lymphoblastic leukemia (B-ALL). Multiplex lymph node involvements of the neck were found by positron emission tomography CT (PET-CT). Treatments, including surgical and chemotherapy, resulted in complete remission. Four years later, systemic bone infiltration was discovered. Surgical resection of the IV rib and intensive chemotherapy led to a complete...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36185218"
                        open
                      >
                        <summary class="article-expander__title">Mutational signature and clonal relatedness of recurrent urothelial carcinomas with aristolochic acid</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36185218/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221010212330&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Urothelial carcinomas (UCs) are malignant tumors that arise from the lower and upper urinary tract and are characterized by multiple recurrences. Aristolochic acid (AA) is a potent nephrotoxin and human carcinogen associated with UC. East Asian populations with a high UC prevalence have an unusual genome-wide AA-induced mutational pattern. To address the genomic differences and clonal relatedness between primary and recurrent tumors in the UCs with AA pattern, we investigated the genomic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36186629"
                        open
                      >
                        <summary class="article-expander__title">Potential role of MAP2K1 mutation in the trans-differentiation of interdigitating dendritic cell sarcoma: Case report and literature review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36186629/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221009214149&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A 5-year-old male child was diagnosed with interdigitating dendritic cell sarcoma (IDCS) during his maintenance therapy for B-cell precursor acute lymphoblastic leukemia (B-ALL). Multiplex lymph node involvements of the neck were found by positron emission tomography CT (PET-CT). Treatments, including surgical and chemotherapy, resulted in complete remission. Four years later, systemic bone infiltration was discovered. Surgical resection of the IV rib and intensive chemotherapy led to a complete...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36185218"
                        open
                      >
                        <summary class="article-expander__title">Mutational signature and clonal relatedness of recurrent urothelial carcinomas with aristolochic acid</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36185218/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221009214149&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Urothelial carcinomas (UCs) are malignant tumors that arise from the lower and upper urinary tract and are characterized by multiple recurrences. Aristolochic acid (AA) is a potent nephrotoxin and human carcinogen associated with UC. East Asian populations with a high UC prevalence have an unusual genome-wide AA-induced mutational pattern. To address the genomic differences and clonal relatedness between primary and recurrent tumors in the UCs with AA pattern, we investigated the genomic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36186629"
                        open
                      >
                        <summary class="article-expander__title">Potential role of MAP2K1 mutation in the trans-differentiation of interdigitating dendritic cell sarcoma: Case report and literature review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36186629/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221008212536&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A 5-year-old male child was diagnosed with interdigitating dendritic cell sarcoma (IDCS) during his maintenance therapy for B-cell precursor acute lymphoblastic leukemia (B-ALL). Multiplex lymph node involvements of the neck were found by positron emission tomography CT (PET-CT). Treatments, including surgical and chemotherapy, resulted in complete remission. Four years later, systemic bone infiltration was discovered. Surgical resection of the IV rib and intensive chemotherapy led to a complete...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36185218"
                        open
                      >
                        <summary class="article-expander__title">Mutational signature and clonal relatedness of recurrent urothelial carcinomas with aristolochic acid</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36185218/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221008212536&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Urothelial carcinomas (UCs) are malignant tumors that arise from the lower and upper urinary tract and are characterized by multiple recurrences. Aristolochic acid (AA) is a potent nephrotoxin and human carcinogen associated with UC. East Asian populations with a high UC prevalence have an unusual genome-wide AA-induced mutational pattern. To address the genomic differences and clonal relatedness between primary and recurrent tumors in the UCs with AA pattern, we investigated the genomic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36186629"
                        open
                      >
                        <summary class="article-expander__title">Potential role of MAP2K1 mutation in the trans-differentiation of interdigitating dendritic cell sarcoma: Case report and literature review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36186629/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221007211807&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A 5-year-old male child was diagnosed with interdigitating dendritic cell sarcoma (IDCS) during his maintenance therapy for B-cell precursor acute lymphoblastic leukemia (B-ALL). Multiplex lymph node involvements of the neck were found by positron emission tomography CT (PET-CT). Treatments, including surgical and chemotherapy, resulted in complete remission. Four years later, systemic bone infiltration was discovered. Surgical resection of the IV rib and intensive chemotherapy led to a complete...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36185218"
                        open
                      >
                        <summary class="article-expander__title">Mutational signature and clonal relatedness of recurrent urothelial carcinomas with aristolochic acid</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36185218/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221007211807&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Urothelial carcinomas (UCs) are malignant tumors that arise from the lower and upper urinary tract and are characterized by multiple recurrences. Aristolochic acid (AA) is a potent nephrotoxin and human carcinogen associated with UC. East Asian populations with a high UC prevalence have an unusual genome-wide AA-induced mutational pattern. To address the genomic differences and clonal relatedness between primary and recurrent tumors in the UCs with AA pattern, we investigated the genomic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36186629"
                        open
                      >
                        <summary class="article-expander__title">Potential role of MAP2K1 mutation in the trans-differentiation of interdigitating dendritic cell sarcoma: Case report and literature review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36186629/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221006214146&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A 5-year-old male child was diagnosed with interdigitating dendritic cell sarcoma (IDCS) during his maintenance therapy for B-cell precursor acute lymphoblastic leukemia (B-ALL). Multiplex lymph node involvements of the neck were found by positron emission tomography CT (PET-CT). Treatments, including surgical and chemotherapy, resulted in complete remission. Four years later, systemic bone infiltration was discovered. Surgical resection of the IV rib and intensive chemotherapy led to a complete...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36185218"
                        open
                      >
                        <summary class="article-expander__title">Mutational signature and clonal relatedness of recurrent urothelial carcinomas with aristolochic acid</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36185218/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221006214146&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Urothelial carcinomas (UCs) are malignant tumors that arise from the lower and upper urinary tract and are characterized by multiple recurrences. Aristolochic acid (AA) is a potent nephrotoxin and human carcinogen associated with UC. East Asian populations with a high UC prevalence have an unusual genome-wide AA-induced mutational pattern. To address the genomic differences and clonal relatedness between primary and recurrent tumors in the UCs with AA pattern, we investigated the genomic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36186629"
                        open
                      >
                        <summary class="article-expander__title">Potential role of MAP2K1 mutation in the trans-differentiation of interdigitating dendritic cell sarcoma: Case report and literature review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36186629/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221005212152&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A 5-year-old male child was diagnosed with interdigitating dendritic cell sarcoma (IDCS) during his maintenance therapy for B-cell precursor acute lymphoblastic leukemia (B-ALL). Multiplex lymph node involvements of the neck were found by positron emission tomography CT (PET-CT). Treatments, including surgical and chemotherapy, resulted in complete remission. Four years later, systemic bone infiltration was discovered. Surgical resection of the IV rib and intensive chemotherapy led to a complete...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36185218"
                        open
                      >
                        <summary class="article-expander__title">Mutational signature and clonal relatedness of recurrent urothelial carcinomas with aristolochic acid</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36185218/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221005212152&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Urothelial carcinomas (UCs) are malignant tumors that arise from the lower and upper urinary tract and are characterized by multiple recurrences. Aristolochic acid (AA) is a potent nephrotoxin and human carcinogen associated with UC. East Asian populations with a high UC prevalence have an unusual genome-wide AA-induced mutational pattern. To address the genomic differences and clonal relatedness between primary and recurrent tumors in the UCs with AA pattern, we investigated the genomic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36186629"
                        open
                      >
                        <summary class="article-expander__title">Potential role of MAP2K1 mutation in the trans-differentiation of interdigitating dendritic cell sarcoma: Case report and literature review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36186629/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221004212702&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A 5-year-old male child was diagnosed with interdigitating dendritic cell sarcoma (IDCS) during his maintenance therapy for B-cell precursor acute lymphoblastic leukemia (B-ALL). Multiplex lymph node involvements of the neck were found by positron emission tomography CT (PET-CT). Treatments, including surgical and chemotherapy, resulted in complete remission. Four years later, systemic bone infiltration was discovered. Surgical resection of the IV rib and intensive chemotherapy led to a complete...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36185218"
                        open
                      >
                        <summary class="article-expander__title">Mutational signature and clonal relatedness of recurrent urothelial carcinomas with aristolochic acid</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36185218/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221004212702&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Urothelial carcinomas (UCs) are malignant tumors that arise from the lower and upper urinary tract and are characterized by multiple recurrences. Aristolochic acid (AA) is a potent nephrotoxin and human carcinogen associated with UC. East Asian populations with a high UC prevalence have an unusual genome-wide AA-induced mutational pattern. To address the genomic differences and clonal relatedness between primary and recurrent tumors in the UCs with AA pattern, we investigated the genomic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36186629"
                        open
                      >
                        <summary class="article-expander__title">Potential role of MAP2K1 mutation in the trans-differentiation of interdigitating dendritic cell sarcoma: Case report and literature review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36186629/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221003213836&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A 5-year-old male child was diagnosed with interdigitating dendritic cell sarcoma (IDCS) during his maintenance therapy for B-cell precursor acute lymphoblastic leukemia (B-ALL). Multiplex lymph node involvements of the neck were found by positron emission tomography CT (PET-CT). Treatments, including surgical and chemotherapy, resulted in complete remission. Four years later, systemic bone infiltration was discovered. Surgical resection of the IV rib and intensive chemotherapy led to a complete...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36185218"
                        open
                      >
                        <summary class="article-expander__title">Mutational signature and clonal relatedness of recurrent urothelial carcinomas with aristolochic acid</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36185218/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221003213836&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Urothelial carcinomas (UCs) are malignant tumors that arise from the lower and upper urinary tract and are characterized by multiple recurrences. Aristolochic acid (AA) is a potent nephrotoxin and human carcinogen associated with UC. East Asian populations with a high UC prevalence have an unusual genome-wide AA-induced mutational pattern. To address the genomic differences and clonal relatedness between primary and recurrent tumors in the UCs with AA pattern, we investigated the genomic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-02 datetime="2022-10-02T10:00:00.000Z">2022-10-02</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-02 datetime="2022-10-02T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.8&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20221128200439&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36184726"
                        open
                      >
                        <summary class="article-expander__title">Age-dependent association of clonal hematopoiesis with COVID-19 mortality in patients over 60years</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36184726/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221016214939&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis, especially that of indeterminate potential (CHIP), has been associated with age-related diseases, such as those contributing to a more severe COVID-19. Four studies have attempted to associate CHIP with COVID-19 severity without conclusive findings. In the present work, we explore the association between CHIP and COVID-19 mortality. Genomic DNA extracted from peripheral blood of COVID-19 patients (n &#x3D; 241 deceased, n &#x3D; 239 survivors) was sequenced with the Myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36184726"
                        open
                      >
                        <summary class="article-expander__title">Age-dependent association of clonal hematopoiesis with COVID-19 mortality in patients over 60years</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36184726/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221015214355&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis, especially that of indeterminate potential (CHIP), has been associated with age-related diseases, such as those contributing to a more severe COVID-19. Four studies have attempted to associate CHIP with COVID-19 severity without conclusive findings. In the present work, we explore the association between CHIP and COVID-19 mortality. Genomic DNA extracted from peripheral blood of COVID-19 patients (n &#x3D; 241 deceased, n &#x3D; 239 survivors) was sequenced with the Myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36184726"
                        open
                      >
                        <summary class="article-expander__title">Age-dependent association of clonal hematopoiesis with COVID-19 mortality in patients over 60years</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36184726/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221014214034&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis, especially that of indeterminate potential (CHIP), has been associated with age-related diseases, such as those contributing to a more severe COVID-19. Four studies have attempted to associate CHIP with COVID-19 severity without conclusive findings. In the present work, we explore the association between CHIP and COVID-19 mortality. Genomic DNA extracted from peripheral blood of COVID-19 patients (n &#x3D; 241 deceased, n &#x3D; 239 survivors) was sequenced with the Myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36183966"
                        open
                      >
                        <summary class="article-expander__title">Role of ASXL1 in hematopoiesis and myeloid diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36183966/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221014214034&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Next-generation sequencing technology, including whole-exome or whole-genome sequencing and target gene sequencing, has allowed the molecular characterization of somatic mutation spectrums in hematologic diseases. Mutations in Additional sex combs-like 1 (ASXL1), a chromatin regulator, are identified in clonal hematopoiesis of indeterminate potential (CHIP), indicating ASXL1 mutations as early events in leukemogenesis. Not surprisingly, they occur at high frequency in myeloid malignancies and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36184726"
                        open
                      >
                        <summary class="article-expander__title">Age-dependent association of clonal hematopoiesis with COVID-19 mortality in patients over 60years</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36184726/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221013214955&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis, especially that of indeterminate potential (CHIP), has been associated with age-related diseases, such as those contributing to a more severe COVID-19. Four studies have attempted to associate CHIP with COVID-19 severity without conclusive findings. In the present work, we explore the association between CHIP and COVID-19 mortality. Genomic DNA extracted from peripheral blood of COVID-19 patients (n &#x3D; 241 deceased, n &#x3D; 239 survivors) was sequenced with the Myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36183966"
                        open
                      >
                        <summary class="article-expander__title">Role of ASXL1 in hematopoiesis and myeloid diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36183966/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221013214955&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Next-generation sequencing technology, including whole-exome or whole-genome sequencing and target gene sequencing, has allowed the molecular characterization of somatic mutation spectrums in hematologic diseases. Mutations in Additional sex combs-like 1 (ASXL1), a chromatin regulator, are identified in clonal hematopoiesis of indeterminate potential (CHIP), indicating ASXL1 mutations as early events in leukemogenesis. Not surprisingly, they occur at high frequency in myeloid malignancies and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36184726"
                        open
                      >
                        <summary class="article-expander__title">Age-dependent association of clonal hematopoiesis with COVID-19 mortality in patients over 60years</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36184726/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221012213728&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis, especially that of indeterminate potential (CHIP), has been associated with age-related diseases, such as those contributing to a more severe COVID-19. Four studies have attempted to associate CHIP with COVID-19 severity without conclusive findings. In the present work, we explore the association between CHIP and COVID-19 mortality. Genomic DNA extracted from peripheral blood of COVID-19 patients (n &#x3D; 241 deceased, n &#x3D; 239 survivors) was sequenced with the Myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36183966"
                        open
                      >
                        <summary class="article-expander__title">Role of ASXL1 in hematopoiesis and myeloid diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36183966/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221012213728&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Next-generation sequencing technology, including whole-exome or whole-genome sequencing and target gene sequencing, has allowed the molecular characterization of somatic mutation spectrums in hematologic diseases. Mutations in Additional sex combs-like 1 (ASXL1), a chromatin regulator, are identified in clonal hematopoiesis of indeterminate potential (CHIP), indicating ASXL1 mutations as early events in leukemogenesis. Not surprisingly, they occur at high frequency in myeloid malignancies and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36184726"
                        open
                      >
                        <summary class="article-expander__title">Age-dependent association of clonal hematopoiesis with COVID-19 mortality in patients over 60years</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36184726/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221011214956&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis, especially that of indeterminate potential (CHIP), has been associated with age-related diseases, such as those contributing to a more severe COVID-19. Four studies have attempted to associate CHIP with COVID-19 severity without conclusive findings. In the present work, we explore the association between CHIP and COVID-19 mortality. Genomic DNA extracted from peripheral blood of COVID-19 patients (n &#x3D; 241 deceased, n &#x3D; 239 survivors) was sequenced with the Myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36183966"
                        open
                      >
                        <summary class="article-expander__title">Role of ASXL1 in hematopoiesis and myeloid diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36183966/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221011214956&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Next-generation sequencing technology, including whole-exome or whole-genome sequencing and target gene sequencing, has allowed the molecular characterization of somatic mutation spectrums in hematologic diseases. Mutations in Additional sex combs-like 1 (ASXL1), a chromatin regulator, are identified in clonal hematopoiesis of indeterminate potential (CHIP), indicating ASXL1 mutations as early events in leukemogenesis. Not surprisingly, they occur at high frequency in myeloid malignancies and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36184726"
                        open
                      >
                        <summary class="article-expander__title">Age-dependent association of clonal hematopoiesis with COVID-19 mortality in patients over 60years</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36184726/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221010212330&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis, especially that of indeterminate potential (CHIP), has been associated with age-related diseases, such as those contributing to a more severe COVID-19. Four studies have attempted to associate CHIP with COVID-19 severity without conclusive findings. In the present work, we explore the association between CHIP and COVID-19 mortality. Genomic DNA extracted from peripheral blood of COVID-19 patients (n &#x3D; 241 deceased, n &#x3D; 239 survivors) was sequenced with the Myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36183966"
                        open
                      >
                        <summary class="article-expander__title">Role of ASXL1 in Hematopoiesis and Myeloid Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36183966/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221010212330&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Next-generation sequencing technology (NGS), including whole-exome or whole-genome sequencing and target gene sequencing, has allowed the molecular characterization of somatic mutation spectrums in hematologic diseases. Mutations in Additional sex combs-like 1 (ASXL1), a chromatin regulator, are identified in clonal hematopoiesis of indeterminate potential (CHIP), indicating ASXL1 mutations as early events in leukemogenesis. Not surprisingly, they occur at high frequency in myeloid malignancies...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36184726"
                        open
                      >
                        <summary class="article-expander__title">Age-dependent association of clonal hematopoiesis with COVID-19 mortality in patients over 60years</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36184726/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221009214149&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis, especially that of indeterminate potential (CHIP), has been associated with age-related diseases, such as those contributing to a more severe COVID-19. Four studies have attempted to associate CHIP with COVID-19 severity without conclusive findings. In the present work, we explore the association between CHIP and COVID-19 mortality. Genomic DNA extracted from peripheral blood of COVID-19 patients (n &#x3D; 241 deceased, n &#x3D; 239 survivors) was sequenced with the Myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36183966"
                        open
                      >
                        <summary class="article-expander__title">Role of ASXL1 in Hematopoiesis and Myeloid Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36183966/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221009214149&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Next-generation sequencing technology (NGS), including whole-exome or whole-genome sequencing and target gene sequencing, has allowed the molecular characterization of somatic mutation spectrums in hematologic diseases. Mutations in Additional sex combs-like 1 (ASXL1), a chromatin regulator, are identified in clonal hematopoiesis of indeterminate potential (CHIP), indicating ASXL1 mutations as early events in leukemogenesis. Not surprisingly, they occur at high frequency in myeloid malignancies...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36184726"
                        open
                      >
                        <summary class="article-expander__title">Age-dependent association of clonal hematopoiesis with COVID-19 mortality in patients over 60years</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36184726/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221008212536&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis, especially that of indeterminate potential (CHIP), has been associated with age-related diseases, such as those contributing to a more severe COVID-19. Four studies have attempted to associate CHIP with COVID-19 severity without conclusive findings. In the present work, we explore the association between CHIP and COVID-19 mortality. Genomic DNA extracted from peripheral blood of COVID-19 patients (n &#x3D; 241 deceased, n &#x3D; 239 survivors) was sequenced with the Myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36183966"
                        open
                      >
                        <summary class="article-expander__title">Role of ASXL1 in Hematopoiesis and Myeloid Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36183966/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221008212536&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Next-generation sequencing technology (NGS), including whole-exome or whole-genome sequencing and target gene sequencing, has allowed the molecular characterization of somatic mutation spectrums in hematologic diseases. Mutations in Additional sex combs-like 1 (ASXL1), a chromatin regulator, are identified in clonal hematopoiesis of indeterminate potential (CHIP), indicating ASXL1 mutations as early events in leukemogenesis. Not surprisingly, they occur at high frequency in myeloid malignancies...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36184726"
                        open
                      >
                        <summary class="article-expander__title">Age-dependent association of clonal hematopoiesis with COVID-19 mortality in patients over 60years</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36184726/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221007211807&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis, especially that of indeterminate potential (CHIP), has been associated with age-related diseases, such as those contributing to a more severe COVID-19. Four studies have attempted to associate CHIP with COVID-19 severity without conclusive findings. In the present work, we explore the association between CHIP and COVID-19 mortality. Genomic DNA extracted from peripheral blood of COVID-19 patients (n &#x3D; 241 deceased, n &#x3D; 239 survivors) was sequenced with the Myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36183966"
                        open
                      >
                        <summary class="article-expander__title">Role of ASXL1 in Hematopoiesis and Myeloid Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36183966/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221007211807&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Next-generation sequencing technology (NGS), including whole-exome or whole-genome sequencing and target gene sequencing, has allowed the molecular characterization of somatic mutation spectrums in hematologic diseases. Mutations in Additional sex combs-like 1 (ASXL1), a chromatin regulator, are identified in clonal hematopoiesis of indeterminate potential (CHIP), indicating ASXL1 mutations as early events in leukemogenesis. Not surprisingly, they occur at high frequency in myeloid malignancies...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36184726"
                        open
                      >
                        <summary class="article-expander__title">Age-dependent association of clonal hematopoiesis with COVID-19 mortality in patients over 60years</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36184726/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221006214146&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis, especially that of indeterminate potential (CHIP), has been associated with age-related diseases, such as those contributing to a more severe COVID-19. Four studies have attempted to associate CHIP with COVID-19 severity without conclusive findings. In the present work, we explore the association between CHIP and COVID-19 mortality. Genomic DNA extracted from peripheral blood of COVID-19 patients (n &#x3D; 241 deceased, n &#x3D; 239 survivors) was sequenced with the Myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36183966"
                        open
                      >
                        <summary class="article-expander__title">Role of ASXL1 in Hematopoiesis and Myeloid Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36183966/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221006214146&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Next-generation sequencing technology (NGS), including whole-exome or whole-genome sequencing and target gene sequencing, has allowed the molecular characterization of somatic mutation spectrums in hematologic diseases. Mutations in Additional sex combs-like 1 (ASXL1), a chromatin regulator, are identified in clonal hematopoiesis of indeterminate potential (CHIP), indicating ASXL1 mutations as early events in leukemogenesis. Not surprisingly, they occur at high frequency in myeloid malignancies...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36184726"
                        open
                      >
                        <summary class="article-expander__title">Age-dependent association of clonal hematopoiesis with COVID-19 mortality in patients over 60years</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36184726/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221005212152&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis, especially that of indeterminate potential (CHIP), has been associated with age-related diseases, such as those contributing to a more severe COVID-19. Four studies have attempted to associate CHIP with COVID-19 severity without conclusive findings. In the present work, we explore the association between CHIP and COVID-19 mortality. Genomic DNA extracted from peripheral blood of COVID-19 patients (n &#x3D; 241 deceased, n &#x3D; 239 survivors) was sequenced with the Myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36183966"
                        open
                      >
                        <summary class="article-expander__title">Role of ASXL1 in Hematopoiesis and Myeloid Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36183966/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221005212152&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Next-generation sequencing technology (NGS), including whole-exome or whole-genome sequencing and target gene sequencing, has allowed the molecular characterization of somatic mutation spectrums in hematologic diseases. Mutations in Additional sex combs-like 1 (ASXL1), a chromatin regulator, are identified in clonal hematopoiesis of indeterminate potential (CHIP), indicating ASXL1 mutations as early events in leukemogenesis. Not surprisingly, they occur at high frequency in myeloid malignancies...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36184726"
                        open
                      >
                        <summary class="article-expander__title">Age-dependent association of clonal hematopoiesis with COVID-19 mortality in patients over 60years</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36184726/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221004212702&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis, especially that of indeterminate potential (CHIP), has been associated with age-related diseases, such as those contributing to a more severe COVID-19. Four studies have attempted to associate CHIP with COVID-19 severity without conclusive findings. In the present work, we explore the association between CHIP and COVID-19 mortality. Genomic DNA extracted from peripheral blood of COVID-19 patients (n &#x3D; 241 deceased, n &#x3D; 239 survivors) was sequenced with the Myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36183966"
                        open
                      >
                        <summary class="article-expander__title">Role of ASXL1 in Hematopoiesis and Myeloid Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36183966/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221004212702&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Next-generation sequencing technology (NGS), including whole-exome or whole-genome sequencing and target gene sequencing, has allowed the molecular characterization of somatic mutation spectrums in hematologic diseases. Mutations in Additional sex combs-like 1 (ASXL1), a chromatin regulator, are identified in clonal hematopoiesis of indeterminate potential (CHIP), indicating ASXL1 mutations as early events in leukemogenesis. Not surprisingly, they occur at high frequency in myeloid malignancies...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36184726"
                        open
                      >
                        <summary class="article-expander__title">Age-dependent association of clonal hematopoiesis with COVID-19 mortality in patients over 60years</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36184726/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221003213836&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis, especially that of indeterminate potential (CHIP), has been associated with age-related diseases, such as those contributing to a more severe COVID-19. Four studies have attempted to associate CHIP with COVID-19 severity without conclusive findings. In the present work, we explore the association between CHIP and COVID-19 mortality. Genomic DNA extracted from peripheral blood of COVID-19 patients (n &#x3D; 241 deceased, n &#x3D; 239 survivors) was sequenced with the Myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36183966"
                        open
                      >
                        <summary class="article-expander__title">Role of ASXL1 in Hematopoiesis and Myeloid Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36183966/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20221003213836&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Next-generation sequencing technology (NGS), including whole-exome or whole-genome sequencing and target gene sequencing, has allowed the molecular characterization of somatic mutation spectrums in hematologic diseases. Mutations in Additional sex combs-like 1 (ASXL1), a chromatin regulator, are identified in clonal hematopoiesis of indeterminate potential (CHIP), indicating ASXL1 mutations as early events in leukemogenesis. Not surprisingly, they occur at high frequency in myeloid malignancies...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221128200439&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.8">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36184613"
                        open
                      >
                        <summary class="article-expander__title">Extrachromosomal circular DNA: biogenesis, structure, functions and diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36184613/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221020212123&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Extrachromosomal circular DNA (eccDNA), ranging in size from tens to millions of base pairs, is independent of conventional chromosomes. Recently, eccDNAs have been considered an unanticipated major source of somatic rearrangements, contributing to genomic remodeling through chimeric circularization and reintegration of circular DNA into the linear genome. In addition, the origin of eccDNA is considered to be associated with essential chromatin-related events, including the formation of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36184613"
                        open
                      >
                        <summary class="article-expander__title">Extrachromosomal circular DNA: biogenesis, structure, functions and diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36184613/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221019212642&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Extrachromosomal circular DNA (eccDNA), ranging in size from tens to millions of base pairs, is independent of conventional chromosomes. Recently, eccDNAs have been considered an unanticipated major source of somatic rearrangements, contributing to genomic remodeling through chimeric circularization and reintegration of circular DNA into the linear genome. In addition, the origin of eccDNA is considered to be associated with essential chromatin-related events, including the formation of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36184613"
                        open
                      >
                        <summary class="article-expander__title">Extrachromosomal circular DNA: biogenesis, structure, functions and diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36184613/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221018213749&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Extrachromosomal circular DNA (eccDNA), ranging in size from tens to millions of base pairs, is independent of conventional chromosomes. Recently, eccDNAs have been considered an unanticipated major source of somatic rearrangements, contributing to genomic remodeling through chimeric circularization and reintegration of circular DNA into the linear genome. In addition, the origin of eccDNA is considered to be associated with essential chromatin-related events, including the formation of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36184613"
                        open
                      >
                        <summary class="article-expander__title">Extrachromosomal circular DNA: biogenesis, structure, functions and diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36184613/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221017214255&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Extrachromosomal circular DNA (eccDNA), ranging in size from tens to millions of base pairs, is independent of conventional chromosomes. Recently, eccDNAs have been considered an unanticipated major source of somatic rearrangements, contributing to genomic remodeling through chimeric circularization and reintegration of circular DNA into the linear genome. In addition, the origin of eccDNA is considered to be associated with essential chromatin-related events, including the formation of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36184613"
                        open
                      >
                        <summary class="article-expander__title">Extrachromosomal circular DNA: biogenesis, structure, functions and diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36184613/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221016214941&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Extrachromosomal circular DNA (eccDNA), ranging in size from tens to millions of base pairs, is independent of conventional chromosomes. Recently, eccDNAs have been considered an unanticipated major source of somatic rearrangements, contributing to genomic remodeling through chimeric circularization and reintegration of circular DNA into the linear genome. In addition, the origin of eccDNA is considered to be associated with essential chromatin-related events, including the formation of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36184613"
                        open
                      >
                        <summary class="article-expander__title">Extrachromosomal circular DNA: biogenesis, structure, functions and diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36184613/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221015214357&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Extrachromosomal circular DNA (eccDNA), ranging in size from tens to millions of base pairs, is independent of conventional chromosomes. Recently, eccDNAs have been considered an unanticipated major source of somatic rearrangements, contributing to genomic remodeling through chimeric circularization and reintegration of circular DNA into the linear genome. In addition, the origin of eccDNA is considered to be associated with essential chromatin-related events, including the formation of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36184613"
                        open
                      >
                        <summary class="article-expander__title">Extrachromosomal circular DNA: biogenesis, structure, functions and diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36184613/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221014214034&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Extrachromosomal circular DNA (eccDNA), ranging in size from tens to millions of base pairs, is independent of conventional chromosomes. Recently, eccDNAs have been considered an unanticipated major source of somatic rearrangements, contributing to genomic remodeling through chimeric circularization and reintegration of circular DNA into the linear genome. In addition, the origin of eccDNA is considered to be associated with essential chromatin-related events, including the formation of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36184613"
                        open
                      >
                        <summary class="article-expander__title">Extrachromosomal circular DNA: biogenesis, structure, functions and diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36184613/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221013214955&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Extrachromosomal circular DNA (eccDNA), ranging in size from tens to millions of base pairs, is independent of conventional chromosomes. Recently, eccDNAs have been considered an unanticipated major source of somatic rearrangements, contributing to genomic remodeling through chimeric circularization and reintegration of circular DNA into the linear genome. In addition, the origin of eccDNA is considered to be associated with essential chromatin-related events, including the formation of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36184613"
                        open
                      >
                        <summary class="article-expander__title">Extrachromosomal circular DNA: biogenesis, structure, functions and diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36184613/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221012213728&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Extrachromosomal circular DNA (eccDNA), ranging in size from tens to millions of base pairs, is independent of conventional chromosomes. Recently, eccDNAs have been considered an unanticipated major source of somatic rearrangements, contributing to genomic remodeling through chimeric circularization and reintegration of circular DNA into the linear genome. In addition, the origin of eccDNA is considered to be associated with essential chromatin-related events, including the formation of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36184613"
                        open
                      >
                        <summary class="article-expander__title">Extrachromosomal circular DNA: biogenesis, structure, functions and diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36184613/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221011214956&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Extrachromosomal circular DNA (eccDNA), ranging in size from tens to millions of base pairs, is independent of conventional chromosomes. Recently, eccDNAs have been considered an unanticipated major source of somatic rearrangements, contributing to genomic remodeling through chimeric circularization and reintegration of circular DNA into the linear genome. In addition, the origin of eccDNA is considered to be associated with essential chromatin-related events, including the formation of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36184613"
                        open
                      >
                        <summary class="article-expander__title">Extrachromosomal circular DNA: biogenesis, structure, functions and diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36184613/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221010212330&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Extrachromosomal circular DNA (eccDNA), ranging in size from tens to millions of base pairs, is independent of conventional chromosomes. Recently, eccDNAs have been considered an unanticipated major source of somatic rearrangements, contributing to genomic remodeling through chimeric circularization and reintegration of circular DNA into the linear genome. In addition, the origin of eccDNA is considered to be associated with essential chromatin-related events, including the formation of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36184613"
                        open
                      >
                        <summary class="article-expander__title">Extrachromosomal circular DNA: biogenesis, structure, functions and diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36184613/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221009214149&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Extrachromosomal circular DNA (eccDNA), ranging in size from tens to millions of base pairs, is independent of conventional chromosomes. Recently, eccDNAs have been considered an unanticipated major source of somatic rearrangements, contributing to genomic remodeling through chimeric circularization and reintegration of circular DNA into the linear genome. In addition, the origin of eccDNA is considered to be associated with essential chromatin-related events, including the formation of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36184613"
                        open
                      >
                        <summary class="article-expander__title">Extrachromosomal circular DNA: biogenesis, structure, functions and diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36184613/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221008212536&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Extrachromosomal circular DNA (eccDNA), ranging in size from tens to millions of base pairs, is independent of conventional chromosomes. Recently, eccDNAs have been considered an unanticipated major source of somatic rearrangements, contributing to genomic remodeling through chimeric circularization and reintegration of circular DNA into the linear genome. In addition, the origin of eccDNA is considered to be associated with essential chromatin-related events, including the formation of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36184613"
                        open
                      >
                        <summary class="article-expander__title">Extrachromosomal circular DNA: biogenesis, structure, functions and diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36184613/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221007211807&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Extrachromosomal circular DNA (eccDNA), ranging in size from tens to millions of base pairs, is independent of conventional chromosomes. Recently, eccDNAs have been considered an unanticipated major source of somatic rearrangements, contributing to genomic remodeling through chimeric circularization and reintegration of circular DNA into the linear genome. In addition, the origin of eccDNA is considered to be associated with essential chromatin-related events, including the formation of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36184613"
                        open
                      >
                        <summary class="article-expander__title">Extrachromosomal circular DNA: biogenesis, structure, functions and diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36184613/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221006214146&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Extrachromosomal circular DNA (eccDNA), ranging in size from tens to millions of base pairs, is independent of conventional chromosomes. Recently, eccDNAs have been considered an unanticipated major source of somatic rearrangements, contributing to genomic remodeling through chimeric circularization and reintegration of circular DNA into the linear genome. In addition, the origin of eccDNA is considered to be associated with essential chromatin-related events, including the formation of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36184613"
                        open
                      >
                        <summary class="article-expander__title">Extrachromosomal circular DNA: biogenesis, structure, functions and diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36184613/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221005212152&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Extrachromosomal circular DNA (eccDNA), ranging in size from tens to millions of base pairs, is independent of conventional chromosomes. Recently, eccDNAs have been considered an unanticipated major source of somatic rearrangements, contributing to genomic remodeling through chimeric circularization and reintegration of circular DNA into the linear genome. In addition, the origin of eccDNA is considered to be associated with essential chromatin-related events, including the formation of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36184613"
                        open
                      >
                        <summary class="article-expander__title">Extrachromosomal circular DNA: biogenesis, structure, functions and diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36184613/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221004212702&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Extrachromosomal circular DNA (eccDNA), ranging in size from tens to millions of base pairs, is independent of conventional chromosomes. Recently, eccDNAs have been considered an unanticipated major source of somatic rearrangements, contributing to genomic remodeling through chimeric circularization and reintegration of circular DNA into the linear genome. In addition, the origin of eccDNA is considered to be associated with essential chromatin-related events, including the formation of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36184613"
                        open
                      >
                        <summary class="article-expander__title">Extrachromosomal circular DNA: biogenesis, structure, functions and diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36184613/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221003213836&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Extrachromosomal circular DNA (eccDNA), ranging in size from tens to millions of base pairs, is independent of conventional chromosomes. Recently, eccDNAs have been considered an unanticipated major source of somatic rearrangements, contributing to genomic remodeling through chimeric circularization and reintegration of circular DNA into the linear genome. In addition, the origin of eccDNA is considered to be associated with essential chromatin-related events, including the formation of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-01 datetime="2022-10-01T10:00:00.000Z">2022-10-01</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-01 datetime="2022-10-01T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221128200439&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.8">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36181657"
                        open
                      >
                        <summary class="article-expander__title">Chronic myeloid leukemia (CML) evolves from Philadelphia chromosome-negative myeloproliferative neoplasms (MPNs) with unexpected frequency</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36181657/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221019212642&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are chronic clonal disorders characterized by overproduction of myeloid-lineage blood cells and potential risk of evolution to acute myeloid leukemia (AML). Chronic myeloid leukemia (CML) is distinct from other MPNs in that its pathophysiology stems from the BCR-ABL fusion protein of the Philadelphia chromosome (Ph +). Though there are known cases of Ph- and Ph + MPNs coexisting in a single patient, overall prevalence has never been quantified in a prospective...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36181657"
                        open
                      >
                        <summary class="article-expander__title">Chronic myeloid leukemia (CML) evolves from Philadelphia chromosome-negative myeloproliferative neoplasms (MPNs) with unexpected frequency</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36181657/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221018213749&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are chronic clonal disorders characterized by overproduction of myeloid-lineage blood cells and potential risk of evolution to acute myeloid leukemia (AML). Chronic myeloid leukemia (CML) is distinct from other MPNs in that its pathophysiology stems from the BCR-ABL fusion protein of the Philadelphia chromosome (Ph +). Though there are known cases of Ph- and Ph + MPNs coexisting in a single patient, overall prevalence has never been quantified in a prospective...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36181657"
                        open
                      >
                        <summary class="article-expander__title">Chronic myeloid leukemia (CML) evolves from Philadelphia chromosome-negative myeloproliferative neoplasms (MPNs) with unexpected frequency</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36181657/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221017214255&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are chronic clonal disorders characterized by overproduction of myeloid-lineage blood cells and potential risk of evolution to acute myeloid leukemia (AML). Chronic myeloid leukemia (CML) is distinct from other MPNs in that its pathophysiology stems from the BCR-ABL fusion protein of the Philadelphia chromosome (Ph +). Though there are known cases of Ph- and Ph + MPNs coexisting in a single patient, overall prevalence has never been quantified in a prospective...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36181657"
                        open
                      >
                        <summary class="article-expander__title">Chronic myeloid leukemia (CML) evolves from Philadelphia chromosome-negative myeloproliferative neoplasms (MPNs) with unexpected frequency</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36181657/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221016214941&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are chronic clonal disorders characterized by overproduction of myeloid-lineage blood cells and potential risk of evolution to acute myeloid leukemia (AML). Chronic myeloid leukemia (CML) is distinct from other MPNs in that its pathophysiology stems from the BCR-ABL fusion protein of the Philadelphia chromosome (Ph +). Though there are known cases of Ph- and Ph + MPNs coexisting in a single patient, overall prevalence has never been quantified in a prospective...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36181657"
                        open
                      >
                        <summary class="article-expander__title">Chronic myeloid leukemia (CML) evolves from Philadelphia chromosome-negative myeloproliferative neoplasms (MPNs) with unexpected frequency</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36181657/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221015214357&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are chronic clonal disorders characterized by overproduction of myeloid-lineage blood cells and potential risk of evolution to acute myeloid leukemia (AML). Chronic myeloid leukemia (CML) is distinct from other MPNs in that its pathophysiology stems from the BCR-ABL fusion protein of the Philadelphia chromosome (Ph +). Though there are known cases of Ph- and Ph + MPNs coexisting in a single patient, overall prevalence has never been quantified in a prospective...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36181657"
                        open
                      >
                        <summary class="article-expander__title">Chronic myeloid leukemia (CML) evolves from Philadelphia chromosome-negative myeloproliferative neoplasms (MPNs) with unexpected frequency</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36181657/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221014214034&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are chronic clonal disorders characterized by overproduction of myeloid-lineage blood cells and potential risk of evolution to acute myeloid leukemia (AML). Chronic myeloid leukemia (CML) is distinct from other MPNs in that its pathophysiology stems from the BCR-ABL fusion protein of the Philadelphia chromosome (Ph +). Though there are known cases of Ph- and Ph + MPNs coexisting in a single patient, overall prevalence has never been quantified in a prospective...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36181657"
                        open
                      >
                        <summary class="article-expander__title">Chronic myeloid leukemia (CML) evolves from Philadelphia chromosome-negative myeloproliferative neoplasms (MPNs) with unexpected frequency</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36181657/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221013214955&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are chronic clonal disorders characterized by overproduction of myeloid-lineage blood cells and potential risk of evolution to acute myeloid leukemia (AML). Chronic myeloid leukemia (CML) is distinct from other MPNs in that its pathophysiology stems from the BCR-ABL fusion protein of the Philadelphia chromosome (Ph +). Though there are known cases of Ph- and Ph + MPNs coexisting in a single patient, overall prevalence has never been quantified in a prospective...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36181657"
                        open
                      >
                        <summary class="article-expander__title">Chronic myeloid leukemia (CML) evolves from Philadelphia chromosome-negative myeloproliferative neoplasms (MPNs) with unexpected frequency</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36181657/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221012213728&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are chronic clonal disorders characterized by overproduction of myeloid-lineage blood cells and potential risk of evolution to acute myeloid leukemia (AML). Chronic myeloid leukemia (CML) is distinct from other MPNs in that its pathophysiology stems from the BCR-ABL fusion protein of the Philadelphia chromosome (Ph +). Though there are known cases of Ph- and Ph + MPNs coexisting in a single patient, overall prevalence has never been quantified in a prospective...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36181657"
                        open
                      >
                        <summary class="article-expander__title">Chronic myeloid leukemia (CML) evolves from Philadelphia chromosome-negative myeloproliferative neoplasms (MPNs) with unexpected frequency</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36181657/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221011214956&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are chronic clonal disorders characterized by overproduction of myeloid-lineage blood cells and potential risk of evolution to acute myeloid leukemia (AML). Chronic myeloid leukemia (CML) is distinct from other MPNs in that its pathophysiology stems from the BCR-ABL fusion protein of the Philadelphia chromosome (Ph +). Though there are known cases of Ph- and Ph + MPNs coexisting in a single patient, overall prevalence has never been quantified in a prospective...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36181657"
                        open
                      >
                        <summary class="article-expander__title">Chronic myeloid leukemia (CML) evolves from Philadelphia chromosome-negative myeloproliferative neoplasms (MPNs) with unexpected frequency</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36181657/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221010212330&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are chronic clonal disorders characterized by overproduction of myeloid-lineage blood cells and potential risk of evolution to acute myeloid leukemia (AML). Chronic myeloid leukemia (CML) is distinct from other MPNs in that its pathophysiology stems from the BCR-ABL fusion protein of the Philadelphia chromosome (Ph +). Though there are known cases of Ph- and Ph + MPNs coexisting in a single patient, overall prevalence has never been quantified in a prospective...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36181657"
                        open
                      >
                        <summary class="article-expander__title">Chronic myeloid leukemia (CML) evolves from Philadelphia chromosome-negative myeloproliferative neoplasms (MPNs) with unexpected frequency</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36181657/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221009214149&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are chronic clonal disorders characterized by overproduction of myeloid-lineage blood cells and potential risk of evolution to acute myeloid leukemia (AML). Chronic myeloid leukemia (CML) is distinct from other MPNs in that its pathophysiology stems from the BCR-ABL fusion protein of the Philadelphia chromosome (Ph +). Though there are known cases of Ph- and Ph + MPNs coexisting in a single patient, overall prevalence has never been quantified in a prospective...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36181657"
                        open
                      >
                        <summary class="article-expander__title">Chronic myeloid leukemia (CML) evolves from Philadelphia chromosome-negative myeloproliferative neoplasms (MPNs) with unexpected frequency</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36181657/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221008212536&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are chronic clonal disorders characterized by overproduction of myeloid-lineage blood cells and potential risk of evolution to acute myeloid leukemia (AML). Chronic myeloid leukemia (CML) is distinct from other MPNs in that its pathophysiology stems from the BCR-ABL fusion protein of the Philadelphia chromosome (Ph +). Though there are known cases of Ph- and Ph + MPNs coexisting in a single patient, overall prevalence has never been quantified in a prospective...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36181657"
                        open
                      >
                        <summary class="article-expander__title">Chronic myeloid leukemia (CML) evolves from Philadelphia chromosome-negative myeloproliferative neoplasms (MPNs) with unexpected frequency</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36181657/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221007211807&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are chronic clonal disorders characterized by overproduction of myeloid-lineage blood cells and potential risk of evolution to acute myeloid leukemia (AML). Chronic myeloid leukemia (CML) is distinct from other MPNs in that its pathophysiology stems from the BCR-ABL fusion protein of the Philadelphia chromosome (Ph +). Though there are known cases of Ph- and Ph + MPNs coexisting in a single patient, overall prevalence has never been quantified in a prospective...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36181657"
                        open
                      >
                        <summary class="article-expander__title">Chronic myeloid leukemia (CML) evolves from Philadelphia chromosome-negative myeloproliferative neoplasms (MPNs) with unexpected frequency</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36181657/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221006214146&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are chronic clonal disorders characterized by overproduction of myeloid-lineage blood cells and potential risk of evolution to acute myeloid leukemia (AML). Chronic myeloid leukemia (CML) is distinct from other MPNs in that its pathophysiology stems from the BCR-ABL fusion protein of the Philadelphia chromosome (Ph +). Though there are known cases of Ph- and Ph + MPNs coexisting in a single patient, overall prevalence has never been quantified in a prospective...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36181657"
                        open
                      >
                        <summary class="article-expander__title">Chronic myeloid leukemia (CML) evolves from Philadelphia chromosome-negative myeloproliferative neoplasms (MPNs) with unexpected frequency</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36181657/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221005212152&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are chronic clonal disorders characterized by overproduction of myeloid-lineage blood cells and potential risk of evolution to acute myeloid leukemia (AML). Chronic myeloid leukemia (CML) is distinct from other MPNs in that its pathophysiology stems from the BCR-ABL fusion protein of the Philadelphia chromosome (Ph +). Though there are known cases of Ph- and Ph + MPNs coexisting in a single patient, overall prevalence has never been quantified in a prospective...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36181657"
                        open
                      >
                        <summary class="article-expander__title">Chronic myeloid leukemia (CML) evolves from Philadelphia chromosome-negative myeloproliferative neoplasms (MPNs) with unexpected frequency</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36181657/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221004212702&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are chronic clonal disorders characterized by overproduction of myeloid-lineage blood cells and potential risk of evolution to acute myeloid leukemia (AML). Chronic myeloid leukemia (CML) is distinct from other MPNs in that its pathophysiology stems from the BCR-ABL fusion protein of the Philadelphia chromosome (Ph +). Though there are known cases of Ph- and Ph + MPNs coexisting in a single patient, overall prevalence has never been quantified in a prospective...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36181657"
                        open
                      >
                        <summary class="article-expander__title">Chronic myeloid leukemia (CML) evolves from Philadelphia chromosome-negative myeloproliferative neoplasms (MPNs) with unexpected frequency</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36181657/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221003213836&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are chronic clonal disorders characterized by overproduction of myeloid-lineage blood cells and potential risk of evolution to acute myeloid leukemia (AML). Chronic myeloid leukemia (CML) is distinct from other MPNs in that its pathophysiology stems from the BCR-ABL fusion protein of the Philadelphia chromosome (Ph +). Though there are known cases of Ph- and Ph + MPNs coexisting in a single patient, overall prevalence has never been quantified in a prospective...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36181657"
                        open
                      >
                        <summary class="article-expander__title">Chronic myeloid leukemia (CML) evolves from Philadelphia chromosome-negative myeloproliferative neoplasms (MPNs) with unexpected frequency</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36181657/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221002212112&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are chronic clonal disorders characterized by overproduction of myeloid-lineage blood cells and potential risk of evolution to acute myeloid leukemia (AML). Chronic myeloid leukemia (CML) is distinct from other MPNs in that its pathophysiology stems from the BCR-ABL fusion protein of the Philadelphia chromosome (Ph +). Though there are known cases of Ph- and Ph + MPNs coexisting in a single patient, overall prevalence has never been quantified in a prospective...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:36181657"
                        open
                      >
                        <summary class="article-expander__title">Chronic myeloid leukemia (CML) evolves from Philadelphia chromosome-negative myeloproliferative neoplasms (MPNs) with unexpected frequency</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/36181657/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20221001214304&amp;v&#x3D;2.17.8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPN) are chronic clonal disorders characterized by overproduction of myeloid-lineage blood cells and potential risk of evolution to acute myeloid leukemia (AML). Chronic myeloid leukemia (CML) is distinct from other MPNs in that its pathophysiology stems from the BCR-ABL fusion protein of the Philadelphia chromosome (Ph +). Though there are known cases of Ph- and Ph + MPNs coexisting in a single patient, overall prevalence has never been quantified in a prospective...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-06-06 datetime="2022-06-06T18:28:04.000Z">2022-06-06</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-06-06 datetime="2022-06-06T18:28:04.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="72a3a11d39c217b1a80cbb551444e6f6"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Nodal interferon exposure drives tumour-specific tolerance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1533878403463405569">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Nodal interferon exposure drives tumour-specific tolerance https://t.co/FT4NeBcFTg pic.twitter.com/NRHkVDUsOr
 NatureReviewsCancer (@NatureRevCancer) June 6, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-30 datetime="2022-05-30T04:37:42.000Z">2022-05-30</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-30 datetime="2022-05-30T04:37:42.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="bb6bc242bcbc356c4a38db850e74d812"
                        open
                      >
                        <summary class="article-expander__title">@VIBConferences: #Cancer #immunotherapy has become a crucial pillar of anticancer therapy. 19 leading researchers at #ImmunoOnco22 will give new insights into the immune response to tumors and tackle unresolved questions.Registration Deadline: Thursday 2 June. Read more</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/VIBConferences/status/1531132720952266752">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>#Cancer #immunotherapy has become a crucial pillar of anticancer therapy. 19 leading researchers at #ImmunoOnco22 will give new insights into the immune response to tumors and tackle unresolved questions.
Registration Deadline: Thursday 2 June. Read more https://t.co/cc0HC5cj6W pic.twitter.com/u8LFuzBbsV
 VIB Conferences (@VIBConferences) May 30, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-27 datetime="2022-05-27T14:45:03.000Z">2022-05-27</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-27 datetime="2022-05-27T14:45:03.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="ed6c996dc1a718afb523de6229a53131"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Nonsense-mediated RNA decay: an emerging modulator of malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1530198403686072320">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Nonsense-mediated RNA decay: an emerging modulator of malignancy https://t.co/wjuHplkpof pic.twitter.com/1SoKYhRsPO
 NatureReviewsCancer (@NatureRevCancer) May 27, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="d15a3a4b24853812d7fa21fd56de5815"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: Our June issue, out NOW! With articles on #therapyresistance, #senescence &amp; targeting CDK4 &amp; CDK6 in #cancer! Plus, a Comment on cancer mRNA vaccines as well as the launch of our new Tools of the Trade articles #NRCToT. So much content </summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1530174481313972224">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Our June issue, out NOW! With articles on #therapyresistance, #senescence &amp; targeting CDK4 &amp; CDK6 in #cancer! Plus, a Comment on cancer mRNA vaccines as well as the launch of our new Tools of the Trade articles #NRCToT. So much content 
 https://t.co/Fo6XRWV3OZ@natrescancer pic.twitter.com/rbxRPqLV6g
 NatureReviewsCancer (@NatureRevCancer) May 27, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-26 datetime="2022-05-26T08:53:50.000Z">2022-05-26</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-26 datetime="2022-05-26T08:53:50.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="fe13755a2693ea65459e5e21df097d2a"
                        open
                      >
                        <summary class="article-expander__title">@UlrikeHarjes: This Review by @MarcusDGon @cantleylab is part of the @NatureRevCancer series on #Diet &amp; #SystemicMetabolism in cancer:  do love this topic in general plus I am handling research at @NatureMedicine now.. get in touch if you have something of interest</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/UlrikeHarjes/status/1529747629936193537">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>This Review by @MarcusDGon @cantleylab is part of the @NatureRevCancer series on #Diet &amp; #SystemicMetabolism in cancer: https://t.co/p3Y2cdNhco
I do love this topic in general plus I am handling research at @NatureMedicine now.. get in touch if you have something of interest https://t.co/qWH2dTv8j9
 Ulrike Harjes (@UlrikeHarjes) May 26, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-25 datetime="2022-05-25T17:35:04.000Z">2022-05-25</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-25 datetime="2022-05-25T17:35:04.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="9ba5279424b31b0122a4ae20c18e6d03"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Developing dietary interventions as therapy for cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1529516411059437568">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Developing dietary interventions as therapy for cancer https://t.co/ijuqctk1wJ pic.twitter.com/KHPpMKnCFa
 NatureReviewsCancer (@NatureRevCancer) May 25, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-18 datetime="2022-05-18T14:11:32.000Z">2022-05-18</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-18 datetime="2022-05-18T14:11:32.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="611d40f2e6ea2796b028bebda41a86dd"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Digging for treasures in the tumour interactome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1526928478490095616">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Digging for treasures in the tumour interactome https://t.co/NhHYAwUgLE
 NatureReviewsCancer (@NatureRevCancer) May 18, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-17 datetime="2022-05-17T13:48:21.000Z">2022-05-17</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-17 datetime="2022-05-17T13:48:21.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="9590a971b00448cd312078183965b21d"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: As part of our overall commitment to diversity, equity and inclusion at Springer Nature, wed like to better understand what DEI means within the global research community. Take the survey here</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1526560254091313156">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>As part of our overall commitment to diversity, equity and inclusion at Springer Nature, wed like to better understand what DEI means within the global research community. Take the survey here https://t.co/03yLiUxFTq
 NatureReviewsCancer (@NatureRevCancer) May 17, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-12 datetime="2022-05-12T15:25:46.000Z">2022-05-12</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-12 datetime="2022-05-12T15:25:46.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="aa38ff4245b93e964773f191844ddc7a"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: Newsflash! We have a new article type in the journal - Tools of the Trade #NRCToT   where we hope to enable more #ECRs to interact with the editorial team &amp; have a bigger voice in our content. For more info, check out the Editorial </summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1524772831904509953">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Newsflash! We have a new article type in the journal - Tools of the Trade #NRCToT   where we hope to enable more #ECRs to interact with the editorial team &amp; have a bigger voice in our content. For more info, check out the Editorial  https://t.co/VbyoUJmTRJ
 NatureReviewsCancer (@NatureRevCancer) May 12, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-10 datetime="2022-05-10T16:46:03.000Z">2022-05-10</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-10 datetime="2022-05-10T16:46:03.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="65aca76b9f794aaa668d84ad32dca4a0"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Tools for the next generation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1524068259712036864">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Tools for the next generation https://t.co/lEzTPFYDf5
 NatureReviewsCancer (@NatureRevCancer) May 10, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="afa9b6784b4ccc7c05ed4efd26ddfb2d"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Hunting down rare drug-tolerant cycling cells with Watermelon</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1524022709386190856">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Hunting down rare drug-tolerant cycling cells with Watermelon https://t.co/XXT0c9SAYX
 NatureReviewsCancer (@NatureRevCancer) May 10, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-04-29 datetime="2022-04-29T14:40:05.000Z">2022-04-29</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-04-29 datetime="2022-04-29T14:40:05.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="43e2d5ccebc9030f737d656f47ac9d98"
                        open
                      >
                        <summary class="article-expander__title">@VarunVenkatara2: Very happy that our latest Perspective is now online @NatureRevCancer ! We discuss implications of malignant, multicellular networks and potential therapeutic strategies how to disconnect them. Thanks to everyone involved ! #CancerNeuroscience #TumorMicrotubes #neurotwitter</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/VarunVenkatara2/status/1520050291986604033">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Very happy that our latest Perspective is now online @NatureRevCancer ! We discuss implications of malignant, multicellular networks and potential therapeutic strategies how to disconnect them. Thanks to everyone involved ! #CancerNeuroscience #TumorMicrotubes #neurotwitter https://t.co/S55cKTGNFd
 Varun Venkataramani (@VarunVenkatara2) April 29, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="9f6b8644378881f4c2e6b6833c93fc2f"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! An inhospitable site</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1520020349068316673">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! An inhospitable site https://t.co/0dvqIKCWxh pic.twitter.com/PtY8J9Fz7r
 NatureReviewsCancer (@NatureRevCancer) April 29, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="f6d91efcd60dae769e2722585bdfa8df"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Disconnecting multicellular networks in brain tumours</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1520020340201598978">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Disconnecting multicellular networks in brain tumours https://t.co/LJrzy8bRma pic.twitter.com/CO7BnooP0r
 NatureReviewsCancer (@NatureRevCancer) April 29, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-04-28 datetime="2022-04-28T14:04:34.000Z">2022-04-28</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-04-28 datetime="2022-04-28T14:04:34.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="76e3b436bc8e43ad21366a80c120e395"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: Our May issue, out NOW! With Reviews on #CRISPR in cancer biology &amp; #therapy, ROS in tumour development &amp; progression, and cancer #proteogenomics, and Comments on esophageal cancer research in Eastern Africa and on Critical Race Theory.</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1519678964364623872">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Our May issue, out NOW! With Reviews on #CRISPR in cancer biology &amp; #therapy, ROS in tumour development &amp; progression, and cancer #proteogenomics, and Comments on esophageal cancer research in Eastern Africa and on Critical Race Theory.
 https://t.co/vQwHFJhpwv@natrescancer pic.twitter.com/6kYgjRYhdT
 NatureReviewsCancer (@NatureRevCancer) April 28, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="fdf224ea293cdf039086d16e5699a59a"
                        open
                      >
                        <summary class="article-expander__title">@tmurraystewart: Our latest publication on #polyamines is online!@HopkinsMedicine</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/tmurraystewart/status/1519413460978704384">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Our latest publication on #polyamines is online!@HopkinsMedicine https://t.co/ehRGgN1w9R
 Tracy Murray Stewart (@tmurraystewart) April 27, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-04-27 datetime="2022-04-27T18:22:03.000Z">2022-04-27</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-04-27 datetime="2022-04-27T18:22:03.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="4f43226ec4dc8e531ff6e64a9c5ab8e6"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Polyamines in cancer: integrating organismal metabolism and antitumour immunity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1519381375874265088">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Polyamines in cancer: integrating organismal metabolism and antitumour immunity https://t.co/YCJX1hRQLO pic.twitter.com/zRUCRNXyxH
 NatureReviewsCancer (@NatureRevCancer) April 27, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="a44ed79b9f76aad377b9272fb392f553"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW #ResearchHighlight! Lv, Liu, Mo &amp; co demonstrate that in pancreatic #PDAC cells, gasdermin E transports YBX1 to the nucleus, which promotes mucin expression, thereby providing tumour cells with a barrier against digestive enzymes.</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1519317944995569664">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW #ResearchHighlight! Lv, Liu, Mo &amp; co demonstrate that in pancreatic #PDAC cells, gasdermin E transports YBX1 to the nucleus, which promotes mucin expression, thereby providing tumour cells with a barrier against digestive enzymes.
 https://t.co/JTUhX0DHQ1@natrescancer pic.twitter.com/2PdCmsDu2p
 NatureReviewsCancer (@NatureRevCancer) April 27, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-04-26 datetime="2022-04-26T14:58:00.000Z">2022-04-26</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-04-26 datetime="2022-04-26T14:58:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="06ff7641d8f5fcc0dff91f835ac3bbc1"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: In the April issue of @NatureRevCancer | Comment, Reimagining cancer research with art, by @CancerSciArtist, Stefani Shoreibah &amp; @tdanino  @natrescancer @Columbia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1518967636561956865">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>In the April issue of @NatureRevCancer | Comment, Reimagining cancer research with art, by @CancerSciArtist, Stefani Shoreibah &amp; @tdanino
 https://t.co/GeTVCMJQsp @natrescancer
 @Columbia pic.twitter.com/ZXW6A0Fe0c
 NatureReviewsCancer (@NatureRevCancer) April 26, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-04-25 datetime="2022-04-25T12:12:32.000Z">2022-04-25</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-04-25 datetime="2022-04-25T12:12:32.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="96b2ddff4eb269cf343213c6a1b1ce95"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Tracking dormant disseminated tumour cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1518563607440621569">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Tracking dormant disseminated tumour cells https://t.co/kSXyWAvCxF
 NatureReviewsCancer (@NatureRevCancer) April 25, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>

    <footer>
      <a class="footer-link" href="https://github.com/MacwanEric/cancer.rss.feed/actions/runs/3570033583">
        <time id="build-timestamp" datetime="2022-11-29T01:04:41.788Z">2022-11-29T01:04:41.788Z</time>
      </a>
      <a class="footer-link" href="https://github.com/osmoscraft/osmosfeed">osmosfeed 1.15.1</a>
    </footer>
    <script src="index.js?v1.14.4"></script>
    <!-- %before-body-end.html% -->
  </body>

</html>