<!DOCTYPE html>
<html lang="en">

  <head>
    <title>CancerFeed</title>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta name="robots" content="noindex, nofollow" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
    <link rel="alternate" type="application/rss+xml" title="CancerFeed" href="feed.atom" />
    <link href="index.css?v1.14.4" rel="stylesheet" />
    <style>
  :root {
    /**
     * Solarized Light
     * Ethan Schoonover (modified by aramisgithub)
     */
    --base00: #fdf6e3;
    --base01: #eee8d5;
    --base02: #93a1a1;
    --base03: #839496;
    --base04: #657b83;
    --base05: #586e75;
    --base06: #073642;
    --base07: #002b36;
    --base08: #dc322f;
    --base09: #cb4b16;
    --base0A: #b58900;
    --base0B: #859900;
    --base0C: #2aa198;
    --base0D: #268bd2;
    --base0E: #6c71c4;
    --base0F: #d33682;

    /**
    * Components
    */
    --source-name-color: var(--base0C);
    /**
   * Typograph
   */
    --font-family-default: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell,
      "Helvetica Neue", sans-serif;
    --font-size-L: 3.6rem;
    --font-size-m: 2.2rem;
    --font-size-s: 1.8rem;
    --font-size-scaler: 52.5%; /* 1rem = 10px */
    --line-height-body: 1.4;
    --line-height-heading: 1.2;

  }
</style>

  </head>

  <body>
    <!-- %after-body-begin.html% -->
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-11-03 datetime="2023-11-03T10:00:00.000Z">2023-11-03</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-11-03 datetime="2023-11-03T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20231105195156">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37921439"
                        open
                      >
                        <summary class="article-expander__title">SRCAP Mutations and Cellular Stressors Promote Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37921439/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>SRCAP mutations promote clonal expansion of hematopoietic stem cells (HSC) in response to cellular stress.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37921439"
                        open
                      >
                        <summary class="article-expander__title">SRCAP Mutations and Cellular Stressors Promote Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37921439/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>SRCAP mutations promote clonal expansion of hematopoietic stem cells (HSC) in response to cellular stress.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37921439"
                        open
                      >
                        <summary class="article-expander__title">SRCAP Mutations and Cellular Stressors Promote Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37921439/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231103204956&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>SRCAP mutations promote clonal expansion of hematopoietic stem cells (HSC) in response to cellular stress.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;ff&#x3D;20231105195156&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37921274"
                        open
                      >
                        <summary class="article-expander__title">Personalised neoantigen-based therapy in colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37921274/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal cancer (CRC) has become one of the most common tumours with high morbidity, mortality and distinctive evolution mechanism. The neoantigens arising from the somatic mutations have become considerable treatment targets in the management of CRC. As cancer-specific aberrant peptides, neoantigens can trigger the robust host immune response and exert anti-tumour effects while minimising the emergence of adverse events commonly associated with alternative therapeutic regimens. In this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37921274"
                        open
                      >
                        <summary class="article-expander__title">Personalised neoantigen-based therapy in colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37921274/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal cancer (CRC) has become one of the most common tumours with high morbidity, mortality and distinctive evolution mechanism. The neoantigens arising from the somatic mutations have become considerable treatment targets in the management of CRC. As cancer-specific aberrant peptides, neoantigens can trigger the robust host immune response and exert anti-tumour effects while minimising the emergence of adverse events commonly associated with alternative therapeutic regimens. In this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37921274"
                        open
                      >
                        <summary class="article-expander__title">Personalised neoantigen-based therapy in colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37921274/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231103204955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal cancer (CRC) has become one of the most common tumours with high morbidity, mortality and distinctive evolution mechanism. The neoantigens arising from the somatic mutations have become considerable treatment targets in the management of CRC. As cancer-specific aberrant peptides, neoantigens can trigger the robust host immune response and exert anti-tumour effects while minimising the emergence of adverse events commonly associated with alternative therapeutic regimens. In this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-11-02 datetime="2023-11-02T10:00:00.000Z">2023-11-02</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-11-02 datetime="2023-11-02T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20231105195156">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37915324"
                        open
                      >
                        <summary class="article-expander__title">Philadelphia chromosome-negative myeloproliferative chronic neoplasms: is clonal hematopoiesis the main determinant of autoimmune and cardio-vascular manifestations?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37915324/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this article, we reviewed the possible mechanisms linking the clonal hematopoiesis of indeterminate potential (CHIP) to chronic myeloproliferative neoplasms (MPNs), autoimmune diseases (ADs), and cardiovascular diseases (CADs). CHIP is characterized by the presence of clonal mutations with an allelic frequency &gt;2% in the peripheral blood without dysplasia, overt hematological neoplasms, or abnormalities in blood cell count. The prevalence may reach 20% of elderly healthy individuals and is...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37915324"
                        open
                      >
                        <summary class="article-expander__title">Philadelphia chromosome-negative myeloproliferative chronic neoplasms: is clonal hematopoiesis the main determinant of autoimmune and cardio-vascular manifestations?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37915324/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this article, we reviewed the possible mechanisms linking the clonal hematopoiesis of indeterminate potential (CHIP) to chronic myeloproliferative neoplasms (MPNs), autoimmune diseases (ADs), and cardiovascular diseases (CADs). CHIP is characterized by the presence of clonal mutations with an allelic frequency &gt;2% in the peripheral blood without dysplasia, overt hematological neoplasms, or abnormalities in blood cell count. The prevalence may reach 20% of elderly healthy individuals and is...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37915324"
                        open
                      >
                        <summary class="article-expander__title">Philadelphia chromosome-negative myeloproliferative chronic neoplasms: is clonal hematopoiesis the main determinant of autoimmune and cardio-vascular manifestations?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37915324/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231103204956&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this article, we reviewed the possible mechanisms linking the clonal hematopoiesis of indeterminate potential (CHIP) to chronic myeloproliferative neoplasms (MPNs), autoimmune diseases (ADs), and cardiovascular diseases (CADs). CHIP is characterized by the presence of clonal mutations with an allelic frequency &gt;2% in the peripheral blood without dysplasia, overt hematological neoplasms, or abnormalities in blood cell count. The prevalence may reach 20% of elderly healthy individuals and is...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37915324"
                        open
                      >
                        <summary class="article-expander__title">Philadelphia chromosome-negative myeloproliferative chronic neoplasms: is clonal hematopoiesis the main determinant of autoimmune and cardio-vascular manifestations?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37915324/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this article, we reviewed the possible mechanisms linking the clonal hematopoiesis of indeterminate potential (CHIP) to chronic myeloproliferative neoplasms (MPNs), autoimmune diseases (ADs), and cardiovascular diseases (CADs). CHIP is characterized by the presence of clonal mutations with an allelic frequency &gt;2% in the peripheral blood without dysplasia, overt hematological neoplasms, or abnormalities in blood cell count. The prevalence may reach 20% of elderly healthy individuals and is...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-11-01 datetime="2023-11-01T10:00:00.000Z">2023-11-01</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-11-01 datetime="2023-11-01T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20231105195156">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37913908"
                        open
                      >
                        <summary class="article-expander__title">Donor-derived malignancy and transplant morbidity: Risks of patient and donor genetics in allogeneic hematopoietic stem cell transplant: Donor-derived malignancy and genetic risks in allo-HSCT</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37913908/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Allogeneic hematopoietic stem cell transplant (allo-HSCT) remains a key treatment option for hematologic malignancies (HMs), although it carries significant risks. Up to 30% of patients relapse after allo-HSCT, of which 2-5% are donor-derived malignancies (DDM). DDM can arise from a germline genetic predisposition allele or clonal hematopoiesis (CH) in the donor. Increasingly, genetic testing reveals that patient and donor genetic factors contribute to the development of DDM and other allo-HSCT...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37909388"
                        open
                      >
                        <summary class="article-expander__title">Somatic TET2 Mutations are Associated with Giant Cell Arteritis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37909388/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH increases risk for development of GCA in a genotype-specific fashion, with greatest risk being conferred by the presence of mutations in TET2. Somatic TET2 mutations likewise increase the risk of GCA-associated vision loss. Integration of somatic genetic testing in GCA diagnostics may be warranted in the future. This article is protected by copyright. All rights reserved.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37913908"
                        open
                      >
                        <summary class="article-expander__title">Donor-derived malignancy and transplant morbidity: Risks of patient and donor genetics in allogeneic hematopoietic stem cell transplant: Donor-derived malignancy and genetic risks in allo-HSCT</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37913908/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Allogeneic hematopoietic stem cell transplant (allo-HSCT) remains a key treatment option for hematologic malignancies (HMs), although it carries significant risks. Up to 30% of patients relapse after allo-HSCT, of which 2-5% are donor-derived malignancies (DDM). DDM can arise from a germline genetic predisposition allele or clonal hematopoiesis (CH) in the donor. Increasingly, genetic testing reveals that patient and donor genetic factors contribute to the development of DDM and other allo-HSCT...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37909388"
                        open
                      >
                        <summary class="article-expander__title">Somatic TET2 Mutations are Associated with Giant Cell Arteritis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37909388/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH increases risk for development of GCA in a genotype-specific fashion, with greatest risk being conferred by the presence of mutations in TET2. Somatic TET2 mutations likewise increase the risk of GCA-associated vision loss. Integration of somatic genetic testing in GCA diagnostics may be warranted in the future. This article is protected by copyright. All rights reserved.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37913908"
                        open
                      >
                        <summary class="article-expander__title">Donor-derived malignancy and transplant morbidity: Risks of patient and donor genetics in allogeneic hematopoietic stem cell transplant: Donor-derived malignancy and genetic risks in allo-HSCT</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37913908/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231103204956&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Allogeneic hematopoietic stem cell transplant (allo-HSCT) remains a key treatment option for hematologic malignancies (HMs), although it carries significant risks. Up to 30% of patients relapse after allo-HSCT, of which 2-5% are donor-derived malignancies (DDM). DDM can arise from a germline genetic predisposition allele or clonal hematopoiesis (CH) in the donor. Increasingly, genetic testing reveals that patient and donor genetic factors contribute to the development of DDM and other allo-HSCT...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37909388"
                        open
                      >
                        <summary class="article-expander__title">Somatic TET2 Mutations are Associated with Giant Cell Arteritis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37909388/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231103204956&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH increases risk for development of GCA in a genotype-specific fashion, with greatest risk being conferred by the presence of mutations in TET2. Somatic TET2 mutations likewise increase the risk of GCA-associated vision loss. Integration of somatic genetic testing in GCA diagnostics may be warranted in the future. This article is protected by copyright. All rights reserved.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37913908"
                        open
                      >
                        <summary class="article-expander__title">Donor-derived malignancy and transplant morbidity: Risks of patient and donor genetics in allogeneic hematopoietic stem cell transplant: Donor-derived malignancy and genetic risks in allo-HSCT</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37913908/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Allogeneic hematopoietic stem cell transplant (allo-HSCT) remains a key treatment option for hematologic malignancies (HMs), although it carries significant risks. Up to 30% of patients relapse after allo-HSCT, of which 2-5% are donor-derived malignancies (DDM). DDM can arise from a germline genetic predisposition allele or clonal hematopoiesis (CH) in the donor. Increasingly, genetic testing reveals that patient and donor genetic factors contribute to the development of DDM and other allo-HSCT...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37909388"
                        open
                      >
                        <summary class="article-expander__title">Somatic TET2 Mutations are Associated with Giant Cell Arteritis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37909388/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH increases risk for development of GCA in a genotype-specific fashion, with greatest risk being conferred by the presence of mutations in TET2. Somatic TET2 mutations likewise increase the risk of GCA-associated vision loss. Integration of somatic genetic testing in GCA diagnostics may be warranted in the future. This article is protected by copyright. All rights reserved.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37909388"
                        open
                      >
                        <summary class="article-expander__title">Somatic TET2 Mutations are Associated with Giant Cell Arteritis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37909388/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH increases risk for development of GCA in a genotype-specific fashion, with greatest risk being conferred by the presence of mutations in TET2. Somatic TET2 mutations likewise increase the risk of GCA-associated vision loss. Integration of somatic genetic testing in GCA diagnostics may be warranted in the future. This article is protected by copyright. All rights reserved.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-10-31 datetime="2023-10-31T10:00:00.000Z">2023-10-31</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-10-31 datetime="2023-10-31T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20231105195156">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37905118"
                        open
                      >
                        <summary class="article-expander__title">Determinants of mosaic chromosomal alteration fitness</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37905118/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is characterized by the acquisition of a somatic mutation in a hematopoietic stem cell that results in a clonal expansion. These driver mutations can be single nucleotide variants in cancer driver genes or larger structural rearrangements called mosaic chromosomal alterations (mCAs). The factors that influence the variations in mCA fitness and ultimately result in different clonal expansion rates are not well-understood. We used the Passenger-Approximated Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904876"
                        open
                      >
                        <summary class="article-expander__title">Germline and somatic drivers in inherited hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904876/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inherited hematologic malignancies are linked to a heterogenous group of genes, knowledge of which is rapidly expanding using panel-based next-generation sequencing (NGS) or whole-exome/whole-genome sequencing. Importantly, the penetrance for these syndromes is incomplete, and disease development, progression or transformation has critical clinical implications. With the earlier detection of healthy carriers and sequential monitoring of these patients, clonal hematopoiesis and somatic driver...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904125"
                        open
                      >
                        <summary class="article-expander__title">Prevalence and significance of clonal hematopoiesis of indeterminate potential in lung transplant recipients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Lung recipients have a higher prevalence of CHIP and a larger variety of genes with CHIP-associated mutations compared with previous reports for the general population. CHIP did not increase the risk of AR, CLAD, or death in lung recipients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37905118"
                        open
                      >
                        <summary class="article-expander__title">Determinants of mosaic chromosomal alteration fitness</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37905118/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is characterized by the acquisition of a somatic mutation in a hematopoietic stem cell that results in a clonal expansion. These driver mutations can be single nucleotide variants in cancer driver genes or larger structural rearrangements called mosaic chromosomal alterations (mCAs). The factors that influence the variations in mCA fitness and ultimately result in different clonal expansion rates are not well-understood. We used the Passenger-Approximated Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904876"
                        open
                      >
                        <summary class="article-expander__title">Germline and somatic drivers in inherited hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904876/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inherited hematologic malignancies are linked to a heterogenous group of genes, knowledge of which is rapidly expanding using panel-based next-generation sequencing (NGS) or whole-exome/whole-genome sequencing. Importantly, the penetrance for these syndromes is incomplete, and disease development, progression or transformation has critical clinical implications. With the earlier detection of healthy carriers and sequential monitoring of these patients, clonal hematopoiesis and somatic driver...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904125"
                        open
                      >
                        <summary class="article-expander__title">Prevalence and significance of clonal hematopoiesis of indeterminate potential in lung transplant recipients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Lung recipients have a higher prevalence of CHIP and a larger variety of genes with CHIP-associated mutations compared with previous reports for the general population. CHIP did not increase the risk of AR, CLAD, or death in lung recipients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37905118"
                        open
                      >
                        <summary class="article-expander__title">Determinants of mosaic chromosomal alteration fitness</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37905118/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231103204956&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is characterized by the acquisition of a somatic mutation in a hematopoietic stem cell that results in a clonal expansion. These driver mutations can be single nucleotide variants in cancer driver genes or larger structural rearrangements called mosaic chromosomal alterations (mCAs). The factors that influence the variations in mCA fitness and ultimately result in different clonal expansion rates are not well-understood. We used the Passenger-Approximated Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904876"
                        open
                      >
                        <summary class="article-expander__title">Germline and somatic drivers in inherited hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904876/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231103204956&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inherited hematologic malignancies are linked to a heterogenous group of genes, knowledge of which is rapidly expanding using panel-based next-generation sequencing (NGS) or whole-exome/whole-genome sequencing. Importantly, the penetrance for these syndromes is incomplete, and disease development, progression or transformation has critical clinical implications. With the earlier detection of healthy carriers and sequential monitoring of these patients, clonal hematopoiesis and somatic driver...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904125"
                        open
                      >
                        <summary class="article-expander__title">Prevalence and significance of clonal hematopoiesis of indeterminate potential in lung transplant recipients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231103204956&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Lung recipients have a higher prevalence of CHIP and a larger variety of genes with CHIP-associated mutations compared with previous reports for the general population. CHIP did not increase the risk of AR, CLAD, or death in lung recipients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37905118"
                        open
                      >
                        <summary class="article-expander__title">Determinants of mosaic chromosomal alteration fitness</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37905118/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is characterized by the acquisition of a somatic mutation in a hematopoietic stem cell that results in a clonal expansion. These driver mutations can be single nucleotide variants in cancer driver genes or larger structural rearrangements called mosaic chromosomal alterations (mCAs). The factors that influence the variations in mCA fitness and ultimately result in different clonal expansion rates are not well-understood. We used the Passenger-Approximated Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904876"
                        open
                      >
                        <summary class="article-expander__title">Germline and somatic drivers in inherited hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904876/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inherited hematologic malignancies are linked to a heterogenous group of genes, knowledge of which is rapidly expanding using panel-based next-generation sequencing (NGS) or whole-exome/whole-genome sequencing. Importantly, the penetrance for these syndromes is incomplete, and disease development, progression or transformation has critical clinical implications. With the earlier detection of healthy carriers and sequential monitoring of these patients, clonal hematopoiesis and somatic driver...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904125"
                        open
                      >
                        <summary class="article-expander__title">Prevalence and significance of clonal hematopoiesis of indeterminate potential in lung transplant recipients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Lung recipients have a higher prevalence of CHIP and a larger variety of genes with CHIP-associated mutations compared with previous reports for the general population. CHIP did not increase the risk of AR, CLAD, or death in lung recipients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37905118"
                        open
                      >
                        <summary class="article-expander__title">Determinants of mosaic chromosomal alteration fitness</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37905118/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is characterized by the acquisition of a somatic mutation in a hematopoietic stem cell that results in a clonal expansion. These driver mutations can be single nucleotide variants in cancer driver genes or larger structural rearrangements called mosaic chromosomal alterations (mCAs). The factors that influence the variations in mCA fitness and ultimately result in different clonal expansion rates are not well-understood. We used the Passenger-Approximated Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904876"
                        open
                      >
                        <summary class="article-expander__title">Germline and somatic drivers in inherited hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904876/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inherited hematologic malignancies are linked to a heterogenous group of genes, knowledge of which is rapidly expanding using panel-based next-generation sequencing (NGS) or whole-exome/whole-genome sequencing. Importantly, the penetrance for these syndromes is incomplete, and disease development, progression or transformation has critical clinical implications. With the earlier detection of healthy carriers and sequential monitoring of these patients, clonal hematopoiesis and somatic driver...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904125"
                        open
                      >
                        <summary class="article-expander__title">Prevalence and significance of clonal hematopoiesis of indeterminate potential in lung transplant recipients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Lung recipients have a higher prevalence of CHIP and a larger variety of genes with CHIP-associated mutations compared with previous reports for the general population. CHIP did not increase the risk of AR, CLAD, or death in lung recipients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37905118"
                        open
                      >
                        <summary class="article-expander__title">Determinants of mosaic chromosomal alteration fitness</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37905118/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is characterized by the acquisition of a somatic mutation in a hematopoietic stem cell that results in a clonal expansion. These driver mutations can be single nucleotide variants in cancer driver genes or larger structural rearrangements called mosaic chromosomal alterations (mCAs). The factors that influence the variations in mCA fitness and ultimately result in different clonal expansion rates are not well-understood. We used the Passenger-Approximated Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904876"
                        open
                      >
                        <summary class="article-expander__title">Germline and somatic drivers in inherited hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904876/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inherited hematologic malignancies are linked to a heterogenous group of genes, knowledge of which is rapidly expanding using panel-based next-generation sequencing (NGS) or whole-exome/whole-genome sequencing. Importantly, the penetrance for these syndromes is incomplete, and disease development, progression or transformation has critical clinical implications. With the earlier detection of healthy carriers and sequential monitoring of these patients, clonal hematopoiesis and somatic driver...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904125"
                        open
                      >
                        <summary class="article-expander__title">Prevalence and significance of clonal hematopoiesis of indeterminate potential in lung transplant recipients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Lung recipients have a higher prevalence of CHIP and a larger variety of genes with CHIP-associated mutations compared with previous reports for the general population. CHIP did not increase the risk of AR, CLAD, or death in lung recipients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;ff&#x3D;20231105195156&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904927"
                        open
                      >
                        <summary class="article-expander__title">Bioinformatics Illustrations Decoded by ChatGPT: The Good, The Bad, and The Ugly</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904927/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging studies underscore the promising capabilities of large language model-based chatbots in conducting fundamental bioinformatics data analyses. The recent feature of accepting image-inputs by ChatGPT motivated us to explore its efficacy in deciphering bioinformatics illustrations. Our evaluation with examples in cancer research, including sequencing data analysis, multimodal network-based drug repositioning, and tumor clonal evolution, revealed that ChatGPT can proficiently explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904927"
                        open
                      >
                        <summary class="article-expander__title">Bioinformatics Illustrations Decoded by ChatGPT: The Good, The Bad, and The Ugly</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904927/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging studies underscore the promising capabilities of large language model-based chatbots in conducting fundamental bioinformatics data analyses. The recent feature of accepting image-inputs by ChatGPT motivated us to explore its efficacy in deciphering bioinformatics illustrations. Our evaluation with examples in cancer research, including sequencing data analysis, multimodal network-based drug repositioning, and tumor clonal evolution, revealed that ChatGPT can proficiently explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904927"
                        open
                      >
                        <summary class="article-expander__title">Bioinformatics Illustrations Decoded by ChatGPT: The Good, The Bad, and The Ugly</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904927/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231103204955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging studies underscore the promising capabilities of large language model-based chatbots in conducting fundamental bioinformatics data analyses. The recent feature of accepting image-inputs by ChatGPT motivated us to explore its efficacy in deciphering bioinformatics illustrations. Our evaluation with examples in cancer research, including sequencing data analysis, multimodal network-based drug repositioning, and tumor clonal evolution, revealed that ChatGPT can proficiently explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904927"
                        open
                      >
                        <summary class="article-expander__title">Bioinformatics Illustrations Decoded by ChatGPT: The Good, The Bad, and The Ugly</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904927/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging studies underscore the promising capabilities of large language model-based chatbots in conducting fundamental bioinformatics data analyses. The recent feature of accepting image-inputs by ChatGPT motivated us to explore its efficacy in deciphering bioinformatics illustrations. Our evaluation with examples in cancer research, including sequencing data analysis, multimodal network-based drug repositioning, and tumor clonal evolution, revealed that ChatGPT can proficiently explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904927"
                        open
                      >
                        <summary class="article-expander__title">Bioinformatics Illustrations Decoded by ChatGPT: The Good, The Bad, and The Ugly</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904927/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging studies underscore the promising capabilities of large language model-based chatbots in conducting fundamental bioinformatics data analyses. The recent feature of accepting image-inputs by ChatGPT motivated us to explore its efficacy in deciphering bioinformatics illustrations. Our evaluation with examples in cancer research, including sequencing data analysis, multimodal network-based drug repositioning, and tumor clonal evolution, revealed that ChatGPT can proficiently explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904927"
                        open
                      >
                        <summary class="article-expander__title">Bioinformatics Illustrations Decoded by ChatGPT: The Good, The Bad, and The Ugly</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904927/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging studies underscore the promising capabilities of large language model-based chatbots in conducting fundamental bioinformatics data analyses. The recent feature of accepting image-inputs by ChatGPT motivated us to explore its efficacy in deciphering bioinformatics illustrations. Our evaluation with examples in cancer research, including sequencing data analysis, multimodal network-based drug repositioning, and tumor clonal evolution, revealed that ChatGPT can proficiently explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-10-30 datetime="2023-10-30T10:00:00.000Z">2023-10-30</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-10-30 datetime="2023-10-30T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20231105195156">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37901004"
                        open
                      >
                        <summary class="article-expander__title">Leukemic transformation during anti-tuberculosis treatment in aplastic anemia-paroxysmal nocturnal hemoglobinuria syndrome: A case report and review of literature</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37901004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Aggravated inflammatory stressors can repress normal and leukemic hematopoiesis, and relieved inflammatory stressors can facilitate penetration of neoplastic hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37901004"
                        open
                      >
                        <summary class="article-expander__title">Leukemic transformation during anti-tuberculosis treatment in aplastic anemia-paroxysmal nocturnal hemoglobinuria syndrome: A case report and review of literature</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37901004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Aggravated inflammatory stressors can repress normal and leukemic hematopoiesis, and relieved inflammatory stressors can facilitate penetration of neoplastic hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37901004"
                        open
                      >
                        <summary class="article-expander__title">Leukemic transformation during anti-tuberculosis treatment in aplastic anemia-paroxysmal nocturnal hemoglobinuria syndrome: A case report and review of literature</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37901004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231103204956&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Aggravated inflammatory stressors can repress normal and leukemic hematopoiesis, and relieved inflammatory stressors can facilitate penetration of neoplastic hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37901004"
                        open
                      >
                        <summary class="article-expander__title">Leukemic transformation during anti-tuberculosis treatment in aplastic anemia-paroxysmal nocturnal hemoglobinuria syndrome: A case report and review of literature</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37901004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Aggravated inflammatory stressors can repress normal and leukemic hematopoiesis, and relieved inflammatory stressors can facilitate penetration of neoplastic hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37901004"
                        open
                      >
                        <summary class="article-expander__title">Leukemic transformation during anti-tuberculosis treatment in aplastic anemia-paroxysmal nocturnal hemoglobinuria syndrome: A case report and review of literature</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37901004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Aggravated inflammatory stressors can repress normal and leukemic hematopoiesis, and relieved inflammatory stressors can facilitate penetration of neoplastic hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37901004"
                        open
                      >
                        <summary class="article-expander__title">Leukemic transformation during anti-tuberculosis treatment in aplastic anemia-paroxysmal nocturnal hemoglobinuria syndrome: A case report and review of literature</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37901004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Aggravated inflammatory stressors can repress normal and leukemic hematopoiesis, and relieved inflammatory stressors can facilitate penetration of neoplastic hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37901004"
                        open
                      >
                        <summary class="article-expander__title">Leukemic transformation during anti-tuberculosis treatment in aplastic anemia-paroxysmal nocturnal hemoglobinuria syndrome: A case report and review of literature</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37901004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Aggravated inflammatory stressors can repress normal and leukemic hematopoiesis, and relieved inflammatory stressors can facilitate penetration of neoplastic hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;ff&#x3D;20231105195156&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37903225"
                        open
                      >
                        <summary class="article-expander__title">Differences in cell shape, motility, and growth reflect chromosomal number variations that can be visualized with live-cell ChReporters</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37903225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosome numbers often change dynamically in tumors and in cultured cells, which complicates therapy as well as understanding genotype-mechanotype relationships. Here we use a live-cell &#x27;ChReporter&#x27; method to identify cells with a single chromosomal loss in efforts to better understand differences in cell shape, motility, and growth. We focus on a standard lung cancer line and first show clonal populations that retain the ChReporter exhibit large differences in cell and nuclear morphology as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37901004"
                        open
                      >
                        <summary class="article-expander__title">Leukemic transformation during anti-tuberculosis treatment in aplastic anemia-paroxysmal nocturnal hemoglobinuria syndrome: A case report and review of literature</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37901004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Aggravated inflammatory stressors can repress normal and leukemic hematopoiesis, and relieved inflammatory stressors can facilitate penetration of neoplastic hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37903225"
                        open
                      >
                        <summary class="article-expander__title">Differences in cell shape, motility, and growth reflect chromosomal number variations that can be visualized with live-cell ChReporters</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37903225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosome numbers often change dynamically in tumors and in cultured cells, which complicates therapy as well as understanding genotype-mechanotype relationships. Here we use a live-cell &#x27;ChReporter&#x27; method to identify cells with a single chromosomal loss in efforts to better understand differences in cell shape, motility, and growth. We focus on a standard lung cancer line and first show clonal populations that retain the ChReporter exhibit large differences in cell and nuclear morphology as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37901004"
                        open
                      >
                        <summary class="article-expander__title">Leukemic transformation during anti-tuberculosis treatment in aplastic anemia-paroxysmal nocturnal hemoglobinuria syndrome: A case report and review of literature</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37901004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Aggravated inflammatory stressors can repress normal and leukemic hematopoiesis, and relieved inflammatory stressors can facilitate penetration of neoplastic hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37903225"
                        open
                      >
                        <summary class="article-expander__title">Differences in cell shape, motility, and growth reflect chromosomal number variations that can be visualized with live-cell ChReporters</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37903225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231103204955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosome numbers often change dynamically in tumors and in cultured cells, which complicates therapy as well as understanding genotype-mechanotype relationships. Here we use a live-cell &#x27;ChReporter&#x27; method to identify cells with a single chromosomal loss in efforts to better understand differences in cell shape, motility, and growth. We focus on a standard lung cancer line and first show clonal populations that retain the ChReporter exhibit large differences in cell and nuclear morphology as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37901004"
                        open
                      >
                        <summary class="article-expander__title">Leukemic transformation during anti-tuberculosis treatment in aplastic anemia-paroxysmal nocturnal hemoglobinuria syndrome: A case report and review of literature</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37901004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231103204955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Aggravated inflammatory stressors can repress normal and leukemic hematopoiesis, and relieved inflammatory stressors can facilitate penetration of neoplastic hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37903225"
                        open
                      >
                        <summary class="article-expander__title">Differences in cell shape, motility, and growth reflect chromosomal number variations that can be visualized with live-cell ChReporters</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37903225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosome numbers often change dynamically in tumors and in cultured cells, which complicates therapy as well as understanding genotype-mechanotype relationships. Here we use a live-cell &#x27;ChReporter&#x27; method to identify cells with a single chromosomal loss in efforts to better understand differences in cell shape, motility, and growth. We focus on a standard lung cancer line and first show clonal populations that retain the ChReporter exhibit large differences in cell and nuclear morphology as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37901004"
                        open
                      >
                        <summary class="article-expander__title">Leukemic transformation during anti-tuberculosis treatment in aplastic anemia-paroxysmal nocturnal hemoglobinuria syndrome: A case report and review of literature</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37901004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Aggravated inflammatory stressors can repress normal and leukemic hematopoiesis, and relieved inflammatory stressors can facilitate penetration of neoplastic hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37903225"
                        open
                      >
                        <summary class="article-expander__title">Differences in cell shape, motility, and growth reflect chromosomal number variations that can be visualized with live-cell ChReporters</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37903225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosome numbers often change dynamically in tumors and in cultured cells, which complicates therapy as well as understanding genotype-mechanotype relationships. Here we use a live-cell &#x27;ChReporter&#x27; method to identify cells with a single chromosomal loss in efforts to better understand differences in cell shape, motility, and growth. We focus on a standard lung cancer line and first show clonal populations that retain the ChReporter exhibit large differences in cell and nuclear morphology as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37901004"
                        open
                      >
                        <summary class="article-expander__title">Leukemic transformation during anti-tuberculosis treatment in aplastic anemia-paroxysmal nocturnal hemoglobinuria syndrome: A case report and review of literature</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37901004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Aggravated inflammatory stressors can repress normal and leukemic hematopoiesis, and relieved inflammatory stressors can facilitate penetration of neoplastic hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37903225"
                        open
                      >
                        <summary class="article-expander__title">Differences in cell shape, motility, and growth reflect chromosomal number variations that can be visualized with live-cell ChReporters</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37903225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosome numbers often change dynamically in tumors and in cultured cells, which complicates therapy as well as understanding genotype-mechanotype relationships. Here we use a live-cell &#x27;ChReporter&#x27; method to identify cells with a single chromosomal loss in efforts to better understand differences in cell shape, motility, and growth. We focus on a standard lung cancer line and first show clonal populations that retain the ChReporter exhibit large differences in cell and nuclear morphology as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37901004"
                        open
                      >
                        <summary class="article-expander__title">Leukemic transformation during anti-tuberculosis treatment in aplastic anemia-paroxysmal nocturnal hemoglobinuria syndrome: A case report and review of literature</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37901004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Aggravated inflammatory stressors can repress normal and leukemic hematopoiesis, and relieved inflammatory stressors can facilitate penetration of neoplastic hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37903225"
                        open
                      >
                        <summary class="article-expander__title">Differences in cell shape, motility, and growth reflect chromosomal number variations that can be visualized with live-cell ChReporters</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37903225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosome numbers often change dynamically in tumors and in cultured cells, which complicates therapy as well as understanding genotype-mechanotype relationships. Here we use a live-cell &#x27;ChReporter&#x27; method to identify cells with a single chromosomal loss in efforts to better understand differences in cell shape, motility, and growth. We focus on a standard lung cancer line and first show clonal populations that retain the ChReporter exhibit large differences in cell and nuclear morphology as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37901004"
                        open
                      >
                        <summary class="article-expander__title">Leukemic transformation during anti-tuberculosis treatment in aplastic anemia-paroxysmal nocturnal hemoglobinuria syndrome: A case report and review of literature</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37901004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Aggravated inflammatory stressors can repress normal and leukemic hematopoiesis, and relieved inflammatory stressors can facilitate penetration of neoplastic hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-10-28 datetime="2023-10-28T10:00:00.000Z">2023-10-28</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-10-28 datetime="2023-10-28T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20231105195156">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895120"
                        open
                      >
                        <summary class="article-expander__title">Deciphering Potential Molecular Signatures to Differentiate Acute Myeloid Leukemia (AML) with BCR::ABL1 from Chronic Myeloid Leukemia (CML) in Blast Crisis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895120/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) with BCR::ABL1 has recently been recognized as a distinct subtype in international classifications. Distinguishing it from myeloid blast crisis chronic myeloid leukemia (BC-CML) without evidence of a chronic phase (CP), remains challenging. We aimed to better characterize this entity by integrating clonal architecture analysis, mutational landscape assessment, and gene expression profiling. We analyzed a large retrospective cohort study including CML and AML...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37892987"
                        open
                      >
                        <summary class="article-expander__title">Emerging Insights into Molecular Mechanisms of Inflammation in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37892987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inflammation impacts human hematopoiesis across physiologic and pathologic conditions, as signals derived from the bone marrow microenvironment, such as pro-inflammatory cytokines and chemokines, have been shown to alter hematopoietic stem cell (HSCs) homeostasis. Dysregulated inflammation can skew HSC fate-related decisions, leading to aberrant hematopoiesis and potentially contributing to the pathogenesis of hematological disorders such as myelodysplastic syndromes (MDS). Recently, emerging...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895120"
                        open
                      >
                        <summary class="article-expander__title">Deciphering Potential Molecular Signatures to Differentiate Acute Myeloid Leukemia (AML) with BCR::ABL1 from Chronic Myeloid Leukemia (CML) in Blast Crisis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895120/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) with BCR::ABL1 has recently been recognized as a distinct subtype in international classifications. Distinguishing it from myeloid blast crisis chronic myeloid leukemia (BC-CML) without evidence of a chronic phase (CP), remains challenging. We aimed to better characterize this entity by integrating clonal architecture analysis, mutational landscape assessment, and gene expression profiling. We analyzed a large retrospective cohort study including CML and AML...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37892987"
                        open
                      >
                        <summary class="article-expander__title">Emerging Insights into Molecular Mechanisms of Inflammation in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37892987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inflammation impacts human hematopoiesis across physiologic and pathologic conditions, as signals derived from the bone marrow microenvironment, such as pro-inflammatory cytokines and chemokines, have been shown to alter hematopoietic stem cell (HSCs) homeostasis. Dysregulated inflammation can skew HSC fate-related decisions, leading to aberrant hematopoiesis and potentially contributing to the pathogenesis of hematological disorders such as myelodysplastic syndromes (MDS). Recently, emerging...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895120"
                        open
                      >
                        <summary class="article-expander__title">Deciphering Potential Molecular Signatures to Differentiate Acute Myeloid Leukemia (AML) with BCR::ABL1 from Chronic Myeloid Leukemia (CML) in Blast Crisis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895120/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231103204956&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) with BCR::ABL1 has recently been recognized as a distinct subtype in international classifications. Distinguishing it from myeloid blast crisis chronic myeloid leukemia (BC-CML) without evidence of a chronic phase (CP), remains challenging. We aimed to better characterize this entity by integrating clonal architecture analysis, mutational landscape assessment, and gene expression profiling. We analyzed a large retrospective cohort study including CML and AML...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37892987"
                        open
                      >
                        <summary class="article-expander__title">Emerging Insights into Molecular Mechanisms of Inflammation in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37892987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231103204956&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inflammation impacts human hematopoiesis across physiologic and pathologic conditions, as signals derived from the bone marrow microenvironment, such as pro-inflammatory cytokines and chemokines, have been shown to alter hematopoietic stem cell (HSCs) homeostasis. Dysregulated inflammation can skew HSC fate-related decisions, leading to aberrant hematopoiesis and potentially contributing to the pathogenesis of hematological disorders such as myelodysplastic syndromes (MDS). Recently, emerging...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895120"
                        open
                      >
                        <summary class="article-expander__title">Deciphering Potential Molecular Signatures to Differentiate Acute Myeloid Leukemia (AML) with BCR::ABL1 from Chronic Myeloid Leukemia (CML) in Blast Crisis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895120/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) with BCR::ABL1 has recently been recognized as a distinct subtype in international classifications. Distinguishing it from myeloid blast crisis chronic myeloid leukemia (BC-CML) without evidence of a chronic phase (CP), remains challenging. We aimed to better characterize this entity by integrating clonal architecture analysis, mutational landscape assessment, and gene expression profiling. We analyzed a large retrospective cohort study including CML and AML...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37892987"
                        open
                      >
                        <summary class="article-expander__title">Emerging Insights into Molecular Mechanisms of Inflammation in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37892987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inflammation impacts human hematopoiesis across physiologic and pathologic conditions, as signals derived from the bone marrow microenvironment, such as pro-inflammatory cytokines and chemokines, have been shown to alter hematopoietic stem cell (HSCs) homeostasis. Dysregulated inflammation can skew HSC fate-related decisions, leading to aberrant hematopoiesis and potentially contributing to the pathogenesis of hematological disorders such as myelodysplastic syndromes (MDS). Recently, emerging...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895120"
                        open
                      >
                        <summary class="article-expander__title">Deciphering Potential Molecular Signatures to Differentiate Acute Myeloid Leukemia (AML) with BCR::ABL1 from Chronic Myeloid Leukemia (CML) in Blast Crisis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895120/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) with BCR::ABL1 has recently been recognized as a distinct subtype in international classifications. Distinguishing it from myeloid blast crisis chronic myeloid leukemia (BC-CML) without evidence of a chronic phase (CP), remains challenging. We aimed to better characterize this entity by integrating clonal architecture analysis, mutational landscape assessment, and gene expression profiling. We analyzed a large retrospective cohort study including CML and AML...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37892987"
                        open
                      >
                        <summary class="article-expander__title">Emerging Insights into Molecular Mechanisms of Inflammation in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37892987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inflammation impacts human hematopoiesis across physiologic and pathologic conditions, as signals derived from the bone marrow microenvironment, such as pro-inflammatory cytokines and chemokines, have been shown to alter hematopoietic stem cell (HSCs) homeostasis. Dysregulated inflammation can skew HSC fate-related decisions, leading to aberrant hematopoiesis and potentially contributing to the pathogenesis of hematological disorders such as myelodysplastic syndromes (MDS). Recently, emerging...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895120"
                        open
                      >
                        <summary class="article-expander__title">Deciphering Potential Molecular Signatures to Differentiate Acute Myeloid Leukemia (AML) with BCR::ABL1 from Chronic Myeloid Leukemia (CML) in Blast Crisis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895120/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) with BCR::ABL1 has recently been recognized as a distinct subtype in international classifications. Distinguishing it from myeloid blast crisis chronic myeloid leukemia (BC-CML) without evidence of a chronic phase (CP), remains challenging. We aimed to better characterize this entity by integrating clonal architecture analysis, mutational landscape assessment, and gene expression profiling. We analyzed a large retrospective cohort study including CML and AML...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37892987"
                        open
                      >
                        <summary class="article-expander__title">Emerging Insights into Molecular Mechanisms of Inflammation in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37892987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inflammation impacts human hematopoiesis across physiologic and pathologic conditions, as signals derived from the bone marrow microenvironment, such as pro-inflammatory cytokines and chemokines, have been shown to alter hematopoietic stem cell (HSCs) homeostasis. Dysregulated inflammation can skew HSC fate-related decisions, leading to aberrant hematopoiesis and potentially contributing to the pathogenesis of hematological disorders such as myelodysplastic syndromes (MDS). Recently, emerging...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895120"
                        open
                      >
                        <summary class="article-expander__title">Deciphering Potential Molecular Signatures to Differentiate Acute Myeloid Leukemia (AML) with BCR::ABL1 from Chronic Myeloid Leukemia (CML) in Blast Crisis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895120/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) with BCR::ABL1 has recently been recognized as a distinct subtype in international classifications. Distinguishing it from myeloid blast crisis chronic myeloid leukemia (BC-CML) without evidence of a chronic phase (CP), remains challenging. We aimed to better characterize this entity by integrating clonal architecture analysis, mutational landscape assessment, and gene expression profiling. We analyzed a large retrospective cohort study including CML and AML...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37892987"
                        open
                      >
                        <summary class="article-expander__title">Emerging Insights into Molecular Mechanisms of Inflammation in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37892987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inflammation impacts human hematopoiesis across physiologic and pathologic conditions, as signals derived from the bone marrow microenvironment, such as pro-inflammatory cytokines and chemokines, have been shown to alter hematopoietic stem cell (HSCs) homeostasis. Dysregulated inflammation can skew HSC fate-related decisions, leading to aberrant hematopoiesis and potentially contributing to the pathogenesis of hematological disorders such as myelodysplastic syndromes (MDS). Recently, emerging...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895120"
                        open
                      >
                        <summary class="article-expander__title">Deciphering Potential Molecular Signatures to Differentiate Acute Myeloid Leukemia (AML) with BCR::ABL1 from Chronic Myeloid Leukemia (CML) in Blast Crisis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895120/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) with BCR::ABL1 has recently been recognized as a distinct subtype in international classifications. Distinguishing it from myeloid blast crisis chronic myeloid leukemia (BC-CML) without evidence of a chronic phase (CP), remains challenging. We aimed to better characterize this entity by integrating clonal architecture analysis, mutational landscape assessment, and gene expression profiling. We analyzed a large retrospective cohort study including CML and AML...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37892987"
                        open
                      >
                        <summary class="article-expander__title">Emerging Insights into Molecular Mechanisms of Inflammation in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37892987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inflammation impacts human hematopoiesis across physiologic and pathologic conditions, as signals derived from the bone marrow microenvironment, such as pro-inflammatory cytokines and chemokines, have been shown to alter hematopoietic stem cell (HSCs) homeostasis. Dysregulated inflammation can skew HSC fate-related decisions, leading to aberrant hematopoiesis and potentially contributing to the pathogenesis of hematological disorders such as myelodysplastic syndromes (MDS). Recently, emerging...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895120"
                        open
                      >
                        <summary class="article-expander__title">Deciphering Potential Molecular Signatures to Differentiate Acute Myeloid Leukemia (AML) with BCR::ABL1 from Chronic Myeloid Leukemia (CML) in Blast Crisis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895120/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) with BCR::ABL1 has recently been recognized as a distinct subtype in international classifications. Distinguishing it from myeloid blast crisis chronic myeloid leukemia (BC-CML) without evidence of a chronic phase (CP), remains challenging. We aimed to better characterize this entity by integrating clonal architecture analysis, mutational landscape assessment, and gene expression profiling. We analyzed a large retrospective cohort study including CML and AML...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37892987"
                        open
                      >
                        <summary class="article-expander__title">Emerging Insights into Molecular Mechanisms of Inflammation in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37892987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inflammation impacts human hematopoiesis across physiologic and pathologic conditions, as signals derived from the bone marrow microenvironment, such as pro-inflammatory cytokines and chemokines, have been shown to alter hematopoietic stem cell (HSCs) homeostasis. Dysregulated inflammation can skew HSC fate-related decisions, leading to aberrant hematopoiesis and potentially contributing to the pathogenesis of hematological disorders such as myelodysplastic syndromes (MDS). Recently, emerging...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;ff&#x3D;20231105195156&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37898477"
                        open
                      >
                        <summary class="article-expander__title">Adaptive therapy: a tumor therapy strategy based on Darwinian evolution theory</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37898477/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer progression is a dynamic process of continuous evolution, in which genetic diversity and heterogeneity are generated by clonal and subclonal amplification based on random mutations. Traditional cancer treatment strategies have a great challenge, which often leads to treatment failure due to drug resistance. Integrating evolutionary dynamics into treatment regimens may be an effective way to overcome the problem of drug resistance. In particular, a potential treatment is adaptive therapy,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895205"
                        open
                      >
                        <summary class="article-expander__title">Decoding Cancer Evolution: Integrating Genetic and Non-Genetic Insights</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The development of cancer begins with cells transitioning from their multicellular nature to a state akin to unicellular organisms. This shift leads to a breakdown in the crucial regulators inherent to multicellularity, resulting in the emergence of diverse cancer cell subpopulations that have enhanced adaptability. The presence of different cell subpopulations within a tumour, known as intratumoural heterogeneity (ITH), poses challenges for cancer treatment. In this review, we delve into the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895004"
                        open
                      >
                        <summary class="article-expander__title">Hematologic Neoplasms Associated with Down Syndrome: Cellular and Molecular Heterogeneity of the Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The molecular basis of Down syndrome (DS) predisposition to leukemia is not fully understood but involves various factors such as chromosomal abnormalities, oncogenic mutations, epigenetic alterations, and changes in selection dynamics. Myeloid leukemia associated with DS (ML-DS) is preceded by a preleukemic phase called transient abnormal myelopoiesis driven by GATA1 gene mutations and progresses to ML-DS via additional mutations in cohesin genes, CTCF, RAS, or JAK/STAT pathway genes....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37894756"
                        open
                      >
                        <summary class="article-expander__title">Overview of Tumor Heterogeneity in High-Grade Serous Ovarian Cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37894756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancers encompass a group of neoplasms originating from germinal tissues and exhibiting distinct clinical, pathological, and molecular features. Among these, epithelial ovarian cancers (EOCs) are the most prevalent, comprising five distinct tumor histotypes. Notably, high-grade serous ovarian cancers (HGSOCs) represent the majority, accounting for over 70% of EOC cases. Due to their silent and asymptomatic behavior, HGSOCs are generally diagnosed in advanced stages with an evolved and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37898477"
                        open
                      >
                        <summary class="article-expander__title">Adaptive therapy: a tumor therapy strategy based on Darwinian evolution theory</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37898477/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer progression is a dynamic process of continuous evolution, in which genetic diversity and heterogeneity are generated by clonal and subclonal amplification based on random mutations. Traditional cancer treatment strategies have a great challenge, which often leads to treatment failure due to drug resistance. Integrating evolutionary dynamics into treatment regimens may be an effective way to overcome the problem of drug resistance. In particular, a potential treatment is adaptive therapy,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895205"
                        open
                      >
                        <summary class="article-expander__title">Decoding Cancer Evolution: Integrating Genetic and Non-Genetic Insights</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The development of cancer begins with cells transitioning from their multicellular nature to a state akin to unicellular organisms. This shift leads to a breakdown in the crucial regulators inherent to multicellularity, resulting in the emergence of diverse cancer cell subpopulations that have enhanced adaptability. The presence of different cell subpopulations within a tumour, known as intratumoural heterogeneity (ITH), poses challenges for cancer treatment. In this review, we delve into the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895004"
                        open
                      >
                        <summary class="article-expander__title">Hematologic Neoplasms Associated with Down Syndrome: Cellular and Molecular Heterogeneity of the Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The molecular basis of Down syndrome (DS) predisposition to leukemia is not fully understood but involves various factors such as chromosomal abnormalities, oncogenic mutations, epigenetic alterations, and changes in selection dynamics. Myeloid leukemia associated with DS (ML-DS) is preceded by a preleukemic phase called transient abnormal myelopoiesis driven by GATA1 gene mutations and progresses to ML-DS via additional mutations in cohesin genes, CTCF, RAS, or JAK/STAT pathway genes....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37894756"
                        open
                      >
                        <summary class="article-expander__title">Overview of Tumor Heterogeneity in High-Grade Serous Ovarian Cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37894756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancers encompass a group of neoplasms originating from germinal tissues and exhibiting distinct clinical, pathological, and molecular features. Among these, epithelial ovarian cancers (EOCs) are the most prevalent, comprising five distinct tumor histotypes. Notably, high-grade serous ovarian cancers (HGSOCs) represent the majority, accounting for over 70% of EOC cases. Due to their silent and asymptomatic behavior, HGSOCs are generally diagnosed in advanced stages with an evolved and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37898477"
                        open
                      >
                        <summary class="article-expander__title">Adaptive therapy: a tumor therapy strategy based on Darwinian evolution theory</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37898477/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231103204955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer progression is a dynamic process of continuous evolution, in which genetic diversity and heterogeneity are generated by clonal and subclonal amplification based on random mutations. Traditional cancer treatment strategies have a great challenge, which often leads to treatment failure due to drug resistance. Integrating evolutionary dynamics into treatment regimens may be an effective way to overcome the problem of drug resistance. In particular, a potential treatment is adaptive therapy,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895205"
                        open
                      >
                        <summary class="article-expander__title">Decoding Cancer Evolution: Integrating Genetic and Non-Genetic Insights</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231103204955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The development of cancer begins with cells transitioning from their multicellular nature to a state akin to unicellular organisms. This shift leads to a breakdown in the crucial regulators inherent to multicellularity, resulting in the emergence of diverse cancer cell subpopulations that have enhanced adaptability. The presence of different cell subpopulations within a tumour, known as intratumoural heterogeneity (ITH), poses challenges for cancer treatment. In this review, we delve into the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895004"
                        open
                      >
                        <summary class="article-expander__title">Hematologic Neoplasms Associated with Down Syndrome: Cellular and Molecular Heterogeneity of the Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231103204955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The molecular basis of Down syndrome (DS) predisposition to leukemia is not fully understood but involves various factors such as chromosomal abnormalities, oncogenic mutations, epigenetic alterations, and changes in selection dynamics. Myeloid leukemia associated with DS (ML-DS) is preceded by a preleukemic phase called transient abnormal myelopoiesis driven by GATA1 gene mutations and progresses to ML-DS via additional mutations in cohesin genes, CTCF, RAS, or JAK/STAT pathway genes....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37894756"
                        open
                      >
                        <summary class="article-expander__title">Overview of Tumor Heterogeneity in High-Grade Serous Ovarian Cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37894756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231103204955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancers encompass a group of neoplasms originating from germinal tissues and exhibiting distinct clinical, pathological, and molecular features. Among these, epithelial ovarian cancers (EOCs) are the most prevalent, comprising five distinct tumor histotypes. Notably, high-grade serous ovarian cancers (HGSOCs) represent the majority, accounting for over 70% of EOC cases. Due to their silent and asymptomatic behavior, HGSOCs are generally diagnosed in advanced stages with an evolved and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37898477"
                        open
                      >
                        <summary class="article-expander__title">Adaptive therapy: a tumor therapy strategy based on Darwinian evolution theory</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37898477/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer progression is a dynamic process of continuous evolution, in which genetic diversity and heterogeneity are generated by clonal and subclonal amplification based on random mutations. Traditional cancer treatment strategies have a great challenge, which often leads to treatment failure due to drug resistance. Integrating evolutionary dynamics into treatment regimens may be an effective way to overcome the problem of drug resistance. In particular, a potential treatment is adaptive therapy,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895205"
                        open
                      >
                        <summary class="article-expander__title">Decoding Cancer Evolution: Integrating Genetic and Non-Genetic Insights</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The development of cancer begins with cells transitioning from their multicellular nature to a state akin to unicellular organisms. This shift leads to a breakdown in the crucial regulators inherent to multicellularity, resulting in the emergence of diverse cancer cell subpopulations that have enhanced adaptability. The presence of different cell subpopulations within a tumour, known as intratumoural heterogeneity (ITH), poses challenges for cancer treatment. In this review, we delve into the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895004"
                        open
                      >
                        <summary class="article-expander__title">Hematologic Neoplasms Associated with Down Syndrome: Cellular and Molecular Heterogeneity of the Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The molecular basis of Down syndrome (DS) predisposition to leukemia is not fully understood but involves various factors such as chromosomal abnormalities, oncogenic mutations, epigenetic alterations, and changes in selection dynamics. Myeloid leukemia associated with DS (ML-DS) is preceded by a preleukemic phase called transient abnormal myelopoiesis driven by GATA1 gene mutations and progresses to ML-DS via additional mutations in cohesin genes, CTCF, RAS, or JAK/STAT pathway genes....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37894756"
                        open
                      >
                        <summary class="article-expander__title">Overview of Tumor Heterogeneity in High-Grade Serous Ovarian Cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37894756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancers encompass a group of neoplasms originating from germinal tissues and exhibiting distinct clinical, pathological, and molecular features. Among these, epithelial ovarian cancers (EOCs) are the most prevalent, comprising five distinct tumor histotypes. Notably, high-grade serous ovarian cancers (HGSOCs) represent the majority, accounting for over 70% of EOC cases. Due to their silent and asymptomatic behavior, HGSOCs are generally diagnosed in advanced stages with an evolved and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37898477"
                        open
                      >
                        <summary class="article-expander__title">Adaptive therapy: a tumor therapy strategy based on Darwinian evolution theory</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37898477/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer progression is a dynamic process of continuous evolution, in which genetic diversity and heterogeneity are generated by clonal and subclonal amplification based on random mutations. Traditional cancer treatment strategies have a great challenge, which often leads to treatment failure due to drug resistance. Integrating evolutionary dynamics into treatment regimens may be an effective way to overcome the problem of drug resistance. In particular, a potential treatment is adaptive therapy,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895205"
                        open
                      >
                        <summary class="article-expander__title">Decoding Cancer Evolution: Integrating Genetic and Non-Genetic Insights</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The development of cancer begins with cells transitioning from their multicellular nature to a state akin to unicellular organisms. This shift leads to a breakdown in the crucial regulators inherent to multicellularity, resulting in the emergence of diverse cancer cell subpopulations that have enhanced adaptability. The presence of different cell subpopulations within a tumour, known as intratumoural heterogeneity (ITH), poses challenges for cancer treatment. In this review, we delve into the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895004"
                        open
                      >
                        <summary class="article-expander__title">Hematologic Neoplasms Associated with Down Syndrome: Cellular and Molecular Heterogeneity of the Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The molecular basis of Down syndrome (DS) predisposition to leukemia is not fully understood but involves various factors such as chromosomal abnormalities, oncogenic mutations, epigenetic alterations, and changes in selection dynamics. Myeloid leukemia associated with DS (ML-DS) is preceded by a preleukemic phase called transient abnormal myelopoiesis driven by GATA1 gene mutations and progresses to ML-DS via additional mutations in cohesin genes, CTCF, RAS, or JAK/STAT pathway genes....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37894756"
                        open
                      >
                        <summary class="article-expander__title">Overview of Tumor Heterogeneity in High-Grade Serous Ovarian Cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37894756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancers encompass a group of neoplasms originating from germinal tissues and exhibiting distinct clinical, pathological, and molecular features. Among these, epithelial ovarian cancers (EOCs) are the most prevalent, comprising five distinct tumor histotypes. Notably, high-grade serous ovarian cancers (HGSOCs) represent the majority, accounting for over 70% of EOC cases. Due to their silent and asymptomatic behavior, HGSOCs are generally diagnosed in advanced stages with an evolved and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37898477"
                        open
                      >
                        <summary class="article-expander__title">Adaptive therapy: a tumor therapy strategy based on Darwinian evolution theory</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37898477/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer progression is a dynamic process of continuous evolution, in which genetic diversity and heterogeneity are generated by clonal and subclonal amplification based on random mutations. Traditional cancer treatment strategies have a great challenge, which often leads to treatment failure due to drug resistance. Integrating evolutionary dynamics into treatment regimens may be an effective way to overcome the problem of drug resistance. In particular, a potential treatment is adaptive therapy,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895205"
                        open
                      >
                        <summary class="article-expander__title">Decoding Cancer Evolution: Integrating Genetic and Non-Genetic Insights</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The development of cancer begins with cells transitioning from their multicellular nature to a state akin to unicellular organisms. This shift leads to a breakdown in the crucial regulators inherent to multicellularity, resulting in the emergence of diverse cancer cell subpopulations that have enhanced adaptability. The presence of different cell subpopulations within a tumour, known as intratumoural heterogeneity (ITH), poses challenges for cancer treatment. In this review, we delve into the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895004"
                        open
                      >
                        <summary class="article-expander__title">Hematologic Neoplasms Associated with Down Syndrome: Cellular and Molecular Heterogeneity of the Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The molecular basis of Down syndrome (DS) predisposition to leukemia is not fully understood but involves various factors such as chromosomal abnormalities, oncogenic mutations, epigenetic alterations, and changes in selection dynamics. Myeloid leukemia associated with DS (ML-DS) is preceded by a preleukemic phase called transient abnormal myelopoiesis driven by GATA1 gene mutations and progresses to ML-DS via additional mutations in cohesin genes, CTCF, RAS, or JAK/STAT pathway genes....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37894756"
                        open
                      >
                        <summary class="article-expander__title">Overview of Tumor Heterogeneity in High-Grade Serous Ovarian Cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37894756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancers encompass a group of neoplasms originating from germinal tissues and exhibiting distinct clinical, pathological, and molecular features. Among these, epithelial ovarian cancers (EOCs) are the most prevalent, comprising five distinct tumor histotypes. Notably, high-grade serous ovarian cancers (HGSOCs) represent the majority, accounting for over 70% of EOC cases. Due to their silent and asymptomatic behavior, HGSOCs are generally diagnosed in advanced stages with an evolved and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37898477"
                        open
                      >
                        <summary class="article-expander__title">Adaptive therapy: a tumor therapy strategy based on Darwinian evolution theory</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37898477/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer progression is a dynamic process of continuous evolution, in which genetic diversity and heterogeneity are generated by clonal and subclonal amplification based on random mutations. Traditional cancer treatment strategies have a great challenge, which often leads to treatment failure due to drug resistance. Integrating evolutionary dynamics into treatment regimens may be an effective way to overcome the problem of drug resistance. In particular, a potential treatment is adaptive therapy,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895205"
                        open
                      >
                        <summary class="article-expander__title">Decoding Cancer Evolution: Integrating Genetic and Non-Genetic Insights</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The development of cancer begins with cells transitioning from their multicellular nature to a state akin to unicellular organisms. This shift leads to a breakdown in the crucial regulators inherent to multicellularity, resulting in the emergence of diverse cancer cell subpopulations that have enhanced adaptability. The presence of different cell subpopulations within a tumour, known as intratumoural heterogeneity (ITH), poses challenges for cancer treatment. In this review, we delve into the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895004"
                        open
                      >
                        <summary class="article-expander__title">Hematologic Neoplasms Associated with Down Syndrome: Cellular and Molecular Heterogeneity of the Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The molecular basis of Down syndrome (DS) predisposition to leukemia is not fully understood but involves various factors such as chromosomal abnormalities, oncogenic mutations, epigenetic alterations, and changes in selection dynamics. Myeloid leukemia associated with DS (ML-DS) is preceded by a preleukemic phase called transient abnormal myelopoiesis driven by GATA1 gene mutations and progresses to ML-DS via additional mutations in cohesin genes, CTCF, RAS, or JAK/STAT pathway genes....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37894756"
                        open
                      >
                        <summary class="article-expander__title">Overview of Tumor Heterogeneity in High-Grade Serous Ovarian Cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37894756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancers encompass a group of neoplasms originating from germinal tissues and exhibiting distinct clinical, pathological, and molecular features. Among these, epithelial ovarian cancers (EOCs) are the most prevalent, comprising five distinct tumor histotypes. Notably, high-grade serous ovarian cancers (HGSOCs) represent the majority, accounting for over 70% of EOC cases. Due to their silent and asymptomatic behavior, HGSOCs are generally diagnosed in advanced stages with an evolved and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37898477"
                        open
                      >
                        <summary class="article-expander__title">Adaptive therapy: a tumor therapy strategy based on Darwinian evolution theory</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37898477/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer progression is a dynamic process of continuous evolution, in which genetic diversity and heterogeneity are generated by clonal and subclonal amplification based on random mutations. Traditional cancer treatment strategies have a great challenge, which often leads to treatment failure due to drug resistance. Integrating evolutionary dynamics into treatment regimens may be an effective way to overcome the problem of drug resistance. In particular, a potential treatment is adaptive therapy,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895205"
                        open
                      >
                        <summary class="article-expander__title">Decoding Cancer Evolution: Integrating Genetic and Non-Genetic Insights</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The development of cancer begins with cells transitioning from their multicellular nature to a state akin to unicellular organisms. This shift leads to a breakdown in the crucial regulators inherent to multicellularity, resulting in the emergence of diverse cancer cell subpopulations that have enhanced adaptability. The presence of different cell subpopulations within a tumour, known as intratumoural heterogeneity (ITH), poses challenges for cancer treatment. In this review, we delve into the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895004"
                        open
                      >
                        <summary class="article-expander__title">Hematologic Neoplasms Associated with Down Syndrome: Cellular and Molecular Heterogeneity of the Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The molecular basis of Down syndrome (DS) predisposition to leukemia is not fully understood but involves various factors such as chromosomal abnormalities, oncogenic mutations, epigenetic alterations, and changes in selection dynamics. Myeloid leukemia associated with DS (ML-DS) is preceded by a preleukemic phase called transient abnormal myelopoiesis driven by GATA1 gene mutations and progresses to ML-DS via additional mutations in cohesin genes, CTCF, RAS, or JAK/STAT pathway genes....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37894756"
                        open
                      >
                        <summary class="article-expander__title">Overview of Tumor Heterogeneity in High-Grade Serous Ovarian Cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37894756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancers encompass a group of neoplasms originating from germinal tissues and exhibiting distinct clinical, pathological, and molecular features. Among these, epithelial ovarian cancers (EOCs) are the most prevalent, comprising five distinct tumor histotypes. Notably, high-grade serous ovarian cancers (HGSOCs) represent the majority, accounting for over 70% of EOC cases. Due to their silent and asymptomatic behavior, HGSOCs are generally diagnosed in advanced stages with an evolved and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895205"
                        open
                      >
                        <summary class="article-expander__title">Decoding Cancer Evolution: Integrating Genetic and Non-Genetic Insights</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The development of cancer begins with cells transitioning from their multicellular nature to a state akin to unicellular organisms. This shift leads to a breakdown in the crucial regulators inherent to multicellularity, resulting in the emergence of diverse cancer cell subpopulations that have enhanced adaptability. The presence of different cell subpopulations within a tumour, known as intratumoural heterogeneity (ITH), poses challenges for cancer treatment. In this review, we delve into the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895004"
                        open
                      >
                        <summary class="article-expander__title">Hematologic Neoplasms Associated with Down Syndrome: Cellular and Molecular Heterogeneity of the Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The molecular basis of Down syndrome (DS) predisposition to leukemia is not fully understood but involves various factors such as chromosomal abnormalities, oncogenic mutations, epigenetic alterations, and changes in selection dynamics. Myeloid leukemia associated with DS (ML-DS) is preceded by a preleukemic phase called transient abnormal myelopoiesis driven by GATA1 gene mutations and progresses to ML-DS via additional mutations in cohesin genes, CTCF, RAS, or JAK/STAT pathway genes....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37894756"
                        open
                      >
                        <summary class="article-expander__title">Overview of Tumor Heterogeneity in High-Grade Serous Ovarian Cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37894756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancers encompass a group of neoplasms originating from germinal tissues and exhibiting distinct clinical, pathological, and molecular features. Among these, epithelial ovarian cancers (EOCs) are the most prevalent, comprising five distinct tumor histotypes. Notably, high-grade serous ovarian cancers (HGSOCs) represent the majority, accounting for over 70% of EOC cases. Due to their silent and asymptomatic behavior, HGSOCs are generally diagnosed in advanced stages with an evolved and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-10-26 datetime="2023-10-26T10:00:00.000Z">2023-10-26</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-10-26 datetime="2023-10-26T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20231105195156">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37884893"
                        open
                      >
                        <summary class="article-expander__title">Alterations of the expression of TET2 and DNA 5-hmC predict poor prognosis in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37884893/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: IPSS-R higher-risk MDS categories and AML-MR showed a reduction in TET2 expression, which was not apparent in lower-risk MDS. DNA 5-hmC levels followed a similar pattern. Overall, a decreased TET2 expression and a low DNA 5-hmC level are predictors of advanced disease and adverse outcome in MDS in the population studied, i.e., MDS patients from India.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37884893"
                        open
                      >
                        <summary class="article-expander__title">Alterations of the expression of TET2 and DNA 5-hmC predict poor prognosis in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37884893/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: IPSS-R higher-risk MDS categories and AML-MR showed a reduction in TET2 expression, which was not apparent in lower-risk MDS. DNA 5-hmC levels followed a similar pattern. Overall, a decreased TET2 expression and a low DNA 5-hmC level are predictors of advanced disease and adverse outcome in MDS in the population studied, i.e., MDS patients from India.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37884893"
                        open
                      >
                        <summary class="article-expander__title">Alterations of the expression of TET2 and DNA 5-hmC predict poor prognosis in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37884893/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231103204956&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: IPSS-R higher-risk MDS categories and AML-MR showed a reduction in TET2 expression, which was not apparent in lower-risk MDS. DNA 5-hmC levels followed a similar pattern. Overall, a decreased TET2 expression and a low DNA 5-hmC level are predictors of advanced disease and adverse outcome in MDS in the population studied, i.e., MDS patients from India.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37884893"
                        open
                      >
                        <summary class="article-expander__title">Alterations of the expression of TET2 and DNA 5-hmC predict poor prognosis in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37884893/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: IPSS-R higher-risk MDS categories and AML-MR showed a reduction in TET2 expression, which was not apparent in lower-risk MDS. DNA 5-hmC levels followed a similar pattern. Overall, a decreased TET2 expression and a low DNA 5-hmC level are predictors of advanced disease and adverse outcome in MDS in the population studied, i.e., MDS patients from India.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37884893"
                        open
                      >
                        <summary class="article-expander__title">Alterations of the expression of TET2 and DNA 5-hmC predict poor prognosis in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37884893/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: IPSS-R higher-risk MDS categories and AML-MR showed a reduction in TET2 expression, which was not apparent in lower-risk MDS. DNA 5-hmC levels followed a similar pattern. Overall, a decreased TET2 expression and a low DNA 5-hmC level are predictors of advanced disease and adverse outcome in MDS in the population studied, i.e., MDS patients from India.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37884893"
                        open
                      >
                        <summary class="article-expander__title">Alterations of the expression of TET2 and DNA 5-hmC predict poor prognosis in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37884893/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: IPSS-R higher-risk MDS categories and AML-MR showed a reduction in TET2 expression, which was not apparent in lower-risk MDS. DNA 5-hmC levels followed a similar pattern. Overall, a decreased TET2 expression and a low DNA 5-hmC level are predictors of advanced disease and adverse outcome in MDS in the population studied, i.e., MDS patients from India.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37884893"
                        open
                      >
                        <summary class="article-expander__title">Alterations of the expression of TET2 and DNA 5-hmC predict poor prognosis in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37884893/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: IPSS-R higher-risk MDS categories and AML-MR showed a reduction in TET2 expression, which was not apparent in lower-risk MDS. DNA 5-hmC levels followed a similar pattern. Overall, a decreased TET2 expression and a low DNA 5-hmC level are predictors of advanced disease and adverse outcome in MDS in the population studied, i.e., MDS patients from India.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37884893"
                        open
                      >
                        <summary class="article-expander__title">Alterations of the expression of TET2 and DNA 5-hmC predict poor prognosis in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37884893/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: IPSS-R higher-risk MDS categories and AML-MR showed a reduction in TET2 expression, which was not apparent in lower-risk MDS. DNA 5-hmC levels followed a similar pattern. Overall, a decreased TET2 expression and a low DNA 5-hmC level are predictors of advanced disease and adverse outcome in MDS in the population studied, i.e., MDS patients from India.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37884893"
                        open
                      >
                        <summary class="article-expander__title">Alterations of the expression of TET2 and DNA 5-hmC predict poor prognosis in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37884893/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: IPSS-R higher-risk MDS categories and AML-MR showed a reduction in TET2 expression, which was not apparent in lower-risk MDS. DNA 5-hmC levels followed a similar pattern. Overall, a decreased TET2 expression and a low DNA 5-hmC level are predictors of advanced disease and adverse outcome in MDS in the population studied, i.e., MDS patients from India.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37884893"
                        open
                      >
                        <summary class="article-expander__title">Alterations of the expression of TET2 and DNA 5-hmC predict poor prognosis in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37884893/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231027204803&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: IPSS-R higher-risk MDS categories and AML-MR showed a reduction in TET2 expression, which was not apparent in lower-risk MDS. DNA 5-hmC levels followed a similar pattern. Overall, a decreased TET2 expression and a low DNA 5-hmC level are predictors of advanced disease and adverse outcome in MDS in the population studied, i.e., MDS patients from India.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-10-25 datetime="2023-10-25T10:00:00.000Z">2023-10-25</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-10-25 datetime="2023-10-25T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20231105195156">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37879493"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and inflammation: A review of mechanisms and clinical implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37879493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is defined by the presence of somatic mutations in hematopoietic stem and progenitor cells (HSPC). CH is associated primarily with advancing age and confers an elevated risk of progression to overt hematologic malignancy and cardiovascular disease. Increasingly, CH is associated with a wide range of diseases driven by, and sequelae of, inflammation. Accordingly, there is great interest in better understanding the pathophysiologic and clinical relationship between CH,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37879493"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and inflammation: A review of mechanisms and clinical implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37879493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is defined by the presence of somatic mutations in hematopoietic stem and progenitor cells (HSPC). CH is associated primarily with advancing age and confers an elevated risk of progression to overt hematologic malignancy and cardiovascular disease. Increasingly, CH is associated with a wide range of diseases driven by, and sequelae of, inflammation. Accordingly, there is great interest in better understanding the pathophysiologic and clinical relationship between CH,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37879493"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and inflammation: A review of mechanisms and clinical implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37879493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231103204956&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is defined by the presence of somatic mutations in hematopoietic stem and progenitor cells (HSPC). CH is associated primarily with advancing age and confers an elevated risk of progression to overt hematologic malignancy and cardiovascular disease. Increasingly, CH is associated with a wide range of diseases driven by, and sequelae of, inflammation. Accordingly, there is great interest in better understanding the pathophysiologic and clinical relationship between CH,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37879493"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Inflammation: A Review of Mechanisms and Clinical Implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37879493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is defined by the presence of somatic mutations in hematopoietic stem and progenitor cells (HSPC). CH is associated primarily with advancing age and confers an elevated risk of progression to overt hematologic malignancy and cardiovascular disease. Increasingly, CH is associated with a wide range of diseases driven by, and sequelae of, inflammation. Accordingly, there is great interest in better understanding the pathophysiologic and clinical relationship between CH,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37879493"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Inflammation: A Review of Mechanisms and Clinical Implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37879493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is defined by the presence of somatic mutations in hematopoietic stem and progenitor cells (HSPC). CH is associated primarily with advancing age and confers an elevated risk of progression to overt hematologic malignancy and cardiovascular disease. Increasingly, CH is associated with a wide range of diseases driven by, and sequelae of, inflammation. Accordingly, there is great interest in better understanding the pathophysiologic and clinical relationship between CH,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37879493"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Inflammation: A Review of Mechanisms and Clinical Implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37879493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is defined by the presence of somatic mutations in hematopoietic stem and progenitor cells (HSPC). CH is associated primarily with advancing age and confers an elevated risk of progression to overt hematologic malignancy and cardiovascular disease. Increasingly, CH is associated with a wide range of diseases driven by, and sequelae of, inflammation. Accordingly, there is great interest in better understanding the pathophysiologic and clinical relationship between CH,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37879493"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Inflammation: A Review of Mechanisms and Clinical Implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37879493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is defined by the presence of somatic mutations in hematopoietic stem and progenitor cells (HSPC). CH is associated primarily with advancing age and confers an elevated risk of progression to overt hematologic malignancy and cardiovascular disease. Increasingly, CH is associated with a wide range of diseases driven by, and sequelae of, inflammation. Accordingly, there is great interest in better understanding the pathophysiologic and clinical relationship between CH,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37879493"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Inflammation: A Review of Mechanisms and Clinical Implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37879493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is defined by the presence of somatic mutations in hematopoietic stem and progenitor cells (HSPC). CH is associated primarily with advancing age and confers an elevated risk of progression to overt hematologic malignancy and cardiovascular disease. Increasingly, CH is associated with a wide range of diseases driven by, and sequelae of, inflammation. Accordingly, there is great interest in better understanding the pathophysiologic and clinical relationship between CH,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37879493"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Inflammation: A Review of Mechanisms and Clinical Implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37879493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is defined by the presence of somatic mutations in hematopoietic stem and progenitor cells (HSPC). CH is associated primarily with advancing age and confers an elevated risk of progression to overt hematologic malignancy and cardiovascular disease. Increasingly, CH is associated with a wide range of diseases driven by, and sequelae of, inflammation. Accordingly, there is great interest in better understanding the pathophysiologic and clinical relationship between CH,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37879493"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Inflammation: A Review of Mechanisms and Clinical Implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37879493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231027204803&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is defined by the presence of somatic mutations in hematopoietic stem and progenitor cells (HSPC). CH is associated primarily with advancing age and confers an elevated risk of progression to overt hematologic malignancy and cardiovascular disease. Increasingly, CH is associated with a wide range of diseases driven by, and sequelae of, inflammation. Accordingly, there is great interest in better understanding the pathophysiologic and clinical relationship between CH,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37879493"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Inflammation: A Review of Mechanisms and Clinical Implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37879493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is defined by the presence of somatic mutations in hematopoietic stem and progenitor cells (HSPC). CH is associated primarily with advancing age and confers an elevated risk of progression to overt hematologic malignancy and cardiovascular disease. Increasingly, CH is associated with a wide range of diseases driven by, and sequelae of, inflammation. Accordingly, there is great interest in better understanding the pathophysiologic and clinical relationship between CH,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;ff&#x3D;20231105195156&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37877809"
                        open
                      >
                        <summary class="article-expander__title">Temporal and structural patterns of hepatitis B virus integrations in hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37877809/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic infection of hepatitis B virus (HBV) is the major cause of hepatocellular carcinoma (HCC). Notably, 90% of HBV-positive HCC cases exhibit detectable HBV integrations, hinting at the potential early entanglement of these viral integrations in tumorigenesis and their subsequent oncogenic implications. Nevertheless, the precise chronology of integration events during HCC tumorigenesis, alongside their sequential structural patterns, has remained elusive thus far. In this study, we applied...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37877809"
                        open
                      >
                        <summary class="article-expander__title">Temporal and structural patterns of hepatitis B virus integrations in hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37877809/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic infection of hepatitis B virus (HBV) is the major cause of hepatocellular carcinoma (HCC). Notably, 90% of HBV-positive HCC cases exhibit detectable HBV integrations, hinting at the potential early entanglement of these viral integrations in tumorigenesis and their subsequent oncogenic implications. Nevertheless, the precise chronology of integration events during HCC tumorigenesis, alongside their sequential structural patterns, has remained elusive thus far. In this study, we applied...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37877809"
                        open
                      >
                        <summary class="article-expander__title">Temporal and structural patterns of hepatitis B virus integrations in hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37877809/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231103204955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic infection of hepatitis B virus (HBV) is the major cause of hepatocellular carcinoma (HCC). Notably, 90% of HBV-positive HCC cases exhibit detectable HBV integrations, hinting at the potential early entanglement of these viral integrations in tumorigenesis and their subsequent oncogenic implications. Nevertheless, the precise chronology of integration events during HCC tumorigenesis, alongside their sequential structural patterns, has remained elusive thus far. In this study, we applied...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37877809"
                        open
                      >
                        <summary class="article-expander__title">Temporal and structural patterns of hepatitis B virus integrations in hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37877809/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic infection of hepatitis B virus (HBV) is the major cause of hepatocellular carcinoma (HCC). Notably, 90% of HBV-positive HCC cases exhibit detectable HBV integrations, hinting at the potential early entanglement of these viral integrations in tumorigenesis and their subsequent oncogenic implications. Nevertheless, the precise chronology of integration events during HCC tumorigenesis, alongside their sequential structural patterns, has remained elusive thus far. In this study, we applied...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37877809"
                        open
                      >
                        <summary class="article-expander__title">Temporal and structural patterns of hepatitis B virus integrations in hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37877809/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic infection of hepatitis B virus (HBV) is the major cause of hepatocellular carcinoma (HCC). Notably, 90% of HBV-positive HCC cases exhibit detectable HBV integrations, hinting at the potential early entanglement of these viral integrations in tumorigenesis and their subsequent oncogenic implications. Nevertheless, the precise chronology of integration events during HCC tumorigenesis, alongside their sequential structural patterns, has remained elusive thus far. In this study, we applied...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37877809"
                        open
                      >
                        <summary class="article-expander__title">Temporal and structural patterns of hepatitis B virus integrations in hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37877809/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic infection of hepatitis B virus (HBV) is the major cause of hepatocellular carcinoma (HCC). Notably, 90% of HBV-positive HCC cases exhibit detectable HBV integrations, hinting at the potential early entanglement of these viral integrations in tumorigenesis and their subsequent oncogenic implications. Nevertheless, the precise chronology of integration events during HCC tumorigenesis, alongside their sequential structural patterns, has remained elusive thus far. In this study, we applied...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37877809"
                        open
                      >
                        <summary class="article-expander__title">Temporal and structural patterns of hepatitis B virus integrations in hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37877809/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic infection of hepatitis B virus (HBV) is the major cause of hepatocellular carcinoma (HCC). Notably, 90% of HBV-positive HCC cases exhibit detectable HBV integrations, hinting at the potential early entanglement of these viral integrations in tumorigenesis and their subsequent oncogenic implications. Nevertheless, the precise chronology of integration events during HCC tumorigenesis, alongside their sequential structural patterns, has remained elusive thus far. In this study, we applied...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37877809"
                        open
                      >
                        <summary class="article-expander__title">Temporal and structural patterns of hepatitis B virus integrations in hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37877809/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic infection of hepatitis B virus (HBV) is the major cause of hepatocellular carcinoma (HCC). Notably, 90% of HBV-positive HCC cases exhibit detectable HBV integrations, hinting at the potential early entanglement of these viral integrations in tumorigenesis and their subsequent oncogenic implications. Nevertheless, the precise chronology of integration events during HCC tumorigenesis, alongside their sequential structural patterns, has remained elusive thus far. In this study, we applied...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37877809"
                        open
                      >
                        <summary class="article-expander__title">Temporal and structural patterns of hepatitis B virus integrations in hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37877809/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic infection of hepatitis B virus (HBV) is the major cause of hepatocellular carcinoma (HCC). Notably, 90% of HBV-positive HCC cases exhibit detectable HBV integrations, hinting at the potential early entanglement of these viral integrations in tumorigenesis and their subsequent oncogenic implications. Nevertheless, the precise chronology of integration events during HCC tumorigenesis, alongside their sequential structural patterns, has remained elusive thus far. In this study, we applied...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37877809"
                        open
                      >
                        <summary class="article-expander__title">Temporal and structural patterns of hepatitis B virus integrations in hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37877809/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231027204802&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic infection of hepatitis B virus (HBV) is the major cause of hepatocellular carcinoma (HCC). Notably, 90% of HBV-positive HCC cases exhibit detectable HBV integrations, hinting at the potential early entanglement of these viral integrations in tumorigenesis and their subsequent oncogenic implications. Nevertheless, the precise chronology of integration events during HCC tumorigenesis, alongside their sequential structural patterns, has remained elusive thus far. In this study, we applied...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37877809"
                        open
                      >
                        <summary class="article-expander__title">Temporal and structural patterns of hepatitis B virus integrations in hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37877809/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic infection of hepatitis B virus (HBV) is the major cause of hepatocellular carcinoma (HCC). Notably, 90% of HBV-positive HCC cases exhibit detectable HBV integrations, hinting at the potential early entanglement of these viral integrations in tumorigenesis and their subsequent oncogenic implications. Nevertheless, the precise chronology of integration events during HCC tumorigenesis, alongside their sequential structural patterns, has remained elusive thus far. In this study, we applied...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37877809"
                        open
                      >
                        <summary class="article-expander__title">Temporal and structural patterns of hepatitis B virus integrations in hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37877809/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic infection of hepatitis B virus (HBV) is the major cause of hepatocellular carcinoma (HCC). Notably, 90% of HBV-positive HCC cases exhibit detectable HBV integrations, hinting at the potential early entanglement of these viral integrations in tumorigenesis and their subsequent oncogenic implications. Nevertheless, the precise chronology of integration events during HCC tumorigenesis, alongside their sequential structural patterns, has remained elusive thus far. In this study, we applied...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-10-24 datetime="2023-10-24T10:00:00.000Z">2023-10-24</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-10-24 datetime="2023-10-24T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;ff&#x3D;20231105195156&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37874744"
                        open
                      >
                        <summary class="article-expander__title">SETD2 mutations do not contribute to clonal fitness in response to chemotherapy in childhood B cell acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37874744/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes encoding epigenetic regulators are commonly observed at relapse in B cell acute lymphoblastic leukemia (B-ALL). Loss-of-function mutations in SETD2, an H3K36 methyltransferase, have been observed in B-ALL and other cancers. Previous studies on mutated SETD2 in solid tumors and acute myelogenous leukemia support a role in promoting resistance to DNA damaging agents. We did not observe chemoresistance, an impaired DNA damage response, nor increased mutation frequency in response...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873488"
                        open
                      >
                        <summary class="article-expander__title">Growth Dynamics of Ductal Carcinoma in Situ Recapitulate Normal Breast Development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873488/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ductal carcinoma in situ (DCIS) and invasive breast cancer share many morphologic, proteomic, and genomic alterations. Yet in contrast to invasive cancer, many DCIS tumors do not progress and may remain indolent over decades. To better understand the heterogenous nature of this disease, we reconstructed the growth dynamics of 18 DCIS tumors based on the geo-spatial distribution of their somatic mutations. The somatic mutation topographies revealed that DCIS is multiclonal and consists of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873375"
                        open
                      >
                        <summary class="article-expander__title">Characterization of Cancer Evolution Landscape Based on Accurate Detection of Somatic Mutations in Single Tumor Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873375/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in single tumor cells is greatly desired as it allows us to quantify the single-cell mutation burden and construct the mutation-based phylogenetic tree. Here we developed scNanoSeq chemistry and profiled 842 single cells from 21 human breast cancer samples. The majority of the mutation-based phylogenetic trees comprise a characteristic stem evolution followed by the clonal sweep. We observed the subtype-dependent lengths in the stem evolution. To explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37872885"
                        open
                      >
                        <summary class="article-expander__title">A guide to epigenetics in leukaemia stem cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37872885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Leukaemia stem cells (LSCs) are the critical seed for the growth of haematological malignancies, driving the clonal expansion that enables disease initiation, relapse and often resistance. Specifically, they display inherent phenotypic and epigenetic plasticity resulting in complex heterogenic diseases. In this review, we discuss the key principles of deregulation of epigenetic processes that shape this disease evolution. We consider measures to define and quantify clonal heterogeneity,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37874744"
                        open
                      >
                        <summary class="article-expander__title">SETD2 mutations do not contribute to clonal fitness in response to chemotherapy in childhood B cell acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37874744/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes encoding epigenetic regulators are commonly observed at relapse in B cell acute lymphoblastic leukemia (B-ALL). Loss-of-function mutations in SETD2, an H3K36 methyltransferase, have been observed in B-ALL and other cancers. Previous studies on mutated SETD2 in solid tumors and acute myelogenous leukemia support a role in promoting resistance to DNA damaging agents. We did not observe chemoresistance, an impaired DNA damage response, nor increased mutation frequency in response...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873488"
                        open
                      >
                        <summary class="article-expander__title">Growth Dynamics of Ductal Carcinoma in Situ Recapitulate Normal Breast Development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873488/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ductal carcinoma in situ (DCIS) and invasive breast cancer share many morphologic, proteomic, and genomic alterations. Yet in contrast to invasive cancer, many DCIS tumors do not progress and may remain indolent over decades. To better understand the heterogenous nature of this disease, we reconstructed the growth dynamics of 18 DCIS tumors based on the geo-spatial distribution of their somatic mutations. The somatic mutation topographies revealed that DCIS is multiclonal and consists of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873375"
                        open
                      >
                        <summary class="article-expander__title">Characterization of Cancer Evolution Landscape Based on Accurate Detection of Somatic Mutations in Single Tumor Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873375/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in single tumor cells is greatly desired as it allows us to quantify the single-cell mutation burden and construct the mutation-based phylogenetic tree. Here we developed scNanoSeq chemistry and profiled 842 single cells from 21 human breast cancer samples. The majority of the mutation-based phylogenetic trees comprise a characteristic stem evolution followed by the clonal sweep. We observed the subtype-dependent lengths in the stem evolution. To explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37872885"
                        open
                      >
                        <summary class="article-expander__title">A guide to epigenetics in leukaemia stem cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37872885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Leukaemia stem cells (LSCs) are the critical seed for the growth of haematological malignancies, driving the clonal expansion that enables disease initiation, relapse and often resistance. Specifically, they display inherent phenotypic and epigenetic plasticity resulting in complex heterogenic diseases. In this review, we discuss the key principles of deregulation of epigenetic processes that shape this disease evolution. We consider measures to define and quantify clonal heterogeneity,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37874744"
                        open
                      >
                        <summary class="article-expander__title">SETD2 mutations do not contribute to clonal fitness in response to chemotherapy in childhood B cell acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37874744/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231103204955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes encoding epigenetic regulators are commonly observed at relapse in B cell acute lymphoblastic leukemia (B-ALL). Loss-of-function mutations in SETD2, an H3K36 methyltransferase, have been observed in B-ALL and other cancers. Previous studies on mutated SETD2 in solid tumors and acute myelogenous leukemia support a role in promoting resistance to DNA damaging agents. We did not observe chemoresistance, an impaired DNA damage response, nor increased mutation frequency in response...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873488"
                        open
                      >
                        <summary class="article-expander__title">Growth Dynamics of Ductal Carcinoma in Situ Recapitulate Normal Breast Development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873488/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231103204955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ductal carcinoma in situ (DCIS) and invasive breast cancer share many morphologic, proteomic, and genomic alterations. Yet in contrast to invasive cancer, many DCIS tumors do not progress and may remain indolent over decades. To better understand the heterogenous nature of this disease, we reconstructed the growth dynamics of 18 DCIS tumors based on the geo-spatial distribution of their somatic mutations. The somatic mutation topographies revealed that DCIS is multiclonal and consists of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873375"
                        open
                      >
                        <summary class="article-expander__title">Characterization of Cancer Evolution Landscape Based on Accurate Detection of Somatic Mutations in Single Tumor Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873375/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231103204955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in single tumor cells is greatly desired as it allows us to quantify the single-cell mutation burden and construct the mutation-based phylogenetic tree. Here we developed scNanoSeq chemistry and profiled 842 single cells from 21 human breast cancer samples. The majority of the mutation-based phylogenetic trees comprise a characteristic stem evolution followed by the clonal sweep. We observed the subtype-dependent lengths in the stem evolution. To explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37872885"
                        open
                      >
                        <summary class="article-expander__title">A guide to epigenetics in leukaemia stem cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37872885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231103204955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Leukaemia stem cells (LSCs) are the critical seed for the growth of haematological malignancies, driving the clonal expansion that enables disease initiation, relapse and often resistance. Specifically, they display inherent phenotypic and epigenetic plasticity resulting in complex heterogenic diseases. In this review, we discuss the key principles of deregulation of epigenetic processes that shape this disease evolution. We consider measures to define and quantify clonal heterogeneity,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37874744"
                        open
                      >
                        <summary class="article-expander__title">SETD2 mutations do not contribute to clonal fitness in response to chemotherapy in childhood B cell acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37874744/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes encoding epigenetic regulators are commonly observed at relapse in B cell acute lymphoblastic leukemia (B-ALL). Loss-of-function mutations in SETD2, an H3K36 methyltransferase, have been observed in B-ALL and other cancers. Previous studies on mutated SETD2 in solid tumors and acute myelogenous leukemia support a role in promoting resistance to DNA damaging agents. We did not observe chemoresistance, an impaired DNA damage response, nor increased mutation frequency in response...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873488"
                        open
                      >
                        <summary class="article-expander__title">Growth Dynamics of Ductal Carcinoma in Situ Recapitulate Normal Breast Development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873488/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ductal carcinoma in situ (DCIS) and invasive breast cancer share many morphologic, proteomic, and genomic alterations. Yet in contrast to invasive cancer, many DCIS tumors do not progress and may remain indolent over decades. To better understand the heterogenous nature of this disease, we reconstructed the growth dynamics of 18 DCIS tumors based on the geo-spatial distribution of their somatic mutations. The somatic mutation topographies revealed that DCIS is multiclonal and consists of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873375"
                        open
                      >
                        <summary class="article-expander__title">Characterization of Cancer Evolution Landscape Based on Accurate Detection of Somatic Mutations in Single Tumor Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873375/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in single tumor cells is greatly desired as it allows us to quantify the single-cell mutation burden and construct the mutation-based phylogenetic tree. Here we developed scNanoSeq chemistry and profiled 842 single cells from 21 human breast cancer samples. The majority of the mutation-based phylogenetic trees comprise a characteristic stem evolution followed by the clonal sweep. We observed the subtype-dependent lengths in the stem evolution. To explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37872885"
                        open
                      >
                        <summary class="article-expander__title">A guide to epigenetics in leukaemia stem cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37872885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Leukaemia stem cells (LSCs) are the critical seed for the growth of haematological malignancies, driving the clonal expansion that enables disease initiation, relapse and often resistance. Specifically, they display inherent phenotypic and epigenetic plasticity resulting in complex heterogenic diseases. In this review, we discuss the key principles of deregulation of epigenetic processes that shape this disease evolution. We consider measures to define and quantify clonal heterogeneity,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37874744"
                        open
                      >
                        <summary class="article-expander__title">SETD2 mutations do not contribute to clonal fitness in response to chemotherapy in childhood B cell acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37874744/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes encoding epigenetic regulators are commonly observed at relapse in B cell acute lymphoblastic leukemia (B-ALL). Loss-of-function mutations in SETD2, an H3K36 methyltransferase, have been observed in B-ALL and other cancers. Previous studies on mutated SETD2 in solid tumors and acute myelogenous leukemia support a role in promoting resistance to DNA damaging agents. We did not observe chemoresistance, an impaired DNA damage response, nor increased mutation frequency in response...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873488"
                        open
                      >
                        <summary class="article-expander__title">Growth Dynamics of Ductal Carcinoma in Situ Recapitulate Normal Breast Development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873488/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ductal carcinoma in situ (DCIS) and invasive breast cancer share many morphologic, proteomic, and genomic alterations. Yet in contrast to invasive cancer, many DCIS tumors do not progress and may remain indolent over decades. To better understand the heterogenous nature of this disease, we reconstructed the growth dynamics of 18 DCIS tumors based on the geo-spatial distribution of their somatic mutations. The somatic mutation topographies revealed that DCIS is multiclonal and consists of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873375"
                        open
                      >
                        <summary class="article-expander__title">Characterization of Cancer Evolution Landscape Based on Accurate Detection of Somatic Mutations in Single Tumor Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873375/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in single tumor cells is greatly desired as it allows us to quantify the single-cell mutation burden and construct the mutation-based phylogenetic tree. Here we developed scNanoSeq chemistry and profiled 842 single cells from 21 human breast cancer samples. The majority of the mutation-based phylogenetic trees comprise a characteristic stem evolution followed by the clonal sweep. We observed the subtype-dependent lengths in the stem evolution. To explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37872885"
                        open
                      >
                        <summary class="article-expander__title">A guide to epigenetics in leukaemia stem cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37872885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Leukaemia stem cells (LSCs) are the critical seed for the growth of haematological malignancies, driving the clonal expansion that enables disease initiation, relapse and often resistance. Specifically, they display inherent phenotypic and epigenetic plasticity resulting in complex heterogenic diseases. In this review, we discuss the key principles of deregulation of epigenetic processes that shape this disease evolution. We consider measures to define and quantify clonal heterogeneity,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37874744"
                        open
                      >
                        <summary class="article-expander__title">SETD2 mutations do not contribute to clonal fitness in response to chemotherapy in childhood B cell acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37874744/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes encoding epigenetic regulators are commonly observed at relapse in B cell acute lymphoblastic leukemia (B-ALL). Loss-of-function mutations in SETD2, an H3K36 methyltransferase, have been observed in B-ALL and other cancers. Previous studies on mutated SETD2 in solid tumors and acute myelogenous leukemia support a role in promoting resistance to DNA damaging agents. We did not observe chemoresistance, an impaired DNA damage response, nor increased mutation frequency in response...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873488"
                        open
                      >
                        <summary class="article-expander__title">Growth Dynamics of Ductal Carcinoma in Situ Recapitulate Normal Breast Development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873488/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ductal carcinoma in situ (DCIS) and invasive breast cancer share many morphologic, proteomic, and genomic alterations. Yet in contrast to invasive cancer, many DCIS tumors do not progress and may remain indolent over decades. To better understand the heterogenous nature of this disease, we reconstructed the growth dynamics of 18 DCIS tumors based on the geo-spatial distribution of their somatic mutations. The somatic mutation topographies revealed that DCIS is multiclonal and consists of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873375"
                        open
                      >
                        <summary class="article-expander__title">Characterization of Cancer Evolution Landscape Based on Accurate Detection of Somatic Mutations in Single Tumor Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873375/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in single tumor cells is greatly desired as it allows us to quantify the single-cell mutation burden and construct the mutation-based phylogenetic tree. Here we developed scNanoSeq chemistry and profiled 842 single cells from 21 human breast cancer samples. The majority of the mutation-based phylogenetic trees comprise a characteristic stem evolution followed by the clonal sweep. We observed the subtype-dependent lengths in the stem evolution. To explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37872885"
                        open
                      >
                        <summary class="article-expander__title">A guide to epigenetics in leukaemia stem cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37872885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Leukaemia stem cells (LSCs) are the critical seed for the growth of haematological malignancies, driving the clonal expansion that enables disease initiation, relapse and often resistance. Specifically, they display inherent phenotypic and epigenetic plasticity resulting in complex heterogenic diseases. In this review, we discuss the key principles of deregulation of epigenetic processes that shape this disease evolution. We consider measures to define and quantify clonal heterogeneity,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37874744"
                        open
                      >
                        <summary class="article-expander__title">SETD2 mutations do not contribute to clonal fitness in response to chemotherapy in childhood B cell acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37874744/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes encoding epigenetic regulators are commonly observed at relapse in B cell acute lymphoblastic leukemia (B-ALL). Loss-of-function mutations in SETD2, an H3K36 methyltransferase, have been observed in B-ALL and other cancers. Previous studies on mutated SETD2 in solid tumors and acute myelogenous leukemia support a role in promoting resistance to DNA damaging agents. We did not observe chemoresistance, an impaired DNA damage response, nor increased mutation frequency in response...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873488"
                        open
                      >
                        <summary class="article-expander__title">Growth Dynamics of Ductal Carcinoma in Situ Recapitulate Normal Breast Development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873488/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ductal carcinoma in situ (DCIS) and invasive breast cancer share many morphologic, proteomic, and genomic alterations. Yet in contrast to invasive cancer, many DCIS tumors do not progress and may remain indolent over decades. To better understand the heterogenous nature of this disease, we reconstructed the growth dynamics of 18 DCIS tumors based on the geo-spatial distribution of their somatic mutations. The somatic mutation topographies revealed that DCIS is multiclonal and consists of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873375"
                        open
                      >
                        <summary class="article-expander__title">Characterization of Cancer Evolution Landscape Based on Accurate Detection of Somatic Mutations in Single Tumor Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873375/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in single tumor cells is greatly desired as it allows us to quantify the single-cell mutation burden and construct the mutation-based phylogenetic tree. Here we developed scNanoSeq chemistry and profiled 842 single cells from 21 human breast cancer samples. The majority of the mutation-based phylogenetic trees comprise a characteristic stem evolution followed by the clonal sweep. We observed the subtype-dependent lengths in the stem evolution. To explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37872885"
                        open
                      >
                        <summary class="article-expander__title">A guide to epigenetics in leukaemia stem cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37872885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Leukaemia stem cells (LSCs) are the critical seed for the growth of haematological malignancies, driving the clonal expansion that enables disease initiation, relapse and often resistance. Specifically, they display inherent phenotypic and epigenetic plasticity resulting in complex heterogenic diseases. In this review, we discuss the key principles of deregulation of epigenetic processes that shape this disease evolution. We consider measures to define and quantify clonal heterogeneity,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37874744"
                        open
                      >
                        <summary class="article-expander__title">SETD2 mutations do not contribute to clonal fitness in response to chemotherapy in childhood B cell acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37874744/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes encoding epigenetic regulators are commonly observed at relapse in B cell acute lymphoblastic leukemia (B-ALL). Loss-of-function mutations in SETD2, an H3K36 methyltransferase, have been observed in B-ALL and other cancers. Previous studies on mutated SETD2 in solid tumors and acute myelogenous leukemia support a role in promoting resistance to DNA damaging agents. We did not observe chemoresistance, an impaired DNA damage response, nor increased mutation frequency in response...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873488"
                        open
                      >
                        <summary class="article-expander__title">Growth Dynamics of Ductal Carcinoma in Situ Recapitulate Normal Breast Development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873488/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ductal carcinoma in situ (DCIS) and invasive breast cancer share many morphologic, proteomic, and genomic alterations. Yet in contrast to invasive cancer, many DCIS tumors do not progress and may remain indolent over decades. To better understand the heterogenous nature of this disease, we reconstructed the growth dynamics of 18 DCIS tumors based on the geo-spatial distribution of their somatic mutations. The somatic mutation topographies revealed that DCIS is multiclonal and consists of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873375"
                        open
                      >
                        <summary class="article-expander__title">Characterization of Cancer Evolution Landscape Based on Accurate Detection of Somatic Mutations in Single Tumor Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873375/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in single tumor cells is greatly desired as it allows us to quantify the single-cell mutation burden and construct the mutation-based phylogenetic tree. Here we developed scNanoSeq chemistry and profiled 842 single cells from 21 human breast cancer samples. The majority of the mutation-based phylogenetic trees comprise a characteristic stem evolution followed by the clonal sweep. We observed the subtype-dependent lengths in the stem evolution. To explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37872885"
                        open
                      >
                        <summary class="article-expander__title">A guide to epigenetics in leukaemia stem cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37872885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Leukaemia stem cells (LSCs) are the critical seed for the growth of haematological malignancies, driving the clonal expansion that enables disease initiation, relapse and often resistance. Specifically, they display inherent phenotypic and epigenetic plasticity resulting in complex heterogenic diseases. In this review, we discuss the key principles of deregulation of epigenetic processes that shape this disease evolution. We consider measures to define and quantify clonal heterogeneity,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37874744"
                        open
                      >
                        <summary class="article-expander__title">SETD2 mutations do not contribute to clonal fitness in response to chemotherapy in childhood B cell acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37874744/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes encoding epigenetic regulators are commonly observed at relapse in B cell acute lymphoblastic leukemia (B-ALL). Loss-of-function mutations in SETD2, an H3K36 methyltransferase, have been observed in B-ALL and other cancers. Previous studies on mutated SETD2 in solid tumors and acute myelogenous leukemia support a role in promoting resistance to DNA damaging agents. We did not observe chemoresistance, an impaired DNA damage response, nor increased mutation frequency in response...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873488"
                        open
                      >
                        <summary class="article-expander__title">Growth Dynamics of Ductal Carcinoma in Situ Recapitulate Normal Breast Development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873488/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ductal carcinoma in situ (DCIS) and invasive breast cancer share many morphologic, proteomic, and genomic alterations. Yet in contrast to invasive cancer, many DCIS tumors do not progress and may remain indolent over decades. To better understand the heterogenous nature of this disease, we reconstructed the growth dynamics of 18 DCIS tumors based on the geo-spatial distribution of their somatic mutations. The somatic mutation topographies revealed that DCIS is multiclonal and consists of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873375"
                        open
                      >
                        <summary class="article-expander__title">Characterization of Cancer Evolution Landscape Based on Accurate Detection of Somatic Mutations in Single Tumor Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873375/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in single tumor cells is greatly desired as it allows us to quantify the single-cell mutation burden and construct the mutation-based phylogenetic tree. Here we developed scNanoSeq chemistry and profiled 842 single cells from 21 human breast cancer samples. The majority of the mutation-based phylogenetic trees comprise a characteristic stem evolution followed by the clonal sweep. We observed the subtype-dependent lengths in the stem evolution. To explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37872885"
                        open
                      >
                        <summary class="article-expander__title">A guide to epigenetics in leukaemia stem cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37872885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Leukaemia stem cells (LSCs) are the critical seed for the growth of haematological malignancies, driving the clonal expansion that enables disease initiation, relapse and often resistance. Specifically, they display inherent phenotypic and epigenetic plasticity resulting in complex heterogenic diseases. In this review, we discuss the key principles of deregulation of epigenetic processes that shape this disease evolution. We consider measures to define and quantify clonal heterogeneity,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37874744"
                        open
                      >
                        <summary class="article-expander__title">SETD2 mutations do not contribute to clonal fitness in response to chemotherapy in childhood B cell acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37874744/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231027204802&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes encoding epigenetic regulators are commonly observed at relapse in B cell acute lymphoblastic leukemia (B-ALL). Loss-of-function mutations in SETD2, an H3K36 methyltransferase, have been observed in B-ALL and other cancers. Previous studies on mutated SETD2 in solid tumors and acute myelogenous leukemia support a role in promoting resistance to DNA damaging agents. We did not observe chemoresistance, an impaired DNA damage response, nor increased mutation frequency in response...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873488"
                        open
                      >
                        <summary class="article-expander__title">Growth Dynamics of Ductal Carcinoma in Situ Recapitulate Normal Breast Development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873488/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231027204802&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ductal carcinoma in situ (DCIS) and invasive breast cancer share many morphologic, proteomic, and genomic alterations. Yet in contrast to invasive cancer, many DCIS tumors do not progress and may remain indolent over decades. To better understand the heterogenous nature of this disease, we reconstructed the growth dynamics of 18 DCIS tumors based on the geo-spatial distribution of their somatic mutations. The somatic mutation topographies revealed that DCIS is multiclonal and consists of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873375"
                        open
                      >
                        <summary class="article-expander__title">Characterization of Cancer Evolution Landscape Based on Accurate Detection of Somatic Mutations in Single Tumor Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873375/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231027204802&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in single tumor cells is greatly desired as it allows us to quantify the single-cell mutation burden and construct the mutation-based phylogenetic tree. Here we developed scNanoSeq chemistry and profiled 842 single cells from 21 human breast cancer samples. The majority of the mutation-based phylogenetic trees comprise a characteristic stem evolution followed by the clonal sweep. We observed the subtype-dependent lengths in the stem evolution. To explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37872885"
                        open
                      >
                        <summary class="article-expander__title">A guide to epigenetics in leukaemia stem cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37872885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231027204802&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Leukaemia stem cells (LSCs) are the critical seed for the growth of haematological malignancies, driving the clonal expansion that enables disease initiation, relapse and often resistance. Specifically, they display inherent phenotypic and epigenetic plasticity resulting in complex heterogenic diseases. In this review, we discuss the key principles of deregulation of epigenetic processes that shape this disease evolution. We consider measures to define and quantify clonal heterogeneity,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37874744"
                        open
                      >
                        <summary class="article-expander__title">SETD2 mutations do not contribute to clonal fitness in response to chemotherapy in childhood B cell acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37874744/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes encoding epigenetic regulators are commonly observed at relapse in B cell acute lymphoblastic leukemia (B-ALL). Loss-of-function mutations in SETD2, an H3K36 methyltransferase, have been observed in B-ALL and other cancers. Previous studies on mutated SETD2 in solid tumors and acute myelogenous leukemia support a role in promoting resistance to DNA damaging agents. We did not observe chemoresistance, an impaired DNA damage response, nor increased mutation frequency in response...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873488"
                        open
                      >
                        <summary class="article-expander__title">Growth Dynamics of Ductal Carcinoma in Situ Recapitulate Normal Breast Development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873488/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ductal carcinoma in situ (DCIS) and invasive breast cancer share many morphologic, proteomic, and genomic alterations. Yet in contrast to invasive cancer, many DCIS tumors do not progress and may remain indolent over decades. To better understand the heterogenous nature of this disease, we reconstructed the growth dynamics of 18 DCIS tumors based on the geo-spatial distribution of their somatic mutations. The somatic mutation topographies revealed that DCIS is multiclonal and consists of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873375"
                        open
                      >
                        <summary class="article-expander__title">Characterization of Cancer Evolution Landscape Based on Accurate Detection of Somatic Mutations in Single Tumor Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873375/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in single tumor cells is greatly desired as it allows us to quantify the single-cell mutation burden and construct the mutation-based phylogenetic tree. Here we developed scNanoSeq chemistry and profiled 842 single cells from 21 human breast cancer samples. The majority of the mutation-based phylogenetic trees comprise a characteristic stem evolution followed by the clonal sweep. We observed the subtype-dependent lengths in the stem evolution. To explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37872885"
                        open
                      >
                        <summary class="article-expander__title">A guide to epigenetics in leukaemia stem cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37872885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Leukaemia stem cells (LSCs) are the critical seed for the growth of haematological malignancies, driving the clonal expansion that enables disease initiation, relapse and often resistance. Specifically, they display inherent phenotypic and epigenetic plasticity resulting in complex heterogenic diseases. In this review, we discuss the key principles of deregulation of epigenetic processes that shape this disease evolution. We consider measures to define and quantify clonal heterogeneity,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37874744"
                        open
                      >
                        <summary class="article-expander__title">SETD2 mutations do not contribute to clonal fitness in response to chemotherapy in childhood B cell acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37874744/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes encoding epigenetic regulators are commonly observed at relapse in B cell acute lymphoblastic leukemia (B-ALL). Loss-of-function mutations in SETD2, an H3K36 methyltransferase, have been observed in B-ALL and other cancers. Previous studies on mutated SETD2 in solid tumors and acute myelogenous leukemia support a role in promoting resistance to DNA damaging agents. We did not observe chemoresistance, an impaired DNA damage response, nor increased mutation frequency in response...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873488"
                        open
                      >
                        <summary class="article-expander__title">Growth Dynamics of Ductal Carcinoma in Situ Recapitulate Normal Breast Development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873488/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ductal carcinoma in situ (DCIS) and invasive breast cancer share many morphologic, proteomic, and genomic alterations. Yet in contrast to invasive cancer, many DCIS tumors do not progress and may remain indolent over decades. To better understand the heterogenous nature of this disease, we reconstructed the growth dynamics of 18 DCIS tumors based on the geo-spatial distribution of their somatic mutations. The somatic mutation topographies revealed that DCIS is multiclonal and consists of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873375"
                        open
                      >
                        <summary class="article-expander__title">Characterization of Cancer Evolution Landscape Based on Accurate Detection of Somatic Mutations in Single Tumor Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873375/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in single tumor cells is greatly desired as it allows us to quantify the single-cell mutation burden and construct the mutation-based phylogenetic tree. Here we developed scNanoSeq chemistry and profiled 842 single cells from 21 human breast cancer samples. The majority of the mutation-based phylogenetic trees comprise a characteristic stem evolution followed by the clonal sweep. We observed the subtype-dependent lengths in the stem evolution. To explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37872885"
                        open
                      >
                        <summary class="article-expander__title">A guide to epigenetics in leukaemia stem cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37872885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Leukaemia stem cells (LSCs) are the critical seed for the growth of haematological malignancies, driving the clonal expansion that enables disease initiation, relapse and often resistance. Specifically, they display inherent phenotypic and epigenetic plasticity resulting in complex heterogenic diseases. In this review, we discuss the key principles of deregulation of epigenetic processes that shape this disease evolution. We consider measures to define and quantify clonal heterogeneity,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37874744"
                        open
                      >
                        <summary class="article-expander__title">SETD2 mutations do not contribute to clonal fitness in response to chemotherapy in childhood B cell acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37874744/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes encoding epigenetic regulators are commonly observed at relapse in B cell acute lymphoblastic leukemia (B-ALL). Loss-of-function mutations in SETD2, an H3K36 methyltransferase, have been observed in B-ALL and other cancers. Previous studies on mutated SETD2 in solid tumors and acute myelogenous leukemia support a role in promoting resistance to DNA damaging agents. We did not observe chemoresistance, an impaired DNA damage response, nor increased mutation frequency in response...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873488"
                        open
                      >
                        <summary class="article-expander__title">Growth Dynamics of Ductal Carcinoma in Situ Recapitulate Normal Breast Development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873488/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ductal carcinoma in situ (DCIS) and invasive breast cancer share many morphologic, proteomic, and genomic alterations. Yet in contrast to invasive cancer, many DCIS tumors do not progress and may remain indolent over decades. To better understand the heterogenous nature of this disease, we reconstructed the growth dynamics of 18 DCIS tumors based on the geo-spatial distribution of their somatic mutations. The somatic mutation topographies revealed that DCIS is multiclonal and consists of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873375"
                        open
                      >
                        <summary class="article-expander__title">Characterization of Cancer Evolution Landscape Based on Accurate Detection of Somatic Mutations in Single Tumor Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873375/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in single tumor cells is greatly desired as it allows us to quantify the single-cell mutation burden and construct the mutation-based phylogenetic tree. Here we developed scNanoSeq chemistry and profiled 842 single cells from 21 human breast cancer samples. The majority of the mutation-based phylogenetic trees comprise a characteristic stem evolution followed by the clonal sweep. We observed the subtype-dependent lengths in the stem evolution. To explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37872885"
                        open
                      >
                        <summary class="article-expander__title">A guide to epigenetics in leukaemia stem cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37872885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Leukaemia stem cells (LSCs) are the critical seed for the growth of haematological malignancies, driving the clonal expansion that enables disease initiation, relapse and often resistance. Specifically, they display inherent phenotypic and epigenetic plasticity resulting in complex heterogenic diseases. In this review, we discuss the key principles of deregulation of epigenetic processes that shape this disease evolution. We consider measures to define and quantify clonal heterogeneity,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-10-23 datetime="2023-10-23T10:00:00.000Z">2023-10-23</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-10-23 datetime="2023-10-23T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20231105195156">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231103204956&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231027204803&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231023204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;ff&#x3D;20231105195156&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37871126"
                        open
                      >
                        <summary class="article-expander__title">Refphase: Multi-sample phasing reveals haplotype-specific copy number heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37871126/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Most computational methods that infer somatic copy number alterations (SCNAs) from bulk sequencing of DNA analyse tumour samples individually. However, the sequencing of multiple tumour samples from a patient&#x27;s disease is an increasingly common practice. We introduce Refphase, an algorithm that leverages this multi-sampling approach to infer haplotype-specific copy numbers through multi-sample phasing. We demonstrate Refphase&#x27;s ability to infer haplotype-specific SCNAs and characterise their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37871126"
                        open
                      >
                        <summary class="article-expander__title">Refphase: Multi-sample phasing reveals haplotype-specific copy number heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37871126/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Most computational methods that infer somatic copy number alterations (SCNAs) from bulk sequencing of DNA analyse tumour samples individually. However, the sequencing of multiple tumour samples from a patient&#x27;s disease is an increasingly common practice. We introduce Refphase, an algorithm that leverages this multi-sampling approach to infer haplotype-specific copy numbers through multi-sample phasing. We demonstrate Refphase&#x27;s ability to infer haplotype-specific SCNAs and characterise their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37871126"
                        open
                      >
                        <summary class="article-expander__title">Refphase: Multi-sample phasing reveals haplotype-specific copy number heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37871126/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231103204955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Most computational methods that infer somatic copy number alterations (SCNAs) from bulk sequencing of DNA analyse tumour samples individually. However, the sequencing of multiple tumour samples from a patient&#x27;s disease is an increasingly common practice. We introduce Refphase, an algorithm that leverages this multi-sampling approach to infer haplotype-specific copy numbers through multi-sample phasing. We demonstrate Refphase&#x27;s ability to infer haplotype-specific SCNAs and characterise their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231103204955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37871126"
                        open
                      >
                        <summary class="article-expander__title">Refphase: Multi-sample phasing reveals haplotype-specific copy number heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37871126/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Most computational methods that infer somatic copy number alterations (SCNAs) from bulk sequencing of DNA analyse tumour samples individually. However, the sequencing of multiple tumour samples from a patient&#x27;s disease is an increasingly common practice. We introduce Refphase, an algorithm that leverages this multi-sampling approach to infer haplotype-specific copy numbers through multi-sample phasing. We demonstrate Refphase&#x27;s ability to infer haplotype-specific SCNAs and characterise their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37871126"
                        open
                      >
                        <summary class="article-expander__title">Refphase: Multi-sample phasing reveals haplotype-specific copy number heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37871126/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Most computational methods that infer somatic copy number alterations (SCNAs) from bulk sequencing of DNA analyse tumour samples individually. However, the sequencing of multiple tumour samples from a patient&#x27;s disease is an increasingly common practice. We introduce Refphase, an algorithm that leverages this multi-sampling approach to infer haplotype-specific copy numbers through multi-sample phasing. We demonstrate Refphase&#x27;s ability to infer haplotype-specific SCNAs and characterise their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37871126"
                        open
                      >
                        <summary class="article-expander__title">Refphase: Multi-sample phasing reveals haplotype-specific copy number heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37871126/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Most computational methods that infer somatic copy number alterations (SCNAs) from bulk sequencing of DNA analyse tumour samples individually. However, the sequencing of multiple tumour samples from a patient&#x27;s disease is an increasingly common practice. We introduce Refphase, an algorithm that leverages this multi-sampling approach to infer haplotype-specific copy numbers through multi-sample phasing. We demonstrate Refphase&#x27;s ability to infer haplotype-specific SCNAs and characterise their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37871126"
                        open
                      >
                        <summary class="article-expander__title">Refphase: Multi-sample phasing reveals haplotype-specific copy number heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37871126/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Most computational methods that infer somatic copy number alterations (SCNAs) from bulk sequencing of DNA analyse tumour samples individually. However, the sequencing of multiple tumour samples from a patient&#x27;s disease is an increasingly common practice. We introduce Refphase, an algorithm that leverages this multi-sampling approach to infer haplotype-specific copy numbers through multi-sample phasing. We demonstrate Refphase&#x27;s ability to infer haplotype-specific SCNAs and characterise their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37871126"
                        open
                      >
                        <summary class="article-expander__title">Refphase: Multi-sample phasing reveals haplotype-specific copy number heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37871126/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Most computational methods that infer somatic copy number alterations (SCNAs) from bulk sequencing of DNA analyse tumour samples individually. However, the sequencing of multiple tumour samples from a patient&#x27;s disease is an increasingly common practice. We introduce Refphase, an algorithm that leverages this multi-sampling approach to infer haplotype-specific copy numbers through multi-sample phasing. We demonstrate Refphase&#x27;s ability to infer haplotype-specific SCNAs and characterise their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37871126"
                        open
                      >
                        <summary class="article-expander__title">Refphase: Multi-sample phasing reveals haplotype-specific copy number heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37871126/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Most computational methods that infer somatic copy number alterations (SCNAs) from bulk sequencing of DNA analyse tumour samples individually. However, the sequencing of multiple tumour samples from a patient&#x27;s disease is an increasingly common practice. We introduce Refphase, an algorithm that leverages this multi-sampling approach to infer haplotype-specific copy numbers through multi-sample phasing. We demonstrate Refphase&#x27;s ability to infer haplotype-specific SCNAs and characterise their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37871126"
                        open
                      >
                        <summary class="article-expander__title">Refphase: Multi-sample phasing reveals haplotype-specific copy number heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37871126/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231027204802&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Most computational methods that infer somatic copy number alterations (SCNAs) from bulk sequencing of DNA analyse tumour samples individually. However, the sequencing of multiple tumour samples from a patient&#x27;s disease is an increasingly common practice. We introduce Refphase, an algorithm that leverages this multi-sampling approach to infer haplotype-specific copy numbers through multi-sample phasing. We demonstrate Refphase&#x27;s ability to infer haplotype-specific SCNAs and characterise their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231027204802&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37871126"
                        open
                      >
                        <summary class="article-expander__title">Refphase: Multi-sample phasing reveals haplotype-specific copy number heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37871126/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Most computational methods that infer somatic copy number alterations (SCNAs) from bulk sequencing of DNA analyse tumour samples individually. However, the sequencing of multiple tumour samples from a patient&#x27;s disease is an increasingly common practice. We introduce Refphase, an algorithm that leverages this multi-sampling approach to infer haplotype-specific copy numbers through multi-sample phasing. We demonstrate Refphase&#x27;s ability to infer haplotype-specific SCNAs and characterise their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37871126"
                        open
                      >
                        <summary class="article-expander__title">Refphase: Multi-sample phasing reveals haplotype-specific copy number heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37871126/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Most computational methods that infer somatic copy number alterations (SCNAs) from bulk sequencing of DNA analyse tumour samples individually. However, the sequencing of multiple tumour samples from a patient&#x27;s disease is an increasingly common practice. We introduce Refphase, an algorithm that leverages this multi-sampling approach to infer haplotype-specific copy numbers through multi-sample phasing. We demonstrate Refphase&#x27;s ability to infer haplotype-specific SCNAs and characterise their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37871126"
                        open
                      >
                        <summary class="article-expander__title">Refphase: Multi-sample phasing reveals haplotype-specific copy number heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37871126/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Most computational methods that infer somatic copy number alterations (SCNAs) from bulk sequencing of DNA analyse tumour samples individually. However, the sequencing of multiple tumour samples from a patient&#x27;s disease is an increasingly common practice. We introduce Refphase, an algorithm that leverages this multi-sampling approach to infer haplotype-specific copy numbers through multi-sample phasing. We demonstrate Refphase&#x27;s ability to infer haplotype-specific SCNAs and characterise their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37871126"
                        open
                      >
                        <summary class="article-expander__title">Refphase: Multi-sample phasing reveals haplotype-specific copy number heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37871126/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231023204959&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Most computational methods that infer somatic copy number alterations (SCNAs) from bulk sequencing of DNA analyse tumour samples individually. However, the sequencing of multiple tumour samples from a patient&#x27;s disease is an increasingly common practice. We introduce Refphase, an algorithm that leverages this multi-sampling approach to infer haplotype-specific copy numbers through multi-sample phasing. We demonstrate Refphase&#x27;s ability to infer haplotype-specific SCNAs and characterise their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231023204959&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-10-21 datetime="2023-10-21T10:00:00.000Z">2023-10-21</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-10-21 datetime="2023-10-21T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20231105195156">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865509"
                        open
                      >
                        <summary class="article-expander__title">Acute Myeloid Leukemia Arising from Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a group of myeloid neoplasms characterized by clonal hematopoiesis and abnormal maturation of hematopoietic cells, resulting in cytopenias. The transformation of MDS to acute myeloid leukemia (AML) reflects a progressive increase in blasts due to impaired maturation of the malignant clone, and thus MDS and many AML subtypes form a biological continuum rather than representing two distinct diseases. Recent data suggest that, in addition to previously described...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865503"
                        open
                      >
                        <summary class="article-expander__title">Premalignant Clonal Hematopoiesis (Clonal Hematopoiesis of Indeterminate Potential and Clonal Cytopenia of Undetermined Significance)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865503/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Premalignant clonal hematopoiesis is the presence of somatic alterations in the blood of otherwise healthy individuals. Although the condition is not considered as a cancer, it carries an increased risk of developing a hematologic malignancy, particularly in those with large neoplastic clones, multiple pathogenic mutations, and high-risk mutations. In addition to the increased risk of malignancy, clonal hematopoiesis carries a markedly increased risk of cardiovascular events and death....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865509"
                        open
                      >
                        <summary class="article-expander__title">Acute Myeloid Leukemia Arising from Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a group of myeloid neoplasms characterized by clonal hematopoiesis and abnormal maturation of hematopoietic cells, resulting in cytopenias. The transformation of MDS to acute myeloid leukemia (AML) reflects a progressive increase in blasts due to impaired maturation of the malignant clone, and thus MDS and many AML subtypes form a biological continuum rather than representing two distinct diseases. Recent data suggest that, in addition to previously described...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865503"
                        open
                      >
                        <summary class="article-expander__title">Premalignant Clonal Hematopoiesis (Clonal Hematopoiesis of Indeterminate Potential and Clonal Cytopenia of Undetermined Significance)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865503/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Premalignant clonal hematopoiesis is the presence of somatic alterations in the blood of otherwise healthy individuals. Although the condition is not considered as a cancer, it carries an increased risk of developing a hematologic malignancy, particularly in those with large neoplastic clones, multiple pathogenic mutations, and high-risk mutations. In addition to the increased risk of malignancy, clonal hematopoiesis carries a markedly increased risk of cardiovascular events and death....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865509"
                        open
                      >
                        <summary class="article-expander__title">Acute Myeloid Leukemia Arising from Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231103204956&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a group of myeloid neoplasms characterized by clonal hematopoiesis and abnormal maturation of hematopoietic cells, resulting in cytopenias. The transformation of MDS to acute myeloid leukemia (AML) reflects a progressive increase in blasts due to impaired maturation of the malignant clone, and thus MDS and many AML subtypes form a biological continuum rather than representing two distinct diseases. Recent data suggest that, in addition to previously described...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865503"
                        open
                      >
                        <summary class="article-expander__title">Premalignant Clonal Hematopoiesis (Clonal Hematopoiesis of Indeterminate Potential and Clonal Cytopenia of Undetermined Significance)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865503/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231103204956&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Premalignant clonal hematopoiesis is the presence of somatic alterations in the blood of otherwise healthy individuals. Although the condition is not considered as a cancer, it carries an increased risk of developing a hematologic malignancy, particularly in those with large neoplastic clones, multiple pathogenic mutations, and high-risk mutations. In addition to the increased risk of malignancy, clonal hematopoiesis carries a markedly increased risk of cardiovascular events and death....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865509"
                        open
                      >
                        <summary class="article-expander__title">Acute Myeloid Leukemia Arising from Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a group of myeloid neoplasms characterized by clonal hematopoiesis and abnormal maturation of hematopoietic cells, resulting in cytopenias. The transformation of MDS to acute myeloid leukemia (AML) reflects a progressive increase in blasts due to impaired maturation of the malignant clone, and thus MDS and many AML subtypes form a biological continuum rather than representing two distinct diseases. Recent data suggest that, in addition to previously described...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865503"
                        open
                      >
                        <summary class="article-expander__title">Premalignant Clonal Hematopoiesis (Clonal Hematopoiesis of Indeterminate Potential and Clonal Cytopenia of Undetermined Significance)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865503/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Premalignant clonal hematopoiesis is the presence of somatic alterations in the blood of otherwise healthy individuals. Although the condition is not considered as a cancer, it carries an increased risk of developing a hematologic malignancy, particularly in those with large neoplastic clones, multiple pathogenic mutations, and high-risk mutations. In addition to the increased risk of malignancy, clonal hematopoiesis carries a markedly increased risk of cardiovascular events and death....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865509"
                        open
                      >
                        <summary class="article-expander__title">Acute Myeloid Leukemia Arising from Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a group of myeloid neoplasms characterized by clonal hematopoiesis and abnormal maturation of hematopoietic cells, resulting in cytopenias. The transformation of MDS to acute myeloid leukemia (AML) reflects a progressive increase in blasts due to impaired maturation of the malignant clone, and thus MDS and many AML subtypes form a biological continuum rather than representing two distinct diseases. Recent data suggest that, in addition to previously described...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865503"
                        open
                      >
                        <summary class="article-expander__title">Premalignant Clonal Hematopoiesis (Clonal Hematopoiesis of Indeterminate Potential and Clonal Cytopenia of Undetermined Significance)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865503/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Premalignant clonal hematopoiesis is the presence of somatic alterations in the blood of otherwise healthy individuals. Although the condition is not considered as a cancer, it carries an increased risk of developing a hematologic malignancy, particularly in those with large neoplastic clones, multiple pathogenic mutations, and high-risk mutations. In addition to the increased risk of malignancy, clonal hematopoiesis carries a markedly increased risk of cardiovascular events and death....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865509"
                        open
                      >
                        <summary class="article-expander__title">Acute Myeloid Leukemia Arising from Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a group of myeloid neoplasms characterized by clonal hematopoiesis and abnormal maturation of hematopoietic cells, resulting in cytopenias. The transformation of MDS to acute myeloid leukemia (AML) reflects a progressive increase in blasts due to impaired maturation of the malignant clone, and thus MDS and many AML subtypes form a biological continuum rather than representing two distinct diseases. Recent data suggest that, in addition to previously described...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865503"
                        open
                      >
                        <summary class="article-expander__title">Premalignant Clonal Hematopoiesis (Clonal Hematopoiesis of Indeterminate Potential and Clonal Cytopenia of Undetermined Significance)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865503/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Premalignant clonal hematopoiesis is the presence of somatic alterations in the blood of otherwise healthy individuals. Although the condition is not considered as a cancer, it carries an increased risk of developing a hematologic malignancy, particularly in those with large neoplastic clones, multiple pathogenic mutations, and high-risk mutations. In addition to the increased risk of malignancy, clonal hematopoiesis carries a markedly increased risk of cardiovascular events and death....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865509"
                        open
                      >
                        <summary class="article-expander__title">Acute Myeloid Leukemia Arising from Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a group of myeloid neoplasms characterized by clonal hematopoiesis and abnormal maturation of hematopoietic cells, resulting in cytopenias. The transformation of MDS to acute myeloid leukemia (AML) reflects a progressive increase in blasts due to impaired maturation of the malignant clone, and thus MDS and many AML subtypes form a biological continuum rather than representing two distinct diseases. Recent data suggest that, in addition to previously described...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865503"
                        open
                      >
                        <summary class="article-expander__title">Premalignant Clonal Hematopoiesis (Clonal Hematopoiesis of Indeterminate Potential and Clonal Cytopenia of Undetermined Significance)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865503/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Premalignant clonal hematopoiesis is the presence of somatic alterations in the blood of otherwise healthy individuals. Although the condition is not considered as a cancer, it carries an increased risk of developing a hematologic malignancy, particularly in those with large neoplastic clones, multiple pathogenic mutations, and high-risk mutations. In addition to the increased risk of malignancy, clonal hematopoiesis carries a markedly increased risk of cardiovascular events and death....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865509"
                        open
                      >
                        <summary class="article-expander__title">Acute Myeloid Leukemia Arising from Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a group of myeloid neoplasms characterized by clonal hematopoiesis and abnormal maturation of hematopoietic cells, resulting in cytopenias. The transformation of MDS to acute myeloid leukemia (AML) reflects a progressive increase in blasts due to impaired maturation of the malignant clone, and thus MDS and many AML subtypes form a biological continuum rather than representing two distinct diseases. Recent data suggest that, in addition to previously described...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865503"
                        open
                      >
                        <summary class="article-expander__title">Premalignant Clonal Hematopoiesis (Clonal Hematopoiesis of Indeterminate Potential and Clonal Cytopenia of Undetermined Significance)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865503/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Premalignant clonal hematopoiesis is the presence of somatic alterations in the blood of otherwise healthy individuals. Although the condition is not considered as a cancer, it carries an increased risk of developing a hematologic malignancy, particularly in those with large neoplastic clones, multiple pathogenic mutations, and high-risk mutations. In addition to the increased risk of malignancy, clonal hematopoiesis carries a markedly increased risk of cardiovascular events and death....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865509"
                        open
                      >
                        <summary class="article-expander__title">Acute Myeloid Leukemia Arising from Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a group of myeloid neoplasms characterized by clonal hematopoiesis and abnormal maturation of hematopoietic cells, resulting in cytopenias. The transformation of MDS to acute myeloid leukemia (AML) reflects a progressive increase in blasts due to impaired maturation of the malignant clone, and thus MDS and many AML subtypes form a biological continuum rather than representing two distinct diseases. Recent data suggest that, in addition to previously described...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865503"
                        open
                      >
                        <summary class="article-expander__title">Premalignant Clonal Hematopoiesis (Clonal Hematopoiesis of Indeterminate Potential and Clonal Cytopenia of Undetermined Significance)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865503/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Premalignant clonal hematopoiesis is the presence of somatic alterations in the blood of otherwise healthy individuals. Although the condition is not considered as a cancer, it carries an increased risk of developing a hematologic malignancy, particularly in those with large neoplastic clones, multiple pathogenic mutations, and high-risk mutations. In addition to the increased risk of malignancy, clonal hematopoiesis carries a markedly increased risk of cardiovascular events and death....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865509"
                        open
                      >
                        <summary class="article-expander__title">Acute Myeloid Leukemia Arising from Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231027204803&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a group of myeloid neoplasms characterized by clonal hematopoiesis and abnormal maturation of hematopoietic cells, resulting in cytopenias. The transformation of MDS to acute myeloid leukemia (AML) reflects a progressive increase in blasts due to impaired maturation of the malignant clone, and thus MDS and many AML subtypes form a biological continuum rather than representing two distinct diseases. Recent data suggest that, in addition to previously described...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865503"
                        open
                      >
                        <summary class="article-expander__title">Premalignant Clonal Hematopoiesis (Clonal Hematopoiesis of Indeterminate Potential and Clonal Cytopenia of Undetermined Significance)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865503/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231027204803&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Premalignant clonal hematopoiesis is the presence of somatic alterations in the blood of otherwise healthy individuals. Although the condition is not considered as a cancer, it carries an increased risk of developing a hematologic malignancy, particularly in those with large neoplastic clones, multiple pathogenic mutations, and high-risk mutations. In addition to the increased risk of malignancy, clonal hematopoiesis carries a markedly increased risk of cardiovascular events and death....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865509"
                        open
                      >
                        <summary class="article-expander__title">Acute Myeloid Leukemia Arising from Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a group of myeloid neoplasms characterized by clonal hematopoiesis and abnormal maturation of hematopoietic cells, resulting in cytopenias. The transformation of MDS to acute myeloid leukemia (AML) reflects a progressive increase in blasts due to impaired maturation of the malignant clone, and thus MDS and many AML subtypes form a biological continuum rather than representing two distinct diseases. Recent data suggest that, in addition to previously described...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865503"
                        open
                      >
                        <summary class="article-expander__title">Premalignant Clonal Hematopoiesis (Clonal Hematopoiesis of Indeterminate Potential and Clonal Cytopenia of Undetermined Significance)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865503/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Premalignant clonal hematopoiesis is the presence of somatic alterations in the blood of otherwise healthy individuals. Although the condition is not considered as a cancer, it carries an increased risk of developing a hematologic malignancy, particularly in those with large neoplastic clones, multiple pathogenic mutations, and high-risk mutations. In addition to the increased risk of malignancy, clonal hematopoiesis carries a markedly increased risk of cardiovascular events and death....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865509"
                        open
                      >
                        <summary class="article-expander__title">Acute Myeloid Leukemia Arising from Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a group of myeloid neoplasms characterized by clonal hematopoiesis and abnormal maturation of hematopoietic cells, resulting in cytopenias. The transformation of MDS to acute myeloid leukemia (AML) reflects a progressive increase in blasts due to impaired maturation of the malignant clone, and thus MDS and many AML subtypes form a biological continuum rather than representing two distinct diseases. Recent data suggest that, in addition to previously described...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865503"
                        open
                      >
                        <summary class="article-expander__title">Premalignant Clonal Hematopoiesis (Clonal Hematopoiesis of Indeterminate Potential and Clonal Cytopenia of Undetermined Significance)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865503/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Premalignant clonal hematopoiesis is the presence of somatic alterations in the blood of otherwise healthy individuals. Although the condition is not considered as a cancer, it carries an increased risk of developing a hematologic malignancy, particularly in those with large neoplastic clones, multiple pathogenic mutations, and high-risk mutations. In addition to the increased risk of malignancy, clonal hematopoiesis carries a markedly increased risk of cardiovascular events and death....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865509"
                        open
                      >
                        <summary class="article-expander__title">Acute Myeloid Leukemia Arising from Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a group of myeloid neoplasms characterized by clonal hematopoiesis and abnormal maturation of hematopoietic cells, resulting in cytopenias. The transformation of MDS to acute myeloid leukemia (AML) reflects a progressive increase in blasts due to impaired maturation of the malignant clone, and thus MDS and many AML subtypes form a biological continuum rather than representing two distinct diseases. Recent data suggest that, in addition to previously described...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865503"
                        open
                      >
                        <summary class="article-expander__title">Premalignant Clonal Hematopoiesis (Clonal Hematopoiesis of Indeterminate Potential and Clonal Cytopenia of Undetermined Significance)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865503/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Premalignant clonal hematopoiesis is the presence of somatic alterations in the blood of otherwise healthy individuals. Although the condition is not considered as a cancer, it carries an increased risk of developing a hematologic malignancy, particularly in those with large neoplastic clones, multiple pathogenic mutations, and high-risk mutations. In addition to the increased risk of malignancy, clonal hematopoiesis carries a markedly increased risk of cardiovascular events and death....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865509"
                        open
                      >
                        <summary class="article-expander__title">Acute Myeloid Leukemia Arising from Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231023204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a group of myeloid neoplasms characterized by clonal hematopoiesis and abnormal maturation of hematopoietic cells, resulting in cytopenias. The transformation of MDS to acute myeloid leukemia (AML) reflects a progressive increase in blasts due to impaired maturation of the malignant clone, and thus MDS and many AML subtypes form a biological continuum rather than representing two distinct diseases. Recent data suggest that, in addition to previously described...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865503"
                        open
                      >
                        <summary class="article-expander__title">Premalignant Clonal Hematopoiesis (Clonal Hematopoiesis of Indeterminate Potential and Clonal Cytopenia of Undetermined Significance)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865503/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231023204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Premalignant clonal hematopoiesis is the presence of somatic alterations in the blood of otherwise healthy individuals. Although the condition is not considered as a cancer, it carries an increased risk of developing a hematologic malignancy, particularly in those with large neoplastic clones, multiple pathogenic mutations, and high-risk mutations. In addition to the increased risk of malignancy, clonal hematopoiesis carries a markedly increased risk of cardiovascular events and death....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865509"
                        open
                      >
                        <summary class="article-expander__title">Acute Myeloid Leukemia Arising from Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231022205027&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a group of myeloid neoplasms characterized by clonal hematopoiesis and abnormal maturation of hematopoietic cells, resulting in cytopenias. The transformation of MDS to acute myeloid leukemia (AML) reflects a progressive increase in blasts due to impaired maturation of the malignant clone, and thus MDS and many AML subtypes form a biological continuum rather than representing two distinct diseases. Recent data suggest that, in addition to previously described...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865503"
                        open
                      >
                        <summary class="article-expander__title">Premalignant Clonal Hematopoiesis (Clonal Hematopoiesis of Indeterminate Potential and Clonal Cytopenia of Undetermined Significance)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865503/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231022205027&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Premalignant clonal hematopoiesis is the presence of somatic alterations in the blood of otherwise healthy individuals. Although the condition is not considered as a cancer, it carries an increased risk of developing a hematologic malignancy, particularly in those with large neoplastic clones, multiple pathogenic mutations, and high-risk mutations. In addition to the increased risk of malignancy, clonal hematopoiesis carries a markedly increased risk of cardiovascular events and death....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;ff&#x3D;20231105195156&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865503"
                        open
                      >
                        <summary class="article-expander__title">Premalignant Clonal Hematopoiesis (Clonal Hematopoiesis of Indeterminate Potential and Clonal Cytopenia of Undetermined Significance)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865503/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Premalignant clonal hematopoiesis is the presence of somatic alterations in the blood of otherwise healthy individuals. Although the condition is not considered as a cancer, it carries an increased risk of developing a hematologic malignancy, particularly in those with large neoplastic clones, multiple pathogenic mutations, and high-risk mutations. In addition to the increased risk of malignancy, clonal hematopoiesis carries a markedly increased risk of cardiovascular events and death....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865503"
                        open
                      >
                        <summary class="article-expander__title">Premalignant Clonal Hematopoiesis (Clonal Hematopoiesis of Indeterminate Potential and Clonal Cytopenia of Undetermined Significance)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865503/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Premalignant clonal hematopoiesis is the presence of somatic alterations in the blood of otherwise healthy individuals. Although the condition is not considered as a cancer, it carries an increased risk of developing a hematologic malignancy, particularly in those with large neoplastic clones, multiple pathogenic mutations, and high-risk mutations. In addition to the increased risk of malignancy, clonal hematopoiesis carries a markedly increased risk of cardiovascular events and death....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-10-20 datetime="2023-10-20T10:00:00.000Z">2023-10-20</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-10-20 datetime="2023-10-20T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20231105195156">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37863054"
                        open
                      >
                        <summary class="article-expander__title">SRCAP mutations drive clonal hematopoiesis through epigenetic and DNA repair dysregulation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37863054/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations accumulate in all cells with age and can confer a selective advantage, leading to clonal expansion over time. In hematopoietic cells, mutations in a subset of genes regulating DNA repair or epigenetics frequently lead to clonal hematopoiesis (CH). Here, we describe the context and mechanisms that lead to enrichment of hematopoietic stem cells (HSCs) with mutations in SRCAP, which encodes a chromatin remodeler that also influences DNA repair. We show that SRCAP mutations confer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37863054"
                        open
                      >
                        <summary class="article-expander__title">SRCAP mutations drive clonal hematopoiesis through epigenetic and DNA repair dysregulation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37863054/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations accumulate in all cells with age and can confer a selective advantage, leading to clonal expansion over time. In hematopoietic cells, mutations in a subset of genes regulating DNA repair or epigenetics frequently lead to clonal hematopoiesis (CH). Here, we describe the context and mechanisms that lead to enrichment of hematopoietic stem cells (HSCs) with mutations in SRCAP, which encodes a chromatin remodeler that also influences DNA repair. We show that SRCAP mutations confer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37863054"
                        open
                      >
                        <summary class="article-expander__title">SRCAP mutations drive clonal hematopoiesis through epigenetic and DNA repair dysregulation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37863054/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations accumulate in all cells with age and can confer a selective advantage, leading to clonal expansion over time. In hematopoietic cells, mutations in a subset of genes regulating DNA repair or epigenetics frequently lead to clonal hematopoiesis (CH). Here, we describe the context and mechanisms that lead to enrichment of hematopoietic stem cells (HSCs) with mutations in SRCAP, which encodes a chromatin remodeler that also influences DNA repair. We show that SRCAP mutations confer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37863054"
                        open
                      >
                        <summary class="article-expander__title">SRCAP mutations drive clonal hematopoiesis through epigenetic and DNA repair dysregulation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37863054/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations accumulate in all cells with age and can confer a selective advantage, leading to clonal expansion over time. In hematopoietic cells, mutations in a subset of genes regulating DNA repair or epigenetics frequently lead to clonal hematopoiesis (CH). Here, we describe the context and mechanisms that lead to enrichment of hematopoietic stem cells (HSCs) with mutations in SRCAP, which encodes a chromatin remodeler that also influences DNA repair. We show that SRCAP mutations confer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37863054"
                        open
                      >
                        <summary class="article-expander__title">SRCAP mutations drive clonal hematopoiesis through epigenetic and DNA repair dysregulation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37863054/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations accumulate in all cells with age and can confer a selective advantage, leading to clonal expansion over time. In hematopoietic cells, mutations in a subset of genes regulating DNA repair or epigenetics frequently lead to clonal hematopoiesis (CH). Here, we describe the context and mechanisms that lead to enrichment of hematopoietic stem cells (HSCs) with mutations in SRCAP, which encodes a chromatin remodeler that also influences DNA repair. We show that SRCAP mutations confer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37863054"
                        open
                      >
                        <summary class="article-expander__title">SRCAP mutations drive clonal hematopoiesis through epigenetic and DNA repair dysregulation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37863054/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations accumulate in all cells with age and can confer a selective advantage, leading to clonal expansion over time. In hematopoietic cells, mutations in a subset of genes regulating DNA repair or epigenetics frequently lead to clonal hematopoiesis (CH). Here, we describe the context and mechanisms that lead to enrichment of hematopoietic stem cells (HSCs) with mutations in SRCAP, which encodes a chromatin remodeler that also influences DNA repair. We show that SRCAP mutations confer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37863054"
                        open
                      >
                        <summary class="article-expander__title">SRCAP mutations drive clonal hematopoiesis through epigenetic and DNA repair dysregulation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37863054/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231027204803&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations accumulate in all cells with age and can confer a selective advantage, leading to clonal expansion over time. In hematopoietic cells, mutations in a subset of genes regulating DNA repair or epigenetics frequently lead to clonal hematopoiesis (CH). Here, we describe the context and mechanisms that lead to enrichment of hematopoietic stem cells (HSCs) with mutations in SRCAP, which encodes a chromatin remodeler that also influences DNA repair. We show that SRCAP mutations confer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37863054"
                        open
                      >
                        <summary class="article-expander__title">SRCAP mutations drive clonal hematopoiesis through epigenetic and DNA repair dysregulation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37863054/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations accumulate in all cells with age and can confer a selective advantage, leading to clonal expansion over time. In hematopoietic cells, mutations in a subset of genes regulating DNA repair or epigenetics frequently lead to clonal hematopoiesis (CH). Here, we describe the context and mechanisms that lead to enrichment of hematopoietic stem cells (HSCs) with mutations in SRCAP, which encodes a chromatin remodeler that also influences DNA repair. We show that SRCAP mutations confer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37863054"
                        open
                      >
                        <summary class="article-expander__title">SRCAP mutations drive clonal hematopoiesis through epigenetic and DNA repair dysregulation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37863054/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations accumulate in all cells with age and can confer a selective advantage, leading to clonal expansion over time. In hematopoietic cells, mutations in a subset of genes regulating DNA repair or epigenetics frequently lead to clonal hematopoiesis (CH). Here, we describe the context and mechanisms that lead to enrichment of hematopoietic stem cells (HSCs) with mutations in SRCAP, which encodes a chromatin remodeler that also influences DNA repair. We show that SRCAP mutations confer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37863054"
                        open
                      >
                        <summary class="article-expander__title">SRCAP mutations drive clonal hematopoiesis through epigenetic and DNA repair dysregulation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37863054/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations accumulate in all cells with age and can confer a selective advantage, leading to clonal expansion over time. In hematopoietic cells, mutations in a subset of genes regulating DNA repair or epigenetics frequently lead to clonal hematopoiesis (CH). Here, we describe the context and mechanisms that lead to enrichment of hematopoietic stem cells (HSCs) with mutations in SRCAP, which encodes a chromatin remodeler that also influences DNA repair. We show that SRCAP mutations confer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37863054"
                        open
                      >
                        <summary class="article-expander__title">SRCAP mutations drive clonal hematopoiesis through epigenetic and DNA repair dysregulation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37863054/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231023204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations accumulate in all cells with age and can confer a selective advantage, leading to clonal expansion over time. In hematopoietic cells, mutations in a subset of genes regulating DNA repair or epigenetics frequently lead to clonal hematopoiesis (CH). Here, we describe the context and mechanisms that lead to enrichment of hematopoietic stem cells (HSCs) with mutations in SRCAP, which encodes a chromatin remodeler that also influences DNA repair. We show that SRCAP mutations confer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37863054"
                        open
                      >
                        <summary class="article-expander__title">SRCAP mutations drive clonal hematopoiesis through epigenetic and DNA repair dysregulation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37863054/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231022205027&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations accumulate in all cells with age and can confer a selective advantage, leading to clonal expansion over time. In hematopoietic cells, mutations in a subset of genes regulating DNA repair or epigenetics frequently lead to clonal hematopoiesis (CH). Here, we describe the context and mechanisms that lead to enrichment of hematopoietic stem cells (HSCs) with mutations in SRCAP, which encodes a chromatin remodeler that also influences DNA repair. We show that SRCAP mutations confer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37863054"
                        open
                      >
                        <summary class="article-expander__title">SRCAP mutations drive clonal hematopoiesis through epigenetic and DNA repair dysregulation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37863054/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231021205333&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations accumulate in all cells with age and can confer a selective advantage, leading to clonal expansion over time. In hematopoietic cells, mutations in a subset of genes regulating DNA repair or epigenetics frequently lead to clonal hematopoiesis (CH). Here, we describe the context and mechanisms that lead to enrichment of hematopoietic stem cells (HSCs) with mutations in SRCAP, which encodes a chromatin remodeler that also influences DNA repair. We show that SRCAP mutations confer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;ff&#x3D;20231105195156&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37858853"
                        open
                      >
                        <summary class="article-expander__title">A pan-cancer landscape of pathogenic somatic copy number variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37858853/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study contributes to elucidate the functional impact of pathogenic CNVs in tumor progression and sheds light on their potential as prognostic markers in specific cancer types.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37858853"
                        open
                      >
                        <summary class="article-expander__title">A pan-cancer landscape of pathogenic somatic copy number variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37858853/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study contributes to elucidate the functional impact of pathogenic CNVs in tumor progression and sheds light on their potential as prognostic markers in specific cancer types.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37858853"
                        open
                      >
                        <summary class="article-expander__title">A pan-cancer landscape of pathogenic somatic copy number variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37858853/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study contributes to elucidate the functional impact of pathogenic CNVs in tumor progression and sheds light on their potential as prognostic markers in specific cancer types.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37858853"
                        open
                      >
                        <summary class="article-expander__title">A pan-cancer landscape of pathogenic somatic copy number variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37858853/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study contributes to elucidate the functional impact of pathogenic CNVs in tumor progression and sheds light on their potential as prognostic markers in specific cancer types.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37858853"
                        open
                      >
                        <summary class="article-expander__title">A pan-cancer landscape of pathogenic somatic copy number variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37858853/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231027204802&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study contributes to elucidate the functional impact of pathogenic CNVs in tumor progression and sheds light on their potential as prognostic markers in specific cancer types.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37858853"
                        open
                      >
                        <summary class="article-expander__title">A pan-cancer landscape of pathogenic somatic copy number variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37858853/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study contributes to elucidate the functional impact of pathogenic CNVs in tumor progression and sheds light on their potential as prognostic markers in specific cancer types.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37858853"
                        open
                      >
                        <summary class="article-expander__title">A pan-cancer landscape of pathogenic somatic copy number variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37858853/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study contributes to elucidate the functional impact of pathogenic CNVs in tumor progression and sheds light on their potential as prognostic markers in specific cancer types.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37858853"
                        open
                      >
                        <summary class="article-expander__title">A pan-cancer landscape of pathogenic somatic copy number variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37858853/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study contributes to elucidate the functional impact of pathogenic CNVs in tumor progression and sheds light on their potential as prognostic markers in specific cancer types.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37858853"
                        open
                      >
                        <summary class="article-expander__title">A pan-cancer landscape of pathogenic somatic copy number variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37858853/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231023204959&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study contributes to elucidate the functional impact of pathogenic CNVs in tumor progression and sheds light on their potential as prognostic markers in specific cancer types.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37858853"
                        open
                      >
                        <summary class="article-expander__title">A pan-cancer landscape of pathogenic somatic copy number variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37858853/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231022205027&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study contributes to elucidate the functional impact of pathogenic CNVs in tumor progression and sheds light on their potential as prognostic markers in specific cancer types.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37858853"
                        open
                      >
                        <summary class="article-expander__title">A pan-cancer landscape of pathogenic somatic copy number variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37858853/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231021205333&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study contributes to elucidate the functional impact of pathogenic CNVs in tumor progression and sheds light on their potential as prognostic markers in specific cancer types.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37858853"
                        open
                      >
                        <summary class="article-expander__title">A pan-cancer landscape of pathogenic somatic copy number variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37858853/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231020204820&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study contributes to elucidate the functional impact of pathogenic CNVs in tumor progression and sheds light on their potential as prognostic markers in specific cancer types.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-10-19 datetime="2023-10-19T10:00:00.000Z">2023-10-19</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-10-19 datetime="2023-10-19T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;ff&#x3D;20231105195156&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37858132"
                        open
                      >
                        <summary class="article-expander__title">Oligo-metastatic neoPlasms from the gastro-intestinal tract: iDentIfiCaTIon of cliNical and molecular drivers: the PREDICTION study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37858132/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Metastatic disease in tumors originating from the gastrointestinal tract can exhibit varying degrees of tumor burden at presentation. Some patients follow a less aggressive disease course, characterized by a limited number of metastatic sites, referred to as &quot;oligo-metastatic disease&quot; (OMD). The precise biological characteristics that define the oligometastatic behavior remain uncertain. In this study, we present a protocol designed to prospectively identify OMD, with the aim of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37858132"
                        open
                      >
                        <summary class="article-expander__title">Oligo-metastatic neoPlasms from the gastro-intestinal tract: iDentIfiCaTIon of cliNical and molecular drivers: the PREDICTION study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37858132/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Metastatic disease in tumors originating from the gastrointestinal tract can exhibit varying degrees of tumor burden at presentation. Some patients follow a less aggressive disease course, characterized by a limited number of metastatic sites, referred to as &quot;oligo-metastatic disease&quot; (OMD). The precise biological characteristics that define the oligometastatic behavior remain uncertain. In this study, we present a protocol designed to prospectively identify OMD, with the aim of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37858132"
                        open
                      >
                        <summary class="article-expander__title">Oligo-metastatic neoPlasms from the gastro-intestinal tract: iDentIfiCaTIon of cliNical and molecular drivers: the PREDICTION study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37858132/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Metastatic disease in tumors originating from the gastrointestinal tract can exhibit varying degrees of tumor burden at presentation. Some patients follow a less aggressive disease course, characterized by a limited number of metastatic sites, referred to as &quot;oligo-metastatic disease&quot; (OMD). The precise biological characteristics that define the oligometastatic behavior remain uncertain. In this study, we present a protocol designed to prospectively identify OMD, with the aim of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37858132"
                        open
                      >
                        <summary class="article-expander__title">Oligo-metastatic neoPlasms from the gastro-intestinal tract: iDentIfiCaTIon of cliNical and molecular drivers: the PREDICTION study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37858132/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231027204802&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Metastatic disease in tumors originating from the gastrointestinal tract can exhibit varying degrees of tumor burden at presentation. Some patients follow a less aggressive disease course, characterized by a limited number of metastatic sites, referred to as &quot;oligo-metastatic disease&quot; (OMD). The precise biological characteristics that define the oligometastatic behavior remain uncertain. In this study, we present a protocol designed to prospectively identify OMD, with the aim of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37858132"
                        open
                      >
                        <summary class="article-expander__title">Oligo-metastatic neoPlasms from the gastro-intestinal tract: iDentIfiCaTIon of cliNical and molecular drivers: the PREDICTION study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37858132/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Metastatic disease in tumors originating from the gastrointestinal tract can exhibit varying degrees of tumor burden at presentation. Some patients follow a less aggressive disease course, characterized by a limited number of metastatic sites, referred to as &quot;oligo-metastatic disease&quot; (OMD). The precise biological characteristics that define the oligometastatic behavior remain uncertain. In this study, we present a protocol designed to prospectively identify OMD, with the aim of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37858132"
                        open
                      >
                        <summary class="article-expander__title">Oligo-metastatic neoPlasms from the gastro-intestinal tract: iDentIfiCaTIon of cliNical and molecular drivers: the PREDICTION study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37858132/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Metastatic disease in tumors originating from the gastrointestinal tract can exhibit varying degrees of tumor burden at presentation. Some patients follow a less aggressive disease course, characterized by a limited number of metastatic sites, referred to as &quot;oligo-metastatic disease&quot; (OMD). The precise biological characteristics that define the oligometastatic behavior remain uncertain. In this study, we present a protocol designed to prospectively identify OMD, with the aim of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37858132"
                        open
                      >
                        <summary class="article-expander__title">Oligo-metastatic neoPlasms from the gastro-intestinal tract: iDentIfiCaTIon of cliNical and molecular drivers: the PREDICTION study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37858132/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Metastatic disease in tumors originating from the gastrointestinal tract can exhibit varying degrees of tumor burden at presentation. Some patients follow a less aggressive disease course, characterized by a limited number of metastatic sites, referred to as &quot;oligo-metastatic disease&quot; (OMD). The precise biological characteristics that define the oligometastatic behavior remain uncertain. In this study, we present a protocol designed to prospectively identify OMD, with the aim of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37858132"
                        open
                      >
                        <summary class="article-expander__title">Oligo-metastatic neoPlasms from the gastro-intestinal tract: iDentIfiCaTIon of cliNical and molecular drivers: the PREDICTION study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37858132/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231023204959&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Metastatic disease in tumors originating from the gastrointestinal tract can exhibit varying degrees of tumor burden at presentation. Some patients follow a less aggressive disease course, characterized by a limited number of metastatic sites, referred to as &quot;oligo-metastatic disease&quot; (OMD). The precise biological characteristics that define the oligometastatic behavior remain uncertain. In this study, we present a protocol designed to prospectively identify OMD, with the aim of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37858132"
                        open
                      >
                        <summary class="article-expander__title">Oligo-metastatic neoPlasms from the gastro-intestinal tract: iDentIfiCaTIon of cliNical and molecular drivers: the PREDICTION study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37858132/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231022205027&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Metastatic disease in tumors originating from the gastrointestinal tract can exhibit varying degrees of tumor burden at presentation. Some patients follow a less aggressive disease course, characterized by a limited number of metastatic sites, referred to as &quot;oligo-metastatic disease&quot; (OMD). The precise biological characteristics that define the oligometastatic behavior remain uncertain. In this study, we present a protocol designed to prospectively identify OMD, with the aim of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37858132"
                        open
                      >
                        <summary class="article-expander__title">Oligo-metastatic neoPlasms from the gastro-intestinal tract: iDentIfiCaTIon of cliNical and molecular drivers: the PREDICTION study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37858132/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231021205333&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Metastatic disease in tumors originating from the gastrointestinal tract can exhibit varying degrees of tumor burden at presentation. Some patients follow a less aggressive disease course, characterized by a limited number of metastatic sites, referred to as &quot;oligo-metastatic disease&quot; (OMD). The precise biological characteristics that define the oligometastatic behavior remain uncertain. In this study, we present a protocol designed to prospectively identify OMD, with the aim of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37858132"
                        open
                      >
                        <summary class="article-expander__title">Oligo-metastatic neoPlasms from the gastro-intestinal tract: iDentIfiCaTIon of cliNical and molecular drivers: the PREDICTION study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37858132/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231020204820&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Metastatic disease in tumors originating from the gastrointestinal tract can exhibit varying degrees of tumor burden at presentation. Some patients follow a less aggressive disease course, characterized by a limited number of metastatic sites, referred to as &quot;oligo-metastatic disease&quot; (OMD). The precise biological characteristics that define the oligometastatic behavior remain uncertain. In this study, we present a protocol designed to prospectively identify OMD, with the aim of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-10-18 datetime="2023-10-18T10:00:00.000Z">2023-10-18</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-10-18 datetime="2023-10-18T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20231105195156">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37852258"
                        open
                      >
                        <summary class="article-expander__title">A mouse model with high clonal barcode diversity for joint lineage, transcriptomic, and epigenomic profiling in single cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37852258/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cellular lineage histories and their molecular states encode fundamental principles of tissue development and homeostasis. Current lineage-recording mouse models have insufficient barcode diversity and single-cell lineage coverage for profiling tissues composed of millions of cells. Here, we developed DARLIN, an inducible Cas9 barcoding mouse line that utilizes terminal deoxynucleotidyl transferase (TdT) and 30 CRISPR target sites. DARLIN is inducible, generates massive lineage barcodes across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37852258"
                        open
                      >
                        <summary class="article-expander__title">A mouse model with high clonal barcode diversity for joint lineage, transcriptomic, and epigenomic profiling in single cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37852258/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cellular lineage histories and their molecular states encode fundamental principles of tissue development and homeostasis. Current lineage-recording mouse models have insufficient barcode diversity and single-cell lineage coverage for profiling tissues composed of millions of cells. Here, we developed DARLIN, an inducible Cas9 barcoding mouse line that utilizes terminal deoxynucleotidyl transferase (TdT) and 30 CRISPR target sites. DARLIN is inducible, generates massive lineage barcodes across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37852258"
                        open
                      >
                        <summary class="article-expander__title">A mouse model with high clonal barcode diversity for joint lineage, transcriptomic, and epigenomic profiling in single cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37852258/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cellular lineage histories and their molecular states encode fundamental principles of tissue development and homeostasis. Current lineage-recording mouse models have insufficient barcode diversity and single-cell lineage coverage for profiling tissues composed of millions of cells. Here, we developed DARLIN, an inducible Cas9 barcoding mouse line that utilizes terminal deoxynucleotidyl transferase (TdT) and 30 CRISPR target sites. DARLIN is inducible, generates massive lineage barcodes across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37852258"
                        open
                      >
                        <summary class="article-expander__title">A mouse model with high clonal barcode diversity for joint lineage, transcriptomic, and epigenomic profiling in single cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37852258/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cellular lineage histories and their molecular states encode fundamental principles of tissue development and homeostasis. Current lineage-recording mouse models have insufficient barcode diversity and single-cell lineage coverage for profiling tissues composed of millions of cells. Here, we developed DARLIN, an inducible Cas9 barcoding mouse line that utilizes terminal deoxynucleotidyl transferase (TdT) and 30 CRISPR target sites. DARLIN is inducible, generates massive lineage barcodes across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37852258"
                        open
                      >
                        <summary class="article-expander__title">A mouse model with high clonal barcode diversity for joint lineage, transcriptomic, and epigenomic profiling in single cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37852258/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cellular lineage histories and their molecular states encode fundamental principles of tissue development and homeostasis. Current lineage-recording mouse models have insufficient barcode diversity and single-cell lineage coverage for profiling tissues composed of millions of cells. Here, we developed DARLIN, an inducible Cas9 barcoding mouse line that utilizes terminal deoxynucleotidyl transferase (TdT) and 30 CRISPR target sites. DARLIN is inducible, generates massive lineage barcodes across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37852258"
                        open
                      >
                        <summary class="article-expander__title">A mouse model with high clonal barcode diversity for joint lineage, transcriptomic, and epigenomic profiling in single cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37852258/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231027204803&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cellular lineage histories and their molecular states encode fundamental principles of tissue development and homeostasis. Current lineage-recording mouse models have insufficient barcode diversity and single-cell lineage coverage for profiling tissues composed of millions of cells. Here, we developed DARLIN, an inducible Cas9 barcoding mouse line that utilizes terminal deoxynucleotidyl transferase (TdT) and 30 CRISPR target sites. DARLIN is inducible, generates massive lineage barcodes across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37852258"
                        open
                      >
                        <summary class="article-expander__title">A mouse model with high clonal barcode diversity for joint lineage, transcriptomic, and epigenomic profiling in single cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37852258/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cellular lineage histories and their molecular states encode fundamental principles of tissue development and homeostasis. Current lineage-recording mouse models have insufficient barcode diversity and single-cell lineage coverage for profiling tissues composed of millions of cells. Here, we developed DARLIN, an inducible Cas9 barcoding mouse line that utilizes terminal deoxynucleotidyl transferase (TdT) and 30 CRISPR target sites. DARLIN is inducible, generates massive lineage barcodes across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37852258"
                        open
                      >
                        <summary class="article-expander__title">A mouse model with high clonal barcode diversity for joint lineage, transcriptomic, and epigenomic profiling in single cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37852258/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cellular lineage histories and their molecular states encode fundamental principles of tissue development and homeostasis. Current lineage-recording mouse models have insufficient barcode diversity and single-cell lineage coverage for profiling tissues composed of millions of cells. Here, we developed DARLIN, an inducible Cas9 barcoding mouse line that utilizes terminal deoxynucleotidyl transferase (TdT) and 30 CRISPR target sites. DARLIN is inducible, generates massive lineage barcodes across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37852258"
                        open
                      >
                        <summary class="article-expander__title">A mouse model with high clonal barcode diversity for joint lineage, transcriptomic, and epigenomic profiling in single cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37852258/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cellular lineage histories and their molecular states encode fundamental principles of tissue development and homeostasis. Current lineage-recording mouse models have insufficient barcode diversity and single-cell lineage coverage for profiling tissues composed of millions of cells. Here, we developed DARLIN, an inducible Cas9 barcoding mouse line that utilizes terminal deoxynucleotidyl transferase (TdT) and 30 CRISPR target sites. DARLIN is inducible, generates massive lineage barcodes across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37852258"
                        open
                      >
                        <summary class="article-expander__title">A mouse model with high clonal barcode diversity for joint lineage, transcriptomic, and epigenomic profiling in single cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37852258/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231023204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cellular lineage histories and their molecular states encode fundamental principles of tissue development and homeostasis. Current lineage-recording mouse models have insufficient barcode diversity and single-cell lineage coverage for profiling tissues composed of millions of cells. Here, we developed DARLIN, an inducible Cas9 barcoding mouse line that utilizes terminal deoxynucleotidyl transferase (TdT) and 30 CRISPR target sites. DARLIN is inducible, generates massive lineage barcodes across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37852258"
                        open
                      >
                        <summary class="article-expander__title">A mouse model with high clonal barcode diversity for joint lineage, transcriptomic, and epigenomic profiling in single cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37852258/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231022205027&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cellular lineage histories and their molecular states encode fundamental principles of tissue development and homeostasis. Current lineage-recording mouse models have insufficient barcode diversity and single-cell lineage coverage for profiling tissues composed of millions of cells. Here, we developed DARLIN, an inducible Cas9 barcoding mouse line that utilizes terminal deoxynucleotidyl transferase (TdT) and 30 CRISPR target sites. DARLIN is inducible, generates massive lineage barcodes across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37852258"
                        open
                      >
                        <summary class="article-expander__title">A mouse model with high clonal barcode diversity for joint lineage, transcriptomic, and epigenomic profiling in single cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37852258/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231021205333&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cellular lineage histories and their molecular states encode fundamental principles of tissue development and homeostasis. Current lineage-recording mouse models have insufficient barcode diversity and single-cell lineage coverage for profiling tissues composed of millions of cells. Here, we developed DARLIN, an inducible Cas9 barcoding mouse line that utilizes terminal deoxynucleotidyl transferase (TdT) and 30 CRISPR target sites. DARLIN is inducible, generates massive lineage barcodes across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37852258"
                        open
                      >
                        <summary class="article-expander__title">A mouse model with high clonal barcode diversity for joint lineage, transcriptomic, and epigenomic profiling in single cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37852258/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231020204820&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cellular lineage histories and their molecular states encode fundamental principles of tissue development and homeostasis. Current lineage-recording mouse models have insufficient barcode diversity and single-cell lineage coverage for profiling tissues composed of millions of cells. Here, we developed DARLIN, an inducible Cas9 barcoding mouse line that utilizes terminal deoxynucleotidyl transferase (TdT) and 30 CRISPR target sites. DARLIN is inducible, generates massive lineage barcodes across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37852258"
                        open
                      >
                        <summary class="article-expander__title">A mouse model with high clonal barcode diversity for joint lineage, transcriptomic, and epigenomic profiling in single cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37852258/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231019204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cellular lineage histories and their molecular states encode fundamental principles of tissue development and homeostasis. Current lineage-recording mouse models have insufficient barcode diversity and single-cell lineage coverage for profiling tissues composed of millions of cells. Here, we developed DARLIN, an inducible Cas9 barcoding mouse line that utilizes terminal deoxynucleotidyl transferase (TdT) and 30 CRISPR target sites. DARLIN is inducible, generates massive lineage barcodes across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-10-14 datetime="2023-10-14T10:00:00.000Z">2023-10-14</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-10-14 datetime="2023-10-14T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20231105195156">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231027204803&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231023204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231022205027&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231021205333&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231020204820&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231019204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231018205013&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231017205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231016205028&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231015205105&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231014205339&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;ff&#x3D;20231105195156&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231027204802&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231023204959&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231022205027&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231021205333&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231020204820&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231019204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231018205013&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231017205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231016205028&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231015205105&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231014205339&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-10-13 datetime="2023-10-13T10:00:00.000Z">2023-10-13</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-10-13 datetime="2023-10-13T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20231105195156">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37833013"
                        open
                      >
                        <summary class="article-expander__title">Technology of genomic balancing of chromatin of autologous hematopoietic stem cells for gene therapy of fatal immune-mediated diseases of civilization, extended life expectancy and sudden human death prevention</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37833013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A biotechnology for personalized ex vivo gene therapy based on molecular genomic balancing of hematopoietic stem cell (HSC) chromatin with nucleosome monomers of human genomic DNA (hDNA^(nmr)) has been developed and implemented in the clinic to change (to &quot;correct&quot;) mutant chromosome loci genomes of dominant HSC clones that form mono- and oligoclonal hematopoiesis during aging and major (oncological, cardiovascular, neurodegenerative and autoimmune) fatal immune-mediated diseases of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37833013"
                        open
                      >
                        <summary class="article-expander__title">Technology of genomic balancing of chromatin of autologous hematopoietic stem cells for gene therapy of fatal immune-mediated diseases of civilization, extended life expectancy and sudden human death prevention</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37833013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A biotechnology for personalized ex vivo gene therapy based on molecular genomic balancing of hematopoietic stem cell (HSC) chromatin with nucleosome monomers of human genomic DNA (hDNA^(nmr)) has been developed and implemented in the clinic to change (to &quot;correct&quot;) mutant chromosome loci genomes of dominant HSC clones that form mono- and oligoclonal hematopoiesis during aging and major (oncological, cardiovascular, neurodegenerative and autoimmune) fatal immune-mediated diseases of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37832948"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis, Inflammation, and Hematologic Malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37832948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic or acquired mutations are postzygotic genetic variations that can occur within any tissue. These mutations accumulate during aging and have classically been linked to malignant processes. Tremendous advancements over the past years have led to a deeper understanding of the role of somatic mutations in benign and malignant age-related diseases. Here, we review the somatic mutations that accumulate in the blood and their connection to disease states, with a particular focus on inflammatory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37833013"
                        open
                      >
                        <summary class="article-expander__title">Technology of genomic balancing of chromatin of autologous hematopoietic stem cells for gene therapy of fatal immune-mediated diseases of civilization, extended life expectancy and sudden human death prevention</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37833013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A biotechnology for personalized ex vivo gene therapy based on molecular genomic balancing of hematopoietic stem cell (HSC) chromatin with nucleosome monomers of human genomic DNA (hDNA^(nmr)) has been developed and implemented in the clinic to change (to &quot;correct&quot;) mutant chromosome loci genomes of dominant HSC clones that form mono- and oligoclonal hematopoiesis during aging and major (oncological, cardiovascular, neurodegenerative and autoimmune) fatal immune-mediated diseases of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37832948"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis, Inflammation, and Hematologic Malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37832948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic or acquired mutations are postzygotic genetic variations that can occur within any tissue. These mutations accumulate during aging and have classically been linked to malignant processes. Tremendous advancements over the past years have led to a deeper understanding of the role of somatic mutations in benign and malignant age-related diseases. Here, we review the somatic mutations that accumulate in the blood and their connection to disease states, with a particular focus on inflammatory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37833013"
                        open
                      >
                        <summary class="article-expander__title">Technology of genomic balancing of chromatin of autologous hematopoietic stem cells for gene therapy of fatal immune-mediated diseases of civilization, extended life expectancy and sudden human death prevention</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37833013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A biotechnology for personalized ex vivo gene therapy based on molecular genomic balancing of hematopoietic stem cell (HSC) chromatin with nucleosome monomers of human genomic DNA (hDNA^(nmr)) has been developed and implemented in the clinic to change (to &quot;correct&quot;) mutant chromosome loci genomes of dominant HSC clones that form mono- and oligoclonal hematopoiesis during aging and major (oncological, cardiovascular, neurodegenerative and autoimmune) fatal immune-mediated diseases of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37832948"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis, Inflammation, and Hematologic Malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37832948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic or acquired mutations are postzygotic genetic variations that can occur within any tissue. These mutations accumulate during aging and have classically been linked to malignant processes. Tremendous advancements over the past years have led to a deeper understanding of the role of somatic mutations in benign and malignant age-related diseases. Here, we review the somatic mutations that accumulate in the blood and their connection to disease states, with a particular focus on inflammatory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37833013"
                        open
                      >
                        <summary class="article-expander__title">Technology of genomic balancing of chromatin of autologous hematopoietic stem cells for gene therapy of fatal immune-mediated diseases of civilization, extended life expectancy and sudden human death prevention</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37833013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231027204803&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A biotechnology for personalized ex vivo gene therapy based on molecular genomic balancing of hematopoietic stem cell (HSC) chromatin with nucleosome monomers of human genomic DNA (hDNA^(nmr)) has been developed and implemented in the clinic to change (to &quot;correct&quot;) mutant chromosome loci genomes of dominant HSC clones that form mono- and oligoclonal hematopoiesis during aging and major (oncological, cardiovascular, neurodegenerative and autoimmune) fatal immune-mediated diseases of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37832948"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis, Inflammation, and Hematologic Malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37832948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231027204803&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic or acquired mutations are postzygotic genetic variations that can occur within any tissue. These mutations accumulate during aging and have classically been linked to malignant processes. Tremendous advancements over the past years have led to a deeper understanding of the role of somatic mutations in benign and malignant age-related diseases. Here, we review the somatic mutations that accumulate in the blood and their connection to disease states, with a particular focus on inflammatory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37833013"
                        open
                      >
                        <summary class="article-expander__title">Technology of genomic balancing of chromatin of autologous hematopoietic stem cells for gene therapy of fatal immune-mediated diseases of civilization, extended life expectancy and sudden human death prevention</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37833013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A biotechnology for personalized ex vivo gene therapy based on molecular genomic balancing of hematopoietic stem cell (HSC) chromatin with nucleosome monomers of human genomic DNA (hDNA^(nmr)) has been developed and implemented in the clinic to change (to &quot;correct&quot;) mutant chromosome loci genomes of dominant HSC clones that form mono- and oligoclonal hematopoiesis during aging and major (oncological, cardiovascular, neurodegenerative and autoimmune) fatal immune-mediated diseases of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37832948"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis, Inflammation, and Hematologic Malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37832948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic or acquired mutations are postzygotic genetic variations that can occur within any tissue. These mutations accumulate during aging and have classically been linked to malignant processes. Tremendous advancements over the past years have led to a deeper understanding of the role of somatic mutations in benign and malignant age-related diseases. Here, we review the somatic mutations that accumulate in the blood and their connection to disease states, with a particular focus on inflammatory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37833013"
                        open
                      >
                        <summary class="article-expander__title">Technology of genomic balancing of chromatin of autologous hematopoietic stem cells for gene therapy of fatal immune-mediated diseases of civilization, extended life expectancy and sudden human death prevention</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37833013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A biotechnology for personalized ex vivo gene therapy based on molecular genomic balancing of hematopoietic stem cell (HSC) chromatin with nucleosome monomers of human genomic DNA (hDNA^(nmr)) has been developed and implemented in the clinic to change (to &quot;correct&quot;) mutant chromosome loci genomes of dominant HSC clones that form mono- and oligoclonal hematopoiesis during aging and major (oncological, cardiovascular, neurodegenerative and autoimmune) fatal immune-mediated diseases of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37832948"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis, Inflammation, and Hematologic Malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37832948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic or acquired mutations are postzygotic genetic variations that can occur within any tissue. These mutations accumulate during aging and have classically been linked to malignant processes. Tremendous advancements over the past years have led to a deeper understanding of the role of somatic mutations in benign and malignant age-related diseases. Here, we review the somatic mutations that accumulate in the blood and their connection to disease states, with a particular focus on inflammatory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37833013"
                        open
                      >
                        <summary class="article-expander__title">Technology of genomic balancing of chromatin of autologous hematopoietic stem cells for gene therapy of fatal immune-mediated diseases of civilization, extended life expectancy and sudden human death prevention</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37833013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A biotechnology for personalized ex vivo gene therapy based on molecular genomic balancing of hematopoietic stem cell (HSC) chromatin with nucleosome monomers of human genomic DNA (hDNA^(nmr)) has been developed and implemented in the clinic to change (to &quot;correct&quot;) mutant chromosome loci genomes of dominant HSC clones that form mono- and oligoclonal hematopoiesis during aging and major (oncological, cardiovascular, neurodegenerative and autoimmune) fatal immune-mediated diseases of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37832948"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis, Inflammation, and Hematologic Malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37832948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic or acquired mutations are postzygotic genetic variations that can occur within any tissue. These mutations accumulate during aging and have classically been linked to malignant processes. Tremendous advancements over the past years have led to a deeper understanding of the role of somatic mutations in benign and malignant age-related diseases. Here, we review the somatic mutations that accumulate in the blood and their connection to disease states, with a particular focus on inflammatory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37833013"
                        open
                      >
                        <summary class="article-expander__title">Technology of genomic balancing of chromatin of autologous hematopoietic stem cells for gene therapy of fatal immune-mediated diseases of civilization, extended life expectancy and sudden human death prevention</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37833013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231023204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A biotechnology for personalized ex vivo gene therapy based on molecular genomic balancing of hematopoietic stem cell (HSC) chromatin with nucleosome monomers of human genomic DNA (hDNA^(nmr)) has been developed and implemented in the clinic to change (to &quot;correct&quot;) mutant chromosome loci genomes of dominant HSC clones that form mono- and oligoclonal hematopoiesis during aging and major (oncological, cardiovascular, neurodegenerative and autoimmune) fatal immune-mediated diseases of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37832948"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis, Inflammation, and Hematologic Malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37832948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231023204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic or acquired mutations are postzygotic genetic variations that can occur within any tissue. These mutations accumulate during aging and have classically been linked to malignant processes. Tremendous advancements over the past years have led to a deeper understanding of the role of somatic mutations in benign and malignant age-related diseases. Here, we review the somatic mutations that accumulate in the blood and their connection to disease states, with a particular focus on inflammatory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37833013"
                        open
                      >
                        <summary class="article-expander__title">Technology of genomic balancing of chromatin of autologous hematopoietic stem cells for gene therapy of fatal immune-mediated diseases of civilization, extended life expectancy and sudden human death prevention</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37833013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231022205027&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A biotechnology for personalized ex vivo gene therapy based on molecular genomic balancing of hematopoietic stem cell (HSC) chromatin with nucleosome monomers of human genomic DNA (hDNA^(nmr)) has been developed and implemented in the clinic to change (to &quot;correct&quot;) mutant chromosome loci genomes of dominant HSC clones that form mono- and oligoclonal hematopoiesis during aging and major (oncological, cardiovascular, neurodegenerative and autoimmune) fatal immune-mediated diseases of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37832948"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis, Inflammation, and Hematologic Malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37832948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231022205027&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic or acquired mutations are postzygotic genetic variations that can occur within any tissue. These mutations accumulate during aging and have classically been linked to malignant processes. Tremendous advancements over the past years have led to a deeper understanding of the role of somatic mutations in benign and malignant age-related diseases. Here, we review the somatic mutations that accumulate in the blood and their connection to disease states, with a particular focus on inflammatory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37833013"
                        open
                      >
                        <summary class="article-expander__title">Technology of genomic balancing of chromatin of autologous hematopoietic stem cells for gene therapy of fatal immune-mediated diseases of civilization, extended life expectancy and sudden human death prevention</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37833013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231021205333&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A biotechnology for personalized ex vivo gene therapy based on molecular genomic balancing of hematopoietic stem cell (HSC) chromatin with nucleosome monomers of human genomic DNA (hDNA^(nmr)) has been developed and implemented in the clinic to change (to &quot;correct&quot;) mutant chromosome loci genomes of dominant HSC clones that form mono- and oligoclonal hematopoiesis during aging and major (oncological, cardiovascular, neurodegenerative and autoimmune) fatal immune-mediated diseases of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37832948"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis, Inflammation, and Hematologic Malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37832948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231021205333&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic or acquired mutations are postzygotic genetic variations that can occur within any tissue. These mutations accumulate during aging and have classically been linked to malignant processes. Tremendous advancements over the past years have led to a deeper understanding of the role of somatic mutations in benign and malignant age-related diseases. Here, we review the somatic mutations that accumulate in the blood and their connection to disease states, with a particular focus on inflammatory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37833013"
                        open
                      >
                        <summary class="article-expander__title">Technology of genomic balancing of chromatin of autologous hematopoietic stem cells for gene therapy of fatal immune-mediated diseases of civilization, extended life expectancy and sudden human death prevention</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37833013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231020204820&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A biotechnology for personalized ex vivo gene therapy based on molecular genomic balancing of hematopoietic stem cell (HSC) chromatin with nucleosome monomers of human genomic DNA (hDNA^(nmr)) has been developed and implemented in the clinic to change (to &quot;correct&quot;) mutant chromosome loci genomes of dominant HSC clones that form mono- and oligoclonal hematopoiesis during aging and major (oncological, cardiovascular, neurodegenerative and autoimmune) fatal immune-mediated diseases of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37832948"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis, Inflammation, and Hematologic Malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37832948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231020204820&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic or acquired mutations are postzygotic genetic variations that can occur within any tissue. These mutations accumulate during aging and have classically been linked to malignant processes. Tremendous advancements over the past years have led to a deeper understanding of the role of somatic mutations in benign and malignant age-related diseases. Here, we review the somatic mutations that accumulate in the blood and their connection to disease states, with a particular focus on inflammatory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37833013"
                        open
                      >
                        <summary class="article-expander__title">Technology of genomic balancing of chromatin of autologous hematopoietic stem cells for gene therapy of fatal immune-mediated diseases of civilization, extended life expectancy and sudden human death prevention</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37833013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231019204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A biotechnology for personalized ex vivo gene therapy based on molecular genomic balancing of hematopoietic stem cell (HSC) chromatin with nucleosome monomers of human genomic DNA (hDNA^(nmr)) has been developed and implemented in the clinic to change (to &quot;correct&quot;) mutant chromosome loci genomes of dominant HSC clones that form mono- and oligoclonal hematopoiesis during aging and major (oncological, cardiovascular, neurodegenerative and autoimmune) fatal immune-mediated diseases of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37832948"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis, Inflammation, and Hematologic Malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37832948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231019204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic or acquired mutations are postzygotic genetic variations that can occur within any tissue. These mutations accumulate during aging and have classically been linked to malignant processes. Tremendous advancements over the past years have led to a deeper understanding of the role of somatic mutations in benign and malignant age-related diseases. Here, we review the somatic mutations that accumulate in the blood and their connection to disease states, with a particular focus on inflammatory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37833013"
                        open
                      >
                        <summary class="article-expander__title">Technology of genomic balancing of chromatin of autologous hematopoietic stem cells for gene therapy of fatal immune-mediated diseases of civilization, extended life expectancy and sudden human death prevention</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37833013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231018205013&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A biotechnology for personalized ex vivo gene therapy based on molecular genomic balancing of hematopoietic stem cell (HSC) chromatin with nucleosome monomers of human genomic DNA (hDNA^(nmr)) has been developed and implemented in the clinic to change (to &quot;correct&quot;) mutant chromosome loci genomes of dominant HSC clones that form mono- and oligoclonal hematopoiesis during aging and major (oncological, cardiovascular, neurodegenerative and autoimmune) fatal immune-mediated diseases of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37832948"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis, Inflammation, and Hematologic Malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37832948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231018205013&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic or acquired mutations are postzygotic genetic variations that can occur within any tissue. These mutations accumulate during aging and have classically been linked to malignant processes. Tremendous advancements over the past years have led to a deeper understanding of the role of somatic mutations in benign and malignant age-related diseases. Here, we review the somatic mutations that accumulate in the blood and their connection to disease states, with a particular focus on inflammatory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37833013"
                        open
                      >
                        <summary class="article-expander__title">Technology of genomic balancing of chromatin of autologous hematopoietic stem cells for gene therapy of fatal immune-mediated diseases of civilization, extended life expectancy and sudden human death prevention</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37833013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231017205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A biotechnology for personalized ex vivo gene therapy based on molecular genomic balancing of hematopoietic stem cell (HSC) chromatin with nucleosome monomers of human genomic DNA (hDNA^(nmr)) has been developed and implemented in the clinic to change (to &quot;correct&quot;) mutant chromosome loci genomes of dominant HSC clones that form mono- and oligoclonal hematopoiesis during aging and major (oncological, cardiovascular, neurodegenerative and autoimmune) fatal immune-mediated diseases of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37832948"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis, Inflammation, and Hematologic Malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37832948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231017205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic or acquired mutations are postzygotic genetic variations that can occur within any tissue. These mutations accumulate during aging and have classically been linked to malignant processes. Tremendous advancements over the past years have led to a deeper understanding of the role of somatic mutations in benign and malignant age-related diseases. Here, we review the somatic mutations that accumulate in the blood and their connection to disease states, with a particular focus on inflammatory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37833013"
                        open
                      >
                        <summary class="article-expander__title">Technology of genomic balancing of chromatin of autologous hematopoietic stem cells for gene therapy of fatal immune-mediated diseases of civilization, extended life expectancy and sudden human death prevention</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37833013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231016205028&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A biotechnology for personalized ex vivo gene therapy based on molecular genomic balancing of hematopoietic stem cell (HSC) chromatin with nucleosome monomers of human genomic DNA (hDNA^(nmr)) has been developed and implemented in the clinic to change (to &quot;correct&quot;) mutant chromosome loci genomes of dominant HSC clones that form mono- and oligoclonal hematopoiesis during aging and major (oncological, cardiovascular, neurodegenerative and autoimmune) fatal immune-mediated diseases of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37832948"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis, Inflammation, and Hematologic Malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37832948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231016205028&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic or acquired mutations are postzygotic genetic variations that can occur within any tissue. These mutations accumulate during aging and have classically been linked to malignant processes. Tremendous advancements over the past years have led to a deeper understanding of the role of somatic mutations in benign and malignant age-related diseases. Here, we review the somatic mutations that accumulate in the blood and their connection to disease states, with a particular focus on inflammatory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37833013"
                        open
                      >
                        <summary class="article-expander__title">Technology of genomic balancing of chromatin of autologous hematopoietic stem cells for gene therapy of fatal immune-mediated diseases of civilization, extended life expectancy and sudden human death prevention</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37833013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231015205105&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A biotechnology for personalized ex vivo gene therapy based on molecular genomic balancing of hematopoietic stem cell (HSC) chromatin with nucleosome monomers of human genomic DNA (hDNA^(nmr)) has been developed and implemented in the clinic to change (to &quot;correct&quot;) mutant chromosome loci genomes of dominant HSC clones that form mono- and oligoclonal hematopoiesis during aging and major (oncological, cardiovascular, neurodegenerative and autoimmune) fatal immune-mediated diseases of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37832948"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis, Inflammation, and Hematologic Malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37832948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231015205105&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic or acquired mutations are postzygotic genetic variations that can occur within any tissue. These mutations accumulate during aging and have classically been linked to malignant processes. Tremendous advancements over the past years have led to a deeper understanding of the role of somatic mutations in benign and malignant age-related diseases. Here, we review the somatic mutations that accumulate in the blood and their connection to disease states, with a particular focus on inflammatory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37833013"
                        open
                      >
                        <summary class="article-expander__title">Technology of genomic balancing of chromatin of autologous hematopoietic stem cells for gene therapy of fatal immune-mediated diseases of civilization, extended life expectancy and sudden human death prevention</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37833013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231014205339&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A biotechnology for personalized ex vivo gene therapy based on molecular genomic balancing of hematopoietic stem cell (HSC) chromatin with nucleosome monomers of human genomic DNA (hDNA^(nmr)) has been developed and implemented in the clinic to change (to &quot;correct&quot;) mutant chromosome loci genomes of dominant HSC clones that form mono- and oligoclonal hematopoiesis during aging and major (oncological, cardiovascular, neurodegenerative and autoimmune) fatal immune-mediated diseases of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37832948"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis, Inflammation, and Hematologic Malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37832948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231014205339&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic or acquired mutations are postzygotic genetic variations that can occur within any tissue. These mutations accumulate during aging and have classically been linked to malignant processes. Tremendous advancements over the past years have led to a deeper understanding of the role of somatic mutations in benign and malignant age-related diseases. Here, we review the somatic mutations that accumulate in the blood and their connection to disease states, with a particular focus on inflammatory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-10-12 datetime="2023-10-12T10:00:00.000Z">2023-10-12</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-10-12 datetime="2023-10-12T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20231105195156">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37827894"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in the application of induced pluripotent stem cell technology to the study of myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37827894/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired myeloid malignancies are a spectrum of clonal disorders known to be caused by sequential acquisition of genetic lesions in hematopoietic stem and progenitor cells, leading to their aberrant self-renewal and differentiation. The increasing use of induced pluripotent stem cell (iPSC) technology to study myeloid malignancies has helped usher a paradigm shift in approaches to disease modeling and drug discovery, especially when combined with gene-editing technology. The process of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37827894"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in the application of induced pluripotent stem cell technology to the study of myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37827894/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired myeloid malignancies are a spectrum of clonal disorders known to be caused by sequential acquisition of genetic lesions in hematopoietic stem and progenitor cells, leading to their aberrant self-renewal and differentiation. The increasing use of induced pluripotent stem cell (iPSC) technology to study myeloid malignancies has helped usher a paradigm shift in approaches to disease modeling and drug discovery, especially when combined with gene-editing technology. The process of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37827894"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in the application of induced pluripotent stem cell technology to the study of myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37827894/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired myeloid malignancies are a spectrum of clonal disorders known to be caused by sequential acquisition of genetic lesions in hematopoietic stem and progenitor cells, leading to their aberrant self-renewal and differentiation. The increasing use of induced pluripotent stem cell (iPSC) technology to study myeloid malignancies has helped usher a paradigm shift in approaches to disease modeling and drug discovery, especially when combined with gene-editing technology. The process of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37827894"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in the application of induced pluripotent stem cell technology to the study of myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37827894/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231027204803&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired myeloid malignancies are a spectrum of clonal disorders known to be caused by sequential acquisition of genetic lesions in hematopoietic stem and progenitor cells, leading to their aberrant self-renewal and differentiation. The increasing use of induced pluripotent stem cell (iPSC) technology to study myeloid malignancies has helped usher a paradigm shift in approaches to disease modeling and drug discovery, especially when combined with gene-editing technology. The process of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37827894"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in the application of induced pluripotent stem cell technology to the study of myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37827894/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired myeloid malignancies are a spectrum of clonal disorders known to be caused by sequential acquisition of genetic lesions in hematopoietic stem and progenitor cells, leading to their aberrant self-renewal and differentiation. The increasing use of induced pluripotent stem cell (iPSC) technology to study myeloid malignancies has helped usher a paradigm shift in approaches to disease modeling and drug discovery, especially when combined with gene-editing technology. The process of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37827894"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in the application of induced pluripotent stem cell technology to the study of myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37827894/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired myeloid malignancies are a spectrum of clonal disorders known to be caused by sequential acquisition of genetic lesions in hematopoietic stem and progenitor cells, leading to their aberrant self-renewal and differentiation. The increasing use of induced pluripotent stem cell (iPSC) technology to study myeloid malignancies has helped usher a paradigm shift in approaches to disease modeling and drug discovery, especially when combined with gene-editing technology. The process of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37827894"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in the application of induced pluripotent stem cell technology to the study of myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37827894/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired myeloid malignancies are a spectrum of clonal disorders known to be caused by sequential acquisition of genetic lesions in hematopoietic stem and progenitor cells, leading to their aberrant self-renewal and differentiation. The increasing use of induced pluripotent stem cell (iPSC) technology to study myeloid malignancies has helped usher a paradigm shift in approaches to disease modeling and drug discovery, especially when combined with gene-editing technology. The process of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37827894"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in the application of induced pluripotent stem cell technology to the study of myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37827894/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231023204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired myeloid malignancies are a spectrum of clonal disorders known to be caused by sequential acquisition of genetic lesions in hematopoietic stem and progenitor cells, leading to their aberrant self-renewal and differentiation. The increasing use of induced pluripotent stem cell (iPSC) technology to study myeloid malignancies has helped usher a paradigm shift in approaches to disease modeling and drug discovery, especially when combined with gene-editing technology. The process of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37827894"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in the application of induced pluripotent stem cell technology to the study of myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37827894/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231022205027&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired myeloid malignancies are a spectrum of clonal disorders known to be caused by sequential acquisition of genetic lesions in hematopoietic stem and progenitor cells, leading to their aberrant self-renewal and differentiation. The increasing use of induced pluripotent stem cell (iPSC) technology to study myeloid malignancies has helped usher a paradigm shift in approaches to disease modeling and drug discovery, especially when combined with gene-editing technology. The process of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37827894"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in the application of induced pluripotent stem cell technology to the study of myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37827894/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231021205333&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired myeloid malignancies are a spectrum of clonal disorders known to be caused by sequential acquisition of genetic lesions in hematopoietic stem and progenitor cells, leading to their aberrant self-renewal and differentiation. The increasing use of induced pluripotent stem cell (iPSC) technology to study myeloid malignancies has helped usher a paradigm shift in approaches to disease modeling and drug discovery, especially when combined with gene-editing technology. The process of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37827894"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in the application of induced pluripotent stem cell technology to the study of myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37827894/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231020204820&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired myeloid malignancies are a spectrum of clonal disorders known to be caused by sequential acquisition of genetic lesions in hematopoietic stem and progenitor cells, leading to their aberrant self-renewal and differentiation. The increasing use of induced pluripotent stem cell (iPSC) technology to study myeloid malignancies has helped usher a paradigm shift in approaches to disease modeling and drug discovery, especially when combined with gene-editing technology. The process of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37827894"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in the application of induced pluripotent stem cell technology to the study of myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37827894/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231019204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired myeloid malignancies are a spectrum of clonal disorders known to be caused by sequential acquisition of genetic lesions in hematopoietic stem and progenitor cells, leading to their aberrant self-renewal and differentiation. The increasing use of induced pluripotent stem cell (iPSC) technology to study myeloid malignancies has helped usher a paradigm shift in approaches to disease modeling and drug discovery, especially when combined with gene-editing technology. The process of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37827894"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in the application of induced pluripotent stem cell technology to the study of myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37827894/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231018205013&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired myeloid malignancies are a spectrum of clonal disorders known to be caused by sequential acquisition of genetic lesions in hematopoietic stem and progenitor cells, leading to their aberrant self-renewal and differentiation. The increasing use of induced pluripotent stem cell (iPSC) technology to study myeloid malignancies has helped usher a paradigm shift in approaches to disease modeling and drug discovery, especially when combined with gene-editing technology. The process of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37827894"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in the application of induced pluripotent stem cell technology to the study of myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37827894/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231017205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired myeloid malignancies are a spectrum of clonal disorders known to be caused by sequential acquisition of genetic lesions in hematopoietic stem and progenitor cells, leading to their aberrant self-renewal and differentiation. The increasing use of induced pluripotent stem cell (iPSC) technology to study myeloid malignancies has helped usher a paradigm shift in approaches to disease modeling and drug discovery, especially when combined with gene-editing technology. The process of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37827894"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in the application of induced pluripotent stem cell technology to the study of myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37827894/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231016205028&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired myeloid malignancies are a spectrum of clonal disorders known to be caused by sequential acquisition of genetic lesions in hematopoietic stem and progenitor cells, leading to their aberrant self-renewal and differentiation. The increasing use of induced pluripotent stem cell (iPSC) technology to study myeloid malignancies has helped usher a paradigm shift in approaches to disease modeling and drug discovery, especially when combined with gene-editing technology. The process of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37827894"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in the application of induced pluripotent stem cell technology to the study of myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37827894/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231015205105&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired myeloid malignancies are a spectrum of clonal disorders known to be caused by sequential acquisition of genetic lesions in hematopoietic stem and progenitor cells, leading to their aberrant self-renewal and differentiation. The increasing use of induced pluripotent stem cell (iPSC) technology to study myeloid malignancies has helped usher a paradigm shift in approaches to disease modeling and drug discovery, especially when combined with gene-editing technology. The process of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37827894"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in the application of induced pluripotent stem cell technology to the study of myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37827894/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231014205339&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired myeloid malignancies are a spectrum of clonal disorders known to be caused by sequential acquisition of genetic lesions in hematopoietic stem and progenitor cells, leading to their aberrant self-renewal and differentiation. The increasing use of induced pluripotent stem cell (iPSC) technology to study myeloid malignancies has helped usher a paradigm shift in approaches to disease modeling and drug discovery, especially when combined with gene-editing technology. The process of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37827894"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in the application of induced pluripotent stem cell technology to the study of myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37827894/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231013204816&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired myeloid malignancies are a spectrum of clonal disorders known to be caused by sequential acquisition of genetic lesions in hematopoietic stem and progenitor cells, leading to their aberrant self-renewal and differentiation. The increasing use of induced pluripotent stem cell (iPSC) technology to study myeloid malignancies has helped usher a paradigm shift in approaches to disease modeling and drug discovery, especially when combined with gene-editing technology. The process of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;ff&#x3D;20231105195156&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37823726"
                        open
                      >
                        <summary class="article-expander__title">Targeted single-cell RNA sequencing analysis reveals metabolic reprogramming and the ferroptosis-resistant state in hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37823726/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematologic malignancies are the most common hematopoietic diseases and a major public health concern. However, the mechanisms underlying myeloid tumors remain unknown owing to the intricate interplay between mutations and diverse clonal evolution patterns, as evidenced by the analysis of bulk cell-derived omics data. Several single-cell omics techniques have been used to characterize the hierarchies and altered immune microenvironments of hematologic malignancies. The comprehensive single-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37823726"
                        open
                      >
                        <summary class="article-expander__title">Targeted single-cell RNA sequencing analysis reveals metabolic reprogramming and the ferroptosis-resistant state in hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37823726/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematologic malignancies are the most common hematopoietic diseases and a major public health concern. However, the mechanisms underlying myeloid tumors remain unknown owing to the intricate interplay between mutations and diverse clonal evolution patterns, as evidenced by the analysis of bulk cell-derived omics data. Several single-cell omics techniques have been used to characterize the hierarchies and altered immune microenvironments of hematologic malignancies. The comprehensive single-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37823726"
                        open
                      >
                        <summary class="article-expander__title">Targeted single-cell RNA sequencing analysis reveals metabolic reprogramming and the ferroptosis-resistant state in hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37823726/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231027204802&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematologic malignancies are the most common hematopoietic diseases and a major public health concern. However, the mechanisms underlying myeloid tumors remain unknown owing to the intricate interplay between mutations and diverse clonal evolution patterns, as evidenced by the analysis of bulk cell-derived omics data. Several single-cell omics techniques have been used to characterize the hierarchies and altered immune microenvironments of hematologic malignancies. The comprehensive single-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37823726"
                        open
                      >
                        <summary class="article-expander__title">Targeted single-cell RNA sequencing analysis reveals metabolic reprogramming and the ferroptosis-resistant state in hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37823726/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematologic malignancies are the most common hematopoietic diseases and a major public health concern. However, the mechanisms underlying myeloid tumors remain unknown owing to the intricate interplay between mutations and diverse clonal evolution patterns, as evidenced by the analysis of bulk cell-derived omics data. Several single-cell omics techniques have been used to characterize the hierarchies and altered immune microenvironments of hematologic malignancies. The comprehensive single-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37823726"
                        open
                      >
                        <summary class="article-expander__title">Targeted single-cell RNA sequencing analysis reveals metabolic reprogramming and the ferroptosis-resistant state in hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37823726/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematologic malignancies are the most common hematopoietic diseases and a major public health concern. However, the mechanisms underlying myeloid tumors remain unknown owing to the intricate interplay between mutations and diverse clonal evolution patterns, as evidenced by the analysis of bulk cell-derived omics data. Several single-cell omics techniques have been used to characterize the hierarchies and altered immune microenvironments of hematologic malignancies. The comprehensive single-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37823726"
                        open
                      >
                        <summary class="article-expander__title">Targeted single-cell RNA sequencing analysis reveals metabolic reprogramming and the ferroptosis-resistant state in hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37823726/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematologic malignancies are the most common hematopoietic diseases and a major public health concern. However, the mechanisms underlying myeloid tumors remain unknown owing to the intricate interplay between mutations and diverse clonal evolution patterns, as evidenced by the analysis of bulk cell-derived omics data. Several single-cell omics techniques have been used to characterize the hierarchies and altered immune microenvironments of hematologic malignancies. The comprehensive single-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37823726"
                        open
                      >
                        <summary class="article-expander__title">Targeted single-cell RNA sequencing analysis reveals metabolic reprogramming and the ferroptosis-resistant state in hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37823726/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231023204959&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematologic malignancies are the most common hematopoietic diseases and a major public health concern. However, the mechanisms underlying myeloid tumors remain unknown owing to the intricate interplay between mutations and diverse clonal evolution patterns, as evidenced by the analysis of bulk cell-derived omics data. Several single-cell omics techniques have been used to characterize the hierarchies and altered immune microenvironments of hematologic malignancies. The comprehensive single-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37823726"
                        open
                      >
                        <summary class="article-expander__title">Targeted single-cell RNA sequencing analysis reveals metabolic reprogramming and the ferroptosis-resistant state in hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37823726/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231022205027&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematologic malignancies are the most common hematopoietic diseases and a major public health concern. However, the mechanisms underlying myeloid tumors remain unknown owing to the intricate interplay between mutations and diverse clonal evolution patterns, as evidenced by the analysis of bulk cell-derived omics data. Several single-cell omics techniques have been used to characterize the hierarchies and altered immune microenvironments of hematologic malignancies. The comprehensive single-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37823726"
                        open
                      >
                        <summary class="article-expander__title">Targeted single-cell RNA sequencing analysis reveals metabolic reprogramming and the ferroptosis-resistant state in hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37823726/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231021205333&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematologic malignancies are the most common hematopoietic diseases and a major public health concern. However, the mechanisms underlying myeloid tumors remain unknown owing to the intricate interplay between mutations and diverse clonal evolution patterns, as evidenced by the analysis of bulk cell-derived omics data. Several single-cell omics techniques have been used to characterize the hierarchies and altered immune microenvironments of hematologic malignancies. The comprehensive single-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37823726"
                        open
                      >
                        <summary class="article-expander__title">Targeted single-cell RNA sequencing analysis reveals metabolic reprogramming and the ferroptosis-resistant state in hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37823726/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231020204820&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematologic malignancies are the most common hematopoietic diseases and a major public health concern. However, the mechanisms underlying myeloid tumors remain unknown owing to the intricate interplay between mutations and diverse clonal evolution patterns, as evidenced by the analysis of bulk cell-derived omics data. Several single-cell omics techniques have been used to characterize the hierarchies and altered immune microenvironments of hematologic malignancies. The comprehensive single-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37823726"
                        open
                      >
                        <summary class="article-expander__title">Targeted single-cell RNA sequencing analysis reveals metabolic reprogramming and the ferroptosis-resistant state in hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37823726/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231019204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematologic malignancies are the most common hematopoietic diseases and a major public health concern. However, the mechanisms underlying myeloid tumors remain unknown owing to the intricate interplay between mutations and diverse clonal evolution patterns, as evidenced by the analysis of bulk cell-derived omics data. Several single-cell omics techniques have been used to characterize the hierarchies and altered immune microenvironments of hematologic malignancies. The comprehensive single-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37823726"
                        open
                      >
                        <summary class="article-expander__title">Targeted single-cell RNA sequencing analysis reveals metabolic reprogramming and the ferroptosis-resistant state in hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37823726/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231018205013&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematologic malignancies are the most common hematopoietic diseases and a major public health concern. However, the mechanisms underlying myeloid tumors remain unknown owing to the intricate interplay between mutations and diverse clonal evolution patterns, as evidenced by the analysis of bulk cell-derived omics data. Several single-cell omics techniques have been used to characterize the hierarchies and altered immune microenvironments of hematologic malignancies. The comprehensive single-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37823726"
                        open
                      >
                        <summary class="article-expander__title">Targeted single-cell RNA sequencing analysis reveals metabolic reprogramming and the ferroptosis-resistant state in hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37823726/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231017205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematologic malignancies are the most common hematopoietic diseases and a major public health concern. However, the mechanisms underlying myeloid tumors remain unknown owing to the intricate interplay between mutations and diverse clonal evolution patterns, as evidenced by the analysis of bulk cell-derived omics data. Several single-cell omics techniques have been used to characterize the hierarchies and altered immune microenvironments of hematologic malignancies. The comprehensive single-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37823726"
                        open
                      >
                        <summary class="article-expander__title">Targeted single-cell RNA sequencing analysis reveals metabolic reprogramming and the ferroptosis-resistant state in hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37823726/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231016205028&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematologic malignancies are the most common hematopoietic diseases and a major public health concern. However, the mechanisms underlying myeloid tumors remain unknown owing to the intricate interplay between mutations and diverse clonal evolution patterns, as evidenced by the analysis of bulk cell-derived omics data. Several single-cell omics techniques have been used to characterize the hierarchies and altered immune microenvironments of hematologic malignancies. The comprehensive single-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37823726"
                        open
                      >
                        <summary class="article-expander__title">Targeted single-cell RNA sequencing analysis reveals metabolic reprogramming and the ferroptosis-resistant state in hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37823726/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231015205105&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematologic malignancies are the most common hematopoietic diseases and a major public health concern. However, the mechanisms underlying myeloid tumors remain unknown owing to the intricate interplay between mutations and diverse clonal evolution patterns, as evidenced by the analysis of bulk cell-derived omics data. Several single-cell omics techniques have been used to characterize the hierarchies and altered immune microenvironments of hematologic malignancies. The comprehensive single-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37823726"
                        open
                      >
                        <summary class="article-expander__title">Targeted single-cell RNA sequencing analysis reveals metabolic reprogramming and the ferroptosis-resistant state in hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37823726/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231014205339&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematologic malignancies are the most common hematopoietic diseases and a major public health concern. However, the mechanisms underlying myeloid tumors remain unknown owing to the intricate interplay between mutations and diverse clonal evolution patterns, as evidenced by the analysis of bulk cell-derived omics data. Several single-cell omics techniques have been used to characterize the hierarchies and altered immune microenvironments of hematologic malignancies. The comprehensive single-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37823726"
                        open
                      >
                        <summary class="article-expander__title">Targeted single-cell RNA sequencing analysis reveals metabolic reprogramming and the ferroptosis-resistant state in hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37823726/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231013204816&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematologic malignancies are the most common hematopoietic diseases and a major public health concern. However, the mechanisms underlying myeloid tumors remain unknown owing to the intricate interplay between mutations and diverse clonal evolution patterns, as evidenced by the analysis of bulk cell-derived omics data. Several single-cell omics techniques have been used to characterize the hierarchies and altered immune microenvironments of hematologic malignancies. The comprehensive single-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37823726"
                        open
                      >
                        <summary class="article-expander__title">Targeted single-cell RNA sequencing analysis reveals metabolic reprogramming and the ferroptosis-resistant state in hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37823726/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231012205121&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematologic malignancies are the most common hematopoietic diseases and a major public health concern. However, the mechanisms underlying myeloid tumors remain unknown owing to the intricate interplay between mutations and diverse clonal evolution patterns, as evidenced by the analysis of bulk cell-derived omics data. Several single-cell omics techniques have been used to characterize the hierarchies and altered immune microenvironments of hematologic malignancies. The comprehensive single-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-10-11 datetime="2023-10-11T10:00:00.000Z">2023-10-11</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-10-11 datetime="2023-10-11T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;ff&#x3D;20231105195156&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37819942"
                        open
                      >
                        <summary class="article-expander__title">Phertilizer: Growing a clonal tree from ultra-low coverage single-cell DNA sequencing of tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37819942/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging ultra-low coverage single-cell DNA sequencing (scDNA-seq) technologies have enabled high resolution evolutionary studies of copy number aberrations (CNAs) within tumors. While these sequencing technologies are well suited for identifying CNAs due to the uniformity of sequencing coverage, the sparsity of coverage poses challenges for the study of single-nucleotide variants (SNVs). In order to maximize the utility of increasingly available ultra-low coverage scDNA-seq data and obtain a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37819942"
                        open
                      >
                        <summary class="article-expander__title">Phertilizer: Growing a clonal tree from ultra-low coverage single-cell DNA sequencing of tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37819942/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231027204802&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging ultra-low coverage single-cell DNA sequencing (scDNA-seq) technologies have enabled high resolution evolutionary studies of copy number aberrations (CNAs) within tumors. While these sequencing technologies are well suited for identifying CNAs due to the uniformity of sequencing coverage, the sparsity of coverage poses challenges for the study of single-nucleotide variants (SNVs). In order to maximize the utility of increasingly available ultra-low coverage scDNA-seq data and obtain a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37819942"
                        open
                      >
                        <summary class="article-expander__title">Phertilizer: Growing a clonal tree from ultra-low coverage single-cell DNA sequencing of tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37819942/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging ultra-low coverage single-cell DNA sequencing (scDNA-seq) technologies have enabled high resolution evolutionary studies of copy number aberrations (CNAs) within tumors. While these sequencing technologies are well suited for identifying CNAs due to the uniformity of sequencing coverage, the sparsity of coverage poses challenges for the study of single-nucleotide variants (SNVs). In order to maximize the utility of increasingly available ultra-low coverage scDNA-seq data and obtain a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37819942"
                        open
                      >
                        <summary class="article-expander__title">Phertilizer: Growing a clonal tree from ultra-low coverage single-cell DNA sequencing of tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37819942/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging ultra-low coverage single-cell DNA sequencing (scDNA-seq) technologies have enabled high resolution evolutionary studies of copy number aberrations (CNAs) within tumors. While these sequencing technologies are well suited for identifying CNAs due to the uniformity of sequencing coverage, the sparsity of coverage poses challenges for the study of single-nucleotide variants (SNVs). In order to maximize the utility of increasingly available ultra-low coverage scDNA-seq data and obtain a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37819942"
                        open
                      >
                        <summary class="article-expander__title">Phertilizer: Growing a clonal tree from ultra-low coverage single-cell DNA sequencing of tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37819942/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging ultra-low coverage single-cell DNA sequencing (scDNA-seq) technologies have enabled high resolution evolutionary studies of copy number aberrations (CNAs) within tumors. While these sequencing technologies are well suited for identifying CNAs due to the uniformity of sequencing coverage, the sparsity of coverage poses challenges for the study of single-nucleotide variants (SNVs). In order to maximize the utility of increasingly available ultra-low coverage scDNA-seq data and obtain a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37819942"
                        open
                      >
                        <summary class="article-expander__title">Phertilizer: Growing a clonal tree from ultra-low coverage single-cell DNA sequencing of tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37819942/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231023204959&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging ultra-low coverage single-cell DNA sequencing (scDNA-seq) technologies have enabled high resolution evolutionary studies of copy number aberrations (CNAs) within tumors. While these sequencing technologies are well suited for identifying CNAs due to the uniformity of sequencing coverage, the sparsity of coverage poses challenges for the study of single-nucleotide variants (SNVs). In order to maximize the utility of increasingly available ultra-low coverage scDNA-seq data and obtain a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37819942"
                        open
                      >
                        <summary class="article-expander__title">Phertilizer: Growing a clonal tree from ultra-low coverage single-cell DNA sequencing of tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37819942/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231022205027&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging ultra-low coverage single-cell DNA sequencing (scDNA-seq) technologies have enabled high resolution evolutionary studies of copy number aberrations (CNAs) within tumors. While these sequencing technologies are well suited for identifying CNAs due to the uniformity of sequencing coverage, the sparsity of coverage poses challenges for the study of single-nucleotide variants (SNVs). In order to maximize the utility of increasingly available ultra-low coverage scDNA-seq data and obtain a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37819942"
                        open
                      >
                        <summary class="article-expander__title">Phertilizer: Growing a clonal tree from ultra-low coverage single-cell DNA sequencing of tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37819942/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231021205333&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging ultra-low coverage single-cell DNA sequencing (scDNA-seq) technologies have enabled high resolution evolutionary studies of copy number aberrations (CNAs) within tumors. While these sequencing technologies are well suited for identifying CNAs due to the uniformity of sequencing coverage, the sparsity of coverage poses challenges for the study of single-nucleotide variants (SNVs). In order to maximize the utility of increasingly available ultra-low coverage scDNA-seq data and obtain a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37819942"
                        open
                      >
                        <summary class="article-expander__title">Phertilizer: Growing a clonal tree from ultra-low coverage single-cell DNA sequencing of tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37819942/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231020204820&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging ultra-low coverage single-cell DNA sequencing (scDNA-seq) technologies have enabled high resolution evolutionary studies of copy number aberrations (CNAs) within tumors. While these sequencing technologies are well suited for identifying CNAs due to the uniformity of sequencing coverage, the sparsity of coverage poses challenges for the study of single-nucleotide variants (SNVs). In order to maximize the utility of increasingly available ultra-low coverage scDNA-seq data and obtain a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37819942"
                        open
                      >
                        <summary class="article-expander__title">Phertilizer: Growing a clonal tree from ultra-low coverage single-cell DNA sequencing of tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37819942/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231019204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging ultra-low coverage single-cell DNA sequencing (scDNA-seq) technologies have enabled high resolution evolutionary studies of copy number aberrations (CNAs) within tumors. While these sequencing technologies are well suited for identifying CNAs due to the uniformity of sequencing coverage, the sparsity of coverage poses challenges for the study of single-nucleotide variants (SNVs). In order to maximize the utility of increasingly available ultra-low coverage scDNA-seq data and obtain a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37819942"
                        open
                      >
                        <summary class="article-expander__title">Phertilizer: Growing a clonal tree from ultra-low coverage single-cell DNA sequencing of tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37819942/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231018205013&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging ultra-low coverage single-cell DNA sequencing (scDNA-seq) technologies have enabled high resolution evolutionary studies of copy number aberrations (CNAs) within tumors. While these sequencing technologies are well suited for identifying CNAs due to the uniformity of sequencing coverage, the sparsity of coverage poses challenges for the study of single-nucleotide variants (SNVs). In order to maximize the utility of increasingly available ultra-low coverage scDNA-seq data and obtain a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37819942"
                        open
                      >
                        <summary class="article-expander__title">Phertilizer: Growing a clonal tree from ultra-low coverage single-cell DNA sequencing of tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37819942/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231017205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging ultra-low coverage single-cell DNA sequencing (scDNA-seq) technologies have enabled high resolution evolutionary studies of copy number aberrations (CNAs) within tumors. While these sequencing technologies are well suited for identifying CNAs due to the uniformity of sequencing coverage, the sparsity of coverage poses challenges for the study of single-nucleotide variants (SNVs). In order to maximize the utility of increasingly available ultra-low coverage scDNA-seq data and obtain a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37819942"
                        open
                      >
                        <summary class="article-expander__title">Phertilizer: Growing a clonal tree from ultra-low coverage single-cell DNA sequencing of tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37819942/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231016205028&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging ultra-low coverage single-cell DNA sequencing (scDNA-seq) technologies have enabled high resolution evolutionary studies of copy number aberrations (CNAs) within tumors. While these sequencing technologies are well suited for identifying CNAs due to the uniformity of sequencing coverage, the sparsity of coverage poses challenges for the study of single-nucleotide variants (SNVs). In order to maximize the utility of increasingly available ultra-low coverage scDNA-seq data and obtain a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37819942"
                        open
                      >
                        <summary class="article-expander__title">Phertilizer: Growing a clonal tree from ultra-low coverage single-cell DNA sequencing of tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37819942/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231015205105&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging ultra-low coverage single-cell DNA sequencing (scDNA-seq) technologies have enabled high resolution evolutionary studies of copy number aberrations (CNAs) within tumors. While these sequencing technologies are well suited for identifying CNAs due to the uniformity of sequencing coverage, the sparsity of coverage poses challenges for the study of single-nucleotide variants (SNVs). In order to maximize the utility of increasingly available ultra-low coverage scDNA-seq data and obtain a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37819942"
                        open
                      >
                        <summary class="article-expander__title">Phertilizer: Growing a clonal tree from ultra-low coverage single-cell DNA sequencing of tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37819942/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231014205339&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging ultra-low coverage single-cell DNA sequencing (scDNA-seq) technologies have enabled high resolution evolutionary studies of copy number aberrations (CNAs) within tumors. While these sequencing technologies are well suited for identifying CNAs due to the uniformity of sequencing coverage, the sparsity of coverage poses challenges for the study of single-nucleotide variants (SNVs). In order to maximize the utility of increasingly available ultra-low coverage scDNA-seq data and obtain a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37819942"
                        open
                      >
                        <summary class="article-expander__title">Phertilizer: Growing a clonal tree from ultra-low coverage single-cell DNA sequencing of tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37819942/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231013204816&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging ultra-low coverage single-cell DNA sequencing (scDNA-seq) technologies have enabled high resolution evolutionary studies of copy number aberrations (CNAs) within tumors. While these sequencing technologies are well suited for identifying CNAs due to the uniformity of sequencing coverage, the sparsity of coverage poses challenges for the study of single-nucleotide variants (SNVs). In order to maximize the utility of increasingly available ultra-low coverage scDNA-seq data and obtain a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37819942"
                        open
                      >
                        <summary class="article-expander__title">Phertilizer: Growing a clonal tree from ultra-low coverage single-cell DNA sequencing of tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37819942/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231012205121&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging ultra-low coverage single-cell DNA sequencing (scDNA-seq) technologies have enabled high resolution evolutionary studies of copy number aberrations (CNAs) within tumors. While these sequencing technologies are well suited for identifying CNAs due to the uniformity of sequencing coverage, the sparsity of coverage poses challenges for the study of single-nucleotide variants (SNVs). In order to maximize the utility of increasingly available ultra-low coverage scDNA-seq data and obtain a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37819942"
                        open
                      >
                        <summary class="article-expander__title">Phertilizer: Growing a clonal tree from ultra-low coverage single-cell DNA sequencing of tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37819942/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231011204744&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging ultra-low coverage single-cell DNA sequencing (scDNA-seq) technologies have enabled high resolution evolutionary studies of copy number aberrations (CNAs) within tumors. While these sequencing technologies are well suited for identifying CNAs due to the uniformity of sequencing coverage, the sparsity of coverage poses challenges for the study of single-nucleotide variants (SNVs). In order to maximize the utility of increasingly available ultra-low coverage scDNA-seq data and obtain a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-10-10 datetime="2023-10-10T10:00:00.000Z">2023-10-10</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-10-10 datetime="2023-10-10T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20231105195156">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37816954"
                        open
                      >
                        <summary class="article-expander__title">STING activation in TET2-mutated hematopoietic stem/progenitor cells contributes to the increased self-renewal and neoplastic transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37816954/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic loss-of-function mutations of the dioxygenase Ten-eleven translocation-2 (TET2) occur frequently in individuals with clonal hematopoiesis (CH) and acute myeloid leukemia (AML). These common hematopoietic disorders can be recapitulated in mouse models. However, the underlying mechanisms by which the deficiency in TET2 promotes these disorders remain unclear. Here we show that the cyclic guanosine monophosphate-adenosine monophosphate synthase (cGAS)-stimulator of interferon genes (STING)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37816954"
                        open
                      >
                        <summary class="article-expander__title">STING activation in TET2-mutated hematopoietic stem/progenitor cells contributes to the increased self-renewal and neoplastic transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37816954/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic loss-of-function mutations of the dioxygenase Ten-eleven translocation-2 (TET2) occur frequently in individuals with clonal hematopoiesis (CH) and acute myeloid leukemia (AML). These common hematopoietic disorders can be recapitulated in mouse models. However, the underlying mechanisms by which the deficiency in TET2 promotes these disorders remain unclear. Here we show that the cyclic guanosine monophosphate-adenosine monophosphate synthase (cGAS)-stimulator of interferon genes (STING)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37816954"
                        open
                      >
                        <summary class="article-expander__title">STING activation in TET2-mutated hematopoietic stem/progenitor cells contributes to the increased self-renewal and neoplastic transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37816954/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic loss-of-function mutations of the dioxygenase Ten-eleven translocation-2 (TET2) occur frequently in individuals with clonal hematopoiesis (CH) and acute myeloid leukemia (AML). These common hematopoietic disorders can be recapitulated in mouse models. However, the underlying mechanisms by which the deficiency in TET2 promotes these disorders remain unclear. Here we show that the cyclic guanosine monophosphate-adenosine monophosphate synthase (cGAS)-stimulator of interferon genes (STING)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37816954"
                        open
                      >
                        <summary class="article-expander__title">STING activation in TET2-mutated hematopoietic stem/progenitor cells contributes to the increased self-renewal and neoplastic transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37816954/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231027204803&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic loss-of-function mutations of the dioxygenase Ten-eleven translocation-2 (TET2) occur frequently in individuals with clonal hematopoiesis (CH) and acute myeloid leukemia (AML). These common hematopoietic disorders can be recapitulated in mouse models. However, the underlying mechanisms by which the deficiency in TET2 promotes these disorders remain unclear. Here we show that the cyclic guanosine monophosphate-adenosine monophosphate synthase (cGAS)-stimulator of interferon genes (STING)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37816954"
                        open
                      >
                        <summary class="article-expander__title">STING activation in TET2-mutated hematopoietic stem/progenitor cells contributes to the increased self-renewal and neoplastic transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37816954/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic loss-of-function mutations of the dioxygenase Ten-eleven translocation-2 (TET2) occur frequently in individuals with clonal hematopoiesis (CH) and acute myeloid leukemia (AML). These common hematopoietic disorders can be recapitulated in mouse models. However, the underlying mechanisms by which the deficiency in TET2 promotes these disorders remain unclear. Here we show that the cyclic guanosine monophosphate-adenosine monophosphate synthase (cGAS)-stimulator of interferon genes (STING)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37816954"
                        open
                      >
                        <summary class="article-expander__title">STING activation in TET2-mutated hematopoietic stem/progenitor cells contributes to the increased self-renewal and neoplastic transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37816954/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic loss-of-function mutations of the dioxygenase Ten-eleven translocation-2 (TET2) occur frequently in individuals with clonal hematopoiesis (CH) and acute myeloid leukemia (AML). These common hematopoietic disorders can be recapitulated in mouse models. However, the underlying mechanisms by which the deficiency in TET2 promotes these disorders remain unclear. Here we show that the cyclic guanosine monophosphate-adenosine monophosphate synthase (cGAS)-stimulator of interferon genes (STING)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37816954"
                        open
                      >
                        <summary class="article-expander__title">STING activation in TET2-mutated hematopoietic stem/progenitor cells contributes to the increased self-renewal and neoplastic transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37816954/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic loss-of-function mutations of the dioxygenase Ten-eleven translocation-2 (TET2) occur frequently in individuals with clonal hematopoiesis (CH) and acute myeloid leukemia (AML). These common hematopoietic disorders can be recapitulated in mouse models. However, the underlying mechanisms by which the deficiency in TET2 promotes these disorders remain unclear. Here we show that the cyclic guanosine monophosphate-adenosine monophosphate synthase (cGAS)-stimulator of interferon genes (STING)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37816954"
                        open
                      >
                        <summary class="article-expander__title">STING activation in TET2-mutated hematopoietic stem/progenitor cells contributes to the increased self-renewal and neoplastic transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37816954/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231023204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic loss-of-function mutations of the dioxygenase Ten-eleven translocation-2 (TET2) occur frequently in individuals with clonal hematopoiesis (CH) and acute myeloid leukemia (AML). These common hematopoietic disorders can be recapitulated in mouse models. However, the underlying mechanisms by which the deficiency in TET2 promotes these disorders remain unclear. Here we show that the cyclic guanosine monophosphate-adenosine monophosphate synthase (cGAS)-stimulator of interferon genes (STING)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37816954"
                        open
                      >
                        <summary class="article-expander__title">STING activation in TET2-mutated hematopoietic stem/progenitor cells contributes to the increased self-renewal and neoplastic transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37816954/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231022205027&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic loss-of-function mutations of the dioxygenase Ten-eleven translocation-2 (TET2) occur frequently in individuals with clonal hematopoiesis (CH) and acute myeloid leukemia (AML). These common hematopoietic disorders can be recapitulated in mouse models. However, the underlying mechanisms by which the deficiency in TET2 promotes these disorders remain unclear. Here we show that the cyclic guanosine monophosphate-adenosine monophosphate synthase (cGAS)-stimulator of interferon genes (STING)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37816954"
                        open
                      >
                        <summary class="article-expander__title">STING activation in TET2-mutated hematopoietic stem/progenitor cells contributes to the increased self-renewal and neoplastic transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37816954/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231021205333&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic loss-of-function mutations of the dioxygenase Ten-eleven translocation-2 (TET2) occur frequently in individuals with clonal hematopoiesis (CH) and acute myeloid leukemia (AML). These common hematopoietic disorders can be recapitulated in mouse models. However, the underlying mechanisms by which the deficiency in TET2 promotes these disorders remain unclear. Here we show that the cyclic guanosine monophosphate-adenosine monophosphate synthase (cGAS)-stimulator of interferon genes (STING)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37816954"
                        open
                      >
                        <summary class="article-expander__title">STING activation in TET2-mutated hematopoietic stem/progenitor cells contributes to the increased self-renewal and neoplastic transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37816954/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231020204820&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic loss-of-function mutations of the dioxygenase Ten-eleven translocation-2 (TET2) occur frequently in individuals with clonal hematopoiesis (CH) and acute myeloid leukemia (AML). These common hematopoietic disorders can be recapitulated in mouse models. However, the underlying mechanisms by which the deficiency in TET2 promotes these disorders remain unclear. Here we show that the cyclic guanosine monophosphate-adenosine monophosphate synthase (cGAS)-stimulator of interferon genes (STING)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37816954"
                        open
                      >
                        <summary class="article-expander__title">STING activation in TET2-mutated hematopoietic stem/progenitor cells contributes to the increased self-renewal and neoplastic transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37816954/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231019204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic loss-of-function mutations of the dioxygenase Ten-eleven translocation-2 (TET2) occur frequently in individuals with clonal hematopoiesis (CH) and acute myeloid leukemia (AML). These common hematopoietic disorders can be recapitulated in mouse models. However, the underlying mechanisms by which the deficiency in TET2 promotes these disorders remain unclear. Here we show that the cyclic guanosine monophosphate-adenosine monophosphate synthase (cGAS)-stimulator of interferon genes (STING)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37816954"
                        open
                      >
                        <summary class="article-expander__title">STING activation in TET2-mutated hematopoietic stem/progenitor cells contributes to the increased self-renewal and neoplastic transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37816954/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231018205013&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic loss-of-function mutations of the dioxygenase Ten-eleven translocation-2 (TET2) occur frequently in individuals with clonal hematopoiesis (CH) and acute myeloid leukemia (AML). These common hematopoietic disorders can be recapitulated in mouse models. However, the underlying mechanisms by which the deficiency in TET2 promotes these disorders remain unclear. Here we show that the cyclic guanosine monophosphate-adenosine monophosphate synthase (cGAS)-stimulator of interferon genes (STING)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37816954"
                        open
                      >
                        <summary class="article-expander__title">STING activation in TET2-mutated hematopoietic stem/progenitor cells contributes to the increased self-renewal and neoplastic transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37816954/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231017205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic loss-of-function mutations of the dioxygenase Ten-eleven translocation-2 (TET2) occur frequently in individuals with clonal hematopoiesis (CH) and acute myeloid leukemia (AML). These common hematopoietic disorders can be recapitulated in mouse models. However, the underlying mechanisms by which the deficiency in TET2 promotes these disorders remain unclear. Here we show that the cyclic guanosine monophosphate-adenosine monophosphate synthase (cGAS)-stimulator of interferon genes (STING)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37816954"
                        open
                      >
                        <summary class="article-expander__title">STING activation in TET2-mutated hematopoietic stem/progenitor cells contributes to the increased self-renewal and neoplastic transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37816954/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231016205028&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic loss-of-function mutations of the dioxygenase Ten-eleven translocation-2 (TET2) occur frequently in individuals with clonal hematopoiesis (CH) and acute myeloid leukemia (AML). These common hematopoietic disorders can be recapitulated in mouse models. However, the underlying mechanisms by which the deficiency in TET2 promotes these disorders remain unclear. Here we show that the cyclic guanosine monophosphate-adenosine monophosphate synthase (cGAS)-stimulator of interferon genes (STING)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37816954"
                        open
                      >
                        <summary class="article-expander__title">STING activation in TET2-mutated hematopoietic stem/progenitor cells contributes to the increased self-renewal and neoplastic transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37816954/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231015205105&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic loss-of-function mutations of the dioxygenase Ten-eleven translocation-2 (TET2) occur frequently in individuals with clonal hematopoiesis (CH) and acute myeloid leukemia (AML). These common hematopoietic disorders can be recapitulated in mouse models. However, the underlying mechanisms by which the deficiency in TET2 promotes these disorders remain unclear. Here we show that the cyclic guanosine monophosphate-adenosine monophosphate synthase (cGAS)-stimulator of interferon genes (STING)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37816954"
                        open
                      >
                        <summary class="article-expander__title">STING activation in TET2-mutated hematopoietic stem/progenitor cells contributes to the increased self-renewal and neoplastic transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37816954/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231014205339&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic loss-of-function mutations of the dioxygenase Ten-eleven translocation-2 (TET2) occur frequently in individuals with clonal hematopoiesis (CH) and acute myeloid leukemia (AML). These common hematopoietic disorders can be recapitulated in mouse models. However, the underlying mechanisms by which the deficiency in TET2 promotes these disorders remain unclear. Here we show that the cyclic guanosine monophosphate-adenosine monophosphate synthase (cGAS)-stimulator of interferon genes (STING)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37816954"
                        open
                      >
                        <summary class="article-expander__title">STING activation in TET2-mutated hematopoietic stem/progenitor cells contributes to the increased self-renewal and neoplastic transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37816954/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231013204816&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic loss-of-function mutations of the dioxygenase Ten-eleven translocation-2 (TET2) occur frequently in individuals with clonal hematopoiesis (CH) and acute myeloid leukemia (AML). These common hematopoietic disorders can be recapitulated in mouse models. However, the underlying mechanisms by which the deficiency in TET2 promotes these disorders remain unclear. Here we show that the cyclic guanosine monophosphate-adenosine monophosphate synthase (cGAS)-stimulator of interferon genes (STING)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37816954"
                        open
                      >
                        <summary class="article-expander__title">STING activation in TET2-mutated hematopoietic stem/progenitor cells contributes to the increased self-renewal and neoplastic transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37816954/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231012205120&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic loss-of-function mutations of the dioxygenase Ten-eleven translocation-2 (TET2) occur frequently in individuals with clonal hematopoiesis (CH) and acute myeloid leukemia (AML). These common hematopoietic disorders can be recapitulated in mouse models. However, the underlying mechanisms by which the deficiency in TET2 promotes these disorders remain unclear. Here we show that the cyclic guanosine monophosphate-adenosine monophosphate synthase (cGAS)-stimulator of interferon genes (STING)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37816954"
                        open
                      >
                        <summary class="article-expander__title">STING activation in TET2-mutated hematopoietic stem/progenitor cells contributes to the increased self-renewal and neoplastic transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37816954/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231011204744&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic loss-of-function mutations of the dioxygenase Ten-eleven translocation-2 (TET2) occur frequently in individuals with clonal hematopoiesis (CH) and acute myeloid leukemia (AML). These common hematopoietic disorders can be recapitulated in mouse models. However, the underlying mechanisms by which the deficiency in TET2 promotes these disorders remain unclear. Here we show that the cyclic guanosine monophosphate-adenosine monophosphate synthase (cGAS)-stimulator of interferon genes (STING)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-10-09 datetime="2023-10-09T10:00:00.000Z">2023-10-09</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-10-09 datetime="2023-10-09T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20231105195156">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37809221"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndrome After Anti-CD19 Chimeric Antigen Receptor T-cell Therapy: A Case Series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37809221/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of CD19-targeted chimeric antigen receptor T-cell (CAR-T cell) therapy in the management of refractory/relapsed B-cell malignancies has increased tremendously in recent times. In addition to cytokine release syndrome (CRS), neurotoxicity, and infections, CAR-T cell patients develop cytopenias, with about 15% of the patients continuing to have severe cytopenias up to three months after treatment. Retrospective reviews have reported the development of myelodysplastic syndrome (MDS) in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37809221"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndrome After Anti-CD19 Chimeric Antigen Receptor T-cell Therapy: A Case Series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37809221/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of CD19-targeted chimeric antigen receptor T-cell (CAR-T cell) therapy in the management of refractory/relapsed B-cell malignancies has increased tremendously in recent times. In addition to cytokine release syndrome (CRS), neurotoxicity, and infections, CAR-T cell patients develop cytopenias, with about 15% of the patients continuing to have severe cytopenias up to three months after treatment. Retrospective reviews have reported the development of myelodysplastic syndrome (MDS) in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37809221"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndrome After Anti-CD19 Chimeric Antigen Receptor T-cell Therapy: A Case Series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37809221/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231027204803&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of CD19-targeted chimeric antigen receptor T-cell (CAR-T cell) therapy in the management of refractory/relapsed B-cell malignancies has increased tremendously in recent times. In addition to cytokine release syndrome (CRS), neurotoxicity, and infections, CAR-T cell patients develop cytopenias, with about 15% of the patients continuing to have severe cytopenias up to three months after treatment. Retrospective reviews have reported the development of myelodysplastic syndrome (MDS) in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37809221"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndrome After Anti-CD19 Chimeric Antigen Receptor T-cell Therapy: A Case Series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37809221/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of CD19-targeted chimeric antigen receptor T-cell (CAR-T cell) therapy in the management of refractory/relapsed B-cell malignancies has increased tremendously in recent times. In addition to cytokine release syndrome (CRS), neurotoxicity, and infections, CAR-T cell patients develop cytopenias, with about 15% of the patients continuing to have severe cytopenias up to three months after treatment. Retrospective reviews have reported the development of myelodysplastic syndrome (MDS) in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37809221"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndrome After Anti-CD19 Chimeric Antigen Receptor T-cell Therapy: A Case Series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37809221/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of CD19-targeted chimeric antigen receptor T-cell (CAR-T cell) therapy in the management of refractory/relapsed B-cell malignancies has increased tremendously in recent times. In addition to cytokine release syndrome (CRS), neurotoxicity, and infections, CAR-T cell patients develop cytopenias, with about 15% of the patients continuing to have severe cytopenias up to three months after treatment. Retrospective reviews have reported the development of myelodysplastic syndrome (MDS) in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37809221"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndrome After Anti-CD19 Chimeric Antigen Receptor T-cell Therapy: A Case Series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37809221/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of CD19-targeted chimeric antigen receptor T-cell (CAR-T cell) therapy in the management of refractory/relapsed B-cell malignancies has increased tremendously in recent times. In addition to cytokine release syndrome (CRS), neurotoxicity, and infections, CAR-T cell patients develop cytopenias, with about 15% of the patients continuing to have severe cytopenias up to three months after treatment. Retrospective reviews have reported the development of myelodysplastic syndrome (MDS) in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37809221"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndrome After Anti-CD19 Chimeric Antigen Receptor T-cell Therapy: A Case Series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37809221/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231023204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of CD19-targeted chimeric antigen receptor T-cell (CAR-T cell) therapy in the management of refractory/relapsed B-cell malignancies has increased tremendously in recent times. In addition to cytokine release syndrome (CRS), neurotoxicity, and infections, CAR-T cell patients develop cytopenias, with about 15% of the patients continuing to have severe cytopenias up to three months after treatment. Retrospective reviews have reported the development of myelodysplastic syndrome (MDS) in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37809221"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndrome After Anti-CD19 Chimeric Antigen Receptor T-cell Therapy: A Case Series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37809221/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231022205027&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of CD19-targeted chimeric antigen receptor T-cell (CAR-T cell) therapy in the management of refractory/relapsed B-cell malignancies has increased tremendously in recent times. In addition to cytokine release syndrome (CRS), neurotoxicity, and infections, CAR-T cell patients develop cytopenias, with about 15% of the patients continuing to have severe cytopenias up to three months after treatment. Retrospective reviews have reported the development of myelodysplastic syndrome (MDS) in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37809221"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndrome After Anti-CD19 Chimeric Antigen Receptor T-cell Therapy: A Case Series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37809221/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231021205333&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of CD19-targeted chimeric antigen receptor T-cell (CAR-T cell) therapy in the management of refractory/relapsed B-cell malignancies has increased tremendously in recent times. In addition to cytokine release syndrome (CRS), neurotoxicity, and infections, CAR-T cell patients develop cytopenias, with about 15% of the patients continuing to have severe cytopenias up to three months after treatment. Retrospective reviews have reported the development of myelodysplastic syndrome (MDS) in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37809221"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndrome After Anti-CD19 Chimeric Antigen Receptor T-cell Therapy: A Case Series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37809221/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231020204820&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of CD19-targeted chimeric antigen receptor T-cell (CAR-T cell) therapy in the management of refractory/relapsed B-cell malignancies has increased tremendously in recent times. In addition to cytokine release syndrome (CRS), neurotoxicity, and infections, CAR-T cell patients develop cytopenias, with about 15% of the patients continuing to have severe cytopenias up to three months after treatment. Retrospective reviews have reported the development of myelodysplastic syndrome (MDS) in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37809221"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndrome After Anti-CD19 Chimeric Antigen Receptor T-cell Therapy: A Case Series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37809221/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231019204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of CD19-targeted chimeric antigen receptor T-cell (CAR-T cell) therapy in the management of refractory/relapsed B-cell malignancies has increased tremendously in recent times. In addition to cytokine release syndrome (CRS), neurotoxicity, and infections, CAR-T cell patients develop cytopenias, with about 15% of the patients continuing to have severe cytopenias up to three months after treatment. Retrospective reviews have reported the development of myelodysplastic syndrome (MDS) in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37809221"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndrome After Anti-CD19 Chimeric Antigen Receptor T-cell Therapy: A Case Series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37809221/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231018205013&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of CD19-targeted chimeric antigen receptor T-cell (CAR-T cell) therapy in the management of refractory/relapsed B-cell malignancies has increased tremendously in recent times. In addition to cytokine release syndrome (CRS), neurotoxicity, and infections, CAR-T cell patients develop cytopenias, with about 15% of the patients continuing to have severe cytopenias up to three months after treatment. Retrospective reviews have reported the development of myelodysplastic syndrome (MDS) in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37809221"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndrome After Anti-CD19 Chimeric Antigen Receptor T-cell Therapy: A Case Series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37809221/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231017205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of CD19-targeted chimeric antigen receptor T-cell (CAR-T cell) therapy in the management of refractory/relapsed B-cell malignancies has increased tremendously in recent times. In addition to cytokine release syndrome (CRS), neurotoxicity, and infections, CAR-T cell patients develop cytopenias, with about 15% of the patients continuing to have severe cytopenias up to three months after treatment. Retrospective reviews have reported the development of myelodysplastic syndrome (MDS) in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37809221"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndrome After Anti-CD19 Chimeric Antigen Receptor T-cell Therapy: A Case Series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37809221/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231016205028&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of CD19-targeted chimeric antigen receptor T-cell (CAR-T cell) therapy in the management of refractory/relapsed B-cell malignancies has increased tremendously in recent times. In addition to cytokine release syndrome (CRS), neurotoxicity, and infections, CAR-T cell patients develop cytopenias, with about 15% of the patients continuing to have severe cytopenias up to three months after treatment. Retrospective reviews have reported the development of myelodysplastic syndrome (MDS) in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37809221"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndrome After Anti-CD19 Chimeric Antigen Receptor T-cell Therapy: A Case Series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37809221/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231015205105&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of CD19-targeted chimeric antigen receptor T-cell (CAR-T cell) therapy in the management of refractory/relapsed B-cell malignancies has increased tremendously in recent times. In addition to cytokine release syndrome (CRS), neurotoxicity, and infections, CAR-T cell patients develop cytopenias, with about 15% of the patients continuing to have severe cytopenias up to three months after treatment. Retrospective reviews have reported the development of myelodysplastic syndrome (MDS) in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37809221"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndrome After Anti-CD19 Chimeric Antigen Receptor T-cell Therapy: A Case Series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37809221/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231014205339&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of CD19-targeted chimeric antigen receptor T-cell (CAR-T cell) therapy in the management of refractory/relapsed B-cell malignancies has increased tremendously in recent times. In addition to cytokine release syndrome (CRS), neurotoxicity, and infections, CAR-T cell patients develop cytopenias, with about 15% of the patients continuing to have severe cytopenias up to three months after treatment. Retrospective reviews have reported the development of myelodysplastic syndrome (MDS) in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37809221"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndrome After Anti-CD19 Chimeric Antigen Receptor T-cell Therapy: A Case Series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37809221/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231013204816&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of CD19-targeted chimeric antigen receptor T-cell (CAR-T cell) therapy in the management of refractory/relapsed B-cell malignancies has increased tremendously in recent times. In addition to cytokine release syndrome (CRS), neurotoxicity, and infections, CAR-T cell patients develop cytopenias, with about 15% of the patients continuing to have severe cytopenias up to three months after treatment. Retrospective reviews have reported the development of myelodysplastic syndrome (MDS) in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37809221"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndrome After Anti-CD19 Chimeric Antigen Receptor T-cell Therapy: A Case Series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37809221/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231012205120&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of CD19-targeted chimeric antigen receptor T-cell (CAR-T cell) therapy in the management of refractory/relapsed B-cell malignancies has increased tremendously in recent times. In addition to cytokine release syndrome (CRS), neurotoxicity, and infections, CAR-T cell patients develop cytopenias, with about 15% of the patients continuing to have severe cytopenias up to three months after treatment. Retrospective reviews have reported the development of myelodysplastic syndrome (MDS) in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37809221"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndrome After Anti-CD19 Chimeric Antigen Receptor T-cell Therapy: A Case Series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37809221/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231011204744&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of CD19-targeted chimeric antigen receptor T-cell (CAR-T cell) therapy in the management of refractory/relapsed B-cell malignancies has increased tremendously in recent times. In addition to cytokine release syndrome (CRS), neurotoxicity, and infections, CAR-T cell patients develop cytopenias, with about 15% of the patients continuing to have severe cytopenias up to three months after treatment. Retrospective reviews have reported the development of myelodysplastic syndrome (MDS) in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37809221"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndrome After Anti-CD19 Chimeric Antigen Receptor T-cell Therapy: A Case Series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37809221/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231010204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of CD19-targeted chimeric antigen receptor T-cell (CAR-T cell) therapy in the management of refractory/relapsed B-cell malignancies has increased tremendously in recent times. In addition to cytokine release syndrome (CRS), neurotoxicity, and infections, CAR-T cell patients develop cytopenias, with about 15% of the patients continuing to have severe cytopenias up to three months after treatment. Retrospective reviews have reported the development of myelodysplastic syndrome (MDS) in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37809221"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndrome After Anti-CD19 Chimeric Antigen Receptor T-cell Therapy: A Case Series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37809221/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231009204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of CD19-targeted chimeric antigen receptor T-cell (CAR-T cell) therapy in the management of refractory/relapsed B-cell malignancies has increased tremendously in recent times. In addition to cytokine release syndrome (CRS), neurotoxicity, and infections, CAR-T cell patients develop cytopenias, with about 15% of the patients continuing to have severe cytopenias up to three months after treatment. Retrospective reviews have reported the development of myelodysplastic syndrome (MDS) in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;ff&#x3D;20231105195156&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37812025"
                        open
                      >
                        <summary class="article-expander__title">Subclonal Somatic Copy-Number Alterations Emerge and Dominate in Recurrent Osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37812025/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231027204802&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple large-scale genomic profiling efforts have been undertaken in osteosarcoma to define the genomic drivers of tumorigenesis, therapeutic response, and disease recurrence. The spatial and temporal intratumor heterogeneity could also play a role in promoting tumor growth and treatment resistance. We conducted longitudinal whole-genome sequencing of 37 tumor samples from 8 patients with relapsed or refractory osteosarcoma. Each patient had at least one sample from a primary site and a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37812025"
                        open
                      >
                        <summary class="article-expander__title">Subclonal Somatic Copy-Number Alterations Emerge and Dominate in Recurrent Osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37812025/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple large-scale genomic profiling efforts have been undertaken in osteosarcoma to define the genomic drivers of tumorigenesis, therapeutic response, and disease recurrence. The spatial and temporal intratumor heterogeneity could also play a role in promoting tumor growth and treatment resistance. We conducted longitudinal whole-genome sequencing of 37 tumor samples from 8 patients with relapsed or refractory osteosarcoma. Each patient had at least one sample from a primary site and a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37812025"
                        open
                      >
                        <summary class="article-expander__title">Subclonal Somatic Copy-Number Alterations Emerge and Dominate in Recurrent Osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37812025/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple large-scale genomic profiling efforts have been undertaken in osteosarcoma to define the genomic drivers of tumorigenesis, therapeutic response, and disease recurrence. The spatial and temporal intratumor heterogeneity could also play a role in promoting tumor growth and treatment resistance. We conducted longitudinal whole-genome sequencing of 37 tumor samples from 8 patients with relapsed or refractory osteosarcoma. Each patient had at least one sample from a primary site and a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37812025"
                        open
                      >
                        <summary class="article-expander__title">Subclonal Somatic Copy-Number Alterations Emerge and Dominate in Recurrent Osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37812025/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple large-scale genomic profiling efforts have been undertaken in osteosarcoma to define the genomic drivers of tumorigenesis, therapeutic response, and disease recurrence. The spatial and temporal intratumor heterogeneity could also play a role in promoting tumor growth and treatment resistance. We conducted longitudinal whole-genome sequencing of 37 tumor samples from 8 patients with relapsed or refractory osteosarcoma. Each patient had at least one sample from a primary site and a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37812025"
                        open
                      >
                        <summary class="article-expander__title">Subclonal Somatic Copy-Number Alterations Emerge and Dominate in Recurrent Osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37812025/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231023204959&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple large-scale genomic profiling efforts have been undertaken in osteosarcoma to define the genomic drivers of tumorigenesis, therapeutic response, and disease recurrence. The spatial and temporal intratumor heterogeneity could also play a role in promoting tumor growth and treatment resistance. We conducted longitudinal whole-genome sequencing of 37 tumor samples from 8 patients with relapsed or refractory osteosarcoma. Each patient had at least one sample from a primary site and a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37812025"
                        open
                      >
                        <summary class="article-expander__title">Subclonal Somatic Copy-Number Alterations Emerge and Dominate in Recurrent Osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37812025/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231022205027&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple large-scale genomic profiling efforts have been undertaken in osteosarcoma to define the genomic drivers of tumorigenesis, therapeutic response, and disease recurrence. The spatial and temporal intratumor heterogeneity could also play a role in promoting tumor growth and treatment resistance. We conducted longitudinal whole-genome sequencing of 37 tumor samples from 8 patients with relapsed or refractory osteosarcoma. Each patient had at least one sample from a primary site and a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37812025"
                        open
                      >
                        <summary class="article-expander__title">Subclonal Somatic Copy-Number Alterations Emerge and Dominate in Recurrent Osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37812025/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231021205333&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple large-scale genomic profiling efforts have been undertaken in osteosarcoma to define the genomic drivers of tumorigenesis, therapeutic response, and disease recurrence. The spatial and temporal intratumor heterogeneity could also play a role in promoting tumor growth and treatment resistance. We conducted longitudinal whole-genome sequencing of 37 tumor samples from 8 patients with relapsed or refractory osteosarcoma. Each patient had at least one sample from a primary site and a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37812025"
                        open
                      >
                        <summary class="article-expander__title">Subclonal Somatic Copy-Number Alterations Emerge and Dominate in Recurrent Osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37812025/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231020204820&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple large-scale genomic profiling efforts have been undertaken in osteosarcoma to define the genomic drivers of tumorigenesis, therapeutic response, and disease recurrence. The spatial and temporal intratumor heterogeneity could also play a role in promoting tumor growth and treatment resistance. We conducted longitudinal whole-genome sequencing of 37 tumor samples from 8 patients with relapsed or refractory osteosarcoma. Each patient had at least one sample from a primary site and a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37812025"
                        open
                      >
                        <summary class="article-expander__title">Subclonal Somatic Copy-Number Alterations Emerge and Dominate in Recurrent Osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37812025/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231019204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple large-scale genomic profiling efforts have been undertaken in osteosarcoma to define the genomic drivers of tumorigenesis, therapeutic response, and disease recurrence. The spatial and temporal intratumor heterogeneity could also play a role in promoting tumor growth and treatment resistance. We conducted longitudinal whole-genome sequencing of 37 tumor samples from 8 patients with relapsed or refractory osteosarcoma. Each patient had at least one sample from a primary site and a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37812025"
                        open
                      >
                        <summary class="article-expander__title">Subclonal Somatic Copy-Number Alterations Emerge and Dominate in Recurrent Osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37812025/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231018205013&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple large-scale genomic profiling efforts have been undertaken in osteosarcoma to define the genomic drivers of tumorigenesis, therapeutic response, and disease recurrence. The spatial and temporal intratumor heterogeneity could also play a role in promoting tumor growth and treatment resistance. We conducted longitudinal whole-genome sequencing of 37 tumor samples from 8 patients with relapsed or refractory osteosarcoma. Each patient had at least one sample from a primary site and a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37812025"
                        open
                      >
                        <summary class="article-expander__title">Subclonal Somatic Copy-Number Alterations Emerge and Dominate in Recurrent Osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37812025/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231017205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple large-scale genomic profiling efforts have been undertaken in osteosarcoma to define the genomic drivers of tumorigenesis, therapeutic response, and disease recurrence. The spatial and temporal intratumor heterogeneity could also play a role in promoting tumor growth and treatment resistance. We conducted longitudinal whole-genome sequencing of 37 tumor samples from 8 patients with relapsed or refractory osteosarcoma. Each patient had at least one sample from a primary site and a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37812025"
                        open
                      >
                        <summary class="article-expander__title">Subclonal Somatic Copy-Number Alterations Emerge and Dominate in Recurrent Osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37812025/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231016205028&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple large-scale genomic profiling efforts have been undertaken in osteosarcoma to define the genomic drivers of tumorigenesis, therapeutic response, and disease recurrence. The spatial and temporal intratumor heterogeneity could also play a role in promoting tumor growth and treatment resistance. We conducted longitudinal whole-genome sequencing of 37 tumor samples from 8 patients with relapsed or refractory osteosarcoma. Each patient had at least one sample from a primary site and a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37812025"
                        open
                      >
                        <summary class="article-expander__title">Subclonal Somatic Copy-Number Alterations Emerge and Dominate in Recurrent Osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37812025/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231015205105&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple large-scale genomic profiling efforts have been undertaken in osteosarcoma to define the genomic drivers of tumorigenesis, therapeutic response, and disease recurrence. The spatial and temporal intratumor heterogeneity could also play a role in promoting tumor growth and treatment resistance. We conducted longitudinal whole-genome sequencing of 37 tumor samples from 8 patients with relapsed or refractory osteosarcoma. Each patient had at least one sample from a primary site and a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37812025"
                        open
                      >
                        <summary class="article-expander__title">Subclonal Somatic Copy-Number Alterations Emerge and Dominate in Recurrent Osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37812025/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231014205339&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple large-scale genomic profiling efforts have been undertaken in osteosarcoma to define the genomic drivers of tumorigenesis, therapeutic response, and disease recurrence. The spatial and temporal intratumor heterogeneity could also play a role in promoting tumor growth and treatment resistance. We conducted longitudinal whole-genome sequencing of 37 tumor samples from 8 patients with relapsed or refractory osteosarcoma. Each patient had at least one sample from a primary site and a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37812025"
                        open
                      >
                        <summary class="article-expander__title">Subclonal Somatic Copy-Number Alterations Emerge and Dominate in Recurrent Osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37812025/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231013204816&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple large-scale genomic profiling efforts have been undertaken in osteosarcoma to define the genomic drivers of tumorigenesis, therapeutic response, and disease recurrence. The spatial and temporal intratumor heterogeneity could also play a role in promoting tumor growth and treatment resistance. We conducted longitudinal whole-genome sequencing of 37 tumor samples from 8 patients with relapsed or refractory osteosarcoma. Each patient had at least one sample from a primary site and a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37812025"
                        open
                      >
                        <summary class="article-expander__title">Subclonal Somatic Copy-Number Alterations Emerge and Dominate in Recurrent Osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37812025/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231012205121&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple large-scale genomic profiling efforts have been undertaken in osteosarcoma to define the genomic drivers of tumorigenesis, therapeutic response, and disease recurrence. The spatial and temporal intratumor heterogeneity could also play a role in promoting tumor growth and treatment resistance. We conducted longitudinal whole-genome sequencing of 37 tumor samples from 8 patients with relapsed or refractory osteosarcoma. Each patient had at least one sample from a primary site and a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37812025"
                        open
                      >
                        <summary class="article-expander__title">Subclonal Somatic Copy-Number Alterations Emerge and Dominate in Recurrent Osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37812025/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231011204744&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple large-scale genomic profiling efforts have been undertaken in osteosarcoma to define the genomic drivers of tumorigenesis, therapeutic response, and disease recurrence. The spatial and temporal intratumor heterogeneity could also play a role in promoting tumor growth and treatment resistance. We conducted longitudinal whole-genome sequencing of 37 tumor samples from 8 patients with relapsed or refractory osteosarcoma. Each patient had at least one sample from a primary site and a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37812025"
                        open
                      >
                        <summary class="article-expander__title">Subclonal Somatic Copy-Number Alterations Emerge and Dominate in Recurrent Osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37812025/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231010204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple large-scale genomic profiling efforts have been undertaken in osteosarcoma to define the genomic drivers of tumorigenesis, therapeutic response, and disease recurrence. The spatial and temporal intratumor heterogeneity could also play a role in promoting tumor growth and treatment resistance. We conducted longitudinal whole-genome sequencing of 37 tumor samples from 8 patients with relapsed or refractory osteosarcoma. Each patient had at least one sample from a primary site and a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37812025"
                        open
                      >
                        <summary class="article-expander__title">Subclonal Somatic Copy-Number Alterations Emerge and Dominate in Recurrent Osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37812025/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231009204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple large-scale genomic profiling efforts have been undertaken in osteosarcoma to define the genomic drivers of tumorigenesis, therapeutic response, and disease recurrence. The spatial and temporal intratumor heterogeneity could also play a role in promoting tumor growth and treatment resistance. We conducted longitudinal whole-genome sequencing of 37 tumor samples from 8 patients with relapsed or refractory osteosarcoma. Each patient had at least one sample from a primary site and a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-10-07 datetime="2023-10-07T10:00:00.000Z">2023-10-07</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-10-07 datetime="2023-10-07T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20231105195156">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37804041"
                        open
                      >
                        <summary class="article-expander__title">Bomedemstat as an investigative treatment for myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37804041/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231027204803&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myeloproliferative neoplasm (MPN) is a heterogeneous group of hematopoietic stem cell disorders characterized by clonal proliferation of one of more of the hematopoietic stem cell lineages. Clinical manifestations result from uncontrolled myeloproliferation, extramedullary hematopoiesis with splenomegaly and excessive inflammatory cytokine production. Currently available therapy improves hematologic parameters and symptoms but does not adequately address the underlying neoplastic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37804041"
                        open
                      >
                        <summary class="article-expander__title">Bomedemstat as an investigative treatment for myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37804041/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myeloproliferative neoplasm (MPN) is a heterogeneous group of hematopoietic stem cell disorders characterized by clonal proliferation of one of more of the hematopoietic stem cell lineages. Clinical manifestations result from uncontrolled myeloproliferation, extramedullary hematopoiesis with splenomegaly and excessive inflammatory cytokine production. Currently available therapy improves hematologic parameters and symptoms but does not adequately address the underlying neoplastic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37804041"
                        open
                      >
                        <summary class="article-expander__title">Bomedemstat as an investigative treatment for myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37804041/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myeloproliferative neoplasm (MPN) is a heterogeneous group of hematopoietic stem cell disorders characterized by clonal proliferation of one of more of the hematopoietic stem cell lineages. Clinical manifestations result from uncontrolled myeloproliferation, extramedullary hematopoiesis with splenomegaly and excessive inflammatory cytokine production. Currently available therapy improves hematologic parameters and symptoms but does not adequately address the underlying neoplastic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37804041"
                        open
                      >
                        <summary class="article-expander__title">Bomedemstat as an investigative treatment for myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37804041/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myeloproliferative neoplasm (MPN) is a heterogeneous group of hematopoietic stem cell disorders characterized by clonal proliferation of one of more of the hematopoietic stem cell lineages. Clinical manifestations result from uncontrolled myeloproliferation, extramedullary hematopoiesis with splenomegaly and excessive inflammatory cytokine production. Currently available therapy improves hematologic parameters and symptoms but does not adequately address the underlying neoplastic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37804041"
                        open
                      >
                        <summary class="article-expander__title">Bomedemstat as an investigative treatment for myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37804041/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231023204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myeloproliferative neoplasm (MPN) is a heterogeneous group of hematopoietic stem cell disorders characterized by clonal proliferation of one of more of the hematopoietic stem cell lineages. Clinical manifestations result from uncontrolled myeloproliferation, extramedullary hematopoiesis with splenomegaly and excessive inflammatory cytokine production. Currently available therapy improves hematologic parameters and symptoms but does not adequately address the underlying neoplastic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37804041"
                        open
                      >
                        <summary class="article-expander__title">Bomedemstat as an investigative treatment for myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37804041/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231022205027&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myeloproliferative neoplasm (MPN) is a heterogeneous group of hematopoietic stem cell disorders characterized by clonal proliferation of one of more of the hematopoietic stem cell lineages. Clinical manifestations result from uncontrolled myeloproliferation, extramedullary hematopoiesis with splenomegaly and excessive inflammatory cytokine production. Currently available therapy improves hematologic parameters and symptoms but does not adequately address the underlying neoplastic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37804041"
                        open
                      >
                        <summary class="article-expander__title">Bomedemstat as an investigative treatment for myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37804041/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231021205333&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myeloproliferative neoplasm (MPN) is a heterogeneous group of hematopoietic stem cell disorders characterized by clonal proliferation of one of more of the hematopoietic stem cell lineages. Clinical manifestations result from uncontrolled myeloproliferation, extramedullary hematopoiesis with splenomegaly and excessive inflammatory cytokine production. Currently available therapy improves hematologic parameters and symptoms but does not adequately address the underlying neoplastic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37804041"
                        open
                      >
                        <summary class="article-expander__title">Bomedemstat as an investigative treatment for myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37804041/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231020204820&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myeloproliferative neoplasm (MPN) is a heterogeneous group of hematopoietic stem cell disorders characterized by clonal proliferation of one of more of the hematopoietic stem cell lineages. Clinical manifestations result from uncontrolled myeloproliferation, extramedullary hematopoiesis with splenomegaly and excessive inflammatory cytokine production. Currently available therapy improves hematologic parameters and symptoms but does not adequately address the underlying neoplastic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37804041"
                        open
                      >
                        <summary class="article-expander__title">Bomedemstat as an investigative treatment for myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37804041/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231019204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myeloproliferative neoplasm (MPN) is a heterogeneous group of hematopoietic stem cell disorders characterized by clonal proliferation of one of more of the hematopoietic stem cell lineages. Clinical manifestations result from uncontrolled myeloproliferation, extramedullary hematopoiesis with splenomegaly and excessive inflammatory cytokine production. Currently available therapy improves hematologic parameters and symptoms but does not adequately address the underlying neoplastic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37804041"
                        open
                      >
                        <summary class="article-expander__title">Bomedemstat as an investigative treatment for myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37804041/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231018205013&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myeloproliferative neoplasm (MPN) is a heterogeneous group of hematopoietic stem cell disorders characterized by clonal proliferation of one of more of the hematopoietic stem cell lineages. Clinical manifestations result from uncontrolled myeloproliferation, extramedullary hematopoiesis with splenomegaly and excessive inflammatory cytokine production. Currently available therapy improves hematologic parameters and symptoms but does not adequately address the underlying neoplastic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37804041"
                        open
                      >
                        <summary class="article-expander__title">Bomedemstat as an investigative treatment for myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37804041/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231017205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myeloproliferative neoplasm (MPN) is a heterogeneous group of hematopoietic stem cell disorders characterized by clonal proliferation of one of more of the hematopoietic stem cell lineages. Clinical manifestations result from uncontrolled myeloproliferation, extramedullary hematopoiesis with splenomegaly and excessive inflammatory cytokine production. Currently available therapy improves hematologic parameters and symptoms but does not adequately address the underlying neoplastic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37804041"
                        open
                      >
                        <summary class="article-expander__title">Bomedemstat as an investigative treatment for myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37804041/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231016205028&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myeloproliferative neoplasm (MPN) is a heterogeneous group of hematopoietic stem cell disorders characterized by clonal proliferation of one of more of the hematopoietic stem cell lineages. Clinical manifestations result from uncontrolled myeloproliferation, extramedullary hematopoiesis with splenomegaly and excessive inflammatory cytokine production. Currently available therapy improves hematologic parameters and symptoms but does not adequately address the underlying neoplastic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37804041"
                        open
                      >
                        <summary class="article-expander__title">Bomedemstat as an investigative treatment for myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37804041/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231015205105&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myeloproliferative neoplasm (MPN) is a heterogeneous group of hematopoietic stem cell disorders characterized by clonal proliferation of one of more of the hematopoietic stem cell lineages. Clinical manifestations result from uncontrolled myeloproliferation, extramedullary hematopoiesis with splenomegaly and excessive inflammatory cytokine production. Currently available therapy improves hematologic parameters and symptoms but does not adequately address the underlying neoplastic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37804041"
                        open
                      >
                        <summary class="article-expander__title">Bomedemstat as an investigative treatment for myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37804041/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231014205339&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myeloproliferative neoplasm (MPN) is a heterogeneous group of hematopoietic stem cell disorders characterized by clonal proliferation of one of more of the hematopoietic stem cell lineages. Clinical manifestations result from uncontrolled myeloproliferation, extramedullary hematopoiesis with splenomegaly and excessive inflammatory cytokine production. Currently available therapy improves hematologic parameters and symptoms but does not adequately address the underlying neoplastic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37804041"
                        open
                      >
                        <summary class="article-expander__title">Bomedemstat as an investigative treatment for myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37804041/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231013204816&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myeloproliferative neoplasm (MPN) is a heterogeneous group of hematopoietic stem cell disorders characterized by clonal proliferation of one of more of the hematopoietic stem cell lineages. Clinical manifestations result from uncontrolled myeloproliferation, extramedullary hematopoiesis with splenomegaly and excessive inflammatory cytokine production. Currently available therapy improves hematologic parameters and symptoms but does not adequately address the underlying neoplastic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37804041"
                        open
                      >
                        <summary class="article-expander__title">Bomedemstat as an investigative treatment for myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37804041/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231012205120&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myeloproliferative neoplasm (MPN) is a heterogeneous group of hematopoietic stem cell disorders characterized by clonal proliferation of one of more of the hematopoietic stem cell lineages. Clinical manifestations result from uncontrolled myeloproliferation, extramedullary hematopoiesis with splenomegaly and excessive inflammatory cytokine production. Currently available therapy improves hematologic parameters and symptoms but does not adequately address the underlying neoplastic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37804041"
                        open
                      >
                        <summary class="article-expander__title">Bomedemstat as an investigative treatment for myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37804041/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231011204744&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myeloproliferative neoplasm (MPN) is a heterogeneous group of hematopoietic stem cell disorders characterized by clonal proliferation of one of more of the hematopoietic stem cell lineages. Clinical manifestations result from uncontrolled myeloproliferation, extramedullary hematopoiesis with splenomegaly and excessive inflammatory cytokine production. Currently available therapy improves hematologic parameters and symptoms but does not adequately address the underlying neoplastic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37804041"
                        open
                      >
                        <summary class="article-expander__title">Bomedemstat as an investigative treatment for myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37804041/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231010204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myeloproliferative neoplasm (MPN) is a heterogeneous group of hematopoietic stem cell disorders characterized by clonal proliferation of one of more of the hematopoietic stem cell lineages. Clinical manifestations result from uncontrolled myeloproliferation, extramedullary hematopoiesis with splenomegaly and excessive inflammatory cytokine production. Currently available therapy improves hematologic parameters and symptoms but does not adequately address the underlying neoplastic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37804041"
                        open
                      >
                        <summary class="article-expander__title">Bomedemstat as an investigative treatment for myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37804041/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231009204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myeloproliferative neoplasm (MPN) are heterogeneous group of hematopoietic stem cell disorders characterized by clonal proliferation of one of more of the hematopoietic stem cell lineages. Clinical manifestations result from uncontrolled myeloproliferation, extramedullary hematopoiesis with splenomegaly and excessive inflammatory cytokine production. Currently available therapy improves hematologic parameters and symptoms but don&#x27;t adequately address the underlying neoplastic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37804041"
                        open
                      >
                        <summary class="article-expander__title">Bomedemstat as an investigative treatment for myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37804041/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231008205021&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myeloproliferative neoplasm (MPN) are heterogeneous group of hematopoietic stem cell disorders characterized by clonal proliferation of one of more of the hematopoietic stem cell lineages. Clinical manifestations result from uncontrolled myeloproliferation, extramedullary hematopoiesis with splenomegaly and excessive inflammatory cytokine production. Currently available therapy improves hematologic parameters and symptoms but don&#x27;t adequately address the underlying neoplastic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37804041"
                        open
                      >
                        <summary class="article-expander__title">Bomedemstat as an investigative treatment for myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37804041/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231007205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myeloproliferative neoplasm (MPN) are heterogeneous group of hematopoietic stem cell disorders characterized by clonal proliferation of one of more of the hematopoietic stem cell lineages. Clinical manifestations result from uncontrolled myeloproliferation, extramedullary hematopoiesis with splenomegaly and excessive inflammatory cytokine production. Currently available therapy improves hematologic parameters and symptoms but don&#x27;t adequately address the underlying neoplastic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-10-06 datetime="2023-10-06T10:00:00.000Z">2023-10-06</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-10-06 datetime="2023-10-06T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20231105195156">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37799345"
                        open
                      >
                        <summary class="article-expander__title">Clinical Implications and Dynamics of Clonal Hematopoiesis in Anti-CD19 CAR T-cell Treated Patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37799345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231027204803&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent evidence revealed important interactions between clonal hematopoiesis (CH) and cellular therapies established for the treatment of hematologic malignancies. The impact of CH on safety, efficacy, and outcome of chimeric antigen receptor (CAR) T-cell therapy is currently under investigation. We analyzed 110 patients with relapsed/refractory B-cell non-Hodgkin lymphoma (n &#x3D; 105) or acute lymphoblastic leukemia (ALL) (n &#x3D; 5), treated with Axicabtagene-Ciloleucel (39%), Tisagenlecleucel (51%),...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37799345"
                        open
                      >
                        <summary class="article-expander__title">Clinical Implications and Dynamics of Clonal Hematopoiesis in Anti-CD19 CAR T-cell Treated Patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37799345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent evidence revealed important interactions between clonal hematopoiesis (CH) and cellular therapies established for the treatment of hematologic malignancies. The impact of CH on safety, efficacy, and outcome of chimeric antigen receptor (CAR) T-cell therapy is currently under investigation. We analyzed 110 patients with relapsed/refractory B-cell non-Hodgkin lymphoma (n &#x3D; 105) or acute lymphoblastic leukemia (ALL) (n &#x3D; 5), treated with Axicabtagene-Ciloleucel (39%), Tisagenlecleucel (51%),...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37799345"
                        open
                      >
                        <summary class="article-expander__title">Clinical Implications and Dynamics of Clonal Hematopoiesis in Anti-CD19 CAR T-cell Treated Patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37799345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent evidence revealed important interactions between clonal hematopoiesis (CH) and cellular therapies established for the treatment of hematologic malignancies. The impact of CH on safety, efficacy, and outcome of chimeric antigen receptor (CAR) T-cell therapy is currently under investigation. We analyzed 110 patients with relapsed/refractory B-cell non-Hodgkin lymphoma (n &#x3D; 105) or acute lymphoblastic leukemia (ALL) (n &#x3D; 5), treated with Axicabtagene-Ciloleucel (39%), Tisagenlecleucel (51%),...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37799345"
                        open
                      >
                        <summary class="article-expander__title">Clinical Implications and Dynamics of Clonal Hematopoiesis in Anti-CD19 CAR T-cell Treated Patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37799345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent evidence revealed important interactions between clonal hematopoiesis (CH) and cellular therapies established for the treatment of hematologic malignancies. The impact of CH on safety, efficacy, and outcome of chimeric antigen receptor (CAR) T-cell therapy is currently under investigation. We analyzed 110 patients with relapsed/refractory B-cell non-Hodgkin lymphoma (n &#x3D; 105) or acute lymphoblastic leukemia (ALL) (n &#x3D; 5), treated with Axicabtagene-Ciloleucel (39%), Tisagenlecleucel (51%),...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37799345"
                        open
                      >
                        <summary class="article-expander__title">Clinical Implications and Dynamics of Clonal Hematopoiesis in Anti-CD19 CAR T-cell Treated Patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37799345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231023204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent evidence revealed important interactions between clonal hematopoiesis (CH) and cellular therapies established for the treatment of hematologic malignancies. The impact of CH on safety, efficacy, and outcome of chimeric antigen receptor (CAR) T-cell therapy is currently under investigation. We analyzed 110 patients with relapsed/refractory B-cell non-Hodgkin lymphoma (n &#x3D; 105) or acute lymphoblastic leukemia (ALL) (n &#x3D; 5), treated with Axicabtagene-Ciloleucel (39%), Tisagenlecleucel (51%),...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37799345"
                        open
                      >
                        <summary class="article-expander__title">Clinical Implications and Dynamics of Clonal Hematopoiesis in Anti-CD19 CAR T-cell Treated Patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37799345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231022205027&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent evidence revealed important interactions between clonal hematopoiesis (CH) and cellular therapies established for the treatment of hematologic malignancies. The impact of CH on safety, efficacy, and outcome of chimeric antigen receptor (CAR) T-cell therapy is currently under investigation. We analyzed 110 patients with relapsed/refractory B-cell non-Hodgkin lymphoma (n &#x3D; 105) or acute lymphoblastic leukemia (ALL) (n &#x3D; 5), treated with Axicabtagene-Ciloleucel (39%), Tisagenlecleucel (51%),...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37799345"
                        open
                      >
                        <summary class="article-expander__title">Clinical Implications and Dynamics of Clonal Hematopoiesis in Anti-CD19 CAR T-cell Treated Patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37799345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231021205333&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent evidence revealed important interactions between clonal hematopoiesis (CH) and cellular therapies established for the treatment of hematologic malignancies. The impact of CH on safety, efficacy, and outcome of chimeric antigen receptor (CAR) T-cell therapy is currently under investigation. We analyzed 110 patients with relapsed/refractory B-cell non-Hodgkin lymphoma (n &#x3D; 105) or acute lymphoblastic leukemia (ALL) (n &#x3D; 5), treated with Axicabtagene-Ciloleucel (39%), Tisagenlecleucel (51%),...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37799345"
                        open
                      >
                        <summary class="article-expander__title">Clinical Implications and Dynamics of Clonal Hematopoiesis in Anti-CD19 CAR T-cell Treated Patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37799345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231020204820&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent evidence revealed important interactions between clonal hematopoiesis (CH) and cellular therapies established for the treatment of hematologic malignancies. The impact of CH on safety, efficacy, and outcome of chimeric antigen receptor (CAR) T-cell therapy is currently under investigation. We analyzed 110 patients with relapsed/refractory B-cell non-Hodgkin lymphoma (n &#x3D; 105) or acute lymphoblastic leukemia (ALL) (n &#x3D; 5), treated with Axicabtagene-Ciloleucel (39%), Tisagenlecleucel (51%),...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37799345"
                        open
                      >
                        <summary class="article-expander__title">Clinical Implications and Dynamics of Clonal Hematopoiesis in Anti-CD19 CAR T-cell Treated Patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37799345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231019204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent evidence revealed important interactions between clonal hematopoiesis (CH) and cellular therapies established for the treatment of hematologic malignancies. The impact of CH on safety, efficacy, and outcome of chimeric antigen receptor (CAR) T-cell therapy is currently under investigation. We analyzed 110 patients with relapsed/refractory B-cell non-Hodgkin lymphoma (n &#x3D; 105) or acute lymphoblastic leukemia (ALL) (n &#x3D; 5), treated with Axicabtagene-Ciloleucel (39%), Tisagenlecleucel (51%),...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37799345"
                        open
                      >
                        <summary class="article-expander__title">Clinical Implications and Dynamics of Clonal Hematopoiesis in Anti-CD19 CAR T-cell Treated Patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37799345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231018205013&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent evidence revealed important interactions between clonal hematopoiesis (CH) and cellular therapies established for the treatment of hematologic malignancies. The impact of CH on safety, efficacy, and outcome of chimeric antigen receptor (CAR) T-cell therapy is currently under investigation. We analyzed 110 patients with relapsed/refractory B-cell non-Hodgkin lymphoma (n &#x3D; 105) or acute lymphoblastic leukemia (ALL) (n &#x3D; 5), treated with Axicabtagene-Ciloleucel (39%), Tisagenlecleucel (51%),...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37799345"
                        open
                      >
                        <summary class="article-expander__title">Clinical Implications and Dynamics of Clonal Hematopoiesis in Anti-CD19 CAR T-cell Treated Patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37799345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231017205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent evidence revealed important interactions between clonal hematopoiesis (CH) and cellular therapies established for the treatment of hematologic malignancies. The impact of CH on safety, efficacy, and outcome of chimeric antigen receptor (CAR) T-cell therapy is currently under investigation. We analyzed 110 patients with relapsed/refractory B-cell non-Hodgkin lymphoma (n &#x3D; 105) or acute lymphoblastic leukemia (ALL) (n &#x3D; 5), treated with Axicabtagene-Ciloleucel (39%), Tisagenlecleucel (51%),...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37799345"
                        open
                      >
                        <summary class="article-expander__title">Clinical Implications and Dynamics of Clonal Hematopoiesis in Anti-CD19 CAR T-cell Treated Patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37799345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231016205028&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent evidence revealed important interactions between clonal hematopoiesis (CH) and cellular therapies established for the treatment of hematologic malignancies. The impact of CH on safety, efficacy, and outcome of chimeric antigen receptor (CAR) T-cell therapy is currently under investigation. We analyzed 110 patients with relapsed/refractory B-cell non-Hodgkin lymphoma (n &#x3D; 105) or acute lymphoblastic leukemia (ALL) (n &#x3D; 5), treated with Axicabtagene-Ciloleucel (39%), Tisagenlecleucel (51%),...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37799345"
                        open
                      >
                        <summary class="article-expander__title">Clinical Implications and Dynamics of Clonal Hematopoiesis in Anti-CD19 CAR T-cell Treated Patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37799345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231015205105&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent evidence revealed important interactions between clonal hematopoiesis (CH) and cellular therapies established for the treatment of hematologic malignancies. The impact of CH on safety, efficacy, and outcome of chimeric antigen receptor (CAR) T-cell therapy is currently under investigation. We analyzed 110 patients with relapsed/refractory B-cell non-Hodgkin lymphoma (n &#x3D; 105) or acute lymphoblastic leukemia (ALL) (n &#x3D; 5), treated with Axicabtagene-Ciloleucel (39%), Tisagenlecleucel (51%),...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37799345"
                        open
                      >
                        <summary class="article-expander__title">Clinical Implications and Dynamics of Clonal Hematopoiesis in Anti-CD19 CAR T-cell Treated Patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37799345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231014205339&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent evidence revealed important interactions between clonal hematopoiesis (CH) and cellular therapies established for the treatment of hematologic malignancies. The impact of CH on safety, efficacy, and outcome of chimeric antigen receptor (CAR) T-cell therapy is currently under investigation. We analyzed 110 patients with relapsed/refractory B-cell non-Hodgkin lymphoma (n &#x3D; 105) or acute lymphoblastic leukemia (ALL) (n &#x3D; 5), treated with Axicabtagene-Ciloleucel (39%), Tisagenlecleucel (51%),...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37799345"
                        open
                      >
                        <summary class="article-expander__title">Clinical Implications and Dynamics of Clonal Hematopoiesis in Anti-CD19 CAR T-cell Treated Patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37799345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231013204816&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent evidence revealed important interactions between clonal hematopoiesis (CH) and cellular therapies established for the treatment of hematologic malignancies. The impact of CH on safety, efficacy, and outcome of chimeric antigen receptor (CAR) T-cell therapy is currently under investigation. We analyzed 110 patients with relapsed/refractory B-cell non-Hodgkin lymphoma (n &#x3D; 105) or acute lymphoblastic leukemia (ALL) (n &#x3D; 5), treated with Axicabtagene-Ciloleucel (39%), Tisagenlecleucel (51%),...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37799345"
                        open
                      >
                        <summary class="article-expander__title">Clinical Implications and Dynamics of Clonal Hematopoiesis in Anti-CD19 CAR T-cell Treated Patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37799345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231012205120&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent evidence revealed important interactions between clonal hematopoiesis (CH) and cellular therapies established for the treatment of hematologic malignancies. The impact of CH on safety, efficacy, and outcome of chimeric antigen receptor (CAR) T-cell therapy is currently under investigation. We analyzed 110 patients with relapsed/refractory B-cell non-Hodgkin lymphoma (n &#x3D; 105) or acute lymphoblastic leukemia (ALL) (n &#x3D; 5), treated with Axicabtagene-Ciloleucel (39%), Tisagenlecleucel (51%),...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37799345"
                        open
                      >
                        <summary class="article-expander__title">Clinical Implications and Dynamics of Clonal Hematopoiesis in Anti-CD19 CAR T-cell Treated Patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37799345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231011204744&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent evidence revealed important interactions between clonal hematopoiesis (CH) and cellular therapies established for the treatment of hematologic malignancies. The impact of CH on safety, efficacy, and outcome of chimeric antigen receptor (CAR) T-cell therapy is currently under investigation. We analyzed 110 patients with relapsed/refractory B-cell non-Hodgkin lymphoma (n &#x3D; 105) or acute lymphoblastic leukemia (ALL) (n &#x3D; 5), treated with Axicabtagene-Ciloleucel (39%), Tisagenlecleucel (51%),...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37799345"
                        open
                      >
                        <summary class="article-expander__title">Clinical Implications and Dynamics of Clonal Hematopoiesis in Anti-CD19 CAR T-cell Treated Patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37799345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231010204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent evidence revealed important interactions between clonal hematopoiesis (CH) and cellular therapies established for the treatment of hematologic malignancies. The impact of CH on safety, efficacy, and outcome of chimeric antigen receptor (CAR) T-cell therapy is currently under investigation. We analyzed 110 patients with relapsed/refractory B-cell non-Hodgkin lymphoma (n &#x3D; 105) or acute lymphoblastic leukemia (ALL) (n &#x3D; 5), treated with Axicabtagene-Ciloleucel (39%), Tisagenlecleucel (51%),...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37799345"
                        open
                      >
                        <summary class="article-expander__title">Clinical Implications and Dynamics of Clonal Hematopoiesis in Anti-CD19 CAR T-cell Treated Patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37799345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231009204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent evidence revealed important interactions between clonal hematopoiesis (CH) and cellular therapies established for the treatment of hematologic malignancies. The impact of CH on safety, efficacy, and outcome of chimeric antigen receptor (CAR) T-cell therapy is currently under investigation. We analyzed 110 patients with relapsed/refractory B-cell non-Hodgkin lymphoma (n &#x3D; 105) or acute lymphoblastic leukemia (ALL) (n &#x3D; 5), treated with Axicabtagene-Ciloleucel (39%), Tisagenlecleucel (51%),...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37799345"
                        open
                      >
                        <summary class="article-expander__title">Clinical Implications and Dynamics of Clonal Hematopoiesis in Anti-CD19 CAR T-cell Treated Patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37799345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231008205021&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent evidence revealed important interactions between clonal hematopoiesis (CH) and cellular therapies established for the treatment of hematologic malignancies. The impact of CH on safety, efficacy, and outcome of chimeric antigen receptor (CAR) T-cell therapy is currently under investigation. We analyzed 110 patients with relapsed/refractory B-cell non-Hodgkin lymphoma (n &#x3D; 105) or acute lymphoblastic leukemia (ALL) (n &#x3D; 5), treated with Axicabtagene-Ciloleucel (39%), Tisagenlecleucel (51%),...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37799345"
                        open
                      >
                        <summary class="article-expander__title">Clinical Implications and Dynamics of Clonal Hematopoiesis in Anti-CD19 CAR T-cell Treated Patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37799345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231007205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent evidence revealed important interactions between clonal hematopoiesis (CH) and cellular therapies established for the treatment of hematologic malignancies. The impact of CH on safety, efficacy, and outcome of chimeric antigen receptor (CAR) T-cell therapy is currently under investigation. We analyzed 110 patients with relapsed/refractory B-cell non-Hodgkin lymphoma (n &#x3D; 105) or acute lymphoblastic leukemia (ALL) (n &#x3D; 5), treated with Axicabtagene-Ciloleucel (39%), Tisagenlecleucel (51%),...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37799345"
                        open
                      >
                        <summary class="article-expander__title">Clinical Implications and Dynamics of Clonal Hematopoiesis in Anti-CD19 CAR T-cell Treated Patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37799345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231006204916&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent evidence revealed important interactions between clonal hematopoiesis (CH) and cellular therapies established for the treatment of hematologic malignancies. The impact of CH on safety, efficacy, and outcome of chimeric antigen receptor (CAR) T-cell therapy is currently under investigation. We analyzed 110 patients with relapsed/refractory B-cell non-Hodgkin lymphoma (n &#x3D; 105) or acute lymphoblastic leukemia (ALL) (n &#x3D; 5), treated with Axicabtagene-Ciloleucel (39%), Tisagenlecleucel (51%),...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;ff&#x3D;20231105195156&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37803464"
                        open
                      >
                        <summary class="article-expander__title">Acute myeloid leukemia: from NGS, through scRNA-seq, to CAR-T. dissect cancer heterogeneity and tailor the treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37803464/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231027204802&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is a malignant blood cancer with marked cellular heterogeneity due to altered maturation and differentiation of myeloid blasts, the possible causes of which are transcriptional or epigenetic alterations, impaired apoptosis, and excessive cell proliferation. This neoplasm has a high rate of resistance to anticancer therapies and thus a high risk of relapse and mortality because of both the biological diversity of the patient and intratumoral heterogeneity due to the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37803452"
                        open
                      >
                        <summary class="article-expander__title">Small extrachromosomal circular DNA harboring targeted tumor suppressor gene mutations supports intratumor heterogeneity in mouse liver cancer induced by multiplexed CRISPR/Cas9</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37803452/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231027204802&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Small eccDNAs carrying TSG mutations may serve as an important source supporting intratumor heterogeneity and tumor evolution in mouse liver cancer induced by multiplexed CRISPR/Cas9.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37803464"
                        open
                      >
                        <summary class="article-expander__title">Acute myeloid leukemia: from NGS, through scRNA-seq, to CAR-T. dissect cancer heterogeneity and tailor the treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37803464/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is a malignant blood cancer with marked cellular heterogeneity due to altered maturation and differentiation of myeloid blasts, the possible causes of which are transcriptional or epigenetic alterations, impaired apoptosis, and excessive cell proliferation. This neoplasm has a high rate of resistance to anticancer therapies and thus a high risk of relapse and mortality because of both the biological diversity of the patient and intratumoral heterogeneity due to the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37803452"
                        open
                      >
                        <summary class="article-expander__title">Small extrachromosomal circular DNA harboring targeted tumor suppressor gene mutations supports intratumor heterogeneity in mouse liver cancer induced by multiplexed CRISPR/Cas9</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37803452/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Small eccDNAs carrying TSG mutations may serve as an important source supporting intratumor heterogeneity and tumor evolution in mouse liver cancer induced by multiplexed CRISPR/Cas9.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37803464"
                        open
                      >
                        <summary class="article-expander__title">Acute myeloid leukemia: from NGS, through scRNA-seq, to CAR-T. dissect cancer heterogeneity and tailor the treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37803464/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is a malignant blood cancer with marked cellular heterogeneity due to altered maturation and differentiation of myeloid blasts, the possible causes of which are transcriptional or epigenetic alterations, impaired apoptosis, and excessive cell proliferation. This neoplasm has a high rate of resistance to anticancer therapies and thus a high risk of relapse and mortality because of both the biological diversity of the patient and intratumoral heterogeneity due to the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37803452"
                        open
                      >
                        <summary class="article-expander__title">Small extrachromosomal circular DNA harboring targeted tumor suppressor gene mutations supports intratumor heterogeneity in mouse liver cancer induced by multiplexed CRISPR/Cas9</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37803452/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Small eccDNAs carrying TSG mutations may serve as an important source supporting intratumor heterogeneity and tumor evolution in mouse liver cancer induced by multiplexed CRISPR/Cas9.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37803464"
                        open
                      >
                        <summary class="article-expander__title">Acute myeloid leukemia: from NGS, through scRNA-seq, to CAR-T. dissect cancer heterogeneity and tailor the treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37803464/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is a malignant blood cancer with marked cellular heterogeneity due to altered maturation and differentiation of myeloid blasts, the possible causes of which are transcriptional or epigenetic alterations, impaired apoptosis, and excessive cell proliferation. This neoplasm has a high rate of resistance to anticancer therapies and thus a high risk of relapse and mortality because of both the biological diversity of the patient and intratumoral heterogeneity due to the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37803452"
                        open
                      >
                        <summary class="article-expander__title">Small extrachromosomal circular DNA harboring targeted tumor suppressor gene mutations supports intratumor heterogeneity in mouse liver cancer induced by multiplexed CRISPR/Cas9</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37803452/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Small eccDNAs carrying TSG mutations may serve as an important source supporting intratumor heterogeneity and tumor evolution in mouse liver cancer induced by multiplexed CRISPR/Cas9.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37803464"
                        open
                      >
                        <summary class="article-expander__title">Acute myeloid leukemia: from NGS, through scRNA-seq, to CAR-T. dissect cancer heterogeneity and tailor the treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37803464/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231023204959&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is a malignant blood cancer with marked cellular heterogeneity due to altered maturation and differentiation of myeloid blasts, the possible causes of which are transcriptional or epigenetic alterations, impaired apoptosis, and excessive cell proliferation. This neoplasm has a high rate of resistance to anticancer therapies and thus a high risk of relapse and mortality because of both the biological diversity of the patient and intratumoral heterogeneity due to the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37803452"
                        open
                      >
                        <summary class="article-expander__title">Small extrachromosomal circular DNA harboring targeted tumor suppressor gene mutations supports intratumor heterogeneity in mouse liver cancer induced by multiplexed CRISPR/Cas9</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37803452/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231023204959&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Small eccDNAs carrying TSG mutations may serve as an important source supporting intratumor heterogeneity and tumor evolution in mouse liver cancer induced by multiplexed CRISPR/Cas9.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37803464"
                        open
                      >
                        <summary class="article-expander__title">Acute myeloid leukemia: from NGS, through scRNA-seq, to CAR-T. dissect cancer heterogeneity and tailor the treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37803464/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231022205027&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is a malignant blood cancer with marked cellular heterogeneity due to altered maturation and differentiation of myeloid blasts, the possible causes of which are transcriptional or epigenetic alterations, impaired apoptosis, and excessive cell proliferation. This neoplasm has a high rate of resistance to anticancer therapies and thus a high risk of relapse and mortality because of both the biological diversity of the patient and intratumoral heterogeneity due to the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37803452"
                        open
                      >
                        <summary class="article-expander__title">Small extrachromosomal circular DNA harboring targeted tumor suppressor gene mutations supports intratumor heterogeneity in mouse liver cancer induced by multiplexed CRISPR/Cas9</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37803452/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231022205027&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Small eccDNAs carrying TSG mutations may serve as an important source supporting intratumor heterogeneity and tumor evolution in mouse liver cancer induced by multiplexed CRISPR/Cas9.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37803464"
                        open
                      >
                        <summary class="article-expander__title">Acute myeloid leukemia: from NGS, through scRNA-seq, to CAR-T. dissect cancer heterogeneity and tailor the treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37803464/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231021205333&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is a malignant blood cancer with marked cellular heterogeneity due to altered maturation and differentiation of myeloid blasts, the possible causes of which are transcriptional or epigenetic alterations, impaired apoptosis, and excessive cell proliferation. This neoplasm has a high rate of resistance to anticancer therapies and thus a high risk of relapse and mortality because of both the biological diversity of the patient and intratumoral heterogeneity due to the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37803452"
                        open
                      >
                        <summary class="article-expander__title">Small extrachromosomal circular DNA harboring targeted tumor suppressor gene mutations supports intratumor heterogeneity in mouse liver cancer induced by multiplexed CRISPR/Cas9</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37803452/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231021205333&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Small eccDNAs carrying TSG mutations may serve as an important source supporting intratumor heterogeneity and tumor evolution in mouse liver cancer induced by multiplexed CRISPR/Cas9.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37803464"
                        open
                      >
                        <summary class="article-expander__title">Acute myeloid leukemia: from NGS, through scRNA-seq, to CAR-T. dissect cancer heterogeneity and tailor the treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37803464/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231020204820&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is a malignant blood cancer with marked cellular heterogeneity due to altered maturation and differentiation of myeloid blasts, the possible causes of which are transcriptional or epigenetic alterations, impaired apoptosis, and excessive cell proliferation. This neoplasm has a high rate of resistance to anticancer therapies and thus a high risk of relapse and mortality because of both the biological diversity of the patient and intratumoral heterogeneity due to the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37803452"
                        open
                      >
                        <summary class="article-expander__title">Small extrachromosomal circular DNA harboring targeted tumor suppressor gene mutations supports intratumor heterogeneity in mouse liver cancer induced by multiplexed CRISPR/Cas9</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37803452/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231020204820&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Small eccDNAs carrying TSG mutations may serve as an important source supporting intratumor heterogeneity and tumor evolution in mouse liver cancer induced by multiplexed CRISPR/Cas9.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37803464"
                        open
                      >
                        <summary class="article-expander__title">Acute myeloid leukemia: from NGS, through scRNA-seq, to CAR-T. dissect cancer heterogeneity and tailor the treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37803464/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231019204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is a malignant blood cancer with marked cellular heterogeneity due to altered maturation and differentiation of myeloid blasts, the possible causes of which are transcriptional or epigenetic alterations, impaired apoptosis, and excessive cell proliferation. This neoplasm has a high rate of resistance to anticancer therapies and thus a high risk of relapse and mortality because of both the biological diversity of the patient and intratumoral heterogeneity due to the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37803452"
                        open
                      >
                        <summary class="article-expander__title">Small extrachromosomal circular DNA harboring targeted tumor suppressor gene mutations supports intratumor heterogeneity in mouse liver cancer induced by multiplexed CRISPR/Cas9</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37803452/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231019204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Small eccDNAs carrying TSG mutations may serve as an important source supporting intratumor heterogeneity and tumor evolution in mouse liver cancer induced by multiplexed CRISPR/Cas9.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37803464"
                        open
                      >
                        <summary class="article-expander__title">Acute myeloid leukemia: from NGS, through scRNA-seq, to CAR-T. dissect cancer heterogeneity and tailor the treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37803464/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231018205013&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is a malignant blood cancer with marked cellular heterogeneity due to altered maturation and differentiation of myeloid blasts, the possible causes of which are transcriptional or epigenetic alterations, impaired apoptosis, and excessive cell proliferation. This neoplasm has a high rate of resistance to anticancer therapies and thus a high risk of relapse and mortality because of both the biological diversity of the patient and intratumoral heterogeneity due to the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37803452"
                        open
                      >
                        <summary class="article-expander__title">Small extrachromosomal circular DNA harboring targeted tumor suppressor gene mutations supports intratumor heterogeneity in mouse liver cancer induced by multiplexed CRISPR/Cas9</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37803452/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231018205013&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Small eccDNAs carrying TSG mutations may serve as an important source supporting intratumor heterogeneity and tumor evolution in mouse liver cancer induced by multiplexed CRISPR/Cas9.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37803464"
                        open
                      >
                        <summary class="article-expander__title">Acute myeloid leukemia: from NGS, through scRNA-seq, to CAR-T. dissect cancer heterogeneity and tailor the treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37803464/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231017205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is a malignant blood cancer with marked cellular heterogeneity due to altered maturation and differentiation of myeloid blasts, the possible causes of which are transcriptional or epigenetic alterations, impaired apoptosis, and excessive cell proliferation. This neoplasm has a high rate of resistance to anticancer therapies and thus a high risk of relapse and mortality because of both the biological diversity of the patient and intratumoral heterogeneity due to the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37803452"
                        open
                      >
                        <summary class="article-expander__title">Small extrachromosomal circular DNA harboring targeted tumor suppressor gene mutations supports intratumor heterogeneity in mouse liver cancer induced by multiplexed CRISPR/Cas9</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37803452/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231017205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Small eccDNAs carrying TSG mutations may serve as an important source supporting intratumor heterogeneity and tumor evolution in mouse liver cancer induced by multiplexed CRISPR/Cas9.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37803464"
                        open
                      >
                        <summary class="article-expander__title">Acute myeloid leukemia: from NGS, through scRNA-seq, to CAR-T. dissect cancer heterogeneity and tailor the treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37803464/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231016205028&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is a malignant blood cancer with marked cellular heterogeneity due to altered maturation and differentiation of myeloid blasts, the possible causes of which are transcriptional or epigenetic alterations, impaired apoptosis, and excessive cell proliferation. This neoplasm has a high rate of resistance to anticancer therapies and thus a high risk of relapse and mortality because of both the biological diversity of the patient and intratumoral heterogeneity due to the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37803452"
                        open
                      >
                        <summary class="article-expander__title">Small extrachromosomal circular DNA harboring targeted tumor suppressor gene mutations supports intratumor heterogeneity in mouse liver cancer induced by multiplexed CRISPR/Cas9</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37803452/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231016205028&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Small eccDNAs carrying TSG mutations may serve as an important source supporting intratumor heterogeneity and tumor evolution in mouse liver cancer induced by multiplexed CRISPR/Cas9.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37803464"
                        open
                      >
                        <summary class="article-expander__title">Acute myeloid leukemia: from NGS, through scRNA-seq, to CAR-T. dissect cancer heterogeneity and tailor the treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37803464/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231015205105&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is a malignant blood cancer with marked cellular heterogeneity due to altered maturation and differentiation of myeloid blasts, the possible causes of which are transcriptional or epigenetic alterations, impaired apoptosis, and excessive cell proliferation. This neoplasm has a high rate of resistance to anticancer therapies and thus a high risk of relapse and mortality because of both the biological diversity of the patient and intratumoral heterogeneity due to the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37803452"
                        open
                      >
                        <summary class="article-expander__title">Small extrachromosomal circular DNA harboring targeted tumor suppressor gene mutations supports intratumor heterogeneity in mouse liver cancer induced by multiplexed CRISPR/Cas9</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37803452/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231015205105&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Small eccDNAs carrying TSG mutations may serve as an important source supporting intratumor heterogeneity and tumor evolution in mouse liver cancer induced by multiplexed CRISPR/Cas9.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37803464"
                        open
                      >
                        <summary class="article-expander__title">Acute myeloid leukemia: from NGS, through scRNA-seq, to CAR-T. dissect cancer heterogeneity and tailor the treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37803464/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231014205339&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is a malignant blood cancer with marked cellular heterogeneity due to altered maturation and differentiation of myeloid blasts, the possible causes of which are transcriptional or epigenetic alterations, impaired apoptosis, and excessive cell proliferation. This neoplasm has a high rate of resistance to anticancer therapies and thus a high risk of relapse and mortality because of both the biological diversity of the patient and intratumoral heterogeneity due to the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37803452"
                        open
                      >
                        <summary class="article-expander__title">Small extrachromosomal circular DNA harboring targeted tumor suppressor gene mutations supports intratumor heterogeneity in mouse liver cancer induced by multiplexed CRISPR/Cas9</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37803452/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231014205339&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Small eccDNAs carrying TSG mutations may serve as an important source supporting intratumor heterogeneity and tumor evolution in mouse liver cancer induced by multiplexed CRISPR/Cas9.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37803464"
                        open
                      >
                        <summary class="article-expander__title">Acute myeloid leukemia: from NGS, through scRNA-seq, to CAR-T. dissect cancer heterogeneity and tailor the treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37803464/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231013204816&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is a malignant blood cancer with marked cellular heterogeneity due to altered maturation and differentiation of myeloid blasts, the possible causes of which are transcriptional or epigenetic alterations, impaired apoptosis, and excessive cell proliferation. This neoplasm has a high rate of resistance to anticancer therapies and thus a high risk of relapse and mortality because of both the biological diversity of the patient and intratumoral heterogeneity due to the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37803452"
                        open
                      >
                        <summary class="article-expander__title">Small extrachromosomal circular DNA harboring targeted tumor suppressor gene mutations supports intratumor heterogeneity in mouse liver cancer induced by multiplexed CRISPR/Cas9</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37803452/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231013204816&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Small eccDNAs carrying TSG mutations may serve as an important source supporting intratumor heterogeneity and tumor evolution in mouse liver cancer induced by multiplexed CRISPR/Cas9.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37803464"
                        open
                      >
                        <summary class="article-expander__title">Acute myeloid leukemia: from NGS, through scRNA-seq, to CAR-T. dissect cancer heterogeneity and tailor the treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37803464/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231012205121&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is a malignant blood cancer with marked cellular heterogeneity due to altered maturation and differentiation of myeloid blasts, the possible causes of which are transcriptional or epigenetic alterations, impaired apoptosis, and excessive cell proliferation. This neoplasm has a high rate of resistance to anticancer therapies and thus a high risk of relapse and mortality because of both the biological diversity of the patient and intratumoral heterogeneity due to the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37803452"
                        open
                      >
                        <summary class="article-expander__title">Small extrachromosomal circular DNA harboring targeted tumor suppressor gene mutations supports intratumor heterogeneity in mouse liver cancer induced by multiplexed CRISPR/Cas9</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37803452/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231012205121&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Small eccDNAs carrying TSG mutations may serve as an important source supporting intratumor heterogeneity and tumor evolution in mouse liver cancer induced by multiplexed CRISPR/Cas9.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37803464"
                        open
                      >
                        <summary class="article-expander__title">Acute myeloid leukemia: from NGS, through scRNA-seq, to CAR-T. dissect cancer heterogeneity and tailor the treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37803464/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231011204744&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is a malignant blood cancer with marked cellular heterogeneity due to altered maturation and differentiation of myeloid blasts, the possible causes of which are transcriptional or epigenetic alterations, impaired apoptosis, and excessive cell proliferation. This neoplasm has a high rate of resistance to anticancer therapies and thus a high risk of relapse and mortality because of both the biological diversity of the patient and intratumoral heterogeneity due to the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37803452"
                        open
                      >
                        <summary class="article-expander__title">Small extrachromosomal circular DNA harboring targeted tumor suppressor gene mutations supports intratumor heterogeneity in mouse liver cancer induced by multiplexed CRISPR/Cas9</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37803452/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231011204744&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Small eccDNAs carrying TSG mutations may serve as an important source supporting intratumor heterogeneity and tumor evolution in mouse liver cancer induced by multiplexed CRISPR/Cas9.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37803464"
                        open
                      >
                        <summary class="article-expander__title">Acute myeloid leukemia: from NGS, through scRNA-seq, to CAR-T. dissect cancer heterogeneity and tailor the treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37803464/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231010204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is a malignant blood cancer with marked cellular heterogeneity due to altered maturation and differentiation of myeloid blasts, the possible causes of which are transcriptional or epigenetic alterations, impaired apoptosis, and excessive cell proliferation. This neoplasm has a high rate of resistance to anticancer therapies and thus a high risk of relapse and mortality because of both the biological diversity of the patient and intratumoral heterogeneity due to the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37803452"
                        open
                      >
                        <summary class="article-expander__title">Small extrachromosomal circular DNA harboring targeted tumor suppressor gene mutations supports intratumor heterogeneity in mouse liver cancer induced by multiplexed CRISPR/Cas9</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37803452/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231010204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Small eccDNAs carrying TSG mutations may serve as an important source supporting intratumor heterogeneity and tumor evolution in mouse liver cancer induced by multiplexed CRISPR/Cas9.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37803464"
                        open
                      >
                        <summary class="article-expander__title">Acute myeloid leukemia: from NGS, through scRNA-seq, to CAR-T. dissect cancer heterogeneity and tailor the treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37803464/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231009204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is a malignant blood cancer with marked cellular heterogeneity due to altered maturation and differentiation of myeloid blasts, the possible causes of which are transcriptional or epigenetic alterations, impaired apoptosis, and excessive cell proliferation. This neoplasm has a high rate of resistance to anticancer therapies and thus a high risk of relapse and mortality because of both the biological diversity of the patient and intratumoral heterogeneity due to the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37803452"
                        open
                      >
                        <summary class="article-expander__title">Small extrachromosomal circular DNA harboring targeted tumor suppressor gene mutations supports intratumor heterogeneity in mouse liver cancer induced by multiplexed CRISPR/Cas9</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37803452/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231009204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Small eccDNAs carrying TSG mutations may serve as an important source supporting intratumor heterogeneity and tumor evolution in mouse liver cancer induced by multiplexed CRISPR/Cas9.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37803464"
                        open
                      >
                        <summary class="article-expander__title">Acute myeloid leukemia: from NGS, through scRNA-seq, to CAR-T. dissect cancer heterogeneity and tailor the treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37803464/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231008205021&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is a malignant blood cancer with marked cellular heterogeneity due to altered maturation and differentiation of myeloid blasts, the possible causes of which are transcriptional or epigenetic alterations, impaired apoptosis, and excessive cell proliferation. This neoplasm has a high rate of resistance to anticancer therapies and thus a high risk of relapse and mortality because of both the biological diversity of the patient and intratumoral heterogeneity due to the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37803452"
                        open
                      >
                        <summary class="article-expander__title">Small extrachromosomal circular DNA harboring targeted tumor suppressor gene mutations supports intratumor heterogeneity in mouse liver cancer induced by multiplexed CRISPR/Cas9</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37803452/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231008205021&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Small eccDNAs carrying TSG mutations may serve as an important source supporting intratumor heterogeneity and tumor evolution in mouse liver cancer induced by multiplexed CRISPR/Cas9.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37803464"
                        open
                      >
                        <summary class="article-expander__title">Acute myeloid leukemia: from NGS, through scRNA-seq, to CAR-T. dissect cancer heterogeneity and tailor the treatment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37803464/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231007205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is a malignant blood cancer with marked cellular heterogeneity due to altered maturation and differentiation of myeloid blasts, the possible causes of which are transcriptional or epigenetic alterations, impaired apoptosis, and excessive cell proliferation. This neoplasm has a high rate of resistance to anticancer therapies and thus a high risk of relapse and mortality because of both the biological diversity of the patient and intratumoral heterogeneity due to the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37803452"
                        open
                      >
                        <summary class="article-expander__title">Small extrachromosomal circular DNA harboring targeted tumor suppressor gene mutations supports intratumor heterogeneity in mouse liver cancer induced by multiplexed CRISPR/Cas9</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37803452/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231007205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Small eccDNAs carrying TSG mutations may serve as an important source supporting intratumor heterogeneity and tumor evolution in mouse liver cancer induced by multiplexed CRISPR/Cas9.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-10-05 datetime="2023-10-05T10:00:00.000Z">2023-10-05</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-10-05 datetime="2023-10-05T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;ff&#x3D;20231105195156&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794586"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive genomic characterization of sporadic synchronous colorectal cancer: Implications for treatment optimization and clinical outcome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794586/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Sporadic synchronous colorectal cancer (SCRC) refers to multiple primary CRC tumors detected simultaneously in an individual without predisposing hereditary conditions, which accounts for the majority of multiple CRCs while lacking a profound understanding of the genomic landscape and evolutionary dynamics to optimize its treatment. In this study, 103 primary tumor samples from 51 patients with SCRC undergo whole-exome sequencing. The germline and somatic mutations and evolutionary and clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794586"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive genomic characterization of sporadic synchronous colorectal cancer: Implications for treatment optimization and clinical outcome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794586/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231023204959&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Sporadic synchronous colorectal cancer (SCRC) refers to multiple primary CRC tumors detected simultaneously in an individual without predisposing hereditary conditions, which accounts for the majority of multiple CRCs while lacking a profound understanding of the genomic landscape and evolutionary dynamics to optimize its treatment. In this study, 103 primary tumor samples from 51 patients with SCRC undergo whole-exome sequencing. The germline and somatic mutations and evolutionary and clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794586"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive genomic characterization of sporadic synchronous colorectal cancer: Implications for treatment optimization and clinical outcome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794586/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231022205027&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Sporadic synchronous colorectal cancer (SCRC) refers to multiple primary CRC tumors detected simultaneously in an individual without predisposing hereditary conditions, which accounts for the majority of multiple CRCs while lacking a profound understanding of the genomic landscape and evolutionary dynamics to optimize its treatment. In this study, 103 primary tumor samples from 51 patients with SCRC undergo whole-exome sequencing. The germline and somatic mutations and evolutionary and clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794586"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive genomic characterization of sporadic synchronous colorectal cancer: Implications for treatment optimization and clinical outcome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794586/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231021205333&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Sporadic synchronous colorectal cancer (SCRC) refers to multiple primary CRC tumors detected simultaneously in an individual without predisposing hereditary conditions, which accounts for the majority of multiple CRCs while lacking a profound understanding of the genomic landscape and evolutionary dynamics to optimize its treatment. In this study, 103 primary tumor samples from 51 patients with SCRC undergo whole-exome sequencing. The germline and somatic mutations and evolutionary and clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794586"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive genomic characterization of sporadic synchronous colorectal cancer: Implications for treatment optimization and clinical outcome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794586/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231020204820&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Sporadic synchronous colorectal cancer (SCRC) refers to multiple primary CRC tumors detected simultaneously in an individual without predisposing hereditary conditions, which accounts for the majority of multiple CRCs while lacking a profound understanding of the genomic landscape and evolutionary dynamics to optimize its treatment. In this study, 103 primary tumor samples from 51 patients with SCRC undergo whole-exome sequencing. The germline and somatic mutations and evolutionary and clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794586"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive genomic characterization of sporadic synchronous colorectal cancer: Implications for treatment optimization and clinical outcome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794586/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231019204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Sporadic synchronous colorectal cancer (SCRC) refers to multiple primary CRC tumors detected simultaneously in an individual without predisposing hereditary conditions, which accounts for the majority of multiple CRCs while lacking a profound understanding of the genomic landscape and evolutionary dynamics to optimize its treatment. In this study, 103 primary tumor samples from 51 patients with SCRC undergo whole-exome sequencing. The germline and somatic mutations and evolutionary and clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794586"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive genomic characterization of sporadic synchronous colorectal cancer: Implications for treatment optimization and clinical outcome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794586/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231018205013&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Sporadic synchronous colorectal cancer (SCRC) refers to multiple primary CRC tumors detected simultaneously in an individual without predisposing hereditary conditions, which accounts for the majority of multiple CRCs while lacking a profound understanding of the genomic landscape and evolutionary dynamics to optimize its treatment. In this study, 103 primary tumor samples from 51 patients with SCRC undergo whole-exome sequencing. The germline and somatic mutations and evolutionary and clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794586"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive genomic characterization of sporadic synchronous colorectal cancer: Implications for treatment optimization and clinical outcome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794586/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231017205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Sporadic synchronous colorectal cancer (SCRC) refers to multiple primary CRC tumors detected simultaneously in an individual without predisposing hereditary conditions, which accounts for the majority of multiple CRCs while lacking a profound understanding of the genomic landscape and evolutionary dynamics to optimize its treatment. In this study, 103 primary tumor samples from 51 patients with SCRC undergo whole-exome sequencing. The germline and somatic mutations and evolutionary and clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794586"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive genomic characterization of sporadic synchronous colorectal cancer: Implications for treatment optimization and clinical outcome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794586/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231016205028&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Sporadic synchronous colorectal cancer (SCRC) refers to multiple primary CRC tumors detected simultaneously in an individual without predisposing hereditary conditions, which accounts for the majority of multiple CRCs while lacking a profound understanding of the genomic landscape and evolutionary dynamics to optimize its treatment. In this study, 103 primary tumor samples from 51 patients with SCRC undergo whole-exome sequencing. The germline and somatic mutations and evolutionary and clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794586"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive genomic characterization of sporadic synchronous colorectal cancer: Implications for treatment optimization and clinical outcome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794586/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231015205105&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Sporadic synchronous colorectal cancer (SCRC) refers to multiple primary CRC tumors detected simultaneously in an individual without predisposing hereditary conditions, which accounts for the majority of multiple CRCs while lacking a profound understanding of the genomic landscape and evolutionary dynamics to optimize its treatment. In this study, 103 primary tumor samples from 51 patients with SCRC undergo whole-exome sequencing. The germline and somatic mutations and evolutionary and clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794586"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive genomic characterization of sporadic synchronous colorectal cancer: Implications for treatment optimization and clinical outcome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794586/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231014205339&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Sporadic synchronous colorectal cancer (SCRC) refers to multiple primary CRC tumors detected simultaneously in an individual without predisposing hereditary conditions, which accounts for the majority of multiple CRCs while lacking a profound understanding of the genomic landscape and evolutionary dynamics to optimize its treatment. In this study, 103 primary tumor samples from 51 patients with SCRC undergo whole-exome sequencing. The germline and somatic mutations and evolutionary and clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794586"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive genomic characterization of sporadic synchronous colorectal cancer: Implications for treatment optimization and clinical outcome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794586/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231013204816&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Sporadic synchronous colorectal cancer (SCRC) refers to multiple primary CRC tumors detected simultaneously in an individual without predisposing hereditary conditions, which accounts for the majority of multiple CRCs while lacking a profound understanding of the genomic landscape and evolutionary dynamics to optimize its treatment. In this study, 103 primary tumor samples from 51 patients with SCRC undergo whole-exome sequencing. The germline and somatic mutations and evolutionary and clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794586"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive genomic characterization of sporadic synchronous colorectal cancer: Implications for treatment optimization and clinical outcome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794586/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231012205121&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Sporadic synchronous colorectal cancer (SCRC) refers to multiple primary CRC tumors detected simultaneously in an individual without predisposing hereditary conditions, which accounts for the majority of multiple CRCs while lacking a profound understanding of the genomic landscape and evolutionary dynamics to optimize its treatment. In this study, 103 primary tumor samples from 51 patients with SCRC undergo whole-exome sequencing. The germline and somatic mutations and evolutionary and clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794586"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive genomic characterization of sporadic synchronous colorectal cancer: Implications for treatment optimization and clinical outcome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794586/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231011204744&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Sporadic synchronous colorectal cancer (SCRC) refers to multiple primary CRC tumors detected simultaneously in an individual without predisposing hereditary conditions, which accounts for the majority of multiple CRCs while lacking a profound understanding of the genomic landscape and evolutionary dynamics to optimize its treatment. In this study, 103 primary tumor samples from 51 patients with SCRC undergo whole-exome sequencing. The germline and somatic mutations and evolutionary and clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794586"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive genomic characterization of sporadic synchronous colorectal cancer: Implications for treatment optimization and clinical outcome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794586/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231010204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Sporadic synchronous colorectal cancer (SCRC) refers to multiple primary CRC tumors detected simultaneously in an individual without predisposing hereditary conditions, which accounts for the majority of multiple CRCs while lacking a profound understanding of the genomic landscape and evolutionary dynamics to optimize its treatment. In this study, 103 primary tumor samples from 51 patients with SCRC undergo whole-exome sequencing. The germline and somatic mutations and evolutionary and clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794586"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive genomic characterization of sporadic synchronous colorectal cancer: Implications for treatment optimization and clinical outcome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794586/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231009204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Sporadic synchronous colorectal cancer (SCRC) refers to multiple primary CRC tumors detected simultaneously in an individual without predisposing hereditary conditions, which accounts for the majority of multiple CRCs while lacking a profound understanding of the genomic landscape and evolutionary dynamics to optimize its treatment. In this study, 103 primary tumor samples from 51 patients with SCRC undergo whole-exome sequencing. The germline and somatic mutations and evolutionary and clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794586"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive genomic characterization of sporadic synchronous colorectal cancer: Implications for treatment optimization and clinical outcome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794586/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231008205021&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Sporadic synchronous colorectal cancer (SCRC) refers to multiple primary CRC tumors detected simultaneously in an individual without predisposing hereditary conditions, which accounts for the majority of multiple CRCs while lacking a profound understanding of the genomic landscape and evolutionary dynamics to optimize its treatment. In this study, 103 primary tumor samples from 51 patients with SCRC undergo whole-exome sequencing. The germline and somatic mutations and evolutionary and clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794586"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive genomic characterization of sporadic synchronous colorectal cancer: Implications for treatment optimization and clinical outcome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794586/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231007205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Sporadic synchronous colorectal cancer (SCRC) refers to multiple primary CRC tumors detected simultaneously in an individual without predisposing hereditary conditions, which accounts for the majority of multiple CRCs while lacking a profound understanding of the genomic landscape and evolutionary dynamics to optimize its treatment. In this study, 103 primary tumor samples from 51 patients with SCRC undergo whole-exome sequencing. The germline and somatic mutations and evolutionary and clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794586"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive genomic characterization of sporadic synchronous colorectal cancer: Implications for treatment optimization and clinical outcome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794586/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231006204916&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Sporadic synchronous colorectal cancer (SCRC) refers to multiple primary CRC tumors detected simultaneously in an individual without predisposing hereditary conditions, which accounts for the majority of multiple CRCs while lacking a profound understanding of the genomic landscape and evolutionary dynamics to optimize its treatment. In this study, 103 primary tumor samples from 51 patients with SCRC undergo whole-exome sequencing. The germline and somatic mutations and evolutionary and clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794586"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive genomic characterization of sporadic synchronous colorectal cancer: Implications for treatment optimization and clinical outcome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794586/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231005204957&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Sporadic synchronous colorectal cancer (SCRC) refers to multiple primary CRC tumors detected simultaneously in an individual without predisposing hereditary conditions, which accounts for the majority of multiple CRCs while lacking a profound understanding of the genomic landscape and evolutionary dynamics to optimize its treatment. In this study, 103 primary tumor samples from 51 patients with SCRC undergo whole-exome sequencing. The germline and somatic mutations and evolutionary and clinical...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-10-04 datetime="2023-10-04T10:00:00.000Z">2023-10-04</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-10-04 datetime="2023-10-04T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20231105195156">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794183"
                        open
                      >
                        <summary class="article-expander__title">Rare variant associations with plasma protein levels in the UK Biobank</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794183/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Integrating human genomics and proteomics can help elucidate disease mechanisms, identify clinical biomarkers and discover drug targets^(1-4). Because previous proteogenomic studies have focused on common variation via genome-wide association studies, the contribution of rare variants to the plasma proteome remains largely unknown. Here we identify associations between rare protein-coding variants and 2,923 plasma protein abundances measured in 49,736 UK Biobank individuals. Our variant-level...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793870"
                        open
                      >
                        <summary class="article-expander__title">Myeloid neoplasms with germline predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793870/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With recent advances in sequencing technologies, novel genes associated with the predisposition to myeloid neoplasms have been discovered, and subsequent studies have shown that the incidence of myeloid neoplasms associated with germline variants is higher than expected. Accordingly, myeloid neoplasms with germline predisposition have represented a unique category in the recent WHO classification and the International Consensus Classification, and DDX41 mutation accounts for 2-5% of myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793865"
                        open
                      >
                        <summary class="article-expander__title">Immune pathophysiology of bone marrow failure</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793865/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bone marrow (BM) failure is a condition characterized by peripheral pancytopenia due to decreased BM function. It includes conditions such as acquired aplastic anemia (AA), myelodysplastic syndrome (MDS), and paroxysmal nocturnal hemoglobinuria (PNH). AA is characterized by pancytopenia and BM hypoplasia, and is primarily caused by an autoimmune mechanism involving cytotoxic T cells that damage hematopoietic stem cells (HSCs). Recent genomic research has revealed that patients with AA often...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793870"
                        open
                      >
                        <summary class="article-expander__title">Myeloid neoplasms with germline predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793870/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With recent advances in sequencing technologies, novel genes associated with the predisposition to myeloid neoplasms have been discovered, and subsequent studies have shown that the incidence of myeloid neoplasms associated with germline variants is higher than expected. Accordingly, myeloid neoplasms with germline predisposition have represented a unique category in the recent WHO classification and the International Consensus Classification, and DDX41 mutation accounts for 2-5% of myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793865"
                        open
                      >
                        <summary class="article-expander__title">Immune pathophysiology of bone marrow failure</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793865/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bone marrow (BM) failure is a condition characterized by peripheral pancytopenia due to decreased BM function. It includes conditions such as acquired aplastic anemia (AA), myelodysplastic syndrome (MDS), and paroxysmal nocturnal hemoglobinuria (PNH). AA is characterized by pancytopenia and BM hypoplasia, and is primarily caused by an autoimmune mechanism involving cytotoxic T cells that damage hematopoietic stem cells (HSCs). Recent genomic research has revealed that patients with AA often...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793870"
                        open
                      >
                        <summary class="article-expander__title">Myeloid neoplasms with germline predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793870/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231023204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With recent advances in sequencing technologies, novel genes associated with the predisposition to myeloid neoplasms have been discovered, and subsequent studies have shown that the incidence of myeloid neoplasms associated with germline variants is higher than expected. Accordingly, myeloid neoplasms with germline predisposition have represented a unique category in the recent WHO classification and the International Consensus Classification, and DDX41 mutation accounts for 2-5% of myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793865"
                        open
                      >
                        <summary class="article-expander__title">Immune pathophysiology of bone marrow failure</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793865/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231023204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bone marrow (BM) failure is a condition characterized by peripheral pancytopenia due to decreased BM function. It includes conditions such as acquired aplastic anemia (AA), myelodysplastic syndrome (MDS), and paroxysmal nocturnal hemoglobinuria (PNH). AA is characterized by pancytopenia and BM hypoplasia, and is primarily caused by an autoimmune mechanism involving cytotoxic T cells that damage hematopoietic stem cells (HSCs). Recent genomic research has revealed that patients with AA often...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793870"
                        open
                      >
                        <summary class="article-expander__title">Myeloid neoplasms with germline predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793870/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231022205027&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With recent advances in sequencing technologies, novel genes associated with the predisposition to myeloid neoplasms have been discovered, and subsequent studies have shown that the incidence of myeloid neoplasms associated with germline variants is higher than expected. Accordingly, myeloid neoplasms with germline predisposition have represented a unique category in the recent WHO classification and the International Consensus Classification, and DDX41 mutation accounts for 2-5% of myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793865"
                        open
                      >
                        <summary class="article-expander__title">Immune pathophysiology of bone marrow failure</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793865/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231022205027&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bone marrow (BM) failure is a condition characterized by peripheral pancytopenia due to decreased BM function. It includes conditions such as acquired aplastic anemia (AA), myelodysplastic syndrome (MDS), and paroxysmal nocturnal hemoglobinuria (PNH). AA is characterized by pancytopenia and BM hypoplasia, and is primarily caused by an autoimmune mechanism involving cytotoxic T cells that damage hematopoietic stem cells (HSCs). Recent genomic research has revealed that patients with AA often...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793858"
                        open
                      >
                        <summary class="article-expander__title">Human diseases and the mechanisms regulating myelopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793858/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231022205027&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelopoiesis is a process that produces myeloid cells including granulocytes and mononuclear phagocytes. The differentiation and proliferation of hematopoietic stem and progenitor cells are tightly regulated to meet demands for such myeloid cells both at steady state and under stressed conditions. CCAAT/enhancer-binding protein family transcription factors are involved not only in the appropriate regulation of myelopoiesis but also in dysregulated myelopoiesis. A recent study has revealed that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793870"
                        open
                      >
                        <summary class="article-expander__title">Myeloid neoplasms with germline predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793870/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231021205333&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With recent advances in sequencing technologies, novel genes associated with the predisposition to myeloid neoplasms have been discovered, and subsequent studies have shown that the incidence of myeloid neoplasms associated with germline variants is higher than expected. Accordingly, myeloid neoplasms with germline predisposition have represented a unique category in the recent WHO classification and the International Consensus Classification, and DDX41 mutation accounts for 2-5% of myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793865"
                        open
                      >
                        <summary class="article-expander__title">Immune pathophysiology of bone marrow failure</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793865/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231021205333&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bone marrow (BM) failure is a condition characterized by peripheral pancytopenia due to decreased BM function. It includes conditions such as acquired aplastic anemia (AA), myelodysplastic syndrome (MDS), and paroxysmal nocturnal hemoglobinuria (PNH). AA is characterized by pancytopenia and BM hypoplasia, and is primarily caused by an autoimmune mechanism involving cytotoxic T cells that damage hematopoietic stem cells (HSCs). Recent genomic research has revealed that patients with AA often...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793858"
                        open
                      >
                        <summary class="article-expander__title">Human diseases and the mechanisms regulating myelopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793858/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231021205333&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelopoiesis is a process that produces myeloid cells including granulocytes and mononuclear phagocytes. The differentiation and proliferation of hematopoietic stem and progenitor cells are tightly regulated to meet demands for such myeloid cells both at steady state and under stressed conditions. CCAAT/enhancer-binding protein family transcription factors are involved not only in the appropriate regulation of myelopoiesis but also in dysregulated myelopoiesis. A recent study has revealed that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793870"
                        open
                      >
                        <summary class="article-expander__title">Myeloid neoplasms with germline predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793870/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231020204820&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With recent advances in sequencing technologies, novel genes associated with the predisposition to myeloid neoplasms have been discovered, and subsequent studies have shown that the incidence of myeloid neoplasms associated with germline variants is higher than expected. Accordingly, myeloid neoplasms with germline predisposition have represented a unique category in the recent WHO classification and the International Consensus Classification, and DDX41 mutation accounts for 2-5% of myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793865"
                        open
                      >
                        <summary class="article-expander__title">Immune pathophysiology of bone marrow failure</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793865/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231020204820&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bone marrow (BM) failure is a condition characterized by peripheral pancytopenia due to decreased BM function. It includes conditions such as acquired aplastic anemia (AA), myelodysplastic syndrome (MDS), and paroxysmal nocturnal hemoglobinuria (PNH). AA is characterized by pancytopenia and BM hypoplasia, and is primarily caused by an autoimmune mechanism involving cytotoxic T cells that damage hematopoietic stem cells (HSCs). Recent genomic research has revealed that patients with AA often...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793858"
                        open
                      >
                        <summary class="article-expander__title">Human diseases and the mechanisms regulating myelopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793858/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231020204820&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelopoiesis is a process that produces myeloid cells including granulocytes and mononuclear phagocytes. The differentiation and proliferation of hematopoietic stem and progenitor cells are tightly regulated to meet demands for such myeloid cells both at steady state and under stressed conditions. CCAAT/enhancer-binding protein family transcription factors are involved not only in the appropriate regulation of myelopoiesis but also in dysregulated myelopoiesis. A recent study has revealed that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793870"
                        open
                      >
                        <summary class="article-expander__title">Myeloid neoplasms with germline predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793870/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231019204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With recent advances in sequencing technologies, novel genes associated with the predisposition to myeloid neoplasms have been discovered, and subsequent studies have shown that the incidence of myeloid neoplasms associated with germline variants is higher than expected. Accordingly, myeloid neoplasms with germline predisposition have represented a unique category in the recent WHO classification and the International Consensus Classification, and DDX41 mutation accounts for 2-5% of myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793865"
                        open
                      >
                        <summary class="article-expander__title">Immune pathophysiology of bone marrow failure</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793865/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231019204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bone marrow (BM) failure is a condition characterized by peripheral pancytopenia due to decreased BM function. It includes conditions such as acquired aplastic anemia (AA), myelodysplastic syndrome (MDS), and paroxysmal nocturnal hemoglobinuria (PNH). AA is characterized by pancytopenia and BM hypoplasia, and is primarily caused by an autoimmune mechanism involving cytotoxic T cells that damage hematopoietic stem cells (HSCs). Recent genomic research has revealed that patients with AA often...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793858"
                        open
                      >
                        <summary class="article-expander__title">Human diseases and the mechanisms regulating myelopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793858/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231019204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelopoiesis is a process that produces myeloid cells including granulocytes and mononuclear phagocytes. The differentiation and proliferation of hematopoietic stem and progenitor cells are tightly regulated to meet demands for such myeloid cells both at steady state and under stressed conditions. CCAAT/enhancer-binding protein family transcription factors are involved not only in the appropriate regulation of myelopoiesis but also in dysregulated myelopoiesis. A recent study has revealed that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793870"
                        open
                      >
                        <summary class="article-expander__title">Myeloid neoplasms with germline predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793870/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231018205013&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With recent advances in sequencing technologies, novel genes associated with the predisposition to myeloid neoplasms have been discovered, and subsequent studies have shown that the incidence of myeloid neoplasms associated with germline variants is higher than expected. Accordingly, myeloid neoplasms with germline predisposition have represented a unique category in the recent WHO classification and the International Consensus Classification, and DDX41 mutation accounts for 2-5% of myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793865"
                        open
                      >
                        <summary class="article-expander__title">Immune pathophysiology of bone marrow failure</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793865/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231018205013&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bone marrow (BM) failure is a condition characterized by peripheral pancytopenia due to decreased BM function. It includes conditions such as acquired aplastic anemia (AA), myelodysplastic syndrome (MDS), and paroxysmal nocturnal hemoglobinuria (PNH). AA is characterized by pancytopenia and BM hypoplasia, and is primarily caused by an autoimmune mechanism involving cytotoxic T cells that damage hematopoietic stem cells (HSCs). Recent genomic research has revealed that patients with AA often...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793858"
                        open
                      >
                        <summary class="article-expander__title">Human diseases and the mechanisms regulating myelopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793858/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231018205013&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelopoiesis is a process that produces myeloid cells including granulocytes and mononuclear phagocytes. The differentiation and proliferation of hematopoietic stem and progenitor cells are tightly regulated to meet demands for such myeloid cells both at steady state and under stressed conditions. CCAAT/enhancer-binding protein family transcription factors are involved not only in the appropriate regulation of myelopoiesis but also in dysregulated myelopoiesis. A recent study has revealed that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793870"
                        open
                      >
                        <summary class="article-expander__title">Myeloid neoplasms with germline predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793870/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231017205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With recent advances in sequencing technologies, novel genes associated with the predisposition to myeloid neoplasms have been discovered, and subsequent studies have shown that the incidence of myeloid neoplasms associated with germline variants is higher than expected. Accordingly, myeloid neoplasms with germline predisposition have represented a unique category in the recent WHO classification and the International Consensus Classification, and DDX41 mutation accounts for 2-5% of myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793865"
                        open
                      >
                        <summary class="article-expander__title">Immune pathophysiology of bone marrow failure</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793865/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231017205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bone marrow (BM) failure is a condition characterized by peripheral pancytopenia due to decreased BM function. It includes conditions such as acquired aplastic anemia (AA), myelodysplastic syndrome (MDS), and paroxysmal nocturnal hemoglobinuria (PNH). AA is characterized by pancytopenia and BM hypoplasia, and is primarily caused by an autoimmune mechanism involving cytotoxic T cells that damage hematopoietic stem cells (HSCs). Recent genomic research has revealed that patients with AA often...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793858"
                        open
                      >
                        <summary class="article-expander__title">Human diseases and the mechanisms regulating myelopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793858/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231017205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelopoiesis is a process that produces myeloid cells including granulocytes and mononuclear phagocytes. The differentiation and proliferation of hematopoietic stem and progenitor cells are tightly regulated to meet demands for such myeloid cells both at steady state and under stressed conditions. CCAAT/enhancer-binding protein family transcription factors are involved not only in the appropriate regulation of myelopoiesis but also in dysregulated myelopoiesis. A recent study has revealed that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793870"
                        open
                      >
                        <summary class="article-expander__title">Myeloid neoplasms with germline predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793870/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231016205028&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With recent advances in sequencing technologies, novel genes associated with the predisposition to myeloid neoplasms have been discovered, and subsequent studies have shown that the incidence of myeloid neoplasms associated with germline variants is higher than expected. Accordingly, myeloid neoplasms with germline predisposition have represented a unique category in the recent WHO classification and the International Consensus Classification, and DDX41 mutation accounts for 2-5% of myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793865"
                        open
                      >
                        <summary class="article-expander__title">Immune pathophysiology of bone marrow failure</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793865/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231016205028&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bone marrow (BM) failure is a condition characterized by peripheral pancytopenia due to decreased BM function. It includes conditions such as acquired aplastic anemia (AA), myelodysplastic syndrome (MDS), and paroxysmal nocturnal hemoglobinuria (PNH). AA is characterized by pancytopenia and BM hypoplasia, and is primarily caused by an autoimmune mechanism involving cytotoxic T cells that damage hematopoietic stem cells (HSCs). Recent genomic research has revealed that patients with AA often...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793858"
                        open
                      >
                        <summary class="article-expander__title">Human diseases and the mechanisms regulating myelopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793858/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231016205028&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelopoiesis is a process that produces myeloid cells including granulocytes and mononuclear phagocytes. The differentiation and proliferation of hematopoietic stem and progenitor cells are tightly regulated to meet demands for such myeloid cells both at steady state and under stressed conditions. CCAAT/enhancer-binding protein family transcription factors are involved not only in the appropriate regulation of myelopoiesis but also in dysregulated myelopoiesis. A recent study has revealed that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793870"
                        open
                      >
                        <summary class="article-expander__title">Myeloid neoplasms with germline predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793870/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231015205105&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With recent advances in sequencing technologies, novel genes associated with the predisposition to myeloid neoplasms have been discovered, and subsequent studies have shown that the incidence of myeloid neoplasms associated with germline variants is higher than expected. Accordingly, myeloid neoplasms with germline predisposition have represented a unique category in the recent WHO classification and the International Consensus Classification, and DDX41 mutation accounts for 2-5% of myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793865"
                        open
                      >
                        <summary class="article-expander__title">Immune pathophysiology of bone marrow failure</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793865/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231015205105&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bone marrow (BM) failure is a condition characterized by peripheral pancytopenia due to decreased BM function. It includes conditions such as acquired aplastic anemia (AA), myelodysplastic syndrome (MDS), and paroxysmal nocturnal hemoglobinuria (PNH). AA is characterized by pancytopenia and BM hypoplasia, and is primarily caused by an autoimmune mechanism involving cytotoxic T cells that damage hematopoietic stem cells (HSCs). Recent genomic research has revealed that patients with AA often...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793858"
                        open
                      >
                        <summary class="article-expander__title">Human diseases and the mechanisms regulating myelopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793858/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231015205105&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelopoiesis is a process that produces myeloid cells including granulocytes and mononuclear phagocytes. The differentiation and proliferation of hematopoietic stem and progenitor cells are tightly regulated to meet demands for such myeloid cells both at steady state and under stressed conditions. CCAAT/enhancer-binding protein family transcription factors are involved not only in the appropriate regulation of myelopoiesis but also in dysregulated myelopoiesis. A recent study has revealed that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793870"
                        open
                      >
                        <summary class="article-expander__title">Myeloid neoplasms with germline predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793870/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231014205339&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With recent advances in sequencing technologies, novel genes associated with the predisposition to myeloid neoplasms have been discovered, and subsequent studies have shown that the incidence of myeloid neoplasms associated with germline variants is higher than expected. Accordingly, myeloid neoplasms with germline predisposition have represented a unique category in the recent WHO classification and the International Consensus Classification, and DDX41 mutation accounts for 2-5% of myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793865"
                        open
                      >
                        <summary class="article-expander__title">Immune pathophysiology of bone marrow failure</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793865/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231014205339&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bone marrow (BM) failure is a condition characterized by peripheral pancytopenia due to decreased BM function. It includes conditions such as acquired aplastic anemia (AA), myelodysplastic syndrome (MDS), and paroxysmal nocturnal hemoglobinuria (PNH). AA is characterized by pancytopenia and BM hypoplasia, and is primarily caused by an autoimmune mechanism involving cytotoxic T cells that damage hematopoietic stem cells (HSCs). Recent genomic research has revealed that patients with AA often...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793858"
                        open
                      >
                        <summary class="article-expander__title">Human diseases and the mechanisms regulating myelopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793858/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231014205339&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelopoiesis is a process that produces myeloid cells including granulocytes and mononuclear phagocytes. The differentiation and proliferation of hematopoietic stem and progenitor cells are tightly regulated to meet demands for such myeloid cells both at steady state and under stressed conditions. CCAAT/enhancer-binding protein family transcription factors are involved not only in the appropriate regulation of myelopoiesis but also in dysregulated myelopoiesis. A recent study has revealed that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793870"
                        open
                      >
                        <summary class="article-expander__title">Myeloid neoplasms with germline predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793870/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231013204816&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With recent advances in sequencing technologies, novel genes associated with the predisposition to myeloid neoplasms have been discovered, and subsequent studies have shown that the incidence of myeloid neoplasms associated with germline variants is higher than expected. Accordingly, myeloid neoplasms with germline predisposition have represented a unique category in the recent WHO classification and the International Consensus Classification, and DDX41 mutation accounts for 2-5% of myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793865"
                        open
                      >
                        <summary class="article-expander__title">Immune pathophysiology of bone marrow failure</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793865/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231013204816&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bone marrow (BM) failure is a condition characterized by peripheral pancytopenia due to decreased BM function. It includes conditions such as acquired aplastic anemia (AA), myelodysplastic syndrome (MDS), and paroxysmal nocturnal hemoglobinuria (PNH). AA is characterized by pancytopenia and BM hypoplasia, and is primarily caused by an autoimmune mechanism involving cytotoxic T cells that damage hematopoietic stem cells (HSCs). Recent genomic research has revealed that patients with AA often...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793858"
                        open
                      >
                        <summary class="article-expander__title">Human diseases and the mechanisms regulating myelopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793858/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231013204816&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelopoiesis is a process that produces myeloid cells including granulocytes and mononuclear phagocytes. The differentiation and proliferation of hematopoietic stem and progenitor cells are tightly regulated to meet demands for such myeloid cells both at steady state and under stressed conditions. CCAAT/enhancer-binding protein family transcription factors are involved not only in the appropriate regulation of myelopoiesis but also in dysregulated myelopoiesis. A recent study has revealed that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793870"
                        open
                      >
                        <summary class="article-expander__title">Myeloid neoplasms with germline predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793870/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231012205120&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With recent advances in sequencing technologies, novel genes associated with the predisposition to myeloid neoplasms have been discovered, and subsequent studies have shown that the incidence of myeloid neoplasms associated with germline variants is higher than expected. Accordingly, myeloid neoplasms with germline predisposition have represented a unique category in the recent WHO classification and the International Consensus Classification, and DDX41 mutation accounts for 2-5% of myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793865"
                        open
                      >
                        <summary class="article-expander__title">Immune pathophysiology of bone marrow failure</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793865/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231012205120&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bone marrow (BM) failure is a condition characterized by peripheral pancytopenia due to decreased BM function. It includes conditions such as acquired aplastic anemia (AA), myelodysplastic syndrome (MDS), and paroxysmal nocturnal hemoglobinuria (PNH). AA is characterized by pancytopenia and BM hypoplasia, and is primarily caused by an autoimmune mechanism involving cytotoxic T cells that damage hematopoietic stem cells (HSCs). Recent genomic research has revealed that patients with AA often...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793858"
                        open
                      >
                        <summary class="article-expander__title">Human diseases and the mechanisms regulating myelopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793858/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231012205120&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelopoiesis is a process that produces myeloid cells including granulocytes and mononuclear phagocytes. The differentiation and proliferation of hematopoietic stem and progenitor cells are tightly regulated to meet demands for such myeloid cells both at steady state and under stressed conditions. CCAAT/enhancer-binding protein family transcription factors are involved not only in the appropriate regulation of myelopoiesis but also in dysregulated myelopoiesis. A recent study has revealed that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793870"
                        open
                      >
                        <summary class="article-expander__title">Myeloid neoplasms with germline predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793870/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231011204744&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With recent advances in sequencing technologies, novel genes associated with the predisposition to myeloid neoplasms have been discovered, and subsequent studies have shown that the incidence of myeloid neoplasms associated with germline variants is higher than expected. Accordingly, myeloid neoplasms with germline predisposition have represented a unique category in the recent WHO classification and the International Consensus Classification, and DDX41 mutation accounts for 2-5% of myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793865"
                        open
                      >
                        <summary class="article-expander__title">Immune pathophysiology of bone marrow failure</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793865/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231011204744&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bone marrow (BM) failure is a condition characterized by peripheral pancytopenia due to decreased BM function. It includes conditions such as acquired aplastic anemia (AA), myelodysplastic syndrome (MDS), and paroxysmal nocturnal hemoglobinuria (PNH). AA is characterized by pancytopenia and BM hypoplasia, and is primarily caused by an autoimmune mechanism involving cytotoxic T cells that damage hematopoietic stem cells (HSCs). Recent genomic research has revealed that patients with AA often...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793858"
                        open
                      >
                        <summary class="article-expander__title">Human diseases and the mechanisms regulating myelopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793858/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231011204744&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelopoiesis is a process that produces myeloid cells including granulocytes and mononuclear phagocytes. The differentiation and proliferation of hematopoietic stem and progenitor cells are tightly regulated to meet demands for such myeloid cells both at steady state and under stressed conditions. CCAAT/enhancer-binding protein family transcription factors are involved not only in the appropriate regulation of myelopoiesis but also in dysregulated myelopoiesis. A recent study has revealed that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793870"
                        open
                      >
                        <summary class="article-expander__title">Myeloid neoplasms with germline predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793870/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231010204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With recent advances in sequencing technologies, novel genes associated with the predisposition to myeloid neoplasms have been discovered, and subsequent studies have shown that the incidence of myeloid neoplasms associated with germline variants is higher than expected. Accordingly, myeloid neoplasms with germline predisposition have represented a unique category in the recent WHO classification and the International Consensus Classification, and DDX41 mutation accounts for 2-5% of myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793865"
                        open
                      >
                        <summary class="article-expander__title">Immune pathophysiology of bone marrow failure</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793865/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231010204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bone marrow (BM) failure is a condition characterized by peripheral pancytopenia due to decreased BM function. It includes conditions such as acquired aplastic anemia (AA), myelodysplastic syndrome (MDS), and paroxysmal nocturnal hemoglobinuria (PNH). AA is characterized by pancytopenia and BM hypoplasia, and is primarily caused by an autoimmune mechanism involving cytotoxic T cells that damage hematopoietic stem cells (HSCs). Recent genomic research has revealed that patients with AA often...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793858"
                        open
                      >
                        <summary class="article-expander__title">Human diseases and the mechanisms regulating myelopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793858/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231010204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelopoiesis is a process that produces myeloid cells including granulocytes and mononuclear phagocytes. The differentiation and proliferation of hematopoietic stem and progenitor cells are tightly regulated to meet demands for such myeloid cells both at steady state and under stressed conditions. CCAAT/enhancer-binding protein family transcription factors are involved not only in the appropriate regulation of myelopoiesis but also in dysregulated myelopoiesis. A recent study has revealed that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793870"
                        open
                      >
                        <summary class="article-expander__title">Myeloid neoplasms with germline predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793870/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231009204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With recent advances in sequencing technologies, novel genes associated with the predisposition to myeloid neoplasms have been discovered, and subsequent studies have shown that the incidence of myeloid neoplasms associated with germline variants is higher than expected. Accordingly, myeloid neoplasms with germline predisposition have represented a unique category in the recent WHO classification and the International Consensus Classification, and DDX41 mutation accounts for 2-5% of myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793865"
                        open
                      >
                        <summary class="article-expander__title">Immune pathophysiology of bone marrow failure</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793865/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231009204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bone marrow (BM) failure is a condition characterized by peripheral pancytopenia due to decreased BM function. It includes conditions such as acquired aplastic anemia (AA), myelodysplastic syndrome (MDS), and paroxysmal nocturnal hemoglobinuria (PNH). AA is characterized by pancytopenia and BM hypoplasia, and is primarily caused by an autoimmune mechanism involving cytotoxic T cells that damage hematopoietic stem cells (HSCs). Recent genomic research has revealed that patients with AA often...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793858"
                        open
                      >
                        <summary class="article-expander__title">Human diseases and the mechanisms regulating myelopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793858/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231009204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelopoiesis is a process that produces myeloid cells including granulocytes and mononuclear phagocytes. The differentiation and proliferation of hematopoietic stem and progenitor cells are tightly regulated to meet demands for such myeloid cells both at steady state and under stressed conditions. CCAAT/enhancer-binding protein family transcription factors are involved not only in the appropriate regulation of myelopoiesis but also in dysregulated myelopoiesis. A recent study has revealed that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793870"
                        open
                      >
                        <summary class="article-expander__title">Myeloid neoplasms with germline predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793870/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231008205021&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With recent advances in sequencing technologies, novel genes associated with the predisposition to myeloid neoplasms have been discovered, and subsequent studies have shown that the incidence of myeloid neoplasms associated with germline variants is higher than expected. Accordingly, myeloid neoplasms with germline predisposition have represented a unique category in the recent WHO classification and the International Consensus Classification, and DDX41 mutation accounts for 2-5% of myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793865"
                        open
                      >
                        <summary class="article-expander__title">Immune pathophysiology of bone marrow failure</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793865/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231008205021&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bone marrow (BM) failure is a condition characterized by peripheral pancytopenia due to decreased BM function. It includes conditions such as acquired aplastic anemia (AA), myelodysplastic syndrome (MDS), and paroxysmal nocturnal hemoglobinuria (PNH). AA is characterized by pancytopenia and BM hypoplasia, and is primarily caused by an autoimmune mechanism involving cytotoxic T cells that damage hematopoietic stem cells (HSCs). Recent genomic research has revealed that patients with AA often...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793858"
                        open
                      >
                        <summary class="article-expander__title">Human diseases and the mechanisms regulating myelopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793858/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231008205021&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelopoiesis is a process that produces myeloid cells including granulocytes and mononuclear phagocytes. The differentiation and proliferation of hematopoietic stem and progenitor cells are tightly regulated to meet demands for such myeloid cells both at steady state and under stressed conditions. CCAAT/enhancer-binding protein family transcription factors are involved not only in the appropriate regulation of myelopoiesis but also in dysregulated myelopoiesis. A recent study has revealed that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793870"
                        open
                      >
                        <summary class="article-expander__title">Myeloid neoplasms with germline predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793870/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231007205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With recent advances in sequencing technologies, novel genes associated with the predisposition to myeloid neoplasms have been discovered, and subsequent studies have shown that the incidence of myeloid neoplasms associated with germline variants is higher than expected. Accordingly, myeloid neoplasms with germline predisposition have represented a unique category in the recent WHO classification and the International Consensus Classification, and DDX41 mutation accounts for 2-5% of myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793865"
                        open
                      >
                        <summary class="article-expander__title">Immune pathophysiology of bone marrow failure</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793865/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231007205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bone marrow (BM) failure is a condition characterized by peripheral pancytopenia due to decreased BM function. It includes conditions such as acquired aplastic anemia (AA), myelodysplastic syndrome (MDS), and paroxysmal nocturnal hemoglobinuria (PNH). AA is characterized by pancytopenia and BM hypoplasia, and is primarily caused by an autoimmune mechanism involving cytotoxic T cells that damage hematopoietic stem cells (HSCs). Recent genomic research has revealed that patients with AA often...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793858"
                        open
                      >
                        <summary class="article-expander__title">Human diseases and the mechanisms regulating myelopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793858/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231007205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelopoiesis is a process that produces myeloid cells including granulocytes and mononuclear phagocytes. The differentiation and proliferation of hematopoietic stem and progenitor cells are tightly regulated to meet demands for such myeloid cells both at steady state and under stressed conditions. CCAAT/enhancer-binding protein family transcription factors are involved not only in the appropriate regulation of myelopoiesis but also in dysregulated myelopoiesis. A recent study has revealed that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793870"
                        open
                      >
                        <summary class="article-expander__title">Myeloid neoplasms with germline predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793870/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231006204916&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With recent advances in sequencing technologies, novel genes associated with the predisposition to myeloid neoplasms have been discovered, and subsequent studies have shown that the incidence of myeloid neoplasms associated with germline variants is higher than expected. Accordingly, myeloid neoplasms with germline predisposition have represented a unique category in the recent WHO classification and the International Consensus Classification, and DDX41 mutation accounts for 2-5% of myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793865"
                        open
                      >
                        <summary class="article-expander__title">Immune pathophysiology of bone marrow failure</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793865/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231006204916&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bone marrow (BM) failure is a condition characterized by peripheral pancytopenia due to decreased BM function. It includes conditions such as acquired aplastic anemia (AA), myelodysplastic syndrome (MDS), and paroxysmal nocturnal hemoglobinuria (PNH). AA is characterized by pancytopenia and BM hypoplasia, and is primarily caused by an autoimmune mechanism involving cytotoxic T cells that damage hematopoietic stem cells (HSCs). Recent genomic research has revealed that patients with AA often...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793858"
                        open
                      >
                        <summary class="article-expander__title">Human diseases and the mechanisms regulating myelopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793858/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231006204916&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelopoiesis is a process that produces myeloid cells including granulocytes and mononuclear phagocytes. The differentiation and proliferation of hematopoietic stem and progenitor cells are tightly regulated to meet demands for such myeloid cells both at steady state and under stressed conditions. CCAAT/enhancer-binding protein family transcription factors are involved not only in the appropriate regulation of myelopoiesis but also in dysregulated myelopoiesis. A recent study has revealed that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793870"
                        open
                      >
                        <summary class="article-expander__title">Myeloid neoplasms with germline predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793870/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231005204957&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With recent advances in sequencing technologies, novel genes associated with the predisposition to myeloid neoplasms have been discovered, and subsequent studies have shown that the incidence of myeloid neoplasms associated with germline variants is higher than expected. Accordingly, myeloid neoplasms with germline predisposition have represented a unique category in the recent WHO classification and the International Consensus Classification, and DDX41 mutation accounts for 2-5% of myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793865"
                        open
                      >
                        <summary class="article-expander__title">Immune pathophysiology of bone marrow failure</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793865/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231005204957&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bone marrow (BM) failure is a condition characterized by peripheral pancytopenia due to decreased BM function. It includes conditions such as acquired aplastic anemia (AA), myelodysplastic syndrome (MDS), and paroxysmal nocturnal hemoglobinuria (PNH). AA is characterized by pancytopenia and BM hypoplasia, and is primarily caused by an autoimmune mechanism involving cytotoxic T cells that damage hematopoietic stem cells (HSCs). Recent genomic research has revealed that patients with AA often...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793858"
                        open
                      >
                        <summary class="article-expander__title">Human diseases and the mechanisms regulating myelopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793858/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231005204957&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelopoiesis is a process that produces myeloid cells including granulocytes and mononuclear phagocytes. The differentiation and proliferation of hematopoietic stem and progenitor cells are tightly regulated to meet demands for such myeloid cells both at steady state and under stressed conditions. CCAAT/enhancer-binding protein family transcription factors are involved not only in the appropriate regulation of myelopoiesis but also in dysregulated myelopoiesis. A recent study has revealed that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;ff&#x3D;20231105195156&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794100"
                        open
                      >
                        <summary class="article-expander__title">Deciphering treatment patterns in non-severe/moderate aplastic anemia: an international observational study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794100/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231023204959&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Non-severe aplastic anemia is a rare bone marrow failure disorder characterized by variable degrees and combination of cytopenias, with limited data on management and outcome. We describe a large multicentric series of 259 patients, focusing on clinical and molecular features, treatment, evolution, and survival. The majority required treatment with cyclosporine (CyA) alone (N &#x3D; 84) or in combination with anti-thymocyte globulin (ATG,44) or eltrombopag (20), eltrombopag alone (10), or others (25)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794034"
                        open
                      >
                        <summary class="article-expander__title">Genomic signatures of past and present chromosomal instability in Barrett&#x27;s esophagus and early esophageal adenocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794034/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231023204959&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The progression of precancerous lesions to malignancy is often accompanied by increasing complexity of chromosomal alterations but how these alterations arise is poorly understood. Here we perform haplotype-specific analysis of chromosomal copy-number evolution in the progression of Barrett&#x27;s esophagus (BE) to esophageal adenocarcinoma (EAC) on multiregional whole-genome sequencing data of BE with dysplasia and microscopic EAC foci. We identify distinct patterns of copy-number evolution...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;17
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793870"
                        open
                      >
                        <summary class="article-expander__title">Myeloid neoplasms with germline predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793870/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231023204959&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With recent advances in sequencing technologies, novel genes associated with the predisposition to myeloid neoplasms have been discovered, and subsequent studies have shown that the incidence of myeloid neoplasms associated with germline variants is higher than expected. Accordingly, myeloid neoplasms with germline predisposition have represented a unique category in the recent WHO classification and the International Consensus Classification, and DDX41 mutation accounts for 2-5% of myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37792958"
                        open
                      >
                        <summary class="article-expander__title">Identifying predictors of glioma evolution from longitudinal sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37792958/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231023204959&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution drives cancer progression and therapeutic resistance. Recent studies have revealed divergent longitudinal trajectories in gliomas, but early molecular features steering posttreatment cancer evolution remain unclear. Here, we collected sequencing and clinical data of initial-recurrent tumor pairs from 544 adult diffuse gliomas and performed multivariate analysis to identify early molecular predictors of tumor evolution in three diffuse glioma subtypes. We found that CDKN2A...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794100"
                        open
                      >
                        <summary class="article-expander__title">Deciphering treatment patterns in non-severe/moderate aplastic anemia: an international observational study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794100/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231022205027&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Non-severe aplastic anemia is a rare bone marrow failure disorder characterized by variable degrees and combination of cytopenias, with limited data on management and outcome. We describe a large multicentric series of 259 patients, focusing on clinical and molecular features, treatment, evolution, and survival. The majority required treatment with cyclosporine (CyA) alone (N &#x3D; 84) or in combination with anti-thymocyte globulin (ATG,44) or eltrombopag (20), eltrombopag alone (10), or others (25)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794034"
                        open
                      >
                        <summary class="article-expander__title">Genomic signatures of past and present chromosomal instability in Barrett&#x27;s esophagus and early esophageal adenocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794034/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231022205027&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The progression of precancerous lesions to malignancy is often accompanied by increasing complexity of chromosomal alterations but how these alterations arise is poorly understood. Here we perform haplotype-specific analysis of chromosomal copy-number evolution in the progression of Barrett&#x27;s esophagus (BE) to esophageal adenocarcinoma (EAC) on multiregional whole-genome sequencing data of BE with dysplasia and microscopic EAC foci. We identify distinct patterns of copy-number evolution...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;17
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793870"
                        open
                      >
                        <summary class="article-expander__title">Myeloid neoplasms with germline predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793870/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231022205027&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With recent advances in sequencing technologies, novel genes associated with the predisposition to myeloid neoplasms have been discovered, and subsequent studies have shown that the incidence of myeloid neoplasms associated with germline variants is higher than expected. Accordingly, myeloid neoplasms with germline predisposition have represented a unique category in the recent WHO classification and the International Consensus Classification, and DDX41 mutation accounts for 2-5% of myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37792958"
                        open
                      >
                        <summary class="article-expander__title">Identifying predictors of glioma evolution from longitudinal sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37792958/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231022205027&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution drives cancer progression and therapeutic resistance. Recent studies have revealed divergent longitudinal trajectories in gliomas, but early molecular features steering posttreatment cancer evolution remain unclear. Here, we collected sequencing and clinical data of initial-recurrent tumor pairs from 544 adult diffuse gliomas and performed multivariate analysis to identify early molecular predictors of tumor evolution in three diffuse glioma subtypes. We found that CDKN2A...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794100"
                        open
                      >
                        <summary class="article-expander__title">Deciphering treatment patterns in non-severe/moderate aplastic anemia: an international observational study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794100/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231021205333&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Non-severe aplastic anemia is a rare bone marrow failure disorder characterized by variable degrees and combination of cytopenias, with limited data on management and outcome. We describe a large multicentric series of 259 patients, focusing on clinical and molecular features, treatment, evolution, and survival. The majority required treatment with cyclosporine (CyA) alone (N &#x3D; 84) or in combination with anti-thymocyte globulin (ATG,44) or eltrombopag (20), eltrombopag alone (10), or others (25)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794034"
                        open
                      >
                        <summary class="article-expander__title">Genomic signatures of past and present chromosomal instability in Barrett&#x27;s esophagus and early esophageal adenocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794034/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231021205333&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The progression of precancerous lesions to malignancy is often accompanied by increasing complexity of chromosomal alterations but how these alterations arise is poorly understood. Here we perform haplotype-specific analysis of chromosomal copy-number evolution in the progression of Barrett&#x27;s esophagus (BE) to esophageal adenocarcinoma (EAC) on multiregional whole-genome sequencing data of BE with dysplasia and microscopic EAC foci. We identify distinct patterns of copy-number evolution...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;17
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793870"
                        open
                      >
                        <summary class="article-expander__title">Myeloid neoplasms with germline predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793870/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231021205333&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With recent advances in sequencing technologies, novel genes associated with the predisposition to myeloid neoplasms have been discovered, and subsequent studies have shown that the incidence of myeloid neoplasms associated with germline variants is higher than expected. Accordingly, myeloid neoplasms with germline predisposition have represented a unique category in the recent WHO classification and the International Consensus Classification, and DDX41 mutation accounts for 2-5% of myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37792958"
                        open
                      >
                        <summary class="article-expander__title">Identifying predictors of glioma evolution from longitudinal sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37792958/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231021205333&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution drives cancer progression and therapeutic resistance. Recent studies have revealed divergent longitudinal trajectories in gliomas, but early molecular features steering posttreatment cancer evolution remain unclear. Here, we collected sequencing and clinical data of initial-recurrent tumor pairs from 544 adult diffuse gliomas and performed multivariate analysis to identify early molecular predictors of tumor evolution in three diffuse glioma subtypes. We found that CDKN2A...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794100"
                        open
                      >
                        <summary class="article-expander__title">Deciphering treatment patterns in non-severe/moderate aplastic anemia: an international observational study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794100/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231020204820&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Non-severe aplastic anemia is a rare bone marrow failure disorder characterized by variable degrees and combination of cytopenias, with limited data on management and outcome. We describe a large multicentric series of 259 patients, focusing on clinical and molecular features, treatment, evolution, and survival. The majority required treatment with cyclosporine (CyA) alone (N &#x3D; 84) or in combination with anti-thymocyte globulin (ATG,44) or eltrombopag (20), eltrombopag alone (10), or others (25)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794034"
                        open
                      >
                        <summary class="article-expander__title">Genomic signatures of past and present chromosomal instability in Barrett&#x27;s esophagus and early esophageal adenocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794034/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231020204820&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The progression of precancerous lesions to malignancy is often accompanied by increasing complexity of chromosomal alterations but how these alterations arise is poorly understood. Here we perform haplotype-specific analysis of chromosomal copy-number evolution in the progression of Barrett&#x27;s esophagus (BE) to esophageal adenocarcinoma (EAC) on multiregional whole-genome sequencing data of BE with dysplasia and microscopic EAC foci. We identify distinct patterns of copy-number evolution...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;17
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793870"
                        open
                      >
                        <summary class="article-expander__title">Myeloid neoplasms with germline predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793870/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231020204820&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With recent advances in sequencing technologies, novel genes associated with the predisposition to myeloid neoplasms have been discovered, and subsequent studies have shown that the incidence of myeloid neoplasms associated with germline variants is higher than expected. Accordingly, myeloid neoplasms with germline predisposition have represented a unique category in the recent WHO classification and the International Consensus Classification, and DDX41 mutation accounts for 2-5% of myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37792958"
                        open
                      >
                        <summary class="article-expander__title">Identifying predictors of glioma evolution from longitudinal sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37792958/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231020204820&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution drives cancer progression and therapeutic resistance. Recent studies have revealed divergent longitudinal trajectories in gliomas, but early molecular features steering posttreatment cancer evolution remain unclear. Here, we collected sequencing and clinical data of initial-recurrent tumor pairs from 544 adult diffuse gliomas and performed multivariate analysis to identify early molecular predictors of tumor evolution in three diffuse glioma subtypes. We found that CDKN2A...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794100"
                        open
                      >
                        <summary class="article-expander__title">Deciphering treatment patterns in non-severe/moderate aplastic anemia: an international observational study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794100/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231019204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Non-severe aplastic anemia is a rare bone marrow failure disorder characterized by variable degrees and combination of cytopenias, with limited data on management and outcome. We describe a large multicentric series of 259 patients, focusing on clinical and molecular features, treatment, evolution, and survival. The majority required treatment with cyclosporine (CyA) alone (N &#x3D; 84) or in combination with anti-thymocyte globulin (ATG,44) or eltrombopag (20), eltrombopag alone (10), or others (25)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794034"
                        open
                      >
                        <summary class="article-expander__title">Genomic signatures of past and present chromosomal instability in Barrett&#x27;s esophagus and early esophageal adenocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794034/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231019204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The progression of precancerous lesions to malignancy is often accompanied by increasing complexity of chromosomal alterations but how these alterations arise is poorly understood. Here we perform haplotype-specific analysis of chromosomal copy-number evolution in the progression of Barrett&#x27;s esophagus (BE) to esophageal adenocarcinoma (EAC) on multiregional whole-genome sequencing data of BE with dysplasia and microscopic EAC foci. We identify distinct patterns of copy-number evolution...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;17
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793870"
                        open
                      >
                        <summary class="article-expander__title">Myeloid neoplasms with germline predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793870/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231019204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With recent advances in sequencing technologies, novel genes associated with the predisposition to myeloid neoplasms have been discovered, and subsequent studies have shown that the incidence of myeloid neoplasms associated with germline variants is higher than expected. Accordingly, myeloid neoplasms with germline predisposition have represented a unique category in the recent WHO classification and the International Consensus Classification, and DDX41 mutation accounts for 2-5% of myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37792958"
                        open
                      >
                        <summary class="article-expander__title">Identifying predictors of glioma evolution from longitudinal sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37792958/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231019204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution drives cancer progression and therapeutic resistance. Recent studies have revealed divergent longitudinal trajectories in gliomas, but early molecular features steering posttreatment cancer evolution remain unclear. Here, we collected sequencing and clinical data of initial-recurrent tumor pairs from 544 adult diffuse gliomas and performed multivariate analysis to identify early molecular predictors of tumor evolution in three diffuse glioma subtypes. We found that CDKN2A...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794100"
                        open
                      >
                        <summary class="article-expander__title">Deciphering treatment patterns in non-severe/moderate aplastic anemia: an international observational study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794100/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231018205013&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Non-severe aplastic anemia is a rare bone marrow failure disorder characterized by variable degrees and combination of cytopenias, with limited data on management and outcome. We describe a large multicentric series of 259 patients, focusing on clinical and molecular features, treatment, evolution, and survival. The majority required treatment with cyclosporine (CyA) alone (N &#x3D; 84) or in combination with anti-thymocyte globulin (ATG,44) or eltrombopag (20), eltrombopag alone (10), or others (25)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794034"
                        open
                      >
                        <summary class="article-expander__title">Genomic signatures of past and present chromosomal instability in Barrett&#x27;s esophagus and early esophageal adenocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794034/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231018205013&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The progression of precancerous lesions to malignancy is often accompanied by increasing complexity of chromosomal alterations but how these alterations arise is poorly understood. Here we perform haplotype-specific analysis of chromosomal copy-number evolution in the progression of Barrett&#x27;s esophagus (BE) to esophageal adenocarcinoma (EAC) on multiregional whole-genome sequencing data of BE with dysplasia and microscopic EAC foci. We identify distinct patterns of copy-number evolution...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;17
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793870"
                        open
                      >
                        <summary class="article-expander__title">Myeloid neoplasms with germline predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793870/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231018205013&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With recent advances in sequencing technologies, novel genes associated with the predisposition to myeloid neoplasms have been discovered, and subsequent studies have shown that the incidence of myeloid neoplasms associated with germline variants is higher than expected. Accordingly, myeloid neoplasms with germline predisposition have represented a unique category in the recent WHO classification and the International Consensus Classification, and DDX41 mutation accounts for 2-5% of myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37792958"
                        open
                      >
                        <summary class="article-expander__title">Identifying predictors of glioma evolution from longitudinal sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37792958/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231018205013&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution drives cancer progression and therapeutic resistance. Recent studies have revealed divergent longitudinal trajectories in gliomas, but early molecular features steering posttreatment cancer evolution remain unclear. Here, we collected sequencing and clinical data of initial-recurrent tumor pairs from 544 adult diffuse gliomas and performed multivariate analysis to identify early molecular predictors of tumor evolution in three diffuse glioma subtypes. We found that CDKN2A...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794100"
                        open
                      >
                        <summary class="article-expander__title">Deciphering treatment patterns in non-severe/moderate aplastic anemia: an international observational study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794100/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231017205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Non-severe aplastic anemia is a rare bone marrow failure disorder characterized by variable degrees and combination of cytopenias, with limited data on management and outcome. We describe a large multicentric series of 259 patients, focusing on clinical and molecular features, treatment, evolution, and survival. The majority required treatment with cyclosporine (CyA) alone (N &#x3D; 84) or in combination with anti-thymocyte globulin (ATG,44) or eltrombopag (20), eltrombopag alone (10), or others (25)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794034"
                        open
                      >
                        <summary class="article-expander__title">Genomic signatures of past and present chromosomal instability in Barrett&#x27;s esophagus and early esophageal adenocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794034/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231017205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The progression of precancerous lesions to malignancy is often accompanied by increasing complexity of chromosomal alterations but how these alterations arise is poorly understood. Here we perform haplotype-specific analysis of chromosomal copy-number evolution in the progression of Barrett&#x27;s esophagus (BE) to esophageal adenocarcinoma (EAC) on multiregional whole-genome sequencing data of BE with dysplasia and microscopic EAC foci. We identify distinct patterns of copy-number evolution...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;17
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793870"
                        open
                      >
                        <summary class="article-expander__title">Myeloid neoplasms with germline predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793870/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231017205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With recent advances in sequencing technologies, novel genes associated with the predisposition to myeloid neoplasms have been discovered, and subsequent studies have shown that the incidence of myeloid neoplasms associated with germline variants is higher than expected. Accordingly, myeloid neoplasms with germline predisposition have represented a unique category in the recent WHO classification and the International Consensus Classification, and DDX41 mutation accounts for 2-5% of myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37792958"
                        open
                      >
                        <summary class="article-expander__title">Identifying predictors of glioma evolution from longitudinal sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37792958/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231017205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution drives cancer progression and therapeutic resistance. Recent studies have revealed divergent longitudinal trajectories in gliomas, but early molecular features steering posttreatment cancer evolution remain unclear. Here, we collected sequencing and clinical data of initial-recurrent tumor pairs from 544 adult diffuse gliomas and performed multivariate analysis to identify early molecular predictors of tumor evolution in three diffuse glioma subtypes. We found that CDKN2A...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794100"
                        open
                      >
                        <summary class="article-expander__title">Deciphering treatment patterns in non-severe/moderate aplastic anemia: an international observational study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794100/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231016205028&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Non-severe aplastic anemia is a rare bone marrow failure disorder characterized by variable degrees and combination of cytopenias, with limited data on management and outcome. We describe a large multicentric series of 259 patients, focusing on clinical and molecular features, treatment, evolution, and survival. The majority required treatment with cyclosporine (CyA) alone (N &#x3D; 84) or in combination with anti-thymocyte globulin (ATG,44) or eltrombopag (20), eltrombopag alone (10), or others (25)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794034"
                        open
                      >
                        <summary class="article-expander__title">Genomic signatures of past and present chromosomal instability in Barrett&#x27;s esophagus and early esophageal adenocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794034/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231016205028&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The progression of precancerous lesions to malignancy is often accompanied by increasing complexity of chromosomal alterations but how these alterations arise is poorly understood. Here we perform haplotype-specific analysis of chromosomal copy-number evolution in the progression of Barrett&#x27;s esophagus (BE) to esophageal adenocarcinoma (EAC) on multiregional whole-genome sequencing data of BE with dysplasia and microscopic EAC foci. We identify distinct patterns of copy-number evolution...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;17
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793870"
                        open
                      >
                        <summary class="article-expander__title">Myeloid neoplasms with germline predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793870/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231016205028&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With recent advances in sequencing technologies, novel genes associated with the predisposition to myeloid neoplasms have been discovered, and subsequent studies have shown that the incidence of myeloid neoplasms associated with germline variants is higher than expected. Accordingly, myeloid neoplasms with germline predisposition have represented a unique category in the recent WHO classification and the International Consensus Classification, and DDX41 mutation accounts for 2-5% of myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37792958"
                        open
                      >
                        <summary class="article-expander__title">Identifying predictors of glioma evolution from longitudinal sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37792958/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231016205028&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution drives cancer progression and therapeutic resistance. Recent studies have revealed divergent longitudinal trajectories in gliomas, but early molecular features steering posttreatment cancer evolution remain unclear. Here, we collected sequencing and clinical data of initial-recurrent tumor pairs from 544 adult diffuse gliomas and performed multivariate analysis to identify early molecular predictors of tumor evolution in three diffuse glioma subtypes. We found that CDKN2A...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794100"
                        open
                      >
                        <summary class="article-expander__title">Deciphering treatment patterns in non-severe/moderate aplastic anemia: an international observational study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794100/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231015205105&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Non-severe aplastic anemia is a rare bone marrow failure disorder characterized by variable degrees and combination of cytopenias, with limited data on management and outcome. We describe a large multicentric series of 259 patients, focusing on clinical and molecular features, treatment, evolution, and survival. The majority required treatment with cyclosporine (CyA) alone (N &#x3D; 84) or in combination with anti-thymocyte globulin (ATG,44) or eltrombopag (20), eltrombopag alone (10), or others (25)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794034"
                        open
                      >
                        <summary class="article-expander__title">Genomic signatures of past and present chromosomal instability in Barrett&#x27;s esophagus and early esophageal adenocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794034/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231015205105&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The progression of precancerous lesions to malignancy is often accompanied by increasing complexity of chromosomal alterations but how these alterations arise is poorly understood. Here we perform haplotype-specific analysis of chromosomal copy-number evolution in the progression of Barrett&#x27;s esophagus (BE) to esophageal adenocarcinoma (EAC) on multiregional whole-genome sequencing data of BE with dysplasia and microscopic EAC foci. We identify distinct patterns of copy-number evolution...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;17
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793870"
                        open
                      >
                        <summary class="article-expander__title">Myeloid neoplasms with germline predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793870/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231015205105&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With recent advances in sequencing technologies, novel genes associated with the predisposition to myeloid neoplasms have been discovered, and subsequent studies have shown that the incidence of myeloid neoplasms associated with germline variants is higher than expected. Accordingly, myeloid neoplasms with germline predisposition have represented a unique category in the recent WHO classification and the International Consensus Classification, and DDX41 mutation accounts for 2-5% of myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37792958"
                        open
                      >
                        <summary class="article-expander__title">Identifying predictors of glioma evolution from longitudinal sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37792958/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231015205105&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution drives cancer progression and therapeutic resistance. Recent studies have revealed divergent longitudinal trajectories in gliomas, but early molecular features steering posttreatment cancer evolution remain unclear. Here, we collected sequencing and clinical data of initial-recurrent tumor pairs from 544 adult diffuse gliomas and performed multivariate analysis to identify early molecular predictors of tumor evolution in three diffuse glioma subtypes. We found that CDKN2A...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794100"
                        open
                      >
                        <summary class="article-expander__title">Deciphering treatment patterns in non-severe/moderate aplastic anemia: an international observational study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794100/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231014205339&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Non-severe aplastic anemia is a rare bone marrow failure disorder characterized by variable degrees and combination of cytopenias, with limited data on management and outcome. We describe a large multicentric series of 259 patients, focusing on clinical and molecular features, treatment, evolution, and survival. The majority required treatment with cyclosporine (CyA) alone (N &#x3D; 84) or in combination with anti-thymocyte globulin (ATG,44) or eltrombopag (20), eltrombopag alone (10), or others (25)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794034"
                        open
                      >
                        <summary class="article-expander__title">Genomic signatures of past and present chromosomal instability in Barrett&#x27;s esophagus and early esophageal adenocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794034/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231014205339&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The progression of precancerous lesions to malignancy is often accompanied by increasing complexity of chromosomal alterations but how these alterations arise is poorly understood. Here we perform haplotype-specific analysis of chromosomal copy-number evolution in the progression of Barrett&#x27;s esophagus (BE) to esophageal adenocarcinoma (EAC) on multiregional whole-genome sequencing data of BE with dysplasia and microscopic EAC foci. We identify distinct patterns of copy-number evolution...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;17
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793870"
                        open
                      >
                        <summary class="article-expander__title">Myeloid neoplasms with germline predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793870/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231014205339&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With recent advances in sequencing technologies, novel genes associated with the predisposition to myeloid neoplasms have been discovered, and subsequent studies have shown that the incidence of myeloid neoplasms associated with germline variants is higher than expected. Accordingly, myeloid neoplasms with germline predisposition have represented a unique category in the recent WHO classification and the International Consensus Classification, and DDX41 mutation accounts for 2-5% of myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37792958"
                        open
                      >
                        <summary class="article-expander__title">Identifying predictors of glioma evolution from longitudinal sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37792958/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231014205339&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution drives cancer progression and therapeutic resistance. Recent studies have revealed divergent longitudinal trajectories in gliomas, but early molecular features steering posttreatment cancer evolution remain unclear. Here, we collected sequencing and clinical data of initial-recurrent tumor pairs from 544 adult diffuse gliomas and performed multivariate analysis to identify early molecular predictors of tumor evolution in three diffuse glioma subtypes. We found that CDKN2A...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794100"
                        open
                      >
                        <summary class="article-expander__title">Deciphering treatment patterns in non-severe/moderate aplastic anemia: an international observational study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794100/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231013204816&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Non-severe aplastic anemia is a rare bone marrow failure disorder characterized by variable degrees and combination of cytopenias, with limited data on management and outcome. We describe a large multicentric series of 259 patients, focusing on clinical and molecular features, treatment, evolution, and survival. The majority required treatment with cyclosporine (CyA) alone (N &#x3D; 84) or in combination with anti-thymocyte globulin (ATG,44) or eltrombopag (20), eltrombopag alone (10), or others (25)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794034"
                        open
                      >
                        <summary class="article-expander__title">Genomic signatures of past and present chromosomal instability in Barrett&#x27;s esophagus and early esophageal adenocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794034/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231013204816&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The progression of precancerous lesions to malignancy is often accompanied by increasing complexity of chromosomal alterations but how these alterations arise is poorly understood. Here we perform haplotype-specific analysis of chromosomal copy-number evolution in the progression of Barrett&#x27;s esophagus (BE) to esophageal adenocarcinoma (EAC) on multiregional whole-genome sequencing data of BE with dysplasia and microscopic EAC foci. We identify distinct patterns of copy-number evolution...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;17
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793870"
                        open
                      >
                        <summary class="article-expander__title">Myeloid neoplasms with germline predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793870/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231013204816&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With recent advances in sequencing technologies, novel genes associated with the predisposition to myeloid neoplasms have been discovered, and subsequent studies have shown that the incidence of myeloid neoplasms associated with germline variants is higher than expected. Accordingly, myeloid neoplasms with germline predisposition have represented a unique category in the recent WHO classification and the International Consensus Classification, and DDX41 mutation accounts for 2-5% of myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37792958"
                        open
                      >
                        <summary class="article-expander__title">Identifying predictors of glioma evolution from longitudinal sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37792958/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231013204816&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution drives cancer progression and therapeutic resistance. Recent studies have revealed divergent longitudinal trajectories in gliomas, but early molecular features steering posttreatment cancer evolution remain unclear. Here, we collected sequencing and clinical data of initial-recurrent tumor pairs from 544 adult diffuse gliomas and performed multivariate analysis to identify early molecular predictors of tumor evolution in three diffuse glioma subtypes. We found that CDKN2A...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794100"
                        open
                      >
                        <summary class="article-expander__title">Deciphering treatment patterns in non-severe/moderate aplastic anemia: an international observational study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794100/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231012205121&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Non-severe aplastic anemia is a rare bone marrow failure disorder characterized by variable degrees and combination of cytopenias, with limited data on management and outcome. We describe a large multicentric series of 259 patients, focusing on clinical and molecular features, treatment, evolution, and survival. The majority required treatment with cyclosporine (CyA) alone (N &#x3D; 84) or in combination with anti-thymocyte globulin (ATG,44) or eltrombopag (20), eltrombopag alone (10), or others (25)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794034"
                        open
                      >
                        <summary class="article-expander__title">Genomic signatures of past and present chromosomal instability in Barrett&#x27;s esophagus and early esophageal adenocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794034/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231012205121&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The progression of precancerous lesions to malignancy is often accompanied by increasing complexity of chromosomal alterations but how these alterations arise is poorly understood. Here we perform haplotype-specific analysis of chromosomal copy-number evolution in the progression of Barrett&#x27;s esophagus (BE) to esophageal adenocarcinoma (EAC) on multiregional whole-genome sequencing data of BE with dysplasia and microscopic EAC foci. We identify distinct patterns of copy-number evolution...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;17
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793870"
                        open
                      >
                        <summary class="article-expander__title">Myeloid neoplasms with germline predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793870/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231012205121&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With recent advances in sequencing technologies, novel genes associated with the predisposition to myeloid neoplasms have been discovered, and subsequent studies have shown that the incidence of myeloid neoplasms associated with germline variants is higher than expected. Accordingly, myeloid neoplasms with germline predisposition have represented a unique category in the recent WHO classification and the International Consensus Classification, and DDX41 mutation accounts for 2-5% of myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37792958"
                        open
                      >
                        <summary class="article-expander__title">Identifying predictors of glioma evolution from longitudinal sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37792958/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231012205121&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution drives cancer progression and therapeutic resistance. Recent studies have revealed divergent longitudinal trajectories in gliomas, but early molecular features steering posttreatment cancer evolution remain unclear. Here, we collected sequencing and clinical data of initial-recurrent tumor pairs from 544 adult diffuse gliomas and performed multivariate analysis to identify early molecular predictors of tumor evolution in three diffuse glioma subtypes. We found that CDKN2A...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794100"
                        open
                      >
                        <summary class="article-expander__title">Deciphering treatment patterns in non-severe/moderate aplastic anemia: an international observational study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794100/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231011204744&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Non-severe aplastic anemia is a rare bone marrow failure disorder characterized by variable degrees and combination of cytopenias, with limited data on management and outcome. We describe a large multicentric series of 259 patients, focusing on clinical and molecular features, treatment, evolution, and survival. The majority required treatment with cyclosporine (CyA) alone (N &#x3D; 84) or in combination with anti-thymocyte globulin (ATG,44) or eltrombopag (20), eltrombopag alone (10), or others (25)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794034"
                        open
                      >
                        <summary class="article-expander__title">Genomic signatures of past and present chromosomal instability in Barrett&#x27;s esophagus and early esophageal adenocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794034/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231011204744&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The progression of precancerous lesions to malignancy is often accompanied by increasing complexity of chromosomal alterations but how these alterations arise is poorly understood. Here we perform haplotype-specific analysis of chromosomal copy-number evolution in the progression of Barrett&#x27;s esophagus (BE) to esophageal adenocarcinoma (EAC) on multiregional whole-genome sequencing data of BE with dysplasia and microscopic EAC foci. We identify distinct patterns of copy-number evolution...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;17
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793870"
                        open
                      >
                        <summary class="article-expander__title">Myeloid neoplasms with germline predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793870/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231011204744&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With recent advances in sequencing technologies, novel genes associated with the predisposition to myeloid neoplasms have been discovered, and subsequent studies have shown that the incidence of myeloid neoplasms associated with germline variants is higher than expected. Accordingly, myeloid neoplasms with germline predisposition have represented a unique category in the recent WHO classification and the International Consensus Classification, and DDX41 mutation accounts for 2-5% of myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37792958"
                        open
                      >
                        <summary class="article-expander__title">Identifying predictors of glioma evolution from longitudinal sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37792958/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231011204744&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution drives cancer progression and therapeutic resistance. Recent studies have revealed divergent longitudinal trajectories in gliomas, but early molecular features steering posttreatment cancer evolution remain unclear. Here, we collected sequencing and clinical data of initial-recurrent tumor pairs from 544 adult diffuse gliomas and performed multivariate analysis to identify early molecular predictors of tumor evolution in three diffuse glioma subtypes. We found that CDKN2A...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794100"
                        open
                      >
                        <summary class="article-expander__title">Deciphering treatment patterns in non-severe/moderate aplastic anemia: an international observational study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794100/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231010204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Non-severe aplastic anemia is a rare bone marrow failure disorder characterized by variable degrees and combination of cytopenias, with limited data on management and outcome. We describe a large multicentric series of 259 patients, focusing on clinical and molecular features, treatment, evolution, and survival. The majority required treatment with cyclosporine (CyA) alone (N &#x3D; 84) or in combination with anti-thymocyte globulin (ATG,44) or eltrombopag (20), eltrombopag alone (10), or others (25)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794034"
                        open
                      >
                        <summary class="article-expander__title">Genomic signatures of past and present chromosomal instability in Barrett&#x27;s esophagus and early esophageal adenocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794034/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231010204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The progression of precancerous lesions to malignancy is often accompanied by increasing complexity of chromosomal alterations but how these alterations arise is poorly understood. Here we perform haplotype-specific analysis of chromosomal copy-number evolution in the progression of Barrett&#x27;s esophagus (BE) to esophageal adenocarcinoma (EAC) on multiregional whole-genome sequencing data of BE with dysplasia and microscopic EAC foci. We identify distinct patterns of copy-number evolution...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;17
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793870"
                        open
                      >
                        <summary class="article-expander__title">Myeloid neoplasms with germline predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793870/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231010204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With recent advances in sequencing technologies, novel genes associated with the predisposition to myeloid neoplasms have been discovered, and subsequent studies have shown that the incidence of myeloid neoplasms associated with germline variants is higher than expected. Accordingly, myeloid neoplasms with germline predisposition have represented a unique category in the recent WHO classification and the International Consensus Classification, and DDX41 mutation accounts for 2-5% of myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37792958"
                        open
                      >
                        <summary class="article-expander__title">Identifying predictors of glioma evolution from longitudinal sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37792958/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231010204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution drives cancer progression and therapeutic resistance. Recent studies have revealed divergent longitudinal trajectories in gliomas, but early molecular features steering posttreatment cancer evolution remain unclear. Here, we collected sequencing and clinical data of initial-recurrent tumor pairs from 544 adult diffuse gliomas and performed multivariate analysis to identify early molecular predictors of tumor evolution in three diffuse glioma subtypes. We found that CDKN2A...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794100"
                        open
                      >
                        <summary class="article-expander__title">Deciphering treatment patterns in non-severe/moderate aplastic anemia: an international observational study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794100/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231009204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Non-severe aplastic anemia is a rare bone marrow failure disorder characterized by variable degrees and combination of cytopenias, with limited data on management and outcome. We describe a large multicentric series of 259 patients, focusing on clinical and molecular features, treatment, evolution, and survival. The majority required treatment with cyclosporine (CyA) alone (N &#x3D; 84) or in combination with anti-thymocyte globulin (ATG,44) or eltrombopag (20), eltrombopag alone (10), or others (25)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794034"
                        open
                      >
                        <summary class="article-expander__title">Genomic signatures of past and present chromosomal instability in Barrett&#x27;s esophagus and early esophageal adenocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794034/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231009204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The progression of precancerous lesions to malignancy is often accompanied by increasing complexity of chromosomal alterations but how these alterations arise is poorly understood. Here we perform haplotype-specific analysis of chromosomal copy-number evolution in the progression of Barrett&#x27;s esophagus (BE) to esophageal adenocarcinoma (EAC) on multiregional whole-genome sequencing data of BE with dysplasia and microscopic EAC foci. We identify distinct patterns of copy-number evolution...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;17
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793870"
                        open
                      >
                        <summary class="article-expander__title">Myeloid neoplasms with germline predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793870/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231009204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With recent advances in sequencing technologies, novel genes associated with the predisposition to myeloid neoplasms have been discovered, and subsequent studies have shown that the incidence of myeloid neoplasms associated with germline variants is higher than expected. Accordingly, myeloid neoplasms with germline predisposition have represented a unique category in the recent WHO classification and the International Consensus Classification, and DDX41 mutation accounts for 2-5% of myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37792958"
                        open
                      >
                        <summary class="article-expander__title">Identifying predictors of glioma evolution from longitudinal sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37792958/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231009204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution drives cancer progression and therapeutic resistance. Recent studies have revealed divergent longitudinal trajectories in gliomas, but early molecular features steering posttreatment cancer evolution remain unclear. Here, we collected sequencing and clinical data of initial-recurrent tumor pairs from 544 adult diffuse gliomas and performed multivariate analysis to identify early molecular predictors of tumor evolution in three diffuse glioma subtypes. We found that CDKN2A...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794100"
                        open
                      >
                        <summary class="article-expander__title">Deciphering treatment patterns in non-severe/moderate aplastic anemia: an international observational study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794100/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231008205021&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Non-severe aplastic anemia is a rare bone marrow failure disorder characterized by variable degrees and combination of cytopenias, with limited data on management and outcome. We describe a large multicentric series of 259 patients, focusing on clinical and molecular features, treatment, evolution, and survival. The majority required treatment with cyclosporine (CyA) alone (N &#x3D; 84) or in combination with anti-thymocyte globulin (ATG,44) or eltrombopag (20), eltrombopag alone (10), or others (25)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794034"
                        open
                      >
                        <summary class="article-expander__title">Genomic signatures of past and present chromosomal instability in Barrett&#x27;s esophagus and early esophageal adenocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794034/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231008205021&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The progression of precancerous lesions to malignancy is often accompanied by increasing complexity of chromosomal alterations but how these alterations arise is poorly understood. Here we perform haplotype-specific analysis of chromosomal copy-number evolution in the progression of Barrett&#x27;s esophagus (BE) to esophageal adenocarcinoma (EAC) on multiregional whole-genome sequencing data of BE with dysplasia and microscopic EAC foci. We identify distinct patterns of copy-number evolution...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;17
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793870"
                        open
                      >
                        <summary class="article-expander__title">Myeloid neoplasms with germline predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793870/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231008205021&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With recent advances in sequencing technologies, novel genes associated with the predisposition to myeloid neoplasms have been discovered, and subsequent studies have shown that the incidence of myeloid neoplasms associated with germline variants is higher than expected. Accordingly, myeloid neoplasms with germline predisposition have represented a unique category in the recent WHO classification and the International Consensus Classification, and DDX41 mutation accounts for 2-5% of myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37792958"
                        open
                      >
                        <summary class="article-expander__title">Identifying predictors of glioma evolution from longitudinal sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37792958/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231008205021&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution drives cancer progression and therapeutic resistance. Recent studies have revealed divergent longitudinal trajectories in gliomas, but early molecular features steering posttreatment cancer evolution remain unclear. Here, we collected sequencing and clinical data of initial-recurrent tumor pairs from 544 adult diffuse gliomas and performed multivariate analysis to identify early molecular predictors of tumor evolution in three diffuse glioma subtypes. We found that CDKN2A...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794100"
                        open
                      >
                        <summary class="article-expander__title">Deciphering treatment patterns in non-severe/moderate aplastic anemia: an international observational study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794100/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231007205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Non-severe aplastic anemia is a rare bone marrow failure disorder characterized by variable degrees and combination of cytopenias, with limited data on management and outcome. We describe a large multicentric series of 259 patients, focusing on clinical and molecular features, treatment, evolution, and survival. The majority required treatment with cyclosporine (CyA) alone (N &#x3D; 84) or in combination with anti-thymocyte globulin (ATG,44) or eltrombopag (20), eltrombopag alone (10), or others (25)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794034"
                        open
                      >
                        <summary class="article-expander__title">Genomic signatures of past and present chromosomal instability in Barrett&#x27;s esophagus and early esophageal adenocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794034/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231007205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The progression of precancerous lesions to malignancy is often accompanied by increasing complexity of chromosomal alterations but how these alterations arise is poorly understood. Here we perform haplotype-specific analysis of chromosomal copy-number evolution in the progression of Barrett&#x27;s esophagus (BE) to esophageal adenocarcinoma (EAC) on multiregional whole-genome sequencing data of BE with dysplasia and microscopic EAC foci. We identify distinct patterns of copy-number evolution...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;17
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793870"
                        open
                      >
                        <summary class="article-expander__title">Myeloid neoplasms with germline predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793870/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231007205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With recent advances in sequencing technologies, novel genes associated with the predisposition to myeloid neoplasms have been discovered, and subsequent studies have shown that the incidence of myeloid neoplasms associated with germline variants is higher than expected. Accordingly, myeloid neoplasms with germline predisposition have represented a unique category in the recent WHO classification and the International Consensus Classification, and DDX41 mutation accounts for 2-5% of myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37792958"
                        open
                      >
                        <summary class="article-expander__title">Identifying predictors of glioma evolution from longitudinal sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37792958/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231007205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution drives cancer progression and therapeutic resistance. Recent studies have revealed divergent longitudinal trajectories in gliomas, but early molecular features steering posttreatment cancer evolution remain unclear. Here, we collected sequencing and clinical data of initial-recurrent tumor pairs from 544 adult diffuse gliomas and performed multivariate analysis to identify early molecular predictors of tumor evolution in three diffuse glioma subtypes. We found that CDKN2A...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794100"
                        open
                      >
                        <summary class="article-expander__title">Deciphering treatment patterns in non-severe/moderate aplastic anemia: an international observational study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794100/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231006204916&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Non-severe aplastic anemia is a rare bone marrow failure disorder characterized by variable degrees and combination of cytopenias, with limited data on management and outcome. We describe a large multicentric series of 259 patients, focusing on clinical and molecular features, treatment, evolution, and survival. The majority required treatment with cyclosporine (CyA) alone (N &#x3D; 84) or in combination with anti-thymocyte globulin (ATG,44) or eltrombopag (20), eltrombopag alone (10), or others (25)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794034"
                        open
                      >
                        <summary class="article-expander__title">Genomic signatures of past and present chromosomal instability in Barrett&#x27;s esophagus and early esophageal adenocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794034/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231006204916&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The progression of precancerous lesions to malignancy is often accompanied by increasing complexity of chromosomal alterations but how these alterations arise is poorly understood. Here we perform haplotype-specific analysis of chromosomal copy-number evolution in the progression of Barrett&#x27;s esophagus (BE) to esophageal adenocarcinoma (EAC) on multiregional whole-genome sequencing data of BE with dysplasia and microscopic EAC foci. We identify distinct patterns of copy-number evolution...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;17
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793870"
                        open
                      >
                        <summary class="article-expander__title">Myeloid neoplasms with germline predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793870/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231006204916&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With recent advances in sequencing technologies, novel genes associated with the predisposition to myeloid neoplasms have been discovered, and subsequent studies have shown that the incidence of myeloid neoplasms associated with germline variants is higher than expected. Accordingly, myeloid neoplasms with germline predisposition have represented a unique category in the recent WHO classification and the International Consensus Classification, and DDX41 mutation accounts for 2-5% of myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37792958"
                        open
                      >
                        <summary class="article-expander__title">Identifying predictors of glioma evolution from longitudinal sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37792958/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231006204916&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution drives cancer progression and therapeutic resistance. Recent studies have revealed divergent longitudinal trajectories in gliomas, but early molecular features steering posttreatment cancer evolution remain unclear. Here, we collected sequencing and clinical data of initial-recurrent tumor pairs from 544 adult diffuse gliomas and performed multivariate analysis to identify early molecular predictors of tumor evolution in three diffuse glioma subtypes. We found that CDKN2A...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37794100"
                        open
                      >
                        <summary class="article-expander__title">Deciphering treatment patterns in non-severe/moderate aplastic anemia: an international observational study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37794100/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231005204957&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Non-severe aplastic anemia is a rare bone marrow failure disorder characterized by variable degrees and combination of cytopenias, with limited data on management and outcome. We describe a large multicentric series of 259 patients, focusing on clinical and molecular features, treatment, evolution, and survival. The majority required treatment with cyclosporine (CyA) alone (N &#x3D; 84) or in combination with anti-thymocyte globulin (ATG,44) or eltrombopag (20), eltrombopag alone (10), or others (25)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37793870"
                        open
                      >
                        <summary class="article-expander__title">Myeloid neoplasms with germline predisposition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37793870/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231005204957&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With recent advances in sequencing technologies, novel genes associated with the predisposition to myeloid neoplasms have been discovered, and subsequent studies have shown that the incidence of myeloid neoplasms associated with germline variants is higher than expected. Accordingly, myeloid neoplasms with germline predisposition have represented a unique category in the recent WHO classification and the International Consensus Classification, and DDX41 mutation accounts for 2-5% of myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37792958"
                        open
                      >
                        <summary class="article-expander__title">Identifying predictors of glioma evolution from longitudinal sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37792958/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231005204957&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution drives cancer progression and therapeutic resistance. Recent studies have revealed divergent longitudinal trajectories in gliomas, but early molecular features steering posttreatment cancer evolution remain unclear. Here, we collected sequencing and clinical data of initial-recurrent tumor pairs from 544 adult diffuse gliomas and performed multivariate analysis to identify early molecular predictors of tumor evolution in three diffuse glioma subtypes. We found that CDKN2A...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37792958"
                        open
                      >
                        <summary class="article-expander__title">Identifying predictors of glioma evolution from longitudinal sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37792958/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231004205006&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution drives cancer progression and therapeutic resistance. Recent studies have revealed divergent longitudinal trajectories in gliomas, but early molecular features steering posttreatment cancer evolution remain unclear. Here, we collected sequencing and clinical data of initial-recurrent tumor pairs from 544 adult diffuse gliomas and performed multivariate analysis to identify early molecular predictors of tumor evolution in three diffuse glioma subtypes. We found that CDKN2A...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-10-03 datetime="2023-10-03T10:00:00.000Z">2023-10-03</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-10-03 datetime="2023-10-03T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20231105195156">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37785441"
                        open
                      >
                        <summary class="article-expander__title">Success of Liquid Tumor Biopsy in Men with Metastatic Prostate Cancer According to Self-Identified Race</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37785441/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231003205046&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Successful cfDNA biopsy in metastatic PrCa is associated with PSA and PSADT, but not related to patient self-identified race. In appropriate clinical scenarios, patients who self-identify as Black or White are equally likely to have PrCa-specific alterations detected on cfDNA testing when evaluating metastatic PrCa patients for local and systemic therapies.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;ff&#x3D;20231105195156&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37786705"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary fingerprints of EMT in pancreatic cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37786705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231022205027&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mesenchymal plasticity has been extensively described in advanced and metastatic epithelial cancers; however, its functional role in malignant progression, metastatic dissemination and therapy response is controversial. More importantly, the role of epithelial mesenchymal transition (EMT) and cell plasticity in tumor heterogeneity, clonal selection and clonal evolution is poorly understood. Functionally, our work clarifies the contribution of EMT to malignant progression and metastasis in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37786705"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary fingerprints of EMT in pancreatic cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37786705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231021205333&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mesenchymal plasticity has been extensively described in advanced and metastatic epithelial cancers; however, its functional role in malignant progression, metastatic dissemination and therapy response is controversial. More importantly, the role of epithelial mesenchymal transition (EMT) and cell plasticity in tumor heterogeneity, clonal selection and clonal evolution is poorly understood. Functionally, our work clarifies the contribution of EMT to malignant progression and metastasis in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37786705"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary fingerprints of EMT in pancreatic cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37786705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231020204820&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mesenchymal plasticity has been extensively described in advanced and metastatic epithelial cancers; however, its functional role in malignant progression, metastatic dissemination and therapy response is controversial. More importantly, the role of epithelial mesenchymal transition (EMT) and cell plasticity in tumor heterogeneity, clonal selection and clonal evolution is poorly understood. Functionally, our work clarifies the contribution of EMT to malignant progression and metastasis in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37786705"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary fingerprints of EMT in pancreatic cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37786705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231019204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mesenchymal plasticity has been extensively described in advanced and metastatic epithelial cancers; however, its functional role in malignant progression, metastatic dissemination and therapy response is controversial. More importantly, the role of epithelial mesenchymal transition (EMT) and cell plasticity in tumor heterogeneity, clonal selection and clonal evolution is poorly understood. Functionally, our work clarifies the contribution of EMT to malignant progression and metastasis in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37786705"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary fingerprints of EMT in pancreatic cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37786705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231018205013&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mesenchymal plasticity has been extensively described in advanced and metastatic epithelial cancers; however, its functional role in malignant progression, metastatic dissemination and therapy response is controversial. More importantly, the role of epithelial mesenchymal transition (EMT) and cell plasticity in tumor heterogeneity, clonal selection and clonal evolution is poorly understood. Functionally, our work clarifies the contribution of EMT to malignant progression and metastasis in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37786705"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary fingerprints of EMT in pancreatic cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37786705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231017205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mesenchymal plasticity has been extensively described in advanced and metastatic epithelial cancers; however, its functional role in malignant progression, metastatic dissemination and therapy response is controversial. More importantly, the role of epithelial mesenchymal transition (EMT) and cell plasticity in tumor heterogeneity, clonal selection and clonal evolution is poorly understood. Functionally, our work clarifies the contribution of EMT to malignant progression and metastasis in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37786705"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary fingerprints of EMT in pancreatic cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37786705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231016205028&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mesenchymal plasticity has been extensively described in advanced and metastatic epithelial cancers; however, its functional role in malignant progression, metastatic dissemination and therapy response is controversial. More importantly, the role of epithelial mesenchymal transition (EMT) and cell plasticity in tumor heterogeneity, clonal selection and clonal evolution is poorly understood. Functionally, our work clarifies the contribution of EMT to malignant progression and metastasis in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37786705"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary fingerprints of EMT in pancreatic cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37786705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231015205105&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mesenchymal plasticity has been extensively described in advanced and metastatic epithelial cancers; however, its functional role in malignant progression, metastatic dissemination and therapy response is controversial. More importantly, the role of epithelial mesenchymal transition (EMT) and cell plasticity in tumor heterogeneity, clonal selection and clonal evolution is poorly understood. Functionally, our work clarifies the contribution of EMT to malignant progression and metastasis in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37786705"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary fingerprints of EMT in pancreatic cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37786705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231014205339&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mesenchymal plasticity has been extensively described in advanced and metastatic epithelial cancers; however, its functional role in malignant progression, metastatic dissemination and therapy response is controversial. More importantly, the role of epithelial mesenchymal transition (EMT) and cell plasticity in tumor heterogeneity, clonal selection and clonal evolution is poorly understood. Functionally, our work clarifies the contribution of EMT to malignant progression and metastasis in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37786705"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary fingerprints of EMT in pancreatic cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37786705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231013204816&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mesenchymal plasticity has been extensively described in advanced and metastatic epithelial cancers; however, its functional role in malignant progression, metastatic dissemination and therapy response is controversial. More importantly, the role of epithelial mesenchymal transition (EMT) and cell plasticity in tumor heterogeneity, clonal selection and clonal evolution is poorly understood. Functionally, our work clarifies the contribution of EMT to malignant progression and metastasis in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37786705"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary fingerprints of EMT in pancreatic cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37786705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231012205121&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mesenchymal plasticity has been extensively described in advanced and metastatic epithelial cancers; however, its functional role in malignant progression, metastatic dissemination and therapy response is controversial. More importantly, the role of epithelial mesenchymal transition (EMT) and cell plasticity in tumor heterogeneity, clonal selection and clonal evolution is poorly understood. Functionally, our work clarifies the contribution of EMT to malignant progression and metastasis in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37786705"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary fingerprints of EMT in pancreatic cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37786705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231011204744&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mesenchymal plasticity has been extensively described in advanced and metastatic epithelial cancers; however, its functional role in malignant progression, metastatic dissemination and therapy response is controversial. More importantly, the role of epithelial mesenchymal transition (EMT) and cell plasticity in tumor heterogeneity, clonal selection and clonal evolution is poorly understood. Functionally, our work clarifies the contribution of EMT to malignant progression and metastasis in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37786705"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary fingerprints of EMT in pancreatic cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37786705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231010204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mesenchymal plasticity has been extensively described in advanced and metastatic epithelial cancers; however, its functional role in malignant progression, metastatic dissemination and therapy response is controversial. More importantly, the role of epithelial mesenchymal transition (EMT) and cell plasticity in tumor heterogeneity, clonal selection and clonal evolution is poorly understood. Functionally, our work clarifies the contribution of EMT to malignant progression and metastasis in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37786705"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary fingerprints of EMT in pancreatic cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37786705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231009204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mesenchymal plasticity has been extensively described in advanced and metastatic epithelial cancers; however, its functional role in malignant progression, metastatic dissemination and therapy response is controversial. More importantly, the role of epithelial mesenchymal transition (EMT) and cell plasticity in tumor heterogeneity, clonal selection and clonal evolution is poorly understood. Functionally, our work clarifies the contribution of EMT to malignant progression and metastasis in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37786705"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary fingerprints of EMT in pancreatic cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37786705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231008205021&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mesenchymal plasticity has been extensively described in advanced and metastatic epithelial cancers; however, its functional role in malignant progression, metastatic dissemination and therapy response is controversial. More importantly, the role of epithelial mesenchymal transition (EMT) and cell plasticity in tumor heterogeneity, clonal selection and clonal evolution is poorly understood. Functionally, our work clarifies the contribution of EMT to malignant progression and metastasis in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37786705"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary fingerprints of EMT in pancreatic cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37786705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231007205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mesenchymal plasticity has been extensively described in advanced and metastatic epithelial cancers; however, its functional role in malignant progression, metastatic dissemination and therapy response is controversial. More importantly, the role of epithelial mesenchymal transition (EMT) and cell plasticity in tumor heterogeneity, clonal selection and clonal evolution is poorly understood. Functionally, our work clarifies the contribution of EMT to malignant progression and metastasis in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37786705"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary fingerprints of EMT in pancreatic cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37786705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231006204916&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mesenchymal plasticity has been extensively described in advanced and metastatic epithelial cancers; however, its functional role in malignant progression, metastatic dissemination and therapy response is controversial. More importantly, the role of epithelial mesenchymal transition (EMT) and cell plasticity in tumor heterogeneity, clonal selection and clonal evolution is poorly understood. Functionally, our work clarifies the contribution of EMT to malignant progression and metastasis in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37786705"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary fingerprints of EMT in pancreatic cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37786705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231005204957&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mesenchymal plasticity has been extensively described in advanced and metastatic epithelial cancers; however, its functional role in malignant progression, metastatic dissemination and therapy response is controversial. More importantly, the role of epithelial mesenchymal transition (EMT) and cell plasticity in tumor heterogeneity, clonal selection and clonal evolution is poorly understood. Functionally, our work clarifies the contribution of EMT to malignant progression and metastasis in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37786705"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary fingerprints of EMT in pancreatic cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37786705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231004205006&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mesenchymal plasticity has been extensively described in advanced and metastatic epithelial cancers; however, its functional role in malignant progression, metastatic dissemination and therapy response is controversial. More importantly, the role of epithelial mesenchymal transition (EMT) and cell plasticity in tumor heterogeneity, clonal selection and clonal evolution is poorly understood. Functionally, our work clarifies the contribution of EMT to malignant progression and metastasis in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37786705"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary fingerprints of EMT in pancreatic cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37786705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231003205044&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mesenchymal plasticity has been extensively described in advanced and metastatic epithelial cancers; however, its functional role in malignant progression, metastatic dissemination and therapy response is controversial. More importantly, the role of epithelial mesenchymal transition (EMT) and cell plasticity in tumor heterogeneity, clonal selection and clonal evolution is poorly understood. Functionally, our work clarifies the contribution of EMT to malignant progression and metastasis in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-10-02 datetime="2023-10-02T10:00:00.000Z">2023-10-02</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-10-02 datetime="2023-10-02T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20231105195156">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37781816"
                        open
                      >
                        <summary class="article-expander__title">BRCC3-Mediated NLRP3 Deubiquitylation Promotes Inflammasome Activation and Atherosclerosis in Tet2 Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37781816/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231021205333&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Hypercholesterolemia and TET2 deficiency converge on a common pathway of NLRP3 inflammasome activation mediated by JNK1 activation and BRCC3-mediated NLRP3 deubiquitylation, with potential therapeutic implications for the prevention of cardiovascular disease in TET2 CH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37781816"
                        open
                      >
                        <summary class="article-expander__title">BRCC3-Mediated NLRP3 Deubiquitylation Promotes Inflammasome Activation and Atherosclerosis in Tet2 Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37781816/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231020204820&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Hypercholesterolemia and TET2 deficiency converge on a common pathway of NLRP3 inflammasome activation mediated by JNK1 activation and BRCC3-mediated NLRP3 deubiquitylation, with potential therapeutic implications for the prevention of cardiovascular disease in TET2 CH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37781816"
                        open
                      >
                        <summary class="article-expander__title">BRCC3-Mediated NLRP3 Deubiquitylation Promotes Inflammasome Activation and Atherosclerosis in Tet2 Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37781816/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231019204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Hypercholesterolemia and TET2 deficiency converge on a common pathway of NLRP3 inflammasome activation mediated by JNK1 activation and BRCC3-mediated NLRP3 deubiquitylation, with potential therapeutic implications for the prevention of cardiovascular disease in TET2 CH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37781816"
                        open
                      >
                        <summary class="article-expander__title">BRCC3-Mediated NLRP3 Deubiquitylation Promotes Inflammasome Activation and Atherosclerosis in Tet2 Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37781816/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231018205013&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Hypercholesterolemia and TET2 deficiency converge on a common pathway of NLRP3 inflammasome activation mediated by JNK1 activation and BRCC3-mediated NLRP3 deubiquitylation, with potential therapeutic implications for the prevention of cardiovascular disease in TET2 CH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37781816"
                        open
                      >
                        <summary class="article-expander__title">BRCC3-Mediated NLRP3 Deubiquitylation Promotes Inflammasome Activation and Atherosclerosis in Tet2 Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37781816/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231017205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Hypercholesterolemia and TET2 deficiency converge on a common pathway of NLRP3 inflammasome activation mediated by JNK1 activation and BRCC3-mediated NLRP3 deubiquitylation, with potential therapeutic implications for the prevention of cardiovascular disease in TET2 CH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37781816"
                        open
                      >
                        <summary class="article-expander__title">BRCC3-Mediated NLRP3 Deubiquitylation Promotes Inflammasome Activation and Atherosclerosis in Tet2 Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37781816/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231016205028&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Hypercholesterolemia and TET2 deficiency converge on a common pathway of NLRP3 inflammasome activation mediated by JNK1 activation and BRCC3-mediated NLRP3 deubiquitylation, with potential therapeutic implications for the prevention of cardiovascular disease in TET2 CH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37781816"
                        open
                      >
                        <summary class="article-expander__title">BRCC3-Mediated NLRP3 Deubiquitylation Promotes Inflammasome Activation and Atherosclerosis in Tet2 Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37781816/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231015205105&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Hypercholesterolemia and TET2 deficiency converge on a common pathway of NLRP3 inflammasome activation mediated by JNK1 activation and BRCC3-mediated NLRP3 deubiquitylation, with potential therapeutic implications for the prevention of cardiovascular disease in TET2 CH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37781816"
                        open
                      >
                        <summary class="article-expander__title">BRCC3-Mediated NLRP3 Deubiquitylation Promotes Inflammasome Activation and Atherosclerosis in Tet2 Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37781816/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231014205339&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Hypercholesterolemia and TET2 deficiency converge on a common pathway of NLRP3 inflammasome activation mediated by JNK1 activation and BRCC3-mediated NLRP3 deubiquitylation, with potential therapeutic implications for the prevention of cardiovascular disease in TET2 CH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37781816"
                        open
                      >
                        <summary class="article-expander__title">BRCC3-Mediated NLRP3 Deubiquitylation Promotes Inflammasome Activation and Atherosclerosis in Tet2 Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37781816/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231013204816&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Hypercholesterolemia and TET2 deficiency converge on a common pathway of NLRP3 inflammasome activation mediated by JNK1 activation and BRCC3-mediated NLRP3 deubiquitylation, with potential therapeutic implications for the prevention of cardiovascular disease in TET2 CH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37781816"
                        open
                      >
                        <summary class="article-expander__title">BRCC3-Mediated NLRP3 Deubiquitylation Promotes Inflammasome Activation and Atherosclerosis in Tet2 Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37781816/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231012205120&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Hypercholesterolemia and TET2 deficiency converge on a common pathway of NLRP3 inflammasome activation mediated by JNK1 activation and BRCC3-mediated NLRP3 deubiquitylation, with potential therapeutic implications for the prevention of cardiovascular disease in TET2 CH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37781816"
                        open
                      >
                        <summary class="article-expander__title">BRCC3-Mediated NLRP3 Deubiquitylation Promotes Inflammasome Activation and Atherosclerosis in Tet2 Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37781816/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231011204744&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Hypercholesterolemia and TET2 deficiency converge on a common pathway of NLRP3 inflammasome activation mediated by JNK1 activation and BRCC3-mediated NLRP3 deubiquitylation, with potential therapeutic implications for the prevention of cardiovascular disease in TET2 CH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37781816"
                        open
                      >
                        <summary class="article-expander__title">BRCC3-Mediated NLRP3 Deubiquitylation Promotes Inflammasome Activation and Atherosclerosis in Tet2 Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37781816/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231010204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Hypercholesterolemia and TET2 deficiency converge on a common pathway of NLRP3 inflammasome activation mediated by JNK1 activation and BRCC3-mediated NLRP3 deubiquitylation, with potential therapeutic implications for the prevention of cardiovascular disease in TET2 CH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37781816"
                        open
                      >
                        <summary class="article-expander__title">BRCC3-Mediated NLRP3 Deubiquitylation Promotes Inflammasome Activation and Atherosclerosis in Tet2 Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37781816/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231009204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Hypercholesterolemia and TET2 deficiency converge on a common pathway of NLRP3 inflammasome activation mediated by JNK1 activation and BRCC3-mediated NLRP3 deubiquitylation, with potential therapeutic implications for the prevention of cardiovascular disease in TET2 CH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37781816"
                        open
                      >
                        <summary class="article-expander__title">BRCC3-Mediated NLRP3 Deubiquitylation Promotes Inflammasome Activation and Atherosclerosis in Tet2 Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37781816/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231008205021&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Hypercholesterolemia and TET2 deficiency converge on a common pathway of NLRP3 inflammasome activation mediated by JNK1 activation and BRCC3-mediated NLRP3 deubiquitylation, with potential therapeutic implications for the prevention of cardiovascular disease in TET2 CH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37781816"
                        open
                      >
                        <summary class="article-expander__title">BRCC3-Mediated NLRP3 Deubiquitylation Promotes Inflammasome Activation and Atherosclerosis in Tet2 Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37781816/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231007205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Hypercholesterolemia and TET2 deficiency converge on a common pathway of NLRP3 inflammasome activation mediated by JNK1 activation and BRCC3-mediated NLRP3 deubiquitylation, with potential therapeutic implications for the prevention of cardiovascular disease in TET2 CH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37781816"
                        open
                      >
                        <summary class="article-expander__title">BRCC3-Mediated NLRP3 Deubiquitylation Promotes Inflammasome Activation and Atherosclerosis in Tet2 Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37781816/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231006204916&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Hypercholesterolemia and TET2 deficiency converge on a common pathway of NLRP3 inflammasome activation mediated by JNK1 activation and BRCC3-mediated NLRP3 deubiquitylation, with potential therapeutic implications for the prevention of cardiovascular disease in TET2 CH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37781816"
                        open
                      >
                        <summary class="article-expander__title">BRCC3-Mediated NLRP3 Deubiquitylation Promotes Inflammasome Activation and Atherosclerosis in Tet2 Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37781816/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231005204957&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Hypercholesterolemia and TET2 deficiency converge on a common pathway of NLRP3 inflammasome activation mediated by JNK1 activation and BRCC3-mediated NLRP3 deubiquitylation, with potential therapeutic implications for the prevention of cardiovascular disease in TET2 CH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37781816"
                        open
                      >
                        <summary class="article-expander__title">BRCC3-Mediated NLRP3 Deubiquitylation Promotes Inflammasome Activation and Atherosclerosis in Tet2 Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37781816/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231004205006&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Hypercholesterolemia and TET2 deficiency converge on a common pathway of NLRP3 inflammasome activation mediated by JNK1 activation and BRCC3-mediated NLRP3 deubiquitylation, with potential therapeutic implications for the prevention of cardiovascular disease in TET2 CH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37781816"
                        open
                      >
                        <summary class="article-expander__title">BRCC3-Mediated NLRP3 Deubiquitylation Promotes Inflammasome Activation and Atherosclerosis in Tet2 Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37781816/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231003205046&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Hypercholesterolemia and TET2 deficiency converge on a common pathway of NLRP3 inflammasome activation mediated by JNK1 activation and BRCC3-mediated NLRP3 deubiquitylation, with potential therapeutic implications for the prevention of cardiovascular disease in TET2 CH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37781816"
                        open
                      >
                        <summary class="article-expander__title">BRCC3-Mediated NLRP3 Deubiquitylation Promotes Inflammasome Activation and Atherosclerosis in Tet2 Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37781816/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231002205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Hypercholesterolemia and TET2 deficiency converge on a common pathway of NLRP3 inflammasome activation mediated by JNK1 activation and BRCC3-mediated NLRP3 deubiquitylation, with potential therapeutic implications for the prevention of cardiovascular disease in TET2 CH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;ff&#x3D;20231105195156&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37783803"
                        open
                      >
                        <summary class="article-expander__title">Somatic evolution of marine transmissible leukemias in the common cockle, Cerastoderma edule</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37783803/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231022205027&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transmissible cancers are malignant cell lineages that spread clonally between individuals. Several such cancers, termed bivalve transmissible neoplasia (BTN), induce leukemia-like disease in marine bivalves. This is the case of BTN lineages affecting the common cockle, Cerastoderma edule, which inhabits the Atlantic coasts of Europe and northwest Africa. To investigate the evolution of cockle BTN, we collected 6,854 cockles, diagnosed 390 BTN tumors, generated a reference genome and assessed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37783803"
                        open
                      >
                        <summary class="article-expander__title">Somatic evolution of marine transmissible leukemias in the common cockle, Cerastoderma edule</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37783803/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231021205333&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transmissible cancers are malignant cell lineages that spread clonally between individuals. Several such cancers, termed bivalve transmissible neoplasia (BTN), induce leukemia-like disease in marine bivalves. This is the case of BTN lineages affecting the common cockle, Cerastoderma edule, which inhabits the Atlantic coasts of Europe and northwest Africa. To investigate the evolution of cockle BTN, we collected 6,854 cockles, diagnosed 390 BTN tumors, generated a reference genome and assessed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37783803"
                        open
                      >
                        <summary class="article-expander__title">Somatic evolution of marine transmissible leukemias in the common cockle, Cerastoderma edule</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37783803/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231020204820&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transmissible cancers are malignant cell lineages that spread clonally between individuals. Several such cancers, termed bivalve transmissible neoplasia (BTN), induce leukemia-like disease in marine bivalves. This is the case of BTN lineages affecting the common cockle, Cerastoderma edule, which inhabits the Atlantic coasts of Europe and northwest Africa. To investigate the evolution of cockle BTN, we collected 6,854 cockles, diagnosed 390 BTN tumors, generated a reference genome and assessed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37783803"
                        open
                      >
                        <summary class="article-expander__title">Somatic evolution of marine transmissible leukemias in the common cockle, Cerastoderma edule</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37783803/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231019204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transmissible cancers are malignant cell lineages that spread clonally between individuals. Several such cancers, termed bivalve transmissible neoplasia (BTN), induce leukemia-like disease in marine bivalves. This is the case of BTN lineages affecting the common cockle, Cerastoderma edule, which inhabits the Atlantic coasts of Europe and northwest Africa. To investigate the evolution of cockle BTN, we collected 6,854 cockles, diagnosed 390 BTN tumors, generated a reference genome and assessed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37783803"
                        open
                      >
                        <summary class="article-expander__title">Somatic evolution of marine transmissible leukemias in the common cockle, Cerastoderma edule</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37783803/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231018205013&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transmissible cancers are malignant cell lineages that spread clonally between individuals. Several such cancers, termed bivalve transmissible neoplasia (BTN), induce leukemia-like disease in marine bivalves. This is the case of BTN lineages affecting the common cockle, Cerastoderma edule, which inhabits the Atlantic coasts of Europe and northwest Africa. To investigate the evolution of cockle BTN, we collected 6,854 cockles, diagnosed 390 BTN tumors, generated a reference genome and assessed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37783803"
                        open
                      >
                        <summary class="article-expander__title">Somatic evolution of marine transmissible leukemias in the common cockle, Cerastoderma edule</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37783803/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231017205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transmissible cancers are malignant cell lineages that spread clonally between individuals. Several such cancers, termed bivalve transmissible neoplasia (BTN), induce leukemia-like disease in marine bivalves. This is the case of BTN lineages affecting the common cockle, Cerastoderma edule, which inhabits the Atlantic coasts of Europe and northwest Africa. To investigate the evolution of cockle BTN, we collected 6,854 cockles, diagnosed 390 BTN tumors, generated a reference genome and assessed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37783803"
                        open
                      >
                        <summary class="article-expander__title">Somatic evolution of marine transmissible leukemias in the common cockle, Cerastoderma edule</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37783803/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231016205028&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transmissible cancers are malignant cell lineages that spread clonally between individuals. Several such cancers, termed bivalve transmissible neoplasia (BTN), induce leukemia-like disease in marine bivalves. This is the case of BTN lineages affecting the common cockle, Cerastoderma edule, which inhabits the Atlantic coasts of Europe and northwest Africa. To investigate the evolution of cockle BTN, we collected 6,854 cockles, diagnosed 390 BTN tumors, generated a reference genome and assessed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37783803"
                        open
                      >
                        <summary class="article-expander__title">Somatic evolution of marine transmissible leukemias in the common cockle, Cerastoderma edule</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37783803/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231015205105&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transmissible cancers are malignant cell lineages that spread clonally between individuals. Several such cancers, termed bivalve transmissible neoplasia (BTN), induce leukemia-like disease in marine bivalves. This is the case of BTN lineages affecting the common cockle, Cerastoderma edule, which inhabits the Atlantic coasts of Europe and northwest Africa. To investigate the evolution of cockle BTN, we collected 6,854 cockles, diagnosed 390 BTN tumors, generated a reference genome and assessed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37783803"
                        open
                      >
                        <summary class="article-expander__title">Somatic evolution of marine transmissible leukemias in the common cockle, Cerastoderma edule</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37783803/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231014205339&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transmissible cancers are malignant cell lineages that spread clonally between individuals. Several such cancers, termed bivalve transmissible neoplasia (BTN), induce leukemia-like disease in marine bivalves. This is the case of BTN lineages affecting the common cockle, Cerastoderma edule, which inhabits the Atlantic coasts of Europe and northwest Africa. To investigate the evolution of cockle BTN, we collected 6,854 cockles, diagnosed 390 BTN tumors, generated a reference genome and assessed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37783803"
                        open
                      >
                        <summary class="article-expander__title">Somatic evolution of marine transmissible leukemias in the common cockle, Cerastoderma edule</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37783803/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231013204816&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transmissible cancers are malignant cell lineages that spread clonally between individuals. Several such cancers, termed bivalve transmissible neoplasia (BTN), induce leukemia-like disease in marine bivalves. This is the case of BTN lineages affecting the common cockle, Cerastoderma edule, which inhabits the Atlantic coasts of Europe and northwest Africa. To investigate the evolution of cockle BTN, we collected 6,854 cockles, diagnosed 390 BTN tumors, generated a reference genome and assessed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37783803"
                        open
                      >
                        <summary class="article-expander__title">Somatic evolution of marine transmissible leukemias in the common cockle, Cerastoderma edule</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37783803/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231012205121&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transmissible cancers are malignant cell lineages that spread clonally between individuals. Several such cancers, termed bivalve transmissible neoplasia (BTN), induce leukemia-like disease in marine bivalves. This is the case of BTN lineages affecting the common cockle, Cerastoderma edule, which inhabits the Atlantic coasts of Europe and northwest Africa. To investigate the evolution of cockle BTN, we collected 6,854 cockles, diagnosed 390 BTN tumors, generated a reference genome and assessed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37783803"
                        open
                      >
                        <summary class="article-expander__title">Somatic evolution of marine transmissible leukemias in the common cockle, Cerastoderma edule</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37783803/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231011204744&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transmissible cancers are malignant cell lineages that spread clonally between individuals. Several such cancers, termed bivalve transmissible neoplasia (BTN), induce leukemia-like disease in marine bivalves. This is the case of BTN lineages affecting the common cockle, Cerastoderma edule, which inhabits the Atlantic coasts of Europe and northwest Africa. To investigate the evolution of cockle BTN, we collected 6,854 cockles, diagnosed 390 BTN tumors, generated a reference genome and assessed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37783803"
                        open
                      >
                        <summary class="article-expander__title">Somatic evolution of marine transmissible leukemias in the common cockle, Cerastoderma edule</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37783803/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231010204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transmissible cancers are malignant cell lineages that spread clonally between individuals. Several such cancers, termed bivalve transmissible neoplasia (BTN), induce leukemia-like disease in marine bivalves. This is the case of BTN lineages affecting the common cockle, Cerastoderma edule, which inhabits the Atlantic coasts of Europe and northwest Africa. To investigate the evolution of cockle BTN, we collected 6,854 cockles, diagnosed 390 BTN tumors, generated a reference genome and assessed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37783803"
                        open
                      >
                        <summary class="article-expander__title">Somatic evolution of marine transmissible leukemias in the common cockle, Cerastoderma edule</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37783803/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231009204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transmissible cancers are malignant cell lineages that spread clonally between individuals. Several such cancers, termed bivalve transmissible neoplasia (BTN), induce leukemia-like disease in marine bivalves. This is the case of BTN lineages affecting the common cockle, Cerastoderma edule, which inhabits the Atlantic coasts of Europe and northwest Africa. To investigate the evolution of cockle BTN, we collected 6,854 cockles, diagnosed 390 BTN tumors, generated a reference genome and assessed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37783803"
                        open
                      >
                        <summary class="article-expander__title">Somatic evolution of marine transmissible leukemias in the common cockle, Cerastoderma edule</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37783803/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231008205021&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transmissible cancers are malignant cell lineages that spread clonally between individuals. Several such cancers, termed bivalve transmissible neoplasia (BTN), induce leukemia-like disease in marine bivalves. This is the case of BTN lineages affecting the common cockle, Cerastoderma edule, which inhabits the Atlantic coasts of Europe and northwest Africa. To investigate the evolution of cockle BTN, we collected 6,854 cockles, diagnosed 390 BTN tumors, generated a reference genome and assessed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37783803"
                        open
                      >
                        <summary class="article-expander__title">Somatic evolution of marine transmissible leukemias in the common cockle, Cerastoderma edule</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37783803/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231007205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transmissible cancers are malignant cell lineages that spread clonally between individuals. Several such cancers, termed bivalve transmissible neoplasia (BTN), induce leukemia-like disease in marine bivalves. This is the case of BTN lineages affecting the common cockle, Cerastoderma edule, which inhabits the Atlantic coasts of Europe and northwest Africa. To investigate the evolution of cockle BTN, we collected 6,854 cockles, diagnosed 390 BTN tumors, generated a reference genome and assessed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37783803"
                        open
                      >
                        <summary class="article-expander__title">Somatic evolution of marine transmissible leukemias in the common cockle, Cerastoderma edule</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37783803/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231006204916&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transmissible cancers are malignant cell lineages that spread clonally between individuals. Several such cancers, termed bivalve transmissible neoplasia (BTN), induce leukemia-like disease in marine bivalves. This is the case of BTN lineages affecting the common cockle, Cerastoderma edule, which inhabits the Atlantic coasts of Europe and northwest Africa. To investigate the evolution of cockle BTN, we collected 6,854 cockles, diagnosed 390 BTN tumors, generated a reference genome and assessed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37783803"
                        open
                      >
                        <summary class="article-expander__title">Somatic evolution of marine transmissible leukemias in the common cockle, Cerastoderma edule</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37783803/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231005204957&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transmissible cancers are malignant cell lineages that spread clonally between individuals. Several such cancers, termed bivalve transmissible neoplasia (BTN), induce leukemia-like disease in marine bivalves. This is the case of BTN lineages affecting the common cockle, Cerastoderma edule, which inhabits the Atlantic coasts of Europe and northwest Africa. To investigate the evolution of cockle BTN, we collected 6,854 cockles, diagnosed 390 BTN tumors, generated a reference genome and assessed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37783803"
                        open
                      >
                        <summary class="article-expander__title">Somatic evolution of marine transmissible leukemias in the common cockle, Cerastoderma edule</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37783803/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231004205006&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transmissible cancers are malignant cell lineages that spread clonally between individuals. Several such cancers, termed bivalve transmissible neoplasia (BTN), induce leukemia-like disease in marine bivalves. This is the case of BTN lineages affecting the common cockle, Cerastoderma edule, which inhabits the Atlantic coasts of Europe and northwest Africa. To investigate the evolution of cockle BTN, we collected 6,854 cockles, diagnosed 390 BTN tumors, generated a reference genome and assessed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37783803"
                        open
                      >
                        <summary class="article-expander__title">Somatic evolution of marine transmissible leukemias in the common cockle, Cerastoderma edule</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37783803/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231003205044&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transmissible cancers are malignant cell lineages that spread clonally between individuals. Several such cancers, termed bivalve transmissible neoplasia (BTN), induce leukemia-like disease in marine bivalves. This is the case of BTN lineages affecting the common cockle, Cerastoderma edule, which inhabits the Atlantic coasts of Europe and northwest Africa. To investigate the evolution of cockle BTN, we collected 6,854 cockles, diagnosed 390 BTN tumors, generated a reference genome and assessed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-10-01 datetime="2023-10-01T10:00:00.000Z">2023-10-01</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-10-01 datetime="2023-10-01T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20231105195156">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37777824"
                        open
                      >
                        <summary class="article-expander__title">Genome-First Approach of the Prevalence and Cancer Phenotypes of Pathogenic or Likely Pathogenic Germline TP53 Variants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37777824/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231021205333&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pathogenic or likely pathogenic (P/LP) germline TP53 variants are the primary cause of Li-Fraumeni syndrome (LFS), a hereditary cancer predisposition disorder characterized by early-onset cancers. The population prevalence of P/LP germline TP53 variants is estimated to be approximately one in every 3,500-20,000 individuals. However, these estimates are likely impacted by ascertainment biases and lack of clinical and genetic data to account for potential confounding factors, such as clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37777824"
                        open
                      >
                        <summary class="article-expander__title">Genome-First Approach of the Prevalence and Cancer Phenotypes of Pathogenic or Likely Pathogenic Germline TP53 Variants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37777824/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231020204820&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pathogenic or likely pathogenic (P/LP) germline TP53 variants are the primary cause of Li-Fraumeni syndrome (LFS), a hereditary cancer predisposition disorder characterized by early-onset cancers. The population prevalence of P/LP germline TP53 variants is estimated to be approximately one in every 3,500-20,000 individuals. However, these estimates are likely impacted by ascertainment biases and lack of clinical and genetic data to account for potential confounding factors, such as clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37777824"
                        open
                      >
                        <summary class="article-expander__title">Genome-First Approach of the Prevalence and Cancer Phenotypes of Pathogenic or Likely Pathogenic Germline TP53 Variants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37777824/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231019204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pathogenic or likely pathogenic (P/LP) germline TP53 variants are the primary cause of Li-Fraumeni syndrome (LFS), a hereditary cancer predisposition disorder characterized by early-onset cancers. The population prevalence of P/LP germline TP53 variants is estimated to be approximately one in every 3,500-20,000 individuals. However, these estimates are likely impacted by ascertainment biases and lack of clinical and genetic data to account for potential confounding factors, such as clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37777824"
                        open
                      >
                        <summary class="article-expander__title">Genome-First Approach of the Prevalence and Cancer Phenotypes of Pathogenic or Likely Pathogenic Germline TP53 Variants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37777824/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231018205013&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pathogenic or likely pathogenic (P/LP) germline TP53 variants are the primary cause of Li-Fraumeni syndrome (LFS), a hereditary cancer predisposition disorder characterized by early-onset cancers. The population prevalence of P/LP germline TP53 variants is estimated to be approximately one in every 3,500-20,000 individuals. However, these estimates are likely impacted by ascertainment biases and lack of clinical and genetic data to account for potential confounding factors, such as clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37777824"
                        open
                      >
                        <summary class="article-expander__title">Genome-First Approach of the Prevalence and Cancer Phenotypes of Pathogenic or Likely Pathogenic Germline TP53 Variants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37777824/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231017205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pathogenic or likely pathogenic (P/LP) germline TP53 variants are the primary cause of Li-Fraumeni syndrome (LFS), a hereditary cancer predisposition disorder characterized by early-onset cancers. The population prevalence of P/LP germline TP53 variants is estimated to be approximately one in every 3,500-20,000 individuals. However, these estimates are likely impacted by ascertainment biases and lack of clinical and genetic data to account for potential confounding factors, such as clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37777824"
                        open
                      >
                        <summary class="article-expander__title">Genome-First Approach of the Prevalence and Cancer Phenotypes of Pathogenic or Likely Pathogenic Germline TP53 Variants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37777824/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231016205028&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pathogenic or likely pathogenic (P/LP) germline TP53 variants are the primary cause of Li-Fraumeni syndrome (LFS), a hereditary cancer predisposition disorder characterized by early-onset cancers. The population prevalence of P/LP germline TP53 variants is estimated to be approximately one in every 3,500-20,000 individuals. However, these estimates are likely impacted by ascertainment biases and lack of clinical and genetic data to account for potential confounding factors, such as clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37777824"
                        open
                      >
                        <summary class="article-expander__title">Genome-First Approach of the Prevalence and Cancer Phenotypes of Pathogenic or Likely Pathogenic Germline TP53 Variants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37777824/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231015205105&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pathogenic or likely pathogenic (P/LP) germline TP53 variants are the primary cause of Li-Fraumeni syndrome (LFS), a hereditary cancer predisposition disorder characterized by early-onset cancers. The population prevalence of P/LP germline TP53 variants is estimated to be approximately one in every 3,500-20,000 individuals. However, these estimates are likely impacted by ascertainment biases and lack of clinical and genetic data to account for potential confounding factors, such as clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37777824"
                        open
                      >
                        <summary class="article-expander__title">Genome-First Approach of the Prevalence and Cancer Phenotypes of Pathogenic or Likely Pathogenic Germline TP53 Variants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37777824/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231014205339&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pathogenic or likely pathogenic (P/LP) germline TP53 variants are the primary cause of Li-Fraumeni syndrome (LFS), a hereditary cancer predisposition disorder characterized by early-onset cancers. The population prevalence of P/LP germline TP53 variants is estimated to be approximately one in every 3,500-20,000 individuals. However, these estimates are likely impacted by ascertainment biases and lack of clinical and genetic data to account for potential confounding factors, such as clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37777824"
                        open
                      >
                        <summary class="article-expander__title">Genome-First Approach of the Prevalence and Cancer Phenotypes of Pathogenic or Likely Pathogenic Germline TP53 Variants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37777824/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231013204816&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pathogenic or likely pathogenic (P/LP) germline TP53 variants are the primary cause of Li-Fraumeni syndrome (LFS), a hereditary cancer predisposition disorder characterized by early-onset cancers. The population prevalence of P/LP germline TP53 variants is estimated to be approximately one in every 3,500-20,000 individuals. However, these estimates are likely impacted by ascertainment biases and lack of clinical and genetic data to account for potential confounding factors, such as clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37777824"
                        open
                      >
                        <summary class="article-expander__title">Genome-First Approach of the Prevalence and Cancer Phenotypes of Pathogenic or Likely Pathogenic Germline TP53 Variants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37777824/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231012205120&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pathogenic or likely pathogenic (P/LP) germline TP53 variants are the primary cause of Li-Fraumeni syndrome (LFS), a hereditary cancer predisposition disorder characterized by early-onset cancers. The population prevalence of P/LP germline TP53 variants is estimated to be approximately one in every 3,500-20,000 individuals. However, these estimates are likely impacted by ascertainment biases and lack of clinical and genetic data to account for potential confounding factors, such as clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37777824"
                        open
                      >
                        <summary class="article-expander__title">Genome-First Approach of the Prevalence and Cancer Phenotypes of Pathogenic or Likely Pathogenic Germline TP53 Variants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37777824/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231011204744&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pathogenic or likely pathogenic (P/LP) germline TP53 variants are the primary cause of Li-Fraumeni syndrome (LFS), a hereditary cancer predisposition disorder characterized by early-onset cancers. The population prevalence of P/LP germline TP53 variants is estimated to be approximately one in every 3,500-20,000 individuals. However, these estimates are likely impacted by ascertainment biases and lack of clinical and genetic data to account for potential confounding factors, such as clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37777824"
                        open
                      >
                        <summary class="article-expander__title">Genome-First Approach of the Prevalence and Cancer Phenotypes of Pathogenic or Likely Pathogenic Germline TP53 Variants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37777824/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231010204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pathogenic or likely pathogenic (P/LP) germline TP53 variants are the primary cause of Li-Fraumeni syndrome (LFS), a hereditary cancer predisposition disorder characterized by early-onset cancers. The population prevalence of P/LP germline TP53 variants is estimated to be approximately one in every 3,500-20,000 individuals. However, these estimates are likely impacted by ascertainment biases and lack of clinical and genetic data to account for potential confounding factors, such as clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37777824"
                        open
                      >
                        <summary class="article-expander__title">Genome-First Approach of the Prevalence and Cancer Phenotypes of Pathogenic or Likely Pathogenic Germline TP53 Variants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37777824/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231009204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pathogenic or likely pathogenic (P/LP) germline TP53 variants are the primary cause of Li-Fraumeni syndrome (LFS), a hereditary cancer predisposition disorder characterized by early-onset cancers. The population prevalence of P/LP germline TP53 variants is estimated to be approximately one in every 3,500-20,000 individuals. However, these estimates are likely impacted by ascertainment biases and lack of clinical and genetic data to account for potential confounding factors, such as clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37777824"
                        open
                      >
                        <summary class="article-expander__title">Genome-First Approach of the Prevalence and Cancer Phenotypes of Pathogenic or Likely Pathogenic Germline TP53 Variants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37777824/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231008205021&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pathogenic or likely pathogenic (P/LP) germline TP53 variants are the primary cause of Li-Fraumeni syndrome (LFS), a hereditary cancer predisposition disorder characterized by early-onset cancers. The population prevalence of P/LP germline TP53 variants is estimated to be approximately one in every 3,500-20,000 individuals. However, these estimates are likely impacted by ascertainment biases and lack of clinical and genetic data to account for potential confounding factors, such as clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37777824"
                        open
                      >
                        <summary class="article-expander__title">Genome-First Approach of the Prevalence and Cancer Phenotypes of Pathogenic or Likely Pathogenic Germline TP53 Variants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37777824/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231007205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pathogenic or likely pathogenic (P/LP) germline TP53 variants are the primary cause of Li-Fraumeni syndrome (LFS), a hereditary cancer predisposition disorder characterized by early-onset cancers. The population prevalence of P/LP germline TP53 variants is estimated to be approximately one in every 3,500-20,000 individuals. However, these estimates are likely impacted by ascertainment biases and lack of clinical and genetic data to account for potential confounding factors, such as clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37777824"
                        open
                      >
                        <summary class="article-expander__title">Genome-First Approach of the Prevalence and Cancer Phenotypes of Pathogenic or Likely Pathogenic Germline TP53 Variants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37777824/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231006204916&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pathogenic or likely pathogenic (P/LP) germline TP53 variants are the primary cause of Li-Fraumeni syndrome (LFS), a hereditary cancer predisposition disorder characterized by early-onset cancers. The population prevalence of P/LP germline TP53 variants is estimated to be approximately one in every 3,500-20,000 individuals. However, these estimates are likely impacted by ascertainment biases and lack of clinical and genetic data to account for potential confounding factors, such as clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37777824"
                        open
                      >
                        <summary class="article-expander__title">Genome-First Approach of the Prevalence and Cancer Phenotypes of Pathogenic or Likely Pathogenic Germline TP53 Variants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37777824/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231005204957&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pathogenic or likely pathogenic (P/LP) germline TP53 variants are the primary cause of Li-Fraumeni syndrome (LFS), a hereditary cancer predisposition disorder characterized by early-onset cancers. The population prevalence of P/LP germline TP53 variants is estimated to be approximately one in every 3,500-20,000 individuals. However, these estimates are likely impacted by ascertainment biases and lack of clinical and genetic data to account for potential confounding factors, such as clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37777824"
                        open
                      >
                        <summary class="article-expander__title">Genome-First Approach of the Prevalence and Cancer Phenotypes of Pathogenic or Likely Pathogenic Germline TP53 Variants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37777824/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231004205006&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pathogenic or likely pathogenic (P/LP) germline TP53 variants are the primary cause of Li-Fraumeni syndrome (LFS), a hereditary cancer predisposition disorder characterized by early-onset cancers. The population prevalence of P/LP germline TP53 variants is estimated to be approximately one in every 3,500-20,000 individuals. However, these estimates are likely impacted by ascertainment biases and lack of clinical and genetic data to account for potential confounding factors, such as clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37777824"
                        open
                      >
                        <summary class="article-expander__title">Genome-First Approach of the Prevalence and Cancer Phenotypes of Pathogenic or Likely Pathogenic Germline TP53 Variants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37777824/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231003205046&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pathogenic or likely pathogenic (P/LP) germline TP53 variants are the primary cause of Li-Fraumeni syndrome (LFS), a hereditary cancer predisposition disorder characterized by early-onset cancers. The population prevalence of P/LP germline TP53 variants is estimated to be approximately one in every 3,500-20,000 individuals. However, these estimates are likely impacted by ascertainment biases and lack of clinical and genetic data to account for potential confounding factors, such as clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37777824"
                        open
                      >
                        <summary class="article-expander__title">Genome-First Approach of the Prevalence and Cancer Phenotypes of Pathogenic or Likely Pathogenic Germline TP53 Variants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37777824/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231002205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pathogenic or likely pathogenic (P/LP) germline TP53 variants are the primary cause of Li-Fraumeni syndrome (LFS), a hereditary cancer predisposition disorder characterized by early-onset cancers. The population prevalence of P/LP germline TP53 variants is estimated to be approximately one in every 3,500-20,000 individuals. However, these estimates are likely impacted by ascertainment biases and lack of clinical and genetic data to account for potential confounding factors, such as clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37777824"
                        open
                      >
                        <summary class="article-expander__title">Genome-First Approach of the Prevalence and Cancer Phenotypes of Pathogenic or Likely Pathogenic Germline TP53 Variants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37777824/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231001205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Pathogenic or likely pathogenic (P/LP) germline TP53 variants are the primary cause of Li-Fraumeni syndrome (LFS), a hereditary cancer predisposition disorder characterized by early-onset cancers. The population prevalence of P/LP germline TP53 variants is estimated to be approximately one in every 3,500-20,000 individuals. However, these estimates are likely impacted by ascertainment biases and lack of clinical and genetic data to account for potential confounding factors, such as clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-09-29 datetime="2023-09-29T10:00:00.000Z">2023-09-29</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-09-29 datetime="2023-09-29T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20231105195156">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37774372"
                        open
                      >
                        <summary class="article-expander__title">Diagnosis and classification of myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37774372/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231020204820&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are neoplastic myeloid proliferations characterized by ineffective hematopoiesis resulting in peripheral blood cytopenias. MDS is distinguished from non-neoplastic clonal myeloid proliferations by the presence of morphologic dysplasia and from acute myeloid leukemia (AML) by a blast threshold of 20%. The diagnosis of MDS can be challenging due to the myriad other causes of cytopenias: accurate diagnosis requires the integration of clinical features with bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37774372"
                        open
                      >
                        <summary class="article-expander__title">Diagnosis and classification of myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37774372/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231019204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are neoplastic myeloid proliferations characterized by ineffective hematopoiesis resulting in peripheral blood cytopenias. MDS is distinguished from non-neoplastic clonal myeloid proliferations by the presence of morphologic dysplasia and from acute myeloid leukemia (AML) by a blast threshold of 20%. The diagnosis of MDS can be challenging due to the myriad other causes of cytopenias: accurate diagnosis requires the integration of clinical features with bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37774372"
                        open
                      >
                        <summary class="article-expander__title">Diagnosis and classification of myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37774372/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231018205013&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are neoplastic myeloid proliferations characterized by ineffective hematopoiesis resulting in peripheral blood cytopenias. MDS is distinguished from non-neoplastic clonal myeloid proliferations by the presence of morphologic dysplasia and from acute myeloid leukemia (AML) by a blast threshold of 20%. The diagnosis of MDS can be challenging due to the myriad other causes of cytopenias: accurate diagnosis requires the integration of clinical features with bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37774372"
                        open
                      >
                        <summary class="article-expander__title">Diagnosis and classification of myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37774372/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231017205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are neoplastic myeloid proliferations characterized by ineffective hematopoiesis resulting in peripheral blood cytopenias. MDS is distinguished from non-neoplastic clonal myeloid proliferations by the presence of morphologic dysplasia and from acute myeloid leukemia (AML) by a blast threshold of 20%. The diagnosis of MDS can be challenging due to the myriad other causes of cytopenias: accurate diagnosis requires the integration of clinical features with bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37774372"
                        open
                      >
                        <summary class="article-expander__title">Diagnosis and classification of myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37774372/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231016205028&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are neoplastic myeloid proliferations characterized by ineffective hematopoiesis resulting in peripheral blood cytopenias. MDS is distinguished from non-neoplastic clonal myeloid proliferations by the presence of morphologic dysplasia and from acute myeloid leukemia (AML) by a blast threshold of 20%. The diagnosis of MDS can be challenging due to the myriad other causes of cytopenias: accurate diagnosis requires the integration of clinical features with bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37774372"
                        open
                      >
                        <summary class="article-expander__title">Diagnosis and classification of myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37774372/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231015205105&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are neoplastic myeloid proliferations characterized by ineffective hematopoiesis resulting in peripheral blood cytopenias. MDS is distinguished from non-neoplastic clonal myeloid proliferations by the presence of morphologic dysplasia and from acute myeloid leukemia (AML) by a blast threshold of 20%. The diagnosis of MDS can be challenging due to the myriad other causes of cytopenias: accurate diagnosis requires the integration of clinical features with bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37774372"
                        open
                      >
                        <summary class="article-expander__title">Diagnosis and classification of myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37774372/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231014205339&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are neoplastic myeloid proliferations characterized by ineffective hematopoiesis resulting in peripheral blood cytopenias. MDS is distinguished from non-neoplastic clonal myeloid proliferations by the presence of morphologic dysplasia and from acute myeloid leukemia (AML) by a blast threshold of 20%. The diagnosis of MDS can be challenging due to the myriad other causes of cytopenias: accurate diagnosis requires the integration of clinical features with bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37774372"
                        open
                      >
                        <summary class="article-expander__title">Diagnosis and classification of myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37774372/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231013204816&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are neoplastic myeloid proliferations characterized by ineffective hematopoiesis resulting in peripheral blood cytopenias. MDS is distinguished from non-neoplastic clonal myeloid proliferations by the presence of morphologic dysplasia and from acute myeloid leukemia (AML) by a blast threshold of 20%. The diagnosis of MDS can be challenging due to the myriad other causes of cytopenias: accurate diagnosis requires the integration of clinical features with bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37774372"
                        open
                      >
                        <summary class="article-expander__title">Diagnosis and classification of myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37774372/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231012205120&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are neoplastic myeloid proliferations characterized by ineffective hematopoiesis resulting in peripheral blood cytopenias. MDS is distinguished from non-neoplastic clonal myeloid proliferations by the presence of morphologic dysplasia and from acute myeloid leukemia (AML) by a blast threshold of 20%. The diagnosis of MDS can be challenging due to the myriad other causes of cytopenias: accurate diagnosis requires the integration of clinical features with bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37774372"
                        open
                      >
                        <summary class="article-expander__title">Diagnosis and classification of myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37774372/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231011204744&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are neoplastic myeloid proliferations characterized by ineffective hematopoiesis resulting in peripheral blood cytopenias. MDS is distinguished from non-neoplastic clonal myeloid proliferations by the presence of morphologic dysplasia and from acute myeloid leukemia (AML) by a blast threshold of 20%. The diagnosis of MDS can be challenging due to the myriad other causes of cytopenias: accurate diagnosis requires the integration of clinical features with bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37774372"
                        open
                      >
                        <summary class="article-expander__title">Diagnosis and classification of myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37774372/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231010204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are neoplastic myeloid proliferations characterized by ineffective hematopoiesis resulting in peripheral blood cytopenias. MDS is distinguished from non-neoplastic clonal myeloid proliferations by the presence of morphologic dysplasia and from acute myeloid leukemia (AML) by a blast threshold of 20%. The diagnosis of MDS can be challenging due to the myriad other causes of cytopenias: accurate diagnosis requires the integration of clinical features with bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37774372"
                        open
                      >
                        <summary class="article-expander__title">Diagnosis and classification of myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37774372/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231009204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are neoplastic myeloid proliferations characterized by ineffective hematopoiesis resulting in peripheral blood cytopenias. MDS is distinguished from non-neoplastic clonal myeloid proliferations by the presence of morphologic dysplasia and from acute myeloid leukemia (AML) by a blast threshold of 20%. The diagnosis of MDS can be challenging due to the myriad other causes of cytopenias: accurate diagnosis requires the integration of clinical features with bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37774372"
                        open
                      >
                        <summary class="article-expander__title">Diagnosis and classification of myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37774372/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231008205021&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are neoplastic myeloid proliferations characterized by ineffective hematopoiesis resulting in peripheral blood cytopenias. MDS is distinguished from non-neoplastic clonal myeloid proliferations by the presence of morphologic dysplasia and from acute myeloid leukemia (AML) by a blast threshold of 20%. The diagnosis of MDS can be challenging due to the myriad other causes of cytopenias: accurate diagnosis requires the integration of clinical features with bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37774372"
                        open
                      >
                        <summary class="article-expander__title">Diagnosis and classification of myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37774372/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231007205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are neoplastic myeloid proliferations characterized by ineffective hematopoiesis resulting in peripheral blood cytopenias. MDS is distinguished from non-neoplastic clonal myeloid proliferations by the presence of morphologic dysplasia and from acute myeloid leukemia (AML) by a blast threshold of 20%. The diagnosis of MDS can be challenging due to the myriad other causes of cytopenias: accurate diagnosis requires the integration of clinical features with bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37774372"
                        open
                      >
                        <summary class="article-expander__title">Diagnosis and classification of myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37774372/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231006204916&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are neoplastic myeloid proliferations characterized by ineffective hematopoiesis resulting in peripheral blood cytopenias. MDS is distinguished from non-neoplastic clonal myeloid proliferations by the presence of morphologic dysplasia and from acute myeloid leukemia (AML) by a blast threshold of 20%. The diagnosis of MDS can be challenging due to the myriad other causes of cytopenias: accurate diagnosis requires the integration of clinical features with bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37774372"
                        open
                      >
                        <summary class="article-expander__title">Diagnosis and classification of myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37774372/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231005204957&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are neoplastic myeloid proliferations characterized by ineffective hematopoiesis resulting in peripheral blood cytopenias. MDS is distinguished from non-neoplastic clonal myeloid proliferations by the presence of morphologic dysplasia and from acute myeloid leukemia (AML) by a blast threshold of 20%. The diagnosis of MDS can be challenging due to the myriad other causes of cytopenias: accurate diagnosis requires the integration of clinical features with bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37774372"
                        open
                      >
                        <summary class="article-expander__title">Diagnosis and classification of myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37774372/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231004205006&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are neoplastic myeloid proliferations characterized by ineffective hematopoiesis resulting in peripheral blood cytopenias. MDS is distinguished from non-neoplastic clonal myeloid proliferations by the presence of morphologic dysplasia and from acute myeloid leukemia (AML) by a blast threshold of 20%. The diagnosis of MDS can be challenging due to the myriad other causes of cytopenias: accurate diagnosis requires the integration of clinical features with bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37774372"
                        open
                      >
                        <summary class="article-expander__title">Diagnosis and classification of myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37774372/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231003205046&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are neoplastic myeloid proliferations characterized by ineffective hematopoiesis resulting in peripheral blood cytopenias. MDS is distinguished from non-neoplastic clonal myeloid proliferations by the presence of morphologic dysplasia and from acute myeloid leukemia (AML) by a blast threshold of 20%. The diagnosis of MDS can be challenging due to the myriad other causes of cytopenias: accurate diagnosis requires the integration of clinical features with bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37774372"
                        open
                      >
                        <summary class="article-expander__title">Diagnosis and classification of myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37774372/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231002205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are neoplastic myeloid proliferations characterized by ineffective hematopoiesis resulting in peripheral blood cytopenias. MDS is distinguished from non-neoplastic clonal myeloid proliferations by the presence of morphologic dysplasia and from acute myeloid leukemia (AML) by a blast threshold of 20%. The diagnosis of MDS can be challenging due to the myriad other causes of cytopenias: accurate diagnosis requires the integration of clinical features with bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37774372"
                        open
                      >
                        <summary class="article-expander__title">Diagnosis and classification of myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37774372/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231001205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are neoplastic myeloid proliferations characterized by ineffective hematopoiesis resulting in peripheral blood cytopenias. MDS is distinguished from non-neoplastic clonal myeloid proliferations by the presence of morphologic dysplasia and from acute myeloid leukemia (AML) by a blast threshold of 20%. The diagnosis of MDS can be challenging due to the myriad other causes of cytopenias: accurate diagnosis requires the integration of clinical features with bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37774372"
                        open
                      >
                        <summary class="article-expander__title">Diagnosis and classification of myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37774372/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230930205630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are neoplastic myeloid proliferations characterized by ineffective hematopoiesis resulting in peripheral blood cytopenias. MDS is distinguished from non-neoplastic clonal myeloid proliferations by the presence of morphologic dysplasia and from acute myeloid leukemia (AML) by a blast threshold of 20%. The diagnosis of MDS can be challenging due to the myriad other causes of cytopenias: accurate diagnosis requires the integration of clinical features with bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37774372"
                        open
                      >
                        <summary class="article-expander__title">Diagnosis and classification of myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37774372/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230929204823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are neoplastic myeloid proliferations characterized by ineffective hematopoiesis resulting in peripheral blood cytopenias. MDS is distinguished from non-neoplastic clonal myeloid proliferations by the presence of morphologic dysplasia and from acute myeloid leukemia (AML) by a blast threshold of 20%. The diagnosis of MDS can be challenging due to the myriad other causes of cytopenias: accurate diagnosis requires the integration of clinical features with bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-09-28 datetime="2023-09-28T10:00:00.000Z">2023-09-28</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-09-28 datetime="2023-09-28T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20231105195156">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37769965"
                        open
                      >
                        <summary class="article-expander__title">The spectrum of glomerular and vascular kidney pathology associated with myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37769965/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231018205013&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A high prevalence of chronic kidney disease (CKD) occurs in patients with myeloproliferative neoplasms (MPN). However, MPN-related glomerulopathy (MPN-RG) may not account for the entirety of CKD risk in this population. The systemic vasculopathy encountered in these patients raises the hypothesis that vascular nephrosclerosis may be a common pattern of injury in patients with MPN and with CKD. In an exhaustive, retrospective, multicenter study of MPN kidney biopsies in four different pathology...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37769965"
                        open
                      >
                        <summary class="article-expander__title">The spectrum of glomerular and vascular kidney pathology associated with myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37769965/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231017205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A high prevalence of chronic kidney disease (CKD) occurs in patients with myeloproliferative neoplasms (MPN). However, MPN-related glomerulopathy (MPN-RG) may not account for the entirety of CKD risk in this population. The systemic vasculopathy encountered in these patients raises the hypothesis that vascular nephrosclerosis may be a common pattern of injury in patients with MPN and with CKD. In an exhaustive, retrospective, multicenter study of MPN kidney biopsies in four different pathology...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37769965"
                        open
                      >
                        <summary class="article-expander__title">The spectrum of glomerular and vascular kidney pathology associated with myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37769965/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231016205028&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A high prevalence of chronic kidney disease (CKD) occurs in patients with myeloproliferative neoplasms (MPN). However, MPN-related glomerulopathy (MPN-RG) may not account for the entirety of CKD risk in this population. The systemic vasculopathy encountered in these patients raises the hypothesis that vascular nephrosclerosis may be a common pattern of injury in patients with MPN and with CKD. In an exhaustive, retrospective, multicenter study of MPN kidney biopsies in four different pathology...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37769965"
                        open
                      >
                        <summary class="article-expander__title">The spectrum of glomerular and vascular kidney pathology associated with myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37769965/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231015205105&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A high prevalence of chronic kidney disease (CKD) occurs in patients with myeloproliferative neoplasms (MPN). However, MPN-related glomerulopathy (MPN-RG) may not account for the entirety of CKD risk in this population. The systemic vasculopathy encountered in these patients raises the hypothesis that vascular nephrosclerosis may be a common pattern of injury in patients with MPN and with CKD. In an exhaustive, retrospective, multicenter study of MPN kidney biopsies in four different pathology...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37769965"
                        open
                      >
                        <summary class="article-expander__title">The spectrum of glomerular and vascular kidney pathology associated with myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37769965/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231014205339&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A high prevalence of chronic kidney disease (CKD) occurs in patients with myeloproliferative neoplasms (MPN). However, MPN-related glomerulopathy (MPN-RG) may not account for the entirety of CKD risk in this population. The systemic vasculopathy encountered in these patients raises the hypothesis that vascular nephrosclerosis may be a common pattern of injury in patients with MPN and with CKD. In an exhaustive, retrospective, multicenter study of MPN kidney biopsies in four different pathology...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37769965"
                        open
                      >
                        <summary class="article-expander__title">The spectrum of glomerular and vascular kidney pathology associated with myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37769965/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231013204816&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A high prevalence of chronic kidney disease (CKD) occurs in patients with myeloproliferative neoplasms (MPN). However, MPN-related glomerulopathy (MPN-RG) may not account for the entirety of CKD risk in this population. The systemic vasculopathy encountered in these patients raises the hypothesis that vascular nephrosclerosis may be a common pattern of injury in patients with MPN and with CKD. In an exhaustive, retrospective, multicenter study of MPN kidney biopsies in four different pathology...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37763205"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential and Cardiovascular Risk in Patients with Chronic Kidney Disease without Previous Cardiac Pathology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37763205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231013204816&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is defined by the clonal expansion of hematopoietic stem cells carrying certain genes associated with an increased risk of hematological malignancies. Our study analyzes the influence of CHIP on the risk of heart disease and cardiovascular events in a population with chronic kidney disease (CKD). A total of 128 patients were prospectively followed up for 18 months to detect major cardiovascular events (MACE). To detect the presence of silent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37760623"
                        open
                      >
                        <summary class="article-expander__title">Exploring the Molecular Landscape of Myelofibrosis, with a Focus on Ras and Mitogen-Activated Protein (MAP) Kinase Signaling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37760623/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231013204816&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is a clonal myeloproliferative neoplasm (MPN) characterized clinically by cytopenias, fatigue, and splenomegaly stemming from extramedullary hematopoiesis. MF commonly arises from mutations in JAK2, MPL, and CALR, which manifests as hyperactive Jak/Stat signaling. Triple-negative MF is diagnosed in the absence of JAK2, MPL, and CALR but when clinical, morphologic criteria are met and other mutation(s) is/are present, including ASXL1, EZH2, and SRSF2. While the clinical and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37769965"
                        open
                      >
                        <summary class="article-expander__title">The spectrum of glomerular and vascular kidney pathology associated with myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37769965/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231012205120&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A high prevalence of chronic kidney disease (CKD) occurs in patients with myeloproliferative neoplasms (MPN). However, MPN-related glomerulopathy (MPN-RG) may not account for the entirety of CKD risk in this population. The systemic vasculopathy encountered in these patients raises the hypothesis that vascular nephrosclerosis may be a common pattern of injury in patients with MPN and with CKD. In an exhaustive, retrospective, multicenter study of MPN kidney biopsies in four different pathology...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37763205"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential and Cardiovascular Risk in Patients with Chronic Kidney Disease without Previous Cardiac Pathology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37763205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231012205120&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is defined by the clonal expansion of hematopoietic stem cells carrying certain genes associated with an increased risk of hematological malignancies. Our study analyzes the influence of CHIP on the risk of heart disease and cardiovascular events in a population with chronic kidney disease (CKD). A total of 128 patients were prospectively followed up for 18 months to detect major cardiovascular events (MACE). To detect the presence of silent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37760623"
                        open
                      >
                        <summary class="article-expander__title">Exploring the Molecular Landscape of Myelofibrosis, with a Focus on Ras and Mitogen-Activated Protein (MAP) Kinase Signaling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37760623/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231012205120&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is a clonal myeloproliferative neoplasm (MPN) characterized clinically by cytopenias, fatigue, and splenomegaly stemming from extramedullary hematopoiesis. MF commonly arises from mutations in JAK2, MPL, and CALR, which manifests as hyperactive Jak/Stat signaling. Triple-negative MF is diagnosed in the absence of JAK2, MPL, and CALR but when clinical, morphologic criteria are met and other mutation(s) is/are present, including ASXL1, EZH2, and SRSF2. While the clinical and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37769965"
                        open
                      >
                        <summary class="article-expander__title">The spectrum of glomerular and vascular kidney pathology associated with myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37769965/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231011204744&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A high prevalence of chronic kidney disease (CKD) occurs in patients with myeloproliferative neoplasms (MPN). However, MPN-related glomerulopathy (MPN-RG) may not account for the entirety of CKD risk in this population. The systemic vasculopathy encountered in these patients raises the hypothesis that vascular nephrosclerosis may be a common pattern of injury in patients with MPN and with CKD. In an exhaustive, retrospective, multicenter study of MPN kidney biopsies in four different pathology...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37763205"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential and Cardiovascular Risk in Patients with Chronic Kidney Disease without Previous Cardiac Pathology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37763205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231011204744&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is defined by the clonal expansion of hematopoietic stem cells carrying certain genes associated with an increased risk of hematological malignancies. Our study analyzes the influence of CHIP on the risk of heart disease and cardiovascular events in a population with chronic kidney disease (CKD). A total of 128 patients were prospectively followed up for 18 months to detect major cardiovascular events (MACE). To detect the presence of silent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37760623"
                        open
                      >
                        <summary class="article-expander__title">Exploring the Molecular Landscape of Myelofibrosis, with a Focus on Ras and Mitogen-Activated Protein (MAP) Kinase Signaling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37760623/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231011204744&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is a clonal myeloproliferative neoplasm (MPN) characterized clinically by cytopenias, fatigue, and splenomegaly stemming from extramedullary hematopoiesis. MF commonly arises from mutations in JAK2, MPL, and CALR, which manifests as hyperactive Jak/Stat signaling. Triple-negative MF is diagnosed in the absence of JAK2, MPL, and CALR but when clinical, morphologic criteria are met and other mutation(s) is/are present, including ASXL1, EZH2, and SRSF2. While the clinical and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37769965"
                        open
                      >
                        <summary class="article-expander__title">The spectrum of glomerular and vascular kidney pathology associated with myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37769965/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231010204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A high prevalence of chronic kidney disease (CKD) occurs in patients with myeloproliferative neoplasms (MPN). However, MPN-related glomerulopathy (MPN-RG) may not account for the entirety of CKD risk in this population. The systemic vasculopathy encountered in these patients raises the hypothesis that vascular nephrosclerosis may be a common pattern of injury in patients with MPN and with CKD. In an exhaustive, retrospective, multicenter study of MPN kidney biopsies in four different pathology...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37763205"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential and Cardiovascular Risk in Patients with Chronic Kidney Disease without Previous Cardiac Pathology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37763205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231010204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is defined by the clonal expansion of hematopoietic stem cells carrying certain genes associated with an increased risk of hematological malignancies. Our study analyzes the influence of CHIP on the risk of heart disease and cardiovascular events in a population with chronic kidney disease (CKD). A total of 128 patients were prospectively followed up for 18 months to detect major cardiovascular events (MACE). To detect the presence of silent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37760623"
                        open
                      >
                        <summary class="article-expander__title">Exploring the Molecular Landscape of Myelofibrosis, with a Focus on Ras and Mitogen-Activated Protein (MAP) Kinase Signaling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37760623/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231010204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is a clonal myeloproliferative neoplasm (MPN) characterized clinically by cytopenias, fatigue, and splenomegaly stemming from extramedullary hematopoiesis. MF commonly arises from mutations in JAK2, MPL, and CALR, which manifests as hyperactive Jak/Stat signaling. Triple-negative MF is diagnosed in the absence of JAK2, MPL, and CALR but when clinical, morphologic criteria are met and other mutation(s) is/are present, including ASXL1, EZH2, and SRSF2. While the clinical and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37769965"
                        open
                      >
                        <summary class="article-expander__title">The spectrum of glomerular and vascular kidney pathology associated with myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37769965/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231009204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A high prevalence of chronic kidney disease (CKD) occurs in patients with myeloproliferative neoplasms (MPN). However, MPN-related glomerulopathy (MPN-RG) may not account for the entirety of CKD risk in this population. The systemic vasculopathy encountered in these patients raises the hypothesis that vascular nephrosclerosis may be a common pattern of injury in patients with MPN and with CKD. In an exhaustive, retrospective, multicenter study of MPN kidney biopsies in four different pathology...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37763205"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential and Cardiovascular Risk in Patients with Chronic Kidney Disease without Previous Cardiac Pathology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37763205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231009204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is defined by the clonal expansion of hematopoietic stem cells carrying certain genes associated with an increased risk of hematological malignancies. Our study analyzes the influence of CHIP on the risk of heart disease and cardiovascular events in a population with chronic kidney disease (CKD). A total of 128 patients were prospectively followed up for 18 months to detect major cardiovascular events (MACE). To detect the presence of silent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37760623"
                        open
                      >
                        <summary class="article-expander__title">Exploring the Molecular Landscape of Myelofibrosis, with a Focus on Ras and Mitogen-Activated Protein (MAP) Kinase Signaling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37760623/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231009204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is a clonal myeloproliferative neoplasm (MPN) characterized clinically by cytopenias, fatigue, and splenomegaly stemming from extramedullary hematopoiesis. MF commonly arises from mutations in JAK2, MPL, and CALR, which manifests as hyperactive Jak/Stat signaling. Triple-negative MF is diagnosed in the absence of JAK2, MPL, and CALR but when clinical, morphologic criteria are met and other mutation(s) is/are present, including ASXL1, EZH2, and SRSF2. While the clinical and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37769965"
                        open
                      >
                        <summary class="article-expander__title">The spectrum of glomerular and vascular kidney pathology associated with myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37769965/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231008205021&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A high prevalence of chronic kidney disease (CKD) occurs in patients with myeloproliferative neoplasms (MPN). However, MPN-related glomerulopathy (MPN-RG) may not account for the entirety of CKD risk in this population. The systemic vasculopathy encountered in these patients raises the hypothesis that vascular nephrosclerosis may be a common pattern of injury in patients with MPN and with CKD. In an exhaustive, retrospective, multicenter study of MPN kidney biopsies in four different pathology...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37763205"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential and Cardiovascular Risk in Patients with Chronic Kidney Disease without Previous Cardiac Pathology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37763205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231008205021&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is defined by the clonal expansion of hematopoietic stem cells carrying certain genes associated with an increased risk of hematological malignancies. Our study analyzes the influence of CHIP on the risk of heart disease and cardiovascular events in a population with chronic kidney disease (CKD). A total of 128 patients were prospectively followed up for 18 months to detect major cardiovascular events (MACE). To detect the presence of silent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37760623"
                        open
                      >
                        <summary class="article-expander__title">Exploring the Molecular Landscape of Myelofibrosis, with a Focus on Ras and Mitogen-Activated Protein (MAP) Kinase Signaling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37760623/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231008205021&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is a clonal myeloproliferative neoplasm (MPN) characterized clinically by cytopenias, fatigue, and splenomegaly stemming from extramedullary hematopoiesis. MF commonly arises from mutations in JAK2, MPL, and CALR, which manifests as hyperactive Jak/Stat signaling. Triple-negative MF is diagnosed in the absence of JAK2, MPL, and CALR but when clinical, morphologic criteria are met and other mutation(s) is/are present, including ASXL1, EZH2, and SRSF2. While the clinical and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37769965"
                        open
                      >
                        <summary class="article-expander__title">The spectrum of glomerular and vascular kidney pathology associated with myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37769965/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231007205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A high prevalence of chronic kidney disease (CKD) occurs in patients with myeloproliferative neoplasms (MPN). However, MPN-related glomerulopathy (MPN-RG) may not account for the entirety of CKD risk in this population. The systemic vasculopathy encountered in these patients raises the hypothesis that vascular nephrosclerosis may be a common pattern of injury in patients with MPN and with CKD. In an exhaustive, retrospective, multicenter study of MPN kidney biopsies in four different pathology...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37763205"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential and Cardiovascular Risk in Patients with Chronic Kidney Disease without Previous Cardiac Pathology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37763205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231007205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is defined by the clonal expansion of hematopoietic stem cells carrying certain genes associated with an increased risk of hematological malignancies. Our study analyzes the influence of CHIP on the risk of heart disease and cardiovascular events in a population with chronic kidney disease (CKD). A total of 128 patients were prospectively followed up for 18 months to detect major cardiovascular events (MACE). To detect the presence of silent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37760623"
                        open
                      >
                        <summary class="article-expander__title">Exploring the Molecular Landscape of Myelofibrosis, with a Focus on Ras and Mitogen-Activated Protein (MAP) Kinase Signaling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37760623/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231007205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is a clonal myeloproliferative neoplasm (MPN) characterized clinically by cytopenias, fatigue, and splenomegaly stemming from extramedullary hematopoiesis. MF commonly arises from mutations in JAK2, MPL, and CALR, which manifests as hyperactive Jak/Stat signaling. Triple-negative MF is diagnosed in the absence of JAK2, MPL, and CALR but when clinical, morphologic criteria are met and other mutation(s) is/are present, including ASXL1, EZH2, and SRSF2. While the clinical and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37769965"
                        open
                      >
                        <summary class="article-expander__title">The spectrum of glomerular and vascular kidney pathology associated with myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37769965/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231006204916&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A high prevalence of chronic kidney disease (CKD) occurs in patients with myeloproliferative neoplasms (MPN). However, MPN-related glomerulopathy (MPN-RG) may not account for the entirety of CKD risk in this population. The systemic vasculopathy encountered in these patients raises the hypothesis that vascular nephrosclerosis may be a common pattern of injury in patients with MPN and with CKD. In an exhaustive, retrospective, multicenter study of MPN kidney biopsies in four different pathology...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37763205"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential and Cardiovascular Risk in Patients with Chronic Kidney Disease without Previous Cardiac Pathology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37763205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231006204916&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is defined by the clonal expansion of hematopoietic stem cells carrying certain genes associated with an increased risk of hematological malignancies. Our study analyzes the influence of CHIP on the risk of heart disease and cardiovascular events in a population with chronic kidney disease (CKD). A total of 128 patients were prospectively followed up for 18 months to detect major cardiovascular events (MACE). To detect the presence of silent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37760623"
                        open
                      >
                        <summary class="article-expander__title">Exploring the Molecular Landscape of Myelofibrosis, with a Focus on Ras and Mitogen-Activated Protein (MAP) Kinase Signaling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37760623/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231006204916&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is a clonal myeloproliferative neoplasm (MPN) characterized clinically by cytopenias, fatigue, and splenomegaly stemming from extramedullary hematopoiesis. MF commonly arises from mutations in JAK2, MPL, and CALR, which manifests as hyperactive Jak/Stat signaling. Triple-negative MF is diagnosed in the absence of JAK2, MPL, and CALR but when clinical, morphologic criteria are met and other mutation(s) is/are present, including ASXL1, EZH2, and SRSF2. While the clinical and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37769965"
                        open
                      >
                        <summary class="article-expander__title">The spectrum of glomerular and vascular kidney pathology associated with myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37769965/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231005204957&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A high prevalence of chronic kidney disease (CKD) occurs in patients with myeloproliferative neoplasms (MPN). However, MPN-related glomerulopathy (MPN-RG) may not account for the entirety of CKD risk in this population. The systemic vasculopathy encountered in these patients raises the hypothesis that vascular nephrosclerosis may be a common pattern of injury in patients with MPN and with CKD. In an exhaustive, retrospective, multicenter study of MPN kidney biopsies in four different pathology...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37763205"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential and Cardiovascular Risk in Patients with Chronic Kidney Disease without Previous Cardiac Pathology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37763205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231005204957&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is defined by the clonal expansion of hematopoietic stem cells carrying certain genes associated with an increased risk of hematological malignancies. Our study analyzes the influence of CHIP on the risk of heart disease and cardiovascular events in a population with chronic kidney disease (CKD). A total of 128 patients were prospectively followed up for 18 months to detect major cardiovascular events (MACE). To detect the presence of silent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37760623"
                        open
                      >
                        <summary class="article-expander__title">Exploring the Molecular Landscape of Myelofibrosis, with a Focus on Ras and Mitogen-Activated Protein (MAP) Kinase Signaling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37760623/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231005204957&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is a clonal myeloproliferative neoplasm (MPN) characterized clinically by cytopenias, fatigue, and splenomegaly stemming from extramedullary hematopoiesis. MF commonly arises from mutations in JAK2, MPL, and CALR, which manifests as hyperactive Jak/Stat signaling. Triple-negative MF is diagnosed in the absence of JAK2, MPL, and CALR but when clinical, morphologic criteria are met and other mutation(s) is/are present, including ASXL1, EZH2, and SRSF2. While the clinical and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37769965"
                        open
                      >
                        <summary class="article-expander__title">The spectrum of glomerular and vascular kidney pathology associated with myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37769965/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231004205006&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A high prevalence of chronic kidney disease (CKD) occurs in patients with myeloproliferative neoplasms (MPN). However, MPN-related glomerulopathy (MPN-RG) may not account for the entirety of CKD risk in this population. The systemic vasculopathy encountered in these patients raises the hypothesis that vascular nephrosclerosis may be a common pattern of injury in patients with MPN and with CKD. In an exhaustive, retrospective, multicenter study of MPN kidney biopsies in four different pathology...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37763205"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential and Cardiovascular Risk in Patients with Chronic Kidney Disease without Previous Cardiac Pathology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37763205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231004205006&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is defined by the clonal expansion of hematopoietic stem cells carrying certain genes associated with an increased risk of hematological malignancies. Our study analyzes the influence of CHIP on the risk of heart disease and cardiovascular events in a population with chronic kidney disease (CKD). A total of 128 patients were prospectively followed up for 18 months to detect major cardiovascular events (MACE). To detect the presence of silent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37760623"
                        open
                      >
                        <summary class="article-expander__title">Exploring the Molecular Landscape of Myelofibrosis, with a Focus on Ras and Mitogen-Activated Protein (MAP) Kinase Signaling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37760623/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231004205006&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is a clonal myeloproliferative neoplasm (MPN) characterized clinically by cytopenias, fatigue, and splenomegaly stemming from extramedullary hematopoiesis. MF commonly arises from mutations in JAK2, MPL, and CALR, which manifests as hyperactive Jak/Stat signaling. Triple-negative MF is diagnosed in the absence of JAK2, MPL, and CALR but when clinical, morphologic criteria are met and other mutation(s) is/are present, including ASXL1, EZH2, and SRSF2. While the clinical and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37769965"
                        open
                      >
                        <summary class="article-expander__title">The spectrum of glomerular and vascular kidney pathology associated with myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37769965/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231003205046&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A high prevalence of chronic kidney disease (CKD) occurs in patients with myeloproliferative neoplasms (MPN). However, MPN-related glomerulopathy (MPN-RG) may not account for the entirety of CKD risk in this population. The systemic vasculopathy encountered in these patients raises the hypothesis that vascular nephrosclerosis may be a common pattern of injury in patients with MPN and with CKD. In an exhaustive, retrospective, multicenter study of MPN kidney biopsies in four different pathology...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37763205"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential and Cardiovascular Risk in Patients with Chronic Kidney Disease without Previous Cardiac Pathology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37763205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231003205046&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is defined by the clonal expansion of hematopoietic stem cells carrying certain genes associated with an increased risk of hematological malignancies. Our study analyzes the influence of CHIP on the risk of heart disease and cardiovascular events in a population with chronic kidney disease (CKD). A total of 128 patients were prospectively followed up for 18 months to detect major cardiovascular events (MACE). To detect the presence of silent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37760623"
                        open
                      >
                        <summary class="article-expander__title">Exploring the Molecular Landscape of Myelofibrosis, with a Focus on Ras and Mitogen-Activated Protein (MAP) Kinase Signaling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37760623/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231003205046&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is a clonal myeloproliferative neoplasm (MPN) characterized clinically by cytopenias, fatigue, and splenomegaly stemming from extramedullary hematopoiesis. MF commonly arises from mutations in JAK2, MPL, and CALR, which manifests as hyperactive Jak/Stat signaling. Triple-negative MF is diagnosed in the absence of JAK2, MPL, and CALR but when clinical, morphologic criteria are met and other mutation(s) is/are present, including ASXL1, EZH2, and SRSF2. While the clinical and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37769965"
                        open
                      >
                        <summary class="article-expander__title">The spectrum of glomerular and vascular kidney pathology associated with myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37769965/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231002205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A high prevalence of chronic kidney disease (CKD) occurs in patients with myeloproliferative neoplasms (MPN). However, MPN-related glomerulopathy (MPN-RG) may not account for the entirety of CKD risk in this population. The systemic vasculopathy encountered in these patients raises the hypothesis that vascular nephrosclerosis may be a common pattern of injury in patients with MPN and with CKD. In an exhaustive, retrospective, multicenter study of MPN kidney biopsies in four different pathology...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37763205"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential and Cardiovascular Risk in Patients with Chronic Kidney Disease without Previous Cardiac Pathology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37763205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231002205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is defined by the clonal expansion of hematopoietic stem cells carrying certain genes associated with an increased risk of hematological malignancies. Our study analyzes the influence of CHIP on the risk of heart disease and cardiovascular events in a population with chronic kidney disease (CKD). A total of 128 patients were prospectively followed up for 18 months to detect major cardiovascular events (MACE). To detect the presence of silent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37760623"
                        open
                      >
                        <summary class="article-expander__title">Exploring the Molecular Landscape of Myelofibrosis, with a Focus on Ras and Mitogen-Activated Protein (MAP) Kinase Signaling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37760623/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231002205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is a clonal myeloproliferative neoplasm (MPN) characterized clinically by cytopenias, fatigue, and splenomegaly stemming from extramedullary hematopoiesis. MF commonly arises from mutations in JAK2, MPL, and CALR, which manifests as hyperactive Jak/Stat signaling. Triple-negative MF is diagnosed in the absence of JAK2, MPL, and CALR but when clinical, morphologic criteria are met and other mutation(s) is/are present, including ASXL1, EZH2, and SRSF2. While the clinical and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37769965"
                        open
                      >
                        <summary class="article-expander__title">The spectrum of glomerular and vascular kidney pathology associated with myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37769965/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231001205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A high prevalence of chronic kidney disease (CKD) occurs in patients with myeloproliferative neoplasms (MPN). However, MPN-related glomerulopathy (MPN-RG) may not account for the entirety of CKD risk in this population. The systemic vasculopathy encountered in these patients raises the hypothesis that vascular nephrosclerosis may be a common pattern of injury in patients with MPN and with CKD. In an exhaustive, retrospective, multicenter study of MPN kidney biopsies in four different pathology...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37763205"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential and Cardiovascular Risk in Patients with Chronic Kidney Disease without Previous Cardiac Pathology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37763205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231001205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is defined by the clonal expansion of hematopoietic stem cells carrying certain genes associated with an increased risk of hematological malignancies. Our study analyzes the influence of CHIP on the risk of heart disease and cardiovascular events in a population with chronic kidney disease (CKD). A total of 128 patients were prospectively followed up for 18 months to detect major cardiovascular events (MACE). To detect the presence of silent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37760623"
                        open
                      >
                        <summary class="article-expander__title">Exploring the Molecular Landscape of Myelofibrosis, with a Focus on Ras and Mitogen-Activated Protein (MAP) Kinase Signaling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37760623/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231001205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is a clonal myeloproliferative neoplasm (MPN) characterized clinically by cytopenias, fatigue, and splenomegaly stemming from extramedullary hematopoiesis. MF commonly arises from mutations in JAK2, MPL, and CALR, which manifests as hyperactive Jak/Stat signaling. Triple-negative MF is diagnosed in the absence of JAK2, MPL, and CALR but when clinical, morphologic criteria are met and other mutation(s) is/are present, including ASXL1, EZH2, and SRSF2. While the clinical and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37769965"
                        open
                      >
                        <summary class="article-expander__title">The spectrum of glomerular and vascular kidney pathology associated with myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37769965/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230930205630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A high prevalence of chronic kidney disease (CKD) occurs in patients with myeloproliferative neoplasms (MPN). However, MPN-related glomerulopathy (MPN-RG) may not account for the entirety of CKD risk in this population. The systemic vasculopathy encountered in these patients raises the hypothesis that vascular nephrosclerosis may be a common pattern of injury in patients with MPN and with CKD. In an exhaustive, retrospective, multicenter study of MPN kidney biopsies in four different pathology...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37763205"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential and Cardiovascular Risk in Patients with Chronic Kidney Disease without Previous Cardiac Pathology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37763205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230930205630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is defined by the clonal expansion of hematopoietic stem cells carrying certain genes associated with an increased risk of hematological malignancies. Our study analyzes the influence of CHIP on the risk of heart disease and cardiovascular events in a population with chronic kidney disease (CKD). A total of 128 patients were prospectively followed up for 18 months to detect major cardiovascular events (MACE). To detect the presence of silent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37760623"
                        open
                      >
                        <summary class="article-expander__title">Exploring the Molecular Landscape of Myelofibrosis, with a Focus on Ras and Mitogen-Activated Protein (MAP) Kinase Signaling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37760623/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230930205630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is a clonal myeloproliferative neoplasm (MPN) characterized clinically by cytopenias, fatigue, and splenomegaly stemming from extramedullary hematopoiesis. MF commonly arises from mutations in JAK2, MPL, and CALR, which manifests as hyperactive Jak/Stat signaling. Triple-negative MF is diagnosed in the absence of JAK2, MPL, and CALR but when clinical, morphologic criteria are met and other mutation(s) is/are present, including ASXL1, EZH2, and SRSF2. While the clinical and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37769965"
                        open
                      >
                        <summary class="article-expander__title">The spectrum of glomerular and vascular kidney pathology associated with myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37769965/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230929204823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A high prevalence of chronic kidney disease (CKD) occurs in patients with myeloproliferative neoplasms (MPN). However, MPN-related glomerulopathy (MPN-RG) may not account for the entirety of CKD risk in this population. The systemic vasculopathy encountered in these patients raises the hypothesis that vascular nephrosclerosis may be a common pattern of injury in patients with MPN and with CKD. In an exhaustive, retrospective, multicenter study of MPN kidney biopsies in four different pathology...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37763205"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential and Cardiovascular Risk in Patients with Chronic Kidney Disease without Previous Cardiac Pathology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37763205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230929204823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is defined by the clonal expansion of hematopoietic stem cells carrying certain genes associated with an increased risk of hematological malignancies. Our study analyzes the influence of CHIP on the risk of heart disease and cardiovascular events in a population with chronic kidney disease (CKD). A total of 128 patients were prospectively followed up for 18 months to detect major cardiovascular events (MACE). To detect the presence of silent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37760623"
                        open
                      >
                        <summary class="article-expander__title">Exploring the Molecular Landscape of Myelofibrosis, with a Focus on Ras and Mitogen-Activated Protein (MAP) Kinase Signaling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37760623/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230929204823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is a clonal myeloproliferative neoplasm (MPN) characterized clinically by cytopenias, fatigue, and splenomegaly stemming from extramedullary hematopoiesis. MF commonly arises from mutations in JAK2, MPL, and CALR, which manifests as hyperactive Jak/Stat signaling. Triple-negative MF is diagnosed in the absence of JAK2, MPL, and CALR but when clinical, morphologic criteria are met and other mutation(s) is/are present, including ASXL1, EZH2, and SRSF2. While the clinical and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37763205"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential and Cardiovascular Risk in Patients with Chronic Kidney Disease without Previous Cardiac Pathology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37763205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230928204947&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is defined by the clonal expansion of hematopoietic stem cells carrying certain genes associated with an increased risk of hematological malignancies. Our study analyzes the influence of CHIP on the risk of heart disease and cardiovascular events in a population with chronic kidney disease (CKD). A total of 128 patients were prospectively followed up for 18 months to detect major cardiovascular events (MACE). To detect the presence of silent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37760623"
                        open
                      >
                        <summary class="article-expander__title">Exploring the Molecular Landscape of Myelofibrosis, with a Focus on Ras and Mitogen-Activated Protein (MAP) Kinase Signaling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37760623/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230928204947&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is a clonal myeloproliferative neoplasm (MPN) characterized clinically by cytopenias, fatigue, and splenomegaly stemming from extramedullary hematopoiesis. MF commonly arises from mutations in JAK2, MPL, and CALR, which manifests as hyperactive Jak/Stat signaling. Triple-negative MF is diagnosed in the absence of JAK2, MPL, and CALR but when clinical, morphologic criteria are met and other mutation(s) is/are present, including ASXL1, EZH2, and SRSF2. While the clinical and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-09-27 datetime="2023-09-27T10:00:00.000Z">2023-09-27</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-09-27 datetime="2023-09-27T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20231105195156">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37756546"
                        open
                      >
                        <summary class="article-expander__title">Runx1-R188Q germline mutation induces inflammation and predisposition to hematologic malignancies in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37756546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231013204816&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline mutations in the RUNX1 gene cause familial platelet disorder (FPD), an inherited disease associated with lifetime risk to hematopoietic malignancies (HM). FPD patients frequently show clonal expansion of pre-malignant cells preceding HM onset. Despite the extensive studies on the role of RUNX1 in hematopoiesis, its function in the pre-malignant bone marrow is not well understood. Here, we characterized the hematopoietic progenitor compartments using a mouse strain carrying an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37756546"
                        open
                      >
                        <summary class="article-expander__title">Runx1-R188Q germline mutation induces inflammation and predisposition to hematologic malignancies in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37756546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231012205120&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline mutations in the RUNX1 gene cause familial platelet disorder (FPD), an inherited disease associated with lifetime risk to hematopoietic malignancies (HM). FPD patients frequently show clonal expansion of pre-malignant cells preceding HM onset. Despite the extensive studies on the role of RUNX1 in hematopoiesis, its function in the pre-malignant bone marrow is not well understood. Here, we characterized the hematopoietic progenitor compartments using a mouse strain carrying an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37756531"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential (CHIP) and Incident Type 2 Diabetes Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37756531/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231012205120&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP was associated with higher incidence of T2D. CHIP mutations located on genes implicated in CHD and mortality were also related to T2D, suggesting shared aging-related pathology.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37756546"
                        open
                      >
                        <summary class="article-expander__title">Runx1-R188Q germline mutation induces inflammation and predisposition to hematologic malignancies in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37756546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231011204744&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline mutations in the RUNX1 gene cause familial platelet disorder (FPD), an inherited disease associated with lifetime risk to hematopoietic malignancies (HM). FPD patients frequently show clonal expansion of pre-malignant cells preceding HM onset. Despite the extensive studies on the role of RUNX1 in hematopoiesis, its function in the pre-malignant bone marrow is not well understood. Here, we characterized the hematopoietic progenitor compartments using a mouse strain carrying an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37756531"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential (CHIP) and Incident Type 2 Diabetes Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37756531/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231011204744&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP was associated with higher incidence of T2D. CHIP mutations located on genes implicated in CHD and mortality were also related to T2D, suggesting shared aging-related pathology.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37756546"
                        open
                      >
                        <summary class="article-expander__title">Runx1-R188Q germline mutation induces inflammation and predisposition to hematologic malignancies in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37756546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231010204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline mutations in the RUNX1 gene cause familial platelet disorder (FPD), an inherited disease associated with lifetime risk to hematopoietic malignancies (HM). FPD patients frequently show clonal expansion of pre-malignant cells preceding HM onset. Despite the extensive studies on the role of RUNX1 in hematopoiesis, its function in the pre-malignant bone marrow is not well understood. Here, we characterized the hematopoietic progenitor compartments using a mouse strain carrying an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37756531"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential (CHIP) and Incident Type 2 Diabetes Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37756531/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231010204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP was associated with higher incidence of T2D. CHIP mutations located on genes implicated in CHD and mortality were also related to T2D, suggesting shared aging-related pathology.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37756546"
                        open
                      >
                        <summary class="article-expander__title">Runx1-R188Q germline mutation induces inflammation and predisposition to hematologic malignancies in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37756546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231009204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline mutations in the RUNX1 gene cause familial platelet disorder (FPD), an inherited disease associated with lifetime risk to hematopoietic malignancies (HM). FPD patients frequently show clonal expansion of pre-malignant cells preceding HM onset. Despite the extensive studies on the role of RUNX1 in hematopoiesis, its function in the pre-malignant bone marrow is not well understood. Here, we characterized the hematopoietic progenitor compartments using a mouse strain carrying an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37756531"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential (CHIP) and Incident Type 2 Diabetes Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37756531/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231009204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP was associated with higher incidence of T2D. CHIP mutations located on genes implicated in CHD and mortality were also related to T2D, suggesting shared aging-related pathology.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37756546"
                        open
                      >
                        <summary class="article-expander__title">Runx1-R188Q germline mutation induces inflammation and predisposition to hematologic malignancies in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37756546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231008205021&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline mutations in the RUNX1 gene cause familial platelet disorder (FPD), an inherited disease associated with lifetime risk to hematopoietic malignancies (HM). FPD patients frequently show clonal expansion of pre-malignant cells preceding HM onset. Despite the extensive studies on the role of RUNX1 in hematopoiesis, its function in the pre-malignant bone marrow is not well understood. Here, we characterized the hematopoietic progenitor compartments using a mouse strain carrying an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37756531"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential (CHIP) and Incident Type 2 Diabetes Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37756531/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231008205021&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP was associated with higher incidence of T2D. CHIP mutations located on genes implicated in CHD and mortality were also related to T2D, suggesting shared aging-related pathology.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37756546"
                        open
                      >
                        <summary class="article-expander__title">Runx1-R188Q germline mutation induces inflammation and predisposition to hematologic malignancies in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37756546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231007205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline mutations in the RUNX1 gene cause familial platelet disorder (FPD), an inherited disease associated with lifetime risk to hematopoietic malignancies (HM). FPD patients frequently show clonal expansion of pre-malignant cells preceding HM onset. Despite the extensive studies on the role of RUNX1 in hematopoiesis, its function in the pre-malignant bone marrow is not well understood. Here, we characterized the hematopoietic progenitor compartments using a mouse strain carrying an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37756531"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential (CHIP) and Incident Type 2 Diabetes Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37756531/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231007205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP was associated with higher incidence of T2D. CHIP mutations located on genes implicated in CHD and mortality were also related to T2D, suggesting shared aging-related pathology.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37756546"
                        open
                      >
                        <summary class="article-expander__title">Runx1-R188Q germline mutation induces inflammation and predisposition to hematologic malignancies in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37756546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231006204916&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline mutations in the RUNX1 gene cause familial platelet disorder (FPD), an inherited disease associated with lifetime risk to hematopoietic malignancies (HM). FPD patients frequently show clonal expansion of pre-malignant cells preceding HM onset. Despite the extensive studies on the role of RUNX1 in hematopoiesis, its function in the pre-malignant bone marrow is not well understood. Here, we characterized the hematopoietic progenitor compartments using a mouse strain carrying an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37756531"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential (CHIP) and Incident Type 2 Diabetes Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37756531/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231006204916&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP was associated with higher incidence of T2D. CHIP mutations located on genes implicated in CHD and mortality were also related to T2D, suggesting shared aging-related pathology.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37756546"
                        open
                      >
                        <summary class="article-expander__title">Runx1-R188Q germline mutation induces inflammation and predisposition to hematologic malignancies in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37756546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231005204957&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline mutations in the RUNX1 gene cause familial platelet disorder (FPD), an inherited disease associated with lifetime risk to hematopoietic malignancies (HM). FPD patients frequently show clonal expansion of pre-malignant cells preceding HM onset. Despite the extensive studies on the role of RUNX1 in hematopoiesis, its function in the pre-malignant bone marrow is not well understood. Here, we characterized the hematopoietic progenitor compartments using a mouse strain carrying an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37756531"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential (CHIP) and Incident Type 2 Diabetes Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37756531/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231005204957&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP was associated with higher incidence of T2D. CHIP mutations located on genes implicated in CHD and mortality were also related to T2D, suggesting shared aging-related pathology.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37756546"
                        open
                      >
                        <summary class="article-expander__title">Runx1-R188Q germline mutation induces inflammation and predisposition to hematologic malignancies in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37756546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231004205006&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline mutations in the RUNX1 gene cause familial platelet disorder (FPD), an inherited disease associated with lifetime risk to hematopoietic malignancies (HM). FPD patients frequently show clonal expansion of pre-malignant cells preceding HM onset. Despite the extensive studies on the role of RUNX1 in hematopoiesis, its function in the pre-malignant bone marrow is not well understood. Here, we characterized the hematopoietic progenitor compartments using a mouse strain carrying an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37756531"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential (CHIP) and Incident Type 2 Diabetes Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37756531/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231004205006&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP was associated with higher incidence of T2D. CHIP mutations located on genes implicated in CHD and mortality were also related to T2D, suggesting shared aging-related pathology.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37756546"
                        open
                      >
                        <summary class="article-expander__title">Runx1-R188Q germline mutation induces inflammation and predisposition to hematologic malignancies in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37756546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231003205046&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline mutations in the RUNX1 gene cause familial platelet disorder (FPD), an inherited disease associated with lifetime risk to hematopoietic malignancies (HM). FPD patients frequently show clonal expansion of pre-malignant cells preceding HM onset. Despite the extensive studies on the role of RUNX1 in hematopoiesis, its function in the pre-malignant bone marrow is not well understood. Here, we characterized the hematopoietic progenitor compartments using a mouse strain carrying an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37756531"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential (CHIP) and Incident Type 2 Diabetes Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37756531/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231003205046&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP was associated with higher incidence of T2D. CHIP mutations located on genes implicated in CHD and mortality were also related to T2D, suggesting shared aging-related pathology.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37756546"
                        open
                      >
                        <summary class="article-expander__title">Runx1-R188Q germline mutation induces inflammation and predisposition to hematologic malignancies in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37756546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231002205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline mutations in the RUNX1 gene cause familial platelet disorder (FPD), an inherited disease associated with lifetime risk to hematopoietic malignancies (HM). FPD patients frequently show clonal expansion of pre-malignant cells preceding HM onset. Despite the extensive studies on the role of RUNX1 in hematopoiesis, its function in the pre-malignant bone marrow is not well understood. Here, we characterized the hematopoietic progenitor compartments using a mouse strain carrying an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37756531"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential (CHIP) and Incident Type 2 Diabetes Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37756531/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231002205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP was associated with higher incidence of T2D. CHIP mutations located on genes implicated in CHD and mortality were also related to T2D, suggesting shared aging-related pathology.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37756546"
                        open
                      >
                        <summary class="article-expander__title">Runx1-R188Q germline mutation induces inflammation and predisposition to hematologic malignancies in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37756546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231001205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline mutations in the RUNX1 gene cause familial platelet disorder (FPD), an inherited disease associated with lifetime risk to hematopoietic malignancies (HM). FPD patients frequently show clonal expansion of pre-malignant cells preceding HM onset. Despite the extensive studies on the role of RUNX1 in hematopoiesis, its function in the pre-malignant bone marrow is not well understood. Here, we characterized the hematopoietic progenitor compartments using a mouse strain carrying an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37756531"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential (CHIP) and Incident Type 2 Diabetes Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37756531/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231001205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP was associated with higher incidence of T2D. CHIP mutations located on genes implicated in CHD and mortality were also related to T2D, suggesting shared aging-related pathology.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37756546"
                        open
                      >
                        <summary class="article-expander__title">Runx1-R188Q germline mutation induces inflammation and predisposition to hematologic malignancies in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37756546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230930205630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline mutations in the RUNX1 gene cause familial platelet disorder (FPD), an inherited disease associated with lifetime risk to hematopoietic malignancies (HM). FPD patients frequently show clonal expansion of pre-malignant cells preceding HM onset. Despite the extensive studies on the role of RUNX1 in hematopoiesis, its function in the pre-malignant bone marrow is not well understood. Here, we characterized the hematopoietic progenitor compartments using a mouse strain carrying an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37756531"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential (CHIP) and Incident Type 2 Diabetes Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37756531/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230930205630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP was associated with higher incidence of T2D. CHIP mutations located on genes implicated in CHD and mortality were also related to T2D, suggesting shared aging-related pathology.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37756546"
                        open
                      >
                        <summary class="article-expander__title">Runx1-R188Q germline mutation induces inflammation and predisposition to hematologic malignancies in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37756546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230929204823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline mutations in the RUNX1 gene cause familial platelet disorder (FPD), an inherited disease associated with lifetime risk to hematopoietic malignancies (HM). FPD patients frequently show clonal expansion of pre-malignant cells preceding HM onset. Despite the extensive studies on the role of RUNX1 in hematopoiesis, its function in the pre-malignant bone marrow is not well understood. Here, we characterized the hematopoietic progenitor compartments using a mouse strain carrying an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37756531"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential (CHIP) and Incident Type 2 Diabetes Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37756531/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230929204823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP was associated with higher incidence of T2D. CHIP mutations located on genes implicated in CHD and mortality were also related to T2D, suggesting shared aging-related pathology.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37756546"
                        open
                      >
                        <summary class="article-expander__title">Runx1-R188Q germline mutation induces inflammation and predisposition to hematologic malignancies in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37756546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230928204947&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline mutations in the RUNX1 gene cause familial platelet disorder (FPD), an inherited disease associated with lifetime risk to hematopoietic malignancies (HM). FPD patients frequently show clonal expansion of pre-malignant cells preceding HM onset. Despite the extensive studies on the role of RUNX1 in hematopoiesis, its function in the pre-malignant bone marrow is not well understood. Here, we characterized the hematopoietic progenitor compartments using a mouse strain carrying an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37756531"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential (CHIP) and Incident Type 2 Diabetes Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37756531/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230928204947&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP was associated with higher incidence of T2D. CHIP mutations located on genes implicated in CHD and mortality were also related to T2D, suggesting shared aging-related pathology.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37756546"
                        open
                      >
                        <summary class="article-expander__title">Runx1-R188Q germline mutation induces inflammation and predisposition to hematologic malignancies in mice</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37756546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230927204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Germline mutations in the RUNX1 gene cause familial platelet disorder (FPD), an inherited disease associated with lifetime risk to hematopoietic malignancies (HM). FPD patients frequently show clonal expansion of pre-malignant cells preceding HM onset. Despite the extensive studies on the role of RUNX1 in hematopoiesis, its function in the pre-malignant bone marrow is not well understood. Here, we characterized the hematopoietic progenitor compartments using a mouse strain carrying an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37756531"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential (CHIP) and Incident Type 2 Diabetes Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37756531/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230927204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CHIP was associated with higher incidence of T2D. CHIP mutations located on genes implicated in CHD and mortality were also related to T2D, suggesting shared aging-related pathology.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;ff&#x3D;20231105195156&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37758711"
                        open
                      >
                        <summary class="article-expander__title">Predictive modelling of response to neoadjuvant therapy in HER2+ breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37758711/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231019204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>HER2-positive (HER2+) breast cancer accounts for 20-25% of all breast cancers. Predictive biomarkers of neoadjuvant therapy response are needed to better identify patients with early stage disease who may benefit from tailored treatments in the adjuvant setting. As part of the TCHL phase-II clinical trial (ICORG10-05/NCT01485926) whole exome DNA sequencing was carried out on normal-tumour pairs collected from 22 patients. Here we report predictive modelling of neoadjuvant therapy response using...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37755647"
                        open
                      >
                        <summary class="article-expander__title">Application of Induced Pluripotent Stem Cells in Malignant Solid Tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37755647/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231019204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the past decade, induced pluripotent stem cells (iPSCs) technology has significantly progressed in studying malignant solid tumors. This technically feasible reprogramming techniques can reawaken sequestered dormant regions that regulate the fate of differentiated cells. Despite the evolving therapeutic modalities for malignant solid tumors, treatment outcomes have not been satisfactory. Recently, scientists attempted to apply induced pluripotent stem cell technology to cancer research, from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37758711"
                        open
                      >
                        <summary class="article-expander__title">Predictive modelling of response to neoadjuvant therapy in HER2+ breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37758711/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231018205013&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>HER2-positive (HER2+) breast cancer accounts for 20-25% of all breast cancers. Predictive biomarkers of neoadjuvant therapy response are needed to better identify patients with early stage disease who may benefit from tailored treatments in the adjuvant setting. As part of the TCHL phase-II clinical trial (ICORG10-05/NCT01485926) whole exome DNA sequencing was carried out on normal-tumour pairs collected from 22 patients. Here we report predictive modelling of neoadjuvant therapy response using...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37755647"
                        open
                      >
                        <summary class="article-expander__title">Application of Induced Pluripotent Stem Cells in Malignant Solid Tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37755647/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231018205013&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the past decade, induced pluripotent stem cells (iPSCs) technology has significantly progressed in studying malignant solid tumors. This technically feasible reprogramming techniques can reawaken sequestered dormant regions that regulate the fate of differentiated cells. Despite the evolving therapeutic modalities for malignant solid tumors, treatment outcomes have not been satisfactory. Recently, scientists attempted to apply induced pluripotent stem cell technology to cancer research, from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37758711"
                        open
                      >
                        <summary class="article-expander__title">Predictive modelling of response to neoadjuvant therapy in HER2+ breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37758711/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231017205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>HER2-positive (HER2+) breast cancer accounts for 20-25% of all breast cancers. Predictive biomarkers of neoadjuvant therapy response are needed to better identify patients with early stage disease who may benefit from tailored treatments in the adjuvant setting. As part of the TCHL phase-II clinical trial (ICORG10-05/NCT01485926) whole exome DNA sequencing was carried out on normal-tumour pairs collected from 22 patients. Here we report predictive modelling of neoadjuvant therapy response using...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37755647"
                        open
                      >
                        <summary class="article-expander__title">Application of Induced Pluripotent Stem Cells in Malignant Solid Tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37755647/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231017205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the past decade, induced pluripotent stem cells (iPSCs) technology has significantly progressed in studying malignant solid tumors. This technically feasible reprogramming techniques can reawaken sequestered dormant regions that regulate the fate of differentiated cells. Despite the evolving therapeutic modalities for malignant solid tumors, treatment outcomes have not been satisfactory. Recently, scientists attempted to apply induced pluripotent stem cell technology to cancer research, from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37758711"
                        open
                      >
                        <summary class="article-expander__title">Predictive modelling of response to neoadjuvant therapy in HER2+ breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37758711/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231016205028&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>HER2-positive (HER2+) breast cancer accounts for 20-25% of all breast cancers. Predictive biomarkers of neoadjuvant therapy response are needed to better identify patients with early stage disease who may benefit from tailored treatments in the adjuvant setting. As part of the TCHL phase-II clinical trial (ICORG10-05/NCT01485926) whole exome DNA sequencing was carried out on normal-tumour pairs collected from 22 patients. Here we report predictive modelling of neoadjuvant therapy response using...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37755647"
                        open
                      >
                        <summary class="article-expander__title">Application of Induced Pluripotent Stem Cells in Malignant Solid Tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37755647/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231016205028&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the past decade, induced pluripotent stem cells (iPSCs) technology has significantly progressed in studying malignant solid tumors. This technically feasible reprogramming techniques can reawaken sequestered dormant regions that regulate the fate of differentiated cells. Despite the evolving therapeutic modalities for malignant solid tumors, treatment outcomes have not been satisfactory. Recently, scientists attempted to apply induced pluripotent stem cell technology to cancer research, from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37758711"
                        open
                      >
                        <summary class="article-expander__title">Predictive modelling of response to neoadjuvant therapy in HER2+ breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37758711/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231015205105&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>HER2-positive (HER2+) breast cancer accounts for 20-25% of all breast cancers. Predictive biomarkers of neoadjuvant therapy response are needed to better identify patients with early stage disease who may benefit from tailored treatments in the adjuvant setting. As part of the TCHL phase-II clinical trial (ICORG10-05/NCT01485926) whole exome DNA sequencing was carried out on normal-tumour pairs collected from 22 patients. Here we report predictive modelling of neoadjuvant therapy response using...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37755647"
                        open
                      >
                        <summary class="article-expander__title">Application of Induced Pluripotent Stem Cells in Malignant Solid Tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37755647/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231015205105&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the past decade, induced pluripotent stem cells (iPSCs) technology has significantly progressed in studying malignant solid tumors. This technically feasible reprogramming techniques can reawaken sequestered dormant regions that regulate the fate of differentiated cells. Despite the evolving therapeutic modalities for malignant solid tumors, treatment outcomes have not been satisfactory. Recently, scientists attempted to apply induced pluripotent stem cell technology to cancer research, from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37758711"
                        open
                      >
                        <summary class="article-expander__title">Predictive modelling of response to neoadjuvant therapy in HER2+ breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37758711/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231014205339&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>HER2-positive (HER2+) breast cancer accounts for 20-25% of all breast cancers. Predictive biomarkers of neoadjuvant therapy response are needed to better identify patients with early stage disease who may benefit from tailored treatments in the adjuvant setting. As part of the TCHL phase-II clinical trial (ICORG10-05/NCT01485926) whole exome DNA sequencing was carried out on normal-tumour pairs collected from 22 patients. Here we report predictive modelling of neoadjuvant therapy response using...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37755647"
                        open
                      >
                        <summary class="article-expander__title">Application of Induced Pluripotent Stem Cells in Malignant Solid Tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37755647/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231014205339&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the past decade, induced pluripotent stem cells (iPSCs) technology has significantly progressed in studying malignant solid tumors. This technically feasible reprogramming techniques can reawaken sequestered dormant regions that regulate the fate of differentiated cells. Despite the evolving therapeutic modalities for malignant solid tumors, treatment outcomes have not been satisfactory. Recently, scientists attempted to apply induced pluripotent stem cell technology to cancer research, from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37758711"
                        open
                      >
                        <summary class="article-expander__title">Predictive modelling of response to neoadjuvant therapy in HER2+ breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37758711/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231013204816&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>HER2-positive (HER2+) breast cancer accounts for 20-25% of all breast cancers. Predictive biomarkers of neoadjuvant therapy response are needed to better identify patients with early stage disease who may benefit from tailored treatments in the adjuvant setting. As part of the TCHL phase-II clinical trial (ICORG10-05/NCT01485926) whole exome DNA sequencing was carried out on normal-tumour pairs collected from 22 patients. Here we report predictive modelling of neoadjuvant therapy response using...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37755647"
                        open
                      >
                        <summary class="article-expander__title">Application of Induced Pluripotent Stem Cells in Malignant Solid Tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37755647/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231013204816&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the past decade, induced pluripotent stem cells (iPSCs) technology has significantly progressed in studying malignant solid tumors. This technically feasible reprogramming techniques can reawaken sequestered dormant regions that regulate the fate of differentiated cells. Despite the evolving therapeutic modalities for malignant solid tumors, treatment outcomes have not been satisfactory. Recently, scientists attempted to apply induced pluripotent stem cell technology to cancer research, from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37758711"
                        open
                      >
                        <summary class="article-expander__title">Predictive modelling of response to neoadjuvant therapy in HER2+ breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37758711/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231012205121&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>HER2-positive (HER2+) breast cancer accounts for 20-25% of all breast cancers. Predictive biomarkers of neoadjuvant therapy response are needed to better identify patients with early stage disease who may benefit from tailored treatments in the adjuvant setting. As part of the TCHL phase-II clinical trial (ICORG10-05/NCT01485926) whole exome DNA sequencing was carried out on normal-tumour pairs collected from 22 patients. Here we report predictive modelling of neoadjuvant therapy response using...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37755647"
                        open
                      >
                        <summary class="article-expander__title">Application of Induced Pluripotent Stem Cells in Malignant Solid Tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37755647/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231012205121&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the past decade, induced pluripotent stem cells (iPSCs) technology has significantly progressed in studying malignant solid tumors. This technically feasible reprogramming techniques can reawaken sequestered dormant regions that regulate the fate of differentiated cells. Despite the evolving therapeutic modalities for malignant solid tumors, treatment outcomes have not been satisfactory. Recently, scientists attempted to apply induced pluripotent stem cell technology to cancer research, from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37758711"
                        open
                      >
                        <summary class="article-expander__title">Predictive modelling of response to neoadjuvant therapy in HER2+ breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37758711/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231011204744&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>HER2-positive (HER2+) breast cancer accounts for 20-25% of all breast cancers. Predictive biomarkers of neoadjuvant therapy response are needed to better identify patients with early stage disease who may benefit from tailored treatments in the adjuvant setting. As part of the TCHL phase-II clinical trial (ICORG10-05/NCT01485926) whole exome DNA sequencing was carried out on normal-tumour pairs collected from 22 patients. Here we report predictive modelling of neoadjuvant therapy response using...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37755647"
                        open
                      >
                        <summary class="article-expander__title">Application of Induced Pluripotent Stem Cells in Malignant Solid Tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37755647/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231011204744&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the past decade, induced pluripotent stem cells (iPSCs) technology has significantly progressed in studying malignant solid tumors. This technically feasible reprogramming techniques can reawaken sequestered dormant regions that regulate the fate of differentiated cells. Despite the evolving therapeutic modalities for malignant solid tumors, treatment outcomes have not been satisfactory. Recently, scientists attempted to apply induced pluripotent stem cell technology to cancer research, from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37758711"
                        open
                      >
                        <summary class="article-expander__title">Predictive modelling of response to neoadjuvant therapy in HER2+ breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37758711/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231010204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>HER2-positive (HER2+) breast cancer accounts for 20-25% of all breast cancers. Predictive biomarkers of neoadjuvant therapy response are needed to better identify patients with early stage disease who may benefit from tailored treatments in the adjuvant setting. As part of the TCHL phase-II clinical trial (ICORG10-05/NCT01485926) whole exome DNA sequencing was carried out on normal-tumour pairs collected from 22 patients. Here we report predictive modelling of neoadjuvant therapy response using...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37755647"
                        open
                      >
                        <summary class="article-expander__title">Application of Induced Pluripotent Stem Cells in Malignant Solid Tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37755647/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231010204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the past decade, induced pluripotent stem cells (iPSCs) technology has significantly progressed in studying malignant solid tumors. This technically feasible reprogramming techniques can reawaken sequestered dormant regions that regulate the fate of differentiated cells. Despite the evolving therapeutic modalities for malignant solid tumors, treatment outcomes have not been satisfactory. Recently, scientists attempted to apply induced pluripotent stem cell technology to cancer research, from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37758711"
                        open
                      >
                        <summary class="article-expander__title">Predictive modelling of response to neoadjuvant therapy in HER2+ breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37758711/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231009204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>HER2-positive (HER2+) breast cancer accounts for 20-25% of all breast cancers. Predictive biomarkers of neoadjuvant therapy response are needed to better identify patients with early stage disease who may benefit from tailored treatments in the adjuvant setting. As part of the TCHL phase-II clinical trial (ICORG10-05/NCT01485926) whole exome DNA sequencing was carried out on normal-tumour pairs collected from 22 patients. Here we report predictive modelling of neoadjuvant therapy response using...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37755647"
                        open
                      >
                        <summary class="article-expander__title">Application of Induced Pluripotent Stem Cells in Malignant Solid Tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37755647/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231009204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the past decade, induced pluripotent stem cells (iPSCs) technology has significantly progressed in studying malignant solid tumors. This technically feasible reprogramming techniques can reawaken sequestered dormant regions that regulate the fate of differentiated cells. Despite the evolving therapeutic modalities for malignant solid tumors, treatment outcomes have not been satisfactory. Recently, scientists attempted to apply induced pluripotent stem cell technology to cancer research, from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37758711"
                        open
                      >
                        <summary class="article-expander__title">Predictive modelling of response to neoadjuvant therapy in HER2+ breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37758711/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231008205021&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>HER2-positive (HER2+) breast cancer accounts for 20-25% of all breast cancers. Predictive biomarkers of neoadjuvant therapy response are needed to better identify patients with early stage disease who may benefit from tailored treatments in the adjuvant setting. As part of the TCHL phase-II clinical trial (ICORG10-05/NCT01485926) whole exome DNA sequencing was carried out on normal-tumour pairs collected from 22 patients. Here we report predictive modelling of neoadjuvant therapy response using...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37755647"
                        open
                      >
                        <summary class="article-expander__title">Application of Induced Pluripotent Stem Cells in Malignant Solid Tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37755647/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231008205021&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the past decade, induced pluripotent stem cells (iPSCs) technology has significantly progressed in studying malignant solid tumors. This technically feasible reprogramming techniques can reawaken sequestered dormant regions that regulate the fate of differentiated cells. Despite the evolving therapeutic modalities for malignant solid tumors, treatment outcomes have not been satisfactory. Recently, scientists attempted to apply induced pluripotent stem cell technology to cancer research, from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37758711"
                        open
                      >
                        <summary class="article-expander__title">Predictive modelling of response to neoadjuvant therapy in HER2+ breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37758711/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231007205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>HER2-positive (HER2+) breast cancer accounts for 20-25% of all breast cancers. Predictive biomarkers of neoadjuvant therapy response are needed to better identify patients with early stage disease who may benefit from tailored treatments in the adjuvant setting. As part of the TCHL phase-II clinical trial (ICORG10-05/NCT01485926) whole exome DNA sequencing was carried out on normal-tumour pairs collected from 22 patients. Here we report predictive modelling of neoadjuvant therapy response using...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37755647"
                        open
                      >
                        <summary class="article-expander__title">Application of Induced Pluripotent Stem Cells in Malignant Solid Tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37755647/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231007205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the past decade, induced pluripotent stem cells (iPSCs) technology has significantly progressed in studying malignant solid tumors. This technically feasible reprogramming techniques can reawaken sequestered dormant regions that regulate the fate of differentiated cells. Despite the evolving therapeutic modalities for malignant solid tumors, treatment outcomes have not been satisfactory. Recently, scientists attempted to apply induced pluripotent stem cell technology to cancer research, from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37758711"
                        open
                      >
                        <summary class="article-expander__title">Predictive modelling of response to neoadjuvant therapy in HER2+ breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37758711/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231006204916&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>HER2-positive (HER2+) breast cancer accounts for 20-25% of all breast cancers. Predictive biomarkers of neoadjuvant therapy response are needed to better identify patients with early stage disease who may benefit from tailored treatments in the adjuvant setting. As part of the TCHL phase-II clinical trial (ICORG10-05/NCT01485926) whole exome DNA sequencing was carried out on normal-tumour pairs collected from 22 patients. Here we report predictive modelling of neoadjuvant therapy response using...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37755647"
                        open
                      >
                        <summary class="article-expander__title">Application of Induced Pluripotent Stem Cells in Malignant Solid Tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37755647/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231006204916&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the past decade, induced pluripotent stem cells (iPSCs) technology has significantly progressed in studying malignant solid tumors. This technically feasible reprogramming techniques can reawaken sequestered dormant regions that regulate the fate of differentiated cells. Despite the evolving therapeutic modalities for malignant solid tumors, treatment outcomes have not been satisfactory. Recently, scientists attempted to apply induced pluripotent stem cell technology to cancer research, from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37758711"
                        open
                      >
                        <summary class="article-expander__title">Predictive modelling of response to neoadjuvant therapy in HER2+ breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37758711/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231005204957&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>HER2-positive (HER2+) breast cancer accounts for 20-25% of all breast cancers. Predictive biomarkers of neoadjuvant therapy response are needed to better identify patients with early stage disease who may benefit from tailored treatments in the adjuvant setting. As part of the TCHL phase-II clinical trial (ICORG10-05/NCT01485926) whole exome DNA sequencing was carried out on normal-tumour pairs collected from 22 patients. Here we report predictive modelling of neoadjuvant therapy response using...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37755647"
                        open
                      >
                        <summary class="article-expander__title">Application of Induced Pluripotent Stem Cells in Malignant Solid Tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37755647/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231005204957&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the past decade, induced pluripotent stem cells (iPSCs) technology has significantly progressed in studying malignant solid tumors. This technically feasible reprogramming techniques can reawaken sequestered dormant regions that regulate the fate of differentiated cells. Despite the evolving therapeutic modalities for malignant solid tumors, treatment outcomes have not been satisfactory. Recently, scientists attempted to apply induced pluripotent stem cell technology to cancer research, from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37758711"
                        open
                      >
                        <summary class="article-expander__title">Predictive modelling of response to neoadjuvant therapy in HER2+ breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37758711/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231004205006&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>HER2-positive (HER2+) breast cancer accounts for 20-25% of all breast cancers. Predictive biomarkers of neoadjuvant therapy response are needed to better identify patients with early stage disease who may benefit from tailored treatments in the adjuvant setting. As part of the TCHL phase-II clinical trial (ICORG10-05/NCT01485926) whole exome DNA sequencing was carried out on normal-tumour pairs collected from 22 patients. Here we report predictive modelling of neoadjuvant therapy response using...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37755647"
                        open
                      >
                        <summary class="article-expander__title">Application of Induced Pluripotent Stem Cells in Malignant Solid Tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37755647/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231004205006&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the past decade, induced pluripotent stem cells (iPSCs) technology has significantly progressed in studying malignant solid tumors. This technically feasible reprogramming techniques can reawaken sequestered dormant regions that regulate the fate of differentiated cells. Despite the evolving therapeutic modalities for malignant solid tumors, treatment outcomes have not been satisfactory. Recently, scientists attempted to apply induced pluripotent stem cell technology to cancer research, from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37758711"
                        open
                      >
                        <summary class="article-expander__title">Predictive modelling of response to neoadjuvant therapy in HER2+ breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37758711/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231003205044&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>HER2-positive (HER2+) breast cancer accounts for 20-25% of all breast cancers. Predictive biomarkers of neoadjuvant therapy response are needed to better identify patients with early stage disease who may benefit from tailored treatments in the adjuvant setting. As part of the TCHL phase-II clinical trial (ICORG10-05/NCT01485926) whole exome DNA sequencing was carried out on normal-tumour pairs collected from 22 patients. Here we report predictive modelling of neoadjuvant therapy response using...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37755647"
                        open
                      >
                        <summary class="article-expander__title">Application of Induced Pluripotent Stem Cells in Malignant Solid Tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37755647/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231003205044&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the past decade, induced pluripotent stem cells (iPSCs) technology has significantly progressed in studying malignant solid tumors. This technically feasible reprogramming techniques can reawaken sequestered dormant regions that regulate the fate of differentiated cells. Despite the evolving therapeutic modalities for malignant solid tumors, treatment outcomes have not been satisfactory. Recently, scientists attempted to apply induced pluripotent stem cell technology to cancer research, from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37758711"
                        open
                      >
                        <summary class="article-expander__title">Predictive modelling of response to neoadjuvant therapy in HER2+ breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37758711/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231002205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>HER2-positive (HER2+) breast cancer accounts for 20-25% of all breast cancers. Predictive biomarkers of neoadjuvant therapy response are needed to better identify patients with early stage disease who may benefit from tailored treatments in the adjuvant setting. As part of the TCHL phase-II clinical trial (ICORG10-05/NCT01485926) whole exome DNA sequencing was carried out on normal-tumour pairs collected from 22 patients. Here we report predictive modelling of neoadjuvant therapy response using...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37755647"
                        open
                      >
                        <summary class="article-expander__title">Application of Induced Pluripotent Stem Cells in Malignant Solid Tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37755647/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231002205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the past decade, induced pluripotent stem cells (iPSCs) technology has significantly progressed in studying malignant solid tumors. This technically feasible reprogramming techniques can reawaken sequestered dormant regions that regulate the fate of differentiated cells. Despite the evolving therapeutic modalities for malignant solid tumors, treatment outcomes have not been satisfactory. Recently, scientists attempted to apply induced pluripotent stem cell technology to cancer research, from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37758711"
                        open
                      >
                        <summary class="article-expander__title">Predictive modelling of response to neoadjuvant therapy in HER2+ breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37758711/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231001205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>HER2-positive (HER2+) breast cancer accounts for 20-25% of all breast cancers. Predictive biomarkers of neoadjuvant therapy response are needed to better identify patients with early stage disease who may benefit from tailored treatments in the adjuvant setting. As part of the TCHL phase-II clinical trial (ICORG10-05/NCT01485926) whole exome DNA sequencing was carried out on normal-tumour pairs collected from 22 patients. Here we report predictive modelling of neoadjuvant therapy response using...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37755647"
                        open
                      >
                        <summary class="article-expander__title">Application of Induced Pluripotent Stem Cells in Malignant Solid Tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37755647/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231001205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the past decade, induced pluripotent stem cells (iPSCs) technology has significantly progressed in studying malignant solid tumors. This technically feasible reprogramming techniques can reawaken sequestered dormant regions that regulate the fate of differentiated cells. Despite the evolving therapeutic modalities for malignant solid tumors, treatment outcomes have not been satisfactory. Recently, scientists attempted to apply induced pluripotent stem cell technology to cancer research, from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37758711"
                        open
                      >
                        <summary class="article-expander__title">Predictive modelling of response to neoadjuvant therapy in HER2+ breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37758711/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230930205630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>HER2-positive (HER2+) breast cancer accounts for 20-25% of all breast cancers. Predictive biomarkers of neoadjuvant therapy response are needed to better identify patients with early stage disease who may benefit from tailored treatments in the adjuvant setting. As part of the TCHL phase-II clinical trial (ICORG10-05/NCT01485926) whole exome DNA sequencing was carried out on normal-tumour pairs collected from 22 patients. Here we report predictive modelling of neoadjuvant therapy response using...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37755647"
                        open
                      >
                        <summary class="article-expander__title">Application of Induced Pluripotent Stem Cells in Malignant Solid Tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37755647/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230930205630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the past decade, induced pluripotent stem cells (iPSCs) technology has significantly progressed in studying malignant solid tumors. This technically feasible reprogramming techniques can reawaken sequestered dormant regions that regulate the fate of differentiated cells. Despite the evolving therapeutic modalities for malignant solid tumors, treatment outcomes have not been satisfactory. Recently, scientists attempted to apply induced pluripotent stem cell technology to cancer research, from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37758711"
                        open
                      >
                        <summary class="article-expander__title">Predictive modelling of response to neoadjuvant therapy in HER2+ breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37758711/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230929204823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>HER2-positive (HER2+) breast cancer accounts for 20-25% of all breast cancers. Predictive biomarkers of neoadjuvant therapy response are needed to better identify patients with early stage disease who may benefit from tailored treatments in the adjuvant setting. As part of the TCHL phase-II clinical trial (ICORG10-05/NCT01485926) whole exome DNA sequencing was carried out on normal-tumour pairs collected from 22 patients. Here we report predictive modelling of neoadjuvant therapy response using...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37755647"
                        open
                      >
                        <summary class="article-expander__title">Application of Induced Pluripotent Stem Cells in Malignant Solid Tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37755647/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230929204823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the past decade, induced pluripotent stem cells (iPSCs) technology has significantly progressed in studying malignant solid tumors. This technically feasible reprogramming techniques can reawaken sequestered dormant regions that regulate the fate of differentiated cells. Despite the evolving therapeutic modalities for malignant solid tumors, treatment outcomes have not been satisfactory. Recently, scientists attempted to apply induced pluripotent stem cell technology to cancer research, from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37758711"
                        open
                      >
                        <summary class="article-expander__title">Predictive modelling of response to neoadjuvant therapy in HER2+ breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37758711/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230928204948&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>HER2-positive (HER2+) breast cancer accounts for 20-25% of all breast cancers. Predictive biomarkers of neoadjuvant therapy response are needed to better identify patients with early stage disease who may benefit from tailored treatments in the adjuvant setting. As part of the TCHL phase-II clinical trial (ICORG10-05/NCT01485926) whole exome DNA sequencing was carried out on normal-tumour pairs collected from 22 patients. Here we report predictive modelling of neoadjuvant therapy response using...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37755647"
                        open
                      >
                        <summary class="article-expander__title">Application of Induced Pluripotent Stem Cells in Malignant Solid Tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37755647/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230928204948&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the past decade, induced pluripotent stem cells (iPSCs) technology has significantly progressed in studying malignant solid tumors. This technically feasible reprogramming techniques can reawaken sequestered dormant regions that regulate the fate of differentiated cells. Despite the evolving therapeutic modalities for malignant solid tumors, treatment outcomes have not been satisfactory. Recently, scientists attempted to apply induced pluripotent stem cell technology to cancer research, from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37755647"
                        open
                      >
                        <summary class="article-expander__title">Application of Induced Pluripotent Stem Cells in Malignant Solid Tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37755647/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230927204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the past decade, induced pluripotent stem cells (iPSCs) technology has significantly progressed in studying malignant solid tumors. This technically feasible reprogramming techniques can reawaken sequestered dormant regions that regulate the fate of differentiated cells. Despite the evolving therapeutic modalities for malignant solid tumors, treatment outcomes have not been satisfactory. Recently, scientists attempted to apply induced pluripotent stem cell technology to cancer research, from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-09-25 datetime="2023-09-25T10:00:00.000Z">2023-09-25</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-09-25 datetime="2023-09-25T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20231105195156">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37749325"
                        open
                      >
                        <summary class="article-expander__title">Newborn and child-like molecular signatures in older adults stem from TCR shifts across human lifespan</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37749325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231010204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CD8^(+) T cells provide robust antiviral immunity, but how epitope-specific T cells evolve across the human lifespan is unclear. Here we defined CD8^(+) T cell immunity directed at the prominent influenza epitope HLA-A*02:01-M1(58-66) (A2/M1(58)) across four age groups at phenotypic, transcriptomic, clonal and functional levels. We identify a linear differentiation trajectory from newborns to children then adults, followed by divergence and a clonal reset in older adults. Gene profiles in older...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37749325"
                        open
                      >
                        <summary class="article-expander__title">Newborn and child-like molecular signatures in older adults stem from TCR shifts across human lifespan</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37749325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231009204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CD8^(+) T cells provide robust antiviral immunity, but how epitope-specific T cells evolve across the human lifespan is unclear. Here we defined CD8^(+) T cell immunity directed at the prominent influenza epitope HLA-A*02:01-M1(58-66) (A2/M1(58)) across four age groups at phenotypic, transcriptomic, clonal and functional levels. We identify a linear differentiation trajectory from newborns to children then adults, followed by divergence and a clonal reset in older adults. Gene profiles in older...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37749325"
                        open
                      >
                        <summary class="article-expander__title">Newborn and child-like molecular signatures in older adults stem from TCR shifts across human lifespan</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37749325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231008205021&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CD8^(+) T cells provide robust antiviral immunity, but how epitope-specific T cells evolve across the human lifespan is unclear. Here we defined CD8^(+) T cell immunity directed at the prominent influenza epitope HLA-A*02:01-M1(58-66) (A2/M1(58)) across four age groups at phenotypic, transcriptomic, clonal and functional levels. We identify a linear differentiation trajectory from newborns to children then adults, followed by divergence and a clonal reset in older adults. Gene profiles in older...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37749269"
                        open
                      >
                        <summary class="article-expander__title">Single-cell lineage capture across genomic modalities with CellTag-multi reveals fate-specific gene regulatory changes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37749269/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231008205021&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Complex gene regulatory mechanisms underlie differentiation and reprogramming. Contemporary single-cell lineage-tracing (scLT) methods use expressed, heritable DNA barcodes to combine cell lineage readout with single-cell transcriptomics. However, reliance on transcriptional profiling limits adaptation to other single-cell assays. With CellTag-multi, we present an approach that enables direct capture of heritable random barcodes expressed as polyadenylated transcripts, in both single-cell RNA...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37749325"
                        open
                      >
                        <summary class="article-expander__title">Newborn and child-like molecular signatures in older adults stem from TCR shifts across human lifespan</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37749325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231007205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CD8^(+) T cells provide robust antiviral immunity, but how epitope-specific T cells evolve across the human lifespan is unclear. Here we defined CD8^(+) T cell immunity directed at the prominent influenza epitope HLA-A*02:01-M1(58-66) (A2/M1(58)) across four age groups at phenotypic, transcriptomic, clonal and functional levels. We identify a linear differentiation trajectory from newborns to children then adults, followed by divergence and a clonal reset in older adults. Gene profiles in older...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37749269"
                        open
                      >
                        <summary class="article-expander__title">Single-cell lineage capture across genomic modalities with CellTag-multi reveals fate-specific gene regulatory changes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37749269/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231007205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Complex gene regulatory mechanisms underlie differentiation and reprogramming. Contemporary single-cell lineage-tracing (scLT) methods use expressed, heritable DNA barcodes to combine cell lineage readout with single-cell transcriptomics. However, reliance on transcriptional profiling limits adaptation to other single-cell assays. With CellTag-multi, we present an approach that enables direct capture of heritable random barcodes expressed as polyadenylated transcripts, in both single-cell RNA...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37749325"
                        open
                      >
                        <summary class="article-expander__title">Newborn and child-like molecular signatures in older adults stem from TCR shifts across human lifespan</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37749325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231006204916&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CD8^(+) T cells provide robust antiviral immunity, but how epitope-specific T cells evolve across the human lifespan is unclear. Here we defined CD8^(+) T cell immunity directed at the prominent influenza epitope HLA-A*02:01-M1(58-66) (A2/M1(58)) across four age groups at phenotypic, transcriptomic, clonal and functional levels. We identify a linear differentiation trajectory from newborns to children then adults, followed by divergence and a clonal reset in older adults. Gene profiles in older...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37749269"
                        open
                      >
                        <summary class="article-expander__title">Single-cell lineage capture across genomic modalities with CellTag-multi reveals fate-specific gene regulatory changes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37749269/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231006204916&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Complex gene regulatory mechanisms underlie differentiation and reprogramming. Contemporary single-cell lineage-tracing (scLT) methods use expressed, heritable DNA barcodes to combine cell lineage readout with single-cell transcriptomics. However, reliance on transcriptional profiling limits adaptation to other single-cell assays. With CellTag-multi, we present an approach that enables direct capture of heritable random barcodes expressed as polyadenylated transcripts, in both single-cell RNA...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37748049"
                        open
                      >
                        <summary class="article-expander__title">Clearing and replacing tissue-resident myeloid cells with an anti-CD45-antibody-drug conjugate</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37748049/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231006204916&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue resident myeloid cells (TRM) in adults have highly variable lifespans and may be derived from early embryonic yolk sac, fetal liver or bone marrow. Some of these TRM are known pathogenic participants in congenital and acquired diseases. Myeloablative conditioning and hematopoietic stem cell transplant can replace long-lived brain TRM resulting in clinical improvements in metabolic storage diseases. With the advent of antibody-drug-conjugate (ADC) targeted cell killing as a cell selective...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37749325"
                        open
                      >
                        <summary class="article-expander__title">Newborn and child-like molecular signatures in older adults stem from TCR shifts across human lifespan</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37749325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231005204957&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CD8^(+) T cells provide robust antiviral immunity, but how epitope-specific T cells evolve across the human lifespan is unclear. Here we defined CD8^(+) T cell immunity directed at the prominent influenza epitope HLA-A*02:01-M1(58-66) (A2/M1(58)) across four age groups at phenotypic, transcriptomic, clonal and functional levels. We identify a linear differentiation trajectory from newborns to children then adults, followed by divergence and a clonal reset in older adults. Gene profiles in older...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37749269"
                        open
                      >
                        <summary class="article-expander__title">Single-cell lineage capture across genomic modalities with CellTag-multi reveals fate-specific gene regulatory changes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37749269/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231005204957&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Complex gene regulatory mechanisms underlie differentiation and reprogramming. Contemporary single-cell lineage-tracing (scLT) methods use expressed, heritable DNA barcodes to combine cell lineage readout with single-cell transcriptomics. However, reliance on transcriptional profiling limits adaptation to other single-cell assays. With CellTag-multi, we present an approach that enables direct capture of heritable random barcodes expressed as polyadenylated transcripts, in both single-cell RNA...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37748049"
                        open
                      >
                        <summary class="article-expander__title">Clearing and replacing tissue-resident myeloid cells with an anti-CD45-antibody-drug conjugate</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37748049/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231005204957&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue resident myeloid cells (TRM) in adults have highly variable lifespans and may be derived from early embryonic yolk sac, fetal liver or bone marrow. Some of these TRM are known pathogenic participants in congenital and acquired diseases. Myeloablative conditioning and hematopoietic stem cell transplant can replace long-lived brain TRM resulting in clinical improvements in metabolic storage diseases. With the advent of antibody-drug-conjugate (ADC) targeted cell killing as a cell selective...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37749325"
                        open
                      >
                        <summary class="article-expander__title">Newborn and child-like molecular signatures in older adults stem from TCR shifts across human lifespan</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37749325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231004205006&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CD8^(+) T cells provide robust antiviral immunity, but how epitope-specific T cells evolve across the human lifespan is unclear. Here we defined CD8^(+) T cell immunity directed at the prominent influenza epitope HLA-A*02:01-M1(58-66) (A2/M1(58)) across four age groups at phenotypic, transcriptomic, clonal and functional levels. We identify a linear differentiation trajectory from newborns to children then adults, followed by divergence and a clonal reset in older adults. Gene profiles in older...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37749269"
                        open
                      >
                        <summary class="article-expander__title">Single-cell lineage capture across genomic modalities with CellTag-multi reveals fate-specific gene regulatory changes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37749269/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231004205006&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Complex gene regulatory mechanisms underlie differentiation and reprogramming. Contemporary single-cell lineage-tracing (scLT) methods use expressed, heritable DNA barcodes to combine cell lineage readout with single-cell transcriptomics. However, reliance on transcriptional profiling limits adaptation to other single-cell assays. With CellTag-multi, we present an approach that enables direct capture of heritable random barcodes expressed as polyadenylated transcripts, in both single-cell RNA...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37748049"
                        open
                      >
                        <summary class="article-expander__title">Clearing and replacing tissue-resident myeloid cells with an anti-CD45-antibody-drug conjugate</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37748049/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231004205006&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue resident myeloid cells (TRM) in adults have highly variable lifespans and may be derived from early embryonic yolk sac, fetal liver or bone marrow. Some of these TRM are known pathogenic participants in congenital and acquired diseases. Myeloablative conditioning and hematopoietic stem cell transplant can replace long-lived brain TRM resulting in clinical improvements in metabolic storage diseases. With the advent of antibody-drug-conjugate (ADC) targeted cell killing as a cell selective...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37749325"
                        open
                      >
                        <summary class="article-expander__title">Newborn and child-like molecular signatures in older adults stem from TCR shifts across human lifespan</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37749325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231003205046&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CD8^(+) T cells provide robust antiviral immunity, but how epitope-specific T cells evolve across the human lifespan is unclear. Here we defined CD8^(+) T cell immunity directed at the prominent influenza epitope HLA-A*02:01-M1(58-66) (A2/M1(58)) across four age groups at phenotypic, transcriptomic, clonal and functional levels. We identify a linear differentiation trajectory from newborns to children then adults, followed by divergence and a clonal reset in older adults. Gene profiles in older...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37749269"
                        open
                      >
                        <summary class="article-expander__title">Single-cell lineage capture across genomic modalities with CellTag-multi reveals fate-specific gene regulatory changes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37749269/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231003205046&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Complex gene regulatory mechanisms underlie differentiation and reprogramming. Contemporary single-cell lineage-tracing (scLT) methods use expressed, heritable DNA barcodes to combine cell lineage readout with single-cell transcriptomics. However, reliance on transcriptional profiling limits adaptation to other single-cell assays. With CellTag-multi, we present an approach that enables direct capture of heritable random barcodes expressed as polyadenylated transcripts, in both single-cell RNA...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37748049"
                        open
                      >
                        <summary class="article-expander__title">Clearing and replacing tissue-resident myeloid cells with an anti-CD45-antibody-drug conjugate</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37748049/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231003205046&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue resident myeloid cells (TRM) in adults have highly variable lifespans and may be derived from early embryonic yolk sac, fetal liver or bone marrow. Some of these TRM are known pathogenic participants in congenital and acquired diseases. Myeloablative conditioning and hematopoietic stem cell transplant can replace long-lived brain TRM resulting in clinical improvements in metabolic storage diseases. With the advent of antibody-drug-conjugate (ADC) targeted cell killing as a cell selective...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37749325"
                        open
                      >
                        <summary class="article-expander__title">Newborn and child-like molecular signatures in older adults stem from TCR shifts across human lifespan</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37749325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231002205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CD8^(+) T cells provide robust antiviral immunity, but how epitope-specific T cells evolve across the human lifespan is unclear. Here we defined CD8^(+) T cell immunity directed at the prominent influenza epitope HLA-A*02:01-M1(58-66) (A2/M1(58)) across four age groups at phenotypic, transcriptomic, clonal and functional levels. We identify a linear differentiation trajectory from newborns to children then adults, followed by divergence and a clonal reset in older adults. Gene profiles in older...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37749269"
                        open
                      >
                        <summary class="article-expander__title">Single-cell lineage capture across genomic modalities with CellTag-multi reveals fate-specific gene regulatory changes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37749269/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231002205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Complex gene regulatory mechanisms underlie differentiation and reprogramming. Contemporary single-cell lineage-tracing (scLT) methods use expressed, heritable DNA barcodes to combine cell lineage readout with single-cell transcriptomics. However, reliance on transcriptional profiling limits adaptation to other single-cell assays. With CellTag-multi, we present an approach that enables direct capture of heritable random barcodes expressed as polyadenylated transcripts, in both single-cell RNA...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37748049"
                        open
                      >
                        <summary class="article-expander__title">Clearing and replacing tissue-resident myeloid cells with an anti-CD45-antibody-drug conjugate</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37748049/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231002205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue resident myeloid cells (TRM) in adults have highly variable lifespans and may be derived from early embryonic yolk sac, fetal liver or bone marrow. Some of these TRM are known pathogenic participants in congenital and acquired diseases. Myeloablative conditioning and hematopoietic stem cell transplant can replace long-lived brain TRM resulting in clinical improvements in metabolic storage diseases. With the advent of antibody-drug-conjugate (ADC) targeted cell killing as a cell selective...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37749325"
                        open
                      >
                        <summary class="article-expander__title">Newborn and child-like molecular signatures in older adults stem from TCR shifts across human lifespan</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37749325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231001205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CD8^(+) T cells provide robust antiviral immunity, but how epitope-specific T cells evolve across the human lifespan is unclear. Here we defined CD8^(+) T cell immunity directed at the prominent influenza epitope HLA-A*02:01-M1(58-66) (A2/M1(58)) across four age groups at phenotypic, transcriptomic, clonal and functional levels. We identify a linear differentiation trajectory from newborns to children then adults, followed by divergence and a clonal reset in older adults. Gene profiles in older...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37749269"
                        open
                      >
                        <summary class="article-expander__title">Single-cell lineage capture across genomic modalities with CellTag-multi reveals fate-specific gene regulatory changes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37749269/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231001205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Complex gene regulatory mechanisms underlie differentiation and reprogramming. Contemporary single-cell lineage-tracing (scLT) methods use expressed, heritable DNA barcodes to combine cell lineage readout with single-cell transcriptomics. However, reliance on transcriptional profiling limits adaptation to other single-cell assays. With CellTag-multi, we present an approach that enables direct capture of heritable random barcodes expressed as polyadenylated transcripts, in both single-cell RNA...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37748049"
                        open
                      >
                        <summary class="article-expander__title">Clearing and replacing tissue-resident myeloid cells with an anti-CD45-antibody-drug conjugate</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37748049/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231001205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue resident myeloid cells (TRM) in adults have highly variable lifespans and may be derived from early embryonic yolk sac, fetal liver or bone marrow. Some of these TRM are known pathogenic participants in congenital and acquired diseases. Myeloablative conditioning and hematopoietic stem cell transplant can replace long-lived brain TRM resulting in clinical improvements in metabolic storage diseases. With the advent of antibody-drug-conjugate (ADC) targeted cell killing as a cell selective...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37749325"
                        open
                      >
                        <summary class="article-expander__title">Newborn and child-like molecular signatures in older adults stem from TCR shifts across human lifespan</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37749325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230930205630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CD8^(+) T cells provide robust antiviral immunity, but how epitope-specific T cells evolve across the human lifespan is unclear. Here we defined CD8^(+) T cell immunity directed at the prominent influenza epitope HLA-A*02:01-M1(58-66) (A2/M1(58)) across four age groups at phenotypic, transcriptomic, clonal and functional levels. We identify a linear differentiation trajectory from newborns to children then adults, followed by divergence and a clonal reset in older adults. Gene profiles in older...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37749269"
                        open
                      >
                        <summary class="article-expander__title">Single-cell lineage capture across genomic modalities with CellTag-multi reveals fate-specific gene regulatory changes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37749269/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230930205630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Complex gene regulatory mechanisms underlie differentiation and reprogramming. Contemporary single-cell lineage-tracing (scLT) methods use expressed, heritable DNA barcodes to combine cell lineage readout with single-cell transcriptomics. However, reliance on transcriptional profiling limits adaptation to other single-cell assays. With CellTag-multi, we present an approach that enables direct capture of heritable random barcodes expressed as polyadenylated transcripts, in both single-cell RNA...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37748049"
                        open
                      >
                        <summary class="article-expander__title">Clearing and replacing tissue-resident myeloid cells with an anti-CD45-antibody-drug conjugate</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37748049/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230930205630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue resident myeloid cells (TRM) in adults have highly variable lifespans and may be derived from early embryonic yolk sac, fetal liver or bone marrow. Some of these TRM are known pathogenic participants in congenital and acquired diseases. Myeloablative conditioning and hematopoietic stem cell transplant can replace long-lived brain TRM resulting in clinical improvements in metabolic storage diseases. With the advent of antibody-drug-conjugate (ADC) targeted cell killing as a cell selective...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37749325"
                        open
                      >
                        <summary class="article-expander__title">Newborn and child-like molecular signatures in older adults stem from TCR shifts across human lifespan</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37749325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230929204823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CD8^(+) T cells provide robust antiviral immunity, but how epitope-specific T cells evolve across the human lifespan is unclear. Here we defined CD8^(+) T cell immunity directed at the prominent influenza epitope HLA-A*02:01-M1(58-66) (A2/M1(58)) across four age groups at phenotypic, transcriptomic, clonal and functional levels. We identify a linear differentiation trajectory from newborns to children then adults, followed by divergence and a clonal reset in older adults. Gene profiles in older...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37749269"
                        open
                      >
                        <summary class="article-expander__title">Single-cell lineage capture across genomic modalities with CellTag-multi reveals fate-specific gene regulatory changes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37749269/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230929204823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Complex gene regulatory mechanisms underlie differentiation and reprogramming. Contemporary single-cell lineage-tracing (scLT) methods use expressed, heritable DNA barcodes to combine cell lineage readout with single-cell transcriptomics. However, reliance on transcriptional profiling limits adaptation to other single-cell assays. With CellTag-multi, we present an approach that enables direct capture of heritable random barcodes expressed as polyadenylated transcripts, in both single-cell RNA...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37748049"
                        open
                      >
                        <summary class="article-expander__title">Clearing and replacing tissue-resident myeloid cells with an anti-CD45-antibody-drug conjugate</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37748049/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230929204823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue resident myeloid cells (TRM) in adults have highly variable lifespans and may be derived from early embryonic yolk sac, fetal liver or bone marrow. Some of these TRM are known pathogenic participants in congenital and acquired diseases. Myeloablative conditioning and hematopoietic stem cell transplant can replace long-lived brain TRM resulting in clinical improvements in metabolic storage diseases. With the advent of antibody-drug-conjugate (ADC) targeted cell killing as a cell selective...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37749325"
                        open
                      >
                        <summary class="article-expander__title">Newborn and child-like molecular signatures in older adults stem from TCR shifts across human lifespan</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37749325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230928204947&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CD8^(+) T cells provide robust antiviral immunity, but how epitope-specific T cells evolve across the human lifespan is unclear. Here we defined CD8^(+) T cell immunity directed at the prominent influenza epitope HLA-A*02:01-M1(58-66) (A2/M1(58)) across four age groups at phenotypic, transcriptomic, clonal and functional levels. We identify a linear differentiation trajectory from newborns to children then adults, followed by divergence and a clonal reset in older adults. Gene profiles in older...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37749269"
                        open
                      >
                        <summary class="article-expander__title">Single-cell lineage capture across genomic modalities with CellTag-multi reveals fate-specific gene regulatory changes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37749269/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230928204947&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Complex gene regulatory mechanisms underlie differentiation and reprogramming. Contemporary single-cell lineage-tracing (scLT) methods use expressed, heritable DNA barcodes to combine cell lineage readout with single-cell transcriptomics. However, reliance on transcriptional profiling limits adaptation to other single-cell assays. With CellTag-multi, we present an approach that enables direct capture of heritable random barcodes expressed as polyadenylated transcripts, in both single-cell RNA...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37748049"
                        open
                      >
                        <summary class="article-expander__title">Clearing and replacing tissue-resident myeloid cells with an anti-CD45-antibody-drug conjugate</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37748049/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230928204947&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue resident myeloid cells (TRM) in adults have highly variable lifespans and may be derived from early embryonic yolk sac, fetal liver or bone marrow. Some of these TRM are known pathogenic participants in congenital and acquired diseases. Myeloablative conditioning and hematopoietic stem cell transplant can replace long-lived brain TRM resulting in clinical improvements in metabolic storage diseases. With the advent of antibody-drug-conjugate (ADC) targeted cell killing as a cell selective...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37749325"
                        open
                      >
                        <summary class="article-expander__title">Newborn and child-like molecular signatures in older adults stem from TCR shifts across human lifespan</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37749325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230927204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CD8^(+) T cells provide robust antiviral immunity, but how epitope-specific T cells evolve across the human lifespan is unclear. Here we defined CD8^(+) T cell immunity directed at the prominent influenza epitope HLA-A*02:01-M1(58-66) (A2/M1(58)) across four age groups at phenotypic, transcriptomic, clonal and functional levels. We identify a linear differentiation trajectory from newborns to children then adults, followed by divergence and a clonal reset in older adults. Gene profiles in older...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37749269"
                        open
                      >
                        <summary class="article-expander__title">Single-cell lineage capture across genomic modalities with CellTag-multi reveals fate-specific gene regulatory changes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37749269/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230927204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Complex gene regulatory mechanisms underlie differentiation and reprogramming. Contemporary single-cell lineage-tracing (scLT) methods use expressed, heritable DNA barcodes to combine cell lineage readout with single-cell transcriptomics. However, reliance on transcriptional profiling limits adaptation to other single-cell assays. With CellTag-multi, we present an approach that enables direct capture of heritable random barcodes expressed as polyadenylated transcripts, in both single-cell RNA...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37748049"
                        open
                      >
                        <summary class="article-expander__title">Clearing and replacing tissue-resident myeloid cells with an anti-CD45-antibody-drug conjugate</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37748049/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230927204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue resident myeloid cells (TRM) in adults have highly variable lifespans and may be derived from early embryonic yolk sac, fetal liver or bone marrow. Some of these TRM are known pathogenic participants in congenital and acquired diseases. Myeloablative conditioning and hematopoietic stem cell transplant can replace long-lived brain TRM resulting in clinical improvements in metabolic storage diseases. With the advent of antibody-drug-conjugate (ADC) targeted cell killing as a cell selective...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37749325"
                        open
                      >
                        <summary class="article-expander__title">Newborn and child-like molecular signatures in older adults stem from TCR shifts across human lifespan</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37749325/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230926205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CD8^(+) T cells provide robust antiviral immunity, but how epitope-specific T cells evolve across the human lifespan is unclear. Here we defined CD8^(+) T cell immunity directed at the prominent influenza epitope HLA-A*02:01-M1(58-66) (A2/M1(58)) across four age groups at phenotypic, transcriptomic, clonal and functional levels. We identify a linear differentiation trajectory from newborns to children then adults, followed by divergence and a clonal reset in older adults. Gene profiles in older...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37749269"
                        open
                      >
                        <summary class="article-expander__title">Single-cell lineage capture across genomic modalities with CellTag-multi reveals fate-specific gene regulatory changes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37749269/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230926205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Complex gene regulatory mechanisms underlie differentiation and reprogramming. Contemporary single-cell lineage-tracing (scLT) methods use expressed, heritable DNA barcodes to combine cell lineage readout with single-cell transcriptomics. However, reliance on transcriptional profiling limits adaptation to other single-cell assays. With CellTag-multi, we present an approach that enables direct capture of heritable random barcodes expressed as polyadenylated transcripts, in both single-cell RNA...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37748049"
                        open
                      >
                        <summary class="article-expander__title">Clearing and replacing tissue-resident myeloid cells with an anti-CD45-antibody-drug conjugate</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37748049/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230926205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue resident myeloid cells (TRM) in adults have highly variable lifespans and may be derived from early embryonic yolk sac, fetal liver or bone marrow. Some of these TRM are known pathogenic participants in congenital and acquired diseases. Myeloablative conditioning and hematopoietic stem cell transplant can replace long-lived brain TRM resulting in clinical improvements in metabolic storage diseases. With the advent of antibody-drug-conjugate (ADC) targeted cell killing as a cell selective...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37748049"
                        open
                      >
                        <summary class="article-expander__title">Clearing and replacing tissue-resident myeloid cells with an anti-CD45-antibody-drug conjugate</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37748049/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230925205009&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue resident myeloid cells (TRM) in adults have highly variable lifespans and may be derived from early embryonic yolk sac, fetal liver or bone marrow. Some of these TRM are known pathogenic participants in congenital and acquired diseases. Myeloablative conditioning and hematopoietic stem cell transplant can replace long-lived brain TRM resulting in clinical improvements in metabolic storage diseases. With the advent of antibody-drug-conjugate (ADC) targeted cell killing as a cell selective...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;ff&#x3D;20231105195156&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37746635"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive and realistic simulation of tumour genomic sequencing data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37746635/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231013204816&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate identification of somatic mutations and allele frequencies in cancer has critical research and clinical applications. Several computational tools have been developed for this purpose but, in the absence of comprehensive &#x27;ground truth&#x27; data, assessing the accuracy of these methods is challenging. We created a computational framework to simulate tumour and matched normal sequencing data for which the source of all loci that contain non-reference bases is known, based on a phased,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37746635"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive and realistic simulation of tumour genomic sequencing data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37746635/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231012205121&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate identification of somatic mutations and allele frequencies in cancer has critical research and clinical applications. Several computational tools have been developed for this purpose but, in the absence of comprehensive &#x27;ground truth&#x27; data, assessing the accuracy of these methods is challenging. We created a computational framework to simulate tumour and matched normal sequencing data for which the source of all loci that contain non-reference bases is known, based on a phased,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37746635"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive and realistic simulation of tumour genomic sequencing data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37746635/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231011204744&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate identification of somatic mutations and allele frequencies in cancer has critical research and clinical applications. Several computational tools have been developed for this purpose but, in the absence of comprehensive &#x27;ground truth&#x27; data, assessing the accuracy of these methods is challenging. We created a computational framework to simulate tumour and matched normal sequencing data for which the source of all loci that contain non-reference bases is known, based on a phased,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37746635"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive and realistic simulation of tumour genomic sequencing data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37746635/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231010204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate identification of somatic mutations and allele frequencies in cancer has critical research and clinical applications. Several computational tools have been developed for this purpose but, in the absence of comprehensive &#x27;ground truth&#x27; data, assessing the accuracy of these methods is challenging. We created a computational framework to simulate tumour and matched normal sequencing data for which the source of all loci that contain non-reference bases is known, based on a phased,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37746635"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive and realistic simulation of tumour genomic sequencing data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37746635/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231009204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate identification of somatic mutations and allele frequencies in cancer has critical research and clinical applications. Several computational tools have been developed for this purpose but, in the absence of comprehensive &#x27;ground truth&#x27; data, assessing the accuracy of these methods is challenging. We created a computational framework to simulate tumour and matched normal sequencing data for which the source of all loci that contain non-reference bases is known, based on a phased,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37746635"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive and realistic simulation of tumour genomic sequencing data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37746635/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231008205021&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate identification of somatic mutations and allele frequencies in cancer has critical research and clinical applications. Several computational tools have been developed for this purpose but, in the absence of comprehensive &#x27;ground truth&#x27; data, assessing the accuracy of these methods is challenging. We created a computational framework to simulate tumour and matched normal sequencing data for which the source of all loci that contain non-reference bases is known, based on a phased,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37746635"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive and realistic simulation of tumour genomic sequencing data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37746635/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231007205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate identification of somatic mutations and allele frequencies in cancer has critical research and clinical applications. Several computational tools have been developed for this purpose but, in the absence of comprehensive &#x27;ground truth&#x27; data, assessing the accuracy of these methods is challenging. We created a computational framework to simulate tumour and matched normal sequencing data for which the source of all loci that contain non-reference bases is known, based on a phased,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37746635"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive and realistic simulation of tumour genomic sequencing data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37746635/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231006204916&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate identification of somatic mutations and allele frequencies in cancer has critical research and clinical applications. Several computational tools have been developed for this purpose but, in the absence of comprehensive &#x27;ground truth&#x27; data, assessing the accuracy of these methods is challenging. We created a computational framework to simulate tumour and matched normal sequencing data for which the source of all loci that contain non-reference bases is known, based on a phased,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37746635"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive and realistic simulation of tumour genomic sequencing data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37746635/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231005204957&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate identification of somatic mutations and allele frequencies in cancer has critical research and clinical applications. Several computational tools have been developed for this purpose but, in the absence of comprehensive &#x27;ground truth&#x27; data, assessing the accuracy of these methods is challenging. We created a computational framework to simulate tumour and matched normal sequencing data for which the source of all loci that contain non-reference bases is known, based on a phased,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37746635"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive and realistic simulation of tumour genomic sequencing data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37746635/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231004205006&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate identification of somatic mutations and allele frequencies in cancer has critical research and clinical applications. Several computational tools have been developed for this purpose but, in the absence of comprehensive &#x27;ground truth&#x27; data, assessing the accuracy of these methods is challenging. We created a computational framework to simulate tumour and matched normal sequencing data for which the source of all loci that contain non-reference bases is known, based on a phased,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37746635"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive and realistic simulation of tumour genomic sequencing data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37746635/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231003205044&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate identification of somatic mutations and allele frequencies in cancer has critical research and clinical applications. Several computational tools have been developed for this purpose but, in the absence of comprehensive &#x27;ground truth&#x27; data, assessing the accuracy of these methods is challenging. We created a computational framework to simulate tumour and matched normal sequencing data for which the source of all loci that contain non-reference bases is known, based on a phased,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37746635"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive and realistic simulation of tumour genomic sequencing data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37746635/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231002205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate identification of somatic mutations and allele frequencies in cancer has critical research and clinical applications. Several computational tools have been developed for this purpose but, in the absence of comprehensive &#x27;ground truth&#x27; data, assessing the accuracy of these methods is challenging. We created a computational framework to simulate tumour and matched normal sequencing data for which the source of all loci that contain non-reference bases is known, based on a phased,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37746635"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive and realistic simulation of tumour genomic sequencing data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37746635/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231001205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate identification of somatic mutations and allele frequencies in cancer has critical research and clinical applications. Several computational tools have been developed for this purpose but, in the absence of comprehensive &#x27;ground truth&#x27; data, assessing the accuracy of these methods is challenging. We created a computational framework to simulate tumour and matched normal sequencing data for which the source of all loci that contain non-reference bases is known, based on a phased,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37746635"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive and realistic simulation of tumour genomic sequencing data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37746635/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230930205630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate identification of somatic mutations and allele frequencies in cancer has critical research and clinical applications. Several computational tools have been developed for this purpose but, in the absence of comprehensive &#x27;ground truth&#x27; data, assessing the accuracy of these methods is challenging. We created a computational framework to simulate tumour and matched normal sequencing data for which the source of all loci that contain non-reference bases is known, based on a phased,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37746635"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive and realistic simulation of tumour genomic sequencing data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37746635/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230929204823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate identification of somatic mutations and allele frequencies in cancer has critical research and clinical applications. Several computational tools have been developed for this purpose but, in the absence of comprehensive &#x27;ground truth&#x27; data, assessing the accuracy of these methods is challenging. We created a computational framework to simulate tumour and matched normal sequencing data for which the source of all loci that contain non-reference bases is known, based on a phased,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37746635"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive and realistic simulation of tumour genomic sequencing data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37746635/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230928204948&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate identification of somatic mutations and allele frequencies in cancer has critical research and clinical applications. Several computational tools have been developed for this purpose but, in the absence of comprehensive &#x27;ground truth&#x27; data, assessing the accuracy of these methods is challenging. We created a computational framework to simulate tumour and matched normal sequencing data for which the source of all loci that contain non-reference bases is known, based on a phased,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37746635"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive and realistic simulation of tumour genomic sequencing data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37746635/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230927204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate identification of somatic mutations and allele frequencies in cancer has critical research and clinical applications. Several computational tools have been developed for this purpose but, in the absence of comprehensive &#x27;ground truth&#x27; data, assessing the accuracy of these methods is challenging. We created a computational framework to simulate tumour and matched normal sequencing data for which the source of all loci that contain non-reference bases is known, based on a phased,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37746635"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive and realistic simulation of tumour genomic sequencing data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37746635/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230926205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate identification of somatic mutations and allele frequencies in cancer has critical research and clinical applications. Several computational tools have been developed for this purpose but, in the absence of comprehensive &#x27;ground truth&#x27; data, assessing the accuracy of these methods is challenging. We created a computational framework to simulate tumour and matched normal sequencing data for which the source of all loci that contain non-reference bases is known, based on a phased,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37746635"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive and realistic simulation of tumour genomic sequencing data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37746635/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230925205014&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate identification of somatic mutations and allele frequencies in cancer has critical research and clinical applications. Several computational tools have been developed for this purpose but, in the absence of comprehensive &#x27;ground truth&#x27; data, assessing the accuracy of these methods is challenging. We created a computational framework to simulate tumour and matched normal sequencing data for which the source of all loci that contain non-reference bases is known, based on a phased,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-09-23 datetime="2023-09-23T10:00:00.000Z">2023-09-23</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-09-23 datetime="2023-09-23T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;ff&#x3D;20231105195156&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37741019"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution of long-term expanding head and neck cancer organoid: Impact on treatment response for personalized therapeutic screening</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37741019/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231011204744&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We report the establishment of HNCO lines derived from 27 patients and demonstrate their genetic concordance with corresponding parental tumors. Furthermore, we show serial changes in mutational profiles of HNCO along with long passage culture and the impact of these clonal evolutions on response to radiotherapy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37741019"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution of long-term expanding head and neck cancer organoid: Impact on treatment response for personalized therapeutic screening</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37741019/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231010204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We report the establishment of HNCO lines derived from 27 patients and demonstrate their genetic concordance with corresponding parental tumors. Furthermore, we show serial changes in mutational profiles of HNCO along with long passage culture and the impact of these clonal evolutions on response to radiotherapy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37741019"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution of long-term expanding head and neck cancer organoid: Impact on treatment response for personalized therapeutic screening</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37741019/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231009204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We report the establishment of HNCO lines derived from 27 patients and demonstrate their genetic concordance with corresponding parental tumors. Furthermore, we show serial changes in mutational profiles of HNCO along with long passage culture and the impact of these clonal evolutions on response to radiotherapy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37741019"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution of long-term expanding head and neck cancer organoid: Impact on treatment response for personalized therapeutic screening</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37741019/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231008205021&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We report the establishment of HNCO lines derived from 27 patients and demonstrate their genetic concordance with corresponding parental tumors. Furthermore, we show serial changes in mutational profiles of HNCO along with long passage culture and the impact of these clonal evolutions on response to radiotherapy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37741019"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution of long-term expanding head and neck cancer organoid: Impact on treatment response for personalized therapeutic screening</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37741019/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231007205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We report the establishment of HNCO lines derived from 27 patients and demonstrate their genetic concordance with corresponding parental tumors. Furthermore, we show serial changes in mutational profiles of HNCO along with long passage culture and the impact of these clonal evolutions on response to radiotherapy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37741019"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution of long-term expanding head and neck cancer organoid: Impact on treatment response for personalized therapeutic screening</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37741019/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231006204916&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We report the establishment of HNCO lines derived from 27 patients and demonstrate their genetic concordance with corresponding parental tumors. Furthermore, we show serial changes in mutational profiles of HNCO along with long passage culture and the impact of these clonal evolutions on response to radiotherapy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37741019"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution of long-term expanding head and neck cancer organoid: Impact on treatment response for personalized therapeutic screening</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37741019/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231005204957&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We report the establishment of HNCO lines derived from 27 patients and demonstrate their genetic concordance with corresponding parental tumors. Furthermore, we show serial changes in mutational profiles of HNCO along with long passage culture and the impact of these clonal evolutions on response to radiotherapy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37741019"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution of long-term expanding head and neck cancer organoid: Impact on treatment response for personalized therapeutic screening</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37741019/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231004205006&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We report the establishment of HNCO lines derived from 27 patients and demonstrate their genetic concordance with corresponding parental tumors. Furthermore, we show serial changes in mutational profiles of HNCO along with long passage culture and the impact of these clonal evolutions on response to radiotherapy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37741019"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution of long-term expanding head and neck cancer organoid: Impact on treatment response for personalized therapeutic screening</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37741019/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231003205044&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We report the establishment of HNCO lines derived from 27 patients and demonstrate their genetic concordance with corresponding parental tumors. Furthermore, we show serial changes in mutational profiles of HNCO along with long passage culture and the impact of these clonal evolutions on response to radiotherapy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37741019"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution of long-term expanding head and neck cancer organoid: Impact on treatment response for personalized therapeutic screening</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37741019/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231002205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We report the establishment of HNCO lines derived from 27 patients and demonstrate their genetic concordance with corresponding parental tumors. Furthermore, we show serial changes in mutational profiles of HNCO along with long passage culture and the impact of these clonal evolutions on response to radiotherapy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37741019"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution of long-term expanding head and neck cancer organoid: Impact on treatment response for personalized therapeutic screening</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37741019/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231001205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We report the establishment of HNCO lines derived from 27 patients and demonstrate their genetic concordance with corresponding parental tumors. Furthermore, we show serial changes in mutational profiles of HNCO along with long passage culture and the impact of these clonal evolutions on response to radiotherapy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37741019"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution of long-term expanding head and neck cancer organoid: Impact on treatment response for personalized therapeutic screening</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37741019/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230930205630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We report the establishment of HNCO lines derived from 27 patients and demonstrate their genetic concordance with corresponding parental tumors. Furthermore, we show serial changes in mutational profiles of HNCO along with long passage culture and the impact of these clonal evolutions on response to radiotherapy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37741019"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution of long-term expanding head and neck cancer organoid: Impact on treatment response for personalized therapeutic screening</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37741019/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230929204823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We report the establishment of HNCO lines derived from 27 patients and demonstrate their genetic concordance with corresponding parental tumors. Furthermore, we show serial changes in mutational profiles of HNCO along with long passage culture and the impact of these clonal evolutions on response to radiotherapy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37741019"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution of long-term expanding head and neck cancer organoid: Impact on treatment response for personalized therapeutic screening</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37741019/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230928204948&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We report the establishment of HNCO lines derived from 27 patients and demonstrate their genetic concordance with corresponding parental tumors. Furthermore, we show serial changes in mutational profiles of HNCO along with long passage culture and the impact of these clonal evolutions on response to radiotherapy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37741019"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution of long-term expanding head and neck cancer organoid: Impact on treatment response for personalized therapeutic screening</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37741019/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230927204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We report the establishment of HNCO lines derived from 27 patients and demonstrate their genetic concordance with corresponding parental tumors. Furthermore, we show serial changes in mutational profiles of HNCO along with long passage culture and the impact of these clonal evolutions on response to radiotherapy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37741019"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution of long-term expanding head and neck cancer organoid: Impact on treatment response for personalized therapeutic screening</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37741019/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230926205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We report the establishment of HNCO lines derived from 27 patients and demonstrate their genetic concordance with corresponding parental tumors. Furthermore, we show serial changes in mutational profiles of HNCO along with long passage culture and the impact of these clonal evolutions on response to radiotherapy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37741019"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution of long-term expanding head and neck cancer organoid: Impact on treatment response for personalized therapeutic screening</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37741019/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230925205014&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We report the establishment of HNCO lines derived from 27 patients and demonstrate their genetic concordance with corresponding parental tumors. Furthermore, we show serial changes in mutational profiles of HNCO along with long passage culture and the impact of these clonal evolutions on response to radiotherapy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37741019"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution of long-term expanding head and neck cancer organoid: Impact on treatment response for personalized therapeutic screening</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37741019/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230924205014&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We report the establishment of HNCO lines derived from 27 patients and demonstrate their genetic concordance with corresponding parental tumors. Furthermore, we show serial changes in mutational profiles of HNCO along with long passage culture and the impact of these clonal evolutions on response to radiotherapy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-09-22 datetime="2023-09-22T10:00:00.000Z">2023-09-22</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-09-22 datetime="2023-09-22T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20231105195156">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37738545"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis as a Molecular Risk Factor for Doxorubicin-Induced Cardiotoxicity: A Proof-of-Concept Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37738545/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231005204957&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study provides preliminary evidence for CH as a predictive risk factor for DIC, which, with further investigation, could serve as an important precision medicine biomarker for the large number of patients with cancer who have CH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37738545"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis as a Molecular Risk Factor for Doxorubicin-Induced Cardiotoxicity: A Proof-of-Concept Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37738545/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231004205006&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study provides preliminary evidence for CH as a predictive risk factor for DIC, which, with further investigation, could serve as an important precision medicine biomarker for the large number of patients with cancer who have CH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37738545"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis as a Molecular Risk Factor for Doxorubicin-Induced Cardiotoxicity: A Proof-of-Concept Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37738545/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231003205046&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study provides preliminary evidence for CH as a predictive risk factor for DIC, which, with further investigation, could serve as an important precision medicine biomarker for the large number of patients with cancer who have CH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37738545"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis as a Molecular Risk Factor for Doxorubicin-Induced Cardiotoxicity: A Proof-of-Concept Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37738545/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231002205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study provides preliminary evidence for CH as a predictive risk factor for DIC, which, with further investigation, could serve as an important precision medicine biomarker for the large number of patients with cancer who have CH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37738545"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis as a Molecular Risk Factor for Doxorubicin-Induced Cardiotoxicity: A Proof-of-Concept Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37738545/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231001205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study provides preliminary evidence for CH as a predictive risk factor for DIC, which, with further investigation, could serve as an important precision medicine biomarker for the large number of patients with cancer who have CH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37738545"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis as a Molecular Risk Factor for Doxorubicin-Induced Cardiotoxicity: A Proof-of-Concept Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37738545/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230930205630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study provides preliminary evidence for CH as a predictive risk factor for DIC, which, with further investigation, could serve as an important precision medicine biomarker for the large number of patients with cancer who have CH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37738545"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis as a Molecular Risk Factor for Doxorubicin-Induced Cardiotoxicity: A Proof-of-Concept Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37738545/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230929204823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study provides preliminary evidence for CH as a predictive risk factor for DIC, which, with further investigation, could serve as an important precision medicine biomarker for the large number of patients with cancer who have CH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37738545"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis as a Molecular Risk Factor for Doxorubicin-Induced Cardiotoxicity: A Proof-of-Concept Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37738545/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230928204947&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study provides preliminary evidence for CH as a predictive risk factor for DIC, which, with further investigation, could serve as an important precision medicine biomarker for the large number of patients with cancer who have CH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37738545"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis as a Molecular Risk Factor for Doxorubicin-Induced Cardiotoxicity: A Proof-of-Concept Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37738545/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230927204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study provides preliminary evidence for CH as a predictive risk factor for DIC, which, with further investigation, could serve as an important precision medicine biomarker for the large number of patients with cancer who have CH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37738545"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis as a Molecular Risk Factor for Doxorubicin-Induced Cardiotoxicity: A Proof-of-Concept Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37738545/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230926205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study provides preliminary evidence for CH as a predictive risk factor for DIC, which, with further investigation, could serve as an important precision medicine biomarker for the large number of patients with cancer who have CH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37738545"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis as a Molecular Risk Factor for Doxorubicin-Induced Cardiotoxicity: A Proof-of-Concept Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37738545/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230925205009&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study provides preliminary evidence for CH as a predictive risk factor for DIC, which, with further investigation, could serve as an important precision medicine biomarker for the large number of patients with cancer who have CH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37738545"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis as a Molecular Risk Factor for Doxorubicin-Induced Cardiotoxicity: A Proof-of-Concept Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37738545/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230924205014&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study provides preliminary evidence for CH as a predictive risk factor for DIC, which, with further investigation, could serve as an important precision medicine biomarker for the large number of patients with cancer who have CH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37738545"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis as a Molecular Risk Factor for Doxorubicin-Induced Cardiotoxicity: A Proof-of-Concept Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37738545/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230923205208&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study provides preliminary evidence for CH as a predictive risk factor for DIC, which, with further investigation, could serve as an important precision medicine biomarker for the large number of patients with cancer who have CH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37738545"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis as a Molecular Risk Factor for Doxorubicin-Induced Cardiotoxicity: A Proof-of-Concept Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37738545/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230922204741&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study provides preliminary evidence for CH as a predictive risk factor for DIC, which, with further investigation, could serve as an important precision medicine biomarker for the large number of patients with cancer who have CH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;ff&#x3D;20231105195156&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37738545"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis as a Molecular Risk Factor for Doxorubicin-Induced Cardiotoxicity: A Proof-of-Concept Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37738545/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231010204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study provides preliminary evidence for CH as a predictive risk factor for DIC, which, with further investigation, could serve as an important precision medicine biomarker for the large number of patients with cancer who have CH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37738545"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis as a Molecular Risk Factor for Doxorubicin-Induced Cardiotoxicity: A Proof-of-Concept Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37738545/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231009204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study provides preliminary evidence for CH as a predictive risk factor for DIC, which, with further investigation, could serve as an important precision medicine biomarker for the large number of patients with cancer who have CH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37738545"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis as a Molecular Risk Factor for Doxorubicin-Induced Cardiotoxicity: A Proof-of-Concept Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37738545/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231008205021&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study provides preliminary evidence for CH as a predictive risk factor for DIC, which, with further investigation, could serve as an important precision medicine biomarker for the large number of patients with cancer who have CH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37738545"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis as a Molecular Risk Factor for Doxorubicin-Induced Cardiotoxicity: A Proof-of-Concept Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37738545/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231007205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study provides preliminary evidence for CH as a predictive risk factor for DIC, which, with further investigation, could serve as an important precision medicine biomarker for the large number of patients with cancer who have CH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37738545"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis as a Molecular Risk Factor for Doxorubicin-Induced Cardiotoxicity: A Proof-of-Concept Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37738545/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231006204916&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study provides preliminary evidence for CH as a predictive risk factor for DIC, which, with further investigation, could serve as an important precision medicine biomarker for the large number of patients with cancer who have CH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37738545"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis as a Molecular Risk Factor for Doxorubicin-Induced Cardiotoxicity: A Proof-of-Concept Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37738545/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231005204957&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study provides preliminary evidence for CH as a predictive risk factor for DIC, which, with further investigation, could serve as an important precision medicine biomarker for the large number of patients with cancer who have CH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37738545"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis as a Molecular Risk Factor for Doxorubicin-Induced Cardiotoxicity: A Proof-of-Concept Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37738545/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231004205006&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study provides preliminary evidence for CH as a predictive risk factor for DIC, which, with further investigation, could serve as an important precision medicine biomarker for the large number of patients with cancer who have CH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37738545"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis as a Molecular Risk Factor for Doxorubicin-Induced Cardiotoxicity: A Proof-of-Concept Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37738545/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231003205044&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study provides preliminary evidence for CH as a predictive risk factor for DIC, which, with further investigation, could serve as an important precision medicine biomarker for the large number of patients with cancer who have CH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37738545"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis as a Molecular Risk Factor for Doxorubicin-Induced Cardiotoxicity: A Proof-of-Concept Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37738545/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231002205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study provides preliminary evidence for CH as a predictive risk factor for DIC, which, with further investigation, could serve as an important precision medicine biomarker for the large number of patients with cancer who have CH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37738545"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis as a Molecular Risk Factor for Doxorubicin-Induced Cardiotoxicity: A Proof-of-Concept Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37738545/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231001205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study provides preliminary evidence for CH as a predictive risk factor for DIC, which, with further investigation, could serve as an important precision medicine biomarker for the large number of patients with cancer who have CH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37738545"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis as a Molecular Risk Factor for Doxorubicin-Induced Cardiotoxicity: A Proof-of-Concept Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37738545/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230930205630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study provides preliminary evidence for CH as a predictive risk factor for DIC, which, with further investigation, could serve as an important precision medicine biomarker for the large number of patients with cancer who have CH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37738545"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis as a Molecular Risk Factor for Doxorubicin-Induced Cardiotoxicity: A Proof-of-Concept Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37738545/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230929204823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study provides preliminary evidence for CH as a predictive risk factor for DIC, which, with further investigation, could serve as an important precision medicine biomarker for the large number of patients with cancer who have CH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37738545"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis as a Molecular Risk Factor for Doxorubicin-Induced Cardiotoxicity: A Proof-of-Concept Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37738545/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230928204948&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study provides preliminary evidence for CH as a predictive risk factor for DIC, which, with further investigation, could serve as an important precision medicine biomarker for the large number of patients with cancer who have CH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37738545"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis as a Molecular Risk Factor for Doxorubicin-Induced Cardiotoxicity: A Proof-of-Concept Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37738545/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230927204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study provides preliminary evidence for CH as a predictive risk factor for DIC, which, with further investigation, could serve as an important precision medicine biomarker for the large number of patients with cancer who have CH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37738545"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis as a Molecular Risk Factor for Doxorubicin-Induced Cardiotoxicity: A Proof-of-Concept Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37738545/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230926205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study provides preliminary evidence for CH as a predictive risk factor for DIC, which, with further investigation, could serve as an important precision medicine biomarker for the large number of patients with cancer who have CH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37738545"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis as a Molecular Risk Factor for Doxorubicin-Induced Cardiotoxicity: A Proof-of-Concept Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37738545/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230925205014&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study provides preliminary evidence for CH as a predictive risk factor for DIC, which, with further investigation, could serve as an important precision medicine biomarker for the large number of patients with cancer who have CH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-09-21 datetime="2023-09-21T10:00:00.000Z">2023-09-21</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-09-21 datetime="2023-09-21T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20231105195156">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37732224"
                        open
                      >
                        <summary class="article-expander__title">CD45-antibody-drug conjugate clears tissue resident myeloid cells from their niches enabling therapeutic adoptive cell transfer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37732224/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231004205006&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue resident myeloid cells (TRM) in adults have highly variable lifespans and may be derived from early embryonic yolk sac, fetal liver or bone marrow. Some of these TRM are known pathogenic participants in congenital and acquired diseases. Myeloablative conditioning and hematopoietic stem cell transplant can replace long-lived brain TRM resulting in clinical improvements in metabolic storage diseases. With the advent of antibody-drug-conjugate (ADC) targeted cell killing as a cell selective...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37732181"
                        open
                      >
                        <summary class="article-expander__title">Long-term longitudinal analysis of 4,187 participants reveals new insights into determinants of incident clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37732181/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231004205006&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH), characterized by blood cells predominantly originating from a single mutated hematopoietic stem cell, is linked to diverse aging-related diseases, including hematologic malignancy and atherosclerotic cardiovascular disease (ASCVD). While CH is common among older adults, the underlying factors driving its development are largely unknown. To address this, we performed whole-exome sequencing on 8,374 blood DNA samples collected from 4,187 Atherosclerosis Risk in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37732224"
                        open
                      >
                        <summary class="article-expander__title">CD45-antibody-drug conjugate clears tissue resident myeloid cells from their niches enabling therapeutic adoptive cell transfer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37732224/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231003205046&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue resident myeloid cells (TRM) in adults have highly variable lifespans and may be derived from early embryonic yolk sac, fetal liver or bone marrow. Some of these TRM are known pathogenic participants in congenital and acquired diseases. Myeloablative conditioning and hematopoietic stem cell transplant can replace long-lived brain TRM resulting in clinical improvements in metabolic storage diseases. With the advent of antibody-drug-conjugate (ADC) targeted cell killing as a cell selective...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37732181"
                        open
                      >
                        <summary class="article-expander__title">Long-term longitudinal analysis of 4,187 participants reveals new insights into determinants of incident clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37732181/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231003205046&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH), characterized by blood cells predominantly originating from a single mutated hematopoietic stem cell, is linked to diverse aging-related diseases, including hematologic malignancy and atherosclerotic cardiovascular disease (ASCVD). While CH is common among older adults, the underlying factors driving its development are largely unknown. To address this, we performed whole-exome sequencing on 8,374 blood DNA samples collected from 4,187 Atherosclerosis Risk in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37732224"
                        open
                      >
                        <summary class="article-expander__title">CD45-antibody-drug conjugate clears tissue resident myeloid cells from their niches enabling therapeutic adoptive cell transfer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37732224/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231002205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue resident myeloid cells (TRM) in adults have highly variable lifespans and may be derived from early embryonic yolk sac, fetal liver or bone marrow. Some of these TRM are known pathogenic participants in congenital and acquired diseases. Myeloablative conditioning and hematopoietic stem cell transplant can replace long-lived brain TRM resulting in clinical improvements in metabolic storage diseases. With the advent of antibody-drug-conjugate (ADC) targeted cell killing as a cell selective...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37732181"
                        open
                      >
                        <summary class="article-expander__title">Long-term longitudinal analysis of 4,187 participants reveals new insights into determinants of incident clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37732181/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231002205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH), characterized by blood cells predominantly originating from a single mutated hematopoietic stem cell, is linked to diverse aging-related diseases, including hematologic malignancy and atherosclerotic cardiovascular disease (ASCVD). While CH is common among older adults, the underlying factors driving its development are largely unknown. To address this, we performed whole-exome sequencing on 8,374 blood DNA samples collected from 4,187 Atherosclerosis Risk in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37732224"
                        open
                      >
                        <summary class="article-expander__title">CD45-antibody-drug conjugate clears tissue resident myeloid cells from their niches enabling therapeutic adoptive cell transfer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37732224/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231001205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue resident myeloid cells (TRM) in adults have highly variable lifespans and may be derived from early embryonic yolk sac, fetal liver or bone marrow. Some of these TRM are known pathogenic participants in congenital and acquired diseases. Myeloablative conditioning and hematopoietic stem cell transplant can replace long-lived brain TRM resulting in clinical improvements in metabolic storage diseases. With the advent of antibody-drug-conjugate (ADC) targeted cell killing as a cell selective...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37732181"
                        open
                      >
                        <summary class="article-expander__title">Long-term longitudinal analysis of 4,187 participants reveals new insights into determinants of incident clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37732181/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231001205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH), characterized by blood cells predominantly originating from a single mutated hematopoietic stem cell, is linked to diverse aging-related diseases, including hematologic malignancy and atherosclerotic cardiovascular disease (ASCVD). While CH is common among older adults, the underlying factors driving its development are largely unknown. To address this, we performed whole-exome sequencing on 8,374 blood DNA samples collected from 4,187 Atherosclerosis Risk in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37732224"
                        open
                      >
                        <summary class="article-expander__title">CD45-antibody-drug conjugate clears tissue resident myeloid cells from their niches enabling therapeutic adoptive cell transfer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37732224/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230930205630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue resident myeloid cells (TRM) in adults have highly variable lifespans and may be derived from early embryonic yolk sac, fetal liver or bone marrow. Some of these TRM are known pathogenic participants in congenital and acquired diseases. Myeloablative conditioning and hematopoietic stem cell transplant can replace long-lived brain TRM resulting in clinical improvements in metabolic storage diseases. With the advent of antibody-drug-conjugate (ADC) targeted cell killing as a cell selective...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37732181"
                        open
                      >
                        <summary class="article-expander__title">Long-term longitudinal analysis of 4,187 participants reveals new insights into determinants of incident clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37732181/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230930205630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH), characterized by blood cells predominantly originating from a single mutated hematopoietic stem cell, is linked to diverse aging-related diseases, including hematologic malignancy and atherosclerotic cardiovascular disease (ASCVD). While CH is common among older adults, the underlying factors driving its development are largely unknown. To address this, we performed whole-exome sequencing on 8,374 blood DNA samples collected from 4,187 Atherosclerosis Risk in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37732224"
                        open
                      >
                        <summary class="article-expander__title">CD45-antibody-drug conjugate clears tissue resident myeloid cells from their niches enabling therapeutic adoptive cell transfer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37732224/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230929204823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue resident myeloid cells (TRM) in adults have highly variable lifespans and may be derived from early embryonic yolk sac, fetal liver or bone marrow. Some of these TRM are known pathogenic participants in congenital and acquired diseases. Myeloablative conditioning and hematopoietic stem cell transplant can replace long-lived brain TRM resulting in clinical improvements in metabolic storage diseases. With the advent of antibody-drug-conjugate (ADC) targeted cell killing as a cell selective...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37732181"
                        open
                      >
                        <summary class="article-expander__title">Long-term longitudinal analysis of 4,187 participants reveals new insights into determinants of incident clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37732181/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230929204823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH), characterized by blood cells predominantly originating from a single mutated hematopoietic stem cell, is linked to diverse aging-related diseases, including hematologic malignancy and atherosclerotic cardiovascular disease (ASCVD). While CH is common among older adults, the underlying factors driving its development are largely unknown. To address this, we performed whole-exome sequencing on 8,374 blood DNA samples collected from 4,187 Atherosclerosis Risk in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37732224"
                        open
                      >
                        <summary class="article-expander__title">CD45-antibody-drug conjugate clears tissue resident myeloid cells from their niches enabling therapeutic adoptive cell transfer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37732224/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230928204947&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue resident myeloid cells (TRM) in adults have highly variable lifespans and may be derived from early embryonic yolk sac, fetal liver or bone marrow. Some of these TRM are known pathogenic participants in congenital and acquired diseases. Myeloablative conditioning and hematopoietic stem cell transplant can replace long-lived brain TRM resulting in clinical improvements in metabolic storage diseases. With the advent of antibody-drug-conjugate (ADC) targeted cell killing as a cell selective...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37732181"
                        open
                      >
                        <summary class="article-expander__title">Long-term longitudinal analysis of 4,187 participants reveals new insights into determinants of incident clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37732181/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230928204947&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH), characterized by blood cells predominantly originating from a single mutated hematopoietic stem cell, is linked to diverse aging-related diseases, including hematologic malignancy and atherosclerotic cardiovascular disease (ASCVD). While CH is common among older adults, the underlying factors driving its development are largely unknown. To address this, we performed whole-exome sequencing on 8,374 blood DNA samples collected from 4,187 Atherosclerosis Risk in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37732224"
                        open
                      >
                        <summary class="article-expander__title">CD45-antibody-drug conjugate clears tissue resident myeloid cells from their niches enabling therapeutic adoptive cell transfer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37732224/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230927204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue resident myeloid cells (TRM) in adults have highly variable lifespans and may be derived from early embryonic yolk sac, fetal liver or bone marrow. Some of these TRM are known pathogenic participants in congenital and acquired diseases. Myeloablative conditioning and hematopoietic stem cell transplant can replace long-lived brain TRM resulting in clinical improvements in metabolic storage diseases. With the advent of antibody-drug-conjugate (ADC) targeted cell killing as a cell selective...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37732181"
                        open
                      >
                        <summary class="article-expander__title">Long-term longitudinal analysis of 4,187 participants reveals new insights into determinants of incident clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37732181/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230927204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH), characterized by blood cells predominantly originating from a single mutated hematopoietic stem cell, is linked to diverse aging-related diseases, including hematologic malignancy and atherosclerotic cardiovascular disease (ASCVD). While CH is common among older adults, the underlying factors driving its development are largely unknown. To address this, we performed whole-exome sequencing on 8,374 blood DNA samples collected from 4,187 Atherosclerosis Risk in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37732224"
                        open
                      >
                        <summary class="article-expander__title">CD45-antibody-drug conjugate clears tissue resident myeloid cells from their niches enabling therapeutic adoptive cell transfer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37732224/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230926205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue resident myeloid cells (TRM) in adults have highly variable lifespans and may be derived from early embryonic yolk sac, fetal liver or bone marrow. Some of these TRM are known pathogenic participants in congenital and acquired diseases. Myeloablative conditioning and hematopoietic stem cell transplant can replace long-lived brain TRM resulting in clinical improvements in metabolic storage diseases. With the advent of antibody-drug-conjugate (ADC) targeted cell killing as a cell selective...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37732181"
                        open
                      >
                        <summary class="article-expander__title">Long-term longitudinal analysis of 4,187 participants reveals new insights into determinants of incident clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37732181/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230926205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH), characterized by blood cells predominantly originating from a single mutated hematopoietic stem cell, is linked to diverse aging-related diseases, including hematologic malignancy and atherosclerotic cardiovascular disease (ASCVD). While CH is common among older adults, the underlying factors driving its development are largely unknown. To address this, we performed whole-exome sequencing on 8,374 blood DNA samples collected from 4,187 Atherosclerosis Risk in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37732224"
                        open
                      >
                        <summary class="article-expander__title">CD45-antibody-drug conjugate clears tissue resident myeloid cells from their niches enabling therapeutic adoptive cell transfer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37732224/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230925205009&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue resident myeloid cells (TRM) in adults have highly variable lifespans and may be derived from early embryonic yolk sac, fetal liver or bone marrow. Some of these TRM are known pathogenic participants in congenital and acquired diseases. Myeloablative conditioning and hematopoietic stem cell transplant can replace long-lived brain TRM resulting in clinical improvements in metabolic storage diseases. With the advent of antibody-drug-conjugate (ADC) targeted cell killing as a cell selective...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37732181"
                        open
                      >
                        <summary class="article-expander__title">Long-term longitudinal analysis of 4,187 participants reveals new insights into determinants of incident clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37732181/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230925205009&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH), characterized by blood cells predominantly originating from a single mutated hematopoietic stem cell, is linked to diverse aging-related diseases, including hematologic malignancy and atherosclerotic cardiovascular disease (ASCVD). While CH is common among older adults, the underlying factors driving its development are largely unknown. To address this, we performed whole-exome sequencing on 8,374 blood DNA samples collected from 4,187 Atherosclerosis Risk in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37732224"
                        open
                      >
                        <summary class="article-expander__title">CD45-antibody-drug conjugate clears tissue resident myeloid cells from their niches enabling therapeutic adoptive cell transfer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37732224/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230924205014&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue resident myeloid cells (TRM) in adults have highly variable lifespans and may be derived from early embryonic yolk sac, fetal liver or bone marrow. Some of these TRM are known pathogenic participants in congenital and acquired diseases. Myeloablative conditioning and hematopoietic stem cell transplant can replace long-lived brain TRM resulting in clinical improvements in metabolic storage diseases. With the advent of antibody-drug-conjugate (ADC) targeted cell killing as a cell selective...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37732181"
                        open
                      >
                        <summary class="article-expander__title">Long-term longitudinal analysis of 4,187 participants reveals new insights into determinants of incident clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37732181/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230924205014&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH), characterized by blood cells predominantly originating from a single mutated hematopoietic stem cell, is linked to diverse aging-related diseases, including hematologic malignancy and atherosclerotic cardiovascular disease (ASCVD). While CH is common among older adults, the underlying factors driving its development are largely unknown. To address this, we performed whole-exome sequencing on 8,374 blood DNA samples collected from 4,187 Atherosclerosis Risk in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37732224"
                        open
                      >
                        <summary class="article-expander__title">CD45-antibody-drug conjugate clears tissue resident myeloid cells from their niches enabling therapeutic adoptive cell transfer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37732224/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230923205208&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue resident myeloid cells (TRM) in adults have highly variable lifespans and may be derived from early embryonic yolk sac, fetal liver or bone marrow. Some of these TRM are known pathogenic participants in congenital and acquired diseases. Myeloablative conditioning and hematopoietic stem cell transplant can replace long-lived brain TRM resulting in clinical improvements in metabolic storage diseases. With the advent of antibody-drug-conjugate (ADC) targeted cell killing as a cell selective...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37732181"
                        open
                      >
                        <summary class="article-expander__title">Long-term longitudinal analysis of 4,187 participants reveals new insights into determinants of incident clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37732181/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230923205208&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH), characterized by blood cells predominantly originating from a single mutated hematopoietic stem cell, is linked to diverse aging-related diseases, including hematologic malignancy and atherosclerotic cardiovascular disease (ASCVD). While CH is common among older adults, the underlying factors driving its development are largely unknown. To address this, we performed whole-exome sequencing on 8,374 blood DNA samples collected from 4,187 Atherosclerosis Risk in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37732224"
                        open
                      >
                        <summary class="article-expander__title">CD45-antibody-drug conjugate clears tissue resident myeloid cells from their niches enabling therapeutic adoptive cell transfer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37732224/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230922204741&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue resident myeloid cells (TRM) in adults have highly variable lifespans and may be derived from early embryonic yolk sac, fetal liver or bone marrow. Some of these TRM are known pathogenic participants in congenital and acquired diseases. Myeloablative conditioning and hematopoietic stem cell transplant can replace long-lived brain TRM resulting in clinical improvements in metabolic storage diseases. With the advent of antibody-drug-conjugate (ADC) targeted cell killing as a cell selective...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37732181"
                        open
                      >
                        <summary class="article-expander__title">Long-term longitudinal analysis of 4,187 participants reveals new insights into determinants of incident clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37732181/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230922204741&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH), characterized by blood cells predominantly originating from a single mutated hematopoietic stem cell, is linked to diverse aging-related diseases, including hematologic malignancy and atherosclerotic cardiovascular disease (ASCVD). While CH is common among older adults, the underlying factors driving its development are largely unknown. To address this, we performed whole-exome sequencing on 8,374 blood DNA samples collected from 4,187 Atherosclerosis Risk in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37732224"
                        open
                      >
                        <summary class="article-expander__title">CD45-antibody-drug conjugate clears tissue resident myeloid cells from their niches enabling therapeutic adoptive cell transfer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37732224/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230921204936&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tissue resident myeloid cells (TRM) in adults have highly variable lifespans and may be derived from early embryonic yolk sac, fetal liver or bone marrow. Some of these TRM are known pathogenic participants in congenital and acquired diseases. Myeloablative conditioning and hematopoietic stem cell transplant can replace long-lived brain TRM resulting in clinical improvements in metabolic storage diseases. With the advent of antibody-drug-conjugate (ADC) targeted cell killing as a cell selective...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37732181"
                        open
                      >
                        <summary class="article-expander__title">Long-term longitudinal analysis of 4,187 participants reveals new insights into determinants of incident clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37732181/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230921204936&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH), characterized by blood cells predominantly originating from a single mutated hematopoietic stem cell, is linked to diverse aging-related diseases, including hematologic malignancy and atherosclerotic cardiovascular disease (ASCVD). While CH is common among older adults, the underlying factors driving its development are largely unknown. To address this, we performed whole-exome sequencing on 8,374 blood DNA samples collected from 4,187 Atherosclerosis Risk in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-09-20 datetime="2023-09-20T10:00:00.000Z">2023-09-20</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-09-20 datetime="2023-09-20T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20231105195156">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37729616"
                        open
                      >
                        <summary class="article-expander__title">Impact of Preexisting Autoimmune Disease on Myelodysplastic Syndromes Outcomes: A population Analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37729616/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231004205006&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Preexisting autoimmune disease affects 10-30% of patients with myelodysplastic syndromes (MDS). Studies comparing outcomes in MDS patients with and without autoimmune disease show discordant results. Using Surveillance, Epidemiology, and End Results (SEER)-Medicare, we conducted a population analysis to define the impact of autoimmunity on MDS outcomes. Cases were ascertained between 2007 - 2017 and claim algorithms employed to identify autoimmune disease, demographic characteristics,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37729616"
                        open
                      >
                        <summary class="article-expander__title">Impact of Preexisting Autoimmune Disease on Myelodysplastic Syndromes Outcomes: A population Analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37729616/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231002205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Preexisting autoimmune disease affects 10-30% of patients with myelodysplastic syndromes (MDS). Studies comparing outcomes in MDS patients with and without autoimmune disease show discordant results. Using Surveillance, Epidemiology, and End Results (SEER)-Medicare, we conducted a population analysis to define the impact of autoimmunity on MDS outcomes. Cases were ascertained between 2007 - 2017 and claim algorithms employed to identify autoimmune disease, demographic characteristics,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37729616"
                        open
                      >
                        <summary class="article-expander__title">Impact of Preexisting Autoimmune Disease on Myelodysplastic Syndromes Outcomes: A population Analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37729616/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231001205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Preexisting autoimmune disease affects 10-30% of patients with myelodysplastic syndromes (MDS). Studies comparing outcomes in MDS patients with and without autoimmune disease show discordant results. Using Surveillance, Epidemiology, and End Results (SEER)-Medicare, we conducted a population analysis to define the impact of autoimmunity on MDS outcomes. Cases were ascertained between 2007 - 2017 and claim algorithms employed to identify autoimmune disease, demographic characteristics,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37729414"
                        open
                      >
                        <summary class="article-expander__title">Single-cell genotypic and phenotypic analysis of measurable residual disease in acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37729414/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231001205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Measurable residual disease (MRD), defined as the population of cancer cells that persist following therapy, serves as the critical reservoir for disease relapse in acute myeloid leukemia and other malignancies. Understanding the biology enabling MRD clones to resist therapy is necessary to guide the development of more effective curative treatments. Discriminating between residual leukemic clones, preleukemic clones, and normal precursors remains a challenge with current MRD tools. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37729616"
                        open
                      >
                        <summary class="article-expander__title">Impact of Preexisting Autoimmune Disease on Myelodysplastic Syndromes Outcomes: A population Analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37729616/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230930205630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Preexisting autoimmune disease affects 10-30% of patients with myelodysplastic syndromes (MDS). Studies comparing outcomes in MDS patients with and without autoimmune disease show discordant results. Using Surveillance, Epidemiology, and End Results (SEER)-Medicare, we conducted a population analysis to define the impact of autoimmunity on MDS outcomes. Cases were ascertained between 2007 - 2017 and claim algorithms employed to identify autoimmune disease, demographic characteristics,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37729414"
                        open
                      >
                        <summary class="article-expander__title">Single-cell genotypic and phenotypic analysis of measurable residual disease in acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37729414/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230930205630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Measurable residual disease (MRD), defined as the population of cancer cells that persist following therapy, serves as the critical reservoir for disease relapse in acute myeloid leukemia and other malignancies. Understanding the biology enabling MRD clones to resist therapy is necessary to guide the development of more effective curative treatments. Discriminating between residual leukemic clones, preleukemic clones, and normal precursors remains a challenge with current MRD tools. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37729616"
                        open
                      >
                        <summary class="article-expander__title">Impact of Preexisting Autoimmune Disease on Myelodysplastic Syndromes Outcomes: A population Analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37729616/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230929204823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Preexisting autoimmune disease affects 10-30% of patients with myelodysplastic syndromes (MDS). Studies comparing outcomes in MDS patients with and without autoimmune disease show discordant results. Using Surveillance, Epidemiology, and End Results (SEER)-Medicare, we conducted a population analysis to define the impact of autoimmunity on MDS outcomes. Cases were ascertained between 2007 - 2017 and claim algorithms employed to identify autoimmune disease, demographic characteristics,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37729414"
                        open
                      >
                        <summary class="article-expander__title">Single-cell genotypic and phenotypic analysis of measurable residual disease in acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37729414/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230929204823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Measurable residual disease (MRD), defined as the population of cancer cells that persist following therapy, serves as the critical reservoir for disease relapse in acute myeloid leukemia and other malignancies. Understanding the biology enabling MRD clones to resist therapy is necessary to guide the development of more effective curative treatments. Discriminating between residual leukemic clones, preleukemic clones, and normal precursors remains a challenge with current MRD tools. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37729616"
                        open
                      >
                        <summary class="article-expander__title">Impact of Preexisting Autoimmune Disease on Myelodysplastic Syndromes Outcomes: A population Analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37729616/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230928204947&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Preexisting autoimmune disease affects 10-30% of patients with myelodysplastic syndromes (MDS). Studies comparing outcomes in MDS patients with and without autoimmune disease show discordant results. Using Surveillance, Epidemiology, and End Results (SEER)-Medicare, we conducted a population analysis to define the impact of autoimmunity on MDS outcomes. Cases were ascertained between 2007 - 2017 and claim algorithms employed to identify autoimmune disease, demographic characteristics,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37729414"
                        open
                      >
                        <summary class="article-expander__title">Single-cell genotypic and phenotypic analysis of measurable residual disease in acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37729414/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230928204947&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Measurable residual disease (MRD), defined as the population of cancer cells that persist following therapy, serves as the critical reservoir for disease relapse in acute myeloid leukemia and other malignancies. Understanding the biology enabling MRD clones to resist therapy is necessary to guide the development of more effective curative treatments. Discriminating between residual leukemic clones, preleukemic clones, and normal precursors remains a challenge with current MRD tools. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37729616"
                        open
                      >
                        <summary class="article-expander__title">Impact of Preexisting Autoimmune Disease on Myelodysplastic Syndromes Outcomes: A population Analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37729616/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230927204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Preexisting autoimmune disease affects 10-30% of patients with myelodysplastic syndromes (MDS). Studies comparing outcomes in MDS patients with and without autoimmune disease show discordant results. Using Surveillance, Epidemiology, and End Results (SEER)-Medicare, we conducted a population analysis to define the impact of autoimmunity on MDS outcomes. Cases were ascertained between 2007 - 2017 and claim algorithms employed to identify autoimmune disease, demographic characteristics,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37729414"
                        open
                      >
                        <summary class="article-expander__title">Single-cell genotypic and phenotypic analysis of measurable residual disease in acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37729414/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230927204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Measurable residual disease (MRD), defined as the population of cancer cells that persist following therapy, serves as the critical reservoir for disease relapse in acute myeloid leukemia and other malignancies. Understanding the biology enabling MRD clones to resist therapy is necessary to guide the development of more effective curative treatments. Discriminating between residual leukemic clones, preleukemic clones, and normal precursors remains a challenge with current MRD tools. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37729616"
                        open
                      >
                        <summary class="article-expander__title">Impact of Preexisting Autoimmune Disease on Myelodysplastic Syndromes Outcomes: A population Analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37729616/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230926205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Preexisting autoimmune disease affects 10-30% of patients with myelodysplastic syndromes (MDS). Studies comparing outcomes in MDS patients with and without autoimmune disease show discordant results. Using Surveillance, Epidemiology, and End Results (SEER)-Medicare, we conducted a population analysis to define the impact of autoimmunity on MDS outcomes. Cases were ascertained between 2007 - 2017 and claim algorithms employed to identify autoimmune disease, demographic characteristics,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37729414"
                        open
                      >
                        <summary class="article-expander__title">Single-cell genotypic and phenotypic analysis of measurable residual disease in acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37729414/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230926205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Measurable residual disease (MRD), defined as the population of cancer cells that persist following therapy, serves as the critical reservoir for disease relapse in acute myeloid leukemia and other malignancies. Understanding the biology enabling MRD clones to resist therapy is necessary to guide the development of more effective curative treatments. Discriminating between residual leukemic clones, preleukemic clones, and normal precursors remains a challenge with current MRD tools. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37729616"
                        open
                      >
                        <summary class="article-expander__title">Impact of Preexisting Autoimmune Disease on Myelodysplastic Syndromes Outcomes: A population Analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37729616/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230925205009&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Preexisting autoimmune disease affects 10-30% of patients with myelodysplastic syndromes (MDS). Studies comparing outcomes in MDS patients with and without autoimmune disease show discordant results. Using Surveillance, Epidemiology, and End Results (SEER)-Medicare, we conducted a population analysis to define the impact of autoimmunity on MDS outcomes. Cases were ascertained between 2007 - 2017 and claim algorithms employed to identify autoimmune disease, demographic characteristics,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37729414"
                        open
                      >
                        <summary class="article-expander__title">Single-cell genotypic and phenotypic analysis of measurable residual disease in acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37729414/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230925205009&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Measurable residual disease (MRD), defined as the population of cancer cells that persist following therapy, serves as the critical reservoir for disease relapse in acute myeloid leukemia and other malignancies. Understanding the biology enabling MRD clones to resist therapy is necessary to guide the development of more effective curative treatments. Discriminating between residual leukemic clones, preleukemic clones, and normal precursors remains a challenge with current MRD tools. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37729616"
                        open
                      >
                        <summary class="article-expander__title">Impact of Preexisting Autoimmune Disease on Myelodysplastic Syndromes Outcomes: A population Analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37729616/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230924205014&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Preexisting autoimmune disease affects 10-30% of patients with myelodysplastic syndromes (MDS). Studies comparing outcomes in MDS patients with and without autoimmune disease show discordant results. Using Surveillance, Epidemiology, and End Results (SEER)-Medicare, we conducted a population analysis to define the impact of autoimmunity on MDS outcomes. Cases were ascertained between 2007 - 2017 and claim algorithms employed to identify autoimmune disease, demographic characteristics,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37729414"
                        open
                      >
                        <summary class="article-expander__title">Single-cell genotypic and phenotypic analysis of measurable residual disease in acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37729414/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230924205014&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Measurable residual disease (MRD), defined as the population of cancer cells that persist following therapy, serves as the critical reservoir for disease relapse in acute myeloid leukemia and other malignancies. Understanding the biology enabling MRD clones to resist therapy is necessary to guide the development of more effective curative treatments. Discriminating between residual leukemic clones, preleukemic clones, and normal precursors remains a challenge with current MRD tools. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37729616"
                        open
                      >
                        <summary class="article-expander__title">Impact of Preexisting Autoimmune Disease on Myelodysplastic Syndromes Outcomes: A population Analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37729616/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230923205208&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Preexisting autoimmune disease affects 10-30% of patients with myelodysplastic syndromes (MDS). Studies comparing outcomes in MDS patients with and without autoimmune disease show discordant results. Using Surveillance, Epidemiology, and End Results (SEER)-Medicare, we conducted a population analysis to define the impact of autoimmunity on MDS outcomes. Cases were ascertained between 2007 - 2017 and claim algorithms employed to identify autoimmune disease, demographic characteristics,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37729414"
                        open
                      >
                        <summary class="article-expander__title">Single-cell genotypic and phenotypic analysis of measurable residual disease in acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37729414/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230923205208&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Measurable residual disease (MRD), defined as the population of cancer cells that persist following therapy, serves as the critical reservoir for disease relapse in acute myeloid leukemia and other malignancies. Understanding the biology enabling MRD clones to resist therapy is necessary to guide the development of more effective curative treatments. Discriminating between residual leukemic clones, preleukemic clones, and normal precursors remains a challenge with current MRD tools. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37729616"
                        open
                      >
                        <summary class="article-expander__title">Impact of Preexisting Autoimmune Disease on Myelodysplastic Syndromes Outcomes: A population Analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37729616/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230922204741&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Preexisting autoimmune disease affects 10-30% of patients with myelodysplastic syndromes (MDS). Studies comparing outcomes in MDS patients with and without autoimmune disease show discordant results. Using Surveillance, Epidemiology, and End Results (SEER)-Medicare, we conducted a population analysis to define the impact of autoimmunity on MDS outcomes. Cases were ascertained between 2007 - 2017 and claim algorithms employed to identify autoimmune disease, demographic characteristics,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37729414"
                        open
                      >
                        <summary class="article-expander__title">Single-cell genotypic and phenotypic analysis of measurable residual disease in acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37729414/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230922204741&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Measurable residual disease (MRD), defined as the population of cancer cells that persist following therapy, serves as the critical reservoir for disease relapse in acute myeloid leukemia and other malignancies. Understanding the biology enabling MRD clones to resist therapy is necessary to guide the development of more effective curative treatments. Discriminating between residual leukemic clones, preleukemic clones, and normal precursors remains a challenge with current MRD tools. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37729616"
                        open
                      >
                        <summary class="article-expander__title">Impact of Preexisting Autoimmune Disease on Myelodysplastic Syndromes Outcomes: A population Analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37729616/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230921204936&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Preexisting autoimmune disease affects 10-30% of patients with myelodysplastic syndromes (MDS). Studies comparing outcomes in MDS patients with and without autoimmune disease show discordant results. Using Surveillance, Epidemiology, and End Results (SEER)-Medicare, we conducted a population analysis to define the impact of autoimmunity on MDS outcomes. Cases were ascertained between 2007 - 2017 and claim algorithms employed to identify autoimmune disease, demographic characteristics,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37729414"
                        open
                      >
                        <summary class="article-expander__title">Single-cell genotypic and phenotypic analysis of measurable residual disease in acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37729414/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230921204936&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Measurable residual disease (MRD), defined as the population of cancer cells that persist following therapy, serves as the critical reservoir for disease relapse in acute myeloid leukemia and other malignancies. Understanding the biology enabling MRD clones to resist therapy is necessary to guide the development of more effective curative treatments. Discriminating between residual leukemic clones, preleukemic clones, and normal precursors remains a challenge with current MRD tools. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37729616"
                        open
                      >
                        <summary class="article-expander__title">Impact of Preexisting Autoimmune Disease on Myelodysplastic Syndromes Outcomes: A population Analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37729616/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230920204905&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Preexisting autoimmune disease affects 10-30% of patients with myelodysplastic syndromes (MDS). Studies comparing outcomes in MDS patients with and without autoimmune disease show discordant results. Using Surveillance, Epidemiology, and End Results (SEER)-Medicare, we conducted a population analysis to define the impact of autoimmunity on MDS outcomes. Cases were ascertained between 2007 - 2017 and claim algorithms employed to identify autoimmune disease, demographic characteristics,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37729414"
                        open
                      >
                        <summary class="article-expander__title">Single-cell genotypic and phenotypic analysis of measurable residual disease in acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37729414/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230920204905&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Measurable residual disease (MRD), defined as the population of cancer cells that persist following therapy, serves as the critical reservoir for disease relapse in acute myeloid leukemia and other malignancies. Understanding the biology enabling MRD clones to resist therapy is necessary to guide the development of more effective curative treatments. Discriminating between residual leukemic clones, preleukemic clones, and normal precursors remains a challenge with current MRD tools. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;ff&#x3D;20231105195156&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37730678"
                        open
                      >
                        <summary class="article-expander__title">Large T cell clones expressing immune checkpoints increase during multiple myeloma evolution and predict treatment resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37730678/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231008205021&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tumor recognition by T cells is essential for antitumor immunity. A comprehensive characterization of T cell diversity may be key to understanding the success of immunomodulatory drugs and failure of PD-1 blockade in tumors such as multiple myeloma (MM). Here, we use single-cell RNA and T cell receptor sequencing to characterize bone marrow T cells from healthy adults (n &#x3D; 4) and patients with precursor (n &#x3D; 8) and full-blown MM (n &#x3D; 10). Large T cell clones from patients with MM expressed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37730678"
                        open
                      >
                        <summary class="article-expander__title">Large T cell clones expressing immune checkpoints increase during multiple myeloma evolution and predict treatment resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37730678/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231007205329&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tumor recognition by T cells is essential for antitumor immunity. A comprehensive characterization of T cell diversity may be key to understanding the success of immunomodulatory drugs and failure of PD-1 blockade in tumors such as multiple myeloma (MM). Here, we use single-cell RNA and T cell receptor sequencing to characterize bone marrow T cells from healthy adults (n &#x3D; 4) and patients with precursor (n &#x3D; 8) and full-blown MM (n &#x3D; 10). Large T cell clones from patients with MM expressed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37730678"
                        open
                      >
                        <summary class="article-expander__title">Large T cell clones expressing immune checkpoints increase during multiple myeloma evolution and predict treatment resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37730678/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231006204916&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tumor recognition by T cells is essential for antitumor immunity. A comprehensive characterization of T cell diversity may be key to understanding the success of immunomodulatory drugs and failure of PD-1 blockade in tumors such as multiple myeloma (MM). Here, we use single-cell RNA and T cell receptor sequencing to characterize bone marrow T cells from healthy adults (n &#x3D; 4) and patients with precursor (n &#x3D; 8) and full-blown MM (n &#x3D; 10). Large T cell clones from patients with MM expressed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37730678"
                        open
                      >
                        <summary class="article-expander__title">Large T cell clones expressing immune checkpoints increase during multiple myeloma evolution and predict treatment resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37730678/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231005204957&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tumor recognition by T cells is essential for antitumor immunity. A comprehensive characterization of T cell diversity may be key to understanding the success of immunomodulatory drugs and failure of PD-1 blockade in tumors such as multiple myeloma (MM). Here, we use single-cell RNA and T cell receptor sequencing to characterize bone marrow T cells from healthy adults (n &#x3D; 4) and patients with precursor (n &#x3D; 8) and full-blown MM (n &#x3D; 10). Large T cell clones from patients with MM expressed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37730678"
                        open
                      >
                        <summary class="article-expander__title">Large T cell clones expressing immune checkpoints increase during multiple myeloma evolution and predict treatment resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37730678/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231004205006&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tumor recognition by T cells is essential for antitumor immunity. A comprehensive characterization of T cell diversity may be key to understanding the success of immunomodulatory drugs and failure of PD-1 blockade in tumors such as multiple myeloma (MM). Here, we use single-cell RNA and T cell receptor sequencing to characterize bone marrow T cells from healthy adults (n &#x3D; 4) and patients with precursor (n &#x3D; 8) and full-blown MM (n &#x3D; 10). Large T cell clones from patients with MM expressed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37730678"
                        open
                      >
                        <summary class="article-expander__title">Large T cell clones expressing immune checkpoints increase during multiple myeloma evolution and predict treatment resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37730678/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231003205044&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tumor recognition by T cells is essential for antitumor immunity. A comprehensive characterization of T cell diversity may be key to understanding the success of immunomodulatory drugs and failure of PD-1 blockade in tumors such as multiple myeloma (MM). Here, we use single-cell RNA and T cell receptor sequencing to characterize bone marrow T cells from healthy adults (n &#x3D; 4) and patients with precursor (n &#x3D; 8) and full-blown MM (n &#x3D; 10). Large T cell clones from patients with MM expressed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37730678"
                        open
                      >
                        <summary class="article-expander__title">Large T cell clones expressing immune checkpoints increase during multiple myeloma evolution and predict treatment resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37730678/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231002205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tumor recognition by T cells is essential for antitumor immunity. A comprehensive characterization of T cell diversity may be key to understanding the success of immunomodulatory drugs and failure of PD-1 blockade in tumors such as multiple myeloma (MM). Here, we use single-cell RNA and T cell receptor sequencing to characterize bone marrow T cells from healthy adults (n &#x3D; 4) and patients with precursor (n &#x3D; 8) and full-blown MM (n &#x3D; 10). Large T cell clones from patients with MM expressed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37730678"
                        open
                      >
                        <summary class="article-expander__title">Large T cell clones expressing immune checkpoints increase during multiple myeloma evolution and predict treatment resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37730678/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231001205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tumor recognition by T cells is essential for antitumor immunity. A comprehensive characterization of T cell diversity may be key to understanding the success of immunomodulatory drugs and failure of PD-1 blockade in tumors such as multiple myeloma (MM). Here, we use single-cell RNA and T cell receptor sequencing to characterize bone marrow T cells from healthy adults (n &#x3D; 4) and patients with precursor (n &#x3D; 8) and full-blown MM (n &#x3D; 10). Large T cell clones from patients with MM expressed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37730678"
                        open
                      >
                        <summary class="article-expander__title">Large T cell clones expressing immune checkpoints increase during multiple myeloma evolution and predict treatment resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37730678/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230930205630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tumor recognition by T cells is essential for antitumor immunity. A comprehensive characterization of T cell diversity may be key to understanding the success of immunomodulatory drugs and failure of PD-1 blockade in tumors such as multiple myeloma (MM). Here, we use single-cell RNA and T cell receptor sequencing to characterize bone marrow T cells from healthy adults (n &#x3D; 4) and patients with precursor (n &#x3D; 8) and full-blown MM (n &#x3D; 10). Large T cell clones from patients with MM expressed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37730678"
                        open
                      >
                        <summary class="article-expander__title">Large T cell clones expressing immune checkpoints increase during multiple myeloma evolution and predict treatment resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37730678/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230929204823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tumor recognition by T cells is essential for antitumor immunity. A comprehensive characterization of T cell diversity may be key to understanding the success of immunomodulatory drugs and failure of PD-1 blockade in tumors such as multiple myeloma (MM). Here, we use single-cell RNA and T cell receptor sequencing to characterize bone marrow T cells from healthy adults (n &#x3D; 4) and patients with precursor (n &#x3D; 8) and full-blown MM (n &#x3D; 10). Large T cell clones from patients with MM expressed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37730678"
                        open
                      >
                        <summary class="article-expander__title">Large T cell clones expressing immune checkpoints increase during multiple myeloma evolution and predict treatment resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37730678/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230928204948&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tumor recognition by T cells is essential for antitumor immunity. A comprehensive characterization of T cell diversity may be key to understanding the success of immunomodulatory drugs and failure of PD-1 blockade in tumors such as multiple myeloma (MM). Here, we use single-cell RNA and T cell receptor sequencing to characterize bone marrow T cells from healthy adults (n &#x3D; 4) and patients with precursor (n &#x3D; 8) and full-blown MM (n &#x3D; 10). Large T cell clones from patients with MM expressed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37730678"
                        open
                      >
                        <summary class="article-expander__title">Large T cell clones expressing immune checkpoints increase during multiple myeloma evolution and predict treatment resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37730678/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230927204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tumor recognition by T cells is essential for antitumor immunity. A comprehensive characterization of T cell diversity may be key to understanding the success of immunomodulatory drugs and failure of PD-1 blockade in tumors such as multiple myeloma (MM). Here, we use single-cell RNA and T cell receptor sequencing to characterize bone marrow T cells from healthy adults (n &#x3D; 4) and patients with precursor (n &#x3D; 8) and full-blown MM (n &#x3D; 10). Large T cell clones from patients with MM expressed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37730678"
                        open
                      >
                        <summary class="article-expander__title">Large T cell clones expressing immune checkpoints increase during multiple myeloma evolution and predict treatment resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37730678/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230926205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tumor recognition by T cells is essential for antitumor immunity. A comprehensive characterization of T cell diversity may be key to understanding the success of immunomodulatory drugs and failure of PD-1 blockade in tumors such as multiple myeloma (MM). Here, we use single-cell RNA and T cell receptor sequencing to characterize bone marrow T cells from healthy adults (n &#x3D; 4) and patients with precursor (n &#x3D; 8) and full-blown MM (n &#x3D; 10). Large T cell clones from patients with MM expressed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37730678"
                        open
                      >
                        <summary class="article-expander__title">Large T cell clones expressing immune checkpoints increase during multiple myeloma evolution and predict treatment resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37730678/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230925205014&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tumor recognition by T cells is essential for antitumor immunity. A comprehensive characterization of T cell diversity may be key to understanding the success of immunomodulatory drugs and failure of PD-1 blockade in tumors such as multiple myeloma (MM). Here, we use single-cell RNA and T cell receptor sequencing to characterize bone marrow T cells from healthy adults (n &#x3D; 4) and patients with precursor (n &#x3D; 8) and full-blown MM (n &#x3D; 10). Large T cell clones from patients with MM expressed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37730678"
                        open
                      >
                        <summary class="article-expander__title">Large T cell clones expressing immune checkpoints increase during multiple myeloma evolution and predict treatment resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37730678/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230924205014&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tumor recognition by T cells is essential for antitumor immunity. A comprehensive characterization of T cell diversity may be key to understanding the success of immunomodulatory drugs and failure of PD-1 blockade in tumors such as multiple myeloma (MM). Here, we use single-cell RNA and T cell receptor sequencing to characterize bone marrow T cells from healthy adults (n &#x3D; 4) and patients with precursor (n &#x3D; 8) and full-blown MM (n &#x3D; 10). Large T cell clones from patients with MM expressed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37730678"
                        open
                      >
                        <summary class="article-expander__title">Large T cell clones expressing immune checkpoints increase during multiple myeloma evolution and predict treatment resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37730678/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230923205208&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tumor recognition by T cells is essential for antitumor immunity. A comprehensive characterization of T cell diversity may be key to understanding the success of immunomodulatory drugs and failure of PD-1 blockade in tumors such as multiple myeloma (MM). Here, we use single-cell RNA and T cell receptor sequencing to characterize bone marrow T cells from healthy adults (n &#x3D; 4) and patients with precursor (n &#x3D; 8) and full-blown MM (n &#x3D; 10). Large T cell clones from patients with MM expressed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37730678"
                        open
                      >
                        <summary class="article-expander__title">Large T cell clones expressing immune checkpoints increase during multiple myeloma evolution and predict treatment resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37730678/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230922204741&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tumor recognition by T cells is essential for antitumor immunity. A comprehensive characterization of T cell diversity may be key to understanding the success of immunomodulatory drugs and failure of PD-1 blockade in tumors such as multiple myeloma (MM). Here, we use single-cell RNA and T cell receptor sequencing to characterize bone marrow T cells from healthy adults (n &#x3D; 4) and patients with precursor (n &#x3D; 8) and full-blown MM (n &#x3D; 10). Large T cell clones from patients with MM expressed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37730678"
                        open
                      >
                        <summary class="article-expander__title">Large T cell clones expressing immune checkpoints increase during multiple myeloma evolution and predict treatment resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37730678/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230921204936&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Tumor recognition by T cells is essential for antitumor immunity. A comprehensive characterization of T cell diversity may be key to understanding the success of immunomodulatory drugs and failure of PD-1 blockade in tumors such as multiple myeloma (MM). Here, we use single-cell RNA and T cell receptor sequencing to characterize bone marrow T cells from healthy adults (n &#x3D; 4) and patients with precursor (n &#x3D; 8) and full-blown MM (n &#x3D; 10). Large T cell clones from patients with MM expressed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-09-19 datetime="2023-09-19T10:00:00.000Z">2023-09-19</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-09-19 datetime="2023-09-19T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;ff&#x3D;20231105195156&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37726835"
                        open
                      >
                        <summary class="article-expander__title">The heterogeneity and clonal evolution analysis of the advanced prostate cancer with castration resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37726835/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231006204916&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our findings revealed potential genetic regulators to predict the castration resistance and provide insights into the castration resistance processes in advanced prostate cancer. The crosstalk between clonal evolution patterns and tumor microenvironment may also play a role in castration resistance. A multicenter-research including larger populations with different background are needed to confirm our conclusion in the future.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37725985"
                        open
                      >
                        <summary class="article-expander__title">Microfluidic design in single-cell sequencing and application to cancer precision medicine</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37725985/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231006204916&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Single-cell sequencing (SCS) is a crucial tool to reveal the genetic and functional heterogeneity of tumors, providing unique insights into the clonal evolution, microenvironment, drug resistance, and metastatic progression of cancers. Microfluidics is a critical component of many SCS technologies and workflows, conferring advantages in throughput, economy, and automation. Here, we review the current landscape of microfluidic architectures and sequencing techniques for single-cell omics analysis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37726835"
                        open
                      >
                        <summary class="article-expander__title">The heterogeneity and clonal evolution analysis of the advanced prostate cancer with castration resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37726835/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231005204957&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our findings revealed potential genetic regulators to predict the castration resistance and provide insights into the castration resistance processes in advanced prostate cancer. The crosstalk between clonal evolution patterns and tumor microenvironment may also play a role in castration resistance. A multicenter-research including larger populations with different background are needed to confirm our conclusion in the future.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37725985"
                        open
                      >
                        <summary class="article-expander__title">Microfluidic design in single-cell sequencing and application to cancer precision medicine</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37725985/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231005204957&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Single-cell sequencing (SCS) is a crucial tool to reveal the genetic and functional heterogeneity of tumors, providing unique insights into the clonal evolution, microenvironment, drug resistance, and metastatic progression of cancers. Microfluidics is a critical component of many SCS technologies and workflows, conferring advantages in throughput, economy, and automation. Here, we review the current landscape of microfluidic architectures and sequencing techniques for single-cell omics analysis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37726835"
                        open
                      >
                        <summary class="article-expander__title">The heterogeneity and clonal evolution analysis of the advanced prostate cancer with castration resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37726835/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231004205006&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our findings revealed potential genetic regulators to predict the castration resistance and provide insights into the castration resistance processes in advanced prostate cancer. The crosstalk between clonal evolution patterns and tumor microenvironment may also play a role in castration resistance. A multicenter-research including larger populations with different background are needed to confirm our conclusion in the future.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37725985"
                        open
                      >
                        <summary class="article-expander__title">Microfluidic design in single-cell sequencing and application to cancer precision medicine</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37725985/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231004205006&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Single-cell sequencing (SCS) is a crucial tool to reveal the genetic and functional heterogeneity of tumors, providing unique insights into the clonal evolution, microenvironment, drug resistance, and metastatic progression of cancers. Microfluidics is a critical component of many SCS technologies and workflows, conferring advantages in throughput, economy, and automation. Here, we review the current landscape of microfluidic architectures and sequencing techniques for single-cell omics analysis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37726835"
                        open
                      >
                        <summary class="article-expander__title">The heterogeneity and clonal evolution analysis of the advanced prostate cancer with castration resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37726835/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231003205044&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our findings revealed potential genetic regulators to predict the castration resistance and provide insights into the castration resistance processes in advanced prostate cancer. The crosstalk between clonal evolution patterns and tumor microenvironment may also play a role in castration resistance. A multicenter-research including larger populations with different background are needed to confirm our conclusion in the future.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37725985"
                        open
                      >
                        <summary class="article-expander__title">Microfluidic design in single-cell sequencing and application to cancer precision medicine</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37725985/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231003205044&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Single-cell sequencing (SCS) is a crucial tool to reveal the genetic and functional heterogeneity of tumors, providing unique insights into the clonal evolution, microenvironment, drug resistance, and metastatic progression of cancers. Microfluidics is a critical component of many SCS technologies and workflows, conferring advantages in throughput, economy, and automation. Here, we review the current landscape of microfluidic architectures and sequencing techniques for single-cell omics analysis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37726835"
                        open
                      >
                        <summary class="article-expander__title">The heterogeneity and clonal evolution analysis of the advanced prostate cancer with castration resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37726835/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231002205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our findings revealed potential genetic regulators to predict the castration resistance and provide insights into the castration resistance processes in advanced prostate cancer. The crosstalk between clonal evolution patterns and tumor microenvironment may also play a role in castration resistance. A multicenter-research including larger populations with different background are needed to confirm our conclusion in the future.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37725985"
                        open
                      >
                        <summary class="article-expander__title">Microfluidic design in single-cell sequencing and application to cancer precision medicine</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37725985/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231002205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Single-cell sequencing (SCS) is a crucial tool to reveal the genetic and functional heterogeneity of tumors, providing unique insights into the clonal evolution, microenvironment, drug resistance, and metastatic progression of cancers. Microfluidics is a critical component of many SCS technologies and workflows, conferring advantages in throughput, economy, and automation. Here, we review the current landscape of microfluidic architectures and sequencing techniques for single-cell omics analysis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37726835"
                        open
                      >
                        <summary class="article-expander__title">The heterogeneity and clonal evolution analysis of the advanced prostate cancer with castration resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37726835/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231001205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our findings revealed potential genetic regulators to predict the castration resistance and provide insights into the castration resistance processes in advanced prostate cancer. The crosstalk between clonal evolution patterns and tumor microenvironment may also play a role in castration resistance. A multicenter-research including larger populations with different background are needed to confirm our conclusion in the future.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37725985"
                        open
                      >
                        <summary class="article-expander__title">Microfluidic design in single-cell sequencing and application to cancer precision medicine</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37725985/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231001205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Single-cell sequencing (SCS) is a crucial tool to reveal the genetic and functional heterogeneity of tumors, providing unique insights into the clonal evolution, microenvironment, drug resistance, and metastatic progression of cancers. Microfluidics is a critical component of many SCS technologies and workflows, conferring advantages in throughput, economy, and automation. Here, we review the current landscape of microfluidic architectures and sequencing techniques for single-cell omics analysis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37726835"
                        open
                      >
                        <summary class="article-expander__title">The heterogeneity and clonal evolution analysis of the advanced prostate cancer with castration resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37726835/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230930205630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our findings revealed potential genetic regulators to predict the castration resistance and provide insights into the castration resistance processes in advanced prostate cancer. The crosstalk between clonal evolution patterns and tumor microenvironment may also play a role in castration resistance. A multicenter-research including larger populations with different background are needed to confirm our conclusion in the future.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37725985"
                        open
                      >
                        <summary class="article-expander__title">Microfluidic design in single-cell sequencing and application to cancer precision medicine</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37725985/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230930205630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Single-cell sequencing (SCS) is a crucial tool to reveal the genetic and functional heterogeneity of tumors, providing unique insights into the clonal evolution, microenvironment, drug resistance, and metastatic progression of cancers. Microfluidics is a critical component of many SCS technologies and workflows, conferring advantages in throughput, economy, and automation. Here, we review the current landscape of microfluidic architectures and sequencing techniques for single-cell omics analysis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37726835"
                        open
                      >
                        <summary class="article-expander__title">The heterogeneity and clonal evolution analysis of the advanced prostate cancer with castration resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37726835/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230929204823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our findings revealed potential genetic regulators to predict the castration resistance and provide insights into the castration resistance processes in advanced prostate cancer. The crosstalk between clonal evolution patterns and tumor microenvironment may also play a role in castration resistance. A multicenter-research including larger populations with different background are needed to confirm our conclusion in the future.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37725985"
                        open
                      >
                        <summary class="article-expander__title">Microfluidic design in single-cell sequencing and application to cancer precision medicine</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37725985/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230929204823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Single-cell sequencing (SCS) is a crucial tool to reveal the genetic and functional heterogeneity of tumors, providing unique insights into the clonal evolution, microenvironment, drug resistance, and metastatic progression of cancers. Microfluidics is a critical component of many SCS technologies and workflows, conferring advantages in throughput, economy, and automation. Here, we review the current landscape of microfluidic architectures and sequencing techniques for single-cell omics analysis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37726835"
                        open
                      >
                        <summary class="article-expander__title">The heterogeneity and clonal evolution analysis of the advanced prostate cancer with castration resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37726835/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230928204948&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our findings revealed potential genetic regulators to predict the castration resistance and provide insights into the castration resistance processes in advanced prostate cancer. The crosstalk between clonal evolution patterns and tumor microenvironment may also play a role in castration resistance. A multicenter-research including larger populations with different background are needed to confirm our conclusion in the future.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37725985"
                        open
                      >
                        <summary class="article-expander__title">Microfluidic design in single-cell sequencing and application to cancer precision medicine</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37725985/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230928204948&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Single-cell sequencing (SCS) is a crucial tool to reveal the genetic and functional heterogeneity of tumors, providing unique insights into the clonal evolution, microenvironment, drug resistance, and metastatic progression of cancers. Microfluidics is a critical component of many SCS technologies and workflows, conferring advantages in throughput, economy, and automation. Here, we review the current landscape of microfluidic architectures and sequencing techniques for single-cell omics analysis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37726835"
                        open
                      >
                        <summary class="article-expander__title">The heterogeneity and clonal evolution analysis of the advanced prostate cancer with castration resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37726835/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230927204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our findings revealed potential genetic regulators to predict the castration resistance and provide insights into the castration resistance processes in advanced prostate cancer. The crosstalk between clonal evolution patterns and tumor microenvironment may also play a role in castration resistance. A multicenter-research including larger populations with different background are needed to confirm our conclusion in the future.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37725985"
                        open
                      >
                        <summary class="article-expander__title">Microfluidic design in single-cell sequencing and application to cancer precision medicine</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37725985/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230927204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Single-cell sequencing (SCS) is a crucial tool to reveal the genetic and functional heterogeneity of tumors, providing unique insights into the clonal evolution, microenvironment, drug resistance, and metastatic progression of cancers. Microfluidics is a critical component of many SCS technologies and workflows, conferring advantages in throughput, economy, and automation. Here, we review the current landscape of microfluidic architectures and sequencing techniques for single-cell omics analysis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37726835"
                        open
                      >
                        <summary class="article-expander__title">The heterogeneity and clonal evolution analysis of the advanced prostate cancer with castration resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37726835/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230926205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our findings revealed potential genetic regulators to predict the castration resistance and provide insights into the castration resistance processes in advanced prostate cancer. The crosstalk between clonal evolution patterns and tumor microenvironment may also play a role in castration resistance. A multicenter-research including larger populations with different background are needed to confirm our conclusion in the future.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37725985"
                        open
                      >
                        <summary class="article-expander__title">Microfluidic design in single-cell sequencing and application to cancer precision medicine</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37725985/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230926205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Single-cell sequencing (SCS) is a crucial tool to reveal the genetic and functional heterogeneity of tumors, providing unique insights into the clonal evolution, microenvironment, drug resistance, and metastatic progression of cancers. Microfluidics is a critical component of many SCS technologies and workflows, conferring advantages in throughput, economy, and automation. Here, we review the current landscape of microfluidic architectures and sequencing techniques for single-cell omics analysis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37726835"
                        open
                      >
                        <summary class="article-expander__title">The heterogeneity and clonal evolution analysis of the advanced prostate cancer with castration resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37726835/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230925205014&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our findings revealed potential genetic regulators to predict the castration resistance and provide insights into the castration resistance processes in advanced prostate cancer. The crosstalk between clonal evolution patterns and tumor microenvironment may also play a role in castration resistance. A multicenter-research including larger populations with different background are needed to confirm our conclusion in the future.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37725985"
                        open
                      >
                        <summary class="article-expander__title">Microfluidic design in single-cell sequencing and application to cancer precision medicine</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37725985/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230925205014&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Single-cell sequencing (SCS) is a crucial tool to reveal the genetic and functional heterogeneity of tumors, providing unique insights into the clonal evolution, microenvironment, drug resistance, and metastatic progression of cancers. Microfluidics is a critical component of many SCS technologies and workflows, conferring advantages in throughput, economy, and automation. Here, we review the current landscape of microfluidic architectures and sequencing techniques for single-cell omics analysis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37726835"
                        open
                      >
                        <summary class="article-expander__title">The heterogeneity and clonal evolution analysis of the advanced prostate cancer with castration resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37726835/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230924205014&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our findings revealed potential genetic regulators to predict the castration resistance and provide insights into the castration resistance processes in advanced prostate cancer. The crosstalk between clonal evolution patterns and tumor microenvironment may also play a role in castration resistance. A multicenter-research including larger populations with different background are needed to confirm our conclusion in the future.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37725985"
                        open
                      >
                        <summary class="article-expander__title">Microfluidic design in single-cell sequencing and application to cancer precision medicine</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37725985/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230924205014&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Single-cell sequencing (SCS) is a crucial tool to reveal the genetic and functional heterogeneity of tumors, providing unique insights into the clonal evolution, microenvironment, drug resistance, and metastatic progression of cancers. Microfluidics is a critical component of many SCS technologies and workflows, conferring advantages in throughput, economy, and automation. Here, we review the current landscape of microfluidic architectures and sequencing techniques for single-cell omics analysis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37726835"
                        open
                      >
                        <summary class="article-expander__title">The heterogeneity and clonal evolution analysis of the advanced prostate cancer with castration resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37726835/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230923205208&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our findings revealed potential genetic regulators to predict the castration resistance and provide insights into the castration resistance processes in advanced prostate cancer. The crosstalk between clonal evolution patterns and tumor microenvironment may also play a role in castration resistance. A multicenter-research including larger populations with different background are needed to confirm our conclusion in the future.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37725985"
                        open
                      >
                        <summary class="article-expander__title">Microfluidic design in single-cell sequencing and application to cancer precision medicine</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37725985/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230923205208&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Single-cell sequencing (SCS) is a crucial tool to reveal the genetic and functional heterogeneity of tumors, providing unique insights into the clonal evolution, microenvironment, drug resistance, and metastatic progression of cancers. Microfluidics is a critical component of many SCS technologies and workflows, conferring advantages in throughput, economy, and automation. Here, we review the current landscape of microfluidic architectures and sequencing techniques for single-cell omics analysis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37726835"
                        open
                      >
                        <summary class="article-expander__title">The heterogeneity and clonal evolution analysis of the advanced prostate cancer with castration resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37726835/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230922204741&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our findings revealed potential genetic regulators to predict the castration resistance and provide insights into the castration resistance processes in advanced prostate cancer. The crosstalk between clonal evolution patterns and tumor microenvironment may also play a role in castration resistance. A multicenter-research including larger populations with different background are needed to confirm our conclusion in the future.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37725985"
                        open
                      >
                        <summary class="article-expander__title">Microfluidic design in single-cell sequencing and application to cancer precision medicine</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37725985/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230922204741&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Single-cell sequencing (SCS) is a crucial tool to reveal the genetic and functional heterogeneity of tumors, providing unique insights into the clonal evolution, microenvironment, drug resistance, and metastatic progression of cancers. Microfluidics is a critical component of many SCS technologies and workflows, conferring advantages in throughput, economy, and automation. Here, we review the current landscape of microfluidic architectures and sequencing techniques for single-cell omics analysis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37726835"
                        open
                      >
                        <summary class="article-expander__title">The heterogeneity and clonal evolution analysis of the advanced prostate cancer with castration resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37726835/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230921204936&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our findings revealed potential genetic regulators to predict the castration resistance and provide insights into the castration resistance processes in advanced prostate cancer. The crosstalk between clonal evolution patterns and tumor microenvironment may also play a role in castration resistance. A multicenter-research including larger populations with different background are needed to confirm our conclusion in the future.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37725985"
                        open
                      >
                        <summary class="article-expander__title">Microfluidic design in single-cell sequencing and application to cancer precision medicine</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37725985/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230921204936&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Single-cell sequencing (SCS) is a crucial tool to reveal the genetic and functional heterogeneity of tumors, providing unique insights into the clonal evolution, microenvironment, drug resistance, and metastatic progression of cancers. Microfluidics is a critical component of many SCS technologies and workflows, conferring advantages in throughput, economy, and automation. Here, we review the current landscape of microfluidic architectures and sequencing techniques for single-cell omics analysis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37726835"
                        open
                      >
                        <summary class="article-expander__title">The heterogeneity and clonal evolution analysis of the advanced prostate cancer with castration resistance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37726835/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230920204904&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our findings revealed potential genetic regulators to predict the castration resistance and provide insights into the castration resistance processes in advanced prostate cancer. The crosstalk between clonal evolution patterns and tumor microenvironment may also play a role in castration resistance. A multicenter-research including larger populations with different background are needed to confirm our conclusion in the future.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37725985"
                        open
                      >
                        <summary class="article-expander__title">Microfluidic design in single-cell sequencing and application to cancer precision medicine</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37725985/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230920204904&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Single-cell sequencing (SCS) is a crucial tool to reveal the genetic and functional heterogeneity of tumors, providing unique insights into the clonal evolution, microenvironment, drug resistance, and metastatic progression of cancers. Microfluidics is a critical component of many SCS technologies and workflows, conferring advantages in throughput, economy, and automation. Here, we review the current landscape of microfluidic architectures and sequencing techniques for single-cell omics analysis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-09-17 datetime="2023-09-17T10:00:00.000Z">2023-09-17</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-09-17 datetime="2023-09-17T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;ff&#x3D;20231105195156&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37717855"
                        open
                      >
                        <summary class="article-expander__title">Genetic intra-tumour heterogeneity remodels the immune microenvironment and induces immune evasion in brain metastasis of lung cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37717855/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231005204957&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our findings provide novel insights into the metastatic process and immune escape mechanisms of brain metastasis and pave the way for precise immunotherapeutic strategies for lung cancer patients with brain metastasis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37717855"
                        open
                      >
                        <summary class="article-expander__title">Genetic intra-tumour heterogeneity remodels the immune microenvironment and induces immune evasion in brain metastasis of lung cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37717855/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231004205006&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our findings provide novel insights into the metastatic process and immune escape mechanisms of brain metastasis and pave the way for precise immunotherapeutic strategies for lung cancer patients with brain metastasis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37717855"
                        open
                      >
                        <summary class="article-expander__title">Genetic intra-tumour heterogeneity remodels the immune microenvironment and induces immune evasion in brain metastasis of lung cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37717855/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231003205044&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our findings provide novel insights into the metastatic process and immune escape mechanisms of brain metastasis and pave the way for precise immunotherapeutic strategies for lung cancer patients with brain metastasis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37717855"
                        open
                      >
                        <summary class="article-expander__title">Genetic intra-tumour heterogeneity remodels the immune microenvironment and induces immune evasion in brain metastasis of lung cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37717855/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231002205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our findings provide novel insights into the metastatic process and immune escape mechanisms of brain metastasis and pave the way for precise immunotherapeutic strategies for lung cancer patients with brain metastasis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37717855"
                        open
                      >
                        <summary class="article-expander__title">Genetic intra-tumour heterogeneity remodels the immune microenvironment and induces immune evasion in brain metastasis of lung cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37717855/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231001205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our findings provide novel insights into the metastatic process and immune escape mechanisms of brain metastasis and pave the way for precise immunotherapeutic strategies for lung cancer patients with brain metastasis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37717855"
                        open
                      >
                        <summary class="article-expander__title">Genetic intra-tumour heterogeneity remodels the immune microenvironment and induces immune evasion in brain metastasis of lung cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37717855/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230930205630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our findings provide novel insights into the metastatic process and immune escape mechanisms of brain metastasis and pave the way for precise immunotherapeutic strategies for lung cancer patients with brain metastasis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37717855"
                        open
                      >
                        <summary class="article-expander__title">Genetic intra-tumour heterogeneity remodels the immune microenvironment and induces immune evasion in brain metastasis of lung cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37717855/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230929204823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our findings provide novel insights into the metastatic process and immune escape mechanisms of brain metastasis and pave the way for precise immunotherapeutic strategies for lung cancer patients with brain metastasis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37717855"
                        open
                      >
                        <summary class="article-expander__title">Genetic intra-tumour heterogeneity remodels the immune microenvironment and induces immune evasion in brain metastasis of lung cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37717855/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230928204948&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our findings provide novel insights into the metastatic process and immune escape mechanisms of brain metastasis and pave the way for precise immunotherapeutic strategies for lung cancer patients with brain metastasis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37717855"
                        open
                      >
                        <summary class="article-expander__title">Genetic intra-tumour heterogeneity remodels the immune microenvironment and induces immune evasion in brain metastasis of lung cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37717855/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230927204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our findings provide novel insights into the metastatic process and immune escape mechanisms of brain metastasis and pave the way for precise immunotherapeutic strategies for lung cancer patients with brain metastasis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37717855"
                        open
                      >
                        <summary class="article-expander__title">Genetic intra-tumour heterogeneity remodels the immune microenvironment and induces immune evasion in brain metastasis of lung cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37717855/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230926205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our findings provide novel insights into the metastatic process and immune escape mechanisms of brain metastasis and pave the way for precise immunotherapeutic strategies for lung cancer patients with brain metastasis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37717855"
                        open
                      >
                        <summary class="article-expander__title">Genetic intra-tumour heterogeneity remodels the immune microenvironment and induces immune evasion in brain metastasis of lung cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37717855/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230925205014&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our findings provide novel insights into the metastatic process and immune escape mechanisms of brain metastasis and pave the way for precise immunotherapeutic strategies for lung cancer patients with brain metastasis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37717855"
                        open
                      >
                        <summary class="article-expander__title">Genetic intra-tumour heterogeneity remodels the immune microenvironment and induces immune evasion in brain metastasis of lung cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37717855/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230924205014&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our findings provide novel insights into the metastatic process and immune escape mechanisms of brain metastasis and pave the way for precise immunotherapeutic strategies for lung cancer patients with brain metastasis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37717855"
                        open
                      >
                        <summary class="article-expander__title">Genetic intra-tumour heterogeneity remodels the immune microenvironment and induces immune evasion in brain metastasis of lung cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37717855/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230923205208&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our findings provide novel insights into the metastatic process and immune escape mechanisms of brain metastasis and pave the way for precise immunotherapeutic strategies for lung cancer patients with brain metastasis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37717855"
                        open
                      >
                        <summary class="article-expander__title">Genetic intra-tumour heterogeneity remodels the immune microenvironment and induces immune evasion in brain metastasis of lung cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37717855/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230922204741&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our findings provide novel insights into the metastatic process and immune escape mechanisms of brain metastasis and pave the way for precise immunotherapeutic strategies for lung cancer patients with brain metastasis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37717855"
                        open
                      >
                        <summary class="article-expander__title">Genetic intra-tumour heterogeneity remodels the immune microenvironment and induces immune evasion in brain metastasis of lung cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37717855/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230921204936&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our findings provide novel insights into the metastatic process and immune escape mechanisms of brain metastasis and pave the way for precise immunotherapeutic strategies for lung cancer patients with brain metastasis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37717855"
                        open
                      >
                        <summary class="article-expander__title">Genetic intra-tumour heterogeneity remodels the immune microenvironment and induces immune evasion in brain metastasis of lung cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37717855/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230920204904&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our findings provide novel insights into the metastatic process and immune escape mechanisms of brain metastasis and pave the way for precise immunotherapeutic strategies for lung cancer patients with brain metastasis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37717855"
                        open
                      >
                        <summary class="article-expander__title">Genetic intra-tumour heterogeneity remodels the immune microenvironment and induces immune evasion in brain metastasis of lung cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37717855/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230919204929&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our findings provide novel insights into the metastatic process and immune escape mechanisms of brain metastasis and pave the way for precise immunotherapeutic strategies for lung cancer patients with brain metastasis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37717855"
                        open
                      >
                        <summary class="article-expander__title">Genetic intra-tumour heterogeneity remodels the immune microenvironment and induces immune evasion in brain metastasis of lung cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37717855/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230918204942&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our findings provide novel insights into the metastatic process and immune escape mechanisms of brain metastasis and pave the way for precise immunotherapeutic strategies for lung cancer patients with brain metastasis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-09-16 datetime="2023-09-16T10:00:00.000Z">2023-09-16</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-09-16 datetime="2023-09-16T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20231105195156">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37716836"
                        open
                      >
                        <summary class="article-expander__title">An Update on Inflammation in Atherosclerosis: How to Effectively Treat Residual Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37716836/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230930205630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE: This study reviewed the contribution of inflammation to atherosclerotic cardiovascular disease (ASCVD), which has gained widespread recognition in recent years.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37716836"
                        open
                      >
                        <summary class="article-expander__title">An Update on Inflammation in Atherosclerosis: How to Effectively Treat Residual Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37716836/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230929204823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE: This study reviewed the contribution of inflammation to atherosclerotic cardiovascular disease (ASCVD), which has gained widespread recognition in recent years.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37716836"
                        open
                      >
                        <summary class="article-expander__title">An Update on Inflammation in Atherosclerosis: How to Effectively Treat Residual Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37716836/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230928204947&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE: This study reviewed the contribution of inflammation to atherosclerotic cardiovascular disease (ASCVD), which has gained widespread recognition in recent years.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37715966"
                        open
                      >
                        <summary class="article-expander__title">Interpretation of ambiguous TP53 test results: Mosaicism, clonal hematopoiesis, and variants of uncertain significance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37715966/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230928204947&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The increased use of next-generation sequencing has led to the detection of pathogenic TP53 variants in the germline setting in patients without a personal or family history consistent with Li-Fraumeni syndrome (LFS). These variants can represent low-penetrance LFS, mosaic LFS, or clonal hematopoiesis of indeterminate potential. Additionally, TP53 variants of uncertain significance can be detected in patients with a history suspicious for LFS. The interpretation of the significance of these...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37716836"
                        open
                      >
                        <summary class="article-expander__title">An Update on Inflammation in Atherosclerosis: How to Effectively Treat Residual Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37716836/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230927204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE: This study reviewed the contribution of inflammation to atherosclerotic cardiovascular disease (ASCVD), which has gained widespread recognition in recent years.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37715966"
                        open
                      >
                        <summary class="article-expander__title">Interpretation of ambiguous TP53 test results: Mosaicism, clonal hematopoiesis, and variants of uncertain significance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37715966/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230927204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The increased use of next-generation sequencing has led to the detection of pathogenic TP53 variants in the germline setting in patients without a personal or family history consistent with Li-Fraumeni syndrome (LFS). These variants can represent low-penetrance LFS, mosaic LFS, or clonal hematopoiesis of indeterminate potential. Additionally, TP53 variants of uncertain significance can be detected in patients with a history suspicious for LFS. The interpretation of the significance of these...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37716836"
                        open
                      >
                        <summary class="article-expander__title">An Update on Inflammation in Atherosclerosis: How to Effectively Treat Residual Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37716836/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230926205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE: This study reviewed the contribution of inflammation to atherosclerotic cardiovascular disease (ASCVD), which has gained widespread recognition in recent years.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37715966"
                        open
                      >
                        <summary class="article-expander__title">Interpretation of ambiguous TP53 test results: Mosaicism, clonal hematopoiesis, and variants of uncertain significance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37715966/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230926205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The increased use of next-generation sequencing has led to the detection of pathogenic TP53 variants in the germline setting in patients without a personal or family history consistent with Li-Fraumeni syndrome (LFS). These variants can represent low-penetrance LFS, mosaic LFS, or clonal hematopoiesis of indeterminate potential. Additionally, TP53 variants of uncertain significance can be detected in patients with a history suspicious for LFS. The interpretation of the significance of these...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37716836"
                        open
                      >
                        <summary class="article-expander__title">An Update on Inflammation in Atherosclerosis: How to Effectively Treat Residual Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37716836/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230925205009&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE: This study reviewed the contribution of inflammation to atherosclerotic cardiovascular disease (ASCVD), which has gained widespread recognition in recent years.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37715966"
                        open
                      >
                        <summary class="article-expander__title">Interpretation of ambiguous TP53 test results: Mosaicism, clonal hematopoiesis, and variants of uncertain significance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37715966/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230925205009&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The increased use of next-generation sequencing has led to the detection of pathogenic TP53 variants in the germline setting in patients without a personal or family history consistent with Li-Fraumeni syndrome (LFS). These variants can represent low-penetrance LFS, mosaic LFS, or clonal hematopoiesis of indeterminate potential. Additionally, TP53 variants of uncertain significance can be detected in patients with a history suspicious for LFS. The interpretation of the significance of these...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37716836"
                        open
                      >
                        <summary class="article-expander__title">An Update on Inflammation in Atherosclerosis: How to Effectively Treat Residual Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37716836/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230924205014&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE: This study reviewed the contribution of inflammation to atherosclerotic cardiovascular disease (ASCVD), which has gained widespread recognition in recent years.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37715966"
                        open
                      >
                        <summary class="article-expander__title">Interpretation of ambiguous TP53 test results: Mosaicism, clonal hematopoiesis, and variants of uncertain significance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37715966/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230924205014&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The increased use of next-generation sequencing has led to the detection of pathogenic TP53 variants in the germline setting in patients without a personal or family history consistent with Li-Fraumeni syndrome (LFS). These variants can represent low-penetrance LFS, mosaic LFS, or clonal hematopoiesis of indeterminate potential. Additionally, TP53 variants of uncertain significance can be detected in patients with a history suspicious for LFS. The interpretation of the significance of these...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37716836"
                        open
                      >
                        <summary class="article-expander__title">An Update on Inflammation in Atherosclerosis: How to Effectively Treat Residual Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37716836/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230923205208&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE: This study reviewed the contribution of inflammation to atherosclerotic cardiovascular disease (ASCVD), which has gained widespread recognition in recent years.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37715966"
                        open
                      >
                        <summary class="article-expander__title">Interpretation of ambiguous TP53 test results: Mosaicism, clonal hematopoiesis, and variants of uncertain significance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37715966/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230923205208&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The increased use of next-generation sequencing has led to the detection of pathogenic TP53 variants in the germline setting in patients without a personal or family history consistent with Li-Fraumeni syndrome (LFS). These variants can represent low-penetrance LFS, mosaic LFS, or clonal hematopoiesis of indeterminate potential. Additionally, TP53 variants of uncertain significance can be detected in patients with a history suspicious for LFS. The interpretation of the significance of these...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37716836"
                        open
                      >
                        <summary class="article-expander__title">An Update on Inflammation in Atherosclerosis: How to Effectively Treat Residual Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37716836/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230922204741&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE: This study reviewed the contribution of inflammation to atherosclerotic cardiovascular disease (ASCVD), which has gained widespread recognition in recent years.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37715966"
                        open
                      >
                        <summary class="article-expander__title">Interpretation of ambiguous TP53 test results: Mosaicism, clonal hematopoiesis, and variants of uncertain significance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37715966/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230922204741&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The increased use of next-generation sequencing has led to the detection of pathogenic TP53 variants in the germline setting in patients without a personal or family history consistent with Li-Fraumeni syndrome (LFS). These variants can represent low-penetrance LFS, mosaic LFS, or clonal hematopoiesis of indeterminate potential. Additionally, TP53 variants of uncertain significance can be detected in patients with a history suspicious for LFS. The interpretation of the significance of these...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37716836"
                        open
                      >
                        <summary class="article-expander__title">An Update on Inflammation in Atherosclerosis: How to Effectively Treat Residual Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37716836/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230921204936&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE: This study reviewed the contribution of inflammation to atherosclerotic cardiovascular disease (ASCVD), which has gained widespread recognition in recent years.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37715966"
                        open
                      >
                        <summary class="article-expander__title">Interpretation of ambiguous TP53 test results: Mosaicism, clonal hematopoiesis, and variants of uncertain significance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37715966/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230921204936&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The increased use of next-generation sequencing has led to the detection of pathogenic TP53 variants in the germline setting in patients without a personal or family history consistent with Li-Fraumeni syndrome (LFS). These variants can represent low-penetrance LFS, mosaic LFS, or clonal hematopoiesis of indeterminate potential. Additionally, TP53 variants of uncertain significance can be detected in patients with a history suspicious for LFS. The interpretation of the significance of these...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37716836"
                        open
                      >
                        <summary class="article-expander__title">An Update on Inflammation in Atherosclerosis: How to Effectively Treat Residual Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37716836/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230920204905&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE: This study reviewed the contribution of inflammation to atherosclerotic cardiovascular disease (ASCVD), which has gained widespread recognition in recent years.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37715966"
                        open
                      >
                        <summary class="article-expander__title">Interpretation of ambiguous TP53 test results: Mosaicism, clonal hematopoiesis, and variants of uncertain significance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37715966/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230920204905&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The increased use of next-generation sequencing has led to the detection of pathogenic TP53 variants in the germline setting in patients without a personal or family history consistent with Li-Fraumeni syndrome (LFS). These variants can represent low-penetrance LFS, mosaic LFS, or clonal hematopoiesis of indeterminate potential. Additionally, TP53 variants of uncertain significance can be detected in patients with a history suspicious for LFS. The interpretation of the significance of these...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37716836"
                        open
                      >
                        <summary class="article-expander__title">An Update on Inflammation in Atherosclerosis: How to Effectively Treat Residual Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37716836/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230919204929&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE: This study reviewed the contribution of inflammation to atherosclerotic cardiovascular disease (ASCVD), which has gained widespread recognition in recent years.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37715966"
                        open
                      >
                        <summary class="article-expander__title">Interpretation of ambiguous TP53 test results: Mosaicism, clonal hematopoiesis, and variants of uncertain significance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37715966/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230919204929&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The increased use of next-generation sequencing has led to the detection of pathogenic TP53 variants in the germline setting in patients without a personal or family history consistent with Li-Fraumeni syndrome (LFS). These variants can represent low-penetrance LFS, mosaic LFS, or clonal hematopoiesis of indeterminate potential. Additionally, TP53 variants of uncertain significance can be detected in patients with a history suspicious for LFS. The interpretation of the significance of these...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37716836"
                        open
                      >
                        <summary class="article-expander__title">An Update on Inflammation in Atherosclerosis: How to Effectively Treat Residual Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37716836/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230918204942&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE: This study reviewed the contribution of inflammation to atherosclerotic cardiovascular disease (ASCVD), which has gained widespread recognition in recent years.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37715966"
                        open
                      >
                        <summary class="article-expander__title">Interpretation of ambiguous TP53 test results: Mosaicism, clonal hematopoiesis, and variants of uncertain significance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37715966/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230918204942&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The increased use of next-generation sequencing has led to the detection of pathogenic TP53 variants in the germline setting in patients without a personal or family history consistent with Li-Fraumeni syndrome (LFS). These variants can represent low-penetrance LFS, mosaic LFS, or clonal hematopoiesis of indeterminate potential. Additionally, TP53 variants of uncertain significance can be detected in patients with a history suspicious for LFS. The interpretation of the significance of these...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37716836"
                        open
                      >
                        <summary class="article-expander__title">An Update on Inflammation in Atherosclerosis: How to Effectively Treat Residual Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37716836/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230917204935&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE: This study reviewed the contribution of inflammation to atherosclerotic cardiovascular disease (ASCVD), which has gained widespread recognition in recent years.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37715966"
                        open
                      >
                        <summary class="article-expander__title">Interpretation of ambiguous TP53 test results: Mosaicism, clonal hematopoiesis, and variants of uncertain significance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37715966/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230917204935&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The increased use of next-generation sequencing has led to the detection of pathogenic TP53 variants in the germline setting in patients without a personal or family history consistent with Li-Fraumeni syndrome (LFS). These variants can represent low-penetrance LFS, mosaic LFS, or clonal hematopoiesis of indeterminate potential. Additionally, TP53 variants of uncertain significance can be detected in patients with a history suspicious for LFS. The interpretation of the significance of these...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37715966"
                        open
                      >
                        <summary class="article-expander__title">Interpretation of ambiguous TP53 test results: Mosaicism, clonal hematopoiesis, and variants of uncertain significance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37715966/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230916205201&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The increased use of next-generation sequencing has led to the detection of pathogenic TP53 variants in the germline setting in patients without a personal or family history consistent with Li-Fraumeni syndrome (LFS). These variants can represent low-penetrance LFS, mosaic LFS, or clonal hematopoiesis of indeterminate potential. Additionally, TP53 variants of uncertain significance can be detected in patients with a history suspicious for LFS. The interpretation of the significance of these...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-09-15 datetime="2023-09-15T10:00:00.000Z">2023-09-15</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-09-15 datetime="2023-09-15T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;ff&#x3D;20231105195156&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37713487"
                        open
                      >
                        <summary class="article-expander__title">Clonal cooperation through soluble metabolite exchange facilitates metastatic outgrowth by modulating Allee effect</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37713487/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231004205006&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers feature substantial intratumoral heterogeneity of genetic and phenotypically distinct lineages. Although interactions between coexisting lineages are emerging as a potential contributor to tumor evolution, the extent and nature of these interactions remain largely unknown. We postulated that tumors develop ecological interactions that sustain diversity and facilitate metastasis. Using a combination of fluorescent barcoding, mathematical modeling, metabolic analysis, and in vivo models,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37713487"
                        open
                      >
                        <summary class="article-expander__title">Clonal cooperation through soluble metabolite exchange facilitates metastatic outgrowth by modulating Allee effect</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37713487/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231003205044&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers feature substantial intratumoral heterogeneity of genetic and phenotypically distinct lineages. Although interactions between coexisting lineages are emerging as a potential contributor to tumor evolution, the extent and nature of these interactions remain largely unknown. We postulated that tumors develop ecological interactions that sustain diversity and facilitate metastasis. Using a combination of fluorescent barcoding, mathematical modeling, metabolic analysis, and in vivo models,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37713487"
                        open
                      >
                        <summary class="article-expander__title">Clonal cooperation through soluble metabolite exchange facilitates metastatic outgrowth by modulating Allee effect</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37713487/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231002205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers feature substantial intratumoral heterogeneity of genetic and phenotypically distinct lineages. Although interactions between coexisting lineages are emerging as a potential contributor to tumor evolution, the extent and nature of these interactions remain largely unknown. We postulated that tumors develop ecological interactions that sustain diversity and facilitate metastasis. Using a combination of fluorescent barcoding, mathematical modeling, metabolic analysis, and in vivo models,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37713487"
                        open
                      >
                        <summary class="article-expander__title">Clonal cooperation through soluble metabolite exchange facilitates metastatic outgrowth by modulating Allee effect</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37713487/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231001205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers feature substantial intratumoral heterogeneity of genetic and phenotypically distinct lineages. Although interactions between coexisting lineages are emerging as a potential contributor to tumor evolution, the extent and nature of these interactions remain largely unknown. We postulated that tumors develop ecological interactions that sustain diversity and facilitate metastasis. Using a combination of fluorescent barcoding, mathematical modeling, metabolic analysis, and in vivo models,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37713487"
                        open
                      >
                        <summary class="article-expander__title">Clonal cooperation through soluble metabolite exchange facilitates metastatic outgrowth by modulating Allee effect</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37713487/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230930205630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers feature substantial intratumoral heterogeneity of genetic and phenotypically distinct lineages. Although interactions between coexisting lineages are emerging as a potential contributor to tumor evolution, the extent and nature of these interactions remain largely unknown. We postulated that tumors develop ecological interactions that sustain diversity and facilitate metastasis. Using a combination of fluorescent barcoding, mathematical modeling, metabolic analysis, and in vivo models,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37713487"
                        open
                      >
                        <summary class="article-expander__title">Clonal cooperation through soluble metabolite exchange facilitates metastatic outgrowth by modulating Allee effect</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37713487/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230929204823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers feature substantial intratumoral heterogeneity of genetic and phenotypically distinct lineages. Although interactions between coexisting lineages are emerging as a potential contributor to tumor evolution, the extent and nature of these interactions remain largely unknown. We postulated that tumors develop ecological interactions that sustain diversity and facilitate metastasis. Using a combination of fluorescent barcoding, mathematical modeling, metabolic analysis, and in vivo models,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37713487"
                        open
                      >
                        <summary class="article-expander__title">Clonal cooperation through soluble metabolite exchange facilitates metastatic outgrowth by modulating Allee effect</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37713487/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230928204948&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers feature substantial intratumoral heterogeneity of genetic and phenotypically distinct lineages. Although interactions between coexisting lineages are emerging as a potential contributor to tumor evolution, the extent and nature of these interactions remain largely unknown. We postulated that tumors develop ecological interactions that sustain diversity and facilitate metastasis. Using a combination of fluorescent barcoding, mathematical modeling, metabolic analysis, and in vivo models,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37713487"
                        open
                      >
                        <summary class="article-expander__title">Clonal cooperation through soluble metabolite exchange facilitates metastatic outgrowth by modulating Allee effect</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37713487/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230927204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers feature substantial intratumoral heterogeneity of genetic and phenotypically distinct lineages. Although interactions between coexisting lineages are emerging as a potential contributor to tumor evolution, the extent and nature of these interactions remain largely unknown. We postulated that tumors develop ecological interactions that sustain diversity and facilitate metastasis. Using a combination of fluorescent barcoding, mathematical modeling, metabolic analysis, and in vivo models,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37713487"
                        open
                      >
                        <summary class="article-expander__title">Clonal cooperation through soluble metabolite exchange facilitates metastatic outgrowth by modulating Allee effect</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37713487/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230926205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers feature substantial intratumoral heterogeneity of genetic and phenotypically distinct lineages. Although interactions between coexisting lineages are emerging as a potential contributor to tumor evolution, the extent and nature of these interactions remain largely unknown. We postulated that tumors develop ecological interactions that sustain diversity and facilitate metastasis. Using a combination of fluorescent barcoding, mathematical modeling, metabolic analysis, and in vivo models,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37713487"
                        open
                      >
                        <summary class="article-expander__title">Clonal cooperation through soluble metabolite exchange facilitates metastatic outgrowth by modulating Allee effect</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37713487/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230925205014&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers feature substantial intratumoral heterogeneity of genetic and phenotypically distinct lineages. Although interactions between coexisting lineages are emerging as a potential contributor to tumor evolution, the extent and nature of these interactions remain largely unknown. We postulated that tumors develop ecological interactions that sustain diversity and facilitate metastasis. Using a combination of fluorescent barcoding, mathematical modeling, metabolic analysis, and in vivo models,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37713487"
                        open
                      >
                        <summary class="article-expander__title">Clonal cooperation through soluble metabolite exchange facilitates metastatic outgrowth by modulating Allee effect</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37713487/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230924205014&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers feature substantial intratumoral heterogeneity of genetic and phenotypically distinct lineages. Although interactions between coexisting lineages are emerging as a potential contributor to tumor evolution, the extent and nature of these interactions remain largely unknown. We postulated that tumors develop ecological interactions that sustain diversity and facilitate metastasis. Using a combination of fluorescent barcoding, mathematical modeling, metabolic analysis, and in vivo models,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37713487"
                        open
                      >
                        <summary class="article-expander__title">Clonal cooperation through soluble metabolite exchange facilitates metastatic outgrowth by modulating Allee effect</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37713487/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230923205208&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers feature substantial intratumoral heterogeneity of genetic and phenotypically distinct lineages. Although interactions between coexisting lineages are emerging as a potential contributor to tumor evolution, the extent and nature of these interactions remain largely unknown. We postulated that tumors develop ecological interactions that sustain diversity and facilitate metastasis. Using a combination of fluorescent barcoding, mathematical modeling, metabolic analysis, and in vivo models,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37713487"
                        open
                      >
                        <summary class="article-expander__title">Clonal cooperation through soluble metabolite exchange facilitates metastatic outgrowth by modulating Allee effect</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37713487/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230922204741&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers feature substantial intratumoral heterogeneity of genetic and phenotypically distinct lineages. Although interactions between coexisting lineages are emerging as a potential contributor to tumor evolution, the extent and nature of these interactions remain largely unknown. We postulated that tumors develop ecological interactions that sustain diversity and facilitate metastasis. Using a combination of fluorescent barcoding, mathematical modeling, metabolic analysis, and in vivo models,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37713487"
                        open
                      >
                        <summary class="article-expander__title">Clonal cooperation through soluble metabolite exchange facilitates metastatic outgrowth by modulating Allee effect</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37713487/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230921204936&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers feature substantial intratumoral heterogeneity of genetic and phenotypically distinct lineages. Although interactions between coexisting lineages are emerging as a potential contributor to tumor evolution, the extent and nature of these interactions remain largely unknown. We postulated that tumors develop ecological interactions that sustain diversity and facilitate metastasis. Using a combination of fluorescent barcoding, mathematical modeling, metabolic analysis, and in vivo models,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37713487"
                        open
                      >
                        <summary class="article-expander__title">Clonal cooperation through soluble metabolite exchange facilitates metastatic outgrowth by modulating Allee effect</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37713487/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230920204904&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers feature substantial intratumoral heterogeneity of genetic and phenotypically distinct lineages. Although interactions between coexisting lineages are emerging as a potential contributor to tumor evolution, the extent and nature of these interactions remain largely unknown. We postulated that tumors develop ecological interactions that sustain diversity and facilitate metastasis. Using a combination of fluorescent barcoding, mathematical modeling, metabolic analysis, and in vivo models,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37713487"
                        open
                      >
                        <summary class="article-expander__title">Clonal cooperation through soluble metabolite exchange facilitates metastatic outgrowth by modulating Allee effect</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37713487/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230919204929&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers feature substantial intratumoral heterogeneity of genetic and phenotypically distinct lineages. Although interactions between coexisting lineages are emerging as a potential contributor to tumor evolution, the extent and nature of these interactions remain largely unknown. We postulated that tumors develop ecological interactions that sustain diversity and facilitate metastasis. Using a combination of fluorescent barcoding, mathematical modeling, metabolic analysis, and in vivo models,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37713487"
                        open
                      >
                        <summary class="article-expander__title">Clonal cooperation through soluble metabolite exchange facilitates metastatic outgrowth by modulating Allee effect</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37713487/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230918204942&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers feature substantial intratumoral heterogeneity of genetic and phenotypically distinct lineages. Although interactions between coexisting lineages are emerging as a potential contributor to tumor evolution, the extent and nature of these interactions remain largely unknown. We postulated that tumors develop ecological interactions that sustain diversity and facilitate metastasis. Using a combination of fluorescent barcoding, mathematical modeling, metabolic analysis, and in vivo models,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37713487"
                        open
                      >
                        <summary class="article-expander__title">Clonal cooperation through soluble metabolite exchange facilitates metastatic outgrowth by modulating Allee effect</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37713487/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230917204936&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers feature substantial intratumoral heterogeneity of genetic and phenotypically distinct lineages. Although interactions between coexisting lineages are emerging as a potential contributor to tumor evolution, the extent and nature of these interactions remain largely unknown. We postulated that tumors develop ecological interactions that sustain diversity and facilitate metastasis. Using a combination of fluorescent barcoding, mathematical modeling, metabolic analysis, and in vivo models,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37713487"
                        open
                      >
                        <summary class="article-expander__title">Clonal cooperation through soluble metabolite exchange facilitates metastatic outgrowth by modulating Allee effect</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37713487/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230916205201&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers feature substantial intratumoral heterogeneity of genetic and phenotypically distinct lineages. Although interactions between coexisting lineages are emerging as a potential contributor to tumor evolution, the extent and nature of these interactions remain largely unknown. We postulated that tumors develop ecological interactions that sustain diversity and facilitate metastasis. Using a combination of fluorescent barcoding, mathematical modeling, metabolic analysis, and in vivo models,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37713487"
                        open
                      >
                        <summary class="article-expander__title">Clonal cooperation through soluble metabolite exchange facilitates metastatic outgrowth by modulating Allee effect</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37713487/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230915204741&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancers feature substantial intratumoral heterogeneity of genetic and phenotypically distinct lineages. Although interactions between coexisting lineages are emerging as a potential contributor to tumor evolution, the extent and nature of these interactions remain largely unknown. We postulated that tumors develop ecological interactions that sustain diversity and facilitate metastasis. Using a combination of fluorescent barcoding, mathematical modeling, metabolic analysis, and in vivo models,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-09-14 datetime="2023-09-14T10:00:00.000Z">2023-09-14</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-09-14 datetime="2023-09-14T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20231105195156">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37709843"
                        open
                      >
                        <summary class="article-expander__title">Ppm1d truncating mutations promote the development of genotoxic stress-induced AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37709843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230928204947&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) ensure blood cell production during the life-time of an organism, and to do so they need to balance self-renewal, proliferation, differentiation, and migration in a steady state as well as in response to stress or injury. Importantly, aberrant proliferation of HSCs leads to hematological malignancies, and thus, tight regulation by various tumor suppressor pathways, including p53, is essential. Protein phosphatase magnesium-dependent 1 delta (PPM1D) is a negative...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37709843"
                        open
                      >
                        <summary class="article-expander__title">Ppm1d truncating mutations promote the development of genotoxic stress-induced AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37709843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230927204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) ensure blood cell production during the life-time of an organism, and to do so they need to balance self-renewal, proliferation, differentiation, and migration in a steady state as well as in response to stress or injury. Importantly, aberrant proliferation of HSCs leads to hematological malignancies, and thus, tight regulation by various tumor suppressor pathways, including p53, is essential. Protein phosphatase magnesium-dependent 1 delta (PPM1D) is a negative...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37705521"
                        open
                      >
                        <summary class="article-expander__title">Bone Marrow Microenvironment Involvement in t-MN: Focus on Mesenchymal Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37705521/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230927204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related myeloid neoplasms (t-MN) are a late complication of cytotoxic therapy (CT) used in the treatment of both malignant and non-malignant diseases. Historically, t-MN has been considered to be a direct consequence of DNA damage induced in normal hematopoietic stem or progenitor cells (HSPC) by CT. However, we now know that treatment-induced mutations in HSC are not the only players involved in t-MN development, but additional factors may contribute to the onset of t-MN. One of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37705244"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplasia after clonal hematopoiesis with APOBEC3-mediated CYBB inactivation in retroviral gene therapy for X-CGD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37705244/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230927204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Stem cell gene therapy using the MFGS-gp91^(phox) retroviral vector was performed on a 27-year-old patient with X-linked chronic granulomatous disease (X-CGD) in 2014. The patient&#x27;s refractory infections were resolved, whereas the oxidase-positive neutrophils disappeared within 6 months. Thirty-two months after gene therapy, the patient developed myelodysplastic syndrome (MDS), and vector integration into the MECOM locus was identified in blast cells. The vector integration into MECOM was...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37709843"
                        open
                      >
                        <summary class="article-expander__title">Ppm1d truncating mutations promote the development of genotoxic stress-induced AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37709843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230926205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) ensure blood cell production during the life-time of an organism, and to do so they need to balance self-renewal, proliferation, differentiation, and migration in a steady state as well as in response to stress or injury. Importantly, aberrant proliferation of HSCs leads to hematological malignancies, and thus, tight regulation by various tumor suppressor pathways, including p53, is essential. Protein phosphatase magnesium-dependent 1 delta (PPM1D) is a negative...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37705521"
                        open
                      >
                        <summary class="article-expander__title">Bone Marrow Microenvironment Involvement in t-MN: Focus on Mesenchymal Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37705521/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230926205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related myeloid neoplasms (t-MN) are a late complication of cytotoxic therapy (CT) used in the treatment of both malignant and non-malignant diseases. Historically, t-MN has been considered to be a direct consequence of DNA damage induced in normal hematopoietic stem or progenitor cells (HSPC) by CT. However, we now know that treatment-induced mutations in HSC are not the only players involved in t-MN development, but additional factors may contribute to the onset of t-MN. One of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37705244"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplasia after clonal hematopoiesis with APOBEC3-mediated CYBB inactivation in retroviral gene therapy for X-CGD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37705244/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230926205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Stem cell gene therapy using the MFGS-gp91^(phox) retroviral vector was performed on a 27-year-old patient with X-linked chronic granulomatous disease (X-CGD) in 2014. The patient&#x27;s refractory infections were resolved, whereas the oxidase-positive neutrophils disappeared within 6 months. Thirty-two months after gene therapy, the patient developed myelodysplastic syndrome (MDS), and vector integration into the MECOM locus was identified in blast cells. The vector integration into MECOM was...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37709843"
                        open
                      >
                        <summary class="article-expander__title">Ppm1d truncating mutations promote the development of genotoxic stress-induced AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37709843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230925205009&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) ensure blood cell production during the life-time of an organism, and to do so they need to balance self-renewal, proliferation, differentiation, and migration in a steady state as well as in response to stress or injury. Importantly, aberrant proliferation of HSCs leads to hematological malignancies, and thus, tight regulation by various tumor suppressor pathways, including p53, is essential. Protein phosphatase magnesium-dependent 1 delta (PPM1D) is a negative...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37705521"
                        open
                      >
                        <summary class="article-expander__title">Bone Marrow Microenvironment Involvement in t-MN: Focus on Mesenchymal Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37705521/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230925205009&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related myeloid neoplasms (t-MN) are a late complication of cytotoxic therapy (CT) used in the treatment of both malignant and non-malignant diseases. Historically, t-MN has been considered to be a direct consequence of DNA damage induced in normal hematopoietic stem or progenitor cells (HSPC) by CT. However, we now know that treatment-induced mutations in HSC are not the only players involved in t-MN development, but additional factors may contribute to the onset of t-MN. One of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37705244"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplasia after clonal hematopoiesis with APOBEC3-mediated CYBB inactivation in retroviral gene therapy for X-CGD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37705244/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230925205009&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Stem cell gene therapy using the MFGS-gp91^(phox) retroviral vector was performed on a 27-year-old patient with X-linked chronic granulomatous disease (X-CGD) in 2014. The patient&#x27;s refractory infections were resolved, whereas the oxidase-positive neutrophils disappeared within 6 months. Thirty-two months after gene therapy, the patient developed myelodysplastic syndrome (MDS), and vector integration into the MECOM locus was identified in blast cells. The vector integration into MECOM was...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37709843"
                        open
                      >
                        <summary class="article-expander__title">Ppm1d truncating mutations promote the development of genotoxic stress-induced AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37709843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230924205014&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) ensure blood cell production during the life-time of an organism, and to do so they need to balance self-renewal, proliferation, differentiation, and migration in a steady state as well as in response to stress or injury. Importantly, aberrant proliferation of HSCs leads to hematological malignancies, and thus, tight regulation by various tumor suppressor pathways, including p53, is essential. Protein phosphatase magnesium-dependent 1 delta (PPM1D) is a negative...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37705521"
                        open
                      >
                        <summary class="article-expander__title">Bone Marrow Microenvironment Involvement in t-MN: Focus on Mesenchymal Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37705521/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230924205014&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related myeloid neoplasms (t-MN) are a late complication of cytotoxic therapy (CT) used in the treatment of both malignant and non-malignant diseases. Historically, t-MN has been considered to be a direct consequence of DNA damage induced in normal hematopoietic stem or progenitor cells (HSPC) by CT. However, we now know that treatment-induced mutations in HSC are not the only players involved in t-MN development, but additional factors may contribute to the onset of t-MN. One of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37705244"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplasia after clonal hematopoiesis with APOBEC3-mediated CYBB inactivation in retroviral gene therapy for X-CGD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37705244/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230924205014&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Stem cell gene therapy using the MFGS-gp91^(phox) retroviral vector was performed on a 27-year-old patient with X-linked chronic granulomatous disease (X-CGD) in 2014. The patient&#x27;s refractory infections were resolved, whereas the oxidase-positive neutrophils disappeared within 6 months. Thirty-two months after gene therapy, the patient developed myelodysplastic syndrome (MDS), and vector integration into the MECOM locus was identified in blast cells. The vector integration into MECOM was...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37709843"
                        open
                      >
                        <summary class="article-expander__title">Ppm1d truncating mutations promote the development of genotoxic stress-induced AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37709843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230923205208&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) ensure blood cell production during the life-time of an organism, and to do so they need to balance self-renewal, proliferation, differentiation, and migration in a steady state as well as in response to stress or injury. Importantly, aberrant proliferation of HSCs leads to hematological malignancies, and thus, tight regulation by various tumor suppressor pathways, including p53, is essential. Protein phosphatase magnesium-dependent 1 delta (PPM1D) is a negative...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37705521"
                        open
                      >
                        <summary class="article-expander__title">Bone Marrow Microenvironment Involvement in t-MN: Focus on Mesenchymal Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37705521/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230923205208&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related myeloid neoplasms (t-MN) are a late complication of cytotoxic therapy (CT) used in the treatment of both malignant and non-malignant diseases. Historically, t-MN has been considered to be a direct consequence of DNA damage induced in normal hematopoietic stem or progenitor cells (HSPC) by CT. However, we now know that treatment-induced mutations in HSC are not the only players involved in t-MN development, but additional factors may contribute to the onset of t-MN. One of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37705244"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplasia after clonal hematopoiesis with APOBEC3-mediated CYBB inactivation in retroviral gene therapy for X-CGD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37705244/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230923205208&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Stem cell gene therapy using the MFGS-gp91^(phox) retroviral vector was performed on a 27-year-old patient with X-linked chronic granulomatous disease (X-CGD) in 2014. The patient&#x27;s refractory infections were resolved, whereas the oxidase-positive neutrophils disappeared within 6 months. Thirty-two months after gene therapy, the patient developed myelodysplastic syndrome (MDS), and vector integration into the MECOM locus was identified in blast cells. The vector integration into MECOM was...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37709843"
                        open
                      >
                        <summary class="article-expander__title">Ppm1d truncating mutations promote the development of genotoxic stress-induced AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37709843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230922204741&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) ensure blood cell production during the life-time of an organism, and to do so they need to balance self-renewal, proliferation, differentiation, and migration in a steady state as well as in response to stress or injury. Importantly, aberrant proliferation of HSCs leads to hematological malignancies, and thus, tight regulation by various tumor suppressor pathways, including p53, is essential. Protein phosphatase magnesium-dependent 1 delta (PPM1D) is a negative...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37705521"
                        open
                      >
                        <summary class="article-expander__title">Bone Marrow Microenvironment Involvement in t-MN: Focus on Mesenchymal Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37705521/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230922204741&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related myeloid neoplasms (t-MN) are a late complication of cytotoxic therapy (CT) used in the treatment of both malignant and non-malignant diseases. Historically, t-MN has been considered to be a direct consequence of DNA damage induced in normal hematopoietic stem or progenitor cells (HSPC) by CT. However, we now know that treatment-induced mutations in HSC are not the only players involved in t-MN development, but additional factors may contribute to the onset of t-MN. One of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37705244"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplasia after clonal hematopoiesis with APOBEC3-mediated CYBB inactivation in retroviral gene therapy for X-CGD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37705244/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230922204741&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Stem cell gene therapy using the MFGS-gp91^(phox) retroviral vector was performed on a 27-year-old patient with X-linked chronic granulomatous disease (X-CGD) in 2014. The patient&#x27;s refractory infections were resolved, whereas the oxidase-positive neutrophils disappeared within 6 months. Thirty-two months after gene therapy, the patient developed a myelodysplastic syndrome (MDS), and vector integration into the MECOM locus was identified in blast cells. The vector integration into MECOM was...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37709843"
                        open
                      >
                        <summary class="article-expander__title">Ppm1d truncating mutations promote the development of genotoxic stress-induced AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37709843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230921204936&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) ensure blood cell production during the life-time of an organism, and to do so they need to balance self-renewal, proliferation, differentiation, and migration in a steady state as well as in response to stress or injury. Importantly, aberrant proliferation of HSCs leads to hematological malignancies, and thus, tight regulation by various tumor suppressor pathways, including p53, is essential. Protein phosphatase magnesium-dependent 1 delta (PPM1D) is a negative...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37705521"
                        open
                      >
                        <summary class="article-expander__title">Bone Marrow Microenvironment Involvement in t-MN: Focus on Mesenchymal Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37705521/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230921204936&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related myeloid neoplasms (t-MN) are a late complication of cytotoxic therapy (CT) used in the treatment of both malignant and non-malignant diseases. Historically, t-MN has been considered to be a direct consequence of DNA damage induced in normal hematopoietic stem or progenitor cells (HSPC) by CT. However, we now know that treatment-induced mutations in HSC are not the only players involved in t-MN development, but additional factors may contribute to the onset of t-MN. One of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37705244"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplasia after clonal hematopoiesis with APOBEC3-mediated CYBB inactivation in retroviral gene therapy for X-CGD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37705244/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230921204936&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Stem cell gene therapy using the MFGS-gp91^(phox) retroviral vector was performed on a 27-year-old patient with X-linked chronic granulomatous disease (X-CGD) in 2014. The patient&#x27;s refractory infections were resolved, whereas the oxidase-positive neutrophils disappeared within 6 months. Thirty-two months after gene therapy, the patient developed a myelodysplastic syndrome (MDS), and vector integration into the MECOM locus was identified in blast cells. The vector integration into MECOM was...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37709843"
                        open
                      >
                        <summary class="article-expander__title">Ppm1d truncating mutations promote the development of genotoxic stress-induced AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37709843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230920204905&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) ensure blood cell production during the life-time of an organism, and to do so they need to balance self-renewal, proliferation, differentiation, and migration in a steady state as well as in response to stress or injury. Importantly, aberrant proliferation of HSCs leads to hematological malignancies, and thus, tight regulation by various tumor suppressor pathways, including p53, is essential. Protein phosphatase magnesium-dependent 1 delta (PPM1D) is a negative...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37705521"
                        open
                      >
                        <summary class="article-expander__title">Bone Marrow Microenvironment Involvement in t-MN: Focus on Mesenchymal Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37705521/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230920204905&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related myeloid neoplasms (t-MN) are a late complication of cytotoxic therapy (CT) used in the treatment of both malignant and non-malignant diseases. Historically, t-MN has been considered to be a direct consequence of DNA damage induced in normal hematopoietic stem or progenitor cells (HSPC) by CT. However, we now know that treatment-induced mutations in HSC are not the only players involved in t-MN development, but additional factors may contribute to the onset of t-MN. One of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37705244"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplasia after clonal hematopoiesis with APOBEC3-mediated CYBB inactivation in retroviral gene therapy for X-CGD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37705244/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230920204905&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Stem cell gene therapy using the MFGS-gp91^(phox) retroviral vector was performed on a 27-year-old patient with X-linked chronic granulomatous disease (X-CGD) in 2014. The patient&#x27;s refractory infections were resolved, whereas the oxidase-positive neutrophils disappeared within 6 months. Thirty-two months after gene therapy, the patient developed a myelodysplastic syndrome (MDS), and vector integration into the MECOM locus was identified in blast cells. The vector integration into MECOM was...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37709843"
                        open
                      >
                        <summary class="article-expander__title">Ppm1d truncating mutations promote the development of genotoxic stress-induced AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37709843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230919204929&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) ensure blood cell production during the life-time of an organism, and to do so they need to balance self-renewal, proliferation, differentiation, and migration in a steady state as well as in response to stress or injury. Importantly, aberrant proliferation of HSCs leads to hematological malignancies, and thus, tight regulation by various tumor suppressor pathways, including p53, is essential. Protein phosphatase magnesium-dependent 1 delta (PPM1D) is a negative...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37705521"
                        open
                      >
                        <summary class="article-expander__title">Bone Marrow Microenvironment Involvement in t-MN: Focus on Mesenchymal Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37705521/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230919204929&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related myeloid neoplasms (t-MN) are a late complication of cytotoxic therapy (CT) used in the treatment of both malignant and non-malignant diseases. Historically, t-MN has been considered to be a direct consequence of DNA damage induced in normal hematopoietic stem or progenitor cells (HSPC) by CT. However, we now know that treatment-induced mutations in HSC are not the only players involved in t-MN development, but additional factors may contribute to the onset of t-MN. One of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37705244"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplasia after clonal hematopoiesis with APOBEC3-mediated CYBB inactivation in retroviral gene therapy for X-CGD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37705244/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230919204929&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Stem cell gene therapy using the MFGS-gp91^(phox) retroviral vector was performed on a 27-year-old patient with X-linked chronic granulomatous disease (X-CGD) in 2014. The patient&#x27;s refractory infections were resolved, whereas the oxidase-positive neutrophils disappeared within 6 months. Thirty-two months after gene therapy, the patient developed a myelodysplastic syndrome (MDS), and vector integration into the MECOM locus was identified in blast cells. The vector integration into MECOM was...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37709843"
                        open
                      >
                        <summary class="article-expander__title">Ppm1d truncating mutations promote the development of genotoxic stress-induced AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37709843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230918204942&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) ensure blood cell production during the life-time of an organism, and to do so they need to balance self-renewal, proliferation, differentiation, and migration in a steady state as well as in response to stress or injury. Importantly, aberrant proliferation of HSCs leads to hematological malignancies, and thus, tight regulation by various tumor suppressor pathways, including p53, is essential. Protein phosphatase magnesium-dependent 1 delta (PPM1D) is a negative...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37705521"
                        open
                      >
                        <summary class="article-expander__title">Bone Marrow Microenvironment Involvement in t-MN: Focus on Mesenchymal Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37705521/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230918204942&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related myeloid neoplasms (t-MN) are a late complication of cytotoxic therapy (CT) used in the treatment of both malignant and non-malignant diseases. Historically, t-MN has been considered to be a direct consequence of DNA damage induced in normal hematopoietic stem or progenitor cells (HSPC) by CT. However, we now know that treatment-induced mutations in HSC are not the only players involved in t-MN development, but additional factors may contribute to the onset of t-MN. One of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37705244"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplasia after clonal hematopoiesis with APOBEC3-mediated CYBB inactivation in retroviral gene therapy for X-CGD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37705244/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230918204942&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Stem cell gene therapy using the MFGS-gp91^(phox) retroviral vector was performed on a 27-year-old patient with X-linked chronic granulomatous disease (X-CGD) in 2014. The patient&#x27;s refractory infections were resolved, whereas the oxidase-positive neutrophils disappeared within 6 months. Thirty-two months after gene therapy, the patient developed a myelodysplastic syndrome (MDS), and vector integration into the MECOM locus was identified in blast cells. The vector integration into MECOM was...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37709843"
                        open
                      >
                        <summary class="article-expander__title">Ppm1d truncating mutations promote the development of genotoxic stress-induced AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37709843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230917204935&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) ensure blood cell production during the life-time of an organism, and to do so they need to balance self-renewal, proliferation, differentiation, and migration in a steady state as well as in response to stress or injury. Importantly, aberrant proliferation of HSCs leads to hematological malignancies, and thus, tight regulation by various tumor suppressor pathways, including p53, is essential. Protein phosphatase magnesium-dependent 1 delta (PPM1D) is a negative...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37705521"
                        open
                      >
                        <summary class="article-expander__title">Bone Marrow Microenvironment Involvement in t-MN: Focus on Mesenchymal Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37705521/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230917204935&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related myeloid neoplasms (t-MN) are a late complication of cytotoxic therapy (CT) used in the treatment of both malignant and non-malignant diseases. Historically, t-MN has been considered to be a direct consequence of DNA damage induced in normal hematopoietic stem or progenitor cells (HSPC) by CT. However, we now know that treatment-induced mutations in HSC are not the only players involved in t-MN development, but additional factors may contribute to the onset of t-MN. One of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37705244"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplasia after clonal hematopoiesis with APOBEC3-mediated CYBB inactivation in retroviral gene therapy for X-CGD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37705244/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230917204935&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Stem cell gene therapy using the MFGS-gp91^(phox) retroviral vector was performed on a 27-year-old patient with X-linked chronic granulomatous disease (X-CGD) in 2014. The patient&#x27;s refractory infections were resolved, whereas the oxidase-positive neutrophils disappeared within 6 months. Thirty-two months after gene therapy, the patient developed a myelodysplastic syndrome (MDS), and vector integration into the MECOM locus was identified in blast cells. The vector integration into MECOM was...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37709843"
                        open
                      >
                        <summary class="article-expander__title">Ppm1d truncating mutations promote the development of genotoxic stress-induced AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37709843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230916205201&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) ensure blood cell production during the life-time of an organism, and to do so they need to balance self-renewal, proliferation, differentiation, and migration in a steady state as well as in response to stress or injury. Importantly, aberrant proliferation of HSCs leads to hematological malignancies, and thus, tight regulation by various tumor suppressor pathways, including p53, is essential. Protein phosphatase magnesium-dependent 1 delta (PPM1D) is a negative...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37705521"
                        open
                      >
                        <summary class="article-expander__title">Bone Marrow Microenvironment Involvement in t-MN: Focus on Mesenchymal Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37705521/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230916205201&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related myeloid neoplasms (t-MN) are a late complication of cytotoxic therapy (CT) used in the treatment of both malignant and non-malignant diseases. Historically, t-MN has been considered to be a direct consequence of DNA damage induced in normal hematopoietic stem or progenitor cells (HSPC) by CT. However, we now know that treatment-induced mutations in HSC are not the only players involved in t-MN development, but additional factors may contribute to the onset of t-MN. One of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37705244"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplasia after clonal hematopoiesis with APOBEC3-mediated CYBB inactivation in retroviral gene therapy for X-CGD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37705244/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230916205201&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Stem cell gene therapy using the MFGS-gp91^(phox) retroviral vector was performed on a 27-year-old patient with X-linked chronic granulomatous disease (X-CGD) in 2014. The patient&#x27;s refractory infections were resolved, whereas the oxidase-positive neutrophils disappeared within 6 months. Thirty-two months after gene therapy, the patient developed a myelodysplastic syndrome (MDS), and vector integration into the MECOM locus was identified in blast cells. The vector integration into MECOM was...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37709843"
                        open
                      >
                        <summary class="article-expander__title">Ppm1d truncating mutations promote the development of genotoxic stress-induced AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37709843/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230915204741&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) ensure blood cell production during the life-time of an organism, and to do so they need to balance self-renewal, proliferation, differentiation, and migration in a steady state as well as in response to stress or injury. Importantly, aberrant proliferation of HSCs leads to hematological malignancies, and thus, tight regulation by various tumor suppressor pathways, including p53, is essential. Protein phosphatase magnesium-dependent 1 delta (PPM1D) is a negative...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37705521"
                        open
                      >
                        <summary class="article-expander__title">Bone Marrow Microenvironment Involvement in t-MN: Focus on Mesenchymal Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37705521/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230915204741&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related myeloid neoplasms (t-MN) are a late complication of cytotoxic therapy (CT) used in the treatment of both malignant and non-malignant diseases. Historically, t-MN has been considered to be a direct consequence of DNA damage induced in normal hematopoietic stem or progenitor cells (HSPC) by CT. However, we now know that treatment-induced mutations in HSC are not the only players involved in t-MN development, but additional factors may contribute to the onset of t-MN. One of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37705244"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplasia after clonal hematopoiesis with APOBEC3-mediated CYBB inactivation in retroviral gene therapy for X-CGD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37705244/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230915204741&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Stem cell gene therapy using the MFGS-gp91^(phox) retroviral vector was performed on a 27-year-old patient with X-linked chronic granulomatous disease (X-CGD) in 2014. The patient&#x27;s refractory infections were resolved, whereas the oxidase-positive neutrophils disappeared within 6 months. Thirty-two months after gene therapy, the patient developed a myelodysplastic syndrome (MDS), and vector integration into the MECOM locus was identified in blast cells. The vector integration into MECOM was...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37705521"
                        open
                      >
                        <summary class="article-expander__title">Bone Marrow Microenvironment Involvement in t-MN: Focus on Mesenchymal Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37705521/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230914204950&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related myeloid neoplasms (t-MN) are a late complication of cytotoxic therapy (CT) used in the treatment of both malignant and non-malignant diseases. Historically, t-MN has been considered to be a direct consequence of DNA damage induced in normal hematopoietic stem or progenitor cells (HSPC) by CT. However, we now know that treatment-induced mutations in HSC are not the only players involved in t-MN development, but additional factors may contribute to the onset of t-MN. One of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37705244"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplasia after clonal hematopoiesis with APOBEC3-mediated CYBB inactivation in retroviral gene therapy for X-CGD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37705244/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230914204950&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Stem cell gene therapy using the MFGS-gp91^(phox) retroviral vector was performed on a 27-year-old patient with X-linked chronic granulomatous disease (X-CGD) in 2014. The patient&#x27;s refractory infections were resolved, whereas the oxidase-positive neutrophils disappeared within 6 months. Thirty-two months after gene therapy, the patient developed a myelodysplastic syndrome (MDS), and vector integration into the MECOM locus was identified in blast cells. The vector integration into MECOM was...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-09-12 datetime="2023-09-12T10:00:00.000Z">2023-09-12</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-09-12 datetime="2023-09-12T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20231105195156">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37699006"
                        open
                      >
                        <summary class="article-expander__title">cloneRate: fast estimation of single-cell clonal dynamics using coalescent theory</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37699006/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230926205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: While evolutionary approaches to medicine show promise, measuring evolution itself is difficult due to experimental constraints and the dynamic nature of body systems. In cancer evolution, continuous observation of clonal architecture is impossible, and longitudinal samples from multiple timepoints are rare. Increasingly available DNA sequencing datasets at single-cell resolution enable the reconstruction of past evolution using mutational history, allowing for a better understanding...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37699006"
                        open
                      >
                        <summary class="article-expander__title">cloneRate: fast estimation of single-cell clonal dynamics using coalescent theory</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37699006/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230925205009&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: While evolutionary approaches to medicine show promise, measuring evolution itself is difficult due to experimental constraints and the dynamic nature of body systems. In cancer evolution, continuous observation of clonal architecture is impossible, and longitudinal samples from multiple timepoints are rare. Increasingly available DNA sequencing datasets at single-cell resolution enable the reconstruction of past evolution using mutational history, allowing for a better understanding...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37699006"
                        open
                      >
                        <summary class="article-expander__title">cloneRate: fast estimation of single-cell clonal dynamics using coalescent theory</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37699006/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230924205014&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: While evolutionary approaches to medicine show promise, measuring evolution itself is difficult due to experimental constraints and the dynamic nature of body systems. In cancer evolution, continuous observation of clonal architecture is impossible, and longitudinal samples from multiple timepoints are rare. Increasingly available DNA sequencing datasets at single-cell resolution enable the reconstruction of past evolution using mutational history, allowing for a better understanding...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37699006"
                        open
                      >
                        <summary class="article-expander__title">cloneRate: fast estimation of single-cell clonal dynamics using coalescent theory</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37699006/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230923205208&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: While evolutionary approaches to medicine show promise, measuring evolution itself is difficult due to experimental constraints and the dynamic nature of body systems. In cancer evolution, continuous observation of clonal architecture is impossible, and longitudinal samples from multiple timepoints are rare. Increasingly available DNA sequencing datasets at single-cell resolution enable the reconstruction of past evolution using mutational history, allowing for a better understanding...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37699006"
                        open
                      >
                        <summary class="article-expander__title">cloneRate: fast estimation of single-cell clonal dynamics using coalescent theory</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37699006/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230922204741&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: While evolutionary approaches to medicine show promise, measuring evolution itself is difficult due to experimental constraints and the dynamic nature of body systems. In cancer evolution, continuous observation of clonal architecture is impossible, and longitudinal samples from multiple timepoints are rare. Increasingly available DNA sequencing datasets at single-cell resolution enable the reconstruction of past evolution using mutational history, allowing for a better understanding...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37699006"
                        open
                      >
                        <summary class="article-expander__title">cloneRate: fast estimation of single-cell clonal dynamics using coalescent theory</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37699006/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230921204936&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: While evolutionary approaches to medicine show promise, measuring evolution itself is difficult due to experimental constraints and the dynamic nature of body systems. In cancer evolution, continuous observation of clonal architecture is impossible, and longitudinal samples from multiple timepoints are rare. Increasingly available DNA sequencing datasets at single-cell resolution enable the reconstruction of past evolution using mutational history, allowing for a better understanding...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37699006"
                        open
                      >
                        <summary class="article-expander__title">cloneRate: fast estimation of single-cell clonal dynamics using coalescent theory</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37699006/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230920204905&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: While evolutionary approaches to medicine show promise, measuring evolution itself is difficult due to experimental constraints and the dynamic nature of body systems. In cancer evolution, continuous observation of clonal architecture is impossible, and longitudinal samples from multiple timepoints are rare. Increasingly available DNA sequencing datasets at single-cell resolution enable the reconstruction of past evolution using mutational history, allowing for a better understanding...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37699006"
                        open
                      >
                        <summary class="article-expander__title">cloneRate: fast estimation of single-cell clonal dynamics using coalescent theory</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37699006/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230919204929&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: While evolutionary approaches to medicine show promise, measuring evolution itself is difficult due to experimental constraints and the dynamic nature of body systems. In cancer evolution, continuous observation of clonal architecture is impossible, and longitudinal samples from multiple timepoints are rare. Increasingly available DNA sequencing datasets at single-cell resolution enable the reconstruction of past evolution using mutational history, allowing for a better understanding...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37699006"
                        open
                      >
                        <summary class="article-expander__title">cloneRate: fast estimation of single-cell clonal dynamics using coalescent theory</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37699006/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230918204942&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: While evolutionary approaches to medicine show promise, measuring evolution itself is difficult due to experimental constraints and the dynamic nature of body systems. In cancer evolution, continuous observation of clonal architecture is impossible, and longitudinal samples from multiple timepoints are rare. Increasingly available DNA sequencing datasets at single-cell resolution enable the reconstruction of past evolution using mutational history, allowing for a better understanding...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37699006"
                        open
                      >
                        <summary class="article-expander__title">cloneRate: fast estimation of single-cell clonal dynamics using coalescent theory</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37699006/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230917204935&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: While evolutionary approaches to medicine show promise, measuring evolution itself is difficult due to experimental constraints and the dynamic nature of body systems. In cancer evolution, continuous observation of clonal architecture is impossible, and longitudinal samples from multiple timepoints are rare. Increasingly available DNA sequencing datasets at single-cell resolution enable the reconstruction of past evolution using mutational history, allowing for a better understanding...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37699006"
                        open
                      >
                        <summary class="article-expander__title">cloneRate: fast estimation of single-cell clonal dynamics using coalescent theory</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37699006/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230916205201&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: While evolutionary approaches to medicine show promise, measuring evolution itself is difficult due to experimental constraints and the dynamic nature of body systems. In cancer evolution, continuous observation of clonal architecture is impossible, and longitudinal samples from multiple timepoints are rare. Increasingly available DNA sequencing datasets at single-cell resolution enable the reconstruction of past evolution using mutational history, allowing for a better understanding...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37699006"
                        open
                      >
                        <summary class="article-expander__title">cloneRate: fast estimation of single-cell clonal dynamics using coalescent theory</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37699006/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230915204741&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: While evolutionary approaches to medicine show promise, measuring evolution itself is difficult due to experimental constraints and the dynamic nature of body systems. In cancer evolution, continuous observation of clonal architecture is impossible, and longitudinal samples from multiple timepoints are rare. Increasingly available DNA sequencing datasets at single-cell resolution enable the reconstruction of past evolution using mutational history, allowing for a better understanding...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37699006"
                        open
                      >
                        <summary class="article-expander__title">cloneRate: fast estimation of single-cell clonal dynamics using coalescent theory</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37699006/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230914204950&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: While evolutionary approaches to medicine show promise, measuring evolution itself is difficult due to experimental constraints and the dynamic nature of body systems. In cancer evolution, continuous observation of clonal architecture is impossible, and longitudinal samples from multiple timepoints are rare. Increasingly available DNA sequencing datasets at single-cell resolution enable the reconstruction of past evolution using mutational history, allowing for a better understanding...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37699006"
                        open
                      >
                        <summary class="article-expander__title">cloneRate: fast estimation of single-cell clonal dynamics using coalescent theory</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37699006/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230913204840&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: While evolutionary approaches to medicine show promise, measuring evolution itself is difficult due to experimental constraints and the dynamic nature of body systems. In cancer evolution, continuous observation of clonal architecture is impossible, and longitudinal samples from multiple timepoints are rare. Increasingly available DNA sequencing datasets at single-cell resolution enable the reconstruction of past evolution using mutational history, allowing for a better understanding...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37699006"
                        open
                      >
                        <summary class="article-expander__title">cloneRate: fast estimation of single-cell clonal dynamics using coalescent theory</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37699006/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230912205016&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: While evolutionary approaches to medicine show promise, measuring evolution itself is difficult due to experimental constraints and the dynamic nature of body systems. In cancer evolution, continuous observation of clonal architecture is impossible, and longitudinal samples from multiple timepoints are rare. Increasingly available DNA sequencing datasets at single-cell resolution enable the reconstruction of past evolution using mutational history, allowing for a better understanding...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-09-11 datetime="2023-09-11T10:00:00.000Z">2023-09-11</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-09-11 datetime="2023-09-11T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;ff&#x3D;20231105195156&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37695905"
                        open
                      >
                        <summary class="article-expander__title">Fully accessible fitness landscape of oncogene-negative lung adenocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37695905/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231004205006&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes are almost invariably complex with genomic alterations cooperating during each step of carcinogenesis. In cancers that lack a single dominant oncogene mutation, cooperation between the inactivation of multiple tumor suppressor genes can drive tumor initiation and growth. Here, we shed light on how the sequential acquisition of genomic alterations generates oncogene-negative lung tumors. We couple tumor barcoding with combinatorial and multiplexed somatic genome editing to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37695905"
                        open
                      >
                        <summary class="article-expander__title">Fully accessible fitness landscape of oncogene-negative lung adenocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37695905/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231003205044&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes are almost invariably complex with genomic alterations cooperating during each step of carcinogenesis. In cancers that lack a single dominant oncogene mutation, cooperation between the inactivation of multiple tumor suppressor genes can drive tumor initiation and growth. Here, we shed light on how the sequential acquisition of genomic alterations generates oncogene-negative lung tumors. We couple tumor barcoding with combinatorial and multiplexed somatic genome editing to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37695905"
                        open
                      >
                        <summary class="article-expander__title">Fully accessible fitness landscape of oncogene-negative lung adenocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37695905/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231002205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes are almost invariably complex with genomic alterations cooperating during each step of carcinogenesis. In cancers that lack a single dominant oncogene mutation, cooperation between the inactivation of multiple tumor suppressor genes can drive tumor initiation and growth. Here, we shed light on how the sequential acquisition of genomic alterations generates oncogene-negative lung tumors. We couple tumor barcoding with combinatorial and multiplexed somatic genome editing to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37695905"
                        open
                      >
                        <summary class="article-expander__title">Fully accessible fitness landscape of oncogene-negative lung adenocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37695905/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231001205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes are almost invariably complex with genomic alterations cooperating during each step of carcinogenesis. In cancers that lack a single dominant oncogene mutation, cooperation between the inactivation of multiple tumor suppressor genes can drive tumor initiation and growth. Here, we shed light on how the sequential acquisition of genomic alterations generates oncogene-negative lung tumors. We couple tumor barcoding with combinatorial and multiplexed somatic genome editing to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37695905"
                        open
                      >
                        <summary class="article-expander__title">Fully accessible fitness landscape of oncogene-negative lung adenocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37695905/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230930205630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes are almost invariably complex with genomic alterations cooperating during each step of carcinogenesis. In cancers that lack a single dominant oncogene mutation, cooperation between the inactivation of multiple tumor suppressor genes can drive tumor initiation and growth. Here, we shed light on how the sequential acquisition of genomic alterations generates oncogene-negative lung tumors. We couple tumor barcoding with combinatorial and multiplexed somatic genome editing to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37695905"
                        open
                      >
                        <summary class="article-expander__title">Fully accessible fitness landscape of oncogene-negative lung adenocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37695905/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230929204823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes are almost invariably complex with genomic alterations cooperating during each step of carcinogenesis. In cancers that lack a single dominant oncogene mutation, cooperation between the inactivation of multiple tumor suppressor genes can drive tumor initiation and growth. Here, we shed light on how the sequential acquisition of genomic alterations generates oncogene-negative lung tumors. We couple tumor barcoding with combinatorial and multiplexed somatic genome editing to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37695905"
                        open
                      >
                        <summary class="article-expander__title">Fully accessible fitness landscape of oncogene-negative lung adenocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37695905/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230928204948&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes are almost invariably complex with genomic alterations cooperating during each step of carcinogenesis. In cancers that lack a single dominant oncogene mutation, cooperation between the inactivation of multiple tumor suppressor genes can drive tumor initiation and growth. Here, we shed light on how the sequential acquisition of genomic alterations generates oncogene-negative lung tumors. We couple tumor barcoding with combinatorial and multiplexed somatic genome editing to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37695905"
                        open
                      >
                        <summary class="article-expander__title">Fully accessible fitness landscape of oncogene-negative lung adenocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37695905/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230927204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes are almost invariably complex with genomic alterations cooperating during each step of carcinogenesis. In cancers that lack a single dominant oncogene mutation, cooperation between the inactivation of multiple tumor suppressor genes can drive tumor initiation and growth. Here, we shed light on how the sequential acquisition of genomic alterations generates oncogene-negative lung tumors. We couple tumor barcoding with combinatorial and multiplexed somatic genome editing to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37695905"
                        open
                      >
                        <summary class="article-expander__title">Fully accessible fitness landscape of oncogene-negative lung adenocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37695905/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230926205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes are almost invariably complex with genomic alterations cooperating during each step of carcinogenesis. In cancers that lack a single dominant oncogene mutation, cooperation between the inactivation of multiple tumor suppressor genes can drive tumor initiation and growth. Here, we shed light on how the sequential acquisition of genomic alterations generates oncogene-negative lung tumors. We couple tumor barcoding with combinatorial and multiplexed somatic genome editing to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37695905"
                        open
                      >
                        <summary class="article-expander__title">Fully accessible fitness landscape of oncogene-negative lung adenocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37695905/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230925205014&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes are almost invariably complex with genomic alterations cooperating during each step of carcinogenesis. In cancers that lack a single dominant oncogene mutation, cooperation between the inactivation of multiple tumor suppressor genes can drive tumor initiation and growth. Here, we shed light on how the sequential acquisition of genomic alterations generates oncogene-negative lung tumors. We couple tumor barcoding with combinatorial and multiplexed somatic genome editing to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37695905"
                        open
                      >
                        <summary class="article-expander__title">Fully accessible fitness landscape of oncogene-negative lung adenocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37695905/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230924205014&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes are almost invariably complex with genomic alterations cooperating during each step of carcinogenesis. In cancers that lack a single dominant oncogene mutation, cooperation between the inactivation of multiple tumor suppressor genes can drive tumor initiation and growth. Here, we shed light on how the sequential acquisition of genomic alterations generates oncogene-negative lung tumors. We couple tumor barcoding with combinatorial and multiplexed somatic genome editing to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37695905"
                        open
                      >
                        <summary class="article-expander__title">Fully accessible fitness landscape of oncogene-negative lung adenocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37695905/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230923205208&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes are almost invariably complex with genomic alterations cooperating during each step of carcinogenesis. In cancers that lack a single dominant oncogene mutation, cooperation between the inactivation of multiple tumor suppressor genes can drive tumor initiation and growth. Here, we shed light on how the sequential acquisition of genomic alterations generates oncogene-negative lung tumors. We couple tumor barcoding with combinatorial and multiplexed somatic genome editing to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37695905"
                        open
                      >
                        <summary class="article-expander__title">Fully accessible fitness landscape of oncogene-negative lung adenocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37695905/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230922204741&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes are almost invariably complex with genomic alterations cooperating during each step of carcinogenesis. In cancers that lack a single dominant oncogene mutation, cooperation between the inactivation of multiple tumor suppressor genes can drive tumor initiation and growth. Here, we shed light on how the sequential acquisition of genomic alterations generates oncogene-negative lung tumors. We couple tumor barcoding with combinatorial and multiplexed somatic genome editing to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37695905"
                        open
                      >
                        <summary class="article-expander__title">Fully accessible fitness landscape of oncogene-negative lung adenocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37695905/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230921204936&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes are almost invariably complex with genomic alterations cooperating during each step of carcinogenesis. In cancers that lack a single dominant oncogene mutation, cooperation between the inactivation of multiple tumor suppressor genes can drive tumor initiation and growth. Here, we shed light on how the sequential acquisition of genomic alterations generates oncogene-negative lung tumors. We couple tumor barcoding with combinatorial and multiplexed somatic genome editing to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37695905"
                        open
                      >
                        <summary class="article-expander__title">Fully accessible fitness landscape of oncogene-negative lung adenocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37695905/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230920204904&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes are almost invariably complex with genomic alterations cooperating during each step of carcinogenesis. In cancers that lack a single dominant oncogene mutation, cooperation between the inactivation of multiple tumor suppressor genes can drive tumor initiation and growth. Here, we shed light on how the sequential acquisition of genomic alterations generates oncogene-negative lung tumors. We couple tumor barcoding with combinatorial and multiplexed somatic genome editing to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37695905"
                        open
                      >
                        <summary class="article-expander__title">Fully accessible fitness landscape of oncogene-negative lung adenocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37695905/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230919204929&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes are almost invariably complex with genomic alterations cooperating during each step of carcinogenesis. In cancers that lack a single dominant oncogene mutation, cooperation between the inactivation of multiple tumor suppressor genes can drive tumor initiation and growth. Here, we shed light on how the sequential acquisition of genomic alterations generates oncogene-negative lung tumors. We couple tumor barcoding with combinatorial and multiplexed somatic genome editing to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37695905"
                        open
                      >
                        <summary class="article-expander__title">Fully accessible fitness landscape of oncogene-negative lung adenocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37695905/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230918204942&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes are almost invariably complex with genomic alterations cooperating during each step of carcinogenesis. In cancers that lack a single dominant oncogene mutation, cooperation between the inactivation of multiple tumor suppressor genes can drive tumor initiation and growth. Here, we shed light on how the sequential acquisition of genomic alterations generates oncogene-negative lung tumors. We couple tumor barcoding with combinatorial and multiplexed somatic genome editing to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37695905"
                        open
                      >
                        <summary class="article-expander__title">Fully accessible fitness landscape of oncogene-negative lung adenocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37695905/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230917204936&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes are almost invariably complex with genomic alterations cooperating during each step of carcinogenesis. In cancers that lack a single dominant oncogene mutation, cooperation between the inactivation of multiple tumor suppressor genes can drive tumor initiation and growth. Here, we shed light on how the sequential acquisition of genomic alterations generates oncogene-negative lung tumors. We couple tumor barcoding with combinatorial and multiplexed somatic genome editing to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37695905"
                        open
                      >
                        <summary class="article-expander__title">Fully accessible fitness landscape of oncogene-negative lung adenocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37695905/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230916205201&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes are almost invariably complex with genomic alterations cooperating during each step of carcinogenesis. In cancers that lack a single dominant oncogene mutation, cooperation between the inactivation of multiple tumor suppressor genes can drive tumor initiation and growth. Here, we shed light on how the sequential acquisition of genomic alterations generates oncogene-negative lung tumors. We couple tumor barcoding with combinatorial and multiplexed somatic genome editing to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37695905"
                        open
                      >
                        <summary class="article-expander__title">Fully accessible fitness landscape of oncogene-negative lung adenocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37695905/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230915204741&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes are almost invariably complex with genomic alterations cooperating during each step of carcinogenesis. In cancers that lack a single dominant oncogene mutation, cooperation between the inactivation of multiple tumor suppressor genes can drive tumor initiation and growth. Here, we shed light on how the sequential acquisition of genomic alterations generates oncogene-negative lung tumors. We couple tumor barcoding with combinatorial and multiplexed somatic genome editing to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37695905"
                        open
                      >
                        <summary class="article-expander__title">Fully accessible fitness landscape of oncogene-negative lung adenocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37695905/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230914204950&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes are almost invariably complex with genomic alterations cooperating during each step of carcinogenesis. In cancers that lack a single dominant oncogene mutation, cooperation between the inactivation of multiple tumor suppressor genes can drive tumor initiation and growth. Here, we shed light on how the sequential acquisition of genomic alterations generates oncogene-negative lung tumors. We couple tumor barcoding with combinatorial and multiplexed somatic genome editing to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37695905"
                        open
                      >
                        <summary class="article-expander__title">Fully accessible fitness landscape of oncogene-negative lung adenocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37695905/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230913204840&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes are almost invariably complex with genomic alterations cooperating during each step of carcinogenesis. In cancers that lack a single dominant oncogene mutation, cooperation between the inactivation of multiple tumor suppressor genes can drive tumor initiation and growth. Here, we shed light on how the sequential acquisition of genomic alterations generates oncogene-negative lung tumors. We couple tumor barcoding with combinatorial and multiplexed somatic genome editing to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37695905"
                        open
                      >
                        <summary class="article-expander__title">Fully accessible fitness landscape of oncogene-negative lung adenocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37695905/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230912205015&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes are almost invariably complex with genomic alterations cooperating during each step of carcinogenesis. In cancers that lack a single dominant oncogene mutation, cooperation between the inactivation of multiple tumor suppressor genes can drive tumor initiation and growth. Here, we shed light on how the sequential acquisition of genomic alterations generates oncogene-negative lung tumors. We couple tumor barcoding with combinatorial and multiplexed somatic genome editing to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37695905"
                        open
                      >
                        <summary class="article-expander__title">Fully accessible fitness landscape of oncogene-negative lung adenocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37695905/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230911204717&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomes are almost invariably complex with genomic alterations cooperating during each step of carcinogenesis. In cancers that lack a single dominant oncogene mutation, cooperation between the inactivation of multiple tumor suppressor genes can drive tumor initiation and growth. Here, we shed light on how the sequential acquisition of genomic alterations generates oncogene-negative lung tumors. We couple tumor barcoding with combinatorial and multiplexed somatic genome editing to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-09-09 datetime="2023-09-09T10:00:00.000Z">2023-09-09</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-09-09 datetime="2023-09-09T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;ff&#x3D;20231105195156&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37686627"
                        open
                      >
                        <summary class="article-expander__title">Challenges and Prospects of Patient-Derived Xenografts for Cancer Research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37686627/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231004205006&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We discuss the importance of the in vivo models in elucidating cancer biology, focusing on the patient-derived xenograft (PDX) models, which are classic and standard functional in vivo platforms for preclinical evaluation. We provide an overview of the most representative models, including cell-derived xenografts (CDX), tumor and metastatic cell-derived xenografts, and PDX models utilizing humanized mice (HM). The orthotopic models, which could reproduce the cancer environment and its...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37686627"
                        open
                      >
                        <summary class="article-expander__title">Challenges and Prospects of Patient-Derived Xenografts for Cancer Research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37686627/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231003205044&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We discuss the importance of the in vivo models in elucidating cancer biology, focusing on the patient-derived xenograft (PDX) models, which are classic and standard functional in vivo platforms for preclinical evaluation. We provide an overview of the most representative models, including cell-derived xenografts (CDX), tumor and metastatic cell-derived xenografts, and PDX models utilizing humanized mice (HM). The orthotopic models, which could reproduce the cancer environment and its...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37685874"
                        open
                      >
                        <summary class="article-expander__title">A Leukemic Target with a Thousand Faces: The Mitochondria</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37685874/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231003205044&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the era of personalized medicine greatly improved by molecular diagnosis and tailor-made therapies, the survival rate of acute myeloid leukemia (AML) at 5 years remains unfortunately low. Indeed, the high heterogeneity of AML clones with distinct metabolic and molecular profiles allows them to survive the chemotherapy-induced changes, thus leading to resistance, clonal evolution, and relapse. Moreover, leukemic stem cells (LSCs), the quiescent reservoir of residual disease, can persist for a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37686627"
                        open
                      >
                        <summary class="article-expander__title">Challenges and Prospects of Patient-Derived Xenografts for Cancer Research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37686627/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231002205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We discuss the importance of the in vivo models in elucidating cancer biology, focusing on the patient-derived xenograft (PDX) models, which are classic and standard functional in vivo platforms for preclinical evaluation. We provide an overview of the most representative models, including cell-derived xenografts (CDX), tumor and metastatic cell-derived xenografts, and PDX models utilizing humanized mice (HM). The orthotopic models, which could reproduce the cancer environment and its...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37685874"
                        open
                      >
                        <summary class="article-expander__title">A Leukemic Target with a Thousand Faces: The Mitochondria</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37685874/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231002205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the era of personalized medicine greatly improved by molecular diagnosis and tailor-made therapies, the survival rate of acute myeloid leukemia (AML) at 5 years remains unfortunately low. Indeed, the high heterogeneity of AML clones with distinct metabolic and molecular profiles allows them to survive the chemotherapy-induced changes, thus leading to resistance, clonal evolution, and relapse. Moreover, leukemic stem cells (LSCs), the quiescent reservoir of residual disease, can persist for a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37686627"
                        open
                      >
                        <summary class="article-expander__title">Challenges and Prospects of Patient-Derived Xenografts for Cancer Research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37686627/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231001205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We discuss the importance of the in vivo models in elucidating cancer biology, focusing on the patient-derived xenograft (PDX) models, which are classic and standard functional in vivo platforms for preclinical evaluation. We provide an overview of the most representative models, including cell-derived xenografts (CDX), tumor and metastatic cell-derived xenografts, and PDX models utilizing humanized mice (HM). The orthotopic models, which could reproduce the cancer environment and its...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37685874"
                        open
                      >
                        <summary class="article-expander__title">A Leukemic Target with a Thousand Faces: The Mitochondria</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37685874/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231001205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the era of personalized medicine greatly improved by molecular diagnosis and tailor-made therapies, the survival rate of acute myeloid leukemia (AML) at 5 years remains unfortunately low. Indeed, the high heterogeneity of AML clones with distinct metabolic and molecular profiles allows them to survive the chemotherapy-induced changes, thus leading to resistance, clonal evolution, and relapse. Moreover, leukemic stem cells (LSCs), the quiescent reservoir of residual disease, can persist for a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37686627"
                        open
                      >
                        <summary class="article-expander__title">Challenges and Prospects of Patient-Derived Xenografts for Cancer Research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37686627/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230930205630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We discuss the importance of the in vivo models in elucidating cancer biology, focusing on the patient-derived xenograft (PDX) models, which are classic and standard functional in vivo platforms for preclinical evaluation. We provide an overview of the most representative models, including cell-derived xenografts (CDX), tumor and metastatic cell-derived xenografts, and PDX models utilizing humanized mice (HM). The orthotopic models, which could reproduce the cancer environment and its...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37685874"
                        open
                      >
                        <summary class="article-expander__title">A Leukemic Target with a Thousand Faces: The Mitochondria</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37685874/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230930205630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the era of personalized medicine greatly improved by molecular diagnosis and tailor-made therapies, the survival rate of acute myeloid leukemia (AML) at 5 years remains unfortunately low. Indeed, the high heterogeneity of AML clones with distinct metabolic and molecular profiles allows them to survive the chemotherapy-induced changes, thus leading to resistance, clonal evolution, and relapse. Moreover, leukemic stem cells (LSCs), the quiescent reservoir of residual disease, can persist for a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37686627"
                        open
                      >
                        <summary class="article-expander__title">Challenges and Prospects of Patient-Derived Xenografts for Cancer Research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37686627/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230929204823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We discuss the importance of the in vivo models in elucidating cancer biology, focusing on the patient-derived xenograft (PDX) models, which are classic and standard functional in vivo platforms for preclinical evaluation. We provide an overview of the most representative models, including cell-derived xenografts (CDX), tumor and metastatic cell-derived xenografts, and PDX models utilizing humanized mice (HM). The orthotopic models, which could reproduce the cancer environment and its...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37685874"
                        open
                      >
                        <summary class="article-expander__title">A Leukemic Target with a Thousand Faces: The Mitochondria</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37685874/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230929204823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the era of personalized medicine greatly improved by molecular diagnosis and tailor-made therapies, the survival rate of acute myeloid leukemia (AML) at 5 years remains unfortunately low. Indeed, the high heterogeneity of AML clones with distinct metabolic and molecular profiles allows them to survive the chemotherapy-induced changes, thus leading to resistance, clonal evolution, and relapse. Moreover, leukemic stem cells (LSCs), the quiescent reservoir of residual disease, can persist for a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37686627"
                        open
                      >
                        <summary class="article-expander__title">Challenges and Prospects of Patient-Derived Xenografts for Cancer Research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37686627/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230928204948&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We discuss the importance of the in vivo models in elucidating cancer biology, focusing on the patient-derived xenograft (PDX) models, which are classic and standard functional in vivo platforms for preclinical evaluation. We provide an overview of the most representative models, including cell-derived xenografts (CDX), tumor and metastatic cell-derived xenografts, and PDX models utilizing humanized mice (HM). The orthotopic models, which could reproduce the cancer environment and its...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37685874"
                        open
                      >
                        <summary class="article-expander__title">A Leukemic Target with a Thousand Faces: The Mitochondria</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37685874/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230928204948&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the era of personalized medicine greatly improved by molecular diagnosis and tailor-made therapies, the survival rate of acute myeloid leukemia (AML) at 5 years remains unfortunately low. Indeed, the high heterogeneity of AML clones with distinct metabolic and molecular profiles allows them to survive the chemotherapy-induced changes, thus leading to resistance, clonal evolution, and relapse. Moreover, leukemic stem cells (LSCs), the quiescent reservoir of residual disease, can persist for a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37686627"
                        open
                      >
                        <summary class="article-expander__title">Challenges and Prospects of Patient-Derived Xenografts for Cancer Research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37686627/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230927204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We discuss the importance of the in vivo models in elucidating cancer biology, focusing on the patient-derived xenograft (PDX) models, which are classic and standard functional in vivo platforms for preclinical evaluation. We provide an overview of the most representative models, including cell-derived xenografts (CDX), tumor and metastatic cell-derived xenografts, and PDX models utilizing humanized mice (HM). The orthotopic models, which could reproduce the cancer environment and its...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37685874"
                        open
                      >
                        <summary class="article-expander__title">A Leukemic Target with a Thousand Faces: The Mitochondria</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37685874/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230927204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the era of personalized medicine greatly improved by molecular diagnosis and tailor-made therapies, the survival rate of acute myeloid leukemia (AML) at 5 years remains unfortunately low. Indeed, the high heterogeneity of AML clones with distinct metabolic and molecular profiles allows them to survive the chemotherapy-induced changes, thus leading to resistance, clonal evolution, and relapse. Moreover, leukemic stem cells (LSCs), the quiescent reservoir of residual disease, can persist for a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37686627"
                        open
                      >
                        <summary class="article-expander__title">Challenges and Prospects of Patient-Derived Xenografts for Cancer Research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37686627/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230926205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We discuss the importance of the in vivo models in elucidating cancer biology, focusing on the patient-derived xenograft (PDX) models, which are classic and standard functional in vivo platforms for preclinical evaluation. We provide an overview of the most representative models, including cell-derived xenografts (CDX), tumor and metastatic cell-derived xenografts, and PDX models utilizing humanized mice (HM). The orthotopic models, which could reproduce the cancer environment and its...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37685874"
                        open
                      >
                        <summary class="article-expander__title">A Leukemic Target with a Thousand Faces: The Mitochondria</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37685874/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230926205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the era of personalized medicine greatly improved by molecular diagnosis and tailor-made therapies, the survival rate of acute myeloid leukemia (AML) at 5 years remains unfortunately low. Indeed, the high heterogeneity of AML clones with distinct metabolic and molecular profiles allows them to survive the chemotherapy-induced changes, thus leading to resistance, clonal evolution, and relapse. Moreover, leukemic stem cells (LSCs), the quiescent reservoir of residual disease, can persist for a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37686627"
                        open
                      >
                        <summary class="article-expander__title">Challenges and Prospects of Patient-Derived Xenografts for Cancer Research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37686627/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230925205014&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We discuss the importance of the in vivo models in elucidating cancer biology, focusing on the patient-derived xenograft (PDX) models, which are classic and standard functional in vivo platforms for preclinical evaluation. We provide an overview of the most representative models, including cell-derived xenografts (CDX), tumor and metastatic cell-derived xenografts, and PDX models utilizing humanized mice (HM). The orthotopic models, which could reproduce the cancer environment and its...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37685874"
                        open
                      >
                        <summary class="article-expander__title">A Leukemic Target with a Thousand Faces: The Mitochondria</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37685874/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230925205014&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the era of personalized medicine greatly improved by molecular diagnosis and tailor-made therapies, the survival rate of acute myeloid leukemia (AML) at 5 years remains unfortunately low. Indeed, the high heterogeneity of AML clones with distinct metabolic and molecular profiles allows them to survive the chemotherapy-induced changes, thus leading to resistance, clonal evolution, and relapse. Moreover, leukemic stem cells (LSCs), the quiescent reservoir of residual disease, can persist for a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37686627"
                        open
                      >
                        <summary class="article-expander__title">Challenges and Prospects of Patient-Derived Xenografts for Cancer Research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37686627/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230924205014&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We discuss the importance of the in vivo models in elucidating cancer biology, focusing on the patient-derived xenograft (PDX) models, which are classic and standard functional in vivo platforms for preclinical evaluation. We provide an overview of the most representative models, including cell-derived xenografts (CDX), tumor and metastatic cell-derived xenografts, and PDX models utilizing humanized mice (HM). The orthotopic models, which could reproduce the cancer environment and its...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37685874"
                        open
                      >
                        <summary class="article-expander__title">A Leukemic Target with a Thousand Faces: The Mitochondria</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37685874/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230924205014&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the era of personalized medicine greatly improved by molecular diagnosis and tailor-made therapies, the survival rate of acute myeloid leukemia (AML) at 5 years remains unfortunately low. Indeed, the high heterogeneity of AML clones with distinct metabolic and molecular profiles allows them to survive the chemotherapy-induced changes, thus leading to resistance, clonal evolution, and relapse. Moreover, leukemic stem cells (LSCs), the quiescent reservoir of residual disease, can persist for a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37686627"
                        open
                      >
                        <summary class="article-expander__title">Challenges and Prospects of Patient-Derived Xenografts for Cancer Research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37686627/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230923205208&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We discuss the importance of the in vivo models in elucidating cancer biology, focusing on the patient-derived xenograft (PDX) models, which are classic and standard functional in vivo platforms for preclinical evaluation. We provide an overview of the most representative models, including cell-derived xenografts (CDX), tumor and metastatic cell-derived xenografts, and PDX models utilizing humanized mice (HM). The orthotopic models, which could reproduce the cancer environment and its...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37685874"
                        open
                      >
                        <summary class="article-expander__title">A Leukemic Target with a Thousand Faces: The Mitochondria</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37685874/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230923205208&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the era of personalized medicine greatly improved by molecular diagnosis and tailor-made therapies, the survival rate of acute myeloid leukemia (AML) at 5 years remains unfortunately low. Indeed, the high heterogeneity of AML clones with distinct metabolic and molecular profiles allows them to survive the chemotherapy-induced changes, thus leading to resistance, clonal evolution, and relapse. Moreover, leukemic stem cells (LSCs), the quiescent reservoir of residual disease, can persist for a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37686627"
                        open
                      >
                        <summary class="article-expander__title">Challenges and Prospects of Patient-Derived Xenografts for Cancer Research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37686627/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230922204741&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We discuss the importance of the in vivo models in elucidating cancer biology, focusing on the patient-derived xenograft (PDX) models, which are classic and standard functional in vivo platforms for preclinical evaluation. We provide an overview of the most representative models, including cell-derived xenografts (CDX), tumor and metastatic cell-derived xenografts, and PDX models utilizing humanized mice (HM). The orthotopic models, which could reproduce the cancer environment and its...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37685874"
                        open
                      >
                        <summary class="article-expander__title">A Leukemic Target with a Thousand Faces: The Mitochondria</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37685874/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230922204741&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the era of personalized medicine greatly improved by molecular diagnosis and tailor-made therapies, the survival rate of acute myeloid leukemia (AML) at 5 years remains unfortunately low. Indeed, the high heterogeneity of AML clones with distinct metabolic and molecular profiles allows them to survive the chemotherapy-induced changes, thus leading to resistance, clonal evolution, and relapse. Moreover, leukemic stem cells (LSCs), the quiescent reservoir of residual disease, can persist for a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37686627"
                        open
                      >
                        <summary class="article-expander__title">Challenges and Prospects of Patient-Derived Xenografts for Cancer Research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37686627/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230921204936&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We discuss the importance of the in vivo models in elucidating cancer biology, focusing on the patient-derived xenograft (PDX) models, which are classic and standard functional in vivo platforms for preclinical evaluation. We provide an overview of the most representative models, including cell-derived xenografts (CDX), tumor and metastatic cell-derived xenografts, and PDX models utilizing humanized mice (HM). The orthotopic models, which could reproduce the cancer environment and its...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37685874"
                        open
                      >
                        <summary class="article-expander__title">A Leukemic Target with a Thousand Faces: The Mitochondria</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37685874/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230921204936&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the era of personalized medicine greatly improved by molecular diagnosis and tailor-made therapies, the survival rate of acute myeloid leukemia (AML) at 5 years remains unfortunately low. Indeed, the high heterogeneity of AML clones with distinct metabolic and molecular profiles allows them to survive the chemotherapy-induced changes, thus leading to resistance, clonal evolution, and relapse. Moreover, leukemic stem cells (LSCs), the quiescent reservoir of residual disease, can persist for a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37686627"
                        open
                      >
                        <summary class="article-expander__title">Challenges and Prospects of Patient-Derived Xenografts for Cancer Research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37686627/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230920204904&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We discuss the importance of the in vivo models in elucidating cancer biology, focusing on the patient-derived xenograft (PDX) models, which are classic and standard functional in vivo platforms for preclinical evaluation. We provide an overview of the most representative models, including cell-derived xenografts (CDX), tumor and metastatic cell-derived xenografts, and PDX models utilizing humanized mice (HM). The orthotopic models, which could reproduce the cancer environment and its...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37685874"
                        open
                      >
                        <summary class="article-expander__title">A Leukemic Target with a Thousand Faces: The Mitochondria</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37685874/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230920204904&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the era of personalized medicine greatly improved by molecular diagnosis and tailor-made therapies, the survival rate of acute myeloid leukemia (AML) at 5 years remains unfortunately low. Indeed, the high heterogeneity of AML clones with distinct metabolic and molecular profiles allows them to survive the chemotherapy-induced changes, thus leading to resistance, clonal evolution, and relapse. Moreover, leukemic stem cells (LSCs), the quiescent reservoir of residual disease, can persist for a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37686627"
                        open
                      >
                        <summary class="article-expander__title">Challenges and Prospects of Patient-Derived Xenografts for Cancer Research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37686627/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230919204929&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We discuss the importance of the in vivo models in elucidating cancer biology, focusing on the patient-derived xenograft (PDX) models, which are classic and standard functional in vivo platforms for preclinical evaluation. We provide an overview of the most representative models, including cell-derived xenografts (CDX), tumor and metastatic cell-derived xenografts, and PDX models utilizing humanized mice (HM). The orthotopic models, which could reproduce the cancer environment and its...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37685874"
                        open
                      >
                        <summary class="article-expander__title">A Leukemic Target with a Thousand Faces: The Mitochondria</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37685874/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230919204929&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the era of personalized medicine greatly improved by molecular diagnosis and tailor-made therapies, the survival rate of acute myeloid leukemia (AML) at 5 years remains unfortunately low. Indeed, the high heterogeneity of AML clones with distinct metabolic and molecular profiles allows them to survive the chemotherapy-induced changes, thus leading to resistance, clonal evolution, and relapse. Moreover, leukemic stem cells (LSCs), the quiescent reservoir of residual disease, can persist for a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37686627"
                        open
                      >
                        <summary class="article-expander__title">Challenges and Prospects of Patient-Derived Xenografts for Cancer Research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37686627/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230918204942&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We discuss the importance of the in vivo models in elucidating cancer biology, focusing on the patient-derived xenograft (PDX) models, which are classic and standard functional in vivo platforms for preclinical evaluation. We provide an overview of the most representative models, including cell-derived xenografts (CDX), tumor and metastatic cell-derived xenografts, and PDX models utilizing humanized mice (HM). The orthotopic models, which could reproduce the cancer environment and its...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37685874"
                        open
                      >
                        <summary class="article-expander__title">A Leukemic Target with a Thousand Faces: The Mitochondria</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37685874/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230918204942&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the era of personalized medicine greatly improved by molecular diagnosis and tailor-made therapies, the survival rate of acute myeloid leukemia (AML) at 5 years remains unfortunately low. Indeed, the high heterogeneity of AML clones with distinct metabolic and molecular profiles allows them to survive the chemotherapy-induced changes, thus leading to resistance, clonal evolution, and relapse. Moreover, leukemic stem cells (LSCs), the quiescent reservoir of residual disease, can persist for a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37686627"
                        open
                      >
                        <summary class="article-expander__title">Challenges and Prospects of Patient-Derived Xenografts for Cancer Research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37686627/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230917204936&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We discuss the importance of the in vivo models in elucidating cancer biology, focusing on the patient-derived xenograft (PDX) models, which are classic and standard functional in vivo platforms for preclinical evaluation. We provide an overview of the most representative models, including cell-derived xenografts (CDX), tumor and metastatic cell-derived xenografts, and PDX models utilizing humanized mice (HM). The orthotopic models, which could reproduce the cancer environment and its...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37685874"
                        open
                      >
                        <summary class="article-expander__title">A Leukemic Target with a Thousand Faces: The Mitochondria</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37685874/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230917204936&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the era of personalized medicine greatly improved by molecular diagnosis and tailor-made therapies, the survival rate of acute myeloid leukemia (AML) at 5 years remains unfortunately low. Indeed, the high heterogeneity of AML clones with distinct metabolic and molecular profiles allows them to survive the chemotherapy-induced changes, thus leading to resistance, clonal evolution, and relapse. Moreover, leukemic stem cells (LSCs), the quiescent reservoir of residual disease, can persist for a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37686627"
                        open
                      >
                        <summary class="article-expander__title">Challenges and Prospects of Patient-Derived Xenografts for Cancer Research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37686627/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230916205201&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We discuss the importance of the in vivo models in elucidating cancer biology, focusing on the patient-derived xenograft (PDX) models, which are classic and standard functional in vivo platforms for preclinical evaluation. We provide an overview of the most representative models, including cell-derived xenografts (CDX), tumor and metastatic cell-derived xenografts, and PDX models utilizing humanized mice (HM). The orthotopic models, which could reproduce the cancer environment and its...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37685874"
                        open
                      >
                        <summary class="article-expander__title">A Leukemic Target with a Thousand Faces: The Mitochondria</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37685874/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230916205201&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the era of personalized medicine greatly improved by molecular diagnosis and tailor-made therapies, the survival rate of acute myeloid leukemia (AML) at 5 years remains unfortunately low. Indeed, the high heterogeneity of AML clones with distinct metabolic and molecular profiles allows them to survive the chemotherapy-induced changes, thus leading to resistance, clonal evolution, and relapse. Moreover, leukemic stem cells (LSCs), the quiescent reservoir of residual disease, can persist for a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37686627"
                        open
                      >
                        <summary class="article-expander__title">Challenges and Prospects of Patient-Derived Xenografts for Cancer Research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37686627/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230915204741&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We discuss the importance of the in vivo models in elucidating cancer biology, focusing on the patient-derived xenograft (PDX) models, which are classic and standard functional in vivo platforms for preclinical evaluation. We provide an overview of the most representative models, including cell-derived xenografts (CDX), tumor and metastatic cell-derived xenografts, and PDX models utilizing humanized mice (HM). The orthotopic models, which could reproduce the cancer environment and its...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37685874"
                        open
                      >
                        <summary class="article-expander__title">A Leukemic Target with a Thousand Faces: The Mitochondria</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37685874/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230915204741&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the era of personalized medicine greatly improved by molecular diagnosis and tailor-made therapies, the survival rate of acute myeloid leukemia (AML) at 5 years remains unfortunately low. Indeed, the high heterogeneity of AML clones with distinct metabolic and molecular profiles allows them to survive the chemotherapy-induced changes, thus leading to resistance, clonal evolution, and relapse. Moreover, leukemic stem cells (LSCs), the quiescent reservoir of residual disease, can persist for a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37686627"
                        open
                      >
                        <summary class="article-expander__title">Challenges and Prospects of Patient-Derived Xenografts for Cancer Research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37686627/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230914204950&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We discuss the importance of the in vivo models in elucidating cancer biology, focusing on the patient-derived xenograft (PDX) models, which are classic and standard functional in vivo platforms for preclinical evaluation. We provide an overview of the most representative models, including cell-derived xenografts (CDX), tumor and metastatic cell-derived xenografts, and PDX models utilizing humanized mice (HM). The orthotopic models, which could reproduce the cancer environment and its...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37685874"
                        open
                      >
                        <summary class="article-expander__title">A Leukemic Target with a Thousand Faces: The Mitochondria</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37685874/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230914204950&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the era of personalized medicine greatly improved by molecular diagnosis and tailor-made therapies, the survival rate of acute myeloid leukemia (AML) at 5 years remains unfortunately low. Indeed, the high heterogeneity of AML clones with distinct metabolic and molecular profiles allows them to survive the chemotherapy-induced changes, thus leading to resistance, clonal evolution, and relapse. Moreover, leukemic stem cells (LSCs), the quiescent reservoir of residual disease, can persist for a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37686627"
                        open
                      >
                        <summary class="article-expander__title">Challenges and Prospects of Patient-Derived Xenografts for Cancer Research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37686627/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230913204840&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We discuss the importance of the in vivo models in elucidating cancer biology, focusing on the patient-derived xenograft (PDX) models, which are classic and standard functional in vivo platforms for preclinical evaluation. We provide an overview of the most representative models, including cell-derived xenografts (CDX), tumor and metastatic cell-derived xenografts, and PDX models utilizing humanized mice (HM). The orthotopic models, which could reproduce the cancer environment and its...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37685874"
                        open
                      >
                        <summary class="article-expander__title">A Leukemic Target with a Thousand Faces: The Mitochondria</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37685874/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230913204840&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the era of personalized medicine greatly improved by molecular diagnosis and tailor-made therapies, the survival rate of acute myeloid leukemia (AML) at 5 years remains unfortunately low. Indeed, the high heterogeneity of AML clones with distinct metabolic and molecular profiles allows them to survive the chemotherapy-induced changes, thus leading to resistance, clonal evolution, and relapse. Moreover, leukemic stem cells (LSCs), the quiescent reservoir of residual disease, can persist for a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37686627"
                        open
                      >
                        <summary class="article-expander__title">Challenges and Prospects of Patient-Derived Xenografts for Cancer Research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37686627/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230912205015&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We discuss the importance of the in vivo models in elucidating cancer biology, focusing on the patient-derived xenograft (PDX) models, which are classic and standard functional in vivo platforms for preclinical evaluation. We provide an overview of the most representative models, including cell-derived xenografts (CDX), tumor and metastatic cell-derived xenografts, and PDX models utilizing humanized mice (HM). The orthotopic models, which could reproduce the cancer environment and its...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37685874"
                        open
                      >
                        <summary class="article-expander__title">A Leukemic Target with a Thousand Faces: The Mitochondria</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37685874/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230912205015&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the era of personalized medicine greatly improved by molecular diagnosis and tailor-made therapies, the survival rate of acute myeloid leukemia (AML) at 5 years remains unfortunately low. Indeed, the high heterogeneity of AML clones with distinct metabolic and molecular profiles allows them to survive the chemotherapy-induced changes, thus leading to resistance, clonal evolution, and relapse. Moreover, leukemic stem cells (LSCs), the quiescent reservoir of residual disease, can persist for a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37686627"
                        open
                      >
                        <summary class="article-expander__title">Challenges and Prospects of Patient-Derived Xenografts for Cancer Research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37686627/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230911204717&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We discuss the importance of the in vivo models in elucidating cancer biology, focusing on the patient-derived xenograft (PDX) models, which are classic and standard functional in vivo platforms for preclinical evaluation. We provide an overview of the most representative models, including cell-derived xenografts (CDX), tumor and metastatic cell-derived xenografts, and PDX models utilizing humanized mice (HM). The orthotopic models, which could reproduce the cancer environment and its...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37685874"
                        open
                      >
                        <summary class="article-expander__title">A Leukemic Target with a Thousand Faces: The Mitochondria</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37685874/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230911204717&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the era of personalized medicine greatly improved by molecular diagnosis and tailor-made therapies, the survival rate of acute myeloid leukemia (AML) at 5 years remains unfortunately low. Indeed, the high heterogeneity of AML clones with distinct metabolic and molecular profiles allows them to survive the chemotherapy-induced changes, thus leading to resistance, clonal evolution, and relapse. Moreover, leukemic stem cells (LSCs), the quiescent reservoir of residual disease, can persist for a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37686627"
                        open
                      >
                        <summary class="article-expander__title">Challenges and Prospects of Patient-Derived Xenografts for Cancer Research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37686627/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230910204912&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We discuss the importance of the in vivo models in elucidating cancer biology, focusing on the patient-derived xenograft (PDX) models, which are classic and standard functional in vivo platforms for preclinical evaluation. We provide an overview of the most representative models, including cell-derived xenografts (CDX), tumor and metastatic cell-derived xenografts, and PDX models utilizing humanized mice (HM). The orthotopic models, which could reproduce the cancer environment and its...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37686627"
                        open
                      >
                        <summary class="article-expander__title">Challenges and Prospects of Patient-Derived Xenografts for Cancer Research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37686627/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230909205145&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We discuss the importance of the in vivo models in elucidating cancer biology, focusing on the patient-derived xenograft (PDX) models, which are classic and standard functional in vivo platforms for preclinical evaluation. We provide an overview of the most representative models, including cell-derived xenografts (CDX), tumor and metastatic cell-derived xenografts, and PDX models utilizing humanized mice (HM). The orthotopic models, which could reproduce the cancer environment and its...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-09-08 datetime="2023-09-08T10:00:00.000Z">2023-09-08</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-09-08 datetime="2023-09-08T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;utm_medium&#x3D;rss&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20231105195156">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37684235"
                        open
                      >
                        <summary class="article-expander__title">Shared and distinct genetic etiologies for different types of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37684235/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230926205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH)-age-related expansion of mutated hematopoietic clones-can differ in frequency and cellular fitness by CH type (e.g., mutations in driver genes (CHIP), gains/losses and copy-neutral loss of chromosomal segments (mCAs), and loss of sex chromosomes). Co-occurring CH raises questions as to their origin, selection, and impact. We integrate sequence and genotype array data in up to 482,378 UK Biobank participants to demonstrate shared genetic architecture across CH types. Our...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37684235"
                        open
                      >
                        <summary class="article-expander__title">Shared and distinct genetic etiologies for different types of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37684235/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230925205009&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH)-age-related expansion of mutated hematopoietic clones-can differ in frequency and cellular fitness by CH type (e.g., mutations in driver genes (CHIP), gains/losses and copy-neutral loss of chromosomal segments (mCAs), and loss of sex chromosomes). Co-occurring CH raises questions as to their origin, selection, and impact. We integrate sequence and genotype array data in up to 482,378 UK Biobank participants to demonstrate shared genetic architecture across CH types. Our...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37681311"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Clinical and Experimental Heart Failure With Preserved Ejection Fraction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37681311/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230925205009&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH is associated with worse heart function and prognosis in patients with HFpEF, and a murine experimental model of Tet2-mediated CH displays greater features of HFpEF.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37684235"
                        open
                      >
                        <summary class="article-expander__title">Shared and distinct genetic etiologies for different types of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37684235/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230924205014&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH)-age-related expansion of mutated hematopoietic clones-can differ in frequency and cellular fitness by CH type (e.g., mutations in driver genes (CHIP), gains/losses and copy-neutral loss of chromosomal segments (mCAs), and loss of sex chromosomes). Co-occurring CH raises questions as to their origin, selection, and impact. We integrate sequence and genotype array data in up to 482,378 UK Biobank participants to demonstrate shared genetic architecture across CH types. Our...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37681311"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Clinical and Experimental Heart Failure With Preserved Ejection Fraction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37681311/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230924205014&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH is associated with worse heart function and prognosis in patients with HFpEF, and a murine experimental model of Tet2-mediated CH displays greater features of HFpEF.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37684235"
                        open
                      >
                        <summary class="article-expander__title">Shared and distinct genetic etiologies for different types of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37684235/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230923205208&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH)-age-related expansion of mutated hematopoietic clones-can differ in frequency and cellular fitness by CH type (e.g., mutations in driver genes (CHIP), gains/losses and copy-neutral loss of chromosomal segments (mCAs), and loss of sex chromosomes). Co-occurring CH raises questions as to their origin, selection, and impact. We integrate sequence and genotype array data in up to 482,378 UK Biobank participants to demonstrate shared genetic architecture across CH types. Our...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37681311"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Clinical and Experimental Heart Failure With Preserved Ejection Fraction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37681311/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230923205208&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH is associated with worse heart function and prognosis in patients with HFpEF, and a murine experimental model of Tet2-mediated CH displays greater features of HFpEF.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37684235"
                        open
                      >
                        <summary class="article-expander__title">Shared and distinct genetic etiologies for different types of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37684235/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230922204741&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH)-age-related expansion of mutated hematopoietic clones-can differ in frequency and cellular fitness by CH type (e.g., mutations in driver genes (CHIP), gains/losses and copy-neutral loss of chromosomal segments (mCAs), and loss of sex chromosomes). Co-occurring CH raises questions as to their origin, selection, and impact. We integrate sequence and genotype array data in up to 482,378 UK Biobank participants to demonstrate shared genetic architecture across CH types. Our...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37681311"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Clinical and Experimental Heart Failure With Preserved Ejection Fraction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37681311/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230922204741&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH is associated with worse heart function and prognosis in patients with HFpEF, and a murine experimental model of Tet2-mediated CH displays greater features of HFpEF.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37684235"
                        open
                      >
                        <summary class="article-expander__title">Shared and distinct genetic etiologies for different types of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37684235/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230921204936&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH)-age-related expansion of mutated hematopoietic clones-can differ in frequency and cellular fitness by CH type (e.g., mutations in driver genes (CHIP), gains/losses and copy-neutral loss of chromosomal segments (mCAs), and loss of sex chromosomes). Co-occurring CH raises questions as to their origin, selection, and impact. We integrate sequence and genotype array data in up to 482,378 UK Biobank participants to demonstrate shared genetic architecture across CH types. Our...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37681311"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Clinical and Experimental Heart Failure With Preserved Ejection Fraction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37681311/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230921204936&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH is associated with worse heart function and prognosis in patients with HFpEF, and a murine experimental model of Tet2-mediated CH displays greater features of HFpEF.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37684235"
                        open
                      >
                        <summary class="article-expander__title">Shared and distinct genetic etiologies for different types of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37684235/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230920204905&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH)-age-related expansion of mutated hematopoietic clones-can differ in frequency and cellular fitness by CH type (e.g., mutations in driver genes (CHIP), gains/losses and copy-neutral loss of chromosomal segments (mCAs), and loss of sex chromosomes). Co-occurring CH raises questions as to their origin, selection, and impact. We integrate sequence and genotype array data in up to 482,378 UK Biobank participants to demonstrate shared genetic architecture across CH types. Our...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37681311"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Clinical and Experimental Heart Failure With Preserved Ejection Fraction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37681311/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230920204905&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH is associated with worse heart function and prognosis in patients with HFpEF, and a murine experimental model of Tet2-mediated CH displays greater features of HFpEF.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37684235"
                        open
                      >
                        <summary class="article-expander__title">Shared and distinct genetic etiologies for different types of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37684235/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230919204929&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH)-age-related expansion of mutated hematopoietic clones-can differ in frequency and cellular fitness by CH type (e.g., mutations in driver genes (CHIP), gains/losses and copy-neutral loss of chromosomal segments (mCAs), and loss of sex chromosomes). Co-occurring CH raises questions as to their origin, selection, and impact. We integrate sequence and genotype array data in up to 482,378 UK Biobank participants to demonstrate shared genetic architecture across CH types. Our...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37681311"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Clinical and Experimental Heart Failure With Preserved Ejection Fraction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37681311/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230919204929&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH is associated with worse heart function and prognosis in patients with HFpEF, and a murine experimental model of Tet2-mediated CH displays greater features of HFpEF.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37684235"
                        open
                      >
                        <summary class="article-expander__title">Shared and distinct genetic etiologies for different types of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37684235/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230918204942&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH)-age-related expansion of mutated hematopoietic clones-can differ in frequency and cellular fitness by CH type (e.g., mutations in driver genes (CHIP), gains/losses and copy-neutral loss of chromosomal segments (mCAs), and loss of sex chromosomes). Co-occurring CH raises questions as to their origin, selection, and impact. We integrate sequence and genotype array data in up to 482,378 UK Biobank participants to demonstrate shared genetic architecture across CH types. Our...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37681311"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Clinical and Experimental Heart Failure With Preserved Ejection Fraction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37681311/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230918204942&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH is associated with worse heart function and prognosis in patients with HFpEF, and a murine experimental model of Tet2-mediated CH displays greater features of HFpEF.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37684235"
                        open
                      >
                        <summary class="article-expander__title">Shared and distinct genetic etiologies for different types of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37684235/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230917204935&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH)-age-related expansion of mutated hematopoietic clones-can differ in frequency and cellular fitness by CH type (e.g., mutations in driver genes (CHIP), gains/losses and copy-neutral loss of chromosomal segments (mCAs), and loss of sex chromosomes). Co-occurring CH raises questions as to their origin, selection, and impact. We integrate sequence and genotype array data in up to 482,378 UK Biobank participants to demonstrate shared genetic architecture across CH types. Our...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37681311"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Clinical and Experimental Heart Failure With Preserved Ejection Fraction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37681311/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230917204935&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH is associated with worse heart function and prognosis in patients with HFpEF, and a murine experimental model of Tet2-mediated CH displays greater features of HFpEF.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37684235"
                        open
                      >
                        <summary class="article-expander__title">Shared and distinct genetic etiologies for different types of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37684235/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230916205201&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH)-age-related expansion of mutated hematopoietic clones-can differ in frequency and cellular fitness by CH type (e.g., mutations in driver genes (CHIP), gains/losses and copy-neutral loss of chromosomal segments (mCAs), and loss of sex chromosomes). Co-occurring CH raises questions as to their origin, selection, and impact. We integrate sequence and genotype array data in up to 482,378 UK Biobank participants to demonstrate shared genetic architecture across CH types. Our...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37681311"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Clinical and Experimental Heart Failure With Preserved Ejection Fraction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37681311/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230916205201&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH is associated with worse heart function and prognosis in patients with HFpEF, and a murine experimental model of Tet2-mediated CH displays greater features of HFpEF.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37684235"
                        open
                      >
                        <summary class="article-expander__title">Shared and distinct genetic etiologies for different types of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37684235/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230915204741&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH)-age-related expansion of mutated hematopoietic clones-can differ in frequency and cellular fitness by CH type (e.g., mutations in driver genes (CHIP), gains/losses and copy-neutral loss of chromosomal segments (mCAs), and loss of sex chromosomes). Co-occurring CH raises questions as to their origin, selection, and impact. We integrate sequence and genotype array data in up to 482,378 UK Biobank participants to demonstrate shared genetic architecture across CH types. Our...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37681311"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Clinical and Experimental Heart Failure With Preserved Ejection Fraction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37681311/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230915204741&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH is associated with worse heart function and prognosis in patients with HFpEF, and a murine experimental model of Tet2-mediated CH displays greater features of HFpEF.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37684235"
                        open
                      >
                        <summary class="article-expander__title">Shared and distinct genetic etiologies for different types of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37684235/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230914204950&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH)-age-related expansion of mutated hematopoietic clones-can differ in frequency and cellular fitness by CH type (e.g., mutations in driver genes (CHIP), gains/losses and copy-neutral loss of chromosomal segments (mCAs), and loss of sex chromosomes). Co-occurring CH raises questions as to their origin, selection, and impact. We integrate sequence and genotype array data in up to 482,378 UK Biobank participants to demonstrate shared genetic architecture across CH types. Our...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37681311"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Clinical and Experimental Heart Failure With Preserved Ejection Fraction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37681311/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230914204950&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH is associated with worse heart function and prognosis in patients with HFpEF, and a murine experimental model of Tet2-mediated CH displays greater features of HFpEF.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37684235"
                        open
                      >
                        <summary class="article-expander__title">Shared and distinct genetic etiologies for different types of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37684235/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230913204840&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH)-age-related expansion of mutated hematopoietic clones-can differ in frequency and cellular fitness by CH type (e.g., mutations in driver genes (CHIP), gains/losses and copy-neutral loss of chromosomal segments (mCAs), and loss of sex chromosomes). Co-occurring CH raises questions as to their origin, selection, and impact. We integrate sequence and genotype array data in up to 482,378 UK Biobank participants to demonstrate shared genetic architecture across CH types. Our...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37681311"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Clinical and Experimental Heart Failure With Preserved Ejection Fraction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37681311/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230913204840&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH is associated with worse heart function and prognosis in patients with HFpEF, and a murine experimental model of Tet2-mediated CH displays greater features of HFpEF.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37684235"
                        open
                      >
                        <summary class="article-expander__title">Shared and distinct genetic etiologies for different types of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37684235/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230912205016&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH)-age-related expansion of mutated hematopoietic clones-can differ in frequency and cellular fitness by CH type (e.g., mutations in driver genes (CHIP), gains/losses and copy-neutral loss of chromosomal segments (mCAs), and loss of sex chromosomes). Co-occurring CH raises questions as to their origin, selection, and impact. We integrate sequence and genotype array data in up to 482,378 UK Biobank participants to demonstrate shared genetic architecture across CH types. Our...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37681311"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Clinical and Experimental Heart Failure With Preserved Ejection Fraction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37681311/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230912205016&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH is associated with worse heart function and prognosis in patients with HFpEF, and a murine experimental model of Tet2-mediated CH displays greater features of HFpEF.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37684235"
                        open
                      >
                        <summary class="article-expander__title">Shared and distinct genetic etiologies for different types of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37684235/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230911204717&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH)-age-related expansion of mutated hematopoietic clones-can differ in frequency and cellular fitness by CH type (e.g., mutations in driver genes (CHIP), gains/losses and copy-neutral loss of chromosomal segments (mCAs), and loss of sex chromosomes). Co-occurring CH raises questions as to their origin, selection, and impact. We integrate sequence and genotype array data in up to 482,378 UK Biobank participants to demonstrate shared genetic architecture across CH types. Our...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37681311"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Clinical and Experimental Heart Failure With Preserved Ejection Fraction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37681311/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230911204717&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH is associated with worse heart function and prognosis in patients with HFpEF, and a murine experimental model of Tet2-mediated CH displays greater features of HFpEF.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37684235"
                        open
                      >
                        <summary class="article-expander__title">Shared and distinct genetic etiologies for different types of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37684235/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230910204912&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH)-age-related expansion of mutated hematopoietic clones-can differ in frequency and cellular fitness by CH type (e.g., mutations in driver genes (CHIP), gains/losses and copy-neutral loss of chromosomal segments (mCAs), and loss of sex chromosomes). Co-occurring CH raises questions as to their origin, selection, and impact. We integrate sequence and genotype array data in up to 482,378 UK Biobank participants to demonstrate shared genetic architecture across CH types. Our...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37681311"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Clinical and Experimental Heart Failure With Preserved Ejection Fraction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37681311/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230910204912&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH is associated with worse heart function and prognosis in patients with HFpEF, and a murine experimental model of Tet2-mediated CH displays greater features of HFpEF.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37684235"
                        open
                      >
                        <summary class="article-expander__title">Shared and distinct genetic etiologies for different types of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37684235/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230909205145&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH)-age-related expansion of mutated hematopoietic clones-can differ in frequency and cellular fitness by CH type (e.g., mutations in driver genes (CHIP), gains/losses and copy-neutral loss of chromosomal segments (mCAs), and loss of sex chromosomes). Co-occurring CH raises questions as to their origin, selection, and impact. We integrate sequence and genotype array data in up to 482,378 UK Biobank participants to demonstrate shared genetic architecture across CH types. Our...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37681311"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Clinical and Experimental Heart Failure With Preserved Ejection Fraction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37681311/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230909205145&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH is associated with worse heart function and prognosis in patients with HFpEF, and a murine experimental model of Tet2-mediated CH displays greater features of HFpEF.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37681311"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis in Clinical and Experimental Heart Failure With Preserved Ejection Fraction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37681311/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20230908204701&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH is associated with worse heart function and prognosis in patients with HFpEF, and a murine experimental model of Tet2-mediated CH displays greater features of HFpEF.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;ff&#x3D;20231105195156&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37684264"
                        open
                      >
                        <summary class="article-expander__title">Potential value of high-throughput single-cell DNA sequencing of Juvenile myelomonocytic leukemia: report of two cases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37684264/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231002205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Juvenile myelomonocytic leukemia (JMML) is a rare myeloproliferative disease of early childhood that develops due to mutations in the genes of the RAS-signaling pathway. Next-generation high throughput sequencing (NGS) enables identification of various secondary molecular genetic events that can facilitate JMML progression and transformation into secondary acute myeloid leukemia (sAML). The methods of single-cell DNA sequencing (scDNA-seq) enable overcoming limitations of bulk NGS and exploring...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37684264"
                        open
                      >
                        <summary class="article-expander__title">Potential value of high-throughput single-cell DNA sequencing of Juvenile myelomonocytic leukemia: report of two cases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37684264/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231001205031&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Juvenile myelomonocytic leukemia (JMML) is a rare myeloproliferative disease of early childhood that develops due to mutations in the genes of the RAS-signaling pathway. Next-generation high throughput sequencing (NGS) enables identification of various secondary molecular genetic events that can facilitate JMML progression and transformation into secondary acute myeloid leukemia (sAML). The methods of single-cell DNA sequencing (scDNA-seq) enable overcoming limitations of bulk NGS and exploring...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37684264"
                        open
                      >
                        <summary class="article-expander__title">Potential value of high-throughput single-cell DNA sequencing of Juvenile myelomonocytic leukemia: report of two cases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37684264/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230930205630&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Juvenile myelomonocytic leukemia (JMML) is a rare myeloproliferative disease of early childhood that develops due to mutations in the genes of the RAS-signaling pathway. Next-generation high throughput sequencing (NGS) enables identification of various secondary molecular genetic events that can facilitate JMML progression and transformation into secondary acute myeloid leukemia (sAML). The methods of single-cell DNA sequencing (scDNA-seq) enable overcoming limitations of bulk NGS and exploring...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37684264"
                        open
                      >
                        <summary class="article-expander__title">Potential value of high-throughput single-cell DNA sequencing of Juvenile myelomonocytic leukemia: report of two cases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37684264/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230929204823&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Juvenile myelomonocytic leukemia (JMML) is a rare myeloproliferative disease of early childhood that develops due to mutations in the genes of the RAS-signaling pathway. Next-generation high throughput sequencing (NGS) enables identification of various secondary molecular genetic events that can facilitate JMML progression and transformation into secondary acute myeloid leukemia (sAML). The methods of single-cell DNA sequencing (scDNA-seq) enable overcoming limitations of bulk NGS and exploring...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37684264"
                        open
                      >
                        <summary class="article-expander__title">Potential value of high-throughput single-cell DNA sequencing of Juvenile myelomonocytic leukemia: report of two cases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37684264/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230928204948&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Juvenile myelomonocytic leukemia (JMML) is a rare myeloproliferative disease of early childhood that develops due to mutations in the genes of the RAS-signaling pathway. Next-generation high throughput sequencing (NGS) enables identification of various secondary molecular genetic events that can facilitate JMML progression and transformation into secondary acute myeloid leukemia (sAML). The methods of single-cell DNA sequencing (scDNA-seq) enable overcoming limitations of bulk NGS and exploring...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37684264"
                        open
                      >
                        <summary class="article-expander__title">Potential value of high-throughput single-cell DNA sequencing of Juvenile myelomonocytic leukemia: report of two cases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37684264/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230927204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Juvenile myelomonocytic leukemia (JMML) is a rare myeloproliferative disease of early childhood that develops due to mutations in the genes of the RAS-signaling pathway. Next-generation high throughput sequencing (NGS) enables identification of various secondary molecular genetic events that can facilitate JMML progression and transformation into secondary acute myeloid leukemia (sAML). The methods of single-cell DNA sequencing (scDNA-seq) enable overcoming limitations of bulk NGS and exploring...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37684264"
                        open
                      >
                        <summary class="article-expander__title">Potential value of high-throughput single-cell DNA sequencing of Juvenile myelomonocytic leukemia: report of two cases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37684264/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230926205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Juvenile myelomonocytic leukemia (JMML) is a rare myeloproliferative disease of early childhood that develops due to mutations in the genes of the RAS-signaling pathway. Next-generation high throughput sequencing (NGS) enables identification of various secondary molecular genetic events that can facilitate JMML progression and transformation into secondary acute myeloid leukemia (sAML). The methods of single-cell DNA sequencing (scDNA-seq) enable overcoming limitations of bulk NGS and exploring...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37684264"
                        open
                      >
                        <summary class="article-expander__title">Potential value of high-throughput single-cell DNA sequencing of Juvenile myelomonocytic leukemia: report of two cases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37684264/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230925205014&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Juvenile myelomonocytic leukemia (JMML) is a rare myeloproliferative disease of early childhood that develops due to mutations in the genes of the RAS-signaling pathway. Next-generation high throughput sequencing (NGS) enables identification of various secondary molecular genetic events that can facilitate JMML progression and transformation into secondary acute myeloid leukemia (sAML). The methods of single-cell DNA sequencing (scDNA-seq) enable overcoming limitations of bulk NGS and exploring...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37684264"
                        open
                      >
                        <summary class="article-expander__title">Potential value of high-throughput single-cell DNA sequencing of Juvenile myelomonocytic leukemia: report of two cases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37684264/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230924205014&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Juvenile myelomonocytic leukemia (JMML) is a rare myeloproliferative disease of early childhood that develops due to mutations in the genes of the RAS-signaling pathway. Next-generation high throughput sequencing (NGS) enables identification of various secondary molecular genetic events that can facilitate JMML progression and transformation into secondary acute myeloid leukemia (sAML). The methods of single-cell DNA sequencing (scDNA-seq) enable overcoming limitations of bulk NGS and exploring...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37684264"
                        open
                      >
                        <summary class="article-expander__title">Potential value of high-throughput single-cell DNA sequencing of Juvenile myelomonocytic leukemia: report of two cases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37684264/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230923205208&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Juvenile myelomonocytic leukemia (JMML) is a rare myeloproliferative disease of early childhood that develops due to mutations in the genes of the RAS-signaling pathway. Next-generation high throughput sequencing (NGS) enables identification of various secondary molecular genetic events that can facilitate JMML progression and transformation into secondary acute myeloid leukemia (sAML). The methods of single-cell DNA sequencing (scDNA-seq) enable overcoming limitations of bulk NGS and exploring...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37684264"
                        open
                      >
                        <summary class="article-expander__title">Potential value of high-throughput single-cell DNA sequencing of Juvenile myelomonocytic leukemia: report of two cases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37684264/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230922204741&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Juvenile myelomonocytic leukemia (JMML) is a rare myeloproliferative disease of early childhood that develops due to mutations in the genes of the RAS-signaling pathway. Next-generation high throughput sequencing (NGS) enables identification of various secondary molecular genetic events that can facilitate JMML progression and transformation into secondary acute myeloid leukemia (sAML). The methods of single-cell DNA sequencing (scDNA-seq) enable overcoming limitations of bulk NGS and exploring...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37684264"
                        open
                      >
                        <summary class="article-expander__title">Potential value of high-throughput single-cell DNA sequencing of Juvenile myelomonocytic leukemia: report of two cases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37684264/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230921204936&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Juvenile myelomonocytic leukemia (JMML) is a rare myeloproliferative disease of early childhood that develops due to mutations in the genes of the RAS-signaling pathway. Next-generation high throughput sequencing (NGS) enables identification of various secondary molecular genetic events that can facilitate JMML progression and transformation into secondary acute myeloid leukemia (sAML). The methods of single-cell DNA sequencing (scDNA-seq) enable overcoming limitations of bulk NGS and exploring...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37684264"
                        open
                      >
                        <summary class="article-expander__title">Potential value of high-throughput single-cell DNA sequencing of Juvenile myelomonocytic leukemia: report of two cases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37684264/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230920204904&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Juvenile myelomonocytic leukemia (JMML) is a rare myeloproliferative disease of early childhood that develops due to mutations in the genes of the RAS-signaling pathway. Next-generation high throughput sequencing (NGS) enables identification of various secondary molecular genetic events that can facilitate JMML progression and transformation into secondary acute myeloid leukemia (sAML). The methods of single-cell DNA sequencing (scDNA-seq) enable overcoming limitations of bulk NGS and exploring...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37684264"
                        open
                      >
                        <summary class="article-expander__title">Potential value of high-throughput single-cell DNA sequencing of Juvenile myelomonocytic leukemia: report of two cases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37684264/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230919204929&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Juvenile myelomonocytic leukemia (JMML) is a rare myeloproliferative disease of early childhood that develops due to mutations in the genes of the RAS-signaling pathway. Next-generation high throughput sequencing (NGS) enables identification of various secondary molecular genetic events that can facilitate JMML progression and transformation into secondary acute myeloid leukemia (sAML). The methods of single-cell DNA sequencing (scDNA-seq) enable overcoming limitations of bulk NGS and exploring...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37684264"
                        open
                      >
                        <summary class="article-expander__title">Potential value of high-throughput single-cell DNA sequencing of Juvenile myelomonocytic leukemia: report of two cases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37684264/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230918204942&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Juvenile myelomonocytic leukemia (JMML) is a rare myeloproliferative disease of early childhood that develops due to mutations in the genes of the RAS-signaling pathway. Next-generation high throughput sequencing (NGS) enables identification of various secondary molecular genetic events that can facilitate JMML progression and transformation into secondary acute myeloid leukemia (sAML). The methods of single-cell DNA sequencing (scDNA-seq) enable overcoming limitations of bulk NGS and exploring...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37684264"
                        open
                      >
                        <summary class="article-expander__title">Potential value of high-throughput single-cell DNA sequencing of Juvenile myelomonocytic leukemia: report of two cases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37684264/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230917204936&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Juvenile myelomonocytic leukemia (JMML) is a rare myeloproliferative disease of early childhood that develops due to mutations in the genes of the RAS-signaling pathway. Next-generation high throughput sequencing (NGS) enables identification of various secondary molecular genetic events that can facilitate JMML progression and transformation into secondary acute myeloid leukemia (sAML). The methods of single-cell DNA sequencing (scDNA-seq) enable overcoming limitations of bulk NGS and exploring...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37684264"
                        open
                      >
                        <summary class="article-expander__title">Potential value of high-throughput single-cell DNA sequencing of Juvenile myelomonocytic leukemia: report of two cases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37684264/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230916205201&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Juvenile myelomonocytic leukemia (JMML) is a rare myeloproliferative disease of early childhood that develops due to mutations in the genes of the RAS-signaling pathway. Next-generation high throughput sequencing (NGS) enables identification of various secondary molecular genetic events that can facilitate JMML progression and transformation into secondary acute myeloid leukemia (sAML). The methods of single-cell DNA sequencing (scDNA-seq) enable overcoming limitations of bulk NGS and exploring...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37684264"
                        open
                      >
                        <summary class="article-expander__title">Potential value of high-throughput single-cell DNA sequencing of Juvenile myelomonocytic leukemia: report of two cases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37684264/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230915204741&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Juvenile myelomonocytic leukemia (JMML) is a rare myeloproliferative disease of early childhood that develops due to mutations in the genes of the RAS-signaling pathway. Next-generation high throughput sequencing (NGS) enables identification of various secondary molecular genetic events that can facilitate JMML progression and transformation into secondary acute myeloid leukemia (sAML). The methods of single-cell DNA sequencing (scDNA-seq) enable overcoming limitations of bulk NGS and exploring...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37684264"
                        open
                      >
                        <summary class="article-expander__title">Potential value of high-throughput single-cell DNA sequencing of Juvenile myelomonocytic leukemia: report of two cases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37684264/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230914204950&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Juvenile myelomonocytic leukemia (JMML) is a rare myeloproliferative disease of early childhood that develops due to mutations in the genes of the RAS-signaling pathway. Next-generation high throughput sequencing (NGS) enables identification of various secondary molecular genetic events that can facilitate JMML progression and transformation into secondary acute myeloid leukemia (sAML). The methods of single-cell DNA sequencing (scDNA-seq) enable overcoming limitations of bulk NGS and exploring...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37684264"
                        open
                      >
                        <summary class="article-expander__title">Potential value of high-throughput single-cell DNA sequencing of Juvenile myelomonocytic leukemia: report of two cases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37684264/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230913204840&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Juvenile myelomonocytic leukemia (JMML) is a rare myeloproliferative disease of early childhood that develops due to mutations in the genes of the RAS-signaling pathway. Next-generation high throughput sequencing (NGS) enables identification of various secondary molecular genetic events that can facilitate JMML progression and transformation into secondary acute myeloid leukemia (sAML). The methods of single-cell DNA sequencing (scDNA-seq) enable overcoming limitations of bulk NGS and exploring...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37684264"
                        open
                      >
                        <summary class="article-expander__title">Potential value of high-throughput single-cell DNA sequencing of Juvenile myelomonocytic leukemia: report of two cases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37684264/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230912205015&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Juvenile myelomonocytic leukemia (JMML) is a rare myeloproliferative disease of early childhood that develops due to mutations in the genes of the RAS-signaling pathway. Next-generation high throughput sequencing (NGS) enables identification of various secondary molecular genetic events that can facilitate JMML progression and transformation into secondary acute myeloid leukemia (sAML). The methods of single-cell DNA sequencing (scDNA-seq) enable overcoming limitations of bulk NGS and exploring...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37684264"
                        open
                      >
                        <summary class="article-expander__title">Potential value of high-throughput single-cell DNA sequencing of Juvenile myelomonocytic leukemia: report of two cases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37684264/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230911204717&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Juvenile myelomonocytic leukemia (JMML) is a rare myeloproliferative disease of early childhood that develops due to mutations in the genes of the RAS-signaling pathway. Next-generation high throughput sequencing (NGS) enables identification of various secondary molecular genetic events that can facilitate JMML progression and transformation into secondary acute myeloid leukemia (sAML). The methods of single-cell DNA sequencing (scDNA-seq) enable overcoming limitations of bulk NGS and exploring...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37684264"
                        open
                      >
                        <summary class="article-expander__title">Potential value of high-throughput single-cell DNA sequencing of Juvenile myelomonocytic leukemia: report of two cases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37684264/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230910204912&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Juvenile myelomonocytic leukemia (JMML) is a rare myeloproliferative disease of early childhood that develops due to mutations in the genes of the RAS-signaling pathway. Next-generation high throughput sequencing (NGS) enables identification of various secondary molecular genetic events that can facilitate JMML progression and transformation into secondary acute myeloid leukemia (sAML). The methods of single-cell DNA sequencing (scDNA-seq) enable overcoming limitations of bulk NGS and exploring...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37684264"
                        open
                      >
                        <summary class="article-expander__title">Potential value of high-throughput single-cell DNA sequencing of Juvenile myelomonocytic leukemia: report of two cases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37684264/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20230909205145&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Juvenile myelomonocytic leukemia (JMML) is a rare myeloproliferative disease of early childhood that develops due to mutations in the genes of the RAS-signaling pathway. Next-generation high throughput sequencing (NGS) enables identification of various secondary molecular genetic events that can facilitate JMML progression and transformation into secondary acute myeloid leukemia (sAML). The methods of single-cell DNA sequencing (scDNA-seq) enable overcoming limitations of bulk NGS and exploring...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-06-06 datetime="2022-06-06T18:28:04.000Z">2022-06-06</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-06-06 datetime="2022-06-06T18:28:04.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="72a3a11d39c217b1a80cbb551444e6f6"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Nodal interferon exposure drives tumour-specific tolerance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1533878403463405569">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Nodal interferon exposure drives tumour-specific tolerance https://t.co/FT4NeBcFTg pic.twitter.com/NRHkVDUsOr
— NatureReviewsCancer (@NatureRevCancer) June 6, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-30 datetime="2022-05-30T04:37:42.000Z">2022-05-30</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-30 datetime="2022-05-30T04:37:42.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="bb6bc242bcbc356c4a38db850e74d812"
                        open
                      >
                        <summary class="article-expander__title">@VIBConferences: #Cancer #immunotherapy has become a crucial pillar of anticancer therapy. 19 leading researchers at #ImmunoOnco22 will give new insights into the immune response to tumors and tackle unresolved questions.⏰Registration Deadline: Thursday 2 June. Read more</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/VIBConferences/status/1531132720952266752">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>#Cancer #immunotherapy has become a crucial pillar of anticancer therapy. 19 leading researchers at #ImmunoOnco22 will give new insights into the immune response to tumors and tackle unresolved questions.
⏰Registration Deadline: Thursday 2 June. Read more https://t.co/cc0HC5cj6W pic.twitter.com/u8LFuzBbsV
— VIB Conferences (@VIBConferences) May 30, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-27 datetime="2022-05-27T14:45:03.000Z">2022-05-27</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-27 datetime="2022-05-27T14:45:03.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="ed6c996dc1a718afb523de6229a53131"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Nonsense-mediated RNA decay: an emerging modulator of malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1530198403686072320">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Nonsense-mediated RNA decay: an emerging modulator of malignancy https://t.co/wjuHplkpof pic.twitter.com/1SoKYhRsPO
— NatureReviewsCancer (@NatureRevCancer) May 27, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="d15a3a4b24853812d7fa21fd56de5815"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: Our June issue, out NOW! With articles on #therapyresistance, #senescence &amp; targeting CDK4 &amp; CDK6 in #cancer! Plus, a Comment on cancer mRNA vaccines as well as the launch of our new Tools of the Trade articles #NRCToT. So much content 👇🏻💌</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1530174481313972224">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Our June issue, out NOW! With articles on #therapyresistance, #senescence &amp; targeting CDK4 &amp; CDK6 in #cancer! Plus, a Comment on cancer mRNA vaccines as well as the launch of our new Tools of the Trade articles #NRCToT. So much content 👇🏻
💌 https://t.co/Fo6XRWV3OZ@natrescancer pic.twitter.com/rbxRPqLV6g
— NatureReviewsCancer (@NatureRevCancer) May 27, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-26 datetime="2022-05-26T08:53:50.000Z">2022-05-26</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-26 datetime="2022-05-26T08:53:50.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="fe13755a2693ea65459e5e21df097d2a"
                        open
                      >
                        <summary class="article-expander__title">@UlrikeHarjes: This Review by @MarcusDGon @cantleylab is part of the @NatureRevCancer series on #Diet &amp; #SystemicMetabolism in cancer:  do love this topic in general plus I am handling research at @NatureMedicine now.. get in touch if you have something of interest☺️</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/UlrikeHarjes/status/1529747629936193537">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>This Review by @MarcusDGon @cantleylab is part of the @NatureRevCancer series on #Diet &amp; #SystemicMetabolism in cancer: https://t.co/p3Y2cdNhco
I do love this topic in general plus I am handling research at @NatureMedicine now.. get in touch if you have something of interest☺️ https://t.co/qWH2dTv8j9
— Ulrike Harjes (@UlrikeHarjes) May 26, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-25 datetime="2022-05-25T17:35:04.000Z">2022-05-25</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-25 datetime="2022-05-25T17:35:04.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="9ba5279424b31b0122a4ae20c18e6d03"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Developing dietary interventions as therapy for cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1529516411059437568">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Developing dietary interventions as therapy for cancer https://t.co/ijuqctk1wJ pic.twitter.com/KHPpMKnCFa
— NatureReviewsCancer (@NatureRevCancer) May 25, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-18 datetime="2022-05-18T14:11:32.000Z">2022-05-18</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-18 datetime="2022-05-18T14:11:32.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="611d40f2e6ea2796b028bebda41a86dd"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Digging for treasures in the tumour interactome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1526928478490095616">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Digging for treasures in the tumour interactome https://t.co/NhHYAwUgLE
— NatureReviewsCancer (@NatureRevCancer) May 18, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-17 datetime="2022-05-17T13:48:21.000Z">2022-05-17</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-17 datetime="2022-05-17T13:48:21.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="9590a971b00448cd312078183965b21d"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: As part of our overall commitment to diversity, equity and inclusion at Springer Nature, we’d like to better understand what DEI means within the global research community. Take the survey here</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1526560254091313156">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>As part of our overall commitment to diversity, equity and inclusion at Springer Nature, we’d like to better understand what DEI means within the global research community. Take the survey here https://t.co/03yLiUxFTq
— NatureReviewsCancer (@NatureRevCancer) May 17, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-12 datetime="2022-05-12T15:25:46.000Z">2022-05-12</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-12 datetime="2022-05-12T15:25:46.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="aa38ff4245b93e964773f191844ddc7a"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: Newsflash! We have a new article type in the journal - Tools of the Trade #NRCToT 🧰 🛠️ where we hope to enable more #ECRs to interact with the editorial team &amp; have a bigger voice in our content. For more info, check out the Editorial 👇🏻</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1524772831904509953">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Newsflash! We have a new article type in the journal - Tools of the Trade #NRCToT 🧰 🛠️ where we hope to enable more #ECRs to interact with the editorial team &amp; have a bigger voice in our content. For more info, check out the Editorial 👇🏻 https://t.co/VbyoUJmTRJ
— NatureReviewsCancer (@NatureRevCancer) May 12, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-10 datetime="2022-05-10T16:46:03.000Z">2022-05-10</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-10 datetime="2022-05-10T16:46:03.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="65aca76b9f794aaa668d84ad32dca4a0"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Tools for the next generation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1524068259712036864">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Tools for the next generation https://t.co/lEzTPFYDf5
— NatureReviewsCancer (@NatureRevCancer) May 10, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="afa9b6784b4ccc7c05ed4efd26ddfb2d"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Hunting down rare drug-tolerant cycling cells with Watermelon</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1524022709386190856">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Hunting down rare drug-tolerant cycling cells with Watermelon https://t.co/XXT0c9SAYX
— NatureReviewsCancer (@NatureRevCancer) May 10, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-04-29 datetime="2022-04-29T14:40:05.000Z">2022-04-29</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-04-29 datetime="2022-04-29T14:40:05.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="43e2d5ccebc9030f737d656f47ac9d98"
                        open
                      >
                        <summary class="article-expander__title">@VarunVenkatara2: Very happy that our latest Perspective is now online @NatureRevCancer ! We discuss implications of malignant, multicellular networks and potential therapeutic strategies how to disconnect them. Thanks to everyone involved ! #CancerNeuroscience #TumorMicrotubes #neurotwitter</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/VarunVenkatara2/status/1520050291986604033">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Very happy that our latest Perspective is now online @NatureRevCancer ! We discuss implications of malignant, multicellular networks and potential therapeutic strategies how to disconnect them. Thanks to everyone involved ! #CancerNeuroscience #TumorMicrotubes #neurotwitter https://t.co/S55cKTGNFd
— Varun Venkataramani (@VarunVenkatara2) April 29, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="9f6b8644378881f4c2e6b6833c93fc2f"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! An inhospitable site</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1520020349068316673">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! An inhospitable site https://t.co/0dvqIKCWxh pic.twitter.com/PtY8J9Fz7r
— NatureReviewsCancer (@NatureRevCancer) April 29, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="f6d91efcd60dae769e2722585bdfa8df"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Disconnecting multicellular networks in brain tumours</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1520020340201598978">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Disconnecting multicellular networks in brain tumours https://t.co/LJrzy8bRma pic.twitter.com/CO7BnooP0r
— NatureReviewsCancer (@NatureRevCancer) April 29, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-04-28 datetime="2022-04-28T14:04:34.000Z">2022-04-28</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-04-28 datetime="2022-04-28T14:04:34.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="76e3b436bc8e43ad21366a80c120e395"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: Our May issue, out NOW! With Reviews on #CRISPR in cancer biology &amp; #therapy, ROS in tumour development &amp; progression, and cancer #proteogenomics, and Comments on esophageal cancer research in Eastern Africa and on Critical Race Theory.💌</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1519678964364623872">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Our May issue, out NOW! With Reviews on #CRISPR in cancer biology &amp; #therapy, ROS in tumour development &amp; progression, and cancer #proteogenomics, and Comments on esophageal cancer research in Eastern Africa and on Critical Race Theory.
💌 https://t.co/vQwHFJhpwv@natrescancer pic.twitter.com/6kYgjRYhdT
— NatureReviewsCancer (@NatureRevCancer) April 28, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="fdf224ea293cdf039086d16e5699a59a"
                        open
                      >
                        <summary class="article-expander__title">@tmurraystewart: Our latest publication on #polyamines is online!@HopkinsMedicine</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/tmurraystewart/status/1519413460978704384">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Our latest publication on #polyamines is online!@HopkinsMedicine https://t.co/ehRGgN1w9R
— Tracy Murray Stewart (@tmurraystewart) April 27, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-04-27 datetime="2022-04-27T18:22:03.000Z">2022-04-27</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-04-27 datetime="2022-04-27T18:22:03.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="4f43226ec4dc8e531ff6e64a9c5ab8e6"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Polyamines in cancer: integrating organismal metabolism and antitumour immunity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1519381375874265088">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Polyamines in cancer: integrating organismal metabolism and antitumour immunity https://t.co/YCJX1hRQLO pic.twitter.com/zRUCRNXyxH
— NatureReviewsCancer (@NatureRevCancer) April 27, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="a44ed79b9f76aad377b9272fb392f553"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW #ResearchHighlight! Lv, Liu, Mo &amp; co demonstrate that in pancreatic #PDAC cells, gasdermin E transports YBX1 to the nucleus, which promotes mucin expression, thereby providing tumour cells with a barrier against digestive enzymes.📯</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1519317944995569664">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW #ResearchHighlight! Lv, Liu, Mo &amp; co demonstrate that in pancreatic #PDAC cells, gasdermin E transports YBX1 to the nucleus, which promotes mucin expression, thereby providing tumour cells with a barrier against digestive enzymes.
📯 https://t.co/JTUhX0DHQ1@natrescancer pic.twitter.com/2PdCmsDu2p
— NatureReviewsCancer (@NatureRevCancer) April 27, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-04-26 datetime="2022-04-26T14:58:00.000Z">2022-04-26</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-04-26 datetime="2022-04-26T14:58:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="06ff7641d8f5fcc0dff91f835ac3bbc1"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: In the April issue of @NatureRevCancer | Comment, Reimagining cancer research with art, by @CancerSciArtist, Stefani Shoreibah &amp; @tdanino  @natrescancer @Columbia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1518967636561956865">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>In the April issue of @NatureRevCancer | Comment, Reimagining cancer research with art, by @CancerSciArtist, Stefani Shoreibah &amp; @tdanino
 https://t.co/GeTVCMJQsp @natrescancer
 @Columbia pic.twitter.com/ZXW6A0Fe0c
— NatureReviewsCancer (@NatureRevCancer) April 26, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-04-25 datetime="2022-04-25T12:12:32.000Z">2022-04-25</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-04-25 datetime="2022-04-25T12:12:32.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="96b2ddff4eb269cf343213c6a1b1ce95"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Tracking dormant disseminated tumour cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1518563607440621569">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Tracking dormant disseminated tumour cells https://t.co/kSXyWAvCxF
— NatureReviewsCancer (@NatureRevCancer) April 25, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>

    <footer>
      <a class="footer-link" href="https://github.com/MacwanEric/cancer.rss.feed/actions/runs/6764898895">
        <time id="build-timestamp" datetime="2023-11-06T00:52:01.857Z">2023-11-06T00:52:01.857Z</time>
      </a>
      <a class="footer-link" href="https://github.com/osmoscraft/osmosfeed">osmosfeed 1.15.1</a>
    </footer>
    <script src="index.js?v1.14.4"></script>
    <!-- %before-body-end.html% -->
  </body>

</html>