<!DOCTYPE html>
<html lang="en">

  <head>
    <title>CancerFeed</title>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta name="robots" content="noindex, nofollow" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
    <link rel="alternate" type="application/rss+xml" title="CancerFeed" href="feed.atom" />
    <link href="index.css?v1.14.4" rel="stylesheet" />
    <style>
  :root {
    /**
     * Solarized Light
     * Ethan Schoonover (modified by aramisgithub)
     */
    --base00: #fdf6e3;
    --base01: #eee8d5;
    --base02: #93a1a1;
    --base03: #839496;
    --base04: #657b83;
    --base05: #586e75;
    --base06: #073642;
    --base07: #002b36;
    --base08: #dc322f;
    --base09: #cb4b16;
    --base0A: #b58900;
    --base0B: #859900;
    --base0C: #2aa198;
    --base0D: #268bd2;
    --base0E: #6c71c4;
    --base0F: #d33682;

    /**
    * Components
    */
    --source-name-color: var(--base0C);
    /**
   * Typograph
   */
    --font-family-default: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell,
      "Helvetica Neue", sans-serif;
    --font-size-L: 3.6rem;
    --font-size-m: 2.2rem;
    --font-size-s: 1.8rem;
    --font-size-scaler: 52.5%; /* 1rem = 10px */
    --line-height-body: 1.4;
    --line-height-heading: 1.2;

  }
</style>

  </head>

  <body>
    <!-- %after-body-begin.html% -->
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-08-23 datetime="2022-08-23T10:00:00.000Z">2022-08-23</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-08-23 datetime="2022-08-23T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608003622&amp;v&#x3D;2.17.7&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35996953"
                        open
                      >
                        <summary class="article-expander__title">Lipids and the cancer stemness regulatory system in acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35996953/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220824212808&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is a heterogeneous disease of impaired myeloid differentiation and a caricature of normal hematopoiesis. Leukemic stem cells (LSCs) are responsible for long-term clonal propagation in AML just as hematopoietic stem cells (HSCs) sustain lifelong hematopoiesis. LSCs are often resistant to standard chemotherapy and are responsible for clinical relapse. Although AML is highly heterogeneous, determinants of stemness are prognostic for AML patient survival and can predict...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35996953"
                        open
                      >
                        <summary class="article-expander__title">Lipids and the cancer stemness regulatory system in acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35996953/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220823212820&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is a heterogeneous disease of impaired myeloid differentiation and a caricature of normal hematopoiesis. Leukemic stem cells (LSCs) are responsible for long-term clonal propagation in AML just as hematopoietic stem cells (HSCs) sustain lifelong hematopoiesis. LSCs are often resistant to standard chemotherapy and are responsible for clinical relapse. Although AML is highly heterogeneous, determinants of stemness are prognostic for AML patient survival and can predict...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-08-22 datetime="2022-08-22T10:00:00.000Z">2022-08-22</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-08-22 datetime="2022-08-22T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608003622&amp;v&#x3D;2.17.7&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35990601"
                        open
                      >
                        <summary class="article-expander__title">Hedgehog/GLI signaling in hematopoietic development and acute myeloid leukemia-From bench to bedside</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35990601/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220824212808&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>While the underlying genetic alterations and biology of acute myeloid leukemia (AML), an aggressive hematologic malignancy characterized by clonal expansion of undifferentiated myeloid cells, have been gradually unraveled in the last decades, translation into clinical treatment approaches has only just begun. High relapse rates remain a major challenge in AML therapy and are to a large extent attributed to the persistence of treatment-resistant leukemic stem cells (LSCs). The Hedgehog (HH)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35990601"
                        open
                      >
                        <summary class="article-expander__title">Hedgehog/GLI signaling in hematopoietic development and acute myeloid leukemia-From bench to bedside</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35990601/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220823212820&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>While the underlying genetic alterations and biology of acute myeloid leukemia (AML), an aggressive hematologic malignancy characterized by clonal expansion of undifferentiated myeloid cells, have been gradually unraveled in the last decades, translation into clinical treatment approaches has only just begun. High relapse rates remain a major challenge in AML therapy and are to a large extent attributed to the persistence of treatment-resistant leukemic stem cells (LSCs). The Hedgehog (HH)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35990601"
                        open
                      >
                        <summary class="article-expander__title">Hedgehog/GLI signaling in hematopoietic development and acute myeloid leukemia-From bench to bedside</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35990601/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220822212717&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>While the underlying genetic alterations and biology of acute myeloid leukemia (AML), an aggressive hematologic malignancy characterized by clonal expansion of undifferentiated myeloid cells, have been gradually unraveled in the last decades, translation into clinical treatment approaches has only just begun. High relapse rates remain a major challenge in AML therapy and are to a large extent attributed to the persistence of treatment-resistant leukemic stem cells (LSCs). The Hedgehog (HH)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608010801&amp;v&#x3D;2.17.7&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35996174"
                        open
                      >
                        <summary class="article-expander__title">Evolution of intra-tumoral heterogeneity across different pathological stages in papillary thyroid carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35996174/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220824212808&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our findings suggest ITH as a potential unfavorable prognostic factor in PTC and highlight the dynamic changes in different clinical stages of PTC, providing some clues for the precision medicine and suggesting different diagnostic decisions depending on the clinical stages of patients. Finally, complete clear guidelines to define risk stratification of PTC patients are lacking; thus, this work could contribute to defining patients who need more aggressive treatments and, in turn,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35996174"
                        open
                      >
                        <summary class="article-expander__title">Evolution of intra-tumoral heterogeneity across different pathological stages in papillary thyroid carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35996174/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220823212820&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our findings suggest ITH as a potential unfavorable prognostic factor in PTC and highlight the dynamic changes in different clinical stages of PTC, providing some clues for the precision medicine and suggesting different diagnostic decisions depending on the clinical stages of patients. Finally, complete clear guidelines to define risk stratification of PTC patients are lacking; thus, this work could contribute to defining patients who need more aggressive treatments and, in turn,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-08-21 datetime="2022-08-21T10:00:00.000Z">2022-08-21</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-08-21 datetime="2022-08-21T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608003622&amp;v&#x3D;2.17.7&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35989137"
                        open
                      >
                        <summary class="article-expander__title">CHIPing away the progression potential of CHIP: A new reality in the making</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35989137/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220824212808&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Over the past few years, we have gained a deeper understanding of clonal hematopoiesis of indeterminate potential (CHIP), especially with regard to the epidemiology, clinical sequelae, and mechanical aspects. However, interventional strategies to prevent or delay the potential negative consequences of CHIP remain underdeveloped. In this review, we highlight the latest updates on clonal hematopoiesis research, including molecular mechanisms and clinical implications, with a particular focus on...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35989137"
                        open
                      >
                        <summary class="article-expander__title">CHIPing away the progression potential of CHIP: A new reality in the making</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35989137/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220823212820&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Over the past few years, we have gained a deeper understanding of clonal hematopoiesis of indeterminate potential (CHIP), especially with regard to the epidemiology, clinical sequelae, and mechanical aspects. However, interventional strategies to prevent or delay the potential negative consequences of CHIP remain underdeveloped. In this review, we highlight the latest updates on clonal hematopoiesis research, including molecular mechanisms and clinical implications, with a particular focus on...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35989137"
                        open
                      >
                        <summary class="article-expander__title">CHIPing away the progression potential of CHIP: A new reality in the making</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35989137/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220822212717&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Over the past few years, we have gained a deeper understanding of clonal hematopoiesis of indeterminate potential (CHIP), especially with regard to the epidemiology, clinical sequelae, and mechanical aspects. However, interventional strategies to prevent or delay the potential negative consequences of CHIP remain underdeveloped. In this review, we highlight the latest updates on clonal hematopoiesis research, including molecular mechanisms and clinical implications, with a particular focus on...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-08-20 datetime="2022-08-20T10:00:00.000Z">2022-08-20</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-08-20 datetime="2022-08-20T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608003622&amp;v&#x3D;2.17.7&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35986863"
                        open
                      >
                        <summary class="article-expander__title">Role of Germline Predisposition to Therapy-Related Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35986863/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220824212808&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Therapy-related myeloid neoplasms (t-MNs) are aggressive leukemias that develop following exposure to DNA-damaging agents. A subset of patients developing t-MN may have an inherited susceptibility to develop myeloid neoplasia. Herein, we review studies reporting t-MN and their association with a germline or inherited predisposition.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35986863"
                        open
                      >
                        <summary class="article-expander__title">Role of Germline Predisposition to Therapy-Related Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35986863/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220823212820&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Therapy-related myeloid neoplasms (t-MNs) are aggressive leukemias that develop following exposure to DNA-damaging agents. A subset of patients developing t-MN may have an inherited susceptibility to develop myeloid neoplasia. Herein, we review studies reporting t-MN and their association with a germline or inherited predisposition.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35986863"
                        open
                      >
                        <summary class="article-expander__title">Role of Germline Predisposition to Therapy-Related Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35986863/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220822212717&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Therapy-related myeloid neoplasms (t-MNs) are aggressive leukemias that develop following exposure to DNA-damaging agents. A subset of patients developing t-MN may have an inherited susceptibility to develop myeloid neoplasia. Herein, we review studies reporting t-MN and their association with a germline or inherited predisposition.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35986863"
                        open
                      >
                        <summary class="article-expander__title">Role of Germline Predisposition to Therapy-Related Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35986863/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220821211551&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Therapy-related myeloid neoplasms (t-MNs) are aggressive leukemias that develop following exposure to DNA-damaging agents. A subset of patients developing t-MN may have an inherited susceptibility to develop myeloid neoplasia. Herein, we review studies reporting t-MN and their association with a germline or inherited predisposition.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35986863"
                        open
                      >
                        <summary class="article-expander__title">Role of Germline Predisposition to Therapy-Related Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35986863/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220820211309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Therapy-related myeloid neoplasms (t-MNs) are aggressive leukemias that develop following exposure to DNA-damaging agents. A subset of patients developing t-MN may have an inherited susceptibility to develop myeloid neoplasia. Herein, we review studies reporting t-MN and their association with a germline or inherited predisposition.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-08-19 datetime="2022-08-19T10:00:00.000Z">2022-08-19</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-08-19 datetime="2022-08-19T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608003622&amp;v&#x3D;2.17.7&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35984598"
                        open
                      >
                        <summary class="article-expander__title">Novel Therapies in Myelofibrosis: Beyond JAK Inhibitors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35984598/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220824212808&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: To discuss the current treatment paradigm, review novel targets, and summarize completed and ongoing clinical trials that may lead to a paradigm shifts in the management of myelofibrosis (MF).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35984598"
                        open
                      >
                        <summary class="article-expander__title">Novel Therapies in Myelofibrosis: Beyond JAK Inhibitors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35984598/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220823212820&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: To discuss the current treatment paradigm, review novel targets, and summarize completed and ongoing clinical trials that may lead to a paradigm shifts in the management of myelofibrosis (MF).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35984598"
                        open
                      >
                        <summary class="article-expander__title">Novel Therapies in Myelofibrosis: Beyond JAK Inhibitors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35984598/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220822212717&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: To discuss the current treatment paradigm, review novel targets, and summarize completed and ongoing clinical trials that may lead to a paradigm shifts in the management of myelofibrosis (MF).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35984598"
                        open
                      >
                        <summary class="article-expander__title">Novel Therapies in Myelofibrosis: Beyond JAK Inhibitors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35984598/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220821211551&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: To discuss the current treatment paradigm, review novel targets, and summarize completed and ongoing clinical trials that may lead to a paradigm shifts in the management of myelofibrosis (MF).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35984598"
                        open
                      >
                        <summary class="article-expander__title">Novel Therapies in Myelofibrosis: Beyond JAK Inhibitors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35984598/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220820211309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: To discuss the current treatment paradigm, review novel targets, and summarize completed and ongoing clinical trials that may lead to a paradigm shifts in the management of myelofibrosis (MF).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35984598"
                        open
                      >
                        <summary class="article-expander__title">Novel Therapies in Myelofibrosis: Beyond JAK Inhibitors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35984598/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220819211457&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: To discuss the current treatment paradigm, review novel targets, and summarize completed and ongoing clinical trials that may lead to a paradigm shifts in the management of myelofibrosis (MF).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608010801&amp;v&#x3D;2.17.7&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35986270"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal single-cell transcriptomics reveals distinct patterns of recurrence in acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35986270/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220824212808&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In summary, the single cell RNA data underpinned the tumor heterogeneity not only amongst patient blasts with similar initiating mutations but also between each Dx-Re pair. Our results suggest alternatively and currently unappreciated and unexplored mechanisms leading to therapeutic resistance and AML recurrence.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35986254"
                        open
                      >
                        <summary class="article-expander__title">SCONCE2: jointly inferring single cell copy number profiles and tumor evolutionary distances</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35986254/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220824212808&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: SCONCE2 jointly estimates copy number profiles and a distance metric for inferring tumor phylogenies in single cell whole genome tumor sequencing across multiple cells, enabling deeper understandings of tumor evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35986270"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal single-cell transcriptomics reveals distinct patterns of recurrence in acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35986270/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220823212820&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In summary, the single cell RNA data underpinned the tumor heterogeneity not only amongst patient blasts with similar initiating mutations but also between each Dx-Re pair. Our results suggest alternatively and currently unappreciated and unexplored mechanisms leading to therapeutic resistance and AML recurrence.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35986254"
                        open
                      >
                        <summary class="article-expander__title">SCONCE2: jointly inferring single cell copy number profiles and tumor evolutionary distances</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35986254/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220823212820&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: SCONCE2 jointly estimates copy number profiles and a distance metric for inferring tumor phylogenies in single cell whole genome tumor sequencing across multiple cells, enabling deeper understandings of tumor evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35986270"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal single-cell transcriptomics reveals distinct patterns of recurrence in acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35986270/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220822212717&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In summary, the single cell RNA data underpinned the tumor heterogeneity not only amongst patient blasts with similar initiating mutations but also between each Dx-Re pair. Our results suggest alternatively and currently unappreciated and unexplored mechanisms leading to therapeutic resistance and AML recurrence.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35986254"
                        open
                      >
                        <summary class="article-expander__title">SCONCE2: jointly inferring single cell copy number profiles and tumor evolutionary distances</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35986254/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220822212717&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: SCONCE2 jointly estimates copy number profiles and a distance metric for inferring tumor phylogenies in single cell whole genome tumor sequencing across multiple cells, enabling deeper understandings of tumor evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35986270"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal single-cell transcriptomics reveals distinct patterns of recurrence in acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35986270/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220821211551&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In summary, the single cell RNA data underpinned the tumor heterogeneity not only amongst patient blasts with similar initiating mutations but also between each Dx-Re pair. Our results suggest alternatively and currently unappreciated and unexplored mechanisms leading to therapeutic resistance and AML recurrence.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35986254"
                        open
                      >
                        <summary class="article-expander__title">SCONCE2: jointly inferring single cell copy number profiles and tumor evolutionary distances</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35986254/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220821211551&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: SCONCE2 jointly estimates copy number profiles and a distance metric for inferring tumor phylogenies in single cell whole genome tumor sequencing across multiple cells, enabling deeper understandings of tumor evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35986270"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal single-cell transcriptomics reveals distinct patterns of recurrence in acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35986270/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220820211309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In summary, the single cell RNA data underpinned the tumor heterogeneity not only amongst patient blasts with similar initiating mutations but also between each Dx-Re pair. Our results suggest alternatively and currently unappreciated and unexplored mechanisms leading to therapeutic resistance and AML recurrence.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35986254"
                        open
                      >
                        <summary class="article-expander__title">SCONCE2: jointly inferring single cell copy number profiles and tumor evolutionary distances</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35986254/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220820211309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: SCONCE2 jointly estimates copy number profiles and a distance metric for inferring tumor phylogenies in single cell whole genome tumor sequencing across multiple cells, enabling deeper understandings of tumor evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-08-18 datetime="2022-08-18T10:00:00.000Z">2022-08-18</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-08-18 datetime="2022-08-18T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608003622&amp;v&#x3D;2.17.7&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35979721"
                        open
                      >
                        <summary class="article-expander__title">Selective inhibition of MCL1 overcomes venetoclax-resistance in a murine model of myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35979721/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220824212808&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Treatment for myelodysplastic syndromes (MDS) remains insufficient due to clonal heterogeneity and lack of effective clinical therapies. Dysregulation of apoptosis is observed across MDS subtypes regardless of mutations and represents an attractive therapeutic opportunity. Venetoclax (VEN), a selective inhibitor of anti-apoptotic protein B-cell lymphoma-2 (BCL2), has yielded impressive responses in older patients with acute myeloid leukemia (AML) and high risk MDS. BCL2 family anti-apoptotic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35979721"
                        open
                      >
                        <summary class="article-expander__title">Selective inhibition of MCL1 overcomes venetoclax-resistance in a murine model of myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35979721/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220823212820&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Treatment for myelodysplastic syndromes (MDS) remains insufficient due to clonal heterogeneity and lack of effective clinical therapies. Dysregulation of apoptosis is observed across MDS subtypes regardless of mutations and represents an attractive therapeutic opportunity. Venetoclax (VEN), a selective inhibitor of anti-apoptotic protein B-cell lymphoma-2 (BCL2), has yielded impressive responses in older patients with acute myeloid leukemia (AML) and high risk MDS. BCL2 family anti-apoptotic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35979721"
                        open
                      >
                        <summary class="article-expander__title">Selective inhibition of MCL1 overcomes venetoclax-resistance in a murine model of myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35979721/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220822212717&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Treatment for myelodysplastic syndromes (MDS) remains insufficient due to clonal heterogeneity and lack of effective clinical therapies. Dysregulation of apoptosis is observed across MDS subtypes regardless of mutations and represents an attractive therapeutic opportunity. Venetoclax (VEN), a selective inhibitor of anti-apoptotic protein B-cell lymphoma-2 (BCL2), has yielded impressive responses in older patients with acute myeloid leukemia (AML) and high risk MDS. BCL2 family anti-apoptotic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35979721"
                        open
                      >
                        <summary class="article-expander__title">Selective inhibition of MCL1 overcomes venetoclax-resistance in a murine model of myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35979721/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220821211551&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Treatment for myelodysplastic syndromes (MDS) remains insufficient due to clonal heterogeneity and lack of effective clinical therapies. Dysregulation of apoptosis is observed across MDS subtypes regardless of mutations and represents an attractive therapeutic opportunity. Venetoclax (VEN), a selective inhibitor of anti-apoptotic protein B-cell lymphoma-2 (BCL2), has yielded impressive responses in older patients with acute myeloid leukemia (AML) and high risk MDS. BCL2 family anti-apoptotic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35979721"
                        open
                      >
                        <summary class="article-expander__title">Selective inhibition of MCL1 overcomes venetoclax-resistance in a murine model of myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35979721/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220820211309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Treatment for myelodysplastic syndromes (MDS) remains insufficient due to clonal heterogeneity and lack of effective clinical therapies. Dysregulation of apoptosis is observed across MDS subtypes regardless of mutations and represents an attractive therapeutic opportunity. Venetoclax (VEN), a selective inhibitor of anti-apoptotic protein B-cell lymphoma-2 (BCL2), has yielded impressive responses in older patients with acute myeloid leukemia (AML) and high risk MDS. BCL2 family anti-apoptotic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35979721"
                        open
                      >
                        <summary class="article-expander__title">Selective inhibition of MCL1 overcomes venetoclax-resistance in a murine model of myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35979721/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220819211457&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Treatment for myelodysplastic syndromes (MDS) remains insufficient due to clonal heterogeneity and lack of effective clinical therapies. Dysregulation of apoptosis is observed across MDS subtypes regardless of mutations and represents an attractive therapeutic opportunity. Venetoclax (VEN), a selective inhibitor of anti-apoptotic protein B-cell lymphoma-2 (BCL2), has yielded impressive responses in older patients with acute myeloid leukemia (AML) and high risk MDS. BCL2 family anti-apoptotic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35979721"
                        open
                      >
                        <summary class="article-expander__title">Selective inhibition of MCL1 overcomes venetoclax-resistance in a murine model of myelodysplastic syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35979721/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220818211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Treatment for myelodysplastic syndromes (MDS) remains insufficient due to clonal heterogeneity and lack of effective clinical therapies. Dysregulation of apoptosis is observed across MDS subtypes regardless of mutations and represents an attractive therapeutic opportunity. Venetoclax (VEN), a selective inhibitor of anti-apoptotic protein B-cell lymphoma-2 (BCL2), has yielded impressive responses in older patients with acute myeloid leukemia (AML) and high risk MDS. BCL2 family anti-apoptotic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-08-17 datetime="2022-08-17T10:00:00.000Z">2022-08-17</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-08-17 datetime="2022-08-17T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608003622&amp;v&#x3D;2.17.7&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35978153"
                        open
                      >
                        <summary class="article-expander__title">Retrospective analysis of somatic mutations and clonal hematopoiesis in astronauts</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35978153/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220824212808&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With planned deep space and commercial spaceflights, gaps remain to address health risks in astronauts. Multiple studies have shown associations between clonal expansion of hematopoietic cells with hematopoietic malignancies and cardiometabolic disease. This expansion of clones in the absence of overt hematopoietic disorders is termed clonal hematopoiesis (CH) of indeterminate potential (CHIP). Using deep, error-corrected, targeted DNA sequencing we assayed for somatic mutations in CH-driver...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35978153"
                        open
                      >
                        <summary class="article-expander__title">Retrospective analysis of somatic mutations and clonal hematopoiesis in astronauts</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35978153/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220823212820&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With planned deep space and commercial spaceflights, gaps remain to address health risks in astronauts. Multiple studies have shown associations between clonal expansion of hematopoietic cells with hematopoietic malignancies and cardiometabolic disease. This expansion of clones in the absence of overt hematopoietic disorders is termed clonal hematopoiesis (CH) of indeterminate potential (CHIP). Using deep, error-corrected, targeted DNA sequencing we assayed for somatic mutations in CH-driver...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35978153"
                        open
                      >
                        <summary class="article-expander__title">Retrospective analysis of somatic mutations and clonal hematopoiesis in astronauts</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35978153/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220822212717&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With planned deep space and commercial spaceflights, gaps remain to address health risks in astronauts. Multiple studies have shown associations between clonal expansion of hematopoietic cells with hematopoietic malignancies and cardiometabolic disease. This expansion of clones in the absence of overt hematopoietic disorders is termed clonal hematopoiesis (CH) of indeterminate potential (CHIP). Using deep, error-corrected, targeted DNA sequencing we assayed for somatic mutations in CH-driver...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35978153"
                        open
                      >
                        <summary class="article-expander__title">Retrospective analysis of somatic mutations and clonal hematopoiesis in astronauts</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35978153/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220821211551&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With planned deep space and commercial spaceflights, gaps remain to address health risks in astronauts. Multiple studies have shown associations between clonal expansion of hematopoietic cells with hematopoietic malignancies and cardiometabolic disease. This expansion of clones in the absence of overt hematopoietic disorders is termed clonal hematopoiesis (CH) of indeterminate potential (CHIP). Using deep, error-corrected, targeted DNA sequencing we assayed for somatic mutations in CH-driver...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35978153"
                        open
                      >
                        <summary class="article-expander__title">Retrospective analysis of somatic mutations and clonal hematopoiesis in astronauts</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35978153/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220820211309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With planned deep space and commercial spaceflights, gaps remain to address health risks in astronauts. Multiple studies have shown associations between clonal expansion of hematopoietic cells with hematopoietic malignancies and cardiometabolic disease. This expansion of clones in the absence of overt hematopoietic disorders is termed clonal hematopoiesis (CH) of indeterminate potential (CHIP). Using deep, error-corrected, targeted DNA sequencing we assayed for somatic mutations in CH-driver...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35978153"
                        open
                      >
                        <summary class="article-expander__title">Retrospective analysis of somatic mutations and clonal hematopoiesis in astronauts</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35978153/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220819211457&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With planned deep space and commercial spaceflights, gaps remain to address health risks in astronauts. Multiple studies have shown associations between clonal expansion of hematopoietic cells with hematopoietic malignancies and cardiometabolic disease. This expansion of clones in the absence of overt hematopoietic disorders is termed clonal hematopoiesis (CH) of indeterminate potential (CHIP). Using deep, error-corrected, targeted DNA sequencing we assayed for somatic mutations in CH-driver...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35978153"
                        open
                      >
                        <summary class="article-expander__title">Retrospective analysis of somatic mutations and clonal hematopoiesis in astronauts</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35978153/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220818211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With planned deep space and commercial spaceflights, gaps remain to address health risks in astronauts. Multiple studies have shown associations between clonal expansion of hematopoietic cells with hematopoietic malignancies and cardiometabolic disease. This expansion of clones in the absence of overt hematopoietic disorders is termed clonal hematopoiesis (CH) of indeterminate potential (CHIP). Using deep, error-corrected, targeted DNA sequencing we assayed for somatic mutations in CH-driver...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608010801&amp;v&#x3D;2.17.7&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35977503"
                        open
                      >
                        <summary class="article-expander__title">Multiregional single-cell proteogenomic analysis of ccRCC reveals cytokine drivers of intratumor spatial heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35977503/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220824212808&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumor heterogeneity (ITH) represents a major challenge for anticancer therapies. An integrated, multidimensional, multiregional approach dissecting ITH of the clear cell renal cell carcinoma (ccRCC) tumor microenvironment (TME) is employed at the single-cell level with mass cytometry (CyTOF), multiplex immunofluorescence (MxIF), and single-nucleus RNA sequencing (snRNA-seq) and at the bulk level with whole-exome sequencing (WES), RNA-seq, and methylation profiling. Multiregional analyses...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35977503"
                        open
                      >
                        <summary class="article-expander__title">Multiregional single-cell proteogenomic analysis of ccRCC reveals cytokine drivers of intratumor spatial heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35977503/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220823212820&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumor heterogeneity (ITH) represents a major challenge for anticancer therapies. An integrated, multidimensional, multiregional approach dissecting ITH of the clear cell renal cell carcinoma (ccRCC) tumor microenvironment (TME) is employed at the single-cell level with mass cytometry (CyTOF), multiplex immunofluorescence (MxIF), and single-nucleus RNA sequencing (snRNA-seq) and at the bulk level with whole-exome sequencing (WES), RNA-seq, and methylation profiling. Multiregional analyses...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35977503"
                        open
                      >
                        <summary class="article-expander__title">Multiregional single-cell proteogenomic analysis of ccRCC reveals cytokine drivers of intratumor spatial heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35977503/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220822212717&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumor heterogeneity (ITH) represents a major challenge for anticancer therapies. An integrated, multidimensional, multiregional approach dissecting ITH of the clear cell renal cell carcinoma (ccRCC) tumor microenvironment (TME) is employed at the single-cell level with mass cytometry (CyTOF), multiplex immunofluorescence (MxIF), and single-nucleus RNA sequencing (snRNA-seq) and at the bulk level with whole-exome sequencing (WES), RNA-seq, and methylation profiling. Multiregional analyses...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35977503"
                        open
                      >
                        <summary class="article-expander__title">Multiregional single-cell proteogenomic analysis of ccRCC reveals cytokine drivers of intratumor spatial heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35977503/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220821211551&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumor heterogeneity (ITH) represents a major challenge for anticancer therapies. An integrated, multidimensional, multiregional approach dissecting ITH of the clear cell renal cell carcinoma (ccRCC) tumor microenvironment (TME) is employed at the single-cell level with mass cytometry (CyTOF), multiplex immunofluorescence (MxIF), and single-nucleus RNA sequencing (snRNA-seq) and at the bulk level with whole-exome sequencing (WES), RNA-seq, and methylation profiling. Multiregional analyses...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35977503"
                        open
                      >
                        <summary class="article-expander__title">Multiregional single-cell proteogenomic analysis of ccRCC reveals cytokine drivers of intratumor spatial heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35977503/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220820211309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumor heterogeneity (ITH) represents a major challenge for anticancer therapies. An integrated, multidimensional, multiregional approach dissecting ITH of the clear cell renal cell carcinoma (ccRCC) tumor microenvironment (TME) is employed at the single-cell level with mass cytometry (CyTOF), multiplex immunofluorescence (MxIF), and single-nucleus RNA sequencing (snRNA-seq) and at the bulk level with whole-exome sequencing (WES), RNA-seq, and methylation profiling. Multiregional analyses...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35977503"
                        open
                      >
                        <summary class="article-expander__title">Multiregional single-cell proteogenomic analysis of ccRCC reveals cytokine drivers of intratumor spatial heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35977503/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220819211458&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumor heterogeneity (ITH) represents a major challenge for anticancer therapies. An integrated, multidimensional, multiregional approach dissecting ITH of the clear cell renal cell carcinoma (ccRCC) tumor microenvironment (TME) is employed at the single-cell level with mass cytometry (CyTOF), multiplex immunofluorescence (MxIF), and single-nucleus RNA sequencing (snRNA-seq) and at the bulk level with whole-exome sequencing (WES), RNA-seq, and methylation profiling. Multiregional analyses...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35977503"
                        open
                      >
                        <summary class="article-expander__title">Multiregional single-cell proteogenomic analysis of ccRCC reveals cytokine drivers of intratumor spatial heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35977503/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220818211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumor heterogeneity (ITH) represents a major challenge for anticancer therapies. An integrated, multidimensional, multiregional approach dissecting ITH of the clear cell renal cell carcinoma (ccRCC) tumor microenvironment (TME) is employed at the single-cell level with mass cytometry (CyTOF), multiplex immunofluorescence (MxIF), and single-nucleus RNA sequencing (snRNA-seq) and at the bulk level with whole-exome sequencing (WES), RNA-seq, and methylation profiling. Multiregional analyses...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-08-16 datetime="2022-08-16T10:00:00.000Z">2022-08-16</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-08-16 datetime="2022-08-16T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608003622&amp;v&#x3D;2.17.7&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35971851"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis confers an increased mortality risk in orthotopic heart transplant recipients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35971851/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220824212808&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Novel risk stratification and non-invasive surveillance methods are needed in orthotopic heart transplant (OHT) to reduce morbidity and mortality post-transplant. Clonal hematopoiesis (CH) refers to the acquisition of specific gene mutations in hematopoietic stem cells linked to enhanced inflammation and worse cardiovascular outcomes. The purpose of this study was to investigate the association between CH and OHT. Blood samples were collected from 127 OHT recipients. Error-corrected sequencing...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35971851"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis confers an increased mortality risk in orthotopic heart transplant recipients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35971851/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220823212820&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Novel risk stratification and non-invasive surveillance methods are needed in orthotopic heart transplant (OHT) to reduce morbidity and mortality post-transplant. Clonal hematopoiesis (CH) refers to the acquisition of specific gene mutations in hematopoietic stem cells linked to enhanced inflammation and worse cardiovascular outcomes. The purpose of this study was to investigate the association between CH and OHT. Blood samples were collected from 127 OHT recipients. Error-corrected sequencing...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35971851"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis confers an increased mortality risk in orthotopic heart transplant recipients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35971851/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220822212717&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Novel risk stratification and non-invasive surveillance methods are needed in orthotopic heart transplant (OHT) to reduce morbidity and mortality post-transplant. Clonal hematopoiesis (CH) refers to the acquisition of specific gene mutations in hematopoietic stem cells linked to enhanced inflammation and worse cardiovascular outcomes. The purpose of this study was to investigate the association between CH and OHT. Blood samples were collected from 127 OHT recipients. Error-corrected sequencing...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35971851"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis confers an increased mortality risk in orthotopic heart transplant recipients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35971851/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220821211551&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Novel risk stratification and non-invasive surveillance methods are needed in orthotopic heart transplant (OHT) to reduce morbidity and mortality post-transplant. Clonal hematopoiesis (CH) refers to the acquisition of specific gene mutations in hematopoietic stem cells linked to enhanced inflammation and worse cardiovascular outcomes. The purpose of this study was to investigate the association between CH and OHT. Blood samples were collected from 127 OHT recipients. Error-corrected sequencing...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35971851"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis confers an increased mortality risk in orthotopic heart transplant recipients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35971851/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220820211309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Novel risk stratification and non-invasive surveillance methods are needed in orthotopic heart transplant (OHT) to reduce morbidity and mortality post-transplant. Clonal hematopoiesis (CH) refers to the acquisition of specific gene mutations in hematopoietic stem cells linked to enhanced inflammation and worse cardiovascular outcomes. The purpose of this study was to investigate the association between CH and OHT. Blood samples were collected from 127 OHT recipients. Error-corrected sequencing...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35971851"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis confers an increased mortality risk in orthotopic heart transplant recipients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35971851/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220819211457&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Novel risk stratification and non-invasive surveillance methods are needed in orthotopic heart transplant (OHT) to reduce morbidity and mortality post-transplant. Clonal hematopoiesis (CH) refers to the acquisition of specific gene mutations in hematopoietic stem cells linked to enhanced inflammation and worse cardiovascular outcomes. The purpose of this study was to investigate the association between CH and OHT. Blood samples were collected from 127 OHT recipients. Error-corrected sequencing...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35971851"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis confers an increased mortality risk in orthotopic heart transplant recipients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35971851/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220818211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Novel risk stratification and non-invasive surveillance methods are needed in orthotopic heart transplant (OHT) to reduce morbidity and mortality post-transplant. Clonal hematopoiesis (CH) refers to the acquisition of specific gene mutations in hematopoietic stem cells linked to enhanced inflammation and worse cardiovascular outcomes. The purpose of this study was to investigate the association between CH and OHT. Blood samples were collected from 127 OHT recipients. Error-corrected sequencing...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35971851"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis confers an increased mortality risk in orthotopic heart transplant recipients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35971851/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220817212606&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Novel risk stratification and non-invasive surveillance methods are needed in orthotopic heart transplant (OHT) to reduce morbidity and mortality post-transplant. Clonal hematopoiesis (CH) refers to the acquisition of specific gene mutations in hematopoietic stem cells linked to enhanced inflammation and worse cardiovascular outcomes. The purpose of this study was to investigate the association between CH and OHT. Blood samples were collected from 127 OHT recipients. Error-corrected sequencing...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35971851"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis confers an increased mortality risk in orthotopic heart transplant recipients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35971851/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220816212355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Novel risk stratification and non-invasive surveillance methods are needed in orthotopic heart transplant (OHT) to reduce morbidity and mortality post-transplant. Clonal hematopoiesis (CH) refers to the acquisition of specific gene mutations in hematopoietic stem cells linked to enhanced inflammation and worse cardiovascular outcomes. The purpose of this study was to investigate the association between CH and OHT. Blood samples were collected from 127 OHT recipients. Error-corrected sequencing...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-08-15 datetime="2022-08-15T10:00:00.000Z">2022-08-15</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-08-15 datetime="2022-08-15T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608003622&amp;v&#x3D;2.17.7&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35970010"
                        open
                      >
                        <summary class="article-expander__title">Geographic disparities in cardiovascular mortality among patients with myelodysplastic syndromes: A population-based analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35970010/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220824212808&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Clonal hematopoiesis, a precursor to myelodysplastic syndromes (MDS), constitutes a novel cardiovascular disease (CVD) risk factor, causing growing interest in cardiovascular outcomes in MDS. Rurality is associated with increased CVD but studies on cardiovascular geographic disparities in MDS are lacking.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35969263"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis: causes and clinical implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35969263/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220824212808&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) refers to hematopoiesis from stem cells with mutations in leukemia-associated driver genes. These confer increased stress tolerance and expansive potential to stem cell clones. Patients with CHIP are hematologically healthy. The main risk factor for the development of CHIP is age or chronic inflammatory processes associated with aging, so-called &quot;inflammaging&quot;. Therefore, the correlation of age-associated comorbidities with the detection of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35970010"
                        open
                      >
                        <summary class="article-expander__title">Geographic disparities in cardiovascular mortality among patients with myelodysplastic syndromes: A population-based analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35970010/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220823212820&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Clonal hematopoiesis, a precursor to myelodysplastic syndromes (MDS), constitutes a novel cardiovascular disease (CVD) risk factor, causing growing interest in cardiovascular outcomes in MDS. Rurality is associated with increased CVD but studies on cardiovascular geographic disparities in MDS are lacking.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35969263"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis: causes and clinical implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35969263/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220823212820&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) refers to hematopoiesis from stem cells with mutations in leukemia-associated driver genes. These confer increased stress tolerance and expansive potential to stem cell clones. Patients with CHIP are hematologically healthy. The main risk factor for the development of CHIP is age or chronic inflammatory processes associated with aging, so-called &quot;inflammaging&quot;. Therefore, the correlation of age-associated comorbidities with the detection of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35970010"
                        open
                      >
                        <summary class="article-expander__title">Geographic disparities in cardiovascular mortality among patients with myelodysplastic syndromes: A population-based analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35970010/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220822212717&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Clonal hematopoiesis, a precursor to myelodysplastic syndromes (MDS), constitutes a novel cardiovascular disease (CVD) risk factor, causing growing interest in cardiovascular outcomes in MDS. Rurality is associated with increased CVD but studies on cardiovascular geographic disparities in MDS are lacking.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35969263"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis: causes and clinical implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35969263/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220822212717&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) refers to hematopoiesis from stem cells with mutations in leukemia-associated driver genes. These confer increased stress tolerance and expansive potential to stem cell clones. Patients with CHIP are hematologically healthy. The main risk factor for the development of CHIP is age or chronic inflammatory processes associated with aging, so-called &quot;inflammaging&quot;. Therefore, the correlation of age-associated comorbidities with the detection of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35970010"
                        open
                      >
                        <summary class="article-expander__title">Geographic disparities in cardiovascular mortality among patients with myelodysplastic syndromes: A population-based analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35970010/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220821211551&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Clonal hematopoiesis, a precursor to myelodysplastic syndromes (MDS), constitutes a novel cardiovascular disease (CVD) risk factor, causing growing interest in cardiovascular outcomes in MDS. Rurality is associated with increased CVD but studies on cardiovascular geographic disparities in MDS are lacking.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35969263"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis: causes and clinical implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35969263/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220821211551&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) refers to hematopoiesis from stem cells with mutations in leukemia-associated driver genes. These confer increased stress tolerance and expansive potential to stem cell clones. Patients with CHIP are hematologically healthy. The main risk factor for the development of CHIP is age or chronic inflammatory processes associated with aging, so-called &quot;inflammaging&quot;. Therefore, the correlation of age-associated comorbidities with the detection of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35970010"
                        open
                      >
                        <summary class="article-expander__title">Geographic disparities in cardiovascular mortality among patients with myelodysplastic syndromes: A population-based analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35970010/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220820211309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Clonal hematopoiesis, a precursor to myelodysplastic syndromes (MDS), constitutes a novel cardiovascular disease (CVD) risk factor, causing growing interest in cardiovascular outcomes in MDS. Rurality is associated with increased CVD but studies on cardiovascular geographic disparities in MDS are lacking.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35969263"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis: causes and clinical implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35969263/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220820211309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) refers to hematopoiesis from stem cells with mutations in leukemia-associated driver genes. These confer increased stress tolerance and expansive potential to stem cell clones. Patients with CHIP are hematologically healthy. The main risk factor for the development of CHIP is age or chronic inflammatory processes associated with aging, so-called &quot;inflammaging&quot;. Therefore, the correlation of age-associated comorbidities with the detection of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35970010"
                        open
                      >
                        <summary class="article-expander__title">Geographic disparities in cardiovascular mortality among patients with myelodysplastic syndromes: A population-based analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35970010/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220819211457&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Clonal hematopoiesis, a precursor to myelodysplastic syndromes (MDS), constitutes a novel cardiovascular disease (CVD) risk factor, causing growing interest in cardiovascular outcomes in MDS. Rurality is associated with increased CVD but studies on cardiovascular geographic disparities in MDS are lacking.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35969263"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis: causes and clinical implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35969263/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220819211457&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) refers to hematopoiesis from stem cells with mutations in leukemia-associated driver genes. These confer increased stress tolerance and expansive potential to stem cell clones. Patients with CHIP are hematologically healthy. The main risk factor for the development of CHIP is age or chronic inflammatory processes associated with aging, so-called &quot;inflammaging&quot;. Therefore, the correlation of age-associated comorbidities with the detection of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35970010"
                        open
                      >
                        <summary class="article-expander__title">Geographic disparities in cardiovascular mortality among patients with myelodysplastic syndromes: A population-based analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35970010/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220818211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Clonal hematopoiesis, a precursor to myelodysplastic syndromes (MDS), constitutes a novel cardiovascular disease (CVD) risk factor, causing growing interest in cardiovascular outcomes in MDS. Rurality is associated with increased CVD but studies on cardiovascular geographic disparities in MDS are lacking.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35969263"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis: causes and clinical implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35969263/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220818211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) refers to hematopoiesis from stem cells with mutations in leukemia-associated driver genes. These confer increased stress tolerance and expansive potential to stem cell clones. Patients with CHIP are hematologically healthy. The main risk factor for the development of CHIP is age or chronic inflammatory processes associated with aging, so-called &quot;inflammaging&quot;. Therefore, the correlation of age-associated comorbidities with the detection of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35970010"
                        open
                      >
                        <summary class="article-expander__title">Geographic disparities in cardiovascular mortality among patients with myelodysplastic syndromes: A population-based analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35970010/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220817212606&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Clonal hematopoiesis, a precursor to myelodysplastic syndromes (MDS), constitutes a novel cardiovascular disease (CVD) risk factor, causing growing interest in cardiovascular outcomes in MDS. Rurality is associated with increased CVD but studies on cardiovascular geographic disparities in MDS are lacking.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35969263"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis: causes and clinical implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35969263/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220817212606&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) refers to hematopoiesis from stem cells with mutations in leukemia-associated driver genes. These confer increased stress tolerance and expansive potential to stem cell clones. Patients with CHIP are hematologically healthy. The main risk factor for the development of CHIP is age or chronic inflammatory processes associated with aging, so-called &quot;inflammaging&quot;. Therefore, the correlation of age-associated comorbidities with the detection of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35970010"
                        open
                      >
                        <summary class="article-expander__title">Geographic disparities in cardiovascular mortality among patients with myelodysplastic syndromes: A population-based analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35970010/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220816212355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Clonal hematopoiesis, a precursor to myelodysplastic syndromes (MDS), constitutes a novel cardiovascular disease (CVD) risk factor, causing growing interest in cardiovascular outcomes in MDS. Rurality is associated with increased CVD but studies on cardiovascular geographic disparities in MDS are lacking.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35969263"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis: causes and clinical implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35969263/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220816212355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) refers to hematopoiesis from stem cells with mutations in leukemia-associated driver genes. These confer increased stress tolerance and expansive potential to stem cell clones. Patients with CHIP are hematologically healthy. The main risk factor for the development of CHIP is age or chronic inflammatory processes associated with aging, so-called &quot;inflammaging&quot;. Therefore, the correlation of age-associated comorbidities with the detection of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35969263"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis: causes and clinical implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35969263/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220815212205&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) refers to hematopoiesis from stem cells with mutations in leukemia-associated driver genes. These confer increased stress tolerance and expansive potential to stem cell clones. Patients with CHIP are hematologically healthy. The main risk factor for the development of CHIP is age or chronic inflammatory processes associated with aging, so-called &quot;inflammaging&quot;. Therefore, the correlation of age-associated comorbidities with the detection of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-08-14 datetime="2022-08-14T10:00:00.000Z">2022-08-14</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-08-14 datetime="2022-08-14T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608010801&amp;v&#x3D;2.17.7&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35964234"
                        open
                      >
                        <summary class="article-expander__title">Histiocytic Sarcoma With CCND1 Gene Rearrangement Clonally Related and Transdifferentiated From Mantle Cell Lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35964234/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220824212808&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: To our knowledge, this is the first report to provide evidence for clonal evolution of histiocytic sarcoma from blastoid mantle cell lymphoma based on cytogenic and molecular findings. The patient&#x27;s protracted therapeutic course may have acted as an evolutionary driver promoting this transdifferentiation process.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35964234"
                        open
                      >
                        <summary class="article-expander__title">Histiocytic Sarcoma With CCND1 Gene Rearrangement Clonally Related and Transdifferentiated From Mantle Cell Lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35964234/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220823212820&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: To our knowledge, this is the first report to provide evidence for clonal evolution of histiocytic sarcoma from blastoid mantle cell lymphoma based on cytogenic and molecular findings. The patient&#x27;s protracted therapeutic course may have acted as an evolutionary driver promoting this transdifferentiation process.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35964234"
                        open
                      >
                        <summary class="article-expander__title">Histiocytic Sarcoma With CCND1 Gene Rearrangement Clonally Related and Transdifferentiated From Mantle Cell Lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35964234/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220822212717&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: To our knowledge, this is the first report to provide evidence for clonal evolution of histiocytic sarcoma from blastoid mantle cell lymphoma based on cytogenic and molecular findings. The patient&#x27;s protracted therapeutic course may have acted as an evolutionary driver promoting this transdifferentiation process.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35964234"
                        open
                      >
                        <summary class="article-expander__title">Histiocytic Sarcoma With CCND1 Gene Rearrangement Clonally Related and Transdifferentiated From Mantle Cell Lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35964234/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220821211551&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: To our knowledge, this is the first report to provide evidence for clonal evolution of histiocytic sarcoma from blastoid mantle cell lymphoma based on cytogenic and molecular findings. The patient&#x27;s protracted therapeutic course may have acted as an evolutionary driver promoting this transdifferentiation process.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35964234"
                        open
                      >
                        <summary class="article-expander__title">Histiocytic Sarcoma With CCND1 Gene Rearrangement Clonally Related and Transdifferentiated From Mantle Cell Lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35964234/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220820211309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: To our knowledge, this is the first report to provide evidence for clonal evolution of histiocytic sarcoma from blastoid mantle cell lymphoma based on cytogenic and molecular findings. The patient&#x27;s protracted therapeutic course may have acted as an evolutionary driver promoting this transdifferentiation process.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35964234"
                        open
                      >
                        <summary class="article-expander__title">Histiocytic Sarcoma With CCND1 Gene Rearrangement Clonally Related and Transdifferentiated From Mantle Cell Lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35964234/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220819211458&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: To our knowledge, this is the first report to provide evidence for clonal evolution of histiocytic sarcoma from blastoid mantle cell lymphoma based on cytogenic and molecular findings. The patient&#x27;s protracted therapeutic course may have acted as an evolutionary driver promoting this transdifferentiation process.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35964234"
                        open
                      >
                        <summary class="article-expander__title">Histiocytic Sarcoma With CCND1 Gene Rearrangement Clonally Related and Transdifferentiated From Mantle Cell Lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35964234/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220818211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: To our knowledge, this is the first report to provide evidence for clonal evolution of histiocytic sarcoma from blastoid mantle cell lymphoma based on cytogenic and molecular findings. The patient&#x27;s protracted therapeutic course may have acted as an evolutionary driver promoting this transdifferentiation process.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35964234"
                        open
                      >
                        <summary class="article-expander__title">Histiocytic Sarcoma With CCND1 Gene Rearrangement Clonally Related and Transdifferentiated From Mantle Cell Lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35964234/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220817212606&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: To our knowledge, this is the first report to provide evidence for clonal evolution of histiocytic sarcoma from blastoid mantle cell lymphoma based on cytogenic and molecular findings. The patient&#x27;s protracted therapeutic course may have acted as an evolutionary driver promoting this transdifferentiation process.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35964234"
                        open
                      >
                        <summary class="article-expander__title">Histiocytic Sarcoma With CCND1 Gene Rearrangement Clonally Related and Transdifferentiated From Mantle Cell Lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35964234/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220816212355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: To our knowledge, this is the first report to provide evidence for clonal evolution of histiocytic sarcoma from blastoid mantle cell lymphoma based on cytogenic and molecular findings. The patient&#x27;s protracted therapeutic course may have acted as an evolutionary driver promoting this transdifferentiation process.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35964234"
                        open
                      >
                        <summary class="article-expander__title">Histiocytic Sarcoma With CCND1 Gene Rearrangement Clonally Related and Transdifferentiated From Mantle Cell Lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35964234/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220815212205&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: To our knowledge, this is the first report to provide evidence for clonal evolution of histiocytic sarcoma from blastoid mantle cell lymphoma based on cytogenic and molecular findings. The patient&#x27;s protracted therapeutic course may have acted as an evolutionary driver promoting this transdifferentiation process.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35964234"
                        open
                      >
                        <summary class="article-expander__title">Histiocytic Sarcoma With CCND1 Gene Rearrangement Clonally Related and Transdifferentiated From Mantle Cell Lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35964234/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220814211915&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: To our knowledge, this is the first report to provide evidence for clonal evolution of histiocytic sarcoma from blastoid mantle cell lymphoma based on cytogenic and molecular findings. The patient&#x27;s protracted therapeutic course may have acted as an evolutionary driver promoting this transdifferentiation process.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-08-12 datetime="2022-08-12T10:00:00.000Z">2022-08-12</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-08-12 datetime="2022-08-12T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608003622&amp;v&#x3D;2.17.7&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35954328"
                        open
                      >
                        <summary class="article-expander__title">A Synopsis Clonal Hematopoiesis of Indeterminate Potential in Hematology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35954328/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220824212808&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential can be defined as genetic mutations that correlate in hematologic neoplasia such as myelodysplastic syndrome. Patients with cytopenia increasingly undergo molecular genetic tests of peripheral blood or bone marrow for diagnostic purposes. Recently, a new entity has been demarcated to lessen the risk of incorrect diagnoses of hematologic malignancies. This new entity is a potential precursor of myeloid diseases, analogous to monoclonal gammopathy of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35954328"
                        open
                      >
                        <summary class="article-expander__title">A Synopsis Clonal Hematopoiesis of Indeterminate Potential in Hematology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35954328/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220823212820&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential can be defined as genetic mutations that correlate in hematologic neoplasia such as myelodysplastic syndrome. Patients with cytopenia increasingly undergo molecular genetic tests of peripheral blood or bone marrow for diagnostic purposes. Recently, a new entity has been demarcated to lessen the risk of incorrect diagnoses of hematologic malignancies. This new entity is a potential precursor of myeloid diseases, analogous to monoclonal gammopathy of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35954328"
                        open
                      >
                        <summary class="article-expander__title">A Synopsis Clonal Hematopoiesis of Indeterminate Potential in Hematology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35954328/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220822212717&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential can be defined as genetic mutations that correlate in hematologic neoplasia such as myelodysplastic syndrome. Patients with cytopenia increasingly undergo molecular genetic tests of peripheral blood or bone marrow for diagnostic purposes. Recently, a new entity has been demarcated to lessen the risk of incorrect diagnoses of hematologic malignancies. This new entity is a potential precursor of myeloid diseases, analogous to monoclonal gammopathy of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35954328"
                        open
                      >
                        <summary class="article-expander__title">A Synopsis Clonal Hematopoiesis of Indeterminate Potential in Hematology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35954328/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220821211551&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential can be defined as genetic mutations that correlate in hematologic neoplasia such as myelodysplastic syndrome. Patients with cytopenia increasingly undergo molecular genetic tests of peripheral blood or bone marrow for diagnostic purposes. Recently, a new entity has been demarcated to lessen the risk of incorrect diagnoses of hematologic malignancies. This new entity is a potential precursor of myeloid diseases, analogous to monoclonal gammopathy of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35954328"
                        open
                      >
                        <summary class="article-expander__title">A Synopsis Clonal Hematopoiesis of Indeterminate Potential in Hematology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35954328/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220820211309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential can be defined as genetic mutations that correlate in hematologic neoplasia such as myelodysplastic syndrome. Patients with cytopenia increasingly undergo molecular genetic tests of peripheral blood or bone marrow for diagnostic purposes. Recently, a new entity has been demarcated to lessen the risk of incorrect diagnoses of hematologic malignancies. This new entity is a potential precursor of myeloid diseases, analogous to monoclonal gammopathy of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35954328"
                        open
                      >
                        <summary class="article-expander__title">A Synopsis Clonal Hematopoiesis of Indeterminate Potential in Hematology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35954328/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220819211457&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential can be defined as genetic mutations that correlate in hematologic neoplasia such as myelodysplastic syndrome. Patients with cytopenia increasingly undergo molecular genetic tests of peripheral blood or bone marrow for diagnostic purposes. Recently, a new entity has been demarcated to lessen the risk of incorrect diagnoses of hematologic malignancies. This new entity is a potential precursor of myeloid diseases, analogous to monoclonal gammopathy of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35954328"
                        open
                      >
                        <summary class="article-expander__title">A Synopsis Clonal Hematopoiesis of Indeterminate Potential in Hematology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35954328/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220818211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential can be defined as genetic mutations that correlate in hematologic neoplasia such as myelodysplastic syndrome. Patients with cytopenia increasingly undergo molecular genetic tests of peripheral blood or bone marrow for diagnostic purposes. Recently, a new entity has been demarcated to lessen the risk of incorrect diagnoses of hematologic malignancies. This new entity is a potential precursor of myeloid diseases, analogous to monoclonal gammopathy of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35954328"
                        open
                      >
                        <summary class="article-expander__title">A Synopsis Clonal Hematopoiesis of Indeterminate Potential in Hematology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35954328/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220817212606&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential can be defined as genetic mutations that correlate in hematologic neoplasia such as myelodysplastic syndrome. Patients with cytopenia increasingly undergo molecular genetic tests of peripheral blood or bone marrow for diagnostic purposes. Recently, a new entity has been demarcated to lessen the risk of incorrect diagnoses of hematologic malignancies. This new entity is a potential precursor of myeloid diseases, analogous to monoclonal gammopathy of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35954328"
                        open
                      >
                        <summary class="article-expander__title">A Synopsis Clonal Hematopoiesis of Indeterminate Potential in Hematology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35954328/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220816212355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential can be defined as genetic mutations that correlate in hematologic neoplasia such as myelodysplastic syndrome. Patients with cytopenia increasingly undergo molecular genetic tests of peripheral blood or bone marrow for diagnostic purposes. Recently, a new entity has been demarcated to lessen the risk of incorrect diagnoses of hematologic malignancies. This new entity is a potential precursor of myeloid diseases, analogous to monoclonal gammopathy of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35954328"
                        open
                      >
                        <summary class="article-expander__title">A Synopsis Clonal Hematopoiesis of Indeterminate Potential in Hematology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35954328/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220815212205&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential can be defined as genetic mutations that correlate in hematologic neoplasia such as myelodysplastic syndrome. Patients with cytopenia increasingly undergo molecular genetic tests of peripheral blood or bone marrow for diagnostic purposes. Recently, a new entity has been demarcated to lessen the risk of incorrect diagnoses of hematologic malignancies. This new entity is a potential precursor of myeloid diseases, analogous to monoclonal gammopathy of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35954328"
                        open
                      >
                        <summary class="article-expander__title">A Synopsis Clonal Hematopoiesis of Indeterminate Potential in Hematology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35954328/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220814211915&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential can be defined as genetic mutations that correlate in hematologic neoplasia such as myelodysplastic syndrome. Patients with cytopenia increasingly undergo molecular genetic tests of peripheral blood or bone marrow for diagnostic purposes. Recently, a new entity has been demarcated to lessen the risk of incorrect diagnoses of hematologic malignancies. This new entity is a potential precursor of myeloid diseases, analogous to monoclonal gammopathy of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35954328"
                        open
                      >
                        <summary class="article-expander__title">A Synopsis Clonal Hematopoiesis of Indeterminate Potential in Hematology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35954328/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220813211549&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential can be defined as genetic mutations that correlate in hematologic neoplasia such as myelodysplastic syndrome. Patients with cytopenia increasingly undergo molecular genetic tests of peripheral blood or bone marrow for diagnostic purposes. Recently, a new entity has been demarcated to lessen the risk of incorrect diagnoses of hematologic malignancies. This new entity is a potential precursor of myeloid diseases, analogous to monoclonal gammopathy of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35954328"
                        open
                      >
                        <summary class="article-expander__title">A Synopsis Clonal Hematopoiesis of Indeterminate Potential in Hematology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35954328/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220812210706&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential can be defined as genetic mutations that correlate in hematologic neoplasia such as myelodysplastic syndrome. Patients with cytopenia increasingly undergo molecular genetic tests of peripheral blood or bone marrow for diagnostic purposes. Recently, a new entity has been demarcated to lessen the risk of incorrect diagnoses of hematologic malignancies. This new entity is a potential precursor of myeloid diseases, analogous to monoclonal gammopathy of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608010801&amp;v&#x3D;2.17.7&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35960050"
                        open
                      >
                        <summary class="article-expander__title">Transformation from polycythemia vera to acute promyelocytic leukemia: Case report and literature review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35960050/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220824212808&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Polycythemia vera can transform into acute promyelocytic leukemia; therefore, it is important to review bone aspiration and other tests to perform a comprehensive assessment and monitor the disease status, to detect disease progression and intervene early when it transforms into acute promyelocytic leukemia.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35954363"
                        open
                      >
                        <summary class="article-expander__title">Targeted Mutational Analysis of Circulating Tumor DNA to Decipher Temporal Heterogeneity of High-Grade Serous Ovarian Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35954363/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220824212808&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We have previously demonstrated that longitudinal untargeted analysis of plasma samples withdrawn from patients with high-grade serous ovarian cancer (HGS-EOC) can intercept the presence of molecular recurrence (TRm) earlier than the diagnosis of clinical recurrence (TRc). This finding opens a clinical important temporal window to acquire through plasma sample analysis a real-time picture of those emerging molecular lesions that will drive and sustain the growth of relapsed disease and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35960050"
                        open
                      >
                        <summary class="article-expander__title">Transformation from polycythemia vera to acute promyelocytic leukemia: Case report and literature review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35960050/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220823212820&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Polycythemia vera can transform into acute promyelocytic leukemia; therefore, it is important to review bone aspiration and other tests to perform a comprehensive assessment and monitor the disease status, to detect disease progression and intervene early when it transforms into acute promyelocytic leukemia.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35954363"
                        open
                      >
                        <summary class="article-expander__title">Targeted Mutational Analysis of Circulating Tumor DNA to Decipher Temporal Heterogeneity of High-Grade Serous Ovarian Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35954363/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220823212820&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We have previously demonstrated that longitudinal untargeted analysis of plasma samples withdrawn from patients with high-grade serous ovarian cancer (HGS-EOC) can intercept the presence of molecular recurrence (TRm) earlier than the diagnosis of clinical recurrence (TRc). This finding opens a clinical important temporal window to acquire through plasma sample analysis a real-time picture of those emerging molecular lesions that will drive and sustain the growth of relapsed disease and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35960050"
                        open
                      >
                        <summary class="article-expander__title">Transformation from polycythemia vera to acute promyelocytic leukemia: Case report and literature review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35960050/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220822212717&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Polycythemia vera can transform into acute promyelocytic leukemia; therefore, it is important to review bone aspiration and other tests to perform a comprehensive assessment and monitor the disease status, to detect disease progression and intervene early when it transforms into acute promyelocytic leukemia.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35954363"
                        open
                      >
                        <summary class="article-expander__title">Targeted Mutational Analysis of Circulating Tumor DNA to Decipher Temporal Heterogeneity of High-Grade Serous Ovarian Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35954363/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220822212717&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We have previously demonstrated that longitudinal untargeted analysis of plasma samples withdrawn from patients with high-grade serous ovarian cancer (HGS-EOC) can intercept the presence of molecular recurrence (TRm) earlier than the diagnosis of clinical recurrence (TRc). This finding opens a clinical important temporal window to acquire through plasma sample analysis a real-time picture of those emerging molecular lesions that will drive and sustain the growth of relapsed disease and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35960050"
                        open
                      >
                        <summary class="article-expander__title">Transformation from polycythemia vera to acute promyelocytic leukemia: Case report and literature review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35960050/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220821211551&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Polycythemia vera can transform into acute promyelocytic leukemia; therefore, it is important to review bone aspiration and other tests to perform a comprehensive assessment and monitor the disease status, to detect disease progression and intervene early when it transforms into acute promyelocytic leukemia.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35954363"
                        open
                      >
                        <summary class="article-expander__title">Targeted Mutational Analysis of Circulating Tumor DNA to Decipher Temporal Heterogeneity of High-Grade Serous Ovarian Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35954363/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220821211551&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We have previously demonstrated that longitudinal untargeted analysis of plasma samples withdrawn from patients with high-grade serous ovarian cancer (HGS-EOC) can intercept the presence of molecular recurrence (TRm) earlier than the diagnosis of clinical recurrence (TRc). This finding opens a clinical important temporal window to acquire through plasma sample analysis a real-time picture of those emerging molecular lesions that will drive and sustain the growth of relapsed disease and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35960050"
                        open
                      >
                        <summary class="article-expander__title">Transformation from polycythemia vera to acute promyelocytic leukemia: Case report and literature review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35960050/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220820211309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Polycythemia vera can transform into acute promyelocytic leukemia; therefore, it is important to review bone aspiration and other tests to perform a comprehensive assessment and monitor the disease status, to detect disease progression and intervene early when it transforms into acute promyelocytic leukemia.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35954363"
                        open
                      >
                        <summary class="article-expander__title">Targeted Mutational Analysis of Circulating Tumor DNA to Decipher Temporal Heterogeneity of High-Grade Serous Ovarian Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35954363/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220820211309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We have previously demonstrated that longitudinal untargeted analysis of plasma samples withdrawn from patients with high-grade serous ovarian cancer (HGS-EOC) can intercept the presence of molecular recurrence (TRm) earlier than the diagnosis of clinical recurrence (TRc). This finding opens a clinical important temporal window to acquire through plasma sample analysis a real-time picture of those emerging molecular lesions that will drive and sustain the growth of relapsed disease and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35960050"
                        open
                      >
                        <summary class="article-expander__title">Transformation from polycythemia vera to acute promyelocytic leukemia: Case report and literature review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35960050/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220819211458&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Polycythemia vera can transform into acute promyelocytic leukemia; therefore, it is important to review bone aspiration and other tests to perform a comprehensive assessment and monitor the disease status, to detect disease progression and intervene early when it transforms into acute promyelocytic leukemia.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35954363"
                        open
                      >
                        <summary class="article-expander__title">Targeted Mutational Analysis of Circulating Tumor DNA to Decipher Temporal Heterogeneity of High-Grade Serous Ovarian Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35954363/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220819211458&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We have previously demonstrated that longitudinal untargeted analysis of plasma samples withdrawn from patients with high-grade serous ovarian cancer (HGS-EOC) can intercept the presence of molecular recurrence (TRm) earlier than the diagnosis of clinical recurrence (TRc). This finding opens a clinical important temporal window to acquire through plasma sample analysis a real-time picture of those emerging molecular lesions that will drive and sustain the growth of relapsed disease and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35960050"
                        open
                      >
                        <summary class="article-expander__title">Transformation from polycythemia vera to acute promyelocytic leukemia: Case report and literature review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35960050/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220818211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Polycythemia vera can transform into acute promyelocytic leukemia; therefore, it is important to review bone aspiration and other tests to perform a comprehensive assessment and monitor the disease status, to detect disease progression and intervene early when it transforms into acute promyelocytic leukemia.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35954363"
                        open
                      >
                        <summary class="article-expander__title">Targeted Mutational Analysis of Circulating Tumor DNA to Decipher Temporal Heterogeneity of High-Grade Serous Ovarian Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35954363/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220818211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We have previously demonstrated that longitudinal untargeted analysis of plasma samples withdrawn from patients with high-grade serous ovarian cancer (HGS-EOC) can intercept the presence of molecular recurrence (TRm) earlier than the diagnosis of clinical recurrence (TRc). This finding opens a clinical important temporal window to acquire through plasma sample analysis a real-time picture of those emerging molecular lesions that will drive and sustain the growth of relapsed disease and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35960050"
                        open
                      >
                        <summary class="article-expander__title">Transformation from polycythemia vera to acute promyelocytic leukemia: Case report and literature review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35960050/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220817212606&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Polycythemia vera can transform into acute promyelocytic leukemia; therefore, it is important to review bone aspiration and other tests to perform a comprehensive assessment and monitor the disease status, to detect disease progression and intervene early when it transforms into acute promyelocytic leukemia.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35954363"
                        open
                      >
                        <summary class="article-expander__title">Targeted Mutational Analysis of Circulating Tumor DNA to Decipher Temporal Heterogeneity of High-Grade Serous Ovarian Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35954363/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220817212606&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We have previously demonstrated that longitudinal untargeted analysis of plasma samples withdrawn from patients with high-grade serous ovarian cancer (HGS-EOC) can intercept the presence of molecular recurrence (TRm) earlier than the diagnosis of clinical recurrence (TRc). This finding opens a clinical important temporal window to acquire through plasma sample analysis a real-time picture of those emerging molecular lesions that will drive and sustain the growth of relapsed disease and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35960050"
                        open
                      >
                        <summary class="article-expander__title">Transformation from polycythemia vera to acute promyelocytic leukemia: Case report and literature review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35960050/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220816212355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Polycythemia vera can transform into acute promyelocytic leukemia; therefore, it is important to review bone aspiration and other tests to perform a comprehensive assessment and monitor the disease status, to detect disease progression and intervene early when it transforms into acute promyelocytic leukemia.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35954363"
                        open
                      >
                        <summary class="article-expander__title">Targeted Mutational Analysis of Circulating Tumor DNA to Decipher Temporal Heterogeneity of High-Grade Serous Ovarian Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35954363/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220816212355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We have previously demonstrated that longitudinal untargeted analysis of plasma samples withdrawn from patients with high-grade serous ovarian cancer (HGS-EOC) can intercept the presence of molecular recurrence (TRm) earlier than the diagnosis of clinical recurrence (TRc). This finding opens a clinical important temporal window to acquire through plasma sample analysis a real-time picture of those emerging molecular lesions that will drive and sustain the growth of relapsed disease and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35954363"
                        open
                      >
                        <summary class="article-expander__title">Targeted Mutational Analysis of Circulating Tumor DNA to Decipher Temporal Heterogeneity of High-Grade Serous Ovarian Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35954363/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220815212205&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We have previously demonstrated that longitudinal untargeted analysis of plasma samples withdrawn from patients with high-grade serous ovarian cancer (HGS-EOC) can intercept the presence of molecular recurrence (TRm) earlier than the diagnosis of clinical recurrence (TRc). This finding opens a clinical important temporal window to acquire through plasma sample analysis a real-time picture of those emerging molecular lesions that will drive and sustain the growth of relapsed disease and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35954363"
                        open
                      >
                        <summary class="article-expander__title">Targeted Mutational Analysis of Circulating Tumor DNA to Decipher Temporal Heterogeneity of High-Grade Serous Ovarian Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35954363/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220814211915&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We have previously demonstrated that longitudinal untargeted analysis of plasma samples withdrawn from patients with high-grade serous ovarian cancer (HGS-EOC) can intercept the presence of molecular recurrence (TRm) earlier than the diagnosis of clinical recurrence (TRc). This finding opens a clinical important temporal window to acquire through plasma sample analysis a real-time picture of those emerging molecular lesions that will drive and sustain the growth of relapsed disease and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35954363"
                        open
                      >
                        <summary class="article-expander__title">Targeted Mutational Analysis of Circulating Tumor DNA to Decipher Temporal Heterogeneity of High-Grade Serous Ovarian Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35954363/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220813211549&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We have previously demonstrated that longitudinal untargeted analysis of plasma samples withdrawn from patients with high-grade serous ovarian cancer (HGS-EOC) can intercept the presence of molecular recurrence (TRm) earlier than the diagnosis of clinical recurrence (TRc). This finding opens a clinical important temporal window to acquire through plasma sample analysis a real-time picture of those emerging molecular lesions that will drive and sustain the growth of relapsed disease and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35954363"
                        open
                      >
                        <summary class="article-expander__title">Targeted Mutational Analysis of Circulating Tumor DNA to Decipher Temporal Heterogeneity of High-Grade Serous Ovarian Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35954363/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220812210706&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We have previously demonstrated that longitudinal untargeted analysis of plasma samples withdrawn from patients with high-grade serous ovarian cancer (HGS-EOC) can intercept the presence of molecular recurrence (TRm) earlier than the diagnosis of clinical recurrence (TRc). This finding opens a clinical important temporal window to acquire through plasma sample analysis a real-time picture of those emerging molecular lesions that will drive and sustain the growth of relapsed disease and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-08-11 datetime="2022-08-11T10:00:00.000Z">2022-08-11</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-08-11 datetime="2022-08-11T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608003622&amp;v&#x3D;2.17.7&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35953207"
                        open
                      >
                        <summary class="article-expander__title">Immunoblotting-assisted assessment of JAK/STAT and PI3K/Akt/mTOR signaling in myeloproliferative neoplasms CD34+ stem cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35953207/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220824212808&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Philadelphia-negative myeloproliferative neoplasms (pH-MPNs) origin from the clonal expansion of hematopoietic stem cells with acquired mutations leading to uncontrolled proliferation of differentiated myeloid cells. The main entities of Ph-MPNs are represented by Polycythemia Vera (PV), Essential Thrombocythemia (ET) and Myelofibrosis (MF) that are characterized by microvascular disorders, thrombosis and bleeding, splenomegaly secondary to extramedullary hematopoiesis, various degree of bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35953207"
                        open
                      >
                        <summary class="article-expander__title">Immunoblotting-assisted assessment of JAK/STAT and PI3K/Akt/mTOR signaling in myeloproliferative neoplasms CD34+ stem cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35953207/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220823212820&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Philadelphia-negative myeloproliferative neoplasms (pH-MPNs) origin from the clonal expansion of hematopoietic stem cells with acquired mutations leading to uncontrolled proliferation of differentiated myeloid cells. The main entities of Ph-MPNs are represented by Polycythemia Vera (PV), Essential Thrombocythemia (ET) and Myelofibrosis (MF) that are characterized by microvascular disorders, thrombosis and bleeding, splenomegaly secondary to extramedullary hematopoiesis, various degree of bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35953207"
                        open
                      >
                        <summary class="article-expander__title">Immunoblotting-assisted assessment of JAK/STAT and PI3K/Akt/mTOR signaling in myeloproliferative neoplasms CD34+ stem cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35953207/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220822212717&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Philadelphia-negative myeloproliferative neoplasms (pH-MPNs) origin from the clonal expansion of hematopoietic stem cells with acquired mutations leading to uncontrolled proliferation of differentiated myeloid cells. The main entities of Ph-MPNs are represented by Polycythemia Vera (PV), Essential Thrombocythemia (ET) and Myelofibrosis (MF) that are characterized by microvascular disorders, thrombosis and bleeding, splenomegaly secondary to extramedullary hematopoiesis, various degree of bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35953207"
                        open
                      >
                        <summary class="article-expander__title">Immunoblotting-assisted assessment of JAK/STAT and PI3K/Akt/mTOR signaling in myeloproliferative neoplasms CD34+ stem cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35953207/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220821211551&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Philadelphia-negative myeloproliferative neoplasms (pH-MPNs) origin from the clonal expansion of hematopoietic stem cells with acquired mutations leading to uncontrolled proliferation of differentiated myeloid cells. The main entities of Ph-MPNs are represented by Polycythemia Vera (PV), Essential Thrombocythemia (ET) and Myelofibrosis (MF) that are characterized by microvascular disorders, thrombosis and bleeding, splenomegaly secondary to extramedullary hematopoiesis, various degree of bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35953207"
                        open
                      >
                        <summary class="article-expander__title">Immunoblotting-assisted assessment of JAK/STAT and PI3K/Akt/mTOR signaling in myeloproliferative neoplasms CD34+ stem cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35953207/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220820211309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Philadelphia-negative myeloproliferative neoplasms (pH-MPNs) origin from the clonal expansion of hematopoietic stem cells with acquired mutations leading to uncontrolled proliferation of differentiated myeloid cells. The main entities of Ph-MPNs are represented by Polycythemia Vera (PV), Essential Thrombocythemia (ET) and Myelofibrosis (MF) that are characterized by microvascular disorders, thrombosis and bleeding, splenomegaly secondary to extramedullary hematopoiesis, various degree of bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35953207"
                        open
                      >
                        <summary class="article-expander__title">Immunoblotting-assisted assessment of JAK/STAT and PI3K/Akt/mTOR signaling in myeloproliferative neoplasms CD34+ stem cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35953207/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220819211457&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Philadelphia-negative myeloproliferative neoplasms (pH-MPNs) origin from the clonal expansion of hematopoietic stem cells with acquired mutations leading to uncontrolled proliferation of differentiated myeloid cells. The main entities of Ph-MPNs are represented by Polycythemia Vera (PV), Essential Thrombocythemia (ET) and Myelofibrosis (MF) that are characterized by microvascular disorders, thrombosis and bleeding, splenomegaly secondary to extramedullary hematopoiesis, various degree of bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35953207"
                        open
                      >
                        <summary class="article-expander__title">Immunoblotting-assisted assessment of JAK/STAT and PI3K/Akt/mTOR signaling in myeloproliferative neoplasms CD34+ stem cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35953207/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220818211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Philadelphia-negative myeloproliferative neoplasms (pH-MPNs) origin from the clonal expansion of hematopoietic stem cells with acquired mutations leading to uncontrolled proliferation of differentiated myeloid cells. The main entities of Ph-MPNs are represented by Polycythemia Vera (PV), Essential Thrombocythemia (ET) and Myelofibrosis (MF) that are characterized by microvascular disorders, thrombosis and bleeding, splenomegaly secondary to extramedullary hematopoiesis, various degree of bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35953207"
                        open
                      >
                        <summary class="article-expander__title">Immunoblotting-assisted assessment of JAK/STAT and PI3K/Akt/mTOR signaling in myeloproliferative neoplasms CD34+ stem cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35953207/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220817212606&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Philadelphia-negative myeloproliferative neoplasms (pH-MPNs) origin from the clonal expansion of hematopoietic stem cells with acquired mutations leading to uncontrolled proliferation of differentiated myeloid cells. The main entities of Ph-MPNs are represented by Polycythemia Vera (PV), Essential Thrombocythemia (ET) and Myelofibrosis (MF) that are characterized by microvascular disorders, thrombosis and bleeding, splenomegaly secondary to extramedullary hematopoiesis, various degree of bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35953207"
                        open
                      >
                        <summary class="article-expander__title">Immunoblotting-assisted assessment of JAK/STAT and PI3K/Akt/mTOR signaling in myeloproliferative neoplasms CD34+ stem cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35953207/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220816212355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Philadelphia-negative myeloproliferative neoplasms (pH-MPNs) origin from the clonal expansion of hematopoietic stem cells with acquired mutations leading to uncontrolled proliferation of differentiated myeloid cells. The main entities of Ph-MPNs are represented by Polycythemia Vera (PV), Essential Thrombocythemia (ET) and Myelofibrosis (MF) that are characterized by microvascular disorders, thrombosis and bleeding, splenomegaly secondary to extramedullary hematopoiesis, various degree of bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35953207"
                        open
                      >
                        <summary class="article-expander__title">Immunoblotting-assisted assessment of JAK/STAT and PI3K/Akt/mTOR signaling in myeloproliferative neoplasms CD34+ stem cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35953207/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220815212205&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Philadelphia-negative myeloproliferative neoplasms (pH-MPNs) origin from the clonal expansion of hematopoietic stem cells with acquired mutations leading to uncontrolled proliferation of differentiated myeloid cells. The main entities of Ph-MPNs are represented by Polycythemia Vera (PV), Essential Thrombocythemia (ET) and Myelofibrosis (MF) that are characterized by microvascular disorders, thrombosis and bleeding, splenomegaly secondary to extramedullary hematopoiesis, various degree of bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35953207"
                        open
                      >
                        <summary class="article-expander__title">Immunoblotting-assisted assessment of JAK/STAT and PI3K/Akt/mTOR signaling in myeloproliferative neoplasms CD34+ stem cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35953207/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220814211915&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Philadelphia-negative myeloproliferative neoplasms (pH-MPNs) origin from the clonal expansion of hematopoietic stem cells with acquired mutations leading to uncontrolled proliferation of differentiated myeloid cells. The main entities of Ph-MPNs are represented by Polycythemia Vera (PV), Essential Thrombocythemia (ET) and Myelofibrosis (MF) that are characterized by microvascular disorders, thrombosis and bleeding, splenomegaly secondary to extramedullary hematopoiesis, various degree of bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35953207"
                        open
                      >
                        <summary class="article-expander__title">Immunoblotting-assisted assessment of JAK/STAT and PI3K/Akt/mTOR signaling in myeloproliferative neoplasms CD34+ stem cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35953207/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220813211549&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Philadelphia-negative myeloproliferative neoplasms (pH-MPNs) origin from the clonal expansion of hematopoietic stem cells with acquired mutations leading to uncontrolled proliferation of differentiated myeloid cells. The main entities of Ph-MPNs are represented by Polycythemia Vera (PV), Essential Thrombocythemia (ET) and Myelofibrosis (MF) that are characterized by microvascular disorders, thrombosis and bleeding, splenomegaly secondary to extramedullary hematopoiesis, various degree of bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35953207"
                        open
                      >
                        <summary class="article-expander__title">Immunoblotting-assisted assessment of JAK/STAT and PI3K/Akt/mTOR signaling in myeloproliferative neoplasms CD34+ stem cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35953207/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220812210706&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Philadelphia-negative myeloproliferative neoplasms (pH-MPNs) origin from the clonal expansion of hematopoietic stem cells with acquired mutations leading to uncontrolled proliferation of differentiated myeloid cells. The main entities of Ph-MPNs are represented by Polycythemia Vera (PV), Essential Thrombocythemia (ET) and Myelofibrosis (MF) that are characterized by microvascular disorders, thrombosis and bleeding, splenomegaly secondary to extramedullary hematopoiesis, various degree of bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-08-10 datetime="2022-08-10T10:00:00.000Z">2022-08-10</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-08-10 datetime="2022-08-10T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608010801&amp;v&#x3D;2.17.7&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35948919"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution in primary breast cancers under sequential epirubicin and docetaxel monotherapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35948919/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220824212808&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Sequential epirubicin and docetaxel monotherapy caused profound redistribution of smaller subclones in primary breast cancer, while early truncal mutations and major subclones generally persisted through treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35948758"
                        open
                      >
                        <summary class="article-expander__title">p73 isoforms meet evolution of metastasis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35948758/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220824212808&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer largely adheres to Darwinian selection. Evolutionary forces are prominent during metastasis, the final and incurable disease stage, where cells acquire combinations of advantageous phenotypic features and interact with a dynamically changing microenvironment, in order to overcome the metastatic bottlenecks, while therapy exerts additional selective pressures. As a strategy to increase their fitness, tumors often co-opt developmental and tissue-homeostasis programs. Herein, 25 years after...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35948708"
                        open
                      >
                        <summary class="article-expander__title">Spatially resolved clonal copy number alterations in benign and malignant tissue</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35948708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220824212808&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Defining the transition from benign to malignant tissue is fundamental to improving early diagnosis of cancer¹. Here we use a systematic approach to study spatial genome integrity in situ and describe previously unidentified clonal relationships. We used spatially resolved transcriptomics² to infer spatial copy number variations in &gt;120,000 regions across multiple organs, in benign and malignant tissues. We demonstrate that genome-wide copy number variation reveals distinct clonal patterns...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;19
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35948919"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution in primary breast cancers under sequential epirubicin and docetaxel monotherapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35948919/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220823212820&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Sequential epirubicin and docetaxel monotherapy caused profound redistribution of smaller subclones in primary breast cancer, while early truncal mutations and major subclones generally persisted through treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35948758"
                        open
                      >
                        <summary class="article-expander__title">p73 isoforms meet evolution of metastasis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35948758/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220823212820&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer largely adheres to Darwinian selection. Evolutionary forces are prominent during metastasis, the final and incurable disease stage, where cells acquire combinations of advantageous phenotypic features and interact with a dynamically changing microenvironment, in order to overcome the metastatic bottlenecks, while therapy exerts additional selective pressures. As a strategy to increase their fitness, tumors often co-opt developmental and tissue-homeostasis programs. Herein, 25 years after...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35948708"
                        open
                      >
                        <summary class="article-expander__title">Spatially resolved clonal copy number alterations in benign and malignant tissue</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35948708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220823212820&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Defining the transition from benign to malignant tissue is fundamental to improving early diagnosis of cancer¹. Here we use a systematic approach to study spatial genome integrity in situ and describe previously unidentified clonal relationships. We used spatially resolved transcriptomics² to infer spatial copy number variations in &gt;120,000 regions across multiple organs, in benign and malignant tissues. We demonstrate that genome-wide copy number variation reveals distinct clonal patterns...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35948919"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution in primary breast cancers under sequential epirubicin and docetaxel monotherapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35948919/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220822212717&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Sequential epirubicin and docetaxel monotherapy caused profound redistribution of smaller subclones in primary breast cancer, while early truncal mutations and major subclones generally persisted through treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35948758"
                        open
                      >
                        <summary class="article-expander__title">p73 isoforms meet evolution of metastasis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35948758/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220822212717&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer largely adheres to Darwinian selection. Evolutionary forces are prominent during metastasis, the final and incurable disease stage, where cells acquire combinations of advantageous phenotypic features and interact with a dynamically changing microenvironment, in order to overcome the metastatic bottlenecks, while therapy exerts additional selective pressures. As a strategy to increase their fitness, tumors often co-opt developmental and tissue-homeostasis programs. Herein, 25 years after...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35948708"
                        open
                      >
                        <summary class="article-expander__title">Spatially resolved clonal copy number alterations in benign and malignant tissue</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35948708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220822212717&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Defining the transition from benign to malignant tissue is fundamental to improving early diagnosis of cancer¹. Here we use a systematic approach to study spatial genome integrity in situ and describe previously unidentified clonal relationships. We used spatially resolved transcriptomics² to infer spatial copy number variations in &gt;120,000 regions across multiple organs, in benign and malignant tissues. We demonstrate that genome-wide copy number variation reveals distinct clonal patterns...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35948919"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution in primary breast cancers under sequential epirubicin and docetaxel monotherapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35948919/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220821211551&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Sequential epirubicin and docetaxel monotherapy caused profound redistribution of smaller subclones in primary breast cancer, while early truncal mutations and major subclones generally persisted through treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35948758"
                        open
                      >
                        <summary class="article-expander__title">p73 isoforms meet evolution of metastasis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35948758/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220821211551&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer largely adheres to Darwinian selection. Evolutionary forces are prominent during metastasis, the final and incurable disease stage, where cells acquire combinations of advantageous phenotypic features and interact with a dynamically changing microenvironment, in order to overcome the metastatic bottlenecks, while therapy exerts additional selective pressures. As a strategy to increase their fitness, tumors often co-opt developmental and tissue-homeostasis programs. Herein, 25 years after...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35948708"
                        open
                      >
                        <summary class="article-expander__title">Spatially resolved clonal copy number alterations in benign and malignant tissue</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35948708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220821211551&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Defining the transition from benign to malignant tissue is fundamental to improving early diagnosis of cancer¹. Here we use a systematic approach to study spatial genome integrity in situ and describe previously unidentified clonal relationships. We used spatially resolved transcriptomics² to infer spatial copy number variations in &gt;120,000 regions across multiple organs, in benign and malignant tissues. We demonstrate that genome-wide copy number variation reveals distinct clonal patterns...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35948919"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution in primary breast cancers under sequential epirubicin and docetaxel monotherapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35948919/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220820211309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Sequential epirubicin and docetaxel monotherapy caused profound redistribution of smaller subclones in primary breast cancer, while early truncal mutations and major subclones generally persisted through treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35948758"
                        open
                      >
                        <summary class="article-expander__title">p73 isoforms meet evolution of metastasis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35948758/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220820211309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer largely adheres to Darwinian selection. Evolutionary forces are prominent during metastasis, the final and incurable disease stage, where cells acquire combinations of advantageous phenotypic features and interact with a dynamically changing microenvironment, in order to overcome the metastatic bottlenecks, while therapy exerts additional selective pressures. As a strategy to increase their fitness, tumors often co-opt developmental and tissue-homeostasis programs. Herein, 25 years after...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35948708"
                        open
                      >
                        <summary class="article-expander__title">Spatially resolved clonal copy number alterations in benign and malignant tissue</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35948708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220820211309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Defining the transition from benign to malignant tissue is fundamental to improving early diagnosis of cancer¹. Here we use a systematic approach to study spatial genome integrity in situ and describe previously unidentified clonal relationships. We used spatially resolved transcriptomics² to infer spatial copy number variations in &gt;120,000 regions across multiple organs, in benign and malignant tissues. We demonstrate that genome-wide copy number variation reveals distinct clonal patterns...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35948919"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution in primary breast cancers under sequential epirubicin and docetaxel monotherapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35948919/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220819211458&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Sequential epirubicin and docetaxel monotherapy caused profound redistribution of smaller subclones in primary breast cancer, while early truncal mutations and major subclones generally persisted through treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35948758"
                        open
                      >
                        <summary class="article-expander__title">p73 isoforms meet evolution of metastasis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35948758/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220819211458&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer largely adheres to Darwinian selection. Evolutionary forces are prominent during metastasis, the final and incurable disease stage, where cells acquire combinations of advantageous phenotypic features and interact with a dynamically changing microenvironment, in order to overcome the metastatic bottlenecks, while therapy exerts additional selective pressures. As a strategy to increase their fitness, tumors often co-opt developmental and tissue-homeostasis programs. Herein, 25 years after...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35948708"
                        open
                      >
                        <summary class="article-expander__title">Spatially resolved clonal copy number alterations in benign and malignant tissue</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35948708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220819211458&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Defining the transition from benign to malignant tissue is fundamental to improving early diagnosis of cancer¹. Here we use a systematic approach to study spatial genome integrity in situ and describe previously unidentified clonal relationships. We used spatially resolved transcriptomics² to infer spatial copy number variations in &gt;120,000 regions across multiple organs, in benign and malignant tissues. We demonstrate that genome-wide copy number variation reveals distinct clonal patterns...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35948919"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution in primary breast cancers under sequential epirubicin and docetaxel monotherapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35948919/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220818211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Sequential epirubicin and docetaxel monotherapy caused profound redistribution of smaller subclones in primary breast cancer, while early truncal mutations and major subclones generally persisted through treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35948758"
                        open
                      >
                        <summary class="article-expander__title">p73 isoforms meet evolution of metastasis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35948758/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220818211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer largely adheres to Darwinian selection. Evolutionary forces are prominent during metastasis, the final and incurable disease stage, where cells acquire combinations of advantageous phenotypic features and interact with a dynamically changing microenvironment, in order to overcome the metastatic bottlenecks, while therapy exerts additional selective pressures. As a strategy to increase their fitness, tumors often co-opt developmental and tissue-homeostasis programs. Herein, 25 years after...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35948708"
                        open
                      >
                        <summary class="article-expander__title">Spatially resolved clonal copy number alterations in benign and malignant tissue</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35948708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220818211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Defining the transition from benign to malignant tissue is fundamental to improving early diagnosis of cancer¹. Here we use a systematic approach to study spatial genome integrity in situ and describe previously unidentified clonal relationships. We used spatially resolved transcriptomics² to infer spatial copy number variations in &gt;120,000 regions across multiple organs, in benign and malignant tissues. We demonstrate that genome-wide copy number variation reveals distinct clonal patterns...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35948919"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution in primary breast cancers under sequential epirubicin and docetaxel monotherapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35948919/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220817212606&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Sequential epirubicin and docetaxel monotherapy caused profound redistribution of smaller subclones in primary breast cancer, while early truncal mutations and major subclones generally persisted through treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35948758"
                        open
                      >
                        <summary class="article-expander__title">p73 isoforms meet evolution of metastasis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35948758/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220817212606&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer largely adheres to Darwinian selection. Evolutionary forces are prominent during metastasis, the final and incurable disease stage, where cells acquire combinations of advantageous phenotypic features and interact with a dynamically changing microenvironment, in order to overcome the metastatic bottlenecks, while therapy exerts additional selective pressures. As a strategy to increase their fitness, tumors often co-opt developmental and tissue-homeostasis programs. Herein, 25 years after...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35948708"
                        open
                      >
                        <summary class="article-expander__title">Spatially resolved clonal copy number alterations in benign and malignant tissue</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35948708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220817212606&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Defining the transition from benign to malignant tissue is fundamental to improving early diagnosis of cancer¹. Here we use a systematic approach to study spatial genome integrity in situ and describe previously unidentified clonal relationships. We used spatially resolved transcriptomics² to infer spatial copy number variations in &gt;120,000 regions across multiple organs, in benign and malignant tissues. We demonstrate that genome-wide copy number variation reveals distinct clonal patterns...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35948919"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution in primary breast cancers under sequential epirubicin and docetaxel monotherapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35948919/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220816212355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Sequential epirubicin and docetaxel monotherapy caused profound redistribution of smaller subclones in primary breast cancer, while early truncal mutations and major subclones generally persisted through treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35948758"
                        open
                      >
                        <summary class="article-expander__title">p73 isoforms meet evolution of metastasis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35948758/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220816212355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer largely adheres to Darwinian selection. Evolutionary forces are prominent during metastasis, the final and incurable disease stage, where cells acquire combinations of advantageous phenotypic features and interact with a dynamically changing microenvironment, in order to overcome the metastatic bottlenecks, while therapy exerts additional selective pressures. As a strategy to increase their fitness, tumors often co-opt developmental and tissue-homeostasis programs. Herein, 25 years after...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35948708"
                        open
                      >
                        <summary class="article-expander__title">Spatially resolved clonal copy number alterations in benign and malignant tissue</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35948708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220816212355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Defining the transition from benign to malignant tissue is fundamental to improving early diagnosis of cancer¹. Here we use a systematic approach to study spatial genome integrity in situ and describe previously unidentified clonal relationships. We used spatially resolved transcriptomics² to infer spatial copy number variations in &gt;120,000 regions across multiple organs, in benign and malignant tissues. We demonstrate that genome-wide copy number variation reveals distinct clonal patterns...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35948919"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution in primary breast cancers under sequential epirubicin and docetaxel monotherapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35948919/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220815212205&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Sequential epirubicin and docetaxel monotherapy caused profound redistribution of smaller subclones in primary breast cancer, while early truncal mutations and major subclones generally persisted through treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35948758"
                        open
                      >
                        <summary class="article-expander__title">p73 isoforms meet evolution of metastasis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35948758/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220815212205&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer largely adheres to Darwinian selection. Evolutionary forces are prominent during metastasis, the final and incurable disease stage, where cells acquire combinations of advantageous phenotypic features and interact with a dynamically changing microenvironment, in order to overcome the metastatic bottlenecks, while therapy exerts additional selective pressures. As a strategy to increase their fitness, tumors often co-opt developmental and tissue-homeostasis programs. Herein, 25 years after...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35948708"
                        open
                      >
                        <summary class="article-expander__title">Spatially resolved clonal copy number alterations in benign and malignant tissue</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35948708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220815212205&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Defining the transition from benign to malignant tissue is fundamental to improving early diagnosis of cancer¹. Here we use a systematic approach to study spatial genome integrity in situ and describe previously unidentified clonal relationships. We used spatially resolved transcriptomics² to infer spatial copy number variations in &gt;120,000 regions across multiple organs, in benign and malignant tissues. We demonstrate that genome-wide copy number variation reveals distinct clonal patterns...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35948919"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution in primary breast cancers under sequential epirubicin and docetaxel monotherapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35948919/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220814211915&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Sequential epirubicin and docetaxel monotherapy caused profound redistribution of smaller subclones in primary breast cancer, while early truncal mutations and major subclones generally persisted through treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35948758"
                        open
                      >
                        <summary class="article-expander__title">p73 isoforms meet evolution of metastasis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35948758/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220814211915&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer largely adheres to Darwinian selection. Evolutionary forces are prominent during metastasis, the final and incurable disease stage, where cells acquire combinations of advantageous phenotypic features and interact with a dynamically changing microenvironment, in order to overcome the metastatic bottlenecks, while therapy exerts additional selective pressures. As a strategy to increase their fitness, tumors often co-opt developmental and tissue-homeostasis programs. Herein, 25 years after...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35948708"
                        open
                      >
                        <summary class="article-expander__title">Spatially resolved clonal copy number alterations in benign and malignant tissue</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35948708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220814211915&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Defining the transition from benign to malignant tissue is fundamental to improving early diagnosis of cancer¹. Here we use a systematic approach to study spatial genome integrity in situ and describe previously unidentified clonal relationships. We used spatially resolved transcriptomics² to infer spatial copy number variations in &gt;120,000 regions across multiple organs, in benign and malignant tissues. We demonstrate that genome-wide copy number variation reveals distinct clonal patterns...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35948919"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution in primary breast cancers under sequential epirubicin and docetaxel monotherapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35948919/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220813211549&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Sequential epirubicin and docetaxel monotherapy caused profound redistribution of smaller subclones in primary breast cancer, while early truncal mutations and major subclones generally persisted through treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35948758"
                        open
                      >
                        <summary class="article-expander__title">p73 isoforms meet evolution of metastasis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35948758/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220813211549&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer largely adheres to Darwinian selection. Evolutionary forces are prominent during metastasis, the final and incurable disease stage, where cells acquire combinations of advantageous phenotypic features and interact with a dynamically changing microenvironment, in order to overcome the metastatic bottlenecks, while therapy exerts additional selective pressures. As a strategy to increase their fitness, tumors often co-opt developmental and tissue-homeostasis programs. Herein, 25 years after...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35948708"
                        open
                      >
                        <summary class="article-expander__title">Spatially resolved clonal copy number alterations in benign and malignant tissue</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35948708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220813211549&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Defining the transition from benign to malignant tissue is fundamental to improving early diagnosis of cancer¹. Here we use a systematic approach to study spatial genome integrity in situ and describe previously unidentified clonal relationships. We used spatially resolved transcriptomics² to infer spatial copy number variations in &gt;120,000 regions across multiple organs, in benign and malignant tissues. We demonstrate that genome-wide copy number variation reveals distinct clonal patterns...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35948919"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution in primary breast cancers under sequential epirubicin and docetaxel monotherapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35948919/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220812210706&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Sequential epirubicin and docetaxel monotherapy caused profound redistribution of smaller subclones in primary breast cancer, while early truncal mutations and major subclones generally persisted through treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35948758"
                        open
                      >
                        <summary class="article-expander__title">p73 isoforms meet evolution of metastasis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35948758/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220812210706&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer largely adheres to Darwinian selection. Evolutionary forces are prominent during metastasis, the final and incurable disease stage, where cells acquire combinations of advantageous phenotypic features and interact with a dynamically changing microenvironment, in order to overcome the metastatic bottlenecks, while therapy exerts additional selective pressures. As a strategy to increase their fitness, tumors often co-opt developmental and tissue-homeostasis programs. Herein, 25 years after...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35948708"
                        open
                      >
                        <summary class="article-expander__title">Spatially resolved clonal copy number alterations in benign and malignant tissue</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35948708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220812210706&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Defining the transition from benign to malignant tissue is fundamental to improving early diagnosis of cancer¹. Here we use a systematic approach to study spatial genome integrity in situ and describe previously unidentified clonal relationships. We used spatially resolved transcriptomics² to infer spatial copy number variations in &gt;120,000 regions across multiple organs, in benign and malignant tissues. We demonstrate that genome-wide copy number variation reveals distinct clonal patterns...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35948919"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution in primary breast cancers under sequential epirubicin and docetaxel monotherapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35948919/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220811211442&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Sequential epirubicin and docetaxel monotherapy caused profound redistribution of smaller subclones in primary breast cancer, while early truncal mutations and major subclones generally persisted through treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35948758"
                        open
                      >
                        <summary class="article-expander__title">p73 isoforms meet evolution of metastasis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35948758/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220811211442&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer largely adheres to Darwinian selection. Evolutionary forces are prominent during metastasis, the final and incurable disease stage, where cells acquire combinations of advantageous phenotypic features and interact with a dynamically changing microenvironment, in order to overcome the metastatic bottlenecks, while therapy exerts additional selective pressures. As a strategy to increase their fitness, tumors often co-opt developmental and tissue-homeostasis programs. Herein, 25 years after...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35948708"
                        open
                      >
                        <summary class="article-expander__title">Spatially resolved clonal copy number alterations in benign and malignant tissue</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35948708/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220811211442&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Defining the transition from benign to malignant tissue is fundamental to improving early diagnosis of cancer¹. Here we use a systematic approach to study spatial genome integrity in situ and describe previously unidentified clonal relationships. We used spatially resolved transcriptomics² to infer spatial copy number variations in &gt;120,000 regions across multiple organs, in benign and malignant tissues. We demonstrate that genome-wide copy number variation reveals distinct clonal patterns...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-08-09 datetime="2022-08-09T10:00:00.000Z">2022-08-09</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-08-09 datetime="2022-08-09T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608003622&amp;v&#x3D;2.17.7&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35945112"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis, inflammation, and cardiovascular disorders: a mitochondrial connection</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35945112/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220824212808&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in two antagonistic regulators of DNA methylation, DNMT3A and TET2, are associated with clonal hematopoiesis and increased risk of cardiovascular disorders. Recently, Cobo et al. traced the mechanistic bases for such links to loss of mitochondrial integrity, cytoplasmic dispersion of mitochondrial DNA, and the subsequent activation of interferon-stimulated genes (ISGs) in macrophages.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35945112"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis, inflammation, and cardiovascular disorders: a mitochondrial connection</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35945112/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220823212820&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in two antagonistic regulators of DNA methylation, DNMT3A and TET2, are associated with clonal hematopoiesis and increased risk of cardiovascular disorders. Recently, Cobo et al. traced the mechanistic bases for such links to loss of mitochondrial integrity, cytoplasmic dispersion of mitochondrial DNA, and the subsequent activation of interferon-stimulated genes (ISGs) in macrophages.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35945112"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis, inflammation, and cardiovascular disorders: a mitochondrial connection</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35945112/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220822212717&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in two antagonistic regulators of DNA methylation, DNMT3A and TET2, are associated with clonal hematopoiesis and increased risk of cardiovascular disorders. Recently, Cobo et al. traced the mechanistic bases for such links to loss of mitochondrial integrity, cytoplasmic dispersion of mitochondrial DNA, and the subsequent activation of interferon-stimulated genes (ISGs) in macrophages.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35945112"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis, inflammation, and cardiovascular disorders: a mitochondrial connection</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35945112/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220821211551&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in two antagonistic regulators of DNA methylation, DNMT3A and TET2, are associated with clonal hematopoiesis and increased risk of cardiovascular disorders. Recently, Cobo et al. traced the mechanistic bases for such links to loss of mitochondrial integrity, cytoplasmic dispersion of mitochondrial DNA, and the subsequent activation of interferon-stimulated genes (ISGs) in macrophages.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35945112"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis, inflammation, and cardiovascular disorders: a mitochondrial connection</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35945112/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220820211309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in two antagonistic regulators of DNA methylation, DNMT3A and TET2, are associated with clonal hematopoiesis and increased risk of cardiovascular disorders. Recently, Cobo et al. traced the mechanistic bases for such links to loss of mitochondrial integrity, cytoplasmic dispersion of mitochondrial DNA, and the subsequent activation of interferon-stimulated genes (ISGs) in macrophages.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35945112"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis, inflammation, and cardiovascular disorders: a mitochondrial connection</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35945112/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220819211457&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in two antagonistic regulators of DNA methylation, DNMT3A and TET2, are associated with clonal hematopoiesis and increased risk of cardiovascular disorders. Recently, Cobo et al. traced the mechanistic bases for such links to loss of mitochondrial integrity, cytoplasmic dispersion of mitochondrial DNA, and the subsequent activation of interferon-stimulated genes (ISGs) in macrophages.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35945112"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis, inflammation, and cardiovascular disorders: a mitochondrial connection</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35945112/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220818211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in two antagonistic regulators of DNA methylation, DNMT3A and TET2, are associated with clonal hematopoiesis and increased risk of cardiovascular disorders. Recently, Cobo et al. traced the mechanistic bases for such links to loss of mitochondrial integrity, cytoplasmic dispersion of mitochondrial DNA, and the subsequent activation of interferon-stimulated genes (ISGs) in macrophages.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35945112"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis, inflammation, and cardiovascular disorders: a mitochondrial connection</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35945112/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220817212606&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in two antagonistic regulators of DNA methylation, DNMT3A and TET2, are associated with clonal hematopoiesis and increased risk of cardiovascular disorders. Recently, Cobo et al. traced the mechanistic bases for such links to loss of mitochondrial integrity, cytoplasmic dispersion of mitochondrial DNA, and the subsequent activation of interferon-stimulated genes (ISGs) in macrophages.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35945112"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis, inflammation, and cardiovascular disorders: a mitochondrial connection</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35945112/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220816212355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in two antagonistic regulators of DNA methylation, DNMT3A and TET2, are associated with clonal hematopoiesis and increased risk of cardiovascular disorders. Recently, Cobo et al. traced the mechanistic bases for such links to loss of mitochondrial integrity, cytoplasmic dispersion of mitochondrial DNA, and the subsequent activation of interferon-stimulated genes (ISGs) in macrophages.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35945112"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis, inflammation, and cardiovascular disorders: a mitochondrial connection</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35945112/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220815212205&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in two antagonistic regulators of DNA methylation, DNMT3A and TET2, are associated with clonal hematopoiesis and increased risk of cardiovascular disorders. Recently, Cobo et al. traced the mechanistic bases for such links to loss of mitochondrial integrity, cytoplasmic dispersion of mitochondrial DNA, and the subsequent activation of interferon-stimulated genes (ISGs) in macrophages.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35945112"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis, inflammation, and cardiovascular disorders: a mitochondrial connection</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35945112/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220814211915&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in two antagonistic regulators of DNA methylation, DNMT3A and TET2, are associated with clonal hematopoiesis and increased risk of cardiovascular disorders. Recently, Cobo et al. traced the mechanistic bases for such links to loss of mitochondrial integrity, cytoplasmic dispersion of mitochondrial DNA, and the subsequent activation of interferon-stimulated genes (ISGs) in macrophages.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35945112"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis, inflammation, and cardiovascular disorders: a mitochondrial connection</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35945112/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220813211549&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in two antagonistic regulators of DNA methylation, DNMT3A and TET2, are associated with clonal hematopoiesis and increased risk of cardiovascular disorders. Recently, Cobo et al. traced the mechanistic bases for such links to loss of mitochondrial integrity, cytoplasmic dispersion of mitochondrial DNA, and the subsequent activation of interferon-stimulated genes (ISGs) in macrophages.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35945112"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis, inflammation, and cardiovascular disorders: a mitochondrial connection</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35945112/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220812210706&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in two antagonistic regulators of DNA methylation, DNMT3A and TET2, are associated with clonal hematopoiesis and increased risk of cardiovascular disorders. Recently, Cobo et al. traced the mechanistic bases for such links to loss of mitochondrial integrity, cytoplasmic dispersion of mitochondrial DNA, and the subsequent activation of interferon-stimulated genes (ISGs) in macrophages.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35945112"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis, inflammation, and cardiovascular disorders: a mitochondrial connection</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35945112/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220811211442&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in two antagonistic regulators of DNA methylation, DNMT3A and TET2, are associated with clonal hematopoiesis and increased risk of cardiovascular disorders. Recently, Cobo et al. traced the mechanistic bases for such links to loss of mitochondrial integrity, cytoplasmic dispersion of mitochondrial DNA, and the subsequent activation of interferon-stimulated genes (ISGs) in macrophages.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35945112"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis, inflammation, and cardiovascular disorders: a mitochondrial connection</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35945112/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220810210900&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in two antagonistic regulators of DNA methylation, DNMT3A and TET2, are associated with clonal hematopoiesis and increased risk of cardiovascular disorders. Recently, Cobo et al. traced the mechanistic bases for such links to loss of mitochondrial integrity, cytoplasmic dispersion of mitochondrial DNA, and the subsequent activation of interferon-stimulated genes (ISGs) in macrophages.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-08-08 datetime="2022-08-08T10:00:00.000Z">2022-08-08</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-08-08 datetime="2022-08-08T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608003622&amp;v&#x3D;2.17.7&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35941268"
                        open
                      >
                        <summary class="article-expander__title">Mechanisms involved in hematopoietic stem cell aging</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35941268/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220824212808&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) undergo progressive functional decline over time due to both internal and external stressors, leading to aging of the hematopoietic system. A comprehensive understanding of the molecular mechanisms underlying HSC aging will be valuable in developing novel therapies for HSC rejuvenation and to prevent the onset of several age-associated diseases and hematological malignancies. This review considers the general causes of HSC aging that range from cell-intrinsic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35940373"
                        open
                      >
                        <summary class="article-expander__title">The Impact of Clonal Hematopoiesis on Cardiovascular Disease in People Living with HIV</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35940373/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220824212808&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) with age-associated somatic mutations that disproportionally contribute to hematopoiesis generate the condition known as clonal hematopoiesis (CH). While CH conveys increased risk of hematologic cancer, there is also strong association between CH and cardiovascular disease (CVD). Accumulating evidence suggests that inflammation mechanistically links CH to CVD, and we hypothesized that CH may be a predictive biomarker of CVD in conditions of chronic inflammation....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35941268"
                        open
                      >
                        <summary class="article-expander__title">Mechanisms involved in hematopoietic stem cell aging</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35941268/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220823212820&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) undergo progressive functional decline over time due to both internal and external stressors, leading to aging of the hematopoietic system. A comprehensive understanding of the molecular mechanisms underlying HSC aging will be valuable in developing novel therapies for HSC rejuvenation and to prevent the onset of several age-associated diseases and hematological malignancies. This review considers the general causes of HSC aging that range from cell-intrinsic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35940373"
                        open
                      >
                        <summary class="article-expander__title">The Impact of Clonal Hematopoiesis on Cardiovascular Disease in People Living with HIV</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35940373/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220823212820&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) with age-associated somatic mutations that disproportionally contribute to hematopoiesis generate the condition known as clonal hematopoiesis (CH). While CH conveys increased risk of hematologic cancer, there is also strong association between CH and cardiovascular disease (CVD). Accumulating evidence suggests that inflammation mechanistically links CH to CVD, and we hypothesized that CH may be a predictive biomarker of CVD in conditions of chronic inflammation....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35941268"
                        open
                      >
                        <summary class="article-expander__title">Mechanisms involved in hematopoietic stem cell aging</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35941268/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220822212717&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) undergo progressive functional decline over time due to both internal and external stressors, leading to aging of the hematopoietic system. A comprehensive understanding of the molecular mechanisms underlying HSC aging will be valuable in developing novel therapies for HSC rejuvenation and to prevent the onset of several age-associated diseases and hematological malignancies. This review considers the general causes of HSC aging that range from cell-intrinsic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35940373"
                        open
                      >
                        <summary class="article-expander__title">The Impact of Clonal Hematopoiesis on Cardiovascular Disease in People Living with HIV</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35940373/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220822212717&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) with age-associated somatic mutations that disproportionally contribute to hematopoiesis generate the condition known as clonal hematopoiesis (CH). While CH conveys increased risk of hematologic cancer, there is also strong association between CH and cardiovascular disease (CVD). Accumulating evidence suggests that inflammation mechanistically links CH to CVD, and we hypothesized that CH may be a predictive biomarker of CVD in conditions of chronic inflammation....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35938951"
                        open
                      >
                        <summary class="article-expander__title">JAK2 V617f in chronic myeloid leukemia: driving force or passive bystander?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35938951/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220822212717&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>OBJECTIVES: BCR-ABL1 and JAK2 V617F coexistence in myeloproliferative neoplasms has been described as concomitant or sequential events. Despite this, we present a unique case of chronic myeloid leukemia (CML) not referable to either of the known scenarios.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35941268"
                        open
                      >
                        <summary class="article-expander__title">Mechanisms involved in hematopoietic stem cell aging</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35941268/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220821211551&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) undergo progressive functional decline over time due to both internal and external stressors, leading to aging of the hematopoietic system. A comprehensive understanding of the molecular mechanisms underlying HSC aging will be valuable in developing novel therapies for HSC rejuvenation and to prevent the onset of several age-associated diseases and hematological malignancies. This review considers the general causes of HSC aging that range from cell-intrinsic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35940373"
                        open
                      >
                        <summary class="article-expander__title">The Impact of Clonal Hematopoiesis on Cardiovascular Disease in People Living with HIV</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35940373/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220821211551&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) with age-associated somatic mutations that disproportionally contribute to hematopoiesis generate the condition known as clonal hematopoiesis (CH). While CH conveys increased risk of hematologic cancer, there is also strong association between CH and cardiovascular disease (CVD). Accumulating evidence suggests that inflammation mechanistically links CH to CVD, and we hypothesized that CH may be a predictive biomarker of CVD in conditions of chronic inflammation....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35938951"
                        open
                      >
                        <summary class="article-expander__title">JAK2 V617f in chronic myeloid leukemia: driving force or passive bystander?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35938951/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220821211551&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>OBJECTIVES: BCR-ABL1 and JAK2 V617F coexistence in myeloproliferative neoplasms has been described as concomitant or sequential events. Despite this, we present a unique case of chronic myeloid leukemia (CML) not referable to either of the known scenarios.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35941268"
                        open
                      >
                        <summary class="article-expander__title">Mechanisms involved in hematopoietic stem cell aging</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35941268/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220820211309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) undergo progressive functional decline over time due to both internal and external stressors, leading to aging of the hematopoietic system. A comprehensive understanding of the molecular mechanisms underlying HSC aging will be valuable in developing novel therapies for HSC rejuvenation and to prevent the onset of several age-associated diseases and hematological malignancies. This review considers the general causes of HSC aging that range from cell-intrinsic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35940373"
                        open
                      >
                        <summary class="article-expander__title">The Impact of Clonal Hematopoiesis on Cardiovascular Disease in People Living with HIV</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35940373/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220820211309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) with age-associated somatic mutations that disproportionally contribute to hematopoiesis generate the condition known as clonal hematopoiesis (CH). While CH conveys increased risk of hematologic cancer, there is also strong association between CH and cardiovascular disease (CVD). Accumulating evidence suggests that inflammation mechanistically links CH to CVD, and we hypothesized that CH may be a predictive biomarker of CVD in conditions of chronic inflammation....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35938951"
                        open
                      >
                        <summary class="article-expander__title">JAK2 V617f in chronic myeloid leukemia: driving force or passive bystander?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35938951/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220820211309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>OBJECTIVES: BCR-ABL1 and JAK2 V617F coexistence in myeloproliferative neoplasms has been described as concomitant or sequential events. Despite this, we present a unique case of chronic myeloid leukemia (CML) not referable to either of the known scenarios.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35941268"
                        open
                      >
                        <summary class="article-expander__title">Mechanisms involved in hematopoietic stem cell aging</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35941268/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220819211457&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) undergo progressive functional decline over time due to both internal and external stressors, leading to aging of the hematopoietic system. A comprehensive understanding of the molecular mechanisms underlying HSC aging will be valuable in developing novel therapies for HSC rejuvenation and to prevent the onset of several age-associated diseases and hematological malignancies. This review considers the general causes of HSC aging that range from cell-intrinsic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35940373"
                        open
                      >
                        <summary class="article-expander__title">The Impact of Clonal Hematopoiesis on Cardiovascular Disease in People Living with HIV</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35940373/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220819211457&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) with age-associated somatic mutations that disproportionally contribute to hematopoiesis generate the condition known as clonal hematopoiesis (CH). While CH conveys increased risk of hematologic cancer, there is also strong association between CH and cardiovascular disease (CVD). Accumulating evidence suggests that inflammation mechanistically links CH to CVD, and we hypothesized that CH may be a predictive biomarker of CVD in conditions of chronic inflammation....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35938951"
                        open
                      >
                        <summary class="article-expander__title">JAK2 V617f in chronic myeloid leukemia: driving force or passive bystander?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35938951/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220819211457&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>OBJECTIVES: BCR-ABL1 and JAK2 V617F coexistence in myeloproliferative neoplasms has been described as concomitant or sequential events. Despite this, we present a unique case of chronic myeloid leukemia (CML) not referable to either of the known scenarios.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35941268"
                        open
                      >
                        <summary class="article-expander__title">Mechanisms involved in hematopoietic stem cell aging</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35941268/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220818211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) undergo progressive functional decline over time due to both internal and external stressors, leading to aging of the hematopoietic system. A comprehensive understanding of the molecular mechanisms underlying HSC aging will be valuable in developing novel therapies for HSC rejuvenation and to prevent the onset of several age-associated diseases and hematological malignancies. This review considers the general causes of HSC aging that range from cell-intrinsic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35940373"
                        open
                      >
                        <summary class="article-expander__title">The Impact of Clonal Hematopoiesis on Cardiovascular Disease in People Living with HIV</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35940373/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220818211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) with age-associated somatic mutations that disproportionally contribute to hematopoiesis generate the condition known as clonal hematopoiesis (CH). While CH conveys increased risk of hematologic cancer, there is also strong association between CH and cardiovascular disease (CVD). Accumulating evidence suggests that inflammation mechanistically links CH to CVD, and we hypothesized that CH may be a predictive biomarker of CVD in conditions of chronic inflammation....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35938951"
                        open
                      >
                        <summary class="article-expander__title">JAK2 V617f in chronic myeloid leukemia: driving force or passive bystander?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35938951/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220818211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>OBJECTIVES: BCR-ABL1 and JAK2 V617F coexistence in myeloproliferative neoplasms has been described as concomitant or sequential events. Despite this, we present a unique case of chronic myeloid leukemia (CML) not referable to either of the known scenarios.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35941268"
                        open
                      >
                        <summary class="article-expander__title">Mechanisms involved in hematopoietic stem cell aging</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35941268/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220817212606&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) undergo progressive functional decline over time due to both internal and external stressors, leading to aging of the hematopoietic system. A comprehensive understanding of the molecular mechanisms underlying HSC aging will be valuable in developing novel therapies for HSC rejuvenation and to prevent the onset of several age-associated diseases and hematological malignancies. This review considers the general causes of HSC aging that range from cell-intrinsic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35940373"
                        open
                      >
                        <summary class="article-expander__title">The Impact of Clonal Hematopoiesis on Cardiovascular Disease in People Living with HIV</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35940373/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220817212606&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) with age-associated somatic mutations that disproportionally contribute to hematopoiesis generate the condition known as clonal hematopoiesis (CH). While CH conveys increased risk of hematologic cancer, there is also strong association between CH and cardiovascular disease (CVD). Accumulating evidence suggests that inflammation mechanistically links CH to CVD, and we hypothesized that CH may be a predictive biomarker of CVD in conditions of chronic inflammation....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35938951"
                        open
                      >
                        <summary class="article-expander__title">JAK2 V617f in chronic myeloid leukemia: driving force or passive bystander?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35938951/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220817212606&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>OBJECTIVES: BCR-ABL1 and JAK2 V617F coexistence in myeloproliferative neoplasms has been described as concomitant or sequential events. Despite this, we present a unique case of chronic myeloid leukemia (CML) not referable to either of the known scenarios.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35941268"
                        open
                      >
                        <summary class="article-expander__title">Mechanisms involved in hematopoietic stem cell aging</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35941268/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220816212355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) undergo progressive functional decline over time due to both internal and external stressors, leading to aging of the hematopoietic system. A comprehensive understanding of the molecular mechanisms underlying HSC aging will be valuable in developing novel therapies for HSC rejuvenation and to prevent the onset of several age-associated diseases and hematological malignancies. This review considers the general causes of HSC aging that range from cell-intrinsic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35940373"
                        open
                      >
                        <summary class="article-expander__title">The Impact of Clonal Hematopoiesis on Cardiovascular Disease in People Living with HIV</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35940373/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220816212355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) with age-associated somatic mutations that disproportionally contribute to hematopoiesis generate the condition known as clonal hematopoiesis (CH). While CH conveys increased risk of hematologic cancer, there is also strong association between CH and cardiovascular disease (CVD). Accumulating evidence suggests that inflammation mechanistically links CH to CVD, and we hypothesized that CH may be a predictive biomarker of CVD in conditions of chronic inflammation....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35938951"
                        open
                      >
                        <summary class="article-expander__title">JAK2 V617f in chronic myeloid leukemia: driving force or passive bystander?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35938951/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220816212355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>OBJECTIVES: BCR-ABL1 and JAK2 V617F coexistence in myeloproliferative neoplasms has been described as concomitant or sequential events. Despite this, we present a unique case of chronic myeloid leukemia (CML) not referable to either of the known scenarios.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35941268"
                        open
                      >
                        <summary class="article-expander__title">Mechanisms involved in hematopoietic stem cell aging</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35941268/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220815212205&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) undergo progressive functional decline over time due to both internal and external stressors, leading to aging of the hematopoietic system. A comprehensive understanding of the molecular mechanisms underlying HSC aging will be valuable in developing novel therapies for HSC rejuvenation and to prevent the onset of several age-associated diseases and hematological malignancies. This review considers the general causes of HSC aging that range from cell-intrinsic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35940373"
                        open
                      >
                        <summary class="article-expander__title">The Impact of Clonal Hematopoiesis on Cardiovascular Disease in People Living with HIV</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35940373/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220815212205&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) with age-associated somatic mutations that disproportionally contribute to hematopoiesis generate the condition known as clonal hematopoiesis (CH). While CH conveys increased risk of hematologic cancer, there is also strong association between CH and cardiovascular disease (CVD). Accumulating evidence suggests that inflammation mechanistically links CH to CVD, and we hypothesized that CH may be a predictive biomarker of CVD in conditions of chronic inflammation....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35938951"
                        open
                      >
                        <summary class="article-expander__title">JAK2 V617f in chronic myeloid leukemia: driving force or passive bystander?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35938951/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220815212205&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>OBJECTIVES: BCR-ABL1 and JAK2 V617F coexistence in myeloproliferative neoplasms has been described as concomitant or sequential events. Despite this, we present a unique case of chronic myeloid leukemia (CML) not referable to either of the known scenarios.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35941268"
                        open
                      >
                        <summary class="article-expander__title">Mechanisms involved in hematopoietic stem cell aging</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35941268/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220814211915&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) undergo progressive functional decline over time due to both internal and external stressors, leading to aging of the hematopoietic system. A comprehensive understanding of the molecular mechanisms underlying HSC aging will be valuable in developing novel therapies for HSC rejuvenation and to prevent the onset of several age-associated diseases and hematological malignancies. This review considers the general causes of HSC aging that range from cell-intrinsic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35940373"
                        open
                      >
                        <summary class="article-expander__title">The Impact of Clonal Hematopoiesis on Cardiovascular Disease in People Living with HIV</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35940373/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220814211915&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) with age-associated somatic mutations that disproportionally contribute to hematopoiesis generate the condition known as clonal hematopoiesis (CH). While CH conveys increased risk of hematologic cancer, there is also strong association between CH and cardiovascular disease (CVD). Accumulating evidence suggests that inflammation mechanistically links CH to CVD, and we hypothesized that CH may be a predictive biomarker of CVD in conditions of chronic inflammation....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35938951"
                        open
                      >
                        <summary class="article-expander__title">JAK2 V617f in chronic myeloid leukemia: driving force or passive bystander?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35938951/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220814211915&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>OBJECTIVES: BCR-ABL1 and JAK2 V617F coexistence in myeloproliferative neoplasms has been described as concomitant or sequential events. Despite this, we present a unique case of chronic myeloid leukemia (CML) not referable to either of the known scenarios.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35941268"
                        open
                      >
                        <summary class="article-expander__title">Mechanisms involved in hematopoietic stem cell aging</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35941268/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220813211549&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) undergo progressive functional decline over time due to both internal and external stressors, leading to aging of the hematopoietic system. A comprehensive understanding of the molecular mechanisms underlying HSC aging will be valuable in developing novel therapies for HSC rejuvenation and to prevent the onset of several age-associated diseases and hematological malignancies. This review considers the general causes of HSC aging that range from cell-intrinsic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35940373"
                        open
                      >
                        <summary class="article-expander__title">The Impact of Clonal Hematopoiesis on Cardiovascular Disease in People Living with HIV</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35940373/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220813211549&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) with age-associated somatic mutations that disproportionally contribute to hematopoiesis generate the condition known as clonal hematopoiesis (CH). While CH conveys increased risk of hematologic cancer, there is also strong association between CH and cardiovascular disease (CVD). Accumulating evidence suggests that inflammation mechanistically links CH to CVD, and we hypothesized that CH may be a predictive biomarker of CVD in conditions of chronic inflammation....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35938951"
                        open
                      >
                        <summary class="article-expander__title">JAK2 V617f in chronic myeloid leukemia: driving force or passive bystander?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35938951/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220813211549&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>OBJECTIVES: BCR-ABL1 and JAK2 V617F coexistence in myeloproliferative neoplasms has been described as concomitant or sequential events. Despite this, we present a unique case of chronic myeloid leukemia (CML) not referable to either of the known scenarios.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35941268"
                        open
                      >
                        <summary class="article-expander__title">Mechanisms involved in hematopoietic stem cell aging</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35941268/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220812210706&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) undergo progressive functional decline over time due to both internal and external stressors, leading to aging of the hematopoietic system. A comprehensive understanding of the molecular mechanisms underlying HSC aging will be valuable in developing novel therapies for HSC rejuvenation and to prevent the onset of several age-associated diseases and hematological malignancies. This review considers the general causes of HSC aging that range from cell-intrinsic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35940373"
                        open
                      >
                        <summary class="article-expander__title">The Impact of Clonal Hematopoiesis on Cardiovascular Disease in People Living with HIV</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35940373/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220812210706&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) with age-associated somatic mutations that disproportionally contribute to hematopoiesis generate the condition known as clonal hematopoiesis (CH). While CH conveys increased risk of hematologic cancer, there is also strong association between CH and cardiovascular disease (CVD). Accumulating evidence suggests that inflammation mechanistically links CH to CVD, and we hypothesized that CH may be a predictive biomarker of CVD in conditions of chronic inflammation....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35938951"
                        open
                      >
                        <summary class="article-expander__title">JAK2 V617f in chronic myeloid leukemia: driving force or passive bystander?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35938951/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220812210706&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>OBJECTIVES: BCR-ABL1 and JAK2 V617F coexistence in myeloproliferative neoplasms has been described as concomitant or sequential events. Despite this, we present a unique case of chronic myeloid leukemia (CML) not referable to either of the known scenarios.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35941268"
                        open
                      >
                        <summary class="article-expander__title">Mechanisms involved in hematopoietic stem cell aging</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35941268/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220811211442&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) undergo progressive functional decline over time due to both internal and external stressors, leading to aging of the hematopoietic system. A comprehensive understanding of the molecular mechanisms underlying HSC aging will be valuable in developing novel therapies for HSC rejuvenation and to prevent the onset of several age-associated diseases and hematological malignancies. This review considers the general causes of HSC aging that range from cell-intrinsic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35940373"
                        open
                      >
                        <summary class="article-expander__title">The Impact of Clonal Hematopoiesis on Cardiovascular Disease in People Living with HIV</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35940373/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220811211442&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) with age-associated somatic mutations that disproportionally contribute to hematopoiesis generate the condition known as clonal hematopoiesis (CH). While CH conveys increased risk of hematologic cancer, there is also strong association between CH and cardiovascular disease (CVD). Accumulating evidence suggests that inflammation mechanistically links CH to CVD, and we hypothesized that CH may be a predictive biomarker of CVD in conditions of chronic inflammation....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35938951"
                        open
                      >
                        <summary class="article-expander__title">JAK2 V617f in chronic myeloid leukemia: driving force or passive bystander?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35938951/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220811211442&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>OBJECTIVES: BCR-ABL1 and JAK2 V617F coexistence in myeloproliferative neoplasms has been described as concomitant or sequential events. Despite this, we present a unique case of chronic myeloid leukemia (CML) not referable to either of the known scenarios.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35941268"
                        open
                      >
                        <summary class="article-expander__title">Mechanisms involved in hematopoietic stem cell aging</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35941268/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220810210900&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) undergo progressive functional decline over time due to both internal and external stressors, leading to aging of the hematopoietic system. A comprehensive understanding of the molecular mechanisms underlying HSC aging will be valuable in developing novel therapies for HSC rejuvenation and to prevent the onset of several age-associated diseases and hematological malignancies. This review considers the general causes of HSC aging that range from cell-intrinsic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35940373"
                        open
                      >
                        <summary class="article-expander__title">The Impact of Clonal Hematopoiesis on Cardiovascular Disease in People Living with HIV</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35940373/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220810210900&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) with age-associated somatic mutations that disproportionally contribute to hematopoiesis generate the condition known as clonal hematopoiesis (CH). While CH conveys increased risk of hematologic cancer, there is also strong association between CH and cardiovascular disease (CVD). Accumulating evidence suggests that inflammation mechanistically links CH to CVD, and we hypothesized that CH may be a predictive biomarker of CVD in conditions of chronic inflammation....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35938951"
                        open
                      >
                        <summary class="article-expander__title">JAK2 V617f in chronic myeloid leukemia: driving force or passive bystander?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35938951/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220810210900&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>OBJECTIVES: BCR-ABL1 and JAK2 V617F coexistence in myeloproliferative neoplasms has been described as concomitant or sequential events. Despite this, we present a unique case of chronic myeloid leukemia (CML) not referable to either of the known scenarios.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35941268"
                        open
                      >
                        <summary class="article-expander__title">Mechanisms involved in hematopoietic stem cell aging</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35941268/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220809211355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) undergo progressive functional decline over time due to both internal and external stressors, leading to aging of the hematopoietic system. A comprehensive understanding of the molecular mechanisms underlying HSC aging will be valuable in developing novel therapies for HSC rejuvenation and to prevent the onset of several age-associated diseases and hematological malignancies. This review considers the general causes of HSC aging that range from cell-intrinsic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35940373"
                        open
                      >
                        <summary class="article-expander__title">The Impact of Clonal Hematopoiesis on Cardiovascular Disease in People Living with HIV</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35940373/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220809211355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) with age-associated somatic mutations that disproportionally contribute to hematopoiesis generate the condition known as clonal hematopoiesis (CH). While CH conveys increased risk of hematologic cancer, there is also strong association between CH and cardiovascular disease (CVD). Accumulating evidence suggests that inflammation mechanistically links CH to CVD, and we hypothesized that CH may be a predictive biomarker of CVD in conditions of chronic inflammation....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35938951"
                        open
                      >
                        <summary class="article-expander__title">JAK2 V617f in chronic myeloid leukemia: driving force or passive bystander?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35938951/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220809211355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>OBJECTIVES: BCR-ABL1 and JAK2 V617F coexistence in myeloproliferative neoplasms has been described as concomitant or sequential events. Despite this, we present a unique case of chronic myeloid leukemia (CML) not referable to either of the known scenarios.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35938951"
                        open
                      >
                        <summary class="article-expander__title">JAK2 V617f in chronic myeloid leukemia: driving force or passive bystander?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35938951/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220808212152&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>OBJECTIVES: BCR-ABL1 and JAK2 V617F coexistence in myeloproliferative neoplasms has been described as concomitant or sequential events. Despite this, we present a unique case of chronic myeloid leukemia (CML) not referable to either of the known scenarios.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608010801&amp;v&#x3D;2.17.7&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35941190"
                        open
                      >
                        <summary class="article-expander__title">Fragment size and dynamics of EGFR-mutated tumor-derived DNA provide prognostic information regarding EGFR-TKI efficacy in patients with EGFR-mutated NSCLC</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35941190/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220824212808&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA)-based next-generation sequencing (NGS) is a complementary and alternative test to tissue-based NGS. We performed NGS analysis of ctDNA samples collected from patients with EGFR-mutated non-small cell lung cancer (NSCLC) who received osimertinib; the samples were collected after second-line treatment, before osimertinib treatment, one week and one month after osimertinib treatment, and at the time of resistance formation. We examinedthe correlation with osimertinib...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35941190"
                        open
                      >
                        <summary class="article-expander__title">Fragment size and dynamics of EGFR-mutated tumor-derived DNA provide prognostic information regarding EGFR-TKI efficacy in patients with EGFR-mutated NSCLC</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35941190/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220823212820&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA)-based next-generation sequencing (NGS) is a complementary and alternative test to tissue-based NGS. We performed NGS analysis of ctDNA samples collected from patients with EGFR-mutated non-small cell lung cancer (NSCLC) who received osimertinib; the samples were collected after second-line treatment, before osimertinib treatment, one week and one month after osimertinib treatment, and at the time of resistance formation. We examinedthe correlation with osimertinib...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35941190"
                        open
                      >
                        <summary class="article-expander__title">Fragment size and dynamics of EGFR-mutated tumor-derived DNA provide prognostic information regarding EGFR-TKI efficacy in patients with EGFR-mutated NSCLC</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35941190/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220822212717&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA)-based next-generation sequencing (NGS) is a complementary and alternative test to tissue-based NGS. We performed NGS analysis of ctDNA samples collected from patients with EGFR-mutated non-small cell lung cancer (NSCLC) who received osimertinib; the samples were collected after second-line treatment, before osimertinib treatment, one week and one month after osimertinib treatment, and at the time of resistance formation. We examinedthe correlation with osimertinib...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35941190"
                        open
                      >
                        <summary class="article-expander__title">Fragment size and dynamics of EGFR-mutated tumor-derived DNA provide prognostic information regarding EGFR-TKI efficacy in patients with EGFR-mutated NSCLC</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35941190/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220821211551&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA)-based next-generation sequencing (NGS) is a complementary and alternative test to tissue-based NGS. We performed NGS analysis of ctDNA samples collected from patients with EGFR-mutated non-small cell lung cancer (NSCLC) who received osimertinib; the samples were collected after second-line treatment, before osimertinib treatment, one week and one month after osimertinib treatment, and at the time of resistance formation. We examinedthe correlation with osimertinib...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35941190"
                        open
                      >
                        <summary class="article-expander__title">Fragment size and dynamics of EGFR-mutated tumor-derived DNA provide prognostic information regarding EGFR-TKI efficacy in patients with EGFR-mutated NSCLC</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35941190/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220820211309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA)-based next-generation sequencing (NGS) is a complementary and alternative test to tissue-based NGS. We performed NGS analysis of ctDNA samples collected from patients with EGFR-mutated non-small cell lung cancer (NSCLC) who received osimertinib; the samples were collected after second-line treatment, before osimertinib treatment, one week and one month after osimertinib treatment, and at the time of resistance formation. We examinedthe correlation with osimertinib...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35941190"
                        open
                      >
                        <summary class="article-expander__title">Fragment size and dynamics of EGFR-mutated tumor-derived DNA provide prognostic information regarding EGFR-TKI efficacy in patients with EGFR-mutated NSCLC</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35941190/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220819211458&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA)-based next-generation sequencing (NGS) is a complementary and alternative test to tissue-based NGS. We performed NGS analysis of ctDNA samples collected from patients with EGFR-mutated non-small cell lung cancer (NSCLC) who received osimertinib; the samples were collected after second-line treatment, before osimertinib treatment, one week and one month after osimertinib treatment, and at the time of resistance formation. We examinedthe correlation with osimertinib...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35941190"
                        open
                      >
                        <summary class="article-expander__title">Fragment size and dynamics of EGFR-mutated tumor-derived DNA provide prognostic information regarding EGFR-TKI efficacy in patients with EGFR-mutated NSCLC</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35941190/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220818211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA)-based next-generation sequencing (NGS) is a complementary and alternative test to tissue-based NGS. We performed NGS analysis of ctDNA samples collected from patients with EGFR-mutated non-small cell lung cancer (NSCLC) who received osimertinib; the samples were collected after second-line treatment, before osimertinib treatment, one week and one month after osimertinib treatment, and at the time of resistance formation. We examinedthe correlation with osimertinib...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35941190"
                        open
                      >
                        <summary class="article-expander__title">Fragment size and dynamics of EGFR-mutated tumor-derived DNA provide prognostic information regarding EGFR-TKI efficacy in patients with EGFR-mutated NSCLC</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35941190/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220817212606&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA)-based next-generation sequencing (NGS) is a complementary and alternative test to tissue-based NGS. We performed NGS analysis of ctDNA samples collected from patients with EGFR-mutated non-small cell lung cancer (NSCLC) who received osimertinib; the samples were collected after second-line treatment, before osimertinib treatment, one week and one month after osimertinib treatment, and at the time of resistance formation. We examinedthe correlation with osimertinib...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35941190"
                        open
                      >
                        <summary class="article-expander__title">Fragment size and dynamics of EGFR-mutated tumor-derived DNA provide prognostic information regarding EGFR-TKI efficacy in patients with EGFR-mutated NSCLC</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35941190/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220816212355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA)-based next-generation sequencing (NGS) is a complementary and alternative test to tissue-based NGS. We performed NGS analysis of ctDNA samples collected from patients with EGFR-mutated non-small cell lung cancer (NSCLC) who received osimertinib; the samples were collected after second-line treatment, before osimertinib treatment, one week and one month after osimertinib treatment, and at the time of resistance formation. We examinedthe correlation with osimertinib...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35941190"
                        open
                      >
                        <summary class="article-expander__title">Fragment size and dynamics of EGFR-mutated tumor-derived DNA provide prognostic information regarding EGFR-TKI efficacy in patients with EGFR-mutated NSCLC</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35941190/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220815212205&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA)-based next-generation sequencing (NGS) is a complementary and alternative test to tissue-based NGS. We performed NGS analysis of ctDNA samples collected from patients with EGFR-mutated non-small cell lung cancer (NSCLC) who received osimertinib; the samples were collected after second-line treatment, before osimertinib treatment, one week and one month after osimertinib treatment, and at the time of resistance formation. We examinedthe correlation with osimertinib...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35941190"
                        open
                      >
                        <summary class="article-expander__title">Fragment size and dynamics of EGFR-mutated tumor-derived DNA provide prognostic information regarding EGFR-TKI efficacy in patients with EGFR-mutated NSCLC</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35941190/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220814211915&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA)-based next-generation sequencing (NGS) is a complementary and alternative test to tissue-based NGS. We performed NGS analysis of ctDNA samples collected from patients with EGFR-mutated non-small cell lung cancer (NSCLC) who received osimertinib; the samples were collected after second-line treatment, before osimertinib treatment, one week and one month after osimertinib treatment, and at the time of resistance formation. We examinedthe correlation with osimertinib...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35941190"
                        open
                      >
                        <summary class="article-expander__title">Fragment size and dynamics of EGFR-mutated tumor-derived DNA provide prognostic information regarding EGFR-TKI efficacy in patients with EGFR-mutated NSCLC</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35941190/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220813211549&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA)-based next-generation sequencing (NGS) is a complementary and alternative test to tissue-based NGS. We performed NGS analysis of ctDNA samples collected from patients with EGFR-mutated non-small cell lung cancer (NSCLC) who received osimertinib; the samples were collected after second-line treatment, before osimertinib treatment, one week and one month after osimertinib treatment, and at the time of resistance formation. We examinedthe correlation with osimertinib...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35941190"
                        open
                      >
                        <summary class="article-expander__title">Fragment size and dynamics of EGFR-mutated tumor-derived DNA provide prognostic information regarding EGFR-TKI efficacy in patients with EGFR-mutated NSCLC</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35941190/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220812210706&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA)-based next-generation sequencing (NGS) is a complementary and alternative test to tissue-based NGS. We performed NGS analysis of ctDNA samples collected from patients with EGFR-mutated non-small cell lung cancer (NSCLC) who received osimertinib; the samples were collected after second-line treatment, before osimertinib treatment, one week and one month after osimertinib treatment, and at the time of resistance formation. We examinedthe correlation with osimertinib...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35941190"
                        open
                      >
                        <summary class="article-expander__title">Fragment size and dynamics of EGFR-mutated tumor-derived DNA provide prognostic information regarding EGFR-TKI efficacy in patients with EGFR-mutated NSCLC</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35941190/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220811211442&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA)-based next-generation sequencing (NGS) is a complementary and alternative test to tissue-based NGS. We performed NGS analysis of ctDNA samples collected from patients with EGFR-mutated non-small cell lung cancer (NSCLC) who received osimertinib; the samples were collected after second-line treatment, before osimertinib treatment, one week and one month after osimertinib treatment, and at the time of resistance formation. We examinedthe correlation with osimertinib...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35941190"
                        open
                      >
                        <summary class="article-expander__title">Fragment size and dynamics of EGFR-mutated tumor-derived DNA provide prognostic information regarding EGFR-TKI efficacy in patients with EGFR-mutated NSCLC</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35941190/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220810210900&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA)-based next-generation sequencing (NGS) is a complementary and alternative test to tissue-based NGS. We performed NGS analysis of ctDNA samples collected from patients with EGFR-mutated non-small cell lung cancer (NSCLC) who received osimertinib; the samples were collected after second-line treatment, before osimertinib treatment, one week and one month after osimertinib treatment, and at the time of resistance formation. We examinedthe correlation with osimertinib...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35941190"
                        open
                      >
                        <summary class="article-expander__title">Fragment size and dynamics of EGFR-mutated tumor-derived DNA provide prognostic information regarding EGFR-TKI efficacy in patients with EGFR-mutated NSCLC</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35941190/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220809211355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA)-based next-generation sequencing (NGS) is a complementary and alternative test to tissue-based NGS. We performed NGS analysis of ctDNA samples collected from patients with EGFR-mutated non-small cell lung cancer (NSCLC) who received osimertinib; the samples were collected after second-line treatment, before osimertinib treatment, one week and one month after osimertinib treatment, and at the time of resistance formation. We examinedthe correlation with osimertinib...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-08-05 datetime="2022-08-05T10:00:00.000Z">2022-08-05</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-08-05 datetime="2022-08-05T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608003622&amp;v&#x3D;2.17.7&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35931086"
                        open
                      >
                        <summary class="article-expander__title">DNA methyltransferase 3 alpha and TET methylcytosine dioxygenase 2 restrain mitochondrial DNA-mediated interferon signaling in macrophages</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35931086/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220821211551&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Deleterious somatic mutations in DNA methyltransferase 3 alpha (DNMT3A) and TET mehtylcytosine dioxygenase 2 (TET2) are associated with clonal expansion of hematopoietic cells and higher risk of cardiovascular disease (CVD). Here, we investigated roles of DNMT3A and TET2 in normal human monocyte-derived macrophages (MDM), in MDM isolated from individuals with DNMT3A or TET2 mutations, and in macrophages isolated from human atherosclerotic plaques. We found that loss of function of DNMT3A or TET2...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35931086"
                        open
                      >
                        <summary class="article-expander__title">DNA methyltransferase 3 alpha and TET methylcytosine dioxygenase 2 restrain mitochondrial DNA-mediated interferon signaling in macrophages</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35931086/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220820211309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Deleterious somatic mutations in DNA methyltransferase 3 alpha (DNMT3A) and TET mehtylcytosine dioxygenase 2 (TET2) are associated with clonal expansion of hematopoietic cells and higher risk of cardiovascular disease (CVD). Here, we investigated roles of DNMT3A and TET2 in normal human monocyte-derived macrophages (MDM), in MDM isolated from individuals with DNMT3A or TET2 mutations, and in macrophages isolated from human atherosclerotic plaques. We found that loss of function of DNMT3A or TET2...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35931086"
                        open
                      >
                        <summary class="article-expander__title">DNA methyltransferase 3 alpha and TET methylcytosine dioxygenase 2 restrain mitochondrial DNA-mediated interferon signaling in macrophages</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35931086/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220819211457&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Deleterious somatic mutations in DNA methyltransferase 3 alpha (DNMT3A) and TET mehtylcytosine dioxygenase 2 (TET2) are associated with clonal expansion of hematopoietic cells and higher risk of cardiovascular disease (CVD). Here, we investigated roles of DNMT3A and TET2 in normal human monocyte-derived macrophages (MDM), in MDM isolated from individuals with DNMT3A or TET2 mutations, and in macrophages isolated from human atherosclerotic plaques. We found that loss of function of DNMT3A or TET2...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35931086"
                        open
                      >
                        <summary class="article-expander__title">DNA methyltransferase 3 alpha and TET methylcytosine dioxygenase 2 restrain mitochondrial DNA-mediated interferon signaling in macrophages</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35931086/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220818211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Deleterious somatic mutations in DNA methyltransferase 3 alpha (DNMT3A) and TET mehtylcytosine dioxygenase 2 (TET2) are associated with clonal expansion of hematopoietic cells and higher risk of cardiovascular disease (CVD). Here, we investigated roles of DNMT3A and TET2 in normal human monocyte-derived macrophages (MDM), in MDM isolated from individuals with DNMT3A or TET2 mutations, and in macrophages isolated from human atherosclerotic plaques. We found that loss of function of DNMT3A or TET2...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35931086"
                        open
                      >
                        <summary class="article-expander__title">DNA methyltransferase 3 alpha and TET methylcytosine dioxygenase 2 restrain mitochondrial DNA-mediated interferon signaling in macrophages</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35931086/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220817212606&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Deleterious somatic mutations in DNA methyltransferase 3 alpha (DNMT3A) and TET mehtylcytosine dioxygenase 2 (TET2) are associated with clonal expansion of hematopoietic cells and higher risk of cardiovascular disease (CVD). Here, we investigated roles of DNMT3A and TET2 in normal human monocyte-derived macrophages (MDM), in MDM isolated from individuals with DNMT3A or TET2 mutations, and in macrophages isolated from human atherosclerotic plaques. We found that loss of function of DNMT3A or TET2...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35931086"
                        open
                      >
                        <summary class="article-expander__title">DNA methyltransferase 3 alpha and TET methylcytosine dioxygenase 2 restrain mitochondrial DNA-mediated interferon signaling in macrophages</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35931086/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220816212355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Deleterious somatic mutations in DNA methyltransferase 3 alpha (DNMT3A) and TET mehtylcytosine dioxygenase 2 (TET2) are associated with clonal expansion of hematopoietic cells and higher risk of cardiovascular disease (CVD). Here, we investigated roles of DNMT3A and TET2 in normal human monocyte-derived macrophages (MDM), in MDM isolated from individuals with DNMT3A or TET2 mutations, and in macrophages isolated from human atherosclerotic plaques. We found that loss of function of DNMT3A or TET2...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35931086"
                        open
                      >
                        <summary class="article-expander__title">DNA methyltransferase 3 alpha and TET methylcytosine dioxygenase 2 restrain mitochondrial DNA-mediated interferon signaling in macrophages</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35931086/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220815212205&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Deleterious somatic mutations in DNA methyltransferase 3 alpha (DNMT3A) and TET mehtylcytosine dioxygenase 2 (TET2) are associated with clonal expansion of hematopoietic cells and higher risk of cardiovascular disease (CVD). Here, we investigated roles of DNMT3A and TET2 in normal human monocyte-derived macrophages (MDM), in MDM isolated from individuals with DNMT3A or TET2 mutations, and in macrophages isolated from human atherosclerotic plaques. We found that loss of function of DNMT3A or TET2...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35931086"
                        open
                      >
                        <summary class="article-expander__title">DNA methyltransferase 3 alpha and TET methylcytosine dioxygenase 2 restrain mitochondrial DNA-mediated interferon signaling in macrophages</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35931086/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220814211915&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Deleterious somatic mutations in DNA methyltransferase 3 alpha (DNMT3A) and TET mehtylcytosine dioxygenase 2 (TET2) are associated with clonal expansion of hematopoietic cells and higher risk of cardiovascular disease (CVD). Here, we investigated roles of DNMT3A and TET2 in normal human monocyte-derived macrophages (MDM), in MDM isolated from individuals with DNMT3A or TET2 mutations, and in macrophages isolated from human atherosclerotic plaques. We found that loss of function of DNMT3A or TET2...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35931086"
                        open
                      >
                        <summary class="article-expander__title">DNA methyltransferase 3 alpha and TET methylcytosine dioxygenase 2 restrain mitochondrial DNA-mediated interferon signaling in macrophages</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35931086/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220813211549&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Deleterious somatic mutations in DNA methyltransferase 3 alpha (DNMT3A) and TET mehtylcytosine dioxygenase 2 (TET2) are associated with clonal expansion of hematopoietic cells and higher risk of cardiovascular disease (CVD). Here, we investigated roles of DNMT3A and TET2 in normal human monocyte-derived macrophages (MDM), in MDM isolated from individuals with DNMT3A or TET2 mutations, and in macrophages isolated from human atherosclerotic plaques. We found that loss of function of DNMT3A or TET2...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35931086"
                        open
                      >
                        <summary class="article-expander__title">DNA methyltransferase 3 alpha and TET methylcytosine dioxygenase 2 restrain mitochondrial DNA-mediated interferon signaling in macrophages</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35931086/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220812210706&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Deleterious somatic mutations in DNA methyltransferase 3 alpha (DNMT3A) and TET mehtylcytosine dioxygenase 2 (TET2) are associated with clonal expansion of hematopoietic cells and higher risk of cardiovascular disease (CVD). Here, we investigated roles of DNMT3A and TET2 in normal human monocyte-derived macrophages (MDM), in MDM isolated from individuals with DNMT3A or TET2 mutations, and in macrophages isolated from human atherosclerotic plaques. We found that loss of function of DNMT3A or TET2...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35931086"
                        open
                      >
                        <summary class="article-expander__title">DNA methyltransferase 3 alpha and TET methylcytosine dioxygenase 2 restrain mitochondrial DNA-mediated interferon signaling in macrophages</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35931086/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220811211442&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Deleterious somatic mutations in DNA methyltransferase 3 alpha (DNMT3A) and TET mehtylcytosine dioxygenase 2 (TET2) are associated with clonal expansion of hematopoietic cells and higher risk of cardiovascular disease (CVD). Here, we investigated roles of DNMT3A and TET2 in normal human monocyte-derived macrophages (MDM), in MDM isolated from individuals with DNMT3A or TET2 mutations, and in macrophages isolated from human atherosclerotic plaques. We found that loss of function of DNMT3A or TET2...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35931086"
                        open
                      >
                        <summary class="article-expander__title">DNA methyltransferase 3 alpha and TET methylcytosine dioxygenase 2 restrain mitochondrial DNA-mediated interferon signaling in macrophages</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35931086/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220810210900&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Deleterious somatic mutations in DNA methyltransferase 3 alpha (DNMT3A) and TET mehtylcytosine dioxygenase 2 (TET2) are associated with clonal expansion of hematopoietic cells and higher risk of cardiovascular disease (CVD). Here, we investigated roles of DNMT3A and TET2 in normal human monocyte-derived macrophages (MDM), in MDM isolated from individuals with DNMT3A or TET2 mutations, and in macrophages isolated from human atherosclerotic plaques. We found that loss of function of DNMT3A or TET2...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35931086"
                        open
                      >
                        <summary class="article-expander__title">DNA methyltransferase 3 alpha and TET methylcytosine dioxygenase 2 restrain mitochondrial DNA-mediated interferon signaling in macrophages</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35931086/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220809211355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Deleterious somatic mutations in DNA methyltransferase 3 alpha (DNMT3A) and TET mehtylcytosine dioxygenase 2 (TET2) are associated with clonal expansion of hematopoietic cells and higher risk of cardiovascular disease (CVD). Here, we investigated roles of DNMT3A and TET2 in normal human monocyte-derived macrophages (MDM), in MDM isolated from individuals with DNMT3A or TET2 mutations, and in macrophages isolated from human atherosclerotic plaques. We found that loss of function of DNMT3A or TET2...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35931086"
                        open
                      >
                        <summary class="article-expander__title">DNA methyltransferase 3 alpha and TET methylcytosine dioxygenase 2 restrain mitochondrial DNA-mediated interferon signaling in macrophages</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35931086/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220808212152&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Deleterious somatic mutations in DNA methyltransferase 3 alpha (DNMT3A) and TET mehtylcytosine dioxygenase 2 (TET2) are associated with clonal expansion of hematopoietic cells and higher risk of cardiovascular disease (CVD). Here, we investigated roles of DNMT3A and TET2 in normal human monocyte-derived macrophages (MDM), in MDM isolated from individuals with DNMT3A or TET2 mutations, and in macrophages isolated from human atherosclerotic plaques. We found that loss of function of DNMT3A or TET2...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35931086"
                        open
                      >
                        <summary class="article-expander__title">DNA methyltransferase 3 alpha and TET methylcytosine dioxygenase 2 restrain mitochondrial DNA-mediated interferon signaling in macrophages</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35931086/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220807211336&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Deleterious somatic mutations in DNA methyltransferase 3 alpha (DNMT3A) and TET mehtylcytosine dioxygenase 2 (TET2) are associated with clonal expansion of hematopoietic cells and higher risk of cardiovascular disease (CVD). Here, we investigated roles of DNMT3A and TET2 in normal human monocyte-derived macrophages (MDM), in MDM isolated from individuals with DNMT3A or TET2 mutations, and in macrophages isolated from human atherosclerotic plaques. We found that loss of function of DNMT3A or TET2...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35931086"
                        open
                      >
                        <summary class="article-expander__title">DNA methyltransferase 3 alpha and TET methylcytosine dioxygenase 2 restrain mitochondrial DNA-mediated interferon signaling in macrophages</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35931086/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220806211830&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Deleterious somatic mutations in DNA methyltransferase 3 alpha (DNMT3A) and TET mehtylcytosine dioxygenase 2 (TET2) are associated with clonal expansion of hematopoietic cells and higher risk of cardiovascular disease (CVD). Here, we investigated roles of DNMT3A and TET2 in normal human monocyte-derived macrophages (MDM), in MDM isolated from individuals with DNMT3A or TET2 mutations, and in macrophages isolated from human atherosclerotic plaques. We found that loss of function of DNMT3A or TET2...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608010801&amp;v&#x3D;2.17.7&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35931318"
                        open
                      >
                        <summary class="article-expander__title">Cancer: Slaying the 9-headed Hydra</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35931318/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220824212808&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Modern medicine continues to evolve, and the treatment armamentarium for various diseases grows more individualized across a breadth of medical disciplines. Cure rates for infectious diseases that were previously pan-fatal approach 100% because of the identification of the specific pathogen(s) involved and the use of appropriate combinations of drugs, where needed, to completely extinguish infection and hence prevent emergence of resistant strains. Similarly, with the assistance of technologies...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35931318"
                        open
                      >
                        <summary class="article-expander__title">Cancer: Slaying the 9-headed Hydra</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35931318/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220823212820&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Modern medicine continues to evolve, and the treatment armamentarium for various diseases grows more individualized across a breadth of medical disciplines. Cure rates for infectious diseases that were previously pan-fatal approach 100% because of the identification of the specific pathogen(s) involved and the use of appropriate combinations of drugs, where needed, to completely extinguish infection and hence prevent emergence of resistant strains. Similarly, with the assistance of technologies...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35931318"
                        open
                      >
                        <summary class="article-expander__title">Cancer: Slaying the 9-headed Hydra</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35931318/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220822212717&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Modern medicine continues to evolve, and the treatment armamentarium for various diseases grows more individualized across a breadth of medical disciplines. Cure rates for infectious diseases that were previously pan-fatal approach 100% because of the identification of the specific pathogen(s) involved and the use of appropriate combinations of drugs, where needed, to completely extinguish infection and hence prevent emergence of resistant strains. Similarly, with the assistance of technologies...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35931318"
                        open
                      >
                        <summary class="article-expander__title">Cancer: Slaying the 9-headed Hydra</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35931318/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220821211551&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Modern medicine continues to evolve, and the treatment armamentarium for various diseases grows more individualized across a breadth of medical disciplines. Cure rates for infectious diseases that were previously pan-fatal approach 100% because of the identification of the specific pathogen(s) involved and the use of appropriate combinations of drugs, where needed, to completely extinguish infection and hence prevent emergence of resistant strains. Similarly, with the assistance of technologies...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35931318"
                        open
                      >
                        <summary class="article-expander__title">Cancer: Slaying the 9-headed Hydra</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35931318/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220820211309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Modern medicine continues to evolve, and the treatment armamentarium for various diseases grows more individualized across a breadth of medical disciplines. Cure rates for infectious diseases that were previously pan-fatal approach 100% because of the identification of the specific pathogen(s) involved and the use of appropriate combinations of drugs, where needed, to completely extinguish infection and hence prevent emergence of resistant strains. Similarly, with the assistance of technologies...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35931318"
                        open
                      >
                        <summary class="article-expander__title">Cancer: Slaying the 9-headed Hydra</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35931318/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220819211458&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Modern medicine continues to evolve, and the treatment armamentarium for various diseases grows more individualized across a breadth of medical disciplines. Cure rates for infectious diseases that were previously pan-fatal approach 100% because of the identification of the specific pathogen(s) involved and the use of appropriate combinations of drugs, where needed, to completely extinguish infection and hence prevent emergence of resistant strains. Similarly, with the assistance of technologies...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35931318"
                        open
                      >
                        <summary class="article-expander__title">Cancer: Slaying the 9-headed Hydra</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35931318/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220818211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Modern medicine continues to evolve, and the treatment armamentarium for various diseases grows more individualized across a breadth of medical disciplines. Cure rates for infectious diseases that were previously pan-fatal approach 100% because of the identification of the specific pathogen(s) involved and the use of appropriate combinations of drugs, where needed, to completely extinguish infection and hence prevent emergence of resistant strains. Similarly, with the assistance of technologies...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35931318"
                        open
                      >
                        <summary class="article-expander__title">Cancer: Slaying the 9-headed Hydra</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35931318/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220817212606&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Modern medicine continues to evolve, and the treatment armamentarium for various diseases grows more individualized across a breadth of medical disciplines. Cure rates for infectious diseases that were previously pan-fatal approach 100% because of the identification of the specific pathogen(s) involved and the use of appropriate combinations of drugs, where needed, to completely extinguish infection and hence prevent emergence of resistant strains. Similarly, with the assistance of technologies...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35931318"
                        open
                      >
                        <summary class="article-expander__title">Cancer: Slaying the 9-headed Hydra</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35931318/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220816212355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Modern medicine continues to evolve, and the treatment armamentarium for various diseases grows more individualized across a breadth of medical disciplines. Cure rates for infectious diseases that were previously pan-fatal approach 100% because of the identification of the specific pathogen(s) involved and the use of appropriate combinations of drugs, where needed, to completely extinguish infection and hence prevent emergence of resistant strains. Similarly, with the assistance of technologies...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35931318"
                        open
                      >
                        <summary class="article-expander__title">Cancer: Slaying the 9-headed Hydra</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35931318/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220815212205&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Modern medicine continues to evolve, and the treatment armamentarium for various diseases grows more individualized across a breadth of medical disciplines. Cure rates for infectious diseases that were previously pan-fatal approach 100% because of the identification of the specific pathogen(s) involved and the use of appropriate combinations of drugs, where needed, to completely extinguish infection and hence prevent emergence of resistant strains. Similarly, with the assistance of technologies...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35931318"
                        open
                      >
                        <summary class="article-expander__title">Cancer: Slaying the 9-headed Hydra</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35931318/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220814211915&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Modern medicine continues to evolve, and the treatment armamentarium for various diseases grows more individualized across a breadth of medical disciplines. Cure rates for infectious diseases that were previously pan-fatal approach 100% because of the identification of the specific pathogen(s) involved and the use of appropriate combinations of drugs, where needed, to completely extinguish infection and hence prevent emergence of resistant strains. Similarly, with the assistance of technologies...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35931318"
                        open
                      >
                        <summary class="article-expander__title">Cancer: Slaying the 9-headed Hydra</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35931318/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220813211549&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Modern medicine continues to evolve, and the treatment armamentarium for various diseases grows more individualized across a breadth of medical disciplines. Cure rates for infectious diseases that were previously pan-fatal approach 100% because of the identification of the specific pathogen(s) involved and the use of appropriate combinations of drugs, where needed, to completely extinguish infection and hence prevent emergence of resistant strains. Similarly, with the assistance of technologies...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35931318"
                        open
                      >
                        <summary class="article-expander__title">Cancer: Slaying the 9-headed Hydra</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35931318/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220812210706&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Modern medicine continues to evolve, and the treatment armamentarium for various diseases grows more individualized across a breadth of medical disciplines. Cure rates for infectious diseases that were previously pan-fatal approach 100% because of the identification of the specific pathogen(s) involved and the use of appropriate combinations of drugs, where needed, to completely extinguish infection and hence prevent emergence of resistant strains. Similarly, with the assistance of technologies...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35931318"
                        open
                      >
                        <summary class="article-expander__title">Cancer: Slaying the 9-headed Hydra</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35931318/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220811211442&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Modern medicine continues to evolve, and the treatment armamentarium for various diseases grows more individualized across a breadth of medical disciplines. Cure rates for infectious diseases that were previously pan-fatal approach 100% because of the identification of the specific pathogen(s) involved and the use of appropriate combinations of drugs, where needed, to completely extinguish infection and hence prevent emergence of resistant strains. Similarly, with the assistance of technologies...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35931318"
                        open
                      >
                        <summary class="article-expander__title">Cancer: Slaying the 9-headed Hydra</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35931318/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220810210900&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Modern medicine continues to evolve, and the treatment armamentarium for various diseases grows more individualized across a breadth of medical disciplines. Cure rates for infectious diseases that were previously pan-fatal approach 100% because of the identification of the specific pathogen(s) involved and the use of appropriate combinations of drugs, where needed, to completely extinguish infection and hence prevent emergence of resistant strains. Similarly, with the assistance of technologies...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35931318"
                        open
                      >
                        <summary class="article-expander__title">Cancer: Slaying the 9-headed Hydra</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35931318/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220809211355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Modern medicine continues to evolve, and the treatment armamentarium for various diseases grows more individualized across a breadth of medical disciplines. Cure rates for infectious diseases that were previously pan-fatal approach 100% because of the identification of the specific pathogen(s) involved and the use of appropriate combinations of drugs, where needed, to completely extinguish infection and hence prevent emergence of resistant strains. Similarly, with the assistance of technologies...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35931318"
                        open
                      >
                        <summary class="article-expander__title">Cancer: Slaying the 9-headed Hydra</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35931318/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220808212152&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Modern medicine continues to evolve, and the treatment armamentarium for various diseases grows more individualized across a breadth of medical disciplines. Cure rates for infectious diseases that were previously pan-fatal approach 100% because of the identification of the specific pathogen(s) involved and the use of appropriate combinations of drugs, where needed, to completely extinguish infection and hence prevent emergence of resistant strains. Similarly, with the assistance of technologies...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35931318"
                        open
                      >
                        <summary class="article-expander__title">Cancer: Slaying the 9-headed Hydra</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35931318/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220807211336&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Modern medicine continues to evolve, and the treatment armamentarium for various diseases grows more individualized across a breadth of medical disciplines. Cure rates for infectious diseases that were previously pan-fatal approach 100% because of the identification of the specific pathogen(s) involved and the use of appropriate combinations of drugs, where needed, to completely extinguish infection and hence prevent emergence of resistant strains. Similarly, with the assistance of technologies...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35931318"
                        open
                      >
                        <summary class="article-expander__title">Cancer: Slaying the 9-headed Hydra</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35931318/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220806211830&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Modern medicine continues to evolve, and the treatment armamentarium for various diseases grows more individualized across a breadth of medical disciplines. Cure rates for infectious diseases that were previously pan-fatal approach 100% because of the identification of the specific pathogen(s) involved and the use of appropriate combinations of drugs, where needed, to completely extinguish infection and hence prevent emergence of resistant strains. Similarly, with the assistance of technologies...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-08-04 datetime="2022-08-04T10:00:00.000Z">2022-08-04</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-08-04 datetime="2022-08-04T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608003622&amp;v&#x3D;2.17.7&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35924979"
                        open
                      >
                        <summary class="article-expander__title">The impact of inflammation-induced tumor plasticity during myeloid transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35924979/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220821211551&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an aging-associated condition characterized by the clonal outgrowth of mutated pre-leukemic cells. Individuals with CH are at an increased risk of developing hematopoietic malignancies. Here, we describe a novel animal model carrying a recurrent TET2 missense mutation, frequently found in CH and leukemic patients. In a fashion similar to CH, animals show signs of disease late in life when they develop a wide range of myeloid neoplasms, including acute myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35924979"
                        open
                      >
                        <summary class="article-expander__title">The impact of inflammation-induced tumor plasticity during myeloid transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35924979/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220820211309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an aging-associated condition characterized by the clonal outgrowth of mutated pre-leukemic cells. Individuals with CH are at an increased risk of developing hematopoietic malignancies. Here, we describe a novel animal model carrying a recurrent TET2 missense mutation, frequently found in CH and leukemic patients. In a fashion similar to CH, animals show signs of disease late in life when they develop a wide range of myeloid neoplasms, including acute myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35924979"
                        open
                      >
                        <summary class="article-expander__title">The impact of inflammation-induced tumor plasticity during myeloid transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35924979/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220819211457&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an aging-associated condition characterized by the clonal outgrowth of mutated pre-leukemic cells. Individuals with CH are at an increased risk of developing hematopoietic malignancies. Here, we describe a novel animal model carrying a recurrent TET2 missense mutation, frequently found in CH and leukemic patients. In a fashion similar to CH, animals show signs of disease late in life when they develop a wide range of myeloid neoplasms, including acute myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35923854"
                        open
                      >
                        <summary class="article-expander__title">Zebrafish models of inflammation in hematopoietic development and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35923854/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220819211457&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Zebrafish offer an excellent tool for studying the vertebrate hematopoietic system thanks to a highly conserved and rapidly developing hematopoietic program, genetic amenability, optical transparency, and experimental accessibility. Zebrafish studies have contributed to our understanding of hematopoiesis, a complex process regulated by signaling cues, inflammation being crucial among them. Hematopoietic stem cells (HSCs) are multipotent cells producing all the functional blood cells, including...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35924979"
                        open
                      >
                        <summary class="article-expander__title">The impact of inflammation-induced tumor plasticity during myeloid transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35924979/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220818211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an aging-associated condition characterized by the clonal outgrowth of mutated pre-leukemic cells. Individuals with CH are at an increased risk of developing hematopoietic malignancies. Here, we describe a novel animal model carrying a recurrent TET2 missense mutation, frequently found in CH and leukemic patients. In a fashion similar to CH, animals show signs of disease late in life when they develop a wide range of myeloid neoplasms, including acute myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35923854"
                        open
                      >
                        <summary class="article-expander__title">Zebrafish models of inflammation in hematopoietic development and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35923854/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220818211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Zebrafish offer an excellent tool for studying the vertebrate hematopoietic system thanks to a highly conserved and rapidly developing hematopoietic program, genetic amenability, optical transparency, and experimental accessibility. Zebrafish studies have contributed to our understanding of hematopoiesis, a complex process regulated by signaling cues, inflammation being crucial among them. Hematopoietic stem cells (HSCs) are multipotent cells producing all the functional blood cells, including...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35924979"
                        open
                      >
                        <summary class="article-expander__title">The impact of inflammation-induced tumor plasticity during myeloid transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35924979/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220817212606&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an aging-associated condition characterized by the clonal outgrowth of mutated pre-leukemic cells. Individuals with CH are at an increased risk of developing hematopoietic malignancies. Here, we describe a novel animal model carrying a recurrent TET2 missense mutation, frequently found in CH and leukemic patients. In a fashion similar to CH, animals show signs of disease late in life when they develop a wide range of myeloid neoplasms, including acute myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35923854"
                        open
                      >
                        <summary class="article-expander__title">Zebrafish models of inflammation in hematopoietic development and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35923854/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220817212606&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Zebrafish offer an excellent tool for studying the vertebrate hematopoietic system thanks to a highly conserved and rapidly developing hematopoietic program, genetic amenability, optical transparency, and experimental accessibility. Zebrafish studies have contributed to our understanding of hematopoiesis, a complex process regulated by signaling cues, inflammation being crucial among them. Hematopoietic stem cells (HSCs) are multipotent cells producing all the functional blood cells, including...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35924979"
                        open
                      >
                        <summary class="article-expander__title">The impact of inflammation-induced tumor plasticity during myeloid transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35924979/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220816212355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an aging-associated condition characterized by the clonal outgrowth of mutated pre-leukemic cells. Individuals with CH are at an increased risk of developing hematopoietic malignancies. Here, we describe a novel animal model carrying a recurrent TET2 missense mutation, frequently found in CH and leukemic patients. In a fashion similar to CH, animals show signs of disease late in life when they develop a wide range of myeloid neoplasms, including acute myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35923854"
                        open
                      >
                        <summary class="article-expander__title">Zebrafish models of inflammation in hematopoietic development and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35923854/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220816212355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Zebrafish offer an excellent tool for studying the vertebrate hematopoietic system thanks to a highly conserved and rapidly developing hematopoietic program, genetic amenability, optical transparency, and experimental accessibility. Zebrafish studies have contributed to our understanding of hematopoiesis, a complex process regulated by signaling cues, inflammation being crucial among them. Hematopoietic stem cells (HSCs) are multipotent cells producing all the functional blood cells, including...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35924979"
                        open
                      >
                        <summary class="article-expander__title">The impact of inflammation-induced tumor plasticity during myeloid transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35924979/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220815212205&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an aging-associated condition characterized by the clonal outgrowth of mutated pre-leukemic cells. Individuals with CH are at an increased risk of developing hematopoietic malignancies. Here, we describe a novel animal model carrying a recurrent TET2 missense mutation, frequently found in CH and leukemic patients. In a fashion similar to CH, animals show signs of disease late in life when they develop a wide range of myeloid neoplasms, including acute myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35923854"
                        open
                      >
                        <summary class="article-expander__title">Zebrafish models of inflammation in hematopoietic development and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35923854/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220815212205&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Zebrafish offer an excellent tool for studying the vertebrate hematopoietic system thanks to a highly conserved and rapidly developing hematopoietic program, genetic amenability, optical transparency, and experimental accessibility. Zebrafish studies have contributed to our understanding of hematopoiesis, a complex process regulated by signaling cues, inflammation being crucial among them. Hematopoietic stem cells (HSCs) are multipotent cells producing all the functional blood cells, including...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35924979"
                        open
                      >
                        <summary class="article-expander__title">The impact of inflammation-induced tumor plasticity during myeloid transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35924979/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220814211915&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an aging-associated condition characterized by the clonal outgrowth of mutated pre-leukemic cells. Individuals with CH are at an increased risk of developing hematopoietic malignancies. Here, we describe a novel animal model carrying a recurrent TET2 missense mutation, frequently found in CH and leukemic patients. In a fashion similar to CH, animals show signs of disease late in life when they develop a wide range of myeloid neoplasms, including acute myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35923854"
                        open
                      >
                        <summary class="article-expander__title">Zebrafish models of inflammation in hematopoietic development and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35923854/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220814211915&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Zebrafish offer an excellent tool for studying the vertebrate hematopoietic system thanks to a highly conserved and rapidly developing hematopoietic program, genetic amenability, optical transparency, and experimental accessibility. Zebrafish studies have contributed to our understanding of hematopoiesis, a complex process regulated by signaling cues, inflammation being crucial among them. Hematopoietic stem cells (HSCs) are multipotent cells producing all the functional blood cells, including...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35924979"
                        open
                      >
                        <summary class="article-expander__title">The impact of inflammation-induced tumor plasticity during myeloid transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35924979/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220813211549&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an aging-associated condition characterized by the clonal outgrowth of mutated pre-leukemic cells. Individuals with CH are at an increased risk of developing hematopoietic malignancies. Here, we describe a novel animal model carrying a recurrent TET2 missense mutation, frequently found in CH and leukemic patients. In a fashion similar to CH, animals show signs of disease late in life when they develop a wide range of myeloid neoplasms, including acute myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35923854"
                        open
                      >
                        <summary class="article-expander__title">Zebrafish models of inflammation in hematopoietic development and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35923854/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220813211549&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Zebrafish offer an excellent tool for studying the vertebrate hematopoietic system thanks to a highly conserved and rapidly developing hematopoietic program, genetic amenability, optical transparency, and experimental accessibility. Zebrafish studies have contributed to our understanding of hematopoiesis, a complex process regulated by signaling cues, inflammation being crucial among them. Hematopoietic stem cells (HSCs) are multipotent cells producing all the functional blood cells, including...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35924979"
                        open
                      >
                        <summary class="article-expander__title">The impact of inflammation-induced tumor plasticity during myeloid transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35924979/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220812210706&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an aging-associated condition characterized by the clonal outgrowth of mutated pre-leukemic cells. Individuals with CH are at an increased risk of developing hematopoietic malignancies. Here, we describe a novel animal model carrying a recurrent TET2 missense mutation, frequently found in CH and leukemic patients. In a fashion similar to CH, animals show signs of disease late in life when they develop a wide range of myeloid neoplasms, including acute myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35923854"
                        open
                      >
                        <summary class="article-expander__title">Zebrafish models of inflammation in hematopoietic development and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35923854/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220812210706&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Zebrafish offer an excellent tool for studying the vertebrate hematopoietic system thanks to a highly conserved and rapidly developing hematopoietic program, genetic amenability, optical transparency, and experimental accessibility. Zebrafish studies have contributed to our understanding of hematopoiesis, a complex process regulated by signaling cues, inflammation being crucial among them. Hematopoietic stem cells (HSCs) are multipotent cells producing all the functional blood cells, including...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35924979"
                        open
                      >
                        <summary class="article-expander__title">The impact of inflammation-induced tumor plasticity during myeloid transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35924979/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220811211442&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an aging-associated condition characterized by the clonal outgrowth of mutated pre-leukemic cells. Individuals with CH are at an increased risk of developing hematopoietic malignancies. Here, we describe a novel animal model carrying a recurrent TET2 missense mutation, frequently found in CH and leukemic patients. In a fashion similar to CH, animals show signs of disease late in life when they develop a wide range of myeloid neoplasms, including acute myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35923854"
                        open
                      >
                        <summary class="article-expander__title">Zebrafish models of inflammation in hematopoietic development and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35923854/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220811211442&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Zebrafish offer an excellent tool for studying the vertebrate hematopoietic system thanks to a highly conserved and rapidly developing hematopoietic program, genetic amenability, optical transparency, and experimental accessibility. Zebrafish studies have contributed to our understanding of hematopoiesis, a complex process regulated by signaling cues, inflammation being crucial among them. Hematopoietic stem cells (HSCs) are multipotent cells producing all the functional blood cells, including...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35924979"
                        open
                      >
                        <summary class="article-expander__title">The impact of inflammation-induced tumor plasticity during myeloid transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35924979/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220810210900&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an aging-associated condition characterized by the clonal outgrowth of mutated pre-leukemic cells. Individuals with CH are at an increased risk of developing hematopoietic malignancies. Here, we describe a novel animal model carrying a recurrent TET2 missense mutation, frequently found in CH and leukemic patients. In a fashion similar to CH, animals show signs of disease late in life when they develop a wide range of myeloid neoplasms, including acute myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35923854"
                        open
                      >
                        <summary class="article-expander__title">Zebrafish models of inflammation in hematopoietic development and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35923854/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220810210900&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Zebrafish offer an excellent tool for studying the vertebrate hematopoietic system thanks to a highly conserved and rapidly developing hematopoietic program, genetic amenability, optical transparency, and experimental accessibility. Zebrafish studies have contributed to our understanding of hematopoiesis, a complex process regulated by signaling cues, inflammation being crucial among them. Hematopoietic stem cells (HSCs) are multipotent cells producing all the functional blood cells, including...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35924979"
                        open
                      >
                        <summary class="article-expander__title">The impact of inflammation-induced tumor plasticity during myeloid transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35924979/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220809211355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an aging-associated condition characterized by the clonal outgrowth of mutated pre-leukemic cells. Individuals with CH are at an increased risk of developing hematopoietic malignancies. Here, we describe a novel animal model carrying a recurrent TET2 missense mutation, frequently found in CH and leukemic patients. In a fashion similar to CH, animals show signs of disease late in life when they develop a wide range of myeloid neoplasms, including acute myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35923854"
                        open
                      >
                        <summary class="article-expander__title">Zebrafish models of inflammation in hematopoietic development and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35923854/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220809211355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Zebrafish offer an excellent tool for studying the vertebrate hematopoietic system thanks to a highly conserved and rapidly developing hematopoietic program, genetic amenability, optical transparency, and experimental accessibility. Zebrafish studies have contributed to our understanding of hematopoiesis, a complex process regulated by signaling cues, inflammation being crucial among them. Hematopoietic stem cells (HSCs) are multipotent cells producing all the functional blood cells, including...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35924979"
                        open
                      >
                        <summary class="article-expander__title">The impact of inflammation-induced tumor plasticity during myeloid transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35924979/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220808212152&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an aging-associated condition characterized by the clonal outgrowth of mutated pre-leukemic cells. Individuals with CH are at an increased risk of developing hematopoietic malignancies. Here, we describe a novel animal model carrying a recurrent TET2 missense mutation, frequently found in CH and leukemic patients. In a fashion similar to CH, animals show signs of disease late in life when they develop a wide range of myeloid neoplasms, including acute myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35923854"
                        open
                      >
                        <summary class="article-expander__title">Zebrafish models of inflammation in hematopoietic development and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35923854/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220808212152&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Zebrafish offer an excellent tool for studying the vertebrate hematopoietic system thanks to a highly conserved and rapidly developing hematopoietic program, genetic amenability, optical transparency, and experimental accessibility. Zebrafish studies have contributed to our understanding of hematopoiesis, a complex process regulated by signaling cues, inflammation being crucial among them. Hematopoietic stem cells (HSCs) are multipotent cells producing all the functional blood cells, including...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35924979"
                        open
                      >
                        <summary class="article-expander__title">The impact of inflammation-induced tumor plasticity during myeloid transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35924979/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220807211336&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an aging-associated condition characterized by the clonal outgrowth of mutated pre-leukemic cells. Individuals with CH are at an increased risk of developing hematopoietic malignancies. Here, we describe a novel animal model carrying a recurrent TET2 missense mutation, frequently found in CH and leukemic patients. In a fashion similar to CH, animals show signs of disease late in life when they develop a wide range of myeloid neoplasms, including acute myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35923854"
                        open
                      >
                        <summary class="article-expander__title">Zebrafish models of inflammation in hematopoietic development and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35923854/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220807211336&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Zebrafish offer an excellent tool for studying the vertebrate hematopoietic system thanks to a highly conserved and rapidly developing hematopoietic program, genetic amenability, optical transparency, and experimental accessibility. Zebrafish studies have contributed to our understanding of hematopoiesis, a complex process regulated by signaling cues, inflammation being crucial among them. Hematopoietic stem cells (HSCs) are multipotent cells producing all the functional blood cells, including...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35924979"
                        open
                      >
                        <summary class="article-expander__title">The impact of inflammation-induced tumor plasticity during myeloid transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35924979/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220806211830&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an aging-associated condition characterized by the clonal outgrowth of mutated pre-leukemic cells. Individuals with CH are at an increased risk of developing hematopoietic malignancies. Here, we describe a novel animal model carrying a recurrent TET2 missense mutation, frequently found in CH and leukemic patients. In a fashion similar to CH, animals show signs of disease late in life when they develop a wide range of myeloid neoplasms, including acute myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35923854"
                        open
                      >
                        <summary class="article-expander__title">Zebrafish models of inflammation in hematopoietic development and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35923854/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220806211830&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Zebrafish offer an excellent tool for studying the vertebrate hematopoietic system thanks to a highly conserved and rapidly developing hematopoietic program, genetic amenability, optical transparency, and experimental accessibility. Zebrafish studies have contributed to our understanding of hematopoiesis, a complex process regulated by signaling cues, inflammation being crucial among them. Hematopoietic stem cells (HSCs) are multipotent cells producing all the functional blood cells, including...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35924979"
                        open
                      >
                        <summary class="article-expander__title">The impact of inflammation-induced tumor plasticity during myeloid transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35924979/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220805210849&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an aging-associated condition characterized by the clonal outgrowth of mutated pre-leukemic cells. Individuals with CH are at an increased risk of developing hematopoietic malignancies. Here, we describe a novel animal model carrying a recurrent TET2 missense mutation, frequently found in CH and leukemic patients. In a fashion similar to CH, animals show signs of disease late in life when they develop a wide range of myeloid neoplasms, including acute myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35923854"
                        open
                      >
                        <summary class="article-expander__title">Zebrafish models of inflammation in hematopoietic development and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35923854/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220805210849&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Zebrafish offer an excellent tool for studying the vertebrate hematopoietic system thanks to a highly conserved and rapidly developing hematopoietic program, genetic amenability, optical transparency, and experimental accessibility. Zebrafish studies have contributed to our understanding of hematopoiesis, a complex process regulated by signaling cues, inflammation being crucial among them. Hematopoietic stem cells (HSCs) are multipotent cells producing all the functional blood cells, including...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35924979"
                        open
                      >
                        <summary class="article-expander__title">The impact of inflammation-induced tumor plasticity during myeloid transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35924979/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220804212147&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an aging-associated condition characterized by the clonal outgrowth of mutated pre-leukemic cells. Individuals with CH are at an increased risk of developing hematopoietic malignancies. Here, we describe a novel animal model carrying a recurrent TET2 missense mutation, frequently found in CH and leukemic patients. In a fashion similar to CH, animals show signs of disease late in life when they develop a wide range of myeloid neoplasms, including acute myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35923854"
                        open
                      >
                        <summary class="article-expander__title">Zebrafish models of inflammation in hematopoietic development and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35923854/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220804212147&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Zebrafish offer an excellent tool for studying the vertebrate hematopoietic system thanks to a highly conserved and rapidly developing hematopoietic program, genetic amenability, optical transparency, and experimental accessibility. Zebrafish studies have contributed to our understanding of hematopoiesis, a complex process regulated by signaling cues, inflammation being crucial among them. Hematopoietic stem cells (HSCs) are multipotent cells producing all the functional blood cells, including...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-08-03 datetime="2022-08-03T10:00:00.000Z">2022-08-03</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-08-03 datetime="2022-08-03T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608003622&amp;v&#x3D;2.17.7&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35921527"
                        open
                      >
                        <summary class="article-expander__title">Clonal germinal center B cells function as a niche for T-cell lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35921527/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220818211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Angioimmunoblastic T-cell lymphoma (AITL) is proposed to be initiated by age-related clonal hematopoiesis (ACH) with TET2 mutations, whereas the G17V RHOA mutation in immature cells with TET2 mutations promotes development of T follicular helper-like tumor cells. Here, we investigated the mechanism by which TET2-mutant immune cells enable AITL development, using mouse models and human samples. Among the two mouse models, mice lacking Tet2 in all the blood cells (Mx-Cre × Tet2flox/flox× G17V RHOA...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35921527"
                        open
                      >
                        <summary class="article-expander__title">Clonal germinal center B cells function as a niche for T-cell lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35921527/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220817212606&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Angioimmunoblastic T-cell lymphoma (AITL) is proposed to be initiated by age-related clonal hematopoiesis (ACH) with TET2 mutations, whereas the G17V RHOA mutation in immature cells with TET2 mutations promotes development of T follicular helper-like tumor cells. Here, we investigated the mechanism by which TET2-mutant immune cells enable AITL development, using mouse models and human samples. Among the two mouse models, mice lacking Tet2 in all the blood cells (Mx-Cre × Tet2flox/flox× G17V RHOA...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35921527"
                        open
                      >
                        <summary class="article-expander__title">Clonal germinal center B cells function as a niche for T-cell lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35921527/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220816212355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Angioimmunoblastic T-cell lymphoma (AITL) is proposed to be initiated by age-related clonal hematopoiesis (ACH) with TET2 mutations, whereas the G17V RHOA mutation in immature cells with TET2 mutations promotes development of T follicular helper-like tumor cells. Here, we investigated the mechanism by which TET2-mutant immune cells enable AITL development, using mouse models and human samples. Among the two mouse models, mice lacking Tet2 in all the blood cells (Mx-Cre × Tet2flox/flox× G17V RHOA...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35921527"
                        open
                      >
                        <summary class="article-expander__title">Clonal germinal center B cells function as a niche for T-cell lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35921527/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220815212205&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Angioimmunoblastic T-cell lymphoma (AITL) is proposed to be initiated by age-related clonal hematopoiesis (ACH) with TET2 mutations, whereas the G17V RHOA mutation in immature cells with TET2 mutations promotes development of T follicular helper-like tumor cells. Here, we investigated the mechanism by which TET2-mutant immune cells enable AITL development, using mouse models and human samples. Among the two mouse models, mice lacking Tet2 in all the blood cells (Mx-Cre × Tet2flox/flox× G17V RHOA...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35921527"
                        open
                      >
                        <summary class="article-expander__title">Clonal germinal center B cells function as a niche for T-cell lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35921527/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220814211915&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Angioimmunoblastic T-cell lymphoma (AITL) is proposed to be initiated by age-related clonal hematopoiesis (ACH) with TET2 mutations, whereas the G17V RHOA mutation in immature cells with TET2 mutations promotes development of T follicular helper-like tumor cells. Here, we investigated the mechanism by which TET2-mutant immune cells enable AITL development, using mouse models and human samples. Among the two mouse models, mice lacking Tet2 in all the blood cells (Mx-Cre × Tet2flox/flox× G17V RHOA...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35921527"
                        open
                      >
                        <summary class="article-expander__title">Clonal germinal center B cells function as a niche for T-cell lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35921527/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220813211549&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Angioimmunoblastic T-cell lymphoma (AITL) is proposed to be initiated by age-related clonal hematopoiesis (ACH) with TET2 mutations, whereas the G17V RHOA mutation in immature cells with TET2 mutations promotes development of T follicular helper-like tumor cells. Here, we investigated the mechanism by which TET2-mutant immune cells enable AITL development, using mouse models and human samples. Among the two mouse models, mice lacking Tet2 in all the blood cells (Mx-Cre × Tet2flox/flox× G17V RHOA...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35921527"
                        open
                      >
                        <summary class="article-expander__title">Clonal germinal center B cells function as a niche for T-cell lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35921527/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220812210706&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Angioimmunoblastic T-cell lymphoma (AITL) is proposed to be initiated by age-related clonal hematopoiesis (ACH) with TET2 mutations, whereas the G17V RHOA mutation in immature cells with TET2 mutations promotes development of T follicular helper-like tumor cells. Here, we investigated the mechanism by which TET2-mutant immune cells enable AITL development, using mouse models and human samples. Among the two mouse models, mice lacking Tet2 in all the blood cells (Mx-Cre × Tet2flox/flox× G17V RHOA...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35921527"
                        open
                      >
                        <summary class="article-expander__title">Clonal germinal center B cells function as a niche for T-cell lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35921527/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220811211442&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Angioimmunoblastic T-cell lymphoma (AITL) is proposed to be initiated by age-related clonal hematopoiesis (ACH) with TET2 mutations, whereas the G17V RHOA mutation in immature cells with TET2 mutations promotes development of T follicular helper-like tumor cells. Here, we investigated the mechanism by which TET2-mutant immune cells enable AITL development, using mouse models and human samples. Among the two mouse models, mice lacking Tet2 in all the blood cells (Mx-Cre × Tet2flox/flox× G17V RHOA...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35921527"
                        open
                      >
                        <summary class="article-expander__title">Clonal germinal center B cells function as a niche for T-cell lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35921527/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220810210900&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Angioimmunoblastic T-cell lymphoma (AITL) is proposed to be initiated by age-related clonal hematopoiesis (ACH) with TET2 mutations, whereas the G17V RHOA mutation in immature cells with TET2 mutations promotes development of T follicular helper-like tumor cells. Here, we investigated the mechanism by which TET2-mutant immune cells enable AITL development, using mouse models and human samples. Among the two mouse models, mice lacking Tet2 in all the blood cells (Mx-Cre × Tet2flox/flox× G17V RHOA...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35921527"
                        open
                      >
                        <summary class="article-expander__title">Clonal germinal center B cells function as a niche for T-cell lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35921527/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220809211355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Angioimmunoblastic T-cell lymphoma (AITL) is proposed to be initiated by age-related clonal hematopoiesis (ACH) with TET2 mutations, whereas the G17V RHOA mutation in immature cells with TET2 mutations promotes development of T follicular helper-like tumor cells. Here, we investigated the mechanism by which TET2-mutant immune cells enable AITL development, using mouse models and human samples. Among the two mouse models, mice lacking Tet2 in all the blood cells (Mx-Cre × Tet2flox/flox× G17V RHOA...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35921527"
                        open
                      >
                        <summary class="article-expander__title">Clonal germinal center B cells function as a niche for T-cell lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35921527/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220808212152&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Angioimmunoblastic T-cell lymphoma (AITL) is proposed to be initiated by age-related clonal hematopoiesis (ACH) with TET2 mutations, whereas the G17V RHOA mutation in immature cells with TET2 mutations promotes development of T follicular helper-like tumor cells. Here, we investigated the mechanism by which TET2-mutant immune cells enable AITL development, using mouse models and human samples. Among the two mouse models, mice lacking Tet2 in all the blood cells (Mx-Cre × Tet2flox/flox× G17V RHOA...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35921527"
                        open
                      >
                        <summary class="article-expander__title">Clonal germinal center B cells function as a niche for T-cell lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35921527/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220807211336&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Angioimmunoblastic T-cell lymphoma (AITL) is proposed to be initiated by age-related clonal hematopoiesis (ACH) with TET2 mutations, whereas the G17V RHOA mutation in immature cells with TET2 mutations promotes development of T follicular helper-like tumor cells. Here, we investigated the mechanism by which TET2-mutant immune cells enable AITL development, using mouse models and human samples. Among the two mouse models, mice lacking Tet2 in all the blood cells (Mx-Cre × Tet2flox/flox× G17V RHOA...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35921527"
                        open
                      >
                        <summary class="article-expander__title">Clonal germinal center B cells function as a niche for T-cell lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35921527/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220806211830&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Angioimmunoblastic T-cell lymphoma (AITL) is proposed to be initiated by age-related clonal hematopoiesis (ACH) with TET2 mutations, whereas the G17V RHOA mutation in immature cells with TET2 mutations promotes development of T follicular helper-like tumor cells. Here, we investigated the mechanism by which TET2-mutant immune cells enable AITL development, using mouse models and human samples. Among the two mouse models, mice lacking Tet2 in all the blood cells (Mx-Cre × Tet2flox/flox× G17V RHOA...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35921527"
                        open
                      >
                        <summary class="article-expander__title">Clonal germinal center B cells function as a niche for T-cell lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35921527/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220805210849&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Angioimmunoblastic T-cell lymphoma (AITL) is proposed to be initiated by age-related clonal hematopoiesis (ACH) with TET2 mutations, whereas the G17V RHOA mutation in immature cells with TET2 mutations promotes development of T follicular helper-like tumor cells. Here, we investigated the mechanism by which TET2-mutant immune cells enable AITL development, using mouse models and human samples. Among the two mouse models, mice lacking Tet2 in all the blood cells (Mx-Cre × Tet2flox/flox× G17V RHOA...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35921527"
                        open
                      >
                        <summary class="article-expander__title">Clonal germinal center B cells function as a niche for T-cell lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35921527/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220804212147&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Angioimmunoblastic T-cell lymphoma (AITL) is proposed to be initiated by age-related clonal hematopoiesis (ACH) with TET2 mutations, whereas the G17V RHOA mutation in immature cells with TET2 mutations promotes development of T follicular helper-like tumor cells. Here, we investigated the mechanism by which TET2-mutant immune cells enable AITL development, using mouse models and human samples. Among the two mouse models, mice lacking Tet2 in all the blood cells (Mx-Cre × Tet2flox/flox× G17V RHOA...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35921527"
                        open
                      >
                        <summary class="article-expander__title">Clonal germinal center B cells function as a niche for T-cell lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35921527/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220803212051&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Angioimmunoblastic T-cell lymphoma (AITL) is proposed to be initiated by age-related clonal hematopoiesis (ACH) with TET2 mutations, whereas the G17V RHOA mutation in immature cells with TET2 mutations promotes development of T follicular helper-like tumor cells. Here, we investigated the mechanism by which TET2-mutant immune cells enable AITL development, using mouse models and human samples. Among the two mouse models, mice lacking Tet2 in all the blood cells (Mx-Cre × Tet2flox/flox× G17V RHOA...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-08-01 datetime="2022-08-01T10:00:00.000Z">2022-08-01</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-08-01 datetime="2022-08-01T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608003622&amp;v&#x3D;2.17.7&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35914495"
                        open
                      >
                        <summary class="article-expander__title">Mediating and maintaining methylation while minimizing mutation: Recent advances on mammalian DNA methyltransferases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35914495/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220817212606&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mammalian genomes are methylated on carbon-5 of many cytosines, mostly in CpG dinucleotides. Methylation patterns are maintained during mitosis via DNMT1, and regulatory factors involved in processes that include histone modifications. Methylation in a sequence longer than CpG can influence the binding of sequence-specific transcription factors, thus affecting gene expression. 5-Methylcytosine deamination results in C-to-T transition. While some mutations are beneficial, most are not; so...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35914227"
                        open
                      >
                        <summary class="article-expander__title">Evaluation of Clonal Hematopoiesis in Pediatric ADA-SCID Gene Therapy Participants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35914227/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220817212606&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Autologous stem cell transplant with gene therapy (ASCT-GT) provides curative therapy while reducing pre-transplant immune-suppressive conditioning and eliminating post-transplant immune suppression. Clonal hematopoiesis of indeterminate potential (CHIP)-associated mutations increase and telomere lengths (TL) shorten with natural aging and DNA damaging processes. It is possible that if CHIP is present before ASCT-GT, or mutagenesis occurs due to busulfan exposure, the HSCs carrying these somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35914495"
                        open
                      >
                        <summary class="article-expander__title">Mediating and maintaining methylation while minimizing mutation: Recent advances on mammalian DNA methyltransferases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35914495/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220816212355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mammalian genomes are methylated on carbon-5 of many cytosines, mostly in CpG dinucleotides. Methylation patterns are maintained during mitosis via DNMT1, and regulatory factors involved in processes that include histone modifications. Methylation in a sequence longer than CpG can influence the binding of sequence-specific transcription factors, thus affecting gene expression. 5-Methylcytosine deamination results in C-to-T transition. While some mutations are beneficial, most are not; so...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35914227"
                        open
                      >
                        <summary class="article-expander__title">Evaluation of Clonal Hematopoiesis in Pediatric ADA-SCID Gene Therapy Participants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35914227/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220816212355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Autologous stem cell transplant with gene therapy (ASCT-GT) provides curative therapy while reducing pre-transplant immune-suppressive conditioning and eliminating post-transplant immune suppression. Clonal hematopoiesis of indeterminate potential (CHIP)-associated mutations increase and telomere lengths (TL) shorten with natural aging and DNA damaging processes. It is possible that if CHIP is present before ASCT-GT, or mutagenesis occurs due to busulfan exposure, the HSCs carrying these somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35914495"
                        open
                      >
                        <summary class="article-expander__title">Mediating and maintaining methylation while minimizing mutation: Recent advances on mammalian DNA methyltransferases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35914495/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220815212205&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mammalian genomes are methylated on carbon-5 of many cytosines, mostly in CpG dinucleotides. Methylation patterns are maintained during mitosis via DNMT1, and regulatory factors involved in processes that include histone modifications. Methylation in a sequence longer than CpG can influence the binding of sequence-specific transcription factors, thus affecting gene expression. 5-Methylcytosine deamination results in C-to-T transition. While some mutations are beneficial, most are not; so...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35914227"
                        open
                      >
                        <summary class="article-expander__title">Evaluation of Clonal Hematopoiesis in Pediatric ADA-SCID Gene Therapy Participants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35914227/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220815212205&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Autologous stem cell transplant with gene therapy (ASCT-GT) provides curative therapy while reducing pre-transplant immune-suppressive conditioning and eliminating post-transplant immune suppression. Clonal hematopoiesis of indeterminate potential (CHIP)-associated mutations increase and telomere lengths (TL) shorten with natural aging and DNA damaging processes. It is possible that if CHIP is present before ASCT-GT, or mutagenesis occurs due to busulfan exposure, the HSCs carrying these somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35914495"
                        open
                      >
                        <summary class="article-expander__title">Mediating and maintaining methylation while minimizing mutation: Recent advances on mammalian DNA methyltransferases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35914495/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220814211915&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mammalian genomes are methylated on carbon-5 of many cytosines, mostly in CpG dinucleotides. Methylation patterns are maintained during mitosis via DNMT1, and regulatory factors involved in processes that include histone modifications. Methylation in a sequence longer than CpG can influence the binding of sequence-specific transcription factors, thus affecting gene expression. 5-Methylcytosine deamination results in C-to-T transition. While some mutations are beneficial, most are not; so...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35914227"
                        open
                      >
                        <summary class="article-expander__title">Evaluation of Clonal Hematopoiesis in Pediatric ADA-SCID Gene Therapy Participants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35914227/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220814211915&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Autologous stem cell transplant with gene therapy (ASCT-GT) provides curative therapy while reducing pre-transplant immune-suppressive conditioning and eliminating post-transplant immune suppression. Clonal hematopoiesis of indeterminate potential (CHIP)-associated mutations increase and telomere lengths (TL) shorten with natural aging and DNA damaging processes. It is possible that if CHIP is present before ASCT-GT, or mutagenesis occurs due to busulfan exposure, the HSCs carrying these somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35914495"
                        open
                      >
                        <summary class="article-expander__title">Mediating and maintaining methylation while minimizing mutation: Recent advances on mammalian DNA methyltransferases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35914495/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220813211549&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mammalian genomes are methylated on carbon-5 of many cytosines, mostly in CpG dinucleotides. Methylation patterns are maintained during mitosis via DNMT1, and regulatory factors involved in processes that include histone modifications. Methylation in a sequence longer than CpG can influence the binding of sequence-specific transcription factors, thus affecting gene expression. 5-Methylcytosine deamination results in C-to-T transition. While some mutations are beneficial, most are not; so...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35914227"
                        open
                      >
                        <summary class="article-expander__title">Evaluation of Clonal Hematopoiesis in Pediatric ADA-SCID Gene Therapy Participants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35914227/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220813211549&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Autologous stem cell transplant with gene therapy (ASCT-GT) provides curative therapy while reducing pre-transplant immune-suppressive conditioning and eliminating post-transplant immune suppression. Clonal hematopoiesis of indeterminate potential (CHIP)-associated mutations increase and telomere lengths (TL) shorten with natural aging and DNA damaging processes. It is possible that if CHIP is present before ASCT-GT, or mutagenesis occurs due to busulfan exposure, the HSCs carrying these somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35914495"
                        open
                      >
                        <summary class="article-expander__title">Mediating and maintaining methylation while minimizing mutation: Recent advances on mammalian DNA methyltransferases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35914495/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220812210706&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mammalian genomes are methylated on carbon-5 of many cytosines, mostly in CpG dinucleotides. Methylation patterns are maintained during mitosis via DNMT1, and regulatory factors involved in processes that include histone modifications. Methylation in a sequence longer than CpG can influence the binding of sequence-specific transcription factors, thus affecting gene expression. 5-Methylcytosine deamination results in C-to-T transition. While some mutations are beneficial, most are not; so...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35914227"
                        open
                      >
                        <summary class="article-expander__title">Evaluation of Clonal Hematopoiesis in Pediatric ADA-SCID Gene Therapy Participants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35914227/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220812210706&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Autologous stem cell transplant with gene therapy (ASCT-GT) provides curative therapy while reducing pre-transplant immune-suppressive conditioning and eliminating post-transplant immune suppression. Clonal hematopoiesis of indeterminate potential (CHIP)-associated mutations increase and telomere lengths (TL) shorten with natural aging and DNA damaging processes. It is possible that if CHIP is present before ASCT-GT, or mutagenesis occurs due to busulfan exposure, the HSCs carrying these somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35914495"
                        open
                      >
                        <summary class="article-expander__title">Mediating and maintaining methylation while minimizing mutation: Recent advances on mammalian DNA methyltransferases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35914495/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220811211442&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mammalian genomes are methylated on carbon-5 of many cytosines, mostly in CpG dinucleotides. Methylation patterns are maintained during mitosis via DNMT1, and regulatory factors involved in processes that include histone modifications. Methylation in a sequence longer than CpG can influence the binding of sequence-specific transcription factors, thus affecting gene expression. 5-Methylcytosine deamination results in C-to-T transition. While some mutations are beneficial, most are not; so...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35914227"
                        open
                      >
                        <summary class="article-expander__title">Evaluation of Clonal Hematopoiesis in Pediatric ADA-SCID Gene Therapy Participants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35914227/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220811211442&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Autologous stem cell transplant with gene therapy (ASCT-GT) provides curative therapy while reducing pre-transplant immune-suppressive conditioning and eliminating post-transplant immune suppression. Clonal hematopoiesis of indeterminate potential (CHIP)-associated mutations increase and telomere lengths (TL) shorten with natural aging and DNA damaging processes. It is possible that if CHIP is present before ASCT-GT, or mutagenesis occurs due to busulfan exposure, the HSCs carrying these somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35914495"
                        open
                      >
                        <summary class="article-expander__title">Mediating and maintaining methylation while minimizing mutation: Recent advances on mammalian DNA methyltransferases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35914495/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220810210900&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mammalian genomes are methylated on carbon-5 of many cytosines, mostly in CpG dinucleotides. Methylation patterns are maintained during mitosis via DNMT1, and regulatory factors involved in processes that include histone modifications. Methylation in a sequence longer than CpG can influence the binding of sequence-specific transcription factors, thus affecting gene expression. 5-Methylcytosine deamination results in C-to-T transition. While some mutations are beneficial, most are not; so...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35914227"
                        open
                      >
                        <summary class="article-expander__title">Evaluation of Clonal Hematopoiesis in Pediatric ADA-SCID Gene Therapy Participants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35914227/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220810210900&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Autologous stem cell transplant with gene therapy (ASCT-GT) provides curative therapy while reducing pre-transplant immune-suppressive conditioning and eliminating post-transplant immune suppression. Clonal hematopoiesis of indeterminate potential (CHIP)-associated mutations increase and telomere lengths (TL) shorten with natural aging and DNA damaging processes. It is possible that if CHIP is present before ASCT-GT, or mutagenesis occurs due to busulfan exposure, the HSCs carrying these somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35914495"
                        open
                      >
                        <summary class="article-expander__title">Mediating and maintaining methylation while minimizing mutation: Recent advances on mammalian DNA methyltransferases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35914495/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220809211355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mammalian genomes are methylated on carbon-5 of many cytosines, mostly in CpG dinucleotides. Methylation patterns are maintained during mitosis via DNMT1, and regulatory factors involved in processes that include histone modifications. Methylation in a sequence longer than CpG can influence the binding of sequence-specific transcription factors, thus affecting gene expression. 5-Methylcytosine deamination results in C-to-T transition. While some mutations are beneficial, most are not; so...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35914227"
                        open
                      >
                        <summary class="article-expander__title">Evaluation of Clonal Hematopoiesis in Pediatric ADA-SCID Gene Therapy Participants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35914227/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220809211355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Autologous stem cell transplant with gene therapy (ASCT-GT) provides curative therapy while reducing pre-transplant immune-suppressive conditioning and eliminating post-transplant immune suppression. Clonal hematopoiesis of indeterminate potential (CHIP)-associated mutations increase and telomere lengths (TL) shorten with natural aging and DNA damaging processes. It is possible that if CHIP is present before ASCT-GT, or mutagenesis occurs due to busulfan exposure, the HSCs carrying these somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35914495"
                        open
                      >
                        <summary class="article-expander__title">Mediating and maintaining methylation while minimizing mutation: Recent advances on mammalian DNA methyltransferases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35914495/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220808212152&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mammalian genomes are methylated on carbon-5 of many cytosines, mostly in CpG dinucleotides. Methylation patterns are maintained during mitosis via DNMT1, and regulatory factors involved in processes that include histone modifications. Methylation in a sequence longer than CpG can influence the binding of sequence-specific transcription factors, thus affecting gene expression. 5-Methylcytosine deamination results in C-to-T transition. While some mutations are beneficial, most are not; so...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35914227"
                        open
                      >
                        <summary class="article-expander__title">Evaluation of Clonal Hematopoiesis in Pediatric ADA-SCID Gene Therapy Participants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35914227/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220808212152&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Autologous stem cell transplant with gene therapy (ASCT-GT) provides curative therapy while reducing pre-transplant immune-suppressive conditioning and eliminating post-transplant immune suppression. Clonal hematopoiesis of indeterminate potential (CHIP)-associated mutations increase and telomere lengths (TL) shorten with natural aging and DNA damaging processes. It is possible that if CHIP is present before ASCT-GT, or mutagenesis occurs due to busulfan exposure, the HSCs carrying these somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35914495"
                        open
                      >
                        <summary class="article-expander__title">Mediating and maintaining methylation while minimizing mutation: Recent advances on mammalian DNA methyltransferases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35914495/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220807211336&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mammalian genomes are methylated on carbon-5 of many cytosines, mostly in CpG dinucleotides. Methylation patterns are maintained during mitosis via DNMT1, and regulatory factors involved in processes that include histone modifications. Methylation in a sequence longer than CpG can influence the binding of sequence-specific transcription factors, thus affecting gene expression. 5-Methylcytosine deamination results in C-to-T transition. While some mutations are beneficial, most are not; so...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35914227"
                        open
                      >
                        <summary class="article-expander__title">Evaluation of Clonal Hematopoiesis in Pediatric ADA-SCID Gene Therapy Participants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35914227/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220807211336&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Autologous stem cell transplant with gene therapy (ASCT-GT) provides curative therapy while reducing pre-transplant immune-suppressive conditioning and eliminating post-transplant immune suppression. Clonal hematopoiesis of indeterminate potential (CHIP)-associated mutations increase and telomere lengths (TL) shorten with natural aging and DNA damaging processes. It is possible that if CHIP is present before ASCT-GT, or mutagenesis occurs due to busulfan exposure, the HSCs carrying these somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35914495"
                        open
                      >
                        <summary class="article-expander__title">Mediating and maintaining methylation while minimizing mutation: Recent advances on mammalian DNA methyltransferases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35914495/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220806211830&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mammalian genomes are methylated on carbon-5 of many cytosines, mostly in CpG dinucleotides. Methylation patterns are maintained during mitosis via DNMT1, and regulatory factors involved in processes that include histone modifications. Methylation in a sequence longer than CpG can influence the binding of sequence-specific transcription factors, thus affecting gene expression. 5-Methylcytosine deamination results in C-to-T transition. While some mutations are beneficial, most are not; so...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35914227"
                        open
                      >
                        <summary class="article-expander__title">Evaluation of Clonal Hematopoiesis in Pediatric ADA-SCID Gene Therapy Participants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35914227/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220806211830&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Autologous stem cell transplant with gene therapy (ASCT-GT) provides curative therapy while reducing pre-transplant immune-suppressive conditioning and eliminating post-transplant immune suppression. Clonal hematopoiesis of indeterminate potential (CHIP)-associated mutations increase and telomere lengths (TL) shorten with natural aging and DNA damaging processes. It is possible that if CHIP is present before ASCT-GT, or mutagenesis occurs due to busulfan exposure, the HSCs carrying these somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35914495"
                        open
                      >
                        <summary class="article-expander__title">Mediating and maintaining methylation while minimizing mutation: Recent advances on mammalian DNA methyltransferases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35914495/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220805210849&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mammalian genomes are methylated on carbon-5 of many cytosines, mostly in CpG dinucleotides. Methylation patterns are maintained during mitosis via DNMT1, and regulatory factors involved in processes that include histone modifications. Methylation in a sequence longer than CpG can influence the binding of sequence-specific transcription factors, thus affecting gene expression. 5-Methylcytosine deamination results in C-to-T transition. While some mutations are beneficial, most are not; so...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35914227"
                        open
                      >
                        <summary class="article-expander__title">Evaluation of Clonal Hematopoiesis in Pediatric ADA-SCID Gene Therapy Participants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35914227/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220805210849&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Autologous stem cell transplant with gene therapy (ASCT-GT) provides curative therapy while reducing pre-transplant immune-suppressive conditioning and eliminating post-transplant immune suppression. Clonal hematopoiesis of indeterminate potential (CHIP)-associated mutations increase and telomere lengths (TL) shorten with natural aging and DNA damaging processes. It is possible that if CHIP is present before ASCT-GT, or mutagenesis occurs due to busulfan exposure, the HSCs carrying these somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35914495"
                        open
                      >
                        <summary class="article-expander__title">Mediating and maintaining methylation while minimizing mutation: Recent advances on mammalian DNA methyltransferases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35914495/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220804212147&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mammalian genomes are methylated on carbon-5 of many cytosines, mostly in CpG dinucleotides. Methylation patterns are maintained during mitosis via DNMT1, and regulatory factors involved in processes that include histone modifications. Methylation in a sequence longer than CpG can influence the binding of sequence-specific transcription factors, thus affecting gene expression. 5-Methylcytosine deamination results in C-to-T transition. While some mutations are beneficial, most are not; so...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35914227"
                        open
                      >
                        <summary class="article-expander__title">Evaluation of Clonal Hematopoiesis in Pediatric ADA-SCID Gene Therapy Participants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35914227/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220804212147&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Autologous stem cell transplant with gene therapy (ASCT-GT) provides curative therapy while reducing pre-transplant immune-suppressive conditioning and eliminating post-transplant immune suppression. Clonal hematopoiesis of indeterminate potential (CHIP)-associated mutations increase and telomere lengths (TL) shorten with natural aging and DNA damaging processes. It is possible that if CHIP is present before ASCT-GT, or mutagenesis occurs due to busulfan exposure, the HSCs carrying these somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35914495"
                        open
                      >
                        <summary class="article-expander__title">Mediating and maintaining methylation while minimizing mutation: Recent advances on mammalian DNA methyltransferases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35914495/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220803212051&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mammalian genomes are methylated on carbon-5 of many cytosines, mostly in CpG dinucleotides. Methylation patterns are maintained during mitosis via DNMT1, and regulatory factors involved in processes that include histone modifications. Methylation in a sequence longer than CpG can influence the binding of sequence-specific transcription factors, thus affecting gene expression. 5-Methylcytosine deamination results in C-to-T transition. While some mutations are beneficial, most are not; so...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35914227"
                        open
                      >
                        <summary class="article-expander__title">Evaluation of Clonal Hematopoiesis in Pediatric ADA-SCID Gene Therapy Participants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35914227/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220803212051&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Autologous stem cell transplant with gene therapy (ASCT-GT) provides curative therapy while reducing pre-transplant immune-suppressive conditioning and eliminating post-transplant immune suppression. Clonal hematopoiesis of indeterminate potential (CHIP)-associated mutations increase and telomere lengths (TL) shorten with natural aging and DNA damaging processes. It is possible that if CHIP is present before ASCT-GT, or mutagenesis occurs due to busulfan exposure, the HSCs carrying these somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35914495"
                        open
                      >
                        <summary class="article-expander__title">Mediating and maintaining methylation while minimizing mutation: Recent advances on mammalian DNA methyltransferases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35914495/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220802212153&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mammalian genomes are methylated on carbon-5 of many cytosines, mostly in CpG dinucleotides. Methylation patterns are maintained during mitosis via DNMT1, and regulatory factors involved in processes that include histone modifications. Methylation in a sequence longer than CpG can influence the binding of sequence-specific transcription factors, thus affecting gene expression. 5-Methylcytosine deamination results in C-to-T transition. While some mutations are beneficial, most are not; so...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35914227"
                        open
                      >
                        <summary class="article-expander__title">Evaluation of Clonal Hematopoiesis in Pediatric ADA-SCID Gene Therapy Participants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35914227/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220802212153&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Autologous stem cell transplant with gene therapy (ASCT-GT) provides curative therapy while reducing pre-transplant immune-suppressive conditioning and eliminating post-transplant immune suppression. Clonal hematopoiesis of indeterminate potential (CHIP)-associated mutations increase and telomere lengths (TL) shorten with natural aging and DNA damaging processes. It is possible that if CHIP is present before ASCT-GT, or mutagenesis occurs due to busulfan exposure, the HSCs carrying these somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35914227"
                        open
                      >
                        <summary class="article-expander__title">Evaluation of Clonal Hematopoiesis in Pediatric ADA-SCID Gene Therapy Participants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35914227/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220801212447&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Autologous stem cell transplant with gene therapy (ASCT-GT) provides curative therapy while reducing pre-transplant immune-suppressive conditioning and eliminating post-transplant immune suppression. Clonal hematopoiesis of indeterminate potential (CHIP)-associated mutations increase and telomere lengths (TL) shorten with natural aging and DNA damaging processes. It is possible that if CHIP is present before ASCT-GT, or mutagenesis occurs due to busulfan exposure, the HSCs carrying these somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-07-31 datetime="2022-07-31T10:00:00.000Z">2022-07-31</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-07-31 datetime="2022-07-31T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608003622&amp;v&#x3D;2.17.7&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35908627"
                        open
                      >
                        <summary class="article-expander__title">Mitochondrial Deep Dive into Hematopoietic Stem Cell Dormancy: Not Much Glycolysis but Plenty of Sluggish Lysosomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35908627/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220815212205&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The discovery of hematopoietic stem cells (HSCs) heterogeneity have had major implications for investigations of hematopoietic stem cell disorders, clonal hematopoiesis and HSC aging. More recent studies of the heterogeneity of HSC organelles have begun to provide additional insights into the HSC behavior with far reaching ramifications for mechanistic understanding of aging of HSCs and stem cell-derived diseases. Mitochondrial heterogeneity has been explored to expose HSC subsets with distinct...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35908627"
                        open
                      >
                        <summary class="article-expander__title">Mitochondrial Deep Dive into Hematopoietic Stem Cell Dormancy: Not Much Glycolysis but Plenty of Sluggish Lysosomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35908627/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220814211915&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The discovery of hematopoietic stem cells (HSCs) heterogeneity have had major implications for investigations of hematopoietic stem cell disorders, clonal hematopoiesis and HSC aging. More recent studies of the heterogeneity of HSC organelles have begun to provide additional insights into the HSC behavior with far reaching ramifications for mechanistic understanding of aging of HSCs and stem cell-derived diseases. Mitochondrial heterogeneity has been explored to expose HSC subsets with distinct...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35908627"
                        open
                      >
                        <summary class="article-expander__title">Mitochondrial Deep Dive into Hematopoietic Stem Cell Dormancy: Not Much Glycolysis but Plenty of Sluggish Lysosomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35908627/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220813211549&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The discovery of hematopoietic stem cells (HSCs) heterogeneity have had major implications for investigations of hematopoietic stem cell disorders, clonal hematopoiesis and HSC aging. More recent studies of the heterogeneity of HSC organelles have begun to provide additional insights into the HSC behavior with far reaching ramifications for mechanistic understanding of aging of HSCs and stem cell-derived diseases. Mitochondrial heterogeneity has been explored to expose HSC subsets with distinct...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35908627"
                        open
                      >
                        <summary class="article-expander__title">Mitochondrial Deep Dive into Hematopoietic Stem Cell Dormancy: Not Much Glycolysis but Plenty of Sluggish Lysosomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35908627/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220812210706&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The discovery of hematopoietic stem cells (HSCs) heterogeneity have had major implications for investigations of hematopoietic stem cell disorders, clonal hematopoiesis and HSC aging. More recent studies of the heterogeneity of HSC organelles have begun to provide additional insights into the HSC behavior with far reaching ramifications for mechanistic understanding of aging of HSCs and stem cell-derived diseases. Mitochondrial heterogeneity has been explored to expose HSC subsets with distinct...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35908627"
                        open
                      >
                        <summary class="article-expander__title">Mitochondrial Deep Dive into Hematopoietic Stem Cell Dormancy: Not Much Glycolysis but Plenty of Sluggish Lysosomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35908627/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220811211442&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The discovery of hematopoietic stem cells (HSCs) heterogeneity have had major implications for investigations of hematopoietic stem cell disorders, clonal hematopoiesis and HSC aging. More recent studies of the heterogeneity of HSC organelles have begun to provide additional insights into the HSC behavior with far reaching ramifications for mechanistic understanding of aging of HSCs and stem cell-derived diseases. Mitochondrial heterogeneity has been explored to expose HSC subsets with distinct...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35908627"
                        open
                      >
                        <summary class="article-expander__title">Mitochondrial Deep Dive into Hematopoietic Stem Cell Dormancy: Not Much Glycolysis but Plenty of Sluggish Lysosomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35908627/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220810210900&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The discovery of hematopoietic stem cells (HSCs) heterogeneity have had major implications for investigations of hematopoietic stem cell disorders, clonal hematopoiesis and HSC aging. More recent studies of the heterogeneity of HSC organelles have begun to provide additional insights into the HSC behavior with far reaching ramifications for mechanistic understanding of aging of HSCs and stem cell-derived diseases. Mitochondrial heterogeneity has been explored to expose HSC subsets with distinct...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35908627"
                        open
                      >
                        <summary class="article-expander__title">Mitochondrial Deep Dive into Hematopoietic Stem Cell Dormancy: Not Much Glycolysis but Plenty of Sluggish Lysosomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35908627/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220809211355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The discovery of hematopoietic stem cells (HSCs) heterogeneity have had major implications for investigations of hematopoietic stem cell disorders, clonal hematopoiesis and HSC aging. More recent studies of the heterogeneity of HSC organelles have begun to provide additional insights into the HSC behavior with far reaching ramifications for mechanistic understanding of aging of HSCs and stem cell-derived diseases. Mitochondrial heterogeneity has been explored to expose HSC subsets with distinct...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35908627"
                        open
                      >
                        <summary class="article-expander__title">Mitochondrial Deep Dive into Hematopoietic Stem Cell Dormancy: Not Much Glycolysis but Plenty of Sluggish Lysosomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35908627/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220808212152&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The discovery of hematopoietic stem cells (HSCs) heterogeneity have had major implications for investigations of hematopoietic stem cell disorders, clonal hematopoiesis and HSC aging. More recent studies of the heterogeneity of HSC organelles have begun to provide additional insights into the HSC behavior with far reaching ramifications for mechanistic understanding of aging of HSCs and stem cell-derived diseases. Mitochondrial heterogeneity has been explored to expose HSC subsets with distinct...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35908627"
                        open
                      >
                        <summary class="article-expander__title">Mitochondrial Deep Dive into Hematopoietic Stem Cell Dormancy: Not Much Glycolysis but Plenty of Sluggish Lysosomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35908627/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220807211336&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The discovery of hematopoietic stem cells (HSCs) heterogeneity have had major implications for investigations of hematopoietic stem cell disorders, clonal hematopoiesis and HSC aging. More recent studies of the heterogeneity of HSC organelles have begun to provide additional insights into the HSC behavior with far reaching ramifications for mechanistic understanding of aging of HSCs and stem cell-derived diseases. Mitochondrial heterogeneity has been explored to expose HSC subsets with distinct...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35908627"
                        open
                      >
                        <summary class="article-expander__title">Mitochondrial Deep Dive into Hematopoietic Stem Cell Dormancy: Not Much Glycolysis but Plenty of Sluggish Lysosomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35908627/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220806211830&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The discovery of hematopoietic stem cells (HSCs) heterogeneity have had major implications for investigations of hematopoietic stem cell disorders, clonal hematopoiesis and HSC aging. More recent studies of the heterogeneity of HSC organelles have begun to provide additional insights into the HSC behavior with far reaching ramifications for mechanistic understanding of aging of HSCs and stem cell-derived diseases. Mitochondrial heterogeneity has been explored to expose HSC subsets with distinct...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35908627"
                        open
                      >
                        <summary class="article-expander__title">Mitochondrial Deep Dive into Hematopoietic Stem Cell Dormancy: Not Much Glycolysis but Plenty of Sluggish Lysosomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35908627/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220805210849&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The discovery of hematopoietic stem cells (HSCs) heterogeneity have had major implications for investigations of hematopoietic stem cell disorders, clonal hematopoiesis and HSC aging. More recent studies of the heterogeneity of HSC organelles have begun to provide additional insights into the HSC behavior with far reaching ramifications for mechanistic understanding of aging of HSCs and stem cell-derived diseases. Mitochondrial heterogeneity has been explored to expose HSC subsets with distinct...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35908627"
                        open
                      >
                        <summary class="article-expander__title">Mitochondrial Deep Dive into Hematopoietic Stem Cell Dormancy: Not Much Glycolysis but Plenty of Sluggish Lysosomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35908627/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220804212147&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The discovery of hematopoietic stem cells (HSCs) heterogeneity have had major implications for investigations of hematopoietic stem cell disorders, clonal hematopoiesis and HSC aging. More recent studies of the heterogeneity of HSC organelles have begun to provide additional insights into the HSC behavior with far reaching ramifications for mechanistic understanding of aging of HSCs and stem cell-derived diseases. Mitochondrial heterogeneity has been explored to expose HSC subsets with distinct...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35908627"
                        open
                      >
                        <summary class="article-expander__title">Mitochondrial Deep Dive into Hematopoietic Stem Cell Dormancy: Not Much Glycolysis but Plenty of Sluggish Lysosomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35908627/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220803212051&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The discovery of hematopoietic stem cells (HSCs) heterogeneity have had major implications for investigations of hematopoietic stem cell disorders, clonal hematopoiesis and HSC aging. More recent studies of the heterogeneity of HSC organelles have begun to provide additional insights into the HSC behavior with far reaching ramifications for mechanistic understanding of aging of HSCs and stem cell-derived diseases. Mitochondrial heterogeneity has been explored to expose HSC subsets with distinct...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35908627"
                        open
                      >
                        <summary class="article-expander__title">Mitochondrial Deep Dive into Hematopoietic Stem Cell Dormancy: Not Much Glycolysis but Plenty of Sluggish Lysosomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35908627/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220802212153&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The discovery of hematopoietic stem cells (HSCs) heterogeneity have had major implications for investigations of hematopoietic stem cell disorders, clonal hematopoiesis and HSC aging. More recent studies of the heterogeneity of HSC organelles have begun to provide additional insights into the HSC behavior with far reaching ramifications for mechanistic understanding of aging of HSCs and stem cell-derived diseases. Mitochondrial heterogeneity has been explored to expose HSC subsets with distinct...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35908627"
                        open
                      >
                        <summary class="article-expander__title">Mitochondrial Deep Dive into Hematopoietic Stem Cell Dormancy: Not Much Glycolysis but Plenty of Sluggish Lysosomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35908627/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220801212447&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The discovery of hematopoietic stem cells (HSCs) heterogeneity have had major implications for investigations of hematopoietic stem cell disorders, clonal hematopoiesis and HSC aging. More recent studies of the heterogeneity of HSC organelles have begun to provide additional insights into the HSC behavior with far reaching ramifications for mechanistic understanding of aging of HSCs and stem cell-derived diseases. Mitochondrial heterogeneity has been explored to expose HSC subsets with distinct...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-07-28 datetime="2022-07-28T10:00:00.000Z">2022-07-28</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-07-28 datetime="2022-07-28T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608003622&amp;v&#x3D;2.17.7&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35899391"
                        open
                      >
                        <summary class="article-expander__title">Acute myeloid leukemia early after anti-CD19 chimeric antigen receptor T cells infusion in a patient with refractory diffuse large B cell lymphoma and pre-existing clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35899391/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220815212205&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Not available.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35899391"
                        open
                      >
                        <summary class="article-expander__title">Acute myeloid leukemia early after anti-CD19 chimeric antigen receptor T cells infusion in a patient with refractory diffuse large B cell lymphoma and pre-existing clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35899391/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220814211915&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Not available.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35899391"
                        open
                      >
                        <summary class="article-expander__title">Acute myeloid leukemia early after anti-CD19 chimeric antigen receptor T cells infusion in a patient with refractory diffuse large B cell lymphoma and pre-existing clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35899391/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220813211549&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Not available.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35899391"
                        open
                      >
                        <summary class="article-expander__title">Acute myeloid leukemia early after anti-CD19 chimeric antigen receptor T cells infusion in a patient with refractory diffuse large B cell lymphoma and pre-existing clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35899391/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220812210706&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Not available.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35899391"
                        open
                      >
                        <summary class="article-expander__title">Acute myeloid leukemia early after anti-CD19 chimeric antigen receptor T cells infusion in a patient with refractory diffuse large B cell lymphoma and pre-existing clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35899391/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220811211442&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Not available.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35899391"
                        open
                      >
                        <summary class="article-expander__title">Acute myeloid leukemia early after anti-CD19 chimeric antigen receptor T cells infusion in a patient with refractory diffuse large B cell lymphoma and pre-existing clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35899391/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220810210900&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Not available.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35899391"
                        open
                      >
                        <summary class="article-expander__title">Acute myeloid leukemia early after anti-CD19 chimeric antigen receptor T cells infusion in a patient with refractory diffuse large B cell lymphoma and pre-existing clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35899391/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220809211355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Not available.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35899391"
                        open
                      >
                        <summary class="article-expander__title">Acute myeloid leukemia early after anti-CD19 chimeric antigen receptor T cells infusion in a patient with refractory diffuse large B cell lymphoma and pre-existing clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35899391/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220808212152&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Not available.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35899391"
                        open
                      >
                        <summary class="article-expander__title">Acute myeloid leukemia early after anti-CD19 chimeric antigen receptor T cells infusion in a patient with refractory diffuse large B cell lymphoma and pre-existing clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35899391/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220807211336&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Not available.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35899391"
                        open
                      >
                        <summary class="article-expander__title">Acute myeloid leukemia early after anti-CD19 chimeric antigen receptor T cells infusion in a patient with refractory diffuse large B cell lymphoma and pre-existing clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35899391/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220806211830&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Not available.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35899391"
                        open
                      >
                        <summary class="article-expander__title">Acute myeloid leukemia early after anti-CD19 chimeric antigen receptor T cells infusion in a patient with refractory diffuse large B cell lymphoma and pre-existing clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35899391/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220805210849&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Not available.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35899391"
                        open
                      >
                        <summary class="article-expander__title">Acute myeloid leukemia early after anti-CD19 chimeric antigen receptor T cells infusion in a patient with refractory diffuse large B cell lymphoma and pre-existing clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35899391/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220804212147&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Not available.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35899391"
                        open
                      >
                        <summary class="article-expander__title">Acute myeloid leukemia early after anti-CD19 chimeric antigen receptor T cells infusion in a patient with refractory diffuse large B cell lymphoma and pre-existing clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35899391/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220803212051&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Not available.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35899391"
                        open
                      >
                        <summary class="article-expander__title">Acute myeloid leukemia early after anti-CD19 chimeric antigen receptor T cells infusion in a patient with refractory diffuse large B cell lymphoma and pre-existing clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35899391/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220802212153&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Not available.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35899391"
                        open
                      >
                        <summary class="article-expander__title">Acute myeloid leukemia early after anti-CD19 chimeric antigen receptor T cells infusion in a patient with refractory diffuse large B cell lymphoma and pre-existing clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35899391/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220801212447&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Not available.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35899391"
                        open
                      >
                        <summary class="article-expander__title">Acute myeloid leukemia early after anti-CD19 chimeric antigen receptor T cells infusion in a patient with refractory diffuse large B cell lymphoma and pre-existing clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35899391/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220731212525&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Not available.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35899391"
                        open
                      >
                        <summary class="article-expander__title">Acute myeloid leukemia early after anti-CD19 chimeric antigen receptor T cells infusion in a patient with refractory diffuse large B cell lymphoma and pre-existing clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35899391/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220730212309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Not available.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35899391"
                        open
                      >
                        <summary class="article-expander__title">Acute myeloid leukemia early after anti-CD19 chimeric antigen receptor T cells infusion in a patient with refractory diffuse large B cell lymphoma and pre-existing clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35899391/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220729211332&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Not available.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35899391"
                        open
                      >
                        <summary class="article-expander__title">Acute myeloid leukemia early after anti-CD19 chimeric antigen receptor T cells infusion in a patient with refractory diffuse large B cell lymphoma and pre-existing clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35899391/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220728211733&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Not available.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608010801&amp;v&#x3D;2.17.7&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35899825"
                        open
                      >
                        <summary class="article-expander__title">Targeted parallel DNA sequencing detects circulating tumor-associated variants of the mitochondrial and nuclear genomes in patients with neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35899825/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220824212808&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In this limited cohort of NB patients clinically informative tumor-associated mt and nuclear circulating variants were detected by targeted parallel sequencing in a minority of patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35899825"
                        open
                      >
                        <summary class="article-expander__title">Targeted parallel DNA sequencing detects circulating tumor-associated variants of the mitochondrial and nuclear genomes in patients with neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35899825/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220823212820&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In this limited cohort of NB patients clinically informative tumor-associated mt and nuclear circulating variants were detected by targeted parallel sequencing in a minority of patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35899825"
                        open
                      >
                        <summary class="article-expander__title">Targeted parallel DNA sequencing detects circulating tumor-associated variants of the mitochondrial and nuclear genomes in patients with neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35899825/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220822212717&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In this limited cohort of NB patients clinically informative tumor-associated mt and nuclear circulating variants were detected by targeted parallel sequencing in a minority of patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35899825"
                        open
                      >
                        <summary class="article-expander__title">Targeted parallel DNA sequencing detects circulating tumor-associated variants of the mitochondrial and nuclear genomes in patients with neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35899825/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220821211551&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In this limited cohort of NB patients clinically informative tumor-associated mt and nuclear circulating variants were detected by targeted parallel sequencing in a minority of patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35899825"
                        open
                      >
                        <summary class="article-expander__title">Targeted parallel DNA sequencing detects circulating tumor-associated variants of the mitochondrial and nuclear genomes in patients with neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35899825/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220820211309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In this limited cohort of NB patients clinically informative tumor-associated mt and nuclear circulating variants were detected by targeted parallel sequencing in a minority of patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35899825"
                        open
                      >
                        <summary class="article-expander__title">Targeted parallel DNA sequencing detects circulating tumor-associated variants of the mitochondrial and nuclear genomes in patients with neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35899825/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220819211458&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In this limited cohort of NB patients clinically informative tumor-associated mt and nuclear circulating variants were detected by targeted parallel sequencing in a minority of patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35899825"
                        open
                      >
                        <summary class="article-expander__title">Targeted parallel DNA sequencing detects circulating tumor-associated variants of the mitochondrial and nuclear genomes in patients with neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35899825/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220818211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In this limited cohort of NB patients clinically informative tumor-associated mt and nuclear circulating variants were detected by targeted parallel sequencing in a minority of patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35899825"
                        open
                      >
                        <summary class="article-expander__title">Targeted parallel DNA sequencing detects circulating tumor-associated variants of the mitochondrial and nuclear genomes in patients with neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35899825/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220817212606&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In this limited cohort of NB patients clinically informative tumor-associated mt and nuclear circulating variants were detected by targeted parallel sequencing in a minority of patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35899825"
                        open
                      >
                        <summary class="article-expander__title">Targeted parallel DNA sequencing detects circulating tumor-associated variants of the mitochondrial and nuclear genomes in patients with neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35899825/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220816212355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In this limited cohort of NB patients clinically informative tumor-associated mt and nuclear circulating variants were detected by targeted parallel sequencing in a minority of patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35899825"
                        open
                      >
                        <summary class="article-expander__title">Targeted parallel DNA sequencing detects circulating tumor-associated variants of the mitochondrial and nuclear genomes in patients with neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35899825/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220815212205&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In this limited cohort of NB patients clinically informative tumor-associated mt and nuclear circulating variants were detected by targeted parallel sequencing in a minority of patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35899825"
                        open
                      >
                        <summary class="article-expander__title">Targeted parallel DNA sequencing detects circulating tumor-associated variants of the mitochondrial and nuclear genomes in patients with neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35899825/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220814211915&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In this limited cohort of NB patients clinically informative tumor-associated mt and nuclear circulating variants were detected by targeted parallel sequencing in a minority of patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35899825"
                        open
                      >
                        <summary class="article-expander__title">Targeted parallel DNA sequencing detects circulating tumor-associated variants of the mitochondrial and nuclear genomes in patients with neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35899825/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220813211549&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In this limited cohort of NB patients clinically informative tumor-associated mt and nuclear circulating variants were detected by targeted parallel sequencing in a minority of patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35899825"
                        open
                      >
                        <summary class="article-expander__title">Targeted parallel DNA sequencing detects circulating tumor-associated variants of the mitochondrial and nuclear genomes in patients with neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35899825/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220812210706&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In this limited cohort of NB patients clinically informative tumor-associated mt and nuclear circulating variants were detected by targeted parallel sequencing in a minority of patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35899825"
                        open
                      >
                        <summary class="article-expander__title">Targeted parallel DNA sequencing detects circulating tumor-associated variants of the mitochondrial and nuclear genomes in patients with neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35899825/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220811211442&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In this limited cohort of NB patients clinically informative tumor-associated mt and nuclear circulating variants were detected by targeted parallel sequencing in a minority of patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35899825"
                        open
                      >
                        <summary class="article-expander__title">Targeted parallel DNA sequencing detects circulating tumor-associated variants of the mitochondrial and nuclear genomes in patients with neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35899825/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220810210900&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In this limited cohort of NB patients clinically informative tumor-associated mt and nuclear circulating variants were detected by targeted parallel sequencing in a minority of patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35899825"
                        open
                      >
                        <summary class="article-expander__title">Targeted parallel DNA sequencing detects circulating tumor-associated variants of the mitochondrial and nuclear genomes in patients with neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35899825/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220809211355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In this limited cohort of NB patients clinically informative tumor-associated mt and nuclear circulating variants were detected by targeted parallel sequencing in a minority of patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35899825"
                        open
                      >
                        <summary class="article-expander__title">Targeted parallel DNA sequencing detects circulating tumor-associated variants of the mitochondrial and nuclear genomes in patients with neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35899825/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220808212152&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In this limited cohort of NB patients clinically informative tumor-associated mt and nuclear circulating variants were detected by targeted parallel sequencing in a minority of patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35899825"
                        open
                      >
                        <summary class="article-expander__title">Targeted parallel DNA sequencing detects circulating tumor-associated variants of the mitochondrial and nuclear genomes in patients with neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35899825/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220807211336&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In this limited cohort of NB patients clinically informative tumor-associated mt and nuclear circulating variants were detected by targeted parallel sequencing in a minority of patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35899825"
                        open
                      >
                        <summary class="article-expander__title">Targeted parallel DNA sequencing detects circulating tumor-associated variants of the mitochondrial and nuclear genomes in patients with neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35899825/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220806211830&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In this limited cohort of NB patients clinically informative tumor-associated mt and nuclear circulating variants were detected by targeted parallel sequencing in a minority of patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35899825"
                        open
                      >
                        <summary class="article-expander__title">Targeted parallel DNA sequencing detects circulating tumor-associated variants of the mitochondrial and nuclear genomes in patients with neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35899825/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220805210849&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In this limited cohort of NB patients clinically informative tumor-associated mt and nuclear circulating variants were detected by targeted parallel sequencing in a minority of patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35899825"
                        open
                      >
                        <summary class="article-expander__title">Targeted parallel DNA sequencing detects circulating tumor-associated variants of the mitochondrial and nuclear genomes in patients with neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35899825/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220804212147&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In this limited cohort of NB patients clinically informative tumor-associated mt and nuclear circulating variants were detected by targeted parallel sequencing in a minority of patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35899825"
                        open
                      >
                        <summary class="article-expander__title">Targeted parallel DNA sequencing detects circulating tumor-associated variants of the mitochondrial and nuclear genomes in patients with neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35899825/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220803212051&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In this limited cohort of NB patients clinically informative tumor-associated mt and nuclear circulating variants were detected by targeted parallel sequencing in a minority of patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35899825"
                        open
                      >
                        <summary class="article-expander__title">Targeted parallel DNA sequencing detects circulating tumor-associated variants of the mitochondrial and nuclear genomes in patients with neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35899825/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220802212153&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In this limited cohort of NB patients clinically informative tumor-associated mt and nuclear circulating variants were detected by targeted parallel sequencing in a minority of patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35899825"
                        open
                      >
                        <summary class="article-expander__title">Targeted parallel DNA sequencing detects circulating tumor-associated variants of the mitochondrial and nuclear genomes in patients with neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35899825/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220801212447&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In this limited cohort of NB patients clinically informative tumor-associated mt and nuclear circulating variants were detected by targeted parallel sequencing in a minority of patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35899825"
                        open
                      >
                        <summary class="article-expander__title">Targeted parallel DNA sequencing detects circulating tumor-associated variants of the mitochondrial and nuclear genomes in patients with neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35899825/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220731212525&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In this limited cohort of NB patients clinically informative tumor-associated mt and nuclear circulating variants were detected by targeted parallel sequencing in a minority of patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35899825"
                        open
                      >
                        <summary class="article-expander__title">Targeted parallel DNA sequencing detects circulating tumor-associated variants of the mitochondrial and nuclear genomes in patients with neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35899825/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220730212309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In this limited cohort of NB patients clinically informative tumor-associated mt and nuclear circulating variants were detected by targeted parallel sequencing in a minority of patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35899825"
                        open
                      >
                        <summary class="article-expander__title">Targeted parallel DNA sequencing detects circulating tumor-associated variants of the mitochondrial and nuclear genomes in patients with neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35899825/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220729211332&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In this limited cohort of NB patients clinically informative tumor-associated mt and nuclear circulating variants were detected by targeted parallel sequencing in a minority of patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35899825"
                        open
                      >
                        <summary class="article-expander__title">Targeted parallel DNA sequencing detects circulating tumor-associated variants of the mitochondrial and nuclear genomes in patients with neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35899825/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220728211733&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In this limited cohort of NB patients clinically informative tumor-associated mt and nuclear circulating variants were detected by targeted parallel sequencing in a minority of patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-07-27 datetime="2022-07-27T10:00:00.000Z">2022-07-27</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-07-27 datetime="2022-07-27T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608003622&amp;v&#x3D;2.17.7&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35896010"
                        open
                      >
                        <summary class="article-expander__title">CAR T-cell Therapy Meets Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35896010/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220814211915&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is common in patients with hematologic malignancies. Recent publications provide evidence that CHIP may affect chimeric antigen receptor T-cell therapy efficacy and that the incidence of treatment-related toxicities such as cytokine release syndrome and immune effector-cell associated neurotoxicity syndrome may be affected. See related article by Saini et al. (8).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35896010"
                        open
                      >
                        <summary class="article-expander__title">CAR T-cell Therapy Meets Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35896010/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220813211549&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is common in patients with hematologic malignancies. Recent publications provide evidence that CHIP may affect chimeric antigen receptor T-cell therapy efficacy and that the incidence of treatment-related toxicities such as cytokine release syndrome and immune effector-cell associated neurotoxicity syndrome may be affected. See related article by Saini et al. (8).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35896010"
                        open
                      >
                        <summary class="article-expander__title">CAR T-cell Therapy Meets Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35896010/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220812210706&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is common in patients with hematologic malignancies. Recent publications provide evidence that CHIP may affect chimeric antigen receptor T-cell therapy efficacy and that the incidence of treatment-related toxicities such as cytokine release syndrome and immune effector-cell associated neurotoxicity syndrome may be affected. See related article by Saini et al. (8).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35896010"
                        open
                      >
                        <summary class="article-expander__title">CAR T-cell Therapy Meets Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35896010/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220811211442&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is common in patients with hematologic malignancies. Recent publications provide evidence that CHIP may affect chimeric antigen receptor T-cell therapy efficacy and that the incidence of treatment-related toxicities such as cytokine release syndrome and immune effector-cell associated neurotoxicity syndrome may be affected. See related article by Saini et al. (8).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885563"
                        open
                      >
                        <summary class="article-expander__title">Diagnosis of Myelodysplastic Syndromes: From Immunological Observations to Clinical Applications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885563/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220811211442&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) constitute a very heterogeneous group of diseases with a high prevalence in elderly patients and a propensity for progression to acute myeloid leukemia. The complexity of these hematopoietic malignancies is revealed by the multiple recurrent somatic mutations involved in MDS pathogenesis and the paradoxical common phenotype observed in these patients characterized by ineffective hematopoiesis and cytopenia. In the context of population aging, the incidence of MDS...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885544"
                        open
                      >
                        <summary class="article-expander__title">The Mesenchymal Niche in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885544/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220811211442&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are clonal disorders characterized by ineffective hematopoiesis, resulting in cytopenias and a risk of developing acute myeloid leukemia. In addition to mutations affecting hematopoietic stem cells (HSCs), numerous studies have highlighted the role of the bone marrow microenvironment (BMME) in the development of MDSs. The mesenchymal niche represents a key component of the BMME. In this review, we discuss the role of the mesenchymal niche in the pathophysiology...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35896010"
                        open
                      >
                        <summary class="article-expander__title">CAR T-cell Therapy Meets Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35896010/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220810210900&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is common in patients with hematologic malignancies. Recent publications provide evidence that CHIP may affect chimeric antigen receptor T-cell therapy efficacy and that the incidence of treatment-related toxicities such as cytokine release syndrome and immune effector-cell associated neurotoxicity syndrome may be affected. See related article by Saini et al. (8).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885563"
                        open
                      >
                        <summary class="article-expander__title">Diagnosis of Myelodysplastic Syndromes: From Immunological Observations to Clinical Applications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885563/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220810210900&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) constitute a very heterogeneous group of diseases with a high prevalence in elderly patients and a propensity for progression to acute myeloid leukemia. The complexity of these hematopoietic malignancies is revealed by the multiple recurrent somatic mutations involved in MDS pathogenesis and the paradoxical common phenotype observed in these patients characterized by ineffective hematopoiesis and cytopenia. In the context of population aging, the incidence of MDS...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885544"
                        open
                      >
                        <summary class="article-expander__title">The Mesenchymal Niche in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885544/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220810210900&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are clonal disorders characterized by ineffective hematopoiesis, resulting in cytopenias and a risk of developing acute myeloid leukemia. In addition to mutations affecting hematopoietic stem cells (HSCs), numerous studies have highlighted the role of the bone marrow microenvironment (BMME) in the development of MDSs. The mesenchymal niche represents a key component of the BMME. In this review, we discuss the role of the mesenchymal niche in the pathophysiology...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35896010"
                        open
                      >
                        <summary class="article-expander__title">CAR T-cell Therapy Meets Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35896010/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220809211355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is common in patients with hematologic malignancies. Recent publications provide evidence that CHIP may affect chimeric antigen receptor T-cell therapy efficacy and that the incidence of treatment-related toxicities such as cytokine release syndrome and immune effector-cell associated neurotoxicity syndrome may be affected. See related article by Saini et al. (8).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885563"
                        open
                      >
                        <summary class="article-expander__title">Diagnosis of Myelodysplastic Syndromes: From Immunological Observations to Clinical Applications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885563/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220809211355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) constitute a very heterogeneous group of diseases with a high prevalence in elderly patients and a propensity for progression to acute myeloid leukemia. The complexity of these hematopoietic malignancies is revealed by the multiple recurrent somatic mutations involved in MDS pathogenesis and the paradoxical common phenotype observed in these patients characterized by ineffective hematopoiesis and cytopenia. In the context of population aging, the incidence of MDS...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885544"
                        open
                      >
                        <summary class="article-expander__title">The Mesenchymal Niche in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885544/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220809211355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are clonal disorders characterized by ineffective hematopoiesis, resulting in cytopenias and a risk of developing acute myeloid leukemia. In addition to mutations affecting hematopoietic stem cells (HSCs), numerous studies have highlighted the role of the bone marrow microenvironment (BMME) in the development of MDSs. The mesenchymal niche represents a key component of the BMME. In this review, we discuss the role of the mesenchymal niche in the pathophysiology...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885518"
                        open
                      >
                        <summary class="article-expander__title">Clinical Significance of Clonal Hematopoiesis of Indeterminate Potential in Hematology and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885518/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220809211355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid profiling uses circulating tumor DNA (ctDNA) for minimal invasive tumor mutational profiling from peripheral blood. The presence of somatic mutations in peripheral blood cells without further evidence of a hematologic neoplasm defines clonal hematopoiesis of indeterminate potential (CHIP). CHIP-mutations can be found in the cell-free DNA (cfDNA) of plasma, are a potential cause of false positive results in liquid profiling, and thus limit its usage in screening settings. Various...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35896010"
                        open
                      >
                        <summary class="article-expander__title">CAR T-cell Therapy Meets Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35896010/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220808212152&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is common in patients with hematologic malignancies. Recent publications provide evidence that CHIP may affect chimeric antigen receptor T-cell therapy efficacy and that the incidence of treatment-related toxicities such as cytokine release syndrome and immune effector-cell associated neurotoxicity syndrome may be affected. See related article by Saini et al. (8).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885563"
                        open
                      >
                        <summary class="article-expander__title">Diagnosis of Myelodysplastic Syndromes: From Immunological Observations to Clinical Applications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885563/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220808212152&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) constitute a very heterogeneous group of diseases with a high prevalence in elderly patients and a propensity for progression to acute myeloid leukemia. The complexity of these hematopoietic malignancies is revealed by the multiple recurrent somatic mutations involved in MDS pathogenesis and the paradoxical common phenotype observed in these patients characterized by ineffective hematopoiesis and cytopenia. In the context of population aging, the incidence of MDS...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885544"
                        open
                      >
                        <summary class="article-expander__title">The Mesenchymal Niche in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885544/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220808212152&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are clonal disorders characterized by ineffective hematopoiesis, resulting in cytopenias and a risk of developing acute myeloid leukemia. In addition to mutations affecting hematopoietic stem cells (HSCs), numerous studies have highlighted the role of the bone marrow microenvironment (BMME) in the development of MDSs. The mesenchymal niche represents a key component of the BMME. In this review, we discuss the role of the mesenchymal niche in the pathophysiology...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885518"
                        open
                      >
                        <summary class="article-expander__title">Clinical Significance of Clonal Hematopoiesis of Indeterminate Potential in Hematology and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885518/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220808212152&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid profiling uses circulating tumor DNA (ctDNA) for minimal invasive tumor mutational profiling from peripheral blood. The presence of somatic mutations in peripheral blood cells without further evidence of a hematologic neoplasm defines clonal hematopoiesis of indeterminate potential (CHIP). CHIP-mutations can be found in the cell-free DNA (cfDNA) of plasma, are a potential cause of false positive results in liquid profiling, and thus limit its usage in screening settings. Various...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885462"
                        open
                      >
                        <summary class="article-expander__title">Automated Detection of Dysplasia: Data Mining from Our Hematology Analyzers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885462/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220808212152&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are clonal hematopoietic diseases of the elderly, characterized by chronic cytopenia, ineffective and dysplastic hematopoiesis, recurrent genetic abnormalities and increased risk of progression to acute myeloid leukemia. Diagnosis on a complete blood count (CBC) can be challenging due to numerous other non-neoplastic causes of cytopenias. New generations of hematology analyzers provide cell population data (CPD) that can be exploited to reliably detect MDSs from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35896010"
                        open
                      >
                        <summary class="article-expander__title">CAR T-cell Therapy Meets Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35896010/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220807211336&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is common in patients with hematologic malignancies. Recent publications provide evidence that CHIP may affect chimeric antigen receptor T-cell therapy efficacy and that the incidence of treatment-related toxicities such as cytokine release syndrome and immune effector-cell associated neurotoxicity syndrome may be affected. See related article by Saini et al. (8).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885563"
                        open
                      >
                        <summary class="article-expander__title">Diagnosis of Myelodysplastic Syndromes: From Immunological Observations to Clinical Applications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885563/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220807211336&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) constitute a very heterogeneous group of diseases with a high prevalence in elderly patients and a propensity for progression to acute myeloid leukemia. The complexity of these hematopoietic malignancies is revealed by the multiple recurrent somatic mutations involved in MDS pathogenesis and the paradoxical common phenotype observed in these patients characterized by ineffective hematopoiesis and cytopenia. In the context of population aging, the incidence of MDS...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885544"
                        open
                      >
                        <summary class="article-expander__title">The Mesenchymal Niche in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885544/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220807211336&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are clonal disorders characterized by ineffective hematopoiesis, resulting in cytopenias and a risk of developing acute myeloid leukemia. In addition to mutations affecting hematopoietic stem cells (HSCs), numerous studies have highlighted the role of the bone marrow microenvironment (BMME) in the development of MDSs. The mesenchymal niche represents a key component of the BMME. In this review, we discuss the role of the mesenchymal niche in the pathophysiology...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885518"
                        open
                      >
                        <summary class="article-expander__title">Clinical Significance of Clonal Hematopoiesis of Indeterminate Potential in Hematology and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885518/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220807211336&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid profiling uses circulating tumor DNA (ctDNA) for minimal invasive tumor mutational profiling from peripheral blood. The presence of somatic mutations in peripheral blood cells without further evidence of a hematologic neoplasm defines clonal hematopoiesis of indeterminate potential (CHIP). CHIP-mutations can be found in the cell-free DNA (cfDNA) of plasma, are a potential cause of false positive results in liquid profiling, and thus limit its usage in screening settings. Various...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885462"
                        open
                      >
                        <summary class="article-expander__title">Automated Detection of Dysplasia: Data Mining from Our Hematology Analyzers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885462/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220807211336&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are clonal hematopoietic diseases of the elderly, characterized by chronic cytopenia, ineffective and dysplastic hematopoiesis, recurrent genetic abnormalities and increased risk of progression to acute myeloid leukemia. Diagnosis on a complete blood count (CBC) can be challenging due to numerous other non-neoplastic causes of cytopenias. New generations of hematology analyzers provide cell population data (CPD) that can be exploited to reliably detect MDSs from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35896010"
                        open
                      >
                        <summary class="article-expander__title">CAR T-cell Therapy Meets Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35896010/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220806211830&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is common in patients with hematologic malignancies. Recent publications provide evidence that CHIP may affect chimeric antigen receptor T-cell therapy efficacy and that the incidence of treatment-related toxicities such as cytokine release syndrome and immune effector-cell associated neurotoxicity syndrome may be affected. See related article by Saini et al. (8).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885563"
                        open
                      >
                        <summary class="article-expander__title">Diagnosis of Myelodysplastic Syndromes: From Immunological Observations to Clinical Applications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885563/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220806211830&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) constitute a very heterogeneous group of diseases with a high prevalence in elderly patients and a propensity for progression to acute myeloid leukemia. The complexity of these hematopoietic malignancies is revealed by the multiple recurrent somatic mutations involved in MDS pathogenesis and the paradoxical common phenotype observed in these patients characterized by ineffective hematopoiesis and cytopenia. In the context of population aging, the incidence of MDS...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885544"
                        open
                      >
                        <summary class="article-expander__title">The Mesenchymal Niche in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885544/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220806211830&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are clonal disorders characterized by ineffective hematopoiesis, resulting in cytopenias and a risk of developing acute myeloid leukemia. In addition to mutations affecting hematopoietic stem cells (HSCs), numerous studies have highlighted the role of the bone marrow microenvironment (BMME) in the development of MDSs. The mesenchymal niche represents a key component of the BMME. In this review, we discuss the role of the mesenchymal niche in the pathophysiology...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885518"
                        open
                      >
                        <summary class="article-expander__title">Clinical Significance of Clonal Hematopoiesis of Indeterminate Potential in Hematology and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885518/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220806211830&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid profiling uses circulating tumor DNA (ctDNA) for minimal invasive tumor mutational profiling from peripheral blood. The presence of somatic mutations in peripheral blood cells without further evidence of a hematologic neoplasm defines clonal hematopoiesis of indeterminate potential (CHIP). CHIP-mutations can be found in the cell-free DNA (cfDNA) of plasma, are a potential cause of false positive results in liquid profiling, and thus limit its usage in screening settings. Various...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885462"
                        open
                      >
                        <summary class="article-expander__title">Automated Detection of Dysplasia: Data Mining from Our Hematology Analyzers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885462/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220806211830&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are clonal hematopoietic diseases of the elderly, characterized by chronic cytopenia, ineffective and dysplastic hematopoiesis, recurrent genetic abnormalities and increased risk of progression to acute myeloid leukemia. Diagnosis on a complete blood count (CBC) can be challenging due to numerous other non-neoplastic causes of cytopenias. New generations of hematology analyzers provide cell population data (CPD) that can be exploited to reliably detect MDSs from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35896010"
                        open
                      >
                        <summary class="article-expander__title">CAR T-cell Therapy Meets Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35896010/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220805210849&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is common in patients with hematologic malignancies. Recent publications provide evidence that CHIP may affect chimeric antigen receptor T-cell therapy efficacy and that the incidence of treatment-related toxicities such as cytokine release syndrome and immune effector-cell associated neurotoxicity syndrome may be affected. See related article by Saini et al. (8).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885563"
                        open
                      >
                        <summary class="article-expander__title">Diagnosis of Myelodysplastic Syndromes: From Immunological Observations to Clinical Applications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885563/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220805210849&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) constitute a very heterogeneous group of diseases with a high prevalence in elderly patients and a propensity for progression to acute myeloid leukemia. The complexity of these hematopoietic malignancies is revealed by the multiple recurrent somatic mutations involved in MDS pathogenesis and the paradoxical common phenotype observed in these patients characterized by ineffective hematopoiesis and cytopenia. In the context of population aging, the incidence of MDS...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885544"
                        open
                      >
                        <summary class="article-expander__title">The Mesenchymal Niche in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885544/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220805210849&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are clonal disorders characterized by ineffective hematopoiesis, resulting in cytopenias and a risk of developing acute myeloid leukemia. In addition to mutations affecting hematopoietic stem cells (HSCs), numerous studies have highlighted the role of the bone marrow microenvironment (BMME) in the development of MDSs. The mesenchymal niche represents a key component of the BMME. In this review, we discuss the role of the mesenchymal niche in the pathophysiology...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885518"
                        open
                      >
                        <summary class="article-expander__title">Clinical Significance of Clonal Hematopoiesis of Indeterminate Potential in Hematology and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885518/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220805210849&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid profiling uses circulating tumor DNA (ctDNA) for minimal invasive tumor mutational profiling from peripheral blood. The presence of somatic mutations in peripheral blood cells without further evidence of a hematologic neoplasm defines clonal hematopoiesis of indeterminate potential (CHIP). CHIP-mutations can be found in the cell-free DNA (cfDNA) of plasma, are a potential cause of false positive results in liquid profiling, and thus limit its usage in screening settings. Various...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885462"
                        open
                      >
                        <summary class="article-expander__title">Automated Detection of Dysplasia: Data Mining from Our Hematology Analyzers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885462/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220805210849&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are clonal hematopoietic diseases of the elderly, characterized by chronic cytopenia, ineffective and dysplastic hematopoiesis, recurrent genetic abnormalities and increased risk of progression to acute myeloid leukemia. Diagnosis on a complete blood count (CBC) can be challenging due to numerous other non-neoplastic causes of cytopenias. New generations of hematology analyzers provide cell population data (CPD) that can be exploited to reliably detect MDSs from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35896010"
                        open
                      >
                        <summary class="article-expander__title">CAR T-cell Therapy Meets Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35896010/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220804212147&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is common in patients with hematologic malignancies. Recent publications provide evidence that CHIP may affect chimeric antigen receptor T-cell therapy efficacy and that the incidence of treatment-related toxicities such as cytokine release syndrome and immune effector-cell associated neurotoxicity syndrome may be affected. See related article by Saini et al. (8).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885563"
                        open
                      >
                        <summary class="article-expander__title">Diagnosis of Myelodysplastic Syndromes: From Immunological Observations to Clinical Applications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885563/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220804212147&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) constitute a very heterogeneous group of diseases with a high prevalence in elderly patients and a propensity for progression to acute myeloid leukemia. The complexity of these hematopoietic malignancies is revealed by the multiple recurrent somatic mutations involved in MDS pathogenesis and the paradoxical common phenotype observed in these patients characterized by ineffective hematopoiesis and cytopenia. In the context of population aging, the incidence of MDS...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885544"
                        open
                      >
                        <summary class="article-expander__title">The Mesenchymal Niche in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885544/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220804212147&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are clonal disorders characterized by ineffective hematopoiesis, resulting in cytopenias and a risk of developing acute myeloid leukemia. In addition to mutations affecting hematopoietic stem cells (HSCs), numerous studies have highlighted the role of the bone marrow microenvironment (BMME) in the development of MDSs. The mesenchymal niche represents a key component of the BMME. In this review, we discuss the role of the mesenchymal niche in the pathophysiology...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885518"
                        open
                      >
                        <summary class="article-expander__title">Clinical Significance of Clonal Hematopoiesis of Indeterminate Potential in Hematology and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885518/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220804212147&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid profiling uses circulating tumor DNA (ctDNA) for minimal invasive tumor mutational profiling from peripheral blood. The presence of somatic mutations in peripheral blood cells without further evidence of a hematologic neoplasm defines clonal hematopoiesis of indeterminate potential (CHIP). CHIP-mutations can be found in the cell-free DNA (cfDNA) of plasma, are a potential cause of false positive results in liquid profiling, and thus limit its usage in screening settings. Various...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885462"
                        open
                      >
                        <summary class="article-expander__title">Automated Detection of Dysplasia: Data Mining from Our Hematology Analyzers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885462/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220804212147&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are clonal hematopoietic diseases of the elderly, characterized by chronic cytopenia, ineffective and dysplastic hematopoiesis, recurrent genetic abnormalities and increased risk of progression to acute myeloid leukemia. Diagnosis on a complete blood count (CBC) can be challenging due to numerous other non-neoplastic causes of cytopenias. New generations of hematology analyzers provide cell population data (CPD) that can be exploited to reliably detect MDSs from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35896010"
                        open
                      >
                        <summary class="article-expander__title">CAR T-cell Therapy Meets Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35896010/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220803212051&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is common in patients with hematologic malignancies. Recent publications provide evidence that CHIP may affect chimeric antigen receptor T-cell therapy efficacy and that the incidence of treatment-related toxicities such as cytokine release syndrome and immune effector-cell associated neurotoxicity syndrome may be affected. See related article by Saini et al. (8).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885563"
                        open
                      >
                        <summary class="article-expander__title">Diagnosis of Myelodysplastic Syndromes: From Immunological Observations to Clinical Applications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885563/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220803212051&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) constitute a very heterogeneous group of diseases with a high prevalence in elderly patients and a propensity for progression to acute myeloid leukemia. The complexity of these hematopoietic malignancies is revealed by the multiple recurrent somatic mutations involved in MDS pathogenesis and the paradoxical common phenotype observed in these patients characterized by ineffective hematopoiesis and cytopenia. In the context of population aging, the incidence of MDS...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885544"
                        open
                      >
                        <summary class="article-expander__title">The Mesenchymal Niche in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885544/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220803212051&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are clonal disorders characterized by ineffective hematopoiesis, resulting in cytopenias and a risk of developing acute myeloid leukemia. In addition to mutations affecting hematopoietic stem cells (HSCs), numerous studies have highlighted the role of the bone marrow microenvironment (BMME) in the development of MDSs. The mesenchymal niche represents a key component of the BMME. In this review, we discuss the role of the mesenchymal niche in the pathophysiology...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885518"
                        open
                      >
                        <summary class="article-expander__title">Clinical Significance of Clonal Hematopoiesis of Indeterminate Potential in Hematology and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885518/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220803212051&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid profiling uses circulating tumor DNA (ctDNA) for minimal invasive tumor mutational profiling from peripheral blood. The presence of somatic mutations in peripheral blood cells without further evidence of a hematologic neoplasm defines clonal hematopoiesis of indeterminate potential (CHIP). CHIP-mutations can be found in the cell-free DNA (cfDNA) of plasma, are a potential cause of false positive results in liquid profiling, and thus limit its usage in screening settings. Various...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885462"
                        open
                      >
                        <summary class="article-expander__title">Automated Detection of Dysplasia: Data Mining from Our Hematology Analyzers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885462/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220803212051&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are clonal hematopoietic diseases of the elderly, characterized by chronic cytopenia, ineffective and dysplastic hematopoiesis, recurrent genetic abnormalities and increased risk of progression to acute myeloid leukemia. Diagnosis on a complete blood count (CBC) can be challenging due to numerous other non-neoplastic causes of cytopenias. New generations of hematology analyzers provide cell population data (CPD) that can be exploited to reliably detect MDSs from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35896010"
                        open
                      >
                        <summary class="article-expander__title">CAR T-cell Therapy Meets Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35896010/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220802212153&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is common in patients with hematologic malignancies. Recent publications provide evidence that CHIP may affect chimeric antigen receptor T-cell therapy efficacy and that the incidence of treatment-related toxicities such as cytokine release syndrome and immune effector-cell associated neurotoxicity syndrome may be affected. See related article by Saini et al. (8).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885563"
                        open
                      >
                        <summary class="article-expander__title">Diagnosis of Myelodysplastic Syndromes: From Immunological Observations to Clinical Applications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885563/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220802212153&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) constitute a very heterogeneous group of diseases with a high prevalence in elderly patients and a propensity for progression to acute myeloid leukemia. The complexity of these hematopoietic malignancies is revealed by the multiple recurrent somatic mutations involved in MDS pathogenesis and the paradoxical common phenotype observed in these patients characterized by ineffective hematopoiesis and cytopenia. In the context of population aging, the incidence of MDS...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885544"
                        open
                      >
                        <summary class="article-expander__title">The Mesenchymal Niche in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885544/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220802212153&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are clonal disorders characterized by ineffective hematopoiesis, resulting in cytopenias and a risk of developing acute myeloid leukemia. In addition to mutations affecting hematopoietic stem cells (HSCs), numerous studies have highlighted the role of the bone marrow microenvironment (BMME) in the development of MDSs. The mesenchymal niche represents a key component of the BMME. In this review, we discuss the role of the mesenchymal niche in the pathophysiology...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885518"
                        open
                      >
                        <summary class="article-expander__title">Clinical Significance of Clonal Hematopoiesis of Indeterminate Potential in Hematology and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885518/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220802212153&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid profiling uses circulating tumor DNA (ctDNA) for minimal invasive tumor mutational profiling from peripheral blood. The presence of somatic mutations in peripheral blood cells without further evidence of a hematologic neoplasm defines clonal hematopoiesis of indeterminate potential (CHIP). CHIP-mutations can be found in the cell-free DNA (cfDNA) of plasma, are a potential cause of false positive results in liquid profiling, and thus limit its usage in screening settings. Various...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885462"
                        open
                      >
                        <summary class="article-expander__title">Automated Detection of Dysplasia: Data Mining from Our Hematology Analyzers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885462/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220802212153&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are clonal hematopoietic diseases of the elderly, characterized by chronic cytopenia, ineffective and dysplastic hematopoiesis, recurrent genetic abnormalities and increased risk of progression to acute myeloid leukemia. Diagnosis on a complete blood count (CBC) can be challenging due to numerous other non-neoplastic causes of cytopenias. New generations of hematology analyzers provide cell population data (CPD) that can be exploited to reliably detect MDSs from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35896010"
                        open
                      >
                        <summary class="article-expander__title">CAR T-cell Therapy Meets Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35896010/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220801212447&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is common in patients with hematologic malignancies. Recent publications provide evidence that CHIP may affect chimeric antigen receptor T-cell therapy efficacy and that the incidence of treatment-related toxicities such as cytokine release syndrome and immune effector-cell associated neurotoxicity syndrome may be affected. See related article by Saini et al. (8).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885563"
                        open
                      >
                        <summary class="article-expander__title">Diagnosis of Myelodysplastic Syndromes: From Immunological Observations to Clinical Applications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885563/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220801212447&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) constitute a very heterogeneous group of diseases with a high prevalence in elderly patients and a propensity for progression to acute myeloid leukemia. The complexity of these hematopoietic malignancies is revealed by the multiple recurrent somatic mutations involved in MDS pathogenesis and the paradoxical common phenotype observed in these patients characterized by ineffective hematopoiesis and cytopenia. In the context of population aging, the incidence of MDS...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885544"
                        open
                      >
                        <summary class="article-expander__title">The Mesenchymal Niche in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885544/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220801212447&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are clonal disorders characterized by ineffective hematopoiesis, resulting in cytopenias and a risk of developing acute myeloid leukemia. In addition to mutations affecting hematopoietic stem cells (HSCs), numerous studies have highlighted the role of the bone marrow microenvironment (BMME) in the development of MDSs. The mesenchymal niche represents a key component of the BMME. In this review, we discuss the role of the mesenchymal niche in the pathophysiology...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885518"
                        open
                      >
                        <summary class="article-expander__title">Clinical Significance of Clonal Hematopoiesis of Indeterminate Potential in Hematology and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885518/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220801212447&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid profiling uses circulating tumor DNA (ctDNA) for minimal invasive tumor mutational profiling from peripheral blood. The presence of somatic mutations in peripheral blood cells without further evidence of a hematologic neoplasm defines clonal hematopoiesis of indeterminate potential (CHIP). CHIP-mutations can be found in the cell-free DNA (cfDNA) of plasma, are a potential cause of false positive results in liquid profiling, and thus limit its usage in screening settings. Various...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885462"
                        open
                      >
                        <summary class="article-expander__title">Automated Detection of Dysplasia: Data Mining from Our Hematology Analyzers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885462/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220801212447&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are clonal hematopoietic diseases of the elderly, characterized by chronic cytopenia, ineffective and dysplastic hematopoiesis, recurrent genetic abnormalities and increased risk of progression to acute myeloid leukemia. Diagnosis on a complete blood count (CBC) can be challenging due to numerous other non-neoplastic causes of cytopenias. New generations of hematology analyzers provide cell population data (CPD) that can be exploited to reliably detect MDSs from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35896010"
                        open
                      >
                        <summary class="article-expander__title">CAR T-cell Therapy Meets Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35896010/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220731212525&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is common in patients with hematologic malignancies. Recent publications provide evidence that CHIP may affect chimeric antigen receptor T-cell therapy efficacy and that the incidence of treatment-related toxicities such as cytokine release syndrome and immune effector-cell associated neurotoxicity syndrome may be affected. See related article by Saini et al. (8).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885563"
                        open
                      >
                        <summary class="article-expander__title">Diagnosis of Myelodysplastic Syndromes: From Immunological Observations to Clinical Applications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885563/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220731212525&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) constitute a very heterogeneous group of diseases with a high prevalence in elderly patients and a propensity for progression to acute myeloid leukemia. The complexity of these hematopoietic malignancies is revealed by the multiple recurrent somatic mutations involved in MDS pathogenesis and the paradoxical common phenotype observed in these patients characterized by ineffective hematopoiesis and cytopenia. In the context of population aging, the incidence of MDS...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885544"
                        open
                      >
                        <summary class="article-expander__title">The Mesenchymal Niche in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885544/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220731212525&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are clonal disorders characterized by ineffective hematopoiesis, resulting in cytopenias and a risk of developing acute myeloid leukemia. In addition to mutations affecting hematopoietic stem cells (HSCs), numerous studies have highlighted the role of the bone marrow microenvironment (BMME) in the development of MDSs. The mesenchymal niche represents a key component of the BMME. In this review, we discuss the role of the mesenchymal niche in the pathophysiology...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885518"
                        open
                      >
                        <summary class="article-expander__title">Clinical Significance of Clonal Hematopoiesis of Indeterminate Potential in Hematology and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885518/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220731212525&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid profiling uses circulating tumor DNA (ctDNA) for minimal invasive tumor mutational profiling from peripheral blood. The presence of somatic mutations in peripheral blood cells without further evidence of a hematologic neoplasm defines clonal hematopoiesis of indeterminate potential (CHIP). CHIP-mutations can be found in the cell-free DNA (cfDNA) of plasma, are a potential cause of false positive results in liquid profiling, and thus limit its usage in screening settings. Various...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885462"
                        open
                      >
                        <summary class="article-expander__title">Automated Detection of Dysplasia: Data Mining from Our Hematology Analyzers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885462/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220731212525&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are clonal hematopoietic diseases of the elderly, characterized by chronic cytopenia, ineffective and dysplastic hematopoiesis, recurrent genetic abnormalities and increased risk of progression to acute myeloid leukemia. Diagnosis on a complete blood count (CBC) can be challenging due to numerous other non-neoplastic causes of cytopenias. New generations of hematology analyzers provide cell population data (CPD) that can be exploited to reliably detect MDSs from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35896010"
                        open
                      >
                        <summary class="article-expander__title">CAR T-cell Therapy Meets Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35896010/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220730212309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is common in patients with hematologic malignancies. Recent publications provide evidence that CHIP may affect chimeric antigen receptor T-cell therapy efficacy and that the incidence of treatment-related toxicities such as cytokine release syndrome and immune effector-cell associated neurotoxicity syndrome may be affected. See related article by Saini et al. (8).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885563"
                        open
                      >
                        <summary class="article-expander__title">Diagnosis of Myelodysplastic Syndromes: From Immunological Observations to Clinical Applications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885563/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220730212309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) constitute a very heterogeneous group of diseases with a high prevalence in elderly patients and a propensity for progression to acute myeloid leukemia. The complexity of these hematopoietic malignancies is revealed by the multiple recurrent somatic mutations involved in MDS pathogenesis and the paradoxical common phenotype observed in these patients characterized by ineffective hematopoiesis and cytopenia. In the context of population aging, the incidence of MDS...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885544"
                        open
                      >
                        <summary class="article-expander__title">The Mesenchymal Niche in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885544/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220730212309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are clonal disorders characterized by ineffective hematopoiesis, resulting in cytopenias and a risk of developing acute myeloid leukemia. In addition to mutations affecting hematopoietic stem cells (HSCs), numerous studies have highlighted the role of the bone marrow microenvironment (BMME) in the development of MDSs. The mesenchymal niche represents a key component of the BMME. In this review, we discuss the role of the mesenchymal niche in the pathophysiology...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885518"
                        open
                      >
                        <summary class="article-expander__title">Clinical Significance of Clonal Hematopoiesis of Indeterminate Potential in Hematology and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885518/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220730212309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid profiling uses circulating tumor DNA (ctDNA) for minimal invasive tumor mutational profiling from peripheral blood. The presence of somatic mutations in peripheral blood cells without further evidence of a hematologic neoplasm defines clonal hematopoiesis of indeterminate potential (CHIP). CHIP-mutations can be found in the cell-free DNA (cfDNA) of plasma, are a potential cause of false positive results in liquid profiling, and thus limit its usage in screening settings. Various...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885462"
                        open
                      >
                        <summary class="article-expander__title">Automated Detection of Dysplasia: Data Mining from Our Hematology Analyzers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885462/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220730212309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are clonal hematopoietic diseases of the elderly, characterized by chronic cytopenia, ineffective and dysplastic hematopoiesis, recurrent genetic abnormalities and increased risk of progression to acute myeloid leukemia. Diagnosis on a complete blood count (CBC) can be challenging due to numerous other non-neoplastic causes of cytopenias. New generations of hematology analyzers provide cell population data (CPD) that can be exploited to reliably detect MDSs from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35896010"
                        open
                      >
                        <summary class="article-expander__title">CAR T-cell Therapy Meets Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35896010/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220729211332&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is common in patients with hematologic malignancies. Recent publications provide evidence that CHIP may affect chimeric antigen receptor T-cell therapy efficacy and that the incidence of treatment-related toxicities such as cytokine release syndrome and immune effector-cell associated neurotoxicity syndrome may be affected. See related article by Saini et al. (8).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885563"
                        open
                      >
                        <summary class="article-expander__title">Diagnosis of Myelodysplastic Syndromes: From Immunological Observations to Clinical Applications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885563/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220729211332&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) constitute a very heterogeneous group of diseases with a high prevalence in elderly patients and a propensity for progression to acute myeloid leukemia. The complexity of these hematopoietic malignancies is revealed by the multiple recurrent somatic mutations involved in MDS pathogenesis and the paradoxical common phenotype observed in these patients characterized by ineffective hematopoiesis and cytopenia. In the context of population aging, the incidence of MDS...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885544"
                        open
                      >
                        <summary class="article-expander__title">The Mesenchymal Niche in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885544/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220729211332&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are clonal disorders characterized by ineffective hematopoiesis, resulting in cytopenias and a risk of developing acute myeloid leukemia. In addition to mutations affecting hematopoietic stem cells (HSCs), numerous studies have highlighted the role of the bone marrow microenvironment (BMME) in the development of MDSs. The mesenchymal niche represents a key component of the BMME. In this review, we discuss the role of the mesenchymal niche in the pathophysiology...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885518"
                        open
                      >
                        <summary class="article-expander__title">Clinical Significance of Clonal Hematopoiesis of Indeterminate Potential in Hematology and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885518/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220729211332&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid profiling uses circulating tumor DNA (ctDNA) for minimal invasive tumor mutational profiling from peripheral blood. The presence of somatic mutations in peripheral blood cells without further evidence of a hematologic neoplasm defines clonal hematopoiesis of indeterminate potential (CHIP). CHIP-mutations can be found in the cell-free DNA (cfDNA) of plasma, are a potential cause of false positive results in liquid profiling, and thus limit its usage in screening settings. Various...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885462"
                        open
                      >
                        <summary class="article-expander__title">Automated Detection of Dysplasia: Data Mining from Our Hematology Analyzers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885462/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220729211332&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are clonal hematopoietic diseases of the elderly, characterized by chronic cytopenia, ineffective and dysplastic hematopoiesis, recurrent genetic abnormalities and increased risk of progression to acute myeloid leukemia. Diagnosis on a complete blood count (CBC) can be challenging due to numerous other non-neoplastic causes of cytopenias. New generations of hematology analyzers provide cell population data (CPD) that can be exploited to reliably detect MDSs from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35896010"
                        open
                      >
                        <summary class="article-expander__title">CAR T-cell Therapy Meets Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35896010/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220728211733&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is common in patients with hematologic malignancies. Recent publications provide evidence that CHIP may affect chimeric antigen receptor T-cell therapy efficacy and that the incidence of treatment-related toxicities such as cytokine release syndrome and immune effector-cell associated neurotoxicity syndrome may be affected. See related article by Saini et al. (8).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885563"
                        open
                      >
                        <summary class="article-expander__title">Diagnosis of Myelodysplastic Syndromes: From Immunological Observations to Clinical Applications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885563/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220728211733&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) constitute a very heterogeneous group of diseases with a high prevalence in elderly patients and a propensity for progression to acute myeloid leukemia. The complexity of these hematopoietic malignancies is revealed by the multiple recurrent somatic mutations involved in MDS pathogenesis and the paradoxical common phenotype observed in these patients characterized by ineffective hematopoiesis and cytopenia. In the context of population aging, the incidence of MDS...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885544"
                        open
                      >
                        <summary class="article-expander__title">The Mesenchymal Niche in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885544/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220728211733&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are clonal disorders characterized by ineffective hematopoiesis, resulting in cytopenias and a risk of developing acute myeloid leukemia. In addition to mutations affecting hematopoietic stem cells (HSCs), numerous studies have highlighted the role of the bone marrow microenvironment (BMME) in the development of MDSs. The mesenchymal niche represents a key component of the BMME. In this review, we discuss the role of the mesenchymal niche in the pathophysiology...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885518"
                        open
                      >
                        <summary class="article-expander__title">Clinical Significance of Clonal Hematopoiesis of Indeterminate Potential in Hematology and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885518/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220728211733&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid profiling uses circulating tumor DNA (ctDNA) for minimal invasive tumor mutational profiling from peripheral blood. The presence of somatic mutations in peripheral blood cells without further evidence of a hematologic neoplasm defines clonal hematopoiesis of indeterminate potential (CHIP). CHIP-mutations can be found in the cell-free DNA (cfDNA) of plasma, are a potential cause of false positive results in liquid profiling, and thus limit its usage in screening settings. Various...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885462"
                        open
                      >
                        <summary class="article-expander__title">Automated Detection of Dysplasia: Data Mining from Our Hematology Analyzers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885462/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220728211733&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are clonal hematopoietic diseases of the elderly, characterized by chronic cytopenia, ineffective and dysplastic hematopoiesis, recurrent genetic abnormalities and increased risk of progression to acute myeloid leukemia. Diagnosis on a complete blood count (CBC) can be challenging due to numerous other non-neoplastic causes of cytopenias. New generations of hematology analyzers provide cell population data (CPD) that can be exploited to reliably detect MDSs from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35896010"
                        open
                      >
                        <summary class="article-expander__title">CAR T-cell Therapy Meets Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35896010/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220727211458&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is common in patients with hematologic malignancies. Recent publications provide evidence that CHIP may affect chimeric antigen receptor T-cell therapy efficacy and that the incidence of treatment-related toxicities such as cytokine release syndrome and immune effector-cell associated neurotoxicity syndrome may be affected. See related article by Saini et al. (8).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885563"
                        open
                      >
                        <summary class="article-expander__title">Diagnosis of Myelodysplastic Syndromes: From Immunological Observations to Clinical Applications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885563/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220727211458&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) constitute a very heterogeneous group of diseases with a high prevalence in elderly patients and a propensity for progression to acute myeloid leukemia. The complexity of these hematopoietic malignancies is revealed by the multiple recurrent somatic mutations involved in MDS pathogenesis and the paradoxical common phenotype observed in these patients characterized by ineffective hematopoiesis and cytopenia. In the context of population aging, the incidence of MDS...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885544"
                        open
                      >
                        <summary class="article-expander__title">The Mesenchymal Niche in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885544/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220727211458&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are clonal disorders characterized by ineffective hematopoiesis, resulting in cytopenias and a risk of developing acute myeloid leukemia. In addition to mutations affecting hematopoietic stem cells (HSCs), numerous studies have highlighted the role of the bone marrow microenvironment (BMME) in the development of MDSs. The mesenchymal niche represents a key component of the BMME. In this review, we discuss the role of the mesenchymal niche in the pathophysiology...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885518"
                        open
                      >
                        <summary class="article-expander__title">Clinical Significance of Clonal Hematopoiesis of Indeterminate Potential in Hematology and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885518/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220727211458&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Liquid profiling uses circulating tumor DNA (ctDNA) for minimal invasive tumor mutational profiling from peripheral blood. The presence of somatic mutations in peripheral blood cells without further evidence of a hematologic neoplasm defines clonal hematopoiesis of indeterminate potential (CHIP). CHIP-mutations can be found in the cell-free DNA (cfDNA) of plasma, are a potential cause of false positive results in liquid profiling, and thus limit its usage in screening settings. Various...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35885462"
                        open
                      >
                        <summary class="article-expander__title">Automated Detection of Dysplasia: Data Mining from Our Hematology Analyzers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35885462/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220727211458&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDSs) are clonal hematopoietic diseases of the elderly, characterized by chronic cytopenia, ineffective and dysplastic hematopoiesis, recurrent genetic abnormalities and increased risk of progression to acute myeloid leukemia. Diagnosis on a complete blood count (CBC) can be challenging due to numerous other non-neoplastic causes of cytopenias. New generations of hematology analyzers provide cell population data (CPD) that can be exploited to reliably detect MDSs from...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608010801&amp;v&#x3D;2.17.7&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35896960"
                        open
                      >
                        <summary class="article-expander__title">Proteomic and phosphoproteomic measurements enhance ability to predict ex vivo drug response in AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35896960/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220824212808&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute Myeloid Leukemia (AML) affects 20,000 patients in the US annually with a five-year survival rate of approximately 25%. One reason for the low survival rate is the high prevalence of clonal evolution that gives rise to heterogeneous sub-populations of leukemic cells with diverse mutation spectra, which eventually leads to disease relapse. This genetic heterogeneity drives the activation of complex signaling pathways that is reflected at the protein level. This diversity makes it difficult...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35893374"
                        open
                      >
                        <summary class="article-expander__title">In Pursuit of Genetic Prognostic Factors and Treatment Approaches in Secondary Acute Myeloid Leukemia-A Narrative Review of Current Knowledge</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35893374/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220824212808&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Secondary acute myeloid leukemia can be divided into two categories: AML evolving from the antecedent hematological condition (AHD-AML) and therapy related AML (t-AML). AHD-AML can evolve from hematological conditions such as myelodysplastic syndromes, myeloproliferative neoplasms, MDS/MPN overlap syndromes, Fanconi anemia, and aplastic anemia. Leukemic transformation occurs as a consequence of the clonal evolution-a process of the acquisition of mutations in clones, while previous mutations are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35896960"
                        open
                      >
                        <summary class="article-expander__title">Proteomic and phosphoproteomic measurements enhance ability to predict ex vivo drug response in AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35896960/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220823212820&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute Myeloid Leukemia (AML) affects 20,000 patients in the US annually with a five-year survival rate of approximately 25%. One reason for the low survival rate is the high prevalence of clonal evolution that gives rise to heterogeneous sub-populations of leukemic cells with diverse mutation spectra, which eventually leads to disease relapse. This genetic heterogeneity drives the activation of complex signaling pathways that is reflected at the protein level. This diversity makes it difficult...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35893374"
                        open
                      >
                        <summary class="article-expander__title">In Pursuit of Genetic Prognostic Factors and Treatment Approaches in Secondary Acute Myeloid Leukemia-A Narrative Review of Current Knowledge</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35893374/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220823212820&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Secondary acute myeloid leukemia can be divided into two categories: AML evolving from the antecedent hematological condition (AHD-AML) and therapy related AML (t-AML). AHD-AML can evolve from hematological conditions such as myelodysplastic syndromes, myeloproliferative neoplasms, MDS/MPN overlap syndromes, Fanconi anemia, and aplastic anemia. Leukemic transformation occurs as a consequence of the clonal evolution-a process of the acquisition of mutations in clones, while previous mutations are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35896960"
                        open
                      >
                        <summary class="article-expander__title">Proteomic and phosphoproteomic measurements enhance ability to predict ex vivo drug response in AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35896960/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220822212717&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute Myeloid Leukemia (AML) affects 20,000 patients in the US annually with a five-year survival rate of approximately 25%. One reason for the low survival rate is the high prevalence of clonal evolution that gives rise to heterogeneous sub-populations of leukemic cells with diverse mutation spectra, which eventually leads to disease relapse. This genetic heterogeneity drives the activation of complex signaling pathways that is reflected at the protein level. This diversity makes it difficult...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35893374"
                        open
                      >
                        <summary class="article-expander__title">In Pursuit of Genetic Prognostic Factors and Treatment Approaches in Secondary Acute Myeloid Leukemia-A Narrative Review of Current Knowledge</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35893374/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220822212717&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Secondary acute myeloid leukemia can be divided into two categories: AML evolving from the antecedent hematological condition (AHD-AML) and therapy related AML (t-AML). AHD-AML can evolve from hematological conditions such as myelodysplastic syndromes, myeloproliferative neoplasms, MDS/MPN overlap syndromes, Fanconi anemia, and aplastic anemia. Leukemic transformation occurs as a consequence of the clonal evolution-a process of the acquisition of mutations in clones, while previous mutations are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35884979"
                        open
                      >
                        <summary class="article-expander__title">Tracking Clonal Evolution of Multiple Myeloma Using Targeted Next-Generation DNA Sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35884979/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220822212717&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution drives treatment failure in multiple myeloma (MM). Here, we used a custom 372-gene panel to track genetic changes occurring during MM progression at different stages of the disease. A tumor-only targeted next-generation DNA sequencing was performed on 69 samples sequentially collected from 30 MM patients. The MAPK/ERK pathway was mostly affected with KRAS mutated in 47% of patients. Acquisition and loss of mutations were observed in 63% and 37% of patients, respectively. Four...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35896960"
                        open
                      >
                        <summary class="article-expander__title">Proteomic and phosphoproteomic measurements enhance ability to predict ex vivo drug response in AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35896960/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220821211551&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute Myeloid Leukemia (AML) affects 20,000 patients in the US annually with a five-year survival rate of approximately 25%. One reason for the low survival rate is the high prevalence of clonal evolution that gives rise to heterogeneous sub-populations of leukemic cells with diverse mutation spectra, which eventually leads to disease relapse. This genetic heterogeneity drives the activation of complex signaling pathways that is reflected at the protein level. This diversity makes it difficult...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35893374"
                        open
                      >
                        <summary class="article-expander__title">In Pursuit of Genetic Prognostic Factors and Treatment Approaches in Secondary Acute Myeloid Leukemia-A Narrative Review of Current Knowledge</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35893374/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220821211551&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Secondary acute myeloid leukemia can be divided into two categories: AML evolving from the antecedent hematological condition (AHD-AML) and therapy related AML (t-AML). AHD-AML can evolve from hematological conditions such as myelodysplastic syndromes, myeloproliferative neoplasms, MDS/MPN overlap syndromes, Fanconi anemia, and aplastic anemia. Leukemic transformation occurs as a consequence of the clonal evolution-a process of the acquisition of mutations in clones, while previous mutations are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35884979"
                        open
                      >
                        <summary class="article-expander__title">Tracking Clonal Evolution of Multiple Myeloma Using Targeted Next-Generation DNA Sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35884979/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220821211551&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution drives treatment failure in multiple myeloma (MM). Here, we used a custom 372-gene panel to track genetic changes occurring during MM progression at different stages of the disease. A tumor-only targeted next-generation DNA sequencing was performed on 69 samples sequentially collected from 30 MM patients. The MAPK/ERK pathway was mostly affected with KRAS mutated in 47% of patients. Acquisition and loss of mutations were observed in 63% and 37% of patients, respectively. Four...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35896960"
                        open
                      >
                        <summary class="article-expander__title">Proteomic and phosphoproteomic measurements enhance ability to predict ex vivo drug response in AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35896960/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220820211309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute Myeloid Leukemia (AML) affects 20,000 patients in the US annually with a five-year survival rate of approximately 25%. One reason for the low survival rate is the high prevalence of clonal evolution that gives rise to heterogeneous sub-populations of leukemic cells with diverse mutation spectra, which eventually leads to disease relapse. This genetic heterogeneity drives the activation of complex signaling pathways that is reflected at the protein level. This diversity makes it difficult...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35893374"
                        open
                      >
                        <summary class="article-expander__title">In Pursuit of Genetic Prognostic Factors and Treatment Approaches in Secondary Acute Myeloid Leukemia-A Narrative Review of Current Knowledge</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35893374/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220820211309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Secondary acute myeloid leukemia can be divided into two categories: AML evolving from the antecedent hematological condition (AHD-AML) and therapy related AML (t-AML). AHD-AML can evolve from hematological conditions such as myelodysplastic syndromes, myeloproliferative neoplasms, MDS/MPN overlap syndromes, Fanconi anemia, and aplastic anemia. Leukemic transformation occurs as a consequence of the clonal evolution-a process of the acquisition of mutations in clones, while previous mutations are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35884979"
                        open
                      >
                        <summary class="article-expander__title">Tracking Clonal Evolution of Multiple Myeloma Using Targeted Next-Generation DNA Sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35884979/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220820211309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution drives treatment failure in multiple myeloma (MM). Here, we used a custom 372-gene panel to track genetic changes occurring during MM progression at different stages of the disease. A tumor-only targeted next-generation DNA sequencing was performed on 69 samples sequentially collected from 30 MM patients. The MAPK/ERK pathway was mostly affected with KRAS mutated in 47% of patients. Acquisition and loss of mutations were observed in 63% and 37% of patients, respectively. Four...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35896960"
                        open
                      >
                        <summary class="article-expander__title">Proteomic and phosphoproteomic measurements enhance ability to predict ex vivo drug response in AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35896960/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220819211458&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute Myeloid Leukemia (AML) affects 20,000 patients in the US annually with a five-year survival rate of approximately 25%. One reason for the low survival rate is the high prevalence of clonal evolution that gives rise to heterogeneous sub-populations of leukemic cells with diverse mutation spectra, which eventually leads to disease relapse. This genetic heterogeneity drives the activation of complex signaling pathways that is reflected at the protein level. This diversity makes it difficult...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35893374"
                        open
                      >
                        <summary class="article-expander__title">In Pursuit of Genetic Prognostic Factors and Treatment Approaches in Secondary Acute Myeloid Leukemia-A Narrative Review of Current Knowledge</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35893374/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220819211458&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Secondary acute myeloid leukemia can be divided into two categories: AML evolving from the antecedent hematological condition (AHD-AML) and therapy related AML (t-AML). AHD-AML can evolve from hematological conditions such as myelodysplastic syndromes, myeloproliferative neoplasms, MDS/MPN overlap syndromes, Fanconi anemia, and aplastic anemia. Leukemic transformation occurs as a consequence of the clonal evolution-a process of the acquisition of mutations in clones, while previous mutations are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35884979"
                        open
                      >
                        <summary class="article-expander__title">Tracking Clonal Evolution of Multiple Myeloma Using Targeted Next-Generation DNA Sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35884979/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220819211458&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution drives treatment failure in multiple myeloma (MM). Here, we used a custom 372-gene panel to track genetic changes occurring during MM progression at different stages of the disease. A tumor-only targeted next-generation DNA sequencing was performed on 69 samples sequentially collected from 30 MM patients. The MAPK/ERK pathway was mostly affected with KRAS mutated in 47% of patients. Acquisition and loss of mutations were observed in 63% and 37% of patients, respectively. Four...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35896960"
                        open
                      >
                        <summary class="article-expander__title">Proteomic and phosphoproteomic measurements enhance ability to predict ex vivo drug response in AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35896960/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220818211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute Myeloid Leukemia (AML) affects 20,000 patients in the US annually with a five-year survival rate of approximately 25%. One reason for the low survival rate is the high prevalence of clonal evolution that gives rise to heterogeneous sub-populations of leukemic cells with diverse mutation spectra, which eventually leads to disease relapse. This genetic heterogeneity drives the activation of complex signaling pathways that is reflected at the protein level. This diversity makes it difficult...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35893374"
                        open
                      >
                        <summary class="article-expander__title">In Pursuit of Genetic Prognostic Factors and Treatment Approaches in Secondary Acute Myeloid Leukemia-A Narrative Review of Current Knowledge</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35893374/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220818211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Secondary acute myeloid leukemia can be divided into two categories: AML evolving from the antecedent hematological condition (AHD-AML) and therapy related AML (t-AML). AHD-AML can evolve from hematological conditions such as myelodysplastic syndromes, myeloproliferative neoplasms, MDS/MPN overlap syndromes, Fanconi anemia, and aplastic anemia. Leukemic transformation occurs as a consequence of the clonal evolution-a process of the acquisition of mutations in clones, while previous mutations are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35884979"
                        open
                      >
                        <summary class="article-expander__title">Tracking Clonal Evolution of Multiple Myeloma Using Targeted Next-Generation DNA Sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35884979/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220818211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution drives treatment failure in multiple myeloma (MM). Here, we used a custom 372-gene panel to track genetic changes occurring during MM progression at different stages of the disease. A tumor-only targeted next-generation DNA sequencing was performed on 69 samples sequentially collected from 30 MM patients. The MAPK/ERK pathway was mostly affected with KRAS mutated in 47% of patients. Acquisition and loss of mutations were observed in 63% and 37% of patients, respectively. Four...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35896960"
                        open
                      >
                        <summary class="article-expander__title">Proteomic and phosphoproteomic measurements enhance ability to predict ex vivo drug response in AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35896960/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220817212606&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute Myeloid Leukemia (AML) affects 20,000 patients in the US annually with a five-year survival rate of approximately 25%. One reason for the low survival rate is the high prevalence of clonal evolution that gives rise to heterogeneous sub-populations of leukemic cells with diverse mutation spectra, which eventually leads to disease relapse. This genetic heterogeneity drives the activation of complex signaling pathways that is reflected at the protein level. This diversity makes it difficult...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35893374"
                        open
                      >
                        <summary class="article-expander__title">In Pursuit of Genetic Prognostic Factors and Treatment Approaches in Secondary Acute Myeloid Leukemia-A Narrative Review of Current Knowledge</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35893374/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220817212606&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Secondary acute myeloid leukemia can be divided into two categories: AML evolving from the antecedent hematological condition (AHD-AML) and therapy related AML (t-AML). AHD-AML can evolve from hematological conditions such as myelodysplastic syndromes, myeloproliferative neoplasms, MDS/MPN overlap syndromes, Fanconi anemia, and aplastic anemia. Leukemic transformation occurs as a consequence of the clonal evolution-a process of the acquisition of mutations in clones, while previous mutations are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35884979"
                        open
                      >
                        <summary class="article-expander__title">Tracking Clonal Evolution of Multiple Myeloma Using Targeted Next-Generation DNA Sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35884979/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220817212606&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution drives treatment failure in multiple myeloma (MM). Here, we used a custom 372-gene panel to track genetic changes occurring during MM progression at different stages of the disease. A tumor-only targeted next-generation DNA sequencing was performed on 69 samples sequentially collected from 30 MM patients. The MAPK/ERK pathway was mostly affected with KRAS mutated in 47% of patients. Acquisition and loss of mutations were observed in 63% and 37% of patients, respectively. Four...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35896960"
                        open
                      >
                        <summary class="article-expander__title">Proteomic and phosphoproteomic measurements enhance ability to predict ex vivo drug response in AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35896960/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220816212355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute Myeloid Leukemia (AML) affects 20,000 patients in the US annually with a five-year survival rate of approximately 25%. One reason for the low survival rate is the high prevalence of clonal evolution that gives rise to heterogeneous sub-populations of leukemic cells with diverse mutation spectra, which eventually leads to disease relapse. This genetic heterogeneity drives the activation of complex signaling pathways that is reflected at the protein level. This diversity makes it difficult...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35893374"
                        open
                      >
                        <summary class="article-expander__title">In Pursuit of Genetic Prognostic Factors and Treatment Approaches in Secondary Acute Myeloid Leukemia-A Narrative Review of Current Knowledge</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35893374/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220816212355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Secondary acute myeloid leukemia can be divided into two categories: AML evolving from the antecedent hematological condition (AHD-AML) and therapy related AML (t-AML). AHD-AML can evolve from hematological conditions such as myelodysplastic syndromes, myeloproliferative neoplasms, MDS/MPN overlap syndromes, Fanconi anemia, and aplastic anemia. Leukemic transformation occurs as a consequence of the clonal evolution-a process of the acquisition of mutations in clones, while previous mutations are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35884979"
                        open
                      >
                        <summary class="article-expander__title">Tracking Clonal Evolution of Multiple Myeloma Using Targeted Next-Generation DNA Sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35884979/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220816212355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution drives treatment failure in multiple myeloma (MM). Here, we used a custom 372-gene panel to track genetic changes occurring during MM progression at different stages of the disease. A tumor-only targeted next-generation DNA sequencing was performed on 69 samples sequentially collected from 30 MM patients. The MAPK/ERK pathway was mostly affected with KRAS mutated in 47% of patients. Acquisition and loss of mutations were observed in 63% and 37% of patients, respectively. Four...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35896960"
                        open
                      >
                        <summary class="article-expander__title">Proteomic and phosphoproteomic measurements enhance ability to predict ex vivo drug response in AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35896960/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220815212205&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute Myeloid Leukemia (AML) affects 20,000 patients in the US annually with a five-year survival rate of approximately 25%. One reason for the low survival rate is the high prevalence of clonal evolution that gives rise to heterogeneous sub-populations of leukemic cells with diverse mutation spectra, which eventually leads to disease relapse. This genetic heterogeneity drives the activation of complex signaling pathways that is reflected at the protein level. This diversity makes it difficult...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35893374"
                        open
                      >
                        <summary class="article-expander__title">In Pursuit of Genetic Prognostic Factors and Treatment Approaches in Secondary Acute Myeloid Leukemia-A Narrative Review of Current Knowledge</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35893374/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220815212205&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Secondary acute myeloid leukemia can be divided into two categories: AML evolving from the antecedent hematological condition (AHD-AML) and therapy related AML (t-AML). AHD-AML can evolve from hematological conditions such as myelodysplastic syndromes, myeloproliferative neoplasms, MDS/MPN overlap syndromes, Fanconi anemia, and aplastic anemia. Leukemic transformation occurs as a consequence of the clonal evolution-a process of the acquisition of mutations in clones, while previous mutations are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35884979"
                        open
                      >
                        <summary class="article-expander__title">Tracking Clonal Evolution of Multiple Myeloma Using Targeted Next-Generation DNA Sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35884979/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220815212205&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution drives treatment failure in multiple myeloma (MM). Here, we used a custom 372-gene panel to track genetic changes occurring during MM progression at different stages of the disease. A tumor-only targeted next-generation DNA sequencing was performed on 69 samples sequentially collected from 30 MM patients. The MAPK/ERK pathway was mostly affected with KRAS mutated in 47% of patients. Acquisition and loss of mutations were observed in 63% and 37% of patients, respectively. Four...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35896960"
                        open
                      >
                        <summary class="article-expander__title">Proteomic and phosphoproteomic measurements enhance ability to predict ex vivo drug response in AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35896960/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220814211915&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute Myeloid Leukemia (AML) affects 20,000 patients in the US annually with a five-year survival rate of approximately 25%. One reason for the low survival rate is the high prevalence of clonal evolution that gives rise to heterogeneous sub-populations of leukemic cells with diverse mutation spectra, which eventually leads to disease relapse. This genetic heterogeneity drives the activation of complex signaling pathways that is reflected at the protein level. This diversity makes it difficult...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35893374"
                        open
                      >
                        <summary class="article-expander__title">In Pursuit of Genetic Prognostic Factors and Treatment Approaches in Secondary Acute Myeloid Leukemia-A Narrative Review of Current Knowledge</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35893374/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220814211915&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Secondary acute myeloid leukemia can be divided into two categories: AML evolving from the antecedent hematological condition (AHD-AML) and therapy related AML (t-AML). AHD-AML can evolve from hematological conditions such as myelodysplastic syndromes, myeloproliferative neoplasms, MDS/MPN overlap syndromes, Fanconi anemia, and aplastic anemia. Leukemic transformation occurs as a consequence of the clonal evolution-a process of the acquisition of mutations in clones, while previous mutations are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35884979"
                        open
                      >
                        <summary class="article-expander__title">Tracking Clonal Evolution of Multiple Myeloma Using Targeted Next-Generation DNA Sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35884979/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220814211915&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution drives treatment failure in multiple myeloma (MM). Here, we used a custom 372-gene panel to track genetic changes occurring during MM progression at different stages of the disease. A tumor-only targeted next-generation DNA sequencing was performed on 69 samples sequentially collected from 30 MM patients. The MAPK/ERK pathway was mostly affected with KRAS mutated in 47% of patients. Acquisition and loss of mutations were observed in 63% and 37% of patients, respectively. Four...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35896960"
                        open
                      >
                        <summary class="article-expander__title">Proteomic and phosphoproteomic measurements enhance ability to predict ex vivo drug response in AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35896960/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220813211549&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute Myeloid Leukemia (AML) affects 20,000 patients in the US annually with a five-year survival rate of approximately 25%. One reason for the low survival rate is the high prevalence of clonal evolution that gives rise to heterogeneous sub-populations of leukemic cells with diverse mutation spectra, which eventually leads to disease relapse. This genetic heterogeneity drives the activation of complex signaling pathways that is reflected at the protein level. This diversity makes it difficult...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35893374"
                        open
                      >
                        <summary class="article-expander__title">In Pursuit of Genetic Prognostic Factors and Treatment Approaches in Secondary Acute Myeloid Leukemia-A Narrative Review of Current Knowledge</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35893374/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220813211549&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Secondary acute myeloid leukemia can be divided into two categories: AML evolving from the antecedent hematological condition (AHD-AML) and therapy related AML (t-AML). AHD-AML can evolve from hematological conditions such as myelodysplastic syndromes, myeloproliferative neoplasms, MDS/MPN overlap syndromes, Fanconi anemia, and aplastic anemia. Leukemic transformation occurs as a consequence of the clonal evolution-a process of the acquisition of mutations in clones, while previous mutations are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35884979"
                        open
                      >
                        <summary class="article-expander__title">Tracking Clonal Evolution of Multiple Myeloma Using Targeted Next-Generation DNA Sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35884979/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220813211549&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution drives treatment failure in multiple myeloma (MM). Here, we used a custom 372-gene panel to track genetic changes occurring during MM progression at different stages of the disease. A tumor-only targeted next-generation DNA sequencing was performed on 69 samples sequentially collected from 30 MM patients. The MAPK/ERK pathway was mostly affected with KRAS mutated in 47% of patients. Acquisition and loss of mutations were observed in 63% and 37% of patients, respectively. Four...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35896960"
                        open
                      >
                        <summary class="article-expander__title">Proteomic and phosphoproteomic measurements enhance ability to predict ex vivo drug response in AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35896960/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220812210706&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute Myeloid Leukemia (AML) affects 20,000 patients in the US annually with a five-year survival rate of approximately 25%. One reason for the low survival rate is the high prevalence of clonal evolution that gives rise to heterogeneous sub-populations of leukemic cells with diverse mutation spectra, which eventually leads to disease relapse. This genetic heterogeneity drives the activation of complex signaling pathways that is reflected at the protein level. This diversity makes it difficult...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35893374"
                        open
                      >
                        <summary class="article-expander__title">In Pursuit of Genetic Prognostic Factors and Treatment Approaches in Secondary Acute Myeloid Leukemia-A Narrative Review of Current Knowledge</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35893374/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220812210706&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Secondary acute myeloid leukemia can be divided into two categories: AML evolving from the antecedent hematological condition (AHD-AML) and therapy related AML (t-AML). AHD-AML can evolve from hematological conditions such as myelodysplastic syndromes, myeloproliferative neoplasms, MDS/MPN overlap syndromes, Fanconi anemia, and aplastic anemia. Leukemic transformation occurs as a consequence of the clonal evolution-a process of the acquisition of mutations in clones, while previous mutations are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35884979"
                        open
                      >
                        <summary class="article-expander__title">Tracking Clonal Evolution of Multiple Myeloma Using Targeted Next-Generation DNA Sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35884979/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220812210706&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution drives treatment failure in multiple myeloma (MM). Here, we used a custom 372-gene panel to track genetic changes occurring during MM progression at different stages of the disease. A tumor-only targeted next-generation DNA sequencing was performed on 69 samples sequentially collected from 30 MM patients. The MAPK/ERK pathway was mostly affected with KRAS mutated in 47% of patients. Acquisition and loss of mutations were observed in 63% and 37% of patients, respectively. Four...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35896960"
                        open
                      >
                        <summary class="article-expander__title">Proteomic and phosphoproteomic measurements enhance ability to predict ex vivo drug response in AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35896960/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220811211442&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute Myeloid Leukemia (AML) affects 20,000 patients in the US annually with a five-year survival rate of approximately 25%. One reason for the low survival rate is the high prevalence of clonal evolution that gives rise to heterogeneous sub-populations of leukemic cells with diverse mutation spectra, which eventually leads to disease relapse. This genetic heterogeneity drives the activation of complex signaling pathways that is reflected at the protein level. This diversity makes it difficult...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35893374"
                        open
                      >
                        <summary class="article-expander__title">In Pursuit of Genetic Prognostic Factors and Treatment Approaches in Secondary Acute Myeloid Leukemia-A Narrative Review of Current Knowledge</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35893374/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220811211442&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Secondary acute myeloid leukemia can be divided into two categories: AML evolving from the antecedent hematological condition (AHD-AML) and therapy related AML (t-AML). AHD-AML can evolve from hematological conditions such as myelodysplastic syndromes, myeloproliferative neoplasms, MDS/MPN overlap syndromes, Fanconi anemia, and aplastic anemia. Leukemic transformation occurs as a consequence of the clonal evolution-a process of the acquisition of mutations in clones, while previous mutations are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35884979"
                        open
                      >
                        <summary class="article-expander__title">Tracking Clonal Evolution of Multiple Myeloma Using Targeted Next-Generation DNA Sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35884979/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220811211442&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution drives treatment failure in multiple myeloma (MM). Here, we used a custom 372-gene panel to track genetic changes occurring during MM progression at different stages of the disease. A tumor-only targeted next-generation DNA sequencing was performed on 69 samples sequentially collected from 30 MM patients. The MAPK/ERK pathway was mostly affected with KRAS mutated in 47% of patients. Acquisition and loss of mutations were observed in 63% and 37% of patients, respectively. Four...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35896960"
                        open
                      >
                        <summary class="article-expander__title">Proteomic and phosphoproteomic measurements enhance ability to predict ex vivo drug response in AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35896960/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220810210900&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute Myeloid Leukemia (AML) affects 20,000 patients in the US annually with a five-year survival rate of approximately 25%. One reason for the low survival rate is the high prevalence of clonal evolution that gives rise to heterogeneous sub-populations of leukemic cells with diverse mutation spectra, which eventually leads to disease relapse. This genetic heterogeneity drives the activation of complex signaling pathways that is reflected at the protein level. This diversity makes it difficult...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35893374"
                        open
                      >
                        <summary class="article-expander__title">In Pursuit of Genetic Prognostic Factors and Treatment Approaches in Secondary Acute Myeloid Leukemia-A Narrative Review of Current Knowledge</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35893374/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220810210900&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Secondary acute myeloid leukemia can be divided into two categories: AML evolving from the antecedent hematological condition (AHD-AML) and therapy related AML (t-AML). AHD-AML can evolve from hematological conditions such as myelodysplastic syndromes, myeloproliferative neoplasms, MDS/MPN overlap syndromes, Fanconi anemia, and aplastic anemia. Leukemic transformation occurs as a consequence of the clonal evolution-a process of the acquisition of mutations in clones, while previous mutations are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35884979"
                        open
                      >
                        <summary class="article-expander__title">Tracking Clonal Evolution of Multiple Myeloma Using Targeted Next-Generation DNA Sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35884979/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220810210900&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution drives treatment failure in multiple myeloma (MM). Here, we used a custom 372-gene panel to track genetic changes occurring during MM progression at different stages of the disease. A tumor-only targeted next-generation DNA sequencing was performed on 69 samples sequentially collected from 30 MM patients. The MAPK/ERK pathway was mostly affected with KRAS mutated in 47% of patients. Acquisition and loss of mutations were observed in 63% and 37% of patients, respectively. Four...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35896960"
                        open
                      >
                        <summary class="article-expander__title">Proteomic and phosphoproteomic measurements enhance ability to predict ex vivo drug response in AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35896960/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220809211355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute Myeloid Leukemia (AML) affects 20,000 patients in the US annually with a five-year survival rate of approximately 25%. One reason for the low survival rate is the high prevalence of clonal evolution that gives rise to heterogeneous sub-populations of leukemic cells with diverse mutation spectra, which eventually leads to disease relapse. This genetic heterogeneity drives the activation of complex signaling pathways that is reflected at the protein level. This diversity makes it difficult...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35893374"
                        open
                      >
                        <summary class="article-expander__title">In Pursuit of Genetic Prognostic Factors and Treatment Approaches in Secondary Acute Myeloid Leukemia-A Narrative Review of Current Knowledge</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35893374/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220809211355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Secondary acute myeloid leukemia can be divided into two categories: AML evolving from the antecedent hematological condition (AHD-AML) and therapy related AML (t-AML). AHD-AML can evolve from hematological conditions such as myelodysplastic syndromes, myeloproliferative neoplasms, MDS/MPN overlap syndromes, Fanconi anemia, and aplastic anemia. Leukemic transformation occurs as a consequence of the clonal evolution-a process of the acquisition of mutations in clones, while previous mutations are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35884979"
                        open
                      >
                        <summary class="article-expander__title">Tracking Clonal Evolution of Multiple Myeloma Using Targeted Next-Generation DNA Sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35884979/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220809211355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution drives treatment failure in multiple myeloma (MM). Here, we used a custom 372-gene panel to track genetic changes occurring during MM progression at different stages of the disease. A tumor-only targeted next-generation DNA sequencing was performed on 69 samples sequentially collected from 30 MM patients. The MAPK/ERK pathway was mostly affected with KRAS mutated in 47% of patients. Acquisition and loss of mutations were observed in 63% and 37% of patients, respectively. Four...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35896960"
                        open
                      >
                        <summary class="article-expander__title">Proteomic and phosphoproteomic measurements enhance ability to predict ex vivo drug response in AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35896960/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220808212152&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute Myeloid Leukemia (AML) affects 20,000 patients in the US annually with a five-year survival rate of approximately 25%. One reason for the low survival rate is the high prevalence of clonal evolution that gives rise to heterogeneous sub-populations of leukemic cells with diverse mutation spectra, which eventually leads to disease relapse. This genetic heterogeneity drives the activation of complex signaling pathways that is reflected at the protein level. This diversity makes it difficult...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35893374"
                        open
                      >
                        <summary class="article-expander__title">In Pursuit of Genetic Prognostic Factors and Treatment Approaches in Secondary Acute Myeloid Leukemia-A Narrative Review of Current Knowledge</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35893374/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220808212152&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Secondary acute myeloid leukemia can be divided into two categories: AML evolving from the antecedent hematological condition (AHD-AML) and therapy related AML (t-AML). AHD-AML can evolve from hematological conditions such as myelodysplastic syndromes, myeloproliferative neoplasms, MDS/MPN overlap syndromes, Fanconi anemia, and aplastic anemia. Leukemic transformation occurs as a consequence of the clonal evolution-a process of the acquisition of mutations in clones, while previous mutations are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35884979"
                        open
                      >
                        <summary class="article-expander__title">Tracking Clonal Evolution of Multiple Myeloma Using Targeted Next-Generation DNA Sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35884979/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220808212152&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution drives treatment failure in multiple myeloma (MM). Here, we used a custom 372-gene panel to track genetic changes occurring during MM progression at different stages of the disease. A tumor-only targeted next-generation DNA sequencing was performed on 69 samples sequentially collected from 30 MM patients. The MAPK/ERK pathway was mostly affected with KRAS mutated in 47% of patients. Acquisition and loss of mutations were observed in 63% and 37% of patients, respectively. Four...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35896960"
                        open
                      >
                        <summary class="article-expander__title">Proteomic and phosphoproteomic measurements enhance ability to predict ex vivo drug response in AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35896960/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220807211336&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute Myeloid Leukemia (AML) affects 20,000 patients in the US annually with a five-year survival rate of approximately 25%. One reason for the low survival rate is the high prevalence of clonal evolution that gives rise to heterogeneous sub-populations of leukemic cells with diverse mutation spectra, which eventually leads to disease relapse. This genetic heterogeneity drives the activation of complex signaling pathways that is reflected at the protein level. This diversity makes it difficult...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35893374"
                        open
                      >
                        <summary class="article-expander__title">In Pursuit of Genetic Prognostic Factors and Treatment Approaches in Secondary Acute Myeloid Leukemia-A Narrative Review of Current Knowledge</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35893374/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220807211336&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Secondary acute myeloid leukemia can be divided into two categories: AML evolving from the antecedent hematological condition (AHD-AML) and therapy related AML (t-AML). AHD-AML can evolve from hematological conditions such as myelodysplastic syndromes, myeloproliferative neoplasms, MDS/MPN overlap syndromes, Fanconi anemia, and aplastic anemia. Leukemic transformation occurs as a consequence of the clonal evolution-a process of the acquisition of mutations in clones, while previous mutations are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35884979"
                        open
                      >
                        <summary class="article-expander__title">Tracking Clonal Evolution of Multiple Myeloma Using Targeted Next-Generation DNA Sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35884979/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220807211336&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution drives treatment failure in multiple myeloma (MM). Here, we used a custom 372-gene panel to track genetic changes occurring during MM progression at different stages of the disease. A tumor-only targeted next-generation DNA sequencing was performed on 69 samples sequentially collected from 30 MM patients. The MAPK/ERK pathway was mostly affected with KRAS mutated in 47% of patients. Acquisition and loss of mutations were observed in 63% and 37% of patients, respectively. Four...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35896960"
                        open
                      >
                        <summary class="article-expander__title">Proteomic and phosphoproteomic measurements enhance ability to predict ex vivo drug response in AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35896960/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220806211830&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute Myeloid Leukemia (AML) affects 20,000 patients in the US annually with a five-year survival rate of approximately 25%. One reason for the low survival rate is the high prevalence of clonal evolution that gives rise to heterogeneous sub-populations of leukemic cells with diverse mutation spectra, which eventually leads to disease relapse. This genetic heterogeneity drives the activation of complex signaling pathways that is reflected at the protein level. This diversity makes it difficult...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35893374"
                        open
                      >
                        <summary class="article-expander__title">In Pursuit of Genetic Prognostic Factors and Treatment Approaches in Secondary Acute Myeloid Leukemia-A Narrative Review of Current Knowledge</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35893374/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220806211830&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Secondary acute myeloid leukemia can be divided into two categories: AML evolving from the antecedent hematological condition (AHD-AML) and therapy related AML (t-AML). AHD-AML can evolve from hematological conditions such as myelodysplastic syndromes, myeloproliferative neoplasms, MDS/MPN overlap syndromes, Fanconi anemia, and aplastic anemia. Leukemic transformation occurs as a consequence of the clonal evolution-a process of the acquisition of mutations in clones, while previous mutations are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35884979"
                        open
                      >
                        <summary class="article-expander__title">Tracking Clonal Evolution of Multiple Myeloma Using Targeted Next-Generation DNA Sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35884979/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220806211830&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution drives treatment failure in multiple myeloma (MM). Here, we used a custom 372-gene panel to track genetic changes occurring during MM progression at different stages of the disease. A tumor-only targeted next-generation DNA sequencing was performed on 69 samples sequentially collected from 30 MM patients. The MAPK/ERK pathway was mostly affected with KRAS mutated in 47% of patients. Acquisition and loss of mutations were observed in 63% and 37% of patients, respectively. Four...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35896960"
                        open
                      >
                        <summary class="article-expander__title">Proteomic and phosphoproteomic measurements enhance ability to predict ex vivo drug response in AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35896960/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220805210849&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute Myeloid Leukemia (AML) affects 20,000 patients in the US annually with a five-year survival rate of approximately 25%. One reason for the low survival rate is the high prevalence of clonal evolution that gives rise to heterogeneous sub-populations of leukemic cells with diverse mutation spectra, which eventually leads to disease relapse. This genetic heterogeneity drives the activation of complex signaling pathways that is reflected at the protein level. This diversity makes it difficult...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35893374"
                        open
                      >
                        <summary class="article-expander__title">In Pursuit of Genetic Prognostic Factors and Treatment Approaches in Secondary Acute Myeloid Leukemia-A Narrative Review of Current Knowledge</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35893374/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220805210849&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Secondary acute myeloid leukemia can be divided into two categories: AML evolving from the antecedent hematological condition (AHD-AML) and therapy related AML (t-AML). AHD-AML can evolve from hematological conditions such as myelodysplastic syndromes, myeloproliferative neoplasms, MDS/MPN overlap syndromes, Fanconi anemia, and aplastic anemia. Leukemic transformation occurs as a consequence of the clonal evolution-a process of the acquisition of mutations in clones, while previous mutations are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35884979"
                        open
                      >
                        <summary class="article-expander__title">Tracking Clonal Evolution of Multiple Myeloma Using Targeted Next-Generation DNA Sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35884979/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220805210849&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution drives treatment failure in multiple myeloma (MM). Here, we used a custom 372-gene panel to track genetic changes occurring during MM progression at different stages of the disease. A tumor-only targeted next-generation DNA sequencing was performed on 69 samples sequentially collected from 30 MM patients. The MAPK/ERK pathway was mostly affected with KRAS mutated in 47% of patients. Acquisition and loss of mutations were observed in 63% and 37% of patients, respectively. Four...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35896960"
                        open
                      >
                        <summary class="article-expander__title">Proteomic and phosphoproteomic measurements enhance ability to predict ex vivo drug response in AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35896960/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220804212147&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute Myeloid Leukemia (AML) affects 20,000 patients in the US annually with a five-year survival rate of approximately 25%. One reason for the low survival rate is the high prevalence of clonal evolution that gives rise to heterogeneous sub-populations of leukemic cells with diverse mutation spectra, which eventually leads to disease relapse. This genetic heterogeneity drives the activation of complex signaling pathways that is reflected at the protein level. This diversity makes it difficult...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35893374"
                        open
                      >
                        <summary class="article-expander__title">In Pursuit of Genetic Prognostic Factors and Treatment Approaches in Secondary Acute Myeloid Leukemia-A Narrative Review of Current Knowledge</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35893374/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220804212147&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Secondary acute myeloid leukemia can be divided into two categories: AML evolving from the antecedent hematological condition (AHD-AML) and therapy related AML (t-AML). AHD-AML can evolve from hematological conditions such as myelodysplastic syndromes, myeloproliferative neoplasms, MDS/MPN overlap syndromes, Fanconi anemia, and aplastic anemia. Leukemic transformation occurs as a consequence of the clonal evolution-a process of the acquisition of mutations in clones, while previous mutations are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35884979"
                        open
                      >
                        <summary class="article-expander__title">Tracking Clonal Evolution of Multiple Myeloma Using Targeted Next-Generation DNA Sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35884979/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220804212147&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution drives treatment failure in multiple myeloma (MM). Here, we used a custom 372-gene panel to track genetic changes occurring during MM progression at different stages of the disease. A tumor-only targeted next-generation DNA sequencing was performed on 69 samples sequentially collected from 30 MM patients. The MAPK/ERK pathway was mostly affected with KRAS mutated in 47% of patients. Acquisition and loss of mutations were observed in 63% and 37% of patients, respectively. Four...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35896960"
                        open
                      >
                        <summary class="article-expander__title">Proteomic and phosphoproteomic measurements enhance ability to predict ex vivo drug response in AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35896960/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220803212051&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute Myeloid Leukemia (AML) affects 20,000 patients in the US annually with a five-year survival rate of approximately 25%. One reason for the low survival rate is the high prevalence of clonal evolution that gives rise to heterogeneous sub-populations of leukemic cells with diverse mutation spectra, which eventually leads to disease relapse. This genetic heterogeneity drives the activation of complex signaling pathways that is reflected at the protein level. This diversity makes it difficult...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35893374"
                        open
                      >
                        <summary class="article-expander__title">In Pursuit of Genetic Prognostic Factors and Treatment Approaches in Secondary Acute Myeloid Leukemia-A Narrative Review of Current Knowledge</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35893374/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220803212051&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Secondary acute myeloid leukemia can be divided into two categories: AML evolving from the antecedent hematological condition (AHD-AML) and therapy related AML (t-AML). AHD-AML can evolve from hematological conditions such as myelodysplastic syndromes, myeloproliferative neoplasms, MDS/MPN overlap syndromes, Fanconi anemia, and aplastic anemia. Leukemic transformation occurs as a consequence of the clonal evolution-a process of the acquisition of mutations in clones, while previous mutations are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35884979"
                        open
                      >
                        <summary class="article-expander__title">Tracking Clonal Evolution of Multiple Myeloma Using Targeted Next-Generation DNA Sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35884979/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220803212051&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution drives treatment failure in multiple myeloma (MM). Here, we used a custom 372-gene panel to track genetic changes occurring during MM progression at different stages of the disease. A tumor-only targeted next-generation DNA sequencing was performed on 69 samples sequentially collected from 30 MM patients. The MAPK/ERK pathway was mostly affected with KRAS mutated in 47% of patients. Acquisition and loss of mutations were observed in 63% and 37% of patients, respectively. Four...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35896960"
                        open
                      >
                        <summary class="article-expander__title">Proteomic and phosphoproteomic measurements enhance ability to predict ex vivo drug response in AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35896960/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220802212153&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute Myeloid Leukemia (AML) affects 20,000 patients in the US annually with a five-year survival rate of approximately 25%. One reason for the low survival rate is the high prevalence of clonal evolution that gives rise to heterogeneous sub-populations of leukemic cells with diverse mutation spectra, which eventually leads to disease relapse. This genetic heterogeneity drives the activation of complex signaling pathways that is reflected at the protein level. This diversity makes it difficult...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35893374"
                        open
                      >
                        <summary class="article-expander__title">In Pursuit of Genetic Prognostic Factors and Treatment Approaches in Secondary Acute Myeloid Leukemia-A Narrative Review of Current Knowledge</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35893374/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220802212153&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Secondary acute myeloid leukemia can be divided into two categories: AML evolving from the antecedent hematological condition (AHD-AML) and therapy related AML (t-AML). AHD-AML can evolve from hematological conditions such as myelodysplastic syndromes, myeloproliferative neoplasms, MDS/MPN overlap syndromes, Fanconi anemia, and aplastic anemia. Leukemic transformation occurs as a consequence of the clonal evolution-a process of the acquisition of mutations in clones, while previous mutations are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35884979"
                        open
                      >
                        <summary class="article-expander__title">Tracking Clonal Evolution of Multiple Myeloma Using Targeted Next-Generation DNA Sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35884979/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220802212153&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution drives treatment failure in multiple myeloma (MM). Here, we used a custom 372-gene panel to track genetic changes occurring during MM progression at different stages of the disease. A tumor-only targeted next-generation DNA sequencing was performed on 69 samples sequentially collected from 30 MM patients. The MAPK/ERK pathway was mostly affected with KRAS mutated in 47% of patients. Acquisition and loss of mutations were observed in 63% and 37% of patients, respectively. Four...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35896960"
                        open
                      >
                        <summary class="article-expander__title">Proteomic and phosphoproteomic measurements enhance ability to predict ex vivo drug response in AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35896960/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220801212447&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute Myeloid Leukemia (AML) affects 20,000 patients in the US annually with a five-year survival rate of approximately 25%. One reason for the low survival rate is the high prevalence of clonal evolution that gives rise to heterogeneous sub-populations of leukemic cells with diverse mutation spectra, which eventually leads to disease relapse. This genetic heterogeneity drives the activation of complex signaling pathways that is reflected at the protein level. This diversity makes it difficult...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35893374"
                        open
                      >
                        <summary class="article-expander__title">In Pursuit of Genetic Prognostic Factors and Treatment Approaches in Secondary Acute Myeloid Leukemia-A Narrative Review of Current Knowledge</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35893374/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220801212447&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Secondary acute myeloid leukemia can be divided into two categories: AML evolving from the antecedent hematological condition (AHD-AML) and therapy related AML (t-AML). AHD-AML can evolve from hematological conditions such as myelodysplastic syndromes, myeloproliferative neoplasms, MDS/MPN overlap syndromes, Fanconi anemia, and aplastic anemia. Leukemic transformation occurs as a consequence of the clonal evolution-a process of the acquisition of mutations in clones, while previous mutations are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35884979"
                        open
                      >
                        <summary class="article-expander__title">Tracking Clonal Evolution of Multiple Myeloma Using Targeted Next-Generation DNA Sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35884979/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220801212447&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution drives treatment failure in multiple myeloma (MM). Here, we used a custom 372-gene panel to track genetic changes occurring during MM progression at different stages of the disease. A tumor-only targeted next-generation DNA sequencing was performed on 69 samples sequentially collected from 30 MM patients. The MAPK/ERK pathway was mostly affected with KRAS mutated in 47% of patients. Acquisition and loss of mutations were observed in 63% and 37% of patients, respectively. Four...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35896960"
                        open
                      >
                        <summary class="article-expander__title">Proteomic and phosphoproteomic measurements enhance ability to predict ex vivo drug response in AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35896960/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220731212525&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute Myeloid Leukemia (AML) affects 20,000 patients in the US annually with a five-year survival rate of approximately 25%. One reason for the low survival rate is the high prevalence of clonal evolution that gives rise to heterogeneous sub-populations of leukemic cells with diverse mutation spectra, which eventually leads to disease relapse. This genetic heterogeneity drives the activation of complex signaling pathways that is reflected at the protein level. This diversity makes it difficult...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35893374"
                        open
                      >
                        <summary class="article-expander__title">In Pursuit of Genetic Prognostic Factors and Treatment Approaches in Secondary Acute Myeloid Leukemia-A Narrative Review of Current Knowledge</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35893374/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220731212525&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Secondary acute myeloid leukemia can be divided into two categories: AML evolving from the antecedent hematological condition (AHD-AML) and therapy related AML (t-AML). AHD-AML can evolve from hematological conditions such as myelodysplastic syndromes, myeloproliferative neoplasms, MDS/MPN overlap syndromes, Fanconi anemia, and aplastic anemia. Leukemic transformation occurs as a consequence of the clonal evolution-a process of the acquisition of mutations in clones, while previous mutations are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35884979"
                        open
                      >
                        <summary class="article-expander__title">Tracking Clonal Evolution of Multiple Myeloma Using Targeted Next-Generation DNA Sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35884979/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220731212525&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution drives treatment failure in multiple myeloma (MM). Here, we used a custom 372-gene panel to track genetic changes occurring during MM progression at different stages of the disease. A tumor-only targeted next-generation DNA sequencing was performed on 69 samples sequentially collected from 30 MM patients. The MAPK/ERK pathway was mostly affected with KRAS mutated in 47% of patients. Acquisition and loss of mutations were observed in 63% and 37% of patients, respectively. Four...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35896960"
                        open
                      >
                        <summary class="article-expander__title">Proteomic and phosphoproteomic measurements enhance ability to predict ex vivo drug response in AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35896960/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220730212309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute Myeloid Leukemia (AML) affects 20,000 patients in the US annually with a five-year survival rate of approximately 25%. One reason for the low survival rate is the high prevalence of clonal evolution that gives rise to heterogeneous sub-populations of leukemic cells with diverse mutation spectra, which eventually leads to disease relapse. This genetic heterogeneity drives the activation of complex signaling pathways that is reflected at the protein level. This diversity makes it difficult...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35893374"
                        open
                      >
                        <summary class="article-expander__title">In Pursuit of Genetic Prognostic Factors and Treatment Approaches in Secondary Acute Myeloid Leukemia-A Narrative Review of Current Knowledge</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35893374/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220730212309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Secondary acute myeloid leukemia can be divided into two categories: AML evolving from the antecedent hematological condition (AHD-AML) and therapy related AML (t-AML). AHD-AML can evolve from hematological conditions such as myelodysplastic syndromes, myeloproliferative neoplasms, MDS/MPN overlap syndromes, Fanconi anemia, and aplastic anemia. Leukemic transformation occurs as a consequence of the clonal evolution-a process of the acquisition of mutations in clones, while previous mutations are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35884979"
                        open
                      >
                        <summary class="article-expander__title">Tracking Clonal Evolution of Multiple Myeloma Using Targeted Next-Generation DNA Sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35884979/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220730212309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution drives treatment failure in multiple myeloma (MM). Here, we used a custom 372-gene panel to track genetic changes occurring during MM progression at different stages of the disease. A tumor-only targeted next-generation DNA sequencing was performed on 69 samples sequentially collected from 30 MM patients. The MAPK/ERK pathway was mostly affected with KRAS mutated in 47% of patients. Acquisition and loss of mutations were observed in 63% and 37% of patients, respectively. Four...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35896960"
                        open
                      >
                        <summary class="article-expander__title">Proteomic and phosphoproteomic measurements enhance ability to predict ex vivo drug response in AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35896960/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220729211332&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute Myeloid Leukemia (AML) affects 20,000 patients in the US annually with a five-year survival rate of approximately 25%. One reason for the low survival rate is the high prevalence of clonal evolution that gives rise to heterogeneous sub-populations of leukemic cells with diverse mutation spectra, which eventually leads to disease relapse. This genetic heterogeneity drives the activation of complex signaling pathways that is reflected at the protein level. This diversity makes it difficult...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35893374"
                        open
                      >
                        <summary class="article-expander__title">In Pursuit of Genetic Prognostic Factors and Treatment Approaches in Secondary Acute Myeloid Leukemia-A Narrative Review of Current Knowledge</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35893374/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220729211332&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Secondary acute myeloid leukemia can be divided into two categories: AML evolving from the antecedent hematological condition (AHD-AML) and therapy related AML (t-AML). AHD-AML can evolve from hematological conditions such as myelodysplastic syndromes, myeloproliferative neoplasms, MDS/MPN overlap syndromes, Fanconi anemia, and aplastic anemia. Leukemic transformation occurs as a consequence of the clonal evolution-a process of the acquisition of mutations in clones, while previous mutations are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35884979"
                        open
                      >
                        <summary class="article-expander__title">Tracking Clonal Evolution of Multiple Myeloma Using Targeted Next-Generation DNA Sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35884979/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220729211332&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution drives treatment failure in multiple myeloma (MM). Here, we used a custom 372-gene panel to track genetic changes occurring during MM progression at different stages of the disease. A tumor-only targeted next-generation DNA sequencing was performed on 69 samples sequentially collected from 30 MM patients. The MAPK/ERK pathway was mostly affected with KRAS mutated in 47% of patients. Acquisition and loss of mutations were observed in 63% and 37% of patients, respectively. Four...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35896960"
                        open
                      >
                        <summary class="article-expander__title">Proteomic and phosphoproteomic measurements enhance ability to predict ex vivo drug response in AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35896960/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220728211733&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute Myeloid Leukemia (AML) affects 20,000 patients in the US annually with a five-year survival rate of approximately 25%. One reason for the low survival rate is the high prevalence of clonal evolution that gives rise to heterogeneous sub-populations of leukemic cells with diverse mutation spectra, which eventually leads to disease relapse. This genetic heterogeneity drives the activation of complex signaling pathways that is reflected at the protein level. This diversity makes it difficult...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35893374"
                        open
                      >
                        <summary class="article-expander__title">In Pursuit of Genetic Prognostic Factors and Treatment Approaches in Secondary Acute Myeloid Leukemia-A Narrative Review of Current Knowledge</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35893374/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220728211733&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Secondary acute myeloid leukemia can be divided into two categories: AML evolving from the antecedent hematological condition (AHD-AML) and therapy related AML (t-AML). AHD-AML can evolve from hematological conditions such as myelodysplastic syndromes, myeloproliferative neoplasms, MDS/MPN overlap syndromes, Fanconi anemia, and aplastic anemia. Leukemic transformation occurs as a consequence of the clonal evolution-a process of the acquisition of mutations in clones, while previous mutations are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35884979"
                        open
                      >
                        <summary class="article-expander__title">Tracking Clonal Evolution of Multiple Myeloma Using Targeted Next-Generation DNA Sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35884979/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220728211733&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution drives treatment failure in multiple myeloma (MM). Here, we used a custom 372-gene panel to track genetic changes occurring during MM progression at different stages of the disease. A tumor-only targeted next-generation DNA sequencing was performed on 69 samples sequentially collected from 30 MM patients. The MAPK/ERK pathway was mostly affected with KRAS mutated in 47% of patients. Acquisition and loss of mutations were observed in 63% and 37% of patients, respectively. Four...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35893374"
                        open
                      >
                        <summary class="article-expander__title">In Pursuit of Genetic Prognostic Factors and Treatment Approaches in Secondary Acute Myeloid Leukemia-A Narrative Review of Current Knowledge</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35893374/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220727211458&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Secondary acute myeloid leukemia can be divided into two categories: AML evolving from the antecedent hematological condition (AHD-AML) and therapy related AML (t-AML). AHD-AML can evolve from hematological conditions such as myelodysplastic syndromes, myeloproliferative neoplasms, MDS/MPN overlap syndromes, Fanconi anemia, and aplastic anemia. Leukemic transformation occurs as a consequence of the clonal evolution-a process of the acquisition of mutations in clones, while previous mutations are...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35884979"
                        open
                      >
                        <summary class="article-expander__title">Tracking Clonal Evolution of Multiple Myeloma Using Targeted Next-Generation DNA Sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35884979/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220727211458&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal evolution drives treatment failure in multiple myeloma (MM). Here, we used a custom 372-gene panel to track genetic changes occurring during MM progression at different stages of the disease. A tumor-only targeted next-generation DNA sequencing was performed on 69 samples sequentially collected from 30 MM patients. The MAPK/ERK pathway was mostly affected with KRAS mutated in 47% of patients. Acquisition and loss of mutations were observed in 63% and 37% of patients, respectively. Four...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-07-26 datetime="2022-07-26T10:00:00.000Z">2022-07-26</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-07-26 datetime="2022-07-26T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608010801&amp;v&#x3D;2.17.7&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35882101"
                        open
                      >
                        <summary class="article-expander__title">Timing evolution of lobular breast cancer through phylogenetic analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35882101/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220819211458&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Late distant recurrence is a challenge for the treatment of invasive lobular carcinoma (ILC) of the breast. Despite in-depth characterisation of primary ILC, the molecular landscape of metastatic ILC is still only partially understood.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35882101"
                        open
                      >
                        <summary class="article-expander__title">Timing evolution of lobular breast cancer through phylogenetic analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35882101/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220818211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Late distant recurrence is a challenge for the treatment of invasive lobular carcinoma (ILC) of the breast. Despite in-depth characterisation of primary ILC, the molecular landscape of metastatic ILC is still only partially understood.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35882101"
                        open
                      >
                        <summary class="article-expander__title">Timing evolution of lobular breast cancer through phylogenetic analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35882101/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220817212606&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Late distant recurrence is a challenge for the treatment of invasive lobular carcinoma (ILC) of the breast. Despite in-depth characterisation of primary ILC, the molecular landscape of metastatic ILC is still only partially understood.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35882101"
                        open
                      >
                        <summary class="article-expander__title">Timing evolution of lobular breast cancer through phylogenetic analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35882101/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220816212355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Late distant recurrence is a challenge for the treatment of invasive lobular carcinoma (ILC) of the breast. Despite in-depth characterisation of primary ILC, the molecular landscape of metastatic ILC is still only partially understood.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35882101"
                        open
                      >
                        <summary class="article-expander__title">Timing evolution of lobular breast cancer through phylogenetic analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35882101/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220815212205&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Late distant recurrence is a challenge for the treatment of invasive lobular carcinoma (ILC) of the breast. Despite in-depth characterisation of primary ILC, the molecular landscape of metastatic ILC is still only partially understood.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35882101"
                        open
                      >
                        <summary class="article-expander__title">Timing evolution of lobular breast cancer through phylogenetic analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35882101/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220814211915&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Late distant recurrence is a challenge for the treatment of invasive lobular carcinoma (ILC) of the breast. Despite in-depth characterisation of primary ILC, the molecular landscape of metastatic ILC is still only partially understood.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35882101"
                        open
                      >
                        <summary class="article-expander__title">Timing evolution of lobular breast cancer through phylogenetic analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35882101/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220813211549&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Late distant recurrence is a challenge for the treatment of invasive lobular carcinoma (ILC) of the breast. Despite in-depth characterisation of primary ILC, the molecular landscape of metastatic ILC is still only partially understood.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35882101"
                        open
                      >
                        <summary class="article-expander__title">Timing evolution of lobular breast cancer through phylogenetic analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35882101/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220812210706&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Late distant recurrence is a challenge for the treatment of invasive lobular carcinoma (ILC) of the breast. Despite in-depth characterisation of primary ILC, the molecular landscape of metastatic ILC is still only partially understood.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35882101"
                        open
                      >
                        <summary class="article-expander__title">Timing evolution of lobular breast cancer through phylogenetic analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35882101/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220811211442&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Late distant recurrence is a challenge for the treatment of invasive lobular carcinoma (ILC) of the breast. Despite in-depth characterisation of primary ILC, the molecular landscape of metastatic ILC is still only partially understood.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35882101"
                        open
                      >
                        <summary class="article-expander__title">Timing evolution of lobular breast cancer through phylogenetic analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35882101/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220810210900&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Late distant recurrence is a challenge for the treatment of invasive lobular carcinoma (ILC) of the breast. Despite in-depth characterisation of primary ILC, the molecular landscape of metastatic ILC is still only partially understood.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35882101"
                        open
                      >
                        <summary class="article-expander__title">Timing evolution of lobular breast cancer through phylogenetic analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35882101/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220809211355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Late distant recurrence is a challenge for the treatment of invasive lobular carcinoma (ILC) of the breast. Despite in-depth characterisation of primary ILC, the molecular landscape of metastatic ILC is still only partially understood.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35882101"
                        open
                      >
                        <summary class="article-expander__title">Timing evolution of lobular breast cancer through phylogenetic analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35882101/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220808212152&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Late distant recurrence is a challenge for the treatment of invasive lobular carcinoma (ILC) of the breast. Despite in-depth characterisation of primary ILC, the molecular landscape of metastatic ILC is still only partially understood.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35882101"
                        open
                      >
                        <summary class="article-expander__title">Timing evolution of lobular breast cancer through phylogenetic analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35882101/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220807211336&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Late distant recurrence is a challenge for the treatment of invasive lobular carcinoma (ILC) of the breast. Despite in-depth characterisation of primary ILC, the molecular landscape of metastatic ILC is still only partially understood.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35882101"
                        open
                      >
                        <summary class="article-expander__title">Timing evolution of lobular breast cancer through phylogenetic analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35882101/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220806211830&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Late distant recurrence is a challenge for the treatment of invasive lobular carcinoma (ILC) of the breast. Despite in-depth characterisation of primary ILC, the molecular landscape of metastatic ILC is still only partially understood.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35882101"
                        open
                      >
                        <summary class="article-expander__title">Timing evolution of lobular breast cancer through phylogenetic analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35882101/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220805210849&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Late distant recurrence is a challenge for the treatment of invasive lobular carcinoma (ILC) of the breast. Despite in-depth characterisation of primary ILC, the molecular landscape of metastatic ILC is still only partially understood.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35882101"
                        open
                      >
                        <summary class="article-expander__title">Timing evolution of lobular breast cancer through phylogenetic analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35882101/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220804212147&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Late distant recurrence is a challenge for the treatment of invasive lobular carcinoma (ILC) of the breast. Despite in-depth characterisation of primary ILC, the molecular landscape of metastatic ILC is still only partially understood.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35882101"
                        open
                      >
                        <summary class="article-expander__title">Timing evolution of lobular breast cancer through phylogenetic analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35882101/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220803212051&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Late distant recurrence is a challenge for the treatment of invasive lobular carcinoma (ILC) of the breast. Despite in-depth characterisation of primary ILC, the molecular landscape of metastatic ILC is still only partially understood.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35882101"
                        open
                      >
                        <summary class="article-expander__title">Timing evolution of lobular breast cancer through phylogenetic analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35882101/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220802212153&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Late distant recurrence is a challenge for the treatment of invasive lobular carcinoma (ILC) of the breast. Despite in-depth characterisation of primary ILC, the molecular landscape of metastatic ILC is still only partially understood.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35882101"
                        open
                      >
                        <summary class="article-expander__title">Timing evolution of lobular breast cancer through phylogenetic analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35882101/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220801212447&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Late distant recurrence is a challenge for the treatment of invasive lobular carcinoma (ILC) of the breast. Despite in-depth characterisation of primary ILC, the molecular landscape of metastatic ILC is still only partially understood.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35882101"
                        open
                      >
                        <summary class="article-expander__title">Timing evolution of lobular breast cancer through phylogenetic analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35882101/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220731212525&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Late distant recurrence is a challenge for the treatment of invasive lobular carcinoma (ILC) of the breast. Despite in-depth characterisation of primary ILC, the molecular landscape of metastatic ILC is still only partially understood.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35882101"
                        open
                      >
                        <summary class="article-expander__title">Timing evolution of lobular breast cancer through phylogenetic analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35882101/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220730212309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Late distant recurrence is a challenge for the treatment of invasive lobular carcinoma (ILC) of the breast. Despite in-depth characterisation of primary ILC, the molecular landscape of metastatic ILC is still only partially understood.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35882101"
                        open
                      >
                        <summary class="article-expander__title">Timing evolution of lobular breast cancer through phylogenetic analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35882101/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220729211332&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Late distant recurrence is a challenge for the treatment of invasive lobular carcinoma (ILC) of the breast. Despite in-depth characterisation of primary ILC, the molecular landscape of metastatic ILC is still only partially understood.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35882101"
                        open
                      >
                        <summary class="article-expander__title">Timing evolution of lobular breast cancer through phylogenetic analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35882101/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220728211733&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Late distant recurrence is a challenge for the treatment of invasive lobular carcinoma (ILC) of the breast. Despite in-depth characterisation of primary ILC, the molecular landscape of metastatic ILC is still only partially understood.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35882101"
                        open
                      >
                        <summary class="article-expander__title">Timing evolution of lobular breast cancer through phylogenetic analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35882101/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220727211458&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Late distant recurrence is a challenge for the treatment of invasive lobular carcinoma (ILC) of the breast. Despite in-depth characterisation of primary ILC, the molecular landscape of metastatic ILC is still only partially understood.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-07-25 datetime="2022-07-25T10:00:00.000Z">2022-07-25</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-07-25 datetime="2022-07-25T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608003622&amp;v&#x3D;2.17.7&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35877223"
                        open
                      >
                        <summary class="article-expander__title">Concurrent Waldenstrom&#x27;s Macroglobulinemia and Myelodysplastic Syndrome with a Sequent t(10;13)(p13;q22) Translocation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35877223/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220808212152&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) and Waldenstrom&#x27;s macroglobulinemia (WM) are rarely synchronous. Ineffective myelopoiesis/hematopoiesis with clonal unilineage or multilineage dysplasia and cytopenias characterize MDS. Despite a myeloid origin, MDS can sometimes lead to decreased production, abnormal apoptosis or dysmaturation of B cells, and the development of lymphoma. WM includes bone marrow involvement by lymphoplasmacytic lymphoma (LPL) secreting monoclonal immunoglobulin M (IgM) with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35877223"
                        open
                      >
                        <summary class="article-expander__title">Concurrent Waldenstrom&#x27;s Macroglobulinemia and Myelodysplastic Syndrome with a Sequent t(10;13)(p13;q22) Translocation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35877223/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220807211336&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) and Waldenstrom&#x27;s macroglobulinemia (WM) are rarely synchronous. Ineffective myelopoiesis/hematopoiesis with clonal unilineage or multilineage dysplasia and cytopenias characterize MDS. Despite a myeloid origin, MDS can sometimes lead to decreased production, abnormal apoptosis or dysmaturation of B cells, and the development of lymphoma. WM includes bone marrow involvement by lymphoplasmacytic lymphoma (LPL) secreting monoclonal immunoglobulin M (IgM) with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35876224"
                        open
                      >
                        <summary class="article-expander__title">Expert Panel Consensus Recommendations on the Use of Circulating Tumor DNA Assays for Patients with Advanced Solid Tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35876224/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220807211336&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Comprehensive genomic profiling is increasingly used to facilitate precision oncology based on molecular stratification. In addition to conventional tissue comprehensive genomic profiling, comprehensive genomic profiling of circulating tumor DNA has become widely utilized in cancer care owing on its advantages, including less invasiveness, rapid turnaround time, and capturing heterogeneity. However, circulating tumor DNA comprehensive genomic profiling has some limitations, mainly false...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35877223"
                        open
                      >
                        <summary class="article-expander__title">Concurrent Waldenstrom&#x27;s Macroglobulinemia and Myelodysplastic Syndrome with a Sequent t(10;13)(p13;q22) Translocation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35877223/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220806211830&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) and Waldenstrom&#x27;s macroglobulinemia (WM) are rarely synchronous. Ineffective myelopoiesis/hematopoiesis with clonal unilineage or multilineage dysplasia and cytopenias characterize MDS. Despite a myeloid origin, MDS can sometimes lead to decreased production, abnormal apoptosis or dysmaturation of B cells, and the development of lymphoma. WM includes bone marrow involvement by lymphoplasmacytic lymphoma (LPL) secreting monoclonal immunoglobulin M (IgM) with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35876224"
                        open
                      >
                        <summary class="article-expander__title">Expert Panel Consensus Recommendations on the Use of Circulating Tumor DNA Assays for Patients with Advanced Solid Tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35876224/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220806211830&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Comprehensive genomic profiling is increasingly used to facilitate precision oncology based on molecular stratification. In addition to conventional tissue comprehensive genomic profiling, comprehensive genomic profiling of circulating tumor DNA has become widely utilized in cancer care owing on its advantages, including less invasiveness, rapid turnaround time, and capturing heterogeneity. However, circulating tumor DNA comprehensive genomic profiling has some limitations, mainly false...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35877223"
                        open
                      >
                        <summary class="article-expander__title">Concurrent Waldenstrom&#x27;s Macroglobulinemia and Myelodysplastic Syndrome with a Sequent t(10;13)(p13;q22) Translocation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35877223/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220805210849&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) and Waldenstrom&#x27;s macroglobulinemia (WM) are rarely synchronous. Ineffective myelopoiesis/hematopoiesis with clonal unilineage or multilineage dysplasia and cytopenias characterize MDS. Despite a myeloid origin, MDS can sometimes lead to decreased production, abnormal apoptosis or dysmaturation of B cells, and the development of lymphoma. WM includes bone marrow involvement by lymphoplasmacytic lymphoma (LPL) secreting monoclonal immunoglobulin M (IgM) with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35876224"
                        open
                      >
                        <summary class="article-expander__title">Expert Panel Consensus Recommendations on the Use of Circulating Tumor DNA Assays for Patients with Advanced Solid Tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35876224/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220805210849&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Comprehensive genomic profiling is increasingly used to facilitate precision oncology based on molecular stratification. In addition to conventional tissue comprehensive genomic profiling, comprehensive genomic profiling of circulating tumor DNA has become widely utilized in cancer care owing on its advantages, including less invasiveness, rapid turnaround time, and capturing heterogeneity. However, circulating tumor DNA comprehensive genomic profiling has some limitations, mainly false...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35872970"
                        open
                      >
                        <summary class="article-expander__title">Using All Our Genomes: Blood-based Liquid Biopsies for the Early Detection of Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35872970/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220805210849&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The pursuit of highly sensitive and specific cancer diagnostics based on cell-free (cf) nucleic acids isolated from minimally invasive liquid biopsies has been an area of intense research and commercial effort for at least two decades. Most of these tests detect cancer-specific mutations or epigenetic modifications on circulating DNA derived from tumor cells (ctDNA). Although recent FDA approvals of both single and multi-analyte liquid biopsy companion diagnostic assays are proof of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35877223"
                        open
                      >
                        <summary class="article-expander__title">Concurrent Waldenstrom&#x27;s Macroglobulinemia and Myelodysplastic Syndrome with a Sequent t(10;13)(p13;q22) Translocation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35877223/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220804212147&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) and Waldenstrom&#x27;s macroglobulinemia (WM) are rarely synchronous. Ineffective myelopoiesis/hematopoiesis with clonal unilineage or multilineage dysplasia and cytopenias characterize MDS. Despite a myeloid origin, MDS can sometimes lead to decreased production, abnormal apoptosis or dysmaturation of B cells, and the development of lymphoma. WM includes bone marrow involvement by lymphoplasmacytic lymphoma (LPL) secreting monoclonal immunoglobulin M (IgM) with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35876224"
                        open
                      >
                        <summary class="article-expander__title">Expert Panel Consensus Recommendations on the Use of Circulating Tumor DNA Assays for Patients with Advanced Solid Tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35876224/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220804212147&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Comprehensive genomic profiling is increasingly used to facilitate precision oncology based on molecular stratification. In addition to conventional tissue comprehensive genomic profiling, comprehensive genomic profiling of circulating tumor DNA has become widely utilized in cancer care owing on its advantages, including less invasiveness, rapid turnaround time, and capturing heterogeneity. However, circulating tumor DNA comprehensive genomic profiling has some limitations, mainly false...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35872970"
                        open
                      >
                        <summary class="article-expander__title">Using All Our Genomes: Blood-based Liquid Biopsies for the Early Detection of Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35872970/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220804212147&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The pursuit of highly sensitive and specific cancer diagnostics based on cell-free (cf) nucleic acids isolated from minimally invasive liquid biopsies has been an area of intense research and commercial effort for at least two decades. Most of these tests detect cancer-specific mutations or epigenetic modifications on circulating DNA derived from tumor cells (ctDNA). Although recent FDA approvals of both single and multi-analyte liquid biopsy companion diagnostic assays are proof of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35877223"
                        open
                      >
                        <summary class="article-expander__title">Concurrent Waldenstrom&#x27;s Macroglobulinemia and Myelodysplastic Syndrome with a Sequent t(10;13)(p13;q22) Translocation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35877223/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220803212051&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) and Waldenstrom&#x27;s macroglobulinemia (WM) are rarely synchronous. Ineffective myelopoiesis/hematopoiesis with clonal unilineage or multilineage dysplasia and cytopenias characterize MDS. Despite a myeloid origin, MDS can sometimes lead to decreased production, abnormal apoptosis or dysmaturation of B cells, and the development of lymphoma. WM includes bone marrow involvement by lymphoplasmacytic lymphoma (LPL) secreting monoclonal immunoglobulin M (IgM) with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35876224"
                        open
                      >
                        <summary class="article-expander__title">Expert Panel Consensus Recommendations on the Use of Circulating Tumor DNA Assays for Patients with Advanced Solid Tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35876224/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220803212051&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Comprehensive genomic profiling is increasingly used to facilitate precision oncology based on molecular stratification. In addition to conventional tissue comprehensive genomic profiling, comprehensive genomic profiling of circulating tumor DNA has become widely utilized in cancer care owing on its advantages, including less invasiveness, rapid turnaround time, and capturing heterogeneity. However, circulating tumor DNA comprehensive genomic profiling has some limitations, mainly false...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35872970"
                        open
                      >
                        <summary class="article-expander__title">Using All Our Genomes: Blood-based Liquid Biopsies for the Early Detection of Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35872970/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220803212051&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The pursuit of highly sensitive and specific cancer diagnostics based on cell-free (cf) nucleic acids isolated from minimally invasive liquid biopsies has been an area of intense research and commercial effort for at least two decades. Most of these tests detect cancer-specific mutations or epigenetic modifications on circulating DNA derived from tumor cells (ctDNA). Although recent FDA approvals of both single and multi-analyte liquid biopsy companion diagnostic assays are proof of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35877223"
                        open
                      >
                        <summary class="article-expander__title">Concurrent Waldenstrom&#x27;s Macroglobulinemia and Myelodysplastic Syndrome with a Sequent t(10;13)(p13;q22) Translocation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35877223/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220802212153&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) and Waldenstrom&#x27;s macroglobulinemia (WM) are rarely synchronous. Ineffective myelopoiesis/hematopoiesis with clonal unilineage or multilineage dysplasia and cytopenias characterize MDS. Despite a myeloid origin, MDS can sometimes lead to decreased production, abnormal apoptosis or dysmaturation of B cells, and the development of lymphoma. WM includes bone marrow involvement by lymphoplasmacytic lymphoma (LPL) secreting monoclonal immunoglobulin M (IgM) with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35876224"
                        open
                      >
                        <summary class="article-expander__title">Expert Panel Consensus Recommendations on the Use of Circulating Tumor DNA Assays for Patients with Advanced Solid Tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35876224/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220802212153&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Comprehensive genomic profiling is increasingly used to facilitate precision oncology based on molecular stratification. In addition to conventional tissue comprehensive genomic profiling, comprehensive genomic profiling of circulating tumor DNA has become widely utilized in cancer care owing on its advantages, including less invasiveness, rapid turnaround time, and capturing heterogeneity. However, circulating tumor DNA comprehensive genomic profiling has some limitations, mainly false...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35872970"
                        open
                      >
                        <summary class="article-expander__title">Using All Our Genomes: Blood-based Liquid Biopsies for the Early Detection of Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35872970/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220802212153&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The pursuit of highly sensitive and specific cancer diagnostics based on cell-free (cf) nucleic acids isolated from minimally invasive liquid biopsies has been an area of intense research and commercial effort for at least two decades. Most of these tests detect cancer-specific mutations or epigenetic modifications on circulating DNA derived from tumor cells (ctDNA). Although recent FDA approvals of both single and multi-analyte liquid biopsy companion diagnostic assays are proof of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35877223"
                        open
                      >
                        <summary class="article-expander__title">Concurrent Waldenstrom&#x27;s Macroglobulinemia and Myelodysplastic Syndrome with a Sequent t(10;13)(p13;q22) Translocation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35877223/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220801212447&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) and Waldenstrom&#x27;s macroglobulinemia (WM) are rarely synchronous. Ineffective myelopoiesis/hematopoiesis with clonal unilineage or multilineage dysplasia and cytopenias characterize MDS. Despite a myeloid origin, MDS can sometimes lead to decreased production, abnormal apoptosis or dysmaturation of B cells, and the development of lymphoma. WM includes bone marrow involvement by lymphoplasmacytic lymphoma (LPL) secreting monoclonal immunoglobulin M (IgM) with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35876224"
                        open
                      >
                        <summary class="article-expander__title">Expert Panel Consensus Recommendations on the Use of Circulating Tumor DNA Assays for Patients with Advanced Solid Tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35876224/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220801212447&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Comprehensive genomic profiling is increasingly used to facilitate precision oncology based on molecular stratification. In addition to conventional tissue comprehensive genomic profiling, comprehensive genomic profiling of circulating tumor DNA has become widely utilized in cancer care owing on its advantages, including less invasiveness, rapid turnaround time, and capturing heterogeneity. However, circulating tumor DNA comprehensive genomic profiling has some limitations, mainly false...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35872970"
                        open
                      >
                        <summary class="article-expander__title">Using All Our Genomes: Blood-based Liquid Biopsies for the Early Detection of Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35872970/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220801212447&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The pursuit of highly sensitive and specific cancer diagnostics based on cell-free (cf) nucleic acids isolated from minimally invasive liquid biopsies has been an area of intense research and commercial effort for at least two decades. Most of these tests detect cancer-specific mutations or epigenetic modifications on circulating DNA derived from tumor cells (ctDNA). Although recent FDA approvals of both single and multi-analyte liquid biopsy companion diagnostic assays are proof of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35877223"
                        open
                      >
                        <summary class="article-expander__title">Concurrent Waldenstrom&#x27;s Macroglobulinemia and Myelodysplastic Syndrome with a Sequent t(10;13)(p13;q22) Translocation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35877223/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220731212525&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) and Waldenstrom&#x27;s macroglobulinemia (WM) are rarely synchronous. Ineffective myelopoiesis/hematopoiesis with clonal unilineage or multilineage dysplasia and cytopenias characterize MDS. Despite a myeloid origin, MDS can sometimes lead to decreased production, abnormal apoptosis or dysmaturation of B cells, and the development of lymphoma. WM includes bone marrow involvement by lymphoplasmacytic lymphoma (LPL) secreting monoclonal immunoglobulin M (IgM) with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35876224"
                        open
                      >
                        <summary class="article-expander__title">Expert Panel Consensus Recommendations on the Use of Circulating Tumor DNA Assays for Patients with Advanced Solid Tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35876224/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220731212525&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Comprehensive genomic profiling is increasingly used to facilitate precision oncology based on molecular stratification. In addition to conventional tissue comprehensive genomic profiling, comprehensive genomic profiling of circulating tumor DNA has become widely utilized in cancer care owing on its advantages, including less invasiveness, rapid turnaround time, and capturing heterogeneity. However, circulating tumor DNA comprehensive genomic profiling has some limitations, mainly false...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35872970"
                        open
                      >
                        <summary class="article-expander__title">Using All Our Genomes: Blood-based Liquid Biopsies for the Early Detection of Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35872970/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220731212525&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The pursuit of highly sensitive and specific cancer diagnostics based on cell-free (cf) nucleic acids isolated from minimally invasive liquid biopsies has been an area of intense research and commercial effort for at least two decades. Most of these tests detect cancer-specific mutations or epigenetic modifications on circulating DNA derived from tumor cells (ctDNA). Although recent FDA approvals of both single and multi-analyte liquid biopsy companion diagnostic assays are proof of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35877223"
                        open
                      >
                        <summary class="article-expander__title">Concurrent Waldenstrom&#x27;s Macroglobulinemia and Myelodysplastic Syndrome with a Sequent t(10;13)(p13;q22) Translocation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35877223/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220730212309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) and Waldenstrom&#x27;s macroglobulinemia (WM) are rarely synchronous. Ineffective myelopoiesis/hematopoiesis with clonal unilineage or multilineage dysplasia and cytopenias characterize MDS. Despite a myeloid origin, MDS can sometimes lead to decreased production, abnormal apoptosis or dysmaturation of B cells, and the development of lymphoma. WM includes bone marrow involvement by lymphoplasmacytic lymphoma (LPL) secreting monoclonal immunoglobulin M (IgM) with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35876224"
                        open
                      >
                        <summary class="article-expander__title">Expert Panel Consensus Recommendations on the Use of Circulating Tumor DNA Assays for Patients with Advanced Solid Tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35876224/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220730212309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Comprehensive genomic profiling is increasingly used to facilitate precision oncology based on molecular stratification. In addition to conventional tissue comprehensive genomic profiling, comprehensive genomic profiling of circulating tumor DNA has become widely utilized in cancer care owing on its advantages, including less invasiveness, rapid turnaround time, and capturing heterogeneity. However, circulating tumor DNA comprehensive genomic profiling has some limitations, mainly false...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35872970"
                        open
                      >
                        <summary class="article-expander__title">Using All Our Genomes: Blood-based Liquid Biopsies for the Early Detection of Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35872970/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220730212309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The pursuit of highly sensitive and specific cancer diagnostics based on cell-free (cf) nucleic acids isolated from minimally invasive liquid biopsies has been an area of intense research and commercial effort for at least two decades. Most of these tests detect cancer-specific mutations or epigenetic modifications on circulating DNA derived from tumor cells (ctDNA). Although recent FDA approvals of both single and multi-analyte liquid biopsy companion diagnostic assays are proof of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35877223"
                        open
                      >
                        <summary class="article-expander__title">Concurrent Waldenstrom&#x27;s Macroglobulinemia and Myelodysplastic Syndrome with a Sequent t(10;13)(p13;q22) Translocation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35877223/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220729211332&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) and Waldenstrom&#x27;s macroglobulinemia (WM) are rarely synchronous. Ineffective myelopoiesis/hematopoiesis with clonal unilineage or multilineage dysplasia and cytopenias characterize MDS. Despite a myeloid origin, MDS can sometimes lead to decreased production, abnormal apoptosis or dysmaturation of B cells, and the development of lymphoma. WM includes bone marrow involvement by lymphoplasmacytic lymphoma (LPL) secreting monoclonal immunoglobulin M (IgM) with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35876224"
                        open
                      >
                        <summary class="article-expander__title">Expert Panel Consensus Recommendations on the Use of Circulating Tumor DNA Assays for Patients with Advanced Solid Tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35876224/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220729211332&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Comprehensive genomic profiling is increasingly used to facilitate precision oncology based on molecular stratification. In addition to conventional tissue comprehensive genomic profiling, comprehensive genomic profiling of circulating tumor DNA has become widely utilized in cancer care owing on its advantages, including less invasiveness, rapid turnaround time, and capturing heterogeneity. However, circulating tumor DNA comprehensive genomic profiling has some limitations, mainly false...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35872970"
                        open
                      >
                        <summary class="article-expander__title">Using All Our Genomes: Blood-based Liquid Biopsies for the Early Detection of Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35872970/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220729211332&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The pursuit of highly sensitive and specific cancer diagnostics based on cell-free (cf) nucleic acids isolated from minimally invasive liquid biopsies has been an area of intense research and commercial effort for at least two decades. Most of these tests detect cancer-specific mutations or epigenetic modifications on circulating DNA derived from tumor cells (ctDNA). Although recent FDA approvals of both single and multi-analyte liquid biopsy companion diagnostic assays are proof of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35877223"
                        open
                      >
                        <summary class="article-expander__title">Concurrent Waldenstrom&#x27;s Macroglobulinemia and Myelodysplastic Syndrome with a Sequent t(10;13)(p13;q22) Translocation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35877223/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220728211733&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) and Waldenstrom&#x27;s macroglobulinemia (WM) are rarely synchronous. Ineffective myelopoiesis/hematopoiesis with clonal unilineage or multilineage dysplasia and cytopenias characterize MDS. Despite a myeloid origin, MDS can sometimes lead to decreased production, abnormal apoptosis or dysmaturation of B cells, and the development of lymphoma. WM includes bone marrow involvement by lymphoplasmacytic lymphoma (LPL) secreting monoclonal immunoglobulin M (IgM) with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35876224"
                        open
                      >
                        <summary class="article-expander__title">Expert Panel Consensus Recommendations on the Use of Circulating Tumor DNA Assays for Patients with Advanced Solid Tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35876224/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220728211733&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Comprehensive genomic profiling is increasingly used to facilitate precision oncology based on molecular stratification. In addition to conventional tissue comprehensive genomic profiling, comprehensive genomic profiling of circulating tumor DNA has become widely utilized in cancer care owing on its advantages, including less invasiveness, rapid turnaround time, and capturing heterogeneity. However, circulating tumor DNA comprehensive genomic profiling has some limitations, mainly false...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35872970"
                        open
                      >
                        <summary class="article-expander__title">Using All Our Genomes: Blood-based Liquid Biopsies for the Early Detection of Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35872970/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220728211733&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The pursuit of highly sensitive and specific cancer diagnostics based on cell-free (cf) nucleic acids isolated from minimally invasive liquid biopsies has been an area of intense research and commercial effort for at least two decades. Most of these tests detect cancer-specific mutations or epigenetic modifications on circulating DNA derived from tumor cells (ctDNA). Although recent FDA approvals of both single and multi-analyte liquid biopsy companion diagnostic assays are proof of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35877223"
                        open
                      >
                        <summary class="article-expander__title">Concurrent Waldenstrom&#x27;s Macroglobulinemia and Myelodysplastic Syndrome with a Sequent t(10;13)(p13;q22) Translocation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35877223/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220727211458&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) and Waldenstrom&#x27;s macroglobulinemia (WM) are rarely synchronous. Ineffective myelopoiesis/hematopoiesis with clonal unilineage or multilineage dysplasia and cytopenias characterize MDS. Despite a myeloid origin, MDS can sometimes lead to decreased production, abnormal apoptosis or dysmaturation of B cells, and the development of lymphoma. WM includes bone marrow involvement by lymphoplasmacytic lymphoma (LPL) secreting monoclonal immunoglobulin M (IgM) with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35876224"
                        open
                      >
                        <summary class="article-expander__title">Expert Panel Consensus Recommendations on the Use of Circulating Tumor DNA Assays for Patients with Advanced Solid Tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35876224/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220727211458&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Comprehensive genomic profiling is increasingly used to facilitate precision oncology based on molecular stratification. In addition to conventional tissue comprehensive genomic profiling, comprehensive genomic profiling of circulating tumor DNA has become widely utilized in cancer care owing on its advantages, including less invasiveness, rapid turnaround time, and capturing heterogeneity. However, circulating tumor DNA comprehensive genomic profiling has some limitations, mainly false...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35872970"
                        open
                      >
                        <summary class="article-expander__title">Using All Our Genomes: Blood-based Liquid Biopsies for the Early Detection of Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35872970/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220727211458&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The pursuit of highly sensitive and specific cancer diagnostics based on cell-free (cf) nucleic acids isolated from minimally invasive liquid biopsies has been an area of intense research and commercial effort for at least two decades. Most of these tests detect cancer-specific mutations or epigenetic modifications on circulating DNA derived from tumor cells (ctDNA). Although recent FDA approvals of both single and multi-analyte liquid biopsy companion diagnostic assays are proof of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35877223"
                        open
                      >
                        <summary class="article-expander__title">Concurrent Waldenstrom&#x27;s Macroglobulinemia and Myelodysplastic Syndrome with a Sequent t(10;13)(p13;q22) Translocation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35877223/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220726212041&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) and Waldenstrom&#x27;s macroglobulinemia (WM) are rarely synchronous. Ineffective myelopoiesis/hematopoiesis with clonal unilineage or multilineage dysplasia and cytopenias characterize MDS. Despite a myeloid origin, MDS can sometimes lead to decreased production, abnormal apoptosis or dysmaturation of B cells, and the development of lymphoma. WM includes bone marrow involvement by lymphoplasmacytic lymphoma (LPL) secreting monoclonal immunoglobulin M (IgM) with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35876224"
                        open
                      >
                        <summary class="article-expander__title">Expert Panel Consensus Recommendations on the Use of Circulating Tumor DNA Assays for Patients with Advanced Solid Tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35876224/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220726212041&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Comprehensive genomic profiling is increasingly used to facilitate precision oncology based on molecular stratification. In addition to conventional tissue comprehensive genomic profiling, comprehensive genomic profiling of circulating tumor DNA has become widely utilized in cancer care owing on its advantages, including less invasiveness, rapid turnaround time, and capturing heterogeneity. However, circulating tumor DNA comprehensive genomic profiling has some limitations, mainly false...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35872970"
                        open
                      >
                        <summary class="article-expander__title">Using All Our Genomes: Blood-based Liquid Biopsies for the Early Detection of Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35872970/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220726212041&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The pursuit of highly sensitive and specific cancer diagnostics based on cell-free (cf) nucleic acids isolated from minimally invasive liquid biopsies has been an area of intense research and commercial effort for at least two decades. Most of these tests detect cancer-specific mutations or epigenetic modifications on circulating DNA derived from tumor cells (ctDNA). Although recent FDA approvals of both single and multi-analyte liquid biopsy companion diagnostic assays are proof of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35877223"
                        open
                      >
                        <summary class="article-expander__title">Concurrent Waldenstrom&#x27;s Macroglobulinemia and Myelodysplastic Syndrome with a Sequent t(10;13)(p13;q22) Translocation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35877223/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220725211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) and Waldenstrom&#x27;s macroglobulinemia (WM) are rarely synchronous. Ineffective myelopoiesis/hematopoiesis with clonal unilineage or multilineage dysplasia and cytopenias characterize MDS. Despite a myeloid origin, MDS can sometimes lead to decreased production, abnormal apoptosis or dysmaturation of B cells, and the development of lymphoma. WM includes bone marrow involvement by lymphoplasmacytic lymphoma (LPL) secreting monoclonal immunoglobulin M (IgM) with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35876224"
                        open
                      >
                        <summary class="article-expander__title">Expert Panel Consensus Recommendations on the Use of Circulating Tumor DNA Assays for Patients with Advanced Solid Tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35876224/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220725211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Comprehensive genomic profiling is increasingly used to facilitate precision oncology based on molecular stratification. In addition to conventional tissue comprehensive genomic profiling, comprehensive genomic profiling of circulating tumor DNA has become widely utilized in cancer care owing on its advantages, including less invasiveness, rapid turnaround time, and capturing heterogeneity. However, circulating tumor DNA comprehensive genomic profiling has some limitations, mainly false...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35872970"
                        open
                      >
                        <summary class="article-expander__title">Using All Our Genomes: Blood-based Liquid Biopsies for the Early Detection of Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35872970/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220725211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The pursuit of highly sensitive and specific cancer diagnostics based on cell-free (cf) nucleic acids isolated from minimally invasive liquid biopsies has been an area of intense research and commercial effort for at least two decades. Most of these tests detect cancer-specific mutations or epigenetic modifications on circulating DNA derived from tumor cells (ctDNA). Although recent FDA approvals of both single and multi-analyte liquid biopsy companion diagnostic assays are proof of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608010801&amp;v&#x3D;2.17.7&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35879542"
                        open
                      >
                        <summary class="article-expander__title">Population genetics of clonally transmissible cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35879542/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220819211458&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Populations of cancer cells are subject to the same core evolutionary processes as asexually reproducing, unicellular organisms. Transmissible cancers are particularly striking examples of these processes. These unusual cancers are clonal lineages that can spread through populations via physical transfer of living cancer cells from one host individual to another, and they have achieved long-term success in the colonization of at least eight different host species. Population genetic theory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35879542"
                        open
                      >
                        <summary class="article-expander__title">Population genetics of clonally transmissible cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35879542/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220818211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Populations of cancer cells are subject to the same core evolutionary processes as asexually reproducing, unicellular organisms. Transmissible cancers are particularly striking examples of these processes. These unusual cancers are clonal lineages that can spread through populations via physical transfer of living cancer cells from one host individual to another, and they have achieved long-term success in the colonization of at least eight different host species. Population genetic theory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35879542"
                        open
                      >
                        <summary class="article-expander__title">Population genetics of clonally transmissible cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35879542/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220817212606&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Populations of cancer cells are subject to the same core evolutionary processes as asexually reproducing, unicellular organisms. Transmissible cancers are particularly striking examples of these processes. These unusual cancers are clonal lineages that can spread through populations via physical transfer of living cancer cells from one host individual to another, and they have achieved long-term success in the colonization of at least eight different host species. Population genetic theory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35879542"
                        open
                      >
                        <summary class="article-expander__title">Population genetics of clonally transmissible cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35879542/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220816212355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Populations of cancer cells are subject to the same core evolutionary processes as asexually reproducing, unicellular organisms. Transmissible cancers are particularly striking examples of these processes. These unusual cancers are clonal lineages that can spread through populations via physical transfer of living cancer cells from one host individual to another, and they have achieved long-term success in the colonization of at least eight different host species. Population genetic theory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35879542"
                        open
                      >
                        <summary class="article-expander__title">Population genetics of clonally transmissible cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35879542/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220815212205&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Populations of cancer cells are subject to the same core evolutionary processes as asexually reproducing, unicellular organisms. Transmissible cancers are particularly striking examples of these processes. These unusual cancers are clonal lineages that can spread through populations via physical transfer of living cancer cells from one host individual to another, and they have achieved long-term success in the colonization of at least eight different host species. Population genetic theory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35879542"
                        open
                      >
                        <summary class="article-expander__title">Population genetics of clonally transmissible cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35879542/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220814211915&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Populations of cancer cells are subject to the same core evolutionary processes as asexually reproducing, unicellular organisms. Transmissible cancers are particularly striking examples of these processes. These unusual cancers are clonal lineages that can spread through populations via physical transfer of living cancer cells from one host individual to another, and they have achieved long-term success in the colonization of at least eight different host species. Population genetic theory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35879542"
                        open
                      >
                        <summary class="article-expander__title">Population genetics of clonally transmissible cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35879542/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220813211549&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Populations of cancer cells are subject to the same core evolutionary processes as asexually reproducing, unicellular organisms. Transmissible cancers are particularly striking examples of these processes. These unusual cancers are clonal lineages that can spread through populations via physical transfer of living cancer cells from one host individual to another, and they have achieved long-term success in the colonization of at least eight different host species. Population genetic theory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35879542"
                        open
                      >
                        <summary class="article-expander__title">Population genetics of clonally transmissible cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35879542/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220812210706&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Populations of cancer cells are subject to the same core evolutionary processes as asexually reproducing, unicellular organisms. Transmissible cancers are particularly striking examples of these processes. These unusual cancers are clonal lineages that can spread through populations via physical transfer of living cancer cells from one host individual to another, and they have achieved long-term success in the colonization of at least eight different host species. Population genetic theory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35879542"
                        open
                      >
                        <summary class="article-expander__title">Population genetics of clonally transmissible cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35879542/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220811211442&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Populations of cancer cells are subject to the same core evolutionary processes as asexually reproducing, unicellular organisms. Transmissible cancers are particularly striking examples of these processes. These unusual cancers are clonal lineages that can spread through populations via physical transfer of living cancer cells from one host individual to another, and they have achieved long-term success in the colonization of at least eight different host species. Population genetic theory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35879542"
                        open
                      >
                        <summary class="article-expander__title">Population genetics of clonally transmissible cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35879542/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220810210900&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Populations of cancer cells are subject to the same core evolutionary processes as asexually reproducing, unicellular organisms. Transmissible cancers are particularly striking examples of these processes. These unusual cancers are clonal lineages that can spread through populations via physical transfer of living cancer cells from one host individual to another, and they have achieved long-term success in the colonization of at least eight different host species. Population genetic theory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35879542"
                        open
                      >
                        <summary class="article-expander__title">Population genetics of clonally transmissible cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35879542/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220809211355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Populations of cancer cells are subject to the same core evolutionary processes as asexually reproducing, unicellular organisms. Transmissible cancers are particularly striking examples of these processes. These unusual cancers are clonal lineages that can spread through populations via physical transfer of living cancer cells from one host individual to another, and they have achieved long-term success in the colonization of at least eight different host species. Population genetic theory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35879542"
                        open
                      >
                        <summary class="article-expander__title">Population genetics of clonally transmissible cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35879542/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220808212152&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Populations of cancer cells are subject to the same core evolutionary processes as asexually reproducing, unicellular organisms. Transmissible cancers are particularly striking examples of these processes. These unusual cancers are clonal lineages that can spread through populations via physical transfer of living cancer cells from one host individual to another, and they have achieved long-term success in the colonization of at least eight different host species. Population genetic theory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35879542"
                        open
                      >
                        <summary class="article-expander__title">Population genetics of clonally transmissible cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35879542/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220807211336&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Populations of cancer cells are subject to the same core evolutionary processes as asexually reproducing, unicellular organisms. Transmissible cancers are particularly striking examples of these processes. These unusual cancers are clonal lineages that can spread through populations via physical transfer of living cancer cells from one host individual to another, and they have achieved long-term success in the colonization of at least eight different host species. Population genetic theory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35879542"
                        open
                      >
                        <summary class="article-expander__title">Population genetics of clonally transmissible cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35879542/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220806211830&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Populations of cancer cells are subject to the same core evolutionary processes as asexually reproducing, unicellular organisms. Transmissible cancers are particularly striking examples of these processes. These unusual cancers are clonal lineages that can spread through populations via physical transfer of living cancer cells from one host individual to another, and they have achieved long-term success in the colonization of at least eight different host species. Population genetic theory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35879542"
                        open
                      >
                        <summary class="article-expander__title">Population genetics of clonally transmissible cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35879542/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220805210849&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Populations of cancer cells are subject to the same core evolutionary processes as asexually reproducing, unicellular organisms. Transmissible cancers are particularly striking examples of these processes. These unusual cancers are clonal lineages that can spread through populations via physical transfer of living cancer cells from one host individual to another, and they have achieved long-term success in the colonization of at least eight different host species. Population genetic theory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-07-23 datetime="2022-07-23T10:00:00.000Z">2022-07-23</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-07-23 datetime="2022-07-23T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608003622&amp;v&#x3D;2.17.7&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35871184"
                        open
                      >
                        <summary class="article-expander__title">Discovering the drivers of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35871184/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220803212051&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes that confer a selective advantage to hematopoietic stem cells (HSCs) drive clonal hematopoiesis (CH). While some CH drivers have been identified, the compendium of all genes able to drive CH upon mutations in HSCs remains incomplete. Exploiting signals of positive selection in blood somatic mutations may be an effective way to identify CH driver genes, analogously to cancer. Using the tumor sample in blood/tumor pairs as reference, we identify blood somatic mutations across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35871184"
                        open
                      >
                        <summary class="article-expander__title">Discovering the drivers of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35871184/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220802212153&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes that confer a selective advantage to hematopoietic stem cells (HSCs) drive clonal hematopoiesis (CH). While some CH drivers have been identified, the compendium of all genes able to drive CH upon mutations in HSCs remains incomplete. Exploiting signals of positive selection in blood somatic mutations may be an effective way to identify CH driver genes, analogously to cancer. Using the tumor sample in blood/tumor pairs as reference, we identify blood somatic mutations across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35871184"
                        open
                      >
                        <summary class="article-expander__title">Discovering the drivers of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35871184/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220801212447&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes that confer a selective advantage to hematopoietic stem cells (HSCs) drive clonal hematopoiesis (CH). While some CH drivers have been identified, the compendium of all genes able to drive CH upon mutations in HSCs remains incomplete. Exploiting signals of positive selection in blood somatic mutations may be an effective way to identify CH driver genes, analogously to cancer. Using the tumor sample in blood/tumor pairs as reference, we identify blood somatic mutations across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35871184"
                        open
                      >
                        <summary class="article-expander__title">Discovering the drivers of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35871184/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220731212525&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes that confer a selective advantage to hematopoietic stem cells (HSCs) drive clonal hematopoiesis (CH). While some CH drivers have been identified, the compendium of all genes able to drive CH upon mutations in HSCs remains incomplete. Exploiting signals of positive selection in blood somatic mutations may be an effective way to identify CH driver genes, analogously to cancer. Using the tumor sample in blood/tumor pairs as reference, we identify blood somatic mutations across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35871184"
                        open
                      >
                        <summary class="article-expander__title">Discovering the drivers of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35871184/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220730212309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes that confer a selective advantage to hematopoietic stem cells (HSCs) drive clonal hematopoiesis (CH). While some CH drivers have been identified, the compendium of all genes able to drive CH upon mutations in HSCs remains incomplete. Exploiting signals of positive selection in blood somatic mutations may be an effective way to identify CH driver genes, analogously to cancer. Using the tumor sample in blood/tumor pairs as reference, we identify blood somatic mutations across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35871184"
                        open
                      >
                        <summary class="article-expander__title">Discovering the drivers of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35871184/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220729211332&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes that confer a selective advantage to hematopoietic stem cells (HSCs) drive clonal hematopoiesis (CH). While some CH drivers have been identified, the compendium of all genes able to drive CH upon mutations in HSCs remains incomplete. Exploiting signals of positive selection in blood somatic mutations may be an effective way to identify CH driver genes, analogously to cancer. Using the tumor sample in blood/tumor pairs as reference, we identify blood somatic mutations across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35871184"
                        open
                      >
                        <summary class="article-expander__title">Discovering the drivers of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35871184/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220728211733&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes that confer a selective advantage to hematopoietic stem cells (HSCs) drive clonal hematopoiesis (CH). While some CH drivers have been identified, the compendium of all genes able to drive CH upon mutations in HSCs remains incomplete. Exploiting signals of positive selection in blood somatic mutations may be an effective way to identify CH driver genes, analogously to cancer. Using the tumor sample in blood/tumor pairs as reference, we identify blood somatic mutations across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35871184"
                        open
                      >
                        <summary class="article-expander__title">Discovering the drivers of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35871184/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220727211458&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes that confer a selective advantage to hematopoietic stem cells (HSCs) drive clonal hematopoiesis (CH). While some CH drivers have been identified, the compendium of all genes able to drive CH upon mutations in HSCs remains incomplete. Exploiting signals of positive selection in blood somatic mutations may be an effective way to identify CH driver genes, analogously to cancer. Using the tumor sample in blood/tumor pairs as reference, we identify blood somatic mutations across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35871184"
                        open
                      >
                        <summary class="article-expander__title">Discovering the drivers of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35871184/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220726212041&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes that confer a selective advantage to hematopoietic stem cells (HSCs) drive clonal hematopoiesis (CH). While some CH drivers have been identified, the compendium of all genes able to drive CH upon mutations in HSCs remains incomplete. Exploiting signals of positive selection in blood somatic mutations may be an effective way to identify CH driver genes, analogously to cancer. Using the tumor sample in blood/tumor pairs as reference, we identify blood somatic mutations across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35871184"
                        open
                      >
                        <summary class="article-expander__title">Discovering the drivers of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35871184/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220725211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes that confer a selective advantage to hematopoietic stem cells (HSCs) drive clonal hematopoiesis (CH). While some CH drivers have been identified, the compendium of all genes able to drive CH upon mutations in HSCs remains incomplete. Exploiting signals of positive selection in blood somatic mutations may be an effective way to identify CH driver genes, analogously to cancer. Using the tumor sample in blood/tumor pairs as reference, we identify blood somatic mutations across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35871184"
                        open
                      >
                        <summary class="article-expander__title">Discovering the drivers of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35871184/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220724211604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes that confer a selective advantage to hematopoietic stem cells (HSCs) drive clonal hematopoiesis (CH). While some CH drivers have been identified, the compendium of all genes able to drive CH upon mutations in HSCs remains incomplete. Exploiting signals of positive selection in blood somatic mutations may be an effective way to identify CH driver genes, analogously to cancer. Using the tumor sample in blood/tumor pairs as reference, we identify blood somatic mutations across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608010801&amp;v&#x3D;2.17.7&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35871184"
                        open
                      >
                        <summary class="article-expander__title">Discovering the drivers of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35871184/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220817212606&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes that confer a selective advantage to hematopoietic stem cells (HSCs) drive clonal hematopoiesis (CH). While some CH drivers have been identified, the compendium of all genes able to drive CH upon mutations in HSCs remains incomplete. Exploiting signals of positive selection in blood somatic mutations may be an effective way to identify CH driver genes, analogously to cancer. Using the tumor sample in blood/tumor pairs as reference, we identify blood somatic mutations across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35871184"
                        open
                      >
                        <summary class="article-expander__title">Discovering the drivers of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35871184/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220816212355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes that confer a selective advantage to hematopoietic stem cells (HSCs) drive clonal hematopoiesis (CH). While some CH drivers have been identified, the compendium of all genes able to drive CH upon mutations in HSCs remains incomplete. Exploiting signals of positive selection in blood somatic mutations may be an effective way to identify CH driver genes, analogously to cancer. Using the tumor sample in blood/tumor pairs as reference, we identify blood somatic mutations across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35871184"
                        open
                      >
                        <summary class="article-expander__title">Discovering the drivers of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35871184/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220815212205&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes that confer a selective advantage to hematopoietic stem cells (HSCs) drive clonal hematopoiesis (CH). While some CH drivers have been identified, the compendium of all genes able to drive CH upon mutations in HSCs remains incomplete. Exploiting signals of positive selection in blood somatic mutations may be an effective way to identify CH driver genes, analogously to cancer. Using the tumor sample in blood/tumor pairs as reference, we identify blood somatic mutations across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35871184"
                        open
                      >
                        <summary class="article-expander__title">Discovering the drivers of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35871184/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220814211915&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes that confer a selective advantage to hematopoietic stem cells (HSCs) drive clonal hematopoiesis (CH). While some CH drivers have been identified, the compendium of all genes able to drive CH upon mutations in HSCs remains incomplete. Exploiting signals of positive selection in blood somatic mutations may be an effective way to identify CH driver genes, analogously to cancer. Using the tumor sample in blood/tumor pairs as reference, we identify blood somatic mutations across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35871184"
                        open
                      >
                        <summary class="article-expander__title">Discovering the drivers of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35871184/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220813211549&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes that confer a selective advantage to hematopoietic stem cells (HSCs) drive clonal hematopoiesis (CH). While some CH drivers have been identified, the compendium of all genes able to drive CH upon mutations in HSCs remains incomplete. Exploiting signals of positive selection in blood somatic mutations may be an effective way to identify CH driver genes, analogously to cancer. Using the tumor sample in blood/tumor pairs as reference, we identify blood somatic mutations across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35871184"
                        open
                      >
                        <summary class="article-expander__title">Discovering the drivers of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35871184/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220812210706&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes that confer a selective advantage to hematopoietic stem cells (HSCs) drive clonal hematopoiesis (CH). While some CH drivers have been identified, the compendium of all genes able to drive CH upon mutations in HSCs remains incomplete. Exploiting signals of positive selection in blood somatic mutations may be an effective way to identify CH driver genes, analogously to cancer. Using the tumor sample in blood/tumor pairs as reference, we identify blood somatic mutations across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35871184"
                        open
                      >
                        <summary class="article-expander__title">Discovering the drivers of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35871184/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220811211442&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes that confer a selective advantage to hematopoietic stem cells (HSCs) drive clonal hematopoiesis (CH). While some CH drivers have been identified, the compendium of all genes able to drive CH upon mutations in HSCs remains incomplete. Exploiting signals of positive selection in blood somatic mutations may be an effective way to identify CH driver genes, analogously to cancer. Using the tumor sample in blood/tumor pairs as reference, we identify blood somatic mutations across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35871184"
                        open
                      >
                        <summary class="article-expander__title">Discovering the drivers of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35871184/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220810210900&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes that confer a selective advantage to hematopoietic stem cells (HSCs) drive clonal hematopoiesis (CH). While some CH drivers have been identified, the compendium of all genes able to drive CH upon mutations in HSCs remains incomplete. Exploiting signals of positive selection in blood somatic mutations may be an effective way to identify CH driver genes, analogously to cancer. Using the tumor sample in blood/tumor pairs as reference, we identify blood somatic mutations across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35871184"
                        open
                      >
                        <summary class="article-expander__title">Discovering the drivers of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35871184/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220809211355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes that confer a selective advantage to hematopoietic stem cells (HSCs) drive clonal hematopoiesis (CH). While some CH drivers have been identified, the compendium of all genes able to drive CH upon mutations in HSCs remains incomplete. Exploiting signals of positive selection in blood somatic mutations may be an effective way to identify CH driver genes, analogously to cancer. Using the tumor sample in blood/tumor pairs as reference, we identify blood somatic mutations across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35871184"
                        open
                      >
                        <summary class="article-expander__title">Discovering the drivers of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35871184/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220808212152&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes that confer a selective advantage to hematopoietic stem cells (HSCs) drive clonal hematopoiesis (CH). While some CH drivers have been identified, the compendium of all genes able to drive CH upon mutations in HSCs remains incomplete. Exploiting signals of positive selection in blood somatic mutations may be an effective way to identify CH driver genes, analogously to cancer. Using the tumor sample in blood/tumor pairs as reference, we identify blood somatic mutations across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35871184"
                        open
                      >
                        <summary class="article-expander__title">Discovering the drivers of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35871184/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220807211336&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes that confer a selective advantage to hematopoietic stem cells (HSCs) drive clonal hematopoiesis (CH). While some CH drivers have been identified, the compendium of all genes able to drive CH upon mutations in HSCs remains incomplete. Exploiting signals of positive selection in blood somatic mutations may be an effective way to identify CH driver genes, analogously to cancer. Using the tumor sample in blood/tumor pairs as reference, we identify blood somatic mutations across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35871184"
                        open
                      >
                        <summary class="article-expander__title">Discovering the drivers of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35871184/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220806211830&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes that confer a selective advantage to hematopoietic stem cells (HSCs) drive clonal hematopoiesis (CH). While some CH drivers have been identified, the compendium of all genes able to drive CH upon mutations in HSCs remains incomplete. Exploiting signals of positive selection in blood somatic mutations may be an effective way to identify CH driver genes, analogously to cancer. Using the tumor sample in blood/tumor pairs as reference, we identify blood somatic mutations across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35871184"
                        open
                      >
                        <summary class="article-expander__title">Discovering the drivers of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35871184/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220805210849&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes that confer a selective advantage to hematopoietic stem cells (HSCs) drive clonal hematopoiesis (CH). While some CH drivers have been identified, the compendium of all genes able to drive CH upon mutations in HSCs remains incomplete. Exploiting signals of positive selection in blood somatic mutations may be an effective way to identify CH driver genes, analogously to cancer. Using the tumor sample in blood/tumor pairs as reference, we identify blood somatic mutations across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35871184"
                        open
                      >
                        <summary class="article-expander__title">Discovering the drivers of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35871184/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220804212147&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes that confer a selective advantage to hematopoietic stem cells (HSCs) drive clonal hematopoiesis (CH). While some CH drivers have been identified, the compendium of all genes able to drive CH upon mutations in HSCs remains incomplete. Exploiting signals of positive selection in blood somatic mutations may be an effective way to identify CH driver genes, analogously to cancer. Using the tumor sample in blood/tumor pairs as reference, we identify blood somatic mutations across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35871184"
                        open
                      >
                        <summary class="article-expander__title">Discovering the drivers of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35871184/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220803212051&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes that confer a selective advantage to hematopoietic stem cells (HSCs) drive clonal hematopoiesis (CH). While some CH drivers have been identified, the compendium of all genes able to drive CH upon mutations in HSCs remains incomplete. Exploiting signals of positive selection in blood somatic mutations may be an effective way to identify CH driver genes, analogously to cancer. Using the tumor sample in blood/tumor pairs as reference, we identify blood somatic mutations across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35871184"
                        open
                      >
                        <summary class="article-expander__title">Discovering the drivers of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35871184/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220802212153&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes that confer a selective advantage to hematopoietic stem cells (HSCs) drive clonal hematopoiesis (CH). While some CH drivers have been identified, the compendium of all genes able to drive CH upon mutations in HSCs remains incomplete. Exploiting signals of positive selection in blood somatic mutations may be an effective way to identify CH driver genes, analogously to cancer. Using the tumor sample in blood/tumor pairs as reference, we identify blood somatic mutations across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35871184"
                        open
                      >
                        <summary class="article-expander__title">Discovering the drivers of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35871184/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220801212447&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes that confer a selective advantage to hematopoietic stem cells (HSCs) drive clonal hematopoiesis (CH). While some CH drivers have been identified, the compendium of all genes able to drive CH upon mutations in HSCs remains incomplete. Exploiting signals of positive selection in blood somatic mutations may be an effective way to identify CH driver genes, analogously to cancer. Using the tumor sample in blood/tumor pairs as reference, we identify blood somatic mutations across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35871184"
                        open
                      >
                        <summary class="article-expander__title">Discovering the drivers of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35871184/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220731212525&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes that confer a selective advantage to hematopoietic stem cells (HSCs) drive clonal hematopoiesis (CH). While some CH drivers have been identified, the compendium of all genes able to drive CH upon mutations in HSCs remains incomplete. Exploiting signals of positive selection in blood somatic mutations may be an effective way to identify CH driver genes, analogously to cancer. Using the tumor sample in blood/tumor pairs as reference, we identify blood somatic mutations across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35871184"
                        open
                      >
                        <summary class="article-expander__title">Discovering the drivers of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35871184/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220730212309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes that confer a selective advantage to hematopoietic stem cells (HSCs) drive clonal hematopoiesis (CH). While some CH drivers have been identified, the compendium of all genes able to drive CH upon mutations in HSCs remains incomplete. Exploiting signals of positive selection in blood somatic mutations may be an effective way to identify CH driver genes, analogously to cancer. Using the tumor sample in blood/tumor pairs as reference, we identify blood somatic mutations across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35871184"
                        open
                      >
                        <summary class="article-expander__title">Discovering the drivers of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35871184/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220729211332&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes that confer a selective advantage to hematopoietic stem cells (HSCs) drive clonal hematopoiesis (CH). While some CH drivers have been identified, the compendium of all genes able to drive CH upon mutations in HSCs remains incomplete. Exploiting signals of positive selection in blood somatic mutations may be an effective way to identify CH driver genes, analogously to cancer. Using the tumor sample in blood/tumor pairs as reference, we identify blood somatic mutations across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35871184"
                        open
                      >
                        <summary class="article-expander__title">Discovering the drivers of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35871184/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220728211733&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes that confer a selective advantage to hematopoietic stem cells (HSCs) drive clonal hematopoiesis (CH). While some CH drivers have been identified, the compendium of all genes able to drive CH upon mutations in HSCs remains incomplete. Exploiting signals of positive selection in blood somatic mutations may be an effective way to identify CH driver genes, analogously to cancer. Using the tumor sample in blood/tumor pairs as reference, we identify blood somatic mutations across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35871184"
                        open
                      >
                        <summary class="article-expander__title">Discovering the drivers of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35871184/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220727211458&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes that confer a selective advantage to hematopoietic stem cells (HSCs) drive clonal hematopoiesis (CH). While some CH drivers have been identified, the compendium of all genes able to drive CH upon mutations in HSCs remains incomplete. Exploiting signals of positive selection in blood somatic mutations may be an effective way to identify CH driver genes, analogously to cancer. Using the tumor sample in blood/tumor pairs as reference, we identify blood somatic mutations across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35871184"
                        open
                      >
                        <summary class="article-expander__title">Discovering the drivers of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35871184/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220726212040&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes that confer a selective advantage to hematopoietic stem cells (HSCs) drive clonal hematopoiesis (CH). While some CH drivers have been identified, the compendium of all genes able to drive CH upon mutations in HSCs remains incomplete. Exploiting signals of positive selection in blood somatic mutations may be an effective way to identify CH driver genes, analogously to cancer. Using the tumor sample in blood/tumor pairs as reference, we identify blood somatic mutations across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35871184"
                        open
                      >
                        <summary class="article-expander__title">Discovering the drivers of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35871184/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220725211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes that confer a selective advantage to hematopoietic stem cells (HSCs) drive clonal hematopoiesis (CH). While some CH drivers have been identified, the compendium of all genes able to drive CH upon mutations in HSCs remains incomplete. Exploiting signals of positive selection in blood somatic mutations may be an effective way to identify CH driver genes, analogously to cancer. Using the tumor sample in blood/tumor pairs as reference, we identify blood somatic mutations across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35871184"
                        open
                      >
                        <summary class="article-expander__title">Discovering the drivers of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35871184/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220724211605&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes that confer a selective advantage to hematopoietic stem cells (HSCs) drive clonal hematopoiesis (CH). While some CH drivers have been identified, the compendium of all genes able to drive CH upon mutations in HSCs remains incomplete. Exploiting signals of positive selection in blood somatic mutations may be an effective way to identify CH driver genes, analogously to cancer. Using the tumor sample in blood/tumor pairs as reference, we identify blood somatic mutations across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-07-22 datetime="2022-07-22T10:00:00.000Z">2022-07-22</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-07-22 datetime="2022-07-22T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608003622&amp;v&#x3D;2.17.7&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35866704"
                        open
                      >
                        <summary class="article-expander__title">Novel Germline Loci Are Associated with Clonal Hematopoiesis Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35866704/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220803212051&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Novel germline loci were associated with the risk of clonal hematopoiesis (CH) within the UK Biobank.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35866704"
                        open
                      >
                        <summary class="article-expander__title">Novel Germline Loci Are Associated with Clonal Hematopoiesis Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35866704/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220802212153&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Novel germline loci were associated with the risk of clonal hematopoiesis (CH) within the UK Biobank.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35866704"
                        open
                      >
                        <summary class="article-expander__title">Novel Germline Loci Are Associated with Clonal Hematopoiesis Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35866704/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220801212447&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Novel germline loci were associated with the risk of clonal hematopoiesis (CH) within the UK Biobank.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35866704"
                        open
                      >
                        <summary class="article-expander__title">Novel Germline Loci Are Associated with Clonal Hematopoiesis Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35866704/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220731212525&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Novel germline loci were associated with the risk of clonal hematopoiesis (CH) within the UK Biobank.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35866704"
                        open
                      >
                        <summary class="article-expander__title">Novel Germline Loci Are Associated with Clonal Hematopoiesis Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35866704/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220730212309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Novel germline loci were associated with the risk of clonal hematopoiesis (CH) within the UK Biobank.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35866704"
                        open
                      >
                        <summary class="article-expander__title">Novel Germline Loci Are Associated with Clonal Hematopoiesis Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35866704/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220729211332&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Novel germline loci were associated with the risk of clonal hematopoiesis (CH) within the UK Biobank.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35866704"
                        open
                      >
                        <summary class="article-expander__title">Novel Germline Loci Are Associated with Clonal Hematopoiesis Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35866704/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220728211733&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Novel germline loci were associated with the risk of clonal hematopoiesis (CH) within the UK Biobank.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35866704"
                        open
                      >
                        <summary class="article-expander__title">Novel Germline Loci Are Associated with Clonal Hematopoiesis Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35866704/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220727211458&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Novel germline loci were associated with the risk of clonal hematopoiesis (CH) within the UK Biobank.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35866704"
                        open
                      >
                        <summary class="article-expander__title">Novel Germline Loci Are Associated with Clonal Hematopoiesis Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35866704/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220726212041&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Novel germline loci were associated with the risk of clonal hematopoiesis (CH) within the UK Biobank.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35866704"
                        open
                      >
                        <summary class="article-expander__title">Novel Germline Loci Are Associated with Clonal Hematopoiesis Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35866704/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220725211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Novel germline loci were associated with the risk of clonal hematopoiesis (CH) within the UK Biobank.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35866704"
                        open
                      >
                        <summary class="article-expander__title">Novel Germline Loci Are Associated with Clonal Hematopoiesis Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35866704/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220724211604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Novel germline loci were associated with the risk of clonal hematopoiesis (CH) within the UK Biobank.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35866704"
                        open
                      >
                        <summary class="article-expander__title">Novel Germline Loci Are Associated with Clonal Hematopoiesis Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35866704/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220723212121&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Novel germline loci were associated with the risk of clonal hematopoiesis (CH) within the UK Biobank.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35866704"
                        open
                      >
                        <summary class="article-expander__title">Novel Germline Loci Are Associated with Clonal Hematopoiesis Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35866704/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220722211347&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Novel germline loci were associated with the risk of clonal hematopoiesis (CH) within the UK Biobank.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608010801&amp;v&#x3D;2.17.7&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35867969"
                        open
                      >
                        <summary class="article-expander__title">Genomic landscapes of canine splenic angiosarcoma (hemangiosarcoma) contain extensive heterogeneity within and between patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35867969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220815212205&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomic heterogeneity presents significant challenges for understanding oncogenic processes and for cancer&#x27;s clinical management. Variation in driver mutation frequency between patients with the same tumor type as well as within an individual patients&#x27; cancer can shape the use of mutations as diagnostic, prognostic, and predictive biomarkers. We have characterized genomic heterogeneity between and within canine splenic hemangiosarcoma (HSA), a common naturally occurring cancer in pet dogs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35867969"
                        open
                      >
                        <summary class="article-expander__title">Genomic landscapes of canine splenic angiosarcoma (hemangiosarcoma) contain extensive heterogeneity within and between patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35867969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220814211915&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomic heterogeneity presents significant challenges for understanding oncogenic processes and for cancer&#x27;s clinical management. Variation in driver mutation frequency between patients with the same tumor type as well as within an individual patients&#x27; cancer can shape the use of mutations as diagnostic, prognostic, and predictive biomarkers. We have characterized genomic heterogeneity between and within canine splenic hemangiosarcoma (HSA), a common naturally occurring cancer in pet dogs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35867969"
                        open
                      >
                        <summary class="article-expander__title">Genomic landscapes of canine splenic angiosarcoma (hemangiosarcoma) contain extensive heterogeneity within and between patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35867969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220813211549&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomic heterogeneity presents significant challenges for understanding oncogenic processes and for cancer&#x27;s clinical management. Variation in driver mutation frequency between patients with the same tumor type as well as within an individual patients&#x27; cancer can shape the use of mutations as diagnostic, prognostic, and predictive biomarkers. We have characterized genomic heterogeneity between and within canine splenic hemangiosarcoma (HSA), a common naturally occurring cancer in pet dogs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35867969"
                        open
                      >
                        <summary class="article-expander__title">Genomic landscapes of canine splenic angiosarcoma (hemangiosarcoma) contain extensive heterogeneity within and between patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35867969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220812210706&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomic heterogeneity presents significant challenges for understanding oncogenic processes and for cancer&#x27;s clinical management. Variation in driver mutation frequency between patients with the same tumor type as well as within an individual patients&#x27; cancer can shape the use of mutations as diagnostic, prognostic, and predictive biomarkers. We have characterized genomic heterogeneity between and within canine splenic hemangiosarcoma (HSA), a common naturally occurring cancer in pet dogs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35867969"
                        open
                      >
                        <summary class="article-expander__title">Genomic landscapes of canine splenic angiosarcoma (hemangiosarcoma) contain extensive heterogeneity within and between patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35867969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220811211442&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomic heterogeneity presents significant challenges for understanding oncogenic processes and for cancer&#x27;s clinical management. Variation in driver mutation frequency between patients with the same tumor type as well as within an individual patients&#x27; cancer can shape the use of mutations as diagnostic, prognostic, and predictive biomarkers. We have characterized genomic heterogeneity between and within canine splenic hemangiosarcoma (HSA), a common naturally occurring cancer in pet dogs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35867969"
                        open
                      >
                        <summary class="article-expander__title">Genomic landscapes of canine splenic angiosarcoma (hemangiosarcoma) contain extensive heterogeneity within and between patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35867969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220810210900&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomic heterogeneity presents significant challenges for understanding oncogenic processes and for cancer&#x27;s clinical management. Variation in driver mutation frequency between patients with the same tumor type as well as within an individual patients&#x27; cancer can shape the use of mutations as diagnostic, prognostic, and predictive biomarkers. We have characterized genomic heterogeneity between and within canine splenic hemangiosarcoma (HSA), a common naturally occurring cancer in pet dogs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35867969"
                        open
                      >
                        <summary class="article-expander__title">Genomic landscapes of canine splenic angiosarcoma (hemangiosarcoma) contain extensive heterogeneity within and between patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35867969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220809211355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomic heterogeneity presents significant challenges for understanding oncogenic processes and for cancer&#x27;s clinical management. Variation in driver mutation frequency between patients with the same tumor type as well as within an individual patients&#x27; cancer can shape the use of mutations as diagnostic, prognostic, and predictive biomarkers. We have characterized genomic heterogeneity between and within canine splenic hemangiosarcoma (HSA), a common naturally occurring cancer in pet dogs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35867969"
                        open
                      >
                        <summary class="article-expander__title">Genomic landscapes of canine splenic angiosarcoma (hemangiosarcoma) contain extensive heterogeneity within and between patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35867969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220808212152&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomic heterogeneity presents significant challenges for understanding oncogenic processes and for cancer&#x27;s clinical management. Variation in driver mutation frequency between patients with the same tumor type as well as within an individual patients&#x27; cancer can shape the use of mutations as diagnostic, prognostic, and predictive biomarkers. We have characterized genomic heterogeneity between and within canine splenic hemangiosarcoma (HSA), a common naturally occurring cancer in pet dogs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35867969"
                        open
                      >
                        <summary class="article-expander__title">Genomic landscapes of canine splenic angiosarcoma (hemangiosarcoma) contain extensive heterogeneity within and between patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35867969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220807211336&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomic heterogeneity presents significant challenges for understanding oncogenic processes and for cancer&#x27;s clinical management. Variation in driver mutation frequency between patients with the same tumor type as well as within an individual patients&#x27; cancer can shape the use of mutations as diagnostic, prognostic, and predictive biomarkers. We have characterized genomic heterogeneity between and within canine splenic hemangiosarcoma (HSA), a common naturally occurring cancer in pet dogs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35867969"
                        open
                      >
                        <summary class="article-expander__title">Genomic landscapes of canine splenic angiosarcoma (hemangiosarcoma) contain extensive heterogeneity within and between patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35867969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220806211830&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomic heterogeneity presents significant challenges for understanding oncogenic processes and for cancer&#x27;s clinical management. Variation in driver mutation frequency between patients with the same tumor type as well as within an individual patients&#x27; cancer can shape the use of mutations as diagnostic, prognostic, and predictive biomarkers. We have characterized genomic heterogeneity between and within canine splenic hemangiosarcoma (HSA), a common naturally occurring cancer in pet dogs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35867969"
                        open
                      >
                        <summary class="article-expander__title">Genomic landscapes of canine splenic angiosarcoma (hemangiosarcoma) contain extensive heterogeneity within and between patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35867969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220805210849&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomic heterogeneity presents significant challenges for understanding oncogenic processes and for cancer&#x27;s clinical management. Variation in driver mutation frequency between patients with the same tumor type as well as within an individual patients&#x27; cancer can shape the use of mutations as diagnostic, prognostic, and predictive biomarkers. We have characterized genomic heterogeneity between and within canine splenic hemangiosarcoma (HSA), a common naturally occurring cancer in pet dogs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35867969"
                        open
                      >
                        <summary class="article-expander__title">Genomic landscapes of canine splenic angiosarcoma (hemangiosarcoma) contain extensive heterogeneity within and between patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35867969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220804212147&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomic heterogeneity presents significant challenges for understanding oncogenic processes and for cancer&#x27;s clinical management. Variation in driver mutation frequency between patients with the same tumor type as well as within an individual patients&#x27; cancer can shape the use of mutations as diagnostic, prognostic, and predictive biomarkers. We have characterized genomic heterogeneity between and within canine splenic hemangiosarcoma (HSA), a common naturally occurring cancer in pet dogs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35867969"
                        open
                      >
                        <summary class="article-expander__title">Genomic landscapes of canine splenic angiosarcoma (hemangiosarcoma) contain extensive heterogeneity within and between patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35867969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220803212051&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomic heterogeneity presents significant challenges for understanding oncogenic processes and for cancer&#x27;s clinical management. Variation in driver mutation frequency between patients with the same tumor type as well as within an individual patients&#x27; cancer can shape the use of mutations as diagnostic, prognostic, and predictive biomarkers. We have characterized genomic heterogeneity between and within canine splenic hemangiosarcoma (HSA), a common naturally occurring cancer in pet dogs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35867969"
                        open
                      >
                        <summary class="article-expander__title">Genomic landscapes of canine splenic angiosarcoma (hemangiosarcoma) contain extensive heterogeneity within and between patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35867969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220802212153&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomic heterogeneity presents significant challenges for understanding oncogenic processes and for cancer&#x27;s clinical management. Variation in driver mutation frequency between patients with the same tumor type as well as within an individual patients&#x27; cancer can shape the use of mutations as diagnostic, prognostic, and predictive biomarkers. We have characterized genomic heterogeneity between and within canine splenic hemangiosarcoma (HSA), a common naturally occurring cancer in pet dogs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35867969"
                        open
                      >
                        <summary class="article-expander__title">Genomic landscapes of canine splenic angiosarcoma (hemangiosarcoma) contain extensive heterogeneity within and between patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35867969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220801212447&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomic heterogeneity presents significant challenges for understanding oncogenic processes and for cancer&#x27;s clinical management. Variation in driver mutation frequency between patients with the same tumor type as well as within an individual patients&#x27; cancer can shape the use of mutations as diagnostic, prognostic, and predictive biomarkers. We have characterized genomic heterogeneity between and within canine splenic hemangiosarcoma (HSA), a common naturally occurring cancer in pet dogs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35867969"
                        open
                      >
                        <summary class="article-expander__title">Genomic landscapes of canine splenic angiosarcoma (hemangiosarcoma) contain extensive heterogeneity within and between patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35867969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220731212525&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomic heterogeneity presents significant challenges for understanding oncogenic processes and for cancer&#x27;s clinical management. Variation in driver mutation frequency between patients with the same tumor type as well as within an individual patients&#x27; cancer can shape the use of mutations as diagnostic, prognostic, and predictive biomarkers. We have characterized genomic heterogeneity between and within canine splenic hemangiosarcoma (HSA), a common naturally occurring cancer in pet dogs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35867969"
                        open
                      >
                        <summary class="article-expander__title">Genomic landscapes of canine splenic angiosarcoma (hemangiosarcoma) contain extensive heterogeneity within and between patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35867969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220730212309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomic heterogeneity presents significant challenges for understanding oncogenic processes and for cancer&#x27;s clinical management. Variation in driver mutation frequency between patients with the same tumor type as well as within an individual patients&#x27; cancer can shape the use of mutations as diagnostic, prognostic, and predictive biomarkers. We have characterized genomic heterogeneity between and within canine splenic hemangiosarcoma (HSA), a common naturally occurring cancer in pet dogs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35867969"
                        open
                      >
                        <summary class="article-expander__title">Genomic landscapes of canine splenic angiosarcoma (hemangiosarcoma) contain extensive heterogeneity within and between patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35867969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220729211332&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomic heterogeneity presents significant challenges for understanding oncogenic processes and for cancer&#x27;s clinical management. Variation in driver mutation frequency between patients with the same tumor type as well as within an individual patients&#x27; cancer can shape the use of mutations as diagnostic, prognostic, and predictive biomarkers. We have characterized genomic heterogeneity between and within canine splenic hemangiosarcoma (HSA), a common naturally occurring cancer in pet dogs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35867969"
                        open
                      >
                        <summary class="article-expander__title">Genomic landscapes of canine splenic angiosarcoma (hemangiosarcoma) contain extensive heterogeneity within and between patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35867969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220728211733&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomic heterogeneity presents significant challenges for understanding oncogenic processes and for cancer&#x27;s clinical management. Variation in driver mutation frequency between patients with the same tumor type as well as within an individual patients&#x27; cancer can shape the use of mutations as diagnostic, prognostic, and predictive biomarkers. We have characterized genomic heterogeneity between and within canine splenic hemangiosarcoma (HSA), a common naturally occurring cancer in pet dogs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35867969"
                        open
                      >
                        <summary class="article-expander__title">Genomic landscapes of canine splenic angiosarcoma (hemangiosarcoma) contain extensive heterogeneity within and between patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35867969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220727211458&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomic heterogeneity presents significant challenges for understanding oncogenic processes and for cancer&#x27;s clinical management. Variation in driver mutation frequency between patients with the same tumor type as well as within an individual patients&#x27; cancer can shape the use of mutations as diagnostic, prognostic, and predictive biomarkers. We have characterized genomic heterogeneity between and within canine splenic hemangiosarcoma (HSA), a common naturally occurring cancer in pet dogs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35867969"
                        open
                      >
                        <summary class="article-expander__title">Genomic landscapes of canine splenic angiosarcoma (hemangiosarcoma) contain extensive heterogeneity within and between patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35867969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220726212040&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomic heterogeneity presents significant challenges for understanding oncogenic processes and for cancer&#x27;s clinical management. Variation in driver mutation frequency between patients with the same tumor type as well as within an individual patients&#x27; cancer can shape the use of mutations as diagnostic, prognostic, and predictive biomarkers. We have characterized genomic heterogeneity between and within canine splenic hemangiosarcoma (HSA), a common naturally occurring cancer in pet dogs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35867969"
                        open
                      >
                        <summary class="article-expander__title">Genomic landscapes of canine splenic angiosarcoma (hemangiosarcoma) contain extensive heterogeneity within and between patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35867969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220725211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomic heterogeneity presents significant challenges for understanding oncogenic processes and for cancer&#x27;s clinical management. Variation in driver mutation frequency between patients with the same tumor type as well as within an individual patients&#x27; cancer can shape the use of mutations as diagnostic, prognostic, and predictive biomarkers. We have characterized genomic heterogeneity between and within canine splenic hemangiosarcoma (HSA), a common naturally occurring cancer in pet dogs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35867969"
                        open
                      >
                        <summary class="article-expander__title">Genomic landscapes of canine splenic angiosarcoma (hemangiosarcoma) contain extensive heterogeneity within and between patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35867969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220724211605&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomic heterogeneity presents significant challenges for understanding oncogenic processes and for cancer&#x27;s clinical management. Variation in driver mutation frequency between patients with the same tumor type as well as within an individual patients&#x27; cancer can shape the use of mutations as diagnostic, prognostic, and predictive biomarkers. We have characterized genomic heterogeneity between and within canine splenic hemangiosarcoma (HSA), a common naturally occurring cancer in pet dogs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35867969"
                        open
                      >
                        <summary class="article-expander__title">Genomic landscapes of canine splenic angiosarcoma (hemangiosarcoma) contain extensive heterogeneity within and between patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35867969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220723212121&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomic heterogeneity presents significant challenges for understanding oncogenic processes and for cancer&#x27;s clinical management. Variation in driver mutation frequency between patients with the same tumor type as well as within an individual patients&#x27; cancer can shape the use of mutations as diagnostic, prognostic, and predictive biomarkers. We have characterized genomic heterogeneity between and within canine splenic hemangiosarcoma (HSA), a common naturally occurring cancer in pet dogs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35867969"
                        open
                      >
                        <summary class="article-expander__title">Genomic landscapes of canine splenic angiosarcoma (hemangiosarcoma) contain extensive heterogeneity within and between patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35867969/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220722211347&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer genomic heterogeneity presents significant challenges for understanding oncogenic processes and for cancer&#x27;s clinical management. Variation in driver mutation frequency between patients with the same tumor type as well as within an individual patients&#x27; cancer can shape the use of mutations as diagnostic, prognostic, and predictive biomarkers. We have characterized genomic heterogeneity between and within canine splenic hemangiosarcoma (HSA), a common naturally occurring cancer in pet dogs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-07-20 datetime="2022-07-20T10:00:00.000Z">2022-07-20</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-07-20 datetime="2022-07-20T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608003622&amp;v&#x3D;2.17.7&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35858618"
                        open
                      >
                        <summary class="article-expander__title">Inflammatory exposure drives long-lived impairment of hematopoietic stem cell self-renewal activity and accelerated aging</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35858618/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220802212153&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) mediate regeneration of the hematopoietic system following injury, such as following infection or inflammation. These challenges impair HSC function, but whether this functional impairment extends beyond the duration of inflammatory exposure is unknown. Unexpectedly, we observed an irreversible depletion of functional HSCs following challenge with inflammation or bacterial infection, with no evidence of any recovery up to 1 year afterward. HSCs from challenged...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35858618"
                        open
                      >
                        <summary class="article-expander__title">Inflammatory exposure drives long-lived impairment of hematopoietic stem cell self-renewal activity and accelerated aging</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35858618/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220801212447&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) mediate regeneration of the hematopoietic system following injury, such as following infection or inflammation. These challenges impair HSC function, but whether this functional impairment extends beyond the duration of inflammatory exposure is unknown. Unexpectedly, we observed an irreversible depletion of functional HSCs following challenge with inflammation or bacterial infection, with no evidence of any recovery up to 1 year afterward. HSCs from challenged...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35857592"
                        open
                      >
                        <summary class="article-expander__title">Hematopoietic loss of Y chromosome leads to cardiac fibrosis and heart failure mortality</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35857592/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220801212447&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic mosaic loss of Y chromosome (mLOY) is associated with increased risk of mortality and age-related diseases in men, but the causal and mechanistic relationships have yet to be established. Here, we show that male mice reconstituted with bone marrow cells lacking the Y chromosome display increased mortality and age-related profibrotic pathologies including reduced cardiac function. Cardiac macrophages lacking the Y chromosome exhibited polarization toward a more fibrotic phenotype,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35858618"
                        open
                      >
                        <summary class="article-expander__title">Inflammatory exposure drives long-lived impairment of hematopoietic stem cell self-renewal activity and accelerated aging</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35858618/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220731212525&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) mediate regeneration of the hematopoietic system following injury, such as following infection or inflammation. These challenges impair HSC function, but whether this functional impairment extends beyond the duration of inflammatory exposure is unknown. Unexpectedly, we observed an irreversible depletion of functional HSCs following challenge with inflammation or bacterial infection, with no evidence of any recovery up to 1 year afterward. HSCs from challenged...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35857592"
                        open
                      >
                        <summary class="article-expander__title">Hematopoietic loss of Y chromosome leads to cardiac fibrosis and heart failure mortality</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35857592/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220731212525&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic mosaic loss of Y chromosome (mLOY) is associated with increased risk of mortality and age-related diseases in men, but the causal and mechanistic relationships have yet to be established. Here, we show that male mice reconstituted with bone marrow cells lacking the Y chromosome display increased mortality and age-related profibrotic pathologies including reduced cardiac function. Cardiac macrophages lacking the Y chromosome exhibited polarization toward a more fibrotic phenotype,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35855153"
                        open
                      >
                        <summary class="article-expander__title">Hidden treasures: Incidental findings in two cases of chronic subdural hematoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35855153/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220731212525&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: EMH is rare and usually found in individuals with profound and chronic anemia. However, this entity may be more common in chronic subdural hematomas. Solitary extraosseous plasmacytoma is exceedingly rare in the cranium, and its presence in chronic subdural hematoma membranes is of uncertain significance. The cytokine milieu that promotes organization of chronic subdural hematomas may play a role in the establishment of both of entities in this location.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35858618"
                        open
                      >
                        <summary class="article-expander__title">Inflammatory exposure drives long-lived impairment of hematopoietic stem cell self-renewal activity and accelerated aging</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35858618/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220730212309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) mediate regeneration of the hematopoietic system following injury, such as following infection or inflammation. These challenges impair HSC function, but whether this functional impairment extends beyond the duration of inflammatory exposure is unknown. Unexpectedly, we observed an irreversible depletion of functional HSCs following challenge with inflammation or bacterial infection, with no evidence of any recovery up to 1 year afterward. HSCs from challenged...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35857592"
                        open
                      >
                        <summary class="article-expander__title">Hematopoietic loss of Y chromosome leads to cardiac fibrosis and heart failure mortality</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35857592/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220730212309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic mosaic loss of Y chromosome (mLOY) is associated with increased risk of mortality and age-related diseases in men, but the causal and mechanistic relationships have yet to be established. Here, we show that male mice reconstituted with bone marrow cells lacking the Y chromosome display increased mortality and age-related profibrotic pathologies including reduced cardiac function. Cardiac macrophages lacking the Y chromosome exhibited polarization toward a more fibrotic phenotype,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35855153"
                        open
                      >
                        <summary class="article-expander__title">Hidden treasures: Incidental findings in two cases of chronic subdural hematoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35855153/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220730212309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: EMH is rare and usually found in individuals with profound and chronic anemia. However, this entity may be more common in chronic subdural hematomas. Solitary extraosseous plasmacytoma is exceedingly rare in the cranium, and its presence in chronic subdural hematoma membranes is of uncertain significance. The cytokine milieu that promotes organization of chronic subdural hematomas may play a role in the establishment of both of entities in this location.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35858618"
                        open
                      >
                        <summary class="article-expander__title">Inflammatory exposure drives long-lived impairment of hematopoietic stem cell self-renewal activity and accelerated aging</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35858618/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220729211332&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) mediate regeneration of the hematopoietic system following injury, such as following infection or inflammation. These challenges impair HSC function, but whether this functional impairment extends beyond the duration of inflammatory exposure is unknown. Unexpectedly, we observed an irreversible depletion of functional HSCs following challenge with inflammation or bacterial infection, with no evidence of any recovery up to 1 year afterward. HSCs from challenged...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35857592"
                        open
                      >
                        <summary class="article-expander__title">Hematopoietic loss of Y chromosome leads to cardiac fibrosis and heart failure mortality</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35857592/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220729211332&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic mosaic loss of Y chromosome (mLOY) is associated with increased risk of mortality and age-related diseases in men, but the causal and mechanistic relationships have yet to be established. Here, we show that male mice reconstituted with bone marrow cells lacking the Y chromosome display increased mortality and age-related profibrotic pathologies including reduced cardiac function. Cardiac macrophages lacking the Y chromosome exhibited polarization toward a more fibrotic phenotype,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35855153"
                        open
                      >
                        <summary class="article-expander__title">Hidden treasures: Incidental findings in two cases of chronic subdural hematoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35855153/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220729211332&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: EMH is rare and usually found in individuals with profound and chronic anemia. However, this entity may be more common in chronic subdural hematomas. Solitary extraosseous plasmacytoma is exceedingly rare in the cranium, and its presence in chronic subdural hematoma membranes is of uncertain significance. The cytokine milieu that promotes organization of chronic subdural hematomas may play a role in the establishment of both of entities in this location.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35858618"
                        open
                      >
                        <summary class="article-expander__title">Inflammatory exposure drives long-lived impairment of hematopoietic stem cell self-renewal activity and accelerated aging</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35858618/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220728211733&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) mediate regeneration of the hematopoietic system following injury, such as following infection or inflammation. These challenges impair HSC function, but whether this functional impairment extends beyond the duration of inflammatory exposure is unknown. Unexpectedly, we observed an irreversible depletion of functional HSCs following challenge with inflammation or bacterial infection, with no evidence of any recovery up to 1 year afterward. HSCs from challenged...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35857592"
                        open
                      >
                        <summary class="article-expander__title">Hematopoietic loss of Y chromosome leads to cardiac fibrosis and heart failure mortality</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35857592/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220728211733&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic mosaic loss of Y chromosome (mLOY) is associated with increased risk of mortality and age-related diseases in men, but the causal and mechanistic relationships have yet to be established. Here, we show that male mice reconstituted with bone marrow cells lacking the Y chromosome display increased mortality and age-related profibrotic pathologies including reduced cardiac function. Cardiac macrophages lacking the Y chromosome exhibited polarization toward a more fibrotic phenotype,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35855153"
                        open
                      >
                        <summary class="article-expander__title">Hidden treasures: Incidental findings in two cases of chronic subdural hematoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35855153/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220728211733&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: EMH is rare and usually found in individuals with profound and chronic anemia. However, this entity may be more common in chronic subdural hematomas. Solitary extraosseous plasmacytoma is exceedingly rare in the cranium, and its presence in chronic subdural hematoma membranes is of uncertain significance. The cytokine milieu that promotes organization of chronic subdural hematomas may play a role in the establishment of both of entities in this location.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35858618"
                        open
                      >
                        <summary class="article-expander__title">Inflammatory exposure drives long-lived impairment of hematopoietic stem cell self-renewal activity and accelerated aging</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35858618/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220727211458&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) mediate regeneration of the hematopoietic system following injury, such as following infection or inflammation. These challenges impair HSC function, but whether this functional impairment extends beyond the duration of inflammatory exposure is unknown. Unexpectedly, we observed an irreversible depletion of functional HSCs following challenge with inflammation or bacterial infection, with no evidence of any recovery up to 1 year afterward. HSCs from challenged...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35857592"
                        open
                      >
                        <summary class="article-expander__title">Hematopoietic loss of Y chromosome leads to cardiac fibrosis and heart failure mortality</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35857592/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220727211458&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic mosaic loss of Y chromosome (mLOY) is associated with increased risk of mortality and age-related diseases in men, but the causal and mechanistic relationships have yet to be established. Here, we show that male mice reconstituted with bone marrow cells lacking the Y chromosome display increased mortality and age-related profibrotic pathologies including reduced cardiac function. Cardiac macrophages lacking the Y chromosome exhibited polarization toward a more fibrotic phenotype,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35855153"
                        open
                      >
                        <summary class="article-expander__title">Hidden treasures: Incidental findings in two cases of chronic subdural hematoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35855153/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220727211458&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: EMH is rare and usually found in individuals with profound and chronic anemia. However, this entity may be more common in chronic subdural hematomas. Solitary extraosseous plasmacytoma is exceedingly rare in the cranium, and its presence in chronic subdural hematoma membranes is of uncertain significance. The cytokine milieu that promotes organization of chronic subdural hematomas may play a role in the establishment of both of entities in this location.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35858618"
                        open
                      >
                        <summary class="article-expander__title">Inflammatory exposure drives long-lived impairment of hematopoietic stem cell self-renewal activity and accelerated aging</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35858618/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220726212041&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) mediate regeneration of the hematopoietic system following injury, such as following infection or inflammation. These challenges impair HSC function, but whether this functional impairment extends beyond the duration of inflammatory exposure is unknown. Unexpectedly, we observed an irreversible depletion of functional HSCs following challenge with inflammation or bacterial infection, with no evidence of any recovery up to 1 year afterward. HSCs from challenged...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35857592"
                        open
                      >
                        <summary class="article-expander__title">Hematopoietic loss of Y chromosome leads to cardiac fibrosis and heart failure mortality</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35857592/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220726212041&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic mosaic loss of Y chromosome (mLOY) is associated with increased risk of mortality and age-related diseases in men, but the causal and mechanistic relationships have yet to be established. Here, we show that male mice reconstituted with bone marrow cells lacking the Y chromosome display increased mortality and age-related profibrotic pathologies including reduced cardiac function. Cardiac macrophages lacking the Y chromosome exhibited polarization toward a more fibrotic phenotype,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35855153"
                        open
                      >
                        <summary class="article-expander__title">Hidden treasures: Incidental findings in two cases of chronic subdural hematoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35855153/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220726212041&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: EMH is rare and usually found in individuals with profound and chronic anemia. However, this entity may be more common in chronic subdural hematomas. Solitary extraosseous plasmacytoma is exceedingly rare in the cranium, and its presence in chronic subdural hematoma membranes is of uncertain significance. The cytokine milieu that promotes organization of chronic subdural hematomas may play a role in the establishment of both of entities in this location.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35858618"
                        open
                      >
                        <summary class="article-expander__title">Inflammatory exposure drives long-lived impairment of hematopoietic stem cell self-renewal activity and accelerated aging</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35858618/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220725211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) mediate regeneration of the hematopoietic system following injury, such as following infection or inflammation. These challenges impair HSC function, but whether this functional impairment extends beyond the duration of inflammatory exposure is unknown. Unexpectedly, we observed an irreversible depletion of functional HSCs following challenge with inflammation or bacterial infection, with no evidence of any recovery up to 1 year afterward. HSCs from challenged...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35857592"
                        open
                      >
                        <summary class="article-expander__title">Hematopoietic loss of Y chromosome leads to cardiac fibrosis and heart failure mortality</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35857592/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220725211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic mosaic loss of Y chromosome (mLOY) is associated with increased risk of mortality and age-related diseases in men, but the causal and mechanistic relationships have yet to be established. Here, we show that male mice reconstituted with bone marrow cells lacking the Y chromosome display increased mortality and age-related profibrotic pathologies including reduced cardiac function. Cardiac macrophages lacking the Y chromosome exhibited polarization toward a more fibrotic phenotype,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35855153"
                        open
                      >
                        <summary class="article-expander__title">Hidden treasures: Incidental findings in two cases of chronic subdural hematoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35855153/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220725211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: EMH is rare and usually found in individuals with profound and chronic anemia. However, this entity may be more common in chronic subdural hematomas. Solitary extraosseous plasmacytoma is exceedingly rare in the cranium, and its presence in chronic subdural hematoma membranes is of uncertain significance. The cytokine milieu that promotes organization of chronic subdural hematomas may play a role in the establishment of both of entities in this location.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35858618"
                        open
                      >
                        <summary class="article-expander__title">Inflammatory exposure drives long-lived impairment of hematopoietic stem cell self-renewal activity and accelerated aging</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35858618/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220724211604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) mediate regeneration of the hematopoietic system following injury, such as following infection or inflammation. These challenges impair HSC function, but whether this functional impairment extends beyond the duration of inflammatory exposure is unknown. Unexpectedly, we observed an irreversible depletion of functional HSCs following challenge with inflammation or bacterial infection, with no evidence of any recovery up to 1 year afterward. HSCs from challenged...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35857592"
                        open
                      >
                        <summary class="article-expander__title">Hematopoietic loss of Y chromosome leads to cardiac fibrosis and heart failure mortality</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35857592/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220724211604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic mosaic loss of Y chromosome (mLOY) is associated with increased risk of mortality and age-related diseases in men, but the causal and mechanistic relationships have yet to be established. Here, we show that male mice reconstituted with bone marrow cells lacking the Y chromosome display increased mortality and age-related profibrotic pathologies including reduced cardiac function. Cardiac macrophages lacking the Y chromosome exhibited polarization toward a more fibrotic phenotype,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35855153"
                        open
                      >
                        <summary class="article-expander__title">Hidden treasures: Incidental findings in two cases of chronic subdural hematoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35855153/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220724211604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: EMH is rare and usually found in individuals with profound and chronic anemia. However, this entity may be more common in chronic subdural hematomas. Solitary extraosseous plasmacytoma is exceedingly rare in the cranium, and its presence in chronic subdural hematoma membranes is of uncertain significance. The cytokine milieu that promotes organization of chronic subdural hematomas may play a role in the establishment of both of entities in this location.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35858618"
                        open
                      >
                        <summary class="article-expander__title">Inflammatory exposure drives long-lived impairment of hematopoietic stem cell self-renewal activity and accelerated aging</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35858618/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220723212121&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) mediate regeneration of the hematopoietic system following injury, such as following infection or inflammation. These challenges impair HSC function, but whether this functional impairment extends beyond the duration of inflammatory exposure is unknown. Unexpectedly, we observed an irreversible depletion of functional HSCs following challenge with inflammation or bacterial infection, with no evidence of any recovery up to 1 year afterward. HSCs from challenged...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35857592"
                        open
                      >
                        <summary class="article-expander__title">Hematopoietic loss of Y chromosome leads to cardiac fibrosis and heart failure mortality</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35857592/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220723212121&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic mosaic loss of Y chromosome (mLOY) is associated with increased risk of mortality and age-related diseases in men, but the causal and mechanistic relationships have yet to be established. Here, we show that male mice reconstituted with bone marrow cells lacking the Y chromosome display increased mortality and age-related profibrotic pathologies including reduced cardiac function. Cardiac macrophages lacking the Y chromosome exhibited polarization toward a more fibrotic phenotype,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35855153"
                        open
                      >
                        <summary class="article-expander__title">Hidden treasures: Incidental findings in two cases of chronic subdural hematoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35855153/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220723212121&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: EMH is rare and usually found in individuals with profound and chronic anemia. However, this entity may be more common in chronic subdural hematomas. Solitary extraosseous plasmacytoma is exceedingly rare in the cranium, and its presence in chronic subdural hematoma membranes is of uncertain significance. The cytokine milieu that promotes organization of chronic subdural hematomas may play a role in the establishment of both of entities in this location.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35858618"
                        open
                      >
                        <summary class="article-expander__title">Inflammatory exposure drives long-lived impairment of hematopoietic stem cell self-renewal activity and accelerated aging</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35858618/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220722211347&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) mediate regeneration of the hematopoietic system following injury, such as following infection or inflammation. These challenges impair HSC function, but whether this functional impairment extends beyond the duration of inflammatory exposure is unknown. Unexpectedly, we observed an irreversible depletion of functional HSCs following challenge with inflammation or bacterial infection, with no evidence of any recovery up to 1 year afterward. HSCs from challenged...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35857592"
                        open
                      >
                        <summary class="article-expander__title">Hematopoietic loss of Y chromosome leads to cardiac fibrosis and heart failure mortality</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35857592/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220722211347&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic mosaic loss of Y chromosome (mLOY) is associated with increased risk of mortality and age-related diseases in men, but the causal and mechanistic relationships have yet to be established. Here, we show that male mice reconstituted with bone marrow cells lacking the Y chromosome display increased mortality and age-related profibrotic pathologies including reduced cardiac function. Cardiac macrophages lacking the Y chromosome exhibited polarization toward a more fibrotic phenotype,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35855153"
                        open
                      >
                        <summary class="article-expander__title">Hidden treasures: Incidental findings in two cases of chronic subdural hematoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35855153/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220722211347&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: EMH is rare and usually found in individuals with profound and chronic anemia. However, this entity may be more common in chronic subdural hematomas. Solitary extraosseous plasmacytoma is exceedingly rare in the cranium, and its presence in chronic subdural hematoma membranes is of uncertain significance. The cytokine milieu that promotes organization of chronic subdural hematomas may play a role in the establishment of both of entities in this location.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35858618"
                        open
                      >
                        <summary class="article-expander__title">Inflammatory exposure drives long-lived impairment of hematopoietic stem cell self-renewal activity and accelerated aging</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35858618/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220721211948&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) mediate regeneration of the hematopoietic system following injury, such as following infection or inflammation. These challenges impair HSC function, but whether this functional impairment extends beyond the duration of inflammatory exposure is unknown. Unexpectedly, we observed an irreversible depletion of functional HSCs following challenge with inflammation or bacterial infection, with no evidence of any recovery up to 1 year afterward. HSCs from challenged...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35857592"
                        open
                      >
                        <summary class="article-expander__title">Hematopoietic loss of Y chromosome leads to cardiac fibrosis and heart failure mortality</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35857592/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220721211948&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic mosaic loss of Y chromosome (mLOY) is associated with increased risk of mortality and age-related diseases in men, but the causal and mechanistic relationships have yet to be established. Here, we show that male mice reconstituted with bone marrow cells lacking the Y chromosome display increased mortality and age-related profibrotic pathologies including reduced cardiac function. Cardiac macrophages lacking the Y chromosome exhibited polarization toward a more fibrotic phenotype,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35855153"
                        open
                      >
                        <summary class="article-expander__title">Hidden treasures: Incidental findings in two cases of chronic subdural hematoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35855153/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220721211948&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: EMH is rare and usually found in individuals with profound and chronic anemia. However, this entity may be more common in chronic subdural hematomas. Solitary extraosseous plasmacytoma is exceedingly rare in the cranium, and its presence in chronic subdural hematoma membranes is of uncertain significance. The cytokine milieu that promotes organization of chronic subdural hematomas may play a role in the establishment of both of entities in this location.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35857592"
                        open
                      >
                        <summary class="article-expander__title">Hematopoietic loss of Y chromosome leads to cardiac fibrosis and heart failure mortality</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35857592/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220720211338&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic mosaic loss of Y chromosome (mLOY) is associated with increased risk of mortality and age-related diseases in men, but the causal and mechanistic relationships have yet to be established. Here, we show that male mice reconstituted with bone marrow cells lacking the Y chromosome display increased mortality and age-related profibrotic pathologies including reduced cardiac function. Cardiac macrophages lacking the Y chromosome exhibited polarization toward a more fibrotic phenotype,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35855153"
                        open
                      >
                        <summary class="article-expander__title">Hidden treasures: Incidental findings in two cases of chronic subdural hematoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35855153/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220720211338&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: EMH is rare and usually found in individuals with profound and chronic anemia. However, this entity may be more common in chronic subdural hematomas. Solitary extraosseous plasmacytoma is exceedingly rare in the cranium, and its presence in chronic subdural hematoma membranes is of uncertain significance. The cytokine milieu that promotes organization of chronic subdural hematomas may play a role in the establishment of both of entities in this location.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608010801&amp;v&#x3D;2.17.7&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35855571"
                        open
                      >
                        <summary class="article-expander__title">Genetics of testicular cancer: a review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35855571/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220813211549&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Testicular germ cell tumours (TGCTs) are the most common solid malignant cancer diagnosed in young males and the incidence is increasing. Understanding the genetic basis of this disease will help us to navigate the challenges of early detection, diagnosis, treatment, surveillance, and long-term outcomes for patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35855571"
                        open
                      >
                        <summary class="article-expander__title">Genetics of testicular cancer: a review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35855571/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220812210706&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Testicular germ cell tumours (TGCTs) are the most common solid malignant cancer diagnosed in young males and the incidence is increasing. Understanding the genetic basis of this disease will help us to navigate the challenges of early detection, diagnosis, treatment, surveillance, and long-term outcomes for patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35855571"
                        open
                      >
                        <summary class="article-expander__title">Genetics of testicular cancer: a review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35855571/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220811211442&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Testicular germ cell tumours (TGCTs) are the most common solid malignant cancer diagnosed in young males and the incidence is increasing. Understanding the genetic basis of this disease will help us to navigate the challenges of early detection, diagnosis, treatment, surveillance, and long-term outcomes for patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35855571"
                        open
                      >
                        <summary class="article-expander__title">Genetics of testicular cancer: a review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35855571/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220810210900&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Testicular germ cell tumours (TGCTs) are the most common solid malignant cancer diagnosed in young males and the incidence is increasing. Understanding the genetic basis of this disease will help us to navigate the challenges of early detection, diagnosis, treatment, surveillance, and long-term outcomes for patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35855571"
                        open
                      >
                        <summary class="article-expander__title">Genetics of testicular cancer: a review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35855571/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220809211355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Testicular germ cell tumours (TGCTs) are the most common solid malignant cancer diagnosed in young males and the incidence is increasing. Understanding the genetic basis of this disease will help us to navigate the challenges of early detection, diagnosis, treatment, surveillance, and long-term outcomes for patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35855571"
                        open
                      >
                        <summary class="article-expander__title">Genetics of testicular cancer: a review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35855571/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220808212152&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Testicular germ cell tumours (TGCTs) are the most common solid malignant cancer diagnosed in young males and the incidence is increasing. Understanding the genetic basis of this disease will help us to navigate the challenges of early detection, diagnosis, treatment, surveillance, and long-term outcomes for patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35855571"
                        open
                      >
                        <summary class="article-expander__title">Genetics of testicular cancer: a review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35855571/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220807211336&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Testicular germ cell tumours (TGCTs) are the most common solid malignant cancer diagnosed in young males and the incidence is increasing. Understanding the genetic basis of this disease will help us to navigate the challenges of early detection, diagnosis, treatment, surveillance, and long-term outcomes for patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35855571"
                        open
                      >
                        <summary class="article-expander__title">Genetics of testicular cancer: a review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35855571/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220806211830&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Testicular germ cell tumours (TGCTs) are the most common solid malignant cancer diagnosed in young males and the incidence is increasing. Understanding the genetic basis of this disease will help us to navigate the challenges of early detection, diagnosis, treatment, surveillance, and long-term outcomes for patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35855571"
                        open
                      >
                        <summary class="article-expander__title">Genetics of testicular cancer: a review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35855571/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220805210849&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Testicular germ cell tumours (TGCTs) are the most common solid malignant cancer diagnosed in young males and the incidence is increasing. Understanding the genetic basis of this disease will help us to navigate the challenges of early detection, diagnosis, treatment, surveillance, and long-term outcomes for patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35855571"
                        open
                      >
                        <summary class="article-expander__title">Genetics of testicular cancer: a review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35855571/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220804212147&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Testicular germ cell tumours (TGCTs) are the most common solid malignant cancer diagnosed in young males and the incidence is increasing. Understanding the genetic basis of this disease will help us to navigate the challenges of early detection, diagnosis, treatment, surveillance, and long-term outcomes for patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35855571"
                        open
                      >
                        <summary class="article-expander__title">Genetics of testicular cancer: a review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35855571/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220803212051&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Testicular germ cell tumours (TGCTs) are the most common solid malignant cancer diagnosed in young males and the incidence is increasing. Understanding the genetic basis of this disease will help us to navigate the challenges of early detection, diagnosis, treatment, surveillance, and long-term outcomes for patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35855571"
                        open
                      >
                        <summary class="article-expander__title">Genetics of testicular cancer: a review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35855571/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220802212153&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Testicular germ cell tumours (TGCTs) are the most common solid malignant cancer diagnosed in young males and the incidence is increasing. Understanding the genetic basis of this disease will help us to navigate the challenges of early detection, diagnosis, treatment, surveillance, and long-term outcomes for patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35855571"
                        open
                      >
                        <summary class="article-expander__title">Genetics of testicular cancer: a review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35855571/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220801212447&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Testicular germ cell tumours (TGCTs) are the most common solid malignant cancer diagnosed in young males and the incidence is increasing. Understanding the genetic basis of this disease will help us to navigate the challenges of early detection, diagnosis, treatment, surveillance, and long-term outcomes for patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35855571"
                        open
                      >
                        <summary class="article-expander__title">Genetics of testicular cancer: a review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35855571/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220731212525&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Testicular germ cell tumours (TGCTs) are the most common solid malignant cancer diagnosed in young males and the incidence is increasing. Understanding the genetic basis of this disease will help us to navigate the challenges of early detection, diagnosis, treatment, surveillance, and long-term outcomes for patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35855571"
                        open
                      >
                        <summary class="article-expander__title">Genetics of testicular cancer: a review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35855571/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220730212309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Testicular germ cell tumours (TGCTs) are the most common solid malignant cancer diagnosed in young males and the incidence is increasing. Understanding the genetic basis of this disease will help us to navigate the challenges of early detection, diagnosis, treatment, surveillance, and long-term outcomes for patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35855571"
                        open
                      >
                        <summary class="article-expander__title">Genetics of testicular cancer: a review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35855571/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220729211332&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Testicular germ cell tumours (TGCTs) are the most common solid malignant cancer diagnosed in young males and the incidence is increasing. Understanding the genetic basis of this disease will help us to navigate the challenges of early detection, diagnosis, treatment, surveillance, and long-term outcomes for patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35855571"
                        open
                      >
                        <summary class="article-expander__title">Genetics of testicular cancer: a review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35855571/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220728211733&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Testicular germ cell tumours (TGCTs) are the most common solid malignant cancer diagnosed in young males and the incidence is increasing. Understanding the genetic basis of this disease will help us to navigate the challenges of early detection, diagnosis, treatment, surveillance, and long-term outcomes for patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35855571"
                        open
                      >
                        <summary class="article-expander__title">Genetics of testicular cancer: a review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35855571/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220727211458&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Testicular germ cell tumours (TGCTs) are the most common solid malignant cancer diagnosed in young males and the incidence is increasing. Understanding the genetic basis of this disease will help us to navigate the challenges of early detection, diagnosis, treatment, surveillance, and long-term outcomes for patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35855571"
                        open
                      >
                        <summary class="article-expander__title">Genetics of testicular cancer: a review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35855571/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220726212040&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Testicular germ cell tumours (TGCTs) are the most common solid malignant cancer diagnosed in young males and the incidence is increasing. Understanding the genetic basis of this disease will help us to navigate the challenges of early detection, diagnosis, treatment, surveillance, and long-term outcomes for patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35855571"
                        open
                      >
                        <summary class="article-expander__title">Genetics of testicular cancer: a review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35855571/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220725211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Testicular germ cell tumours (TGCTs) are the most common solid malignant cancer diagnosed in young males and the incidence is increasing. Understanding the genetic basis of this disease will help us to navigate the challenges of early detection, diagnosis, treatment, surveillance, and long-term outcomes for patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35855571"
                        open
                      >
                        <summary class="article-expander__title">Genetics of testicular cancer: a review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35855571/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220724211605&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Testicular germ cell tumours (TGCTs) are the most common solid malignant cancer diagnosed in young males and the incidence is increasing. Understanding the genetic basis of this disease will help us to navigate the challenges of early detection, diagnosis, treatment, surveillance, and long-term outcomes for patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35855571"
                        open
                      >
                        <summary class="article-expander__title">Genetics of testicular cancer: a review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35855571/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220723212121&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Testicular germ cell tumours (TGCTs) are the most common solid malignant cancer diagnosed in young males and the incidence is increasing. Understanding the genetic basis of this disease will help us to navigate the challenges of early detection, diagnosis, treatment, surveillance, and long-term outcomes for patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35855571"
                        open
                      >
                        <summary class="article-expander__title">Genetics of testicular cancer: a review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35855571/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220722211347&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Testicular germ cell tumours (TGCTs) are the most common solid malignant cancer diagnosed in young males and the incidence is increasing. Understanding the genetic basis of this disease will help us to navigate the challenges of early detection, diagnosis, treatment, surveillance, and long-term outcomes for patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35855571"
                        open
                      >
                        <summary class="article-expander__title">Genetics of testicular cancer: a review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35855571/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220721211948&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Testicular germ cell tumours (TGCTs) are the most common solid malignant cancer diagnosed in young males and the incidence is increasing. Understanding the genetic basis of this disease will help us to navigate the challenges of early detection, diagnosis, treatment, surveillance, and long-term outcomes for patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35855571"
                        open
                      >
                        <summary class="article-expander__title">Genetics of testicular cancer: a review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35855571/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220720211338&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Testicular germ cell tumours (TGCTs) are the most common solid malignant cancer diagnosed in young males and the incidence is increasing. Understanding the genetic basis of this disease will help us to navigate the challenges of early detection, diagnosis, treatment, surveillance, and long-term outcomes for patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-07-19 datetime="2022-07-19T10:00:00.000Z">2022-07-19</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-07-19 datetime="2022-07-19T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608003622&amp;v&#x3D;2.17.7&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35852781"
                        open
                      >
                        <summary class="article-expander__title">Hypertension in Cancer Survivors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35852781/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220731212525&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: With increasing survival after cancer treatment, there is a need for long-term management of risk factors and chronic medical conditions to realize the full benefit of improvement of outcomes. Hypertension is a major risk factor for cardiovascular disease and has a higher prevalence in cancer survivors compared to the general population. In this review article, we discuss the burden of hypertension in cancer survivors and how this impacts their long-term outcomes and risk of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35852781"
                        open
                      >
                        <summary class="article-expander__title">Hypertension in Cancer Survivors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35852781/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220730212309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: With increasing survival after cancer treatment, there is a need for long-term management of risk factors and chronic medical conditions to realize the full benefit of improvement of outcomes. Hypertension is a major risk factor for cardiovascular disease and has a higher prevalence in cancer survivors compared to the general population. In this review article, we discuss the burden of hypertension in cancer survivors and how this impacts their long-term outcomes and risk of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35852781"
                        open
                      >
                        <summary class="article-expander__title">Hypertension in Cancer Survivors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35852781/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220729211332&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: With increasing survival after cancer treatment, there is a need for long-term management of risk factors and chronic medical conditions to realize the full benefit of improvement of outcomes. Hypertension is a major risk factor for cardiovascular disease and has a higher prevalence in cancer survivors compared to the general population. In this review article, we discuss the burden of hypertension in cancer survivors and how this impacts their long-term outcomes and risk of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35852781"
                        open
                      >
                        <summary class="article-expander__title">Hypertension in Cancer Survivors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35852781/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220728211733&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: With increasing survival after cancer treatment, there is a need for long-term management of risk factors and chronic medical conditions to realize the full benefit of improvement of outcomes. Hypertension is a major risk factor for cardiovascular disease and has a higher prevalence in cancer survivors compared to the general population. In this review article, we discuss the burden of hypertension in cancer survivors and how this impacts their long-term outcomes and risk of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35852781"
                        open
                      >
                        <summary class="article-expander__title">Hypertension in Cancer Survivors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35852781/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220727211458&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: With increasing survival after cancer treatment, there is a need for long-term management of risk factors and chronic medical conditions to realize the full benefit of improvement of outcomes. Hypertension is a major risk factor for cardiovascular disease and has a higher prevalence in cancer survivors compared to the general population. In this review article, we discuss the burden of hypertension in cancer survivors and how this impacts their long-term outcomes and risk of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35852781"
                        open
                      >
                        <summary class="article-expander__title">Hypertension in Cancer Survivors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35852781/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220726212041&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: With increasing survival after cancer treatment, there is a need for long-term management of risk factors and chronic medical conditions to realize the full benefit of improvement of outcomes. Hypertension is a major risk factor for cardiovascular disease and has a higher prevalence in cancer survivors compared to the general population. In this review article, we discuss the burden of hypertension in cancer survivors and how this impacts their long-term outcomes and risk of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35852781"
                        open
                      >
                        <summary class="article-expander__title">Hypertension in Cancer Survivors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35852781/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220725211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: With increasing survival after cancer treatment, there is a need for long-term management of risk factors and chronic medical conditions to realize the full benefit of improvement of outcomes. Hypertension is a major risk factor for cardiovascular disease and has a higher prevalence in cancer survivors compared to the general population. In this review article, we discuss the burden of hypertension in cancer survivors and how this impacts their long-term outcomes and risk of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35852781"
                        open
                      >
                        <summary class="article-expander__title">Hypertension in Cancer Survivors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35852781/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220724211604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: With increasing survival after cancer treatment, there is a need for long-term management of risk factors and chronic medical conditions to realize the full benefit of improvement of outcomes. Hypertension is a major risk factor for cardiovascular disease and has a higher prevalence in cancer survivors compared to the general population. In this review article, we discuss the burden of hypertension in cancer survivors and how this impacts their long-term outcomes and risk of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35852781"
                        open
                      >
                        <summary class="article-expander__title">Hypertension in Cancer Survivors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35852781/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220723212121&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: With increasing survival after cancer treatment, there is a need for long-term management of risk factors and chronic medical conditions to realize the full benefit of improvement of outcomes. Hypertension is a major risk factor for cardiovascular disease and has a higher prevalence in cancer survivors compared to the general population. In this review article, we discuss the burden of hypertension in cancer survivors and how this impacts their long-term outcomes and risk of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35852781"
                        open
                      >
                        <summary class="article-expander__title">Hypertension in Cancer Survivors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35852781/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220722211347&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: With increasing survival after cancer treatment, there is a need for long-term management of risk factors and chronic medical conditions to realize the full benefit of improvement of outcomes. Hypertension is a major risk factor for cardiovascular disease and has a higher prevalence in cancer survivors compared to the general population. In this review article, we discuss the burden of hypertension in cancer survivors and how this impacts their long-term outcomes and risk of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35852781"
                        open
                      >
                        <summary class="article-expander__title">Hypertension in Cancer Survivors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35852781/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220721211948&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: With increasing survival after cancer treatment, there is a need for long-term management of risk factors and chronic medical conditions to realize the full benefit of improvement of outcomes. Hypertension is a major risk factor for cardiovascular disease and has a higher prevalence in cancer survivors compared to the general population. In this review article, we discuss the burden of hypertension in cancer survivors and how this impacts their long-term outcomes and risk of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35852781"
                        open
                      >
                        <summary class="article-expander__title">Hypertension in Cancer Survivors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35852781/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220720211338&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: With increasing survival after cancer treatment, there is a need for long-term management of risk factors and chronic medical conditions to realize the full benefit of improvement of outcomes. Hypertension is a major risk factor for cardiovascular disease and has a higher prevalence in cancer survivors compared to the general population. In this review article, we discuss the burden of hypertension in cancer survivors and how this impacts their long-term outcomes and risk of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35852781"
                        open
                      >
                        <summary class="article-expander__title">Hypertension in Cancer Survivors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35852781/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220719211909&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: With increasing survival after cancer treatment, there is a need for long-term management of risk factors and chronic medical conditions to realize the full benefit of improvement of outcomes. Hypertension is a major risk factor for cardiovascular disease and has a higher prevalence in cancer survivors compared to the general population. In this review article, we discuss the burden of hypertension in cancer survivors and how this impacts their long-term outcomes and risk of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608010801&amp;v&#x3D;2.17.7&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35853232"
                        open
                      >
                        <summary class="article-expander__title">Geospatial immune heterogeneity reflects the diverse tumor-immune interactions in intrahepatic cholangiocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35853232/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220811211442&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intrahepatic cholangiocarcinoma (iCCA) exhibits extensive intratumoral heterogeneity and extremely high mortality rate. Here, we performed WES, RNA-seq, TCR-seq and multiplexed immunofluorescence on 207 tumor regions from 45 iCCA patients. Over half of iCCA displayed intratumoral heterogeneity of immune infiltration, and iCCA were classified into sparsely, heterogeneously, and highly infiltrated subgroups with distinct immunogenomic characteristics. Sparsely infiltrated tumors displayed active...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35853232"
                        open
                      >
                        <summary class="article-expander__title">Geospatial immune heterogeneity reflects the diverse tumor-immune interactions in intrahepatic cholangiocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35853232/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220810210900&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intrahepatic cholangiocarcinoma (iCCA) exhibits extensive intratumoral heterogeneity and extremely high mortality rate. Here, we performed WES, RNA-seq, TCR-seq and multiplexed immunofluorescence on 207 tumor regions from 45 iCCA patients. Over half of iCCA displayed intratumoral heterogeneity of immune infiltration, and iCCA were classified into sparsely, heterogeneously, and highly infiltrated subgroups with distinct immunogenomic characteristics. Sparsely infiltrated tumors displayed active...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35850404"
                        open
                      >
                        <summary class="article-expander__title">Game of clones: Battles in the field of carcinogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35850404/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220810210900&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent advances in bulk sequencing approaches as well as genomic decoding at the single-cell level have revealed surprisingly high somatic mutational burdens in normal tissues, as well as increased our understanding of the landscape of &quot;field cancerization&quot;, that is, molecular and immune alterations in mutagen-exposed normal-appearing tissues that recapitulated those present in tumors. Charting the somatic mutational landscapes in normal tissues can have strong implications on our understanding...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35853232"
                        open
                      >
                        <summary class="article-expander__title">Geospatial immune heterogeneity reflects the diverse tumor-immune interactions in intrahepatic cholangiocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35853232/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220809211355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intrahepatic cholangiocarcinoma (iCCA) exhibits extensive intratumoral heterogeneity and extremely high mortality rate. Here, we performed WES, RNA-seq, TCR-seq and multiplexed immunofluorescence on 207 tumor regions from 45 iCCA patients. Over half of iCCA displayed intratumoral heterogeneity of immune infiltration, and iCCA were classified into sparsely, heterogeneously, and highly infiltrated subgroups with distinct immunogenomic characteristics. Sparsely infiltrated tumors displayed active...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35850404"
                        open
                      >
                        <summary class="article-expander__title">Game of clones: Battles in the field of carcinogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35850404/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220809211355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent advances in bulk sequencing approaches as well as genomic decoding at the single-cell level have revealed surprisingly high somatic mutational burdens in normal tissues, as well as increased our understanding of the landscape of &quot;field cancerization&quot;, that is, molecular and immune alterations in mutagen-exposed normal-appearing tissues that recapitulated those present in tumors. Charting the somatic mutational landscapes in normal tissues can have strong implications on our understanding...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35853232"
                        open
                      >
                        <summary class="article-expander__title">Geospatial immune heterogeneity reflects the diverse tumor-immune interactions in intrahepatic cholangiocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35853232/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220808212152&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intrahepatic cholangiocarcinoma (iCCA) exhibits extensive intratumoral heterogeneity and extremely high mortality rate. Here, we performed WES, RNA-seq, TCR-seq and multiplexed immunofluorescence on 207 tumor regions from 45 iCCA patients. Over half of iCCA displayed intratumoral heterogeneity of immune infiltration, and iCCA were classified into sparsely, heterogeneously, and highly infiltrated subgroups with distinct immunogenomic characteristics. Sparsely infiltrated tumors displayed active...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35850404"
                        open
                      >
                        <summary class="article-expander__title">Game of clones: Battles in the field of carcinogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35850404/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220808212152&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent advances in bulk sequencing approaches as well as genomic decoding at the single-cell level have revealed surprisingly high somatic mutational burdens in normal tissues, as well as increased our understanding of the landscape of &quot;field cancerization&quot;, that is, molecular and immune alterations in mutagen-exposed normal-appearing tissues that recapitulated those present in tumors. Charting the somatic mutational landscapes in normal tissues can have strong implications on our understanding...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35853232"
                        open
                      >
                        <summary class="article-expander__title">Geospatial immune heterogeneity reflects the diverse tumor-immune interactions in intrahepatic cholangiocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35853232/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220807211336&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intrahepatic cholangiocarcinoma (iCCA) exhibits extensive intratumoral heterogeneity and extremely high mortality rate. Here, we performed WES, RNA-seq, TCR-seq and multiplexed immunofluorescence on 207 tumor regions from 45 iCCA patients. Over half of iCCA displayed intratumoral heterogeneity of immune infiltration, and iCCA were classified into sparsely, heterogeneously, and highly infiltrated subgroups with distinct immunogenomic characteristics. Sparsely infiltrated tumors displayed active...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35850404"
                        open
                      >
                        <summary class="article-expander__title">Game of clones: Battles in the field of carcinogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35850404/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220807211336&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent advances in bulk sequencing approaches as well as genomic decoding at the single-cell level have revealed surprisingly high somatic mutational burdens in normal tissues, as well as increased our understanding of the landscape of &quot;field cancerization&quot;, that is, molecular and immune alterations in mutagen-exposed normal-appearing tissues that recapitulated those present in tumors. Charting the somatic mutational landscapes in normal tissues can have strong implications on our understanding...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35853232"
                        open
                      >
                        <summary class="article-expander__title">Geospatial immune heterogeneity reflects the diverse tumor-immune interactions in intrahepatic cholangiocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35853232/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220806211830&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intrahepatic cholangiocarcinoma (iCCA) exhibits extensive intratumoral heterogeneity and extremely high mortality rate. Here, we performed WES, RNA-seq, TCR-seq and multiplexed immunofluorescence on 207 tumor regions from 45 iCCA patients. Over half of iCCA displayed intratumoral heterogeneity of immune infiltration, and iCCA were classified into sparsely, heterogeneously, and highly infiltrated subgroups with distinct immunogenomic characteristics. Sparsely infiltrated tumors displayed active...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35850404"
                        open
                      >
                        <summary class="article-expander__title">Game of clones: Battles in the field of carcinogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35850404/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220806211830&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent advances in bulk sequencing approaches as well as genomic decoding at the single-cell level have revealed surprisingly high somatic mutational burdens in normal tissues, as well as increased our understanding of the landscape of &quot;field cancerization&quot;, that is, molecular and immune alterations in mutagen-exposed normal-appearing tissues that recapitulated those present in tumors. Charting the somatic mutational landscapes in normal tissues can have strong implications on our understanding...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35853232"
                        open
                      >
                        <summary class="article-expander__title">Geospatial immune heterogeneity reflects the diverse tumor-immune interactions in intrahepatic cholangiocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35853232/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220805210849&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intrahepatic cholangiocarcinoma (iCCA) exhibits extensive intratumoral heterogeneity and extremely high mortality rate. Here, we performed WES, RNA-seq, TCR-seq and multiplexed immunofluorescence on 207 tumor regions from 45 iCCA patients. Over half of iCCA displayed intratumoral heterogeneity of immune infiltration, and iCCA were classified into sparsely, heterogeneously, and highly infiltrated subgroups with distinct immunogenomic characteristics. Sparsely infiltrated tumors displayed active...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35850404"
                        open
                      >
                        <summary class="article-expander__title">Game of clones: Battles in the field of carcinogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35850404/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220805210849&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent advances in bulk sequencing approaches as well as genomic decoding at the single-cell level have revealed surprisingly high somatic mutational burdens in normal tissues, as well as increased our understanding of the landscape of &quot;field cancerization&quot;, that is, molecular and immune alterations in mutagen-exposed normal-appearing tissues that recapitulated those present in tumors. Charting the somatic mutational landscapes in normal tissues can have strong implications on our understanding...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35853232"
                        open
                      >
                        <summary class="article-expander__title">Geospatial immune heterogeneity reflects the diverse tumor-immune interactions in intrahepatic cholangiocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35853232/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220804212147&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intrahepatic cholangiocarcinoma (iCCA) exhibits extensive intratumoral heterogeneity and extremely high mortality rate. Here, we performed WES, RNA-seq, TCR-seq and multiplexed immunofluorescence on 207 tumor regions from 45 iCCA patients. Over half of iCCA displayed intratumoral heterogeneity of immune infiltration, and iCCA were classified into sparsely, heterogeneously, and highly infiltrated subgroups with distinct immunogenomic characteristics. Sparsely infiltrated tumors displayed active...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35850404"
                        open
                      >
                        <summary class="article-expander__title">Game of clones: Battles in the field of carcinogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35850404/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220804212147&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent advances in bulk sequencing approaches as well as genomic decoding at the single-cell level have revealed surprisingly high somatic mutational burdens in normal tissues, as well as increased our understanding of the landscape of &quot;field cancerization&quot;, that is, molecular and immune alterations in mutagen-exposed normal-appearing tissues that recapitulated those present in tumors. Charting the somatic mutational landscapes in normal tissues can have strong implications on our understanding...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35853232"
                        open
                      >
                        <summary class="article-expander__title">Geospatial immune heterogeneity reflects the diverse tumor-immune interactions in intrahepatic cholangiocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35853232/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220803212051&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intrahepatic cholangiocarcinoma (iCCA) exhibits extensive intratumoral heterogeneity and extremely high mortality rate. Here, we performed WES, RNA-seq, TCR-seq and multiplexed immunofluorescence on 207 tumor regions from 45 iCCA patients. Over half of iCCA displayed intratumoral heterogeneity of immune infiltration, and iCCA were classified into sparsely, heterogeneously, and highly infiltrated subgroups with distinct immunogenomic characteristics. Sparsely infiltrated tumors displayed active...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35850404"
                        open
                      >
                        <summary class="article-expander__title">Game of clones: Battles in the field of carcinogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35850404/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220803212051&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent advances in bulk sequencing approaches as well as genomic decoding at the single-cell level have revealed surprisingly high somatic mutational burdens in normal tissues, as well as increased our understanding of the landscape of &quot;field cancerization&quot;, that is, molecular and immune alterations in mutagen-exposed normal-appearing tissues that recapitulated those present in tumors. Charting the somatic mutational landscapes in normal tissues can have strong implications on our understanding...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35853232"
                        open
                      >
                        <summary class="article-expander__title">Geospatial immune heterogeneity reflects the diverse tumor-immune interactions in intrahepatic cholangiocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35853232/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220802212153&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intrahepatic cholangiocarcinoma (iCCA) exhibits extensive intratumoral heterogeneity and extremely high mortality rate. Here, we performed WES, RNA-seq, TCR-seq and multiplexed immunofluorescence on 207 tumor regions from 45 iCCA patients. Over half of iCCA displayed intratumoral heterogeneity of immune infiltration, and iCCA were classified into sparsely, heterogeneously, and highly infiltrated subgroups with distinct immunogenomic characteristics. Sparsely infiltrated tumors displayed active...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35850404"
                        open
                      >
                        <summary class="article-expander__title">Game of clones: Battles in the field of carcinogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35850404/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220802212153&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent advances in bulk sequencing approaches as well as genomic decoding at the single-cell level have revealed surprisingly high somatic mutational burdens in normal tissues, as well as increased our understanding of the landscape of &quot;field cancerization&quot;, that is, molecular and immune alterations in mutagen-exposed normal-appearing tissues that recapitulated those present in tumors. Charting the somatic mutational landscapes in normal tissues can have strong implications on our understanding...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35853232"
                        open
                      >
                        <summary class="article-expander__title">Geospatial immune heterogeneity reflects the diverse tumor-immune interactions in intrahepatic cholangiocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35853232/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220801212447&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intrahepatic cholangiocarcinoma (iCCA) exhibits extensive intratumoral heterogeneity and extremely high mortality rate. Here, we performed WES, RNA-seq, TCR-seq and multiplexed immunofluorescence on 207 tumor regions from 45 iCCA patients. Over half of iCCA displayed intratumoral heterogeneity of immune infiltration, and iCCA were classified into sparsely, heterogeneously, and highly infiltrated subgroups with distinct immunogenomic characteristics. Sparsely infiltrated tumors displayed active...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35850404"
                        open
                      >
                        <summary class="article-expander__title">Game of clones: Battles in the field of carcinogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35850404/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220801212447&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent advances in bulk sequencing approaches as well as genomic decoding at the single-cell level have revealed surprisingly high somatic mutational burdens in normal tissues, as well as increased our understanding of the landscape of &quot;field cancerization&quot;, that is, molecular and immune alterations in mutagen-exposed normal-appearing tissues that recapitulated those present in tumors. Charting the somatic mutational landscapes in normal tissues can have strong implications on our understanding...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35853232"
                        open
                      >
                        <summary class="article-expander__title">Geospatial immune heterogeneity reflects the diverse tumor-immune interactions in intrahepatic cholangiocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35853232/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220731212525&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intrahepatic cholangiocarcinoma (iCCA) exhibits extensive intratumoral heterogeneity and extremely high mortality rate. Here, we performed WES, RNA-seq, TCR-seq and multiplexed immunofluorescence on 207 tumor regions from 45 iCCA patients. Over half of iCCA displayed intratumoral heterogeneity of immune infiltration, and iCCA were classified into sparsely, heterogeneously, and highly infiltrated subgroups with distinct immunogenomic characteristics. Sparsely infiltrated tumors displayed active...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35850404"
                        open
                      >
                        <summary class="article-expander__title">Game of clones: Battles in the field of carcinogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35850404/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220731212525&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent advances in bulk sequencing approaches as well as genomic decoding at the single-cell level have revealed surprisingly high somatic mutational burdens in normal tissues, as well as increased our understanding of the landscape of &quot;field cancerization&quot;, that is, molecular and immune alterations in mutagen-exposed normal-appearing tissues that recapitulated those present in tumors. Charting the somatic mutational landscapes in normal tissues can have strong implications on our understanding...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35853232"
                        open
                      >
                        <summary class="article-expander__title">Geospatial immune heterogeneity reflects the diverse tumor-immune interactions in intrahepatic cholangiocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35853232/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220730212309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intrahepatic cholangiocarcinoma (iCCA) exhibits extensive intratumoral heterogeneity and extremely high mortality rate. Here, we performed WES, RNA-seq, TCR-seq and multiplexed immunofluorescence on 207 tumor regions from 45 iCCA patients. Over half of iCCA displayed intratumoral heterogeneity of immune infiltration, and iCCA were classified into sparsely, heterogeneously, and highly infiltrated subgroups with distinct immunogenomic characteristics. Sparsely infiltrated tumors displayed active...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35850404"
                        open
                      >
                        <summary class="article-expander__title">Game of clones: Battles in the field of carcinogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35850404/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220730212309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent advances in bulk sequencing approaches as well as genomic decoding at the single-cell level have revealed surprisingly high somatic mutational burdens in normal tissues, as well as increased our understanding of the landscape of &quot;field cancerization&quot;, that is, molecular and immune alterations in mutagen-exposed normal-appearing tissues that recapitulated those present in tumors. Charting the somatic mutational landscapes in normal tissues can have strong implications on our understanding...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35853232"
                        open
                      >
                        <summary class="article-expander__title">Geospatial immune heterogeneity reflects the diverse tumor-immune interactions in intrahepatic cholangiocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35853232/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220729211332&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intrahepatic cholangiocarcinoma (iCCA) exhibits extensive intratumoral heterogeneity and extremely high mortality rate. Here, we performed WES, RNA-seq, TCR-seq and multiplexed immunofluorescence on 207 tumor regions from 45 iCCA patients. Over half of iCCA displayed intratumoral heterogeneity of immune infiltration, and iCCA were classified into sparsely, heterogeneously, and highly infiltrated subgroups with distinct immunogenomic characteristics. Sparsely infiltrated tumors displayed active...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35850404"
                        open
                      >
                        <summary class="article-expander__title">Game of clones: Battles in the field of carcinogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35850404/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220729211332&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent advances in bulk sequencing approaches as well as genomic decoding at the single-cell level have revealed surprisingly high somatic mutational burdens in normal tissues, as well as increased our understanding of the landscape of &quot;field cancerization&quot;, that is, molecular and immune alterations in mutagen-exposed normal-appearing tissues that recapitulated those present in tumors. Charting the somatic mutational landscapes in normal tissues can have strong implications on our understanding...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35853232"
                        open
                      >
                        <summary class="article-expander__title">Geospatial immune heterogeneity reflects the diverse tumor-immune interactions in intrahepatic cholangiocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35853232/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220728211733&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intrahepatic cholangiocarcinoma (iCCA) exhibits extensive intratumoral heterogeneity and extremely high mortality rate. Here, we performed WES, RNA-seq, TCR-seq and multiplexed immunofluorescence on 207 tumor regions from 45 iCCA patients. Over half of iCCA displayed intratumoral heterogeneity of immune infiltration, and iCCA were classified into sparsely, heterogeneously, and highly infiltrated subgroups with distinct immunogenomic characteristics. Sparsely infiltrated tumors displayed active...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35850404"
                        open
                      >
                        <summary class="article-expander__title">Game of clones: Battles in the field of carcinogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35850404/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220728211733&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent advances in bulk sequencing approaches as well as genomic decoding at the single-cell level have revealed surprisingly high somatic mutational burdens in normal tissues, as well as increased our understanding of the landscape of &quot;field cancerization&quot;, that is, molecular and immune alterations in mutagen-exposed normal-appearing tissues that recapitulated those present in tumors. Charting the somatic mutational landscapes in normal tissues can have strong implications on our understanding...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35853232"
                        open
                      >
                        <summary class="article-expander__title">Geospatial immune heterogeneity reflects the diverse tumor-immune interactions in intrahepatic cholangiocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35853232/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220727211458&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intrahepatic cholangiocarcinoma (iCCA) exhibits extensive intratumoral heterogeneity and extremely high mortality rate. Here, we performed WES, RNA-seq, TCR-seq and multiplexed immunofluorescence on 207 tumor regions from 45 iCCA patients. Over half of iCCA displayed intratumoral heterogeneity of immune infiltration, and iCCA were classified into sparsely, heterogeneously, and highly infiltrated subgroups with distinct immunogenomic characteristics. Sparsely infiltrated tumors displayed active...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35850404"
                        open
                      >
                        <summary class="article-expander__title">Game of clones: Battles in the field of carcinogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35850404/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220727211458&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent advances in bulk sequencing approaches as well as genomic decoding at the single-cell level have revealed surprisingly high somatic mutational burdens in normal tissues, as well as increased our understanding of the landscape of &quot;field cancerization&quot;, that is, molecular and immune alterations in mutagen-exposed normal-appearing tissues that recapitulated those present in tumors. Charting the somatic mutational landscapes in normal tissues can have strong implications on our understanding...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35853232"
                        open
                      >
                        <summary class="article-expander__title">Geospatial immune heterogeneity reflects the diverse tumor-immune interactions in intrahepatic cholangiocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35853232/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220726212040&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intrahepatic cholangiocarcinoma (iCCA) exhibits extensive intratumoral heterogeneity and extremely high mortality rate. Here, we performed WES, RNA-seq, TCR-seq and multiplexed immunofluorescence on 207 tumor regions from 45 iCCA patients. Over half of iCCA displayed intratumoral heterogeneity of immune infiltration, and iCCA were classified into sparsely, heterogeneously, and highly infiltrated subgroups with distinct immunogenomic characteristics. Sparsely infiltrated tumors displayed active...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35850404"
                        open
                      >
                        <summary class="article-expander__title">Game of clones: Battles in the field of carcinogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35850404/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220726212040&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent advances in bulk sequencing approaches as well as genomic decoding at the single-cell level have revealed surprisingly high somatic mutational burdens in normal tissues, as well as increased our understanding of the landscape of &quot;field cancerization&quot;, that is, molecular and immune alterations in mutagen-exposed normal-appearing tissues that recapitulated those present in tumors. Charting the somatic mutational landscapes in normal tissues can have strong implications on our understanding...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35853232"
                        open
                      >
                        <summary class="article-expander__title">Geospatial immune heterogeneity reflects the diverse tumor-immune interactions in intrahepatic cholangiocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35853232/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220725211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intrahepatic cholangiocarcinoma (iCCA) exhibits extensive intratumoral heterogeneity and extremely high mortality rate. Here, we performed WES, RNA-seq, TCR-seq and multiplexed immunofluorescence on 207 tumor regions from 45 iCCA patients. Over half of iCCA displayed intratumoral heterogeneity of immune infiltration, and iCCA were classified into sparsely, heterogeneously, and highly infiltrated subgroups with distinct immunogenomic characteristics. Sparsely infiltrated tumors displayed active...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35850404"
                        open
                      >
                        <summary class="article-expander__title">Game of clones: Battles in the field of carcinogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35850404/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220725211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent advances in bulk sequencing approaches as well as genomic decoding at the single-cell level have revealed surprisingly high somatic mutational burdens in normal tissues, as well as increased our understanding of the landscape of &quot;field cancerization&quot;, that is, molecular and immune alterations in mutagen-exposed normal-appearing tissues that recapitulated those present in tumors. Charting the somatic mutational landscapes in normal tissues can have strong implications on our understanding...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35853232"
                        open
                      >
                        <summary class="article-expander__title">Geospatial immune heterogeneity reflects the diverse tumor-immune interactions in intrahepatic cholangiocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35853232/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220724211605&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intrahepatic cholangiocarcinoma (iCCA) exhibits extensive intratumoral heterogeneity and extremely high mortality rate. Here, we performed WES, RNA-seq, TCR-seq and multiplexed immunofluorescence on 207 tumor regions from 45 iCCA patients. Over half of iCCA displayed intratumoral heterogeneity of immune infiltration, and iCCA were classified into sparsely, heterogeneously, and highly infiltrated subgroups with distinct immunogenomic characteristics. Sparsely infiltrated tumors displayed active...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35850404"
                        open
                      >
                        <summary class="article-expander__title">Game of clones: Battles in the field of carcinogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35850404/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220724211605&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent advances in bulk sequencing approaches as well as genomic decoding at the single-cell level have revealed surprisingly high somatic mutational burdens in normal tissues, as well as increased our understanding of the landscape of &quot;field cancerization&quot;, that is, molecular and immune alterations in mutagen-exposed normal-appearing tissues that recapitulated those present in tumors. Charting the somatic mutational landscapes in normal tissues can have strong implications on our understanding...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35853232"
                        open
                      >
                        <summary class="article-expander__title">Geospatial immune heterogeneity reflects the diverse tumor-immune interactions in intrahepatic cholangiocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35853232/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220723212121&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intrahepatic cholangiocarcinoma (iCCA) exhibits extensive intratumoral heterogeneity and extremely high mortality rate. Here, we performed WES, RNA-seq, TCR-seq and multiplexed immunofluorescence on 207 tumor regions from 45 iCCA patients. Over half of iCCA displayed intratumoral heterogeneity of immune infiltration, and iCCA were classified into sparsely, heterogeneously, and highly infiltrated subgroups with distinct immunogenomic characteristics. Sparsely infiltrated tumors displayed active...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35850404"
                        open
                      >
                        <summary class="article-expander__title">Game of clones: Battles in the field of carcinogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35850404/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220723212121&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent advances in bulk sequencing approaches as well as genomic decoding at the single-cell level have revealed surprisingly high somatic mutational burdens in normal tissues, as well as increased our understanding of the landscape of &quot;field cancerization&quot;, that is, molecular and immune alterations in mutagen-exposed normal-appearing tissues that recapitulated those present in tumors. Charting the somatic mutational landscapes in normal tissues can have strong implications on our understanding...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35853232"
                        open
                      >
                        <summary class="article-expander__title">Geospatial immune heterogeneity reflects the diverse tumor-immune interactions in intrahepatic cholangiocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35853232/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220722211347&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intrahepatic cholangiocarcinoma (iCCA) exhibits extensive intratumoral heterogeneity and extremely high mortality rate. Here, we performed WES, RNA-seq, TCR-seq and multiplexed immunofluorescence on 207 tumor regions from 45 iCCA patients. Over half of iCCA displayed intratumoral heterogeneity of immune infiltration, and iCCA were classified into sparsely, heterogeneously, and highly infiltrated subgroups with distinct immunogenomic characteristics. Sparsely infiltrated tumors displayed active...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35850404"
                        open
                      >
                        <summary class="article-expander__title">Game of clones: Battles in the field of carcinogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35850404/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220722211347&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent advances in bulk sequencing approaches as well as genomic decoding at the single-cell level have revealed surprisingly high somatic mutational burdens in normal tissues, as well as increased our understanding of the landscape of &quot;field cancerization&quot;, that is, molecular and immune alterations in mutagen-exposed normal-appearing tissues that recapitulated those present in tumors. Charting the somatic mutational landscapes in normal tissues can have strong implications on our understanding...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35853232"
                        open
                      >
                        <summary class="article-expander__title">Geospatial immune heterogeneity reflects the diverse tumor-immune interactions in intrahepatic cholangiocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35853232/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220721211948&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intrahepatic cholangiocarcinoma (iCCA) exhibits extensive intratumoral heterogeneity and extremely high mortality rate. Here, we performed WES, RNA-seq, TCR-seq and multiplexed immunofluorescence on 207 tumor regions from 45 iCCA patients. Over half of iCCA displayed intratumoral heterogeneity of immune infiltration, and iCCA were classified into sparsely, heterogeneously, and highly infiltrated subgroups with distinct immunogenomic characteristics. Sparsely infiltrated tumors displayed active...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35850404"
                        open
                      >
                        <summary class="article-expander__title">Game of clones: Battles in the field of carcinogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35850404/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220721211948&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent advances in bulk sequencing approaches as well as genomic decoding at the single-cell level have revealed surprisingly high somatic mutational burdens in normal tissues, as well as increased our understanding of the landscape of &quot;field cancerization&quot;, that is, molecular and immune alterations in mutagen-exposed normal-appearing tissues that recapitulated those present in tumors. Charting the somatic mutational landscapes in normal tissues can have strong implications on our understanding...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35853232"
                        open
                      >
                        <summary class="article-expander__title">Geospatial immune heterogeneity reflects the diverse tumor-immune interactions in intrahepatic cholangiocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35853232/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220720211338&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intrahepatic cholangiocarcinoma (iCCA) exhibits extensive intratumoral heterogeneity and extremely high mortality rate. Here, we performed WES, RNA-seq, TCR-seq and multiplexed immunofluorescence on 207 tumor regions from 45 iCCA patients. Over half of iCCA displayed intratumoral heterogeneity of immune infiltration, and iCCA were classified into sparsely, heterogeneously, and highly infiltrated subgroups with distinct immunogenomic characteristics. Sparsely infiltrated tumors displayed active...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35850404"
                        open
                      >
                        <summary class="article-expander__title">Game of clones: Battles in the field of carcinogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35850404/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220720211338&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent advances in bulk sequencing approaches as well as genomic decoding at the single-cell level have revealed surprisingly high somatic mutational burdens in normal tissues, as well as increased our understanding of the landscape of &quot;field cancerization&quot;, that is, molecular and immune alterations in mutagen-exposed normal-appearing tissues that recapitulated those present in tumors. Charting the somatic mutational landscapes in normal tissues can have strong implications on our understanding...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35853232"
                        open
                      >
                        <summary class="article-expander__title">Geospatial immune heterogeneity reflects the diverse tumor-immune interactions in intrahepatic cholangiocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35853232/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220719211909&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intrahepatic cholangiocarcinoma (iCCA) exhibits extensive intratumoral heterogeneity and extremely high mortality rate. Here, we performed WES, RNA-seq, TCR-seq and multiplexed immunofluorescence on 207 tumor regions from 45 iCCA patients. Over half of iCCA displayed intratumoral heterogeneity of immune infiltration, and iCCA were classified into sparsely, heterogeneously, and highly infiltrated subgroups with distinct immunogenomic characteristics. Sparsely infiltrated tumors displayed active...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35850404"
                        open
                      >
                        <summary class="article-expander__title">Game of clones: Battles in the field of carcinogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35850404/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220719211909&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent advances in bulk sequencing approaches as well as genomic decoding at the single-cell level have revealed surprisingly high somatic mutational burdens in normal tissues, as well as increased our understanding of the landscape of &quot;field cancerization&quot;, that is, molecular and immune alterations in mutagen-exposed normal-appearing tissues that recapitulated those present in tumors. Charting the somatic mutational landscapes in normal tissues can have strong implications on our understanding...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-07-18 datetime="2022-07-18T10:00:00.000Z">2022-07-18</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-07-18 datetime="2022-07-18T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608003622&amp;v&#x3D;2.17.7&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35844680"
                        open
                      >
                        <summary class="article-expander__title">Examining disease boundaries: Genetics of myelodysplastic/myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35844680/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220727211458&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic/myeloproliferative neoplasms (MDS/MPN) are clonal myeloid malignancies that are characterized by dysplasia resulting in cytopenias as well as proliferative features such as thrombocytosis or splenomegaly. Recent studies have better defined the genetics underlying this diverse group of disorders. Trisomy 8, monosomy 7, and loss of Y chromosome are the most common cytogenetic abnormalities seen. Chronic myelomonocytic leukemia (CMML) likely develops from early clones with TET2...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35844680"
                        open
                      >
                        <summary class="article-expander__title">Examining disease boundaries: Genetics of myelodysplastic/myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35844680/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220726212041&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic/myeloproliferative neoplasms (MDS/MPN) are clonal myeloid malignancies that are characterized by dysplasia resulting in cytopenias as well as proliferative features such as thrombocytosis or splenomegaly. Recent studies have better defined the genetics underlying this diverse group of disorders. Trisomy 8, monosomy 7, and loss of Y chromosome are the most common cytogenetic abnormalities seen. Chronic myelomonocytic leukemia (CMML) likely develops from early clones with TET2...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35844680"
                        open
                      >
                        <summary class="article-expander__title">Examining disease boundaries: Genetics of myelodysplastic/myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35844680/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220725211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic/myeloproliferative neoplasms (MDS/MPN) are clonal myeloid malignancies that are characterized by dysplasia resulting in cytopenias as well as proliferative features such as thrombocytosis or splenomegaly. Recent studies have better defined the genetics underlying this diverse group of disorders. Trisomy 8, monosomy 7, and loss of Y chromosome are the most common cytogenetic abnormalities seen. Chronic myelomonocytic leukemia (CMML) likely develops from early clones with TET2...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35844680"
                        open
                      >
                        <summary class="article-expander__title">Examining disease boundaries: Genetics of myelodysplastic/myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35844680/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220724211604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic/myeloproliferative neoplasms (MDS/MPN) are clonal myeloid malignancies that are characterized by dysplasia resulting in cytopenias as well as proliferative features such as thrombocytosis or splenomegaly. Recent studies have better defined the genetics underlying this diverse group of disorders. Trisomy 8, monosomy 7, and loss of Y chromosome are the most common cytogenetic abnormalities seen. Chronic myelomonocytic leukemia (CMML) likely develops from early clones with TET2...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35844680"
                        open
                      >
                        <summary class="article-expander__title">Examining disease boundaries: Genetics of myelodysplastic/myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35844680/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220723212121&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic/myeloproliferative neoplasms (MDS/MPN) are clonal myeloid malignancies that are characterized by dysplasia resulting in cytopenias as well as proliferative features such as thrombocytosis or splenomegaly. Recent studies have better defined the genetics underlying this diverse group of disorders. Trisomy 8, monosomy 7, and loss of Y chromosome are the most common cytogenetic abnormalities seen. Chronic myelomonocytic leukemia (CMML) likely develops from early clones with TET2...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35844680"
                        open
                      >
                        <summary class="article-expander__title">Examining disease boundaries: Genetics of myelodysplastic/myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35844680/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220722211347&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic/myeloproliferative neoplasms (MDS/MPN) are clonal myeloid malignancies that are characterized by dysplasia resulting in cytopenias as well as proliferative features such as thrombocytosis or splenomegaly. Recent studies have better defined the genetics underlying this diverse group of disorders. Trisomy 8, monosomy 7, and loss of Y chromosome are the most common cytogenetic abnormalities seen. Chronic myelomonocytic leukemia (CMML) likely develops from early clones with TET2...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35844680"
                        open
                      >
                        <summary class="article-expander__title">Examining disease boundaries: Genetics of myelodysplastic/myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35844680/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220721211948&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic/myeloproliferative neoplasms (MDS/MPN) are clonal myeloid malignancies that are characterized by dysplasia resulting in cytopenias as well as proliferative features such as thrombocytosis or splenomegaly. Recent studies have better defined the genetics underlying this diverse group of disorders. Trisomy 8, monosomy 7, and loss of Y chromosome are the most common cytogenetic abnormalities seen. Chronic myelomonocytic leukemia (CMML) likely develops from early clones with TET2...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35844680"
                        open
                      >
                        <summary class="article-expander__title">Examining disease boundaries: Genetics of myelodysplastic/myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35844680/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220720211338&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic/myeloproliferative neoplasms (MDS/MPN) are clonal myeloid malignancies that are characterized by dysplasia resulting in cytopenias as well as proliferative features such as thrombocytosis or splenomegaly. Recent studies have better defined the genetics underlying this diverse group of disorders. Trisomy 8, monosomy 7, and loss of Y chromosome are the most common cytogenetic abnormalities seen. Chronic myelomonocytic leukemia (CMML) likely develops from early clones with TET2...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35844680"
                        open
                      >
                        <summary class="article-expander__title">Examining disease boundaries: Genetics of myelodysplastic/myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35844680/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220719211909&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic/myeloproliferative neoplasms (MDS/MPN) are clonal myeloid malignancies that are characterized by dysplasia resulting in cytopenias as well as proliferative features such as thrombocytosis or splenomegaly. Recent studies have better defined the genetics underlying this diverse group of disorders. Trisomy 8, monosomy 7, and loss of Y chromosome are the most common cytogenetic abnormalities seen. Chronic myelomonocytic leukemia (CMML) likely develops from early clones with TET2...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35844680"
                        open
                      >
                        <summary class="article-expander__title">Examining disease boundaries: Genetics of myelodysplastic/myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35844680/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220718212208&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic/myeloproliferative neoplasms (MDS/MPN) are clonal myeloid malignancies that are characterized by dysplasia resulting in cytopenias as well as proliferative features such as thrombocytosis or splenomegaly. Recent studies have better defined the genetics underlying this diverse group of disorders. Trisomy 8, monosomy 7, and loss of Y chromosome are the most common cytogenetic abnormalities seen. Chronic myelomonocytic leukemia (CMML) likely develops from early clones with TET2...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608010801&amp;v&#x3D;2.17.7&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35844680"
                        open
                      >
                        <summary class="article-expander__title">Examining disease boundaries: Genetics of myelodysplastic/myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35844680/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220801212447&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic/myeloproliferative neoplasms (MDS/MPN) are clonal myeloid malignancies that are characterized by dysplasia resulting in cytopenias as well as proliferative features such as thrombocytosis or splenomegaly. Recent studies have better defined the genetics underlying this diverse group of disorders. Trisomy 8, monosomy 7, and loss of Y chromosome are the most common cytogenetic abnormalities seen. Chronic myelomonocytic leukemia (CMML) likely develops from early clones with TET2...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35844680"
                        open
                      >
                        <summary class="article-expander__title">Examining disease boundaries: Genetics of myelodysplastic/myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35844680/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220731212525&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic/myeloproliferative neoplasms (MDS/MPN) are clonal myeloid malignancies that are characterized by dysplasia resulting in cytopenias as well as proliferative features such as thrombocytosis or splenomegaly. Recent studies have better defined the genetics underlying this diverse group of disorders. Trisomy 8, monosomy 7, and loss of Y chromosome are the most common cytogenetic abnormalities seen. Chronic myelomonocytic leukemia (CMML) likely develops from early clones with TET2...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35844680"
                        open
                      >
                        <summary class="article-expander__title">Examining disease boundaries: Genetics of myelodysplastic/myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35844680/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220730212309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic/myeloproliferative neoplasms (MDS/MPN) are clonal myeloid malignancies that are characterized by dysplasia resulting in cytopenias as well as proliferative features such as thrombocytosis or splenomegaly. Recent studies have better defined the genetics underlying this diverse group of disorders. Trisomy 8, monosomy 7, and loss of Y chromosome are the most common cytogenetic abnormalities seen. Chronic myelomonocytic leukemia (CMML) likely develops from early clones with TET2...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35844680"
                        open
                      >
                        <summary class="article-expander__title">Examining disease boundaries: Genetics of myelodysplastic/myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35844680/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220729211332&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic/myeloproliferative neoplasms (MDS/MPN) are clonal myeloid malignancies that are characterized by dysplasia resulting in cytopenias as well as proliferative features such as thrombocytosis or splenomegaly. Recent studies have better defined the genetics underlying this diverse group of disorders. Trisomy 8, monosomy 7, and loss of Y chromosome are the most common cytogenetic abnormalities seen. Chronic myelomonocytic leukemia (CMML) likely develops from early clones with TET2...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35844680"
                        open
                      >
                        <summary class="article-expander__title">Examining disease boundaries: Genetics of myelodysplastic/myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35844680/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220728211733&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic/myeloproliferative neoplasms (MDS/MPN) are clonal myeloid malignancies that are characterized by dysplasia resulting in cytopenias as well as proliferative features such as thrombocytosis or splenomegaly. Recent studies have better defined the genetics underlying this diverse group of disorders. Trisomy 8, monosomy 7, and loss of Y chromosome are the most common cytogenetic abnormalities seen. Chronic myelomonocytic leukemia (CMML) likely develops from early clones with TET2...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35844680"
                        open
                      >
                        <summary class="article-expander__title">Examining disease boundaries: Genetics of myelodysplastic/myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35844680/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220727211458&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic/myeloproliferative neoplasms (MDS/MPN) are clonal myeloid malignancies that are characterized by dysplasia resulting in cytopenias as well as proliferative features such as thrombocytosis or splenomegaly. Recent studies have better defined the genetics underlying this diverse group of disorders. Trisomy 8, monosomy 7, and loss of Y chromosome are the most common cytogenetic abnormalities seen. Chronic myelomonocytic leukemia (CMML) likely develops from early clones with TET2...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35844680"
                        open
                      >
                        <summary class="article-expander__title">Examining disease boundaries: Genetics of myelodysplastic/myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35844680/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220726212040&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic/myeloproliferative neoplasms (MDS/MPN) are clonal myeloid malignancies that are characterized by dysplasia resulting in cytopenias as well as proliferative features such as thrombocytosis or splenomegaly. Recent studies have better defined the genetics underlying this diverse group of disorders. Trisomy 8, monosomy 7, and loss of Y chromosome are the most common cytogenetic abnormalities seen. Chronic myelomonocytic leukemia (CMML) likely develops from early clones with TET2...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35844680"
                        open
                      >
                        <summary class="article-expander__title">Examining disease boundaries: Genetics of myelodysplastic/myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35844680/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220725211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic/myeloproliferative neoplasms (MDS/MPN) are clonal myeloid malignancies that are characterized by dysplasia resulting in cytopenias as well as proliferative features such as thrombocytosis or splenomegaly. Recent studies have better defined the genetics underlying this diverse group of disorders. Trisomy 8, monosomy 7, and loss of Y chromosome are the most common cytogenetic abnormalities seen. Chronic myelomonocytic leukemia (CMML) likely develops from early clones with TET2...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35844680"
                        open
                      >
                        <summary class="article-expander__title">Examining disease boundaries: Genetics of myelodysplastic/myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35844680/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220724211605&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic/myeloproliferative neoplasms (MDS/MPN) are clonal myeloid malignancies that are characterized by dysplasia resulting in cytopenias as well as proliferative features such as thrombocytosis or splenomegaly. Recent studies have better defined the genetics underlying this diverse group of disorders. Trisomy 8, monosomy 7, and loss of Y chromosome are the most common cytogenetic abnormalities seen. Chronic myelomonocytic leukemia (CMML) likely develops from early clones with TET2...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35844680"
                        open
                      >
                        <summary class="article-expander__title">Examining disease boundaries: Genetics of myelodysplastic/myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35844680/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220723212121&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic/myeloproliferative neoplasms (MDS/MPN) are clonal myeloid malignancies that are characterized by dysplasia resulting in cytopenias as well as proliferative features such as thrombocytosis or splenomegaly. Recent studies have better defined the genetics underlying this diverse group of disorders. Trisomy 8, monosomy 7, and loss of Y chromosome are the most common cytogenetic abnormalities seen. Chronic myelomonocytic leukemia (CMML) likely develops from early clones with TET2...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35844680"
                        open
                      >
                        <summary class="article-expander__title">Examining disease boundaries: Genetics of myelodysplastic/myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35844680/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220722211347&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic/myeloproliferative neoplasms (MDS/MPN) are clonal myeloid malignancies that are characterized by dysplasia resulting in cytopenias as well as proliferative features such as thrombocytosis or splenomegaly. Recent studies have better defined the genetics underlying this diverse group of disorders. Trisomy 8, monosomy 7, and loss of Y chromosome are the most common cytogenetic abnormalities seen. Chronic myelomonocytic leukemia (CMML) likely develops from early clones with TET2...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35844680"
                        open
                      >
                        <summary class="article-expander__title">Examining disease boundaries: Genetics of myelodysplastic/myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35844680/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220721211948&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic/myeloproliferative neoplasms (MDS/MPN) are clonal myeloid malignancies that are characterized by dysplasia resulting in cytopenias as well as proliferative features such as thrombocytosis or splenomegaly. Recent studies have better defined the genetics underlying this diverse group of disorders. Trisomy 8, monosomy 7, and loss of Y chromosome are the most common cytogenetic abnormalities seen. Chronic myelomonocytic leukemia (CMML) likely develops from early clones with TET2...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35844680"
                        open
                      >
                        <summary class="article-expander__title">Examining disease boundaries: Genetics of myelodysplastic/myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35844680/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220720211338&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic/myeloproliferative neoplasms (MDS/MPN) are clonal myeloid malignancies that are characterized by dysplasia resulting in cytopenias as well as proliferative features such as thrombocytosis or splenomegaly. Recent studies have better defined the genetics underlying this diverse group of disorders. Trisomy 8, monosomy 7, and loss of Y chromosome are the most common cytogenetic abnormalities seen. Chronic myelomonocytic leukemia (CMML) likely develops from early clones with TET2...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35844680"
                        open
                      >
                        <summary class="article-expander__title">Examining disease boundaries: Genetics of myelodysplastic/myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35844680/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220719211909&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic/myeloproliferative neoplasms (MDS/MPN) are clonal myeloid malignancies that are characterized by dysplasia resulting in cytopenias as well as proliferative features such as thrombocytosis or splenomegaly. Recent studies have better defined the genetics underlying this diverse group of disorders. Trisomy 8, monosomy 7, and loss of Y chromosome are the most common cytogenetic abnormalities seen. Chronic myelomonocytic leukemia (CMML) likely develops from early clones with TET2...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35844680"
                        open
                      >
                        <summary class="article-expander__title">Examining disease boundaries: Genetics of myelodysplastic/myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35844680/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220718212208&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic/myeloproliferative neoplasms (MDS/MPN) are clonal myeloid malignancies that are characterized by dysplasia resulting in cytopenias as well as proliferative features such as thrombocytosis or splenomegaly. Recent studies have better defined the genetics underlying this diverse group of disorders. Trisomy 8, monosomy 7, and loss of Y chromosome are the most common cytogenetic abnormalities seen. Chronic myelomonocytic leukemia (CMML) likely develops from early clones with TET2...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-07-15 datetime="2022-07-15T10:00:00.000Z">2022-07-15</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-07-15 datetime="2022-07-15T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608003622&amp;v&#x3D;2.17.7&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35840818"
                        open
                      >
                        <summary class="article-expander__title">Molecular characterization of triple-negative myeloproliferative neoplasms by next-generation sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35840818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220726212041&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The role of next-generation sequencing (NGS) in identifying mutations in the driver, epigenetic regulator, RNA splicing, and signaling pathway genes in myeloproliferative neoplasms (MPNs) has contributed substantially to our understanding of the disease pathogenesis as well as disease evolution. NGS aids in determining the clonal nature of the disease in a subset of these disorders where mutations in the driver genes are not detected. There is a paucity of real-world data on the utility of this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35840818"
                        open
                      >
                        <summary class="article-expander__title">Molecular characterization of triple-negative myeloproliferative neoplasms by next-generation sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35840818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220725211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The role of next-generation sequencing (NGS) in identifying mutations in the driver, epigenetic regulator, RNA splicing, and signaling pathway genes in myeloproliferative neoplasms (MPNs) has contributed substantially to our understanding of the disease pathogenesis as well as disease evolution. NGS aids in determining the clonal nature of the disease in a subset of these disorders where mutations in the driver genes are not detected. There is a paucity of real-world data on the utility of this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35840818"
                        open
                      >
                        <summary class="article-expander__title">Molecular characterization of triple-negative myeloproliferative neoplasms by next-generation sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35840818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220724211604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The role of next-generation sequencing (NGS) in identifying mutations in the driver, epigenetic regulator, RNA splicing, and signaling pathway genes in myeloproliferative neoplasms (MPNs) has contributed substantially to our understanding of the disease pathogenesis as well as disease evolution. NGS aids in determining the clonal nature of the disease in a subset of these disorders where mutations in the driver genes are not detected. There is a paucity of real-world data on the utility of this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35840818"
                        open
                      >
                        <summary class="article-expander__title">Molecular characterization of triple-negative myeloproliferative neoplasms by next-generation sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35840818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220723212121&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The role of next-generation sequencing (NGS) in identifying mutations in the driver, epigenetic regulator, RNA splicing, and signaling pathway genes in myeloproliferative neoplasms (MPNs) has contributed substantially to our understanding of the disease pathogenesis as well as disease evolution. NGS aids in determining the clonal nature of the disease in a subset of these disorders where mutations in the driver genes are not detected. There is a paucity of real-world data on the utility of this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35840818"
                        open
                      >
                        <summary class="article-expander__title">Molecular characterization of triple-negative myeloproliferative neoplasms by next-generation sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35840818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220722211347&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The role of next-generation sequencing (NGS) in identifying mutations in the driver, epigenetic regulator, RNA splicing, and signaling pathway genes in myeloproliferative neoplasms (MPNs) has contributed substantially to our understanding of the disease pathogenesis as well as disease evolution. NGS aids in determining the clonal nature of the disease in a subset of these disorders where mutations in the driver genes are not detected. There is a paucity of real-world data on the utility of this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35840818"
                        open
                      >
                        <summary class="article-expander__title">Molecular characterization of triple-negative myeloproliferative neoplasms by next-generation sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35840818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220721211948&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The role of next-generation sequencing (NGS) in identifying mutations in the driver, epigenetic regulator, RNA splicing, and signaling pathway genes in myeloproliferative neoplasms (MPNs) has contributed substantially to our understanding of the disease pathogenesis as well as disease evolution. NGS aids in determining the clonal nature of the disease in a subset of these disorders where mutations in the driver genes are not detected. There is a paucity of real-world data on the utility of this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35840818"
                        open
                      >
                        <summary class="article-expander__title">Molecular characterization of triple-negative myeloproliferative neoplasms by next-generation sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35840818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220720211338&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The role of next-generation sequencing (NGS) in identifying mutations in the driver, epigenetic regulator, RNA splicing, and signaling pathway genes in myeloproliferative neoplasms (MPNs) has contributed substantially to our understanding of the disease pathogenesis as well as disease evolution. NGS aids in determining the clonal nature of the disease in a subset of these disorders where mutations in the driver genes are not detected. There is a paucity of real-world data on the utility of this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35840818"
                        open
                      >
                        <summary class="article-expander__title">Molecular characterization of triple-negative myeloproliferative neoplasms by next-generation sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35840818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220719211909&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The role of next-generation sequencing (NGS) in identifying mutations in the driver, epigenetic regulator, RNA splicing, and signaling pathway genes in myeloproliferative neoplasms (MPNs) has contributed substantially to our understanding of the disease pathogenesis as well as disease evolution. NGS aids in determining the clonal nature of the disease in a subset of these disorders where mutations in the driver genes are not detected. There is a paucity of real-world data on the utility of this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35840818"
                        open
                      >
                        <summary class="article-expander__title">Molecular characterization of triple-negative myeloproliferative neoplasms by next-generation sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35840818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220718212208&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The role of next-generation sequencing (NGS) in identifying mutations in the driver, epigenetic regulator, RNA splicing, and signaling pathway genes in myeloproliferative neoplasms (MPNs) has contributed substantially to our understanding of the disease pathogenesis as well as disease evolution. NGS aids in determining the clonal nature of the disease in a subset of these disorders where mutations in the driver genes are not detected. There is a paucity of real-world data on the utility of this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35840818"
                        open
                      >
                        <summary class="article-expander__title">Molecular characterization of triple-negative myeloproliferative neoplasms by next-generation sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35840818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220717211412&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The role of next-generation sequencing (NGS) in identifying mutations in the driver, epigenetic regulator, RNA splicing, and signaling pathway genes in myeloproliferative neoplasms (MPNs) has contributed substantially to our understanding of the disease pathogenesis as well as disease evolution. NGS aids in determining the clonal nature of the disease in a subset of these disorders where mutations in the driver genes are not detected. There is a paucity of real-world data on the utility of this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35840818"
                        open
                      >
                        <summary class="article-expander__title">Molecular characterization of triple-negative myeloproliferative neoplasms by next-generation sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35840818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220716212224&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The role of next-generation sequencing (NGS) in identifying mutations in the driver, epigenetic regulator, RNA splicing, and signaling pathway genes in myeloproliferative neoplasms (MPNs) has contributed substantially to our understanding of the disease pathogenesis as well as disease evolution. NGS aids in determining the clonal nature of the disease in a subset of these disorders where mutations in the driver genes are not detected. There is a paucity of real-world data on the utility of this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608010801&amp;v&#x3D;2.17.7&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35840818"
                        open
                      >
                        <summary class="article-expander__title">Molecular characterization of triple-negative myeloproliferative neoplasms by next-generation sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35840818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220810210900&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The role of next-generation sequencing (NGS) in identifying mutations in the driver, epigenetic regulator, RNA splicing, and signaling pathway genes in myeloproliferative neoplasms (MPNs) has contributed substantially to our understanding of the disease pathogenesis as well as disease evolution. NGS aids in determining the clonal nature of the disease in a subset of these disorders where mutations in the driver genes are not detected. There is a paucity of real-world data on the utility of this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35840818"
                        open
                      >
                        <summary class="article-expander__title">Molecular characterization of triple-negative myeloproliferative neoplasms by next-generation sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35840818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220809211355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The role of next-generation sequencing (NGS) in identifying mutations in the driver, epigenetic regulator, RNA splicing, and signaling pathway genes in myeloproliferative neoplasms (MPNs) has contributed substantially to our understanding of the disease pathogenesis as well as disease evolution. NGS aids in determining the clonal nature of the disease in a subset of these disorders where mutations in the driver genes are not detected. There is a paucity of real-world data on the utility of this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35840818"
                        open
                      >
                        <summary class="article-expander__title">Molecular characterization of triple-negative myeloproliferative neoplasms by next-generation sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35840818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220808212152&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The role of next-generation sequencing (NGS) in identifying mutations in the driver, epigenetic regulator, RNA splicing, and signaling pathway genes in myeloproliferative neoplasms (MPNs) has contributed substantially to our understanding of the disease pathogenesis as well as disease evolution. NGS aids in determining the clonal nature of the disease in a subset of these disorders where mutations in the driver genes are not detected. There is a paucity of real-world data on the utility of this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35840818"
                        open
                      >
                        <summary class="article-expander__title">Molecular characterization of triple-negative myeloproliferative neoplasms by next-generation sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35840818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220807211336&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The role of next-generation sequencing (NGS) in identifying mutations in the driver, epigenetic regulator, RNA splicing, and signaling pathway genes in myeloproliferative neoplasms (MPNs) has contributed substantially to our understanding of the disease pathogenesis as well as disease evolution. NGS aids in determining the clonal nature of the disease in a subset of these disorders where mutations in the driver genes are not detected. There is a paucity of real-world data on the utility of this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35840818"
                        open
                      >
                        <summary class="article-expander__title">Molecular characterization of triple-negative myeloproliferative neoplasms by next-generation sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35840818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220806211830&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The role of next-generation sequencing (NGS) in identifying mutations in the driver, epigenetic regulator, RNA splicing, and signaling pathway genes in myeloproliferative neoplasms (MPNs) has contributed substantially to our understanding of the disease pathogenesis as well as disease evolution. NGS aids in determining the clonal nature of the disease in a subset of these disorders where mutations in the driver genes are not detected. There is a paucity of real-world data on the utility of this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35840818"
                        open
                      >
                        <summary class="article-expander__title">Molecular characterization of triple-negative myeloproliferative neoplasms by next-generation sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35840818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220805210849&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The role of next-generation sequencing (NGS) in identifying mutations in the driver, epigenetic regulator, RNA splicing, and signaling pathway genes in myeloproliferative neoplasms (MPNs) has contributed substantially to our understanding of the disease pathogenesis as well as disease evolution. NGS aids in determining the clonal nature of the disease in a subset of these disorders where mutations in the driver genes are not detected. There is a paucity of real-world data on the utility of this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35840818"
                        open
                      >
                        <summary class="article-expander__title">Molecular characterization of triple-negative myeloproliferative neoplasms by next-generation sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35840818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220804212147&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The role of next-generation sequencing (NGS) in identifying mutations in the driver, epigenetic regulator, RNA splicing, and signaling pathway genes in myeloproliferative neoplasms (MPNs) has contributed substantially to our understanding of the disease pathogenesis as well as disease evolution. NGS aids in determining the clonal nature of the disease in a subset of these disorders where mutations in the driver genes are not detected. There is a paucity of real-world data on the utility of this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35840818"
                        open
                      >
                        <summary class="article-expander__title">Molecular characterization of triple-negative myeloproliferative neoplasms by next-generation sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35840818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220803212051&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The role of next-generation sequencing (NGS) in identifying mutations in the driver, epigenetic regulator, RNA splicing, and signaling pathway genes in myeloproliferative neoplasms (MPNs) has contributed substantially to our understanding of the disease pathogenesis as well as disease evolution. NGS aids in determining the clonal nature of the disease in a subset of these disorders where mutations in the driver genes are not detected. There is a paucity of real-world data on the utility of this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35840818"
                        open
                      >
                        <summary class="article-expander__title">Molecular characterization of triple-negative myeloproliferative neoplasms by next-generation sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35840818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220802212153&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The role of next-generation sequencing (NGS) in identifying mutations in the driver, epigenetic regulator, RNA splicing, and signaling pathway genes in myeloproliferative neoplasms (MPNs) has contributed substantially to our understanding of the disease pathogenesis as well as disease evolution. NGS aids in determining the clonal nature of the disease in a subset of these disorders where mutations in the driver genes are not detected. There is a paucity of real-world data on the utility of this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35840818"
                        open
                      >
                        <summary class="article-expander__title">Molecular characterization of triple-negative myeloproliferative neoplasms by next-generation sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35840818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220801212447&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The role of next-generation sequencing (NGS) in identifying mutations in the driver, epigenetic regulator, RNA splicing, and signaling pathway genes in myeloproliferative neoplasms (MPNs) has contributed substantially to our understanding of the disease pathogenesis as well as disease evolution. NGS aids in determining the clonal nature of the disease in a subset of these disorders where mutations in the driver genes are not detected. There is a paucity of real-world data on the utility of this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35840818"
                        open
                      >
                        <summary class="article-expander__title">Molecular characterization of triple-negative myeloproliferative neoplasms by next-generation sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35840818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220731212525&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The role of next-generation sequencing (NGS) in identifying mutations in the driver, epigenetic regulator, RNA splicing, and signaling pathway genes in myeloproliferative neoplasms (MPNs) has contributed substantially to our understanding of the disease pathogenesis as well as disease evolution. NGS aids in determining the clonal nature of the disease in a subset of these disorders where mutations in the driver genes are not detected. There is a paucity of real-world data on the utility of this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35840818"
                        open
                      >
                        <summary class="article-expander__title">Molecular characterization of triple-negative myeloproliferative neoplasms by next-generation sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35840818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220730212309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The role of next-generation sequencing (NGS) in identifying mutations in the driver, epigenetic regulator, RNA splicing, and signaling pathway genes in myeloproliferative neoplasms (MPNs) has contributed substantially to our understanding of the disease pathogenesis as well as disease evolution. NGS aids in determining the clonal nature of the disease in a subset of these disorders where mutations in the driver genes are not detected. There is a paucity of real-world data on the utility of this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35840818"
                        open
                      >
                        <summary class="article-expander__title">Molecular characterization of triple-negative myeloproliferative neoplasms by next-generation sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35840818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220729211332&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The role of next-generation sequencing (NGS) in identifying mutations in the driver, epigenetic regulator, RNA splicing, and signaling pathway genes in myeloproliferative neoplasms (MPNs) has contributed substantially to our understanding of the disease pathogenesis as well as disease evolution. NGS aids in determining the clonal nature of the disease in a subset of these disorders where mutations in the driver genes are not detected. There is a paucity of real-world data on the utility of this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35840818"
                        open
                      >
                        <summary class="article-expander__title">Molecular characterization of triple-negative myeloproliferative neoplasms by next-generation sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35840818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220728211733&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The role of next-generation sequencing (NGS) in identifying mutations in the driver, epigenetic regulator, RNA splicing, and signaling pathway genes in myeloproliferative neoplasms (MPNs) has contributed substantially to our understanding of the disease pathogenesis as well as disease evolution. NGS aids in determining the clonal nature of the disease in a subset of these disorders where mutations in the driver genes are not detected. There is a paucity of real-world data on the utility of this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35840818"
                        open
                      >
                        <summary class="article-expander__title">Molecular characterization of triple-negative myeloproliferative neoplasms by next-generation sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35840818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220727211458&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The role of next-generation sequencing (NGS) in identifying mutations in the driver, epigenetic regulator, RNA splicing, and signaling pathway genes in myeloproliferative neoplasms (MPNs) has contributed substantially to our understanding of the disease pathogenesis as well as disease evolution. NGS aids in determining the clonal nature of the disease in a subset of these disorders where mutations in the driver genes are not detected. There is a paucity of real-world data on the utility of this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35840818"
                        open
                      >
                        <summary class="article-expander__title">Molecular characterization of triple-negative myeloproliferative neoplasms by next-generation sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35840818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220726212040&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The role of next-generation sequencing (NGS) in identifying mutations in the driver, epigenetic regulator, RNA splicing, and signaling pathway genes in myeloproliferative neoplasms (MPNs) has contributed substantially to our understanding of the disease pathogenesis as well as disease evolution. NGS aids in determining the clonal nature of the disease in a subset of these disorders where mutations in the driver genes are not detected. There is a paucity of real-world data on the utility of this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35840818"
                        open
                      >
                        <summary class="article-expander__title">Molecular characterization of triple-negative myeloproliferative neoplasms by next-generation sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35840818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220725211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The role of next-generation sequencing (NGS) in identifying mutations in the driver, epigenetic regulator, RNA splicing, and signaling pathway genes in myeloproliferative neoplasms (MPNs) has contributed substantially to our understanding of the disease pathogenesis as well as disease evolution. NGS aids in determining the clonal nature of the disease in a subset of these disorders where mutations in the driver genes are not detected. There is a paucity of real-world data on the utility of this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35840818"
                        open
                      >
                        <summary class="article-expander__title">Molecular characterization of triple-negative myeloproliferative neoplasms by next-generation sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35840818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220724211605&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The role of next-generation sequencing (NGS) in identifying mutations in the driver, epigenetic regulator, RNA splicing, and signaling pathway genes in myeloproliferative neoplasms (MPNs) has contributed substantially to our understanding of the disease pathogenesis as well as disease evolution. NGS aids in determining the clonal nature of the disease in a subset of these disorders where mutations in the driver genes are not detected. There is a paucity of real-world data on the utility of this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35840818"
                        open
                      >
                        <summary class="article-expander__title">Molecular characterization of triple-negative myeloproliferative neoplasms by next-generation sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35840818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220723212121&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The role of next-generation sequencing (NGS) in identifying mutations in the driver, epigenetic regulator, RNA splicing, and signaling pathway genes in myeloproliferative neoplasms (MPNs) has contributed substantially to our understanding of the disease pathogenesis as well as disease evolution. NGS aids in determining the clonal nature of the disease in a subset of these disorders where mutations in the driver genes are not detected. There is a paucity of real-world data on the utility of this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35840818"
                        open
                      >
                        <summary class="article-expander__title">Molecular characterization of triple-negative myeloproliferative neoplasms by next-generation sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35840818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220722211347&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The role of next-generation sequencing (NGS) in identifying mutations in the driver, epigenetic regulator, RNA splicing, and signaling pathway genes in myeloproliferative neoplasms (MPNs) has contributed substantially to our understanding of the disease pathogenesis as well as disease evolution. NGS aids in determining the clonal nature of the disease in a subset of these disorders where mutations in the driver genes are not detected. There is a paucity of real-world data on the utility of this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35840818"
                        open
                      >
                        <summary class="article-expander__title">Molecular characterization of triple-negative myeloproliferative neoplasms by next-generation sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35840818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220721211948&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The role of next-generation sequencing (NGS) in identifying mutations in the driver, epigenetic regulator, RNA splicing, and signaling pathway genes in myeloproliferative neoplasms (MPNs) has contributed substantially to our understanding of the disease pathogenesis as well as disease evolution. NGS aids in determining the clonal nature of the disease in a subset of these disorders where mutations in the driver genes are not detected. There is a paucity of real-world data on the utility of this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35840818"
                        open
                      >
                        <summary class="article-expander__title">Molecular characterization of triple-negative myeloproliferative neoplasms by next-generation sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35840818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220720211338&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The role of next-generation sequencing (NGS) in identifying mutations in the driver, epigenetic regulator, RNA splicing, and signaling pathway genes in myeloproliferative neoplasms (MPNs) has contributed substantially to our understanding of the disease pathogenesis as well as disease evolution. NGS aids in determining the clonal nature of the disease in a subset of these disorders where mutations in the driver genes are not detected. There is a paucity of real-world data on the utility of this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35840818"
                        open
                      >
                        <summary class="article-expander__title">Molecular characterization of triple-negative myeloproliferative neoplasms by next-generation sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35840818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220719211909&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The role of next-generation sequencing (NGS) in identifying mutations in the driver, epigenetic regulator, RNA splicing, and signaling pathway genes in myeloproliferative neoplasms (MPNs) has contributed substantially to our understanding of the disease pathogenesis as well as disease evolution. NGS aids in determining the clonal nature of the disease in a subset of these disorders where mutations in the driver genes are not detected. There is a paucity of real-world data on the utility of this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35840818"
                        open
                      >
                        <summary class="article-expander__title">Molecular characterization of triple-negative myeloproliferative neoplasms by next-generation sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35840818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220718212208&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The role of next-generation sequencing (NGS) in identifying mutations in the driver, epigenetic regulator, RNA splicing, and signaling pathway genes in myeloproliferative neoplasms (MPNs) has contributed substantially to our understanding of the disease pathogenesis as well as disease evolution. NGS aids in determining the clonal nature of the disease in a subset of these disorders where mutations in the driver genes are not detected. There is a paucity of real-world data on the utility of this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35840818"
                        open
                      >
                        <summary class="article-expander__title">Molecular characterization of triple-negative myeloproliferative neoplasms by next-generation sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35840818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220717211412&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The role of next-generation sequencing (NGS) in identifying mutations in the driver, epigenetic regulator, RNA splicing, and signaling pathway genes in myeloproliferative neoplasms (MPNs) has contributed substantially to our understanding of the disease pathogenesis as well as disease evolution. NGS aids in determining the clonal nature of the disease in a subset of these disorders where mutations in the driver genes are not detected. There is a paucity of real-world data on the utility of this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35840818"
                        open
                      >
                        <summary class="article-expander__title">Molecular characterization of triple-negative myeloproliferative neoplasms by next-generation sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35840818/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220716212224&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The role of next-generation sequencing (NGS) in identifying mutations in the driver, epigenetic regulator, RNA splicing, and signaling pathway genes in myeloproliferative neoplasms (MPNs) has contributed substantially to our understanding of the disease pathogenesis as well as disease evolution. NGS aids in determining the clonal nature of the disease in a subset of these disorders where mutations in the driver genes are not detected. There is a paucity of real-world data on the utility of this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-07-14 datetime="2022-07-14T10:00:00.000Z">2022-07-14</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-07-14 datetime="2022-07-14T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608003622&amp;v&#x3D;2.17.7&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35835912"
                        open
                      >
                        <summary class="article-expander__title">Genome-wide analyses of 200,453 individuals yield new insights into the causes and consequences of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35835912/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220726212041&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH), the clonal expansion of a blood stem cell and its progeny driven by somatic driver mutations, affects over a third of people, yet remains poorly understood. Here we analyze genetic data from 200,453 UK Biobank participants to map the landscape of inherited predisposition to CH, increasing the number of germline associations with CH in European-ancestry populations from 4 to 14. Genes at new loci implicate DNA damage repair (PARP1, ATM, CHEK2), hematopoietic stem cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35832562"
                        open
                      >
                        <summary class="article-expander__title">A Novel Prognostic Scoring Model for Myelodysplastic Syndrome Patients With SF3B1 Mutation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35832562/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220726212041&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The outcomes of myelodysplastic syndrome (MDS) patients with SF3B1 mutation, despite identified as a favorable prognostic biomarker, are variable. To comprehend the heterogeneity in clinical characteristics and outcomes, we reviewed 140 MDS patients with SF3B1 mutation in Zhejiang province of China. Seventy-three (52.1%) patients diagnosed as MDS with ring sideroblasts (MDS-RS) following the 2016 World Health Organization (WHO) classification and 118 (84.3%) patients belonged to lower risk...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35835912"
                        open
                      >
                        <summary class="article-expander__title">Genome-wide analyses of 200,453 individuals yield new insights into the causes and consequences of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35835912/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220725211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH), the clonal expansion of a blood stem cell and its progeny driven by somatic driver mutations, affects over a third of people, yet remains poorly understood. Here we analyze genetic data from 200,453 UK Biobank participants to map the landscape of inherited predisposition to CH, increasing the number of germline associations with CH in European-ancestry populations from 4 to 14. Genes at new loci implicate DNA damage repair (PARP1, ATM, CHEK2), hematopoietic stem cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35832562"
                        open
                      >
                        <summary class="article-expander__title">A Novel Prognostic Scoring Model for Myelodysplastic Syndrome Patients With SF3B1 Mutation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35832562/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220725211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The outcomes of myelodysplastic syndrome (MDS) patients with SF3B1 mutation, despite identified as a favorable prognostic biomarker, are variable. To comprehend the heterogeneity in clinical characteristics and outcomes, we reviewed 140 MDS patients with SF3B1 mutation in Zhejiang province of China. Seventy-three (52.1%) patients diagnosed as MDS with ring sideroblasts (MDS-RS) following the 2016 World Health Organization (WHO) classification and 118 (84.3%) patients belonged to lower risk...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35835912"
                        open
                      >
                        <summary class="article-expander__title">Genome-wide analyses of 200,453 individuals yield new insights into the causes and consequences of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35835912/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220724211604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH), the clonal expansion of a blood stem cell and its progeny driven by somatic driver mutations, affects over a third of people, yet remains poorly understood. Here we analyze genetic data from 200,453 UK Biobank participants to map the landscape of inherited predisposition to CH, increasing the number of germline associations with CH in European-ancestry populations from 4 to 14. Genes at new loci implicate DNA damage repair (PARP1, ATM, CHEK2), hematopoietic stem cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35832562"
                        open
                      >
                        <summary class="article-expander__title">A Novel Prognostic Scoring Model for Myelodysplastic Syndrome Patients With SF3B1 Mutation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35832562/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220724211604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The outcomes of myelodysplastic syndrome (MDS) patients with SF3B1 mutation, despite identified as a favorable prognostic biomarker, are variable. To comprehend the heterogeneity in clinical characteristics and outcomes, we reviewed 140 MDS patients with SF3B1 mutation in Zhejiang province of China. Seventy-three (52.1%) patients diagnosed as MDS with ring sideroblasts (MDS-RS) following the 2016 World Health Organization (WHO) classification and 118 (84.3%) patients belonged to lower risk...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35835912"
                        open
                      >
                        <summary class="article-expander__title">Genome-wide analyses of 200,453 individuals yield new insights into the causes and consequences of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35835912/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220723212121&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH), the clonal expansion of a blood stem cell and its progeny driven by somatic driver mutations, affects over a third of people, yet remains poorly understood. Here we analyze genetic data from 200,453 UK Biobank participants to map the landscape of inherited predisposition to CH, increasing the number of germline associations with CH in European-ancestry populations from 4 to 14. Genes at new loci implicate DNA damage repair (PARP1, ATM, CHEK2), hematopoietic stem cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35832562"
                        open
                      >
                        <summary class="article-expander__title">A Novel Prognostic Scoring Model for Myelodysplastic Syndrome Patients With SF3B1 Mutation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35832562/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220723212121&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The outcomes of myelodysplastic syndrome (MDS) patients with SF3B1 mutation, despite identified as a favorable prognostic biomarker, are variable. To comprehend the heterogeneity in clinical characteristics and outcomes, we reviewed 140 MDS patients with SF3B1 mutation in Zhejiang province of China. Seventy-three (52.1%) patients diagnosed as MDS with ring sideroblasts (MDS-RS) following the 2016 World Health Organization (WHO) classification and 118 (84.3%) patients belonged to lower risk...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35835912"
                        open
                      >
                        <summary class="article-expander__title">Genome-wide analyses of 200,453 individuals yield new insights into the causes and consequences of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35835912/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220722211347&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH), the clonal expansion of a blood stem cell and its progeny driven by somatic driver mutations, affects over a third of people, yet remains poorly understood. Here we analyze genetic data from 200,453 UK Biobank participants to map the landscape of inherited predisposition to CH, increasing the number of germline associations with CH in European-ancestry populations from 4 to 14. Genes at new loci implicate DNA damage repair (PARP1, ATM, CHEK2), hematopoietic stem cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35832562"
                        open
                      >
                        <summary class="article-expander__title">A Novel Prognostic Scoring Model for Myelodysplastic Syndrome Patients With SF3B1 Mutation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35832562/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220722211347&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The outcomes of myelodysplastic syndrome (MDS) patients with SF3B1 mutation, despite identified as a favorable prognostic biomarker, are variable. To comprehend the heterogeneity in clinical characteristics and outcomes, we reviewed 140 MDS patients with SF3B1 mutation in Zhejiang province of China. Seventy-three (52.1%) patients diagnosed as MDS with ring sideroblasts (MDS-RS) following the 2016 World Health Organization (WHO) classification and 118 (84.3%) patients belonged to lower risk...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35835912"
                        open
                      >
                        <summary class="article-expander__title">Genome-wide analyses of 200,453 individuals yield new insights into the causes and consequences of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35835912/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220721211948&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH), the clonal expansion of a blood stem cell and its progeny driven by somatic driver mutations, affects over a third of people, yet remains poorly understood. Here we analyze genetic data from 200,453 UK Biobank participants to map the landscape of inherited predisposition to CH, increasing the number of germline associations with CH in European-ancestry populations from 4 to 14. Genes at new loci implicate DNA damage repair (PARP1, ATM, CHEK2), hematopoietic stem cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35832562"
                        open
                      >
                        <summary class="article-expander__title">A Novel Prognostic Scoring Model for Myelodysplastic Syndrome Patients With SF3B1 Mutation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35832562/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220721211948&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The outcomes of myelodysplastic syndrome (MDS) patients with SF3B1 mutation, despite identified as a favorable prognostic biomarker, are variable. To comprehend the heterogeneity in clinical characteristics and outcomes, we reviewed 140 MDS patients with SF3B1 mutation in Zhejiang province of China. Seventy-three (52.1%) patients diagnosed as MDS with ring sideroblasts (MDS-RS) following the 2016 World Health Organization (WHO) classification and 118 (84.3%) patients belonged to lower risk...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35835912"
                        open
                      >
                        <summary class="article-expander__title">Genome-wide analyses of 200,453 individuals yield new insights into the causes and consequences of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35835912/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220720211338&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH), the clonal expansion of a blood stem cell and its progeny driven by somatic driver mutations, affects over a third of people, yet remains poorly understood. Here we analyze genetic data from 200,453 UK Biobank participants to map the landscape of inherited predisposition to CH, increasing the number of germline associations with CH in European-ancestry populations from 4 to 14. Genes at new loci implicate DNA damage repair (PARP1, ATM, CHEK2), hematopoietic stem cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35832562"
                        open
                      >
                        <summary class="article-expander__title">A Novel Prognostic Scoring Model for Myelodysplastic Syndrome Patients With SF3B1 Mutation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35832562/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220720211338&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The outcomes of myelodysplastic syndrome (MDS) patients with SF3B1 mutation, despite identified as a favorable prognostic biomarker, are variable. To comprehend the heterogeneity in clinical characteristics and outcomes, we reviewed 140 MDS patients with SF3B1 mutation in Zhejiang province of China. Seventy-three (52.1%) patients diagnosed as MDS with ring sideroblasts (MDS-RS) following the 2016 World Health Organization (WHO) classification and 118 (84.3%) patients belonged to lower risk...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35835912"
                        open
                      >
                        <summary class="article-expander__title">Genome-wide analyses of 200,453 individuals yield new insights into the causes and consequences of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35835912/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220719211909&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH), the clonal expansion of a blood stem cell and its progeny driven by somatic driver mutations, affects over a third of people, yet remains poorly understood. Here we analyze genetic data from 200,453 UK Biobank participants to map the landscape of inherited predisposition to CH, increasing the number of germline associations with CH in European-ancestry populations from 4 to 14. Genes at new loci implicate DNA damage repair (PARP1, ATM, CHEK2), hematopoietic stem cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35832562"
                        open
                      >
                        <summary class="article-expander__title">A Novel Prognostic Scoring Model for Myelodysplastic Syndrome Patients With SF3B1 Mutation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35832562/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220719211909&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The outcomes of myelodysplastic syndrome (MDS) patients with SF3B1 mutation, despite identified as a favorable prognostic biomarker, are variable. To comprehend the heterogeneity in clinical characteristics and outcomes, we reviewed 140 MDS patients with SF3B1 mutation in Zhejiang province of China. Seventy-three (52.1%) patients diagnosed as MDS with ring sideroblasts (MDS-RS) following the 2016 World Health Organization (WHO) classification and 118 (84.3%) patients belonged to lower risk...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35835912"
                        open
                      >
                        <summary class="article-expander__title">Genome-wide analyses of 200,453 individuals yield new insights into the causes and consequences of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35835912/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220718212208&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH), the clonal expansion of a blood stem cell and its progeny driven by somatic driver mutations, affects over a third of people, yet remains poorly understood. Here we analyze genetic data from 200,453 UK Biobank participants to map the landscape of inherited predisposition to CH, increasing the number of germline associations with CH in European-ancestry populations from 4 to 14. Genes at new loci implicate DNA damage repair (PARP1, ATM, CHEK2), hematopoietic stem cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35832562"
                        open
                      >
                        <summary class="article-expander__title">A Novel Prognostic Scoring Model for Myelodysplastic Syndrome Patients With SF3B1 Mutation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35832562/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220718212208&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The outcomes of myelodysplastic syndrome (MDS) patients with SF3B1 mutation, despite identified as a favorable prognostic biomarker, are variable. To comprehend the heterogeneity in clinical characteristics and outcomes, we reviewed 140 MDS patients with SF3B1 mutation in Zhejiang province of China. Seventy-three (52.1%) patients diagnosed as MDS with ring sideroblasts (MDS-RS) following the 2016 World Health Organization (WHO) classification and 118 (84.3%) patients belonged to lower risk...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35835912"
                        open
                      >
                        <summary class="article-expander__title">Genome-wide analyses of 200,453 individuals yield new insights into the causes and consequences of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35835912/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220717211412&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH), the clonal expansion of a blood stem cell and its progeny driven by somatic driver mutations, affects over a third of people, yet remains poorly understood. Here we analyze genetic data from 200,453 UK Biobank participants to map the landscape of inherited predisposition to CH, increasing the number of germline associations with CH in European-ancestry populations from 4 to 14. Genes at new loci implicate DNA damage repair (PARP1, ATM, CHEK2), hematopoietic stem cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35832562"
                        open
                      >
                        <summary class="article-expander__title">A Novel Prognostic Scoring Model for Myelodysplastic Syndrome Patients With SF3B1 Mutation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35832562/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220717211412&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The outcomes of myelodysplastic syndrome (MDS) patients with SF3B1 mutation, despite identified as a favorable prognostic biomarker, are variable. To comprehend the heterogeneity in clinical characteristics and outcomes, we reviewed 140 MDS patients with SF3B1 mutation in Zhejiang province of China. Seventy-three (52.1%) patients diagnosed as MDS with ring sideroblasts (MDS-RS) following the 2016 World Health Organization (WHO) classification and 118 (84.3%) patients belonged to lower risk...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35835912"
                        open
                      >
                        <summary class="article-expander__title">Genome-wide analyses of 200,453 individuals yield new insights into the causes and consequences of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35835912/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220716212224&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH), the clonal expansion of a blood stem cell and its progeny driven by somatic driver mutations, affects over a third of people, yet remains poorly understood. Here we analyze genetic data from 200,453 UK Biobank participants to map the landscape of inherited predisposition to CH, increasing the number of germline associations with CH in European-ancestry populations from 4 to 14. Genes at new loci implicate DNA damage repair (PARP1, ATM, CHEK2), hematopoietic stem cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35832562"
                        open
                      >
                        <summary class="article-expander__title">A Novel Prognostic Scoring Model for Myelodysplastic Syndrome Patients With SF3B1 Mutation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35832562/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220716212224&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The outcomes of myelodysplastic syndrome (MDS) patients with SF3B1 mutation, despite identified as a favorable prognostic biomarker, are variable. To comprehend the heterogeneity in clinical characteristics and outcomes, we reviewed 140 MDS patients with SF3B1 mutation in Zhejiang province of China. Seventy-three (52.1%) patients diagnosed as MDS with ring sideroblasts (MDS-RS) following the 2016 World Health Organization (WHO) classification and 118 (84.3%) patients belonged to lower risk...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35835912"
                        open
                      >
                        <summary class="article-expander__title">Genome-wide analyses of 200,453 individuals yield new insights into the causes and consequences of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35835912/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220715211735&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH), the clonal expansion of a blood stem cell and its progeny driven by somatic driver mutations, affects over a third of people, yet remains poorly understood. Here we analyze genetic data from 200,453 UK Biobank participants to map the landscape of inherited predisposition to CH, increasing the number of germline associations with CH in European-ancestry populations from 4 to 14. Genes at new loci implicate DNA damage repair (PARP1, ATM, CHEK2), hematopoietic stem cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35832562"
                        open
                      >
                        <summary class="article-expander__title">A Novel Prognostic Scoring Model for Myelodysplastic Syndrome Patients With SF3B1 Mutation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35832562/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220715211735&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The outcomes of myelodysplastic syndrome (MDS) patients with SF3B1 mutation, despite identified as a favorable prognostic biomarker, are variable. To comprehend the heterogeneity in clinical characteristics and outcomes, we reviewed 140 MDS patients with SF3B1 mutation in Zhejiang province of China. Seventy-three (52.1%) patients diagnosed as MDS with ring sideroblasts (MDS-RS) following the 2016 World Health Organization (WHO) classification and 118 (84.3%) patients belonged to lower risk...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35832562"
                        open
                      >
                        <summary class="article-expander__title">A Novel Prognostic Scoring Model for Myelodysplastic Syndrome Patients With SF3B1 Mutation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35832562/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The outcomes of myelodysplastic syndrome (MDS) patients with SF3B1 mutation, despite identified as a favorable prognostic biomarker, are variable. To comprehend the heterogeneity in clinical characteristics and outcomes, we reviewed 140 MDS patients with SF3B1 mutation in Zhejiang province of China. Seventy-three (52.1%) patients diagnosed as MDS with ring sideroblasts (MDS-RS) following the 2016 World Health Organization (WHO) classification and 118 (84.3%) patients belonged to lower risk...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608010801&amp;v&#x3D;2.17.7&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35835912"
                        open
                      >
                        <summary class="article-expander__title">Genome-wide analyses of 200,453 individuals yield new insights into the causes and consequences of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35835912/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220810210900&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH), the clonal expansion of a blood stem cell and its progeny driven by somatic driver mutations, affects over a third of people, yet remains poorly understood. Here we analyze genetic data from 200,453 UK Biobank participants to map the landscape of inherited predisposition to CH, increasing the number of germline associations with CH in European-ancestry populations from 4 to 14. Genes at new loci implicate DNA damage repair (PARP1, ATM, CHEK2), hematopoietic stem cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35835912"
                        open
                      >
                        <summary class="article-expander__title">Genome-wide analyses of 200,453 individuals yield new insights into the causes and consequences of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35835912/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220809211355&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH), the clonal expansion of a blood stem cell and its progeny driven by somatic driver mutations, affects over a third of people, yet remains poorly understood. Here we analyze genetic data from 200,453 UK Biobank participants to map the landscape of inherited predisposition to CH, increasing the number of germline associations with CH in European-ancestry populations from 4 to 14. Genes at new loci implicate DNA damage repair (PARP1, ATM, CHEK2), hematopoietic stem cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;17
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35832612"
                        open
                      >
                        <summary class="article-expander__title">Gene utility recapitulates chromosomal aberrancies in advanced stage neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35832612/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220808212152&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Neuroblastoma (NB) is the most common extracranial solid tumor in children. Although only a few recurrent somatic mutations have been identified, chromosomal abnormalities, including the loss of heterozygosity (LOH) at the chromosome 1p and gains of chromosome 17q, are often seen in the high-risk cases. The biological basis and evolutionary forces that drive such genetic abnormalities remain enigmatic. Here, we conceptualize the Gene Utility Model (GUM) that seeks to identify genes driving...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35832612"
                        open
                      >
                        <summary class="article-expander__title">Gene utility recapitulates chromosomal aberrancies in advanced stage neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35832612/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220807211336&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Neuroblastoma (NB) is the most common extracranial solid tumor in children. Although only a few recurrent somatic mutations have been identified, chromosomal abnormalities, including the loss of heterozygosity (LOH) at the chromosome 1p and gains of chromosome 17q, are often seen in the high-risk cases. The biological basis and evolutionary forces that drive such genetic abnormalities remain enigmatic. Here, we conceptualize the Gene Utility Model (GUM) that seeks to identify genes driving...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35832612"
                        open
                      >
                        <summary class="article-expander__title">Gene utility recapitulates chromosomal aberrancies in advanced stage neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35832612/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220806211830&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Neuroblastoma (NB) is the most common extracranial solid tumor in children. Although only a few recurrent somatic mutations have been identified, chromosomal abnormalities, including the loss of heterozygosity (LOH) at the chromosome 1p and gains of chromosome 17q, are often seen in the high-risk cases. The biological basis and evolutionary forces that drive such genetic abnormalities remain enigmatic. Here, we conceptualize the Gene Utility Model (GUM) that seeks to identify genes driving...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35832612"
                        open
                      >
                        <summary class="article-expander__title">Gene utility recapitulates chromosomal aberrancies in advanced stage neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35832612/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220805210849&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Neuroblastoma (NB) is the most common extracranial solid tumor in children. Although only a few recurrent somatic mutations have been identified, chromosomal abnormalities, including the loss of heterozygosity (LOH) at the chromosome 1p and gains of chromosome 17q, are often seen in the high-risk cases. The biological basis and evolutionary forces that drive such genetic abnormalities remain enigmatic. Here, we conceptualize the Gene Utility Model (GUM) that seeks to identify genes driving...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35832612"
                        open
                      >
                        <summary class="article-expander__title">Gene utility recapitulates chromosomal aberrancies in advanced stage neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35832612/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220804212147&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Neuroblastoma (NB) is the most common extracranial solid tumor in children. Although only a few recurrent somatic mutations have been identified, chromosomal abnormalities, including the loss of heterozygosity (LOH) at the chromosome 1p and gains of chromosome 17q, are often seen in the high-risk cases. The biological basis and evolutionary forces that drive such genetic abnormalities remain enigmatic. Here, we conceptualize the Gene Utility Model (GUM) that seeks to identify genes driving...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35832612"
                        open
                      >
                        <summary class="article-expander__title">Gene utility recapitulates chromosomal aberrancies in advanced stage neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35832612/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220803212051&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Neuroblastoma (NB) is the most common extracranial solid tumor in children. Although only a few recurrent somatic mutations have been identified, chromosomal abnormalities, including the loss of heterozygosity (LOH) at the chromosome 1p and gains of chromosome 17q, are often seen in the high-risk cases. The biological basis and evolutionary forces that drive such genetic abnormalities remain enigmatic. Here, we conceptualize the Gene Utility Model (GUM) that seeks to identify genes driving...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35832612"
                        open
                      >
                        <summary class="article-expander__title">Gene utility recapitulates chromosomal aberrancies in advanced stage neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35832612/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220802212153&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Neuroblastoma (NB) is the most common extracranial solid tumor in children. Although only a few recurrent somatic mutations have been identified, chromosomal abnormalities, including the loss of heterozygosity (LOH) at the chromosome 1p and gains of chromosome 17q, are often seen in the high-risk cases. The biological basis and evolutionary forces that drive such genetic abnormalities remain enigmatic. Here, we conceptualize the Gene Utility Model (GUM) that seeks to identify genes driving...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35832612"
                        open
                      >
                        <summary class="article-expander__title">Gene utility recapitulates chromosomal aberrancies in advanced stage neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35832612/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220801212447&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Neuroblastoma (NB) is the most common extracranial solid tumor in children. Although only a few recurrent somatic mutations have been identified, chromosomal abnormalities, including the loss of heterozygosity (LOH) at the chromosome 1p and gains of chromosome 17q, are often seen in the high-risk cases. The biological basis and evolutionary forces that drive such genetic abnormalities remain enigmatic. Here, we conceptualize the Gene Utility Model (GUM) that seeks to identify genes driving...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35832612"
                        open
                      >
                        <summary class="article-expander__title">Gene utility recapitulates chromosomal aberrancies in advanced stage neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35832612/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220731212525&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Neuroblastoma (NB) is the most common extracranial solid tumor in children. Although only a few recurrent somatic mutations have been identified, chromosomal abnormalities, including the loss of heterozygosity (LOH) at the chromosome 1p and gains of chromosome 17q, are often seen in the high-risk cases. The biological basis and evolutionary forces that drive such genetic abnormalities remain enigmatic. Here, we conceptualize the Gene Utility Model (GUM) that seeks to identify genes driving...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35832612"
                        open
                      >
                        <summary class="article-expander__title">Gene utility recapitulates chromosomal aberrancies in advanced stage neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35832612/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220730212309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Neuroblastoma (NB) is the most common extracranial solid tumor in children. Although only a few recurrent somatic mutations have been identified, chromosomal abnormalities, including the loss of heterozygosity (LOH) at the chromosome 1p and gains of chromosome 17q, are often seen in the high-risk cases. The biological basis and evolutionary forces that drive such genetic abnormalities remain enigmatic. Here, we conceptualize the Gene Utility Model (GUM) that seeks to identify genes driving...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35832612"
                        open
                      >
                        <summary class="article-expander__title">Gene utility recapitulates chromosomal aberrancies in advanced stage neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35832612/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220729211332&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Neuroblastoma (NB) is the most common extracranial solid tumor in children. Although only a few recurrent somatic mutations have been identified, chromosomal abnormalities, including the loss of heterozygosity (LOH) at the chromosome 1p and gains of chromosome 17q, are often seen in the high-risk cases. The biological basis and evolutionary forces that drive such genetic abnormalities remain enigmatic. Here, we conceptualize the Gene Utility Model (GUM) that seeks to identify genes driving...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35832612"
                        open
                      >
                        <summary class="article-expander__title">Gene utility recapitulates chromosomal aberrancies in advanced stage neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35832612/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220728211733&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Neuroblastoma (NB) is the most common extracranial solid tumor in children. Although only a few recurrent somatic mutations have been identified, chromosomal abnormalities, including the loss of heterozygosity (LOH) at the chromosome 1p and gains of chromosome 17q, are often seen in the high-risk cases. The biological basis and evolutionary forces that drive such genetic abnormalities remain enigmatic. Here, we conceptualize the Gene Utility Model (GUM) that seeks to identify genes driving...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35832612"
                        open
                      >
                        <summary class="article-expander__title">Gene utility recapitulates chromosomal aberrancies in advanced stage neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35832612/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220727211458&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Neuroblastoma (NB) is the most common extracranial solid tumor in children. Although only a few recurrent somatic mutations have been identified, chromosomal abnormalities, including the loss of heterozygosity (LOH) at the chromosome 1p and gains of chromosome 17q, are often seen in the high-risk cases. The biological basis and evolutionary forces that drive such genetic abnormalities remain enigmatic. Here, we conceptualize the Gene Utility Model (GUM) that seeks to identify genes driving...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35832612"
                        open
                      >
                        <summary class="article-expander__title">Gene utility recapitulates chromosomal aberrancies in advanced stage neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35832612/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220726212040&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Neuroblastoma (NB) is the most common extracranial solid tumor in children. Although only a few recurrent somatic mutations have been identified, chromosomal abnormalities, including the loss of heterozygosity (LOH) at the chromosome 1p and gains of chromosome 17q, are often seen in the high-risk cases. The biological basis and evolutionary forces that drive such genetic abnormalities remain enigmatic. Here, we conceptualize the Gene Utility Model (GUM) that seeks to identify genes driving...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35832612"
                        open
                      >
                        <summary class="article-expander__title">Gene utility recapitulates chromosomal aberrancies in advanced stage neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35832612/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220725211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Neuroblastoma (NB) is the most common extracranial solid tumor in children. Although only a few recurrent somatic mutations have been identified, chromosomal abnormalities, including the loss of heterozygosity (LOH) at the chromosome 1p and gains of chromosome 17q, are often seen in the high-risk cases. The biological basis and evolutionary forces that drive such genetic abnormalities remain enigmatic. Here, we conceptualize the Gene Utility Model (GUM) that seeks to identify genes driving...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35832612"
                        open
                      >
                        <summary class="article-expander__title">Gene utility recapitulates chromosomal aberrancies in advanced stage neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35832612/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220724211605&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Neuroblastoma (NB) is the most common extracranial solid tumor in children. Although only a few recurrent somatic mutations have been identified, chromosomal abnormalities, including the loss of heterozygosity (LOH) at the chromosome 1p and gains of chromosome 17q, are often seen in the high-risk cases. The biological basis and evolutionary forces that drive such genetic abnormalities remain enigmatic. Here, we conceptualize the Gene Utility Model (GUM) that seeks to identify genes driving...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35832612"
                        open
                      >
                        <summary class="article-expander__title">Gene utility recapitulates chromosomal aberrancies in advanced stage neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35832612/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220723212121&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Neuroblastoma (NB) is the most common extracranial solid tumor in children. Although only a few recurrent somatic mutations have been identified, chromosomal abnormalities, including the loss of heterozygosity (LOH) at the chromosome 1p and gains of chromosome 17q, are often seen in the high-risk cases. The biological basis and evolutionary forces that drive such genetic abnormalities remain enigmatic. Here, we conceptualize the Gene Utility Model (GUM) that seeks to identify genes driving...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35832612"
                        open
                      >
                        <summary class="article-expander__title">Gene utility recapitulates chromosomal aberrancies in advanced stage neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35832612/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220722211347&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Neuroblastoma (NB) is the most common extracranial solid tumor in children. Although only a few recurrent somatic mutations have been identified, chromosomal abnormalities, including the loss of heterozygosity (LOH) at the chromosome 1p and gains of chromosome 17q, are often seen in the high-risk cases. The biological basis and evolutionary forces that drive such genetic abnormalities remain enigmatic. Here, we conceptualize the Gene Utility Model (GUM) that seeks to identify genes driving...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35832612"
                        open
                      >
                        <summary class="article-expander__title">Gene utility recapitulates chromosomal aberrancies in advanced stage neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35832612/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220721211948&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Neuroblastoma (NB) is the most common extracranial solid tumor in children. Although only a few recurrent somatic mutations have been identified, chromosomal abnormalities, including the loss of heterozygosity (LOH) at the chromosome 1p and gains of chromosome 17q, are often seen in the high-risk cases. The biological basis and evolutionary forces that drive such genetic abnormalities remain enigmatic. Here, we conceptualize the Gene Utility Model (GUM) that seeks to identify genes driving...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35832612"
                        open
                      >
                        <summary class="article-expander__title">Gene utility recapitulates chromosomal aberrancies in advanced stage neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35832612/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220720211338&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Neuroblastoma (NB) is the most common extracranial solid tumor in children. Although only a few recurrent somatic mutations have been identified, chromosomal abnormalities, including the loss of heterozygosity (LOH) at the chromosome 1p and gains of chromosome 17q, are often seen in the high-risk cases. The biological basis and evolutionary forces that drive such genetic abnormalities remain enigmatic. Here, we conceptualize the Gene Utility Model (GUM) that seeks to identify genes driving...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35832612"
                        open
                      >
                        <summary class="article-expander__title">Gene utility recapitulates chromosomal aberrancies in advanced stage neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35832612/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220719211909&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Neuroblastoma (NB) is the most common extracranial solid tumor in children. Although only a few recurrent somatic mutations have been identified, chromosomal abnormalities, including the loss of heterozygosity (LOH) at the chromosome 1p and gains of chromosome 17q, are often seen in the high-risk cases. The biological basis and evolutionary forces that drive such genetic abnormalities remain enigmatic. Here, we conceptualize the Gene Utility Model (GUM) that seeks to identify genes driving...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35832612"
                        open
                      >
                        <summary class="article-expander__title">Gene utility recapitulates chromosomal aberrancies in advanced stage neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35832612/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220718212208&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Neuroblastoma (NB) is the most common extracranial solid tumor in children. Although only a few recurrent somatic mutations have been identified, chromosomal abnormalities, including the loss of heterozygosity (LOH) at the chromosome 1p and gains of chromosome 17q, are often seen in the high-risk cases. The biological basis and evolutionary forces that drive such genetic abnormalities remain enigmatic. Here, we conceptualize the Gene Utility Model (GUM) that seeks to identify genes driving...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35832612"
                        open
                      >
                        <summary class="article-expander__title">Gene utility recapitulates chromosomal aberrancies in advanced stage neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35832612/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220717211412&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Neuroblastoma (NB) is the most common extracranial solid tumor in children. Although only a few recurrent somatic mutations have been identified, chromosomal abnormalities, including the loss of heterozygosity (LOH) at the chromosome 1p and gains of chromosome 17q, are often seen in the high-risk cases. The biological basis and evolutionary forces that drive such genetic abnormalities remain enigmatic. Here, we conceptualize the Gene Utility Model (GUM) that seeks to identify genes driving...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35832612"
                        open
                      >
                        <summary class="article-expander__title">Gene utility recapitulates chromosomal aberrancies in advanced stage neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35832612/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220716212224&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Neuroblastoma (NB) is the most common extracranial solid tumor in children. Although only a few recurrent somatic mutations have been identified, chromosomal abnormalities, including the loss of heterozygosity (LOH) at the chromosome 1p and gains of chromosome 17q, are often seen in the high-risk cases. The biological basis and evolutionary forces that drive such genetic abnormalities remain enigmatic. Here, we conceptualize the Gene Utility Model (GUM) that seeks to identify genes driving...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35832612"
                        open
                      >
                        <summary class="article-expander__title">Gene utility recapitulates chromosomal aberrancies in advanced stage neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35832612/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220715211735&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Neuroblastoma (NB) is the most common extracranial solid tumor in children. Although only a few recurrent somatic mutations have been identified, chromosomal abnormalities, including the loss of heterozygosity (LOH) at the chromosome 1p and gains of chromosome 17q, are often seen in the high-risk cases. The biological basis and evolutionary forces that drive such genetic abnormalities remain enigmatic. Here, we conceptualize the Gene Utility Model (GUM) that seeks to identify genes driving...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35832612"
                        open
                      >
                        <summary class="article-expander__title">Gene utility recapitulates chromosomal aberrancies in advanced stage neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35832612/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Neuroblastoma (NB) is the most common extracranial solid tumor in children. Although only a few recurrent somatic mutations have been identified, chromosomal abnormalities, including the loss of heterozygosity (LOH) at the chromosome 1p and gains of chromosome 17q, are often seen in the high-risk cases. The biological basis and evolutionary forces that drive such genetic abnormalities remain enigmatic. Here, we conceptualize the Gene Utility Model (GUM) that seeks to identify genes driving...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-07-13 datetime="2022-07-13T10:00:00.000Z">2022-07-13</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-07-13 datetime="2022-07-13T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608003622&amp;v&#x3D;2.17.7&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35831189"
                        open
                      >
                        <summary class="article-expander__title">Pathophysiology of hematological malignancies associated with ASXL1 mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35831189/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220726212041&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations in the epigenetic regulator ASXL1 are considered a poor prognostic factor in myeloid malignancies, including myelodysplastic syndrome (MDS) and acute myeloid leukemia (AML). ASXL1 mutations coexist with other mutations in majority of patients, suggesting that its mutation alone is insufficient to cause cancer. ASXL1 mutations have been detected in age-related clonal hematopoiesis (CH), which has been linked to an increased risk of hematological malignancies. Therefore, ASXL1...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35831189"
                        open
                      >
                        <summary class="article-expander__title">Pathophysiology of hematological malignancies associated with ASXL1 mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35831189/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220725211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations in the epigenetic regulator ASXL1 are considered a poor prognostic factor in myeloid malignancies, including myelodysplastic syndrome (MDS) and acute myeloid leukemia (AML). ASXL1 mutations coexist with other mutations in majority of patients, suggesting that its mutation alone is insufficient to cause cancer. ASXL1 mutations have been detected in age-related clonal hematopoiesis (CH), which has been linked to an increased risk of hematological malignancies. Therefore, ASXL1...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35822030"
                        open
                      >
                        <summary class="article-expander__title">Stem Cells in the Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35822030/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220725211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The myelodysplastic syndromes (MDS) represent a group of clonal disorders characterized by ineffective hematopoiesis, resulting in peripheral cytopenias and frequent transformation to acute myeloid leukemia (AML). We and others have demonstrated that MDS arises in, and is propagated by malignant stem cells (MDS-SCs), that arise due to the sequential acquisition of genetic and epigenetic alterations in normal hematopoietic stem cells (HSCs). This review focuses on recent advancements in the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35831189"
                        open
                      >
                        <summary class="article-expander__title">Pathophysiology of hematological malignancies associated with ASXL1 mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35831189/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220724211604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations in the epigenetic regulator ASXL1 are considered a poor prognostic factor in myeloid malignancies, including myelodysplastic syndrome (MDS) and acute myeloid leukemia (AML). ASXL1 mutations coexist with other mutations in majority of patients, suggesting that its mutation alone is insufficient to cause cancer. ASXL1 mutations have been detected in age-related clonal hematopoiesis (CH), which has been linked to an increased risk of hematological malignancies. Therefore, ASXL1...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35822030"
                        open
                      >
                        <summary class="article-expander__title">Stem Cells in the Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35822030/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220724211604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The myelodysplastic syndromes (MDS) represent a group of clonal disorders characterized by ineffective hematopoiesis, resulting in peripheral cytopenias and frequent transformation to acute myeloid leukemia (AML). We and others have demonstrated that MDS arises in, and is propagated by malignant stem cells (MDS-SCs), that arise due to the sequential acquisition of genetic and epigenetic alterations in normal hematopoietic stem cells (HSCs). This review focuses on recent advancements in the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35821803"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis Analyses in Clinical, Epidemiologic, and Genetic Aging Studies to Unravel Underlying Mechanisms of Age-Related Dysfunction in Humans</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35821803/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220724211604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aging is characterized by increased mortality, functional decline, and exponential increases in the incidence of diseases such as cancer, stroke, cardiovascular disease, neurological disease, respiratory disease, etc. Though the role of aging in these diseases is widely accepted and considered to be a common denominator, the underlying mechanisms are largely unknown. A significant age-related feature observed in many population cohorts is somatic mosaicism, the detectable accumulation of somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35831189"
                        open
                      >
                        <summary class="article-expander__title">Pathophysiology of hematological malignancies associated with ASXL1 mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35831189/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220723212121&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations in the epigenetic regulator ASXL1 are considered a poor prognostic factor in myeloid malignancies, including myelodysplastic syndrome (MDS) and acute myeloid leukemia (AML). ASXL1 mutations coexist with other mutations in majority of patients, suggesting that its mutation alone is insufficient to cause cancer. ASXL1 mutations have been detected in age-related clonal hematopoiesis (CH), which has been linked to an increased risk of hematological malignancies. Therefore, ASXL1...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35822030"
                        open
                      >
                        <summary class="article-expander__title">Stem Cells in the Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35822030/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220723212121&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The myelodysplastic syndromes (MDS) represent a group of clonal disorders characterized by ineffective hematopoiesis, resulting in peripheral cytopenias and frequent transformation to acute myeloid leukemia (AML). We and others have demonstrated that MDS arises in, and is propagated by malignant stem cells (MDS-SCs), that arise due to the sequential acquisition of genetic and epigenetic alterations in normal hematopoietic stem cells (HSCs). This review focuses on recent advancements in the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35821803"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis Analyses in Clinical, Epidemiologic, and Genetic Aging Studies to Unravel Underlying Mechanisms of Age-Related Dysfunction in Humans</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35821803/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220723212121&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aging is characterized by increased mortality, functional decline, and exponential increases in the incidence of diseases such as cancer, stroke, cardiovascular disease, neurological disease, respiratory disease, etc. Though the role of aging in these diseases is widely accepted and considered to be a common denominator, the underlying mechanisms are largely unknown. A significant age-related feature observed in many population cohorts is somatic mosaicism, the detectable accumulation of somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35831189"
                        open
                      >
                        <summary class="article-expander__title">Pathophysiology of hematological malignancies associated with ASXL1 mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35831189/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220722211347&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations in the epigenetic regulator ASXL1 are considered a poor prognostic factor in myeloid malignancies, including myelodysplastic syndrome (MDS) and acute myeloid leukemia (AML). ASXL1 mutations coexist with other mutations in majority of patients, suggesting that its mutation alone is insufficient to cause cancer. ASXL1 mutations have been detected in age-related clonal hematopoiesis (CH), which has been linked to an increased risk of hematological malignancies. Therefore, ASXL1...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35822030"
                        open
                      >
                        <summary class="article-expander__title">Stem Cells in the Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35822030/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220722211347&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The myelodysplastic syndromes (MDS) represent a group of clonal disorders characterized by ineffective hematopoiesis, resulting in peripheral cytopenias and frequent transformation to acute myeloid leukemia (AML). We and others have demonstrated that MDS arises in, and is propagated by malignant stem cells (MDS-SCs), that arise due to the sequential acquisition of genetic and epigenetic alterations in normal hematopoietic stem cells (HSCs). This review focuses on recent advancements in the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35821803"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis Analyses in Clinical, Epidemiologic, and Genetic Aging Studies to Unravel Underlying Mechanisms of Age-Related Dysfunction in Humans</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35821803/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220722211347&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aging is characterized by increased mortality, functional decline, and exponential increases in the incidence of diseases such as cancer, stroke, cardiovascular disease, neurological disease, respiratory disease, etc. Though the role of aging in these diseases is widely accepted and considered to be a common denominator, the underlying mechanisms are largely unknown. A significant age-related feature observed in many population cohorts is somatic mosaicism, the detectable accumulation of somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35831189"
                        open
                      >
                        <summary class="article-expander__title">Pathophysiology of hematological malignancies associated with ASXL1 mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35831189/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220721211948&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations in the epigenetic regulator ASXL1 are considered a poor prognostic factor in myeloid malignancies, including myelodysplastic syndrome (MDS) and acute myeloid leukemia (AML). ASXL1 mutations coexist with other mutations in majority of patients, suggesting that its mutation alone is insufficient to cause cancer. ASXL1 mutations have been detected in age-related clonal hematopoiesis (CH), which has been linked to an increased risk of hematological malignancies. Therefore, ASXL1...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35822030"
                        open
                      >
                        <summary class="article-expander__title">Stem Cells in the Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35822030/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220721211948&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The myelodysplastic syndromes (MDS) represent a group of clonal disorders characterized by ineffective hematopoiesis, resulting in peripheral cytopenias and frequent transformation to acute myeloid leukemia (AML). We and others have demonstrated that MDS arises in, and is propagated by malignant stem cells (MDS-SCs), that arise due to the sequential acquisition of genetic and epigenetic alterations in normal hematopoietic stem cells (HSCs). This review focuses on recent advancements in the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35821803"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis Analyses in Clinical, Epidemiologic, and Genetic Aging Studies to Unravel Underlying Mechanisms of Age-Related Dysfunction in Humans</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35821803/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220721211948&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aging is characterized by increased mortality, functional decline, and exponential increases in the incidence of diseases such as cancer, stroke, cardiovascular disease, neurological disease, respiratory disease, etc. Though the role of aging in these diseases is widely accepted and considered to be a common denominator, the underlying mechanisms are largely unknown. A significant age-related feature observed in many population cohorts is somatic mosaicism, the detectable accumulation of somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35831189"
                        open
                      >
                        <summary class="article-expander__title">Pathophysiology of hematological malignancies associated with ASXL1 mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35831189/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220720211338&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations in the epigenetic regulator ASXL1 are considered a poor prognostic factor in myeloid malignancies, including myelodysplastic syndrome (MDS) and acute myeloid leukemia (AML). ASXL1 mutations coexist with other mutations in majority of patients, suggesting that its mutation alone is insufficient to cause cancer. ASXL1 mutations have been detected in age-related clonal hematopoiesis (CH), which has been linked to an increased risk of hematological malignancies. Therefore, ASXL1...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35822030"
                        open
                      >
                        <summary class="article-expander__title">Stem Cells in the Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35822030/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220720211338&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The myelodysplastic syndromes (MDS) represent a group of clonal disorders characterized by ineffective hematopoiesis, resulting in peripheral cytopenias and frequent transformation to acute myeloid leukemia (AML). We and others have demonstrated that MDS arises in, and is propagated by malignant stem cells (MDS-SCs), that arise due to the sequential acquisition of genetic and epigenetic alterations in normal hematopoietic stem cells (HSCs). This review focuses on recent advancements in the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35821803"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis Analyses in Clinical, Epidemiologic, and Genetic Aging Studies to Unravel Underlying Mechanisms of Age-Related Dysfunction in Humans</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35821803/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220720211338&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aging is characterized by increased mortality, functional decline, and exponential increases in the incidence of diseases such as cancer, stroke, cardiovascular disease, neurological disease, respiratory disease, etc. Though the role of aging in these diseases is widely accepted and considered to be a common denominator, the underlying mechanisms are largely unknown. A significant age-related feature observed in many population cohorts is somatic mosaicism, the detectable accumulation of somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35831189"
                        open
                      >
                        <summary class="article-expander__title">Pathophysiology of hematological malignancies associated with ASXL1 mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35831189/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220719211909&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations in the epigenetic regulator ASXL1 are considered a poor prognostic factor in myeloid malignancies, including myelodysplastic syndrome (MDS) and acute myeloid leukemia (AML). ASXL1 mutations coexist with other mutations in majority of patients, suggesting that its mutation alone is insufficient to cause cancer. ASXL1 mutations have been detected in age-related clonal hematopoiesis (CH), which has been linked to an increased risk of hematological malignancies. Therefore, ASXL1...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35822030"
                        open
                      >
                        <summary class="article-expander__title">Stem Cells in the Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35822030/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220719211909&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The myelodysplastic syndromes (MDS) represent a group of clonal disorders characterized by ineffective hematopoiesis, resulting in peripheral cytopenias and frequent transformation to acute myeloid leukemia (AML). We and others have demonstrated that MDS arises in, and is propagated by malignant stem cells (MDS-SCs), that arise due to the sequential acquisition of genetic and epigenetic alterations in normal hematopoietic stem cells (HSCs). This review focuses on recent advancements in the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35821803"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis Analyses in Clinical, Epidemiologic, and Genetic Aging Studies to Unravel Underlying Mechanisms of Age-Related Dysfunction in Humans</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35821803/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220719211909&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aging is characterized by increased mortality, functional decline, and exponential increases in the incidence of diseases such as cancer, stroke, cardiovascular disease, neurological disease, respiratory disease, etc. Though the role of aging in these diseases is widely accepted and considered to be a common denominator, the underlying mechanisms are largely unknown. A significant age-related feature observed in many population cohorts is somatic mosaicism, the detectable accumulation of somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35831189"
                        open
                      >
                        <summary class="article-expander__title">Pathophysiology of hematological malignancies associated with ASXL1 mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35831189/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220718212208&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations in the epigenetic regulator ASXL1 are considered a poor prognostic factor in myeloid malignancies, including myelodysplastic syndrome (MDS) and acute myeloid leukemia (AML). ASXL1 mutations coexist with other mutations in majority of patients, suggesting that its mutation alone is insufficient to cause cancer. ASXL1 mutations have been detected in age-related clonal hematopoiesis (CH), which has been linked to an increased risk of hematological malignancies. Therefore, ASXL1...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35822030"
                        open
                      >
                        <summary class="article-expander__title">Stem Cells in the Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35822030/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220718212208&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The myelodysplastic syndromes (MDS) represent a group of clonal disorders characterized by ineffective hematopoiesis, resulting in peripheral cytopenias and frequent transformation to acute myeloid leukemia (AML). We and others have demonstrated that MDS arises in, and is propagated by malignant stem cells (MDS-SCs), that arise due to the sequential acquisition of genetic and epigenetic alterations in normal hematopoietic stem cells (HSCs). This review focuses on recent advancements in the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35821803"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis Analyses in Clinical, Epidemiologic, and Genetic Aging Studies to Unravel Underlying Mechanisms of Age-Related Dysfunction in Humans</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35821803/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220718212208&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aging is characterized by increased mortality, functional decline, and exponential increases in the incidence of diseases such as cancer, stroke, cardiovascular disease, neurological disease, respiratory disease, etc. Though the role of aging in these diseases is widely accepted and considered to be a common denominator, the underlying mechanisms are largely unknown. A significant age-related feature observed in many population cohorts is somatic mosaicism, the detectable accumulation of somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35831189"
                        open
                      >
                        <summary class="article-expander__title">Pathophysiology of hematological malignancies associated with ASXL1 mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35831189/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220717211412&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations in the epigenetic regulator ASXL1 are considered a poor prognostic factor in myeloid malignancies, including myelodysplastic syndrome (MDS) and acute myeloid leukemia (AML). ASXL1 mutations coexist with other mutations in majority of patients, suggesting that its mutation alone is insufficient to cause cancer. ASXL1 mutations have been detected in age-related clonal hematopoiesis (CH), which has been linked to an increased risk of hematological malignancies. Therefore, ASXL1...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35822030"
                        open
                      >
                        <summary class="article-expander__title">Stem Cells in the Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35822030/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220717211412&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The myelodysplastic syndromes (MDS) represent a group of clonal disorders characterized by ineffective hematopoiesis, resulting in peripheral cytopenias and frequent transformation to acute myeloid leukemia (AML). We and others have demonstrated that MDS arises in, and is propagated by malignant stem cells (MDS-SCs), that arise due to the sequential acquisition of genetic and epigenetic alterations in normal hematopoietic stem cells (HSCs). This review focuses on recent advancements in the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35821803"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis Analyses in Clinical, Epidemiologic, and Genetic Aging Studies to Unravel Underlying Mechanisms of Age-Related Dysfunction in Humans</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35821803/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220717211412&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aging is characterized by increased mortality, functional decline, and exponential increases in the incidence of diseases such as cancer, stroke, cardiovascular disease, neurological disease, respiratory disease, etc. Though the role of aging in these diseases is widely accepted and considered to be a common denominator, the underlying mechanisms are largely unknown. A significant age-related feature observed in many population cohorts is somatic mosaicism, the detectable accumulation of somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35831189"
                        open
                      >
                        <summary class="article-expander__title">Pathophysiology of hematological malignancies associated with ASXL1 mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35831189/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220716212224&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations in the epigenetic regulator ASXL1 are considered a poor prognostic factor in myeloid malignancies, including myelodysplastic syndrome (MDS) and acute myeloid leukemia (AML). ASXL1 mutations coexist with other mutations in majority of patients, suggesting that its mutation alone is insufficient to cause cancer. ASXL1 mutations have been detected in age-related clonal hematopoiesis (CH), which has been linked to an increased risk of hematological malignancies. Therefore, ASXL1...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35822030"
                        open
                      >
                        <summary class="article-expander__title">Stem Cells in the Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35822030/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220716212224&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The myelodysplastic syndromes (MDS) represent a group of clonal disorders characterized by ineffective hematopoiesis, resulting in peripheral cytopenias and frequent transformation to acute myeloid leukemia (AML). We and others have demonstrated that MDS arises in, and is propagated by malignant stem cells (MDS-SCs), that arise due to the sequential acquisition of genetic and epigenetic alterations in normal hematopoietic stem cells (HSCs). This review focuses on recent advancements in the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35821803"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis Analyses in Clinical, Epidemiologic, and Genetic Aging Studies to Unravel Underlying Mechanisms of Age-Related Dysfunction in Humans</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35821803/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220716212224&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aging is characterized by increased mortality, functional decline, and exponential increases in the incidence of diseases such as cancer, stroke, cardiovascular disease, neurological disease, respiratory disease, etc. Though the role of aging in these diseases is widely accepted and considered to be a common denominator, the underlying mechanisms are largely unknown. A significant age-related feature observed in many population cohorts is somatic mosaicism, the detectable accumulation of somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35831189"
                        open
                      >
                        <summary class="article-expander__title">Pathophysiology of hematological malignancies associated with ASXL1 mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35831189/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220715211735&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations in the epigenetic regulator ASXL1 are considered a poor prognostic factor in myeloid malignancies, including myelodysplastic syndrome (MDS) and acute myeloid leukemia (AML). ASXL1 mutations coexist with other mutations in majority of patients, suggesting that its mutation alone is insufficient to cause cancer. ASXL1 mutations have been detected in age-related clonal hematopoiesis (CH), which has been linked to an increased risk of hematological malignancies. Therefore, ASXL1...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35822030"
                        open
                      >
                        <summary class="article-expander__title">Stem Cells in the Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35822030/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220715211735&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The myelodysplastic syndromes (MDS) represent a group of clonal disorders characterized by ineffective hematopoiesis, resulting in peripheral cytopenias and frequent transformation to acute myeloid leukemia (AML). We and others have demonstrated that MDS arises in, and is propagated by malignant stem cells (MDS-SCs), that arise due to the sequential acquisition of genetic and epigenetic alterations in normal hematopoietic stem cells (HSCs). This review focuses on recent advancements in the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35821803"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis Analyses in Clinical, Epidemiologic, and Genetic Aging Studies to Unravel Underlying Mechanisms of Age-Related Dysfunction in Humans</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35821803/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220715211735&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aging is characterized by increased mortality, functional decline, and exponential increases in the incidence of diseases such as cancer, stroke, cardiovascular disease, neurological disease, respiratory disease, etc. Though the role of aging in these diseases is widely accepted and considered to be a common denominator, the underlying mechanisms are largely unknown. A significant age-related feature observed in many population cohorts is somatic mosaicism, the detectable accumulation of somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35831189"
                        open
                      >
                        <summary class="article-expander__title">Pathophysiology of hematological malignancies associated with ASXL1 mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35831189/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations in the epigenetic regulator ASXL1 are considered a poor prognostic factor in myeloid malignancies, including myelodysplastic syndrome (MDS) and acute myeloid leukemia (AML). ASXL1 mutations coexist with other mutations in majority of patients, suggesting that its mutation alone is insufficient to cause cancer. ASXL1 mutations have been detected in age-related clonal hematopoiesis (CH), which has been linked to an increased risk of hematological malignancies. Therefore, ASXL1...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35822030"
                        open
                      >
                        <summary class="article-expander__title">Stem Cells in the Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35822030/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The myelodysplastic syndromes (MDS) represent a group of clonal disorders characterized by ineffective hematopoiesis, resulting in peripheral cytopenias and frequent transformation to acute myeloid leukemia (AML). We and others have demonstrated that MDS arises in, and is propagated by malignant stem cells (MDS-SCs), that arise due to the sequential acquisition of genetic and epigenetic alterations in normal hematopoietic stem cells (HSCs). This review focuses on recent advancements in the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35821803"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis Analyses in Clinical, Epidemiologic, and Genetic Aging Studies to Unravel Underlying Mechanisms of Age-Related Dysfunction in Humans</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35821803/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aging is characterized by increased mortality, functional decline, and exponential increases in the incidence of diseases such as cancer, stroke, cardiovascular disease, neurological disease, respiratory disease, etc. Though the role of aging in these diseases is widely accepted and considered to be a common denominator, the underlying mechanisms are largely unknown. A significant age-related feature observed in many population cohorts is somatic mosaicism, the detectable accumulation of somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35822030"
                        open
                      >
                        <summary class="article-expander__title">Stem Cells in the Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35822030/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The myelodysplastic syndromes (MDS) represent a group of clonal disorders characterized by ineffective hematopoiesis, resulting in peripheral cytopenias and frequent transformation to acute myeloid leukemia (AML). We and others have demonstrated that MDS arises in, and is propagated by malignant stem cells (MDS-SCs), that arise due to the sequential acquisition of genetic and epigenetic alterations in normal hematopoietic stem cells (HSCs). This review focuses on recent advancements in the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35821803"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis Analyses in Clinical, Epidemiologic, and Genetic Aging Studies to Unravel Underlying Mechanisms of Age-Related Dysfunction in Humans</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35821803/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aging is characterized by increased mortality, functional decline, and exponential increases in the incidence of diseases such as cancer, stroke, cardiovascular disease, neurological disease, respiratory disease, etc. Though the role of aging in these diseases is widely accepted and considered to be a common denominator, the underlying mechanisms are largely unknown. A significant age-related feature observed in many population cohorts is somatic mosaicism, the detectable accumulation of somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608010801&amp;v&#x3D;2.17.7&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35831448"
                        open
                      >
                        <summary class="article-expander__title">Single-cell DNA sequencing identifies risk-associated clonal complexity and evolutionary trajectories in childhood medulloblastoma development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35831448/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220808212152&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We reconstructed the natural history and temporal evolution of the most common childhood brain malignancy, medulloblastoma, by single-cell whole-genome sequencing (sc-WGS) of tumours representing its major molecular sub-classes and clinical risk groups. Favourable-risk disease sub-types assessed (MB(WNT) and infant desmoplastic/nodular MB(SHH)) typically comprised a single clone with no evidence of further evolution. In contrast, highest risk sub-classes (MYC-amplified MB(Group3) and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35831448"
                        open
                      >
                        <summary class="article-expander__title">Single-cell DNA sequencing identifies risk-associated clonal complexity and evolutionary trajectories in childhood medulloblastoma development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35831448/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220807211336&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We reconstructed the natural history and temporal evolution of the most common childhood brain malignancy, medulloblastoma, by single-cell whole-genome sequencing (sc-WGS) of tumours representing its major molecular sub-classes and clinical risk groups. Favourable-risk disease sub-types assessed (MB(WNT) and infant desmoplastic/nodular MB(SHH)) typically comprised a single clone with no evidence of further evolution. In contrast, highest risk sub-classes (MYC-amplified MB(Group3) and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35831448"
                        open
                      >
                        <summary class="article-expander__title">Single-cell DNA sequencing identifies risk-associated clonal complexity and evolutionary trajectories in childhood medulloblastoma development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35831448/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220806211830&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We reconstructed the natural history and temporal evolution of the most common childhood brain malignancy, medulloblastoma, by single-cell whole-genome sequencing (sc-WGS) of tumours representing its major molecular sub-classes and clinical risk groups. Favourable-risk disease sub-types assessed (MB(WNT) and infant desmoplastic/nodular MB(SHH)) typically comprised a single clone with no evidence of further evolution. In contrast, highest risk sub-classes (MYC-amplified MB(Group3) and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35831448"
                        open
                      >
                        <summary class="article-expander__title">Single-cell DNA sequencing identifies risk-associated clonal complexity and evolutionary trajectories in childhood medulloblastoma development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35831448/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220805210849&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We reconstructed the natural history and temporal evolution of the most common childhood brain malignancy, medulloblastoma, by single-cell whole-genome sequencing (sc-WGS) of tumours representing its major molecular sub-classes and clinical risk groups. Favourable-risk disease sub-types assessed (MB(WNT) and infant desmoplastic/nodular MB(SHH)) typically comprised a single clone with no evidence of further evolution. In contrast, highest risk sub-classes (MYC-amplified MB(Group3) and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35822488"
                        open
                      >
                        <summary class="article-expander__title">Targeting stem cells in myelodysplastic syndromes and acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35822488/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220805210849&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The genetic architecture of cancer has been delineated through advances in high-throughput next-generation sequencing, where the sequential acquisition of recurrent driver mutations initially targeted towards normal cells ultimately leads to malignant transformation. Myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML) are hematologic malignancies frequently initiated by mutations in the normal hematopoietic stem cell compartment leading to the establishment of leukemic stem cells....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35831448"
                        open
                      >
                        <summary class="article-expander__title">Single-cell DNA sequencing identifies risk-associated clonal complexity and evolutionary trajectories in childhood medulloblastoma development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35831448/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220804212147&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We reconstructed the natural history and temporal evolution of the most common childhood brain malignancy, medulloblastoma, by single-cell whole-genome sequencing (sc-WGS) of tumours representing its major molecular sub-classes and clinical risk groups. Favourable-risk disease sub-types assessed (MB(WNT) and infant desmoplastic/nodular MB(SHH)) typically comprised a single clone with no evidence of further evolution. In contrast, highest risk sub-classes (MYC-amplified MB(Group3) and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35822488"
                        open
                      >
                        <summary class="article-expander__title">Targeting stem cells in myelodysplastic syndromes and acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35822488/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220804212147&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The genetic architecture of cancer has been delineated through advances in high-throughput next-generation sequencing, where the sequential acquisition of recurrent driver mutations initially targeted towards normal cells ultimately leads to malignant transformation. Myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML) are hematologic malignancies frequently initiated by mutations in the normal hematopoietic stem cell compartment leading to the establishment of leukemic stem cells....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35831448"
                        open
                      >
                        <summary class="article-expander__title">Single-cell DNA sequencing identifies risk-associated clonal complexity and evolutionary trajectories in childhood medulloblastoma development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35831448/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220803212051&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We reconstructed the natural history and temporal evolution of the most common childhood brain malignancy, medulloblastoma, by single-cell whole-genome sequencing (sc-WGS) of tumours representing its major molecular sub-classes and clinical risk groups. Favourable-risk disease sub-types assessed (MB(WNT) and infant desmoplastic/nodular MB(SHH)) typically comprised a single clone with no evidence of further evolution. In contrast, highest risk sub-classes (MYC-amplified MB(Group3) and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35822488"
                        open
                      >
                        <summary class="article-expander__title">Targeting stem cells in myelodysplastic syndromes and acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35822488/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220803212051&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The genetic architecture of cancer has been delineated through advances in high-throughput next-generation sequencing, where the sequential acquisition of recurrent driver mutations initially targeted towards normal cells ultimately leads to malignant transformation. Myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML) are hematologic malignancies frequently initiated by mutations in the normal hematopoietic stem cell compartment leading to the establishment of leukemic stem cells....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35831448"
                        open
                      >
                        <summary class="article-expander__title">Single-cell DNA sequencing identifies risk-associated clonal complexity and evolutionary trajectories in childhood medulloblastoma development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35831448/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220802212153&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We reconstructed the natural history and temporal evolution of the most common childhood brain malignancy, medulloblastoma, by single-cell whole-genome sequencing (sc-WGS) of tumours representing its major molecular sub-classes and clinical risk groups. Favourable-risk disease sub-types assessed (MB(WNT) and infant desmoplastic/nodular MB(SHH)) typically comprised a single clone with no evidence of further evolution. In contrast, highest risk sub-classes (MYC-amplified MB(Group3) and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35822488"
                        open
                      >
                        <summary class="article-expander__title">Targeting stem cells in myelodysplastic syndromes and acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35822488/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220802212153&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The genetic architecture of cancer has been delineated through advances in high-throughput next-generation sequencing, where the sequential acquisition of recurrent driver mutations initially targeted towards normal cells ultimately leads to malignant transformation. Myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML) are hematologic malignancies frequently initiated by mutations in the normal hematopoietic stem cell compartment leading to the establishment of leukemic stem cells....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35831448"
                        open
                      >
                        <summary class="article-expander__title">Single-cell DNA sequencing identifies risk-associated clonal complexity and evolutionary trajectories in childhood medulloblastoma development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35831448/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220801212447&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We reconstructed the natural history and temporal evolution of the most common childhood brain malignancy, medulloblastoma, by single-cell whole-genome sequencing (sc-WGS) of tumours representing its major molecular sub-classes and clinical risk groups. Favourable-risk disease sub-types assessed (MB(WNT) and infant desmoplastic/nodular MB(SHH)) typically comprised a single clone with no evidence of further evolution. In contrast, highest risk sub-classes (MYC-amplified MB(Group3) and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35822488"
                        open
                      >
                        <summary class="article-expander__title">Targeting stem cells in myelodysplastic syndromes and acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35822488/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220801212447&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The genetic architecture of cancer has been delineated through advances in high-throughput next-generation sequencing, where the sequential acquisition of recurrent driver mutations initially targeted towards normal cells ultimately leads to malignant transformation. Myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML) are hematologic malignancies frequently initiated by mutations in the normal hematopoietic stem cell compartment leading to the establishment of leukemic stem cells....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35831448"
                        open
                      >
                        <summary class="article-expander__title">Single-cell DNA sequencing identifies risk-associated clonal complexity and evolutionary trajectories in childhood medulloblastoma development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35831448/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220731212525&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We reconstructed the natural history and temporal evolution of the most common childhood brain malignancy, medulloblastoma, by single-cell whole-genome sequencing (sc-WGS) of tumours representing its major molecular sub-classes and clinical risk groups. Favourable-risk disease sub-types assessed (MB(WNT) and infant desmoplastic/nodular MB(SHH)) typically comprised a single clone with no evidence of further evolution. In contrast, highest risk sub-classes (MYC-amplified MB(Group3) and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35822488"
                        open
                      >
                        <summary class="article-expander__title">Targeting stem cells in myelodysplastic syndromes and acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35822488/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220731212525&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The genetic architecture of cancer has been delineated through advances in high-throughput next-generation sequencing, where the sequential acquisition of recurrent driver mutations initially targeted towards normal cells ultimately leads to malignant transformation. Myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML) are hematologic malignancies frequently initiated by mutations in the normal hematopoietic stem cell compartment leading to the establishment of leukemic stem cells....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35831448"
                        open
                      >
                        <summary class="article-expander__title">Single-cell DNA sequencing identifies risk-associated clonal complexity and evolutionary trajectories in childhood medulloblastoma development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35831448/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220730212309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We reconstructed the natural history and temporal evolution of the most common childhood brain malignancy, medulloblastoma, by single-cell whole-genome sequencing (sc-WGS) of tumours representing its major molecular sub-classes and clinical risk groups. Favourable-risk disease sub-types assessed (MB(WNT) and infant desmoplastic/nodular MB(SHH)) typically comprised a single clone with no evidence of further evolution. In contrast, highest risk sub-classes (MYC-amplified MB(Group3) and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35822488"
                        open
                      >
                        <summary class="article-expander__title">Targeting stem cells in myelodysplastic syndromes and acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35822488/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220730212309&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The genetic architecture of cancer has been delineated through advances in high-throughput next-generation sequencing, where the sequential acquisition of recurrent driver mutations initially targeted towards normal cells ultimately leads to malignant transformation. Myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML) are hematologic malignancies frequently initiated by mutations in the normal hematopoietic stem cell compartment leading to the establishment of leukemic stem cells....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35831448"
                        open
                      >
                        <summary class="article-expander__title">Single-cell DNA sequencing identifies risk-associated clonal complexity and evolutionary trajectories in childhood medulloblastoma development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35831448/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220729211332&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We reconstructed the natural history and temporal evolution of the most common childhood brain malignancy, medulloblastoma, by single-cell whole-genome sequencing (sc-WGS) of tumours representing its major molecular sub-classes and clinical risk groups. Favourable-risk disease sub-types assessed (MB(WNT) and infant desmoplastic/nodular MB(SHH)) typically comprised a single clone with no evidence of further evolution. In contrast, highest risk sub-classes (MYC-amplified MB(Group3) and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35822488"
                        open
                      >
                        <summary class="article-expander__title">Targeting stem cells in myelodysplastic syndromes and acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35822488/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220729211332&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The genetic architecture of cancer has been delineated through advances in high-throughput next-generation sequencing, where the sequential acquisition of recurrent driver mutations initially targeted towards normal cells ultimately leads to malignant transformation. Myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML) are hematologic malignancies frequently initiated by mutations in the normal hematopoietic stem cell compartment leading to the establishment of leukemic stem cells....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35831448"
                        open
                      >
                        <summary class="article-expander__title">Single-cell DNA sequencing identifies risk-associated clonal complexity and evolutionary trajectories in childhood medulloblastoma development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35831448/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220728211733&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We reconstructed the natural history and temporal evolution of the most common childhood brain malignancy, medulloblastoma, by single-cell whole-genome sequencing (sc-WGS) of tumours representing its major molecular sub-classes and clinical risk groups. Favourable-risk disease sub-types assessed (MB(WNT) and infant desmoplastic/nodular MB(SHH)) typically comprised a single clone with no evidence of further evolution. In contrast, highest risk sub-classes (MYC-amplified MB(Group3) and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35822488"
                        open
                      >
                        <summary class="article-expander__title">Targeting stem cells in myelodysplastic syndromes and acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35822488/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220728211733&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The genetic architecture of cancer has been delineated through advances in high-throughput next-generation sequencing, where the sequential acquisition of recurrent driver mutations initially targeted towards normal cells ultimately leads to malignant transformation. Myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML) are hematologic malignancies frequently initiated by mutations in the normal hematopoietic stem cell compartment leading to the establishment of leukemic stem cells....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35831448"
                        open
                      >
                        <summary class="article-expander__title">Single-cell DNA sequencing identifies risk-associated clonal complexity and evolutionary trajectories in childhood medulloblastoma development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35831448/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220727211458&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We reconstructed the natural history and temporal evolution of the most common childhood brain malignancy, medulloblastoma, by single-cell whole-genome sequencing (sc-WGS) of tumours representing its major molecular sub-classes and clinical risk groups. Favourable-risk disease sub-types assessed (MB(WNT) and infant desmoplastic/nodular MB(SHH)) typically comprised a single clone with no evidence of further evolution. In contrast, highest risk sub-classes (MYC-amplified MB(Group3) and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35822488"
                        open
                      >
                        <summary class="article-expander__title">Targeting stem cells in myelodysplastic syndromes and acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35822488/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220727211458&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The genetic architecture of cancer has been delineated through advances in high-throughput next-generation sequencing, where the sequential acquisition of recurrent driver mutations initially targeted towards normal cells ultimately leads to malignant transformation. Myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML) are hematologic malignancies frequently initiated by mutations in the normal hematopoietic stem cell compartment leading to the establishment of leukemic stem cells....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35831448"
                        open
                      >
                        <summary class="article-expander__title">Single-cell DNA sequencing identifies risk-associated clonal complexity and evolutionary trajectories in childhood medulloblastoma development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35831448/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220726212040&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We reconstructed the natural history and temporal evolution of the most common childhood brain malignancy, medulloblastoma, by single-cell whole-genome sequencing (sc-WGS) of tumours representing its major molecular sub-classes and clinical risk groups. Favourable-risk disease sub-types assessed (MB(WNT) and infant desmoplastic/nodular MB(SHH)) typically comprised a single clone with no evidence of further evolution. In contrast, highest risk sub-classes (MYC-amplified MB(Group3) and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35822488"
                        open
                      >
                        <summary class="article-expander__title">Targeting stem cells in myelodysplastic syndromes and acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35822488/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220726212040&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The genetic architecture of cancer has been delineated through advances in high-throughput next-generation sequencing, where the sequential acquisition of recurrent driver mutations initially targeted towards normal cells ultimately leads to malignant transformation. Myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML) are hematologic malignancies frequently initiated by mutations in the normal hematopoietic stem cell compartment leading to the establishment of leukemic stem cells....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35831448"
                        open
                      >
                        <summary class="article-expander__title">Single-cell DNA sequencing identifies risk-associated clonal complexity and evolutionary trajectories in childhood medulloblastoma development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35831448/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220725211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We reconstructed the natural history and temporal evolution of the most common childhood brain malignancy, medulloblastoma, by single-cell whole-genome sequencing (sc-WGS) of tumours representing its major molecular sub-classes and clinical risk groups. Favourable-risk disease sub-types assessed (MB(WNT) and infant desmoplastic/nodular MB(SHH)) typically comprised a single clone with no evidence of further evolution. In contrast, highest risk sub-classes (MYC-amplified MB(Group3) and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35822488"
                        open
                      >
                        <summary class="article-expander__title">Targeting stem cells in myelodysplastic syndromes and acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35822488/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220725211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The genetic architecture of cancer has been delineated through advances in high-throughput next-generation sequencing, where the sequential acquisition of recurrent driver mutations initially targeted towards normal cells ultimately leads to malignant transformation. Myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML) are hematologic malignancies frequently initiated by mutations in the normal hematopoietic stem cell compartment leading to the establishment of leukemic stem cells....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35831448"
                        open
                      >
                        <summary class="article-expander__title">Single-cell DNA sequencing identifies risk-associated clonal complexity and evolutionary trajectories in childhood medulloblastoma development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35831448/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220724211605&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We reconstructed the natural history and temporal evolution of the most common childhood brain malignancy, medulloblastoma, by single-cell whole-genome sequencing (sc-WGS) of tumours representing its major molecular sub-classes and clinical risk groups. Favourable-risk disease sub-types assessed (MB(WNT) and infant desmoplastic/nodular MB(SHH)) typically comprised a single clone with no evidence of further evolution. In contrast, highest risk sub-classes (MYC-amplified MB(Group3) and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35822488"
                        open
                      >
                        <summary class="article-expander__title">Targeting stem cells in myelodysplastic syndromes and acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35822488/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220724211605&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The genetic architecture of cancer has been delineated through advances in high-throughput next-generation sequencing, where the sequential acquisition of recurrent driver mutations initially targeted towards normal cells ultimately leads to malignant transformation. Myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML) are hematologic malignancies frequently initiated by mutations in the normal hematopoietic stem cell compartment leading to the establishment of leukemic stem cells....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35831448"
                        open
                      >
                        <summary class="article-expander__title">Single-cell DNA sequencing identifies risk-associated clonal complexity and evolutionary trajectories in childhood medulloblastoma development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35831448/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220723212121&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We reconstructed the natural history and temporal evolution of the most common childhood brain malignancy, medulloblastoma, by single-cell whole-genome sequencing (sc-WGS) of tumours representing its major molecular sub-classes and clinical risk groups. Favourable-risk disease sub-types assessed (MB(WNT) and infant desmoplastic/nodular MB(SHH)) typically comprised a single clone with no evidence of further evolution. In contrast, highest risk sub-classes (MYC-amplified MB(Group3) and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35822488"
                        open
                      >
                        <summary class="article-expander__title">Targeting stem cells in myelodysplastic syndromes and acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35822488/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220723212121&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The genetic architecture of cancer has been delineated through advances in high-throughput next-generation sequencing, where the sequential acquisition of recurrent driver mutations initially targeted towards normal cells ultimately leads to malignant transformation. Myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML) are hematologic malignancies frequently initiated by mutations in the normal hematopoietic stem cell compartment leading to the establishment of leukemic stem cells....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35831448"
                        open
                      >
                        <summary class="article-expander__title">Single-cell DNA sequencing identifies risk-associated clonal complexity and evolutionary trajectories in childhood medulloblastoma development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35831448/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220722211347&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We reconstructed the natural history and temporal evolution of the most common childhood brain malignancy, medulloblastoma, by single-cell whole-genome sequencing (sc-WGS) of tumours representing its major molecular sub-classes and clinical risk groups. Favourable-risk disease sub-types assessed (MB(WNT) and infant desmoplastic/nodular MB(SHH)) typically comprised a single clone with no evidence of further evolution. In contrast, highest risk sub-classes (MYC-amplified MB(Group3) and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35822488"
                        open
                      >
                        <summary class="article-expander__title">Targeting stem cells in myelodysplastic syndromes and acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35822488/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220722211347&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The genetic architecture of cancer has been delineated through advances in high-throughput next-generation sequencing, where the sequential acquisition of recurrent driver mutations initially targeted towards normal cells ultimately leads to malignant transformation. Myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML) are hematologic malignancies frequently initiated by mutations in the normal hematopoietic stem cell compartment leading to the establishment of leukemic stem cells....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35831448"
                        open
                      >
                        <summary class="article-expander__title">Single-cell DNA sequencing identifies risk-associated clonal complexity and evolutionary trajectories in childhood medulloblastoma development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35831448/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220721211948&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We reconstructed the natural history and temporal evolution of the most common childhood brain malignancy, medulloblastoma, by single-cell whole-genome sequencing (sc-WGS) of tumours representing its major molecular sub-classes and clinical risk groups. Favourable-risk disease sub-types assessed (MB(WNT) and infant desmoplastic/nodular MB(SHH)) typically comprised a single clone with no evidence of further evolution. In contrast, highest risk sub-classes (MYC-amplified MB(Group3) and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35831448"
                        open
                      >
                        <summary class="article-expander__title">Single-cell DNA sequencing identifies risk-associated clonal complexity and evolutionary trajectories in childhood medulloblastoma development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35831448/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220720211338&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We reconstructed the natural history and temporal evolution of the most common childhood brain malignancy, medulloblastoma, by single-cell whole-genome sequencing (sc-WGS) of tumours representing its major molecular sub-classes and clinical risk groups. Favourable-risk disease sub-types assessed (MB(WNT) and infant desmoplastic/nodular MB(SHH)) typically comprised a single clone with no evidence of further evolution. In contrast, highest risk sub-classes (MYC-amplified MB(Group3) and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35831448"
                        open
                      >
                        <summary class="article-expander__title">Single-cell DNA sequencing identifies risk-associated clonal complexity and evolutionary trajectories in childhood medulloblastoma development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35831448/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220719211909&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We reconstructed the natural history and temporal evolution of the most common childhood brain malignancy, medulloblastoma, by single-cell whole-genome sequencing (sc-WGS) of tumours representing its major molecular sub-classes and clinical risk groups. Favourable-risk disease sub-types assessed (MB(WNT) and infant desmoplastic/nodular MB(SHH)) typically comprised a single clone with no evidence of further evolution. In contrast, highest risk sub-classes (MYC-amplified MB(Group3) and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35831448"
                        open
                      >
                        <summary class="article-expander__title">Single-cell DNA sequencing identifies risk-associated clonal complexity and evolutionary trajectories in childhood medulloblastoma development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35831448/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220718212208&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We reconstructed the natural history and temporal evolution of the most common childhood brain malignancy, medulloblastoma, by single-cell whole-genome sequencing (sc-WGS) of tumours representing its major molecular sub-classes and clinical risk groups. Favourable-risk disease sub-types assessed (MB(WNT) and infant desmoplastic/nodular MB(SHH)) typically comprised a single clone with no evidence of further evolution. In contrast, highest risk sub-classes (MYC-amplified MB(Group3) and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35831448"
                        open
                      >
                        <summary class="article-expander__title">Single-cell DNA sequencing identifies risk-associated clonal complexity and evolutionary trajectories in childhood medulloblastoma development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35831448/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220717211412&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We reconstructed the natural history and temporal evolution of the most common childhood brain malignancy, medulloblastoma, by single-cell whole-genome sequencing (sc-WGS) of tumours representing its major molecular sub-classes and clinical risk groups. Favourable-risk disease sub-types assessed (MB(WNT) and infant desmoplastic/nodular MB(SHH)) typically comprised a single clone with no evidence of further evolution. In contrast, highest risk sub-classes (MYC-amplified MB(Group3) and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35831448"
                        open
                      >
                        <summary class="article-expander__title">Single-cell DNA sequencing identifies risk-associated clonal complexity and evolutionary trajectories in childhood medulloblastoma development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35831448/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220716212224&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We reconstructed the natural history and temporal evolution of the most common childhood brain malignancy, medulloblastoma, by single-cell whole-genome sequencing (sc-WGS) of tumours representing its major molecular sub-classes and clinical risk groups. Favourable-risk disease sub-types assessed (MB(WNT) and infant desmoplastic/nodular MB(SHH)) typically comprised a single clone with no evidence of further evolution. In contrast, highest risk sub-classes (MYC-amplified MB(Group3) and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35831448"
                        open
                      >
                        <summary class="article-expander__title">Single-cell DNA sequencing identifies risk-associated clonal complexity and evolutionary trajectories in childhood medulloblastoma development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35831448/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220715211735&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We reconstructed the natural history and temporal evolution of the most common childhood brain malignancy, medulloblastoma, by single-cell whole-genome sequencing (sc-WGS) of tumours representing its major molecular sub-classes and clinical risk groups. Favourable-risk disease sub-types assessed (MB(WNT) and infant desmoplastic/nodular MB(SHH)) typically comprised a single clone with no evidence of further evolution. In contrast, highest risk sub-classes (MYC-amplified MB(Group3) and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35831448"
                        open
                      >
                        <summary class="article-expander__title">Single-cell DNA sequencing identifies risk-associated clonal complexity and evolutionary trajectories in childhood medulloblastoma development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35831448/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We reconstructed the natural history and temporal evolution of the most common childhood brain malignancy, medulloblastoma, by single-cell whole-genome sequencing (sc-WGS) of tumours representing its major molecular sub-classes and clinical risk groups. Favourable-risk disease sub-types assessed (MB(WNT) and infant desmoplastic/nodular MB(SHH)) typically comprised a single clone with no evidence of further evolution. In contrast, highest risk sub-classes (MYC-amplified MB(Group3) and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-07-12 datetime="2022-07-12T10:00:00.000Z">2022-07-12</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-07-12 datetime="2022-07-12T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608003622&amp;v&#x3D;2.17.7&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35818539"
                        open
                      >
                        <summary class="article-expander__title">Cardiovascular Disease in Myeloproliferative Neoplasms: JACC: CardioOncology State-of-the-Art Review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35818539/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220724211604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms are associated with increased risk for thrombotic complications. These conditions most commonly involve somatic mutations in genes that lead to constitutive activation of the Janus-associated kinase signaling pathway (eg, Janus kinase 2, calreticulin, myeloproliferative leukemia protein). Acquired gain-of-function mutations in these genes, particularly Janus kinase 2, can cause a spectrum of disorders, ranging from clonal hematopoiesis of indeterminate potential, a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35818539"
                        open
                      >
                        <summary class="article-expander__title">Cardiovascular Disease in Myeloproliferative Neoplasms: JACC: CardioOncology State-of-the-Art Review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35818539/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220723212121&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms are associated with increased risk for thrombotic complications. These conditions most commonly involve somatic mutations in genes that lead to constitutive activation of the Janus-associated kinase signaling pathway (eg, Janus kinase 2, calreticulin, myeloproliferative leukemia protein). Acquired gain-of-function mutations in these genes, particularly Janus kinase 2, can cause a spectrum of disorders, ranging from clonal hematopoiesis of indeterminate potential, a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35818539"
                        open
                      >
                        <summary class="article-expander__title">Cardiovascular Disease in Myeloproliferative Neoplasms: JACC: CardioOncology State-of-the-Art Review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35818539/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220722211347&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms are associated with increased risk for thrombotic complications. These conditions most commonly involve somatic mutations in genes that lead to constitutive activation of the Janus-associated kinase signaling pathway (eg, Janus kinase 2, calreticulin, myeloproliferative leukemia protein). Acquired gain-of-function mutations in these genes, particularly Janus kinase 2, can cause a spectrum of disorders, ranging from clonal hematopoiesis of indeterminate potential, a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35818539"
                        open
                      >
                        <summary class="article-expander__title">Cardiovascular Disease in Myeloproliferative Neoplasms: JACC: CardioOncology State-of-the-Art Review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35818539/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220721211948&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms are associated with increased risk for thrombotic complications. These conditions most commonly involve somatic mutations in genes that lead to constitutive activation of the Janus-associated kinase signaling pathway (eg, Janus kinase 2, calreticulin, myeloproliferative leukemia protein). Acquired gain-of-function mutations in these genes, particularly Janus kinase 2, can cause a spectrum of disorders, ranging from clonal hematopoiesis of indeterminate potential, a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35818539"
                        open
                      >
                        <summary class="article-expander__title">Cardiovascular Disease in Myeloproliferative Neoplasms: JACC: CardioOncology State-of-the-Art Review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35818539/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220720211338&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms are associated with increased risk for thrombotic complications. These conditions most commonly involve somatic mutations in genes that lead to constitutive activation of the Janus-associated kinase signaling pathway (eg, Janus kinase 2, calreticulin, myeloproliferative leukemia protein). Acquired gain-of-function mutations in these genes, particularly Janus kinase 2, can cause a spectrum of disorders, ranging from clonal hematopoiesis of indeterminate potential, a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35818539"
                        open
                      >
                        <summary class="article-expander__title">Cardiovascular Disease in Myeloproliferative Neoplasms: JACC: CardioOncology State-of-the-Art Review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35818539/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220719211909&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms are associated with increased risk for thrombotic complications. These conditions most commonly involve somatic mutations in genes that lead to constitutive activation of the Janus-associated kinase signaling pathway (eg, Janus kinase 2, calreticulin, myeloproliferative leukemia protein). Acquired gain-of-function mutations in these genes, particularly Janus kinase 2, can cause a spectrum of disorders, ranging from clonal hematopoiesis of indeterminate potential, a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35818539"
                        open
                      >
                        <summary class="article-expander__title">Cardiovascular Disease in Myeloproliferative Neoplasms: JACC: CardioOncology State-of-the-Art Review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35818539/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220718212208&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms are associated with increased risk for thrombotic complications. These conditions most commonly involve somatic mutations in genes that lead to constitutive activation of the Janus-associated kinase signaling pathway (eg, Janus kinase 2, calreticulin, myeloproliferative leukemia protein). Acquired gain-of-function mutations in these genes, particularly Janus kinase 2, can cause a spectrum of disorders, ranging from clonal hematopoiesis of indeterminate potential, a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35818539"
                        open
                      >
                        <summary class="article-expander__title">Cardiovascular Disease in Myeloproliferative Neoplasms: JACC: CardioOncology State-of-the-Art Review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35818539/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220717211412&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms are associated with increased risk for thrombotic complications. These conditions most commonly involve somatic mutations in genes that lead to constitutive activation of the Janus-associated kinase signaling pathway (eg, Janus kinase 2, calreticulin, myeloproliferative leukemia protein). Acquired gain-of-function mutations in these genes, particularly Janus kinase 2, can cause a spectrum of disorders, ranging from clonal hematopoiesis of indeterminate potential, a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35818539"
                        open
                      >
                        <summary class="article-expander__title">Cardiovascular Disease in Myeloproliferative Neoplasms: JACC: CardioOncology State-of-the-Art Review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35818539/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220716212224&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms are associated with increased risk for thrombotic complications. These conditions most commonly involve somatic mutations in genes that lead to constitutive activation of the Janus-associated kinase signaling pathway (eg, Janus kinase 2, calreticulin, myeloproliferative leukemia protein). Acquired gain-of-function mutations in these genes, particularly Janus kinase 2, can cause a spectrum of disorders, ranging from clonal hematopoiesis of indeterminate potential, a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35818539"
                        open
                      >
                        <summary class="article-expander__title">Cardiovascular Disease in Myeloproliferative Neoplasms: JACC: CardioOncology State-of-the-Art Review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35818539/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220715211735&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms are associated with increased risk for thrombotic complications. These conditions most commonly involve somatic mutations in genes that lead to constitutive activation of the Janus-associated kinase signaling pathway (eg, Janus kinase 2, calreticulin, myeloproliferative leukemia protein). Acquired gain-of-function mutations in these genes, particularly Janus kinase 2, can cause a spectrum of disorders, ranging from clonal hematopoiesis of indeterminate potential, a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35818539"
                        open
                      >
                        <summary class="article-expander__title">Cardiovascular Disease in Myeloproliferative Neoplasms: JACC: CardioOncology State-of-the-Art Review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35818539/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms are associated with increased risk for thrombotic complications. These conditions most commonly involve somatic mutations in genes that lead to constitutive activation of the Janus-associated kinase signaling pathway (eg, Janus kinase 2, calreticulin, myeloproliferative leukemia protein). Acquired gain-of-function mutations in these genes, particularly Janus kinase 2, can cause a spectrum of disorders, ranging from clonal hematopoiesis of indeterminate potential, a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35818539"
                        open
                      >
                        <summary class="article-expander__title">Cardiovascular Disease in Myeloproliferative Neoplasms: JACC: CardioOncology State-of-the-Art Review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35818539/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms are associated with increased risk for thrombotic complications. These conditions most commonly involve somatic mutations in genes that lead to constitutive activation of the Janus-associated kinase signaling pathway (eg, Janus kinase 2, calreticulin, myeloproliferative leukemia protein). Acquired gain-of-function mutations in these genes, particularly Janus kinase 2, can cause a spectrum of disorders, ranging from clonal hematopoiesis of indeterminate potential, a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35818539"
                        open
                      >
                        <summary class="article-expander__title">Cardiovascular Disease in Myeloproliferative Neoplasms: JACC: CardioOncology State-of-the-Art Review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35818539/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms are associated with increased risk for thrombotic complications. These conditions most commonly involve somatic mutations in genes that lead to constitutive activation of the Janus-associated kinase signaling pathway (eg, Janus kinase 2, calreticulin, myeloproliferative leukemia protein). Acquired gain-of-function mutations in these genes, particularly Janus kinase 2, can cause a spectrum of disorders, ranging from clonal hematopoiesis of indeterminate potential, a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608010801&amp;v&#x3D;2.17.7&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35820396"
                        open
                      >
                        <summary class="article-expander__title">Immune pressure sculps tumor cells and trims high-quality mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35820396/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220804212147&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immunoediting, the loss of tumor (neo)antigens due to T cell-dependent selection, sculpts tumor immunogenicity. In Nature, Łuksza et al. conceive a model to score neoantigens&#x27; immunogenicity and predict tumor clonal evolution. With this model, they demonstrate that durable tumor control associates with selective limited acquisition of high-quality mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35820396"
                        open
                      >
                        <summary class="article-expander__title">Immune pressure sculps tumor cells and trims high-quality mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35820396/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220803212051&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immunoediting, the loss of tumor (neo)antigens due to T cell-dependent selection, sculpts tumor immunogenicity. In Nature, Łuksza et al. conceive a model to score neoantigens&#x27; immunogenicity and predict tumor clonal evolution. With this model, they demonstrate that durable tumor control associates with selective limited acquisition of high-quality mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35820396"
                        open
                      >
                        <summary class="article-expander__title">Immune pressure sculps tumor cells and trims high-quality mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35820396/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220802212153&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immunoediting, the loss of tumor (neo)antigens due to T cell-dependent selection, sculpts tumor immunogenicity. In Nature, Łuksza et al. conceive a model to score neoantigens&#x27; immunogenicity and predict tumor clonal evolution. With this model, they demonstrate that durable tumor control associates with selective limited acquisition of high-quality mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35820396"
                        open
                      >
                        <summary class="article-expander__title">Immune pressure sculps tumor cells and trims high-quality mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35820396/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220727211458&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immunoediting, the loss of tumor (neo)antigens due to T cell-dependent selection, sculpts tumor immunogenicity. In Nature, Łuksza et al. conceive a model to score neoantigens&#x27; immunogenicity and predict tumor clonal evolution. With this model, they demonstrate that durable tumor control associates with selective limited acquisition of high-quality mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35820396"
                        open
                      >
                        <summary class="article-expander__title">Immune pressure sculps tumor cells and trims high-quality mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35820396/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220726212040&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immunoediting, the loss of tumor (neo)antigens due to T cell-dependent selection, sculpts tumor immunogenicity. In Nature, Łuksza et al. conceive a model to score neoantigens&#x27; immunogenicity and predict tumor clonal evolution. With this model, they demonstrate that durable tumor control associates with selective limited acquisition of high-quality mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35820396"
                        open
                      >
                        <summary class="article-expander__title">Immune pressure sculps tumor cells and trims high-quality mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35820396/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220725211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immunoediting, the loss of tumor (neo)antigens due to T cell-dependent selection, sculpts tumor immunogenicity. In Nature, Łuksza et al. conceive a model to score neoantigens&#x27; immunogenicity and predict tumor clonal evolution. With this model, they demonstrate that durable tumor control associates with selective limited acquisition of high-quality mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35820396"
                        open
                      >
                        <summary class="article-expander__title">Immune pressure sculps tumor cells and trims high-quality mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35820396/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220724211605&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immunoediting, the loss of tumor (neo)antigens due to T cell-dependent selection, sculpts tumor immunogenicity. In Nature, Łuksza et al. conceive a model to score neoantigens&#x27; immunogenicity and predict tumor clonal evolution. With this model, they demonstrate that durable tumor control associates with selective limited acquisition of high-quality mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35820396"
                        open
                      >
                        <summary class="article-expander__title">Immune pressure sculps tumor cells and trims high-quality mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35820396/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220723212121&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immunoediting, the loss of tumor (neo)antigens due to T cell-dependent selection, sculpts tumor immunogenicity. In Nature, Łuksza et al. conceive a model to score neoantigens&#x27; immunogenicity and predict tumor clonal evolution. With this model, they demonstrate that durable tumor control associates with selective limited acquisition of high-quality mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35820396"
                        open
                      >
                        <summary class="article-expander__title">Immune pressure sculps tumor cells and trims high-quality mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35820396/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220722211347&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immunoediting, the loss of tumor (neo)antigens due to T cell-dependent selection, sculpts tumor immunogenicity. In Nature, Łuksza et al. conceive a model to score neoantigens&#x27; immunogenicity and predict tumor clonal evolution. With this model, they demonstrate that durable tumor control associates with selective limited acquisition of high-quality mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35820396"
                        open
                      >
                        <summary class="article-expander__title">Immune pressure sculps tumor cells and trims high-quality mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35820396/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220721211948&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immunoediting, the loss of tumor (neo)antigens due to T cell-dependent selection, sculpts tumor immunogenicity. In Nature, Łuksza et al. conceive a model to score neoantigens&#x27; immunogenicity and predict tumor clonal evolution. With this model, they demonstrate that durable tumor control associates with selective limited acquisition of high-quality mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35820396"
                        open
                      >
                        <summary class="article-expander__title">Immune pressure sculps tumor cells and trims high-quality mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35820396/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220720211338&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immunoediting, the loss of tumor (neo)antigens due to T cell-dependent selection, sculpts tumor immunogenicity. In Nature, Łuksza et al. conceive a model to score neoantigens&#x27; immunogenicity and predict tumor clonal evolution. With this model, they demonstrate that durable tumor control associates with selective limited acquisition of high-quality mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35820396"
                        open
                      >
                        <summary class="article-expander__title">Immune pressure sculps tumor cells and trims high-quality mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35820396/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220719211909&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immunoediting, the loss of tumor (neo)antigens due to T cell-dependent selection, sculpts tumor immunogenicity. In Nature, Łuksza et al. conceive a model to score neoantigens&#x27; immunogenicity and predict tumor clonal evolution. With this model, they demonstrate that durable tumor control associates with selective limited acquisition of high-quality mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35820396"
                        open
                      >
                        <summary class="article-expander__title">Immune pressure sculps tumor cells and trims high-quality mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35820396/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220718212208&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immunoediting, the loss of tumor (neo)antigens due to T cell-dependent selection, sculpts tumor immunogenicity. In Nature, Łuksza et al. conceive a model to score neoantigens&#x27; immunogenicity and predict tumor clonal evolution. With this model, they demonstrate that durable tumor control associates with selective limited acquisition of high-quality mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35820396"
                        open
                      >
                        <summary class="article-expander__title">Immune pressure sculps tumor cells and trims high-quality mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35820396/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220717211412&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immunoediting, the loss of tumor (neo)antigens due to T cell-dependent selection, sculpts tumor immunogenicity. In Nature, Łuksza et al. conceive a model to score neoantigens&#x27; immunogenicity and predict tumor clonal evolution. With this model, they demonstrate that durable tumor control associates with selective limited acquisition of high-quality mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35820396"
                        open
                      >
                        <summary class="article-expander__title">Immune pressure sculps tumor cells and trims high-quality mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35820396/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220716212224&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immunoediting, the loss of tumor (neo)antigens due to T cell-dependent selection, sculpts tumor immunogenicity. In Nature, Łuksza et al. conceive a model to score neoantigens&#x27; immunogenicity and predict tumor clonal evolution. With this model, they demonstrate that durable tumor control associates with selective limited acquisition of high-quality mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35820396"
                        open
                      >
                        <summary class="article-expander__title">Immune pressure sculps tumor cells and trims high-quality mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35820396/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220715211735&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immunoediting, the loss of tumor (neo)antigens due to T cell-dependent selection, sculpts tumor immunogenicity. In Nature, Łuksza et al. conceive a model to score neoantigens&#x27; immunogenicity and predict tumor clonal evolution. With this model, they demonstrate that durable tumor control associates with selective limited acquisition of high-quality mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35820396"
                        open
                      >
                        <summary class="article-expander__title">Immune pressure sculps tumor cells and trims high-quality mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35820396/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immunoediting, the loss of tumor (neo)antigens due to T cell-dependent selection, sculpts tumor immunogenicity. In Nature, Łuksza et al. conceive a model to score neoantigens&#x27; immunogenicity and predict tumor clonal evolution. With this model, they demonstrate that durable tumor control associates with selective limited acquisition of high-quality mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35820396"
                        open
                      >
                        <summary class="article-expander__title">Immune pressure sculps tumor cells and trims high-quality mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35820396/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immunoediting, the loss of tumor (neo)antigens due to T cell-dependent selection, sculpts tumor immunogenicity. In Nature, Łuksza et al. conceive a model to score neoantigens&#x27; immunogenicity and predict tumor clonal evolution. With this model, they demonstrate that durable tumor control associates with selective limited acquisition of high-quality mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-07-11 datetime="2022-07-11T10:00:00.000Z">2022-07-11</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-07-11 datetime="2022-07-11T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608003622&amp;v&#x3D;2.17.7&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35815881"
                        open
                      >
                        <summary class="article-expander__title">Targeted sequencing of candidate gene regions for myelofibrosis in dogs</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35815881/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220724211604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS AND CLINICAL IMPORTANCE: Somatic variants might initiate or perpetuate myelofibrosis in dogs. Our findings suggest the occurrence of clonal hematopoiesis in dogs, with increasing incidence with age, as observed in humans.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35815881"
                        open
                      >
                        <summary class="article-expander__title">Targeted sequencing of candidate gene regions for myelofibrosis in dogs</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35815881/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220723212121&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS AND CLINICAL IMPORTANCE: Somatic variants might initiate or perpetuate myelofibrosis in dogs. Our findings suggest the occurrence of clonal hematopoiesis in dogs, with increasing incidence with age, as observed in humans.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35815881"
                        open
                      >
                        <summary class="article-expander__title">Targeted sequencing of candidate gene regions for myelofibrosis in dogs</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35815881/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220722211347&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS AND CLINICAL IMPORTANCE: Somatic variants might initiate or perpetuate myelofibrosis in dogs. Our findings suggest the occurrence of clonal hematopoiesis in dogs, with increasing incidence with age, as observed in humans.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35815881"
                        open
                      >
                        <summary class="article-expander__title">Targeted sequencing of candidate gene regions for myelofibrosis in dogs</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35815881/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220721211948&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS AND CLINICAL IMPORTANCE: Somatic variants might initiate or perpetuate myelofibrosis in dogs. Our findings suggest the occurrence of clonal hematopoiesis in dogs, with increasing incidence with age, as observed in humans.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35815881"
                        open
                      >
                        <summary class="article-expander__title">Targeted sequencing of candidate gene regions for myelofibrosis in dogs</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35815881/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220720211338&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS AND CLINICAL IMPORTANCE: Somatic variants might initiate or perpetuate myelofibrosis in dogs. Our findings suggest the occurrence of clonal hematopoiesis in dogs, with increasing incidence with age, as observed in humans.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35815881"
                        open
                      >
                        <summary class="article-expander__title">Targeted sequencing of candidate gene regions for myelofibrosis in dogs</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35815881/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220719211909&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS AND CLINICAL IMPORTANCE: Somatic variants might initiate or perpetuate myelofibrosis in dogs. Our findings suggest the occurrence of clonal hematopoiesis in dogs, with increasing incidence with age, as observed in humans.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35815881"
                        open
                      >
                        <summary class="article-expander__title">Targeted sequencing of candidate gene regions for myelofibrosis in dogs</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35815881/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220718212208&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS AND CLINICAL IMPORTANCE: Somatic variants might initiate or perpetuate myelofibrosis in dogs. Our findings suggest the occurrence of clonal hematopoiesis in dogs, with increasing incidence with age, as observed in humans.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35815881"
                        open
                      >
                        <summary class="article-expander__title">Targeted sequencing of candidate gene regions for myelofibrosis in dogs</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35815881/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220717211412&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS AND CLINICAL IMPORTANCE: Somatic variants might initiate or perpetuate myelofibrosis in dogs. Our findings suggest the occurrence of clonal hematopoiesis in dogs, with increasing incidence with age, as observed in humans.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35815881"
                        open
                      >
                        <summary class="article-expander__title">Targeted sequencing of candidate gene regions for myelofibrosis in dogs</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35815881/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220716212224&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS AND CLINICAL IMPORTANCE: Somatic variants might initiate or perpetuate myelofibrosis in dogs. Our findings suggest the occurrence of clonal hematopoiesis in dogs, with increasing incidence with age, as observed in humans.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35815881"
                        open
                      >
                        <summary class="article-expander__title">Targeted sequencing of candidate gene regions for myelofibrosis in dogs</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35815881/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220715211735&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS AND CLINICAL IMPORTANCE: Somatic variants might initiate or perpetuate myelofibrosis in dogs. Our findings suggest the occurrence of clonal hematopoiesis in dogs, with increasing incidence with age, as observed in humans.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35815881"
                        open
                      >
                        <summary class="article-expander__title">Targeted sequencing of candidate gene regions for myelofibrosis in dogs</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35815881/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS AND CLINICAL IMPORTANCE: Somatic variants might initiate or perpetuate myelofibrosis in dogs. Our findings suggest the occurrence of clonal hematopoiesis in dogs, with increasing incidence with age, as observed in humans.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35815881"
                        open
                      >
                        <summary class="article-expander__title">Targeted sequencing of candidate gene regions for myelofibrosis in dogs</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35815881/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS AND CLINICAL IMPORTANCE: Somatic variants might initiate or perpetuate myelofibrosis in dogs. Our findings suggest the occurrence of clonal hematopoiesis in dogs, with increasing incidence with age, as observed in humans.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35815881"
                        open
                      >
                        <summary class="article-expander__title">Targeted sequencing of candidate gene regions for myelofibrosis in dogs</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35815881/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS AND CLINICAL IMPORTANCE: Somatic variants might initiate or perpetuate myelofibrosis in dogs. Our findings suggest the occurrence of clonal hematopoiesis in dogs, with increasing incidence with age, as observed in humans.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35815881"
                        open
                      >
                        <summary class="article-expander__title">Targeted sequencing of candidate gene regions for myelofibrosis in dogs</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35815881/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS AND CLINICAL IMPORTANCE: Somatic variants might initiate or perpetuate myelofibrosis in dogs. Our findings suggest the occurrence of clonal hematopoiesis in dogs, with increasing incidence with age, as observed in humans.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608010801&amp;v&#x3D;2.17.7&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35815661"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive genetic analysis of histological components of combined small cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35815661/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220727211458&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: We conclude that the morphological evolution of heterogeneous histological components in cSCLC may be associated with differences in ASCL1 expression levels, but not in acquired somatic gene mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35815661"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive genetic analysis of histological components of combined small cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35815661/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220726212040&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: We conclude that the morphological evolution of heterogeneous histological components in cSCLC may be associated with differences in ASCL1 expression levels, but not in acquired somatic gene mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35815661"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive genetic analysis of histological components of combined small cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35815661/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220725211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: We conclude that the morphological evolution of heterogeneous histological components in cSCLC may be associated with differences in ASCL1 expression levels, but not in acquired somatic gene mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35815661"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive genetic analysis of histological components of combined small cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35815661/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220724211605&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: We conclude that the morphological evolution of heterogeneous histological components in cSCLC may be associated with differences in ASCL1 expression levels, but not in acquired somatic gene mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35815661"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive genetic analysis of histological components of combined small cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35815661/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220723212121&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: We conclude that the morphological evolution of heterogeneous histological components in cSCLC may be associated with differences in ASCL1 expression levels, but not in acquired somatic gene mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35815661"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive genetic analysis of histological components of combined small cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35815661/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220722211347&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: We conclude that the morphological evolution of heterogeneous histological components in cSCLC may be associated with differences in ASCL1 expression levels, but not in acquired somatic gene mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35815661"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive genetic analysis of histological components of combined small cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35815661/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220721211948&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: We conclude that the morphological evolution of heterogeneous histological components in cSCLC may be associated with differences in ASCL1 expression levels, but not in acquired somatic gene mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35815661"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive genetic analysis of histological components of combined small cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35815661/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220720211338&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: We conclude that the morphological evolution of heterogeneous histological components in cSCLC may be associated with differences in ASCL1 expression levels, but not in acquired somatic gene mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35815661"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive genetic analysis of histological components of combined small cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35815661/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220719211909&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: We conclude that the morphological evolution of heterogeneous histological components in cSCLC may be associated with differences in ASCL1 expression levels, but not in acquired somatic gene mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35815661"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive genetic analysis of histological components of combined small cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35815661/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220718212208&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: We conclude that the morphological evolution of heterogeneous histological components in cSCLC may be associated with differences in ASCL1 expression levels, but not in acquired somatic gene mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35815661"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive genetic analysis of histological components of combined small cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35815661/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220717211412&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: We conclude that the morphological evolution of heterogeneous histological components in cSCLC may be associated with differences in ASCL1 expression levels, but not in acquired somatic gene mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35815661"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive genetic analysis of histological components of combined small cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35815661/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220716212224&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: We conclude that the morphological evolution of heterogeneous histological components in cSCLC may be associated with differences in ASCL1 expression levels, but not in acquired somatic gene mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35815661"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive genetic analysis of histological components of combined small cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35815661/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220715211735&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: We conclude that the morphological evolution of heterogeneous histological components in cSCLC may be associated with differences in ASCL1 expression levels, but not in acquired somatic gene mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35815661"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive genetic analysis of histological components of combined small cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35815661/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: We conclude that the morphological evolution of heterogeneous histological components in cSCLC may be associated with differences in ASCL1 expression levels, but not in acquired somatic gene mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35815661"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive genetic analysis of histological components of combined small cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35815661/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: We conclude that the morphological evolution of heterogeneous histological components in cSCLC may be associated with differences in ASCL1 expression levels, but not in acquired somatic gene mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35815661"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive genetic analysis of histological components of combined small cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35815661/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: We conclude that the morphological evolution of heterogeneous histological components in cSCLC may be associated with differences in ASCL1 expression levels, but not in acquired somatic gene mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35815661"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive genetic analysis of histological components of combined small cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35815661/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: We conclude that the morphological evolution of heterogeneous histological components in cSCLC may be associated with differences in ASCL1 expression levels, but not in acquired somatic gene mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-07-09 datetime="2022-07-09T10:00:00.000Z">2022-07-09</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-07-09 datetime="2022-07-09T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608003622&amp;v&#x3D;2.17.7&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35805204"
                        open
                      >
                        <summary class="article-expander__title">Association of Clonal Hematopoiesis of Indeterminate Potential with Inflammatory Gene Expression in Patients with COPD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35805204/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220723212121&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic obstructive pulmonary disease (COPD) is a disease with an inflammatory phenotype with increasing prevalence in the elderly. Expanded population of mutant blood cells carrying somatic mutations is termed clonal hematopoiesis of indeterminate potential (CHIP). The association between CHIP and COPD and its relevant effects on DNA methylation in aging are mainly unknown. Analyzing the deep-targeted amplicon sequencing from 125 COPD patients, we found enhanced incidence of CHIP mutations...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35805204"
                        open
                      >
                        <summary class="article-expander__title">Association of Clonal Hematopoiesis of Indeterminate Potential with Inflammatory Gene Expression in Patients with COPD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35805204/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220722211347&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic obstructive pulmonary disease (COPD) is a disease with an inflammatory phenotype with increasing prevalence in the elderly. Expanded population of mutant blood cells carrying somatic mutations is termed clonal hematopoiesis of indeterminate potential (CHIP). The association between CHIP and COPD and its relevant effects on DNA methylation in aging are mainly unknown. Analyzing the deep-targeted amplicon sequencing from 125 COPD patients, we found enhanced incidence of CHIP mutations...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35805204"
                        open
                      >
                        <summary class="article-expander__title">Association of Clonal Hematopoiesis of Indeterminate Potential with Inflammatory Gene Expression in Patients with COPD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35805204/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220721211948&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic obstructive pulmonary disease (COPD) is a disease with an inflammatory phenotype with increasing prevalence in the elderly. Expanded population of mutant blood cells carrying somatic mutations is termed clonal hematopoiesis of indeterminate potential (CHIP). The association between CHIP and COPD and its relevant effects on DNA methylation in aging are mainly unknown. Analyzing the deep-targeted amplicon sequencing from 125 COPD patients, we found enhanced incidence of CHIP mutations...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35805204"
                        open
                      >
                        <summary class="article-expander__title">Association of Clonal Hematopoiesis of Indeterminate Potential with Inflammatory Gene Expression in Patients with COPD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35805204/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220720211338&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic obstructive pulmonary disease (COPD) is a disease with an inflammatory phenotype with increasing prevalence in the elderly. Expanded population of mutant blood cells carrying somatic mutations is termed clonal hematopoiesis of indeterminate potential (CHIP). The association between CHIP and COPD and its relevant effects on DNA methylation in aging are mainly unknown. Analyzing the deep-targeted amplicon sequencing from 125 COPD patients, we found enhanced incidence of CHIP mutations...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35805204"
                        open
                      >
                        <summary class="article-expander__title">Association of Clonal Hematopoiesis of Indeterminate Potential with Inflammatory Gene Expression in Patients with COPD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35805204/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220719211909&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic obstructive pulmonary disease (COPD) is a disease with an inflammatory phenotype with increasing prevalence in the elderly. Expanded population of mutant blood cells carrying somatic mutations is termed clonal hematopoiesis of indeterminate potential (CHIP). The association between CHIP and COPD and its relevant effects on DNA methylation in aging are mainly unknown. Analyzing the deep-targeted amplicon sequencing from 125 COPD patients, we found enhanced incidence of CHIP mutations...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35805204"
                        open
                      >
                        <summary class="article-expander__title">Association of Clonal Hematopoiesis of Indeterminate Potential with Inflammatory Gene Expression in Patients with COPD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35805204/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220718212208&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic obstructive pulmonary disease (COPD) is a disease with an inflammatory phenotype with increasing prevalence in the elderly. Expanded population of mutant blood cells carrying somatic mutations is termed clonal hematopoiesis of indeterminate potential (CHIP). The association between CHIP and COPD and its relevant effects on DNA methylation in aging are mainly unknown. Analyzing the deep-targeted amplicon sequencing from 125 COPD patients, we found enhanced incidence of CHIP mutations...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35805204"
                        open
                      >
                        <summary class="article-expander__title">Association of Clonal Hematopoiesis of Indeterminate Potential with Inflammatory Gene Expression in Patients with COPD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35805204/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220717211412&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic obstructive pulmonary disease (COPD) is a disease with an inflammatory phenotype with increasing prevalence in the elderly. Expanded population of mutant blood cells carrying somatic mutations is termed clonal hematopoiesis of indeterminate potential (CHIP). The association between CHIP and COPD and its relevant effects on DNA methylation in aging are mainly unknown. Analyzing the deep-targeted amplicon sequencing from 125 COPD patients, we found enhanced incidence of CHIP mutations...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35805204"
                        open
                      >
                        <summary class="article-expander__title">Association of Clonal Hematopoiesis of Indeterminate Potential with Inflammatory Gene Expression in Patients with COPD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35805204/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220716212224&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic obstructive pulmonary disease (COPD) is a disease with an inflammatory phenotype with increasing prevalence in the elderly. Expanded population of mutant blood cells carrying somatic mutations is termed clonal hematopoiesis of indeterminate potential (CHIP). The association between CHIP and COPD and its relevant effects on DNA methylation in aging are mainly unknown. Analyzing the deep-targeted amplicon sequencing from 125 COPD patients, we found enhanced incidence of CHIP mutations...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35805204"
                        open
                      >
                        <summary class="article-expander__title">Association of Clonal Hematopoiesis of Indeterminate Potential with Inflammatory Gene Expression in Patients with COPD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35805204/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220715211735&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic obstructive pulmonary disease (COPD) is a disease with an inflammatory phenotype with increasing prevalence in the elderly. Expanded population of mutant blood cells carrying somatic mutations is termed clonal hematopoiesis of indeterminate potential (CHIP). The association between CHIP and COPD and its relevant effects on DNA methylation in aging are mainly unknown. Analyzing the deep-targeted amplicon sequencing from 125 COPD patients, we found enhanced incidence of CHIP mutations...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35805204"
                        open
                      >
                        <summary class="article-expander__title">Association of Clonal Hematopoiesis of Indeterminate Potential with Inflammatory Gene Expression in Patients with COPD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35805204/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic obstructive pulmonary disease (COPD) is a disease with an inflammatory phenotype with increasing prevalence in the elderly. Expanded population of mutant blood cells carrying somatic mutations is termed clonal hematopoiesis of indeterminate potential (CHIP). The association between CHIP and COPD and its relevant effects on DNA methylation in aging are mainly unknown. Analyzing the deep-targeted amplicon sequencing from 125 COPD patients, we found enhanced incidence of CHIP mutations...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35805204"
                        open
                      >
                        <summary class="article-expander__title">Association of Clonal Hematopoiesis of Indeterminate Potential with Inflammatory Gene Expression in Patients with COPD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35805204/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic obstructive pulmonary disease (COPD) is a disease with an inflammatory phenotype with increasing prevalence in the elderly. Expanded population of mutant blood cells carrying somatic mutations is termed clonal hematopoiesis of indeterminate potential (CHIP). The association between CHIP and COPD and its relevant effects on DNA methylation in aging are mainly unknown. Analyzing the deep-targeted amplicon sequencing from 125 COPD patients, we found enhanced incidence of CHIP mutations...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35805204"
                        open
                      >
                        <summary class="article-expander__title">Association of Clonal Hematopoiesis of Indeterminate Potential with Inflammatory Gene Expression in Patients with COPD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35805204/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic obstructive pulmonary disease (COPD) is a disease with an inflammatory phenotype with increasing prevalence in the elderly. Expanded population of mutant blood cells carrying somatic mutations is termed clonal hematopoiesis of indeterminate potential (CHIP). The association between CHIP and COPD and its relevant effects on DNA methylation in aging are mainly unknown. Analyzing the deep-targeted amplicon sequencing from 125 COPD patients, we found enhanced incidence of CHIP mutations...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35805204"
                        open
                      >
                        <summary class="article-expander__title">Association of Clonal Hematopoiesis of Indeterminate Potential with Inflammatory Gene Expression in Patients with COPD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35805204/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic obstructive pulmonary disease (COPD) is a disease with an inflammatory phenotype with increasing prevalence in the elderly. Expanded population of mutant blood cells carrying somatic mutations is termed clonal hematopoiesis of indeterminate potential (CHIP). The association between CHIP and COPD and its relevant effects on DNA methylation in aging are mainly unknown. Analyzing the deep-targeted amplicon sequencing from 125 COPD patients, we found enhanced incidence of CHIP mutations...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35805204"
                        open
                      >
                        <summary class="article-expander__title">Association of Clonal Hematopoiesis of Indeterminate Potential with Inflammatory Gene Expression in Patients with COPD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35805204/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic obstructive pulmonary disease (COPD) is a disease with an inflammatory phenotype with increasing prevalence in the elderly. Expanded population of mutant blood cells carrying somatic mutations is termed clonal hematopoiesis of indeterminate potential (CHIP). The association between CHIP and COPD and its relevant effects on DNA methylation in aging are mainly unknown. Analyzing the deep-targeted amplicon sequencing from 125 COPD patients, we found enhanced incidence of CHIP mutations...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35805204"
                        open
                      >
                        <summary class="article-expander__title">Association of Clonal Hematopoiesis of Indeterminate Potential with Inflammatory Gene Expression in Patients with COPD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35805204/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic obstructive pulmonary disease (COPD) is a disease with an inflammatory phenotype with increasing prevalence in the elderly. Expanded population of mutant blood cells carrying somatic mutations is termed clonal hematopoiesis of indeterminate potential (CHIP). The association between CHIP and COPD and its relevant effects on DNA methylation in aging are mainly unknown. Analyzing the deep-targeted amplicon sequencing from 125 COPD patients, we found enhanced incidence of CHIP mutations...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-07-08 datetime="2022-07-08T10:00:00.000Z">2022-07-08</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-07-08 datetime="2022-07-08T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608003622&amp;v&#x3D;2.17.7&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35802878"
                        open
                      >
                        <summary class="article-expander__title">JAK inhibitors in the treatment of myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35802878/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220721211948&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is a myeloproliferative neoplasm driven by constitutive activation of the JAK/STAT pathway, resulting in clonal hematopoiesis, fibrotic replacement of the bone marrow, extramedullary hematopoiesis, splenomegaly, and debilitating constitutional symptoms. The advent of JAK inhibitors has changed the landscape of treatment options for patients with MF, providing relatively tolerable drug options that control symptoms, reduce splenomegaly, and improve quality of life, but often at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35802878"
                        open
                      >
                        <summary class="article-expander__title">JAK inhibitors in the treatment of myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35802878/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220720211338&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is a myeloproliferative neoplasm driven by constitutive activation of the JAK/STAT pathway, resulting in clonal hematopoiesis, fibrotic replacement of the bone marrow, extramedullary hematopoiesis, splenomegaly, and debilitating constitutional symptoms. The advent of JAK inhibitors has changed the landscape of treatment options for patients with MF, providing relatively tolerable drug options that control symptoms, reduce splenomegaly, and improve quality of life, but often at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35802878"
                        open
                      >
                        <summary class="article-expander__title">JAK inhibitors in the treatment of myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35802878/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220719211909&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is a myeloproliferative neoplasm driven by constitutive activation of the JAK/STAT pathway, resulting in clonal hematopoiesis, fibrotic replacement of the bone marrow, extramedullary hematopoiesis, splenomegaly, and debilitating constitutional symptoms. The advent of JAK inhibitors has changed the landscape of treatment options for patients with MF, providing relatively tolerable drug options that control symptoms, reduce splenomegaly, and improve quality of life, but often at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35802878"
                        open
                      >
                        <summary class="article-expander__title">JAK inhibitors in the treatment of myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35802878/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220718212208&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is a myeloproliferative neoplasm driven by constitutive activation of the JAK/STAT pathway, resulting in clonal hematopoiesis, fibrotic replacement of the bone marrow, extramedullary hematopoiesis, splenomegaly, and debilitating constitutional symptoms. The advent of JAK inhibitors has changed the landscape of treatment options for patients with MF, providing relatively tolerable drug options that control symptoms, reduce splenomegaly, and improve quality of life, but often at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35802878"
                        open
                      >
                        <summary class="article-expander__title">JAK inhibitors in the treatment of myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35802878/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220717211412&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is a myeloproliferative neoplasm driven by constitutive activation of the JAK/STAT pathway, resulting in clonal hematopoiesis, fibrotic replacement of the bone marrow, extramedullary hematopoiesis, splenomegaly, and debilitating constitutional symptoms. The advent of JAK inhibitors has changed the landscape of treatment options for patients with MF, providing relatively tolerable drug options that control symptoms, reduce splenomegaly, and improve quality of life, but often at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35802878"
                        open
                      >
                        <summary class="article-expander__title">JAK inhibitors in the treatment of myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35802878/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220716212224&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is a myeloproliferative neoplasm driven by constitutive activation of the JAK/STAT pathway, resulting in clonal hematopoiesis, fibrotic replacement of the bone marrow, extramedullary hematopoiesis, splenomegaly, and debilitating constitutional symptoms. The advent of JAK inhibitors has changed the landscape of treatment options for patients with MF, providing relatively tolerable drug options that control symptoms, reduce splenomegaly, and improve quality of life, but often at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35802878"
                        open
                      >
                        <summary class="article-expander__title">JAK inhibitors in the treatment of myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35802878/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220715211735&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is a myeloproliferative neoplasm driven by constitutive activation of the JAK/STAT pathway, resulting in clonal hematopoiesis, fibrotic replacement of the bone marrow, extramedullary hematopoiesis, splenomegaly, and debilitating constitutional symptoms. The advent of JAK inhibitors has changed the landscape of treatment options for patients with MF, providing relatively tolerable drug options that control symptoms, reduce splenomegaly, and improve quality of life, but often at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35802878"
                        open
                      >
                        <summary class="article-expander__title">JAK inhibitors in the treatment of myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35802878/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is a myeloproliferative neoplasm driven by constitutive activation of the JAK/STAT pathway, resulting in clonal hematopoiesis, fibrotic replacement of the bone marrow, extramedullary hematopoiesis, splenomegaly, and debilitating constitutional symptoms. The advent of JAK inhibitors has changed the landscape of treatment options for patients with MF, providing relatively tolerable drug options that control symptoms, reduce splenomegaly, and improve quality of life, but often at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35802878"
                        open
                      >
                        <summary class="article-expander__title">JAK inhibitors in the treatment of myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35802878/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is a myeloproliferative neoplasm driven by constitutive activation of the JAK/STAT pathway, resulting in clonal hematopoiesis, fibrotic replacement of the bone marrow, extramedullary hematopoiesis, splenomegaly, and debilitating constitutional symptoms. The advent of JAK inhibitors has changed the landscape of treatment options for patients with MF, providing relatively tolerable drug options that control symptoms, reduce splenomegaly, and improve quality of life, but often at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35802878"
                        open
                      >
                        <summary class="article-expander__title">JAK inhibitors in the treatment of myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35802878/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is a myeloproliferative neoplasm driven by constitutive activation of the JAK/STAT pathway, resulting in clonal hematopoiesis, fibrotic replacement of the bone marrow, extramedullary hematopoiesis, splenomegaly, and debilitating constitutional symptoms. The advent of JAK inhibitors has changed the landscape of treatment options for patients with MF, providing relatively tolerable drug options that control symptoms, reduce splenomegaly, and improve quality of life, but often at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35802878"
                        open
                      >
                        <summary class="article-expander__title">JAK inhibitors in the treatment of myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35802878/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is a myeloproliferative neoplasm driven by constitutive activation of the JAK/STAT pathway, resulting in clonal hematopoiesis, fibrotic replacement of the bone marrow, extramedullary hematopoiesis, splenomegaly, and debilitating constitutional symptoms. The advent of JAK inhibitors has changed the landscape of treatment options for patients with MF, providing relatively tolerable drug options that control symptoms, reduce splenomegaly, and improve quality of life, but often at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35802878"
                        open
                      >
                        <summary class="article-expander__title">JAK inhibitors in the treatment of myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35802878/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is a myeloproliferative neoplasm driven by constitutive activation of the JAK/STAT pathway, resulting in clonal hematopoiesis, fibrotic replacement of the bone marrow, extramedullary hematopoiesis, splenomegaly, and debilitating constitutional symptoms. The advent of JAK inhibitors has changed the landscape of treatment options for patients with MF, providing relatively tolerable drug options that control symptoms, reduce splenomegaly, and improve quality of life, but often at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35802878"
                        open
                      >
                        <summary class="article-expander__title">JAK inhibitors in the treatment of myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35802878/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is a myeloproliferative neoplasm driven by constitutive activation of the JAK/STAT pathway, resulting in clonal hematopoiesis, fibrotic replacement of the bone marrow, extramedullary hematopoiesis, splenomegaly, and debilitating constitutional symptoms. The advent of JAK inhibitors has changed the landscape of treatment options for patients with MF, providing relatively tolerable drug options that control symptoms, reduce splenomegaly, and improve quality of life, but often at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35802878"
                        open
                      >
                        <summary class="article-expander__title">JAK inhibitors in the treatment of myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35802878/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220708210406&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is a myeloproliferative neoplasm driven by constitutive activation of the JAK/STAT pathway, resulting in clonal hematopoiesis, fibrotic replacement of the bone marrow, extramedullary hematopoiesis, splenomegaly, and debilitating constitutional symptoms. The advent of JAK inhibitors has changed the landscape of treatment options for patients with MF, providing relatively tolerable drug options that control symptoms, reduce splenomegaly, and improve quality of life, but often at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608010801&amp;v&#x3D;2.17.7&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35803921"
                        open
                      >
                        <summary class="article-expander__title">Therapy-related clonal cytopenia as a precursor to therapy-related myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35803921/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220726212040&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related myeloid neoplasms (t-MN) are aggressive leukemia that develops as a complication of prior exposure to DNA-damaging agents. Clonal cytopenia of undetermined significance (CCUS) is a precursor of de novo myeloid neoplasms. Characteristics of CCUS that develop following cytotoxic therapies (therapy-related clonal cytopenia, t-CC) and outcomes following t-CC have not been described. We identified 33 patients with t-CC and compared to a cohort of the WHO-defined t-MN (n &#x3D; 309). t-CC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35803921"
                        open
                      >
                        <summary class="article-expander__title">Therapy-related clonal cytopenia as a precursor to therapy-related myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35803921/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220725211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related myeloid neoplasms (t-MN) are aggressive leukemia that develops as a complication of prior exposure to DNA-damaging agents. Clonal cytopenia of undetermined significance (CCUS) is a precursor of de novo myeloid neoplasms. Characteristics of CCUS that develop following cytotoxic therapies (therapy-related clonal cytopenia, t-CC) and outcomes following t-CC have not been described. We identified 33 patients with t-CC and compared to a cohort of the WHO-defined t-MN (n &#x3D; 309). t-CC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35803921"
                        open
                      >
                        <summary class="article-expander__title">Therapy-related clonal cytopenia as a precursor to therapy-related myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35803921/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220724211605&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related myeloid neoplasms (t-MN) are aggressive leukemia that develops as a complication of prior exposure to DNA-damaging agents. Clonal cytopenia of undetermined significance (CCUS) is a precursor of de novo myeloid neoplasms. Characteristics of CCUS that develop following cytotoxic therapies (therapy-related clonal cytopenia, t-CC) and outcomes following t-CC have not been described. We identified 33 patients with t-CC and compared to a cohort of the WHO-defined t-MN (n &#x3D; 309). t-CC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35803921"
                        open
                      >
                        <summary class="article-expander__title">Therapy-related clonal cytopenia as a precursor to therapy-related myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35803921/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220723212121&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related myeloid neoplasms (t-MN) are aggressive leukemia that develops as a complication of prior exposure to DNA-damaging agents. Clonal cytopenia of undetermined significance (CCUS) is a precursor of de novo myeloid neoplasms. Characteristics of CCUS that develop following cytotoxic therapies (therapy-related clonal cytopenia, t-CC) and outcomes following t-CC have not been described. We identified 33 patients with t-CC and compared to a cohort of the WHO-defined t-MN (n &#x3D; 309). t-CC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35803921"
                        open
                      >
                        <summary class="article-expander__title">Therapy-related clonal cytopenia as a precursor to therapy-related myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35803921/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220722211347&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related myeloid neoplasms (t-MN) are aggressive leukemia that develops as a complication of prior exposure to DNA-damaging agents. Clonal cytopenia of undetermined significance (CCUS) is a precursor of de novo myeloid neoplasms. Characteristics of CCUS that develop following cytotoxic therapies (therapy-related clonal cytopenia, t-CC) and outcomes following t-CC have not been described. We identified 33 patients with t-CC and compared to a cohort of the WHO-defined t-MN (n &#x3D; 309). t-CC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35803921"
                        open
                      >
                        <summary class="article-expander__title">Therapy-related clonal cytopenia as a precursor to therapy-related myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35803921/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220721211948&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related myeloid neoplasms (t-MN) are aggressive leukemia that develops as a complication of prior exposure to DNA-damaging agents. Clonal cytopenia of undetermined significance (CCUS) is a precursor of de novo myeloid neoplasms. Characteristics of CCUS that develop following cytotoxic therapies (therapy-related clonal cytopenia, t-CC) and outcomes following t-CC have not been described. We identified 33 patients with t-CC and compared to a cohort of the WHO-defined t-MN (n &#x3D; 309). t-CC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35803921"
                        open
                      >
                        <summary class="article-expander__title">Therapy-related clonal cytopenia as a precursor to therapy-related myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35803921/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220720211338&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related myeloid neoplasms (t-MN) are aggressive leukemia that develops as a complication of prior exposure to DNA-damaging agents. Clonal cytopenia of undetermined significance (CCUS) is a precursor of de novo myeloid neoplasms. Characteristics of CCUS that develop following cytotoxic therapies (therapy-related clonal cytopenia, t-CC) and outcomes following t-CC have not been described. We identified 33 patients with t-CC and compared to a cohort of the WHO-defined t-MN (n &#x3D; 309). t-CC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35803921"
                        open
                      >
                        <summary class="article-expander__title">Therapy-related clonal cytopenia as a precursor to therapy-related myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35803921/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220719211909&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related myeloid neoplasms (t-MN) are aggressive leukemia that develops as a complication of prior exposure to DNA-damaging agents. Clonal cytopenia of undetermined significance (CCUS) is a precursor of de novo myeloid neoplasms. Characteristics of CCUS that develop following cytotoxic therapies (therapy-related clonal cytopenia, t-CC) and outcomes following t-CC have not been described. We identified 33 patients with t-CC and compared to a cohort of the WHO-defined t-MN (n &#x3D; 309). t-CC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35803921"
                        open
                      >
                        <summary class="article-expander__title">Therapy-related clonal cytopenia as a precursor to therapy-related myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35803921/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220718212208&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related myeloid neoplasms (t-MN) are aggressive leukemia that develops as a complication of prior exposure to DNA-damaging agents. Clonal cytopenia of undetermined significance (CCUS) is a precursor of de novo myeloid neoplasms. Characteristics of CCUS that develop following cytotoxic therapies (therapy-related clonal cytopenia, t-CC) and outcomes following t-CC have not been described. We identified 33 patients with t-CC and compared to a cohort of the WHO-defined t-MN (n &#x3D; 309). t-CC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35803921"
                        open
                      >
                        <summary class="article-expander__title">Therapy-related clonal cytopenia as a precursor to therapy-related myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35803921/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220717211412&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related myeloid neoplasms (t-MN) are aggressive leukemia that develops as a complication of prior exposure to DNA-damaging agents. Clonal cytopenia of undetermined significance (CCUS) is a precursor of de novo myeloid neoplasms. Characteristics of CCUS that develop following cytotoxic therapies (therapy-related clonal cytopenia, t-CC) and outcomes following t-CC have not been described. We identified 33 patients with t-CC and compared to a cohort of the WHO-defined t-MN (n &#x3D; 309). t-CC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35803921"
                        open
                      >
                        <summary class="article-expander__title">Therapy-related clonal cytopenia as a precursor to therapy-related myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35803921/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220716212224&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related myeloid neoplasms (t-MN) are aggressive leukemia that develops as a complication of prior exposure to DNA-damaging agents. Clonal cytopenia of undetermined significance (CCUS) is a precursor of de novo myeloid neoplasms. Characteristics of CCUS that develop following cytotoxic therapies (therapy-related clonal cytopenia, t-CC) and outcomes following t-CC have not been described. We identified 33 patients with t-CC and compared to a cohort of the WHO-defined t-MN (n &#x3D; 309). t-CC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35803921"
                        open
                      >
                        <summary class="article-expander__title">Therapy-related clonal cytopenia as a precursor to therapy-related myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35803921/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220715211735&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related myeloid neoplasms (t-MN) are aggressive leukemia that develops as a complication of prior exposure to DNA-damaging agents. Clonal cytopenia of undetermined significance (CCUS) is a precursor of de novo myeloid neoplasms. Characteristics of CCUS that develop following cytotoxic therapies (therapy-related clonal cytopenia, t-CC) and outcomes following t-CC have not been described. We identified 33 patients with t-CC and compared to a cohort of the WHO-defined t-MN (n &#x3D; 309). t-CC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35803921"
                        open
                      >
                        <summary class="article-expander__title">Therapy-related clonal cytopenia as a precursor to therapy-related myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35803921/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related myeloid neoplasms (t-MN) are aggressive leukemia that develops as a complication of prior exposure to DNA-damaging agents. Clonal cytopenia of undetermined significance (CCUS) is a precursor of de novo myeloid neoplasms. Characteristics of CCUS that develop following cytotoxic therapies (therapy-related clonal cytopenia, t-CC) and outcomes following t-CC have not been described. We identified 33 patients with t-CC and compared to a cohort of the WHO-defined t-MN (n &#x3D; 309). t-CC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35803921"
                        open
                      >
                        <summary class="article-expander__title">Therapy-related clonal cytopenia as a precursor to therapy-related myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35803921/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related myeloid neoplasms (t-MN) are aggressive leukemia that develops as a complication of prior exposure to DNA-damaging agents. Clonal cytopenia of undetermined significance (CCUS) is a precursor of de novo myeloid neoplasms. Characteristics of CCUS that develop following cytotoxic therapies (therapy-related clonal cytopenia, t-CC) and outcomes following t-CC have not been described. We identified 33 patients with t-CC and compared to a cohort of the WHO-defined t-MN (n &#x3D; 309). t-CC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35803921"
                        open
                      >
                        <summary class="article-expander__title">Therapy-related clonal cytopenia as a precursor to therapy-related myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35803921/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related myeloid neoplasms (t-MN) are aggressive leukemia that develops as a complication of prior exposure to DNA-damaging agents. Clonal cytopenia of undetermined significance (CCUS) is a precursor of de novo myeloid neoplasms. Characteristics of CCUS that develop following cytotoxic therapies (therapy-related clonal cytopenia, t-CC) and outcomes following t-CC have not been described. We identified 33 patients with t-CC and compared to a cohort of the WHO-defined t-MN (n &#x3D; 309). t-CC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35803921"
                        open
                      >
                        <summary class="article-expander__title">Therapy-related clonal cytopenia as a precursor to therapy-related myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35803921/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related myeloid neoplasms (t-MN) are aggressive leukemia that develops as a complication of prior exposure to DNA-damaging agents. Clonal cytopenia of undetermined significance (CCUS) is a precursor of de novo myeloid neoplasms. Characteristics of CCUS that develop following cytotoxic therapies (therapy-related clonal cytopenia, t-CC) and outcomes following t-CC have not been described. We identified 33 patients with t-CC and compared to a cohort of the WHO-defined t-MN (n &#x3D; 309). t-CC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35803921"
                        open
                      >
                        <summary class="article-expander__title">Therapy-related clonal cytopenia as a precursor to therapy-related myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35803921/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related myeloid neoplasms (t-MN) are aggressive leukemia that develops as a complication of prior exposure to DNA-damaging agents. Clonal cytopenia of undetermined significance (CCUS) is a precursor of de novo myeloid neoplasms. Characteristics of CCUS that develop following cytotoxic therapies (therapy-related clonal cytopenia, t-CC) and outcomes following t-CC have not been described. We identified 33 patients with t-CC and compared to a cohort of the WHO-defined t-MN (n &#x3D; 309). t-CC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35803921"
                        open
                      >
                        <summary class="article-expander__title">Therapy-related clonal cytopenia as a precursor to therapy-related myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35803921/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related myeloid neoplasms (t-MN) are aggressive leukemia that develops as a complication of prior exposure to DNA-damaging agents. Clonal cytopenia of undetermined significance (CCUS) is a precursor of de novo myeloid neoplasms. Characteristics of CCUS that develop following cytotoxic therapies (therapy-related clonal cytopenia, t-CC) and outcomes following t-CC have not been described. We identified 33 patients with t-CC and compared to a cohort of the WHO-defined t-MN (n &#x3D; 309). t-CC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-07-07 datetime="2022-07-07T10:00:00.000Z">2022-07-07</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-07-07 datetime="2022-07-07T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608003622&amp;v&#x3D;2.17.7&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35797508"
                        open
                      >
                        <summary class="article-expander__title">Matched Molecular Profiling of Cell-Free DNA and Tumor Tissue in Patients With Advanced Clear Cell Renal Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35797508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220720211338&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: cfDNA-based NGS profiling yielded low detection rates in this metastatic ccRCC cohort. Concordance with tumor profiling was low, even for truncal mutations such as VHL, and some findings in peripheral blood may represent clonal hematopoiesis. Routine cfDNA panel testing is not supported, and its application in biomarker efforts must account for these limitations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35797508"
                        open
                      >
                        <summary class="article-expander__title">Matched Molecular Profiling of Cell-Free DNA and Tumor Tissue in Patients With Advanced Clear Cell Renal Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35797508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220719211909&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: cfDNA-based NGS profiling yielded low detection rates in this metastatic ccRCC cohort. Concordance with tumor profiling was low, even for truncal mutations such as VHL, and some findings in peripheral blood may represent clonal hematopoiesis. Routine cfDNA panel testing is not supported, and its application in biomarker efforts must account for these limitations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35795468"
                        open
                      >
                        <summary class="article-expander__title">CSF3R T618I mutant myelodysplastic/myeloproliferative neoplasm in the elderly: An age-related disease with unfavorable prognosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35795468/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220719211909&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We reported two MDS/MPN-U and one CMML patients with CSF3R T618I mutation. There were two males and one female, with a median age of 84 years. Three patients presented with leukocytosis and anemia, two with thrombocytopenia, and one with monocytosis. In all the patients, bone marrow showed hypercellularity with myeloid or erythroid predominant trilineage hematopoiesis and dysplasia. Two cases carried -7/-7q abnormalities. In addition to CSF3R T618/I mutation, each case carried 3-5 additional...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35797508"
                        open
                      >
                        <summary class="article-expander__title">Matched Molecular Profiling of Cell-Free DNA and Tumor Tissue in Patients With Advanced Clear Cell Renal Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35797508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220718212208&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: cfDNA-based NGS profiling yielded low detection rates in this metastatic ccRCC cohort. Concordance with tumor profiling was low, even for truncal mutations such as VHL, and some findings in peripheral blood may represent clonal hematopoiesis. Routine cfDNA panel testing is not supported, and its application in biomarker efforts must account for these limitations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35795468"
                        open
                      >
                        <summary class="article-expander__title">CSF3R T618I mutant myelodysplastic/myeloproliferative neoplasm in the elderly: An age-related disease with unfavorable prognosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35795468/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220718212208&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We reported two MDS/MPN-U and one CMML patients with CSF3R T618I mutation. There were two males and one female, with a median age of 84 years. Three patients presented with leukocytosis and anemia, two with thrombocytopenia, and one with monocytosis. In all the patients, bone marrow showed hypercellularity with myeloid or erythroid predominant trilineage hematopoiesis and dysplasia. Two cases carried -7/-7q abnormalities. In addition to CSF3R T618/I mutation, each case carried 3-5 additional...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35797508"
                        open
                      >
                        <summary class="article-expander__title">Matched Molecular Profiling of Cell-Free DNA and Tumor Tissue in Patients With Advanced Clear Cell Renal Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35797508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220717211412&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: cfDNA-based NGS profiling yielded low detection rates in this metastatic ccRCC cohort. Concordance with tumor profiling was low, even for truncal mutations such as VHL, and some findings in peripheral blood may represent clonal hematopoiesis. Routine cfDNA panel testing is not supported, and its application in biomarker efforts must account for these limitations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35795468"
                        open
                      >
                        <summary class="article-expander__title">CSF3R T618I mutant myelodysplastic/myeloproliferative neoplasm in the elderly: An age-related disease with unfavorable prognosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35795468/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220717211412&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We reported two MDS/MPN-U and one CMML patients with CSF3R T618I mutation. There were two males and one female, with a median age of 84 years. Three patients presented with leukocytosis and anemia, two with thrombocytopenia, and one with monocytosis. In all the patients, bone marrow showed hypercellularity with myeloid or erythroid predominant trilineage hematopoiesis and dysplasia. Two cases carried -7/-7q abnormalities. In addition to CSF3R T618/I mutation, each case carried 3-5 additional...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35797508"
                        open
                      >
                        <summary class="article-expander__title">Matched Molecular Profiling of Cell-Free DNA and Tumor Tissue in Patients With Advanced Clear Cell Renal Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35797508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220716212224&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: cfDNA-based NGS profiling yielded low detection rates in this metastatic ccRCC cohort. Concordance with tumor profiling was low, even for truncal mutations such as VHL, and some findings in peripheral blood may represent clonal hematopoiesis. Routine cfDNA panel testing is not supported, and its application in biomarker efforts must account for these limitations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35795468"
                        open
                      >
                        <summary class="article-expander__title">CSF3R T618I mutant myelodysplastic/myeloproliferative neoplasm in the elderly: An age-related disease with unfavorable prognosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35795468/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220716212224&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We reported two MDS/MPN-U and one CMML patients with CSF3R T618I mutation. There were two males and one female, with a median age of 84 years. Three patients presented with leukocytosis and anemia, two with thrombocytopenia, and one with monocytosis. In all the patients, bone marrow showed hypercellularity with myeloid or erythroid predominant trilineage hematopoiesis and dysplasia. Two cases carried -7/-7q abnormalities. In addition to CSF3R T618/I mutation, each case carried 3-5 additional...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35797508"
                        open
                      >
                        <summary class="article-expander__title">Matched Molecular Profiling of Cell-Free DNA and Tumor Tissue in Patients With Advanced Clear Cell Renal Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35797508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220715211735&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: cfDNA-based NGS profiling yielded low detection rates in this metastatic ccRCC cohort. Concordance with tumor profiling was low, even for truncal mutations such as VHL, and some findings in peripheral blood may represent clonal hematopoiesis. Routine cfDNA panel testing is not supported, and its application in biomarker efforts must account for these limitations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35795468"
                        open
                      >
                        <summary class="article-expander__title">CSF3R T618I mutant myelodysplastic/myeloproliferative neoplasm in the elderly: An age-related disease with unfavorable prognosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35795468/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220715211735&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We reported two MDS/MPN-U and one CMML patients with CSF3R T618I mutation. There were two males and one female, with a median age of 84 years. Three patients presented with leukocytosis and anemia, two with thrombocytopenia, and one with monocytosis. In all the patients, bone marrow showed hypercellularity with myeloid or erythroid predominant trilineage hematopoiesis and dysplasia. Two cases carried -7/-7q abnormalities. In addition to CSF3R T618/I mutation, each case carried 3-5 additional...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35797508"
                        open
                      >
                        <summary class="article-expander__title">Matched Molecular Profiling of Cell-Free DNA and Tumor Tissue in Patients With Advanced Clear Cell Renal Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35797508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: cfDNA-based NGS profiling yielded low detection rates in this metastatic ccRCC cohort. Concordance with tumor profiling was low, even for truncal mutations such as VHL, and some findings in peripheral blood may represent clonal hematopoiesis. Routine cfDNA panel testing is not supported, and its application in biomarker efforts must account for these limitations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35795468"
                        open
                      >
                        <summary class="article-expander__title">CSF3R T618I mutant myelodysplastic/myeloproliferative neoplasm in the elderly: An age-related disease with unfavorable prognosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35795468/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We reported two MDS/MPN-U and one CMML patients with CSF3R T618I mutation. There were two males and one female, with a median age of 84 years. Three patients presented with leukocytosis and anemia, two with thrombocytopenia, and one with monocytosis. In all the patients, bone marrow showed hypercellularity with myeloid or erythroid predominant trilineage hematopoiesis and dysplasia. Two cases carried -7/-7q abnormalities. In addition to CSF3R T618/I mutation, each case carried 3-5 additional...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35797508"
                        open
                      >
                        <summary class="article-expander__title">Matched Molecular Profiling of Cell-Free DNA and Tumor Tissue in Patients With Advanced Clear Cell Renal Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35797508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: cfDNA-based NGS profiling yielded low detection rates in this metastatic ccRCC cohort. Concordance with tumor profiling was low, even for truncal mutations such as VHL, and some findings in peripheral blood may represent clonal hematopoiesis. Routine cfDNA panel testing is not supported, and its application in biomarker efforts must account for these limitations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35795468"
                        open
                      >
                        <summary class="article-expander__title">CSF3R T618I mutant myelodysplastic/myeloproliferative neoplasm in the elderly: An age-related disease with unfavorable prognosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35795468/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We reported two MDS/MPN-U and one CMML patients with CSF3R T618I mutation. There were two males and one female, with a median age of 84 years. Three patients presented with leukocytosis and anemia, two with thrombocytopenia, and one with monocytosis. In all the patients, bone marrow showed hypercellularity with myeloid or erythroid predominant trilineage hematopoiesis and dysplasia. Two cases carried -7/-7q abnormalities. In addition to CSF3R T618/I mutation, each case carried 3-5 additional...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35797508"
                        open
                      >
                        <summary class="article-expander__title">Matched Molecular Profiling of Cell-Free DNA and Tumor Tissue in Patients With Advanced Clear Cell Renal Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35797508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: cfDNA-based NGS profiling yielded low detection rates in this metastatic ccRCC cohort. Concordance with tumor profiling was low, even for truncal mutations such as VHL, and some findings in peripheral blood may represent clonal hematopoiesis. Routine cfDNA panel testing is not supported, and its application in biomarker efforts must account for these limitations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35795468"
                        open
                      >
                        <summary class="article-expander__title">CSF3R T618I mutant myelodysplastic/myeloproliferative neoplasm in the elderly: An age-related disease with unfavorable prognosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35795468/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We reported two MDS/MPN-U and one CMML patients with CSF3R T618I mutation. There were two males and one female, with a median age of 84 years. Three patients presented with leukocytosis and anemia, two with thrombocytopenia, and one with monocytosis. In all the patients, bone marrow showed hypercellularity with myeloid or erythroid predominant trilineage hematopoiesis and dysplasia. Two cases carried -7/-7q abnormalities. In addition to CSF3R T618/I mutation, each case carried 3-5 additional...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35797508"
                        open
                      >
                        <summary class="article-expander__title">Matched Molecular Profiling of Cell-Free DNA and Tumor Tissue in Patients With Advanced Clear Cell Renal Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35797508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: cfDNA-based NGS profiling yielded low detection rates in this metastatic ccRCC cohort. Concordance with tumor profiling was low, even for truncal mutations such as VHL, and some findings in peripheral blood may represent clonal hematopoiesis. Routine cfDNA panel testing is not supported, and its application in biomarker efforts must account for these limitations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35795468"
                        open
                      >
                        <summary class="article-expander__title">CSF3R T618I mutant myelodysplastic/myeloproliferative neoplasm in the elderly: An age-related disease with unfavorable prognosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35795468/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We reported two MDS/MPN-U and one CMML patients with CSF3R T618I mutation. There were two males and one female, with a median age of 84 years. Three patients presented with leukocytosis and anemia, two with thrombocytopenia, and one with monocytosis. In all the patients, bone marrow showed hypercellularity with myeloid or erythroid predominant trilineage hematopoiesis and dysplasia. Two cases carried -7/-7q abnormalities. In addition to CSF3R T618/I mutation, each case carried 3-5 additional...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35797508"
                        open
                      >
                        <summary class="article-expander__title">Matched Molecular Profiling of Cell-Free DNA and Tumor Tissue in Patients With Advanced Clear Cell Renal Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35797508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: cfDNA-based NGS profiling yielded low detection rates in this metastatic ccRCC cohort. Concordance with tumor profiling was low, even for truncal mutations such as VHL, and some findings in peripheral blood may represent clonal hematopoiesis. Routine cfDNA panel testing is not supported, and its application in biomarker efforts must account for these limitations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35795468"
                        open
                      >
                        <summary class="article-expander__title">CSF3R T618I mutant myelodysplastic/myeloproliferative neoplasm in the elderly: An age-related disease with unfavorable prognosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35795468/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We reported two MDS/MPN-U and one CMML patients with CSF3R T618I mutation. There were two males and one female, with a median age of 84 years. Three patients presented with leukocytosis and anemia, two with thrombocytopenia, and one with monocytosis. In all the patients, bone marrow showed hypercellularity with myeloid or erythroid predominant trilineage hematopoiesis and dysplasia. Two cases carried -7/-7q abnormalities. In addition to CSF3R T618/I mutation, each case carried 3-5 additional...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35797508"
                        open
                      >
                        <summary class="article-expander__title">Matched Molecular Profiling of Cell-Free DNA and Tumor Tissue in Patients With Advanced Clear Cell Renal Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35797508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: cfDNA-based NGS profiling yielded low detection rates in this metastatic ccRCC cohort. Concordance with tumor profiling was low, even for truncal mutations such as VHL, and some findings in peripheral blood may represent clonal hematopoiesis. Routine cfDNA panel testing is not supported, and its application in biomarker efforts must account for these limitations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35795468"
                        open
                      >
                        <summary class="article-expander__title">CSF3R T618I mutant myelodysplastic/myeloproliferative neoplasm in the elderly: An age-related disease with unfavorable prognosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35795468/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We reported two MDS/MPN-U and one CMML patients with CSF3R T618I mutation. There were two males and one female, with a median age of 84 years. Three patients presented with leukocytosis and anemia, two with thrombocytopenia, and one with monocytosis. In all the patients, bone marrow showed hypercellularity with myeloid or erythroid predominant trilineage hematopoiesis and dysplasia. Two cases carried -7/-7q abnormalities. In addition to CSF3R T618/I mutation, each case carried 3-5 additional...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35797508"
                        open
                      >
                        <summary class="article-expander__title">Matched Molecular Profiling of Cell-Free DNA and Tumor Tissue in Patients With Advanced Clear Cell Renal Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35797508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220708210406&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: cfDNA-based NGS profiling yielded low detection rates in this metastatic ccRCC cohort. Concordance with tumor profiling was low, even for truncal mutations such as VHL, and some findings in peripheral blood may represent clonal hematopoiesis. Routine cfDNA panel testing is not supported, and its application in biomarker efforts must account for these limitations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35795468"
                        open
                      >
                        <summary class="article-expander__title">CSF3R T618I mutant myelodysplastic/myeloproliferative neoplasm in the elderly: An age-related disease with unfavorable prognosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35795468/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220708210406&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We reported two MDS/MPN-U and one CMML patients with CSF3R T618I mutation. There were two males and one female, with a median age of 84 years. Three patients presented with leukocytosis and anemia, two with thrombocytopenia, and one with monocytosis. In all the patients, bone marrow showed hypercellularity with myeloid or erythroid predominant trilineage hematopoiesis and dysplasia. Two cases carried -7/-7q abnormalities. In addition to CSF3R T618/I mutation, each case carried 3-5 additional...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35797508"
                        open
                      >
                        <summary class="article-expander__title">Matched Molecular Profiling of Cell-Free DNA and Tumor Tissue in Patients With Advanced Clear Cell Renal Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35797508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220707211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: cfDNA-based NGS profiling yielded low detection rates in this metastatic ccRCC cohort. Concordance with tumor profiling was low, even for truncal mutations such as VHL, and some findings in peripheral blood may represent clonal hematopoiesis. Routine cfDNA panel testing is not supported, and its application in biomarker efforts must account for these limitations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35795468"
                        open
                      >
                        <summary class="article-expander__title">CSF3R T618I mutant myelodysplastic/myeloproliferative neoplasm in the elderly: An age-related disease with unfavorable prognosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35795468/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220707211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We reported two MDS/MPN-U and one CMML patients with CSF3R T618I mutation. There were two males and one female, with a median age of 84 years. Three patients presented with leukocytosis and anemia, two with thrombocytopenia, and one with monocytosis. In all the patients, bone marrow showed hypercellularity with myeloid or erythroid predominant trilineage hematopoiesis and dysplasia. Two cases carried -7/-7q abnormalities. In addition to CSF3R T618/I mutation, each case carried 3-5 additional...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608010801&amp;v&#x3D;2.17.7&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35797509"
                        open
                      >
                        <summary class="article-expander__title">Multidimensional Molecular Profiling of Metastatic Triple-Negative Breast Cancer and Immune Checkpoint Inhibitor Benefit</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35797509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220726212040&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study identified potential biomarkers of response to ICIs among patients with mTNBC: high tumor mutational burden; presence of CD8+, CD4 memory T cells, follicular helper T cells, and M1 macrophages; and inflammatory gene expression pathways. Pretreatment deficiencies in the homologous recombination DNA damage repair pathway and the absence of or minimal clonal evolution post-treatment may be associated with worse outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35797509"
                        open
                      >
                        <summary class="article-expander__title">Multidimensional Molecular Profiling of Metastatic Triple-Negative Breast Cancer and Immune Checkpoint Inhibitor Benefit</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35797509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220725211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study identified potential biomarkers of response to ICIs among patients with mTNBC: high tumor mutational burden; presence of CD8+, CD4 memory T cells, follicular helper T cells, and M1 macrophages; and inflammatory gene expression pathways. Pretreatment deficiencies in the homologous recombination DNA damage repair pathway and the absence of or minimal clonal evolution post-treatment may be associated with worse outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35797509"
                        open
                      >
                        <summary class="article-expander__title">Multidimensional Molecular Profiling of Metastatic Triple-Negative Breast Cancer and Immune Checkpoint Inhibitor Benefit</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35797509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220724211605&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study identified potential biomarkers of response to ICIs among patients with mTNBC: high tumor mutational burden; presence of CD8+, CD4 memory T cells, follicular helper T cells, and M1 macrophages; and inflammatory gene expression pathways. Pretreatment deficiencies in the homologous recombination DNA damage repair pathway and the absence of or minimal clonal evolution post-treatment may be associated with worse outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35797509"
                        open
                      >
                        <summary class="article-expander__title">Multidimensional Molecular Profiling of Metastatic Triple-Negative Breast Cancer and Immune Checkpoint Inhibitor Benefit</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35797509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220723212121&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study identified potential biomarkers of response to ICIs among patients with mTNBC: high tumor mutational burden; presence of CD8+, CD4 memory T cells, follicular helper T cells, and M1 macrophages; and inflammatory gene expression pathways. Pretreatment deficiencies in the homologous recombination DNA damage repair pathway and the absence of or minimal clonal evolution post-treatment may be associated with worse outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35797509"
                        open
                      >
                        <summary class="article-expander__title">Multidimensional Molecular Profiling of Metastatic Triple-Negative Breast Cancer and Immune Checkpoint Inhibitor Benefit</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35797509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220722211347&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study identified potential biomarkers of response to ICIs among patients with mTNBC: high tumor mutational burden; presence of CD8+, CD4 memory T cells, follicular helper T cells, and M1 macrophages; and inflammatory gene expression pathways. Pretreatment deficiencies in the homologous recombination DNA damage repair pathway and the absence of or minimal clonal evolution post-treatment may be associated with worse outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35797509"
                        open
                      >
                        <summary class="article-expander__title">Multidimensional Molecular Profiling of Metastatic Triple-Negative Breast Cancer and Immune Checkpoint Inhibitor Benefit</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35797509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220721211948&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study identified potential biomarkers of response to ICIs among patients with mTNBC: high tumor mutational burden; presence of CD8+, CD4 memory T cells, follicular helper T cells, and M1 macrophages; and inflammatory gene expression pathways. Pretreatment deficiencies in the homologous recombination DNA damage repair pathway and the absence of or minimal clonal evolution post-treatment may be associated with worse outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35797509"
                        open
                      >
                        <summary class="article-expander__title">Multidimensional Molecular Profiling of Metastatic Triple-Negative Breast Cancer and Immune Checkpoint Inhibitor Benefit</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35797509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220720211338&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study identified potential biomarkers of response to ICIs among patients with mTNBC: high tumor mutational burden; presence of CD8+, CD4 memory T cells, follicular helper T cells, and M1 macrophages; and inflammatory gene expression pathways. Pretreatment deficiencies in the homologous recombination DNA damage repair pathway and the absence of or minimal clonal evolution post-treatment may be associated with worse outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35797509"
                        open
                      >
                        <summary class="article-expander__title">Multidimensional Molecular Profiling of Metastatic Triple-Negative Breast Cancer and Immune Checkpoint Inhibitor Benefit</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35797509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220719211909&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study identified potential biomarkers of response to ICIs among patients with mTNBC: high tumor mutational burden; presence of CD8+, CD4 memory T cells, follicular helper T cells, and M1 macrophages; and inflammatory gene expression pathways. Pretreatment deficiencies in the homologous recombination DNA damage repair pathway and the absence of or minimal clonal evolution post-treatment may be associated with worse outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35797509"
                        open
                      >
                        <summary class="article-expander__title">Multidimensional Molecular Profiling of Metastatic Triple-Negative Breast Cancer and Immune Checkpoint Inhibitor Benefit</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35797509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220718212208&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study identified potential biomarkers of response to ICIs among patients with mTNBC: high tumor mutational burden; presence of CD8+, CD4 memory T cells, follicular helper T cells, and M1 macrophages; and inflammatory gene expression pathways. Pretreatment deficiencies in the homologous recombination DNA damage repair pathway and the absence of or minimal clonal evolution post-treatment may be associated with worse outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35797509"
                        open
                      >
                        <summary class="article-expander__title">Multidimensional Molecular Profiling of Metastatic Triple-Negative Breast Cancer and Immune Checkpoint Inhibitor Benefit</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35797509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220717211412&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study identified potential biomarkers of response to ICIs among patients with mTNBC: high tumor mutational burden; presence of CD8+, CD4 memory T cells, follicular helper T cells, and M1 macrophages; and inflammatory gene expression pathways. Pretreatment deficiencies in the homologous recombination DNA damage repair pathway and the absence of or minimal clonal evolution post-treatment may be associated with worse outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35797509"
                        open
                      >
                        <summary class="article-expander__title">Multidimensional Molecular Profiling of Metastatic Triple-Negative Breast Cancer and Immune Checkpoint Inhibitor Benefit</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35797509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220716212224&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study identified potential biomarkers of response to ICIs among patients with mTNBC: high tumor mutational burden; presence of CD8+, CD4 memory T cells, follicular helper T cells, and M1 macrophages; and inflammatory gene expression pathways. Pretreatment deficiencies in the homologous recombination DNA damage repair pathway and the absence of or minimal clonal evolution post-treatment may be associated with worse outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35797509"
                        open
                      >
                        <summary class="article-expander__title">Multidimensional Molecular Profiling of Metastatic Triple-Negative Breast Cancer and Immune Checkpoint Inhibitor Benefit</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35797509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220715211735&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study identified potential biomarkers of response to ICIs among patients with mTNBC: high tumor mutational burden; presence of CD8+, CD4 memory T cells, follicular helper T cells, and M1 macrophages; and inflammatory gene expression pathways. Pretreatment deficiencies in the homologous recombination DNA damage repair pathway and the absence of or minimal clonal evolution post-treatment may be associated with worse outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35797509"
                        open
                      >
                        <summary class="article-expander__title">Multidimensional Molecular Profiling of Metastatic Triple-Negative Breast Cancer and Immune Checkpoint Inhibitor Benefit</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35797509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study identified potential biomarkers of response to ICIs among patients with mTNBC: high tumor mutational burden; presence of CD8+, CD4 memory T cells, follicular helper T cells, and M1 macrophages; and inflammatory gene expression pathways. Pretreatment deficiencies in the homologous recombination DNA damage repair pathway and the absence of or minimal clonal evolution post-treatment may be associated with worse outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35797509"
                        open
                      >
                        <summary class="article-expander__title">Multidimensional Molecular Profiling of Metastatic Triple-Negative Breast Cancer and Immune Checkpoint Inhibitor Benefit</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35797509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study identified potential biomarkers of response to ICIs among patients with mTNBC: high tumor mutational burden; presence of CD8+, CD4 memory T cells, follicular helper T cells, and M1 macrophages; and inflammatory gene expression pathways. Pretreatment deficiencies in the homologous recombination DNA damage repair pathway and the absence of or minimal clonal evolution post-treatment may be associated with worse outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35797509"
                        open
                      >
                        <summary class="article-expander__title">Multidimensional Molecular Profiling of Metastatic Triple-Negative Breast Cancer and Immune Checkpoint Inhibitor Benefit</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35797509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study identified potential biomarkers of response to ICIs among patients with mTNBC: high tumor mutational burden; presence of CD8+, CD4 memory T cells, follicular helper T cells, and M1 macrophages; and inflammatory gene expression pathways. Pretreatment deficiencies in the homologous recombination DNA damage repair pathway and the absence of or minimal clonal evolution post-treatment may be associated with worse outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35797509"
                        open
                      >
                        <summary class="article-expander__title">Multidimensional Molecular Profiling of Metastatic Triple-Negative Breast Cancer and Immune Checkpoint Inhibitor Benefit</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35797509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study identified potential biomarkers of response to ICIs among patients with mTNBC: high tumor mutational burden; presence of CD8+, CD4 memory T cells, follicular helper T cells, and M1 macrophages; and inflammatory gene expression pathways. Pretreatment deficiencies in the homologous recombination DNA damage repair pathway and the absence of or minimal clonal evolution post-treatment may be associated with worse outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35797509"
                        open
                      >
                        <summary class="article-expander__title">Multidimensional Molecular Profiling of Metastatic Triple-Negative Breast Cancer and Immune Checkpoint Inhibitor Benefit</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35797509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study identified potential biomarkers of response to ICIs among patients with mTNBC: high tumor mutational burden; presence of CD8+, CD4 memory T cells, follicular helper T cells, and M1 macrophages; and inflammatory gene expression pathways. Pretreatment deficiencies in the homologous recombination DNA damage repair pathway and the absence of or minimal clonal evolution post-treatment may be associated with worse outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35797509"
                        open
                      >
                        <summary class="article-expander__title">Multidimensional Molecular Profiling of Metastatic Triple-Negative Breast Cancer and Immune Checkpoint Inhibitor Benefit</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35797509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study identified potential biomarkers of response to ICIs among patients with mTNBC: high tumor mutational burden; presence of CD8+, CD4 memory T cells, follicular helper T cells, and M1 macrophages; and inflammatory gene expression pathways. Pretreatment deficiencies in the homologous recombination DNA damage repair pathway and the absence of or minimal clonal evolution post-treatment may be associated with worse outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35797509"
                        open
                      >
                        <summary class="article-expander__title">Multidimensional Molecular Profiling of Metastatic Triple-Negative Breast Cancer and Immune Checkpoint Inhibitor Benefit</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35797509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220708210406&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study identified potential biomarkers of response to ICIs among patients with mTNBC: high tumor mutational burden; presence of CD8+, CD4 memory T cells, follicular helper T cells, and M1 macrophages; and inflammatory gene expression pathways. Pretreatment deficiencies in the homologous recombination DNA damage repair pathway and the absence of or minimal clonal evolution post-treatment may be associated with worse outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35797509"
                        open
                      >
                        <summary class="article-expander__title">Multidimensional Molecular Profiling of Metastatic Triple-Negative Breast Cancer and Immune Checkpoint Inhibitor Benefit</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35797509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220707211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study identified potential biomarkers of response to ICIs among patients with mTNBC: high tumor mutational burden; presence of CD8+, CD4 memory T cells, follicular helper T cells, and M1 macrophages; and inflammatory gene expression pathways. Pretreatment deficiencies in the homologous recombination DNA damage repair pathway and the absence of or minimal clonal evolution post-treatment may be associated with worse outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-07-05 datetime="2022-07-05T10:00:00.000Z">2022-07-05</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-07-05 datetime="2022-07-05T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608003622&amp;v&#x3D;2.17.7&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788175"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal dynamics of clonal hematopoiesis identifies gene-specific fitness effects</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788175/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220719211909&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) increases rapidly in prevalence beyond age 60 and has been associated with increased risk for malignancy, heart disease and ischemic stroke. CHIP is driven by somatic mutations in hematopoietic stem and progenitor cells (HSPCs). Because mutations in HSPCs often drive leukemia, we hypothesized that HSPC fitness substantially contributes to transformation from CHIP to leukemia. HSPC fitness is defined as the proliferative advantage over cells...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788175"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal dynamics of clonal hematopoiesis identifies gene-specific fitness effects</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788175/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220718212208&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) increases rapidly in prevalence beyond age 60 and has been associated with increased risk for malignancy, heart disease and ischemic stroke. CHIP is driven by somatic mutations in hematopoietic stem and progenitor cells (HSPCs). Because mutations in HSPCs often drive leukemia, we hypothesized that HSPC fitness substantially contributes to transformation from CHIP to leukemia. HSPC fitness is defined as the proliferative advantage over cells...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787548"
                        open
                      >
                        <summary class="article-expander__title">Role of p53 in regulation of hematopoiesis in health and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787548/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220718212208&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Human aging is associated with an exponential increase in the occurrence of clonal hematopoiesis of indeterminate potential (CHIP). CHIP is associated with increased risks of de novo and therapy-related hematologic neoplasms and serves as a reservoir for leukemic relapse. Somatic mutations in the TP53 gene, which encodes the tumor suppressor protein p53, rank in the top five among genes that were mutated in CHIP. TP53 mutations in CHIP are associated with an increased...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788175"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal dynamics of clonal hematopoiesis identifies gene-specific fitness effects</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788175/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220717211412&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) increases rapidly in prevalence beyond age 60 and has been associated with increased risk for malignancy, heart disease and ischemic stroke. CHIP is driven by somatic mutations in hematopoietic stem and progenitor cells (HSPCs). Because mutations in HSPCs often drive leukemia, we hypothesized that HSPC fitness substantially contributes to transformation from CHIP to leukemia. HSPC fitness is defined as the proliferative advantage over cells...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787548"
                        open
                      >
                        <summary class="article-expander__title">Role of p53 in regulation of hematopoiesis in health and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787548/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220717211412&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Human aging is associated with an exponential increase in the occurrence of clonal hematopoiesis of indeterminate potential (CHIP). CHIP is associated with increased risks of de novo and therapy-related hematologic neoplasms and serves as a reservoir for leukemic relapse. Somatic mutations in the TP53 gene, which encodes the tumor suppressor protein p53, rank in the top five among genes that were mutated in CHIP. TP53 mutations in CHIP are associated with an increased...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787546"
                        open
                      >
                        <summary class="article-expander__title">No time to die? Intrinsic apoptosis signaling in hematopoietic stem and progenitor cells and therapeutic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220717211412&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Dysregulated apoptosis contributes to the pathogenesis of many hematologic malignancies. BH3-mimetics, antagonists of antiapoptotic BCL-2 proteins, represent novel, and promising cancer drugs. While the acute myelosuppressive effects of Venetoclax, the first Food and Drug Administration approved BCL-2 inhibitor, are fairly well described, little is known about side effects of novel BH3-mimetics and effects of chronic Venetoclax treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788175"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal dynamics of clonal hematopoiesis identifies gene-specific fitness effects</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788175/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220716212224&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) increases rapidly in prevalence beyond age 60 and has been associated with increased risk for malignancy, heart disease and ischemic stroke. CHIP is driven by somatic mutations in hematopoietic stem and progenitor cells (HSPCs). Because mutations in HSPCs often drive leukemia, we hypothesized that HSPC fitness substantially contributes to transformation from CHIP to leukemia. HSPC fitness is defined as the proliferative advantage over cells...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787548"
                        open
                      >
                        <summary class="article-expander__title">Role of p53 in regulation of hematopoiesis in health and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787548/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220716212224&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Human aging is associated with an exponential increase in the occurrence of clonal hematopoiesis of indeterminate potential (CHIP). CHIP is associated with increased risks of de novo and therapy-related hematologic neoplasms and serves as a reservoir for leukemic relapse. Somatic mutations in the TP53 gene, which encodes the tumor suppressor protein p53, rank in the top five among genes that were mutated in CHIP. TP53 mutations in CHIP are associated with an increased...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787546"
                        open
                      >
                        <summary class="article-expander__title">No time to die? Intrinsic apoptosis signaling in hematopoietic stem and progenitor cells and therapeutic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220716212224&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Dysregulated apoptosis contributes to the pathogenesis of many hematologic malignancies. BH3-mimetics, antagonists of antiapoptotic BCL-2 proteins, represent novel, and promising cancer drugs. While the acute myelosuppressive effects of Venetoclax, the first Food and Drug Administration approved BCL-2 inhibitor, are fairly well described, little is known about side effects of novel BH3-mimetics and effects of chronic Venetoclax treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788175"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal dynamics of clonal hematopoiesis identifies gene-specific fitness effects</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788175/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220715211735&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) increases rapidly in prevalence beyond age 60 and has been associated with increased risk for malignancy, heart disease and ischemic stroke. CHIP is driven by somatic mutations in hematopoietic stem and progenitor cells (HSPCs). Because mutations in HSPCs often drive leukemia, we hypothesized that HSPC fitness substantially contributes to transformation from CHIP to leukemia. HSPC fitness is defined as the proliferative advantage over cells...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787548"
                        open
                      >
                        <summary class="article-expander__title">Role of p53 in regulation of hematopoiesis in health and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787548/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220715211735&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Human aging is associated with an exponential increase in the occurrence of clonal hematopoiesis of indeterminate potential (CHIP). CHIP is associated with increased risks of de novo and therapy-related hematologic neoplasms and serves as a reservoir for leukemic relapse. Somatic mutations in the TP53 gene, which encodes the tumor suppressor protein p53, rank in the top five among genes that were mutated in CHIP. TP53 mutations in CHIP are associated with an increased...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787546"
                        open
                      >
                        <summary class="article-expander__title">No time to die? Intrinsic apoptosis signaling in hematopoietic stem and progenitor cells and therapeutic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220715211735&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Dysregulated apoptosis contributes to the pathogenesis of many hematologic malignancies. BH3-mimetics, antagonists of antiapoptotic BCL-2 proteins, represent novel, and promising cancer drugs. While the acute myelosuppressive effects of Venetoclax, the first Food and Drug Administration approved BCL-2 inhibitor, are fairly well described, little is known about side effects of novel BH3-mimetics and effects of chronic Venetoclax treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787092"
                        open
                      >
                        <summary class="article-expander__title">Fedratinib, the first selective JAK2 inhibitor approved for treatment of myelofibrosis - an option beyond ruxolitinib</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220715211735&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myelofibrosis, a life shortening clonal disorder, presents with a constellation of features: bone marrow fibrosis, abnormal blood counts, extramedullary hematopoiesis, splenomegaly, thrombohemorrhagic complications, and constitutional symptoms. Until recently Ruxolitinib, a JAK1 and 2 inhibitor, has been the only targeted therapy available for transplant-ineligible patients requiring treatment for splenomegaly and disease-related symptoms. However, most patients discontinue...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788175"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal dynamics of clonal hematopoiesis identifies gene-specific fitness effects</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788175/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) increases rapidly in prevalence beyond age 60 and has been associated with increased risk for malignancy, heart disease and ischemic stroke. CHIP is driven by somatic mutations in hematopoietic stem and progenitor cells (HSPCs). Because mutations in HSPCs often drive leukemia, we hypothesized that HSPC fitness substantially contributes to transformation from CHIP to leukemia. HSPC fitness is defined as the proliferative advantage over cells...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787548"
                        open
                      >
                        <summary class="article-expander__title">Role of p53 in regulation of hematopoiesis in health and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787548/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Human aging is associated with an exponential increase in the occurrence of clonal hematopoiesis of indeterminate potential (CHIP). CHIP is associated with increased risks of de novo and therapy-related hematologic neoplasms and serves as a reservoir for leukemic relapse. Somatic mutations in the TP53 gene, which encodes the tumor suppressor protein p53, rank in the top five among genes that were mutated in CHIP. TP53 mutations in CHIP are associated with an increased...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787546"
                        open
                      >
                        <summary class="article-expander__title">No time to die? Intrinsic apoptosis signaling in hematopoietic stem and progenitor cells and therapeutic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Dysregulated apoptosis contributes to the pathogenesis of many hematologic malignancies. BH3-mimetics, antagonists of antiapoptotic BCL-2 proteins, represent novel, and promising cancer drugs. While the acute myelosuppressive effects of Venetoclax, the first Food and Drug Administration approved BCL-2 inhibitor, are fairly well described, little is known about side effects of novel BH3-mimetics and effects of chronic Venetoclax treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787092"
                        open
                      >
                        <summary class="article-expander__title">Fedratinib, the first selective JAK2 inhibitor approved for treatment of myelofibrosis - an option beyond ruxolitinib</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myelofibrosis, a life shortening clonal disorder, presents with a constellation of features: bone marrow fibrosis, abnormal blood counts, extramedullary hematopoiesis, splenomegaly, thrombohemorrhagic complications, and constitutional symptoms. Until recently Ruxolitinib, a JAK1 and 2 inhibitor, has been the only targeted therapy available for transplant-ineligible patients requiring treatment for splenomegaly and disease-related symptoms. However, most patients discontinue...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35786159"
                        open
                      >
                        <summary class="article-expander__title">Current and emerging drugs for the treatment of atherosclerosis: the evidence to date</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35786159/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Atherosclerosis can be considered a chronic inflammatory process that stands out as a dominant cause of cardiovascular disease (CVD). Since blood lipids are the leading risk factor for atherosclerosis development, lowering low-density lipoprotein cholesterol (LDL-C) and other apolipoprotein B-containing lipoproteins reduces the risk of future cardiovascular events. However, there has been significant progress in developing lipid-lowering drugs for aggressive management of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788175"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal dynamics of clonal hematopoiesis identifies gene-specific fitness effects</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788175/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) increases rapidly in prevalence beyond age 60 and has been associated with increased risk for malignancy, heart disease and ischemic stroke. CHIP is driven by somatic mutations in hematopoietic stem and progenitor cells (HSPCs). Because mutations in HSPCs often drive leukemia, we hypothesized that HSPC fitness substantially contributes to transformation from CHIP to leukemia. HSPC fitness is defined as the proliferative advantage over cells...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787548"
                        open
                      >
                        <summary class="article-expander__title">Role of p53 in regulation of hematopoiesis in health and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787548/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Human aging is associated with an exponential increase in the occurrence of clonal hematopoiesis of indeterminate potential (CHIP). CHIP is associated with increased risks of de novo and therapy-related hematologic neoplasms and serves as a reservoir for leukemic relapse. Somatic mutations in the TP53 gene, which encodes the tumor suppressor protein p53, rank in the top five among genes that were mutated in CHIP. TP53 mutations in CHIP are associated with an increased...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787546"
                        open
                      >
                        <summary class="article-expander__title">No time to die? Intrinsic apoptosis signaling in hematopoietic stem and progenitor cells and therapeutic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Dysregulated apoptosis contributes to the pathogenesis of many hematologic malignancies. BH3-mimetics, antagonists of antiapoptotic BCL-2 proteins, represent novel, and promising cancer drugs. While the acute myelosuppressive effects of Venetoclax, the first Food and Drug Administration approved BCL-2 inhibitor, are fairly well described, little is known about side effects of novel BH3-mimetics and effects of chronic Venetoclax treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787092"
                        open
                      >
                        <summary class="article-expander__title">Fedratinib, the first selective JAK2 inhibitor approved for treatment of myelofibrosis - an option beyond ruxolitinib</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myelofibrosis, a life shortening clonal disorder, presents with a constellation of features: bone marrow fibrosis, abnormal blood counts, extramedullary hematopoiesis, splenomegaly, thrombohemorrhagic complications, and constitutional symptoms. Until recently Ruxolitinib, a JAK1 and 2 inhibitor, has been the only targeted therapy available for transplant-ineligible patients requiring treatment for splenomegaly and disease-related symptoms. However, most patients discontinue...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35786159"
                        open
                      >
                        <summary class="article-expander__title">Current and emerging drugs for the treatment of atherosclerosis: the evidence to date</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35786159/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Atherosclerosis can be considered a chronic inflammatory process that stands out as a dominant cause of cardiovascular disease (CVD). Since blood lipids are the leading risk factor for atherosclerosis development, lowering low-density lipoprotein cholesterol (LDL-C) and other apolipoprotein B-containing lipoproteins reduces the risk of future cardiovascular events. However, there has been significant progress in developing lipid-lowering drugs for aggressive management of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788175"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal dynamics of clonal hematopoiesis identifies gene-specific fitness effects</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788175/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) increases rapidly in prevalence beyond age 60 and has been associated with increased risk for malignancy, heart disease and ischemic stroke. CHIP is driven by somatic mutations in hematopoietic stem and progenitor cells (HSPCs). Because mutations in HSPCs often drive leukemia, we hypothesized that HSPC fitness substantially contributes to transformation from CHIP to leukemia. HSPC fitness is defined as the proliferative advantage over cells...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787548"
                        open
                      >
                        <summary class="article-expander__title">Role of p53 in regulation of hematopoiesis in health and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787548/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Human aging is associated with an exponential increase in the occurrence of clonal hematopoiesis of indeterminate potential (CHIP). CHIP is associated with increased risks of de novo and therapy-related hematologic neoplasms and serves as a reservoir for leukemic relapse. Somatic mutations in the TP53 gene, which encodes the tumor suppressor protein p53, rank in the top five among genes that were mutated in CHIP. TP53 mutations in CHIP are associated with an increased...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787546"
                        open
                      >
                        <summary class="article-expander__title">No time to die? Intrinsic apoptosis signaling in hematopoietic stem and progenitor cells and therapeutic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Dysregulated apoptosis contributes to the pathogenesis of many hematologic malignancies. BH3-mimetics, antagonists of antiapoptotic BCL-2 proteins, represent novel, and promising cancer drugs. While the acute myelosuppressive effects of Venetoclax, the first Food and Drug Administration approved BCL-2 inhibitor, are fairly well described, little is known about side effects of novel BH3-mimetics and effects of chronic Venetoclax treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787092"
                        open
                      >
                        <summary class="article-expander__title">Fedratinib, the first selective JAK2 inhibitor approved for treatment of myelofibrosis - an option beyond ruxolitinib</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myelofibrosis, a life shortening clonal disorder, presents with a constellation of features: bone marrow fibrosis, abnormal blood counts, extramedullary hematopoiesis, splenomegaly, thrombohemorrhagic complications, and constitutional symptoms. Until recently Ruxolitinib, a JAK1 and 2 inhibitor, has been the only targeted therapy available for transplant-ineligible patients requiring treatment for splenomegaly and disease-related symptoms. However, most patients discontinue...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35786159"
                        open
                      >
                        <summary class="article-expander__title">Current and emerging drugs for the treatment of atherosclerosis: the evidence to date</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35786159/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Atherosclerosis can be considered a chronic inflammatory process that stands out as a dominant cause of cardiovascular disease (CVD). Since blood lipids are the leading risk factor for atherosclerosis development, lowering low-density lipoprotein cholesterol (LDL-C) and other apolipoprotein B-containing lipoproteins reduces the risk of future cardiovascular events. However, there has been significant progress in developing lipid-lowering drugs for aggressive management of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788175"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal dynamics of clonal hematopoiesis identifies gene-specific fitness effects</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788175/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) increases rapidly in prevalence beyond age 60 and has been associated with increased risk for malignancy, heart disease and ischemic stroke. CHIP is driven by somatic mutations in hematopoietic stem and progenitor cells (HSPCs). Because mutations in HSPCs often drive leukemia, we hypothesized that HSPC fitness substantially contributes to transformation from CHIP to leukemia. HSPC fitness is defined as the proliferative advantage over cells...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787548"
                        open
                      >
                        <summary class="article-expander__title">Role of p53 in regulation of hematopoiesis in health and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787548/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Human aging is associated with an exponential increase in the occurrence of clonal hematopoiesis of indeterminate potential (CHIP). CHIP is associated with increased risks of de novo and therapy-related hematologic neoplasms and serves as a reservoir for leukemic relapse. Somatic mutations in the TP53 gene, which encodes the tumor suppressor protein p53, rank in the top five among genes that were mutated in CHIP. TP53 mutations in CHIP are associated with an increased...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787546"
                        open
                      >
                        <summary class="article-expander__title">No time to die? Intrinsic apoptosis signaling in hematopoietic stem and progenitor cells and therapeutic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Dysregulated apoptosis contributes to the pathogenesis of many hematologic malignancies. BH3-mimetics, antagonists of antiapoptotic BCL-2 proteins, represent novel, and promising cancer drugs. While the acute myelosuppressive effects of Venetoclax, the first Food and Drug Administration approved BCL-2 inhibitor, are fairly well described, little is known about side effects of novel BH3-mimetics and effects of chronic Venetoclax treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787092"
                        open
                      >
                        <summary class="article-expander__title">Fedratinib, the first selective JAK2 inhibitor approved for treatment of myelofibrosis - an option beyond ruxolitinib</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myelofibrosis, a life shortening clonal disorder, presents with a constellation of features: bone marrow fibrosis, abnormal blood counts, extramedullary hematopoiesis, splenomegaly, thrombohemorrhagic complications, and constitutional symptoms. Until recently Ruxolitinib, a JAK1 and 2 inhibitor, has been the only targeted therapy available for transplant-ineligible patients requiring treatment for splenomegaly and disease-related symptoms. However, most patients discontinue...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35786159"
                        open
                      >
                        <summary class="article-expander__title">Current and emerging drugs for the treatment of atherosclerosis: the evidence to date</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35786159/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Atherosclerosis can be considered a chronic inflammatory process that stands out as a dominant cause of cardiovascular disease (CVD). Since blood lipids are the leading risk factor for atherosclerosis development, lowering low-density lipoprotein cholesterol (LDL-C) and other apolipoprotein B-containing lipoproteins reduces the risk of future cardiovascular events. However, there has been significant progress in developing lipid-lowering drugs for aggressive management of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788175"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal dynamics of clonal hematopoiesis identifies gene-specific fitness effects</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788175/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) increases rapidly in prevalence beyond age 60 and has been associated with increased risk for malignancy, heart disease and ischemic stroke. CHIP is driven by somatic mutations in hematopoietic stem and progenitor cells (HSPCs). Because mutations in HSPCs often drive leukemia, we hypothesized that HSPC fitness substantially contributes to transformation from CHIP to leukemia. HSPC fitness is defined as the proliferative advantage over cells...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787548"
                        open
                      >
                        <summary class="article-expander__title">Role of p53 in regulation of hematopoiesis in health and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787548/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Human aging is associated with an exponential increase in the occurrence of clonal hematopoiesis of indeterminate potential (CHIP). CHIP is associated with increased risks of de novo and therapy-related hematologic neoplasms and serves as a reservoir for leukemic relapse. Somatic mutations in the TP53 gene, which encodes the tumor suppressor protein p53, rank in the top five among genes that were mutated in CHIP. TP53 mutations in CHIP are associated with an increased...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787546"
                        open
                      >
                        <summary class="article-expander__title">No time to die? Intrinsic apoptosis signaling in hematopoietic stem and progenitor cells and therapeutic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Dysregulated apoptosis contributes to the pathogenesis of many hematologic malignancies. BH3-mimetics, antagonists of antiapoptotic BCL-2 proteins, represent novel, and promising cancer drugs. While the acute myelosuppressive effects of Venetoclax, the first Food and Drug Administration approved BCL-2 inhibitor, are fairly well described, little is known about side effects of novel BH3-mimetics and effects of chronic Venetoclax treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787092"
                        open
                      >
                        <summary class="article-expander__title">Fedratinib, the first selective JAK2 inhibitor approved for treatment of myelofibrosis - an option beyond ruxolitinib</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Introduction: Myelofibrosis, a life shortening clonal disorder, presents with a constellation of features: bone marrow fibrosis, abnormal blood counts, extramedullary hematopoiesis, splenomegaly, thrombohemorrhagic complications and constitutional symptoms. Until recently Ruxolitinib, a JAK1 and 2 inhibitor, has been the only targeted therapy available for transplant-ineligible patients requiring treatment for splenomegaly and disease related symptoms. However, most patients discontinue...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35786159"
                        open
                      >
                        <summary class="article-expander__title">Current and emerging drugs for the treatment of atherosclerosis: the evidence to date</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35786159/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Atherosclerosis can be considered a chronic inflammatory process that stands out as a dominant cause of cardiovascular disease (CVD). Since blood lipids are the leading risk factor for atherosclerosis development, lowering low-density lipoprotein cholesterol (LDL-C) and other apolipoprotein B-containing lipoproteins reduces the risk of future cardiovascular events. However, there has been significant progress in developing lipid-lowering drugs for aggressive management of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788175"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal dynamics of clonal hematopoiesis identifies gene-specific fitness effects</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788175/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) increases rapidly in prevalence beyond age 60 and has been associated with increased risk for malignancy, heart disease and ischemic stroke. CHIP is driven by somatic mutations in hematopoietic stem and progenitor cells (HSPCs). Because mutations in HSPCs often drive leukemia, we hypothesized that HSPC fitness substantially contributes to transformation from CHIP to leukemia. HSPC fitness is defined as the proliferative advantage over cells...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787548"
                        open
                      >
                        <summary class="article-expander__title">Role of p53 in regulation of hematopoiesis in health and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787548/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Human aging is associated with an exponential increase in the occurrence of clonal hematopoiesis of indeterminate potential (CHIP). CHIP is associated with increased risks of de novo and therapy-related hematologic neoplasms and serves as a reservoir for leukemic relapse. Somatic mutations in the TP53 gene, which encodes the tumor suppressor protein p53, rank in the top five among genes that were mutated in CHIP. TP53 mutations in CHIP are associated with an increased...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787546"
                        open
                      >
                        <summary class="article-expander__title">No time to die? Intrinsic apoptosis signaling in hematopoietic stem and progenitor cells and therapeutic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Dysregulated apoptosis contributes to the pathogenesis of many hematologic malignancies. BH3-mimetics, antagonists of antiapoptotic BCL-2 proteins, represent novel, and promising cancer drugs. While the acute myelosuppressive effects of Venetoclax, the first Food and Drug Administration approved BCL-2 inhibitor, are fairly well described, little is known about side effects of novel BH3-mimetics and effects of chronic Venetoclax treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787092"
                        open
                      >
                        <summary class="article-expander__title">Fedratinib, the first selective JAK2 inhibitor approved for treatment of myelofibrosis - an option beyond ruxolitinib</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Introduction: Myelofibrosis, a life shortening clonal disorder, presents with a constellation of features: bone marrow fibrosis, abnormal blood counts, extramedullary hematopoiesis, splenomegaly, thrombohemorrhagic complications and constitutional symptoms. Until recently Ruxolitinib, a JAK1 and 2 inhibitor, has been the only targeted therapy available for transplant-ineligible patients requiring treatment for splenomegaly and disease related symptoms. However, most patients discontinue...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35786159"
                        open
                      >
                        <summary class="article-expander__title">Current and emerging drugs for the treatment of atherosclerosis: the evidence to date</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35786159/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Atherosclerosis can be considered a chronic inflammatory process that stands out as a dominant cause of cardiovascular disease (CVD). Since blood lipids are the leading risk factor for atherosclerosis development, lowering low-density lipoprotein cholesterol (LDL-C) and other apolipoprotein B-containing lipoproteins reduces the risk of future cardiovascular events. However, there has been significant progress in developing lipid-lowering drugs for aggressive management of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788175"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal dynamics of clonal hematopoiesis identifies gene-specific fitness effects</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788175/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220708210406&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) increases rapidly in prevalence beyond age 60 and has been associated with increased risk for malignancy, heart disease and ischemic stroke. CHIP is driven by somatic mutations in hematopoietic stem and progenitor cells (HSPCs). Because mutations in HSPCs often drive leukemia, we hypothesized that HSPC fitness substantially contributes to transformation from CHIP to leukemia. HSPC fitness is defined as the proliferative advantage over cells...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787548"
                        open
                      >
                        <summary class="article-expander__title">Role of p53 in regulation of hematopoiesis in health and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787548/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220708210406&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Human aging is associated with an exponential increase in the occurrence of clonal hematopoiesis of indeterminate potential (CHIP). CHIP is associated with increased risks of de novo and therapy-related hematologic neoplasms and serves as a reservoir for leukemic relapse. Somatic mutations in the TP53 gene, which encodes the tumor suppressor protein p53, rank in the top five among genes that were mutated in CHIP. TP53 mutations in CHIP are associated with an increased...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787546"
                        open
                      >
                        <summary class="article-expander__title">No time to die? Intrinsic apoptosis signaling in hematopoietic stem and progenitor cells and therapeutic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220708210406&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Dysregulated apoptosis contributes to the pathogenesis of many hematologic malignancies. BH3-mimetics, antagonists of antiapoptotic BCL-2 proteins, represent novel, and promising cancer drugs. While the acute myelosuppressive effects of Venetoclax, the first Food and Drug Administration approved BCL-2 inhibitor, are fairly well described, little is known about side effects of novel BH3-mimetics and effects of chronic Venetoclax treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787092"
                        open
                      >
                        <summary class="article-expander__title">Fedratinib, the first selective JAK2 inhibitor approved for treatment of myelofibrosis - an option beyond ruxolitinib</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220708210406&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Introduction: Myelofibrosis, a life shortening clonal disorder, presents with a constellation of features: bone marrow fibrosis, abnormal blood counts, extramedullary hematopoiesis, splenomegaly, thrombohemorrhagic complications and constitutional symptoms. Until recently Ruxolitinib, a JAK1 and 2 inhibitor, has been the only targeted therapy available for transplant-ineligible patients requiring treatment for splenomegaly and disease related symptoms. However, most patients discontinue...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35786159"
                        open
                      >
                        <summary class="article-expander__title">Current and emerging drugs for the treatment of atherosclerosis: the evidence to date</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35786159/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220708210406&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Atherosclerosis can be considered a chronic inflammatory process that stands out as a dominant cause of cardiovascular disease (CVD). Since blood lipids are the leading risk factor for atherosclerosis development, lowering low-density lipoprotein cholesterol (LDL-C) and other apolipoprotein B-containing lipoproteins reduces the risk of future cardiovascular events. However, there has been significant progress in developing lipid-lowering drugs for aggressive management of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788175"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal dynamics of clonal hematopoiesis identifies gene-specific fitness effects</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788175/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220707211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) increases rapidly in prevalence beyond age 60 and has been associated with increased risk for malignancy, heart disease and ischemic stroke. CHIP is driven by somatic mutations in hematopoietic stem and progenitor cells (HSPCs). Because mutations in HSPCs often drive leukemia, we hypothesized that HSPC fitness substantially contributes to transformation from CHIP to leukemia. HSPC fitness is defined as the proliferative advantage over cells...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787548"
                        open
                      >
                        <summary class="article-expander__title">Role of p53 in regulation of hematopoiesis in health and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787548/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220707211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Human aging is associated with an exponential increase in the occurrence of clonal hematopoiesis of indeterminate potential (CHIP). CHIP is associated with increased risks of de novo and therapy-related hematologic neoplasms and serves as a reservoir for leukemic relapse. Somatic mutations in the TP53 gene, which encodes the tumor suppressor protein p53, rank in the top five among genes that were mutated in CHIP. TP53 mutations in CHIP are associated with an increased...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787546"
                        open
                      >
                        <summary class="article-expander__title">No time to die? Intrinsic apoptosis signaling in hematopoietic stem and progenitor cells and therapeutic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220707211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Dysregulated apoptosis contributes to the pathogenesis of many hematologic malignancies. BH3-mimetics, antagonists of antiapoptotic BCL-2 proteins, represent novel, and promising cancer drugs. While the acute myelosuppressive effects of Venetoclax, the first Food and Drug Administration approved BCL-2 inhibitor, are fairly well described, little is known about side effects of novel BH3-mimetics and effects of chronic Venetoclax treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787092"
                        open
                      >
                        <summary class="article-expander__title">Fedratinib, the first selective JAK2 inhibitor approved for treatment of myelofibrosis - an option beyond ruxolitinib</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220707211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Introduction: Myelofibrosis, a life shortening clonal disorder, presents with a constellation of features: bone marrow fibrosis, abnormal blood counts, extramedullary hematopoiesis, splenomegaly, thrombohemorrhagic complications and constitutional symptoms. Until recently Ruxolitinib, a JAK1 and 2 inhibitor, has been the only targeted therapy available for transplant-ineligible patients requiring treatment for splenomegaly and disease related symptoms. However, most patients discontinue...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35786159"
                        open
                      >
                        <summary class="article-expander__title">Current and emerging drugs for the treatment of atherosclerosis: the evidence to date</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35786159/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220707211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Atherosclerosis can be considered a chronic inflammatory process that stands out as a dominant cause of cardiovascular disease (CVD). Since blood lipids are the leading risk factor for atherosclerosis development, lowering low-density lipoprotein cholesterol (LDL-C) and other apolipoprotein B-containing lipoproteins reduces the risk of future cardiovascular events. However, there has been significant progress in developing lipid-lowering drugs for aggressive management of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788175"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal dynamics of clonal hematopoiesis identifies gene-specific fitness effects</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788175/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220706212003&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) increases rapidly in prevalence beyond age 60 and has been associated with increased risk for malignancy, heart disease and ischemic stroke. CHIP is driven by somatic mutations in hematopoietic stem and progenitor cells (HSPCs). Because mutations in HSPCs often drive leukemia, we hypothesized that HSPC fitness substantially contributes to transformation from CHIP to leukemia. HSPC fitness is defined as the proliferative advantage over cells...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787548"
                        open
                      >
                        <summary class="article-expander__title">Role of p53 in regulation of hematopoiesis in health and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787548/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220706212003&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Human aging is associated with an exponential increase in the occurrence of clonal hematopoiesis of indeterminate potential (CHIP). CHIP is associated with increased risks of de novo and therapy-related hematologic neoplasms and serves as a reservoir for leukemic relapse. Somatic mutations in the TP53 gene, which encodes the tumor suppressor protein p53, rank in the top five among genes that were mutated in CHIP. TP53 mutations in CHIP are associated with an increased...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787546"
                        open
                      >
                        <summary class="article-expander__title">No time to die? Intrinsic apoptosis signaling in hematopoietic stem and progenitor cells and therapeutic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220706212003&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Dysregulated apoptosis contributes to the pathogenesis of many hematologic malignancies. BH3-mimetics, antagonists of antiapoptotic BCL-2 proteins, represent novel, and promising cancer drugs. While the acute myelosuppressive effects of Venetoclax, the first Food and Drug Administration approved BCL-2 inhibitor, are fairly well described, little is known about side effects of novel BH3-mimetics and effects of chronic Venetoclax treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787092"
                        open
                      >
                        <summary class="article-expander__title">Fedratinib, the first selective JAK2 inhibitor approved for treatment of myelofibrosis - an option beyond ruxolitinib</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220706212003&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Introduction: Myelofibrosis, a life shortening clonal disorder, presents with a constellation of features: bone marrow fibrosis, abnormal blood counts, extramedullary hematopoiesis, splenomegaly, thrombohemorrhagic complications and constitutional symptoms. Until recently Ruxolitinib, a JAK1 and 2 inhibitor, has been the only targeted therapy available for transplant-ineligible patients requiring treatment for splenomegaly and disease related symptoms. However, most patients discontinue...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35786159"
                        open
                      >
                        <summary class="article-expander__title">Current and emerging drugs for the treatment of atherosclerosis: the evidence to date</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35786159/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220706212003&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Atherosclerosis can be considered a chronic inflammatory process that stands out as a dominant cause of cardiovascular disease (CVD). Since blood lipids are the leading risk factor for atherosclerosis development, lowering low-density lipoprotein cholesterol (LDL-C) and other apolipoprotein B-containing lipoproteins reduces the risk of future cardiovascular events. However, there has been significant progress in developing lipid-lowering drugs for aggressive management of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787548"
                        open
                      >
                        <summary class="article-expander__title">Role of p53 in regulation of hematopoiesis in health and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787548/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220705212255&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Human aging is associated with an exponential increase in the occurrence of clonal hematopoiesis of indeterminate potential (CHIP). CHIP is associated with increased risks of de novo and therapy-related hematologic neoplasms and serves as a reservoir for leukemic relapse. Somatic mutations in the TP53 gene, which encodes the tumor suppressor protein p53, rank in the top five among genes that were mutated in CHIP. TP53 mutations in CHIP are associated with an increased...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787546"
                        open
                      >
                        <summary class="article-expander__title">No time to die? Intrinsic apoptosis signaling in hematopoietic stem and progenitor cells and therapeutic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220705212255&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Dysregulated apoptosis contributes to the pathogenesis of many hematologic malignancies. BH3-mimetics, antagonists of antiapoptotic BCL-2 proteins, represent novel, and promising cancer drugs. While the acute myelosuppressive effects of Venetoclax, the first Food and Drug Administration approved BCL-2 inhibitor, are fairly well described, little is known about side effects of novel BH3-mimetics and effects of chronic Venetoclax treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787092"
                        open
                      >
                        <summary class="article-expander__title">Fedratinib, the first selective JAK2 inhibitor approved for treatment of myelofibrosis - an option beyond ruxolitinib</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220705212255&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Introduction: Myelofibrosis, a life shortening clonal disorder, presents with a constellation of features: bone marrow fibrosis, abnormal blood counts, extramedullary hematopoiesis, splenomegaly, thrombohemorrhagic complications and constitutional symptoms. Until recently Ruxolitinib, a JAK1 and 2 inhibitor, has been the only targeted therapy available for transplant-ineligible patients requiring treatment for splenomegaly and disease related symptoms. However, most patients discontinue...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35786159"
                        open
                      >
                        <summary class="article-expander__title">Current and emerging drugs for the treatment of atherosclerosis: the evidence to date</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35786159/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220705212255&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Atherosclerosis can be considered a chronic inflammatory process that stands out as a dominant cause of cardiovascular disease (CVD). Since blood lipids are the leading risk factor for atherosclerosis development, lowering low-density lipoprotein cholesterol (LDL-C) and other apolipoprotein B-containing lipoproteins reduces the risk of future cardiovascular events. However, there has been significant progress in developing lipid-lowering drugs for aggressive management of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608010801&amp;v&#x3D;2.17.7&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788590"
                        open
                      >
                        <summary class="article-expander__title">Clonal barcoding with qPCR detection enables live cell functional analyses for cancer research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788590/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220726212040&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Single-cell analysis methods are valuable tools; however, current approaches do not easily enable live cell retrieval. That is a particular issue when further study of cells that were eliminated during experimentation could provide critical information. We report a clonal molecular barcoding method, called SunCatcher, that enables longitudinal tracking and live cell functional analysis. From complex cell populations, we generate single cell-derived clonal populations, infect each with a unique...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788590"
                        open
                      >
                        <summary class="article-expander__title">Clonal barcoding with qPCR detection enables live cell functional analyses for cancer research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788590/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220725211845&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Single-cell analysis methods are valuable tools; however, current approaches do not easily enable live cell retrieval. That is a particular issue when further study of cells that were eliminated during experimentation could provide critical information. We report a clonal molecular barcoding method, called SunCatcher, that enables longitudinal tracking and live cell functional analysis. From complex cell populations, we generate single cell-derived clonal populations, infect each with a unique...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788590"
                        open
                      >
                        <summary class="article-expander__title">Clonal barcoding with qPCR detection enables live cell functional analyses for cancer research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788590/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220724211605&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Single-cell analysis methods are valuable tools; however, current approaches do not easily enable live cell retrieval. That is a particular issue when further study of cells that were eliminated during experimentation could provide critical information. We report a clonal molecular barcoding method, called SunCatcher, that enables longitudinal tracking and live cell functional analysis. From complex cell populations, we generate single cell-derived clonal populations, infect each with a unique...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788590"
                        open
                      >
                        <summary class="article-expander__title">Clonal barcoding with qPCR detection enables live cell functional analyses for cancer research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788590/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220723212121&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Single-cell analysis methods are valuable tools; however, current approaches do not easily enable live cell retrieval. That is a particular issue when further study of cells that were eliminated during experimentation could provide critical information. We report a clonal molecular barcoding method, called SunCatcher, that enables longitudinal tracking and live cell functional analysis. From complex cell populations, we generate single cell-derived clonal populations, infect each with a unique...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35785618"
                        open
                      >
                        <summary class="article-expander__title">Single-cell sequencing reveals CD133+CD44--originating evolution and novel stemness related variants in human colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35785618/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220723212121&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Tumor heterogeneity of human colorectal cancer (CRC)-initiating cells (CRCICs) in cancer tissues often represents aggressive features of cancer progression. For high-resolution examination of CRCICs, we performed single-cell whole-exome sequencing (scWES) and bulk cell targeted exome sequencing (TES) of CRCICs to investigate stemness-specific somatic alterations or clonal evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788590"
                        open
                      >
                        <summary class="article-expander__title">Clonal barcoding with qPCR detection enables live cell functional analyses for cancer research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788590/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220722211347&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Single-cell analysis methods are valuable tools; however, current approaches do not easily enable live cell retrieval. That is a particular issue when further study of cells that were eliminated during experimentation could provide critical information. We report a clonal molecular barcoding method, called SunCatcher, that enables longitudinal tracking and live cell functional analysis. From complex cell populations, we generate single cell-derived clonal populations, infect each with a unique...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35785618"
                        open
                      >
                        <summary class="article-expander__title">Single-cell sequencing reveals CD133+CD44--originating evolution and novel stemness related variants in human colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35785618/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220722211347&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Tumor heterogeneity of human colorectal cancer (CRC)-initiating cells (CRCICs) in cancer tissues often represents aggressive features of cancer progression. For high-resolution examination of CRCICs, we performed single-cell whole-exome sequencing (scWES) and bulk cell targeted exome sequencing (TES) of CRCICs to investigate stemness-specific somatic alterations or clonal evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788590"
                        open
                      >
                        <summary class="article-expander__title">Clonal barcoding with qPCR detection enables live cell functional analyses for cancer research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788590/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220721211948&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Single-cell analysis methods are valuable tools; however, current approaches do not easily enable live cell retrieval. That is a particular issue when further study of cells that were eliminated during experimentation could provide critical information. We report a clonal molecular barcoding method, called SunCatcher, that enables longitudinal tracking and live cell functional analysis. From complex cell populations, we generate single cell-derived clonal populations, infect each with a unique...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35785618"
                        open
                      >
                        <summary class="article-expander__title">Single-cell sequencing reveals CD133+CD44--originating evolution and novel stemness related variants in human colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35785618/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220721211948&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Tumor heterogeneity of human colorectal cancer (CRC)-initiating cells (CRCICs) in cancer tissues often represents aggressive features of cancer progression. For high-resolution examination of CRCICs, we performed single-cell whole-exome sequencing (scWES) and bulk cell targeted exome sequencing (TES) of CRCICs to investigate stemness-specific somatic alterations or clonal evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35781188"
                        open
                      >
                        <summary class="article-expander__title">Germline and Somatic Defects in DDX41 and its Impact on Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35781188/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220721211948&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: While DDX41 mutation (m) is one of the most prevalent predisposition genes in adult myelodysplastic syndrome (MDS)/acute myeloid leukemia (AML), most patients do not always present with a family history of MDS/AML. In this review, we will be highlighting epidemiological data on DDX41m, roles of DDX41 in oncogenesis, mechanisms of clonal evolution with somatic DDX41m, and clinical phenotypes and management of MDS/AML in patients harboring DDX41m.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788590"
                        open
                      >
                        <summary class="article-expander__title">Clonal barcoding with qPCR detection enables live cell functional analyses for cancer research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788590/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220720211338&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Single-cell analysis methods are valuable tools; however, current approaches do not easily enable live cell retrieval. That is a particular issue when further study of cells that were eliminated during experimentation could provide critical information. We report a clonal molecular barcoding method, called SunCatcher, that enables longitudinal tracking and live cell functional analysis. From complex cell populations, we generate single cell-derived clonal populations, infect each with a unique...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35785618"
                        open
                      >
                        <summary class="article-expander__title">Single-cell sequencing reveals CD133+CD44--originating evolution and novel stemness related variants in human colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35785618/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220720211338&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Tumor heterogeneity of human colorectal cancer (CRC)-initiating cells (CRCICs) in cancer tissues often represents aggressive features of cancer progression. For high-resolution examination of CRCICs, we performed single-cell whole-exome sequencing (scWES) and bulk cell targeted exome sequencing (TES) of CRCICs to investigate stemness-specific somatic alterations or clonal evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35781188"
                        open
                      >
                        <summary class="article-expander__title">Germline and Somatic Defects in DDX41 and its Impact on Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35781188/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220720211338&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: While DDX41 mutation (m) is one of the most prevalent predisposition genes in adult myelodysplastic syndrome (MDS)/acute myeloid leukemia (AML), most patients do not always present with a family history of MDS/AML. In this review, we will be highlighting epidemiological data on DDX41m, roles of DDX41 in oncogenesis, mechanisms of clonal evolution with somatic DDX41m, and clinical phenotypes and management of MDS/AML in patients harboring DDX41m.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788590"
                        open
                      >
                        <summary class="article-expander__title">Clonal barcoding with qPCR detection enables live cell functional analyses for cancer research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788590/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220719211909&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Single-cell analysis methods are valuable tools; however, current approaches do not easily enable live cell retrieval. That is a particular issue when further study of cells that were eliminated during experimentation could provide critical information. We report a clonal molecular barcoding method, called SunCatcher, that enables longitudinal tracking and live cell functional analysis. From complex cell populations, we generate single cell-derived clonal populations, infect each with a unique...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35785618"
                        open
                      >
                        <summary class="article-expander__title">Single-cell sequencing reveals CD133+CD44--originating evolution and novel stemness related variants in human colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35785618/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220719211909&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Tumor heterogeneity of human colorectal cancer (CRC)-initiating cells (CRCICs) in cancer tissues often represents aggressive features of cancer progression. For high-resolution examination of CRCICs, we performed single-cell whole-exome sequencing (scWES) and bulk cell targeted exome sequencing (TES) of CRCICs to investigate stemness-specific somatic alterations or clonal evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35781188"
                        open
                      >
                        <summary class="article-expander__title">Germline and Somatic Defects in DDX41 and its Impact on Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35781188/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220719211909&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: While DDX41 mutation (m) is one of the most prevalent predisposition genes in adult myelodysplastic syndrome (MDS)/acute myeloid leukemia (AML), most patients do not always present with a family history of MDS/AML. In this review, we will be highlighting epidemiological data on DDX41m, roles of DDX41 in oncogenesis, mechanisms of clonal evolution with somatic DDX41m, and clinical phenotypes and management of MDS/AML in patients harboring DDX41m.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788590"
                        open
                      >
                        <summary class="article-expander__title">Clonal barcoding with qPCR detection enables live cell functional analyses for cancer research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788590/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220718212208&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Single-cell analysis methods are valuable tools; however, current approaches do not easily enable live cell retrieval. That is a particular issue when further study of cells that were eliminated during experimentation could provide critical information. We report a clonal molecular barcoding method, called SunCatcher, that enables longitudinal tracking and live cell functional analysis. From complex cell populations, we generate single cell-derived clonal populations, infect each with a unique...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35785618"
                        open
                      >
                        <summary class="article-expander__title">Single-cell sequencing reveals CD133+CD44--originating evolution and novel stemness related variants in human colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35785618/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220718212208&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Tumor heterogeneity of human colorectal cancer (CRC)-initiating cells (CRCICs) in cancer tissues often represents aggressive features of cancer progression. For high-resolution examination of CRCICs, we performed single-cell whole-exome sequencing (scWES) and bulk cell targeted exome sequencing (TES) of CRCICs to investigate stemness-specific somatic alterations or clonal evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35781188"
                        open
                      >
                        <summary class="article-expander__title">Germline and Somatic Defects in DDX41 and its Impact on Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35781188/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220718212208&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: While DDX41 mutation (m) is one of the most prevalent predisposition genes in adult myelodysplastic syndrome (MDS)/acute myeloid leukemia (AML), most patients do not always present with a family history of MDS/AML. In this review, we will be highlighting epidemiological data on DDX41m, roles of DDX41 in oncogenesis, mechanisms of clonal evolution with somatic DDX41m, and clinical phenotypes and management of MDS/AML in patients harboring DDX41m.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788590"
                        open
                      >
                        <summary class="article-expander__title">Clonal barcoding with qPCR detection enables live cell functional analyses for cancer research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788590/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220717211412&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Single-cell analysis methods are valuable tools; however, current approaches do not easily enable live cell retrieval. That is a particular issue when further study of cells that were eliminated during experimentation could provide critical information. We report a clonal molecular barcoding method, called SunCatcher, that enables longitudinal tracking and live cell functional analysis. From complex cell populations, we generate single cell-derived clonal populations, infect each with a unique...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35785618"
                        open
                      >
                        <summary class="article-expander__title">Single-cell sequencing reveals CD133+CD44--originating evolution and novel stemness related variants in human colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35785618/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220717211412&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Tumor heterogeneity of human colorectal cancer (CRC)-initiating cells (CRCICs) in cancer tissues often represents aggressive features of cancer progression. For high-resolution examination of CRCICs, we performed single-cell whole-exome sequencing (scWES) and bulk cell targeted exome sequencing (TES) of CRCICs to investigate stemness-specific somatic alterations or clonal evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35781188"
                        open
                      >
                        <summary class="article-expander__title">Germline and Somatic Defects in DDX41 and its Impact on Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35781188/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220717211412&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: While DDX41 mutation (m) is one of the most prevalent predisposition genes in adult myelodysplastic syndrome (MDS)/acute myeloid leukemia (AML), most patients do not always present with a family history of MDS/AML. In this review, we will be highlighting epidemiological data on DDX41m, roles of DDX41 in oncogenesis, mechanisms of clonal evolution with somatic DDX41m, and clinical phenotypes and management of MDS/AML in patients harboring DDX41m.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788590"
                        open
                      >
                        <summary class="article-expander__title">Clonal barcoding with qPCR detection enables live cell functional analyses for cancer research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788590/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220716212224&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Single-cell analysis methods are valuable tools; however, current approaches do not easily enable live cell retrieval. That is a particular issue when further study of cells that were eliminated during experimentation could provide critical information. We report a clonal molecular barcoding method, called SunCatcher, that enables longitudinal tracking and live cell functional analysis. From complex cell populations, we generate single cell-derived clonal populations, infect each with a unique...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35785618"
                        open
                      >
                        <summary class="article-expander__title">Single-cell sequencing reveals CD133+CD44--originating evolution and novel stemness related variants in human colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35785618/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220716212224&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Tumor heterogeneity of human colorectal cancer (CRC)-initiating cells (CRCICs) in cancer tissues often represents aggressive features of cancer progression. For high-resolution examination of CRCICs, we performed single-cell whole-exome sequencing (scWES) and bulk cell targeted exome sequencing (TES) of CRCICs to investigate stemness-specific somatic alterations or clonal evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35781188"
                        open
                      >
                        <summary class="article-expander__title">Germline and Somatic Defects in DDX41 and its Impact on Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35781188/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220716212224&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: While DDX41 mutation (m) is one of the most prevalent predisposition genes in adult myelodysplastic syndrome (MDS)/acute myeloid leukemia (AML), most patients do not always present with a family history of MDS/AML. In this review, we will be highlighting epidemiological data on DDX41m, roles of DDX41 in oncogenesis, mechanisms of clonal evolution with somatic DDX41m, and clinical phenotypes and management of MDS/AML in patients harboring DDX41m.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788590"
                        open
                      >
                        <summary class="article-expander__title">Clonal barcoding with qPCR detection enables live cell functional analyses for cancer research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788590/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220715211735&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Single-cell analysis methods are valuable tools; however, current approaches do not easily enable live cell retrieval. That is a particular issue when further study of cells that were eliminated during experimentation could provide critical information. We report a clonal molecular barcoding method, called SunCatcher, that enables longitudinal tracking and live cell functional analysis. From complex cell populations, we generate single cell-derived clonal populations, infect each with a unique...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35785618"
                        open
                      >
                        <summary class="article-expander__title">Single-cell sequencing reveals CD133+CD44--originating evolution and novel stemness related variants in human colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35785618/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220715211735&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Tumor heterogeneity of human colorectal cancer (CRC)-initiating cells (CRCICs) in cancer tissues often represents aggressive features of cancer progression. For high-resolution examination of CRCICs, we performed single-cell whole-exome sequencing (scWES) and bulk cell targeted exome sequencing (TES) of CRCICs to investigate stemness-specific somatic alterations or clonal evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35781188"
                        open
                      >
                        <summary class="article-expander__title">Germline and Somatic Defects in DDX41 and its Impact on Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35781188/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220715211735&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: While DDX41 mutation (m) is one of the most prevalent predisposition genes in adult myelodysplastic syndrome (MDS)/acute myeloid leukemia (AML), most patients do not always present with a family history of MDS/AML. In this review, we will be highlighting epidemiological data on DDX41m, roles of DDX41 in oncogenesis, mechanisms of clonal evolution with somatic DDX41m, and clinical phenotypes and management of MDS/AML in patients harboring DDX41m.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788590"
                        open
                      >
                        <summary class="article-expander__title">Clonal barcoding with qPCR detection enables live cell functional analyses for cancer research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788590/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Single-cell analysis methods are valuable tools; however, current approaches do not easily enable live cell retrieval. That is a particular issue when further study of cells that were eliminated during experimentation could provide critical information. We report a clonal molecular barcoding method, called SunCatcher, that enables longitudinal tracking and live cell functional analysis. From complex cell populations, we generate single cell-derived clonal populations, infect each with a unique...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35785618"
                        open
                      >
                        <summary class="article-expander__title">Single-cell sequencing reveals CD133+CD44--originating evolution and novel stemness related variants in human colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35785618/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Tumor heterogeneity of human colorectal cancer (CRC)-initiating cells (CRCICs) in cancer tissues often represents aggressive features of cancer progression. For high-resolution examination of CRCICs, we performed single-cell whole-exome sequencing (scWES) and bulk cell targeted exome sequencing (TES) of CRCICs to investigate stemness-specific somatic alterations or clonal evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35781188"
                        open
                      >
                        <summary class="article-expander__title">Germline and Somatic Defects in DDX41 and its Impact on Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35781188/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: While DDX41 mutation (m) is one of the most prevalent predisposition genes in adult myelodysplastic syndrome (MDS)/acute myeloid leukemia (AML), most patients do not always present with a family history of MDS/AML. In this review, we will be highlighting epidemiological data on DDX41m, roles of DDX41 in oncogenesis, mechanisms of clonal evolution with somatic DDX41m, and clinical phenotypes and management of MDS/AML in patients harboring DDX41m.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788590"
                        open
                      >
                        <summary class="article-expander__title">Clonal barcoding with qPCR detection enables live cell functional analyses for cancer research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788590/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Single-cell analysis methods are valuable tools; however, current approaches do not easily enable live cell retrieval. That is a particular issue when further study of cells that were eliminated during experimentation could provide critical information. We report a clonal molecular barcoding method, called SunCatcher, that enables longitudinal tracking and live cell functional analysis. From complex cell populations, we generate single cell-derived clonal populations, infect each with a unique...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35785618"
                        open
                      >
                        <summary class="article-expander__title">Single-cell sequencing reveals CD133+CD44--originating evolution and novel stemness related variants in human colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35785618/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Tumor heterogeneity of human colorectal cancer (CRC)-initiating cells (CRCICs) in cancer tissues often represents aggressive features of cancer progression. For high-resolution examination of CRCICs, we performed single-cell whole-exome sequencing (scWES) and bulk cell targeted exome sequencing (TES) of CRCICs to investigate stemness-specific somatic alterations or clonal evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35781188"
                        open
                      >
                        <summary class="article-expander__title">Germline and Somatic Defects in DDX41 and its Impact on Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35781188/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: While DDX41 mutation (m) is one of the most prevalent predisposition genes in adult myelodysplastic syndrome (MDS)/acute myeloid leukemia (AML), most patients do not always present with a family history of MDS/AML. In this review, we will be highlighting epidemiological data on DDX41m, roles of DDX41 in oncogenesis, mechanisms of clonal evolution with somatic DDX41m, and clinical phenotypes and management of MDS/AML in patients harboring DDX41m.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788590"
                        open
                      >
                        <summary class="article-expander__title">Clonal barcoding with qPCR detection enables live cell functional analyses for cancer research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788590/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Single-cell analysis methods are valuable tools; however, current approaches do not easily enable live cell retrieval. That is a particular issue when further study of cells that were eliminated during experimentation could provide critical information. We report a clonal molecular barcoding method, called SunCatcher, that enables longitudinal tracking and live cell functional analysis. From complex cell populations, we generate single cell-derived clonal populations, infect each with a unique...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35785618"
                        open
                      >
                        <summary class="article-expander__title">Single-cell sequencing reveals CD133+CD44--originating evolution and novel stemness related variants in human colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35785618/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Tumor heterogeneity of human colorectal cancer (CRC)-initiating cells (CRCICs) in cancer tissues often represents aggressive features of cancer progression. For high-resolution examination of CRCICs, we performed single-cell whole-exome sequencing (scWES) and bulk cell targeted exome sequencing (TES) of CRCICs to investigate stemness-specific somatic alterations or clonal evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35781188"
                        open
                      >
                        <summary class="article-expander__title">Germline and Somatic Defects in DDX41 and its Impact on Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35781188/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: While DDX41 mutation (m) is one of the most prevalent predisposition genes in adult myelodysplastic syndrome (MDS)/acute myeloid leukemia (AML), most patients do not always present with a family history of MDS/AML. In this review, we will be highlighting epidemiological data on DDX41m, roles of DDX41 in oncogenesis, mechanisms of clonal evolution with somatic DDX41m, and clinical phenotypes and management of MDS/AML in patients harboring DDX41m.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788590"
                        open
                      >
                        <summary class="article-expander__title">Clonal barcoding with qPCR detection enables live cell functional analyses for cancer research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788590/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Single-cell analysis methods are valuable tools; however, current approaches do not easily enable live cell retrieval. That is a particular issue when further study of cells that were eliminated during experimentation could provide critical information. We report a clonal molecular barcoding method, called SunCatcher, that enables longitudinal tracking and live cell functional analysis. From complex cell populations, we generate single cell-derived clonal populations, infect each with a unique...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35785618"
                        open
                      >
                        <summary class="article-expander__title">Single-cell sequencing reveals CD133+CD44--originating evolution and novel stemness related variants in human colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35785618/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Tumor heterogeneity of human colorectal cancer (CRC)-initiating cells (CRCICs) in cancer tissues often represents aggressive features of cancer progression. For high-resolution examination of CRCICs, we performed single-cell whole-exome sequencing (scWES) and bulk cell targeted exome sequencing (TES) of CRCICs to investigate stemness-specific somatic alterations or clonal evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35781188"
                        open
                      >
                        <summary class="article-expander__title">Germline and Somatic Defects in DDX41 and its Impact on Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35781188/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: While DDX41 mutation (m) is one of the most prevalent predisposition genes in adult myelodysplastic syndrome (MDS)/acute myeloid leukemia (AML), most patients do not always present with a family history of MDS/AML. In this review, we will be highlighting epidemiological data on DDX41m, roles of DDX41 in oncogenesis, mechanisms of clonal evolution with somatic DDX41m, and clinical phenotypes and management of MDS/AML in patients harboring DDX41m.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788590"
                        open
                      >
                        <summary class="article-expander__title">Clonal barcoding with qPCR detection enables live cell functional analyses for cancer research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788590/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Single-cell analysis methods are valuable tools; however, current approaches do not easily enable live cell retrieval. That is a particular issue when further study of cells that were eliminated during experimentation could provide critical information. We report a clonal molecular barcoding method, called SunCatcher, that enables longitudinal tracking and live cell functional analysis. From complex cell populations, we generate single cell-derived clonal populations, infect each with a unique...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35785618"
                        open
                      >
                        <summary class="article-expander__title">Single-cell sequencing reveals CD133+CD44--originating evolution and novel stemness related variants in human colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35785618/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Tumor heterogeneity of human colorectal cancer (CRC)-initiating cells (CRCICs) in cancer tissues often represents aggressive features of cancer progression. For high-resolution examination of CRCICs, we performed single-cell whole-exome sequencing (scWES) and bulk cell targeted exome sequencing (TES) of CRCICs to investigate stemness-specific somatic alterations or clonal evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35781188"
                        open
                      >
                        <summary class="article-expander__title">Germline and Somatic Defects in DDX41 and its Impact on Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35781188/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: While DDX41 mutation (m) is one of the most prevalent predisposition genes in adult myelodysplastic syndrome (MDS)/acute myeloid leukemia (AML), most patients do not always present with a family history of MDS/AML. In this review, we will be highlighting epidemiological data on DDX41m, roles of DDX41 in oncogenesis, mechanisms of clonal evolution with somatic DDX41m, and clinical phenotypes and management of MDS/AML in patients harboring DDX41m.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788590"
                        open
                      >
                        <summary class="article-expander__title">Clonal barcoding with qPCR detection enables live cell functional analyses for cancer research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788590/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Single-cell analysis methods are valuable tools; however, current approaches do not easily enable live cell retrieval. That is a particular issue when further study of cells that were eliminated during experimentation could provide critical information. We report a clonal molecular barcoding method, called SunCatcher, that enables longitudinal tracking and live cell functional analysis. From complex cell populations, we generate single cell-derived clonal populations, infect each with a unique...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35785618"
                        open
                      >
                        <summary class="article-expander__title">Single-cell sequencing reveals CD133+CD44--originating evolution and novel stemness related variants in human colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35785618/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Tumor heterogeneity of human colorectal cancer (CRC)-initiating cells (CRCICs) in cancer tissues often represents aggressive features of cancer progression. For high-resolution examination of CRCICs, we performed single-cell whole-exome sequencing (scWES) and bulk cell targeted exome sequencing (TES) of CRCICs to investigate stemness-specific somatic alterations or clonal evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35781188"
                        open
                      >
                        <summary class="article-expander__title">Germline and Somatic Defects in DDX41 and its Impact on Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35781188/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: While DDX41 mutation (m) is one of the most prevalent predisposition genes in adult myelodysplastic syndrome (MDS)/acute myeloid leukemia (AML), most patients do not always present with a family history of MDS/AML. In this review, we will be highlighting epidemiological data on DDX41m, roles of DDX41 in oncogenesis, mechanisms of clonal evolution with somatic DDX41m, and clinical phenotypes and management of MDS/AML in patients harboring DDX41m.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788590"
                        open
                      >
                        <summary class="article-expander__title">Clonal barcoding with qPCR detection enables live cell functional analyses for cancer research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788590/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220708210406&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Single-cell analysis methods are valuable tools; however, current approaches do not easily enable live cell retrieval. That is a particular issue when further study of cells that were eliminated during experimentation could provide critical information. We report a clonal molecular barcoding method, called SunCatcher, that enables longitudinal tracking and live cell functional analysis. From complex cell populations, we generate single cell-derived clonal populations, infect each with a unique...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35785618"
                        open
                      >
                        <summary class="article-expander__title">Single-cell sequencing reveals CD133+CD44--originating evolution and novel stemness related variants in human colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35785618/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220708210406&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Tumor heterogeneity of human colorectal cancer (CRC)-initiating cells (CRCICs) in cancer tissues often represents aggressive features of cancer progression. For high-resolution examination of CRCICs, we performed single-cell whole-exome sequencing (scWES) and bulk cell targeted exome sequencing (TES) of CRCICs to investigate stemness-specific somatic alterations or clonal evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35781188"
                        open
                      >
                        <summary class="article-expander__title">Germline and Somatic Defects in DDX41 and its Impact on Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35781188/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220708210406&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: While DDX41 mutation (m) is one of the most prevalent predisposition genes in adult myelodysplastic syndrome (MDS)/acute myeloid leukemia (AML), most patients do not always present with a family history of MDS/AML. In this review, we will be highlighting epidemiological data on DDX41m, roles of DDX41 in oncogenesis, mechanisms of clonal evolution with somatic DDX41m, and clinical phenotypes and management of MDS/AML in patients harboring DDX41m.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788590"
                        open
                      >
                        <summary class="article-expander__title">Clonal barcoding with qPCR detection enables live cell functional analyses for cancer research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788590/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220707211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Single-cell analysis methods are valuable tools; however, current approaches do not easily enable live cell retrieval. That is a particular issue when further study of cells that were eliminated during experimentation could provide critical information. We report a clonal molecular barcoding method, called SunCatcher, that enables longitudinal tracking and live cell functional analysis. From complex cell populations, we generate single cell-derived clonal populations, infect each with a unique...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35785618"
                        open
                      >
                        <summary class="article-expander__title">Single-cell sequencing reveals CD133+CD44--originating evolution and novel stemness related variants in human colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35785618/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220707211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Tumor heterogeneity of human colorectal cancer (CRC)-initiating cells (CRCICs) in cancer tissues often represents aggressive features of cancer progression. For high-resolution examination of CRCICs, we performed single-cell whole-exome sequencing (scWES) and bulk cell targeted exome sequencing (TES) of CRCICs to investigate stemness-specific somatic alterations or clonal evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35781188"
                        open
                      >
                        <summary class="article-expander__title">Germline and Somatic Defects in DDX41 and its Impact on Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35781188/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220707211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: While DDX41 mutation (m) is one of the most prevalent predisposition genes in adult myelodysplastic syndrome (MDS)/acute myeloid leukemia (AML), most patients do not always present with a family history of MDS/AML. In this review, we will be highlighting epidemiological data on DDX41m, roles of DDX41 in oncogenesis, mechanisms of clonal evolution with somatic DDX41m, and clinical phenotypes and management of MDS/AML in patients harboring DDX41m.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788590"
                        open
                      >
                        <summary class="article-expander__title">Clonal barcoding with qPCR detection enables live cell functional analyses for cancer research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788590/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220706212003&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Single-cell analysis methods are valuable tools; however, current approaches do not easily enable live cell retrieval. That is a particular issue when further study of cells that were eliminated during experimentation could provide critical information. We report a clonal molecular barcoding method, called SunCatcher, that enables longitudinal tracking and live cell functional analysis. From complex cell populations, we generate single cell-derived clonal populations, infect each with a unique...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35785618"
                        open
                      >
                        <summary class="article-expander__title">Single-cell sequencing reveals CD133+CD44--originating evolution and novel stemness related variants in human colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35785618/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220706212003&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Tumor heterogeneity of human colorectal cancer (CRC)-initiating cells (CRCICs) in cancer tissues often represents aggressive features of cancer progression. For high-resolution examination of CRCICs, we performed single-cell whole-exome sequencing (scWES) and bulk cell targeted exome sequencing (TES) of CRCICs to investigate stemness-specific somatic alterations or clonal evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35781188"
                        open
                      >
                        <summary class="article-expander__title">Germline and Somatic Defects in DDX41 and its Impact on Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35781188/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220706212003&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: While DDX41 mutation (m) is one of the most prevalent predisposition genes in adult myelodysplastic syndrome (MDS)/acute myeloid leukemia (AML), most patients do not always present with a family history of MDS/AML. In this review, we will be highlighting epidemiological data on DDX41m, roles of DDX41 in oncogenesis, mechanisms of clonal evolution with somatic DDX41m, and clinical phenotypes and management of MDS/AML in patients harboring DDX41m.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35785618"
                        open
                      >
                        <summary class="article-expander__title">Single-cell sequencing reveals CD133+CD44--originating evolution and novel stemness related variants in human colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35785618/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220705212255&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Tumor heterogeneity of human colorectal cancer (CRC)-initiating cells (CRCICs) in cancer tissues often represents aggressive features of cancer progression. For high-resolution examination of CRCICs, we performed single-cell whole-exome sequencing (scWES) and bulk cell targeted exome sequencing (TES) of CRCICs to investigate stemness-specific somatic alterations or clonal evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35781188"
                        open
                      >
                        <summary class="article-expander__title">Germline and Somatic Defects in DDX41 and its Impact on Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35781188/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220705212255&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: While DDX41 mutation (m) is one of the most prevalent predisposition genes in adult myelodysplastic syndrome (MDS)/acute myeloid leukemia (AML), most patients do not always present with a family history of MDS/AML. In this review, we will be highlighting epidemiological data on DDX41m, roles of DDX41 in oncogenesis, mechanisms of clonal evolution with somatic DDX41m, and clinical phenotypes and management of MDS/AML in patients harboring DDX41m.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-06-30 datetime="2022-06-30T10:00:00.000Z">2022-06-30</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-06-30 datetime="2022-06-30T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608010801&amp;v&#x3D;2.17.7&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772431"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic syndrome and autoimmune disorders: two sides of the same coin?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772431/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220721211948&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Systemic inflammatory and autoimmune diseases and myelodysplastic syndromes have been linked in individual patients and in larger case series for at least 25 years. These associations frequently include thyroid disease, neutrophilic dermatoses, polyarthritis, connective tissue diseases, vasculitis, and autoimmune cytopenias. Studies have found that autoimmune disease (or its therapy) is a risk factor for the development of myelodysplastic syndromes, but such syndromes might also be an instigator...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772431"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic syndrome and autoimmune disorders: two sides of the same coin?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772431/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220720211338&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Systemic inflammatory and autoimmune diseases and myelodysplastic syndromes have been linked in individual patients and in larger case series for at least 25 years. These associations frequently include thyroid disease, neutrophilic dermatoses, polyarthritis, connective tissue diseases, vasculitis, and autoimmune cytopenias. Studies have found that autoimmune disease (or its therapy) is a risk factor for the development of myelodysplastic syndromes, but such syndromes might also be an instigator...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772431"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic syndrome and autoimmune disorders: two sides of the same coin?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772431/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220719211909&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Systemic inflammatory and autoimmune diseases and myelodysplastic syndromes have been linked in individual patients and in larger case series for at least 25 years. These associations frequently include thyroid disease, neutrophilic dermatoses, polyarthritis, connective tissue diseases, vasculitis, and autoimmune cytopenias. Studies have found that autoimmune disease (or its therapy) is a risk factor for the development of myelodysplastic syndromes, but such syndromes might also be an instigator...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772351"
                        open
                      >
                        <summary class="article-expander__title">Langerhans cell histiocytosis and associated malignancies: A retrospective analysis of 270 patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772351/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220719211909&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The largest collection of data on LCH-AM to date clearly indicates inherent relationships between specific types of AM and LCH, which may be due to therapy effects, clonal evolution, and germ-line predisposition, respectively. Prospective thorough genetic analysis is warranted and will hopefully shed light on the association of LCH and second neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772431"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic syndrome and autoimmune disorders: two sides of the same coin?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772431/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220718212208&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Systemic inflammatory and autoimmune diseases and myelodysplastic syndromes have been linked in individual patients and in larger case series for at least 25 years. These associations frequently include thyroid disease, neutrophilic dermatoses, polyarthritis, connective tissue diseases, vasculitis, and autoimmune cytopenias. Studies have found that autoimmune disease (or its therapy) is a risk factor for the development of myelodysplastic syndromes, but such syndromes might also be an instigator...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772351"
                        open
                      >
                        <summary class="article-expander__title">Langerhans cell histiocytosis and associated malignancies: A retrospective analysis of 270 patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772351/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220718212208&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The largest collection of data on LCH-AM to date clearly indicates inherent relationships between specific types of AM and LCH, which may be due to therapy effects, clonal evolution, and germ-line predisposition, respectively. Prospective thorough genetic analysis is warranted and will hopefully shed light on the association of LCH and second neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35771847"
                        open
                      >
                        <summary class="article-expander__title">Interferon-alpha2 treatment of patients with polycythemia vera and related neoplasms favorably impacts deregulation of oxidative stress genes and antioxidative defense mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35771847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220718212208&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic inflammation is considered a major driving force for clonal expansion and evolution in the Philadelphia-negative myeloproliferative neoplasms, which include essential thrombocythemia, polycythemia vera and primary myelofibrosis (MPNs). One of the key mutation drivers is the JAK2V617F mutation, which has been shown to induce the generation of reactive oxygen species (ROS). Using whole blood gene expression profiling, deregulation of several oxidative stress and anti-oxidative defense...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35770493"
                        open
                      >
                        <summary class="article-expander__title">To B- or not to B-: A review of lineage switched acute leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35770493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220718212208&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute leukemia is a heterogeneous disorder of hematologic malignancies composed primarily of hematopoietic precursors that have acquired unregulated self-renewal and proliferation. Hematology classification systems typically divide these neoplasms into lymphoid (B- or T-) and myeloid-lineage subtypes, with therapy dependent upon this distinction. Infrequently, certain acute leukemias may undergo a complete lineage switch at relapse, subsequently complicating the diagnosis and treatment of these...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772431"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic syndrome and autoimmune disorders: two sides of the same coin?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772431/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220717211412&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Systemic inflammatory and autoimmune diseases and myelodysplastic syndromes have been linked in individual patients and in larger case series for at least 25 years. These associations frequently include thyroid disease, neutrophilic dermatoses, polyarthritis, connective tissue diseases, vasculitis, and autoimmune cytopenias. Studies have found that autoimmune disease (or its therapy) is a risk factor for the development of myelodysplastic syndromes, but such syndromes might also be an instigator...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772351"
                        open
                      >
                        <summary class="article-expander__title">Langerhans cell histiocytosis and associated malignancies: A retrospective analysis of 270 patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772351/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220717211412&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The largest collection of data on LCH-AM to date clearly indicates inherent relationships between specific types of AM and LCH, which may be due to therapy effects, clonal evolution, and germ-line predisposition, respectively. Prospective thorough genetic analysis is warranted and will hopefully shed light on the association of LCH and second neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35771847"
                        open
                      >
                        <summary class="article-expander__title">Interferon-alpha2 treatment of patients with polycythemia vera and related neoplasms favorably impacts deregulation of oxidative stress genes and antioxidative defense mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35771847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220717211412&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic inflammation is considered a major driving force for clonal expansion and evolution in the Philadelphia-negative myeloproliferative neoplasms, which include essential thrombocythemia, polycythemia vera and primary myelofibrosis (MPNs). One of the key mutation drivers is the JAK2V617F mutation, which has been shown to induce the generation of reactive oxygen species (ROS). Using whole blood gene expression profiling, deregulation of several oxidative stress and anti-oxidative defense...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35770493"
                        open
                      >
                        <summary class="article-expander__title">To B- or not to B-: A review of lineage switched acute leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35770493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220717211412&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute leukemia is a heterogeneous disorder of hematologic malignancies composed primarily of hematopoietic precursors that have acquired unregulated self-renewal and proliferation. Hematology classification systems typically divide these neoplasms into lymphoid (B- or T-) and myeloid-lineage subtypes, with therapy dependent upon this distinction. Infrequently, certain acute leukemias may undergo a complete lineage switch at relapse, subsequently complicating the diagnosis and treatment of these...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772431"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic syndrome and autoimmune disorders: two sides of the same coin?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772431/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220716212224&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Systemic inflammatory and autoimmune diseases and myelodysplastic syndromes have been linked in individual patients and in larger case series for at least 25 years. These associations frequently include thyroid disease, neutrophilic dermatoses, polyarthritis, connective tissue diseases, vasculitis, and autoimmune cytopenias. Studies have found that autoimmune disease (or its therapy) is a risk factor for the development of myelodysplastic syndromes, but such syndromes might also be an instigator...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772351"
                        open
                      >
                        <summary class="article-expander__title">Langerhans cell histiocytosis and associated malignancies: A retrospective analysis of 270 patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772351/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220716212224&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The largest collection of data on LCH-AM to date clearly indicates inherent relationships between specific types of AM and LCH, which may be due to therapy effects, clonal evolution, and germ-line predisposition, respectively. Prospective thorough genetic analysis is warranted and will hopefully shed light on the association of LCH and second neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35771847"
                        open
                      >
                        <summary class="article-expander__title">Interferon-alpha2 treatment of patients with polycythemia vera and related neoplasms favorably impacts deregulation of oxidative stress genes and antioxidative defense mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35771847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220716212224&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic inflammation is considered a major driving force for clonal expansion and evolution in the Philadelphia-negative myeloproliferative neoplasms, which include essential thrombocythemia, polycythemia vera and primary myelofibrosis (MPNs). One of the key mutation drivers is the JAK2V617F mutation, which has been shown to induce the generation of reactive oxygen species (ROS). Using whole blood gene expression profiling, deregulation of several oxidative stress and anti-oxidative defense...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35770493"
                        open
                      >
                        <summary class="article-expander__title">To B- or not to B-: A review of lineage switched acute leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35770493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220716212224&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute leukemia is a heterogeneous disorder of hematologic malignancies composed primarily of hematopoietic precursors that have acquired unregulated self-renewal and proliferation. Hematology classification systems typically divide these neoplasms into lymphoid (B- or T-) and myeloid-lineage subtypes, with therapy dependent upon this distinction. Infrequently, certain acute leukemias may undergo a complete lineage switch at relapse, subsequently complicating the diagnosis and treatment of these...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772431"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic syndrome and autoimmune disorders: two sides of the same coin?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772431/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220715211735&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Systemic inflammatory and autoimmune diseases and myelodysplastic syndromes have been linked in individual patients and in larger case series for at least 25 years. These associations frequently include thyroid disease, neutrophilic dermatoses, polyarthritis, connective tissue diseases, vasculitis, and autoimmune cytopenias. Studies have found that autoimmune disease (or its therapy) is a risk factor for the development of myelodysplastic syndromes, but such syndromes might also be an instigator...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772351"
                        open
                      >
                        <summary class="article-expander__title">Langerhans cell histiocytosis and associated malignancies: A retrospective analysis of 270 patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772351/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220715211735&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The largest collection of data on LCH-AM to date clearly indicates inherent relationships between specific types of AM and LCH, which may be due to therapy effects, clonal evolution, and germ-line predisposition, respectively. Prospective thorough genetic analysis is warranted and will hopefully shed light on the association of LCH and second neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35771847"
                        open
                      >
                        <summary class="article-expander__title">Interferon-alpha2 treatment of patients with polycythemia vera and related neoplasms favorably impacts deregulation of oxidative stress genes and antioxidative defense mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35771847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220715211735&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic inflammation is considered a major driving force for clonal expansion and evolution in the Philadelphia-negative myeloproliferative neoplasms, which include essential thrombocythemia, polycythemia vera and primary myelofibrosis (MPNs). One of the key mutation drivers is the JAK2V617F mutation, which has been shown to induce the generation of reactive oxygen species (ROS). Using whole blood gene expression profiling, deregulation of several oxidative stress and anti-oxidative defense...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35770493"
                        open
                      >
                        <summary class="article-expander__title">To B- or not to B-: A review of lineage switched acute leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35770493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220715211735&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute leukemia is a heterogeneous disorder of hematologic malignancies composed primarily of hematopoietic precursors that have acquired unregulated self-renewal and proliferation. Hematology classification systems typically divide these neoplasms into lymphoid (B- or T-) and myeloid-lineage subtypes, with therapy dependent upon this distinction. Infrequently, certain acute leukemias may undergo a complete lineage switch at relapse, subsequently complicating the diagnosis and treatment of these...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772431"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic syndrome and autoimmune disorders: two sides of the same coin?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772431/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Systemic inflammatory and autoimmune diseases and myelodysplastic syndromes have been linked in individual patients and in larger case series for at least 25 years. These associations frequently include thyroid disease, neutrophilic dermatoses, polyarthritis, connective tissue diseases, vasculitis, and autoimmune cytopenias. Studies have found that autoimmune disease (or its therapy) is a risk factor for the development of myelodysplastic syndromes, but such syndromes might also be an instigator...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772351"
                        open
                      >
                        <summary class="article-expander__title">Langerhans cell histiocytosis and associated malignancies: A retrospective analysis of 270 patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772351/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The largest collection of data on LCH-AM to date clearly indicates inherent relationships between specific types of AM and LCH, which may be due to therapy effects, clonal evolution, and germ-line predisposition, respectively. Prospective thorough genetic analysis is warranted and will hopefully shed light on the association of LCH and second neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35771847"
                        open
                      >
                        <summary class="article-expander__title">Interferon-alpha2 treatment of patients with polycythemia vera and related neoplasms favorably impacts deregulation of oxidative stress genes and antioxidative defense mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35771847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic inflammation is considered a major driving force for clonal expansion and evolution in the Philadelphia-negative myeloproliferative neoplasms, which include essential thrombocythemia, polycythemia vera and primary myelofibrosis (MPNs). One of the key mutation drivers is the JAK2V617F mutation, which has been shown to induce the generation of reactive oxygen species (ROS). Using whole blood gene expression profiling, deregulation of several oxidative stress and anti-oxidative defense...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35770493"
                        open
                      >
                        <summary class="article-expander__title">To B- or not to B-: A review of lineage switched acute leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35770493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute leukemia is a heterogeneous disorder of hematologic malignancies composed primarily of hematopoietic precursors that have acquired unregulated self-renewal and proliferation. Hematology classification systems typically divide these neoplasms into lymphoid (B- or T-) and myeloid-lineage subtypes, with therapy dependent upon this distinction. Infrequently, certain acute leukemias may undergo a complete lineage switch at relapse, subsequently complicating the diagnosis and treatment of these...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772431"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic syndrome and autoimmune disorders: two sides of the same coin?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772431/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Systemic inflammatory and autoimmune diseases and myelodysplastic syndromes have been linked in individual patients and in larger case series for at least 25 years. These associations frequently include thyroid disease, neutrophilic dermatoses, polyarthritis, connective tissue diseases, vasculitis, and autoimmune cytopenias. Studies have found that autoimmune disease (or its therapy) is a risk factor for the development of myelodysplastic syndromes, but such syndromes might also be an instigator...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772351"
                        open
                      >
                        <summary class="article-expander__title">Langerhans cell histiocytosis and associated malignancies: A retrospective analysis of 270 patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772351/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The largest collection of data on LCH-AM to date clearly indicates inherent relationships between specific types of AM and LCH, which may be due to therapy effects, clonal evolution, and germ-line predisposition, respectively. Prospective thorough genetic analysis is warranted and will hopefully shed light on the association of LCH and second neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35771847"
                        open
                      >
                        <summary class="article-expander__title">Interferon-alpha2 treatment of patients with polycythemia vera and related neoplasms favorably impacts deregulation of oxidative stress genes and antioxidative defense mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35771847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic inflammation is considered a major driving force for clonal expansion and evolution in the Philadelphia-negative myeloproliferative neoplasms, which include essential thrombocythemia, polycythemia vera and primary myelofibrosis (MPNs). One of the key mutation drivers is the JAK2V617F mutation, which has been shown to induce the generation of reactive oxygen species (ROS). Using whole blood gene expression profiling, deregulation of several oxidative stress and anti-oxidative defense...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35770493"
                        open
                      >
                        <summary class="article-expander__title">To B- or not to B-: A review of lineage switched acute leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35770493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute leukemia is a heterogeneous disorder of hematologic malignancies composed primarily of hematopoietic precursors that have acquired unregulated self-renewal and proliferation. Hematology classification systems typically divide these neoplasms into lymphoid (B- or T-) and myeloid-lineage subtypes, with therapy dependent upon this distinction. Infrequently, certain acute leukemias may undergo a complete lineage switch at relapse, subsequently complicating the diagnosis and treatment of these...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772431"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic syndrome and autoimmune disorders: two sides of the same coin?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772431/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Systemic inflammatory and autoimmune diseases and myelodysplastic syndromes have been linked in individual patients and in larger case series for at least 25 years. These associations frequently include thyroid disease, neutrophilic dermatoses, polyarthritis, connective tissue diseases, vasculitis, and autoimmune cytopenias. Studies have found that autoimmune disease (or its therapy) is a risk factor for the development of myelodysplastic syndromes, but such syndromes might also be an instigator...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772351"
                        open
                      >
                        <summary class="article-expander__title">Langerhans cell histiocytosis and associated malignancies: A retrospective analysis of 270 patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772351/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The largest collection of data on LCH-AM to date clearly indicates inherent relationships between specific types of AM and LCH, which may be due to therapy effects, clonal evolution, and germ-line predisposition, respectively. Prospective thorough genetic analysis is warranted and will hopefully shed light on the association of LCH and second neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35771847"
                        open
                      >
                        <summary class="article-expander__title">Interferon-alpha2 treatment of patients with polycythemia vera and related neoplasms favorably impacts deregulation of oxidative stress genes and antioxidative defense mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35771847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic inflammation is considered a major driving force for clonal expansion and evolution in the Philadelphia-negative myeloproliferative neoplasms, which include essential thrombocythemia, polycythemia vera and primary myelofibrosis (MPNs). One of the key mutation drivers is the JAK2V617F mutation, which has been shown to induce the generation of reactive oxygen species (ROS). Using whole blood gene expression profiling, deregulation of several oxidative stress and anti-oxidative defense...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35770493"
                        open
                      >
                        <summary class="article-expander__title">To B- or not to B-: A review of lineage switched acute leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35770493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute leukemia is a heterogeneous disorder of hematologic malignancies composed primarily of hematopoietic precursors that have acquired unregulated self-renewal and proliferation. Hematology classification systems typically divide these neoplasms into lymphoid (B- or T-) and myeloid-lineage subtypes, with therapy dependent upon this distinction. Infrequently, certain acute leukemias may undergo a complete lineage switch at relapse, subsequently complicating the diagnosis and treatment of these...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772431"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic syndrome and autoimmune disorders: two sides of the same coin?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772431/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Systemic inflammatory and autoimmune diseases and myelodysplastic syndromes have been linked in individual patients and in larger case series for at least 25 years. These associations frequently include thyroid disease, neutrophilic dermatoses, polyarthritis, connective tissue diseases, vasculitis, and autoimmune cytopenias. Studies have found that autoimmune disease (or its therapy) is a risk factor for the development of myelodysplastic syndromes, but such syndromes might also be an instigator...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772351"
                        open
                      >
                        <summary class="article-expander__title">Langerhans cell histiocytosis and associated malignancies: A retrospective analysis of 270 patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772351/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The largest collection of data on LCH-AM to date clearly indicates inherent relationships between specific types of AM and LCH, which may be due to therapy effects, clonal evolution, and germ-line predisposition, respectively. Prospective thorough genetic analysis is warranted and will hopefully shed light on the association of LCH and second neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35771847"
                        open
                      >
                        <summary class="article-expander__title">Interferon-alpha2 treatment of patients with polycythemia vera and related neoplasms favorably impacts deregulation of oxidative stress genes and antioxidative defense mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35771847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic inflammation is considered a major driving force for clonal expansion and evolution in the Philadelphia-negative myeloproliferative neoplasms, which include essential thrombocythemia, polycythemia vera and primary myelofibrosis (MPNs). One of the key mutation drivers is the JAK2V617F mutation, which has been shown to induce the generation of reactive oxygen species (ROS). Using whole blood gene expression profiling, deregulation of several oxidative stress and anti-oxidative defense...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35770493"
                        open
                      >
                        <summary class="article-expander__title">To B- or not to B-: A review of lineage switched acute leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35770493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute leukemia is a heterogeneous disorder of hematologic malignancies composed primarily of hematopoietic precursors that have acquired unregulated self-renewal and proliferation. Hematology classification systems typically divide these neoplasms into lymphoid (B- or T-) and myeloid-lineage subtypes, with therapy dependent upon this distinction. Infrequently, certain acute leukemias may undergo a complete lineage switch at relapse, subsequently complicating the diagnosis and treatment of these...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772431"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic syndrome and autoimmune disorders: two sides of the same coin?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772431/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Systemic inflammatory and autoimmune diseases and myelodysplastic syndromes have been linked in individual patients and in larger case series for at least 25 years. These associations frequently include thyroid disease, neutrophilic dermatoses, polyarthritis, connective tissue diseases, vasculitis, and autoimmune cytopenias. Studies have found that autoimmune disease (or its therapy) is a risk factor for the development of myelodysplastic syndromes, but such syndromes might also be an instigator...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772351"
                        open
                      >
                        <summary class="article-expander__title">Langerhans cell histiocytosis and associated malignancies: A retrospective analysis of 270 patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772351/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The largest collection of data on LCH-AM to date clearly indicates inherent relationships between specific types of AM and LCH, which may be due to therapy effects, clonal evolution, and germ-line predisposition, respectively. Prospective thorough genetic analysis is warranted and will hopefully shed light on the association of LCH and second neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35771847"
                        open
                      >
                        <summary class="article-expander__title">Interferon-alpha2 treatment of patients with polycythemia vera and related neoplasms favorably impacts deregulation of oxidative stress genes and antioxidative defense mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35771847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic inflammation is considered a major driving force for clonal expansion and evolution in the Philadelphia-negative myeloproliferative neoplasms, which include essential thrombocythemia, polycythemia vera and primary myelofibrosis (MPNs). One of the key mutation drivers is the JAK2V617F mutation, which has been shown to induce the generation of reactive oxygen species (ROS). Using whole blood gene expression profiling, deregulation of several oxidative stress and anti-oxidative defense...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35770493"
                        open
                      >
                        <summary class="article-expander__title">To B- or not to B-: A review of lineage switched acute leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35770493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute leukemia is a heterogeneous disorder of hematologic malignancies composed primarily of hematopoietic precursors that have acquired unregulated self-renewal and proliferation. Hematology classification systems typically divide these neoplasms into lymphoid (B- or T-) and myeloid-lineage subtypes, with therapy dependent upon this distinction. Infrequently, certain acute leukemias may undergo a complete lineage switch at relapse, subsequently complicating the diagnosis and treatment of these...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772431"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic syndrome and autoimmune disorders: two sides of the same coin?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772431/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Systemic inflammatory and autoimmune diseases and myelodysplastic syndromes have been linked in individual patients and in larger case series for at least 25 years. These associations frequently include thyroid disease, neutrophilic dermatoses, polyarthritis, connective tissue diseases, vasculitis, and autoimmune cytopenias. Studies have found that autoimmune disease (or its therapy) is a risk factor for the development of myelodysplastic syndromes, but such syndromes might also be an instigator...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772351"
                        open
                      >
                        <summary class="article-expander__title">Langerhans cell histiocytosis and associated malignancies: A retrospective analysis of 270 patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772351/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The largest collection of data on LCH-AM to date clearly indicates inherent relationships between specific types of AM and LCH, which may be due to therapy effects, clonal evolution, and germ-line predisposition, respectively. Prospective thorough genetic analysis is warranted and will hopefully shed light on the association of LCH and second neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35771847"
                        open
                      >
                        <summary class="article-expander__title">Interferon-alpha2 treatment of patients with polycythemia vera and related neoplasms favorably impacts deregulation of oxidative stress genes and antioxidative defense mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35771847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic inflammation is considered a major driving force for clonal expansion and evolution in the Philadelphia-negative myeloproliferative neoplasms, which include essential thrombocythemia, polycythemia vera and primary myelofibrosis (MPNs). One of the key mutation drivers is the JAK2V617F mutation, which has been shown to induce the generation of reactive oxygen species (ROS). Using whole blood gene expression profiling, deregulation of several oxidative stress and anti-oxidative defense...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35770493"
                        open
                      >
                        <summary class="article-expander__title">To B- or not to B-: A review of lineage switched acute leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35770493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute leukemia is a heterogeneous disorder of hematologic malignancies composed primarily of hematopoietic precursors that have acquired unregulated self-renewal and proliferation. Hematology classification systems typically divide these neoplasms into lymphoid (B- or T-) and myeloid-lineage subtypes, with therapy dependent upon this distinction. Infrequently, certain acute leukemias may undergo a complete lineage switch at relapse, subsequently complicating the diagnosis and treatment of these...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772431"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic syndrome and autoimmune disorders: two sides of the same coin?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772431/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220708210406&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Systemic inflammatory and autoimmune diseases and myelodysplastic syndromes have been linked in individual patients and in larger case series for at least 25 years. These associations frequently include thyroid disease, neutrophilic dermatoses, polyarthritis, connective tissue diseases, vasculitis, and autoimmune cytopenias. Studies have found that autoimmune disease (or its therapy) is a risk factor for the development of myelodysplastic syndromes, but such syndromes might also be an instigator...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772351"
                        open
                      >
                        <summary class="article-expander__title">Langerhans cell histiocytosis and associated malignancies: A retrospective analysis of 270 patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772351/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220708210406&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The largest collection of data on LCH-AM to date clearly indicates inherent relationships between specific types of AM and LCH, which may be due to therapy effects, clonal evolution, and germ-line predisposition, respectively. Prospective thorough genetic analysis is warranted and will hopefully shed light on the association of LCH and second neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35771847"
                        open
                      >
                        <summary class="article-expander__title">Interferon-alpha2 treatment of patients with polycythemia vera and related neoplasms favorably impacts deregulation of oxidative stress genes and antioxidative defense mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35771847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220708210406&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic inflammation is considered a major driving force for clonal expansion and evolution in the Philadelphia-negative myeloproliferative neoplasms, which include essential thrombocythemia, polycythemia vera and primary myelofibrosis (MPNs). One of the key mutation drivers is the JAK2V617F mutation, which has been shown to induce the generation of reactive oxygen species (ROS). Using whole blood gene expression profiling, deregulation of several oxidative stress and anti-oxidative defense...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35770493"
                        open
                      >
                        <summary class="article-expander__title">To B- or not to B-: A review of lineage switched acute leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35770493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220708210406&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute leukemia is a heterogeneous disorder of hematologic malignancies composed primarily of hematopoietic precursors that have acquired unregulated self-renewal and proliferation. Hematology classification systems typically divide these neoplasms into lymphoid (B- or T-) and myeloid-lineage subtypes, with therapy dependent upon this distinction. Infrequently, certain acute leukemias may undergo a complete lineage switch at relapse, subsequently complicating the diagnosis and treatment of these...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772431"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic syndrome and autoimmune disorders: two sides of the same coin?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772431/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220707211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Systemic inflammatory and autoimmune diseases and myelodysplastic syndromes have been linked in individual patients and in larger case series for at least 25 years. These associations frequently include thyroid disease, neutrophilic dermatoses, polyarthritis, connective tissue diseases, vasculitis, and autoimmune cytopenias. Studies have found that autoimmune disease (or its therapy) is a risk factor for the development of myelodysplastic syndromes, but such syndromes might also be an instigator...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772351"
                        open
                      >
                        <summary class="article-expander__title">Langerhans cell histiocytosis and associated malignancies: A retrospective analysis of 270 patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772351/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220707211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The largest collection of data on LCH-AM to date clearly indicates inherent relationships between specific types of AM and LCH, which may be due to therapy effects, clonal evolution, and germ-line predisposition, respectively. Prospective thorough genetic analysis is warranted and will hopefully shed light on the association of LCH and second neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35771847"
                        open
                      >
                        <summary class="article-expander__title">Interferon-alpha2 treatment of patients with polycythemia vera and related neoplasms favorably impacts deregulation of oxidative stress genes and antioxidative defense mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35771847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220707211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic inflammation is considered a major driving force for clonal expansion and evolution in the Philadelphia-negative myeloproliferative neoplasms, which include essential thrombocythemia, polycythemia vera and primary myelofibrosis (MPNs). One of the key mutation drivers is the JAK2V617F mutation, which has been shown to induce the generation of reactive oxygen species (ROS). Using whole blood gene expression profiling, deregulation of several oxidative stress and anti-oxidative defense...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35770493"
                        open
                      >
                        <summary class="article-expander__title">To B- or not to B-: A review of lineage switched acute leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35770493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220707211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute leukemia is a heterogeneous disorder of hematologic malignancies composed primarily of hematopoietic precursors that have acquired unregulated self-renewal and proliferation. Hematology classification systems typically divide these neoplasms into lymphoid (B- or T-) and myeloid-lineage subtypes, with therapy dependent upon this distinction. Infrequently, certain acute leukemias may undergo a complete lineage switch at relapse, subsequently complicating the diagnosis and treatment of these...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772431"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic syndrome and autoimmune disorders: two sides of the same coin?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772431/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220706212003&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Systemic inflammatory and autoimmune diseases and myelodysplastic syndromes have been linked in individual patients and in larger case series for at least 25 years. These associations frequently include thyroid disease, neutrophilic dermatoses, polyarthritis, connective tissue diseases, vasculitis, and autoimmune cytopenias. Studies have found that autoimmune disease (or its therapy) is a risk factor for the development of myelodysplastic syndromes, but such syndromes might also be an instigator...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772351"
                        open
                      >
                        <summary class="article-expander__title">Langerhans cell histiocytosis and associated malignancies: A retrospective analysis of 270 patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772351/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220706212003&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The largest collection of data on LCH-AM to date clearly indicates inherent relationships between specific types of AM and LCH, which may be due to therapy effects, clonal evolution, and germ-line predisposition, respectively. Prospective thorough genetic analysis is warranted and will hopefully shed light on the association of LCH and second neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35771847"
                        open
                      >
                        <summary class="article-expander__title">Interferon-alpha2 treatment of patients with polycythemia vera and related neoplasms favorably impacts deregulation of oxidative stress genes and antioxidative defense mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35771847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220706212003&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic inflammation is considered a major driving force for clonal expansion and evolution in the Philadelphia-negative myeloproliferative neoplasms, which include essential thrombocythemia, polycythemia vera and primary myelofibrosis (MPNs). One of the key mutation drivers is the JAK2V617F mutation, which has been shown to induce the generation of reactive oxygen species (ROS). Using whole blood gene expression profiling, deregulation of several oxidative stress and anti-oxidative defense...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35770493"
                        open
                      >
                        <summary class="article-expander__title">To B- or not to B-: A review of lineage switched acute leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35770493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220706212003&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute leukemia is a heterogeneous disorder of hematologic malignancies composed primarily of hematopoietic precursors that have acquired unregulated self-renewal and proliferation. Hematology classification systems typically divide these neoplasms into lymphoid (B- or T-) and myeloid-lineage subtypes, with therapy dependent upon this distinction. Infrequently, certain acute leukemias may undergo a complete lineage switch at relapse, subsequently complicating the diagnosis and treatment of these...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772431"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic syndrome and autoimmune disorders: two sides of the same coin?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772431/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220705212255&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Systemic inflammatory and autoimmune diseases and myelodysplastic syndromes have been linked in individual patients and in larger case series for at least 25 years. These associations frequently include thyroid disease, neutrophilic dermatoses, polyarthritis, connective tissue diseases, vasculitis, and autoimmune cytopenias. Studies have found that autoimmune disease (or its therapy) is a risk factor for the development of myelodysplastic syndromes, but such syndromes might also be an instigator...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772351"
                        open
                      >
                        <summary class="article-expander__title">Langerhans cell histiocytosis and associated malignancies: A retrospective analysis of 270 patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772351/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220705212255&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The largest collection of data on LCH-AM to date clearly indicates inherent relationships between specific types of AM and LCH, which may be due to therapy effects, clonal evolution, and germ-line predisposition, respectively. Prospective thorough genetic analysis is warranted and will hopefully shed light on the association of LCH and second neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35771847"
                        open
                      >
                        <summary class="article-expander__title">Interferon-alpha2 treatment of patients with polycythemia vera and related neoplasms favorably impacts deregulation of oxidative stress genes and antioxidative defense mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35771847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220705212255&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic inflammation is considered a major driving force for clonal expansion and evolution in the Philadelphia-negative myeloproliferative neoplasms, which include essential thrombocythemia, polycythemia vera and primary myelofibrosis (MPNs). One of the key mutation drivers is the JAK2V617F mutation, which has been shown to induce the generation of reactive oxygen species (ROS). Using whole blood gene expression profiling, deregulation of several oxidative stress and anti-oxidative defense...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35770493"
                        open
                      >
                        <summary class="article-expander__title">To B- or not to B-: A review of lineage switched acute leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35770493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220705212255&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute leukemia is a heterogeneous disorder of hematologic malignancies composed primarily of hematopoietic precursors that have acquired unregulated self-renewal and proliferation. Hematology classification systems typically divide these neoplasms into lymphoid (B- or T-) and myeloid-lineage subtypes, with therapy dependent upon this distinction. Infrequently, certain acute leukemias may undergo a complete lineage switch at relapse, subsequently complicating the diagnosis and treatment of these...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772431"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic syndrome and autoimmune disorders: two sides of the same coin?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772431/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220704211122&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Systemic inflammatory and autoimmune diseases and myelodysplastic syndromes have been linked in individual patients and in larger case series for at least 25 years. These associations frequently include thyroid disease, neutrophilic dermatoses, polyarthritis, connective tissue diseases, vasculitis, and autoimmune cytopenias. Studies have found that autoimmune disease (or its therapy) is a risk factor for the development of myelodysplastic syndromes, but such syndromes might also be an instigator...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772351"
                        open
                      >
                        <summary class="article-expander__title">Langerhans cell histiocytosis and associated malignancies: A retrospective analysis of 270 patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772351/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220704211122&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The largest collection of data on LCH-AM to date clearly indicates inherent relationships between specific types of AM and LCH, which may be due to therapy effects, clonal evolution, and germ-line predisposition, respectively. Prospective thorough genetic analysis is warranted and will hopefully shed light on the association of LCH and second neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35771847"
                        open
                      >
                        <summary class="article-expander__title">Interferon-alpha2 treatment of patients with polycythemia vera and related neoplasms favorably impacts deregulation of oxidative stress genes and antioxidative defense mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35771847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220704211122&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic inflammation is considered a major driving force for clonal expansion and evolution in the Philadelphia-negative myeloproliferative neoplasms, which include essential thrombocythemia, polycythemia vera and primary myelofibrosis (MPNs). One of the key mutation drivers is the JAK2V617F mutation, which has been shown to induce the generation of reactive oxygen species (ROS). Using whole blood gene expression profiling, deregulation of several oxidative stress and anti-oxidative defense...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35770493"
                        open
                      >
                        <summary class="article-expander__title">To B- or not to B-: A review of lineage switched acute leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35770493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220704211122&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute leukemia is a heterogeneous disorder of hematologic malignancies composed primarily of hematopoietic precursors that have acquired unregulated self-renewal and proliferation. Hematology classification systems typically divide these neoplasms into lymphoid (B- or T-) and myeloid-lineage subtypes, with therapy dependent upon this distinction. Infrequently, certain acute leukemias may undergo a complete lineage switch at relapse, subsequently complicating the diagnosis and treatment of these...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772431"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic syndrome and autoimmune disorders: two sides of the same coin?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772431/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220703212221&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Systemic inflammatory and autoimmune diseases and myelodysplastic syndromes have been linked in individual patients and in larger case series for at least 25 years. These associations frequently include thyroid disease, neutrophilic dermatoses, polyarthritis, connective tissue diseases, vasculitis, and autoimmune cytopenias. Studies have found that autoimmune disease (or its therapy) is a risk factor for the development of myelodysplastic syndromes, but such syndromes might also be an instigator...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772351"
                        open
                      >
                        <summary class="article-expander__title">Langerhans cell histiocytosis and associated malignancies: A retrospective analysis of 270 patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772351/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220703212221&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The largest collection of data on LCH-AM to date clearly indicates inherent relationships between specific types of AM and LCH, which may be due to therapy effects, clonal evolution, and germ-line predisposition, respectively. Prospective thorough genetic analysis is warranted and will hopefully shed light on the association of LCH and second neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35771847"
                        open
                      >
                        <summary class="article-expander__title">Interferon-alpha2 treatment of patients with polycythemia vera and related neoplasms favorably impacts deregulation of oxidative stress genes and antioxidative defense mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35771847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220703212221&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic inflammation is considered a major driving force for clonal expansion and evolution in the Philadelphia-negative myeloproliferative neoplasms, which include essential thrombocythemia, polycythemia vera and primary myelofibrosis (MPNs). One of the key mutation drivers is the JAK2V617F mutation, which has been shown to induce the generation of reactive oxygen species (ROS). Using whole blood gene expression profiling, deregulation of several oxidative stress and anti-oxidative defense...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35770493"
                        open
                      >
                        <summary class="article-expander__title">To B- or not to B-: A review of lineage switched acute leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35770493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220703212221&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute leukemia is a heterogeneous disorder of hematologic malignancies composed primarily of hematopoietic precursors that have acquired unregulated self-renewal and proliferation. Hematology classification systems typically divide these neoplasms into lymphoid (B- or T-) and myeloid-lineage subtypes, with therapy dependent upon this distinction. Infrequently, certain acute leukemias may undergo a complete lineage switch at relapse, subsequently complicating the diagnosis and treatment of these...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772431"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic syndrome and autoimmune disorders: two sides of the same coin?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772431/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220702211519&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Systemic inflammatory and autoimmune diseases and myelodysplastic syndromes have been linked in individual patients and in larger case series for at least 25 years. These associations frequently include thyroid disease, neutrophilic dermatoses, polyarthritis, connective tissue diseases, vasculitis, and autoimmune cytopenias. Studies have found that autoimmune disease (or its therapy) is a risk factor for the development of myelodysplastic syndromes, but such syndromes might also be an instigator...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772351"
                        open
                      >
                        <summary class="article-expander__title">Langerhans cell histiocytosis and associated malignancies: A retrospective analysis of 270 patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772351/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220702211519&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The largest collection of data on LCH-AM to date clearly indicates inherent relationships between specific types of AM and LCH, which may be due to therapy effects, clonal evolution, and germ-line predisposition, respectively. Prospective thorough genetic analysis is warranted and will hopefully shed light on the association of LCH and second neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35771847"
                        open
                      >
                        <summary class="article-expander__title">Interferon-alpha2 treatment of patients with polycythemia vera and related neoplasms favorably impacts deregulation of oxidative stress genes and antioxidative defense mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35771847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220702211519&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic inflammation is considered a major driving force for clonal expansion and evolution in the Philadelphia-negative myeloproliferative neoplasms, which include essential thrombocythemia, polycythemia vera and primary myelofibrosis (MPNs). One of the key mutation drivers is the JAK2V617F mutation, which has been shown to induce the generation of reactive oxygen species (ROS). Using whole blood gene expression profiling, deregulation of several oxidative stress and anti-oxidative defense...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35770493"
                        open
                      >
                        <summary class="article-expander__title">To B- or not to B-: A review of lineage switched acute leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35770493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220702211519&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute leukemia is a heterogeneous disorder of hematologic malignancies composed primarily of hematopoietic precursors that have acquired unregulated self-renewal and proliferation. Hematology classification systems typically divide these neoplasms into lymphoid (B- or T-) and myeloid-lineage subtypes, with therapy dependent upon this distinction. Infrequently, certain acute leukemias may undergo a complete lineage switch at relapse, subsequently complicating the diagnosis and treatment of these...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772431"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic syndrome and autoimmune disorders: two sides of the same coin?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772431/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220701211233&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Systemic inflammatory and autoimmune diseases and myelodysplastic syndromes have been linked in individual patients and in larger case series for at least 25 years. These associations frequently include thyroid disease, neutrophilic dermatoses, polyarthritis, connective tissue diseases, vasculitis, and autoimmune cytopenias. Studies have found that autoimmune disease (or its therapy) is a risk factor for the development of myelodysplastic syndromes, but such syndromes might also be an instigator...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772351"
                        open
                      >
                        <summary class="article-expander__title">Langerhans cell histiocytosis and associated malignancies: A retrospective analysis of 270 patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772351/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220701211233&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The largest collection of data on LCH-AM to date clearly indicates inherent relationships between specific types of AM and LCH, which may be due to therapy effects, clonal evolution, and germ-line predisposition, respectively. Prospective thorough genetic analysis is warranted and will hopefully shed light on the association of LCH and second neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35771847"
                        open
                      >
                        <summary class="article-expander__title">Interferon-alpha2 treatment of patients with polycythemia vera and related neoplasms favorably impacts deregulation of oxidative stress genes and antioxidative defense mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35771847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220701211233&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic inflammation is considered a major driving force for clonal expansion and evolution in the Philadelphia-negative myeloproliferative neoplasms, which include essential thrombocythemia, polycythemia vera and primary myelofibrosis (MPNs). One of the key mutation drivers is the JAK2V617F mutation, which has been shown to induce the generation of reactive oxygen species (ROS). Using whole blood gene expression profiling, deregulation of several oxidative stress and anti-oxidative defense...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35770493"
                        open
                      >
                        <summary class="article-expander__title">To B- or not to B-: A review of lineage switched acute leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35770493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220701211233&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute leukemia is a heterogeneous disorder of hematologic malignancies composed primarily of hematopoietic precursors that have acquired unregulated self-renewal and proliferation. Hematology classification systems typically divide these neoplasms into lymphoid (B- or T-) and myeloid-lineage subtypes, with therapy dependent upon this distinction. Infrequently, certain acute leukemias may undergo a complete lineage switch at relapse, subsequently complicating the diagnosis and treatment of these...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35771847"
                        open
                      >
                        <summary class="article-expander__title">Interferon-alpha2 treatment of patients with polycythemia vera and related neoplasms favorably impacts deregulation of oxidative stress genes and antioxidative defense mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35771847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220630212102&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic inflammation is considered a major driving force for clonal expansion and evolution in the Philadelphia-negative myeloproliferative neoplasms, which include essential thrombocythemia, polycythemia vera and primary myelofibrosis (MPNs). One of the key mutation drivers is the JAK2V617F mutation, which has been shown to induce the generation of reactive oxygen species (ROS). Using whole blood gene expression profiling, deregulation of several oxidative stress and anti-oxidative defense...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35770493"
                        open
                      >
                        <summary class="article-expander__title">To B- or not to B-: A review of lineage switched acute leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35770493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220630212102&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute leukemia is a heterogeneous disorder of hematologic malignancies composed primarily of hematopoietic precursors that have acquired unregulated self-renewal and proliferation. Hematology classification systems typically divide these neoplasms into lymphoid (B- or T-) and myeloid-lineage subtypes, with therapy dependent upon this distinction. Infrequently, certain acute leukemias may undergo a complete lineage switch at relapse, subsequently complicating the diagnosis and treatment of these...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-06-29 datetime="2022-06-29T10:00:00.000Z">2022-06-29</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-06-29 datetime="2022-06-29T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608010801&amp;v&#x3D;2.17.7&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35767937"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive analysis of intratumoural heterogeneity of somatic copy number alterations in diffuse glioma reveals clonality-dependent prognostic patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35767937/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220717211412&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The systematic analysis of ITH-SCNAs in large cohorts of DGs highlighted the importance of considering the clonality of SCNA in discovery of tumour prognostic markers.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35767937"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive analysis of intratumoural heterogeneity of somatic copy number alterations in diffuse glioma reveals clonality-dependent prognostic patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35767937/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220716212224&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The systematic analysis of ITH-SCNAs in large cohorts of DGs highlighted the importance of considering the clonality of SCNA in discovery of tumour prognostic markers.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35767937"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive analysis of intratumoural heterogeneity of somatic copy number alterations in diffuse glioma reveals clonality-dependent prognostic patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35767937/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220715211735&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The systematic analysis of ITH-SCNAs in large cohorts of DGs highlighted the importance of considering the clonality of SCNA in discovery of tumour prognostic markers.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35767937"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive analysis of intratumoural heterogeneity of somatic copy number alterations in diffuse glioma reveals clonality-dependent prognostic patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35767937/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The systematic analysis of ITH-SCNAs in large cohorts of DGs highlighted the importance of considering the clonality of SCNA in discovery of tumour prognostic markers.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35767937"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive analysis of intratumoural heterogeneity of somatic copy number alterations in diffuse glioma reveals clonality-dependent prognostic patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35767937/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The systematic analysis of ITH-SCNAs in large cohorts of DGs highlighted the importance of considering the clonality of SCNA in discovery of tumour prognostic markers.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35767937"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive analysis of intratumoural heterogeneity of somatic copy number alterations in diffuse glioma reveals clonality-dependent prognostic patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35767937/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The systematic analysis of ITH-SCNAs in large cohorts of DGs highlighted the importance of considering the clonality of SCNA in discovery of tumour prognostic markers.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35767937"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive analysis of intratumoural heterogeneity of somatic copy number alterations in diffuse glioma reveals clonality-dependent prognostic patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35767937/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The systematic analysis of ITH-SCNAs in large cohorts of DGs highlighted the importance of considering the clonality of SCNA in discovery of tumour prognostic markers.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35767937"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive analysis of intratumoural heterogeneity of somatic copy number alterations in diffuse glioma reveals clonality-dependent prognostic patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35767937/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The systematic analysis of ITH-SCNAs in large cohorts of DGs highlighted the importance of considering the clonality of SCNA in discovery of tumour prognostic markers.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35767937"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive analysis of intratumoural heterogeneity of somatic copy number alterations in diffuse glioma reveals clonality-dependent prognostic patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35767937/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The systematic analysis of ITH-SCNAs in large cohorts of DGs highlighted the importance of considering the clonality of SCNA in discovery of tumour prognostic markers.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35767937"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive analysis of intratumoural heterogeneity of somatic copy number alterations in diffuse glioma reveals clonality-dependent prognostic patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35767937/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220708210406&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The systematic analysis of ITH-SCNAs in large cohorts of DGs highlighted the importance of considering the clonality of SCNA in discovery of tumour prognostic markers.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35767937"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive analysis of intratumoural heterogeneity of somatic copy number alterations in diffuse glioma reveals clonality-dependent prognostic patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35767937/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220707211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The systematic analysis of ITH-SCNAs in large cohorts of DGs highlighted the importance of considering the clonality of SCNA in discovery of tumour prognostic markers.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35767937"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive Analysis of Intratumoural Heterogeneity of Somatic Copy Number Alterations in Diffuse Glioma Reveals Clonality-dependent Prognostic Patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35767937/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220706212003&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The systematic analysis of ITH-SCNAs in large cohorts of DGs highlighted the importance of considering the clonality of SCNA in discovery of tumour prognostic markers.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35767937"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive Analysis of Intratumoural Heterogeneity of Somatic Copy Number Alterations in Diffuse Glioma Reveals Clonality-dependent Prognostic Patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35767937/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220705212255&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The systematic analysis of ITH-SCNAs in large cohorts of DGs highlighted the importance of considering the clonality of SCNA in discovery of tumour prognostic markers.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35767937"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive Analysis of Intratumoural Heterogeneity of Somatic Copy Number Alterations in Diffuse Glioma Reveals Clonality-dependent Prognostic Patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35767937/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220704211122&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The systematic analysis of ITH-SCNAs in large cohorts of DGs highlighted the importance of considering the clonality of SCNA in discovery of tumour prognostic markers.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35767937"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive Analysis of Intratumoural Heterogeneity of Somatic Copy Number Alterations in Diffuse Glioma Reveals Clonality-dependent Prognostic Patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35767937/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220703212221&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The systematic analysis of ITH-SCNAs in large cohorts of DGs highlighted the importance of considering the clonality of SCNA in discovery of tumour prognostic markers.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35767937"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive Analysis of Intratumoural Heterogeneity of Somatic Copy Number Alterations in Diffuse Glioma Reveals Clonality-dependent Prognostic Patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35767937/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220702211519&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The systematic analysis of ITH-SCNAs in large cohorts of DGs highlighted the importance of considering the clonality of SCNA in discovery of tumour prognostic markers.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35767937"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive Analysis of Intratumoural Heterogeneity of Somatic Copy Number Alterations in Diffuse Glioma Reveals Clonality-dependent Prognostic Patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35767937/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220701211233&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The systematic analysis of ITH-SCNAs in large cohorts of DGs highlighted the importance of considering the clonality of SCNA in discovery of tumour prognostic markers.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35767937"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive Analysis of Intratumoural Heterogeneity of Somatic Copy Number Alterations in Diffuse Glioma Reveals Clonality-dependent Prognostic Patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35767937/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220630212102&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The systematic analysis of ITH-SCNAs in large cohorts of DGs highlighted the importance of considering the clonality of SCNA in discovery of tumour prognostic markers.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-06-28 datetime="2022-06-28T10:00:00.000Z">2022-06-28</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-06-28 datetime="2022-06-28T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608003622&amp;v&#x3D;2.17.7&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35764632"
                        open
                      >
                        <summary class="article-expander__title">Clonal reconstruction from co-occurrence of vector integration sites accurately quantifies expanding clones in vivo</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35764632/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>High transduction rates of viral vectors in gene therapies (GT) and experimental hematopoiesis ensure a high frequency of gene delivery, although multiple integration events can occur in the same cell. Therefore, tracing of integration sites (IS) leads to mis-quantification of the true clonal spectrum and limits safety considerations in GT. Hence, we use correlations between repeated measurements of IS abundances to estimate their mutual similarity and identify clusters of co-occurring IS, for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35764632"
                        open
                      >
                        <summary class="article-expander__title">Clonal reconstruction from co-occurrence of vector integration sites accurately quantifies expanding clones in vivo</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35764632/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>High transduction rates of viral vectors in gene therapies (GT) and experimental hematopoiesis ensure a high frequency of gene delivery, although multiple integration events can occur in the same cell. Therefore, tracing of integration sites (IS) leads to mis-quantification of the true clonal spectrum and limits safety considerations in GT. Hence, we use correlations between repeated measurements of IS abundances to estimate their mutual similarity and identify clusters of co-occurring IS, for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35764632"
                        open
                      >
                        <summary class="article-expander__title">Clonal reconstruction from co-occurrence of vector integration sites accurately quantifies expanding clones in vivo</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35764632/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>High transduction rates of viral vectors in gene therapies (GT) and experimental hematopoiesis ensure a high frequency of gene delivery, although multiple integration events can occur in the same cell. Therefore, tracing of integration sites (IS) leads to mis-quantification of the true clonal spectrum and limits safety considerations in GT. Hence, we use correlations between repeated measurements of IS abundances to estimate their mutual similarity and identify clusters of co-occurring IS, for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35764632"
                        open
                      >
                        <summary class="article-expander__title">Clonal reconstruction from co-occurrence of vector integration sites accurately quantifies expanding clones in vivo</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35764632/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>High transduction rates of viral vectors in gene therapies (GT) and experimental hematopoiesis ensure a high frequency of gene delivery, although multiple integration events can occur in the same cell. Therefore, tracing of integration sites (IS) leads to mis-quantification of the true clonal spectrum and limits safety considerations in GT. Hence, we use correlations between repeated measurements of IS abundances to estimate their mutual similarity and identify clusters of co-occurring IS, for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35764632"
                        open
                      >
                        <summary class="article-expander__title">Clonal reconstruction from co-occurrence of vector integration sites accurately quantifies expanding clones in vivo</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35764632/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>High transduction rates of viral vectors in gene therapies (GT) and experimental hematopoiesis ensure a high frequency of gene delivery, although multiple integration events can occur in the same cell. Therefore, tracing of integration sites (IS) leads to mis-quantification of the true clonal spectrum and limits safety considerations in GT. Hence, we use correlations between repeated measurements of IS abundances to estimate their mutual similarity and identify clusters of co-occurring IS, for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35764632"
                        open
                      >
                        <summary class="article-expander__title">Clonal reconstruction from co-occurrence of vector integration sites accurately quantifies expanding clones in vivo</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35764632/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220708210406&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>High transduction rates of viral vectors in gene therapies (GT) and experimental hematopoiesis ensure a high frequency of gene delivery, although multiple integration events can occur in the same cell. Therefore, tracing of integration sites (IS) leads to mis-quantification of the true clonal spectrum and limits safety considerations in GT. Hence, we use correlations between repeated measurements of IS abundances to estimate their mutual similarity and identify clusters of co-occurring IS, for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35764632"
                        open
                      >
                        <summary class="article-expander__title">Clonal reconstruction from co-occurrence of vector integration sites accurately quantifies expanding clones in vivo</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35764632/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220707211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>High transduction rates of viral vectors in gene therapies (GT) and experimental hematopoiesis ensure a high frequency of gene delivery, although multiple integration events can occur in the same cell. Therefore, tracing of integration sites (IS) leads to mis-quantification of the true clonal spectrum and limits safety considerations in GT. Hence, we use correlations between repeated measurements of IS abundances to estimate their mutual similarity and identify clusters of co-occurring IS, for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35764632"
                        open
                      >
                        <summary class="article-expander__title">Clonal reconstruction from co-occurrence of vector integration sites accurately quantifies expanding clones in vivo</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35764632/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220706212003&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>High transduction rates of viral vectors in gene therapies (GT) and experimental hematopoiesis ensure a high frequency of gene delivery, although multiple integration events can occur in the same cell. Therefore, tracing of integration sites (IS) leads to mis-quantification of the true clonal spectrum and limits safety considerations in GT. Hence, we use correlations between repeated measurements of IS abundances to estimate their mutual similarity and identify clusters of co-occurring IS, for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35764632"
                        open
                      >
                        <summary class="article-expander__title">Clonal reconstruction from co-occurrence of vector integration sites accurately quantifies expanding clones in vivo</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35764632/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220705212255&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>High transduction rates of viral vectors in gene therapies (GT) and experimental hematopoiesis ensure a high frequency of gene delivery, although multiple integration events can occur in the same cell. Therefore, tracing of integration sites (IS) leads to mis-quantification of the true clonal spectrum and limits safety considerations in GT. Hence, we use correlations between repeated measurements of IS abundances to estimate their mutual similarity and identify clusters of co-occurring IS, for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35764632"
                        open
                      >
                        <summary class="article-expander__title">Clonal reconstruction from co-occurrence of vector integration sites accurately quantifies expanding clones in vivo</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35764632/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220704211122&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>High transduction rates of viral vectors in gene therapies (GT) and experimental hematopoiesis ensure a high frequency of gene delivery, although multiple integration events can occur in the same cell. Therefore, tracing of integration sites (IS) leads to mis-quantification of the true clonal spectrum and limits safety considerations in GT. Hence, we use correlations between repeated measurements of IS abundances to estimate their mutual similarity and identify clusters of co-occurring IS, for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35764632"
                        open
                      >
                        <summary class="article-expander__title">Clonal reconstruction from co-occurrence of vector integration sites accurately quantifies expanding clones in vivo</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35764632/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220703212221&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>High transduction rates of viral vectors in gene therapies (GT) and experimental hematopoiesis ensure a high frequency of gene delivery, although multiple integration events can occur in the same cell. Therefore, tracing of integration sites (IS) leads to mis-quantification of the true clonal spectrum and limits safety considerations in GT. Hence, we use correlations between repeated measurements of IS abundances to estimate their mutual similarity and identify clusters of co-occurring IS, for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35764632"
                        open
                      >
                        <summary class="article-expander__title">Clonal reconstruction from co-occurrence of vector integration sites accurately quantifies expanding clones in vivo</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35764632/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220702211519&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>High transduction rates of viral vectors in gene therapies (GT) and experimental hematopoiesis ensure a high frequency of gene delivery, although multiple integration events can occur in the same cell. Therefore, tracing of integration sites (IS) leads to mis-quantification of the true clonal spectrum and limits safety considerations in GT. Hence, we use correlations between repeated measurements of IS abundances to estimate their mutual similarity and identify clusters of co-occurring IS, for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35764632"
                        open
                      >
                        <summary class="article-expander__title">Clonal reconstruction from co-occurrence of vector integration sites accurately quantifies expanding clones in vivo</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35764632/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220701211233&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>High transduction rates of viral vectors in gene therapies (GT) and experimental hematopoiesis ensure a high frequency of gene delivery, although multiple integration events can occur in the same cell. Therefore, tracing of integration sites (IS) leads to mis-quantification of the true clonal spectrum and limits safety considerations in GT. Hence, we use correlations between repeated measurements of IS abundances to estimate their mutual similarity and identify clusters of co-occurring IS, for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35764632"
                        open
                      >
                        <summary class="article-expander__title">Clonal reconstruction from co-occurrence of vector integration sites accurately quantifies expanding clones in vivo</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35764632/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220630212102&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>High transduction rates of viral vectors in gene therapies (GT) and experimental hematopoiesis ensure a high frequency of gene delivery, although multiple integration events can occur in the same cell. Therefore, tracing of integration sites (IS) leads to mis-quantification of the true clonal spectrum and limits safety considerations in GT. Hence, we use correlations between repeated measurements of IS abundances to estimate their mutual similarity and identify clusters of co-occurring IS, for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35764632"
                        open
                      >
                        <summary class="article-expander__title">Clonal reconstruction from co-occurrence of vector integration sites accurately quantifies expanding clones in vivo</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35764632/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220629210920&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>High transduction rates of viral vectors in gene therapies (GT) and experimental hematopoiesis ensure a high frequency of gene delivery, although multiple integration events can occur in the same cell. Therefore, tracing of integration sites (IS) leads to mis-quantification of the true clonal spectrum and limits safety considerations in GT. Hence, we use correlations between repeated measurements of IS abundances to estimate their mutual similarity and identify clusters of co-occurring IS, for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35764632"
                        open
                      >
                        <summary class="article-expander__title">Clonal reconstruction from co-occurrence of vector integration sites accurately quantifies expanding clones in vivo</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35764632/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220629090955&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>High transduction rates of viral vectors in gene therapies (GT) and experimental hematopoiesis ensure a high frequency of gene delivery, although multiple integration events can occur in the same cell. Therefore, tracing of integration sites (IS) leads to mis-quantification of the true clonal spectrum and limits safety considerations in GT. Hence, we use correlations between repeated measurements of IS abundances to estimate their mutual similarity and identify clusters of co-occurring IS, for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-06-27 datetime="2022-06-27T10:00:00.000Z">2022-06-27</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-06-27 datetime="2022-06-27T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220824212808&amp;fc&#x3D;20220608010801&amp;v&#x3D;2.17.7&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35758777"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing tumor clonal lineage trees incorporating single-nucleotide variants, copy number alterations and structural variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35758777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220715211735&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: Cancer develops through a process of clonal evolution in which an initially healthy cell gives rise to progeny gradually differentiating through the accumulation of genetic and epigenetic mutations. These mutations can take various forms, including single-nucleotide variants (SNVs), copy number alterations (CNAs) or structural variations (SVs), with each variant type providing complementary insights into tumor evolution as well as offering distinct challenges to phylogenetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35758777"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing tumor clonal lineage trees incorporating single-nucleotide variants, copy number alterations and structural variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35758777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: Cancer develops through a process of clonal evolution in which an initially healthy cell gives rise to progeny gradually differentiating through the accumulation of genetic and epigenetic mutations. These mutations can take various forms, including single-nucleotide variants (SNVs), copy number alterations (CNAs) or structural variations (SVs), with each variant type providing complementary insights into tumor evolution as well as offering distinct challenges to phylogenetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35758777"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing tumor clonal lineage trees incorporating single-nucleotide variants, copy number alterations and structural variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35758777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: Cancer develops through a process of clonal evolution in which an initially healthy cell gives rise to progeny gradually differentiating through the accumulation of genetic and epigenetic mutations. These mutations can take various forms, including single-nucleotide variants (SNVs), copy number alterations (CNAs) or structural variations (SVs), with each variant type providing complementary insights into tumor evolution as well as offering distinct challenges to phylogenetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35758777"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing tumor clonal lineage trees incorporating single-nucleotide variants, copy number alterations and structural variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35758777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: Cancer develops through a process of clonal evolution in which an initially healthy cell gives rise to progeny gradually differentiating through the accumulation of genetic and epigenetic mutations. These mutations can take various forms, including single-nucleotide variants (SNVs), copy number alterations (CNAs) or structural variations (SVs), with each variant type providing complementary insights into tumor evolution as well as offering distinct challenges to phylogenetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35758777"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing tumor clonal lineage trees incorporating single-nucleotide variants, copy number alterations and structural variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35758777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: Cancer develops through a process of clonal evolution in which an initially healthy cell gives rise to progeny gradually differentiating through the accumulation of genetic and epigenetic mutations. These mutations can take various forms, including single-nucleotide variants (SNVs), copy number alterations (CNAs) or structural variations (SVs), with each variant type providing complementary insights into tumor evolution as well as offering distinct challenges to phylogenetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35758777"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing tumor clonal lineage trees incorporating single-nucleotide variants, copy number alterations and structural variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35758777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: Cancer develops through a process of clonal evolution in which an initially healthy cell gives rise to progeny gradually differentiating through the accumulation of genetic and epigenetic mutations. These mutations can take various forms, including single-nucleotide variants (SNVs), copy number alterations (CNAs) or structural variations (SVs), with each variant type providing complementary insights into tumor evolution as well as offering distinct challenges to phylogenetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35758777"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing tumor clonal lineage trees incorporating single-nucleotide variants, copy number alterations and structural variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35758777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: Cancer develops through a process of clonal evolution in which an initially healthy cell gives rise to progeny gradually differentiating through the accumulation of genetic and epigenetic mutations. These mutations can take various forms, including single-nucleotide variants (SNVs), copy number alterations (CNAs) or structural variations (SVs), with each variant type providing complementary insights into tumor evolution as well as offering distinct challenges to phylogenetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35758777"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing tumor clonal lineage trees incorporating single-nucleotide variants, copy number alterations and structural variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35758777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220708210406&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: Cancer develops through a process of clonal evolution in which an initially healthy cell gives rise to progeny gradually differentiating through the accumulation of genetic and epigenetic mutations. These mutations can take various forms, including single-nucleotide variants (SNVs), copy number alterations (CNAs) or structural variations (SVs), with each variant type providing complementary insights into tumor evolution as well as offering distinct challenges to phylogenetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35758777"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing tumor clonal lineage trees incorporating single-nucleotide variants, copy number alterations and structural variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35758777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220707211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: Cancer develops through a process of clonal evolution in which an initially healthy cell gives rise to progeny gradually differentiating through the accumulation of genetic and epigenetic mutations. These mutations can take various forms, including single-nucleotide variants (SNVs), copy number alterations (CNAs) or structural variations (SVs), with each variant type providing complementary insights into tumor evolution as well as offering distinct challenges to phylogenetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35758777"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing tumor clonal lineage trees incorporating single-nucleotide variants, copy number alterations and structural variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35758777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220706212003&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: Cancer develops through a process of clonal evolution in which an initially healthy cell gives rise to progeny gradually differentiating through the accumulation of genetic and epigenetic mutations. These mutations can take various forms, including single-nucleotide variants (SNVs), copy number alterations (CNAs) or structural variations (SVs), with each variant type providing complementary insights into tumor evolution as well as offering distinct challenges to phylogenetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35758777"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing tumor clonal lineage trees incorporating single-nucleotide variants, copy number alterations and structural variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35758777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220705212255&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: Cancer develops through a process of clonal evolution in which an initially healthy cell gives rise to progeny gradually differentiating through the accumulation of genetic and epigenetic mutations. These mutations can take various forms, including single-nucleotide variants (SNVs), copy number alterations (CNAs) or structural variations (SVs), with each variant type providing complementary insights into tumor evolution as well as offering distinct challenges to phylogenetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35758777"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing tumor clonal lineage trees incorporating single-nucleotide variants, copy number alterations and structural variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35758777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220704211122&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: Cancer develops through a process of clonal evolution in which an initially healthy cell gives rise to progeny gradually differentiating through the accumulation of genetic and epigenetic mutations. These mutations can take various forms, including single-nucleotide variants (SNVs), copy number alterations (CNAs) or structural variations (SVs), with each variant type providing complementary insights into tumor evolution as well as offering distinct challenges to phylogenetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35758777"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing tumor clonal lineage trees incorporating single-nucleotide variants, copy number alterations and structural variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35758777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220703212221&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: Cancer develops through a process of clonal evolution in which an initially healthy cell gives rise to progeny gradually differentiating through the accumulation of genetic and epigenetic mutations. These mutations can take various forms, including single-nucleotide variants (SNVs), copy number alterations (CNAs) or structural variations (SVs), with each variant type providing complementary insights into tumor evolution as well as offering distinct challenges to phylogenetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35758777"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing tumor clonal lineage trees incorporating single-nucleotide variants, copy number alterations and structural variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35758777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220702211519&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: Cancer develops through a process of clonal evolution in which an initially healthy cell gives rise to progeny gradually differentiating through the accumulation of genetic and epigenetic mutations. These mutations can take various forms, including single-nucleotide variants (SNVs), copy number alterations (CNAs) or structural variations (SVs), with each variant type providing complementary insights into tumor evolution as well as offering distinct challenges to phylogenetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35758777"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing tumor clonal lineage trees incorporating single-nucleotide variants, copy number alterations and structural variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35758777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220701211233&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: Cancer develops through a process of clonal evolution in which an initially healthy cell gives rise to progeny gradually differentiating through the accumulation of genetic and epigenetic mutations. These mutations can take various forms, including single-nucleotide variants (SNVs), copy number alterations (CNAs) or structural variations (SVs), with each variant type providing complementary insights into tumor evolution as well as offering distinct challenges to phylogenetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35758777"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing tumor clonal lineage trees incorporating single-nucleotide variants, copy number alterations and structural variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35758777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220630212102&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: Cancer develops through a process of clonal evolution in which an initially healthy cell gives rise to progeny gradually differentiating through the accumulation of genetic and epigenetic mutations. These mutations can take various forms, including single-nucleotide variants (SNVs), copy number alterations (CNAs) or structural variations (SVs), with each variant type providing complementary insights into tumor evolution as well as offering distinct challenges to phylogenetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35758777"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing tumor clonal lineage trees incorporating single-nucleotide variants, copy number alterations and structural variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35758777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220629210920&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: Cancer develops through a process of clonal evolution in which an initially healthy cell gives rise to progeny gradually differentiating through the accumulation of genetic and epigenetic mutations. These mutations can take various forms, including single-nucleotide variants (SNVs), copy number alterations (CNAs) or structural variations (SVs), with each variant type providing complementary insights into tumor evolution as well as offering distinct challenges to phylogenetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35758777"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing tumor clonal lineage trees incorporating single-nucleotide variants, copy number alterations and structural variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35758777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220629090955&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: Cancer develops through a process of clonal evolution in which an initially healthy cell gives rise to progeny gradually differentiating through the accumulation of genetic and epigenetic mutations. These mutations can take various forms, including single-nucleotide variants (SNVs), copy number alterations (CNAs) or structural variations (SVs), with each variant type providing complementary insights into tumor evolution as well as offering distinct challenges to phylogenetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35758777"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing tumor clonal lineage trees incorporating single-nucleotide variants, copy number alterations and structural variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35758777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220628211035&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: Cancer develops through a process of clonal evolution in which an initially healthy cell gives rise to progeny gradually differentiating through the accumulation of genetic and epigenetic mutations. These mutations can take various forms, including single-nucleotide variants (SNVs), copy number alterations (CNAs) or structural variations (SVs), with each variant type providing complementary insights into tumor evolution as well as offering distinct challenges to phylogenetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35758777"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing tumor clonal lineage trees incorporating single-nucleotide variants, copy number alterations and structural variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35758777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220627211523&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: Cancer develops through a process of clonal evolution in which an initially healthy cell gives rise to progeny gradually differentiating through the accumulation of genetic and epigenetic mutations. These mutations can take various forms, including single-nucleotide variants (SNVs), copy number alterations (CNAs) or structural variations (SVs), with each variant type providing complementary insights into tumor evolution as well as offering distinct challenges to phylogenetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-06-06 datetime="2022-06-06T18:28:04.000Z">2022-06-06</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-06-06 datetime="2022-06-06T18:28:04.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="72a3a11d39c217b1a80cbb551444e6f6"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Nodal interferon exposure drives tumour-specific tolerance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1533878403463405569">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Nodal interferon exposure drives tumour-specific tolerance https://t.co/FT4NeBcFTg pic.twitter.com/NRHkVDUsOr
— NatureReviewsCancer (@NatureRevCancer) June 6, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-30 datetime="2022-05-30T04:37:42.000Z">2022-05-30</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-30 datetime="2022-05-30T04:37:42.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="bb6bc242bcbc356c4a38db850e74d812"
                        open
                      >
                        <summary class="article-expander__title">@VIBConferences: #Cancer #immunotherapy has become a crucial pillar of anticancer therapy. 19 leading researchers at #ImmunoOnco22 will give new insights into the immune response to tumors and tackle unresolved questions.⏰Registration Deadline: Thursday 2 June. Read more</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/VIBConferences/status/1531132720952266752">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>#Cancer #immunotherapy has become a crucial pillar of anticancer therapy. 19 leading researchers at #ImmunoOnco22 will give new insights into the immune response to tumors and tackle unresolved questions.
⏰Registration Deadline: Thursday 2 June. Read more https://t.co/cc0HC5cj6W pic.twitter.com/u8LFuzBbsV
— VIB Conferences (@VIBConferences) May 30, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-27 datetime="2022-05-27T14:45:03.000Z">2022-05-27</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-27 datetime="2022-05-27T14:45:03.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="ed6c996dc1a718afb523de6229a53131"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Nonsense-mediated RNA decay: an emerging modulator of malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1530198403686072320">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Nonsense-mediated RNA decay: an emerging modulator of malignancy https://t.co/wjuHplkpof pic.twitter.com/1SoKYhRsPO
— NatureReviewsCancer (@NatureRevCancer) May 27, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="d15a3a4b24853812d7fa21fd56de5815"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: Our June issue, out NOW! With articles on #therapyresistance, #senescence &amp; targeting CDK4 &amp; CDK6 in #cancer! Plus, a Comment on cancer mRNA vaccines as well as the launch of our new Tools of the Trade articles #NRCToT. So much content 👇🏻💌</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1530174481313972224">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Our June issue, out NOW! With articles on #therapyresistance, #senescence &amp; targeting CDK4 &amp; CDK6 in #cancer! Plus, a Comment on cancer mRNA vaccines as well as the launch of our new Tools of the Trade articles #NRCToT. So much content 👇🏻
💌 https://t.co/Fo6XRWV3OZ@natrescancer pic.twitter.com/rbxRPqLV6g
— NatureReviewsCancer (@NatureRevCancer) May 27, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-26 datetime="2022-05-26T08:53:50.000Z">2022-05-26</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-26 datetime="2022-05-26T08:53:50.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="fe13755a2693ea65459e5e21df097d2a"
                        open
                      >
                        <summary class="article-expander__title">@UlrikeHarjes: This Review by @MarcusDGon @cantleylab is part of the @NatureRevCancer series on #Diet &amp; #SystemicMetabolism in cancer:  do love this topic in general plus I am handling research at @NatureMedicine now.. get in touch if you have something of interest☺️</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/UlrikeHarjes/status/1529747629936193537">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>This Review by @MarcusDGon @cantleylab is part of the @NatureRevCancer series on #Diet &amp; #SystemicMetabolism in cancer: https://t.co/p3Y2cdNhco
I do love this topic in general plus I am handling research at @NatureMedicine now.. get in touch if you have something of interest☺️ https://t.co/qWH2dTv8j9
— Ulrike Harjes (@UlrikeHarjes) May 26, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-25 datetime="2022-05-25T17:35:04.000Z">2022-05-25</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-25 datetime="2022-05-25T17:35:04.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="9ba5279424b31b0122a4ae20c18e6d03"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Developing dietary interventions as therapy for cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1529516411059437568">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Developing dietary interventions as therapy for cancer https://t.co/ijuqctk1wJ pic.twitter.com/KHPpMKnCFa
— NatureReviewsCancer (@NatureRevCancer) May 25, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-18 datetime="2022-05-18T14:11:32.000Z">2022-05-18</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-18 datetime="2022-05-18T14:11:32.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="611d40f2e6ea2796b028bebda41a86dd"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Digging for treasures in the tumour interactome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1526928478490095616">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Digging for treasures in the tumour interactome https://t.co/NhHYAwUgLE
— NatureReviewsCancer (@NatureRevCancer) May 18, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-17 datetime="2022-05-17T13:48:21.000Z">2022-05-17</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-17 datetime="2022-05-17T13:48:21.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="9590a971b00448cd312078183965b21d"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: As part of our overall commitment to diversity, equity and inclusion at Springer Nature, we’d like to better understand what DEI means within the global research community. Take the survey here</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1526560254091313156">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>As part of our overall commitment to diversity, equity and inclusion at Springer Nature, we’d like to better understand what DEI means within the global research community. Take the survey here https://t.co/03yLiUxFTq
— NatureReviewsCancer (@NatureRevCancer) May 17, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-12 datetime="2022-05-12T15:25:46.000Z">2022-05-12</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-12 datetime="2022-05-12T15:25:46.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="aa38ff4245b93e964773f191844ddc7a"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: Newsflash! We have a new article type in the journal - Tools of the Trade #NRCToT 🧰 🛠️ where we hope to enable more #ECRs to interact with the editorial team &amp; have a bigger voice in our content. For more info, check out the Editorial 👇🏻</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1524772831904509953">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Newsflash! We have a new article type in the journal - Tools of the Trade #NRCToT 🧰 🛠️ where we hope to enable more #ECRs to interact with the editorial team &amp; have a bigger voice in our content. For more info, check out the Editorial 👇🏻 https://t.co/VbyoUJmTRJ
— NatureReviewsCancer (@NatureRevCancer) May 12, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-10 datetime="2022-05-10T16:46:03.000Z">2022-05-10</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-10 datetime="2022-05-10T16:46:03.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="65aca76b9f794aaa668d84ad32dca4a0"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Tools for the next generation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1524068259712036864">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Tools for the next generation https://t.co/lEzTPFYDf5
— NatureReviewsCancer (@NatureRevCancer) May 10, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="afa9b6784b4ccc7c05ed4efd26ddfb2d"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Hunting down rare drug-tolerant cycling cells with Watermelon</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1524022709386190856">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Hunting down rare drug-tolerant cycling cells with Watermelon https://t.co/XXT0c9SAYX
— NatureReviewsCancer (@NatureRevCancer) May 10, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-04-29 datetime="2022-04-29T14:40:05.000Z">2022-04-29</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-04-29 datetime="2022-04-29T14:40:05.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="43e2d5ccebc9030f737d656f47ac9d98"
                        open
                      >
                        <summary class="article-expander__title">@VarunVenkatara2: Very happy that our latest Perspective is now online @NatureRevCancer ! We discuss implications of malignant, multicellular networks and potential therapeutic strategies how to disconnect them. Thanks to everyone involved ! #CancerNeuroscience #TumorMicrotubes #neurotwitter</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/VarunVenkatara2/status/1520050291986604033">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Very happy that our latest Perspective is now online @NatureRevCancer ! We discuss implications of malignant, multicellular networks and potential therapeutic strategies how to disconnect them. Thanks to everyone involved ! #CancerNeuroscience #TumorMicrotubes #neurotwitter https://t.co/S55cKTGNFd
— Varun Venkataramani (@VarunVenkatara2) April 29, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="9f6b8644378881f4c2e6b6833c93fc2f"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! An inhospitable site</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1520020349068316673">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! An inhospitable site https://t.co/0dvqIKCWxh pic.twitter.com/PtY8J9Fz7r
— NatureReviewsCancer (@NatureRevCancer) April 29, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="f6d91efcd60dae769e2722585bdfa8df"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Disconnecting multicellular networks in brain tumours</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1520020340201598978">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Disconnecting multicellular networks in brain tumours https://t.co/LJrzy8bRma pic.twitter.com/CO7BnooP0r
— NatureReviewsCancer (@NatureRevCancer) April 29, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-04-28 datetime="2022-04-28T14:04:34.000Z">2022-04-28</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-04-28 datetime="2022-04-28T14:04:34.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="76e3b436bc8e43ad21366a80c120e395"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: Our May issue, out NOW! With Reviews on #CRISPR in cancer biology &amp; #therapy, ROS in tumour development &amp; progression, and cancer #proteogenomics, and Comments on esophageal cancer research in Eastern Africa and on Critical Race Theory.💌</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1519678964364623872">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Our May issue, out NOW! With Reviews on #CRISPR in cancer biology &amp; #therapy, ROS in tumour development &amp; progression, and cancer #proteogenomics, and Comments on esophageal cancer research in Eastern Africa and on Critical Race Theory.
💌 https://t.co/vQwHFJhpwv@natrescancer pic.twitter.com/6kYgjRYhdT
— NatureReviewsCancer (@NatureRevCancer) April 28, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="fdf224ea293cdf039086d16e5699a59a"
                        open
                      >
                        <summary class="article-expander__title">@tmurraystewart: Our latest publication on #polyamines is online!@HopkinsMedicine</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/tmurraystewart/status/1519413460978704384">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Our latest publication on #polyamines is online!@HopkinsMedicine https://t.co/ehRGgN1w9R
— Tracy Murray Stewart (@tmurraystewart) April 27, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-04-27 datetime="2022-04-27T18:22:03.000Z">2022-04-27</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-04-27 datetime="2022-04-27T18:22:03.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="4f43226ec4dc8e531ff6e64a9c5ab8e6"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Polyamines in cancer: integrating organismal metabolism and antitumour immunity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1519381375874265088">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Polyamines in cancer: integrating organismal metabolism and antitumour immunity https://t.co/YCJX1hRQLO pic.twitter.com/zRUCRNXyxH
— NatureReviewsCancer (@NatureRevCancer) April 27, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="a44ed79b9f76aad377b9272fb392f553"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW #ResearchHighlight! Lv, Liu, Mo &amp; co demonstrate that in pancreatic #PDAC cells, gasdermin E transports YBX1 to the nucleus, which promotes mucin expression, thereby providing tumour cells with a barrier against digestive enzymes.📯</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1519317944995569664">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW #ResearchHighlight! Lv, Liu, Mo &amp; co demonstrate that in pancreatic #PDAC cells, gasdermin E transports YBX1 to the nucleus, which promotes mucin expression, thereby providing tumour cells with a barrier against digestive enzymes.
📯 https://t.co/JTUhX0DHQ1@natrescancer pic.twitter.com/2PdCmsDu2p
— NatureReviewsCancer (@NatureRevCancer) April 27, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-04-26 datetime="2022-04-26T14:58:00.000Z">2022-04-26</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-04-26 datetime="2022-04-26T14:58:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="06ff7641d8f5fcc0dff91f835ac3bbc1"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: In the April issue of @NatureRevCancer | Comment, Reimagining cancer research with art, by @CancerSciArtist, Stefani Shoreibah &amp; @tdanino  @natrescancer @Columbia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1518967636561956865">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>In the April issue of @NatureRevCancer | Comment, Reimagining cancer research with art, by @CancerSciArtist, Stefani Shoreibah &amp; @tdanino
 https://t.co/GeTVCMJQsp @natrescancer
 @Columbia pic.twitter.com/ZXW6A0Fe0c
— NatureReviewsCancer (@NatureRevCancer) April 26, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-04-25 datetime="2022-04-25T12:12:32.000Z">2022-04-25</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-04-25 datetime="2022-04-25T12:12:32.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="96b2ddff4eb269cf343213c6a1b1ce95"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Tracking dormant disseminated tumour cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1518563607440621569">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Tracking dormant disseminated tumour cells https://t.co/kSXyWAvCxF
— NatureReviewsCancer (@NatureRevCancer) April 25, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>

    <footer>
      <a class="footer-link" href="https://github.com/MacwanEric/cancer.rss.feed/actions/runs/2923389669">
        <time id="build-timestamp" datetime="2022-08-25T01:28:10.781Z">2022-08-25T01:28:10.781Z</time>
      </a>
      <a class="footer-link" href="https://github.com/osmoscraft/osmosfeed">osmosfeed 1.15.1</a>
    </footer>
    <script src="index.js?v1.14.4"></script>
    <!-- %before-body-end.html% -->
  </body>

</html>